--- aungier-1674-e3-p1.psd.vert	2025-09-10 13:51:11
+++ aungier-1674-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -13 +13 @@
-Lond.))
+Lond.#NA))
@@ -19 +19 @@
-Mar.)
+Mar.#NA)
@@ -21 +21 @@
-y=e=)
+y=e=#the_adj)
@@ -23 +23 @@
-15=th=)
+15=th=#NA)
@@ -27 +27 @@
-1672-3)
+1672-3#NA)
@@ -36 +36 @@
-May)
+May#may_v1)
@@ -39 +39 @@
-it))
+it#it_pron))
@@ -41 +41 @@
-please)
+please#please_v)
@@ -44 +44 @@
-y=r=)
+y=r=#your_pron)
@@ -46 +46 @@
-Ex=cye=)))
+Ex=cye=#excellency_n)))
@@ -51 +51 @@
-I))
+I#i_pron))
@@ -53 +53 @@
-am)
+am#be_v)
@@ -56 +56 @@
-but)
+but#but_prep)
@@ -58 +58 @@
-just)
+just#just_adv)
@@ -60 +60 @@
-now))
+now#now_adv))
@@ -62 +62 @@
-return'd)
+return'd#return_v1)
@@ -65 +65 @@
-out)
+out#out_adv)
@@ -67 +67 @@
-of)
+of#of_prep)
@@ -70 +70 @@
-y=e=)
+y=e=#the_adj)
@@ -72 +72 @@
-country)
+country#country_n)
@@ -78 +78 @@
-where))
+where#where_adv))
@@ -86 +86 @@
-I))
+I#i_pron))
@@ -88 +88 @@
-have)
+have#have_v)
@@ -90 +90 @@
-spent)
+spent#spend_v1)
@@ -93 +93 @@
-fower)
+fower#four_adj)
@@ -95 +95 @@
-dayes))))))
+dayes#day_n))))))
@@ -103 +103 @@
-&)
+&#and_conj1)
@@ -108 +108 @@
-therefore))
+therefore#therefore_adv))
@@ -110 +110 @@
-am)
+am#be_v)
@@ -112 +112 @@
-not)
+not#not_adv)
@@ -116 +116 @@
-so)
+so#so_adv)
@@ -118 +118 @@
-well))
+well#well_adv))
@@ -120 +120 @@
-instructed)
+instructed#instructed_adj)
@@ -123 +123 @@
-in)
+in#in_prep)
@@ -126 +126 @@
-ye)
+ye#the_adj)
@@ -128 +128 @@
-affaires)
+affaires#affair_n)
@@ -132 +132 @@
-w=ch=))
+w=ch=#which_adj))
@@ -139 +139 @@
-have)
+have#have_v)
@@ -141 +141 @@
-pass'd)
+pass'd#pass_v)
@@ -144 +144 @@
-in)
+in#in_prep)
@@ -147 +147 @@
-Parliam=t=)))
+Parliam=t=#NA)))
@@ -150 +150 @@
-since)
+since#since_adv)
@@ -153 +153 @@
-my)
+my#my_adj)
@@ -155 +155 @@
-last)))))))
+last#last_adv)))))))
@@ -158 +158 @@
-as)
+as#as_adv)
@@ -161 +161 @@
-to)
+to#to_prep)
@@ -163 +163 @@
-be)
+be#be_v)
@@ -166 +166 @@
-able)
+able#able_adj)
@@ -169 +169 @@
-to)
+to#to_prep)
@@ -171 +171 @@
-give)
+give#give_v)
@@ -174 +174 @@
-y=r=)
+y=r=#your_pron)
@@ -176 +176 @@
-Ex=cye=))
+Ex=cye=#excellency_n))
@@ -179 +179 @@
-a)
+a#a_adj2)
@@ -181 +181 @@
-particular)
+particular#particular_adj)
@@ -183 +183 @@
-acc=t=)
+acc=t=#account_n)
@@ -186 +186 @@
-of)
+of#of_prep)
@@ -189 +189 @@
-them)))))))))
+them#them_pron)))))))))
@@ -197 +197 @@
-But)
+But#but_prep)
@@ -200 +200 @@
-calling)
+calling#call_v)
@@ -203 +203 @@
-upon)
+upon#upon_prep)
@@ -206 +206 @@
-S=r=)
+S=r=#NA)
@@ -208 +208 @@
-Arthur)
+Arthur#NA)
@@ -210 +210 @@
-Forbes))))
+Forbes#NA))))
@@ -215 +215 @@
-I))
+I#i_pron))
@@ -217 +217 @@
-have)
+have#have_v)
@@ -219 +219 @@
-glean'd)
+glean'd#glean_v)
@@ -222 +222 @@
-from)
+from#from_prep)
@@ -225 +225 @@
-him)))
+him#him_pron)))
@@ -230 +230 @@
-what))
+what#what_pron))
@@ -236 +236 @@
-I))
+I#i_pron))
@@ -238 +238 @@
-am)
+am#be_v)
@@ -241 +241 @@
-now))
+now#now_adv))
@@ -246 +246 @@
-to)
+to#to_prep)
@@ -248 +248 @@
-tell)
+tell#tell_v)
@@ -251 +251 @@
-y=r=)
+y=r=#your_pron)
@@ -253 +253 @@
-Ex=c=ye=)))))
+Ex=c=ye=#excellency_n)))))
@@ -258 +258 @@
-viz.)
+viz.#NA)
@@ -261 +261 @@
-1=st=))
+1=st=#first_adj))
@@ -263 +263 @@
-That)
+That#that_conj)
@@ -268 +268 @@
-y=e=)
+y=e=#the_adj)
@@ -270 +270 @@
-Bill)
+Bill#bill_n3)
@@ -273 +273 @@
-for)
+for#for_prep)
@@ -277 +277 @@
-his)
+his#his_adj)
@@ -279 +279 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -281 +281 @@
-supply))))
+supply#supply_n))))
@@ -283 +283 @@
-was)
+was#be_v)
@@ -286 +286 @@
-this)
+this#this_pron)
@@ -288 +288 @@
-day))
+day#day_n))
@@ -290 +290 @@
-reported)
+reported#report_v)
@@ -293 +293 @@
-from)
+from#from_prep)
@@ -296 +296 @@
-y=e=)
+y=e=#the_adj)
@@ -298 +298 @@
-Committee))))
+Committee#committee_n.II))))
@@ -303 +303 @@
-&)
+&#and_conj1)
@@ -306 +306 @@
-ordered)
+ordered#order_v)
@@ -312 +312 @@
-to)
+to#to_prep)
@@ -314 +314 @@
-be)
+be#be_v)
@@ -316 +316 @@
-engrost))
+engrost#engross_v))
@@ -321 +321 @@
-but)
+but#but_prep)
@@ -326 +326 @@
-not)
+not#not_adv)
@@ -328 +328 @@
-to)
+to#to_prep)
@@ -330 +330 @@
-have)
+have#have_v)
@@ -333 +333 @@
-its)
+its#its_adj)
@@ -335 +335 @@
-last)
+last#last_adv)
@@ -337 +337 @@
-reading))
+reading#reading_n2))
@@ -340 +340 @@
-till)
+till#till_prep)
@@ -343 +343 @@
-Fryday)
+Fryday#NA)
@@ -346 +346 @@
-next)))))))))
+next#next_adj)))))))))
@@ -352 +352 @@
-by)
+by#by_prep)
@@ -355 +355 @@
-w=ch=)
+w=ch=#which_adj)
@@ -357 +357 @@
-time)))
+time#time_n)))
@@ -365 +365 @@
-y=e=)
+y=e=#the_adj)
@@ -367 +367 @@
-House)
+House#NA)
@@ -370 +370 @@
-of)
+of#of_prep)
@@ -373 +373 @@
-Com~ons))))
+Com~ons#NA))))
@@ -375 +375 @@
-will)
+will#will_v1)
@@ -377 +377 @@
-see)
+see#see_v)
@@ -381 +381 @@
-what)
+what#what_pron)
@@ -383 +383 @@
-fortune))
+fortune#fortune_n))
@@ -391 +391 @@
-y=e=)
+y=e=#the_adj)
@@ -393 +393 @@
-Bill)
+Bill#bill_n3)
@@ -396 +396 @@
-ag=t=)
+ag=t=#against_prep)
@@ -399 +399 @@
-y=e=)
+y=e=#the_adj)
@@ -401 +401 @@
-groweth)
+groweth#growth_n1)
@@ -404 +404 @@
-of)
+of#of_prep)
@@ -407 +407 @@
-Popery))))))
+Popery#popery_n))))))
@@ -409 +409 @@
-is)
+is#be_v)
@@ -412 +412 @@
-like)
+like#like_adj)
@@ -417 +417 @@
-to)
+to#to_prep)
@@ -419 +419 @@
-have)
+have#have_v)
@@ -422 +422 @@
-in)
+in#in_prep)
@@ -425 +425 @@
-the)
+the#the_adj)
@@ -427 +427 @@
-Howse)
+Howse#NA)
@@ -430 +430 @@
-of)
+of#of_prep)
@@ -433 +433 @@
-Lords)))))))
+Lords#NA)))))))
@@ -439 +439 @@
-from)
+from#from_prep)
@@ -442 +442 @@
-whence)))
+whence#whence_adv)))
@@ -450 +450 @@
-they))
+they#they_pron))
@@ -452 +452 @@
-will)
+will#will_v1)
@@ -454 +454 @@
-take)
+take#take_v)
@@ -457 +457 @@
-their)
+their#their_adj)
@@ -459 +459 @@
-future)
+future#future_adj)
@@ -461 +461 @@
-measures)))))))))))
+measures#measure_n)))))))))))
@@ -470 +470 @@
-2=ly=))
+2=ly=#secondly_adv))
@@ -473 +473 @@
-That)
+That#that_conj)
@@ -477 +477 @@
-there))
+there#there_adv))
@@ -479 +479 @@
-have)
+have#have_v)
@@ -481 +481 @@
-beene)
+beene#be_v)
@@ -484 +484 @@
-some)
+some#some_pron)
@@ -486 +486 @@
-hints)
+hints#hint_n)
@@ -489 +489 @@
-given)
+given#give_v)
@@ -492 +492 @@
-in)
+in#in_prep)
@@ -495 +495 @@
-y=e=)
+y=e=#the_adj)
@@ -497 +497 @@
-howse)))
+howse#NA)))
@@ -500 +500 @@
-of)
+of#of_prep)
@@ -503 +503 @@
-y=e=)
+y=e=#the_adj)
@@ -505 +505 @@
-late)
+late#late_adj1)
@@ -507 +507 @@
-encouragem=t=)
+encouragem=t=#encouragement_n)
@@ -511 +511 @@
-w=ch=))
+w=ch=#which_adj))
@@ -518 +518 @@
-has)
+has#have_v)
@@ -520 +520 @@
-beene)
+beene#be_v)
@@ -522 +522 @@
-given)
+given#give_v)
@@ -525 +525 @@
-to)
+to#to_prep)
@@ -528 +528 @@
-y=e=)
+y=e=#the_adj)
@@ -530 +530 @@
-Papists)
+Papists#NA)
@@ -533 +533 @@
-in)
+in#in_prep)
@@ -536 +536 @@
-Ire=ld=)))))
+Ire=ld=#NA)))))
@@ -539 +539 @@
-by)
+by#by_prep)
@@ -542 +542 @@
-admitting)
+admitting#admit_v)
@@ -545 +545 @@
-them))
+them#them_pron))
@@ -548 +548 @@
-into)
+into#into_prep)
@@ -551 +551 @@
-y=e=)
+y=e=#the_adj)
@@ -553 +553 @@
-Com=r=)
+Com=r=#NA)
@@ -556 +556 @@
-of)
+of#of_prep)
@@ -559 +559 @@
-Peace)
+Peace#NA)
@@ -561 +561 @@
-&)
+&#and_conj1)
@@ -563 +563 @@
-Corporations)))))))))))))))
+Corporations#NA)))))))))))))))
@@ -571 +571 @@
-And)
+And#and_conj1)
@@ -574 +574 @@
-that)
+that#that_conj)
@@ -580 +580 @@
-my)
+my#my_adj)
@@ -582 +582 @@
-Lord)
+Lord#lord_n)
@@ -585 +585 @@
-Arundell)
+Arundell#NA)
@@ -588 +588 @@
-of)
+of#of_prep)
@@ -591 +591 @@
-Warder)))))
+Warder#NA)))))
@@ -597 +597 @@
-Father)
+Father#NA)
@@ -599 +599 @@
-Patrick)))
+Patrick#NA)))
@@ -604 +604 @@
-&)
+&#and_conj1)
@@ -607 +607 @@
-Co=ll=)
+Co=ll=#NA)
@@ -609 +609 @@
-Rich=d=)
+Rich=d=#NA)
@@ -611 +611 @@
-Talbot))))
+Talbot#NA))))
@@ -613 +613 @@
-were)
+were#be_v)
@@ -616 +616 @@
-this)
+this#this_pron)
@@ -618 +618 @@
-day))
+day#day_n))
@@ -620 +620 @@
-named)
+named#name_v)
@@ -623 +623 @@
-in)
+in#in_prep)
@@ -626 +626 @@
-the)
+the#the_adj)
@@ -628 +628 @@
-Howse)))
+Howse#NA)))
@@ -631 +631 @@
-as)
+as#as_adv)
@@ -634 +634 @@
-y=e=)
+y=e=#the_adj)
@@ -636 +636 @@
-Cheife)
+Cheife#chief_adj)
@@ -638 +638 @@
-promoters)
+promoters#promoter_n)
@@ -641 +641 @@
-of)
+of#of_prep)
@@ -644 +644 @@
-it))))))
+it#it_pron))))))
@@ -649 +649 @@
-&)
+&#and_conj1)
@@ -653 +653 @@
-consequently))
+consequently#consequently_adv))
@@ -657 +657 @@
-very)
+very#very_adj)
@@ -659 +659 @@
-unfit))
+unfit#unfit_adj))
@@ -661 +661 @@
-persons)
+persons#person_n)
@@ -664 +664 @@
-to)
+to#to_prep)
@@ -666 +666 @@
-be)
+be#be_v)
@@ -669 +669 @@
-so)
+so#so_adv)
@@ -671 +671 @@
-neare)
+neare#near_adj)
@@ -676 +676 @@
-His)
+His#his_adj)
@@ -678 +678 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -681 +681 @@
-&)
+&#and_conj1)
@@ -684 +684 @@
-His)
+His#his_adj)
@@ -686 +686 @@
-Royall)
+Royall#royal_adj)
@@ -688 +688 @@
-Highnesses))))
+Highnesses#highness_n))))
@@ -690 +690 @@
-persons)))))))))
+persons#person_n)))))))))
@@ -699 +699 @@
-But)
+But#but_prep)
@@ -702 +702 @@
-this))
+this#this_pron))
@@ -704 +704 @@
-was)
+was#be_v)
@@ -706 +706 @@
-only)
+only#only_adj)
@@ -708 +708 @@
-sayd)
+sayd#say_v1)
@@ -711 +711 @@
-by)
+by#by_prep)
@@ -714 +714 @@
-one)
+one#one_adj)
@@ -716 +716 @@
-member)
+member#member_n)
@@ -718 +718 @@
-only))))
+only#only_adj))))
@@ -723 +723 @@
-&)
+&#and_conj1)
@@ -726 +726 @@
-not)
+not#not_adv)
@@ -728 +728 @@
-seconded)
+seconded#second_v1)
@@ -731 +731 @@
-by)
+by#by_prep)
@@ -734 +734 @@
-any)
+any#any_adj)
@@ -737 +737 @@
-of)
+of#of_prep)
@@ -740 +740 @@
-y=e=)
+y=e=#the_adj)
@@ -742 +742 @@
-Howse)))))))
+Howse#NA)))))))
@@ -751 +751 @@
-These)
+These#these_pron)
@@ -753 +753 @@
-particulars))
+particulars#particular_adj))
@@ -756 +756 @@
-S=r=)
+S=r=#NA)
@@ -758 +758 @@
-Arthur))
+Arthur#NA))
@@ -760 +760 @@
-desired)
+desired#desire_v)
@@ -763 +763 @@
-me))
+me#me_pron1))
@@ -766 +766 @@
-to)
+to#to_prep)
@@ -768 +768 @@
-acquainte)
+acquainte#acquaint_v)
@@ -771 +771 @@
-y=r=)
+y=r=#your_pron)
@@ -773 +773 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -776 +776 @@
-w=th=)
+w=th=#with_prep)
@@ -784 +784 @@
-he))
+he#he_pron))
@@ -786 +786 @@
-being)
+being#be_v)
@@ -789 +789 @@
-not)
+not#not_adv)
@@ -791 +791 @@
-yett))
+yett#yet_adv))
@@ -795 +795 @@
-so)
+so#so_adv)
@@ -797 +797 @@
-well))
+well#well_adv))
@@ -799 +799 @@
-recovered)
+recovered#recovered_adj)
@@ -802 +802 @@
-as)
+as#as_adv)
@@ -805 +805 @@
-to)
+to#to_prep)
@@ -807 +807 @@
-be)
+be#be_v)
@@ -810 +810 @@
-able)
+able#able_adj)
@@ -813 +813 @@
-to)
+to#to_prep)
@@ -815 +815 @@
-write)
+write#write_v)
@@ -818 +818 @@
-himselfe))))))))
+himselfe#himself_pron))))))))
@@ -826 +826 @@
-and)
+and#and_conj1)
@@ -829 +829 @@
-he))
+he#he_pron))
@@ -832 +832 @@
-further))
+further#further_adj))
@@ -834 +834 @@
-intreated)
+intreated#entreat_v)
@@ -836 +836 @@
-me)
+me#be_v)
@@ -840 +840 @@
-in)
+in#in_prep)
@@ -843 +843 @@
-his)
+his#his_adj)
@@ -845 +845 @@
-name)))
+name#name_n)))
@@ -847 +847 @@
-to)
+to#to_prep)
@@ -849 +849 @@
-tell)
+tell#tell_v)
@@ -852 +852 @@
-y=r=)
+y=r=#your_pron)
@@ -854 +854 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -857 +857 @@
-that)
+that#that_conj)
@@ -861 +861 @@
-since)
+since#since_adv)
@@ -870 +870 @@
-y=e=)
+y=e=#the_adj)
@@ -872 +872 @@
-Papists))
+Papists#NA))
@@ -874 +874 @@
-are)
+are#be_v)
@@ -876 +876 @@
-faln)
+faln#fall_v)
@@ -879 +879 @@
-upon)
+upon#upon_prep)
@@ -884 +884 @@
-by)
+by#by_prep)
@@ -887 +887 @@
-y=e=)
+y=e=#the_adj)
@@ -889 +889 @@
-Parliam=t=))))))
+Parliam=t=#NA))))))
@@ -894 +894 @@
-he))
+he#he_pron))
@@ -896 +896 @@
-fears)
+fears#fear_v)
@@ -904 +904 @@
-They))
+They#they_pron))
@@ -906 +906 @@
-may)
+may#may_v1)
@@ -908 +908 @@
-grow)
+grow#grow_v)
@@ -911 +911 @@
-desperate)))
+desperate#desperate_adj)))
@@ -914 +914 @@
-and)
+and#and_conj1)
@@ -917 +917 @@
-endeavour)
+endeavour#endeavour_v)
@@ -920 +920 @@
-to)
+to#to_prep)
@@ -922 +922 @@
-disturb)
+disturb#disturb_v)
@@ -925 +925 @@
-y=e=)
+y=e=#the_adj)
@@ -927 +927 @@
-govern=t=)
+govern=t=#government_n)
@@ -930 +930 @@
-There)))))))))))
+There#there_adv)))))))))))
@@ -940 +940 @@
-And)
+And#and_conj1)
@@ -943 +943 @@
-therefore))
+therefore#therefore_adv))
@@ -946 +946 @@
-he))
+he#he_pron))
@@ -949 +949 @@
-humbly))
+humbly#humbly_adv))
@@ -951 +951 @@
-submitts)
+submitts#submit_v)
@@ -954 +954 @@
-it))
+it#it_pron))
@@ -957 +957 @@
-to)
+to#to_prep)
@@ -961 +961 @@
-y=r=)
+y=r=#your_pron)
@@ -963 +963 @@
-Ex=cys=))
+Ex=cys=#excellency_n))
@@ -965 +965 @@
-consideration)))
+consideration#consideration_n)))
@@ -970 +970 @@
-whether)
+whether#whether_pron)
@@ -976 +976 @@
-in)
+in#in_prep)
@@ -979 +979 @@
-this)
+this#this_pron)
@@ -981 +981 @@
-conjuncture)))
+conjuncture#conjuncture_n)))
@@ -984 +984 @@
-it))
+it#it_pron))
@@ -986 +986 @@
-be)
+be#be_v)
@@ -988 +988 @@
-not)
+not#not_adv)
@@ -991 +991 @@
-necessary))
+necessary#necessary_adj))
@@ -994 +994 @@
-for)
+for#for_prep)
@@ -997 +997 @@
-y=r=)
+y=r=#your_pron)
@@ -999 +999 @@
-Ex=cye=))
+Ex=cye=#excellency_n))
@@ -1001 +1001 @@
-to)
+to#to_prep)
@@ -1005 +1005 @@
-order)
+order#order_v)
@@ -1008 +1008 @@
-y=e=)
+y=e=#the_adj)
@@ -1010 +1010 @@
-Officers)
+Officers#officer_n)
@@ -1013 +1013 @@
-of)
+of#of_prep)
@@ -1016 +1016 @@
-y=e=)
+y=e=#the_adj)
@@ -1018 +1018 @@
-Army))))
+Army#army_n))))
@@ -1022 +1022 @@
-to)
+to#to_prep)
@@ -1025 +1025 @@
-their)
+their#their_adj)
@@ -1027 +1027 @@
-charges)))
+charges#charge_n)))
@@ -1030 +1030 @@
-and)
+and#and_conj1)
@@ -1033 +1033 @@
-to)
+to#to_prep)
@@ -1035 +1035 @@
-mind)
+mind#mind_v)
@@ -1038 +1038 @@
-carefully))
+carefully#carefully_adv))
@@ -1041 +1041 @@
-their)
+their#their_adj)
@@ -1043 +1043 @@
-guards))))))
+guards#guard_n))))))
@@ -1051 +1051 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1057 +1057 @@
-I))
+I#i_pron))
@@ -1061 +1061 @@
-presume)
+presume#presume_v)
@@ -1066 +1066 @@
-to)
+to#to_prep)
@@ -1068 +1068 @@
-write)
+write#write_v)
@@ -1071 +1071 @@
-to)
+to#to_prep)
@@ -1074 +1074 @@
-y=r=)
+y=r=#your_pron)
@@ -1076 +1076 @@
-Ex=cye=)))
+Ex=cye=#excellency_n)))
@@ -1079 +1079 @@
-in)
+in#in_prep)
@@ -1082 +1082 @@
-Cypher))))
+Cypher#cipher_n))))
@@ -1087 +1087 @@
-because)
+because#because_adv)
@@ -1094 +1094 @@
-he))
+he#he_pron))
@@ -1096 +1096 @@
-desired)
+desired#desire_v)
@@ -1098 +1098 @@
-me)
+me#be_v)
@@ -1103 +1103 @@
-so))
+so#so_adv))
@@ -1105 +1105 @@
-to)
+to#to_prep)
@@ -1107 +1107 @@
-doe))
+doe#do_v))
@@ -1112 +1112 @@
-&)
+&#and_conj1)
@@ -1115 +1115 @@
-to)
+to#to_prep)
@@ -1117 +1117 @@
-beseech)
+beseech#beseech_v)
@@ -1120 +1120 @@
-y=r=)
+y=r=#your_pron)
@@ -1122 +1122 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -1125 +1125 @@
-to)
+to#to_prep)
@@ -1127 +1127 @@
-conceale)
+conceale#conceal_v)
@@ -1130 +1130 @@
-y=e=)
+y=e=#the_adj)
@@ -1132 +1132 @@
-advizer)))))))))))))
+advizer#adviser_n)))))))))))))
@@ -1141 +1141 @@
-I))
+I#i_pron))
@@ -1143 +1143 @@
-have)
+have#have_v)
@@ -1145 +1145 @@
-directed)
+directed#direct_v)
@@ -1148 +1148 @@
-my)
+my#my_adj)
@@ -1150 +1150 @@
-Brother)
+Brother#brother_n)
@@ -1153 +1153 @@
-Cuffe)))
+Cuffe#NA)))
@@ -1156 +1156 @@
-to)
+to#to_prep)
@@ -1158 +1158 @@
-attend)
+attend#attend_v)
@@ -1161 +1161 @@
-y=r=)
+y=r=#your_pron)
@@ -1163 +1163 @@
-Ex=cy=)
+Ex=cy=#excellency_n)
@@ -1166 +1166 @@
-w=th=)
+w=th=#with_prep)
@@ -1169 +1169 @@
-his)
+his#his_adj)
@@ -1171 +1171 @@
-Cypher)
+Cypher#cipher_n)
@@ -1177 +1177 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1184 +1184 @@
-will)
+will#will_v1)
@@ -1187 +1187 @@
-w=th=out)
+w=th=out#without_adv)
@@ -1190 +1190 @@
-giveing)
+giveing#give_v)
@@ -1193 +1193 @@
-y=r=)
+y=r=#your_pron)
@@ -1195 +1195 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -1198 +1198 @@
-any)
+any#any_adj)
@@ -1200 +1200 @@
-greate)
+greate#great_adj)
@@ -1202 +1202 @@
-trouble))))
+trouble#trouble_n))))
@@ -1204 +1204 @@
-unlocke)
+unlocke#unlock_v)
@@ -1209 +1209 @@
-what))
+what#what_pron))
@@ -1217 +1217 @@
-I))
+I#i_pron))
@@ -1219 +1219 @@
-have)
+have#have_v)
@@ -1221 +1221 @@
-written)))
+written#write_v)))
@@ -1227 +1227 @@
-for)
+for#for_prep)
@@ -1230 +1230 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -1238 +1238 @@
-notw=th=standing))
+notw=th=standing#notwithstanding_prep))
@@ -1241 +1241 @@
-I))
+I#i_pron))
@@ -1244 +1244 @@
-humbly))
+humbly#humbly_adv))
@@ -1246 +1246 @@
-aske)
+aske#ask_v)
@@ -1250 +1250 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1252 +1252 @@
-Ex=cyes=))
+Ex=cyes=#excellency_n))
@@ -1254 +1254 @@
-pardon)))))))))))
+pardon#pardon_n1)))))))))))
@@ -1268 +1268 @@
-Lond.))
+Lond.#NA))
@@ -1274 +1274 @@
-Apr.)
+Apr.#NA)
@@ -1276 +1276 @@
-12=th=)
+12=th=#NA)
@@ -1280 +1280 @@
-1673)
+1673#NA)
@@ -1289 +1289 @@
-May)
+May#may_v1)
@@ -1292 +1292 @@
-it))
+it#it_pron))
@@ -1294 +1294 @@
-please)
+please#please_v)
@@ -1297 +1297 @@
-yor)
+yor#your_pron)
@@ -1299 +1299 @@
-Ex=cye=)))
+Ex=cye=#excellency_n)))
@@ -1304 +1304 @@
-On)
+On#on_prep)
@@ -1307 +1307 @@
-Wenesday)
+Wenesday#NA)
@@ -1310 +1310 @@
-last))))
+last#last_adv))))
@@ -1313 +1313 @@
-Co=ll=)
+Co=ll=#NA)
@@ -1315 +1315 @@
-Talbot))
+Talbot#NA))
@@ -1317 +1317 @@
-appeared)
+appeared#appear_v)
@@ -1320 +1320 @@
-publiquely))
+publiquely#publicly_adv))
@@ -1323 +1323 @@
-in)
+in#in_prep)
@@ -1327 +1327 @@
-His)
+His#his_adj)
@@ -1329 +1329 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -1331 +1331 @@
-Bedchamber)
+Bedchamber#bedchamber_n)
@@ -1337 +1337 @@
-where))
+where#where_adv))
@@ -1345 +1345 @@
-he))
+he#he_pron))
@@ -1347 +1347 @@
-attended)
+attended#attend_v)
@@ -1350 +1350 @@
-to)
+to#to_prep)
@@ -1352 +1352 @@
-deliver)
+deliver#deliver_v1)
@@ -1355 +1355 @@
-his)
+his#his_adj)
@@ -1357 +1357 @@
-petition))
+petition#petition_n))
@@ -1363 +1363 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1371 +1371 @@
-having)
+having#have_v)
@@ -1373 +1373 @@
-don)
+don#do_v)
@@ -1376 +1376 @@
-as)
+as#as_adv)
@@ -1383 +1383 @@
-his)
+his#his_adj)
@@ -1385 +1385 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -1387 +1387 @@
-was)
+was#be_v)
@@ -1389 +1389 @@
-passing)
+passing#pass_v)
@@ -1392 +1392 @@
-through)
+through#through_prep)
@@ -1394 +1394 @@
-to)
+to#to_prep)
@@ -1397 +1397 @@
-Councill)))))))
+Councill#NA)))))))
@@ -1402 +1402 @@
-he))
+he#he_pron))
@@ -1405 +1405 @@
-there))
+there#there_adv))
@@ -1407 +1407 @@
-waited)
+waited#wait_v1)
@@ -1411 +1411 @@
-His)
+His#his_adj)
@@ -1413 +1413 @@
-Ma=ties=)
+Ma=ties=#majesty_n)
@@ -1416 +1416 @@
-&)
+&#and_conj1)
@@ -1419 +1419 @@
-Councill@)))
+Councill@#NA)))
@@ -1421 +1421 @@
-@'s))
+@'s#NA))
@@ -1423 +1423 @@
-resolution)
+resolution#resolution_n)
@@ -1426 +1426 @@
-upon)
+upon#upon_prep)
@@ -1429 +1429 @@
-it)))))))))))
+it#it_pron)))))))))))
@@ -1438 +1438 @@
-His)
+His#his_adj)
@@ -1440 +1440 @@
-Petition))
+Petition#petition_n))
@@ -1442 +1442 @@
-purported)
+purported#purport_v)
@@ -1445 +1445 @@
-his)
+his#his_adj)
@@ -1447 +1447 @@
-greate)
+greate#great_adj)
@@ -1449 +1449 @@
-surprize)
+surprize#surprise_n)
@@ -1452 +1452 @@
-at)
+at#at_prep)
@@ -1455 +1455 @@
-y=e=)
+y=e=#the_adj)
@@ -1457 +1457 @@
-severe)
+severe#severe_adj)
@@ -1459 +1459 @@
-Sentence)
+Sentence#sentence_n)
@@ -1462 +1462 @@
-of)
+of#of_prep)
@@ -1465 +1465 @@
-y=e=)
+y=e=#the_adj)
@@ -1467 +1467 @@
-Howse)
+Howse#NA)
@@ -1470 +1470 @@
-of)
+of#of_prep)
@@ -1475 +1475 @@
-Com~ons)))))
+Com~ons#NA)))))
@@ -1478 +1478 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1481 +1481 @@
-him)
+him#him_pron)
@@ -1484 +1484 @@
-unheard))))
+unheard#unheard_adj))))
@@ -1487 +1487 @@
-specifyed)
+specifyed#specify_v)
@@ -1490 +1490 @@
-in)
+in#in_prep)
@@ -1493 +1493 @@
-their)
+their#their_adj)
@@ -1495 +1495 @@
-addresse)
+addresse#address_n)
@@ -1498 +1498 @@
-to)
+to#to_prep)
@@ -1501 +1501 @@
-His)
+His#his_adj)
@@ -1503 +1503 @@
-Ma=tie=))))))
+Ma=tie=#majesty_n))))))
@@ -1509 +1509 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1517 +1517 @@
-he))
+he#he_pron))
@@ -1520 +1520 @@
-humbly))
+humbly#humbly_adv))
@@ -1522 +1522 @@
-beg'd)
+beg'd#beg_v)
@@ -1529 +1529 @@
-His)
+His#his_adj)
@@ -1531,3 +1531,3 @@
-Ma=tie=))
-(MD
-would)
+Ma=tie=#majesty_n))
+(MD
+would#will_v1)
@@ -1535 +1535 @@
-not)
+not#not_adv)
@@ -1537 +1537 @@
-comply)
+comply#comply_v)
@@ -1540 +1540 @@
-in)
+in#in_prep)
@@ -1543 +1543 @@
-putting)
+putting#put_v)
@@ -1546 +1546 @@
-it))
+it#it_pron))
@@ -1549 +1549 @@
-in)
+in#in_prep)
@@ -1552 +1552 @@
-execution)))
+execution#execution_n)))
@@ -1555 +1555 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1558 +1558 @@
-him))))))))))))
+him#him_pron))))))))))))
@@ -1567 +1567 @@
-Because)
+Because#because_adv)
@@ -1576 +1576 @@
-not)
+not#not_adv)
@@ -1578 +1578 @@
-being)
+being#be_v)
@@ -1581 +1581 @@
-conscious)
+conscious#conscious_adj)
@@ -1584 +1584 @@
-to)
+to#to_prep)
@@ -1587 +1587 @@
-himselfe)))
+himselfe#himself_pron)))
@@ -1590 +1590 @@
-of)
+of#of_prep)
@@ -1593 +1593 @@
-haveing)
+haveing#have_v)
@@ -1595 +1595 @@
-com~itted)
+com~itted#commit_v)
@@ -1598 +1598 @@
-any)
+any#any_adj)
@@ -1600 +1600 @@
-crime))))))
+crime#crime_n))))))
@@ -1605 +1605 @@
-he))
+he#he_pron))
@@ -1607 +1607 @@
-would)
+would#will_v1)
@@ -1609 +1609 @@
-give)
+give#give_v)
@@ -1612 +1612 @@
-securitye)
+securitye#security_n)
@@ -1615 +1615 @@
-to)
+to#to_prep)
@@ -1618 +1618 @@
-stande)
+stande#stand_v)
@@ -1620 +1620 @@
-and)
+and#and_conj1)
@@ -1622 +1622 @@
-abide))
+abide#abide_v))
@@ -1625 +1625 @@
-the)
+the#the_adj)
@@ -1627 +1627 @@
-judge=mt=)
+judge=mt=#judgement_n)
@@ -1630 +1630 @@
-of)
+of#of_prep)
@@ -1633 +1633 @@
-y=e=)
+y=e=#the_adj)
@@ -1635 +1635 @@
-Howse)
+Howse#NA)
@@ -1638 +1638 @@
-of)
+of#of_prep)
@@ -1641 +1641 @@
-Com~ons)
+Com~ons#NA)
@@ -1646 +1646 @@
-at)
+at#at_prep)
@@ -1649 +1649 @@
-their)
+their#their_adj)
@@ -1651 +1651 @@
-next)
+next#next_adj)
@@ -1653 +1653 @@
-meeting))))
+meeting#meeting_n))))
@@ -1660 +1660 @@
-then)
+then#than_conj)
@@ -1663 +1663 @@
-whome)))
+whome#whom_pron)))
@@ -1669 +1669 @@
-he))
+he#he_pron))
@@ -1671 +1671 @@
-did)
+did#do_v)
@@ -1673 +1673 @@
-not)
+not#not_adv)
@@ -1675 +1675 @@
-desire)
+desire#desire_v)
@@ -1678 +1678 @@
-to)
+to#to_prep)
@@ -1680 +1680 @@
-appeare)
+appeare#appear_v)
@@ -1683 +1683 @@
-to)
+to#to_prep)
@@ -1687 +1687 @@
-other)
+other#other_adj)
@@ -1689 +1689 @@
-or)
+or#or_conj2)
@@ -1691 +1691 @@
-better))
+better#better_adj))
@@ -1693 +1693 @@
-judges)
+judges#judge_n)
@@ -1700 +1700 @@
-&)
+&#and_conj1)
@@ -1704 +1704 @@
-to)
+to#to_prep)
@@ -1707 +1707 @@
-whose)
+whose#whose_pron)
@@ -1709 +1709 @@
-judgem=t=)))
+judgem=t=#judgement_n)))
@@ -1717 +1717 @@
-he))
+he#he_pron))
@@ -1719 +1719 @@
-had)
+had#have_v)
@@ -1722 +1722 @@
-at)
+at#at_prep)
@@ -1725 +1725 @@
-their)
+their#their_adj)
@@ -1727 +1727 @@
-last)
+last#last_adv)
@@ -1729 +1729 @@
-meeting)))
+meeting#meeting_n)))
@@ -1731 +1731 @@
-submitted)
+submitted#submit_v)
@@ -1734 +1734 @@
-himselfe))
+himselfe#himself_pron))
@@ -1739 +1739 @@
-if)
+if#if_conj)
@@ -1746 +1746 @@
-there))
+there#there_adv))
@@ -1748 +1748 @@
-had)
+had#have_v)
@@ -1750 +1750 @@
-beene)
+beene#be_v)
@@ -1753 +1753 @@
-time)
+time#time_n)
@@ -1755 +1755 @@
-or)
+or#or_conj2)
@@ -1757 +1757 @@
-opportunitye)
+opportunitye#opportunity_n)
@@ -1760 +1760 @@
-for)
+for#for_prep)
@@ -1763 +1763 @@
-it)))
+it#it_pron)))
@@ -1766 +1766 @@
-before)
+before#before_adv)
@@ -1769 +1769 @@
-their)
+their#their_adj)
@@ -1771 +1771 @@
-riseing))))))))))))))))
+riseing#rising_n))))))))))))))))
@@ -1780 +1780 @@
-He))
+He#he_pron))
@@ -1783 +1783 @@
-therefore))
+therefore#therefore_adv))
@@ -1786 +1786 @@
-humbly))
+humbly#humbly_adv))
@@ -1788 +1788 @@
-beseeched)
+beseeched#beseech_v)
@@ -1791 +1791 @@
-His)
+His#his_adj)
@@ -1793 +1793 @@
-Ma=tie=)
+Ma=tie=#majesty_n)
@@ -1799 +1799 @@
-who))
+who#who_pron))
@@ -1806 +1806 @@
-was)
+was#be_v)
@@ -1809 +1809 @@
-no)
+no#no_adj)
@@ -1811 +1811 @@
-stranger)
+stranger#stranger_n)
@@ -1814 +1814 @@
-to)
+to#to_prep)
@@ -1817 +1817 @@
-his)
+his#his_adj)
@@ -1819 +1819 @@
-loyalltye)
+loyalltye#loyalty_n)
@@ -1822 +1822 @@
-&)
+&#and_conj1)
@@ -1825 +1825 @@
-services)))))))))
+services#service_n1)))))))))
@@ -1830 +1830 @@
-to)
+to#to_prep)
@@ -1832 +1832 @@
-suspend)
+suspend#suspend_v)
@@ -1835 +1835 @@
-y=e=)
+y=e=#the_adj)
@@ -1837 +1837 @@
-execution)
+execution#execution_n)
@@ -1840 +1840 @@
-of)
+of#of_prep)
@@ -1843 +1843 @@
-y=e=)
+y=e=#the_adj)
@@ -1845 +1845 @@
-severe)
+severe#severe_adj)
@@ -1847 +1847 @@
-sentence)
+sentence#sentence_n)
@@ -1850 +1850 @@
-of)
+of#of_prep)
@@ -1853 +1853 @@
-Banishm=t=)
+Banishm=t=#banishment_n)
@@ -1856 +1856 @@
-out)
+out#out_adv)
@@ -1858 +1858 @@
-of)
+of#of_prep)
@@ -1861 +1861 @@
-his)
+his#his_adj)
@@ -1863 +1863 @@
-Royall)
+Royall#royal_adj)
@@ -1865 +1865 @@
-presence))))))))
+presence#presence_n))))))))
@@ -1870 +1870 @@
-till)
+till#till_prep)
@@ -1873 +1873 @@
-y=e=)
+y=e=#the_adj)
@@ -1875 +1875 @@
-meeting)
+meeting#meeting_n)
@@ -1878 +1878 @@
-of)
+of#of_prep)
@@ -1881 +1881 @@
-y=e=)
+y=e=#the_adj)
@@ -1883 +1883 @@
-Parliam=t=)
+Parliam=t=#NA)
@@ -1889 +1889 @@
-before)
+before#before_adv)
@@ -1892 +1892 @@
-whome)))
+whome#whom_pron)))
@@ -1898 +1898 @@
-he))
+he#he_pron))
@@ -1900 +1900 @@
-was)
+was#be_v)
@@ -1903 +1903 @@
-willing)
+willing#willing_adj)
@@ -1908 +1908 @@
-to)
+to#to_prep)
@@ -1910 +1910 @@
-appeare))))))))))
+appeare#appear_v))))))))))
@@ -1918 +1918 @@
-&)
+&#and_conj1)
@@ -1922 +1922 @@
-did)
+did#do_v)
@@ -1924 +1924 @@
-not)
+not#not_adv)
@@ -1926 +1926 @@
-doubt)
+doubt#doubt_v)
@@ -1929 +1929 @@
-but)
+but#but_prep)
@@ -1931 +1931 @@
-to)
+to#to_prep)
@@ -1934 +1934 @@
-cleare)
+cleare#clear_v)
@@ -1936 +1936 @@
-&)
+&#and_conj1)
@@ -1938 +1938 @@
-justifye))
+justifye#justify_v))
@@ -1941 +1941 @@
-himselfe))
+himselfe#himself_pron))
@@ -1944 +1944 @@
-in)
+in#in_prep)
@@ -1947 +1947 @@
-any)
+any#any_adj)
@@ -1949 +1949 @@
-crime)
+crime#crime_n)
@@ -1954 +1954 @@
-that)
+that#that_conj)
@@ -1959 +1959 @@
-should)
+should#shall_v)
@@ -1961 +1961 @@
-be)
+be#be_v)
@@ -1963 +1963 @@
-objected)
+objected#object_v)
@@ -1966 +1966 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1969 +1969 @@
-him))))))))
+him#him_pron))))))))
@@ -1978 +1978 @@
-His)
+His#his_adj)
@@ -1980 +1980 @@
-Petition))
+Petition#petition_n))
@@ -1982 +1982 @@
-was)
+was#be_v)
@@ -1985 +1985 @@
-that)
+that#that_pron)
@@ -1987 +1987 @@
-day))
+day#day_n))
@@ -1989 +1989 @@
-read)
+read#read_v)
@@ -1992 +1992 @@
-at)
+at#at_prep)
@@ -1995 +1995 @@
-Councill)))
+Councill#NA)))
@@ -2003 +2003 @@
-but)
+but#but_prep)
@@ -2007 +2007 @@
-other)
+other#other_adj)
@@ -2009 +2009 @@
-affaires)
+affaires#affair_n)
@@ -2012 +2012 @@
-of)
+of#of_prep)
@@ -2015 +2015 @@
-greater)
+greater#great_adj)
@@ -2017 +2017 @@
-importance))))
+importance#importance_n))))
@@ -2019 +2019 @@
-intervening))
+intervening#intervene_v))
@@ -2024 +2024 @@
-the)
+the#the_adj)
@@ -2026 +2026 @@
-further)
+further#further_adj)
@@ -2028 +2028 @@
-consideration)
+consideration#consideration_n)
@@ -2031 +2031 @@
-of)
+of#of_prep)
@@ -2034 +2034 @@
-it))))
+it#it_pron))))
@@ -2036 +2036 @@
-was)
+was#be_v)
@@ -2038 +2038 @@
-putt)
+putt#put_v)
@@ -2040 +2040 @@
-of)
+of#off_adv)
@@ -2043 +2043 @@
-till)
+till#till_prep)
@@ -2046 +2046 @@
-yesterday)))
+yesterday#yesterday_adv)))
@@ -2051 +2051 @@
-when)
+when#when_adv)
@@ -2059 +2059 @@
-the)
+the#the_adj)
@@ -2061 +2061 @@
-addresse))
+addresse#address_n))
@@ -2063 +2063 @@
-being)
+being#be_v)
@@ -2065 +2065 @@
-taken)
+taken#take_v)
@@ -2068 +2068 @@
-into)
+into#into_prep)
@@ -2071 +2071 @@
-consideration))))
+consideration#consideration_n))))
@@ -2076 +2076 @@
-it))
+it#it_pron))
@@ -2078 +2078 @@
-is)
+is#be_v)
@@ -2080 +2080 @@
-sayd)
+sayd#say_v1)
@@ -2087 +2087 @@
-the)
+the#the_adj)
@@ -2089 +2089 @@
-Preamble)
+Preamble#preamble_n)
@@ -2093 +2093 @@
-wh=ch=))
+wh=ch=#which_adj))
@@ -2100 +2100 @@
-mentions)
+mentions#mention_v)
@@ -2103 +2103 @@
-y=e=)
+y=e=#the_adj)
@@ -2105 +2105 @@
-late)
+late#late_adj1)
@@ -2107 +2107 @@
-insolencyes)
+insolencyes#insolency_n)
@@ -2110 +2110 @@
-of)
+of#of_prep)
@@ -2113 +2113 @@
-y=e=)
+y=e=#the_adj)
@@ -2115 +2115 @@
-Papists)))
+Papists#NA)))
@@ -2118 +2118 @@
-grounded)
+grounded#ground_v)
@@ -2121 +2121 @@
-upon)
+upon#upon_prep)
@@ -2125 +2125 @@
-His)
+His#his_adj)
@@ -2127 +2127 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -2129 +2129 @@
-Indulgencies))))))))
+Indulgencies#indulgency_n))))))))
@@ -2131 +2131 @@
-will)
+will#will_v1)
@@ -2133 +2133 @@
-be)
+be#be_v)
@@ -2135 +2135 @@
-referred)
+referred#refer_v)
@@ -2138 +2138 @@
-to)
+to#to_prep)
@@ -2142 +2142 @@
-y=r=)
+y=r=#your_pron)
@@ -2144 +2144 @@
-Ex=cyes=))
+Ex=cyes=#excellency_n))
@@ -2146 +2146 @@
-Examination)
+Examination#examination_n)
@@ -2149 +2149 @@
-&)
+&#and_conj1)
@@ -2154 +2154 @@
-Reporte)))
+Reporte#report_n)))
@@ -2157 +2157 @@
-thereupon)))))))))
+thereupon#thereupon_adv)))))))))
@@ -2166 +2166 @@
-As)
+As#as_adv)
@@ -2169 +2169 @@
-for)
+for#for_prep)
@@ -2172 +2172 @@
-the)
+the#the_adj)
@@ -2174 +2174 @@
-Com=n=)
+Com=n=#commission_n1)
@@ -2177 +2177 @@
-of)
+of#of_prep)
@@ -2180 +2180 @@
-Inquirye))))))
+Inquirye#inquiry_n))))))
@@ -2185 +2185 @@
-the)
+the#the_adj)
@@ -2187 +2187 @@
-extraordinary)
+extraordinary#extraordinary_adj)
@@ -2189 +2189 @@
-powers)
+powers#power_n1)
@@ -2193 +2193 @@
-wh=ch=))
+wh=ch=#which_adj))
@@ -2199 +2199 @@
-the)
+the#the_adj)
@@ -2201 +2201 @@
-Parliam=t=))
+Parliam=t=#NA))
@@ -2203 +2203 @@
-observ'd)
+observ'd#observe_v)
@@ -2208 +2208 @@
-to)
+to#to_prep)
@@ -2210 +2210 @@
-be)
+be#be_v)
@@ -2213 +2213 @@
-in)
+in#in_prep)
@@ -2216 +2216 @@
-it)))))))
+it#it_pron)))))))
@@ -2218 +2218 @@
-are)
+are#be_v)
@@ -2220 +2220 @@
-referred)
+referred#refer_v)
@@ -2223 +2223 @@
-to)
+to#to_prep)
@@ -2226 +2226 @@
-y=e=)
+y=e=#the_adj)
@@ -2228 +2228 @@
-Lords)
+Lords#lord_n)
@@ -2230 +2230 @@
-Com=rs=)
+Com=rs=#commissioner_n)
@@ -2236 +2236 @@
-who))
+who#who_pron))
@@ -2244 +2244 @@
-calling)
+calling#call_v)
@@ -2247 +2247 @@
-to)
+to#to_prep)
@@ -2250 +2250 @@
-their)
+their#their_adj)
@@ -2252 +2252 @@
-assistance)))
+assistance#assistance_n)))
@@ -2255 +2255 @@
-those)
+those#those_pron)
@@ -2258 +2258 @@
-of)
+of#of_prep)
@@ -2262 +2262 @@
-His)
+His#his_adj)
@@ -2264 +2264 @@
-M=ties=))
+M=ties=#majesty_n))
@@ -2266 +2266 @@
-Privy)
+Privy#NA)
@@ -2268 +2268 @@
-Councill)
+Councill#NA)
@@ -2271 +2271 @@
-of)
+of#of_prep)
@@ -2274 +2274 @@
-Ire=ld=)))))
+Ire=ld=#NA)))))
@@ -2278 +2278 @@
-who))
+who#who_pron))
@@ -2285 +2285 @@
-are)
+are#be_v)
@@ -2288 +2288 @@
-Members)
+Members#member_n)
@@ -2291 +2291 @@
-of)
+of#of_prep)
@@ -2294 +2294 @@
-y=e=)
+y=e=#the_adj)
@@ -2296 +2296 @@
-Howse)
+Howse#NA)
@@ -2299 +2299 @@
-of)
+of#of_prep)
@@ -2302 +2302 @@
-Com~ons))))))))))
+Com~ons#NA))))))))))
@@ -2306 +2306 @@
-are)
+are#be_v)
@@ -2309 +2309 @@
-on)
+on#on_prep)
@@ -2312 +2312 @@
-Thursday)
+Thursday#NA)
@@ -2315 +2315 @@
-next))))
+next#next_adj))))
@@ -2319 +2319 @@
-to)
+to#to_prep)
@@ -2321 +2321 @@
-take)
+take#take_v)
@@ -2324 +2324 @@
-them))
+them#them_pron))
@@ -2327 +2327 @@
-into)
+into#into_prep)
@@ -2330 +2330 @@
-consideration))))
+consideration#consideration_n))))
@@ -2333 +2333 @@
-&)
+&#and_conj1)
@@ -2338 +2338 @@
-reporte)
+reporte#report_v)
@@ -2341 +2341 @@
-their)
+their#their_adj)
@@ -2343 +2343 @@
-opinion))
+opinion#opinion_n))
@@ -2346 +2346 @@
-to)
+to#to_prep)
@@ -2350 +2350 @@
-His)
+His#his_adj)
@@ -2352 +2352 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -2355 +2355 @@
-and)
+and#and_conj1)
@@ -2358 +2358 @@
-Councill))))))))))))
+Councill#NA))))))))))))
@@ -2367 +2367 @@
-As)
+As#as_adv)
@@ -2370 +2370 @@
-for)
+for#for_prep)
@@ -2373 +2373 @@
-the)
+the#the_adj)
@@ -2375 +2375 @@
-disarming)
+disarming#disarming_n)
@@ -2378 +2378 @@
-of)
+of#of_prep)
@@ -2381 +2381 @@
-y=e=)
+y=e=#the_adj)
@@ -2383 +2383 @@
-Papists))))))
+Papists#NA))))))
@@ -2388 +2388 @@
-His)
+His#his_adj)
@@ -2390 +2390 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -2392 +2392 @@
-declared)
+declared#declare_v)
@@ -2395 +2395 @@
-his)
+his#his_adj)
@@ -2397 +2397 @@
-resolution)
+resolution#resolution_n)
@@ -2400 +2400 @@
-to)
+to#to_prep)
@@ -2402 +2402 @@
-comply)
+comply#comply_v)
@@ -2405 +2405 @@
-w=th=)
+w=th=#with_prep)
@@ -2408 +2408 @@
-the)
+the#the_adj)
@@ -2410 +2410 @@
-Howse)
+Howse#NA)
@@ -2413 +2413 @@
-of)
+of#of_prep)
@@ -2416 +2416 @@
-Com~ons)))))
+Com~ons#NA)))))
@@ -2419 +2419 @@
-in)
+in#in_prep)
@@ -2422 +2422 @@
-y=t=)
+y=t=#that_pron)
@@ -2424 +2424 @@
-particular)))))
+particular#particular_adj)))))
@@ -2432 +2432 @@
-&)
+&#and_conj1)
@@ -2435 +2435 @@
-in)
+in#in_prep)
@@ -2438 +2438 @@
-order)
+order#order_n)
@@ -2441 +2441 @@
-to)
+to#to_prep)
@@ -2444 +2444 @@
-it)))))
+it#it_pron)))))
@@ -2447 +2447 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2449 +2449 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -2451 +2451 @@
-will)
+will#will_v1)
@@ -2453 +2453 @@
-have)
+have#have_v)
@@ -2456 +2456 @@
-directions)
+directions#direction_n)
@@ -2459 +2459 @@
-to)
+to#to_prep)
@@ -2461 +2461 @@
-disarme)
+disarme#disarm_v)
@@ -2465 +2465 @@
-theme))
+theme#them_pron))
@@ -2470 +2470 @@
-and)
+and#and_conj1)
@@ -2473 +2473 @@
-all)
+all#all_adj)
@@ -2475 +2475 @@
-others)
+others#other_adj)
@@ -2479 +2479 @@
-whom))
+whom#whom_pron))
@@ -2485 +2485 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2487 +2487 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -2489 +2489 @@
-shall)
+shall#shall_v)
@@ -2491 +2491 @@
-finde)
+finde#find_v)
@@ -2494 +2494 @@
-cause)
+cause#cause_n)
@@ -2499 +2499 @@
-to)
+to#to_prep)
@@ -2501 +2501 @@
-suspect))))))))))
+suspect#suspect_v))))))))))
@@ -2510 +2510 @@
-As)
+As#as_adv)
@@ -2513 +2513 @@
-for)
+for#for_prep)
@@ -2516 +2516 @@
-the)
+the#the_adj)
@@ -2518 +2518 @@
-discharge)
+discharge#discharge_n)
@@ -2521 +2521 @@
-of)
+of#of_prep)
@@ -2526 +2526 @@
-all)
+all#all_adj)
@@ -2528 +2528 @@
-Papists)))
+Papists#NA)))
@@ -2531 +2531 @@
-out)
+out#out_adv)
@@ -2533 +2533 @@
-of)
+of#of_prep)
@@ -2536 +2536 @@
-all)
+all#all_adj)
@@ -2538 +2538 @@
-Com~ands)
+Com~ands#command_n)
@@ -2541 +2541 @@
-both)
+both#both_pron)
@@ -2543 +2543 @@
-Military)
+Military#military_adj)
@@ -2545 +2545 @@
-and)
+and#and_conj1)
@@ -2547 +2547 @@
-Civill)))))))
+Civill#civil_adj)))))))
@@ -2552 +2552 @@
-His)
+His#his_adj)
@@ -2554 +2554 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -2556 +2556 @@
-observed)
+observed#observe_v)
@@ -2566 +2566 @@
-though)
+though#though_adv)
@@ -2573 +2573 @@
-the)
+the#the_adj)
@@ -2575 +2575 @@
-Howse)
+Howse#NA)
@@ -2578 +2578 @@
-of)
+of#of_prep)
@@ -2581 +2581 @@
-Com~ons))))
+Com~ons#NA))))
@@ -2583 +2583 @@
-had)
+had#have_v)
@@ -2586 +2586 @@
-lesse)
+lesse#less_adj)
@@ -2588 +2588 @@
-cause)
+cause#cause_n)
@@ -2591 +2591 @@
-to)
+to#to_prep)
@@ -2593 +2593 @@
-complaine)
+complaine#complain_v)
@@ -2596 +2596 @@
-of)
+of#of_prep)
@@ -2599 +2599 @@
-this)))
+this#this_pron)))
@@ -2602 +2602 @@
-now))
+now#now_adv))
@@ -2605 +2605 @@
-then)
+then#than_conj)
@@ -2608 +2608 @@
-in)
+in#in_prep)
@@ -2611 +2611 @@
-y=e=)
+y=e=#the_adj)
@@ -2613 +2613 @@
-Reignes)
+Reignes#reign_n)
@@ -2616 +2616 @@
-of)
+of#of_prep)
@@ -2619 +2619 @@
-any)
+any#any_adj)
@@ -2622 +2622 @@
-of)
+of#of_prep)
@@ -2625 +2625 @@
-his)
+his#his_adj)
@@ -2627 +2627 @@
-Auncestors)))))))))))))
+Auncestors#ancestor_n)))))))))))))
@@ -2632 +2632 @@
-there))
+there#there_adv))
@@ -2634 +2634 @@
-be)
+be#be_v)
@@ -2637 +2637 @@
-only)
+only#only_adj)
@@ -2639 +2639 @@
-one)
+one#one_adj)
@@ -2642 +2642 @@
-&)
+&#and_conj1)
@@ -2645 +2645 @@
-y=t=))
+y=t=#that_pron))
@@ -2648 +2648 @@
-Co=ll=)
+Co=ll=#NA)
@@ -2650 +2650 @@
-Talbot)))
+Talbot#NA)))
@@ -2653 +2653 @@
-in)
+in#in_prep)
@@ -2656 +2656 @@
-y=e=)
+y=e=#the_adj)
@@ -2658 +2658 @@
-Armye))))
+Armye#army_n))))
@@ -2663 +2663 @@
-whereas)
+whereas#whereas_adv)
@@ -2670 +2670 @@
-in)
+in#in_prep)
@@ -2675 +2675 @@
-Queene)
+Queene#NA)
@@ -2677 +2677 @@
-Elizabeth's))
+Elizabeth's#NA))
@@ -2679 +2679 @@
-time))
+time#time_n))
@@ -2686 +2686 @@
-King)
+King#NA)
@@ -2688 +2688 @@
-James))))
+James#NA))))
@@ -2693 +2693 @@
-&)
+&#and_conj1)
@@ -2697 +2697 @@
-his)
+his#his_adj)
@@ -2699 +2699 @@
-Father's))))))
+Father's#father_n))))))
@@ -2704 +2704 @@
-there))
+there#there_adv))
@@ -2706 +2706 @@
-were)
+were#be_v)
@@ -2709 +2709 @@
-at)
+at#at_prep)
@@ -2712 +2712 @@
-all)
+all#all_adj)
@@ -2714 +2714 @@
-times)))
+times#time_n)))
@@ -2717 +2717 @@
-severall)
+severall#several_adj)
@@ -2720 +2720 @@
-of)
+of#of_prep)
@@ -2723 +2723 @@
-y=e=)
+y=e=#the_adj)
@@ -2725 +2725 @@
-Popish)
+Popish#popish_adj)
@@ -2727 +2727 @@
-Religion))))
+Religion#religion_n))))
@@ -2730 +2730 @@
-in)
+in#in_prep)
@@ -2733 +2733 @@
-y=e=)
+y=e=#the_adj)
@@ -2735 +2735 @@
-Standing)
+Standing#standing_adj)
@@ -2737 +2737 @@
-Armye))))))))
+Armye#army_n))))))))
@@ -2742 +2742 @@
-and)
+and#and_conj1)
@@ -2749 +2749 @@
-though)
+though#though_adv)
@@ -2756 +2756 @@
-he))
+he#he_pron))
@@ -2758 +2758 @@
-were)
+were#be_v)
@@ -2761 +2761 @@
-very)
+very#very_adj)
@@ -2763 +2763 @@
-well))
+well#well_adv))
@@ -2765 +2765 @@
-satisfyed)
+satisfyed#satisfy_v)
@@ -2768 +2768 @@
-w=th=)
+w=th=#with_prep)
@@ -2771 +2771 @@
-Co=ll=)
+Co=ll=#NA)
@@ -2773 +2773 @@
-Talbot)))
+Talbot#NA)))
@@ -2776 +2776 @@
-for)
+for#for_prep)
@@ -2779 +2779 @@
-his)
+his#his_adj)
@@ -2781 +2781 @@
-loyalty)
+loyalty#loyalty_n)
@@ -2784 +2784 @@
-&)
+&#and_conj1)
@@ -2787 +2787 @@
-good)
+good#good_adj)
@@ -2789 +2789 @@
-Services))))))))
+Services#service_n1))))))))
@@ -2794 +2794 @@
-yett))
+yett#yet_adv))
@@ -2797 +2797 @@
-he))
+he#he_pron))
@@ -2799 +2799 @@
-was)
+was#be_v)
@@ -2801 +2801 @@
-resolv'd)
+resolv'd#resolve_v)
@@ -2803 +2803 @@
-alsoe)
+alsoe#also_adv)
@@ -2806 +2806 @@
-to)
+to#to_prep)
@@ -2808 +2808 @@
-give)
+give#give_v)
@@ -2811 +2811 @@
-the)
+the#the_adj)
@@ -2813 +2813 @@
-Howse)
+Howse#NA)
@@ -2816 +2816 @@
-of)
+of#of_prep)
@@ -2819 +2819 @@
-Com~ons))))
+Com~ons#NA))))
@@ -2822 +2822 @@
-Satisfaction)
+Satisfaction#satisfaction_n)
@@ -2825 +2825 @@
-in)
+in#in_prep)
@@ -2828 +2828 @@
-y=t=)
+y=t=#that_pron)
@@ -2830 +2830 @@
-poynt))))
+poynt#point_n1))))
@@ -2833 +2833 @@
-by)
+by#by_prep)
@@ -2836 +2836 @@
-takeing)
+takeing#take_v)
@@ -2839 +2839 @@
-away))
+away#away_adv))
@@ -2842 +2842 @@
-his)
+his#his_adj)
@@ -2844 +2844 @@
-comand)
+comand#command_n)
@@ -2848 +2848 @@
-for)
+for#for_prep)
@@ -2851 +2851 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -2857 +2857 @@
-he))
+he#he_pron))
@@ -2859 +2859 @@
-would)
+would#will_v1)
@@ -2861 +2861 @@
-make)
+make#make_v1)
@@ -2864 +2864 @@
-him))
+him#him_pron))
@@ -2867 +2867 @@
-a)
+a#a_adj2)
@@ -2869 +2869 @@
-recompence)
+recompence#recompense_n)
@@ -2874 +2874 @@
-some)
+some#some_pron)
@@ -2876 +2876 @@
-other)
+other#other_adj)
@@ -2878 +2878 @@
-way))))))))))))
+way#way_n1))))))))))))
@@ -2887 +2887 @@
-As)
+As#as_adv)
@@ -2890 +2890 @@
-for)
+for#for_prep)
@@ -2893 +2893 @@
-y=e=)
+y=e=#the_adj)
@@ -2895 +2895 @@
-remaineing)
+remaineing#remaining_adj)
@@ -2897 +2897 @@
-particulars)
+particulars#particular_adj)
@@ -2900 +2900 @@
-of)
+of#of_prep)
@@ -2903 +2903 @@
-the)
+the#the_adj)
@@ -2905 +2905 @@
-addresse))))))
+addresse#address_n))))))
@@ -2908 +2908 @@
-the)
+the#the_adj)
@@ -2910 +2910 @@
-consideration)
+consideration#consideration_n)
@@ -2913 +2913 @@
-of)
+of#of_prep)
@@ -2916 +2916 @@
-it))))
+it#it_pron))))
@@ -2918 +2918 @@
-is)
+is#be_v)
@@ -2920 +2920 @@
-putt)
+putt#put_v)
@@ -2922 +2922 @@
-of)
+of#off_adv)
@@ -2925 +2925 @@
-till)
+till#till_prep)
@@ -2928 +2928 @@
-Wenesday)
+Wenesday#NA)
@@ -2931 +2931 @@
-next))))
+next#next_adj))))
@@ -2936 +2936 @@
-when)
+when#when_adv)
@@ -2943 +2943 @@
-His)
+His#his_adj)
@@ -2945 +2945 @@
-Maj=tie=))
+Maj=tie=#majesty_n))
@@ -2947 +2947 @@
-will)
+will#will_v1)
@@ -2949 +2949 @@
-declare)
+declare#declare_v)
@@ -2952 +2952 @@
-his)
+his#his_adj)
@@ -2954 +2954 @@
-pleasure)
+pleasure#pleasure_n)
@@ -2957 +2957 @@
-in)
+in#in_prep)
@@ -2960 +2960 @@
-them))))
+them#them_pron))))
@@ -2962 +2962 @@
-alsoe))))
+alsoe#also_adv))))
@@ -2971 +2971 @@
-I))
+I#i_pron))
@@ -2973 +2973 @@
-had)
+had#have_v)
@@ -2976 +2976 @@
-almost))
+almost#almost_adv))
@@ -2978 +2978 @@
-forgott)
+forgott#forget_v)
@@ -2981 +2981 @@
-to)
+to#to_prep)
@@ -2983 +2983 @@
-observe)
+observe#observe_v)
@@ -2986 +2986 @@
-to)
+to#to_prep)
@@ -2989 +2989 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2991 +2991 @@
-Ex=cy=)))
+Ex=cy=#excellency_n)))
@@ -2996 +2996 @@
-that)
+that#that_conj)
@@ -3000 +3000 @@
-it))
+it#it_pron))
@@ -3002 +3002 @@
-is)
+is#be_v)
@@ -3004 +3004 @@
-sayd)
+sayd#say_v1)
@@ -3011 +3011 @@
-Co=ll=)
+Co=ll=#NA)
@@ -3013 +3013 @@
-Talbot))
+Talbot#NA))
@@ -3015 +3015 @@
-withdrew)
+withdrew#withdraw_v)
@@ -3018 +3018 @@
-his)
+his#his_adj)
@@ -3020 +3020 @@
-Petition))
+Petition#petition_n))
@@ -3025 +3025 @@
-a)
+a#a_adj2)
@@ -3027 +3027 @@
-few))
+few#few_adj))
@@ -3029 +3029 @@
-minutes))
+minutes#minute_n1))
@@ -3031 +3031 @@
-before)
+before#before_adv)
@@ -3038 +3038 @@
-His)
+His#his_adj)
@@ -3040 +3040 @@
-Maj=tie=))
+Maj=tie=#majesty_n))
@@ -3042 +3042 @@
-went)
+went#go_v)
@@ -3045 +3045 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -3048 +3048 @@
-to)
+to#to_prep)
@@ -3051 +3051 @@
-Councill))))))
+Councill#NA))))))
@@ -3057 +3057 @@
-w=ch=))
+w=ch=#which_adj))
@@ -3064 +3064 @@
-makes)
+makes#make_v1)
@@ -3068 +3068 @@
-some))
+some#some_pron))
@@ -3070 +3070 @@
-conjecture)
+conjecture#conjecture_v)
@@ -3077 +3077 @@
-his)
+his#his_adj)
@@ -3080 +3080 @@
-appeareing)
+appeareing#appear_v)
@@ -3083 +3083 @@
-in)
+in#in_prep)
@@ -3086 +3086 @@
-Courte)))
+Courte#court_n1)))
@@ -3089 +3089 @@
-for)
+for#for_prep)
@@ -3092 +3092 @@
-y=e=)
+y=e=#the_adj)
@@ -3094 +3094 @@
-future)))))
+future#future_adj)))))
@@ -3096 +3096 @@
-will)
+will#will_v1)
@@ -3098 +3098 @@
-not)
+not#not_adv)
@@ -3100 +3100 @@
-be)
+be#be_v)
@@ -3103 +3103 @@
-very)
+very#very_adj)
@@ -3105 +3105 @@
-frequent))))))))))))
+frequent#frequent_adj))))))))))))
@@ -3114 +3114 @@
-I))
+I#i_pron))
@@ -3116 +3116 @@
-beg)
+beg#beg_v)
@@ -3120 +3120 @@
-yo=r=)
+yo=r=#your_pron)
@@ -3122 +3122 @@
-Ex=cyes=))
+Ex=cyes=#excellency_n))
@@ -3124 +3124 @@
-pardon)
+pardon#pardon_n1)
@@ -3127 +3127 @@
-for)
+for#for_prep)
@@ -3130 +3130 @@
-this)
+this#this_pron)
@@ -3132 +3132 @@
-long)
+long#long_adj1)
@@ -3134 +3134 @@
-lr~e))))
+lr~e#letter_n1))))
@@ -3142 +3142 @@
-&)
+&#and_conj1)
@@ -3146 +3146 @@
-crave)
+crave#crave_v)
@@ -3149 +3149 @@
-leave)
+leave#leave_n1)
@@ -3152 +3152 @@
-to)
+to#to_prep)
@@ -3154 +3154 @@
-honor)
+honor#honour_v)
@@ -3157 +3157 @@
-myselfe))
+myselfe#myself_pron))
@@ -3160 +3160 @@
-w=th=)
+w=th=#with_prep)
@@ -3163 +3163 @@
-the)
+the#the_adj)
@@ -3165 +3165 @@
-title)
+title#title_n)
@@ -3168 +3168 @@
-of)
+of#of_prep)
@@ -3179 +3179 @@
-The)
+The#the_adj)
@@ -3181 +3181 @@
-Speaker)
+Speaker#speaker_n)
@@ -3184 +3184 @@
-of)
+of#of_prep)
@@ -3187 +3187 @@
-y=e=)
+y=e=#the_adj)
@@ -3189 +3189 @@
-Howse)
+Howse#NA)
@@ -3192 +3192 @@
-of)
+of#of_prep)
@@ -3195 +3195 @@
-Com~ns))))))
+Com~ns#NA))))))
@@ -3197 +3197 @@
-was)
+was#be_v)
@@ -3200 +3200 @@
-on)
+on#on_prep)
@@ -3203 +3203 @@
-Wenesday)
+Wenesday#NA)
@@ -3206 +3206 @@
-last))))
+last#last_adv))))
@@ -3208 +3208 @@
-sworne)
+sworne#swear_v)
@@ -3211 +3211 @@
-of)
+of#of_prep)
@@ -3215 +3215 @@
-His)
+His#his_adj)
@@ -3217 +3217 @@
-Maj=ties=))
+Maj=ties=#majesty_n))
@@ -3219 +3219 @@
-Privy)
+Privy#NA)
@@ -3221 +3221 @@
-Councill)))
+Councill#NA)))
