--- chatton-1691-e3-p1.psd.vert	2025-09-10 13:51:11
+++ chatton-1691-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -13 +13 @@
-April)
+April#NA)
@@ -15 +15 @@
-26)
+26#NA)
@@ -19 +19 @@
-1690)
+1690#NA)
@@ -30 +30 @@
-My)
+My#my_adj)
@@ -32 +32 @@
-L=d=))
+L=d=#lord_n))
@@ -37 +37 @@
-Some)
+Some#some_pron)
@@ -40 +40 @@
-of)
+of#of_prep)
@@ -43 +43 @@
-my)
+my#my_adj)
@@ -45 +45 @@
-old)
+old#old_adj)
@@ -47 +47 @@
-botanick)
+botanick#botanic_adj)
@@ -49 +49 @@
-acquaintances))))
+acquaintances#acquaintance_n))))
@@ -54 +54 @@
-having)
+having#have_v)
@@ -56 +56 @@
-made)
+made#make_v1)
@@ -59 +59 @@
-an)
+an#a_adj2)
@@ -61 +61 @@
-appointm=t=)
+appointm=t=#appointment_n)
@@ -64 +64 @@
-to)
+to#to_prep)
@@ -66 +66 @@
-goe)
+goe#go_v)
@@ -69 +69 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -72 +72 @@
-to)
+to#to_prep)
@@ -75 +75 @@
-Hampton)
+Hampton#NA)
@@ -77 +77 @@
-Court)))
+Court#NA)))
@@ -82 +82 @@
-to)
+to#to_prep)
@@ -84 +84 @@
-see)
+see#see_v)
@@ -87 +87 @@
-y=e=)
+y=e=#the_adj)
@@ -89 +89 @@
-famous)
+famous#famous_adj)
@@ -91 +91 @@
-collection)
+collection#collection_n)
@@ -94 +94 @@
-ther))
+ther#there_adv))
@@ -97 +97 @@
-of)
+of#of_prep)
@@ -100 +100 @@
-y=e=)
+y=e=#the_adj)
@@ -102 +102 @@
-rare)
+rare#rare_adj1)
@@ -104 +104 @@
-Indian)
+Indian#indian_adj)
@@ -106 +106 @@
-plantes)
+plantes#plant_n1)
@@ -110 +110 @@
-w=ch=))
+w=ch=#which_adj))
@@ -118 +118 @@
-mine)
+mine#mine_adj)
@@ -120 +120 @@
-Heer)
+Heer#NA)
@@ -122 +122 @@
-Fagel))
+Fagel#NA))
@@ -124 +124 @@
-had)
+had#have_v)
@@ -126 +126 @@
-gathered)
+gathered#gather_v)
@@ -129 +129 @@
-together)))))))))))
+together#together_adv)))))))))))
@@ -135 +135 @@
-called)
+called#call_v)
@@ -138 +138 @@
-on)
+on#on_prep)
@@ -141 +141 @@
-me)))
+me#me_pron1)))
@@ -149 +149 @@
-and)
+and#and_conj1)
@@ -152 +152 @@
-I))
+I#i_pron))
@@ -155 +155 @@
-heartily))
+heartily#heartily_adv))
@@ -157 +157 @@
-wish'd)
+wish'd#wish_v)
@@ -164 +164 @@
-y=r=)
+y=r=#your_pron)
@@ -166 +166 @@
-Lo=ppe=))
+Lo=ppe=#lordship_n))
@@ -168 +168 @@
-had)
+had#have_v)
@@ -170 +170 @@
-been)
+been#be_v)
@@ -173 +173 @@
-ther))))
+ther#there_adv))))
@@ -181 +181 @@
-for)
+for#for_prep)
@@ -184 +184 @@
-there))
+there#there_adv))
@@ -186 +186 @@
-is)
+is#be_v)
@@ -190 +190 @@
-about)
+about#about_adv)
@@ -192 +192 @@
-400))
+400#NA))
@@ -194 +194 @@
-rare)
+rare#rare_adj1)
@@ -196 +196 @@
-Indian)
+Indian#indian_adj)
@@ -198 +198 @@
-plantes)
+plantes#plant_n1)
@@ -202 +202 @@
-w=ch=))
+w=ch=#which_adj))
@@ -209 +209 @@
-were)
+were#be_v)
@@ -212 +212 @@
-never))
+never#never_adv))
@@ -214 +214 @@
-seen)
+seen#see_v)
@@ -217 +217 @@
-in)
+in#in_prep)
@@ -220 +220 @@
-England))))))
+England#NA))))))
@@ -228 +228 @@
-and)
+and#and_conj1)
@@ -231 +231 @@
-there))
+there#there_adv))
@@ -233 +233 @@
-is)
+is#be_v)
@@ -236 +236 @@
-scarce))
+scarce#scarce_adj))
@@ -239 +239 @@
-any)
+any#any_adj)
@@ -241 +241 @@
-desirable)
+desirable#desirable_adj)
@@ -243 +243 @@
-Indian)
+Indian#indian_adj)
@@ -245 +245 @@
-plant))
+plant#plant_n1))
@@ -250 +250 @@
-but)
+but#but_prep)
@@ -257 +257 @@
-a)
+a#a_adj2)
@@ -259 +259 @@
-specimen))
+specimen#specimen_n))
@@ -261 +261 @@
-may)
+may#may_v1)
@@ -263 +263 @@
-be)
+be#be_v)
@@ -265 +265 @@
-seen)
+seen#see_v)
@@ -268 +268 @@
-there)))))
+there#there_adv)))))
@@ -276 +276 @@
-and)
+and#and_conj1)
@@ -279 +279 @@
-some)
+some#some_pron)
@@ -282 +282 @@
-very)
+very#very_adj)
@@ -284 +284 @@
-curious))
+curious#curious_adj))
@@ -286 +286 @@
-Indian)
+Indian#indian_adj)
@@ -288 +288 @@
-plants))
+plants#plant_n1))
@@ -290 +290 @@
-are)
+are#be_v)
@@ -293 +293 @@
-in)
+in#in_prep)
@@ -297 +297 @@
-so)
+so#so_adv)
@@ -299 +299 @@
-great))
+great#great_adj))
@@ -301 +301 @@
-perfection)
+perfection#perfection_n)
@@ -304 +304 @@
-y=t=)
+y=t=#that_conj)
@@ -308 +308 @@
-it))
+it#it_pron))
@@ -310 +310 @@
-is)
+is#be_v)
@@ -314 +314 @@
-very)
+very#very_adj)
@@ -316 +316 @@
-wonderfull))
+wonderfull#wonderful_adj))
@@ -319 +319 @@
-and)
+and#and_conj1)
@@ -322 +322 @@
-scarce)
+scarce#scarce_adj)
@@ -324 +324 @@
-credible))))))))
+credible#credible_adj))))))))
@@ -333 +333 @@
-The)
+The#the_adj)
@@ -335 +335 @@
-stoves)
+stoves#stove_n1)
@@ -339 +339 @@
-in)
+in#in_prep)
@@ -342 +342 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -350 +350 @@
-they))
+they#they_pron))
@@ -352 +352 @@
-are)
+are#be_v)
@@ -354 +354 @@
-kept))))
+kept#keep_v))))
@@ -356 +356 @@
-are)
+are#be_v)
@@ -359 +359 @@
-much)
+much#much_adj)
@@ -361 +361 @@
-better)
+better#better_adj)
@@ -366 +366 @@
-contrived)
+contrived#contrive_v1)
@@ -368 +368 @@
-and)
+and#and_conj1)
@@ -370 +370 @@
-built))
+built#build_v))
@@ -373 +373 @@
-then)
+then#than_conj)
@@ -376 +376 @@
-any)
+any#any_adj)
@@ -378 +378 @@
-other)
+other#other_adj)
@@ -381 +381 @@
-in)
+in#in_prep)
@@ -384 +384 @@
-England)))))
+England#NA)))))
@@ -393 +393 @@
-Wee))
+Wee#we_pron))
@@ -395 +395 @@
-went)
+went#go_v)
@@ -398 +398 @@
-by)
+by#by_prep)
@@ -401 +401 @@
-water))))
+water#water_n))))
@@ -407 +407 @@
-and)
+and#and_conj1)
@@ -411 +411 @@
-called)
+called#call_v)
@@ -414 +414 @@
-at)
+at#at_prep)
@@ -418 +418 @@
-a)
+a#a_adj2)
@@ -420 +420 @@
-D=rs=))
+D=rs=#doctor_n))
@@ -422 +422 @@
-house)
+house#house_n1)
@@ -427 +427 @@
-by)
+by#by_prep)
@@ -430 +430 @@
-y=e=)
+y=e=#the_adj)
@@ -432 +432 @@
-way)))
+way#way_n1)))
@@ -438 +438 @@
-wher))
+wher#where_adv))
@@ -446 +446 @@
-I))
+I#i_pron))
@@ -448 +448 @@
-saw)
+saw#see_v)
@@ -451 +451 @@
-a)
+a#a_adj2)
@@ -454 +454 @@
-very)
+very#very_adj)
@@ -456 +456 @@
-pretty))
+pretty#pretty_adj))
@@ -458 +458 @@
-floewring)
+floewring#flowering_adj)
@@ -460 +460 @@
-shrubb))
+shrubb#shrub_n1))
@@ -463 +463 @@
-in)
+in#in_prep)
@@ -466 +466 @@
-his)
+his#his_adj)
@@ -468 +468 @@
-greenhouse)))))
+greenhouse#greenhouse_n)))))
@@ -476 +476 @@
-and)
+and#and_conj1)
@@ -479 +479 @@
-I))
+I#i_pron))
@@ -481 +481 @@
-have)
+have#have_v)
@@ -483 +483 @@
-enclosed)
+enclosed#enclose_v)
@@ -487 +487 @@
-2)
+2#two_adj)
@@ -489 +489 @@
-or)
+or#or_conj2)
@@ -491 +491 @@
-3))
+3#three_adj))
@@ -493 +493 @@
-seeds)
+seeds#seed_n)
@@ -498 +498 @@
-all)
+all#all_adj)
@@ -509 +509 @@
-I))
+I#i_pron))
@@ -511 +511 @@
-cou'd)
+cou'd#can_v1)
@@ -513 +513 @@
-get)))))
+get#get_v)))))
@@ -521 +521 @@
-and)
+and#and_conj1)
@@ -524 +524 @@
-if)
+if#if_conj)
@@ -528 +528 @@
-carefully))
+carefully#carefully_adv))
@@ -530 +530 @@
-sowed)
+sowed#sow_v1)
@@ -533 +533 @@
-on)
+on#on_prep)
@@ -536 +536 @@
-a)
+a#a_adj2)
@@ -538 +538 @@
-hotbed)))))
+hotbed#hotbed_n)))))
@@ -543 +543 @@
-I))
+I#i_pron))
@@ -545 +545 @@
-doubt)
+doubt#doubt_v)
@@ -547 +547 @@
-not)
+not#not_adv)
@@ -550 +550 @@
-but)
+but#but_prep)
@@ -556 +556 @@
-they))
+they#they_pron))
@@ -558 +558 @@
-will)
+will#will_v1)
@@ -560 +560 @@
-come)
+come#come_v)
@@ -562 +562 @@
-up)))
+up#up_adv1)))
@@ -571 +571 @@
-The)
+The#the_adj)
@@ -573 +573 @@
-greatest)
+greatest#great_adj)
@@ -575 +575 @@
-news)
+news#news_n)
@@ -586 +586 @@
-I))
+I#i_pron))
@@ -588 +588 @@
-can)
+can#can_v1)
@@ -590 +590 @@
-send)
+send#send_v1)
@@ -593 +593 @@
-you))
+you#you_pron))
@@ -596 +596 @@
-from)
+from#from_prep)
@@ -599 +599 @@
-here))))))
+here#here_adv))))))
@@ -601 +601 @@
-is)
+is#be_v)
@@ -604 +604 @@
-y=t=)
+y=t=#that_conj)
@@ -608 +608 @@
-y=e=)
+y=e=#the_adj)
@@ -610 +610 @@
-Abjuration)
+Abjuration#NA)
@@ -614 +614 @@
-Bill))
+Bill#NA))
@@ -616 +616 @@
-is)
+is#be_v)
@@ -618 +618 @@
-flung)
+flung#fling_v)
@@ -621 +621 @@
-out)
+out#out_adv)
@@ -623 +623 @@
-of)
+of#of_prep)
@@ -626 +626 @@
-y=e=)
+y=e=#the_adj)
@@ -628 +628 @@
-House)
+House#NA)
@@ -631 +631 @@
-of)
+of#of_prep)
@@ -634 +634 @@
-Commons)))))
+Commons#NA)))))
@@ -640 +640 @@
-at)
+at#at_prep)
@@ -643 +643 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -652 +652 @@
-it))
+it#it_pron))
@@ -654 +654 @@
-is)
+is#be_v)
@@ -656 +656 @@
-said))
+said#say_v1))
@@ -661 +661 @@
-some))
+some#some_pron))
@@ -663 +663 @@
-are)
+are#be_v)
@@ -666 +666 @@
-soe)
+soe#so_adv)
@@ -668 +668 @@
-displeased)
+displeased#displeased_adj)
@@ -673 +673 @@
-y=t=)
+y=t=#that_conj)
@@ -677 +677 @@
-ther))
+ther#there_adv))
@@ -679 +679 @@
-will)
+will#will_v1)
@@ -681 +681 @@
-be)
+be#be_v)
@@ -684 +684 @@
-another)
+another#another_pron)
@@ -686 +686 @@
-bill))
+bill#bill_n3))
@@ -688 +688 @@
-brought)
+brought#bring_v)
@@ -690 +690 @@
-in)
+in#in_adv)
@@ -693 +693 @@
-much)
+much#much_adj)
@@ -695 +695 @@
-to)
+to#to_prep)
@@ -698 +698 @@
-y=e=)
+y=e=#the_adj)
@@ -700 +700 @@
-same)
+same#same_adj)
@@ -702 +702 @@
-effect))))))))))
+effect#effect_n))))))))))
@@ -712 +712 @@
-My)
+My#my_adj)
@@ -714 +714 @@
-L=d=)
+L=d=#lord_n)
@@ -717 +717 @@
-Nottingham@))
+Nottingham@#NA))
@@ -719 +719 @@
-@'s))
+@'s#NA))
@@ -721 +721 @@
-abdication))
+abdication#abdication_n))
@@ -723 +723 @@
-is)
+is#be_v)
@@ -726 +726 @@
-now))
+now#now_adv))
@@ -728 +728 @@
-contradicted)
+contradicted#contradict_v)
@@ -731 +731 @@
-by)
+by#by_prep)
@@ -734 +734 @@
-y=e=)
+y=e=#the_adj)
@@ -736 +736 @@
-coffeehouse)
+coffeehouse#coffee-house_n)
@@ -738 +738 @@
-newsmongers)))
+newsmongers#newsmonger_n)))
@@ -747 +747 @@
-Wee))
+Wee#we_pron))
@@ -749 +749 @@
-are)
+are#be_v)
@@ -751 +751 @@
-all)
+all#all_adj)
@@ -754 +754 @@
-here))
+here#here_adv))
@@ -756 +756 @@
-both)
+both#NA)
@@ -759 +759 @@
-to)
+to#to_prep)
@@ -765 +765 @@
-y=r=)
+y=r=#your_pron)
@@ -767 +767 @@
-Lo=ppe=))
+Lo=ppe=#lordship_n))
@@ -770 +770 @@
-and)
+and#and_conj1)
@@ -773 +773 @@
-all)
+all#all_adj)
@@ -776 +776 @@
-at)
+at#at_prep)
@@ -779 +779 @@
-Kirby)))))))
+Kirby#NA)))))))
@@ -784 +784 @@
-most)
+most#most_adj)
@@ -786 +786 @@
-faithfull))
+faithfull#faithful_adj))
@@ -789 +789 @@
-and)
+and#and_conj1)
@@ -792 +792 @@
-very)
+very#very_adj)
@@ -794 +794 @@
-humble))))
+humble#humble_adj))))
@@ -796 +796 @@
-servants))
+servants#servant_n))
@@ -804 +804 @@
-C.)
+C.#NA)
@@ -806 +806 @@
-Hatton)
+Hatton#NA)
@@ -817 +817 @@
-May)
+May#NA)
@@ -819 +819 @@
-15)
+15#fifteen_adj)
@@ -823 +823 @@
-1690)
+1690#NA)
@@ -834 +834 @@
-Wee))
+Wee#we_pron))
@@ -836 +836 @@
-are)
+are#be_v)
@@ -839 +839 @@
-likely)
+likely#likely_adj)
@@ -842 +842 @@
-to)
+to#to_prep)
@@ -844 +844 @@
-have)
+have#have_v)
@@ -847 +847 @@
-here))
+here#here_adv))
@@ -850 +850 @@
-great)
+great#great_adj)
@@ -852 +852 @@
-bussle)
+bussle#bustle_n1)
@@ -855 +855 @@
-betwixt)
+betwixt#betwixt_prep)
@@ -859 +859 @@
-y=e=)
+y=e=#the_adj)
@@ -861 +861 @@
-Wiggs))
+Wiggs#NA))
@@ -864 +864 @@
-and)
+and#and_conj1)
@@ -867 +867 @@
-modern)
+modern#modern_adj)
@@ -869 +869 @@
-Tories))))))))
+Tories#NA))))))))
@@ -877 +877 @@
-for)
+for#for_prep)
@@ -880 +880 @@
-y=e=)
+y=e=#the_adj)
@@ -882 +882 @@
-Lords))
+Lords#lord_n))
@@ -884 +884 @@
-have)
+have#have_v)
@@ -886 +886 @@
-passed)
+passed#pass_v)
@@ -889 +889 @@
-y=e=)
+y=e=#the_adj)
@@ -891 +891 @@
-bill)
+bill#bill_n3)
@@ -894 +894 @@
-for)
+for#for_prep)
@@ -898 +898 @@
-invalidating)
+invalidating#invalidate_v)
@@ -901 +901 @@
-y=e=)
+y=e=#the_adj)
@@ -903 +903 @@
-judgm=t=)
+judgm=t=#judgement_n)
@@ -906 +906 @@
-ag=t=)
+ag=t=#against_prep)
@@ -909 +909 @@
-y=e=)
+y=e=#the_adj)
@@ -911 +911 @@
-charter)
+charter#charter_n1)
@@ -914 +914 @@
-of)
+of#of_prep)
@@ -917 +917 @@
-London)))))))
+London#NA)))))))
@@ -920 +920 @@
-and)
+and#and_conj1)
@@ -923 +923 @@
-restoring)
+restoring#restore_v)
@@ -926 +926 @@
-y=e=)
+y=e=#the_adj)
@@ -928 +928 @@
-old)
+old#old_adj)
@@ -930 +930 @@
-charter))))))
+charter#charter_n1))))))
@@ -936 +936 @@
-by)
+by#by_prep)
@@ -939 +939 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -947 +947 @@
-all)
+all#all_adj)
@@ -949 +949 @@
-y=e=)
+y=e=#the_adj)
@@ -951 +951 @@
-old)
+old#old_adj)
@@ -953 +953 @@
-aldermen)
+aldermen#alderman_n)
@@ -956 +956 @@
-and)
+and#and_conj1)
@@ -959 +959 @@
-magistrates)))
+magistrates#magistrate_n)))
@@ -962 +962 @@
-of)
+of#of_prep)
@@ -965 +965 @@
-London)))
+London#NA)))
@@ -971 +971 @@
-who))
+who#who_pron))
@@ -978 +978 @@
-were)
+were#be_v)
@@ -980 +980 @@
-displaced)
+displaced#displace_v)
@@ -983 +983 @@
-when)
+when#when_adv)
@@ -990 +990 @@
-y=e=)
+y=e=#the_adj)
@@ -992 +992 @@
-charter)
+charter#charter_n1)
@@ -995 +995 @@
-of)
+of#of_prep)
@@ -998 +998 @@
-London))))
+London#NA))))
@@ -1000 +1000 @@
-wase)
+wase#be_v)
@@ -1002 +1002 @@
-judg'd)
+judg'd#judge_v)
@@ -1007 +1007 @@
-forfeited))))))))
+forfeited#forfeit_v))))))))
@@ -1011 +1011 @@
-are)
+are#be_v)
@@ -1013 +1013 @@
-restored))))
+restored#restore_v))))
@@ -1021 +1021 @@
-and)
+and#and_conj1)
@@ -1024 +1024 @@
-all)
+all#all_adj)
@@ -1026 +1026 @@
-vacancies)
+vacancies#vacancy_n)
@@ -1029 +1029 @@
-since)
+since#since_adv)
@@ -1034 +1034 @@
-by)
+by#by_prep)
@@ -1037 +1037 @@
-the)
+the#the_adj)
@@ -1039 +1039 @@
-26)
+26#NA)
@@ -1042 +1042 @@
-of)
+of#of_prep)
@@ -1045 +1045 @@
-this)
+this#this_pron)
@@ -1047 +1047 @@
-month)))))))
+month#month_n1)))))))
@@ -1051 +1051 @@
-are)
+are#be_v)
@@ -1055 +1055 @@
-by)
+by#by_prep)
@@ -1058 +1058 @@
-new)
+new#new_adj)
@@ -1060 +1060 @@
-elections)))
+elections#election_n)))
@@ -1062 +1062 @@
-to)
+to#to_prep)
@@ -1064 +1064 @@
-be)
+be#be_v)
@@ -1066 +1066 @@
-filled)
+filled#fill_v)
@@ -1068 +1068 @@
-up))
+up#up_adv1))
@@ -1073 +1073 @@
-so)
+so#so_adv)
@@ -1076 +1076 @@
-y=t=)
+y=t=#that_conj)
@@ -1080 +1080 @@
-by)
+by#by_prep)
@@ -1083 +1083 @@
-y=e=)
+y=e=#the_adj)
@@ -1085 +1085 @@
-26=th=)))
+26=th=#NA)))
@@ -1089 +1089 @@
-a)
+a#a_adj2)
@@ -1091 +1091 @@
-new)
+new#new_adj)
@@ -1093 +1093 @@
-L=d=)
+L=d=#lord_n)
@@ -1095 +1095 @@
-Mayor))
+Mayor#mayor_n))
@@ -1101 +1101 @@
-new)
+new#new_adj)
@@ -1103 +1103 @@
-sheriffs)))
+sheriffs#sheriff_n)))
@@ -1109 +1109 @@
-severall)
+severall#several_adj)
@@ -1111 +1111 @@
-new)
+new#new_adj)
@@ -1113 +1113 @@
-aldermen)))
+aldermen#alderman_n)))
@@ -1119 +1119 @@
-a)
+a#a_adj2)
@@ -1121 +1121 @@
-new)
+new#new_adj)
@@ -1123 +1123 @@
-common)
+common#common_adj)
@@ -1125 +1125 @@
-councell))))
+councell#council_n))))
@@ -1129 +1129 @@
-are)
+are#be_v)
@@ -1132 +1132 @@
-to)
+to#to_prep)
@@ -1134 +1134 @@
-be)
+be#be_v)
@@ -1136 +1136 @@
-chosen))
+chosen#choose_v))
@@ -1142 +1142 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1148 +1148 @@
-you))
+you#you_pron))
@@ -1150 +1150 @@
-will)
+will#will_v1)
@@ -1153 +1153 @@
-easily))
+easily#easily_adv))
@@ -1155 +1155 @@
-imagine)
+imagine#imagine_v)
@@ -1163 +1163 @@
-will)
+will#will_v1)
@@ -1165 +1165 @@
-occasion)
+occasion#occasion_v)
@@ -1168 +1168 @@
-warm)
+warm#warm_adj)
@@ -1170 +1170 @@
-disputes)))))))))
+disputes#dispute_n)))))))))
@@ -1179 +1179 @@
-Yesterday))
+Yesterday#yesterday_adv))
@@ -1182 +1182 @@
-S=r=)
+S=r=#NA)
@@ -1184 +1184 @@
-Ed.)
+Ed.#NA)
@@ -1186 +1186 @@
-Seymour))
+Seymour#NA))
@@ -1189 +1189 @@
-in)
+in#in_prep)
@@ -1192 +1192 @@
-y=e=)
+y=e=#the_adj)
@@ -1195 +1195 @@
-H.)
+H.#NA)
@@ -1197 +1197 @@
-of)
+of#of_prep)
@@ -1200 +1200 @@
-Commons)))))
+Commons#NA)))))
@@ -1202 +1202 @@
-tooke)
+tooke#take_v)
@@ -1205 +1205 @@
-occasion))
+occasion#occasion_n1))
@@ -1208 +1208 @@
-to)
+to#to_prep)
@@ -1210 +1210 @@
-fall)
+fall#fall_v)
@@ -1214 +1214 @@
-a)
+a#a_adj2)
@@ -1216 +1216 @@
-little))
+little#little_adj))
@@ -1218 +1218 @@
-foul))
+foul#foul_adj))
@@ -1221 +1221 @@
-upon)
+upon#upon_prep)
@@ -1224 +1224 @@
-y=e=)
+y=e=#the_adj)
@@ -1226 +1226 @@
-white)
+white#white_adj)
@@ -1228 +1228 @@
-marquise))))
+marquise#marquise_n))))
@@ -1236 +1236 @@
-and)
+and#and_conj1)
@@ -1240 +1240 @@
-wase)
+wase#be_v)
@@ -1242 +1242 @@
-seconded)
+seconded#second_v1)
@@ -1245 +1245 @@
-by)
+by#by_prep)
@@ -1248 +1248 @@
-Capt=n=)
+Capt=n=#NA)
@@ -1250 +1250 @@
-Granville)
+Granville#NA)
@@ -1256 +1256 @@
-my)
+my#my_adj)
@@ -1258 +1258 @@
-L=d=)
+L=d=#lord_n)
@@ -1261 +1261 @@
-Bath@))
+Bath@#NA))
@@ -1263 +1263 @@
-@s))
+@s#NA))
@@ -1265 +1265 @@
-son))))
+son#son_n1))))
@@ -1274 +1274 @@
-In)
+In#in_prep)
@@ -1277 +1277 @@
-return)))
+return#return_n)))
@@ -1282 +1282 @@
-some)
+some#some_pron)
@@ -1284 +1284 @@
-friends)
+friends#friend_n)
@@ -1287 +1287 @@
-to)
+to#to_prep)
@@ -1290 +1290 @@
-the)
+the#the_adj)
@@ -1294 +1294 @@
-Marquise))))
+Marquise#marquise_n))))
@@ -1296 +1296 @@
-made)
+made#make_v1)
@@ -1299 +1299 @@
-some)
+some#some_pron)
@@ -1301 +1301 @@
-sever)
+sever#severe_adj)
@@ -1303 +1303 @@
-reflections))
+reflections#reflection_n))
@@ -1306 +1306 @@
-on)
+on#on_prep)
@@ -1309 +1309 @@
-y=e=)
+y=e=#the_adj)
@@ -1311 +1311 @@
-Earle)
+Earle#earl_n)
@@ -1314 +1314 @@
-of)
+of#of_prep)
@@ -1317 +1317 @@
-Bath)))))
+Bath#NA)))))
@@ -1325 +1325 @@
-and)
+and#and_conj1)
@@ -1330 +1330 @@
-after)
+after#after_adv)
@@ -1333 +1333 @@
-some)
+some#some_pron)
@@ -1335 +1335 @@
-wrangling)))
+wrangling#wrangling_n)))
@@ -1340 +1340 @@
-y=e=)
+y=e=#the_adj)
@@ -1342 +1342 @@
-business))
+business#business_n))
@@ -1344 +1344 @@
-fell)
+fell#fall_v)
@@ -1352 +1352 @@
-and)
+and#and_conj1)
@@ -1356 +1356 @@
-y=e=)
+y=e=#the_adj)
@@ -1358 +1358 @@
-Marquises)
+Marquises#marquis_n1)
@@ -1361 +1361 @@
-and)
+and#and_conj1)
@@ -1364 +1364 @@
-Earle's))))
+Earle's#earl_n))))
@@ -1366 +1366 @@
-honour))
+honour#honour_n))
@@ -1371 +1371 @@
-notwithstanding)
+notwithstanding#notwithstanding_prep)
@@ -1374 +1374 @@
-all)
+all#all_adj)
@@ -1376 +1376 @@
-reflections)))
+reflections#reflection_n)))
@@ -1380 +1380 @@
-are)
+are#be_v)
@@ -1383 +1383 @@
-in)
+in#NA)
@@ -1385 +1385 @@
-statu)
+statu#NA)
@@ -1387 +1387 @@
-quo))
+quo#NA))
@@ -1395 +1395 @@
-C.)
+C.#NA)
@@ -1397 +1397 @@
-Hatton)
+Hatton#NA)
@@ -1411 +1411 @@
-June)
+June#NA)
@@ -1413 +1413 @@
-7)
+7#seven_adj)
@@ -1417 +1417 @@
-1690)
+1690#NA)
@@ -1428 +1428 @@
-I))
+I#i_pron))
@@ -1430 +1430 @@
-hope)
+hope#hope_v)
@@ -1435 +1435 @@
-my)
+my#my_adj)
@@ -1437 +1437 @@
-L=d=))
+L=d=#lord_n))
@@ -1446 +1446 @@
-it))
+it#it_pron))
@@ -1448 +1448 @@
-wase)
+wase#be_v)
@@ -1450 +1450 @@
-not)
+not#not_adv)
@@ -1453 +1453 @@
-any)
+any#any_adj)
@@ -1455 +1455 @@
-increase)
+increase#increase_n)
@@ -1458 +1458 @@
-of)
+of#of_prep)
@@ -1461 +1461 @@
-y=r=)
+y=r=#your_pron)
@@ -1463 +1463 @@
-indisposition)))
+indisposition#indisposition_n)))
@@ -1467 +1467 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1474 +1474 @@
-wase)
+wase#be_v)
@@ -1477 +1477 @@
-y=e=)
+y=e=#the_adj)
@@ -1479 +1479 @@
-occasion)
+occasion#occasion_n1)
@@ -1490 +1490 @@
-I))
+I#i_pron))
@@ -1492 +1492 @@
-heard)
+heard#hear_v)
@@ -1495 +1495 @@
-nothing))
+nothing#nothing_pron))
@@ -1498 +1498 @@
-from)
+from#from_prep)
@@ -1501 +1501 @@
-Kirby)))
+Kirby#NA)))
@@ -1504 +1504 @@
-y=e=)
+y=e=#the_adj)
@@ -1506 +1506 @@
-last)
+last#last_adv)
@@ -1508 +1508 @@
-post))))))))))
+post#post_n2))))))))))
@@ -1516 +1516 @@
-And)
+And#and_conj1)
@@ -1521 +1521 @@
-tho')
+tho'#though_adv)
@@ -1528 +1528 @@
-I))
+I#i_pron))
@@ -1530 +1530 @@
-have)
+have#have_v)
@@ -1532 +1532 @@
-not)
+not#not_adv)
@@ -1535 +1535 @@
-much)
+much#much_adj)
@@ -1537 +1537 @@
-news)
+news#news_n)
@@ -1545 +1545 @@
-to)
+to#to_prep)
@@ -1547 +1547 @@
-acquaint)
+acquaint#acquaint_v)
@@ -1550 +1550 @@
-y=r=)
+y=r=#your_pron)
@@ -1552 +1552 @@
-Lo=ppe=))))))))
+Lo=ppe=#lordship_n))))))))
@@ -1557 +1557 @@
-yet))
+yet#yet_adv))
@@ -1560 +1560 @@
-I))
+I#i_pron))
@@ -1562 +1562 @@
-will)
+will#will_v1)
@@ -1564 +1564 @@
-send)
+send#send_v1)
@@ -1569 +1569 @@
-w=t=))
+w=t=#what_pron))
@@ -1575 +1575 @@
-I))
+I#i_pron))
@@ -1577 +1577 @@
-heare)
+heare#hear_v)
@@ -1582 +1582 @@
-stirring)))))
+stirring#stir_v)))))
@@ -1590 +1590 @@
-and)
+and#and_conj1)
@@ -1595 +1595 @@
-in)
+in#in_prep)
@@ -1598 +1598 @@
-y=e=)
+y=e=#the_adj)
@@ -1600 +1600 @@
-first)
+first#first_adj)
@@ -1602 +1602 @@
-place)))
+place#place_n.II-VII)))
@@ -1604 +1604 @@
-tell)
+tell#tell_v)
@@ -1607 +1607 @@
-you))
+you#you_pron))
@@ -1612 +1612 @@
-w=t=))
+w=t=#what_pron))
@@ -1619 +1619 @@
-will)
+will#will_v1)
@@ -1621 +1621 @@
-be)
+be#be_v)
@@ -1624 +1624 @@
-more)
+more#more_adj)
@@ -1626 +1626 @@
-surprising)
+surprising#surprising_adj)
@@ -1629 +1629 @@
-then)
+then#than_conj)
@@ -1632 +1632 @@
-afflicting)))
+afflicting#afflicting_adj)))
@@ -1635 +1635 @@
-to)
+to#to_prep)
@@ -1638 +1638 @@
-you)))))))
+you#you_pron)))))))
@@ -1647 +1647 @@
-The)
+The#the_adj)
@@ -1649 +1649 @@
-famed)
+famed#famed_adj)
@@ -1651 +1651 @@
-Ferguson))
+Ferguson#NA))
@@ -1653 +1653 @@
-is)
+is#be_v)
@@ -1655 +1655 @@
-committed)
+committed#commit_v)
@@ -1658 +1658 @@
-to)
+to#to_prep)
@@ -1662 +1662 @@
-a)
+a#a_adj2)
@@ -1664 +1664 @@
-messenger's))
+messenger's#messenger_n))
@@ -1666 +1666 @@
-hand)))
+hand#hand_n)))
@@ -1669 +1669 @@
-on)
+on#on_prep)
@@ -1672 +1672 @@
-suspition)
+suspition#suspicion_n)
@@ -1675 +1675 @@
-of)
+of#of_prep)
@@ -1678 +1678 @@
-treason)
+treason#treason_n)
@@ -1681 +1681 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1684 +1684 @@
-y=e=)
+y=e=#the_adj)
@@ -1686 +1686 @@
-present)
+present#present_adj)
@@ -1688 +1688 @@
-governm=t=)))))))
+governm=t=#government_n)))))))
@@ -1698 +1698 @@
-This)
+This#this_pron)
@@ -1700 +1700 @@
-day))
+day#day_n))
@@ -1703 +1703 @@
-Capt=n=)
+Capt=n=#NA)
@@ -1705 +1705 @@
-Crone))
+Crone#NA))
@@ -1707 +1707 @@
-wase)
+wase#be_v)
@@ -1710 +1710 @@
-y=e=)
+y=e=#the_adj)
@@ -1712 +1712 @@
-3=d=)
+3=d=#third_adj)
@@ -1714 +1714 @@
-time))
+time#time_n))
@@ -1716 +1716 @@
-tryed))
+tryed#try_v))
@@ -1719 +1719 @@
-and)
+and#and_conj1)
@@ -1722 +1722 @@
-found)
+found#find_v)
@@ -1728 +1728 @@
-guilty)
+guilty#guilty_adj)
@@ -1731 +1731 @@
-of)
+of#of_prep)
@@ -1734 +1734 @@
-high)
+high#high_adj)
@@ -1736 +1736 @@
-treason)))))))
+treason#treason_n)))))))
@@ -1744 +1744 @@
-but)
+but#but_prep)
@@ -1749 +1749 @@
-before)
+before#before_adv)
@@ -1756 +1756 @@
-y=e=)
+y=e=#the_adj)
@@ -1758 +1758 @@
-jury))
+jury#jury_n))
@@ -1760 +1760 @@
-brought)
+brought#bring_v)
@@ -1763 +1763 @@
-him))
+him#him_pron))
@@ -1765 +1765 @@
-in)
+in#in_adv)
@@ -1768 +1768 @@
-guilty)))))
+guilty#guilty_adj)))))
@@ -1773 +1773 @@
-they))
+they#they_pron))
@@ -1775 +1775 @@
-were)
+were#be_v)
@@ -1777 +1777 @@
-shut)
+shut#shut_v)
@@ -1779 +1779 @@
-up)
+up#up_adv1)
@@ -1783 +1783 @@
-from)
+from#from_prep)
@@ -1786 +1786 @@
-3)
+3#three_adj)
@@ -1789 +1789 @@
-of)
+of#of_prep)
@@ -1792 +1792 @@
-y=e=)
+y=e=#the_adj)
@@ -1794 +1794 @@
-clocke)))
+clocke#clock_n2)))
@@ -1797 +1797 @@
-yesterday)
+yesterday#yesterday_adv)
@@ -1800 +1800 @@
-in)
+in#in_prep)
@@ -1803 +1803 @@
-y=e=)
+y=e=#the_adj)
@@ -1805 +1805 @@
-afternoone))))))
+afternoone#afternoon_n))))))
@@ -1808 +1808 @@
-till)
+till#till_prep)
@@ -1811 +1811 @@
-one)
+one#one_adj)
@@ -1814 +1814 @@
-this)
+this#this_pron)
@@ -1816 +1816 @@
-afternoone)))))
+afternoone#afternoon_n)))))
@@ -1822 +1822 @@
-after)
+after#after_adv)
@@ -1825 +1825 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -1833 +1833 @@
-in)
+in#in_prep)
@@ -1836 +1836 @@
-arrest)
+arrest#arrest_n1)
@@ -1839 +1839 @@
-of)
+of#of_prep)
@@ -1842 +1842 @@
-judgm=t=)))))
+judgm=t=#judgement_n)))))
@@ -1845 +1845 @@
-he))
+he#he_pron))
@@ -1847 +1847 @@
-pleaded)
+pleaded#plead_v)
@@ -1850 +1850 @@
-an)
+an#a_adj2)
@@ -1852 +1852 @@
-errour)
+errour#error_n)
@@ -1855 +1855 @@
-in)
+in#in_prep)
@@ -1858 +1858 @@
-y=e=)
+y=e=#the_adj)
@@ -1860 +1860 @@
-indictm=t=)
+indictm=t=#indictment_n)
@@ -1863 +1863 @@
-as)
+as#as_adv)
@@ -1866 +1866 @@
-found)
+found#find_v)
@@ -1869 +1869 @@
-by)
+by#by_prep)
@@ -1872 +1872 @@
-y=e=)
+y=e=#the_adj)
@@ -1874 +1874 @@
-grand)
+grand#grand_adj)
@@ -1876 +1876 @@
-jury))))))))))
+jury#jury_n))))))))))
@@ -1884 +1884 @@
-for)
+for#for_prep)
@@ -1887 +1887 @@
-they))
+they#they_pron))
@@ -1889 +1889 @@
-signed)
+signed#sign_v1)
@@ -1892 +1892 @@
-it))
+it#it_pron))
@@ -1896 +1896 @@
-Billa)
+Billa#NA)
@@ -1898 +1898 @@
-Verra)))
+Verra#NA)))
@@ -1906 +1906 @@
-and)
+and#and_conj1)
@@ -1909 +1909 @@
-the)
+the#the_adj)
@@ -1911 +1911 @@
-judges))
+judges#judge_n))
@@ -1913 +1913 @@
-have)
+have#have_v)
@@ -1915 +1915 @@
-appointed)
+appointed#appoint_v)
@@ -1918 +1918 @@
-to)
+to#to_prep)
@@ -1920 +1920 @@
-have)
+have#have_v)
@@ -1924 +1924 @@
-it))
+it#it_pron))
@@ -1926 +1926 @@
-argued)
+argued#argue_v)
@@ -1929 +1929 @@
-by)
+by#by_prep)
@@ -1932 +1932 @@
-councell)))
+councell#counsel_n)))
@@ -1935 +1935 @@
-before)
+before#before_adv)
@@ -1938 +1938 @@
-y=m=)))
+y=m=#them_pron)))
@@ -1941 +1941 @@
-next)
+next#next_adj)
@@ -1943 +1943 @@
-Thursday))))
+Thursday#NA))))
@@ -1952 +1952 @@
-It))
+It#it_pron))
@@ -1954 +1954 @@
-is)
+is#be_v)
@@ -1956 +1956 @@
-reported)
+reported#report_v)
@@ -1959 +1959 @@
-y=t=)
+y=t=#that_conj)
@@ -1963 +1963 @@
-y=e=)
+y=e=#the_adj)
@@ -1965 +1965 @@
-Queen)
+Queen#queen_n)
@@ -1967 +1967 @@
-Dowager))
+Dowager#dowager_n))
@@ -1969 +1969 @@
-is)
+is#be_v)
@@ -1971 +1971 @@
-ordered)
+ordered#order_v)
@@ -1974 +1974 @@
-to)
+to#to_prep)
@@ -1976 +1976 @@
-goe)
+goe#go_v)
@@ -1978 +1978 @@
-either)
+either#NA)
@@ -1981 +1981 @@
-to)
+to#to_prep)
@@ -1986 +1986 @@
-Windsor)
+Windsor#NA)
@@ -1988 +1988 @@
-or)
+or#or_conj2)
@@ -1990 +1990 @@
-Audle-End))))
+Audle-End#NA))))
@@ -1995 +1995 @@
-because)
+because#because_adv)
@@ -2002 +2002 @@
-y=e=)
+y=e=#the_adj)
@@ -2004 +2004 @@
-Papists))
+Papists#NA))
@@ -2009 +2009 @@
-on)
+on#on_prep)
@@ -2012 +2012 @@
-pretence)
+pretence#pretence_n)
@@ -2015 +2015 @@
-of)
+of#of_prep)
@@ -2018 +2018 @@
-going)
+going#go_v)
@@ -2021 +2021 @@
-to)
+to#to_prep)
@@ -2024 +2024 @@
-her)
+her#her_pron2)
@@ -2026 +2026 @@
-chappell)))))))
+chappell#chapel_n)))))))
@@ -2030 +2030 @@
-doe)
+doe#do_v)
@@ -2033 +2033 @@
-meet)
+meet#meet_v)
@@ -2035 +2035 @@
-and)
+and#and_conj1)
@@ -2037 +2037 @@
-caball)))))))
+caball#cabal_v)))))))
@@ -2046 +2046 @@
-The)
+The#the_adj)
@@ -2048 +2048 @@
-truth))
+truth#truth_n))
@@ -2050 +2050 @@
-is)
+is#be_v)
@@ -2060 +2060 @@
-y=e=)
+y=e=#the_adj)
@@ -2062 +2062 @@
-popish)
+popish#popish_adj)
@@ -2064 +2064 @@
-party))
+party#party_n))
@@ -2066 +2066 @@
-are)
+are#be_v)
@@ -2069 +2069 @@
-very)
+very#very_adj)
@@ -2071 +2071 @@
-insolent)))
+insolent#insolent_adj)))
@@ -2074 +2074 @@
-and)
+and#and_conj1)
@@ -2080 +2080 @@
-impudently))
+impudently#impudently_adv))
@@ -2082 +2082 @@
-give)
+give#give_v)
@@ -2085 +2085 @@
-y=e=)
+y=e=#the_adj)
@@ -2087 +2087 @@
-ly))
+ly#lie_n1))
@@ -2090 +2090 @@
-to)
+to#to_prep)
@@ -2093 +2093 @@
-all)
+all#all_adj)
@@ -2095 +2095 @@
-other)
+other#other_adj)
@@ -2097 +2097 @@
-publick)
+publick#public_adj)
@@ -2099 +2099 @@
-news)))))))
+news#news_n)))))))
@@ -2107 +2107 @@
-and)
+and#and_conj1)
@@ -2112 +2112 @@
-tho')
+tho'#though_adv)
@@ -2119 +2119 @@
-every)
+every#every_adj)
@@ -2121 +2121 @@
-publick)
+publick#public_adj)
@@ -2123 +2123 @@
-intelligence))
+intelligence#intelligence_n))
@@ -2125 +2125 @@
-tells)
+tells#tell_v)
@@ -2128 +2128 @@
-us))
+us#us_pron))
@@ -2132 +2132 @@
-how))
+how#how_adv))
@@ -2140 +2140 @@
-y=e=)
+y=e=#the_adj)
@@ -2142 +2142 @@
-Highlanders))
+Highlanders#NA))
@@ -2144 +2144 @@
-are)
+are#be_v)
@@ -2146 +2146 @@
-run)
+run#run_v)
@@ -2148 +2148 @@
-down))))))
+down#down_adv))))))
@@ -2153 +2153 @@
-y=t=))
+y=t=#yet_adv))
@@ -2156 +2156 @@
-y=e=)
+y=e=#the_adj)
@@ -2158 +2158 @@
-popish)
+popish#popish_adj)
@@ -2160 +2160 @@
-party))
+party#party_n))
@@ -2162 +2162 @@
-give)
+give#give_v)
@@ -2164 +2164 @@
-out)
+out#out_adv)
@@ -2167 +2167 @@
-y=t=)
+y=t=#that_conj)
@@ -2172 +2172 @@
-they))
+they#they_pron))
@@ -2174 +2174 @@
-are)
+are#be_v)
@@ -2179 +2179 @@
-very)
+very#very_adj)
@@ -2181 +2181 @@
-strong)))
+strong#strong_adj)))
@@ -2184 +2184 @@
-and)
+and#and_conj1)
@@ -2189 +2189 @@
-have)
+have#have_v)
@@ -2192 +2192 @@
-lately))
+lately#lately_adv))
@@ -2194 +2194 @@
-received)
+received#receive_v)
@@ -2197 +2197 @@
-great)
+great#great_adj)
@@ -2199 +2199 @@
-recruits))
+recruits#recruit_n))
@@ -2202 +2202 @@
-from)
+from#from_prep)
@@ -2205 +2205 @@
-Ireland)))))))
+Ireland#NA)))))))
@@ -2214 +2214 @@
-Wee))
+Wee#we_pron))
@@ -2216 +2216 @@
-shall)
+shall#shall_v)
@@ -2219 +2219 @@
-never))
+never#never_adv))
@@ -2221 +2221 @@
-be)
+be#be_v)
@@ -2224 +2224 @@
-happy))
+happy#happy_adj))
@@ -2227 +2227 @@
-till)
+till#till_prep)
@@ -2234 +2234 @@
-all)
+all#all_adj)
@@ -2236 +2236 @@
-our)
+our#our_pron)
@@ -2238 +2238 @@
-damnable)
+damnable#damnable_adj)
@@ -2240 +2240 @@
-lyers))
+lyers#liar_n))
@@ -2242 +2242 @@
-are)
+are#be_v)
@@ -2244 +2244 @@
-confounded))))
+confounded#confound_v))))
@@ -2250 +2250 @@
-w=ch=))
+w=ch=#which_adj))
@@ -2258 +2258 @@
-God))
+God#NA))
@@ -2261 +2261 @@
-speedily))
+speedily#speedily_adv))
@@ -2263 +2263 @@
-grant)))
+grant#grant_v)))
@@ -2273 +2273 @@
-I))
+I#i_pron))
@@ -2276 +2276 @@
-and)
+and#and_conj1)
@@ -2279 +2279 @@
-all)
+all#all_adj)
@@ -2282 +2282 @@
-here)))))
+here#here_adv)))))
@@ -2284 +2284 @@
-are)
+are#be_v)
@@ -2289 +2289 @@
-w=th=)
+w=th=#with_prep)
@@ -2292 +2292 @@
-great)
+great#great_adj)
@@ -2294 +2294 @@
-sincerity)))
+sincerity#sincerity_n)))
@@ -2299 +2299 @@
-to)
+to#to_prep)
@@ -2303 +2303 @@
-y=r=)
+y=r=#your_pron)
@@ -2305 +2305 @@
-Lo=ppe=))
+Lo=ppe=#lordship_n))
@@ -2308 +2308 @@
-and)
+and#and_conj1)
@@ -2311 +2311 @@
-all)
+all#all_adj)
@@ -2314 +2314 @@
-at)
+at#at_prep)
@@ -2317 +2317 @@
-Kirby)))))))
+Kirby#NA)))))))
@@ -2321 +2321 @@
-most)
+most#most_adj)
@@ -2323 +2323 @@
-faithfull)
+faithfull#faithful_adj)
@@ -2326 +2326 @@
-and)
+and#and_conj1)
@@ -2329 +2329 @@
-humble))))
+humble#humble_adj))))
@@ -2331 +2331 @@
-servants))
+servants#servant_n))
@@ -2339 +2339 @@
-C.)
+C.#NA)
@@ -2341 +2341 @@
-Hatton)
+Hatton#NA)
@@ -2355 +2355 @@
-June)
+June#NA)
@@ -2357 +2357 @@
-17)
+17#seventeen_adj)
@@ -2361 +2361 @@
-1690)
+1690#NA)
@@ -2372 +2372 @@
-Wee))
+Wee#we_pron))
@@ -2374 +2374 @@
-have)
+have#have_v)
@@ -2377 +2377 @@
-here))
+here#here_adv))
@@ -2381 +2381 @@
-very)
+very#very_adj)
@@ -2383 +2383 @@
-little))
+little#little_adj))
@@ -2385 +2385 @@
-news)
+news#news_n)
@@ -2388 +2388 @@
-stirring)))
+stirring#stir_v)))
@@ -2397 +2397 @@
-These)
+These#these_pron)
@@ -2399 +2399 @@
-infidells)
+infidells#infidel_adj)
@@ -2404 +2404 @@
-y=e=)
+y=e=#the_adj)
@@ -2406 +2406 @@
-Jacobites)))
+Jacobites#NA)))
@@ -2410 +2410 @@
-goe)
+goe#go_v)
@@ -2412 +2412 @@
-about)
+about#about_adv)
@@ -2415 +2415 @@
-to)
+to#to_prep)
@@ -2417 +2417 @@
-shamme)
+shamme#shame_v)
@@ -2420 +2420 @@
-all)
+all#all_adj)
@@ -2422 +2422 @@
-y=e=)
+y=e=#the_adj)
@@ -2424 +2424 @@
-glorious)
+glorious#glorious_adj)
@@ -2426 +2426 @@
-defeat)
+defeat#defeat_n)
@@ -2429 +2429 @@
-of)
+of#of_prep)
@@ -2432 +2432 @@
-y=e=)
+y=e=#the_adj)
@@ -2434 +2434 @@
-Thoulon)
+Thoulon#NA)
@@ -2436 +2436 @@
-fleet)))))
+fleet#fleet_n1)))))
@@ -2444 +2444 @@
-and)
+and#and_conj1)
@@ -2448 +2448 @@
-wou'd)
+wou'd#will_v1)
@@ -2450 +2450 @@
-have)
+have#have_v)
@@ -2454 +2454 @@
-it))
+it#it_pron))
@@ -2456 +2456 @@
-pass)
+pass#pass_v)
@@ -2459 +2459 @@
-for)
+for#for_prep)
@@ -2462 +2462 @@
-a)
+a#a_adj2)
@@ -2464 +2464 @@
-shamme))))
+shamme#sham_n1))))
@@ -2472 +2472 @@
-and)
+and#and_conj1)
@@ -2475 +2475 @@
-they))
+they#they_pron))
@@ -2478 +2478 @@
-wickedly))
+wickedly#wickedly_adv))
@@ -2480 +2480 @@
-endeavour)
+endeavour#endeavour_v)
@@ -2483 +2483 @@
-to)
+to#to_prep)
@@ -2485 +2485 @@
-persuade)
+persuade#persuade_v)
@@ -2488 +2488 @@
-y=e=)
+y=e=#the_adj)
@@ -2490 +2490 @@
-people))
+people#people_n))
@@ -2497 +2497 @@
-our)
+our#our_pron)
@@ -2499 +2499 @@
-gazeteer))
+gazeteer#gazetteer_n))
@@ -2501 +2501 @@
-is)
+is#be_v)
@@ -2505 +2505 @@
-noe)
+noe#no_adj)
@@ -2507 +2507 @@
-more))
+more#more_adj))
@@ -2509 +2509 @@
-infallibel)
+infallibel#infallible_adj)
@@ -2512 +2512 @@
-than)
+than#than_conj)
@@ -2515 +2515 @@
-y=e=)
+y=e=#the_adj)
@@ -2517 +2517 @@
-pope)))))))
+pope#pope_n1)))))))
@@ -2525 +2525 @@
-C.)
+C.#NA)
@@ -2527 +2527 @@
-Hatton)
+Hatton#NA)
