--- dell-1688-e3-p1.psd.vert	2025-09-10 13:51:11
+++ dell-1688-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -19 +19 @@
-London))
+London#NA))
@@ -25 +25 @@
-18)
+18#eighteen_adj)
@@ -27 +27 @@
-Dec.)
+Dec.#NA)
@@ -29 +29 @@
-1688)
+1688#NA)
@@ -38 +38 @@
-Sir))
+Sir#sir_n))
@@ -41 +41 @@
-Common)
+Common#common_adj)
@@ -43 +43 @@
-experience))
+experience#experience_n))
@@ -45 +45 @@
-tells)
+tells#tell_v)
@@ -48 +48 @@
-us))
+us#us_pron))
@@ -51 +51 @@
-that)
+that#that_conj)
@@ -55 +55 @@
-when)
+when#when_adv)
@@ -62 +62 @@
-people))
+people#people_n))
@@ -65 +65 @@
-earnestly))
+earnestly#earnestly_adv))
@@ -67 +67 @@
-desire)
+desire#desire_v)
@@ -70 +70 @@
-any)
+any#any_adj)
@@ -72 +72 @@
-thing)))))
+thing#thing_n1)))))
@@ -77 +77 @@
-they))
+they#they_pron))
@@ -79 +79 @@
-are)
+are#be_v)
@@ -82 +82 @@
-apt)
+apt#apt_adj)
@@ -85 +85 @@
-to)
+to#to_prep)
@@ -87 +87 @@
-fancy)
+fancy#fancy_v)
@@ -94 +94 @@
-the)
+the#the_adj)
@@ -96 +96 @@
-success))
+success#success_n))
@@ -98 +98 @@
-will)
+will#will_v1)
@@ -100 +100 @@
-be)
+be#be_v)
@@ -103 +103 @@
-answerable)
+answerable#answerable_adj)
@@ -106 +106 @@
-to)
+to#to_prep)
@@ -109 +109 @@
-their)
+their#their_adj)
@@ -111 +111 @@
-expectation))))))))
+expectation#expectation_n))))))))
@@ -117 +117 @@
-which))
+which#which_adj))
@@ -124 +124 @@
-is)
+is#be_v)
@@ -127 +127 @@
-the)
+the#the_adj)
@@ -129 +129 @@
-reason)
+reason#reason_n1)
@@ -133 +133 @@
-why))
+why#why_adv))
@@ -141 +141 @@
-the)
+the#the_adj)
@@ -143 +143 @@
-time)
+time#time_n)
@@ -146 +146 @@
-of)
+of#of_prep)
@@ -150 +150 @@
-the)
+the#the_adj)
@@ -152 +152 @@
-Prince)
+Prince#prince_n)
@@ -155 +155 @@
-of)
+of#of_prep)
@@ -158 +158 @@
-Orange@)))
+Orange@#NA)))
@@ -160 +160 @@
-@'s))
+@'s#NA))
@@ -162 +162 @@
-coming))))
+coming#coming_n))))
@@ -164 +164 @@
-has)
+has#have_v)
@@ -166 +166 @@
-been)
+been#be_v)
@@ -169 +169 @@
-so)
+so#so_adv)
@@ -171 +171 @@
-often))
+often#often_adv))
@@ -174 +174 @@
-positively))
+positively#positively_adv))
@@ -176 +176 @@
-determined))))))))
+determined#determine_v))))))))
@@ -184 +184 @@
-but)
+but#but_prep)
@@ -187 +187 @@
-I))
+I#i_pron))
@@ -189 +189 @@
-can)
+can#can_v1)
@@ -192 +192 @@
-now))
+now#now_adv))
@@ -195 +195 @@
-certainly))
+certainly#certainly_adv))
@@ -197 +197 @@
-assure)
+assure#assure_v)
@@ -200 +200 @@
-you))
+you#you_pron))
@@ -203 +203 @@
-that)
+that#that_conj)
@@ -207 +207 @@
-he))
+he#he_pron))
@@ -209 +209 @@
-arrived)
+arrived#arrive_v)
@@ -212 +212 @@
-at)
+at#at_prep)
@@ -216 +216 @@
-S=t=.)
+S=t=.#NA)
@@ -218 +218 @@
-James's))))
+James's#NA))))
@@ -222 +222 @@
-about)
+about#about_adv)
@@ -224 +224 @@
-two))
+two#two_adj))
@@ -227 +227 @@
-a)
+a#a_prep2)
@@ -230 +230 @@
-clock)))
+clock#clock_n1)))
@@ -233 +233 @@
-this)
+this#this_pron)
@@ -235 +235 @@
-afternoon)))
+afternoon#afternoon_n)))
@@ -240 +240 @@
-tho')
+tho'#though_adv)
@@ -247 +247 @@
-I))
+I#i_pron))
@@ -249 +249 @@
-will)
+will#will_v1)
@@ -251 +251 @@
-not)
+not#not_adv)
@@ -253 +253 @@
-pretend)
+pretend#pretend_v)
@@ -256 +256 @@
-to)
+to#to_prep)
@@ -258 +258 @@
-describe)
+describe#describe_v)
@@ -261 +261 @@
-the)
+the#the_adj)
@@ -263 +263 @@
-Manner)
+Manner#manner_n)
@@ -266 +266 @@
-of)
+of#of_prep)
@@ -269 +269 @@
-his)
+his#his_adj)
@@ -271 +271 @@
-Entrance))))
+Entrance#entrance_n))))
@@ -277 +277 @@
-it))
+it#it_pron))
@@ -279 +279 @@
-being)
+being#be_v)
@@ -282 +282 @@
-as)
+as#as_adv)
@@ -284 +284 @@
-privately)
+privately#privately_adv)
@@ -287 +287 @@
-as)
+as#as_adv)
@@ -298 +298 @@
-with)
+with#with_prep)
@@ -301 +301 @@
-respect)
+respect#respect_n)
@@ -304 +304 @@
-to)
+to#to_prep)
@@ -307 +307 @@
-his)
+his#his_adj)
@@ -309 +309 @@
-attendance)))))
+attendance#attendance_n)))))
@@ -311 +311 @@
-was)
+was#be_v)
@@ -314 +314 @@
-possible)))))))
+possible#possible_adj)))))))
@@ -319 +319 @@
-being)
+being#be_v)
@@ -321 +321 @@
-met)
+met#meet_v)
@@ -326 +326 @@
-out)
+out#out_adv)
@@ -328 +328 @@
-of)
+of#of_prep)
@@ -331 +331 @@
-Town)))
+Town#town_n)))
@@ -336 +336 @@
-by)
+by#by_prep)
@@ -339 +339 @@
-the)
+the#the_adj)
@@ -341 +341 @@
-Sherifs)
+Sherifs#sheriff_n)
@@ -344 +344 @@
-of)
+of#of_prep)
@@ -347 +347 @@
-London)
+London#NA)
@@ -349 +349 @@
-and)
+and#and_conj1)
@@ -351 +351 @@
-Middlesex)))
+Middlesex#NA)))
@@ -357 +357 @@
-who))
+who#who_pron))
@@ -364 +364 @@
-surrendered)
+surrendered#surrender_v)
@@ -367 +367 @@
-their)
+their#their_adj)
@@ -369 +369 @@
-white)
+white#NA)
@@ -371 +371 @@
-staves))
+staves#NA))
@@ -374 +374 @@
-to)
+to#to_prep)
@@ -377 +377 @@
-his)
+his#his_adj)
@@ -379 +379 @@
-Highness)
+Highness#highness_n)
@@ -385 +385 @@
-which))
+which#which_adj))
@@ -392 +392 @@
-were)
+were#be_v)
@@ -395 +395 @@
-immediately))
+immediately#immediately_adv))
@@ -397 +397 @@
-returned)
+returned#return_v1)
@@ -400 +400 @@
-to)
+to#to_prep)
@@ -403 +403 @@
-them))))))))))))))))))
+them#them_pron))))))))))))))))))
@@ -412 +412 @@
-It))
+It#it_pron))
@@ -414 +414 @@
-will)
+will#will_v1)
@@ -416 +416 @@
-be)
+be#be_v)
@@ -419 +419 @@
-unnecessary))
+unnecessary#unnecessary_adj))
@@ -422 +422 @@
-to)
+to#to_prep)
@@ -424 +424 @@
-acquaint)
+acquaint#acquaint_v)
@@ -427 +427 @@
-you))
+you#you_pron))
@@ -430 +430 @@
-that)
+that#that_conj)
@@ -435 +435 @@
-he))
+he#he_pron))
@@ -437 +437 @@
-was)
+was#be_v)
@@ -440 +440 @@
-preceded)
+preceded#precede_v.1-7)
@@ -442 +442 @@
-and)
+and#and_conj1)
@@ -444 +444 @@
-guarded))
+guarded#guard_v))
@@ -447 +447 @@
-by)
+by#by_prep)
@@ -450 +450 @@
-a)
+a#a_adj2)
@@ -452 +452 @@
-numerous)
+numerous#numerous_adj)
@@ -454 +454 @@
-company)
+company#company_n)
@@ -457 +457 @@
-of)
+of#of_prep)
@@ -460 +460 @@
-his)
+his#his_adj)
@@ -462 +462 @@
-army))))))
+army#army_n))))))
@@ -467 +467 @@
-and)
+and#and_conj1)
@@ -470 +470 @@
-accompanied)
+accompanied#accompany_v)
@@ -473 +473 @@
-by)
+by#by_prep)
@@ -476 +476 @@
-multitudes)
+multitudes#multitude_n)
@@ -479 +479 @@
-of)
+of#of_prep)
@@ -482 +482 @@
-all)
+all#all_adj)
@@ -484 +484 @@
-ranks)))
+ranks#rank_n1)))
@@ -487 +487 @@
-on)
+on#on_prep)
@@ -490 +490 @@
-horseback))))))))))
+horseback#horseback_n))))))))))
@@ -496 +496 @@
-these))
+these#these_pron))
@@ -498 +498 @@
-being)
+being#be_v)
@@ -501 +501 @@
-inseperable)
+inseperable#inseparable_adj)
@@ -503 +503 @@
-consequences)
+consequences#consequence_n)
@@ -506 +506 @@
-of)
+of#of_prep)
@@ -509 +509 @@
-such)
+such#such_adj)
@@ -511 +511 @@
-occurrences)))))
+occurrences#occurrence_n)))))
@@ -519 +519 @@
-and)
+and#and_conj1)
@@ -522 +522 @@
-the)
+the#the_adj)
@@ -524 +524 @@
-excessive)
+excessive#excessive_adj)
@@ -526 +526 @@
-rain)
+rain#rain_n1)
@@ -530 +530 @@
-which))
+which#which_adj))
@@ -537 +537 @@
-fell)
+fell#fall_v)
@@ -540 +540 @@
-this)
+this#this_pron)
@@ -542 +542 @@
-day)))))
+day#day_n)))))
@@ -548 +548 @@
-I))
+I#i_pron))
@@ -550 +550 @@
-may)
+may#may_v1)
@@ -553 +553 @@
-almost))
+almost#almost_adv))
@@ -555 +555 @@
-say)
+say#say_v1)
@@ -560 +560 @@
-without)
+without#without_adv)
@@ -563 +563 @@
-intermission))))
+intermission#intermission_n1))))
@@ -567 +567 @@
-could)
+could#can_v1)
@@ -569 +569 @@
-not)
+not#not_adv)
@@ -571 +571 @@
-detain)
+detain#detain_v)
@@ -574 +574 @@
-the)
+the#the_adj)
@@ -576 +576 @@
-generality))
+generality#generality_n))
@@ -579 +579 @@
-within)
+within#within_adv)
@@ -582 +582 @@
-doors)))
+doors#door_n)))
@@ -587 +587 @@
-notwithstanding)
+notwithstanding#notwithstanding_prep)
@@ -594 +594 @@
-the)
+the#the_adj)
@@ -596 +596 @@
-streets))
+streets#street_n))
@@ -599 +599 @@
-by)
+by#by_prep)
@@ -602 +602 @@
-reason)
+reason#reason_n1)
@@ -605 +605 @@
-of)
+of#of_prep)
@@ -608 +608 @@
-the)
+the#the_adj)
@@ -610 +610 @@
-extraordinary)
+extraordinary#extraordinary_adj)
@@ -612 +612 @@
-crowd)))))
+crowd#crowd_n3)))))
@@ -614 +614 @@
-restrained)
+restrained#restrain_v)
@@ -617 +617 @@
-them))
+them#them_pron))
@@ -620 +620 @@
-from)
+from#from_prep)
@@ -623 +623 @@
-an)
+an#a_adj2)
@@ -625 +625 @@
-absolute)
+absolute#absolute_adj)
@@ -627 +627 @@
-liberty))))))
+liberty#liberty_n))))))
@@ -633 +633 @@
-so)
+so#so_adv)
@@ -635 +635 @@
-powerful))
+powerful#powerful_adj))
@@ -637 +637 @@
-is)
+is#be_v)
@@ -640 +640 @@
-the)
+the#the_adj)
@@ -642 +642 @@
-concupiscence)
+concupiscence#concupiscence_n)
@@ -645 +645 @@
-of)
+of#of_prep)
@@ -648 +648 @@
-satisfying)
+satisfying#satisfy_v)
@@ -651 +651 @@
-ones)
+ones#one_adj)
@@ -653 +653 @@
-curiosity))))))
+curiosity#curiosity_n))))))
@@ -662 +662 @@
-You))
+You#you_pron))
@@ -664 +664 @@
-may)
+may#may_v1)
@@ -666 +666 @@
-imagine)
+imagine#imagine_v)
@@ -673 +673 @@
-it))
+it#it_pron))
@@ -675 +675 @@
-was)
+was#be_v)
@@ -678 +678 @@
-no)
+no#no_adj)
@@ -680 +680 @@
-unsurprising)
+unsurprising#unsurprising_adj)
@@ -682 +682 @@
-spectacle))
+spectacle#spectacle_n1))
@@ -685 +685 @@
-to)
+to#to_prep)
@@ -687 +687 @@
-see)
+see#see_v)
@@ -692 +692 @@
-if)
+if#if_conj)
@@ -699 +699 @@
-I))
+I#i_pron))
@@ -701 +701 @@
-may)
+may#may_v1)
@@ -704 +704 @@
-so))
+so#so_adv))
@@ -706 +706 @@
-phrase)
+phrase#phrase_v)
@@ -709 +709 @@
-it)))))
+it#it_pron)))))
@@ -715 +715 @@
-a)
+a#a_adj2)
@@ -719 +719 @@
-foreign)
+foreign#foreign_adj)
@@ -721 +721 @@
-enemy))
+enemy#enemy_n))
@@ -724 +724 @@
-in)
+in#in_prep)
@@ -727 +727 @@
-an)
+an#a_adj2)
@@ -729 +729 @@
-hostile)
+hostile#hostile_adj)
@@ -731 +731 @@
-manner)))
+manner#manner_n)))
@@ -733 +733 @@
-march)
+march#march_v2)
@@ -736 +736 @@
-through)
+through#through_prep)
@@ -739 +739 @@
-the)
+the#the_adj)
@@ -741 +741 @@
-metropolis)
+metropolis#metropolis_n)
@@ -744 +744 @@
-of)
+of#of_prep)
@@ -747 +747 @@
-the)
+the#the_adj)
@@ -749 +749 @@
-Kingdom)))))
+Kingdom#kingdom_n)))))
@@ -752 +752 @@
-with)
+with#with_prep)
@@ -755 +755 @@
-no)
+no#no_adj)
@@ -757 +757 @@
-other)
+other#other_adj)
@@ -759 +759 @@
-diversion)
+diversion#diversion_n)
@@ -762 +762 @@
-than)
+than#than_conj)
@@ -765 +765 @@
-the)
+the#the_adj)
@@ -767 +767 @@
-repeated)
+repeated#repeated_adj)
@@ -769 +769 @@
-huzzas)
+huzzas#huzza_int)
@@ -772 +772 @@
-and)
+and#and_conj1)
@@ -775 +775 @@
-loud)
+loud#loud_adj)
@@ -777 +777 @@
-acclamations)))
+acclamations#acclamation_n)))
@@ -780 +780 @@
-of)
+of#of_prep)
@@ -783 +783 @@
-the)
+the#the_adj)
@@ -785 +785 @@
-inhabitants)
+inhabitants#inhabitant_adj)
@@ -791 +791 @@
-who))
+who#who_pron))
@@ -798 +798 @@
-shewed)
+shewed#show_v)
@@ -801 +801 @@
-no)
+no#no_adj)
@@ -803 +803 @@
-other)
+other#other_adj)
@@ -805 +805 @@
-concern)
+concern#concern_n)
@@ -808 +808 @@
-for)
+for#for_prep)
@@ -811 +811 @@
-the)
+the#the_adj)
@@ -813 +813 @@
-Revolution)))
+Revolution#revolution_n)))
@@ -816 +816 @@
-but)
+but#but_prep)
@@ -821 +821 @@
-what))
+what#what_pron))
@@ -828 +828 @@
-might)
+might#may_v1)
@@ -830 +830 @@
-express)
+express#express_v1)
@@ -833 +833 @@
-their)
+their#their_adj)
@@ -835 +835 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -838 +838 @@
-and)
+and#and_conj1)
@@ -841 +841 @@
-approbation)))))))))))))))))))))
+approbation#approbation_n)))))))))))))))))))))
@@ -850 +850 @@
-In)
+In#in_prep)
@@ -853 +853 @@
-a)
+a#a_adj2)
@@ -855 +855 @@
-word)))
+word#word_n)))
@@ -858 +858 @@
-all)
+all#all_adj)
@@ -860 +860 @@
-things))
+things#thing_n1))
@@ -862 +862 @@
-conspired)
+conspired#conspire_v)
@@ -865 +865 @@
-to)
+to#to_prep)
@@ -867 +867 @@
-testify)
+testify#testify_v)
@@ -870 +870 @@
-their)
+their#their_adj)
@@ -872 +872 @@
-joy))
+joy#joy_n))
@@ -875 +875 @@
-on)
+on#on_prep)
@@ -878 +878 @@
-this)
+this#this_pron)
@@ -880 +880 @@
-occasion))))
+occasion#occasion_n1))))
@@ -889 +889 @@
-The)
+The#the_adj)
@@ -891 +891 @@
-King))
+King#king_n))
@@ -894 +894 @@
-being)
+being#be_v)
@@ -896 +896 @@
-prevented)
+prevented#prevent_v)
@@ -901 +901 @@
-as)
+as#as_adv)
@@ -912 +912 @@
-I))
+I#i_pron))
@@ -915 +915 @@
-formerly))
+formerly#formerly_adv))
@@ -917 +917 @@
-hinted))))
+hinted#hint_v))))
@@ -922 +922 @@
-from)
+from#from_prep)
@@ -925 +925 @@
-withdrawing)
+withdrawing#withdraw_v)
@@ -928 +928 @@
-himself))
+himself#himself_pron))
@@ -931 +931 @@
-as)
+as#as_adv)
@@ -942 +942 @@
-he))
+he#he_pron))
@@ -944 +944 @@
-intended)
+intended#intend_v)
@@ -947 +947 @@
-on)
+on#on_prep)
@@ -950 +950 @@
-Sunday)
+Sunday#NA)
@@ -952 +952 @@
-evening)))))))))
+evening#evening_n1)))))))))
@@ -956 +956 @@
-returned)
+returned#return_v1)
@@ -959 +959 @@
-in)
+in#in_prep)
@@ -962 +962 @@
-his)
+his#his_adj)
@@ -964 +964 @@
-Coach)))
+Coach#coach_n)))
@@ -970 +970 @@
-preceded)
+preceded#precede_v.1-7)
@@ -973 +973 @@
-by)
+by#by_prep)
@@ -976 +976 @@
-several)
+several#several_adj)
@@ -978 +978 @@
-gentlemen)
+gentlemen#gentleman_n)
@@ -981 +981 @@
-bare_headed))
+bare_headed#bare-headed_adj))
@@ -984 +984 @@
-on)
+on#on_prep)
@@ -987 +987 @@
-horse-back))))))
+horse-back#horseback_n))))))
@@ -992 +992 @@
-and)
+and#and_conj1)
@@ -995 +995 @@
-followed)
+followed#follow_v)
@@ -998 +998 @@
-by)
+by#by_prep)
@@ -1001 +1001 @@
-some)
+some#some_pron)
@@ -1004 +1004 @@
-of)
+of#of_prep)
@@ -1007 +1007 @@
-his)
+his#his_adj)
@@ -1009 +1009 @@
-Guards)
+Guards#guard_n)
@@ -1012 +1012 @@
-sent)
+sent#send_v1)
@@ -1015 +1015 @@
-by)
+by#by_prep)
@@ -1018 +1018 @@
-the)
+the#the_adj)
@@ -1020 +1020 @@
-Lords)
+Lords#lord_n)
@@ -1023 +1023 @@
-of)
+of#of_prep)
@@ -1026 +1026 @@
-the)
+the#the_adj)
@@ -1028 +1028 @@
-Council)))
+Council#council_n)))
@@ -1031 +1031 @@
-and)
+and#and_conj1)
@@ -1034 +1034 @@
-Peers)))))
+Peers#peer_n)))))
@@ -1037 +1037 @@
-for)
+for#for_prep)
@@ -1040 +1040 @@
-that)
+that#that_pron)
@@ -1042 +1042 @@
-purpose)))))))))))
+purpose#purpose_n)))))))))))
@@ -1052 +1052 @@
-came)
+came#come_v)
@@ -1055 +1055 @@
-back)
+back#back_adv)
@@ -1057 +1057 @@
-to)
+to#to_prep)
@@ -1060 +1060 @@
-White)
+White#NA)
@@ -1062 +1062 @@
-Hall)))
+Hall#NA)))
@@ -1065 +1065 @@
-through)
+through#through_prep)
@@ -1068 +1068 @@
-the)
+the#the_adj)
@@ -1070 +1070 @@
-City)))
+City#NA)))
@@ -1075 +1075 @@
-not)
+not#not_adv)
@@ -1077 +1077 @@
-without)
+without#without_adv)
@@ -1080 +1080 @@
-some)
+some#some_pron)
@@ -1082 +1082 @@
-demonstration)
+demonstration#demonstration_n)
@@ -1085 +1085 @@
-of)
+of#of_prep)
@@ -1088 +1088 @@
-rejoycing)))
+rejoycing#rejoicing_n)))
@@ -1091 +1091 @@
-expressed)
+expressed#express_v1)
@@ -1094 +1094 @@
-by)
+by#by_prep)
@@ -1097 +1097 @@
-bonfires)
+bonfires#bonfire_n)
@@ -1099 +1099 @@
-and)
+and#and_conj1)
@@ -1101 +1101 @@
-shoutings))))))
+shoutings#shouting_n))))))
@@ -1109 +1109 @@
-and)
+and#and_conj1)
@@ -1113 +1113 @@
-was)
+was#be_v)
@@ -1116 +1116 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -1119 +1119 @@
-publicly))
+publicly#publicly_adv))
@@ -1122 +1122 @@
-at)
+at#at_prep)
@@ -1125 +1125 @@
-Mass)))
+Mass#mass_n1)))
@@ -1130 +1130 @@
-solemnized)
+solemnized#solemnize_v)
@@ -1133 +1133 @@
-in)
+in#in_prep)
@@ -1136 +1136 @@
-his)
+his#his_adj)
@@ -1138 +1138 @@
-Chapel)
+Chapel#chapel_n)
@@ -1143 +1143 @@
-by)
+by#by_prep)
@@ -1146 +1146 @@
-some)
+some#some_pron)
@@ -1149 +1149 @@
-of)
+of#of_prep)
@@ -1153 +1153 @@
-the)
+the#the_adj)
@@ -1155 +1155 @@
-Spanish)
+Spanish#spanish_adj)
@@ -1157 +1157 @@
-ambassadors))
+ambassadors#ambassador_n))
@@ -1159 +1159 @@
-priests)))))
+priests#priest_n)))))
@@ -1165 +1165 @@
-where))
+where#where_adv))
@@ -1173 +1173 @@
-there))
+there#there_adv))
@@ -1175 +1175 @@
-was)
+was#be_v)
@@ -1178 +1178 @@
-an)
+an#a_adj2)
@@ -1180 +1180 @@
-unexpected)
+unexpected#unexpected_adj)
@@ -1182 +1182 @@
-confluence)
+confluence#confluence_n)
@@ -1185 +1185 @@
-of)
+of#of_prep)
@@ -1188 +1188 @@
-Papists))))
+Papists#NA))))
@@ -1193 +1193 @@
-insomuch)
+insomuch#insomuch_adv)
@@ -1196 +1196 @@
-that)
+that#that_conj)
@@ -1200 +1200 @@
-it))
+it#it_pron))
@@ -1202 +1202 @@
-was)
+was#be_v)
@@ -1205 +1205 @@
-difficult))
+difficult#difficult_adj))
@@ -1208 +1208 @@
-to)
+to#to_prep)
@@ -1210 +1210 @@
-come)
+come#come_v)
@@ -1213 +1213 @@
-near)
+near#near_adj)
@@ -1216 +1216 @@
-his)
+his#his_adj)
@@ -1218 +1218 @@
-person))))))))))
+person#person_n))))))))))
@@ -1224 +1224 @@
-of)
+of#of_prep)
@@ -1227 +1227 @@
-which)))
+which#which_adj)))
@@ -1233 +1233 @@
-the)
+the#the_adj)
@@ -1235 +1235 @@
-Prince))
+Prince#prince_n))
@@ -1238 +1238 @@
-having)
+having#have_v)
@@ -1241 +1241 @@
-intelligence)
+intelligence#intelligence_n)
@@ -1247 +1247 @@
-sent)
+sent#send_v1)
@@ -1250 +1250 @@
-the)
+the#the_adj)
@@ -1252 +1252 @@
-Lords)
+Lords#lord_n)
@@ -1255 +1255 @@
-Macclesfield)
+Macclesfield#NA)
@@ -1257 +1257 @@
-and)
+and#and_conj1)
@@ -1259 +1259 @@
-Delamere)))
+Delamere#NA)))
@@ -1262 +1262 @@
-with)
+with#with_prep)
@@ -1265 +1265 @@
-a)
+a#a_adj2)
@@ -1268 +1268 @@
-very)
+very#very_adj)
@@ -1270 +1270 @@
-sharp))
+sharp#sharp_adj))
@@ -1272 +1272 @@
-Letter)))
+Letter#letter_n1)))
@@ -1275 +1275 @@
-to)
+to#to_prep)
@@ -1278 +1278 @@
-his)
+his#his_adj)
@@ -1280 +1280 @@
-Majesty)))
+Majesty#majesty_n)))
@@ -1285 +1285 @@
-signifying)
+signifying#signify_v)
@@ -1288 +1288 @@
-that)
+that#that_conj)
@@ -1292 +1292 @@
-he))
+he#he_pron))
@@ -1294 +1294 @@
-expected)
+expected#expect_v)
@@ -1297 +1297 @@
-his)
+his#his_adj)
@@ -1299 +1299 @@
-immediate)
+immediate#immediate_adj)
@@ -1301 +1301 @@
-departure)
+departure#departure_n)
@@ -1304 +1304 @@
-to)
+to#to_prep)
@@ -1307 +1307 @@
-Ham))))
+Ham#NA))))
@@ -1313 +1313 @@
-I))
+I#i_pron))
@@ -1315 +1315 @@
-suppose)
+suppose#suppose_v)
@@ -1318 +1318 @@
-in)
+in#in_prep)
@@ -1321 +1321 @@
-Essex)))))))))
+Essex#NA)))))))))
@@ -1331 +1331 @@
-nor)
+nor#nor_conj1)
@@ -1333 +1333 @@
-were)
+were#be_v)
@@ -1336 +1336 @@
-the)
+the#the_adj)
@@ -1338 +1338 @@
-Peers))
+Peers#peer_n))
@@ -1341 +1341 @@
-wanting)
+wanting#wanting_adj)
@@ -1344 +1344 @@
-in)
+in#in_prep)
@@ -1347 +1347 @@
-expressing)
+expressing#express_v1)
@@ -1350 +1350 @@
-their)
+their#their_adj)
@@ -1352 +1352 @@
-resentment)
+resentment#resentment_n)
@@ -1355 +1355 @@
-of)
+of#of_prep)
@@ -1358 +1358 @@
-that)
+that#that_pron)
@@ -1360 +1360 @@
-action))))
+action#action_n))))
@@ -1363 +1363 @@
-in)
+in#in_prep)
@@ -1367 +1367 @@
-so)
+so#so_adv)
@@ -1369 +1369 @@
-improper))
+improper#improper_adj))
@@ -1371 +1371 @@
-a)
+a#a_adj2)
@@ -1373 +1373 @@
-conjuncture))))))
+conjuncture#conjuncture_n))))))
@@ -1381 +1381 @@
-but)
+but#but_prep)
@@ -1385 +1385 @@
-the)
+the#the_adj)
@@ -1387 +1387 @@
-King))
+King#king_n))
@@ -1389 +1389 @@
-desiring)
+desiring#desire_v)
@@ -1392 +1392 @@
-to)
+to#to_prep)
@@ -1394 +1394 @@
-know)
+know#know_v)
@@ -1397 +1397 @@
-if)
+if#if_conj)
@@ -1403 +1403 @@
-he))
+he#he_pron))
@@ -1405 +1405 @@
-might)
+might#may_v1)
@@ -1407 +1407 @@
-not)
+not#not_adv)
@@ -1409 +1409 @@
-be)
+be#be_v)
@@ -1411 +1411 @@
-permitted)
+permitted#permit_v)
@@ -1414 +1414 @@
-to)
+to#to_prep)
@@ -1416 +1416 @@
-retire)
+retire#retire_v)
@@ -1421 +1421 @@
-whither))
+whither#whither_adv))
@@ -1431 +1431 @@
-he))
+he#he_pron))
@@ -1433 +1433 @@
-pleased)))))))))
+pleased#please_v)))))))))
@@ -1438 +1438 @@
-the)
+the#the_adj)
@@ -1440 +1440 @@
-Lords))
+Lords#lord_n))
@@ -1443 +1443 @@
-after)
+after#after_adv)
@@ -1446 +1446 @@
-some)
+some#some_pron)
@@ -1448 +1448 @@
-debate)))
+debate#debate_n1)))
@@ -1450 +1450 @@
-assented)
+assented#assent_v)
@@ -1453 +1453 @@
-to)
+to#to_prep)
@@ -1456 +1456 @@
-his)
+his#his_adj)
@@ -1458 +1458 @@
-liberty)))
+liberty#liberty_n)))
@@ -1464 +1464 @@
-whereupon))
+whereupon#whereupon_adv))
@@ -1472 +1472 @@
-this)
+this#this_pron)
@@ -1474 +1474 @@
-morning))
+morning#morning_n))
@@ -1477 +1477 @@
-he))
+he#he_pron))
@@ -1479 +1479 @@
-went)
+went#go_v)
@@ -1482 +1482 @@
-in)
+in#in_prep)
@@ -1485 +1485 @@
-his)
+his#his_adj)
@@ -1487 +1487 @@
-barge)))
+barge#barge_n)))
@@ -1490 +1490 @@
-to)
+to#to_prep)
@@ -1493 +1493 @@
-Gravesend)))
+Gravesend#NA)))
@@ -1496 +1496 @@
-attended)
+attended#attend_v)
@@ -1501 +1501 @@
-in)
+in#in_prep)
@@ -1504 +1504 @@
-boats)))
+boats#boat_n)))
@@ -1509 +1509 @@
-by)
+by#by_prep)
@@ -1513 +1513 @@
-about)
+about#about_adv)
@@ -1515 +1515 @@
-a_hundred))
+a_hundred#NA))
@@ -1518 +1518 @@
-of)
+of#of_prep)
@@ -1522 +1522 @@
-the)
+the#the_adj)
@@ -1524 +1524 @@
-Princes))
+Princes#prince_n))
@@ -1526 +1526 @@
-Foot))))))
+Foot#foot_n))))))
@@ -1534 +1534 @@
-two_hundred)
+two_hundred#NA)
@@ -1537 +1537 @@
-of)
+of#of_prep)
@@ -1540 +1540 @@
-his)
+his#his_adj)
@@ -1542 +1542 @@
-Horse)))))
+Horse#horse_n)))))
@@ -1544 +1544 @@
-being)
+being#be_v)
@@ -1547 +1547 @@
-likewise))
+likewise#likewise_adv))
@@ -1549 +1549 @@
-sent)
+sent#send_v1)
@@ -1552 +1552 @@
-to)
+to#to_prep)
@@ -1555,3 +1555,3 @@
-meet)
-(CONJ
-and)
+meet#meet_v)
+(CONJ
+and#and_conj1)
@@ -1559 +1559 @@
-convey))
+convey#convey_v1))
@@ -1562 +1562 @@
-him))
+him#him_pron))
@@ -1565 +1565 @@
-to)
+to#to_prep)
@@ -1568 +1568 @@
-Rochester)))))
+Rochester#NA)))))
@@ -1576 +1576 @@
-all)
+all#all_adj)
@@ -1578 +1578 @@
-the)
+the#the_adj)
@@ -1580 +1580 @@
-Kings))
+Kings#king_n))
@@ -1582 +1582 @@
-soldiers))
+soldiers#soldier_n))
@@ -1587 +1587 @@
-if)
+if#if_conj)
@@ -1594 +1594 @@
-they))
+they#they_pron))
@@ -1596 +1596 @@
-may)
+may#may_v1)
@@ -1598 +1598 @@
-be)
+be#be_v)
@@ -1601 +1601 @@
-so))
+so#so_adv))
@@ -1603 +1603 @@
-accounted))))
+accounted#account_v))))
@@ -1607 +1607 @@
-being)
+being#be_v)
@@ -1609 +1609 @@
-commanded)
+commanded#command_v)
@@ -1612 +1612 @@
-to)
+to#to_prep)
@@ -1615 +1615 @@
-places)
+places#place_n.II-VII)
@@ -1618 +1618 @@
-assigned)
+assigned#assign_v)
@@ -1621 +1621 @@
-by)
+by#by_prep)
@@ -1624 +1624 @@
-the)
+the#the_adj)
@@ -1626 +1626 @@
-Prince)))
+Prince#prince_n)))
@@ -1631 +1631 @@
-some)
+some#some_pron)
@@ -1633 +1633 @@
-distance)
+distance#distance_n)
@@ -1636 +1636 @@
-from)
+from#from_prep)
@@ -1639 +1639 @@
-the)
+the#the_adj)
@@ -1641 +1641 @@
-City)))))))))
+City#NA)))))))))
@@ -1648 +1648 @@
-The)
+The#the_adj)
@@ -1650 +1650 @@
-Dutchmen))
+Dutchmen#NA))
@@ -1652 +1652 @@
-mounting)
+mounting#mount_v)
@@ -1655 +1655 @@
-the)
+the#the_adj)
@@ -1657 +1657 @@
-guard))
+guard#guard_n))
@@ -1660 +1660 @@
-at)
+at#at_prep)
@@ -1664 +1664 @@
-Whitehall))
+Whitehall#NA))
@@ -1667 +1667 @@
-and)
+and#and_conj1)
@@ -1670 +1670 @@
-the)
+the#the_adj)
@@ -1672 +1672 @@
-other)
+other#other_adj)
@@ -1674 +1674 @@
-usual)
+usual#usual_adj)
@@ -1676 +1676 @@
-places))))))))))
+places#place_n.II-VII))))))))))
@@ -1685 +1685 @@
-The)
+The#the_adj)
@@ -1687 +1687 @@
-Earl)
+Earl#earl_n)
@@ -1690 +1690 @@
-of)
+of#of_prep)
@@ -1693 +1693 @@
-Feversham)))
+Feversham#NA)))
@@ -1697 +1697 @@
-who))
+who#who_pron))
@@ -1704 +1704 @@
-went)
+went#go_v)
@@ -1708 +1708 @@
-from)
+from#from_prep)
@@ -1711 +1711 @@
-the)
+the#the_adj)
@@ -1713 +1713 @@
-King)))
+King#king_n)))
@@ -1716 +1716 @@
-to)
+to#to_prep)
@@ -1719 +1719 @@
-Windsor))))
+Windsor#NA))))
@@ -1722 +1722 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -1727 +1727 @@
-to)
+to#to_prep)
@@ -1729 +1729 @@
-invite)
+invite#invite_v)
@@ -1732 +1732 @@
-the)
+the#the_adj)
@@ -1734 +1734 @@
-Prince))
+Prince#prince_n))
@@ -1737 +1737 @@
-hither))))))
+hither#hither_adv))))))
@@ -1741 +1741 @@
-was)
+was#be_v)
@@ -1743 +1743 @@
-detained)
+detained#detain_v)
@@ -1746 +1746 @@
-for)
+for#for_prep)
@@ -1749 +1749 @@
-High)
+High#high_adj)
@@ -1751 +1751 @@
-Treason)))
+Treason#treason_n)))
@@ -1760 +1760 @@
-Amongst)
+Amongst#amongst_prep)
@@ -1763 +1763 @@
-the)
+the#the_adj)
@@ -1765 +1765 @@
-many)
+many#many_adj)
@@ -1767 +1767 @@
-persons)
+persons#person_n)
@@ -1771 +1771 @@
-now))
+now#now_adv))
@@ -1774 +1774 @@
-in)
+in#in_prep)
@@ -1777 +1777 @@
-custody))))))
+custody#custody_n))))))
@@ -1782 +1782 @@
-Sir)
+Sir#NA)
@@ -1784 +1784 @@
-Roger)
+Roger#NA)
@@ -1786 +1786 @@
-le)
+le#NA)
@@ -1788 +1788 @@
-Strange))
+Strange#NA))
@@ -1790 +1790 @@
-is)
+is#be_v)
@@ -1792 +1792 @@
-not)
+not#not_adv)
@@ -1795 +1795 @@
-of)
+of#of_prep)
@@ -1798 +1798 @@
-the)
+the#the_adj)
@@ -1800 +1800 @@
-least)
+least#least_adj)
@@ -1802 +1802 @@
-eminency)))
+eminency#eminency_n)))
@@ -1811 +1811 @@
-I))
+I#i_pron))
@@ -1813 +1813 @@
-could)
+could#can_v1)
@@ -1815 +1815 @@
-not)
+not#not_adv)
@@ -1817 +1817 @@
-write)
+write#write_v)
@@ -1820 +1820 @@
-to)
+to#to_prep)
@@ -1823 +1823 @@
-you)))
+you#you_pron)))
@@ -1826 +1826 @@
-last)
+last#last_adv)
@@ -1828 +1828 @@
-Post))
+Post#post_n2))
@@ -1831 +1831 @@
-as)
+as#as_adv)
@@ -1842 +1842 @@
-I))
+I#i_pron))
@@ -1844 +1844 @@
-promised))))
+promised#promise_v))))
@@ -1852 +1852 @@
-and)
+and#and_conj1)
@@ -1856 +1856 @@
-can)
+can#can_v1)
@@ -1859 +1859 @@
-now))
+now#now_adv))
@@ -1861 +1861 @@
-only)
+only#only_adj)
@@ -1863 +1863 @@
-add)
+add#add_v)
@@ -1866 +1866 @@
-that)
+that#that_conj)
@@ -1870 +1870 @@
-I))
+I#i_pron))
@@ -1872 +1872 @@
-am)
+am#be_v)
@@ -1875 +1875 @@
-Sir))
+Sir#sir_n))
@@ -1880 +1880 @@
-Yours)
+Yours#yours_pron)
@@ -1882 +1882 @@
-&c.)))))
+&c.#et-cetera_n)))))
