--- eoxinden-1658-e3-p1.psd.vert	2025-09-10 13:51:11
+++ eoxinden-1658-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -17 +17 @@
-Deare)
+Deare#dear_adj1)
@@ -19 +19 @@
-Mother))
+Mother#mother_n1))
@@ -24 +24 @@
-This))
+This#this_pron))
@@ -26 +26 @@
-is)
+is#be_v)
@@ -29 +29 @@
-to)
+to#to_prep)
@@ -31 +31 @@
-let)
+let#let_v1)
@@ -35 +35 @@
-you))
+you#you_pron))
@@ -37 +37 @@
-know)
+know#know_v)
@@ -40 +40 @@
-that)
+that#that_conj)
@@ -44 +44 @@
-wee))
+wee#we_pron))
@@ -46 +46 @@
-got)
+got#get_v)
@@ -49 +49 @@
-very)
+very#very_adj)
@@ -51 +51 @@
-well))
+well#well_adv))
@@ -54 +54 @@
-to)
+to#to_prep)
@@ -57 +57 @@
-Sandwich)))))))
+Sandwich#NA)))))))
@@ -65 +65 @@
-my)
+my#my_adj)
@@ -67 +67 @@
-humble)
+humble#humble_adj)
@@ -69 +69 @@
-thanks)
+thanks#thank_n)
@@ -72 +72 @@
-to)
+to#to_prep)
@@ -75 +75 @@
-you)))
+you#you_pron)))
@@ -78 +78 @@
-for)
+for#for_prep)
@@ -81 +81 @@
-all)
+all#all_adj)
@@ -83 +83 @@
-those)
+those#those_pron)
@@ -85 +85 @@
-presents)
+presents#present_n2)
@@ -96 +96 @@
-I))
+I#i_pron))
@@ -98 +98 @@
-have)
+have#have_v)
@@ -100 +100 @@
-recived)
+recived#receive_v)
@@ -103 +103 @@
-by)
+by#by_prep)
@@ -106 +106 @@
-Mary)))))))
+Mary#NA)))))))
@@ -115 +115 @@
-Deare)
+Deare#dear_adj1)
@@ -117 +117 @@
-Mother))
+Mother#mother_n1))
@@ -120 +120 @@
-you))
+you#you_pron))
@@ -122 +122 @@
-load)
+load#load_v)
@@ -125 +125 @@
-me))
+me#me_pron1))
@@ -128 +128 @@
-with)
+with#with_prep)
@@ -131 +131 @@
-varieties)
+varieties#variety_n)
@@ -134 +134 @@
-of)
+of#of_prep)
@@ -137 +137 @@
-favours))))))
+favours#favour_n))))))
@@ -143 +143 @@
-and)
+and#and_conj1)
@@ -146 +146 @@
-all)
+all#all_adj)
@@ -151 +151 @@
-that)
+that#that_conj)
@@ -157 +157 @@
-I))
+I#i_pron))
@@ -159 +159 @@
-can)
+can#can_v1)
@@ -161 +161 @@
-doe)
+doe#do_v)
@@ -164 +164 @@
-in)
+in#in_prep)
@@ -167 +167 @@
-requitall))))))
+requitall#requital_n))))))
@@ -169 +169 @@
-is)
+is#be_v)
@@ -172 +172 @@
-to)
+to#to_prep)
@@ -174 +174 @@
-give)
+give#give_v)
@@ -177 +177 @@
-you))
+you#you_pron))
@@ -180 +180 @@
-thanks)))
+thanks#thank_n)))
@@ -189 +189 @@
-Dick))
+Dick#NA))
@@ -192 +192 @@
-as)
+as#as_adv)
@@ -194 +194 @@
-soone)
+soone#soon_adv)
@@ -197 +197 @@
-as)
+as#as_adv)
@@ -208 +208 @@
-he))
+he#he_pron))
@@ -210 +210 @@
-awaked)
+awaked#awake_v)
@@ -213 +213 @@
-today))))))
+today#today_adv))))))
@@ -215 +215 @@
-aske)
+aske#ask_v)
@@ -218 +218 @@
-to)
+to#to_prep)
@@ -220 +220 @@
-goe)
+goe#go_v)
@@ -223 +223 @@
-home))
+home#home_n1))
@@ -226 +226 @@
-to)
+to#to_prep)
@@ -229 +229 @@
-his)
+his#his_adj)
@@ -231 +231 @@
-grandmother)))))
+grandmother#grandmother_n)))))
@@ -237 +237 @@
-and)
+and#and_conj1)
@@ -241 +241 @@
-did)
+did#do_v)
@@ -244 +244 @@
-much))
+much#much_adj))
@@ -246 +246 @@
-joy)
+joy#joy_v)
@@ -249 +249 @@
-upon)
+upon#upon_prep)
@@ -253 +253 @@
-Mary))
+Mary#NA))
@@ -255 +255 @@
-coming)))
+coming#come_v)))
@@ -258 +258 @@
-thinking)
+thinking#think_v2)
@@ -265 +265 @@
-he))
+he#he_pron))
@@ -267 +267 @@
-should)
+should#shall_v)
@@ -269 +269 @@
-goe)
+goe#go_v)
@@ -272 +272 @@
-home))
+home#home_n1))
@@ -275 +275 @@
-with)
+with#with_prep)
@@ -278 +278 @@
-her))))))
+her#her_pron1))))))
@@ -286 +286 @@
-and)
+and#and_conj1)
@@ -291 +291 @@
-many)
+many#many_adj)
@@ -293 +293 @@
-times)
+times#time_n)
@@ -296 +296 @@
-when)
+when#when_adv)
@@ -303 +303 @@
-he))
+he#he_pron))
@@ -305 +305 @@
-is)
+is#be_v)
@@ -308 +308 @@
-earnestly))
+earnestly#earnestly_adv))
@@ -311 +311 @@
-a)
+a#a_prep1)
@@ -314 +314 @@
-talking)))))))
+talking#talking_n)))))))
@@ -316 +316 @@
-calls)
+calls#call_v)
@@ -320 +320 @@
-me))
+me#me_pron1))
@@ -323 +323 @@
-grandmother)))
+grandmother#grandmother_n)))
@@ -327 +327 @@
-he))
+he#he_pron))
@@ -329 +329 @@
-is)
+is#be_v)
@@ -332 +332 @@
-so)
+so#so_adv)
@@ -334 +334 @@
-mindfull)
+mindfull#mindful_adj)
@@ -337 +337 @@
-of)
+of#of_prep)
@@ -340 +340 @@
-you)))))
+you#you_pron)))))
@@ -349 +349 @@
-You))
+You#you_pron))
@@ -351 +351 @@
-shall)
+shall#shall_v)
@@ -353 +353 @@
-have)
+have#have_v)
@@ -356 +356 @@
-your)
+your#your_pron)
@@ -358 +358 @@
-haire)
+haire#hair_n)
@@ -360 +360 @@
-locket))
+locket#NA))
@@ -363 +363 @@
-as)
+as#as_adv)
@@ -365 +365 @@
-soone)
+soone#soon_adv)
@@ -368 +368 @@
-as)
+as#as_adv)
@@ -371 +371 @@
-posible)
+posible#possible_adj)
@@ -382 +382 @@
-I))
+I#i_pron))
@@ -384 +384 @@
-can)
+can#can_v1)
@@ -394 +394 @@
-and)
+and#and_conj1)
@@ -397 +397 @@
-Deare)
+Deare#dear_adj1)
@@ -399 +399 @@
-mother))
+mother#mother_n1))
@@ -402 +402 @@
-anythinge)
+anythinge#anything_pron)
@@ -413 +413 @@
-I))
+I#i_pron))
@@ -415 +415 @@
-can)
+can#can_v1)
@@ -417 +417 @@
-doe))))
+doe#do_v))))
@@ -422 +422 @@
-pray))
+pray#pray_v))
@@ -424 +424 @@
-command))
+command#command_v))
@@ -430 +430 @@
-and)
+and#and_conj1)
@@ -432 +432 @@
-doe)
+doe#do_v)
@@ -434 +434 @@
-not)
+not#not_adv)
@@ -438 +438 @@
-soe)
+soe#so_adv)
@@ -440 +440 @@
-much))
+much#much_adj))
@@ -443 +443 @@
-as)))
+as#as_adv)))
@@ -445 +445 @@
-harbour)
+harbour#harbour_v)
@@ -448 +448 @@
-shuch)
+shuch#such_adj)
@@ -450 +450 @@
-a)
+a#a_adj2)
@@ -452 +452 @@
-thought)
+thought#thought_n)
@@ -455 +455 @@
-as)
+as#as_adv)
@@ -458 +458 @@
-to)
+to#to_prep)
@@ -460 +460 @@
-thinke)
+thinke#think_v2)
@@ -463 +463 @@
-that)
+that#that_conj)
@@ -467 +467 @@
-any)
+any#any_adj)
@@ -469 +469 @@
-thinge)
+thinge#thing_n1)
@@ -480 +480 @@
-I))
+I#i_pron))
@@ -482 +482 @@
-can)
+can#can_v1)
@@ -484 +484 @@
-doe)
+doe#do_v)
@@ -487 +487 @@
-for)
+for#for_prep)
@@ -490 +490 @@
-you))))))
+you#you_pron))))))
@@ -492 +492 @@
-is)
+is#be_v)
@@ -495 +495 @@
-any)
+any#any_adj)
@@ -497 +497 @@
-trouble)))))))
+trouble#trouble_n)))))))
@@ -505 +505 @@
-for)
+for#for_prep)
@@ -508 +508 @@
-it))
+it#it_pron))
@@ -510 +510 @@
-is)
+is#be_v)
@@ -513 +513 @@
-my)
+my#my_adj)
@@ -515 +515 @@
-only)
+only#only_adj)
@@ -517 +517 @@
-ambition))
+ambition#ambition_n))
@@ -520 +520 @@
-to)
+to#to_prep)
@@ -522 +522 @@
-find)
+find#find_v)
@@ -524 +524 @@
-out)
+out#out_adv)
@@ -527 +527 @@
-some)
+some#some_pron)
@@ -529 +529 @@
-way)
+way#way_n1)
@@ -533 +533 @@
-wherein))
+wherein#wherein_adv))
@@ -541 +541 @@
-I))
+I#i_pron))
@@ -543 +543 @@
-may)
+may#may_v1)
@@ -545 +545 @@
-serve)
+serve#serve_v1)
@@ -548 +548 @@
-you)))))))
+you#you_pron)))))))
@@ -554 +554 @@
-and)
+and#and_conj1)
@@ -557 +557 @@
-any)
+any#any_adj)
@@ -559 +559 @@
-time)
+time#time_n)
@@ -563 +563 @@
-when))
+when#when_adv))
@@ -571 +571 @@
-you))
+you#you_pron))
@@ -573 +573 @@
-will)
+will#will_v1)
@@ -575 +575 @@
-have)
+have#have_v)
@@ -578 +578 @@
-any)
+any#any_adj)
@@ -580 +580 @@
-buttons)))))
+buttons#button_n)))))
@@ -585 +585 @@
-pray))
+pray#pray_v))
@@ -587 +587 @@
-let)
+let#let_v1)
@@ -591 +591 @@
-me))
+me#me_pron1))
@@ -593 +593 @@
-desire)
+desire#desire_v)
@@ -596 +596 @@
-you))
+you#you_pron))
@@ -601 +601 @@
-to)
+to#to_prep)
@@ -603 +603 @@
-send)
+send#send_v1)
@@ -606 +606 @@
-to)
+to#to_prep)
@@ -609 +609 @@
-me)))))
+me#me_pron1)))))
@@ -618 +618 @@
-I))
+I#i_pron))
@@ -620 +620 @@
-desire)
+desire#desire_v)
@@ -623 +623 @@
-you))
+you#you_pron))
@@ -626 +626 @@
-to)
+to#to_prep)
@@ -628 +628 @@
-pardon)
+pardon#pardon_v)
@@ -631 +631 @@
-my)
+my#my_adj)
@@ -633 +633 @@
-great)
+great#great_adj)
@@ -635 +635 @@
-incivility))
+incivility#incivility_n))
@@ -638 +638 @@
-in)
+in#in_prep)
@@ -641 +641 @@
-coming)
+coming#come_v)
@@ -644 +644 @@
-away))
+away#away_adv))
@@ -647 +647 @@
-in)
+in#in_prep)
@@ -650 +650 @@
-shuch)
+shuch#such_adj)
@@ -652 +652 @@
-a)
+a#a_adj2)
@@ -654 +654 @@
-strang)
+strang#strange_adj)
@@ -656 +656 @@
-maner)))
+maner#manner_n)))
@@ -659 +659 @@
-without)
+without#without_adv)
@@ -662 +662 @@
-giving)
+giving#give_v)
@@ -665 +665 @@
-you))
+you#you_pron))
@@ -668 +668 @@
-thanks)
+thanks#thank_n)
@@ -671 +671 @@
-for)
+for#for_prep)
@@ -674 +674 @@
-all)
+all#all_adj)
@@ -676 +676 @@
-your)
+your#your_pron)
@@ -678 +678 @@
-kindness)
+kindness#kindness_n)
@@ -681 +681 @@
-to)
+to#to_prep)
@@ -685 +685 @@
-me))
+me#me_pron1))
@@ -688 +688 @@
-and)
+and#and_conj1)
@@ -691 +691 @@
-mine)))))
+mine#mine_adj)))))
@@ -694 +694 @@
-while)
+while#while_adv)
@@ -701 +701 @@
-I))
+I#i_pron))
@@ -703 +703 @@
-were)
+were#be_v)
@@ -706 +706 @@
-with)
+with#with_prep)
@@ -709 +709 @@
-you))))))))))))))
+you#you_pron))))))))))))))
@@ -717 +717 @@
-but)
+but#but_prep)
@@ -720 +720 @@
-really))
+really#really_adv2))
@@ -724 +724 @@
-my)
+my#my_adj)
@@ -726 +726 @@
-Father))
+Father#father_n))
@@ -728 +728 @@
-being)
+being#be_v)
@@ -731 +731 @@
-soe))
+soe#so_adv))
@@ -734 +734 @@
-out)
+out#out_adv)
@@ -736 +736 @@
-of)
+of#of_prep)
@@ -739 +739 @@
-humer))))
+humer#humour_n))))
@@ -741 +741 @@
-put)
+put#put_v)
@@ -745 +745 @@
-me))
+me#me_pron1))
@@ -747 +747 @@
-to)
+to#to_prep)
@@ -749 +749 @@
-be)
+be#be_v)
@@ -752 +752 @@
-strangly)
+strangly#strangely_adv)
@@ -754 +754 @@
-forgetfull)))
+forgetfull#forgetful_adj)))
@@ -763 +763 @@
-My)
+My#my_adj)
@@ -765 +765 @@
-sister)
+sister#sister_n)
@@ -768 +768 @@
-Wood)))
+Wood#NA)))
@@ -770 +770 @@
-had)
+had#have_v)
@@ -773 +773 @@
-desires)
+desires#desire_n)
@@ -777 +777 @@
-to)
+to#to_prep)
@@ -779 +779 @@
-kiss)
+kiss#kiss_v)
@@ -782 +782 @@
-your)
+your#your_pron)
@@ -784 +784 @@
-hand))
+hand#hand_n))
@@ -787 +787 @@
-with)
+with#with_prep)
@@ -791 +791 @@
-a)
+a#a_adj2)
@@ -793 +793 @@
-few))
+few#few_adj))
@@ -795 +795 @@
-lines))))
+lines#line_n2))))
@@ -800 +800 @@
-and)
+and#and_conj1)
@@ -803 +803 @@
-to)
+to#to_prep)
@@ -805 +805 @@
-give)
+give#give_v)
@@ -808 +808 @@
-you))
+you#you_pron))
@@ -811 +811 @@
-thanks))
+thanks#thank_n))
@@ -814 +814 @@
-under)
+under#under_prep)
@@ -817 +817 @@
-her)
+her#her_pron2)
@@ -819 +819 @@
-one)
+one#own_adj)
@@ -821 +821 @@
-hand)))))))
+hand#hand_n)))))))
@@ -829 +829 @@
-but)
+but#but_prep)
@@ -833 +833 @@
-she))
+she#she_pron))
@@ -835 +835 @@
-being)
+being#be_v)
@@ -837 +837 @@
-prevented)
+prevented#prevent_v)
@@ -840 +840 @@
-by)
+by#by_prep)
@@ -844 +844 @@
-your)
+your#your_pron)
@@ -846 +846 @@
-cosen)
+cosen#cousin_n)
@@ -849 +849 @@
-denward@))
+denward@#NA))
@@ -851 +851 @@
-@'s))
+@'s#NA))
@@ -854 +854 @@
-coming)
+coming#come_v)
@@ -856 +856 @@
-in)))))
+in#in_adv)))))
@@ -861 +861 @@
-she))
+she#she_pron))
@@ -863 +863 @@
-desires)
+desires#desire_v)
@@ -870 +870 @@
-this))
+this#this_pron))
@@ -872 +872 @@
-may)
+may#may_v1)
@@ -874 +874 @@
-be)
+be#be_v)
@@ -876 +876 @@
-axcepted)
+axcepted#accept_v)
@@ -879 +879 @@
-with)
+with#with_prep)
@@ -882 +882 @@
-you)))))
+you#you_pron)))))
@@ -891 +891 @@
-she))
+she#she_pron))
@@ -893 +893 @@
-hath)
+hath#have_v)
@@ -895 +895 @@
-sent)
+sent#send_v1)
@@ -898 +898 @@
-you))
+you#you_pron))
@@ -902 +902 @@
-a)
+a#a_adj2)
@@ -904 +904 @@
-few))
+few#few_adj))
@@ -906 +906 @@
-goosberrys))
+goosberrys#gooseberry_n))
@@ -915 +915 @@
-She)
+She#she_pron)
@@ -917 +917 @@
-and)
+and#and_conj1)
@@ -919 +919 @@
-I))
+I#i_pron))
@@ -921 +921 @@
-am)
+am#be_v)
@@ -924 +924 @@
-very)
+very#very_adj)
@@ -926 +926 @@
-sory)
+sory#sorry_adj)
@@ -929 +929 @@
-that)
+that#that_conj)
@@ -933 +933 @@
-the)
+the#the_adj)
@@ -935 +935 @@
-towne))
+towne#town_n))
@@ -937 +937 @@
-will)
+will#will_v1)
@@ -939 +939 @@
-afoord)
+afoord#afford_v)
@@ -942 +942 @@
-nothing)
+nothing#nothing_pron)
@@ -945 +945 @@
-worth)
+worth#worth_adj)
@@ -948 +948 @@
-your)
+your#your_pron)
@@ -950 +950 @@
-acceptance)))))))
+acceptance#acceptance_n)))))))
@@ -959 +959 @@
-My)
+My#my_adj)
@@ -961 +961 @@
-humble)
+humble#humble_adj)
@@ -963 +963 @@
-Duty)
+Duty#duty_n)
@@ -966 +966 @@
-to)
+to#to_prep)
@@ -969 +969 @@
-you))))
+you#you_pron))))
@@ -972 +972 @@
-and)
+and#and_conj1)
@@ -975 +975 @@
-Dickes))
+Dickes#NA))
@@ -977 +977 @@
-also))
+also#also_adv))
@@ -985 +985 @@
-and)
+and#and_conj1)
@@ -988 +988 @@
-I))
+I#i_pron))
@@ -990 +990 @@
-conclude)
+conclude#conclude_v)
@@ -993 +993 @@
-at)
+at#at_prep)
@@ -996 +996 @@
-present)))
+present#present_adj)))
@@ -999 +999 @@
-Your)
+Your#your_pron)
@@ -1001 +1001 @@
-obedient)
+obedient#obedient_adj)
@@ -1003 +1003 @@
-Daughter)
+Daughter#daughter_n)
@@ -1006 +1006 @@
-Elizabeth)
+Elizabeth#NA)
@@ -1008 +1008 @@
-Oxinden))))
+Oxinden#NA))))
@@ -1018 +1018 @@
-I))
+I#i_pron))
@@ -1020 +1020 @@
-desire)
+desire#desire_v)
@@ -1023 +1023 @@
-you))
+you#you_pron))
@@ -1026 +1026 @@
-to)
+to#to_prep)
@@ -1028 +1028 @@
-send)
+send#send_v1)
@@ -1031 +1031 @@
-that)
+that#that_pron)
@@ -1033 +1033 @@
-bundle))
+bundle#bundle_n))
@@ -1036 +1036 @@
-when)
+when#when_adv)
@@ -1043 +1043 @@
-you))
+you#you_pron))
@@ -1045 +1045 @@
-heare)
+heare#hear_v)
@@ -1048 +1048 @@
-of)
+of#of_prep)
@@ -1051 +1051 @@
-anybody)
+anybody#anybody_pron)
@@ -1054 +1054 @@
-come)
+come#come_v)
@@ -1057 +1057 @@
-to)
+to#to_prep)
@@ -1060 +1060 @@
-this)
+this#this_pron)
@@ -1062 +1062 @@
-Towne))))))))))
+Towne#town_n))))))))))
@@ -1071 +1071 @@
-Adams)
+Adams#NA)
@@ -1074 +1074 @@
-of)
+of#of_prep)
@@ -1077 +1077 @@
-denhill))))
+denhill#NA))))
@@ -1079 +1079 @@
-comes)
+comes#come_v)
@@ -1082 +1082 @@
-next)
+next#next_adj)
@@ -1084 +1084 @@
-weeke))
+weeke#week_n))
@@ -1092 +1092 @@
-and)
+and#and_conj1)
@@ -1095 +1095 @@
-John)
+John#NA)
@@ -1097 +1097 @@
-Say))
+Say#NA))
@@ -1099 +1099 @@
-would)
+would#will_v1)
@@ -1101 +1101 @@
-bring)
+bring#bring_v)
@@ -1104 +1104 @@
-it))
+it#it_pron))
@@ -1107 +1107 @@
-if)
+if#if_conj)
@@ -1114 +1114 @@
-he))
+he#he_pron))
@@ -1116 +1116 @@
-could)
+could#can_v1)
@@ -1118 +1118 @@
-have)
+have#have_v)
@@ -1122 +1122 @@
-it))
+it#it_pron))
@@ -1124 +1124 @@
-brought)
+brought#bring_v)
@@ -1127 +1127 @@
-to)
+to#to_prep)
@@ -1130 +1130 @@
-his)
+his#his_adj)
@@ -1132 +1132 @@
-house)))))))
+house#house_n1)))))))
@@ -1140 +1140 @@
-E.)
+E.#NA)
@@ -1142 +1142 @@
-O.))
+O.#NA))
