--- eoxinden-1665-e3-p1.psd.vert	2025-09-10 13:51:11
+++ eoxinden-1665-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -17 +17 @@
-Deare)
+Deare#dear_adj1)
@@ -19 +19 @@
-Mother))
+Mother#mother_n1))
@@ -24 +24 @@
-I))
+I#i_pron))
@@ -26 +26 @@
-recived)
+recived#receive_v)
@@ -30 +30 @@
-your)
+your#your_pron)
@@ -32 +32 @@
-letter)
+letter#letter_n1)
@@ -37 +37 @@
-that)
+that#that_conj)
@@ -42 +42 @@
-came)
+came#come_v)
@@ -45 +45 @@
-by)
+by#by_prep)
@@ -48 +48 @@
-Mr.)
+Mr.#NA)
@@ -50 +50 @@
-Lunn))))))
+Lunn#NA))))))
@@ -53 +53 @@
-and)
+and#and_conj1)
@@ -56 +56 @@
-your)
+your#your_pron)
@@ -58 +58 @@
-other)
+other#other_adj)
@@ -61 +61 @@
-before)))))
+before#before_adv)))))
@@ -69 +69 @@
-and)
+and#and_conj1)
@@ -72 +72 @@
-in)
+in#in_prep)
@@ -75 +75 @@
-obeidence)
+obeidence#obedience_n)
@@ -78 +78 @@
-to)
+to#to_prep)
@@ -81 +81 @@
-your)
+your#your_pron)
@@ -83 +83 @@
-commands)))))
+commands#command_n)))))
@@ -87 +87 @@
-my)
+my#my_adj)
@@ -89 +89 @@
-sister)
+sister#sister_n)
@@ -92 +92 @@
-Wood)))
+Wood#NA)))
@@ -95 +95 @@
-and)
+and#and_conj1)
@@ -98 +98 @@
-myself))))
+myself#myself_pron))))
@@ -100 +100 @@
-went)
+went#go_v)
@@ -103 +103 @@
-a)
+a#a_adj2)
@@ -105 +105 @@
-whole)
+whole#whole_adj)
@@ -107 +107 @@
-week))
+week#week_n))
@@ -110 +110 @@
-to)
+to#to_prep)
@@ -112 +112 @@
-meet)
+meet#meet_v)
@@ -115 +115 @@
-the)
+the#the_adj)
@@ -117 +117 @@
-coaches))))
+coaches#coach_n))))
@@ -123 +123 @@
-but)
+but#but_prep)
@@ -127 +127 @@
-were)
+were#be_v)
@@ -129 +129 @@
-deprived)
+deprived#deprive_v)
@@ -132 +132 @@
-of)
+of#of_prep)
@@ -135 +135 @@
-the)
+the#the_adj)
@@ -137 +137 @@
-hapinesse)
+hapinesse#happiness_n)
@@ -140 +140 @@
-of)
+of#of_prep)
@@ -143 +143 @@
-seeing)
+seeing#see_v)
@@ -146 +146 @@
-you))))))
+you#you_pron))))))
@@ -155 +155 @@
-Your)
+Your#your_pron)
@@ -157 +157 @@
-letter)
+letter#letter_n1)
@@ -162 +162 @@
-that)
+that#that_conj)
@@ -167 +167 @@
-came)
+came#come_v)
@@ -170 +170 @@
-then)))))
+then#then_adv)))))
@@ -172 +172 @@
-had)
+had#have_v)
@@ -174 +174 @@
-been)
+been#be_v)
@@ -177 +177 @@
-twice))
+twice#twice_adv))
@@ -179 +179 @@
-broke)
+broke#break_v)
@@ -182 +182 @@
-open))
+open#open_adj))
@@ -191 +191 @@
-They)
+They#they_pron)
@@ -196 +196 @@
-that)
+that#that_conj)
@@ -201 +201 @@
-did)
+did#do_v)
@@ -207 +207 @@
-it))
+it#it_pron))
@@ -209 +209 @@
-is)
+is#be_v)
@@ -212 +212 @@
-posible)))
+posible#possible_adj)))
@@ -214 +214 @@
-hath)
+hath#have_v)
@@ -216 +216 @@
-been)
+been#be_v)
@@ -218 +218 @@
-used)
+used#use_v)
@@ -221 +221 @@
-to)
+to#to_prep)
@@ -224 +224 @@
-the)
+the#the_adj)
@@ -226 +226 @@
-trade)))
+trade#trade_n)))
@@ -229 +229 @@
-a)
+a#a_adj2)
@@ -231 +231 @@
-great)
+great#great_adj)
@@ -233 +233 @@
-while))
+while#while_n))
@@ -241 +241 @@
-but)
+but#but_prep)
@@ -244 +244 @@
-yett))
+yett#yet_adv))
@@ -247 +247 @@
-they))
+they#they_pron))
@@ -249 +249 @@
-are)
+are#be_v)
@@ -251 +251 @@
-not)
+not#not_adv)
@@ -255 +255 @@
-their)
+their#their_adj)
@@ -257 +257 @@
-craft's))
+craft's#craft_n))
@@ -259 +259 @@
-masters)
+masters#master_n1)
@@ -262 +262 @@
-at)
+at#at_prep)
@@ -265 +265 @@
-it))))
+it#it_pron))))
@@ -274 +274 @@
-My)
+My#my_adj)
@@ -276 +276 @@
-father))
+father#father_n))
@@ -278 +278 @@
-called)
+called#call_v)
@@ -281 +281 @@
-heare))
+heare#here_adv))
@@ -284 +284 @@
-one)
+one#one_adj)
@@ -286 +286 @@
-Monday)))
+Monday#NA)))
@@ -292 +292 @@
-and)
+and#and_conj1)
@@ -297 +297 @@
-went)
+went#go_v)
@@ -300 +300 @@
-for)
+for#for_prep)
@@ -303 +303 @@
-London)))
+London#NA)))
@@ -306 +306 @@
-early)
+early#early_adv)
@@ -309 +309 @@
-one)
+one#one_adj)
@@ -311 +311 @@
-tusday)
+tusday#NA)
@@ -313 +313 @@
-morning))))
+morning#morning_n))))
@@ -316 +316 @@
-and)
+and#and_conj1)
@@ -320 +320 @@
-Mr)
+Mr#NA)
@@ -322 +322 @@
-Cater))
+Cater#NA))
@@ -325 +325 @@
-with)
+with#with_prep)
@@ -328 +328 @@
-him)))))
+him#him_pron)))))
@@ -337 +337 @@
-Mr.)
+Mr.#NA)
@@ -339 +339 @@
-Cater))
+Cater#NA))
@@ -341 +341 @@
-goeth)
+goeth#go_v)
@@ -344 +344 @@
-about)
+about#about_adv)
@@ -347 +347 @@
-Preston)
+Preston#NA)
@@ -349 +349 @@
-liueing)))
+liueing#living_n.3-5)))
@@ -358 +358 @@
-the)
+the#the_adj)
@@ -360 +360 @@
-minister)
+minister#minister_n)
@@ -363 +363 @@
-there)))
+there#there_adv)))
@@ -365 +365 @@
-hath)
+hath#have_v)
@@ -367 +367 @@
-laid)
+laid#lay_v1)
@@ -369 +369 @@
-down)
+down#down_adv)
@@ -379 +379 @@
-He))
+He#he_pron))
@@ -382 +382 @@
-and)
+and#and_conj1)
@@ -385 +385 @@
-his)
+his#his_adj)
@@ -387 +387 @@
-wife))))
+wife#wife_n))))
@@ -389 +389 @@
-were)
+were#be_v)
@@ -392 +392 @@
-there))
+there#there_adv))
@@ -395 +395 @@
-last)
+last#last_adv)
@@ -397 +397 @@
-week))
+week#week_n))
@@ -400 +400 @@
-about)
+about#about_adv)
@@ -403 +403 @@
-it)))
+it#it_pron)))
@@ -412 +412 @@
-she))
+she#she_pron))
@@ -414 +414 @@
-went)
+went#go_v)
@@ -417 +417 @@
-to)
+to#to_prep)
@@ -420 +420 @@
-her)
+her#her_pron2)
@@ -422 +422 @@
-cosin)
+cosin#cousin_n)
@@ -425 +425 @@
-cockle))))
+cockle#NA))))
@@ -428 +428 @@
-to)
+to#to_prep)
@@ -430 +430 @@
-give)
+give#give_v)
@@ -433 +433 @@
-it))
+it#it_pron))
@@ -436 +436 @@
-to)
+to#to_prep)
@@ -439 +439 @@
-her)
+her#her_pron2)
@@ -441 +441 @@
-husband)))
+husband#husband_n)))
@@ -447 +447 @@
-he))
+he#he_pron))
@@ -449 +449 @@
-being)
+being#be_v)
@@ -452 +452 @@
-a)
+a#a_adj2)
@@ -454 +454 @@
-great)
+great#great_adj)
@@ -456 +456 @@
-man)
+man#man_n1)
@@ -459 +459 @@
-with)
+with#with_prep)
@@ -462 +462 @@
-the)
+the#the_adj)
@@ -464 +464 @@
-bishop))))))
+bishop#bishop_n))))))
@@ -473 +473 @@
-It))
+It#it_pron))
@@ -475 +475 @@
-is)
+is#be_v)
@@ -478 +478 @@
-all)
+all#all_adj)
@@ -483 +483 @@
-that)
+that#that_conj)
@@ -487 +487 @@
-people))
+people#people_n))
@@ -489 +489 @@
-aime)
+aime#aim_v)
@@ -492 +492 @@
-at)
+at#at_prep)
@@ -497 +497 @@
-to)
+to#to_prep)
@@ -499 +499 @@
-be)
+be#be_v)
@@ -502 +502 @@
-great)
+great#great_adj)
@@ -505 +505 @@
-in)
+in#in_prep)
@@ -508 +508 @@
-this)
+this#this_pron)
@@ -510 +510 @@
-world)))))
+world#world_n)))))
@@ -519 +519 @@
-Mr.)
+Mr.#NA)
@@ -521 +521 @@
-Cater))
+Cater#NA))
@@ -523 +523 @@
-hath)
+hath#have_v)
@@ -525 +525 @@
-been)
+been#be_v)
@@ -528 +528 @@
-glad)
+glad#glad_adj)
@@ -531 +531 @@
-at)
+at#at_prep)
@@ -534 +534 @@
-half)
+half#half_adj)
@@ -538 +538 @@
-what))
+what#what_pron))
@@ -546 +546 @@
-he))
+he#he_pron))
@@ -548 +548 @@
-hath)
+hath#have_v)
@@ -551 +551 @@
-now)))))))
+now#now_adv)))))))
@@ -560 +560 @@
-When)
+When#when_adv)
@@ -567 +567 @@
-your)
+your#your_pron)
@@ -569 +569 @@
-letter)
+letter#letter_n1)
@@ -572 +572 @@
-before)
+before#before_adv)
@@ -575 +575 @@
-this))))
+this#this_pron))))
@@ -577 +577 @@
-came))))
+came#come_v))))
@@ -583 +583 @@
-my)
+my#my_adj)
@@ -585 +585 @@
-sister)
+sister#sister_n)
@@ -588 +588 @@
-Wood)))
+Wood#NA)))
@@ -591 +591 @@
-and)
+and#and_conj1)
@@ -594 +594 @@
-myself))))
+myself#myself_pron))))
@@ -596 +596 @@
-were)
+were#be_v)
@@ -599 +599 @@
-at)
+at#at_prep)
@@ -602 +602 @@
-Mr.)
+Mr.#NA)
@@ -604 +604 @@
-Master)))
+Master#NA)))
@@ -607 +607 @@
-at)
+at#at_prep)
@@ -610 +610 @@
-Ospringes)
+Ospringes#NA)
@@ -614 +614 @@
-where))
+where#where_adv))
@@ -622 +622 @@
-we))
+we#we_pron))
@@ -624 +624 @@
-lay)
+lay#lie_v1)
@@ -627 +627 @@
-five)
+five#five_v)
@@ -629 +629 @@
-night))))))
+night#night_n))))))
@@ -637 +637 @@
-and)
+and#and_conj1)
@@ -640 +640 @@
-while)
+while#while_adv)
@@ -647 +647 @@
-I))
+I#i_pron))
@@ -649 +649 @@
-were)
+were#be_v)
@@ -652 +652 @@
-there)))))
+there#there_adv)))))
@@ -655 +655 @@
-I))
+I#i_pron))
@@ -657 +657 @@
-heard)
+heard#hear_v)
@@ -664 +664 @@
-I))
+I#i_pron))
@@ -666 +666 @@
-had)
+had#have_v)
@@ -669 +669 @@
-a)
+a#a_adj2)
@@ -671 +671 @@
-letter)))))
+letter#letter_n1)))))
@@ -677 +677 @@
-and)
+and#and_conj1)
@@ -680 +680 @@
-I))
+I#i_pron))
@@ -682 +682 @@
-did)
+did#do_v)
@@ -684 +684 @@
-imagine)
+imagine#imagine_v)
@@ -691 +691 @@
-it))
+it#it_pron))
@@ -693 +693 @@
-was)
+was#be_v)
@@ -696 +696 @@
-from)
+from#from_prep)
@@ -699 +699 @@
-you)))))
+you#you_pron)))))
@@ -705 +705 @@
-which))
+which#which_adj))
@@ -712 +712 @@
-drawed)
+drawed#draw_v)
@@ -715 +715 @@
-me))
+me#me_pron1))
@@ -718 +718 @@
-home))
+home#home_n1))
@@ -722 +722 @@
-a)
+a#a_adj2)
@@ -724 +724 @@
-week))
+week#week_n))
@@ -726 +726 @@
-before)
+before#before_adv)
@@ -733 +733 @@
-I))
+I#i_pron))
@@ -735 +735 @@
-should)
+should#shall_v)
@@ -746 +746 @@
-Wee))
+Wee#we_pron))
@@ -748 +748 @@
-were)
+were#be_v)
@@ -750 +750 @@
-entertained)
+entertained#entertain_v)
@@ -753 +753 @@
-with)
+with#with_prep)
@@ -756 +756 @@
-the)
+the#the_adj)
@@ -758 +758 @@
-greatest)
+greatest#great_adj)
@@ -760 +760 @@
-kindnesse)
+kindnesse#kindness_n)
@@ -763 +763 @@
-and)
+and#and_conj1)
@@ -766 +766 @@
-civility)))
+civility#civility_n)))
@@ -771 +771 @@
-that)
+that#that_conj)
@@ -776 +776 @@
-were)
+were#be_v)
@@ -779 +779 @@
-possible))))))
+possible#possible_adj))))))
@@ -788 +788 @@
-it))
+it#it_pron))
@@ -790 +790 @@
-would)
+would#will_v1)
@@ -792 +792 @@
-take)
+take#take_v)
@@ -794 +794 @@
-up)
+up#up_adv1)
@@ -797 +797 @@
-this)
+this#this_pron)
@@ -799 +799 @@
-whole)
+whole#whole_adj)
@@ -801 +801 @@
-paper))
+paper#paper_n))
@@ -804 +804 @@
-to)
+to#to_prep)
@@ -806 +806 @@
-relate)
+relate#relate_v)
@@ -809 +809 @@
-it))
+it#it_pron))
@@ -812 +812 @@
-unto)
+unto#unto_prep)
@@ -815 +815 @@
-you))))
+you#you_pron))))
@@ -824 +824 @@
-therefore))
+therefore#therefore_adv))
@@ -827 +827 @@
-I))
+I#i_pron))
@@ -829 +829 @@
-shall)
+shall#shall_v)
@@ -831 +831 @@
-refer)
+refer#refer_v)
@@ -835 +835 @@
-it))
+it#it_pron))
@@ -838 +838 @@
-and)
+and#and_conj1)
@@ -841 +841 @@
-all)
+all#all_adj)
@@ -843 +843 @@
-thinges))))
+thinges#thing_n1))))
@@ -846 +846 @@
-untill)
+untill#until_prep)
@@ -853 +853 @@
-I))
+I#i_pron))
@@ -855 +855 @@
-see)
+see#see_v)
@@ -858 +858 @@
-you))
+you#you_pron))
@@ -862 +862 @@
-which))
+which#which_adj))
@@ -869 +869 @@
-will)
+will#will_v1)
@@ -871 +871 @@
-be)
+be#be_v)
@@ -874 +874 @@
-within)
+within#within_adv)
@@ -877 +877 @@
-this)
+this#this_pron)
@@ -879 +879 @@
-three)
+three#three_adj)
@@ -881 +881 @@
-week)))
+week#week_n)))
@@ -887 +887 @@
-God))
+God#NA))
@@ -889 +889 @@
-willing)))))))
+willing#will_v2)))))))
@@ -898 +898 @@
-My)
+My#my_adj)
@@ -900 +900 @@
-father))
+father#father_n))
@@ -902 +902 @@
-is)
+is#be_v)
@@ -905 +905 @@
-very)
+very#very_adj)
@@ -907 +907 @@
-desireous)
+desireous#desirous_adj)
@@ -910 +910 @@
-to)
+to#to_prep)
@@ -912 +912 @@
-have)
+have#have_v)
@@ -916 +916 @@
-Dick))
+Dick#NA))
@@ -918 +918 @@
-meet)
+meet#meet_v)
@@ -921 +921 @@
-me))
+me#me_pron1))
@@ -924 +924 @@
-at)
+at#at_prep)
@@ -927 +927 @@
-his)
+his#his_adj)
@@ -929 +929 @@
-house))))))
+house#house_n1))))))
@@ -935 +935 @@
-which))
+which#which_adj))
@@ -941 +941 @@
-hee))
+hee#he_pron))
@@ -943 +943 @@
-bid)
+bid#bid_v)
@@ -947 +947 @@
-me))
+me#me_pron1))
@@ -949 +949 @@
-acquaint)
+acquaint#acquaint_v)
@@ -952 +952 @@
-you))
+you#you_pron))
@@ -955 +955 @@
-with)
+with#with_prep)
@@ -960 +960 @@
-if)
+if#if_conj)
@@ -967 +967 @@
-I))
+I#i_pron))
@@ -969 +969 @@
-write)
+write#write_v)
@@ -972 +972 @@
-to)
+to#to_prep)
@@ -975 +975 @@
-you)))
+you#you_pron)))
@@ -978 +978 @@
-before)
+before#before_adv)
@@ -985 +985 @@
-he))
+he#he_pron))
@@ -987 +987 @@
-returnd)))))))))
+returnd#return_v1)))))))))
@@ -995 +995 @@
-but)
+but#but_prep)
@@ -998 +998 @@
-I))
+I#i_pron))
@@ -1000 +1000 @@
-feare)
+feare#fear_v)
@@ -1007 +1007 @@
-he))
+he#he_pron))
@@ -1009 +1009 @@
-was)
+was#be_v)
@@ -1012 +1012 @@
-very)
+very#very_adj)
@@ -1014 +1014 @@
-troublesome)
+troublesome#troublesome_adj)
@@ -1017 +1017 @@
-to)
+to#to_prep)
@@ -1020 +1020 @@
-you))))
+you#you_pron))))
@@ -1023 +1023 @@
-when)
+when#when_adv)
@@ -1030 +1030 @@
-he))
+he#he_pron))
@@ -1032 +1032 @@
-was)
+was#be_v)
@@ -1035 +1035 @@
-last))
+last#last_adv))
@@ -1038 +1038 @@
-there)))))))
+there#there_adv)))))))
@@ -1046 +1046 @@
-but)
+but#but_prep)
@@ -1049 +1049 @@
-it))
+it#it_pron))
@@ -1051 +1051 @@
-is)
+is#be_v)
@@ -1055 +1055 @@
-my)
+my#my_adj)
@@ -1057 +1057 @@
-father's))
+father's#father_n))
@@ -1059 +1059 @@
-desire)))
+desire#desire_n)))
@@ -1066 +1066 @@
-therefore))
+therefore#therefore_adv))
@@ -1069 +1069 @@
-I))
+I#i_pron))
@@ -1071 +1071 @@
-hope)
+hope#hope_v)
@@ -1078 +1078 @@
-you))
+you#you_pron))
@@ -1080 +1080 @@
-will)
+will#will_v1)
@@ -1082 +1082 @@
-pardon)
+pardon#pardon_v)
@@ -1085 +1085 @@
-it))))
+it#it_pron))))
@@ -1094 +1094 @@
-Mis)
+Mis#NA)
@@ -1096 +1096 @@
-Hugusson))
+Hugusson#NA))
@@ -1098 +1098 @@
-is)
+is#be_v)
@@ -1100 +1100 @@
-returned)
+returned#return_v1)
@@ -1103 +1103 @@
-from)
+from#from_prep)
@@ -1106 +1106 @@
-London)))
+London#NA)))
@@ -1118 +1118 @@
-I))
+I#i_pron))
@@ -1120 +1120 @@
-had)
+had#have_v)
@@ -1123 +1123 @@
-a)
+a#a_adj2)
@@ -1125 +1125 @@
-letter))
+letter#letter_n1))
@@ -1128 +1128 @@
-from)
+from#from_prep)
@@ -1131 +1131 @@
-her)))
+her#her_pron1)))
@@ -1134 +1134 @@
-while)
+while#while_adv)
@@ -1141 +1141 @@
-she))
+she#she_pron))
@@ -1143 +1143 @@
-was)
+was#be_v)
@@ -1146 +1146 @@
-there)))))
+there#there_adv)))))
@@ -1155 +1155 @@
-excepting)
+excepting#except_v)
@@ -1158 +1158 @@
-your)
+your#your_pron)
@@ -1160 +1160 @@
-self)))
+self#self_pron)))
@@ -1163 +1163 @@
-deare)
+deare#dear_adj1)
@@ -1165 +1165 @@
-Mother))
+Mother#mother_n1))
@@ -1168 +1168 @@
-I))
+I#i_pron))
@@ -1170 +1170 @@
-doe)
+doe#do_v)
@@ -1172 +1172 @@
-not)
+not#not_adv)
@@ -1174 +1174 @@
-know)
+know#know_v)
@@ -1177 +1177 @@
-her)
+her#her_pron2)
@@ -1179 +1179 @@
-fellow)
+fellow#fellow_n)
@@ -1182 +1182 @@
-for)
+for#for_prep)
@@ -1185 +1185 @@
-goodnesse)))
+goodnesse#goodness_n)))
@@ -1188 +1188 @@
-in)
+in#in_prep)
@@ -1191 +1191 @@
-the)
+the#the_adj)
@@ -1193 +1193 @@
-world))))
+world#world_n))))
@@ -1202 +1202 @@
-I))
+I#i_pron))
@@ -1204 +1204 @@
-am)
+am#be_v)
@@ -1206 +1206 @@
-not)
+not#not_adv)
@@ -1209 +1209 @@
-able)
+able#able_adj)
@@ -1212 +1212 @@
-to)
+to#to_prep)
@@ -1214 +1214 @@
-commend)
+commend#commend_v)
@@ -1217 +1217 @@
-her))
+her#her_pron1))
@@ -1220 +1220 @@
-comparable)
+comparable#comparably_adv)
@@ -1223 +1223 @@
-to)
+to#to_prep)
@@ -1226 +1226 @@
-her)
+her#her_pron2)
@@ -1228 +1228 @@
-meritt)))))))
+meritt#merit_n)))))))
@@ -1235 +1235 @@
-therefore))
+therefore#therefore_adv))
@@ -1238 +1238 @@
-I))
+I#i_pron))
@@ -1240 +1240 @@
-will)
+will#will_v1)
@@ -1242 +1242 @@
-say)
+say#say_v1)
@@ -1245 +1245 @@
-nothing))
+nothing#nothing_pron))
@@ -1254 +1254 @@
-Mis)
+Mis#NA)
@@ -1256 +1256 @@
-Wekerlin))
+Wekerlin#NA))
@@ -1258 +1258 @@
-goeth)
+goeth#go_v)
@@ -1261 +1261 @@
-to)
+to#to_prep)
@@ -1264 +1264 @@
-house-keeping)))
+house-keeping#housekeeping_n)))
@@ -1267 +1267 @@
-this)
+this#this_pron)
@@ -1269 +1269 @@
-Michalmas))
+Michalmas#NA))
@@ -1272 +1272 @@
-heare)
+heare#here_adv)
@@ -1275 +1275 @@
-in)
+in#in_prep)
@@ -1278 +1278 @@
-this)
+this#this_pron)
@@ -1280 +1280 @@
-parrish))))
+parrish#parish_n))))
@@ -1289 +1289 @@
-She))
+She#she_pron))
@@ -1292 +1292 @@
-never))
+never#never_adv))
@@ -1294 +1294 @@
-kene)
+kene#can_v1)
@@ -1296 +1296 @@
-have)
+have#have_v)
@@ -1299 +1299 @@
-since)
+since#since_adv)
@@ -1306 +1306 @@
-she))
+she#she_pron))
@@ -1308 +1308 @@
-was)
+was#be_v)
@@ -1310 +1310 @@
-married)
+married#marry_v)
@@ -1313 +1313 @@
-before)))))
+before#before_adv)))))
@@ -1322 +1322 @@
-As)
+As#as_adv)
@@ -1333 +1333 @@
-I))
+I#i_pron))
@@ -1335 +1335 @@
-remember))))
+remember#remember_v))))
@@ -1338 +1338 @@
-I))
+I#i_pron))
@@ -1340 +1340 @@
-sent)
+sent#send_v1)
@@ -1343 +1343 @@
-you))
+you#you_pron))
@@ -1346 +1346 @@
-word))
+word#word_n))
@@ -1349 +1349 @@
-before)
+before#before_adv)
@@ -1357 +1357 @@
-the)
+the#the_adj)
@@ -1359 +1359 @@
-Lady's))
+Lady's#lady_n))
@@ -1363 +1363 @@
-land))
+land#land_n1))
@@ -1365 +1365 @@
-was)
+was#be_v)
@@ -1368 +1368 @@
-gone)))))
+gone#gone_adj)))))
@@ -1377 +1377 @@
-She))
+She#she_pron))
@@ -1379 +1379 @@
-hath)
+hath#have_v)
@@ -1381 +1381 @@
-bought)
+bought#buy_v)
@@ -1384 +1384 @@
-her))
+her#her_pron1))
@@ -1387 +1387 @@
-a)
+a#a_adj2)
@@ -1389 +1389 @@
-fine)
+fine#fine_adj)
@@ -1391 +1391 @@
-dimon)
+dimon#diamond_n)
@@ -1393 +1393 @@
-ring)
+ring#ring_n1)
@@ -1398 +1398 @@
-with)
+with#with_prep)
@@ -1401 +1401 @@
-part)
+part#part_n)
@@ -1404 +1404 @@
-of)
+of#of_prep)
@@ -1407 +1407 @@
-the)
+the#the_adj)
@@ -1409 +1409 @@
-mony)))))
+mony#money_n)))))
@@ -1415 +1415 @@
-which))
+which#which_adj))
@@ -1422 +1422 @@
-is)
+is#be_v)
@@ -1424 +1424 @@
-thought)
+thought#think_v2)
@@ -1433 +1433 @@
-in)
+in#in_prep)
@@ -1436 +1436 @@
-short)
+short#short_adj)
@@ -1438 +1438 @@
-time)))
+time#time_n)))
@@ -1440 +1440 @@
-will)
+will#will_v1)
@@ -1442 +1442 @@
-be)
+be#be_v)
@@ -1445 +1445 @@
-her)
+her#her_pron2)
@@ -1447 +1447 @@
-greatest)
+greatest#great_adj)
@@ -1449 +1449 @@
-riches))))))
+riches#riches_n))))))
@@ -1458 +1458 @@
-It))
+It#it_pron))
@@ -1460 +1460 @@
-is)
+is#be_v)
@@ -1462 +1462 @@
-thought)
+thought#think_v2)
@@ -1465 +1465 @@
-that)
+that#that_conj)
@@ -1469 +1469 @@
-he))
+he#he_pron))
@@ -1471 +1471 @@
-is)
+is#be_v)
@@ -1474 +1474 @@
-in)
+in#in_prep)
@@ -1477 +1477 @@
-a)
+a#a_adj2)
@@ -1479 +1479 @@
-Consumption)))))
+Consumption#consumption_n)))))
@@ -1488 +1488 @@
-it))
+it#it_pron))
@@ -1490 +1490 @@
-is)
+is#be_v)
@@ -1492 +1492 @@
-said)
+said#say_v1)
@@ -1495 +1495 @@
-be)
+be#by_prep)
@@ -1500 +1500 @@
-some)
+some#some_pron)
@@ -1505 +1505 @@
-that)
+that#that_conj)
@@ -1510 +1510 @@
-hath)
+hath#have_v)
@@ -1513 +1513 @@
-lately))
+lately#lately_adv))
@@ -1515 +1515 @@
-seen)
+seen#see_v)
@@ -1518 +1518 @@
-him))))))
+him#him_pron))))))
@@ -1521 +1521 @@
-that)
+that#that_conj)
@@ -1525 +1525 @@
-he))
+he#he_pron))
@@ -1527 +1527 @@
-is)
+is#be_v)
@@ -1531 +1531 @@
-soe)
+soe#so_adv)
@@ -1533 +1533 @@
-much))
+much#much_adj))
@@ -1535 +1535 @@
-altered)
+altered#altered_adj)
@@ -1538 +1538 @@
-that)
+that#that_conj)
@@ -1542 +1542 @@
-one))
+one#one_adj))
@@ -1544 +1544 @@
-would)
+would#will_v1)
@@ -1547 +1547 @@
-scarce))
+scarce#scarce_adj))
@@ -1549 +1549 @@
-know)
+know#know_v)
@@ -1552 +1552 @@
-him))))
+him#him_pron))))
@@ -1557 +1557 @@
-soe)
+soe#so_adv)
@@ -1559 +1559 @@
-lean)
+lean#lean_adj)
@@ -1562 +1562 @@
-and)
+and#and_conj1)
@@ -1565 +1565 @@
-pale)))))))
+pale#pale_adj)))))))
@@ -1574 +1574 @@
-Though)
+Though#though_adv)
@@ -1581 +1581 @@
-they))
+they#they_pron))
@@ -1583 +1583 @@
-have)
+have#have_v)
@@ -1586 +1586 @@
-money)))))
+money#money_n)))))
@@ -1589 +1589 @@
-I))
+I#i_pron))
@@ -1591 +1591 @@
-beleeve)
+beleeve#believe_v)
@@ -1598 +1598 @@
-there))
+there#there_adv))
@@ -1600 +1600 @@
-is)
+is#be_v)
@@ -1603 +1603 @@
-little)
+little#little_adj)
@@ -1605 +1605 @@
-pleasure)
+pleasure#pleasure_n)
@@ -1608 +1608 @@
-to)
+to#to_prep)
@@ -1610 +1610 @@
-be)
+be#be_v)
@@ -1612 +1612 @@
-taken)
+taken#take_v)
@@ -1615 +1615 @@
-in)
+in#in_prep)
@@ -1618 +1618 @@
-such)
+such#such_adj)
@@ -1620 +1620 @@
-a)
+a#a_adj2)
@@ -1622 +1622 @@
-course)
+course#course_n1)
@@ -1625 +1625 @@
-of)
+of#of_prep)
@@ -1628 +1628 @@
-life)))))))))
+life#life_n)))))))))
@@ -1639 +1639 @@
-Pray))
+Pray#pray_v))
@@ -1641 +1641 @@
-give)
+give#give_v)
@@ -1644 +1644 @@
-my)
+my#my_adj)
@@ -1646 +1646 @@
-service))
+service#service_n1))
@@ -1649 +1649 @@
-to)
+to#to_prep)
@@ -1652 +1652 @@
-my)
+my#my_adj)
@@ -1654 +1654 @@
-wife)
+wife#wife_n)
@@ -1658 +1658 @@
-whom))
+whom#whom_pron))
@@ -1664 +1664 @@
-I))
+I#i_pron))
@@ -1666 +1666 @@
-hope)
+hope#hope_v)
@@ -1671 +1671 @@
-to)
+to#to_prep)
@@ -1673 +1673 @@
-see)
+see#see_v)
@@ -1676 +1676 @@
-ere)
+ere#ere_adv)
@@ -1679 +1679 @@
-long))))))))
+long#long_adj1))))))))
@@ -1688 +1688 @@
-I))
+I#i_pron))
@@ -1690 +1690 @@
-shall)
+shall#shall_v)
@@ -1692 +1692 @@
-not)
+not#not_adv)
@@ -1694 +1694 @@
-wright)
+wright#write_v)
@@ -1698 +1698 @@
-above)
+above#above_adv)
@@ -1700 +1700 @@
-once))
+once#once_adv))
@@ -1703 +1703 @@
-more)))
+more#more_adj)))
@@ -1706 +1706 @@
-before)
+before#before_adv)
@@ -1713 +1713 @@
-I))
+I#i_pron))
@@ -1715 +1715 @@
-see)
+see#see_v)
@@ -1718 +1718 @@
-you)))))
+you#you_pron)))))
@@ -1727 +1727 @@
-in)
+in#in_prep)
@@ -1730 +1730 @@
-the)
+the#the_adj)
@@ -1732 +1732 @@
-meantime)))
+meantime#meantime_n)))
@@ -1735 +1735 @@
-I))
+I#i_pron))
@@ -1737 +1737 @@
-am)
+am#be_v)
@@ -1740 +1740 @@
-Your)
+Your#your_pron)
@@ -1742 +1742 @@
-obeident)
+obeident#obedient_adj)
@@ -1744 +1744 @@
-Daughter)
+Daughter#daughter_n)
@@ -1747 +1747 @@
-and)
+and#and_conj1)
@@ -1750 +1750 @@
-servant)))
+servant#servant_n)))
@@ -1753 +1753 @@
-Elizabeth)
+Elizabeth#NA)
@@ -1755 +1755 @@
-Oxinden))))
+Oxinden#NA))))
@@ -1764 +1764 @@
-Pray))
+Pray#pray_v))
@@ -1766 +1766 @@
-remember)
+remember#remember_v)
@@ -1769 +1769 @@
-me))
+me#me_pron1))
@@ -1772 +1772 @@
-to)
+to#to_prep)
@@ -1776 +1776 @@
-Mall))
+Mall#NA))
@@ -1779 +1779 @@
-and)
+and#and_conj1)
@@ -1782 +1782 @@
-the)
+the#the_adj)
@@ -1784 +1784 @@
-countesse)))
+countesse#countess_n)))
@@ -1787 +1787 @@
-and)
+and#and_conj1)
@@ -1790 +1790 @@
-my)
+my#my_adj)
@@ -1792 +1792 @@
-neighbour)))))
+neighbour#neighbour_n)))))
