--- grey-1523-e1-p1.psd.vert	2025-09-10 13:51:11
+++ grey-1523-e1-p1.lem.vert	2025-09-10 13:50:58
@@ -17 +17 @@
-Cromwell))
+Cromwell#NA))
@@ -22 +22 @@
-I))
+I#i_pron))
@@ -24 +24 @@
-woll)
+woll#will_v2)
@@ -27 +27 @@
-that)
+that#that_conj)
@@ -31 +31 @@
-yow))
+yow#you_pron))
@@ -33 +33 @@
-send)
+send#send_v1)
@@ -36 +36 @@
-to)
+to#to_prep)
@@ -39 +39 @@
-me)))
+me#me_pron1)))
@@ -42 +42 @@
-in)
+in#in_prep)
@@ -45 +45 @@
-hast)))
+hast#haste_n)))
@@ -49 +49 @@
-the)
+the#the_adj)
@@ -51 +51 @@
-trussynn)
+trussynn#truss_v)
@@ -53 +53 @@
-bed)
+bed#bed_n)
@@ -56 +56 @@
-of)
+of#of_prep)
@@ -59 +59 @@
-cloth)
+cloth#cloth_n)
@@ -62 +62 @@
-of)
+of#of_prep)
@@ -65 +65 @@
-tyssewe))))))
+tyssewe#tissue_n))))))
@@ -70 +70 @@
-and)
+and#and_conj1)
@@ -73 +73 @@
-the)
+the#the_adj)
@@ -75 +75 @@
-fether)
+fether#feather_n)
@@ -77 +77 @@
-bed)
+bed#bed_n)
@@ -82 +82 @@
-wyth)
+wyth#with_prep)
@@ -86 +86 @@
-the)
+the#the_adj)
@@ -88 +88 @@
-fustyons))
+fustyons#fustian_n))
@@ -91 +91 @@
-and)
+and#and_conj1)
@@ -94 +94 @@
-a)
+a#a_adj2)
@@ -96 +96 @@
-materas)
+materas#mattress_n1)
@@ -99 +99 @@
-longynn)
+longynn#longing_adj1)
@@ -102 +102 @@
-to)
+to#to_prep)
@@ -105 +105 @@
-the)
+the#the_adj)
@@ -107 +107 @@
-same))))))))
+same#same_adj))))))))
@@ -110 +110 @@
-wyth)
+wyth#with_prep)
@@ -113 +113 @@
-the)
+the#the_adj)
@@ -115 +115 @@
-cownterpoynt))))))))
+cownterpoynt#counterpoint_n2))))))))
@@ -123 +123 @@
-also)
+also#also_adv)
@@ -126 +126 @@
-I))
+I#i_pron))
@@ -128 +128 @@
-woll)
+woll#will_v2)
@@ -131 +131 @@
-that)
+that#that_conj)
@@ -135 +135 @@
-you))
+you#you_pron))
@@ -137 +137 @@
-delyver)
+delyver#deliver_v1)
@@ -140 +140 @@
-all)
+all#all_adj)
@@ -142 +142 @@
-such)
+such#such_adj)
@@ -144 +144 @@
-tents)
+tents#tent_n1)
@@ -150 +150 @@
-pavylyons)))
+pavylyons#pavilion_n)))
@@ -155 +155 @@
-and)
+and#and_conj1)
@@ -158 +158 @@
-hales)))
+hales#hall_n1)))
@@ -161 +161 @@
-as)
+as#as_adv)
@@ -172 +172 @@
-you))
+you#you_pron))
@@ -174 +174 @@
-have))))
+have#have_v))))
@@ -177 +177 @@
-of)
+of#of_prep)
@@ -180 +180 @@
-myne))))
+myne#mine_adj))))
@@ -183 +183 @@
-on)
+on#on_adv)
@@ -185 +185 @@
-to)
+to#to_prep)
@@ -188 +188 @@
-my)
+my#my_adj)
@@ -190 +190 @@
-sonne)
+sonne#son_n1)
@@ -193 +193 @@
-Lenard))))
+Lenard#NA))))
@@ -198 +198 @@
-as)
+as#as_adv)
@@ -209 +209 @@
-you))
+you#you_pron))
@@ -211 +211 @@
-tender)
+tender#tender_v2)
@@ -214 +214 @@
-my)
+my#my_adj)
@@ -216 +216 @@
-plesur)))))))
+plesur#pleasure_n)))))))
@@ -224 +224 @@
-And)
+And#and_conj1)
@@ -227 +227 @@
-thys))
+thys#this_pron))
@@ -229 +229 @@
-shall)
+shall#shall_v)
@@ -231 +231 @@
-be)
+be#be_v)
@@ -234 +234 @@
-your)
+your#your_pron)
@@ -236 +236 @@
-suffycyant)
+suffycyant#sufficient_adj)
@@ -238 +238 @@
-warrant)
+warrant#warrant_n1)
@@ -241 +241 @@
-and)
+and#and_conj1)
@@ -244 +244 @@
-dyscharge)))
+dyscharge#discharge_n)))
@@ -247 +247 @@
-att)
+att#at_prep)
@@ -250 +250 @@
-all)
+all#all_adj)
@@ -252 +252 @@
-tymes))))
+tymes#time_n))))
@@ -260 +260 @@
-Wrytyn)
+Wrytyn#write_v)
@@ -263 +263 @@
-at)
+at#at_prep)
@@ -266 +266 @@
-Bedwell)))
+Bedwell#NA)))
@@ -269 +269 @@
-thys)
+thys#this_pron)
@@ -271 +271 @@
-present)
+present#present_adj)
@@ -273 +273 @@
-Thorysdaye)
+Thorysdaye#NA)
@@ -276 +276 @@
-by_foore)
+by_foore#before_adv)
@@ -279 +279 @@
-our)
+our#NA)
@@ -281 +281 @@
-Lady)
+Lady#NA)
@@ -283 +283 @@
-daye)
+daye#NA)
@@ -286 +286 @@
-the)
+the#the_adj)
@@ -288 +288 @@
-Assumpcyon)))))
+Assumpcyon#NA)))))
@@ -296 +296 @@
-Cecyl)
+Cecyl#NA)
@@ -298 +298 @@
-Dorsett))
+Dorsett#NA))
