--- jetaylormeas-1662-e3-p1.psd.vert	2025-09-10 13:51:11
+++ jetaylormeas-1662-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -11 +11 @@
-Madam))
+Madam#madam_n))
@@ -16 +16 @@
-The)
+The#the_adj)
@@ -18 +18 @@
-wise)
+wise#wise_adj)
@@ -22 +22 @@
-Bensirach))
+Bensirach#NA))
@@ -26 +26 @@
-advised)
+advised#advise_v)
@@ -29 +29 @@
-that)
+that#that_conj)
@@ -34 +34 @@
-we))
+we#we_pron))
@@ -36 +36 @@
-should)
+should#shall_v)
@@ -38 +38 @@
-not)
+not#not_adv)
@@ -40 +40 @@
-consult)
+consult#consult_v)
@@ -43 +43 @@
-with)
+with#with_prep)
@@ -46 +46 @@
-a)
+a#a_adj2)
@@ -48 +48 @@
-woman)))
+woman#woman_n)))
@@ -51 +51 @@
-concerning)
+concerning#concerning_prep)
@@ -54 +54 @@
-her)
+her#her_pron1)
@@ -58 +58 @@
-of)
+of#of_prep)
@@ -61 +61 @@
-whom)))
+whom#whom_pron)))
@@ -67 +67 @@
-she))
+she#she_pron))
@@ -69 +69 @@
-is)
+is#be_v)
@@ -72 +72 @@
-jealous)
+jealous#jealous_adj)
@@ -79 +79 @@
-neither)
+neither#neither_adv)
@@ -83 +83 @@
-with)
+with#with_prep)
@@ -86 +86 @@
-a)
+a#a_adj2)
@@ -88 +88 @@
-coward)))
+coward#coward_n)))
@@ -91 +91 @@
-in)
+in#in_prep)
@@ -94 +94 @@
-matters)
+matters#matter_n1)
@@ -97 +97 @@
-of)
+of#of_prep)
@@ -100 +100 @@
-warre)))))))
+warre#war_n1)))))))
@@ -105 +105 @@
-nor)
+nor#nor_conj1)
@@ -109 +109 @@
-with)
+with#with_prep)
@@ -112 +112 @@
-a)
+a#a_adj2)
@@ -114 +114 @@
-merchant)))
+merchant#merchant_n)))
@@ -117 +117 @@
-concerning)
+concerning#concerning_prep)
@@ -120 +120 @@
-exchange)))))))
+exchange#exchange_n)))))))
@@ -128 +128 @@
-and)
+and#and_conj1)
@@ -131 +131 @@
-some)
+some#some_pron)
@@ -133 +133 @@
-other)
+other#other_adj)
@@ -135 +135 @@
-instances))
+instances#instance_n))
@@ -138 +138 @@
-he))
+he#he_pron))
@@ -140 +140 @@
-gives)
+gives#give_v)
@@ -143 +143 @@
-of)
+of#of_prep)
@@ -146 +146 @@
-interested)
+interested#interested_adj)
@@ -148 +148 @@
-persons)
+persons#person_n)
@@ -154 +154 @@
-to)
+to#to_prep)
@@ -157 +157 @@
-whom)))
+whom#whom_pron)))
@@ -163 +163 @@
-he))
+he#he_pron))
@@ -167 +167 @@
-would)
+would#will_v1)
@@ -169 +169 @@
-not)
+not#not_adv)
@@ -171 +171 @@
-have)
+have#have_v)
@@ -177 +177 @@
-us))
+us#us_pron))
@@ -179 +179 @@
-hearken)
+hearken#hearken_v)
@@ -182 +182 @@
-in)
+in#in_prep)
@@ -185 +185 @@
-any)
+any#any_adj)
@@ -187 +187 @@
-matter)
+matter#matter_n1)
@@ -190 +190 @@
-of)
+of#of_prep)
@@ -193 +193 @@
-Counsel))))))))))
+Counsel#counsel_n))))))))))
@@ -201 +201 @@
-For)
+For#for_prep)
@@ -206 +206 @@
-where)
+where#where_adv)
@@ -208 +208 @@
-ever))
+ever#ever_adv))
@@ -216 +216 @@
-the)
+the#the_adj)
@@ -218 +218 @@
-interest))
+interest#interest_n))
@@ -220 +220 @@
-is)
+is#be_v)
@@ -225 +225 @@
-secular)
+secular#secular_adj)
@@ -229 +229 @@
-or)
+or#or_conj2)
@@ -233 +233 @@
-vitious)))))
+vitious#vicious_adj)))))
@@ -240 +240 @@
-there))
+there#there_adv))
@@ -243 +243 @@
-the)
+the#the_adj)
@@ -245 +245 @@
-bias))
+bias#bias_n))
@@ -247 +247 @@
-is)
+is#be_v)
@@ -249 +249 @@
-not)
+not#not_adv)
@@ -252 +252 @@
-on)
+on#on_prep)
@@ -255 +255 @@
-the)
+the#the_adj)
@@ -257 +257 @@
-side)
+side#side_n1)
@@ -260 +260 @@
-of)
+of#of_prep)
@@ -265 +265 @@
-truth)
+truth#truth_n)
@@ -269 +269 @@
-or)
+or#or_conj2)
@@ -273 +273 @@
-reason)))))
+reason#reason_n1)))))
@@ -280 +280 @@
-because)
+because#because_adv)
@@ -289 +289 @@
-these))
+these#these_pron))
@@ -293 +293 @@
-are)
+are#be_v)
@@ -296 +296 @@
-seldom))
+seldom#seldom_adv))
@@ -298 +298 @@
-serv'd)
+serv'd#serve_v1)
@@ -301 +301 @@
-by)
+by#by_prep)
@@ -305 +305 @@
-profit))
+profit#profit_n))
@@ -308 +308 @@
-and)
+and#and_conj1)
@@ -311 +311 @@
-low)
+low#low_adj)
@@ -313 +313 @@
-regards))))))))
+regards#regard_n))))))))
@@ -321 +321 @@
-But)
+But#but_prep)
@@ -324 +324 @@
-to)
+to#to_prep)
@@ -326 +326 @@
-consult)
+consult#consult_v)
@@ -329 +329 @@
-with)
+with#with_prep)
@@ -332 +332 @@
-a)
+a#a_adj2)
@@ -334 +334 @@
-friend)))
+friend#friend_n)))
@@ -337 +337 @@
-in)
+in#in_prep)
@@ -340 +340 @@
-matters)
+matters#matter_n1)
@@ -343 +343 @@
-of)
+of#of_prep)
@@ -346 +346 @@
-friendship))))))
+friendship#friendship_n))))))
@@ -348 +348 @@
-is)
+is#be_v)
@@ -351 +351 @@
-like)
+like#like_adj)
@@ -354 +354 @@
-consulting)
+consulting#consult_v)
@@ -357 +357 @@
-with)
+with#with_prep)
@@ -360 +360 @@
-a)
+a#a_adj2)
@@ -362 +362 @@
-spiritual)
+spiritual#spiritual_adj)
@@ -364 +364 @@
-person)))
+person#person_n)))
@@ -367 +367 @@
-in)
+in#in_prep)
@@ -370 +370 @@
-Religion)))))
+Religion#religion_n)))))
@@ -379 +379 @@
-they)
+they#they_pron)
@@ -383 +383 @@
-who))
+who#who_pron))
@@ -390 +390 @@
-understand)
+understand#understand_v)
@@ -394 +394 @@
-the)
+the#the_adj)
@@ -396 +396 @@
-secrets)
+secrets#secret_adj)
@@ -399 +399 @@
-of)
+of#of_prep)
@@ -402 +402 @@
-Religion))))
+Religion#religion_n))))
@@ -405 +405 @@
-or)
+or#or_conj2)
@@ -408 +408 @@
-the)
+the#the_adj)
@@ -410 +410 @@
-interior)
+interior#interior_adj)
@@ -412 +412 @@
-beauties)
+beauties#beauty_n)
@@ -415 +415 @@
-of)
+of#of_prep)
@@ -418 +418 @@
-friendship)))))))))
+friendship#friendship_n)))))))))
@@ -420 +420 @@
-are)
+are#be_v)
@@ -423 +423 @@
-the)
+the#the_adj)
@@ -425 +425 @@
-fittest)
+fittest#fit_adj)
@@ -433 +433 @@
-to)
+to#to_prep)
@@ -435 +435 @@
-give)
+give#give_v)
@@ -438 +438 @@
-answers)
+answers#answer_n)
@@ -441 +441 @@
-in)
+in#in_prep)
@@ -444 +444 @@
-all)
+all#all_adj)
@@ -446 +446 @@
-inquiries)
+inquiries#inquiry_n)
@@ -449 +449 @@
-concerning)
+concerning#concerning_prep)
@@ -452 +452 @@
-the)
+the#the_adj)
@@ -454 +454 @@
-respective)
+respective#respective_adj)
@@ -456 +456 @@
-subjects)))))))))
+subjects#subject_n)))))))))
@@ -461 +461 @@
-because)
+because#because_adv)
@@ -470 +470 @@
-reason)
+reason#reason_n1)
@@ -474 +474 @@
-and)
+and#and_conj1)
@@ -478 +478 @@
-experience))
+experience#experience_n))
@@ -482 +482 @@
-are)
+are#be_v)
@@ -485 +485 @@
-on)
+on#on_prep)
@@ -488 +488 @@
-the)
+the#the_adj)
@@ -490 +490 @@
-side)
+side#side_n1)
@@ -493 +493 @@
-of)
+of#of_prep)
@@ -498 +498 @@
-interest))))))))
+interest#interest_n))))))))
@@ -508 +508 @@
-and)
+and#and_conj1)
@@ -511 +511 @@
-that)
+that#that_pron)
@@ -515 +515 @@
-which))
+which#which_adj))
@@ -523 +523 @@
-in)
+in#in_prep)
@@ -526 +526 @@
-friendship)))
+friendship#friendship_n)))
@@ -528 +528 @@
-is)
+is#be_v)
@@ -532 +532 @@
-most)
+most#most_adj)
@@ -536 +536 @@
-pleasing))
+pleasing#pleasing_adj))
@@ -541 +541 @@
-and)
+and#and_conj1)
@@ -544 +544 @@
-most)
+most#most_adj)
@@ -548 +548 @@
-useful)))))))
+useful#useful_adj)))))))
@@ -554 +554 @@
-is)
+is#be_v)
@@ -556 +556 @@
-also)
+also#also_adv)
@@ -560 +560 @@
-most)
+most#most_adj)
@@ -564 +564 @@
-reasonable))
+reasonable#reasonable_adj))
@@ -569 +569 @@
-and)
+and#and_conj1)
@@ -572 +572 @@
-most)
+most#most_adj)
@@ -576 +576 @@
-true))))
+true#true_adj))))
@@ -586 +586 @@
-and)
+and#and_conj1)
@@ -590 +590 @@
-a)
+a#a_adj2)
@@ -592 +592 @@
-friends))
+friends#friend_n))
@@ -594 +594 @@
-fairest)
+fairest#fair_adj)
@@ -596 +596 @@
-interest))
+interest#interest_n))
@@ -598 +598 @@
-is)
+is#be_v)
@@ -601 +601 @@
-the)
+the#the_adj)
@@ -603 +603 @@
-best)
+best#best_adj)
@@ -605 +605 @@
-measure)
+measure#measure_n)
@@ -608 +608 @@
-of)
+of#of_prep)
@@ -611 +611 @@
-the)
+the#the_adj)
@@ -614 +614 @@
-conducting)
+conducting#conduct_v)
@@ -617 +617 @@
-friendships))))))
+friendships#friendship_n))))))
@@ -625 +625 @@
-and)
+and#and_conj1)
@@ -628 +628 @@
-therefore))
+therefore#therefore_adv))
@@ -631 +631 @@
-you)
+you#you_pron)
@@ -635 +635 @@
-who))
+who#who_pron))
@@ -642 +642 @@
-are)
+are#be_v)
@@ -645 +645 @@
-so)
+so#so_adv)
@@ -647 +647 @@
-eminent)
+eminent#eminent_adj)
@@ -650 +650 @@
-in)
+in#in_prep)
@@ -653 +653 @@
-friendships)))))))
+friendships#friendship_n)))))))
@@ -655 +655 @@
-could)
+could#can_v1)
@@ -657 +657 @@
-also)
+also#also_adv)
@@ -659 +659 @@
-have)
+have#have_v)
@@ -661 +661 @@
-given)
+given#give_v)
@@ -666 +666 @@
-the)
+the#the_adj)
@@ -668 +668 @@
-best)
+best#best_adj)
@@ -670 +670 @@
-answer)
+answer#answer_n)
@@ -673 +673 @@
-to)
+to#to_prep)
@@ -676 +676 @@
-your)
+your#your_pron)
@@ -678 +678 @@
-own)
+own#own_adj)
@@ -680 +680 @@
-inquiries))))
+inquiries#inquiry_n))))
@@ -688 +688 @@
-and)
+and#and_conj1)
@@ -691 +691 @@
-you))
+you#you_pron))
@@ -693 +693 @@
-could)
+could#can_v1)
@@ -695 +695 @@
-have)
+have#have_v)
@@ -697 +697 @@
-trusted)
+trusted#trust_v)
@@ -700 +700 @@
-your)
+your#your_pron)
@@ -702 +702 @@
-own)
+own#own_adj)
@@ -704 +704 @@
-reason))
+reason#reason_n1))
@@ -709 +709 @@
-because)
+because#because_adv)
@@ -716 +716 @@
-it))
+it#it_pron))
@@ -718 +718 @@
-is)
+is#be_v)
@@ -720 +720 @@
-not)
+not#not_adv)
@@ -722 +722 @@
-onely)
+onely#NA)
@@ -725 +725 @@
-greatly))
+greatly#greatly_adv))
@@ -727 +727 @@
-instructed)
+instructed#instruct_v)
@@ -735 +735 @@
-by)
+by#by_prep)
@@ -738 +738 @@
-the)
+the#the_adj)
@@ -740 +740 @@
-direct)
+direct#direct_adj)
@@ -742 +742 @@
-notices)
+notices#notice_n)
@@ -745 +745 @@
-of)
+of#of_prep)
@@ -748 +748 @@
-things)))))
+things#thing_n1)))))
@@ -753 +753 @@
-but)
+but#but_prep)
@@ -755 +755 @@
-also)
+also#also_adv)
@@ -758 +758 @@
-by)
+by#by_prep)
@@ -761 +761 @@
-great)
+great#great_adj)
@@ -763 +763 @@
-experience)
+experience#experience_n)
@@ -766 +766 @@
-in)
+in#in_prep)
@@ -769 +769 @@
-the)
+the#the_adj)
@@ -771 +771 @@
-matter)
+matter#matter_n1)
@@ -775 +775 @@
-of)
+of#of_prep)
@@ -778 +778 @@
-which)))
+which#which_adj)))
@@ -786 +786 @@
-you))
+you#you_pron))
@@ -789 +789 @@
-now))
+now#now_adv))
@@ -791 +791 @@
-inquire))))))))))))
+inquire#inquire_v))))))))))))
@@ -799 +799 @@
-But)
+But#but_prep)
@@ -802 +802 @@
-because)
+because#because_adv)
@@ -809 +809 @@
-I))
+I#i_pron))
@@ -811 +811 @@
-will)
+will#will_v1)
@@ -813 +813 @@
-not)
+not#not_adv)
@@ -815 +815 @@
-use)
+use#use_v)
@@ -818 +818 @@
-any)
+any#any_adj)
@@ -820 +820 @@
-thing)
+thing#thing_n1)
@@ -825 +825 @@
-that)
+that#that_conj)
@@ -830 +830 @@
-shall)
+shall#shall_v)
@@ -832 +832 @@
-look)
+look#look_v)
@@ -835 +835 @@
-like)
+like#like_adj)
@@ -838 +838 @@
-an)
+an#a_adj2)
@@ -840 +840 @@
-excuse)))))))))
+excuse#excuse_n)))))))))
@@ -845 +845 @@
-I))
+I#i_pron))
@@ -847 +847 @@
-will)
+will#will_v1)
@@ -850 +850 @@
-rather)
+rather#rather_adv)
@@ -854 +854 @@
-give)
+give#give_v)
@@ -857 +857 @@
-you))
+you#you_pron))
@@ -860 +860 @@
-such)
+such#such_adj)
@@ -862 +862 @@
-an)
+an#a_adj2)
@@ -864 +864 @@
-account)
+account#account_n)
@@ -868 +868 @@
-which))
+which#which_adj))
@@ -876 +876 @@
-you))
+you#you_pron))
@@ -878 +878 @@
-can)
+can#can_v1)
@@ -881 +881 @@
-easily))
+easily#easily_adv))
@@ -883 +883 @@
-reprove))))
+reprove#reprove_v1))))
@@ -888 +888 @@
-then)
+then#than_conj)
@@ -892 +892 @@
-by)
+by#by_prep)
@@ -895 +895 @@
-declining)
+declining#decline_v)
@@ -898 +898 @@
-your)
+your#your_pron)
@@ -900 +900 @@
-commands))))
+commands#command_n))))
@@ -904 +904 @@
-seem)
+seem#seem_v2)
@@ -907 +907 @@
-more)
+more#more_adj)
@@ -909 +909 @@
-safe)
+safe#safe_adj)
@@ -912 +912 @@
-in)
+in#in_prep)
@@ -915 +915 @@
-my)
+my#my_adj)
@@ -917 +917 @@
-prudence)))
+prudence#prudence_n)))
@@ -922 +922 @@
-then)
+then#than_conj)
@@ -925 +925 @@
-open)
+open#open_adj)
@@ -928 +928 @@
-and)
+and#and_conj1)
@@ -931 +931 @@
-communicative)))
+communicative#communicative_adj)))
@@ -934 +934 @@
-in)
+in#in_prep)
@@ -937 +937 @@
-my)
+my#my_adj)
@@ -939 +939 @@
-friendship)
+friendship#friendship_n)
@@ -942 +942 @@
-to)
+to#to_prep)
@@ -945 +945 @@
-you))))))))))
+you#you_pron))))))))))
@@ -954 +954 @@
-You))
+You#you_pron))
@@ -957 +957 @@
-first))
+first#first_adj))
@@ -959 +959 @@
-inquire)
+inquire#inquire_v)
@@ -964 +964 @@
-how)
+how#how_adv)
@@ -966 +966 @@
-far)))
+far#far_adj)))
@@ -975 +975 @@
-a)
+a#a_adj2)
@@ -977 +977 @@
-Dear))
+Dear#dear_adj1))
@@ -980 +980 @@
-and)
+and#and_conj1)
@@ -983 +983 @@
-a)
+a#a_adj2)
@@ -985 +985 @@
-perfect)
+perfect#perfect_adj)
@@ -987 +987 @@
-friendship))))
+friendship#friendship_n))))
@@ -989 +989 @@
-is)
+is#be_v)
@@ -991 +991 @@
-authoriz'd)
+authoriz'd#authorize_v)
@@ -994 +994 @@
-by)
+by#by_prep)
@@ -997 +997 @@
-the)
+the#the_adj)
@@ -999 +999 @@
-principles)
+principles#principle_n)
@@ -1002 +1002 @@
-of)
+of#of_prep)
@@ -1005 +1005 @@
-Christianity)))))))
+Christianity#christianity_n)))))))
@@ -1014 +1014 @@
-To)
+To#to_prep)
@@ -1017 +1017 @@
-this)))
+this#this_pron)))
@@ -1020 +1020 @@
-I))
+I#i_pron))
@@ -1022 +1022 @@
-answer)
+answer#answer_v)
@@ -1025 +1025 @@
-that)
+that#that_conj)
@@ -1029 +1029 @@
-the)
+the#the_adj)
@@ -1031 +1031 @@
-word)
+word#word_n)
@@ -1036 +1036 @@
-Friendship))
+Friendship#friendship_n))
@@ -1041 +1041 @@
-in)
+in#in_prep)
@@ -1044 +1044 @@
-the)
+the#the_adj)
@@ -1046 +1046 @@
-sense)
+sense#sense_n)
@@ -1057 +1057 @@
-we))
+we#we_pron))
@@ -1060 +1060 @@
-commonly))
+commonly#commonly_adv))
@@ -1062 +1062 @@
-mean)
+mean#mean_v1)
@@ -1065 +1065 @@
-by)
+by#by_prep)
@@ -1068 +1068 @@
-it))))))))
+it#it_pron))))))))
@@ -1072 +1072 @@
-is)
+is#be_v)
@@ -1074 +1074 @@
-not)
+not#not_adv)
@@ -1078 +1078 @@
-so)
+so#so_adv)
@@ -1080 +1080 @@
-much))
+much#much_adj))
@@ -1085 +1085 @@
-as)))
+as#as_adv)))
@@ -1087 +1087 @@
-named)
+named#name_v)
@@ -1090 +1090 @@
-in)
+in#in_prep)
@@ -1093 +1093 @@
-the)
+the#the_adj)
@@ -1095 +1095 @@
-New-Testament)))))
+New-Testament#NA)))))
@@ -1103 +1103 @@
-&)
+&#and_conj1)
@@ -1106 +1106 @@
-our)
+our#our_pron)
@@ -1108 +1108 @@
-Religion))
+Religion#religion_n))
@@ -1110 +1110 @@
-takes)
+takes#take_v)
@@ -1113 +1113 @@
-no)
+no#no_adj)
@@ -1115 +1115 @@
-notice))
+notice#notice_n))
@@ -1118 +1118 @@
-of)
+of#of_prep)
@@ -1121 +1121 @@
-it)))
+it#it_pron)))
@@ -1130 +1130 @@
-You))
+You#you_pron))
@@ -1132 +1132 @@
-think)
+think#think_v2)
@@ -1136 +1136 @@
-it))
+it#it_pron))
@@ -1139 +1139 @@
-strange)))
+strange#strange_adj)))
@@ -1147 +1147 @@
-but)
+but#but_prep)
@@ -1149 +1149 @@
-read)
+read#read_v)
@@ -1151 +1151 @@
-on)
+on#on_adv)
@@ -1154 +1154 @@
-before)
+before#before_adv)
@@ -1161 +1161 @@
-you))
+you#you_pron))
@@ -1163 +1163 @@
-spend)
+spend#spend_v1)
@@ -1167 +1167 @@
-so)
+so#so_adv)
@@ -1169 +1169 @@
-much))
+much#much_adj))
@@ -1172 +1172 @@
-as)
+as#as_adv)
@@ -1175 +1175 @@
-the)
+the#the_adj)
@@ -1177 +1177 @@
-beginning)
+beginning#beginning_n)
@@ -1180 +1180 @@
-of)
+of#of_prep)
@@ -1184 +1184 @@
-a)
+a#a_adj2)
@@ -1186 +1186 @@
-passion))
+passion#passion_n))
@@ -1189 +1189 @@
-or)
+or#or_conj2)
@@ -1192 +1192 @@
-a)
+a#a_adj2)
@@ -1194 +1194 @@
-Wonder))))))))
+Wonder#wonder_n))))))))
@@ -1197 +1197 @@
-upon)
+upon#upon_prep)
@@ -1200 +1200 @@
-it))))))
+it#it_pron))))))
@@ -1209 +1209 @@
-There))
+There#there_adv))
@@ -1211 +1211 @@
-is)
+is#be_v)
@@ -1214 +1214 @@
-mention)
+mention#mention_n)
@@ -1217 +1217 @@
-of)
+of#of_prep)
@@ -1224 +1224 @@
-Friendship)
+Friendship#friendship_n)
@@ -1227 +1227 @@
-with)
+with#with_prep)
@@ -1230 +1230 @@
-the)
+the#the_adj)
@@ -1232 +1232 @@
-world))))))
+world#world_n))))))
@@ -1244 +1244 @@
-and)
+and#and_conj1)
@@ -1247 +1247 @@
-it))
+it#it_pron))
@@ -1249 +1249 @@
-is)
+is#be_v)
@@ -1251 +1251 @@
-said)
+said#say_v1)
@@ -1256 +1256 @@
-to)
+to#to_prep)
@@ -1258 +1258 @@
-be)
+be#be_v)
@@ -1263 +1263 @@
-enmity)
+enmity#enmity_n)
@@ -1266 +1266 @@
-with)
+with#with_prep)
@@ -1269 +1269 @@
-God)))))
+God#NA)))))
@@ -1279 +1279 @@
-but)
+but#but_prep)
@@ -1282 +1282 @@
-the)
+the#the_adj)
@@ -1284 +1284 @@
-word))
+word#word_n))
@@ -1286 +1286 @@
-is)
+is#be_v)
@@ -1289 +1289 @@
-no)
+no#no_adj)
@@ -1291 +1291 @@
-where)
+where#where_adv)
@@ -1294 +1294 @@
-else)))
+else#else_adv)))
@@ -1296 +1296 @@
-named)
+named#name_v)
@@ -1301 +1301 @@
-or)
+or#or_conj2)
@@ -1304 +1304 @@
-to)
+to#to_prep)
@@ -1307 +1307 @@
-any)
+any#any_adj)
@@ -1309 +1309 @@
-other)
+other#other_adj)
@@ -1311 +1311 @@
-purpose))))
+purpose#purpose_n))))
@@ -1314 +1314 @@
-in)
+in#in_prep)
@@ -1317 +1317 @@
-all)
+all#all_adj)
@@ -1319 +1319 @@
-the)
+the#the_adj)
@@ -1321 +1321 @@
-New)
+New#NA)
@@ -1323 +1323 @@
-Testament)))
+Testament#NA)))
@@ -1332 +1332 @@
-It))
+It#it_pron))
@@ -1334 +1334 @@
-speaks)
+speaks#speak_v)
@@ -1337 +1337 @@
-of)
+of#of_prep)
@@ -1340 +1340 @@
-Friends)))
+Friends#friend_n)))
@@ -1343 +1343 @@
-often))
+often#often_adv))
@@ -1351 +1351 @@
-but)
+but#but_prep)
@@ -1354 +1354 @@
-by)
+by#by_prep)
@@ -1359 +1359 @@
-Friends)))
+Friends#friend_n)))
@@ -1363 +1363 @@
-are)
+are#be_v)
@@ -1365 +1365 @@
-meant)
+meant#mean_v1)
@@ -1369 +1369 @@
-our)
+our#our_pron)
@@ -1371 +1371 @@
-acquaintance))
+acquaintance#acquaintance_n))
@@ -1376 +1376 @@
-or)
+or#or_conj2)
@@ -1379 +1379 @@
-our)
+our#our_pron)
@@ -1381 +1381 @@
-Kindred)))
+Kindred#kindred_n)))
@@ -1387 +1387 @@
-the)
+the#the_adj)
@@ -1389 +1389 @@
-relatives)
+relatives#relative_n)
@@ -1392 +1392 @@
-of)
+of#of_prep)
@@ -1396 +1396 @@
-our)
+our#our_pron)
@@ -1398 +1398 @@
-family))
+family#family_n))
@@ -1401 +1401 @@
-or)
+or#or_conj2)
@@ -1404 +1404 @@
-our)
+our#our_pron)
@@ -1406 +1406 @@
-fortune)))))))
+fortune#fortune_n)))))))
@@ -1411 +1411 @@
-or)
+or#or_conj2)
@@ -1414 +1414 @@
-our)
+our#our_pron)
@@ -1416 +1416 @@
-sect))))
+sect#sect_n1))))
@@ -1426 +1426 @@
-something)
+something#something_n)
@@ -1429 +1429 @@
-of)
+of#of_prep)
@@ -1432 +1432 @@
-society))))
+society#society_n))))
@@ -1437 +1437 @@
-or)
+or#or_conj2)
@@ -1440 +1440 @@
-something)
+something#something_n)
@@ -1443 +1443 @@
-of)
+of#of_prep)
@@ -1446 +1446 @@
-kindness)))))
+kindness#kindness_n)))))
@@ -1451 +1451 @@
-there))
+there#there_adv))
@@ -1453 +1453 @@
-is)
+is#be_v)
@@ -1456 +1456 @@
-in)
+in#in_prep)
@@ -1459 +1459 @@
-it)))
+it#it_pron)))
@@ -1465 +1465 @@
-a)
+a#a_adj2)
@@ -1467 +1467 @@
-tenderness)
+tenderness#tenderness_n)
@@ -1470 +1470 @@
-of)
+of#of_prep)
@@ -1473 +1473 @@
-appellation)
+appellation#appellation_n)
@@ -1475 +1475 @@
-and)
+and#and_conj1)
@@ -1477 +1477 @@
-civility))))
+civility#civility_n))))
@@ -1483 +1483 @@
-a)
+a#a_adj2)
@@ -1485 +1485 @@
-relation)
+relation#relation_n)
@@ -1488 +1488 @@
-made)
+made#make_v1)
@@ -1492 +1492 @@
-by)
+by#by_prep)
@@ -1495 +1495 @@
-gifts)))
+gifts#gift_n)))
@@ -1500 +1500 @@
-or)
+or#or_conj2)
@@ -1503 +1503 @@
-by)
+by#by_prep)
@@ -1506 +1506 @@
-duty))))
+duty#duty_n))))
@@ -1512 +1512 @@
-by)
+by#by_prep)
@@ -1515 +1515 @@
-services)
+services#service_n1)
@@ -1517 +1517 @@
-and)
+and#and_conj1)
@@ -1519 +1519 @@
-subjection)))))))))
+subjection#subjection_n)))))))))
@@ -1527 +1527 @@
-and)
+and#and_conj1)
@@ -1530 +1530 @@
-I))
+I#i_pron))
@@ -1532 +1532 @@
-think)
+think#think_v2)
@@ -1541 +1541 @@
-I))
+I#i_pron))
@@ -1543 +1543 @@
-have)
+have#have_v)
@@ -1546 +1546 @@
-reason)
+reason#reason_n1)
@@ -1549 +1549 @@
-to)
+to#to_prep)
@@ -1551 +1551 @@
-be)
+be#be_v)
@@ -1554 +1554 @@
-confident)
+confident#confident_adj)
@@ -1559 +1559 @@
-that)
+that#that_conj)
@@ -1563 +1563 @@
-the)
+the#the_adj)
@@ -1565 +1565 @@
-word)
+word#word_n)
@@ -1569 +1569 @@
-Friend))
+Friend#friend_n))
@@ -1576 +1576 @@
-speaking)
+speaking#speak_v)
@@ -1579 +1579 @@
-of)
+of#of_prep)
@@ -1582 +1582 @@
-humane)
+humane#human_adj)
@@ -1584 +1584 @@
-intercourse))))
+intercourse#intercourse_n))))
@@ -1590 +1590 @@
-is)
+is#be_v)
@@ -1592 +1592 @@
-no)
+no#no_adv2)
@@ -1595 +1595 @@
-other-wayes))
+other-wayes#otherways_adv))
@@ -1597 +1597 @@
-used)
+used#use_v)
@@ -1600 +1600 @@
-in)
+in#in_prep)
@@ -1604 +1604 @@
-the)
+the#the_adj)
@@ -1606 +1606 @@
-Gospels)
+Gospels#NA)
@@ -1609 +1609 @@
-or)
+or#or_conj2)
@@ -1612 +1612 @@
-Epistles))))
+Epistles#NA))))
@@ -1617 +1617 @@
-or)
+or#or_conj2)
@@ -1620 +1620 @@
-Acts)
+Acts#NA)
@@ -1623 +1623 @@
-of)
+of#of_prep)
@@ -1626 +1626 @@
-the)
+the#the_adj)
@@ -1628 +1628 @@
-Apostles))))))))))))))
+Apostles#NA))))))))))))))
@@ -1636 +1636 @@
-and)
+and#and_conj1)
@@ -1639 +1639 @@
-the)
+the#the_adj)
@@ -1641 +1641 @@
-reason)
+reason#reason_n1)
@@ -1644 +1644 @@
-of)
+of#of_prep)
@@ -1647 +1647 @@
-it))))
+it#it_pron))))
@@ -1649 +1649 @@
-is)
+is#be_v)
@@ -1658 +1658 @@
-the)
+the#the_adj)
@@ -1660 +1660 @@
-word)
+word#word_n)
@@ -1664 +1664 @@
-Friend))
+Friend#friend_n))
@@ -1668 +1668 @@
-is)
+is#be_v)
@@ -1671 +1671 @@
-of)
+of#of_prep)
@@ -1674 +1674 @@
-a)
+a#a_adj2)
@@ -1676 +1676 @@
-large)
+large#large_adj)
@@ -1678 +1678 @@
-signification)))))
+signification#signification_n)))))
@@ -1686 +1686 @@
-and)
+and#and_conj1)
@@ -1690 +1690 @@
-means)
+means#mean_v1)
@@ -1694 +1694 @@
-all)
+all#all_adj)
@@ -1696 +1696 @@
-relations)
+relations#relation_n)
@@ -1699 +1699 @@
-and)
+and#and_conj1)
@@ -1702 +1702 @@
-societies))))
+societies#society_n))))
@@ -1707 +1707 @@
-and)
+and#and_conj1)
@@ -1712 +1712 @@
-whatsoever))
+whatsoever#whatsoever_pron))
@@ -1719 +1719 @@
-is)
+is#be_v)
@@ -1721 +1721 @@
-not)
+not#not_adv)
@@ -1726 +1726 @@
-enemy)))))))
+enemy#enemy_n)))))))
@@ -1736 +1736 @@
-but)
+but#but_prep)
@@ -1739 +1739 @@
-by)
+by#by_prep)
@@ -1744 +1744 @@
-friendships)))
+friendships#friendship_n)))
@@ -1751 +1751 @@
-I))
+I#i_pron))
@@ -1753 +1753 @@
-suppose)
+suppose#suppose_v)
@@ -1760 +1760 @@
-you))
+you#you_pron))
@@ -1762 +1762 @@
-mean)
+mean#mean_v1)
@@ -1770 +1770 @@
-the)
+the#the_adj)
@@ -1772 +1772 @@
-greatest)
+greatest#great_adj)
@@ -1774 +1774 @@
-love))
+love#love_n1))
@@ -1779 +1779 @@
-and)
+and#and_conj1)
@@ -1782 +1782 @@
-the)
+the#the_adj)
@@ -1784 +1784 @@
-greatest)
+greatest#great_adj)
@@ -1786 +1786 @@
-usefulnesse)))
+usefulnesse#usefulness_n)))
@@ -1791 +1791 @@
-and)
+and#and_conj1)
@@ -1794 +1794 @@
-the)
+the#the_adj)
@@ -1797 +1797 @@
-most)
+most#most_adj)
@@ -1799 +1799 @@
-open))
+open#open_adj))
@@ -1801 +1801 @@
-communication)))
+communication#communication_n)))
@@ -1806 +1806 @@
-and)
+and#and_conj1)
@@ -1809 +1809 @@
-the)
+the#the_adj)
@@ -1811 +1811 @@
-noble)
+noble#noble_adj)
@@ -1813 +1813 @@
-sufferings)))
+sufferings#suffering_n)))
@@ -1818 +1818 @@
-and)
+and#and_conj1)
@@ -1821 +1821 @@
-the)
+the#the_adj)
@@ -1824 +1824 @@
-most)
+most#most_adj)
@@ -1826 +1826 @@
-exemplar))
+exemplar#exemplar_adj))
@@ -1828 +1828 @@
-faithfulness)))
+faithfulness#faithfulness_n)))
@@ -1833 +1833 @@
-and)
+and#and_conj1)
@@ -1836 +1836 @@
-the)
+the#the_adj)
@@ -1838 +1838 @@
-severest)
+severest#severe_adj)
@@ -1840 +1840 @@
-truth)))
+truth#truth_n)))
@@ -1845 +1845 @@
-and)
+and#and_conj1)
@@ -1848 +1848 @@
-the)
+the#the_adj)
@@ -1850 +1850 @@
-heartest)
+heartest#hard_adj)
@@ -1854 +1854 @@
-counsel)))
+counsel#counsel_n)))
@@ -1859 +1859 @@
-and)
+and#and_conj1)
@@ -1862 +1862 @@
-the)
+the#the_adj)
@@ -1864 +1864 @@
-greatest)
+greatest#great_adj)
@@ -1866 +1866 @@
-union)
+union#union_n1)
@@ -1869 +1869 @@
-of)
+of#of_prep)
@@ -1872 +1872 @@
-minds)))))
+minds#mind_n1)))))
@@ -1880 +1880 @@
-of)
+of#of_prep)
@@ -1883 +1883 @@
-which)))
+which#which_adj)))
@@ -1889 +1889 @@
-brave)
+brave#brave_adj)
@@ -1891 +1891 @@
-men)
+men#man_n1)
@@ -1894 +1894 @@
-and)
+and#and_conj1)
@@ -1897 +1897 @@
-women))))
+women#woman_n))))
@@ -1899 +1899 @@
-are)
+are#be_v)
@@ -1902 +1902 @@
-capable)
+capable#capable_adj)
@@ -1912 +1912 @@
-But)
+But#but_prep)
@@ -1915 +1915 @@
-then))
+then#then_adv))
@@ -1918 +1918 @@
-I))
+I#i_pron))
@@ -1920 +1920 @@
-must)
+must#must_v1)
@@ -1922 +1922 @@
-tell)
+tell#tell_v)
@@ -1925 +1925 @@
-you))
+you#you_pron))
@@ -1928 +1928 @@
-that)
+that#that_conj)
@@ -1932 +1932 @@
-Christianity))
+Christianity#christianity_n))
@@ -1934 +1934 @@
-hath)
+hath#have_v)
@@ -1937 +1937 @@
-new))
+new#new_adv))
@@ -1939 +1939 @@
-christened)
+christened#christen_v)
@@ -1942 +1942 @@
-it))))
+it#it_pron))))
@@ -1950 +1950 @@
-and)
+and#and_conj1)
@@ -1954 +1954 @@
-calls)
+calls#call_v)
@@ -1958 +1958 @@
-this))
+this#this_pron))
@@ -1963 +1963 @@
-Charity)))
+Charity#charity_n)))
@@ -1974 +1974 @@
-The)
+The#the_adj)
@@ -1976 +1976 @@
-Christian))
+Christian#NA))
@@ -1978 +1978 @@
-knowes)
+knowes#know_v)
@@ -1981 +1981 @@
-no)
+no#no_adj)
@@ -1983 +1983 @@
-enemy)
+enemy#enemy_n)
@@ -1994 +1994 @@
-he))
+he#he_pron))
@@ -1996 +1996 @@
-hath))))
+hath#have_v))))
@@ -2006 +2006 @@
-that))
+that#that_pron))
@@ -2008 +2008 @@
-is))
+is#be_v))
@@ -2013 +2013 @@
-though)
+though#though_adv)
@@ -2020 +2020 @@
-persons))
+persons#person_n))
@@ -2022 +2022 @@
-may)
+may#may_v1)
@@ -2024 +2024 @@
-be)
+be#be_v)
@@ -2028 +2028 @@
-injurious)
+injurious#injurious_adj)
@@ -2031 +2031 @@
-to)
+to#to_prep)
@@ -2034 +2034 @@
-him))))
+him#him_pron))))
@@ -2039 +2039 @@
-and)
+and#and_conj1)
@@ -2042 +2042 @@
-unworthy)
+unworthy#unworthy_adj)
@@ -2045 +2045 @@
-in)
+in#in_prep)
@@ -2048 +2048 @@
-themselves)))))))))
+themselves#themselves_pron)))))))))
@@ -2053 +2053 @@
-yet))
+yet#yet_adv))
@@ -2056 +2056 @@
-he))
+he#he_pron))
@@ -2058 +2058 @@
-knowes)
+knowes#know_v)
@@ -2061 +2061 @@
-none)
+none#none_pron)
@@ -2065 +2065 @@
-whom))
+whom#whom_pron))
@@ -2071 +2071 @@
-he))
+he#he_pron))
@@ -2073 +2073 @@
-is)
+is#be_v)
@@ -2075 +2075 @@
-not)
+not#not_adv)
@@ -2078 +2078 @@
-first))
+first#first_adj))
@@ -2080 +2080 @@
-bound)
+bound#bind_v)
@@ -2085 +2085 @@
-to)
+to#to_prep)
@@ -2087 +2087 @@
-forgive)
+forgive#forgive_v)
@@ -2093 +2093 @@
-which))
+which#which_adj))
@@ -2100 +2100 @@
-is)
+is#be_v)
@@ -2103 +2103 @@
-indeed))
+indeed#indeed_adv))
@@ -2108 +2108 @@
-to)
+to#to_prep)
@@ -2110 +2110 @@
-make)
+make#make_v1)
@@ -2114 +2114 @@
-them))
+them#them_pron))
@@ -2117 +2117 @@
-on)
+on#on_prep)
@@ -2120 +2120 @@
-his)
+his#his_adj)
@@ -2122 +2122 @@
-part)))
+part#part_n)))
@@ -2124 +2124 @@
-to)
+to#to_prep)
@@ -2126 +2126 @@
-be)
+be#be_v)
@@ -2129 +2129 @@
-no)
+no#no_adj)
@@ -2131 +2131 @@
-enemies))))))))))
+enemies#enemy_n))))))))))
@@ -2140 +2140 @@
-that))
+that#that_pron))
@@ -2142 +2142 @@
-is)
+is#be_v)
@@ -2147 +2147 @@
-to)
+to#to_prep)
@@ -2149 +2149 @@
-make)
+make#make_v1)
@@ -2152 +2152 @@
-that)
+that#that_conj)
@@ -2156 +2156 @@
-the)
+the#the_adj)
@@ -2158 +2158 @@
-word)
+word#word_n)
@@ -2162 +2162 @@
-enemy))
+enemy#enemy_n))
@@ -2166 +2166 @@
-shall)
+shall#shall_v)
@@ -2168 +2168 @@
-not)
+not#not_adv)
@@ -2170 +2170 @@
-be)
+be#be_v)
@@ -2173 +2173 @@
-perfectly))
+perfectly#perfectly_adv))
@@ -2176 +2176 @@
-contrary)
+contrary#contrary_adj)
@@ -2179 +2179 @@
-to)
+to#to_prep)
@@ -2184 +2184 @@
-Friend)))))))
+Friend#friend_n)))))))
@@ -2196 +2196 @@
-it))
+it#it_pron))
@@ -2198 +2198 @@
-shall)
+shall#shall_v)
@@ -2200 +2200 @@
-not)
+not#not_adv)
@@ -2202 +2202 @@
-be)
+be#be_v)
@@ -2205 +2205 @@
-a)
+a#a_adj2)
@@ -2207 +2207 @@
-relative)
+relative#relative_n)
@@ -2209 +2209 @@
-term)))
+term#term_n)))
@@ -2212 +2212 @@
-and)
+and#and_conj1)
@@ -2215 +2215 @@
-signifie)
+signifie#signify_v)
@@ -2218 +2218 @@
-something)
+something#something_n)
@@ -2221 +2221 @@
-on)
+on#on_prep)
@@ -2224 +2224 @@
-each)
+each#each_adj)
@@ -2226 +2226 @@
-hand)))
+hand#hand_n)))
@@ -2231 +2231 @@
-a)
+a#a_adj2)
@@ -2235 +2235 @@
-relative)
+relative#relative_n)
@@ -2240 +2240 @@
-and)
+and#and_conj1)
@@ -2243 +2243 @@
-a)
+a#a_adj2)
@@ -2247 +2247 @@
-correlative)))))))
+correlative#correlative_adj)))))))
@@ -2257 +2257 @@
-and)
+and#and_conj1)
@@ -2260 +2260 @@
-then))
+then#then_adv))
@@ -2263 +2263 @@
-he))
+he#he_pron))
@@ -2265 +2265 @@
-knowes)
+knowes#know_v)
@@ -2268 +2268 @@
-none)
+none#none_pron)
@@ -2272 +2272 @@
-whom))
+whom#whom_pron))
@@ -2278 +2278 @@
-he))
+he#he_pron))
@@ -2280 +2280 @@
-is)
+is#be_v)
@@ -2282 +2282 @@
-not)
+not#not_adv)
@@ -2284 +2284 @@
-bound)
+bound#bind_v)
@@ -2290 +2290 @@
-to)
+to#to_prep)
@@ -2292 +2292 @@
-love))
+love#love_v1))
@@ -2295 +2295 @@
-&)
+&#and_conj1)
@@ -2300 +2300 @@
-pray)
+pray#pray_v)
@@ -2303 +2303 @@
-for)
+for#for_prep)
@@ -2313 +2313 @@
-to)
+to#to_prep)
@@ -2315 +2315 @@
-treat)
+treat#treat_v)
@@ -2319,3 +2319,3 @@
-kindly)
-(CONJ
-and)
+kindly#kindly_adv)
+(CONJ
+and#and_conj1)
@@ -2323 +2323 @@
-justly))
+justly#justly_adv))
@@ -2329 +2329 @@
-liberally)
+liberally#liberally_adv)
@@ -2333 +2333 @@
-and)
+and#and_conj1)
@@ -2335 +2335 @@
-obligingly))))))))))
+obligingly#obligingly_adv))))))))))
@@ -2344 +2344 @@
-Christian)
+Christian#christian_adj)
@@ -2346 +2346 @@
-Charity))
+Charity#charity_n))
@@ -2348 +2348 @@
-is)
+is#be_v)
@@ -2351 +2351 @@
-Friendship)
+Friendship#friendship_n)
@@ -2354 +2354 @@
-to)
+to#to_prep)
@@ -2357 +2357 @@
-all)
+all#all_adj)
@@ -2359 +2359 @@
-the)
+the#the_adj)
@@ -2361 +2361 @@
-world))))
+world#world_n))))
@@ -2369 +2369 @@
-and)
+and#and_conj1)
@@ -2372 +2372 @@
-when)
+when#when_adv)
@@ -2379 +2379 @@
-Friendships))
+Friendships#friendship_n))
@@ -2381 +2381 @@
-were)
+were#be_v)
@@ -2384 +2384 @@
-the)
+the#the_adj)
@@ -2386 +2386 @@
-noblest)
+noblest#noble_adj)
@@ -2388 +2388 @@
-things))
+things#thing_n1))
@@ -2391 +2391 @@
-in)
+in#in_prep)
@@ -2394 +2394 @@
-the)
+the#the_adj)
@@ -2396 +2396 @@
-world))))))
+world#world_n))))))
@@ -2401 +2401 @@
-Charity))
+Charity#charity_n))
@@ -2403 +2403 @@
-was)
+was#be_v)
@@ -2406 +2406 @@
-little))
+little#little_adj))
@@ -2411 +2411 @@
-like)
+like#like_adj)
@@ -2415 +2415 @@
-the)
+the#the_adj)
@@ -2417 +2417 @@
-Sun)
+Sun#sun_n)
@@ -2420 +2420 @@
-drawn)
+drawn#draw_v)
@@ -2422 +2422 @@
-in)
+in#in_adv)
@@ -2425 +2425 @@
-at)
+at#at_prep)
@@ -2428 +2428 @@
-a)
+a#a_adj2)
@@ -2430 +2430 @@
-chink)))))
+chink#chink_n2)))))
@@ -2435 +2435 @@
-or)
+or#or_conj2)
@@ -2438 +2438 @@
-his)
+his#his_adj)
@@ -2440 +2440 @@
-beams)
+beams#beam_n1)
@@ -2443 +2443 @@
-drawn)
+drawn#draw_v)
@@ -2446 +2446 @@
-into)
+into#into_prep)
@@ -2449 +2449 @@
-the)
+the#the_adj)
@@ -2451 +2451 @@
-centre)
+centre#centre_n1)
@@ -2454 +2454 @@
-of)
+of#of_prep)
@@ -2457 +2457 @@
-a)
+a#a_adj2)
@@ -2459 +2459 @@
-Burning)
+Burning#burning_n)
@@ -2461 +2461 @@
-glass))))))))))
+glass#glass_n1))))))))))
@@ -2469 +2469 @@
-but)
+but#but_prep)
@@ -2472 +2472 @@
-Christian)
+Christian#christian_adj)
@@ -2474 +2474 @@
-Charity))
+Charity#charity_n))
@@ -2476 +2476 @@
-is)
+is#be_v)
@@ -2479 +2479 @@
-Friendship)
+Friendship#friendship_n)
@@ -2484 +2484 @@
-expanded)
+expanded#expand_v)
@@ -2487 +2487 @@
-like)
+like#like_adj)
@@ -2490 +2490 @@
-the)
+the#the_adj)
@@ -2492 +2492 @@
-face)
+face#face_n)
@@ -2495 +2495 @@
-of)
+of#of_prep)
@@ -2498 +2498 @@
-the)
+the#the_adj)
@@ -2500 +2500 @@
-Sun)
+Sun#sun_n)
@@ -2503 +2503 @@
-when)
+when#when_adv)
@@ -2510 +2510 @@
-it))
+it#it_pron))
@@ -2512 +2512 @@
-mounts)
+mounts#mount_v)
@@ -2515 +2515 @@
-above)
+above#above_adv)
@@ -2518 +2518 @@
-the)
+the#the_adj)
@@ -2520 +2520 @@
-Eastern)
+Eastern#eastern_adj)
@@ -2522 +2522 @@
-hills))))))))))))
+hills#hill_n))))))))))))
@@ -2530 +2530 @@
-and)
+and#and_conj1)
@@ -2533 +2533 @@
-I))
+I#i_pron))
@@ -2535 +2535 @@
-was)
+was#be_v)
@@ -2538 +2538 @@
-strangely))
+strangely#strangely_adv))
@@ -2540 +2540 @@
-pleased)
+pleased#please_v)
@@ -2543 +2543 @@
-when)
+when#when_adv)
@@ -2550 +2550 @@
-I))
+I#i_pron))
@@ -2552 +2552 @@
-saw)
+saw#see_v)
@@ -2555 +2555 @@
-something)
+something#something_n)
@@ -2558 +2558 @@
-of)
+of#of_prep)
@@ -2561 +2561 @@
-this))))
+this#this_pron))))
@@ -2564 +2564 @@
-in)
+in#in_prep)
@@ -2567 +2567 @@
-Cicero))))))
+Cicero#NA))))))
@@ -2575 +2575 @@
-for)
+for#for_prep)
@@ -2578 +2578 @@
-I))
+I#i_pron))
@@ -2580 +2580 @@
-have)
+have#have_v)
@@ -2582 +2582 @@
-been)
+been#be_v)
@@ -2585 +2585 @@
-so)
+so#so_adv)
@@ -2591 +2591 @@
-push'd)
+push'd#push_v)
@@ -2594 +2594 @@
-at)
+at#at_prep)
@@ -2599 +2599 @@
-by)
+by#by_prep)
@@ -2602 +2602 @@
-herds)
+herds#herd_n1)
@@ -2604 +2604 @@
-and)
+and#and_conj1)
@@ -2606 +2606 @@
-flockes)
+flockes#flock_n1)
@@ -2609 +2609 @@
-of)
+of#of_prep)
@@ -2612 +2612 @@
-people)
+people#people_n)
@@ -2617 +2617 @@
-that)
+that#that_conj)
@@ -2622 +2622 @@
-follow)
+follow#follow_v)
@@ -2625 +2625 @@
-any)
+any#any_adj)
@@ -2627 +2627 @@
-body)
+body#body_n)
@@ -2632 +2632 @@
-that)
+that#that_conj)
@@ -2638 +2638 @@
-whistles)
+whistles#whistle_v)
@@ -2641 +2641 @@
-to)
+to#to_prep)
@@ -2644 +2644 @@
-them))))
+them#them_pron))))
@@ -2649 +2649 @@
-or)
+or#or_conj2)
@@ -2654 +2654 @@
-drives)
+drives#drive_v)
@@ -2657 +2657 @@
-them))
+them#them_pron))
@@ -2660 +2660 @@
-to)
+to#to_prep)
@@ -2663 +2663 @@
-pasture))))))))))))))
+pasture#pasture_n))))))))))))))
@@ -2668 +2668 @@
-that)
+that#that_conj)
@@ -2672 +2672 @@
-I))
+I#i_pron))
@@ -2674 +2674 @@
-am)
+am#be_v)
@@ -2676 +2676 @@
-grown)
+grown#grow_v)
@@ -2679 +2679 @@
-afraid)
+afraid#afraid_adj)
@@ -2682 +2682 @@
-of)
+of#of_prep)
@@ -2685 +2685 @@
-any)
+any#any_adj)
@@ -2687 +2687 @@
-Truth)
+Truth#truth_n)
@@ -2692 +2692 @@
-that)
+that#that_conj)
@@ -2697 +2697 @@
-seems)
+seems#seem_v2)
@@ -2700 +2700 @@
-chargeable)
+chargeable#chargeable_adj)
@@ -2703 +2703 @@
-with)
+with#with_prep)
@@ -2706 +2706 @@
-singularity)))))))))))
+singularity#NA)))))))))))
@@ -2714 +2714 @@
-but)
+but#but_prep)
@@ -2717 +2717 @@
-therefore)
+therefore#therefore_adv)
@@ -2724 +2724 @@
-I))
+I#i_pron))
@@ -2726 +2726 @@
-say)
+say#say_v1)
@@ -2735 +2735 @@
-glad))
+glad#glad_adj))
@@ -2738 +2738 @@
-I))
+I#i_pron))
@@ -2740 +2740 @@
-was)
+was#be_v)
@@ -2743 +2743 @@
-when)
+when#when_adv)
@@ -2750 +2750 @@
-I))
+I#i_pron))
@@ -2752 +2752 @@
-saw)
+saw#see_v)
@@ -2758 +2758 @@
-Laelius))
+Laelius#NA))
@@ -2763 +2763 @@
-in)
+in#in_prep)
@@ -2768 +2768 @@
-Cicero)))
+Cicero#NA)))
@@ -2772 +2772 @@
-discourse)
+discourse#discourse_v)
@@ -2775 +2775 @@
-thus))
+thus#thus_adv))
@@ -2783 +2783 @@
-Amicitia)
+Amicitia#NA)
@@ -2785 +2785 @@
-ex)
+ex#NA)
@@ -2787 +2787 @@
-infinitate)
+infinitate#NA)
@@ -2789 +2789 @@
-generis)
+generis#NA)
@@ -2791 +2791 @@
-humani)
+humani#NA)
@@ -2793 +2793 @@
-quam)
+quam#NA)
@@ -2795 +2795 @@
-conciliavit)
+conciliavit#NA)
@@ -2797 +2797 @@
-ipsa)
+ipsa#NA)
@@ -2799 +2799 @@
-natura)
+natura#NA)
@@ -2803 +2803 @@
-contracta)
+contracta#NA)
@@ -2805 +2805 @@
-res)
+res#NA)
@@ -2807 +2807 @@
-est)
+est#NA)
@@ -2811 +2811 @@
-&)
+&#NA)
@@ -2813 +2813 @@
-adducta)
+adducta#NA)
@@ -2815 +2815 @@
-in)
+in#NA)
@@ -2817 +2817 @@
-angustum)
+angustum#NA)
@@ -2821 +2821 @@
-ut)
+ut#NA)
@@ -2823 +2823 @@
-omnis)
+omnis#NA)
@@ -2825 +2825 @@
-charitas)
+charitas#NA)
@@ -2829 +2829 @@
-aut)
+aut#NA)
@@ -2831 +2831 @@
-inter)
+inter#NA)
@@ -2833 +2833 @@
-duos)
+duos#NA)
@@ -2837 +2837 @@
-aut)
+aut#NA)
@@ -2839 +2839 @@
-inter)
+inter#NA)
@@ -2841 +2841 @@
-paucos)
+paucos#NA)
@@ -2843 +2843 @@
-jungeretur))))))))))))
+jungeretur#NA))))))))))))
@@ -2854 +2854 @@
-Nature))
+Nature#nature_n))
@@ -2856 +2856 @@
-hath)
+hath#have_v)
@@ -2858 +2858 @@
-made)
+made#make_v1)
@@ -2861 +2861 @@
-Friendships)
+Friendships#friendship_n)
@@ -2865 +2865 @@
-and)
+and#and_conj1)
@@ -2867 +2867 @@
-societies)
+societies#society_n)
@@ -2873 +2873 @@
-relations)
+relations#relation_n)
@@ -2875 +2875 @@
-and)
+and#and_conj1)
@@ -2877 +2877 @@
-endearments))
+endearments#endearment_n))
@@ -2885 +2885 @@
-and)
+and#and_conj1)
@@ -2888 +2888 @@
-by)
+by#by_prep)
@@ -2891,3 +2891,3 @@
-something)
-(CONJ
-or)
+something#something_n)
+(CONJ
+or#or_conj2)
@@ -2895 +2895 @@
-other)))
+other#other_adj)))
@@ -2898 +2898 @@
-we))
+we#we_pron))
@@ -2900 +2900 @@
-relate)
+relate#relate_v)
@@ -2903 +2903 @@
-to)
+to#to_prep)
@@ -2906 +2906 @@
-all)
+all#all_adj)
@@ -2908 +2908 @@
-the)
+the#the_adj)
@@ -2910 +2910 @@
-world)))
+world#world_n)))
@@ -2919 +2919 @@
-there))
+there#there_adv))
@@ -2921 +2921 @@
-is)
+is#be_v)
@@ -2924 +2924 @@
-enough)
+enough#enough_adj)
@@ -2929 +2929 @@
-in)
+in#in_prep)
@@ -2932 +2932 @@
-every)
+every#every_adj)
@@ -2934 +2934 @@
-man)))
+man#man_n1)))
@@ -2939 +2939 @@
-that)
+that#that_conj)
@@ -2944 +2944 @@
-is)
+is#be_v)
@@ -2947 +2947 @@
-willing)
+willing#willing_adj)
@@ -2952 +2952 @@
-to)
+to#to_prep)
@@ -2954 +2954 @@
-make)
+make#make_v1)
@@ -2958 +2958 @@
-him))
+him#him_pron))
@@ -2960 +2960 @@
-becom)
+becom#become_v)
@@ -2963 +2963 @@
-our)
+our#our_pron)
@@ -2965 +2965 @@
-friend)))))))
+friend#friend_n)))))))
@@ -2973 +2973 @@
-but)
+but#but_prep)
@@ -2976 +2976 @@
-when)
+when#when_adv)
@@ -2983 +2983 @@
-men))
+men#man_n1))
@@ -2985 +2985 @@
-contract)
+contract#contract_v)
@@ -2988 +2988 @@
-Friendships)))))
+Friendships#friendship_n)))))
@@ -2993 +2993 @@
-they))
+they#they_pron))
@@ -2995 +2995 @@
-inclose)
+inclose#enclose_v)
@@ -2998 +2998 @@
-the)
+the#the_adj)
@@ -3000 +3000 @@
-Commons))
+Commons#common_n1))
@@ -3008 +3008 @@
-and)
+and#and_conj1)
@@ -3013 +3013 @@
-what))
+what#what_pron))
@@ -3019 +3019 @@
-Nature))
+Nature#nature_n))
@@ -3021 +3021 @@
-intended)
+intended#intend_v)
@@ -3029 +3029 @@
-should)
+should#shall_v)
@@ -3031 +3031 @@
-be)
+be#be_v)
@@ -3035 +3035 @@
-every)
+every#every_adj)
@@ -3037 +3037 @@
-mans))))))))
+mans#man_n1))))))))
@@ -3042 +3042 @@
-we))
+we#we_pron))
@@ -3044 +3044 @@
-make)
+make#make_v1)
@@ -3050 +3050 @@
-proper)
+proper#proper_adj)
@@ -3053 +3053 @@
-to)
+to#to_prep)
@@ -3057 +3057 @@
-two)
+two#two_adj)
@@ -3059 +3059 @@
-or)
+or#or_conj2)
@@ -3063 +3063 @@
-three))))))
+three#three_adj))))))
@@ -3072 +3072 @@
-Friendship))
+Friendship#friendship_n))
@@ -3074 +3074 @@
-is)
+is#be_v)
@@ -3077 +3077 @@
-like)
+like#like_adj)
@@ -3081 +3081 @@
-rivers))
+rivers#river_n1))
@@ -3084 +3084 @@
-and)
+and#and_conj1)
@@ -3087 +3087 @@
-the)
+the#the_adj)
@@ -3089 +3089 @@
-strand)
+strand#strand_n1)
@@ -3092 +3092 @@
-of)
+of#of_prep)
@@ -3095 +3095 @@
-seas)))))
+seas#sea_n)))))
@@ -3100 +3100 @@
-and)
+and#and_conj1)
@@ -3103 +3103 @@
-the)
+the#the_adj)
@@ -3105 +3105 @@
-ayre)))))
+ayre#air_n)))))
@@ -3110 +3110 @@
-common)
+common#common_adj)
@@ -3113 +3113 @@
-to)
+to#to_prep)
@@ -3116 +3116 @@
-all)
+all#all_adj)
@@ -3118 +3118 @@
-the)
+the#the_adj)
@@ -3120 +3120 @@
-world))))
+world#world_n))))
@@ -3128 +3128 @@
-but)
+but#but_prep)
@@ -3132 +3132 @@
-Tyrants))
+Tyrants#tyrant_n))
@@ -3137 +3137 @@
-and)
+and#and_conj1)
@@ -3140 +3140 @@
-civil)
+civil#civil_adj)
@@ -3142 +3142 @@
-Customs)))
+Customs#custom_n)))
@@ -3148 +3148 @@
-Warres)))
+Warres#war_n1)))
@@ -3153 +3153 @@
-and)
+and#and_conj1)
@@ -3156 +3156 @@
-want)
+want#want_n2)
@@ -3159 +3159 @@
-of)
+of#of_prep)
@@ -3162 +3162 @@
-Love))))))
+Love#love_n1))))))
@@ -3164 +3164 @@
-have)
+have#have_v)
@@ -3166 +3166 @@
-made)
+made#make_v1)
@@ -3170 +3170 @@
-them))
+them#them_pron))
@@ -3173 +3173 @@
-proper)
+proper#proper_adj)
@@ -3175 +3175 @@
-and)
+and#and_conj1)
@@ -3177 +3177 @@
-peculiar)))
+peculiar#peculiar_adj)))
@@ -3185 +3185 @@
-But)
+But#but_prep)
@@ -3188 +3188 @@
-when)
+when#when_adv)
@@ -3195 +3195 @@
-Christianity))
+Christianity#christianity_n))
@@ -3197 +3197 @@
-came)
+came#come_v)
@@ -3201 +3201 @@
-to)
+to#to_prep)
@@ -3203 +3203 @@
-renew)
+renew#renew_v1)
@@ -3206 +3206 @@
-our)
+our#our_pron)
@@ -3208 +3208 @@
-nature)))
+nature#nature_n)))
@@ -3213 +3213 @@
-and)
+and#and_conj1)
@@ -3216 +3216 @@
-to)
+to#to_prep)
@@ -3218 +3218 @@
-rest)
+rest#rest_v2)
@@ -3221 +3221 @@
-ore)
+ore#or_adv1)
@@ -3224 +3224 @@
-our)
+our#our_pron)
@@ -3226 +3226 @@
-lawes)))))
+lawes#law_n1)))))
@@ -3231 +3231 @@
-and)
+and#and_conj1)
@@ -3234 +3234 @@
-to)
+to#to_prep)
@@ -3236 +3236 @@
-increase)
+increase#increase_v)
@@ -3239 +3239 @@
-her)
+her#her_pron2)
@@ -3241 +3241 @@
-priviledges))))
+priviledges#privilege_n))))
@@ -3246 +3246 @@
-and)
+and#and_conj1)
@@ -3249 +3249 @@
-to)
+to#to_prep)
@@ -3251 +3251 @@
-make)
+make#make_v1)
@@ -3255 +3255 @@
-her)
+her#her_pron2)
@@ -3257 +3257 @@
-aptness))
+aptness#aptness_n))
@@ -3259 +3259 @@
-to)
+to#to_prep)
@@ -3261 +3261 @@
-becom)
+becom#become_v)
@@ -3264 +3264 @@
-Religion)))))))))
+Religion#religion_n)))))))))
@@ -3269 +3269 @@
-then))
+then#then_adv))
@@ -3272 +3272 @@
-it))
+it#it_pron))
@@ -3274 +3274 @@
-was)
+was#be_v)
@@ -3276 +3276 @@
-declared)
+declared#declare_v)
@@ -3279 +3279 @@
-that)
+that#that_conj)
@@ -3283 +3283 @@
-our)
+our#our_pron)
@@ -3285 +3285 @@
-Friendships))
+Friendships#friendship_n))
@@ -3287 +3287 @@
-were)
+were#be_v)
@@ -3290 +3290 @@
-to)
+to#to_prep)
@@ -3292 +3292 @@
-be)
+be#be_v)
@@ -3295 +3295 @@
-as)
+as#as_adv)
@@ -3297 +3297 @@
-universal)
+universal#universal_adj)
@@ -3300 +3300 @@
-as)
+as#as_adv)
@@ -3303 +3303 @@
-our)
+our#our_pron)
@@ -3305 +3305 @@
-conversation)))))))
+conversation#conversation_n)))))))
@@ -3314 +3314 @@
-that))
+that#that_pron))
@@ -3316 +3316 @@
-is)
+is#be_v)
@@ -3324 +3324 @@
-actual)
+actual#actual_adj)
@@ -3329 +3329 @@
-to)
+to#to_prep)
@@ -3332 +3332 @@
-all)
+all#all_adj)
@@ -3336 +3336 @@
-with)
+with#with_prep)
@@ -3339 +3339 @@
-whom)))
+whom#whom_pron)))
@@ -3347 +3347 @@
-we))
+we#we_pron))
@@ -3349 +3349 @@
-converse))))))
+converse#converse_v))))))
@@ -3354 +3354 @@
-and)
+and#and_conj1)
@@ -3360 +3360 @@
-potentially)
+potentially#potentially_adv)
@@ -3362 +3362 @@
-extended))
+extended#extend_v))
@@ -3367 +3367 @@
-unto)
+unto#unto_prep)
@@ -3370 +3370 @@
-those)
+those#those_pron)
@@ -3374 +3374 @@
-with)
+with#with_prep)
@@ -3377 +3377 @@
-whom)))
+whom#whom_pron)))
@@ -3385 +3385 @@
-we))
+we#we_pron))
@@ -3387 +3387 @@
-did)
+did#do_v)
@@ -3389 +3389 @@
-not)
+not#not_adv)
@@ -3399 +3399 @@
-For)
+For#for_prep)
@@ -3402 +3402 @@
-he)
+he#he_pron)
@@ -3406 +3406 @@
-who))
+who#who_pron))
@@ -3413 +3413 @@
-was)
+was#be_v)
@@ -3416 +3416 @@
-to)
+to#to_prep)
@@ -3418 +3418 @@
-treat)
+treat#treat_v)
@@ -3421 +3421 @@
-his)
+his#his_adj)
@@ -3423 +3423 @@
-enemies))
+enemies#enemy_n))
@@ -3426 +3426 @@
-with)
+with#with_prep)
@@ -3429 +3429 @@
-forgiveness)
+forgiveness#forgiveness_n)
@@ -3431 +3431 @@
-and)
+and#and_conj1)
@@ -3433 +3433 @@
-prayers)
+prayers#prayer_n1)
@@ -3437 +3437 @@
-and)
+and#and_conj1)
@@ -3439 +3439 @@
-love)
+love#love_n1)
@@ -3443 +3443 @@
-and)
+and#and_conj1)
@@ -3445 +3445 @@
-beneficence)))))))
+beneficence#beneficence_n)))))))
@@ -3447 +3447 @@
-was)
+was#be_v)
@@ -3450 +3450 @@
-indeed))
+indeed#indeed_adv))
@@ -3454 +3454 @@
-to)
+to#to_prep)
@@ -3456 +3456 @@
-have)
+have#have_v)
@@ -3459 +3459 @@
-no)
+no#no_adj)
@@ -3461 +3461 @@
-enemies)))
+enemies#enemy_n)))
@@ -3466 +3466 @@
-and)
+and#and_conj1)
@@ -3469 +3469 @@
-to)
+to#to_prep)
@@ -3471 +3471 @@
-have)
+have#have_v)
@@ -3474 +3474 @@
-all)
+all#all_adj)
@@ -3476 +3476 @@
-friends)))))
+friends#friend_n)))))
@@ -3484 +3484 @@
-So)
+So#so_adv)
@@ -3487 +3487 @@
-that)
+that#that_conj)
@@ -3491 +3491 @@
-to)
+to#to_prep)
@@ -3494 +3494 @@
-your)
+your#your_pron)
@@ -3496 +3496 @@
-question)
+question#question_n)
@@ -3503 +3503 @@
-how)
+how#how_adv)
@@ -3505 +3505 @@
-far)))
+far#far_adj)))
@@ -3513 +3513 @@
-a)
+a#a_adj2)
@@ -3516 +3516 @@
-Dear)
+Dear#dear_adj1)
@@ -3518 +3518 @@
-&)
+&#and_conj1)
@@ -3520 +3520 @@
-perfect))
+perfect#perfect_adj))
@@ -3522 +3522 @@
-Friendship))
+Friendship#friendship_n))
@@ -3524 +3524 @@
-is)
+is#be_v)
@@ -3526 +3526 @@
-authoriz'd)
+authoriz'd#authorize_v)
@@ -3531 +3531 @@
-by)
+by#by_prep)
@@ -3534 +3534 @@
-the)
+the#the_adj)
@@ -3536 +3536 @@
-principles)
+principles#principle_n)
@@ -3539 +3539 @@
-of)
+of#of_prep)
@@ -3542 +3542 @@
-Christianity)))))))))
+Christianity#christianity_n)))))))))
@@ -3547 +3547 @@
-The)
+The#the_adj)
@@ -3549 +3549 @@
-answer))
+answer#answer_n))
@@ -3551 +3551 @@
-is)
+is#be_v)
@@ -3554 +3554 @@
-ready)
+ready#ready_adj)
@@ -3556 +3556 @@
-and)
+and#and_conj1)
@@ -3558 +3558 @@
-easie))))
+easie#easy_adj))))
@@ -3567 +3567 @@
-It))
+It#it_pron))
@@ -3569 +3569 @@
-is)
+is#be_v)
@@ -3571 +3571 @@
-warranted)
+warranted#warrant_v)
@@ -3574 +3574 @@
-to)
+to#to_prep)
@@ -3576 +3576 @@
-extend)
+extend#extend_v)
@@ -3579 +3579 @@
-to)
+to#to_prep)
@@ -3582 +3582 @@
-all)
+all#all_adj)
@@ -3584 +3584 @@
-Mankind))))
+Mankind#mankind_n))))
@@ -3592 +3592 @@
-and)
+and#and_conj1)
@@ -3596 +3596 @@
-the)
+the#the_adj)
@@ -3598 +3598 @@
-more))
+more#more_adj))
@@ -3609 +3609 @@
-we))
+we#we_pron))
@@ -3611 +3611 @@
-love))))
+love#love_v1))))
@@ -3616 +3616 @@
-the)
+the#the_adj)
@@ -3618 +3618 @@
-better))
+better#better_adj))
@@ -3621 +3621 @@
-we))
+we#we_pron))
@@ -3623 +3623 @@
-are)
+are#be_v)
@@ -3631 +3631 @@
-and)
+and#and_conj1)
@@ -3634 +3634 @@
-the)
+the#the_adj)
@@ -3636 +3636 @@
-greater)
+greater#great_adj)
@@ -3647 +3647 @@
-our)
+our#our_pron)
@@ -3649 +3649 @@
-friendships))
+friendships#friendship_n))
@@ -3651 +3651 @@
-are))))
+are#be_v))))
@@ -3656 +3656 @@
-the)
+the#the_adj)
@@ -3658 +3658 @@
-dearer))
+dearer#dear_adj1))
@@ -3661 +3661 @@
-we))
+we#we_pron))
@@ -3663 +3663 @@
-are)
+are#be_v)
@@ -3666 +3666 @@
-to)
+to#to_prep)
@@ -3669 +3669 @@
-God)))
+God#NA)))
@@ -3677 +3677 @@
-let)
+let#let_v1)
@@ -3681 +3681 @@
-them))
+them#them_pron))
@@ -3683 +3683 @@
-be)
+be#be_v)
@@ -3686 +3686 @@
-as)
+as#as_adv)
@@ -3688 +3688 @@
-Dear)
+Dear#dear_adj1)
@@ -3693 +3693 @@
-and)
+and#and_conj1)
@@ -3695 +3695 @@
-let)
+let#let_v1)
@@ -3699 +3699 @@
-them))
+them#them_pron))
@@ -3701 +3701 @@
-be)
+be#be_v)
@@ -3704 +3704 @@
-as)
+as#as_adv)
@@ -3706 +3706 @@
-perfect))))
+perfect#perfect_adj))))
@@ -3711 +3711 @@
-and)
+and#and_conj1)
@@ -3713 +3713 @@
-let)
+let#let_v1)
@@ -3717 +3717 @@
-them))
+them#them_pron))
@@ -3719 +3719 @@
-be)
+be#be_v)
@@ -3723 +3723 @@
-as)
+as#as_adv)
@@ -3725 +3725 @@
-many)))))
+many#many_adj)))))
@@ -3728 +3728 @@
-as)
+as#as_adv)
@@ -3739 +3739 @@
-you))
+you#you_pron))
@@ -3741 +3741 @@
-can))))))
+can#can_v1))))))
@@ -3750 +3750 @@
-there))
+there#there_adv))
@@ -3752 +3752 @@
-is)
+is#be_v)
@@ -3755 +3755 @@
-no)
+no#no_adj)
@@ -3757 +3757 @@
-danger))
+danger#danger_n))
@@ -3760 +3760 @@
-in)
+in#in_prep)
@@ -3763 +3763 @@
-it)))
+it#it_pron)))
@@ -3772 +3772 @@
-onely)
+onely#only_adj)
@@ -3776 +3776 @@
-where))
+where#where_adv))
@@ -3784 +3784 @@
-the)
+the#the_adj)
@@ -3786 +3786 @@
-restraint))
+restraint#restraint_n))
@@ -3788 +3788 @@
-begins))))
+begins#begin_v1))))
@@ -3793 +3793 @@
-there))
+there#there_adv))
@@ -3795 +3795 @@
-begins)
+begins#begin_v1)
@@ -3798 +3798 @@
-our)
+our#our_pron)
@@ -3800 +3800 @@
-imperfection))
+imperfection#imperfection_n))
@@ -3809 +3809 @@
-it))
+it#it_pron))
@@ -3811 +3811 @@
-is)
+is#be_v)
@@ -3813 +3813 @@
-not)
+not#not_adv)
@@ -3816 +3816 @@
-ill))
+ill#ill_adj))
@@ -3819 +3819 @@
-that)
+that#that_conj)
@@ -3823 +3823 @@
-you))
+you#you_pron))
@@ -3825 +3825 @@
-entertain)
+entertain#entertain_v)
@@ -3829 +3829 @@
-brave)
+brave#brave_adj)
@@ -3831 +3831 @@
-Friendships))
+Friendships#friendship_n))
@@ -3834 +3834 @@
-and)
+and#and_conj1)
@@ -3837 +3837 @@
-worthy)
+worthy#worthy_adj)
@@ -3839 +3839 @@
-societies))))))
+societies#society_n))))))
@@ -3848 +3848 @@
-it))
+it#it_pron))
@@ -3850 +3850 @@
-were)
+were#be_v)
@@ -3853 +3853 @@
-well))
+well#well_adj))
@@ -3856 +3856 @@
-if)
+if#if_conj)
@@ -3863 +3863 @@
-you))
+you#you_pron))
@@ -3865 +3865 @@
-could)
+could#can_v1)
@@ -3869 +3869 @@
-love)
+love#love_v1)
@@ -3876 +3876 @@
-and)
+and#and_conj1)
@@ -3879 +3879 @@
-if)
+if#if_conj)
@@ -3886 +3886 @@
-you))
+you#you_pron))
@@ -3888 +3888 @@
-could)
+could#can_v1)
@@ -3892 +3892 @@
-benefit)))))
+benefit#benefit_v)))))
@@ -3897 +3897 @@
-all)
+all#all_adj)
@@ -3899 +3899 @@
-Mankind)))))
+Mankind#mankind_n)))))
@@ -3907 +3907 @@
-for)
+for#for_prep)
@@ -3910 +3910 @@
-I))
+I#i_pron))
@@ -3912 +3912 @@
-conceive)
+conceive#conceive_v)
@@ -3919 +3919 @@
-that))
+that#that_pron))
@@ -3921 +3921 @@
-is)
+is#be_v)
@@ -3924 +3924 @@
-the)
+the#the_adj)
@@ -3926 +3926 @@
-sum)
+sum#some_pron)
@@ -3929 +3929 @@
-of)
+of#of_prep)
@@ -3932 +3932 @@
-all)
+all#all_adj)
@@ -3934 +3934 @@
-Friendships))))))
+Friendships#friendship_n))))))
@@ -3943 +3943 @@
-I))
+I#i_pron))
@@ -3945 +3945 @@
-confess)
+confess#confess_v)
@@ -3952 +3952 @@
-this))
+this#this_pron))
@@ -3954 +3954 @@
-is)
+is#be_v)
@@ -3956 +3956 @@
-not)
+not#not_adv)
@@ -3959 +3959 @@
-to)
+to#to_prep)
@@ -3961 +3961 @@
-be)
+be#be_v)
@@ -3963 +3963 @@
-expected)
+expected#expect_v)
@@ -3966 +3966 @@
-of)
+of#of_prep)
@@ -3969 +3969 @@
-us)))
+us#us_pron)))
@@ -3972 +3972 @@
-in)
+in#in_prep)
@@ -3975 +3975 @@
-this)
+this#this_pron)
@@ -3977 +3977 @@
-world))))))
+world#world_n))))))
@@ -3985 +3985 @@
-but)
+but#but_prep)
@@ -3988 +3988 @@
-as)
+as#as_adv)
@@ -3999 +3999 @@
-all)
+all#all_adj)
@@ -4001 +4001 @@
-our)
+our#our_pron)
@@ -4003 +4003 @@
-graces)
+graces#grace_n)
@@ -4006 +4006 @@
-here)))
+here#here_adv)))
@@ -4008 +4008 @@
-are)
+are#be_v)
@@ -4011 +4011 @@
-but)
+but#but_prep)
@@ -4013 +4013 @@
-imperfect))
+imperfect#imperfect_adj))
@@ -4019 +4019 @@
-that))
+that#that_pron))
@@ -4021 +4021 @@
-is)
+is#be_v)
@@ -4027 +4027 @@
-at)
+at#at_prep)
@@ -4030 +4030 @@
-the)
+the#the_adj)
@@ -4032 +4032 @@
-best)))
+best#best_adj)))
@@ -4035 +4035 @@
-they))
+they#they_pron))
@@ -4037 +4037 @@
-are)
+are#be_v)
@@ -4040 +4040 @@
-but)
+but#but_prep)
@@ -4042 +4042 @@
-tendencies)
+tendencies#tendency_n)
@@ -4045 +4045 @@
-to)
+to#to_prep)
@@ -4048 +4048 @@
-glory)))))))))
+glory#glory_n)))))))))
@@ -4053 +4053 @@
-so))
+so#so_adv))
@@ -4056 +4056 @@
-our)
+our#our_pron)
@@ -4060 +4060 @@
-Friendships))
+Friendships#friendship_n))
@@ -4062 +4062 @@
-are)
+are#be_v)
@@ -4066 +4066 @@
-imperfect))
+imperfect#imperfect_adj))
@@ -4071 +4071 @@
-too))
+too#too_adv))
@@ -4076 +4076 @@
-and)
+and#and_conj1)
@@ -4079 +4079 @@
-but)
+but#but_prep)
@@ -4081 +4081 @@
-beginnings)
+beginnings#beginning_n)
@@ -4084 +4084 @@
-of)
+of#of_prep)
@@ -4087 +4087 @@
-a)
+a#a_adj2)
@@ -4089 +4089 @@
-coelestial)
+coelestial#celestial_adj)
@@ -4091 +4091 @@
-Friendship)
+Friendship#friendship_n)
@@ -4097 +4097 @@
-by)
+by#by_prep)
@@ -4100 +4100 @@
-which)))
+which#which_adj)))
@@ -4108 +4108 @@
-we))
+we#we_pron))
@@ -4110 +4110 @@
-shall)
+shall#shall_v)
@@ -4112 +4112 @@
-love)
+love#love_v1)
@@ -4115 +4115 @@
-every)
+every#every_adj)
@@ -4117 +4117 @@
-one))
+one#one_adj))
@@ -4121 +4121 @@
-as)
+as#as_adv)
@@ -4123 +4123 @@
-much))
+much#much_adj))
@@ -4126 +4126 @@
-as)
+as#as_adv)
@@ -4137 +4137 @@
-they))
+they#they_pron))
@@ -4139 +4139 @@
-can)
+can#can_v1)
@@ -4141 +4141 @@
-be)
+be#be_v)
@@ -4143 +4143 @@
-loved)))))))))))
+loved#love_v1)))))))))))
@@ -4151 +4151 @@
-But)
+But#but_prep)
@@ -4154 +4154 @@
-then))
+then#then_adv))
@@ -4157 +4157 @@
-so))
+so#so_adv))
@@ -4160 +4160 @@
-we))
+we#we_pron))
@@ -4162 +4162 @@
-must)
+must#must_v1)
@@ -4167 +4167 @@
-here))
+here#here_adv))
@@ -4172 +4172 @@
-in)
+in#in_prep)
@@ -4175 +4175 @@
-our)
+our#our_pron)
@@ -4177 +4177 @@
-proportion)))
+proportion#proportion_n)))
@@ -4187 +4187 @@
-and)
+and#and_conj1)
@@ -4190 +4190 @@
-indeed))
+indeed#indeed_adv))
@@ -4193 +4193 @@
-that))
+that#that_pron))
@@ -4195 +4195 @@
-is)
+is#be_v)
@@ -4198 +4198 @@
-it))
+it#it_pron))
@@ -4203 +4203 @@
-that)
+that#that_conj)
@@ -4208 +4208 @@
-can)
+can#can_v1)
@@ -4210 +4210 @@
-make)
+make#make_v1)
@@ -4213 +4213 @@
-the)
+the#the_adj)
@@ -4215 +4215 @@
-difference))))
+difference#difference_n1))))
@@ -4224 +4224 @@
-we))
+we#we_pron))
@@ -4226 +4226 @@
-must)
+must#must_v1)
@@ -4228 +4228 @@
-be)
+be#be_v)
@@ -4231 +4231 @@
-friends)
+friends#friend_n)
@@ -4234 +4234 @@
-to)
+to#to_prep)
@@ -4237 +4237 @@
-all))))
+all#all_adj))))
@@ -4246 +4246 @@
-That))
+That#that_pron))
@@ -4248 +4248 @@
-is)
+is#be_v)
@@ -4254 +4254 @@
-apt)
+apt#apt_adj)
@@ -4257 +4257 @@
-to)
+to#to_prep)
@@ -4259 +4259 @@
-do)
+do#do_v)
@@ -4262 +4262 @@
-good))))
+good#good_adj))))
@@ -4268 +4268 @@
-loving)
+loving#love_v1)
@@ -4271 +4271 @@
-them))
+them#them_pron))
@@ -4274 +4274 @@
-really))))
+really#really_adv2))))
@@ -4279 +4279 @@
-and)
+and#and_conj1)
@@ -4282 +4282 @@
-doing)
+doing#do_v)
@@ -4285 +4285 @@
-to)
+to#to_prep)
@@ -4288 +4288 @@
-them)))
+them#them_pron)))
@@ -4291 +4291 @@
-all)
+all#all_adj)
@@ -4293 +4293 @@
-the)
+the#the_adj)
@@ -4295 +4295 @@
-benefits)
+benefits#benefit_n)
@@ -4300 +4300 @@
-which))
+which#which_adj))
@@ -4308 +4308 @@
-we))
+we#we_pron))
@@ -4310 +4310 @@
-can)
+can#can_v1)
@@ -4317 +4317 @@
-and)
+and#and_conj1)
@@ -4321 +4321 @@
-which))
+which#which_adj))
@@ -4327 +4327 @@
-they))
+they#they_pron))
@@ -4329 +4329 @@
-are)
+are#be_v)
@@ -4332 +4332 @@
-capable)
+capable#capable_adj)
@@ -4335 +4335 @@
-of)
+of#of_prep)
@@ -4346 +4346 @@
-The)
+The#the_adj)
@@ -4348 +4348 @@
-Friendship))
+Friendship#friendship_n))
@@ -4350 +4350 @@
-is)
+is#be_v)
@@ -4353 +4353 @@
-equall)
+equall#equal_adj)
@@ -4356 +4356 @@
-to)
+to#to_prep)
@@ -4359 +4359 @@
-all)
+all#all_adj)
@@ -4361 +4361 @@
-the)
+the#the_adj)
@@ -4363 +4363 @@
-World))))
+World#world_n))))
@@ -4371 +4371 @@
-and)
+and#and_conj1)
@@ -4374 +4374 @@
-of)
+of#of_prep)
@@ -4377 +4377 @@
-it)
+it#it_pron)
@@ -4379 +4379 @@
-self)))
+self#self_pron)))
@@ -4383 +4383 @@
-hath)
+hath#have_v)
@@ -4386 +4386 @@
-no)
+no#no_adj)
@@ -4388 +4388 @@
-difference))
+difference#difference_n1))
@@ -4396 +4396 @@
-but)
+but#but_prep)
@@ -4400 +4400 @@
-is)
+is#be_v)
@@ -4402 +4402 @@
-differenced)
+differenced#difference_v)
@@ -4406 +4406 @@
-onely)
+onely#only_adj)
@@ -4408 +4408 @@
-by)
+by#by_prep)
@@ -4411 +4411 @@
-accidents)))
+accidents#accident_n)))
@@ -4416 +4416 @@
-and)
+and#and_conj1)
@@ -4419 +4419 @@
-by)
+by#by_prep)
@@ -4422 +4422 @@
-the)
+the#the_adj)
@@ -4424 +4424 @@
-capacity)
+capacity#capacity_n)
@@ -4427 +4427 @@
-or)
+or#or_conj2)
@@ -4430 +4430 @@
-incapacity)))
+incapacity#incapacity_n)))
@@ -4433 +4433 @@
-of)
+of#of_prep)
@@ -4436 +4436 @@
-them)
+them#them_pron)
@@ -4441 +4441 @@
-that)
+that#that_conj)
@@ -4446 +4446 @@
-receive)
+receive#receive_v)
@@ -4449 +4449 @@
-it))))))))))
+it#it_pron))))))))))
