--- jopinney-1688-e3-p1.psd.vert	2025-09-10 13:51:11
+++ jopinney-1688-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-Dear)
+Dear#dear_adj1)
@@ -16 +16 @@
-Hester))
+Hester#NA))
@@ -19 +19 @@
-the)
+the#the_adj)
@@ -22 +22 @@
-many)
+many#many_adj)
@@ -24 +24 @@
-&)
+&#and_conj1)
@@ -26 +26 @@
-suddain))
+suddain#sudden_adj))
@@ -28 +28 @@
-troubles)
+troubles#trouble_n)
@@ -33 +33 @@
-y=t=)
+y=t=#that_conj)
@@ -38 +38 @@
-come)
+come#come_v)
@@ -41 +41 @@
-on)
+on#on_prep)
@@ -44 +44 @@
-yo=r=)
+yo=r=#your_pron)
@@ -46 +46 @@
-Aged)
+Aged#aged_adj)
@@ -48 +48 @@
-Parents))))))
+Parents#parent_n))))))
@@ -50 +50 @@
-are)
+are#be_v)
@@ -53 +53 @@
-enough)
+enough#enough_adj)
@@ -56 +56 @@
-to)
+to#to_prep)
@@ -58 +58 @@
-sink)
+sink#sink_v)
@@ -61 +61 @@
-them)))))
+them#them_pron)))))
@@ -67 +67 @@
-&)
+&#and_conj1)
@@ -71 +71 @@
-these)
+these#these_pron)
@@ -73 +73 @@
-2)
+2#two_adj)
@@ -75 +75 @@
-monthes))
+monthes#month_n1))
@@ -77 +77 @@
-uneasy)
+uneasy#uneasy_adj)
@@ -79 +79 @@
-wandering))
+wandering#wandering_n))
@@ -81 +81 @@
-hath)
+hath#have_v)
@@ -83 +83 @@
-bin)
+bin#be_v)
@@ -86 +86 @@
-a)
+a#a_adj2)
@@ -88 +88 @@
-great)
+great#great_adj)
@@ -90 +90 @@
-trouble)
+trouble#trouble_n)
@@ -93 +93 @@
-&)
+&#and_conj1)
@@ -96 +96 @@
-sorrow)))
+sorrow#sorrow_n)))
@@ -99 +99 @@
-to)
+to#to_prep)
@@ -102 +102 @@
-us))))
+us#us_pron))))
@@ -107 +107 @@
-Especially))
+Especially#especially_adv))
@@ -109 +109 @@
-for)
+for#for_prep)
@@ -112 +112 @@
-yo=r=)
+yo=r=#your_pron)
@@ -114 +114 @@
-mother))
+mother#mother_n1))
@@ -116 +116 @@
-to)
+to#to_prep)
@@ -118 +118 @@
-be)
+be#be_v)
@@ -121 +121 @@
-in)
+in#in_prep)
@@ -124 +124 @@
-Dublin))))
+Dublin#NA))))
@@ -127 +127 @@
-&)
+&#and_conj1)
@@ -131 +131 @@
-I))
+I#i_pron))
@@ -134 +134 @@
-in)
+in#in_prep)
@@ -137 +137 @@
-this)
+this#this_pron)
@@ -139 +139 @@
-place)
+place#place_n.II-VII)
@@ -143 +143 @@
-from)
+from#from_prep)
@@ -146 +146 @@
-whence)))
+whence#whence_adv)))
@@ -152 +152 @@
-I))
+I#i_pron))
@@ -154 +154 @@
-intend)
+intend#intend_v)
@@ -159 +159 @@
-to)
+to#to_prep)
@@ -161 +161 @@
-goe)
+goe#go_v)
@@ -164 +164 @@
-to)
+to#to_prep)
@@ -167 +167 @@
-her)))
+her#her_pron1)))
@@ -170 +170 @@
-by)
+by#by_prep)
@@ -173 +173 @@
-y=e=)
+y=e=#the_adj)
@@ -175 +175 @@
-ferst)
+ferst#first_adj)
@@ -177 +177 @@
-vessell)
+vessell#vessel_n1)
@@ -182 +182 @@
-y=t=)
+y=t=#that_conj)
@@ -187 +187 @@
-goes)
+goes#go_v)
@@ -190 +190 @@
-hence)))))))
+hence#hence_adv)))))))
@@ -196 +196 @@
-here))
+here#here_adv))
@@ -200 +200 @@
-being)
+being#be_v)
@@ -203 +203 @@
-5)
+5#five_v)
@@ -208 +208 @@
-y=t=)
+y=t=#that_conj)
@@ -213 +213 @@
-lye)
+lye#lie_v1)
@@ -216 +216 @@
-wind_bound)))))))))))))
+wind_bound#wind-bound_adj)))))))))))))
@@ -225 +225 @@
-This)
+This#this_pron)
@@ -227 +227 @@
-inclosed))
+inclosed#enclosed_adj))
@@ -229 +229 @@
-give)
+give#give_v)
@@ -232 +232 @@
-yo=r=)
+yo=r=#your_pron)
@@ -234 +234 @@
-brother)))
+brother#brother_n)))
@@ -240 +240 @@
-or)
+or#or_conj2)
@@ -243 +243 @@
-send)
+send#send_v1)
@@ -245 +245 @@
-&)
+&#and_conj1)
@@ -249 +249 @@
-supscribe))
+supscribe#subscribe_v))
@@ -252 +252 @@
-it))
+it#it_pron))
@@ -255 +255 @@
-for)
+for#for_prep)
@@ -258 +258 @@
-him)))
+him#him_pron)))
@@ -267 +267 @@
-I))
+I#i_pron))
@@ -269 +269 @@
-left)
+left#leave_v1)
@@ -272 +272 @@
-all))
+all#all_adj))
@@ -275 +275 @@
-when)
+when#when_adv)
@@ -282 +282 @@
-I))
+I#i_pron))
@@ -284 +284 @@
-came)
+came#come_v)
@@ -287 +287 @@
-of)
+of#of_prep)
@@ -290 +290 @@
-Dublin)))
+Dublin#NA)))
@@ -293 +293 @@
-by)
+by#by_prep)
@@ -296 +296 @@
-y=e=)
+y=e=#the_adj)
@@ -298 +298 @@
-Instigation)
+Instigation#instigation_n)
@@ -301 +301 @@
-of)
+of#of_prep)
@@ -304 +304 @@
-yong)
+yong#young_adj)
@@ -306 +306 @@
-Damer)
+Damer#NA)
@@ -309 +309 @@
-a)
+a#a_adj2)
@@ -311 +311 @@
-very)
+very#very_adj)
@@ -313 +313 @@
-Rascall)))))))))
+Rascall#rascal_n)))))))))
@@ -322 +322 @@
-his)
+his#his_adj)
@@ -324 +324 @@
-Uncle))
+Uncle#uncle_n))
@@ -326 +326 @@
-was)
+was#be_v)
@@ -329 +329 @@
-allwayes))
+allwayes#always_adv))
@@ -332 +332 @@
-civill)
+civill#civil_adj)
@@ -335 +335 @@
-to)
+to#to_prep)
@@ -338 +338 @@
-me)))))
+me#me_pron1)))))
@@ -344 +344 @@
-but)
+but#but_prep)
@@ -347 +347 @@
-y=e=)
+y=e=#the_adj)
@@ -349 +349 @@
-young)
+young#young_adj)
@@ -351 +351 @@
-villaine))
+villaine#villain_n))
@@ -353 +353 @@
-hath)
+hath#have_v)
@@ -355 +355 @@
-abused)
+abused#abuse_v)
@@ -359 +359 @@
-yo=r=)
+yo=r=#your_pron)
@@ -361 +361 @@
-Mother))
+Mother#mother_n1))
@@ -364 +364 @@
-&)
+&#and_conj1)
@@ -367 +367 @@
-me))))
+me#me_pron1))))
@@ -370 +370 @@
-to)
+to#to_prep)
@@ -373 +373 @@
-a)
+a#a_adj2)
@@ -375 +375 @@
-great)
+great#great_adj)
@@ -377 +377 @@
-degree)))
+degree#degree_n)))
@@ -380 +380 @@
-In_so_much)
+In_so_much#insomuch_adv)
@@ -383 +383 @@
-that)
+that#that_conj)
@@ -387 +387 @@
-I))
+I#i_pron))
@@ -390 +390 @@
-once))
+once#once_adv))
@@ -392 +392 @@
-gave)
+gave#give_v)
@@ -395 +395 @@
-him))
+him#him_pron))
@@ -398 +398 @@
-a)
+a#a_adj2)
@@ -400 +400 @@
-box)
+box#box_n3)
@@ -403 +403 @@
-on)
+on#on_prep)
@@ -406 +406 @@
-y=e=)
+y=e=#the_adj)
@@ -408 +408 @@
-ear))))
+ear#ear_n1))))
@@ -411 +411 @@
-to)
+to#to_prep)
@@ -414 +414 @@
-y=e=)
+y=e=#the_adj)
@@ -416 +416 @@
-purpose))))))
+purpose#purpose_n))))))
@@ -425 +425 @@
-you))
+you#you_pron))
@@ -427 +427 @@
-may)
+may#may_v1)
@@ -429 +429 @@
-see)
+see#see_v)
@@ -433 +433 @@
-what))
+what#what_pron))
@@ -441 +441 @@
-I))
+I#i_pron))
@@ -443 +443 @@
-have)
+have#have_v)
@@ -445 +445 @@
-writ)
+writ#write_v)
@@ -448 +448 @@
-more)
+more#more_adj)
@@ -450 +450 @@
-fully))
+fully#fully_adv))
@@ -453 +453 @@
-to)
+to#to_prep)
@@ -456 +456 @@
-yo=r=)
+yo=r=#your_pron)
@@ -458 +458 @@
-brother)))))
+brother#brother_n)))))
@@ -466 +466 @@
-&)
+&#and_conj1)
@@ -470 +470 @@
-shall)
+shall#shall_v)
@@ -472 +472 @@
-not)
+not#not_adv)
@@ -474 +474 @@
-enlarge)
+enlarge#enlarge_v)
@@ -477 +477 @@
-to)
+to#to_prep)
@@ -480 +480 @@
-you)))
+you#you_pron)))
@@ -483 +483 @@
-only)
+only#only_adj)
@@ -485 +485 @@
-Praying)
+Praying#pray_v)
@@ -488 +488 @@
-God))
+God#NA))
@@ -492 +492 @@
-to)
+to#to_prep)
@@ -494 +494 @@
-Bless)
+Bless#bless_v1)
@@ -497 +497 @@
-you)))
+you#you_pron)))
@@ -500 +500 @@
-&)
+&#and_conj1)
@@ -503 +503 @@
-to)
+to#to_prep)
@@ -505 +505 @@
-continue)
+continue#continue_v)
@@ -508 +508 @@
-to)
+to#to_prep)
@@ -510 +510 @@
-be)
+be#be_v)
@@ -513 +513 @@
-a)
+a#a_adj2)
@@ -515 +515 @@
-comfort)
+comfort#comfort_n)
@@ -518 +518 @@
-to)
+to#to_prep)
@@ -521 +521 @@
-you))))))))))
+you#you_pron))))))))))
@@ -527 +527 @@
-yo=r=)
+yo=r=#your_pron)
@@ -529 +529 @@
-Aged)
+Aged#aged_adj)
@@ -531 +531 @@
-p~ents)
+p~ents#parent_n)
@@ -534 +534 @@
-Jo)
+Jo#NA)
@@ -536 +536 @@
-Penney)))
+Penney#NA)))
@@ -546 +546 @@
-My)
+My#my_adj)
@@ -548 +548 @@
-dear)
+dear#dear_adj1)
@@ -550 +550 @@
-Hester))
+Hester#NA))
@@ -553 +553 @@
-Yours)
+Yours#yours_pron)
@@ -556 +556 @@
-of)
+of#of_prep)
@@ -559 +559 @@
-June)
+June#NA)
@@ -561 +561 @@
-21))))
+21#NA))))
@@ -564 +564 @@
-I))
+I#i_pron))
@@ -566 +566 @@
-rec=d=)
+rec=d=#receive_v)
@@ -569 +569 @@
-July)
+July#NA)
@@ -571 +571 @@
-4=th=)
+4=th=#NA)
@@ -574 +574 @@
-present)))
+present#present_adj)))
@@ -582 +582 @@
-But)
+But#but_prep)
@@ -586 +586 @@
-hear)
+hear#hear_v)
@@ -588 +588 @@
-not)
+not#not_adv)
@@ -591 +591 @@
-of)
+of#of_prep)
@@ -595 +595 @@
-your)
+your#your_pron)
@@ -597 +597 @@
-Mothers))
+Mothers#mother_n1))
@@ -600 +600 @@
-comming)
+comming#come_v)
@@ -603 +603 @@
-hither))))))
+hither#hither_adv))))))
@@ -609 +609 @@
-&)
+&#and_conj1)
@@ -612 +612 @@
-if)
+if#if_conj)
@@ -619 +619 @@
-she))
+she#she_pron))
@@ -621 +621 @@
-doe)
+doe#do_v)
@@ -623 +623 @@
-come))))
+come#come_v))))
@@ -628 +628 @@
-it@))
+it@#it_pron))
@@ -630 +630 @@
-@s)
+@s#be_v)
@@ -633 +633 @@
-strang)
+strang#strange_adj)
@@ -636 +636 @@
-to)
+to#to_prep)
@@ -639 +639 @@
-me))))
+me#me_pron1))))
@@ -646 +646 @@
-you))
+you#you_pron))
@@ -648 +648 @@
-had)
+had#have_v)
@@ -650 +650 @@
-not)
+not#not_adv)
@@ -652 +652 @@
-come)
+come#come_v)
@@ -655 +655 @@
-with)
+with#with_prep)
@@ -658 +658 @@
-her)
+her#her_pron1)
@@ -663 +663 @@
-A)
+A#a_adj2)
@@ -665 +665 @@
-woeman)
+woeman#woman_n)
@@ -669 +669 @@
-now))
+now#now_adv))
@@ -672 +672 @@
-in)
+in#in_prep)
@@ -675 +675 @@
-the)
+the#the_adj)
@@ -677 +677 @@
-70=th=)
+70=th=#NA)
@@ -679 +679 @@
-year)
+year#year_n)
@@ -682 +682 @@
-of)
+of#of_prep)
@@ -685 +685 @@
-her)
+her#her_pron2)
@@ -687 +687 @@
-age)))))))))))
+age#age_n)))))))))))
@@ -696 +696 @@
-from)
+from#from_prep)
@@ -699 +699 @@
-your)
+your#your_pron)
@@ -701 +701 @@
-Brother)))
+Brother#brother_n)))
@@ -704 +704 @@
-I))
+I#i_pron))
@@ -706 +706 @@
-rec=d=)
+rec=d=#receive_v)
@@ -709 +709 @@
-a)
+a#a_adj2)
@@ -711 +711 @@
-l=r=)
+l=r=#letter_n1)
@@ -714 +714 @@
-from)
+from#from_prep)
@@ -717 +717 @@
-Bristol))))
+Bristol#NA))))
@@ -720 +720 @@
-intending)
+intending#intend_v)
@@ -723 +723 @@
-to)
+to#to_prep)
@@ -725 +725 @@
-goe)
+goe#go_v)
@@ -728 +728 @@
-into)
+into#into_prep)
@@ -731 +731 @@
-the)
+the#the_adj)
@@ -733 +733 @@
-West)))))
+West#west_adv)))))
@@ -742 +742 @@
-I))
+I#i_pron))
@@ -744 +744 @@
-hope)
+hope#hope_v)
@@ -751 +751 @@
-all)
+all#all_adj)
@@ -753 +753 @@
-thinges)
+thinges#thing_n1)
@@ -756 +756 @@
-there)))
+there#there_adv)))
@@ -758 +758 @@
-are)
+are#be_v)
@@ -760 +760 @@
-composed)))
+composed#compose_v)))
@@ -769 +769 @@
-The)
+The#the_adj)
@@ -771 +771 @@
-death)
+death#death_n)
@@ -774 +774 @@
-of)
+of#of_prep)
@@ -777 +777 @@
-Rachels)
+Rachels#NA)
@@ -779 +779 @@
-husband))))
+husband#husband_n))))
@@ -782 +782 @@
-I))
+I#i_pron))
@@ -784 +784 @@
-heard)
+heard#hear_v)
@@ -787 +787 @@
-off)
+off#of_prep)
@@ -795 +795 @@
-who))
+who#who_pron))
@@ -801 +801 @@
-I))
+I#i_pron))
@@ -803 +803 @@
-think)
+think#think_v2)
@@ -811 +811 @@
-hast)
+hast#have_v)
@@ -813 +813 @@
-wasted)
+wasted#waste_v)
@@ -816 +816 @@
-her)
+her#her_pron2)
@@ -818 +818 @@
-substance))))))
+substance#substance_n))))))
@@ -827 +827 @@
-If)
+If#if_conj)
@@ -834 +834 @@
-Children))
+Children#child_n))
@@ -836 +836 @@
-will)
+will#will_v1)
@@ -838 +838 @@
-ruin)
+ruin#ruin_v)
@@ -841 +841 @@
-themselves))
+themselves#themselves_pron))
@@ -845 +845 @@
-without)
+without#without_adv)
@@ -848 +848 @@
-the)
+the#the_adj)
@@ -850 +850 @@
-Advice)))
+Advice#advice_n)))
@@ -853 +853 @@
-&)
+&#and_conj1)
@@ -856 +856 @@
-against)
+against#against_prep)
@@ -859 +859 @@
-the)
+the#the_adj)
@@ -861 +861 @@
-minde))))
+minde#mind_n1))))
@@ -864 +864 @@
-of)
+of#of_prep)
@@ -867 +867 @@
-their)
+their#their_adj)
@@ -869 +869 @@
-parents)))))))
+parents#parent_n)))))))
@@ -872 +872 @@
-they))
+they#they_pron))
@@ -874 +874 @@
-must)
+must#must_v1)
@@ -876 +876 @@
-suffer)
+suffer#suffer_v)
@@ -879 +879 @@
-for)
+for#for_prep)
@@ -882 +882 @@
-it)))
+it#it_pron)))
@@ -892 +892 @@
-one))
+one#one_adj))
@@ -894 +894 @@
-lost)
+lost#lose_v1)
@@ -897 +897 @@
-her)
+her#her_pron2)
@@ -899 +899 @@
-life)))
+life#life_n)))
@@ -906 +906 @@
-another))
+another#another_pron))
@@ -909 +909 @@
-his)
+his#his_adj)
@@ -911 +911 @@
-liberty))))
+liberty#liberty_n))))
@@ -916 +916 @@
-&)
+&#and_conj1)
@@ -920 +920 @@
-her))
+her#her_pron1))
@@ -923 +923 @@
-the)
+the#the_adj)
@@ -925 +925 @@
-substance)
+substance#substance_n)
@@ -929 +929 @@
-which))
+which#which_adj))
@@ -937 +937 @@
-she))
+she#she_pron))
@@ -939 +939 @@
-had))))
+had#have_v))))
@@ -942 +942 @@
-by)
+by#by_prep)
@@ -945 +945 @@
-marrying)
+marrying#marry_v)
@@ -948 +948 @@
-against)
+against#against_prep)
@@ -951 +951 @@
-our)
+our#our_pron)
@@ -953 +953 @@
-Advice)))))))
+Advice#advice_n)))))))
@@ -961 +961 @@
-take)
+take#take_v)
@@ -964 +964 @@
-you)
+you#you_pron)
@@ -969 +969 @@
-heed))
+heed#heed_n))
@@ -974 +974 @@
-that)
+that#that_conj)
@@ -979 +979 @@
-have)
+have#have_v)
@@ -981 +981 @@
-seen)
+seen#see_v)
@@ -985 +985 @@
-so)
+so#so_adv)
@@ -987 +987 @@
-many))
+many#many_adj))
@@ -989 +989 @@
-Instances)
+Instances#instance_n)
@@ -992 +992 @@
-of)
+of#of_prep)
@@ -995 +995 @@
-it)))
+it#it_pron)))
@@ -998 +998 @@
-&)
+&#and_conj1)
@@ -1001 +1001 @@
-Warnings))))))
+Warnings#warning_n1))))))
@@ -1010 +1010 @@
-I))
+I#i_pron))
@@ -1012 +1012 @@
-should)
+should#shall_v)
@@ -1014 +1014 @@
-have)
+have#have_v)
@@ -1016 +1016 @@
-bin)
+bin#be_v)
@@ -1019 +1019 @@
-glad)
+glad#glad_adj)
@@ -1022 +1022 @@
-to)
+to#to_prep)
@@ -1024 +1024 @@
-have)
+have#have_v)
@@ -1026 +1026 @@
-seen)
+seen#see_v)
@@ -1029 +1029 @@
-you))
+you#you_pron))
@@ -1032 +1032 @@
-here))))
+here#here_adv))))
@@ -1041 +1041 @@
-My)
+My#my_adj)
@@ -1043 +1043 @@
-going)
+going#going_n)
@@ -1046 +1046 @@
-hence)))
+hence#hence_adv)))
@@ -1048 +1048 @@
-will)
+will#will_v1)
@@ -1050 +1050 @@
-be)
+be#be_v)
@@ -1053 +1053 @@
-Impossible))
+Impossible#impossible_adj))
@@ -1056 +1056 @@
-as)
+as#as_adv)
@@ -1059 +1059 @@
-yet)))
+yet#yet_adv)))
@@ -1069 +1069 @@
-I))
+I#i_pron))
@@ -1071 +1071 @@
-can@)
+can@#can_v1)
@@ -1073 +1073 @@
-@not)
+@not#not_adv)
@@ -1077 +1077 @@
-disoblige)
+disoblige#disoblige_v)
@@ -1080 +1080 @@
-them)
+them#them_pron)
@@ -1085 +1085 @@
-that)
+that#that_conj)
@@ -1090 +1090 @@
-have)
+have#have_v)
@@ -1092 +1092 @@
-bin)
+bin#be_v)
@@ -1095 +1095 @@
-so)
+so#so_adv)
@@ -1097 +1097 @@
-kinde)
+kinde#kind_adj)
@@ -1100 +1100 @@
-to)
+to#to_prep)
@@ -1103 +1103 @@
-me))))))))
+me#me_pron1))))))))
@@ -1106 +1106 @@
-&)
+&#and_conj1)
@@ -1109 +1109 @@
-ruin)
+ruin#ruin_v)
@@ -1112 +1112 @@
-them)
+them#them_pron)
@@ -1117 +1117 @@
-that)
+that#that_conj)
@@ -1122 +1122 @@
-have)
+have#have_v)
@@ -1124 +1124 @@
-succoured)
+succoured#succour_v)
@@ -1127 +1127 @@
-me)))))))
+me#me_pron1)))))))
@@ -1137 +1137 @@
-I))
+I#i_pron))
@@ -1139 +1139 @@
-know)
+know#know_v)
@@ -1146 +1146 @@
-our)
+our#our_pron)
@@ -1148 +1148 @@
-Dangers)
+Dangers#danger_n)
@@ -1151 +1151 @@
-here)))
+here#here_adv)))
@@ -1153 +1153 @@
-are)
+are#be_v)
@@ -1157 +1157 @@
-many))))))
+many#many_adj))))))
@@ -1160 +1160 @@
-and)
+and#and_conj1)
@@ -1164 +1164 @@
-apparent))
+apparent#apparent_adj))
@@ -1168 +1168 @@
-what))
+what#what_pron))
@@ -1175 +1175 @@
-will)
+will#will_v1)
@@ -1177 +1177 @@
-be)
+be#be_v)
@@ -1180 +1180 @@
-the)
+the#the_adj)
@@ -1182 +1182 @@
-Issue)))))))
+Issue#issue_n)))))))
@@ -1189 +1189 @@
-God))
+God#NA))
@@ -1192 +1192 @@
-knowes)
+knowes#know_v)
@@ -1194 +1194 @@
-&)
+&#and_conj1)
@@ -1196 +1196 @@
-orders))
+orders#order_v))
@@ -1199 +1199 @@
-all))
+all#all_adj))
@@ -1208 +1208 @@
-I))
+I#i_pron))
@@ -1210 +1210 @@
-think)
+think#think_v2)
@@ -1217 +1217 @@
-it@))
+it@#it_pron))
@@ -1219 +1219 @@
-@s)
+@s#be_v)
@@ -1222 +1222 @@
-a)
+a#a_adj2)
@@ -1224 +1224 @@
-great)
+great#great_adj)
@@ -1226 +1226 @@
-mercy)
+mercy#mercy_n)
@@ -1229 +1229 @@
-to)
+to#to_prep)
@@ -1232 +1232 @@
-your)
+your#your_pron)
@@ -1234 +1234 @@
-sister))))
+sister#sister_n))))
@@ -1237 +1237 @@
-that)
+that#that_conj)
@@ -1241 +1241 @@
-she))
+she#she_pron))
@@ -1243 +1243 @@
-is)
+is#be_v)
@@ -1245 +1245 @@
-delivered)
+delivered#deliver_v1)
@@ -1248 +1248 @@
-from)
+from#from_prep)
@@ -1251 +1251 @@
-such)
+such#such_adj)
@@ -1253 +1253 @@
-a)
+a#a_adj2)
@@ -1255 +1255 @@
-vile)
+vile#vile_adj)
@@ -1257 +1257 @@
-monster)))))))
+monster#monster_n)))))))
@@ -1266 +1266 @@
-I))
+I#i_pron))
@@ -1268 +1268 @@
-hope)
+hope#hope_v)
@@ -1275 +1275 @@
-she))
+she#she_pron))
@@ -1277 +1277 @@
-will)
+will#will_v1)
@@ -1279 +1279 @@
-be)
+be#be_v)
@@ -1282 +1282 @@
-more)
+more#more_adj)
@@ -1284 +1284 @@
-carefull))))
+carefull#careful_adj))))
@@ -1293 +1293 @@
-If)
+If#if_conj)
@@ -1300 +1300 @@
-you))
+you#you_pron))
@@ -1302 +1302 @@
-have)
+have#have_v)
@@ -1305 +1305 @@
-any)
+any#any_adj)
@@ -1307 +1307 @@
-minde))
+minde#mind_n1))
@@ -1310 +1310 @@
-to)
+to#to_prep)
@@ -1312 +1312 @@
-come)
+come#come_v)
@@ -1315 +1315 @@
-hither))))))
+hither#hither_adv))))))
@@ -1318 +1318 @@
-you))
+you#you_pron))
@@ -1320 +1320 @@
-have)
+have#have_v)
@@ -1323 +1323 @@
-good)
+good#good_adj)
@@ -1325 +1325 @@
-Company))
+Company#company_n))
@@ -1328 +1328 @@
-to)
+to#to_prep)
@@ -1330 +1330 @@
-come)
+come#come_v)
@@ -1333 +1333 @@
-with)
+with#with_prep)
@@ -1336 +1336 @@
-the)
+the#the_adj)
@@ -1338 +1338 @@
-Lady)
+Lady#lady_n)
@@ -1341 +1341 @@
-Sands))
+Sands#NA))
@@ -1346 +1346 @@
-that)
+that#that_conj)
@@ -1351 +1351 @@
-is)
+is#be_v)
@@ -1354 +1354 @@
-now))
+now#now_adv))
@@ -1357 +1357 @@
-in)
+in#in_prep)
@@ -1360 +1360 @@
-London))))))))
+London#NA))))))))
@@ -1369 +1369 @@
-Somebody))
+Somebody#somebody_n))
@@ -1372 +1372 @@
-I))
+I#i_pron))
@@ -1374 +1374 @@
-want)
+want#want_v)
@@ -1377 +1377 @@
-here)))
+here#here_adv)))
@@ -1383 +1383 @@
-for)
+for#for_prep)
@@ -1386 +1386 @@
-my)
+my#my_adj)
@@ -1388 +1388 @@
-linnen))
+linnen#linen_adj))
@@ -1390 +1390 @@
-is)
+is#be_v)
@@ -1392 +1392 @@
-all)
+all#all_adj)
@@ -1394 +1394 @@
-torn)
+torn#tear_v1)
@@ -1403 +1403 @@
-As)
+As#as_adv)
@@ -1406 +1406 @@
-for)
+for#for_prep)
@@ -1409 +1409 @@
-the)
+the#the_adj)
@@ -1411 +1411 @@
-Scot)
+Scot#NA)
@@ -1416 +1416 @@
-that)
+that#that_conj)
@@ -1421 +1421 @@
-is)
+is#be_v)
@@ -1425 +1425 @@
-gon)
+gon#go_v)
@@ -1428 +1428 @@
-to)
+to#to_prep)
@@ -1431 +1431 @@
-London))))))))
+London#NA))))))))
@@ -1435 +1435 @@
-let)
+let#let_v1)
@@ -1439 +1439 @@
-her))
+her#her_pron1))
@@ -1441 +1441 @@
-shift)
+shift#shift_v)
@@ -1444 +1444 @@
-for)
+for#for_prep)
@@ -1447 +1447 @@
-herselfe)))
+herselfe#herself_pron)))
@@ -1450 +1450 @@
-as)
+as#as_adv)
@@ -1461 +1461 @@
-she))
+she#she_pron))
@@ -1463 +1463 @@
-did)
+did#do_v)
@@ -1466 +1466 @@
-before))))))
+before#before_adv))))))
@@ -1475 +1475 @@
-I))
+I#i_pron))
@@ -1477 +1477 @@
-know)
+know#know_v)
@@ -1481 +1481 @@
-what)
+what#what_pron)
@@ -1483 +1483 @@
-an)
+an#a_adj2)
@@ -1485 +1485 @@
-unhappiness))
+unhappiness#unhappiness_n))
@@ -1493 +1493 @@
-she))
+she#she_pron))
@@ -1495 +1495 @@
-hath)
+hath#have_v)
@@ -1497 +1497 @@
-bin)
+bin#be_v)
@@ -1500 +1500 @@
-to)
+to#to_prep)
@@ -1503 +1503 @@
-us)
+us#us_pron)
@@ -1506 +1506 @@
-all))))))
+all#all_adj))))))
@@ -1515 +1515 @@
-I))
+I#i_pron))
@@ -1517 +1517 @@
-have)
+have#have_v)
@@ -1519 +1519 @@
-not)
+not#not_adv)
@@ -1523 +1523 @@
-tyme)
+tyme#time_n)
@@ -1526 +1526 @@
-to)
+to#to_prep)
@@ -1528 +1528 @@
-write)
+write#write_v)
@@ -1531 +1531 @@
-long))))
+long#long_adj1))))
@@ -1535 +1535 @@
-nor)
+nor#nor_conj1)
@@ -1537 +1537 @@
-strenght))))
+strenght#strength_n))))
@@ -1545 +1545 @@
-But)
+But#but_prep)
@@ -1549 +1549 @@
-doe)
+doe#do_v)
@@ -1551 +1551 @@
-charge)
+charge#charge_v)
@@ -1554 +1554 @@
-you))
+you#you_pron))
@@ -1558 +1558 @@
-to)
+to#to_prep)
@@ -1560 +1560 @@
-live)
+live#live_v1)
@@ -1563 +1563 @@
-in)
+in#in_prep)
@@ -1566 +1566 @@
-the)
+the#the_adj)
@@ -1568 +1568 @@
-fear)
+fear#fear_n)
@@ -1571 +1571 @@
-of)
+of#of_prep)
@@ -1574 +1574 @@
-God))))))
+God#NA))))))
@@ -1580 +1580 @@
-not)
+not#not_adv)
@@ -1582 +1582 @@
-to)
+to#to_prep)
@@ -1584 +1584 @@
-dispose)
+dispose#dispose_v)
@@ -1587 +1587 @@
-of)
+of#of_prep)
@@ -1590 +1590 @@
-yourselfe)))
+yourselfe#yourself_pron)))
@@ -1593 +1593 @@
-without)
+without#without_adv)
@@ -1596 +1596 @@
-our)
+our#our_pron)
@@ -1598 +1598 @@
-Advise)))))))
+Advise#advice_n)))))))
@@ -1605 +1605 @@
-We))
+We#we_pron))
@@ -1607 +1607 @@
-may)
+may#may_v1)
@@ -1609 +1609 @@
-have)
+have#have_v)
@@ -1612 +1612 @@
-comfort))
+comfort#comfort_n))
@@ -1615 +1615 @@
-in)
+in#in_prep)
@@ -1618 +1618 @@
-you)))
+you#you_pron)))
@@ -1621 +1621 @@
-through)
+through#through_prep)
@@ -1624 +1624 @@
-Gods)
+Gods#NA)
@@ -1626 +1626 @@
-Blessing)
+Blessing#blessing_n)
@@ -1629 +1629 @@
-on)
+on#on_prep)
@@ -1632 +1632 @@
-you)))
+you#you_pron)))
@@ -1636 +1636 @@
-which))
+which#which_adj))
@@ -1643 +1643 @@
-is)
+is#be_v)
@@ -1646 +1646 @@
-Dayly))
+Dayly#daily_adv))
@@ -1649 +1649 @@
-the)
+the#the_adj)
@@ -1651 +1651 @@
-prayer)
+prayer#prayer_n1)
@@ -1654 +1654 @@
-of)
+of#of_prep)
@@ -1657 +1657 @@
-the)
+the#the_adj)
@@ -1659 +1659 @@
-longing)
+longing#longing_adj1)
@@ -1661 +1661 @@
-Soul)
+Soul#soul_n)
@@ -1664 +1664 @@
-of)
+of#of_prep)
@@ -1667 +1667 @@
-your)
+your#your_pron)
@@ -1670 +1670 @@
-Aged)
+Aged#aged_adj)
@@ -1672 +1672 @@
-&)
+&#and_conj1)
@@ -1674 +1674 @@
-infirme))
+infirme#infirm_adj))
@@ -1676 +1676 @@
-father)
+father#father_n)
@@ -1679 +1679 @@
-John)
+John#NA)
@@ -1681 +1681 @@
-Pinney))))))))))))
+Pinney#NA))))))))))))
@@ -1688 +1688 @@
-I))
+I#i_pron))
@@ -1690 +1690 @@
-recd)
+recd#receive_v)
@@ -1693 +1693 @@
-allso)
+allso#also_adv)
@@ -1695 +1695 @@
-a)
+a#a_adj2)
@@ -1697 +1697 @@
-letter)
+letter#letter_n1)
@@ -1700 +1700 @@
-from)
+from#from_prep)
@@ -1703 +1703 @@
-Rachel)
+Rachel#NA)
@@ -1710 +1710 @@
-not)
+not#not_adv)
@@ -1712 +1712 @@
-long)))
+long#long_adj1)))
@@ -1714 +1714 @@
-since))
+since#since_adv))
@@ -1720 +1720 @@
-to)
+to#to_prep)
@@ -1723 +1723 @@
-whom)))
+whom#whom_pron)))
@@ -1731 +1731 @@
-I))
+I#i_pron))
@@ -1733 +1733 @@
-would)
+would#will_v1)
@@ -1735 +1735 @@
-be)
+be#be_v)
@@ -1737 +1737 @@
-Remembred)))
+Remembred#remember_v)))
