--- jpinney-1686-e3-p1.psd.vert	2025-09-10 13:51:11
+++ jpinney-1686-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-dare)
+dare#dear_adj1)
@@ -16 +16 @@
-childe))
+childe#child_n))
@@ -20 +20 @@
-I))
+I#i_pron))
@@ -22 +22 @@
-blase)
+blase#bless_v1)
@@ -25 +25 @@
-god)))
+god#NA)))
@@ -28 +28 @@
-I))
+I#i_pron))
@@ -30 +30 @@
-ame)
+ame#be_v)
@@ -33 +33 @@
-safe))
+safe#safe_adj))
@@ -35 +35 @@
-com)
+com#come_v)
@@ -38 +38 @@
-heare))
+heare#here_adv))
@@ -41 +41 @@
-this)
+this#this_pron)
@@ -43 +43 @@
-day)
+day#day_n)
@@ -46 +46 @@
-fortnigt))))
+fortnigt#fortnight_n))))
@@ -52 +52 @@
-and)
+and#and_conj1)
@@ -56 +56 @@
-found)
+found#find_v)
@@ -60 +60 @@
-yo=r=)
+yo=r=#your_pron)
@@ -62 +62 @@
-father))
+father#father_n))
@@ -66 +66 @@
-very)
+very#very_adj)
@@ -68 +68 @@
-well))
+well#well_adj))
@@ -71 +71 @@
-but)
+but#but_prep)
@@ -74 +74 @@
-littill)
+littill#little_adj)
@@ -76 +76 @@
-joyed)
+joyed#SEE_joy_v)
@@ -79 +79 @@
-of)
+of#of_prep)
@@ -84 +84 @@
-gettinge)
+gettinge#get_v)
@@ -87 +87 @@
-him))
+him#him_pron))
@@ -90 +90 @@
-home))))))))
+home#home_n1))))))))
@@ -98 +98 @@
-for)
+for#for_prep)
@@ -102 +102 @@
-the)
+the#the_adj)
@@ -104 +104 @@
-paypill))
+paypill#people_n))
@@ -106 +106 @@
-are)
+are#be_v)
@@ -108 +108 @@
-not)
+not#not_adv)
@@ -111 +111 @@
-willinge)
+willinge#willing_adj)
@@ -114 +114 @@
-to)
+to#to_prep)
@@ -116 +116 @@
-parte)
+parte#part_v)
@@ -119 +119 @@
-with)
+with#with_prep)
@@ -122 +122 @@
-him))))))
+him#him_pron))))))
@@ -125 +125 @@
-and)
+and#and_conj1)
@@ -129 +129 @@
-hee))
+hee#he_pron))
@@ -131 +131 @@
-not)
+not#not_adv)
@@ -134 +134 @@
-willinge)
+willinge#willing_adj)
@@ -137 +137 @@
-to)
+to#to_prep)
@@ -139 +139 @@
-goe)
+goe#go_v)
@@ -142 +142 @@
-from)
+from#from_prep)
@@ -145 +145 @@
-thim)))))))
+thim#them_pron)))))))
@@ -153 +153 @@
-but)
+but#but_prep)
@@ -157 +157 @@
-sayes)
+sayes#say_v1)
@@ -161 +161 @@
-how))
+how#how_adv))
@@ -166 +166 @@
-shold)
+shold#shall_v)
@@ -169 +169 @@
-hee))
+hee#he_pron))
@@ -171 +171 @@
-goe)
+goe#go_v)
@@ -174 +174 @@
-from)
+from#from_prep)
@@ -177 +177 @@
-thim)
+thim#them_pron)
@@ -182 +182 @@
-that)
+that#that_conj)
@@ -187 +187 @@
-have)
+have#have_v)
@@ -189 +189 @@
-ben)
+ben#be_v)
@@ -192 +192 @@
-so)
+so#so_adv)
@@ -194 +194 @@
-kinde)
+kinde#kind_adj)
@@ -197 +197 @@
-to)
+to#to_prep)
@@ -200 +200 @@
-him))))))))))
+him#him_pron))))))))))
@@ -209 +209 @@
-I))
+I#i_pron))
@@ -211 +211 @@
-wish)
+wish#wish_v)
@@ -218 +218 @@
-sum)
+sum#some_pron)
@@ -221 +221 @@
-of)
+of#of_prep)
@@ -224 +224 @@
-you)
+you#you_pron)
@@ -227 +227 @@
-his)
+his#his_adj)
@@ -229 +229 @@
-chilldern)))))
+chilldern#child_n)))))
@@ -231 +231 @@
-ware)
+ware#be_v)
@@ -234 +234 @@
-heare)))))
+heare#here_adv)))))
@@ -240 +240 @@
-for)
+for#for_prep)
@@ -243 +243 @@
-I))
+I#i_pron))
@@ -245 +245 @@
-am)
+am#be_v)
@@ -248 +248 @@
-like)
+like#like_adj)
@@ -251 +251 @@
-to)
+to#to_prep)
@@ -253 +253 @@
-doe)
+doe#do_v)
@@ -256 +256 @@
-no)
+no#no_adj)
@@ -258 +258 @@
-good))
+good#good_adj))
@@ -261 +261 @@
-to)
+to#to_prep)
@@ -263 +263 @@
-perswade)
+perswade#persuade_v)
@@ -266 +266 @@
-him)))))
+him#him_pron)))))
@@ -275 +275 @@
-the)
+the#the_adj)
@@ -277 +277 @@
-paypill))
+paypill#people_n))
@@ -279 +279 @@
-are)
+are#be_v)
@@ -281 +281 @@
-all)
+all#all_adj)
@@ -284 +284 @@
-so)
+so#so_adv)
@@ -286 +286 @@
-mad)
+mad#mad_adj)
@@ -289 +289 @@
-to)
+to#to_prep)
@@ -291 +291 @@
-keepe)
+keepe#keep_v)
@@ -294 +294 @@
-him))
+him#him_pron))
@@ -297 +297 @@
-heare))))
+heare#here_adv))))
@@ -312 +312 @@
-Heare))
+Heare#here_adv))
@@ -315 +315 @@
-you))
+you#you_pron))
@@ -317 +317 @@
-see)
+see#see_v)
@@ -321 +321 @@
-what))
+what#what_pron))
@@ -329 +329 @@
-your)
+your#your_pron)
@@ -331 +331 @@
-father))
+father#father_n))
@@ -333 +333 @@
-doe)
+doe#do_v)
@@ -335 +335 @@
-write)))
+write#write_v)))
@@ -344 +344 @@
-I))
+I#i_pron))
@@ -346 +346 @@
-would)
+would#will_v1)
@@ -348 +348 @@
-have)
+have#have_v)
@@ -352 +352 @@
-you))
+you#you_pron))
@@ -354 +354 @@
-send)
+send#send_v1)
@@ -357 +357 @@
-hither))
+hither#hither_adv))
@@ -360 +360 @@
-Mr.)
+Mr.#NA)
@@ -362 +362 @@
-Pammer))
+Pammer#NA))
@@ -365 +365 @@
-spedily))
+spedily#speedily_adv))
@@ -368 +368 @@
-ere)
+ere#ere_adv)
@@ -375 +375 @@
-the)
+the#the_adj)
@@ -379 +379 @@
-wentter))
+wentter#winter_n1))
@@ -381 +381 @@
-doe)
+doe#do_v)
@@ -383 +383 @@
-com)))))
+com#come_v)))))
@@ -386 +386 @@
-for)
+for#for_prep)
@@ -389 +389 @@
-your)
+your#your_pron)
@@ -391 +391 @@
-father))
+father#father_n))
@@ -393 +393 @@
-have)
+have#have_v)
@@ -395 +395 @@
-got)
+got#get_v)
@@ -398 +398 @@
-one)
+one#one_adj)
@@ -403 +403 @@
-into)
+into#into_prep)
@@ -406 +406 @@
-house)))
+house#house_n1)))
@@ -409 +409 @@
-with)
+with#with_prep)
@@ -412 +412 @@
-him)))
+him#him_pron)))
@@ -417 +417 @@
-that)
+that#that_conj)
@@ -421 +421 @@
-wee))
+wee#we_pron))
@@ -423 +423 @@
-can@)
+can@#can_v1)
@@ -425 +425 @@
-@not)
+@not#not_adv)
@@ -427 +427 @@
-be)
+be#be_v)
@@ -430 +430 @@
-red)
+red#red_adj)
@@ -433 +433 @@
-of)
+of#of_prep)
@@ -438 +438 @@
-till)
+till#till_prep)
@@ -445 +445 @@
-som)
+som#some_pron)
@@ -447 +447 @@
-one)
+one#one_adj)
@@ -450 +450 @@
-of)
+of#of_prep)
@@ -453 +453 @@
-you))))
+you#you_pron))))
@@ -455 +455 @@
-com)
+com#come_v)
@@ -458 +458 @@
-here))))))))
+here#here_adv))))))))
@@ -467 +467 @@
-It))
+It#it_pron))
@@ -469 +469 @@
-is)
+is#be_v)
@@ -472 +472 @@
-Dammers)
+Dammers#NA)
@@ -474 +474 @@
-son)
+son#son_n1)
@@ -477 +477 @@
-of)
+of#of_prep)
@@ -480 +480 @@
-Dorchister))))
+Dorchister#NA))))
@@ -488 +488 @@
-to)
+to#to_prep)
@@ -490 +490 @@
-be)
+be#be_v)
@@ -493 +493 @@
-a)
+a#a_adj2)
@@ -495 +495 @@
-menester)))))
+menester#minister_n)))))
@@ -501 +501 @@
-and)
+and#and_conj1)
@@ -504 +504 @@
-hee))
+hee#he_pron))
@@ -506 +506 @@
-is)
+is#be_v)
@@ -509 +509 @@
-wery)
+wery#weary_adj)
@@ -512 +512 @@
-of)
+of#of_prep)
@@ -515 +515 @@
-him))))
+him#him_pron))))
@@ -518 +518 @@
-as)
+as#as_adv)
@@ -520 +520 @@
-well)
+well#well_adv)
@@ -523 +523 @@
-as)
+as#as_adv)
@@ -526 +526 @@
-I))))
+I#i_pron))))
@@ -536 +536 @@
-You))
+You#you_pron))
@@ -538 +538 @@
-may)
+may#may_v1)
@@ -540 +540 @@
-write)
+write#write_v)
@@ -543 +543 @@
-to)
+to#to_prep)
@@ -546 +546 @@
-her))))
+her#her_pron1))))
@@ -549 +549 @@
-and)
+and#and_conj1)
@@ -552 +552 @@
-diricke)
+diricke#direct_v)
@@ -555 +555 @@
-the)
+the#the_adj)
@@ -557 +557 @@
-letter))
+letter#letter_n1))
@@ -560 +560 @@
-to)
+to#to_prep)
@@ -563 +563 @@
-old)
+old#old_adj)
@@ -565 +565 @@
-Burridge))))))
+Burridge#NA))))))
@@ -572 +572 @@
-his)
+his#his_adj)
@@ -574 +574 @@
-name))
+name#name_n))
@@ -576 +576 @@
-is)
+is#be_v)
@@ -579 +579 @@
-Mikel)
+Mikel#NA)
@@ -581 +581 @@
-Burredge))
+Burredge#NA))
@@ -590 +590 @@
-So))
+So#so_adv))
@@ -593 +593 @@
-with)
+with#with_prep)
@@ -596 +596 @@
-my)
+my#my_adj)
@@ -598 +598 @@
-respickes)
+respickes#respect_n)
@@ -601 +601 @@
-to)
+to#to_prep)
@@ -604 +604 @@
-you)
+you#you_pron)
@@ -607 +607 @@
-both))))))
+both#both_pron))))))
@@ -610 +610 @@
-I))
+I#i_pron))
@@ -612 +612 @@
-remane)
+remane#remain_v)
@@ -615 +615 @@
-Your)
+Your#your_pron)
@@ -617 +617 @@
-loving)
+loving#loving_adj)
@@ -619 +619 @@
-mother)
+mother#mother_n1)
@@ -622 +622 @@
-Jane)
+Jane#NA)
@@ -624 +624 @@
-Pinney))))
+Pinney#NA))))
