--- mhatton-1678-e3-p1.psd.vert	2025-09-10 13:51:11
+++ mhatton-1678-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -13 +13 @@
-July)
+July#NA)
@@ -16 +16 @@
-y=e=)
+y=e=#the_adj)
@@ -18 +18 @@
-17))
+17#seventeen_adj))
@@ -24 +24 @@
-1678)
+1678#NA)
@@ -35 +35 @@
-Dearest)
+Dearest#dear_adj1)
@@ -37 +37 @@
-Brother))
+Brother#brother_n))
@@ -42 +42 @@
-My)
+My#my_adj)
@@ -44 +44 @@
-designes))
+designes#design_n))
@@ -49 +49 @@
-as)
+as#as_adv)
@@ -60 +60 @@
-you))
+you#you_pron))
@@ -62 +62 @@
-know))))
+know#know_v))))
@@ -66 +66 @@
-are)
+are#be_v)
@@ -68 +68 @@
-broke)
+broke#break_v)
@@ -73 +73 @@
-since)
+since#since_adv)
@@ -80 +80 @@
-a)
+a#a_adj2)
@@ -82 +82 @@
-peace))
+peace#peace_n))
@@ -84 +84 @@
-is)
+is#be_v)
@@ -87 +87 @@
-absolutly))
+absolutly#absolutely_adv))
@@ -89 +89 @@
-determined))))
+determined#determine_v))))
@@ -98 +98 @@
-The)
+The#the_adj)
@@ -100 +100 @@
-K.))
+K.#king_n))
@@ -102 +102 @@
-consents)
+consents#consent_v)
@@ -105 +105 @@
-to)
+to#to_prep)
@@ -107 +107 @@
-disband)
+disband#disband_v)
@@ -110 +110 @@
-the)
+the#the_adj)
@@ -112 +112 @@
-army)))
+army#army_n)))
@@ -118 +118 @@
-onely)
+onely#only_adj)
@@ -120 +120 @@
-that)
+that#that_pron)
@@ -123 +123 @@
-in)
+in#in_prep)
@@ -126 +126 @@
-garrisons)))
+garrisons#garrison_n)))
@@ -131 +131 @@
-in)
+in#in_prep)
@@ -134 +134 @@
-Flanders))))
+Flanders#NA))))
@@ -138 +138 @@
-to)
+to#to_prep)
@@ -140 +140 @@
-stand)
+stand#stand_v)
@@ -143 +143 @@
-till)
+till#till_prep)
@@ -150 +150 @@
-peace))
+peace#peace_n))
@@ -152 +152 @@
-is)
+is#be_v)
@@ -154 +154 @@
-determined)
+determined#determine_v)
@@ -157 +157 @@
-generaly))))))
+generaly#generally_adv))))))
@@ -166 +166 @@
-S=r=)
+S=r=#NA)
@@ -168 +168 @@
-Robert)
+Robert#NA)
@@ -170 +170 @@
-Carr))
+Carr#NA))
@@ -172 +172 @@
-is)
+is#be_v)
@@ -174 +174 @@
-turned)
+turned#turn_v)
@@ -176 +176 @@
-out)
+out#out_adv)
@@ -179 +179 @@
-from)
+from#from_prep)
@@ -182 +182 @@
-being)
+being#be_v)
@@ -185 +185 @@
-Chancelour)
+Chancelour#chancellor_n)
@@ -188 +188 @@
-of)
+of#of_prep)
@@ -191 +191 @@
-the)
+the#the_adj)
@@ -193 +193 @@
-Dutchy))))))
+Dutchy#duchy_n))))))
@@ -202 +202 @@
-He))
+He#he_pron))
@@ -204 +204 @@
-was)
+was#be_v)
@@ -208 +208 @@
-too)
+too#too_adv)
@@ -210 +210 @@
-much))
+much#much_adj))
@@ -215 +215 @@
-a)
+a#a_adj2)
@@ -217 +217 @@
-freind)
+freind#friend_n)
@@ -220 +220 @@
-to)
+to#to_prep)
@@ -223 +223 @@
-L=d=)
+L=d=#NA)
@@ -225 +225 @@
-Arlington))))
+Arlington#NA))))
@@ -228 +228 @@
-to)
+to#to_prep)
@@ -230 +230 @@
-stand))
+stand#stand_v))
@@ -239 +239 @@
-The)
+The#the_adj)
@@ -241 +241 @@
-next)
+next#next_adj)
@@ -245 +245 @@
-news)
+news#news_n)
@@ -256 +256 @@
-he))
+he#he_pron))
@@ -258 +258 @@
-gives)
+gives#give_v)
@@ -261 +261 @@
-me)))))
+me#me_pron1)))))
@@ -263 +263 @@
-is)
+is#be_v)
@@ -266 +266 @@
-a)
+a#a_adj2)
@@ -268 +268 @@
-pleasant)
+pleasant#pleasant_adj)
@@ -270 +270 @@
-storry)
+storry#story_n)
@@ -275 +275 @@
-that)
+that#that_conj)
@@ -280 +280 @@
-begings)
+begings#begin_v1)
@@ -283 +283 @@
-thus)))))
+thus#thus_adv)))))
@@ -293 +293 @@
-Daincourt))
+Daincourt#NA))
@@ -299 +299 @@
-Lady)
+Lady#NA)
@@ -301 +301 @@
-Gray)))
+Gray#NA)))
@@ -306 +306 @@
-and)
+and#and_conj1)
@@ -309 +309 @@
-M=is=)
+M=is=#NA)
@@ -311 +311 @@
-Smith))))
+Smith#NA))))
@@ -313 +313 @@
-walked)
+walked#walk_v)
@@ -316 +316 @@
-still))
+still#still_adv))
@@ -318 +318 @@
-on)
+on#on_adv)
@@ -327 +327 @@
-In)
+In#in_prep)
@@ -330 +330 @@
-the)
+the#the_adj)
@@ -332 +332 @@
-Park)))
+Park#park_n)))
@@ -337 +337 @@
-the)
+the#the_adj)
@@ -339 +339 @@
-first))
+first#first_adj))
@@ -341 +341 @@
-left)
+left#leave_v1)
@@ -344 +344 @@
-them))
+them#them_pron))
@@ -352 +352 @@
-and)
+and#and_conj1)
@@ -355 +355 @@
-going)
+going#go_v)
@@ -358 +358 @@
-home))
+home#home_n1))
@@ -361 +361 @@
-in)
+in#in_prep)
@@ -364 +364 @@
-her)
+her#her_pron2)
@@ -366 +366 @@
-chair))))
+chair#chair_n))))
@@ -371 +371 @@
-the)
+the#the_adj)
@@ -373 +373 @@
-L=d=)
+L=d=#lord_n)
@@ -376 +376 @@
-of)
+of#of_prep)
@@ -379 +379 @@
-Monmouth))))
+Monmouth#NA))))
@@ -381 +381 @@
-mistaks)
+mistaks#mistake_v)
@@ -384 +384 @@
-the)
+the#the_adj)
@@ -386 +386 @@
-Lady)
+Lady#lady_n)
@@ -389 +389 @@
-Gray))))
+Gray#NA))))
@@ -395 +395 @@
-and)
+and#and_conj1)
@@ -399 +399 @@
-gives)
+gives#give_v)
@@ -402 +402 @@
-her))
+her#her_pron1))
@@ -405 +405 @@
-a)
+a#a_adj2)
@@ -407 +407 @@
-billet))
+billet#billet_n1))
@@ -416 +416 @@
-When)
+When#when_adv)
@@ -423 +423 @@
-she))
+she#she_pron))
@@ -425 +425 @@
-came)
+came#come_v)
@@ -428 +428 @@
-home)))))
+home#home_n1)))))
@@ -433 +433 @@
-she))
+she#she_pron))
@@ -435 +435 @@
-gives)
+gives#give_v)
@@ -438 +438 @@
-it))
+it#it_pron))
@@ -441 +441 @@
-her)
+her#her_pron2)
@@ -443 +443 @@
-husband)
+husband#husband_n)
@@ -449 +449 @@
-who))
+who#who_pron))
@@ -456 +456 @@
-was)
+was#be_v)
@@ -459 +459 @@
-angry)
+angry#angry_adj)
@@ -462 +462 @@
-with)
+with#with_prep)
@@ -465 +465 @@
-her))))
+her#her_pron1))))
@@ -470 +470 @@
-least)
+least#lest_conj)
@@ -477 +477 @@
-y=e=)
+y=e=#the_adj)
@@ -479 +479 @@
-L=d=))
+L=d=#lord_n))
@@ -481 +481 @@
-should)
+should#shall_v)
@@ -483 +483 @@
-be)
+be#be_v)
@@ -486 +486 @@
-displeased)
+displeased#displeased_adj)
@@ -489 +489 @@
-at)
+at#at_prep)
@@ -492 +492 @@
-the)
+the#the_adj)
@@ -494 +494 @@
-misfortun)
+misfortun#misfortune_n)
@@ -497 +497 @@
-of)
+of#of_prep)
@@ -500 +500 @@
-his)
+his#his_adj)
@@ -502 +502 @@
-billet))))))))))))
+billet#billet_n1))))))))))))
@@ -511 +511 @@
-he))
+he#he_pron))
@@ -513 +513 @@
-caryes)
+caryes#carry_v)
@@ -516 +516 @@
-it))
+it#it_pron))
@@ -519 +519 @@
-presently))
+presently#presently_adv))
@@ -522 +522 @@
-to)
+to#to_prep)
@@ -525 +525 @@
-him)))
+him#him_pron)))
@@ -535 +535 @@
-beggs)
+beggs#beg_v)
@@ -538 +538 @@
-his)
+his#his_adj)
@@ -540 +540 @@
-pardon)
+pardon#pardon_n1)
@@ -543 +543 @@
-in)
+in#in_prep)
@@ -546 +546 @@
-the)
+the#the_adj)
@@ -548 +548 @@
-mistake))))
+mistake#mistake_n))))
@@ -557 +557 @@
-The)
+The#the_adj)
@@ -559 +559 @@
-L=d=))
+L=d=#lord_n))
@@ -561 +561 @@
-beggs)
+beggs#beg_v)
@@ -564 +564 @@
-his))
+his#his_pron1))
@@ -569 +569 @@
-lest)
+lest#lest_conj)
@@ -576 +576 @@
-he))
+he#he_pron))
@@ -578 +578 @@
-should)
+should#shall_v)
@@ -580 +580 @@
-take)
+take#take_v)
@@ -583 +583 @@
-the)
+the#the_adj)
@@ -585 +585 @@
-giving)
+giving#giving_n)
@@ -588 +588 @@
-of)
+of#of_prep)
@@ -591 +591 @@
-the)
+the#the_adj)
@@ -593 +593 @@
-letter))))
+letter#letter_n1))))
@@ -596 +596 @@
-ill))
+ill#ill_adv))
@@ -599 +599 @@
-to)
+to#to_prep)
@@ -602 +602 @@
-his)
+his#his_adj)
@@ -604 +604 @@
-wife))))))
+wife#wife_n))))))
@@ -612 +612 @@
-And)
+And#and_conj1)
@@ -616 +616 @@
-thus)
+thus#thus_adv)
@@ -618 +618 @@
-much)))
+much#much_adj)))
@@ -621 +621 @@
-for)
+for#for_prep)
@@ -624 +624 @@
-Inglish)
+Inglish#english_adj)
@@ -626 +626 @@
-news)))
+news#news_n)))
@@ -637 +637 @@
-What))
+What#what_pron))
@@ -643 +643 @@
-I))
+I#i_pron))
@@ -645 +645 @@
-have)
+have#have_v)
@@ -648 +648 @@
-to)
+to#to_prep)
@@ -650 +650 @@
-aquaint)
+aquaint#acquaint_v)
@@ -653 +653 @@
-you))
+you#you_pron))
@@ -656 +656 @@
-w=th=all)
+w=th=all#withal_adv)
@@ -661 +661 @@
-of)
+of#of_prep)
@@ -664 +664 @@
-Paris)
+Paris#NA)
@@ -666 +666 @@
-news)))))))
+news#news_n)))))))
@@ -668 +668 @@
-is)
+is#be_v)
@@ -672 +672 @@
-our)
+our#our_pron)
@@ -674 +674 @@
-cosin)
+cosin#cousin_n)
@@ -677 +677 @@
-Montague@))
+Montague@#NA))
@@ -679 +679 @@
-@s))
+@s#NA))
@@ -682 +682 @@
-being)
+being#be_v)
@@ -684 +684 @@
-gon)
+gon#go_v)
@@ -687 +687 @@
-last)
+last#last_adv)
@@ -689 +689 @@
-Monday))
+Monday#NA))
@@ -692 +692 @@
-post))
+post#post_n2))
@@ -695 +695 @@
-towards)
+towards#towards_prep)
@@ -698 +698 @@
-Ingland)))
+Ingland#NA)))
@@ -703 +703 @@
-opon)
+opon#upon_prep)
@@ -709 +709 @@
-my)
+my#my_adj)
@@ -711 +711 @@
-Lord)
+Lord#lord_n)
@@ -714 +714 @@
-Sunderland@))
+Sunderland@#NA))
@@ -716 +716 @@
-@'s))
+@'s#NA))
@@ -719 +719 @@
-being)
+being#be_v)
@@ -721 +721 @@
-sent)
+sent#send_v1)
@@ -724 +724 @@
-hither))
+hither#hither_adv))
@@ -727 +727 @@
-ambassador)))))))
+ambassador#ambassador_n)))))))
@@ -733 +733 @@
-which)
+which#which_adj)
@@ -735 +735 @@
-bussness))
+bussness#business_n))
@@ -741 +741 @@
-they))
+they#they_pron))
@@ -743 +743 @@
-say)
+say#say_v1)
@@ -752 +752 @@
-my)
+my#my_adj)
@@ -754 +754 @@
-Lady)
+Lady#lady_n)
@@ -757 +757 @@
-Cleavland)))
+Cleavland#NA)))
@@ -759 +759 @@
-has)
+has#have_v)
@@ -761 +761 @@
-intrigued)
+intrigued#intrigue_v)
@@ -766 +766 @@
-out)
+out#out_adv)
@@ -768 +768 @@
-of)
+of#of_prep)
@@ -771 +771 @@
-revenge)
+revenge#revenge_n)
@@ -774 +774 @@
-to)
+to#to_prep)
@@ -777 +777 @@
-the)
+the#the_adj)
@@ -779 +779 @@
-ambass.)))
+ambass.#ambassador_n)))
@@ -782 +782 @@
-for)
+for#for_prep)
@@ -785 +785 @@
-being)
+being#be_v)
@@ -788 +788 @@
-soe)
+soe#so_adv)
@@ -790 +790 @@
-jealous)
+jealous#jealous_adj)
@@ -793 +793 @@
-of)
+of#of_prep)
@@ -796 +796 @@
-her)))
+her#her_pron1)))
@@ -799 +799 @@
-for)
+for#for_prep)
@@ -802 +802 @@
-one)
+one#one_adj)
@@ -804 +804 @@
-Chevalier)
+Chevalier#NA)
@@ -806 +806 @@
-Chatillon)))
+Chatillon#NA)))
@@ -809 +809 @@
-as)
+as#as_adv)
@@ -812 +812 @@
-to)
+to#to_prep)
@@ -814 +814 @@
-wright)
+wright#write_v)
@@ -817 +817 @@
-it))
+it#it_pron))
@@ -822 +822 @@
-wheire))
+wheire#where_adv))
@@ -828 +828 @@
-he))
+he#he_pron))
@@ -830 +830 @@
-thought)
+thought#think_v2)
@@ -839 +839 @@
-it))
+it#it_pron))
@@ -841 +841 @@
-might)
+might#may_v1)
@@ -843 +843 @@
-doe)
+doe#do_v)
@@ -846 +846 @@
-her))
+her#her_pron1))
@@ -849 +849 @@
-most)
+most#most_adj)
@@ -851 +851 @@
-prejudice)))))))
+prejudice#prejudice_n)))))))
@@ -857 +857 @@
-which))
+which#which_adj))
@@ -867 +867 @@
-she))
+she#she_pron))
@@ -869 +869 @@
-being)
+being#be_v)
@@ -871 +871 @@
-advertised)
+advertised#advertise_v)
@@ -874 +874 @@
-of)
+of#of_prep)
@@ -881 +881 @@
-and)
+and#and_conj1)
@@ -886 +886 @@
-attributing)
+attributing#attribute_v)
@@ -889 +889 @@
-to)
+to#to_prep)
@@ -892 +892 @@
-it)))
+it#it_pron)))
@@ -895 +895 @@
-the)
+the#the_adj)
@@ -897 +897 @@
-cold)
+cold#cold_adj)
@@ -899 +899 @@
-reception)
+reception#reception_n)
@@ -910 +910 @@
-she))
+she#she_pron))
@@ -912 +912 @@
-found)
+found#find_v)
@@ -915 +915 @@
-when)
+when#when_adv)
@@ -922 +922 @@
-she))
+she#she_pron))
@@ -924 +924 @@
-was)
+was#be_v)
@@ -927 +927 @@
-laitly))
+laitly#late_adv))
@@ -930 +930 @@
-in)
+in#in_prep)
@@ -933 +933 @@
-Ingland))))))))))))
+Ingland#NA))))))))))))
@@ -937 +937 @@
-has)
+has#have_v)
@@ -942 +942 @@
-as)
+as#as_adv)
@@ -953 +953 @@
-they))
+they#they_pron))
@@ -955 +955 @@
-say))))
+say#say_v1))))
@@ -959 +959 @@
-acussed)
+acussed#accuse_v)
@@ -962 +962 @@
-him))
+him#him_pron))
@@ -965 +965 @@
-of)
+of#of_prep)
@@ -968 +968 @@
-not)
+not#not_adv)
@@ -970 +970 @@
-being)
+being#be_v)
@@ -973 +973 @@
-faithfull)
+faithfull#faithful_adj)
@@ -976 +976 @@
-to)
+to#to_prep)
@@ -979 +979 @@
-his)
+his#his_adj)
@@ -981 +981 @@
-master)))
+master#master_n1)))
@@ -984 +984 @@
-in)
+in#in_prep)
@@ -987 +987 @@
-the)
+the#the_adj)
@@ -989 +989 @@
-imployment)
+imployment#employment_n)
@@ -1000 +1000 @@
-he))
+he#he_pron))
@@ -1002 +1002 @@
-gave)
+gave#give_v)
@@ -1005 +1005 @@
-him))
+him#him_pron))
@@ -1008 +1008 @@
-here))))))))))))))))))))))
+here#here_adv))))))))))))))))))))))
@@ -1014 +1014 @@
-too)
+too#to_prep)
@@ -1017 +1017 @@
-which)))
+which#which_adj)))
@@ -1025 +1025 @@
-there))
+there#there_adv))
@@ -1027 +1027 @@
-is)
+is#be_v)
@@ -1030 +1030 @@
-another)
+another#another_pron)
@@ -1032 +1032 @@
-particular)
+particular#particular_adj)
@@ -1037 +1037 @@
-that)
+that#that_conj)
@@ -1042 +1042 @@
-dus)
+dus#do_v)
@@ -1045 +1045 @@
-much))
+much#much_adj))
@@ -1047 +1047 @@
-agravate)
+agravate#aggravate_v)
@@ -1050 +1050 @@
-her)))))))
+her#her_pron1)))))))
@@ -1058 +1058 @@
-and)
+and#and_conj1)
@@ -1061 +1061 @@
-that))
+that#that_pron))
@@ -1063 +1063 @@
-is)
+is#be_v)
@@ -1066 +1066 @@
-that)
+that#that_conj)
@@ -1072 +1072 @@
-whillest)
+whillest#whilst_adv)
@@ -1079 +1079 @@
-she))
+she#she_pron))
@@ -1081 +1081 @@
-was)
+was#be_v)
@@ -1084 +1084 @@
-in)
+in#in_prep)
@@ -1087 +1087 @@
-Ingland))))))
+Ingland#NA))))))
@@ -1092 +1092 @@
-the)
+the#the_adj)
@@ -1094 +1094 @@
-ambas.))
+ambas.#ambassador_n))
@@ -1096 +1096 @@
-was)
+was#be_v)
@@ -1099 +1099 @@
-every)
+every#every_adj)
@@ -1101 +1101 @@
-day))
+day#day_n))
@@ -1104 +1104 @@
-with)
+with#with_prep)
@@ -1107 +1107 @@
-her)
+her#her_pron2)
@@ -1109 +1109 @@
-daughter)
+daughter#daughter_n)
@@ -1112 +1112 @@
-Sussex))))
+Sussex#NA))))
@@ -1118 +1118 @@
-which))
+which#which_adj))
@@ -1125 +1125 @@
-has)
+has#have_v)
@@ -1127 +1127 @@
-ocationed)
+ocationed#occasion_v)
@@ -1130 +1130 @@
-such)
+such#such_adj)
@@ -1132 +1132 @@
-jealously)
+jealously#jealousy_n)
@@ -1135 +1135 @@
-of)
+of#of_prep)
@@ -1138 +1138 @@
-all)
+all#all_adj)
@@ -1140 +1140 @@
-sides)))
+sides#side_n1)))
@@ -1143 +1143 @@
-that)
+that#that_conj)
@@ -1149 +1149 @@
-for)
+for#for_prep)
@@ -1152 +1152 @@
-the)
+the#the_adj)
@@ -1154 +1154 @@
-saffty)
+saffty#safety_n)
@@ -1157 +1157 @@
-of)
+of#of_prep)
@@ -1160 +1160 @@
-my)
+my#my_adj)
@@ -1162 +1162 @@
-Lady)
+Lady#lady_n)
@@ -1165 +1165 @@
-Sussex))))))
+Sussex#NA))))))
@@ -1170 +1170 @@
-it))
+it#it_pron))
@@ -1172 +1172 @@
-is)
+is#be_v)
@@ -1174 +1174 @@
-reported)
+reported#report_v)
@@ -1182 +1182 @@
-the)
+the#the_adj)
@@ -1184 +1184 @@
-ambas.))
+ambas.#ambassador_n))
@@ -1186 +1186 @@
-advissed)
+advissed#advise_v)
@@ -1189 +1189 @@
-her))
+her#her_pron1))
@@ -1192 +1192 @@
-to)
+to#to_prep)
@@ -1195 +1195 @@
-a)
+a#a_adj2)
@@ -1197 +1197 @@
-nunnery))))
+nunnery#nunnery_n))))
@@ -1202 +1202 @@
-and)
+and#and_conj1)
@@ -1207 +1207 @@
-made)
+made#make_v1)
@@ -1210 +1210 @@
-choice)
+choice#choice_n)
@@ -1213 +1213 @@
-of)
+of#of_prep)
@@ -1216 +1216 @@
-Belle)
+Belle#NA)
@@ -1218 +1218 @@
-Chase)
+Chase#NA)
@@ -1223 +1223 @@
-for)
+for#for_prep)
@@ -1226 +1226 @@
-her)))
+her#her_pron1)))
@@ -1232 +1232 @@
-where))
+where#where_adv))
@@ -1241 +1241 @@
-she))
+she#she_pron))
@@ -1243 +1243 @@
-is)
+is#be_v)
@@ -1246 +1246 @@
-at)
+at#at_prep)
@@ -1249 +1249 @@
-present))))
+present#present_adj))))
@@ -1252 +1252 @@
-and)
+and#and_conj1)
@@ -1259 +1259 @@
-will)
+will#will_v1)
@@ -1261 +1261 @@
-not)
+not#not_adv)
@@ -1263 +1263 @@
-see)
+see#see_v)
@@ -1266 +1266 @@
-her)
+her#her_pron2)
@@ -1268 +1268 @@
-mother)))))))))))))))))
+mother#mother_n1)))))))))))))))))
@@ -1277 +1277 @@
-The)
+The#the_adj)
@@ -1279 +1279 @@
-Chevalier)
+Chevalier#NA)
@@ -1282 +1282 @@
-Chatillon)))
+Chatillon#NA)))
@@ -1284 +1284 @@
-is)
+is#be_v)
@@ -1287 +1287 @@
-a)
+a#a_adj2)
@@ -1289 +1289 @@
-person)
+person#person_n)
@@ -1292 +1292 @@
-of)
+of#of_prep)
@@ -1295 +1295 @@
-quallity)))
+quallity#quality_n)))
@@ -1300 +1300 @@
-young)
+young#young_adj)
@@ -1302 +1302 @@
-and)
+and#and_conj1)
@@ -1304 +1304 @@
-handsome)))
+handsome#handsome_adj)))
@@ -1312 +1312 @@
-but)
+but#but_prep)
@@ -1315 +1315 @@
-noe)
+noe#no_adj)
@@ -1317 +1317 @@
-istate))
+istate#estate_n))
@@ -1321 +1321 @@
-and)
+and#and_conj1)
@@ -1324 +1324 @@
-therefore))
+therefore#therefore_adv))
@@ -1326 +1326 @@
-&c.)
+&c.#et-cetera_n)
@@ -1334 +1334 @@
-pour)
+pour#NA)
@@ -1336 +1336 @@
-la)
+la#NA)
@@ -1340 +1340 @@
-bigotte)
+bigotte#NA)
@@ -1344 +1344 @@
-si)
+si#NA)
@@ -1346 +1346 @@
-elle)
+elle#NA)
@@ -1348 +1348 @@
-ne)
+ne#NA)
@@ -1350 +1350 @@
-peut)
+peut#NA)
@@ -1352 +1352 @@
-courrir)
+courrir#NA)
@@ -1356 +1356 @@
-quelle)
+quelle#NA)
@@ -1358 +1358 @@
-trotte)
+trotte#NA)
@@ -1364 +1364 @@
-with)
+with#with_prep)
@@ -1367 +1367 @@
-which)
+which#which_adj)
@@ -1369 +1369 @@
-French)
+French#french_adj)
@@ -1371 +1371 @@
-proverb)))
+proverb#proverb_n)))
@@ -1380 +1380 @@
-I))
+I#i_pron))
@@ -1382 +1382 @@
-will)
+will#will_v1)
@@ -1385 +1385 @@
-now))
+now#now_adv))
@@ -1387 +1387 @@
-eaise)
+eaise#ease_v)
@@ -1390 +1390 @@
-you))
+you#you_pron))
@@ -1393 +1393 @@
-of)
+of#of_prep)
@@ -1396 +1396 @@
-this)
+this#this_pron)
@@ -1398 +1398 @@
-trouble))))
+trouble#trouble_n))))
@@ -1403 +1403 @@
-and)
+and#and_conj1)
@@ -1406 +1406 @@
-onely)
+onely#only_adj)
@@ -1408 +1408 @@
-begg)
+begg#beg_v)
@@ -1411 +1411 @@
-one)
+one#one_adj)
@@ -1413 +1413 @@
-favour)
+favour#favour_n)
@@ -1416 +1416 @@
-more))
+more#more_adj))
@@ -1421 +1421 @@
-of)
+of#of_prep)
@@ -1424 +1424 @@
-you)))
+you#you_pron)))
@@ -1429 +1429 @@
-that)
+that#that_conj)
@@ -1433 +1433 @@
-you))
+you#you_pron))
@@ -1435 +1435 @@
-will)
+will#will_v1)
@@ -1438 +1438 @@
-ever))
+ever#ever_adv))
@@ -1440 +1440 @@
-beleeve)
+beleeve#believe_v)
@@ -1444 +1444 @@
-me))
+me#me_pron1))
@@ -1451 +1451 @@
-what))
+what#what_pron))
@@ -1459 +1459 @@
-sincerly))
+sincerly#sincerely_adv))
@@ -1462 +1462 @@
-I))
+I#i_pron))
@@ -1464 +1464 @@
-am)))
+am#be_v)))
@@ -1469 +1469 @@
-Deare)
+Deare#dear_adj1)
@@ -1471 +1471 @@
-brother))
+brother#brother_n))
@@ -1476 +1476 @@
-Yr)
+Yr#your_pron)
@@ -1479 +1479 @@
-most)
+most#most_adj)
@@ -1481 +1481 @@
-affec.))
+affec.#affectionate_adj))
@@ -1483 +1483 @@
-sister)
+sister#sister_n)
@@ -1486 +1486 @@
-and)
+and#and_conj1)
@@ -1489 +1489 @@
-humble)
+humble#humble_adj)
@@ -1491 +1491 @@
-servant)
+servant#servant_n)
@@ -1499 +1499 @@
-to)
+to#to_prep)
@@ -1501 +1501 @@
-command)))))
+command#command_v)))))
@@ -1506 +1506 @@
-M.)
+M.#NA)
@@ -1508 +1508 @@
-H.))))))))))))
+H.#NA))))))))))))
@@ -1515 +1515 @@
-Deare)
+Deare#dear_adj1)
@@ -1517 +1517 @@
-brother))
+brother#brother_n))
@@ -1522 +1522 @@
-I))
+I#i_pron))
@@ -1524 +1524 @@
-am)
+am#be_v)
@@ -1526 +1526 @@
-not)
+not#not_adv)
@@ -1529 +1529 @@
-afraid)
+afraid#afraid_adj)
@@ -1532 +1532 @@
-of)
+of#of_prep)
@@ -1535 +1535 @@
-y=r=)
+y=r=#your_pron)
@@ -1538 +1538 @@
-getting)
+getting#get_v)
@@ -1541 +1541 @@
-the)
+the#the_adj)
@@ -1543 +1543 @@
-small_pox))))))
+small_pox#smallpox_n))))))
@@ -1551 +1551 @@
-but)
+but#but_prep)
@@ -1554 +1554 @@
-for)
+for#for_prep)
@@ -1557 +1557 @@
-God)
+God#NA)
@@ -1559 +1559 @@
-saike)))
+saike#sake_n1)))
@@ -1561 +1561 @@
-have)
+have#have_v)
@@ -1564 +1564 @@
-a)
+a#a_adj2)
@@ -1566 +1566 @@
-care))
+care#care_n1))
@@ -1569 +1569 @@
-of)
+of#of_prep)
@@ -1572 +1572 @@
-coming)
+coming#come_v)
@@ -1575 +1575 @@
-neare)
+neare#near_adj)
@@ -1578 +1578 @@
-those)
+those#those_pron)
@@ -1583 +1583 @@
-that)
+that#that_conj)
@@ -1588 +1588 @@
-have)
+have#have_v)
@@ -1591 +1591 @@
-the)
+the#the_adj)
@@ -1593 +1593 @@
-feavour))))))))
+feavour#fever_n1))))))))
@@ -1604 +1604 @@
-Sept.)
+Sept.#NA)
@@ -1607 +1607 @@
-y=e=)
+y=e=#the_adj)
@@ -1609 +1609 @@
-6th))
+6th#NA))
@@ -1613 +1613 @@
-1678?)
+1678?#NA)
@@ -1622 +1622 @@
-It))
+It#it_pron))
@@ -1624 +1624 @@
-comes)
+comes#come_v)
@@ -1627 +1627 @@
-in)
+in#in_prep)
@@ -1630 +1630 @@
-my)
+my#my_adj)
@@ -1632 +1632 @@
-mind)))
+mind#mind_n1)))
@@ -1635 +1635 @@
-to)
+to#to_prep)
@@ -1637 +1637 @@
-aske)
+aske#ask_v)
@@ -1640 +1640 @@
-you))
+you#you_pron))
@@ -1643 +1643 @@
-if)
+if#if_conj)
@@ -1649 +1649 @@
-you))
+you#you_pron))
@@ -1651 +1651 @@
-have)
+have#have_v)
@@ -1656 +1656 @@
-in)
+in#in_prep)
@@ -1659 +1659 @@
-England)))
+England#NA)))
@@ -1664 +1664 @@
-stel)
+stel#steel_n1)
@@ -1666 +1666 @@
-penns)))))
+penns#pen_n2)))))
@@ -1671 +1671 @@
-because)
+because#because_adv)
@@ -1680 +1680 @@
-if)
+if#if_conj)
@@ -1687 +1687 @@
-you))
+you#you_pron))
@@ -1689 +1689 @@
-have)
+have#have_v)
@@ -1691 +1691 @@
-not))))
+not#not_adv))))
@@ -1696 +1696 @@
-I))
+I#i_pron))
@@ -1698 +1698 @@
-will)
+will#will_v1)
@@ -1700 +1700 @@
-indevour)
+indevour#endeavour_v)
@@ -1703 +1703 @@
-to)
+to#to_prep)
@@ -1705 +1705 @@
-gett)
+gett#get_v)
@@ -1708 +1708 @@
-you))
+you#you_pron))
@@ -1711 +1711 @@
-some))
+some#some_pron))
@@ -1714 +1714 @@
-by)
+by#by_prep)
@@ -1717 +1717 @@
-one)
+one#one_adj)
@@ -1722 +1722 @@
-that)
+that#that_conj)
@@ -1728 +1728 @@
-told)
+told#tell_v)
@@ -1731 +1731 @@
-me))
+me#me_pron1))
@@ -1734 +1734 @@
-of)
+of#of_prep)
@@ -1737 +1737 @@
-them))))
+them#them_pron))))
@@ -1742 +1742 @@
-and)
+and#and_conj1)
@@ -1747 +1747 @@
-did)
+did#do_v)
@@ -1749 +1749 @@
-asseur)
+asseur#assure_v)
@@ -1752 +1752 @@
-me))
+me#me_pron1))
@@ -1759 +1759 @@
-neither)
+neither#neither_adv)
@@ -1762 +1762 @@
-the)
+the#the_adj)
@@ -1764 +1764 @@
-glass)
+glass#glass_n1)
@@ -1766 +1766 @@
-penns))
+penns#pen_n2))
@@ -1769 +1769 @@
-nor)
+nor#nor_conj1)
@@ -1772 +1772 @@
-any)
+any#any_adj)
@@ -1774 +1774 @@
-other)
+other#other_adj)
@@ -1776 +1776 @@
-sorts))))
+sorts#sort_n2))))
@@ -1778 +1778 @@
-are)
+are#be_v)
@@ -1782 +1782 @@
-neare))
+neare#near_adj))
@@ -1784 +1784 @@
-soe)
+soe#so_adv)
@@ -1786 +1786 @@
-good))))))))))))))
+good#good_adj))))))))))))))
@@ -1795 +1795 @@
-The)
+The#the_adj)
@@ -1797 +1797 @@
-Duchesse)
+Duchesse#duchess_n)
@@ -1800 +1800 @@
-of)
+of#of_prep)
@@ -1803 +1803 @@
-Monmouth)))
+Monmouth#NA)))
@@ -1811 +1811 @@
-laitly))
+laitly#late_adv))
@@ -1813 +1813 @@
-arrived)
+arrived#arrive_v)
@@ -1816 +1816 @@
-at)
+at#at_prep)
@@ -1819 +1819 @@
-Paris)))))
+Paris#NA)))))
@@ -1823 +1823 @@
-went)
+went#go_v)
@@ -1826 +1826 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -1829 +1829 @@
-towards)
+towards#towards_prep)
@@ -1832 +1832 @@
-Bourbon)))
+Bourbon#NA)))
@@ -1841 +1841 @@
-She))
+She#she_pron))
@@ -1843 +1843 @@
-dus)
+dus#do_v)
@@ -1845 +1845 @@
-gaine)
+gaine#gain_v2)
@@ -1848 +1848 @@
-upon)
+upon#upon_prep)
@@ -1851 +1851 @@
-all)
+all#all_adj)
@@ -1856 +1856 @@
-that)
+that#that_conj)
@@ -1861 +1861 @@
-visitts)
+visitts#visit_v)
@@ -1864 +1864 @@
-her))
+her#her_pron1))
@@ -1867 +1867 @@
-here))
+here#here_adv))
@@ -1870 +1870 @@
-for)
+for#for_prep)
@@ -1873 +1873 @@
-her)
+her#her_pron2)
@@ -1875 +1875 @@
-tender)
+tender#tender_adj)
@@ -1877 +1877 @@
-hart)
+hart#heart_n)
@@ -1880 +1880 @@
-to)
+to#to_prep)
@@ -1883 +1883 @@
-the)
+the#the_adj)
@@ -1885 +1885 @@
-Romain)
+Romain#NA)
@@ -1887 +1887 @@
-Catho.)
+Catho.#NA)
@@ -1890 +1890 @@
-in)
+in#in_prep)
@@ -1893 +1893 @@
-Ingland)))))))))))
+Ingland#NA)))))))))))
@@ -1902 +1902 @@
-Doctor)
+Doctor#NA)
@@ -1904 +1904 @@
-Goffe))
+Goffe#NA))
@@ -1906 +1906 @@
-taks)
+taks#take_v)
@@ -1909 +1909 @@
-ye)
+ye#the_adj)
@@ -1911 +1911 @@
-care))
+care#care_n1))
@@ -1914 +1914 @@
-of)
+of#of_prep)
@@ -1917 +1917 @@
-all)
+all#all_adj)
@@ -1919 +1919 @@
-her)
+her#her_pron2)
@@ -1921 +1921 @@
-affaires)
+affaires#affair_n)
@@ -1924 +1924 @@
-here))))
+here#here_adv))))
@@ -1933 +1933 @@
-Y=r=)
+Y=r=#your_pron)
@@ -1935 +1935 @@
-lo=p=))
+lo=p=#lordship_n))
@@ -1938 +1938 @@
-most)
+most#most_adj)
@@ -1940 +1940 @@
-affec.))
+affec.#affectionate_adj))
@@ -1942 +1942 @@
-sister)
+sister#sister_n)
@@ -1945 +1945 @@
-and)
+and#and_conj1)
@@ -1948 +1948 @@
-humble)
+humble#humble_adj)
@@ -1950 +1950 @@
-servant)
+servant#servant_n)
@@ -1958 +1958 @@
-to)
+to#to_prep)
@@ -1960 +1960 @@
-command)))))
+command#command_v)))))
