--- moxinden-1640-e2-p1.psd.vert	2025-09-10 13:51:11
+++ moxinden-1640-e2-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-Good)
+Good#good_adj)
@@ -16 +16 @@
-Nephew))
+Nephew#nephew_n))
@@ -21 +21 @@
-I))
+I#i_pron))
@@ -23 +23 @@
-am)
+am#be_v)
@@ -26 +26 @@
-very)
+very#very_adj)
@@ -28 +28 @@
-sorry)
+sorry#sorry_adj)
@@ -31 +31 @@
-to)
+to#to_prep)
@@ -33 +33 @@
-hear)
+hear#hear_v)
@@ -36 +36 @@
-that)
+that#that_conj)
@@ -40 +40 @@
-my)
+my#my_adj)
@@ -42 +42 @@
-Neece))
+Neece#niece_n))
@@ -44 +44 @@
-is)
+is#be_v)
@@ -47 +47 @@
-so)
+so#so_adv)
@@ -49 +49 @@
-ill))
+ill#ill_adj))
@@ -52 +52 @@
-still))))))
+still#still_adv))))))
@@ -60 +60 @@
-but)
+but#but_prep)
@@ -63 +63 @@
-shuch)
+shuch#such_adj)
@@ -67 +67 @@
-is)
+is#be_v)
@@ -70 +70 @@
-the)
+the#the_adj)
@@ -72 +72 @@
-Nature)
+Nature#nature_n)
@@ -75 +75 @@
-of)
+of#of_prep)
@@ -78 +78 @@
-this)
+this#this_pron)
@@ -80 +80 @@
-kind)
+kind#kind_n)
@@ -83 +83 @@
-of)
+of#of_prep)
@@ -86 +86 @@
-sikneses))))))
+sikneses#sickness_n))))))
@@ -89 +89 @@
-that)
+that#that_conj)
@@ -93 +93 @@
-I))
+I#i_pron))
@@ -95 +95 @@
-am)
+am#be_v)
@@ -98 +98 @@
-veryly))
+veryly#verily_adv))
@@ -100 +100 @@
-perswayd)
+perswayd#persway_v)
@@ -107 +107 @@
-it))
+it#it_pron))
@@ -109 +109 @@
-is)
+is#be_v)
@@ -111 +111 @@
-not)
+not#not_adv)
@@ -114 +114 @@
-in)
+in#in_prep)
@@ -117 +117 @@
-the)
+the#the_adj)
@@ -119 +119 @@
-powr)
+powr#power_n1)
@@ -122 +122 @@
-of)
+of#of_prep)
@@ -125 +125 @@
-any)
+any#any_adj)
@@ -127 +127 @@
-phisition)))
+phisition#physician_n)))
@@ -133 +133 @@
-to)
+to#to_prep)
@@ -135 +135 @@
-alter)
+alter#alter_v)
@@ -146 +146 @@
-they))
+they#they_pron))
@@ -148 +148 @@
-are)
+are#be_v)
@@ -151 +151 @@
-the)
+the#the_adj)
@@ -153 +153 @@
-helpes)
+helpes#help_n)
@@ -156 +156 @@
-of)
+of#of_prep)
@@ -159 +159 @@
-natur)))
+natur#nature_n)))
@@ -163 +163 @@
-which))
+which#which_adj))
@@ -171 +171 @@
-if)
+if#if_conj)
@@ -178 +178 @@
-thay))
+thay#they_pron))
@@ -180 +180 @@
-be)
+be#be_v)
@@ -183 +183 @@
-to)
+to#too_adv)
@@ -185 +185 @@
-beisi)))))
+beisi#busy_adj)))))
@@ -187 +187 @@
-prove)
+prove#prove_v)
@@ -190 +190 @@
-the)
+the#the_adj)
@@ -192 +192 @@
-destriors)))))
+destriors#destroyer_n)))))
@@ -201 +201 @@
-Natur))
+Natur#nature_n))
@@ -203 +203 @@
-helps)
+helps#help_v)
@@ -206 +206 @@
-her)
+her#her_pron1)
@@ -208 +208 @@
-self))
+self#self_pron))
@@ -211 +211 @@
-by)
+by#by_prep)
@@ -214 +214 @@
-the)
+the#the_adj)
@@ -216 +216 @@
-cof)
+cof#cough_n)
@@ -221 +221 @@
-that)
+that#that_conj)
@@ -227 +227 @@
-my)
+my#my_adj)
@@ -229 +229 @@
-cosin))
+cosin#cousin_n))
@@ -231 +231 @@
-hath)))
+hath#have_v)))
@@ -235 +235 @@
-which))
+which#which_adj))
@@ -238 +238 @@
-desier)
+desier#desire_v)
@@ -241 +241 @@
-her))
+her#her_pron1))
@@ -246 +246 @@
-to)
+to#to_prep)
@@ -248 +248 @@
-bear)
+bear#bear_v1)
@@ -251 +251 @@
-with)
+with#with_prep)
@@ -254 +254 @@
-as)
+as#as_adv)
@@ -256 +256 @@
-much)
+much#much_adj)
@@ -258 +258 @@
-cherfullnes)
+cherfullnes#cheerfulness_n)
@@ -261 +261 @@
-as)
+as#as_adv)
@@ -272 +272 @@
-she))
+she#she_pron))
@@ -274 +274 @@
-may)
+may#may_v1)
@@ -285 +285 @@
-I))
+I#i_pron))
@@ -287 +287 @@
-send)
+send#send_v1)
@@ -290 +290 @@
-her))
+her#her_pron1))
@@ -293 +293 @@
-heer))
+heer#here_adv))
@@ -296 +296 @@
-an)
+an#a_adj2)
@@ -298 +298 @@
-oyntmen)
+oyntmen#ointment_n)
@@ -302 +302 @@
-which))
+which#which_adj))
@@ -308 +308 @@
-I))
+I#i_pron))
@@ -310 +310 @@
-desire)
+desire#desire_v)
@@ -313 +313 @@
-her))
+her#her_pron1))
@@ -317 +317 @@
-to)
+to#to_prep)
@@ -319 +319 @@
-anoynt)
+anoynt#anoint_v)
@@ -322 +322 @@
-her)
+her#her_pron2)
@@ -324 +324 @@
-forhed))
+forhed#forehead_n))
@@ -327 +327 @@
-with)
+with#with_prep)
@@ -332 +332 @@
-when)
+when#when_adv)
@@ -339 +339 @@
-she))
+she#she_pron))
@@ -341 +341 @@
-goeth)
+goeth#go_v)
@@ -344 +344 @@
-to)
+to#to_prep)
@@ -347 +347 @@
-bead)))))))
+bead#bead_n)))))))
@@ -350 +350 @@
-and)
+and#and_conj1)
@@ -355 +355 @@
-take)
+take#take_v)
@@ -358 +358 @@
-this)
+this#this_pron)
@@ -360 +360 @@
-Cordiall))
+Cordiall#cordial_n))
@@ -363 +363 @@
-to)
+to#to_prep)
@@ -365 +365 @@
-procur)
+procur#procure_v)
@@ -368 +368 @@
-rest)))))))))
+rest#rest_n1)))))))))
@@ -377 +377 @@
-I))
+I#i_pron))
@@ -379 +379 @@
-dare)
+dare#dare_v1)
@@ -381 +381 @@
-not)
+not#not_adv)
@@ -383 +383 @@
-go)
+go#go_v)
@@ -386 +386 @@
-beyond)
+beyond#beyond_adv)
@@ -389 +389 @@
-a)
+a#a_adj2)
@@ -391 +391 @@
-Cordiall))
+Cordiall#cordial_n))
@@ -395 +395 @@
-whear))
+whear#where_adv))
@@ -403 +403 @@
-ther))
+ther#there_adv))
@@ -405 +405 @@
-is)
+is#be_v)
@@ -408 +408 @@
-so)
+so#so_adv)
@@ -410 +410 @@
-much)
+much#much_adj)
@@ -412 +412 @@
-illness))))))
+illness#illness_n))))))
@@ -418 +418 @@
-butt)
+butt#but_prep)
@@ -421 +421 @@
-this))
+this#this_pron))
@@ -423 +423 @@
-pray)
+pray#pray_v)
@@ -426 +426 @@
-her))
+her#her_pron1))
@@ -429 +429 @@
-to)
+to#to_prep)
@@ -431 +431 @@
-be)
+be#be_v)
@@ -434 +434 @@
-confidend)
+confidend#confiding_adj)
@@ -443 +443 @@
-the)
+the#the_adj)
@@ -445 +445 @@
-queene))
+queene#queen_n))
@@ -447 +447 @@
-may)
+may#may_v1)
@@ -449 +449 @@
-take)
+take#take_v)
@@ -452 +452 @@
-at)
+at#at_prep)
@@ -455 +455 @@
-my)
+my#my_adj)
@@ -457 +457 @@
-hands)))))))
+hands#hand_n)))))))
@@ -467 +467 @@
-I))
+I#i_pron))
@@ -469 +469 @@
-will)
+will#will_v1)
@@ -471 +471 @@
-make)
+make#make_v1)
@@ -474 +474 @@
-her))
+her#her_pron1))
@@ -477 +477 @@
-a)
+a#a_adj2)
@@ -479 +479 @@
-tisain)
+tisain#tisane_n)
@@ -484 +484 @@
-to)
+to#to_prep)
@@ -487 +487 @@
-morow))))
+morow#morrow_n))))
@@ -490 +490 @@
-and)
+and#and_conj1)
@@ -493 +493 @@
-send)
+send#send_v1)
@@ -496 +496 @@
-her))
+her#her_pron1))
@@ -499 +499 @@
-to)
+to#to_prep)
@@ -501 +501 @@
-eas)
+eas#ease_v)
@@ -504 +504 @@
-the)
+the#the_adj)
@@ -506 +506 @@
-paynfullnes)
+paynfullnes#painfulness_n)
@@ -509 +509 @@
-of)
+of#of_prep)
@@ -512 +512 @@
-her)
+her#her_pron2)
@@ -514 +514 @@
-coff)))))))
+coff#cough_n)))))))
@@ -520 +520 @@
-which))
+which#which_adj))
@@ -526 +526 @@
-you))
+you#you_pron))
@@ -528 +528 @@
-shall)
+shall#shall_v)
@@ -530 +530 @@
-have)
+have#have_v)
@@ -535 +535 @@
-sent)
+sent#send_v1)
@@ -538 +538 @@
-you))
+you#you_pron))
@@ -541 +541 @@
-by)
+by#by_prep)
@@ -545,3 +545,3 @@
-to)
-(CONJ
-or)
+to#two_adj)
+(CONJ
+or#or_conj2)
@@ -549 +549 @@
-thre))
+thre#three_adj))
@@ -552 +552 @@
-of)
+of#of_prep)
@@ -555 +555 @@
-the)
+the#the_adj)
@@ -557 +557 @@
-clok))))))))
+clok#clock_n1))))))))
@@ -567 +567 @@
-so))
+so#so_adv))
@@ -569 +569 @@
-praying)
+praying#pray_v)
@@ -572 +572 @@
-god))
+god#NA))
@@ -575 +575 @@
-to)
+to#to_prep)
@@ -577 +577 @@
-send)
+send#send_v1)
@@ -580 +580 @@
-her))
+her#her_pron1))
@@ -583 +583 @@
-ese))))
+ese#ease_n))))
@@ -586 +586 @@
-I))
+I#i_pron))
@@ -588 +588 @@
-am)
+am#be_v)
@@ -591 +591 @@
-Your)
+Your#your_pron)
@@ -593 +593 @@
-afectionat)
+afectionat#affectionate_adj)
@@ -595 +595 @@
-Ant)
+Ant#aunt_n)
@@ -598 +598 @@
-M.)
+M.#NA)
@@ -600 +600 @@
-O.))))
+O.#NA))))
@@ -613 +613 @@
-Good)
+Good#good_adj)
@@ -615 +615 @@
-Cosin))
+Cosin#cousin_n))
@@ -620 +620 @@
-I))
+I#i_pron))
@@ -622 +622 @@
-am)
+am#be_v)
@@ -625 +625 @@
-exceding)
+exceding#exceeding_adj)
@@ -627 +627 @@
-sory)
+sory#sorry_adj)
@@ -634 +634 @@
-my)
+my#my_adj)
@@ -636 +636 @@
-sister)
+sister#sister_n)
@@ -639 +639 @@
-Oxinden)))
+Oxinden#NA)))
@@ -641 +641 @@
-is)
+is#be_v)
@@ -644 +644 @@
-so)
+so#so_adv)
@@ -646 +646 @@
-ill)))))
+ill#ill_adj)))))
@@ -655 +655 @@
-I))
+I#i_pron))
@@ -657,3 +657,3 @@
-will)
-(NEG
-not)
+will#will_v1)
+(NEG
+not#not_adv)
@@ -661 +661 @@
-fayell)
+fayell#fail_v)
@@ -664 +664 @@
-to)
+to#to_prep)
@@ -666 +666 @@
-visit)
+visit#visit_v)
@@ -669 +669 @@
-her))
+her#her_pron1))
@@ -672 +672 @@
-so)
+so#so_adv)
@@ -674 +674 @@
-sone)
+sone#soon_adv)
@@ -677 +677 @@
-as)
+as#as_adv)
@@ -688 +688 @@
-I))
+I#i_pron))
@@ -690 +690 @@
-can)
+can#can_v1)
@@ -693 +693 @@
-posible))
+posible#possible_adj))
@@ -704 +704 @@
-in)
+in#in_prep)
@@ -707 +707 @@
-the)
+the#the_adj)
@@ -709 +709 @@
-mene)
+mene#mean_adj2)
@@ -711 +711 @@
-time)))
+time#time_n)))
@@ -714 +714 @@
-I))
+I#i_pron))
@@ -716 +716 @@
-have)
+have#have_v)
@@ -718 +718 @@
-sent)
+sent#send_v1)
@@ -721 +721 @@
-her))
+her#her_pron1))
@@ -724 +724 @@
-a)
+a#a_adj2)
@@ -726 +726 @@
-water)
+water#water_n)
@@ -731 +731 @@
-for)
+for#for_prep)
@@ -734 +734 @@
-wind)))
+wind#wind_n1)))
@@ -739 +739 @@
-that)
+that#that_conj)
@@ -743 +743 @@
-I))
+I#i_pron))
@@ -745 +745 @@
-have)
+have#have_v)
@@ -747 +747 @@
-found)
+found#find_v)
@@ -751 +751 @@
-very)
+very#very_adj)
@@ -753 +753 @@
-excellent))
+excellent#excellent_adj))
@@ -755 +755 @@
-efects))
+efects#effect_n))
@@ -758 +758 @@
-of)
+of#of_prep)
@@ -769 +769 @@
-I))
+I#i_pron))
@@ -771 +771 @@
-desire)
+desire#desire_v)
@@ -774 +774 @@
-her))
+her#her_pron1))
@@ -778 +778 @@
-to)
+to#to_prep)
@@ -780 +780 @@
-take)
+take#take_v)
@@ -783 +783 @@
-it))
+it#it_pron))
@@ -786 +786 @@
-with)
+with#with_prep)
@@ -789 +789 @@
-shuger)))
+shuger#sugar_n)))
@@ -795 +795 @@
-a)
+a#a_adj2)
@@ -799 +799 @@
-spoonfull)
+spoonfull#spoonful_n)
@@ -802 +802 @@
-of)
+of#of_prep)
@@ -805 +805 @@
-the)
+the#the_adj)
@@ -807 +807 @@
-water)
+water#water_n)
@@ -810 +810 @@
-filled)
+filled#fill_v)
@@ -813 +813 @@
-full)
+full#full_adj)
@@ -816 +816 @@
-of)
+of#of_prep)
@@ -819 +819 @@
-shuger))))))))))
+shuger#sugar_n))))))))))
@@ -822 +822 @@
-and)
+and#and_conj1)
@@ -826 +826 @@
-so))
+so#so_adv))
@@ -829 +829 @@
-rather)
+rather#rather_adv)
@@ -835 +835 @@
-cay)
+cay#NA)
@@ -838 +838 @@
-it))
+it#it_pron))
@@ -841 +841 @@
-then)
+then#than_conj)
@@ -846 +846 @@
-drink)
+drink#drink_v1)
@@ -849 +849 @@
-it)))))))
+it#it_pron)))))))
@@ -858 +858 @@
-I))
+I#i_pron))
@@ -860 +860 @@
-send)
+send#send_v1)
@@ -863 +863 @@
-her))
+her#her_pron1))
@@ -865 +865 @@
-allso)
+allso#also_adv)
@@ -868 +868 @@
-a)
+a#a_adj2)
@@ -870 +870 @@
-powder)
+powder#powder_n1)
@@ -874 +874 @@
-which))
+which#which_adj))
@@ -880 +880 @@
-I))
+I#i_pron))
@@ -882 +882 @@
-wold)
+wold#will_v1)
@@ -884 +884 @@
-have)
+have#have_v)
@@ -890 +890 @@
-her))
+her#her_pron1))
@@ -892 +892 @@
-take)
+take#take_v)
@@ -895 +895 @@
-in)
+in#in_prep)
@@ -899 +899 @@
-a)
+a#a_adj2)
@@ -901 +901 @@
-litel))
+litel#little_adj))
@@ -903 +903 @@
-beer)
+beer#beer_n1)
@@ -906 +906 @@
-or)
+or#or_conj2)
@@ -909 +909 @@
-posit)))
+posit#posset_n)))
@@ -915 +915 @@
-which))
+which#which_adj))
@@ -923 +923 @@
-she))
+she#she_pron))
@@ -925 +925 @@
-likes)
+likes#like_v1)
@@ -928 +928 @@
-best))))))))))
+best#best_adj))))))))))
@@ -938 +938 @@
-as)
+as#as_adv)
@@ -940 +940 @@
-much))
+much#much_adj))
@@ -943 +943 @@
-as)
+as#as_adv)
@@ -953 +953 @@
-will)
+will#will_v1)
@@ -955 +955 @@
-ly)
+ly#lie_v1)
@@ -958 +958 @@
-a_pon)
+a_pon#upon_prep)
@@ -961 +961 @@
-3@)
+3@#three_adj)
@@ -963 +963 @@
-@d.)))))))
+@d.#penny_n)))))))
@@ -965 +965 @@
-will)
+will#will_v1)
@@ -967 +967 @@
-be)
+be#be_v)
@@ -970 +970 @@
-enuf))
+enuf#enough_adj))
@@ -973 +973 @@
-at)
+at#at_prep)
@@ -976 +976 @@
-a)
+a#a_adj2)
@@ -978 +978 @@
-time)))
+time#time_n)))
@@ -988 +988 @@
-that))
+that#that_pron))
@@ -991 +991 @@
-or)
+or#or_conj2)
@@ -994 +994 @@
-the)
+the#the_adj)
@@ -996 +996 @@
-water))))
+water#water_n))))
@@ -998 +998 @@
-may@)
+may@#may_v1)
@@ -1000 +1000 @@
-@be)
+@be#be_v)
@@ -1002 +1002 @@
-taken)
+taken#take_v)
@@ -1005 +1005 @@
-at)
+at#at_prep)
@@ -1008 +1008 @@
-any)
+any#any_adj)
@@ -1010 +1010 @@
-tim)))
+tim#time_n)))
@@ -1013 +1013 @@
-when)
+when#when_adv)
@@ -1020 +1020 @@
-she))
+she#she_pron))
@@ -1022 +1022 @@
-is)
+is#be_v)
@@ -1025 +1025 @@
-ill)))))
+ill#ill_adj)))))
@@ -1034 +1034 @@
-She))
+She#she_pron))
@@ -1036 +1036 @@
-may)
+may#may_v1)
@@ -1038 +1038 @@
-take)
+take#take_v)
@@ -1041 +1041 @@
-this)
+this#this_pron)
@@ -1043 +1043 @@
-water))
+water#water_n))
@@ -1046 +1046 @@
-with)
+with#with_prep)
@@ -1049 +1049 @@
-heat)))
+heat#heat_n)))
@@ -1052 +1052 @@
-as)
+as#as_adv)
@@ -1063 +1063 @@
-other)
+other#other_adj)
@@ -1065 +1065 @@
-hot)
+hot#hot_adj)
@@ -1067 +1067 @@
-water))
+water#water_n))
@@ -1069 +1069 @@
-is)
+is#be_v)
@@ -1071 +1071 @@
-takin))))
+takin#take_v))))
@@ -1082 +1082 @@
-so))
+so#so_adv))
@@ -1084 +1084 @@
-wishing)
+wishing#wish_v)
@@ -1087 +1087 @@
-her))
+her#her_pron1))
@@ -1090 +1090 @@
-health)))
+health#health_n)))
@@ -1093 +1093 @@
-and)
+and#and_conj1)
@@ -1097 +1097 @@
-you))
+you#you_pron))
@@ -1100 +1100 @@
-all)
+all#all_adj)
@@ -1102 +1102 @@
-hapines)))))
+hapines#happiness_n)))))
@@ -1105 +1105 @@
-I))
+I#i_pron))
@@ -1107 +1107 @@
-rest)
+rest#rest_v2)
@@ -1110 +1110 @@
-Your)
+Your#your_pron)
@@ -1112 +1112 @@
-afectionat)
+afectionat#affectionate_adj)
@@ -1114 +1114 @@
-frend)
+frend#friend_n)
@@ -1117 +1117 @@
-and)
+and#and_conj1)
@@ -1120 +1120 @@
-Ant)))
+Ant#aunt_n)))
@@ -1123 +1123 @@
-Marg:)
+Marg:#NA)
@@ -1125 +1125 @@
-Oxinden)
+Oxinden#NA)
@@ -1127 +1127 @@
-Dean))))
+Dean#NA))))
@@ -1133 +1133 @@
-the)
+the#the_adj)
@@ -1135 +1135 @@
-14)
+14#fourteen_adj)
@@ -1137 +1137 @@
-1640))
+1640#NA))
