--- mtudor-1515-e1-p1.psd.vert	2025-09-10 13:51:11
+++ mtudor-1515-e1-p1.lem.vert	2025-09-10 13:50:58
@@ -17 +17 @@
-My)
+My#my_adj)
@@ -19 +19 @@
-good)
+good#good_adj)
@@ -21 +21 @@
-Brother))
+Brother#brother_n))
@@ -24 +24 @@
-as)
+as#as_adv)
@@ -26 +26 @@
-hertly)
+hertly#heartily_adv)
@@ -29 +29 @@
-as)
+as#as_adv)
@@ -40 +40 @@
-I))
+I#i_pron))
@@ -42 +42 @@
-can)
+can#can_v1)
@@ -47 +47 @@
-I))
+I#i_pron))
@@ -49 +49 @@
-recomaund)
+recomaund#recommend_v1)
@@ -52 +52 @@
-me))
+me#me_pron1))
@@ -55 +55 @@
-unto)
+unto#unto_prep)
@@ -58 +58 @@
-youre)
+youre#your_pron)
@@ -60 +60 @@
-Grace)))
+Grace#grace_n)))
@@ -65 +65 @@
-mervelynge)
+mervelynge#marvel_v)
@@ -68 +68 @@
-moch))
+moch#much_adj))
@@ -71 +71 @@
-that)
+that#that_conj)
@@ -75 +75 @@
-I))
+I#i_pron))
@@ -78 +78 @@
-never))
+never#never_adv))
@@ -80 +80 @@
-heard)
+heard#hear_v)
@@ -83 +83 @@
-from)
+from#from_prep)
@@ -86 +86 @@
-you)))
+you#you_pron)))
@@ -89 +89 @@
-syns)
+syns#since_adv)
@@ -92 +92 @@
-...re)
+...re#your_pron)
@@ -94 +94 @@
-depertynge)))
+depertynge#departing_n)))
@@ -99 +99 @@
-so)
+so#so_adv)
@@ -101 +101 @@
-often)
+often#often_adv)
@@ -104 +104 @@
-as)
+as#as_adv)
@@ -115 +115 @@
-I))
+I#i_pron))
@@ -117 +117 @@
-have)
+have#have_v)
@@ -120 +120 @@
-sent)
+sent#send_v1)
@@ -122 +122 @@
-and)
+and#and_conj1)
@@ -124 +124 @@
-wrytten))
+wrytten#write_v))
@@ -127 +127 @@
-to)
+to#to_prep)
@@ -130 +130 @@
-you))))))))))
+you#you_pron))))))))))
@@ -138 +138 @@
-And)
+And#and_conj1)
@@ -141 +141 @@
-now))
+now#now_adv))
@@ -143 +143 @@
-am)
+am#be_v)
@@ -146 +146 @@
-I))
+I#i_pron))
@@ -148 +148 @@
-left)
+left#leave_v1)
@@ -151 +151 @@
-post_a_lone))
+post_a_lone#post-alone_adv))
@@ -154 +154 @@
-in)
+in#in_prep)
@@ -157 +157 @@
-effect)))
+effect#effect_n)))
@@ -166 +166 @@
-for)
+for#for_prep)
@@ -169 +169 @@
-on)
+on#on_prep)
@@ -172 +172 @@
-the)
+the#the_adj)
@@ -174 +174 @@
-morn)
+morn#morn_n)
@@ -177 +177 @@
-next)
+next#next_adj)
@@ -179 +179 @@
-after)
+after#after_adv)
@@ -182 +182 @@
-...e)
+...e#the_adj)
@@ -184 +184 @@
-maryage)))))
+maryage#marriage_n)))))
@@ -187 +187 @@
-my)
+my#my_adj)
@@ -189 +189 @@
-chambirlayn)
+chambirlayn#chamberlain_n)
@@ -192 +192 @@
-with)
+with#with_prep)
@@ -195 +195 @@
-all)
+all#all_adj)
@@ -197 +197 @@
-other)
+other#other_adj)
@@ -199 +199 @@
-men)
+men#man_n1)
@@ -201 +201 @@
-servants))))
+servants#servant_n))))
@@ -203 +203 @@
-wer)
+wer#be_v)
@@ -205 +205 @@
-dischargd))
+dischargd#discharge_v))
@@ -208 +208 @@
-and)
+and#and_conj1)
@@ -212 +212 @@
-in)
+in#in_prep)
@@ -215 +215 @@
-lyke)
+lyke#like_adj)
@@ -217 +217 @@
-wyse)))
+wyse#wise_n1)))
@@ -220 +220 @@
-my)
+my#my_adj)
@@ -222 +222 @@
-mother)
+mother#mother_n1)
@@ -225 +225 @@
-Guldeford)))
+Guldeford#NA)))
@@ -228 +228 @@
-with)
+with#with_prep)
@@ -231 +231 @@
-other)
+other#other_adj)
@@ -233 +233 @@
-my)
+my#my_adj)
@@ -235 +235 @@
-women)
+women#woman_n)
@@ -238 +238 @@
-and)
+and#and_conj1)
@@ -241 +241 @@
-maydyns)))))
+maydyns#maiden_n)))))
@@ -246 +246 @@
-except)
+except#except_adj)
@@ -249 +249 @@
-such)
+such#such_adj)
@@ -252 +252 @@
-as)
+as#as_adv)
@@ -263 +263 @@
-never))
+never#never_adv))
@@ -265 +265 @@
-had)
+had#have_v)
@@ -268 +268 @@
-experiens)
+experiens#experience_n)
@@ -270 +270 @@
-nor)
+nor#nor_conj1)
@@ -272 +272 @@
-knowlech)
+knowlech#knowledge_n)
@@ -276 +276 @@
-how))
+how#how_adv))
@@ -281 +281 @@
-to)
+to#to_prep)
@@ -284 +284 @@
-advertyse)
+advertyse#advertise_v)
@@ -286 +286 @@
-or)
+or#or_conj2)
@@ -288 +288 @@
-gyfe))
+gyfe#give_v))
@@ -291 +291 @@
-me))
+me#me_pron1))
@@ -294 +294 @@
-counsell))
+counsell#counsel_n))
@@ -297 +297 @@
-yn)
+yn#in_prep)
@@ -300 +300 @@
-any)
+any#any_adj)
@@ -302 +302 @@
-time)
+time#time_n)
@@ -305 +305 @@
-of)
+of#of_prep)
@@ -308 +308 @@
-nede)))
+nede#need_n1)))
@@ -314 +314 @@
-which))
+which#which_adj))
@@ -321 +321 @@
-is)
+is#be_v)
@@ -324 +324 @@
-to)
+to#to_prep)
@@ -326 +326 @@
-be)
+be#be_v)
@@ -328 +328 @@
-fered)
+fered#fear_v)
@@ -331 +331 @@
-more)
+more#more_adj)
@@ -333 +333 @@
-schortly)
+schortly#shortly_adv)
@@ -336 +336 @@
-then)
+then#than_conj)
@@ -347 +347 @@
-your)
+your#your_pron)
@@ -349 +349 @@
-Grace))
+Grace#grace_n))
@@ -351 +351 @@
-thought)
+thought#think_v2)
@@ -354 +354 @@
-at)
+at#at_prep)
@@ -357 +357 @@
-the)
+the#the_adj)
@@ -359 +359 @@
-tyme)
+tyme#time_n)
@@ -362 +362 @@
-of)
+of#of_prep)
@@ -365 +365 @@
-my)
+my#my_adj)
@@ -367 +367 @@
-departynge)))))))))
+departynge#departing_n)))))))))
@@ -372 +372 @@
-as)
+as#as_adv)
@@ -383 +383 @@
-my)
+my#my_adj)
@@ -385 +385 @@
-mother)
+mother#mother_n1)
@@ -388 +388 @@
-Guldeford))
+Guldeford#NA))
@@ -392 +392 @@
-can)
+can#can_v1)
@@ -395 +395 @@
-more)
+more#more_adj)
@@ -397 +397 @@
-playnly)
+playnly#plainly_adv1)
@@ -401 +401 @@
-schew)
+schew#show_v)
@@ -404 +404 @@
-your)
+your#your_pron)
@@ -406 +406 @@
-Grace))
+Grace#grace_n))
@@ -409 +409 @@
-then)
+then#than_conj)
@@ -420 +420 @@
-I))
+I#i_pron))
@@ -422 +422 @@
-cann)
+cann#can_v1)
@@ -424 +424 @@
-wryt))))
+wryt#write_v))))
@@ -430 +430 @@
-to)
+to#to_prep)
@@ -433 +433 @@
-whom)))
+whom#whom_pron)))
@@ -439 +439 @@
-I))
+I#i_pron))
@@ -441 +441 @@
-beseche)
+beseche#beseech_v)
@@ -444 +444 @@
-you))
+you#you_pron))
@@ -449 +449 @@
-to)
+to#to_prep)
@@ -451 +451 @@
-gyve)
+gyve#give_v)
@@ -454 +454 @@
-credens)))))))))))))))))))))))
+credens#credence_n)))))))))))))))))))))))
@@ -462 +462 @@
-And)
+And#and_conj1)
@@ -465 +465 @@
-yf)
+yf#if_conj)
@@ -472 +472 @@
-hit))
+hit#it_pron))
@@ -474 +474 @@
-may)
+may#may_v1)
@@ -476 +476 @@
-be)
+be#be_v)
@@ -479 +479 @@
-by)
+by#by_prep)
@@ -482 +482 @@
-eny)
+eny#any_adj)
@@ -484 +484 @@
-meane)))
+meane#mean_n2)))
@@ -487 +487 @@
-possible)))))
+possible#possible_adj)))))
@@ -492 +492 @@
-I))
+I#i_pron))
@@ -495 +495 @@
-humbly))
+humbly#humbly_adv))
@@ -497 +497 @@
-requyr)
+requyr#require_v)
@@ -500 +500 @@
-you))
+you#you_pron))
@@ -503 +503 @@
-to)
+to#to_prep)
@@ -505 +505 @@
-cause)
+cause#cause_v1)
@@ -508 +508 @@
-my)
+my#my_adj)
@@ -510 +510 @@
-seyd)
+seyd#said_adj)
@@ -512 +512 @@
-mother)
+mother#mother_n1)
@@ -515 +515 @@
-Guldeford)))
+Guldeford#NA)))
@@ -518 +518 @@
-to)
+to#to_prep)
@@ -520 +520 @@
-repayr)
+repayr#repair_v1)
@@ -523 +523 @@
-hither))
+hither#hither_adv))
@@ -528 +528 @@
-once)
+once#once_adv)
@@ -530 +530 @@
-agayn))))
+agayn#again_adv))))
@@ -538 +538 @@
-For)
+For#for_prep)
@@ -541 +541 @@
-ells))
+ells#else_adv))
@@ -544 +544 @@
-yf)
+yf#if_conj)
@@ -551 +551 @@
-any)
+any#any_adj)
@@ -553 +553 @@
-chauns))
+chauns#chance_n))
@@ -555 +555 @@
-happe)
+happe#hap_v1)
@@ -558 +558 @@
-other)
+other#other_adj)
@@ -561 +561 @@
-then)
+then#than_conj)
@@ -564 +564 @@
-weale)))))))
+weale#well_adv)))))))
@@ -567 +567 @@
-I))
+I#i_pron))
@@ -569 +569 @@
-schall)
+schall#shall_v)
@@ -571 +571 @@
-not)
+not#not_adv)
@@ -573 +573 @@
-knowe)
+knowe#know_v)
@@ -578 +578 @@
-wher))
+wher#where_adv))
@@ -581 +581 @@
-nor)
+nor#nor_conj1)
@@ -584 +584 @@
-of)
+of#of_prep)
@@ -587 +587 @@
-whom)))))
+whom#whom_pron)))))
@@ -593 +593 @@
-to)
+to#to_prep)
@@ -595 +595 @@
-aske)
+aske#ask_v)
@@ -598 +598 @@
-any)
+any#any_adj)
@@ -600 +600 @@
-good)
+good#good_adj)
@@ -602 +602 @@
-counsell))
+counsell#counsel_n))
@@ -605 +605 @@
-to)
+to#to_prep)
@@ -608 +608 @@
-your)
+your#your_pron)
@@ -610 +610 @@
-pleasur))))
+pleasur#pleasure_n))))
@@ -615 +615 @@
-nor)
+nor#nor_conj1)
@@ -619 +619 @@
-yet))
+yet#yet_adv))
@@ -622 +622 @@
-to)
+to#to_prep)
@@ -625 +625 @@
-myn)
+myn#mine_adj)
@@ -627 +627 @@
-own)
+own#own_adj)
@@ -629 +629 @@
-proffit)))))))
+proffit#profit_n)))))))
@@ -638 +638 @@
-I))
+I#i_pron))
@@ -640 +640 @@
-merveill)
+merveill#marvel_v)
@@ -643 +643 @@
-moche))
+moche#much_adj))
@@ -646 +646 @@
-that)
+that#that_conj)
@@ -650 +650 @@
-my)
+my#my_adj)
@@ -652 +652 @@
-Lord)
+Lord#lord_n)
@@ -655 +655 @@
-of)
+of#of_prep)
@@ -658 +658 @@
-Northfolke))))
+Northfolke#NA))))
@@ -660 +660 @@
-wold)
+wold#will_v1)
@@ -663 +663 @@
-at)
+at#at_prep)
@@ -666 +666 @@
-all)
+all#all_adj)
@@ -668 +668 @@
-tymes)))
+tymes#time_n)))
@@ -671 +671 @@
-so)
+so#so_adv)
@@ -673 +673 @@
-lyghtly))
+lyghtly#lightly_adv))
@@ -675 +675 @@
-grant)
+grant#grant_v)
@@ -678 +678 @@
-every)
+every#every_adj)
@@ -680 +680 @@
-thynge))
+thynge#thing_n1))
@@ -683 +683 @@
-at)
+at#at_prep)
@@ -686 +686 @@
-ther)
+ther#their_adj)
@@ -688 +688 @@
-reqwests)))
+reqwests#request_n1)))
@@ -691 +691 @@
-here))))
+here#here_adv))))
@@ -700 +700 @@
-I))
+I#i_pron))
@@ -702 +702 @@
-am)
+am#be_v)
@@ -705 +705 @@
-weale))
+weale#well_adv))
@@ -707 +707 @@
-assured)
+assured#assure_v)
@@ -710 +710 @@
-that)
+that#that_conj)
@@ -714 +714 @@
-when)
+when#when_adv)
@@ -721 +721 @@
-ze))
+ze#ye_pron))
@@ -723 +723 @@
-know)
+know#know_v)
@@ -726 +726 @@
-the)
+the#the_adj)
@@ -728 +728 @@
-trouth)
+trouth#troth_n)
@@ -731 +731 @@
-of)
+of#of_prep)
@@ -734 +734 @@
-every)
+every#every_adj)
@@ -736 +736 @@
-thyng))))
+thyng#thing_n1))))
@@ -739 +739 @@
-as)
+as#as_adv)
@@ -750 +750 @@
-my)
+my#my_adj)
@@ -752 +752 @@
-mother)
+mother#mother_n1)
@@ -755 +755 @@
-Guldeford)))
+Guldeford#NA)))
@@ -757 +757 @@
-can)
+can#can_v1)
@@ -759 +759 @@
-schew)
+schew#show_v)
@@ -762 +762 @@
-you))))))))
+you#you_pron))))))))
@@ -767 +767 @@
-ze))
+ze#ye_pron))
@@ -769 +769 @@
-wold)
+wold#will_v1)
@@ -773 +773 @@
-full)
+full#full_adj)
@@ -775 +775 @@
-lyttyll)))
+lyttyll#little_adj)))
@@ -777 +777 @@
-have)
+have#have_v)
@@ -779 +779 @@
-thowght)
+thowght#think_v2)
@@ -786 +786 @@
-I))
+I#i_pron))
@@ -788 +788 @@
-schold)
+schold#shall_v)
@@ -790 +790 @@
-have)
+have#have_v)
@@ -792 +792 @@
-ben)
+ben#be_v)
@@ -795 +795 @@
-thus)
+thus#thus_adv)
@@ -799 +799 @@
-intreated)
+intreated#entreat_v)
@@ -804 +804 @@
-that)
+that#that_conj)
@@ -809 +809 @@
-wold)
+wold#will_v1)
@@ -812 +812 @@
-God))
+God#NA))
@@ -819 +819 @@
-my)
+my#my_adj)
@@ -821 +821 @@
-Lord)
+Lord#lord_n)
@@ -824 +824 @@
-of)
+of#of_prep)
@@ -827 +827 @@
-Zorke))))
+Zorke#NA))))
@@ -829 +829 @@
-had)
+had#have_v)
@@ -831 +831 @@
-come)
+come#come_v)
@@ -834 +834 @@
-with)
+with#with_prep)
@@ -837 +837 @@
-me)))
+me#me_pron1)))
@@ -840 +840 @@
-yn)
+yn#in_prep)
@@ -843 +843 @@
-the)
+the#the_adj)
@@ -845 +845 @@
-rome)
+rome#room_n1)
@@ -848 +848 @@
-of)
+of#of_prep)
@@ -851 +851 @@
-Northfolke)))))))))))))
+Northfolke#NA)))))))))))))
@@ -859 +859 @@
-for)
+for#for_prep)
@@ -862 +862 @@
-then))
+then#then_adv))
@@ -864 +864 @@
-am)
+am#be_v)
@@ -867 +867 @@
-I))
+I#i_pron))
@@ -870 +870 @@
-sure))
+sure#sure_adj))
@@ -877 +877 @@
-I))
+I#i_pron))
@@ -879 +879 @@
-schuld)
+schuld#shall_v)
@@ -881 +881 @@
-have)
+have#have_v)
@@ -883 +883 @@
-bene)
+bene#be_v)
@@ -885 +885 @@
-left)
+left#leave_v1)
@@ -889 +889 @@
-moch)
+moch#much_adj)
@@ -891 +891 @@
-more))
+more#more_adj))
@@ -896 +896 @@
-at)
+at#at_prep)
@@ -899 +899 @@
-my)
+my#my_adj)
@@ -901 +901 @@
-herti)
+herti#hearty_adj)
@@ -903 +903 @@
-...)))
+...#NA)))
@@ -906 +906 @@
-then)
+then#than_conj)
@@ -917 +917 @@
-I))
+I#i_pron))
@@ -919 +919 @@
-am)
+am#be_v)
@@ -924 +924 @@
-now)))))))
+now#now_adv)))))))
@@ -932 +932 @@
-And)
+And#and_conj1)
@@ -935 +935 @@
-thus))
+thus#thus_adv))
@@ -938 +938 @@
-I))
+I#i_pron))
@@ -940 +940 @@
-byd)
+byd#bid_v)
@@ -943 +943 @@
-your)
+your#your_pron)
@@ -945 +945 @@
-Grace))
+Grace#grace_n))
@@ -948 +948 @@
-fare_weale))
+fare_weale#farewell_int))
@@ -951 +951 @@
-with)
+with#with_prep)
@@ -955 +955 @@
-...)
+...#NA)
@@ -958 +958 @@
-as)
+as#as_adv)
@@ -969 +969 @@
-ever))
+ever#ever_adv))
@@ -971 +971 @@
-had)
+had#have_v)
@@ -974 +974 @@
-Prince))))))
+Prince#prince_n))))))
@@ -979 +979 @@
-and)
+and#and_conj1)
@@ -982 +982 @@
-more)
+more#more_adj)
@@ -984 +984 @@
-hertis)
+hertis#heart_n)
@@ -986 +986 @@
-ease)
+ease#ease_n)
@@ -989 +989 @@
-then)
+then#than_conj)
@@ -1000 +1000 @@
-I))
+I#i_pron))
@@ -1002 +1002 @@
-have)
+have#have_v)
@@ -1005 +1005 @@
-now)))))))))
+now#now_adv)))))))))
@@ -1013 +1013 @@
-From)
+From#from_prep)
@@ -1016 +1016 @@
-Abvile)))
+Abvile#NA)))
@@ -1022 +1022 @@
-the)
+the#the_adj)
@@ -1024 +1024 @@
-xij.=th=)
+xij.=th=#NA)
@@ -1026 +1026 @@
-daye)
+daye#day_n)
@@ -1029 +1029 @@
-of)
+of#of_prep)
@@ -1032 +1032 @@
-October)))
+October#NA)))
@@ -1040 +1040 @@
-...)
+...#NA)
@@ -1042 +1042 @@
-gef)
+gef#give_v)
@@ -1045 +1045 @@
-gredens))
+gredens#greeting_n))
@@ -1048 +1048 @@
-to)
+to#to_prep)
@@ -1051 +1051 @@
-my)
+my#my_adj)
@@ -1053 +1053 @@
-mowder)
+mowder#mother_n1)
@@ -1056 +1056 @@
-Geldeford))))
+Geldeford#NA))))
@@ -1064 +1064 @@
-Be)
+Be#by_prep)
@@ -1067 +1067 @@
-your)
+your#your_pron)
@@ -1069 +1069 @@
-lowyng)
+lowyng#loving_adj)
@@ -1071 +1071 @@
-syster)
+syster#sister_n)
@@ -1074 +1074 @@
-Mary))
+Mary#NA))
@@ -1077 +1077 @@
-Quene)
+Quene#NA)
@@ -1080 +1080 @@
-of)
+of#of_prep)
@@ -1083 +1083 @@
-France)))))
+France#NA)))))
@@ -1094 +1094 @@
-...)
+...#NA)
@@ -1097 +1097 @@
-I))
+I#i_pron))
@@ -1099 +1099 @@
-recomaund)
+recomaund#recommend_v1)
@@ -1102 +1102 @@
-me))
+me#me_pron1))
@@ -1105 +1105 @@
-un)
+un#on_prep)
@@ -1107 +1107 @@
-to)
+to#to_prep)
@@ -1110 +1110 @@
-you)))
+you#you_pron)))
@@ -1114 +1114 @@
-as)
+as#as_adv)
@@ -1116 +1116 @@
-hertly)
+hertly#heartily_adv)
@@ -1119 +1119 @@
-as)
+as#as_adv)
@@ -1130 +1130 @@
-I))
+I#i_pron))
@@ -1132 +1132 @@
-can)
+can#can_v1)
@@ -1139 +1139 @@
-and)
+and#and_conj1)
@@ -1142 +1142 @@
-as)
+as#as_adv)
@@ -1144 +1144 @@
-schoth)
+schoth#NA)
@@ -1147 +1147 @@
-...)
+...#NA)
@@ -1149 +1149 @@
-intreated))
+intreated#entreat_v))
@@ -1152 +1152 @@
-as)
+as#as_adv)
@@ -1164 +1164 @@
-the)
+the#the_adj)
@@ -1166 +1166 @@
-kynge))
+kynge#king_n))
@@ -1169 +1169 @@
-and)
+and#and_conj1)
@@ -1172 +1172 @@
-you))))
+you#you_pron))))
@@ -1176 +1176 @@
-thought)
+thought#think_v2)
@@ -1183,3 +1183,3 @@
-I))
-(MD
-schuld)
+I#i_pron))
+(MD
+schuld#shall_v)
@@ -1187 +1187 @@
-have)
+have#have_v)
@@ -1189 +1189 @@
-ben)
+ben#be_v)
@@ -1199 +1199 @@
-for)
+for#for_prep)
@@ -1201 +1201 @@
-...)
+...#NA)
@@ -1204 +1204 @@
-the)
+the#the_adj)
@@ -1206 +1206 @@
-morn)
+morn#morn_n)
@@ -1209 +1209 @@
-next)
+next#next_adj)
@@ -1212 +1212 @@
-after)
+after#after_adv)
@@ -1215 +1215 @@
-the)
+the#the_adj)
@@ -1217 +1217 @@
-maryage)))))
+maryage#marriage_n)))))
@@ -1222 +1222 @@
-all)
+all#all_adj)
@@ -1224 +1224 @@
-my)
+my#my_adj)
@@ -1226 +1226 @@
-servants)
+servants#servant_n)
@@ -1231 +1231 @@
-both)
+both#both_pron)
@@ -1233 +1233 @@
-men)
+men#man_n1)
@@ -1235 +1235 @@
-and)
+and#and_conj1)
@@ -1237 +1237 @@
-women)))
+women#woman_n)))
@@ -1241 +1241 @@
-were)
+were#be_v)
@@ -1245 +1245 @@
-dyscharged)
+dyscharged#discharge_v)
@@ -1250 +1250 @@
-Insomoch)
+Insomoch#insomuch_adv)
@@ -1253 +1253 @@
-that)
+that#that_conj)
@@ -1257 +1257 @@
-my)
+my#my_adj)
@@ -1259 +1259 @@
-mother)
+mother#mother_n1)
@@ -1262 +1262 @@
-Guldeford))
+Guldeford#NA))
@@ -1266 +1266 @@
-was)
+was#be_v)
@@ -1268 +1268 @@
-also)
+also#also_adv)
@@ -1270 +1270 @@
-dischargyd)
+dischargyd#discharge_v)
@@ -1279 +1279 @@
-whom)))
+whom#whom_pron)))
@@ -1285 +1285 @@
-as)
+as#as_adv)
@@ -1296 +1296 @@
-ze))
+ze#ye_pron))
@@ -1298 +1298 @@
-knowe))))
+knowe#know_v))))
@@ -1302 +1302 @@
-the)
+the#the_adj)
@@ -1304 +1304 @@
-kynge))
+kynge#king_n))
@@ -1307 +1307 @@
-and)
+and#and_conj1)
@@ -1310 +1310 @@
-zou))))
+zou#you_pron))))
@@ -1312 +1312 @@
-willed)
+willed#will_v2)
@@ -1318 +1318 @@
-me))
+me#me_pron1))
@@ -1321 +1321 @@
-in)
+in#in_prep)
@@ -1324 +1324 @@
-eny)
+eny#any_adj)
@@ -1326 +1326 @@
-wyse)))
+wyse#wise_n1)))
@@ -1328 +1328 @@
-to)
+to#to_prep)
@@ -1330 +1330 @@
-be)
+be#be_v)
@@ -1332 +1332 @@
-cowncelled)))))))
+cowncelled#counsel_v)))))))
@@ -1340 +1340 @@
-But)
+But#but_prep)
@@ -1343 +1343 @@
-for)
+for#for_prep)
@@ -1346 +1346 @@
-eny)
+eny#any_adj)
@@ -1348 +1348 @@
-thynge)
+thynge#thing_n1)
@@ -1359 +1359 @@
-I))
+I#i_pron))
@@ -1361 +1361 @@
-myght)
+myght#may_v1)
@@ -1363 +1363 @@
-do)))))
+do#do_v)))))
@@ -1368 +1368 @@
-yn)
+yn#in_prep)
@@ -1371 +1371 @@
-no)
+no#no_adj)
@@ -1373 +1373 @@
-wyse)))
+wyse#wise_n1)))
@@ -1375 +1375 @@
-myght)
+myght#may_v1)
@@ -1378 +1378 @@
-I))
+I#i_pron))
@@ -1380 +1380 @@
-have)
+have#have_v)
@@ -1383 +1383 @@
-any)
+any#any_adj)
@@ -1385 +1385 @@
-graunt)
+graunt#grant_n1)
@@ -1388 +1388 @@
-for)
+for#for_prep)
@@ -1391 +1391 @@
-her)
+her#her_pron2)
@@ -1393 +1393 @@
-abode)
+abode#abode_n1)
@@ -1396 +1396 @@
-here)))))
+here#here_adv)))))
@@ -1402 +1402 @@
-which))
+which#which_adj))
@@ -1408 +1408 @@
-I))
+I#i_pron))
@@ -1410 +1410 @@
-assure)
+assure#assure_v)
@@ -1413 +1413 @@
-you))
+you#you_pron))
@@ -1416 +1416 @@
-my)
+my#my_adj)
@@ -1418 +1418 @@
-lord))
+lord#lord_n))
@@ -1426 +1426 @@
-is)
+is#be_v)
@@ -1429 +1429 @@
-moch)
+moch#much_adj)
@@ -1431 +1431 @@
-to)
+to#to_prep)
@@ -1434 +1434 @@
-my)
+my#my_adj)
@@ -1436 +1436 @@
-discomffort)))
+discomffort#discomfort_n)))
@@ -1441 +1441 @@
-besyd)
+besyd#beside_adv)
@@ -1444 +1444 @@
-meny)
+meny#many_adj)
@@ -1446 +1446 @@
-other)
+other#other_adj)
@@ -1448 +1448 @@
-discomffortis)
+discomffortis#discomfort_n)
@@ -1453 +1453 @@
-that)
+that#that_conj)
@@ -1459 +1459 @@
-ze))
+ze#ye_pron))
@@ -1461 +1461 @@
-wold)
+wold#will_v1)
@@ -1465,3 +1465,3 @@
-full)
-(Q
-lyttyll)))
+full#full_adj)
+(Q
+lyttyll#little_adj)))
@@ -1469 +1469 @@
-have)
+have#have_v)
@@ -1471 +1471 @@
-thought)))))))))
+thought#think_v2)))))))))
@@ -1480 +1480 @@
-I))
+I#i_pron))
@@ -1482 +1482 @@
-have)
+have#have_v)
@@ -1484 +1484 @@
-not)
+not#not_adv)
@@ -1487 +1487 @@
-zet))
+zet#yet_adv))
@@ -1489 +1489 @@
-seen)
+seen#see_v)
@@ -1492 +1492 @@
-yn)
+yn#in_prep)
@@ -1495 +1495 @@
-Fraunce)))
+Fraunce#NA)))
@@ -1498 +1498 @@
-eny)
+eny#any_adj)
@@ -1500 +1500 @@
-lady)
+lady#lady_n)
@@ -1503 +1503 @@
-or)
+or#or_conj2)
@@ -1506 +1506 @@
-jentill)
+jentill#gentle_adj)
@@ -1508 +1508 @@
-woman)))
+woman#woman_n)))
@@ -1512 +1512 @@
-so)
+so#so_adv)
@@ -1514 +1514 @@
-necessary)
+necessary#necessary_adj)
@@ -1517 +1517 @@
-for)
+for#for_prep)
@@ -1520 +1520 @@
-me)))
+me#me_pron1)))
@@ -1523 +1523 @@
-as)
+as#as_adv)
@@ -1534 +1534 @@
-sche))
+sche#she_pron))
@@ -1536 +1536 @@
-ys)))))
+ys#be_v)))))
@@ -1539 +1539 @@
-nor)
+nor#nor_conj1)
@@ -1543 +1543 @@
-zet))
+zet#yet_adv))
@@ -1546 +1546 @@
-so)
+so#so_adv)
@@ -1548 +1548 @@
-mete)
+mete#meet_adj)
@@ -1551 +1551 @@
-to)
+to#to_prep)
@@ -1553 +1553 @@
-do)
+do#do_v)
@@ -1556 +1556 @@
-the)
+the#the_adj)
@@ -1558 +1558 @@
-kynge)
+kynge#king_n)
@@ -1561 +1561 @@
-my)
+my#my_adj)
@@ -1563 +1563 @@
-brother)))
+brother#brother_n)))
@@ -1566 +1566 @@
-service)))
+service#service_n1)))
@@ -1569 +1569 @@
-as)
+as#as_adv)
@@ -1580 +1580 @@
-sche))
+sche#she_pron))
@@ -1582 +1582 @@
-ys)))))))))
+ys#be_v)))))))))
@@ -1590 +1590 @@
-And)
+And#and_conj1)
@@ -1593 +1593 @@
-for)
+for#for_prep)
@@ -1596 +1596 @@
-my)
+my#my_adj)
@@ -1598 +1598 @@
-part)))
+part#part_n)))
@@ -1601 +1601 @@
-my)
+my#my_adj)
@@ -1603 +1603 @@
-lord))
+lord#lord_n))
@@ -1608 +1608 @@
-as)
+as#as_adv)
@@ -1619 +1619 @@
-ze))
+ze#ye_pron))
@@ -1621 +1621 @@
-love)
+love#love_v1)
@@ -1625 +1625 @@
-the)
+the#the_adj)
@@ -1627 +1627 @@
-kynge)
+kynge#king_n)
@@ -1630 +1630 @@
-my)
+my#my_adj)
@@ -1632 +1632 @@
-broder)))
+broder#brother_n)))
@@ -1635 +1635 @@
-and)
+and#and_conj1)
@@ -1638 +1638 @@
-me)))))))
+me#me_pron1)))))))
@@ -1642 +1642 @@
-fynd)
+fynd#find_v)
@@ -1645 +1645 @@
-the)
+the#the_adj)
@@ -1647 +1647 @@
-meanes)
+meanes#mean_n2)
@@ -1652 +1652 @@
-that)
+that#that_conj)
@@ -1658 +1658 @@
-sche))
+sche#she_pron))
@@ -1660 +1660 @@
-may)
+may#may_v1)
@@ -1663 +1663 @@
-yn)
+yn#in_prep)
@@ -1666 +1666 @@
-all)
+all#all_adj)
@@ -1668 +1668 @@
-hast)))
+hast#haste_n)))
@@ -1670 +1670 @@
-com)
+com#come_v)
@@ -1673 +1673 @@
-hither))
+hither#hither_adv))
@@ -1676 +1676 @@
-agayn)))))
+agayn#again_adv)))))
@@ -1684 +1684 @@
-for)
+for#for_prep)
@@ -1687 +1687 @@
-I))
+I#i_pron))
@@ -1689 +1689 @@
-had)
+had#have_v)
@@ -1692 +1692 @@
-as)
+as#as_adv)
@@ -1694 +1694 @@
-lefe)
+lefe#lief_adj)
@@ -1698 +1698 @@
-lose)
+lose#lose_v1)
@@ -1701 +1701 @@
-the)
+the#the_adj)
@@ -1703 +1703 @@
-wynninge)
+wynninge#winning_n1)
@@ -1714 +1714 @@
-I))
+I#i_pron))
@@ -1716 +1716 @@
-schall)
+schall#shall_v)
@@ -1718 +1718 @@
-have)
+have#have_v)
@@ -1721 +1721 @@
-yn)
+yn#in_prep)
@@ -1724 +1724 @@
-France))))))
+France#NA))))))
@@ -1727 +1727 @@
-as)
+as#as_adv)
@@ -1730 +1730 @@
-to)
+to#to_prep)
@@ -1732 +1732 @@
-lose)
+lose#lose_v1)
@@ -1735 +1735 @@
-her)
+her#her_pron2)
@@ -1737 +1737 @@
-counsell)
+counsell#counsel_n)
@@ -1742 +1742 @@
-when)
+when#when_adv)
@@ -1749 +1749 @@
-I))
+I#i_pron))
@@ -1751 +1751 @@
-schall)
+schall#shall_v)
@@ -1753 +1753 @@
-lacke)
+lacke#lack_v1)
@@ -1756 +1756 @@
-it)))))
+it#it_pron)))))
@@ -1762 +1762 @@
-which))
+which#which_adj))
@@ -1769 +1769 @@
-is)
+is#be_v)
@@ -1771 +1771 @@
-not)
+not#not_adv)
@@ -1774 +1774 @@
-like)
+like#like_adj)
@@ -1778 +1778 @@
-long))
+long#long_adj1))
@@ -1780 +1780 @@
-to)
+to#to_prep)
@@ -1782 +1782 @@
-be)
+be#be_v)
@@ -1784 +1784 @@
-required)))
+required#require_v)))
@@ -1789 +1789 @@
-as)
+as#as_adv)
@@ -1798 +1798 @@
-I))
+I#i_pron))
@@ -1800 +1800 @@
-am)
+am#be_v)
@@ -1803 +1803 @@
-sure)
+sure#sure_adj)
@@ -1812 +1812 @@
-the)
+the#the_adj)
@@ -1814 +1814 @@
-nobill)
+nobill#noble_adj)
@@ -1816 +1816 @@
-men)
+men#man_n1)
@@ -1819 +1819 @@
-and)
+and#and_conj1)
@@ -1822 +1822 @@
-jentillmen))))
+jentillmen#gentleman_n))))
@@ -1824 +1824 @@
-can)
+can#can_v1)
@@ -1826 +1826 @@
-shewe)
+shewe#show_v)
@@ -1829 +1829 @@
-you))
+you#you_pron))
@@ -1832 +1832 @@
-more)
+more#more_adj)
@@ -1835 +1835 @@
-then)
+then#than_conj)
@@ -1845 +1845 @@
-becometh)
+becometh#become_v)
@@ -1848 +1848 @@
-me))
+me#me_pron1))
@@ -1851 +1851 @@
-to)
+to#to_prep)
@@ -1853 +1853 @@
-wryte)
+wryte#write_v)
@@ -1856 +1856 @@
-yn)
+yn#in_prep)
@@ -1859 +1859 @@
-this)
+this#this_pron)
@@ -1861 +1861 @@
-matter))))))))))))))))))
+matter#matter_n1))))))))))))))))))
@@ -1871 +1871 @@
-I))
+I#i_pron))
@@ -1873 +1873 @@
-pray)
+pray#pray_v)
@@ -1876 +1876 @@
-you)))
+you#you_pron)))
@@ -1879 +1879 @@
-my)
+my#my_adj)
@@ -1881 +1881 @@
-Lord))
+Lord#lord_n))
@@ -1883 +1883 @@
-gyf)
+gyf#give_v)
@@ -1886 +1886 @@
-credens))
+credens#credence_n))
@@ -1889 +1889 @@
-forther))
+forther#further_adj))
@@ -1892 +1892 @@
-to)
+to#to_prep)
@@ -1895 +1895 @@
-my)
+my#my_adj)
@@ -1897 +1897 @@
-moder)
+moder#mother_n1)
@@ -1900 +1900 @@
-Guldeford))))
+Guldeford#NA))))
@@ -1903 +1903 @@
-yn)
+yn#in_prep)
@@ -1906 +1906 @@
-every)
+every#every_adj)
@@ -1908 +1908 @@
-thyng)
+thyng#thing_n1)
@@ -1911 +1911 @@
-concernynge)
+concernynge#concerning_prep)
@@ -1914 +1914 @@
-thys)
+thys#this_pron)
@@ -1916 +1916 @@
-matter)))))
+matter#matter_n1)))))
@@ -1924 +1924 @@
-And)
+And#and_conj1)
@@ -1927 +1927 @@
-albehit)
+albehit#albeit_conj)
@@ -1934 +1934 @@
-my)
+my#my_adj)
@@ -1936 +1936 @@
-Lord)
+Lord#lord_n)
@@ -1939 +1939 @@
-of)
+of#of_prep)
@@ -1942 +1942 @@
-Northfollke))))
+Northfollke#NA))))
@@ -1944 +1944 @@
-hath)
+hath#have_v)
@@ -1946 +1946 @@
-nethyr)
+nethyr#neither_adv)
@@ -1948 +1948 @@
-deled)
+deled#devil_v)
@@ -1951 +1951 @@
-best))
+best#best_adj))
@@ -1954 +1954 @@
-with)
+with#with_prep)
@@ -1957 +1957 @@
-me)))
+me#me_pron1)))
@@ -1960 +1960 @@
-nor)
+nor#nor_conj1)
@@ -1963 +1963 @@
-zet))
+zet#yet_adv))
@@ -1966 +1966 @@
-with)
+with#with_prep)
@@ -1969 +1969 @@
-her))))
+her#her_pron1))))
@@ -1972 +1972 @@
-at)
+at#at_prep)
@@ -1975 +1975 @@
-thys)
+thys#this_pron)
@@ -1977 +1977 @@
-tyme))))))
+tyme#time_n))))))
@@ -1982 +1982 @@
-zet))
+zet#yet_adv))
@@ -1985 +1985 @@
-I))
+I#i_pron))
@@ -1987 +1987 @@
-pray)
+pray#pray_v)
@@ -1990 +1990 @@
-you))
+you#you_pron))
@@ -1994 +1994 @@
-allwayes))
+allwayes#always_adv))
@@ -1996 +1996 @@
-to)
+to#to_prep)
@@ -1998 +1998 @@
-be)
+be#be_v)
@@ -2001 +2001 @@
-good)
+good#good_adj)
@@ -2003 +2003 @@
-lord)
+lord#lord_n)
@@ -2006 +2006 @@
-un)
+un#on_prep)
@@ -2008 +2008 @@
-to)
+to#to_prep)
@@ -2011 +2011 @@
-her)))))
+her#her_pron1)))))
@@ -2019 +2019 @@
-And)
+And#and_conj1)
@@ -2023 +2023 @@
-wold)
+wold#will_v1)
@@ -2026 +2026 @@
-to)
+to#to_prep)
@@ -2029 +2029 @@
-God)))
+God#NA)))
@@ -2036 +2036 @@
-my)
+my#my_adj)
@@ -2038 +2038 @@
-...))
+...#NA))
@@ -2040 +2040 @@
-had)
+had#have_v)
@@ -2042 +2042 @@
-ben)
+ben#be_v)
@@ -2045 +2045 @@
-so)
+so#so_adv)
@@ -2047 +2047 @@
-good)
+good#good_adj)
@@ -2050 +2050 @@
-to)
+to#to_prep)
@@ -2052 +2052 @@
-have)
+have#have_v)
@@ -2054 +2054 @@
-had)
+had#have_v)
@@ -2057 +2057 @@
-zou))
+zou#you_pron))
@@ -2060 +2060 @@
-with)
+with#with_prep)
@@ -2063 +2063 @@
-me)))
+me#me_pron1)))
@@ -2066 +2066 @@
-hither))
+hither#hither_adv))
@@ -2069 +2069 @@
-when)
+when#when_adv)
@@ -2076 +2076 @@
-I))
+I#i_pron))
@@ -2078 +2078 @@
-had)
+had#have_v)
@@ -2083 +2083 @@
-my)
+my#my_adj)
@@ -2087 +2087 @@
-lord)
+lord#lord_n)
@@ -2092 +2092 @@
-of)
+of#of_prep)
@@ -2095 +2095 @@
-Northfolke)))))))))))
+Northfolke#NA)))))))))))
@@ -2103 +2103 @@
-And)
+And#and_conj1)
@@ -2106 +2106 @@
-thus))
+thus#thus_adv))
@@ -2108 +2108 @@
-fare)
+fare#fare_v1)
@@ -2111 +2111 @@
-ze))
+ze#ye_pron))
@@ -2114 +2114 @@
-weale))
+weale#well_adv))
@@ -2119 +2119 @@
-My)
+My#my_adj)
@@ -2121 +2121 @@
-Lord))
+Lord#lord_n))
@@ -2129 +2129 @@
-Wrytten)
+Wrytten#write_v)
@@ -2134 +2134 @@
-from)
+from#from_prep)
@@ -2139 +2139 @@
-Abbeville)))
+Abbeville#NA)))
@@ -2144 +2144 @@
-the)
+the#the_adj)
@@ -2146 +2146 @@
-xij.=the=)
+xij.=the=#NA)
@@ -2148 +2148 @@
-daye)
+daye#day_n)
@@ -2151 +2151 @@
-of)
+of#of_prep)
@@ -2154 +2154 @@
-Octobr))))
+Octobr#NA))))
@@ -2163 +2163 @@
-My)
+My#my_adj)
@@ -2165 +2165 @@
-Lord))
+Lord#lord_n))
@@ -2169 +2169 @@
-I))
+I#i_pron))
@@ -2171 +2171 @@
-pray)
+pray#pray_v)
@@ -2174 +2174 @@
-you)))
+you#you_pron)))
@@ -2176 +2176 @@
-gyve)
+gyve#give_v)
@@ -2179 +2179 @@
-credens))
+credens#credence_n))
@@ -2182 +2182 @@
-to)
+to#to_prep)
@@ -2185 +2185 @@
-my)
+my#my_adj)
@@ -2189 +2189 @@
-ord)))
+ord#NA)))
@@ -2193 +2193 @@
-yn)
+yn#in_prep)
@@ -2196 +2196 @@
-my)
+my#my_adj)
@@ -2198 +2198 @@
-sorows)))
+sorows#sorrow_n)))
@@ -2201 +2201 @@
-she))
+she#she_pron))
@@ -2203 +2203 @@
-have)
+have#have_v)
@@ -2205 +2205 @@
-delyve))
+delyve#deliver_v1))
@@ -2213 +2213 @@
-Yowr)
+Yowr#your_pron)
@@ -2215 +2215 @@
-on)
+on#own_adj)
@@ -2218 +2218 @@
-while)
+while#while_adv)
@@ -2225 +2225 @@
-I))
+I#i_pron))
@@ -2227 +2227 @@
-lefe))))
+lefe#live_v1))))
@@ -2230 +2230 @@
-Mary)))
+Mary#NA)))
@@ -2236 +2236 @@
-To)
+To#to_prep)
@@ -2239 +2239 @@
-my)
+my#my_adj)
@@ -2241 +2241 @@
-lovynge)
+lovynge#loving_adj)
@@ -2243 +2243 @@
-frend)
+frend#friend_n)
@@ -2246 +2246 @@
-Th'@)
+Th'@#the_adj)
@@ -2248 +2248 @@
-@archebischop)
+@archebischop#archbishop_n)
@@ -2251 +2251 @@
-of)
+of#of_prep)
@@ -2254 +2254 @@
-Zorke)))))
+Zorke#NA)))))
@@ -2272 +2272 @@
-My)
+My#my_adj)
@@ -2274 +2274 @@
-nanne)
+nanne#nown_adj)
@@ -2276 +2276 @@
-good)
+good#good_adj)
@@ -2278 +2278 @@
-Lord))
+Lord#lord_n))
@@ -2283 +2283 @@
-I))
+I#i_pron))
@@ -2285 +2285 @@
-recomend)
+recomend#recommend_v1)
@@ -2288 +2288 @@
-me))
+me#me_pron1))
@@ -2291 +2291 @@
-to)
+to#to_prep)
@@ -2294 +2294 @@
-yow))))
+yow#you_pron))))
@@ -2300 +2300 @@
-and)
+and#and_conj1)
@@ -2303 +2303 @@
-thankyng)
+thankyng#thank_v)
@@ -2306 +2306 @@
-yow))
+yow#you_pron))
@@ -2309 +2309 @@
-for)
+for#for_prep)
@@ -2312 +2312 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2315 +2315 @@
-kynde)
+kynde#kind_adj)
@@ -2317 +2317 @@
-and)
+and#and_conj1)
@@ -2319 +2319 @@
-lovyng))
+lovyng#loving_adj))
@@ -2321 +2321 @@
-Letter))))
+Letter#letter_n1))))
@@ -2326 +2326 @@
-dyssyryng)
+dyssyryng#desire_v)
@@ -2329 +2329 @@
-yow))
+yow#you_pron))
@@ -2332 +2332 @@
-of)
+of#of_prep)
@@ -2335 +2335 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2337 +2337 @@
-good)
+good#good_adj)
@@ -2339 +2339 @@
-contenevans)
+contenevans#continuance_n)
@@ -2342 +2342 @@
-and)
+and#and_conj1)
@@ -2345 +2345 @@
-good)
+good#good_adj)
@@ -2347 +2347 @@
-lessones)))
+lessones#lesson_n)))
@@ -2352 +2352 @@
-that)
+that#that_conj)
@@ -2358 +2358 @@
-yow))
+yow#you_pron))
@@ -2360 +2360 @@
-hathe)
+hathe#have_v)
@@ -2362 +2362 @@
-gyffen)
+gyffen#give_v)
@@ -2365 +2365 @@
-to)
+to#to_prep)
@@ -2368 +2368 @@
-me))))))))
+me#me_pron1))))))))
@@ -2377 +2377 @@
-my)
+my#my_adj)
@@ -2379 +2379 @@
-lord))
+lord#lord_n))
@@ -2384 +2384 @@
-I))
+I#i_pron))
@@ -2386 +2386 @@
-pray)
+pray#pray_v)
@@ -2389 +2389 @@
-yow))
+yow#you_pron))
@@ -2392 +2392 @@
-as)
+as#as_adv)
@@ -2403 +2403 @@
-my)
+my#my_adj)
@@ -2405 +2405 @@
-trust))
+trust#trust_n))
@@ -2407 +2407 @@
-ys)
+ys#be_v)
@@ -2410 +2410 @@
-in)
+in#in_prep)
@@ -2413 +2413 @@
-yow))))))
+yow#you_pron))))))
@@ -2418 +2418 @@
-for)
+for#for_prep)
@@ -2420 +2420 @@
-to)
+to#to_prep)
@@ -2422 +2422 @@
-remember)
+remember#remember_v)
@@ -2425 +2425 @@
-me))
+me#me_pron1))
@@ -2428 +2428 @@
-to)
+to#to_prep)
@@ -2431 +2431 @@
-the)
+the#the_adj)
@@ -2433 +2433 @@
-Kyng)
+Kyng#king_n)
@@ -2436 +2436 @@
-my)
+my#my_adj)
@@ -2438 +2438 @@
-brother))))
+brother#brother_n))))
@@ -2443 +2443 @@
-for)
+for#for_prep)
@@ -2446 +2446 @@
-sowche)
+sowche#such_adj)
@@ -2448 +2448 @@
-causses)
+causses#cause_n)
@@ -2451 +2451 @@
-and)
+and#and_conj1)
@@ -2454 +2454 @@
-bessynes)))
+bessynes#business_n)))
@@ -2457 +2457 @@
-as)
+as#as_adv)
@@ -2466 +2466 @@
-I))
+I#i_pron))
@@ -2468 +2468 @@
-have)
+have#have_v)
@@ -2473 +2473 @@
-for)
+for#for_prep)
@@ -2475 +2475 @@
-to)
+to#to_prep)
@@ -2477 +2477 @@
-do))))))))
+do#do_v))))))))
@@ -2485 +2485 @@
-for)
+for#for_prep)
@@ -2488 +2488 @@
-as)
+as#as_adv)
@@ -2491 +2491 @@
-now)))
+now#now_adv)))
@@ -2494 +2494 @@
-I))
+I#i_pron))
@@ -2496 +2496 @@
-have)
+have#have_v)
@@ -2499 +2499 @@
-no)
+no#no_adj)
@@ -2501 +2501 @@
-nother)
+nother#other_adj)
@@ -2507 +2507 @@
-to)
+to#to_prep)
@@ -2509 +2509 @@
-put)
+put#put_v)
@@ -2512 +2512 @@
-my)
+my#my_adj)
@@ -2514 +2514 @@
-trust))
+trust#trust_n))
@@ -2517 +2517 @@
-in)
+in#in_prep)
@@ -2522 +2522 @@
-but)
+but#but_prep)
@@ -2526 +2526 @@
-the)
+the#the_adj)
@@ -2528 +2528 @@
-Kyng)
+Kyng#king_n)
@@ -2531 +2531 @@
-my)
+my#my_adj)
@@ -2533 +2533 @@
-brother)))
+brother#brother_n)))
@@ -2536 +2536 @@
-and)
+and#and_conj1)
@@ -2539 +2539 @@
-yow))))))
+yow#you_pron))))))
@@ -2550 +2550 @@
-And)
+And#and_conj1)
@@ -2553 +2553 @@
-as)
+as#as_adv)
@@ -2564 +2564 @@
-yt))
+yt#it_pron))
@@ -2566 +2566 @@
-shall)
+shall#shall_v)
@@ -2568 +2568 @@
-ples)
+ples#please_v)
@@ -2572 +2572 @@
-the)
+the#the_adj)
@@ -2574 +2574 @@
-Kyng)
+Kyng#king_n)
@@ -2577 +2577 @@
-my)
+my#my_adj)
@@ -2579 +2579 @@
-brother)))
+brother#brother_n)))
@@ -2582 +2582 @@
-and)
+and#and_conj1)
@@ -2585 +2585 @@
-hys)
+hys#his_adj)
@@ -2587 +2587 @@
-Counsell)))))))
+Counsell#counsel_n)))))))
@@ -2590 +2590 @@
-I))
+I#i_pron))
@@ -2592 +2592 @@
-wol@)
+wol@#will_v1)
@@ -2594 +2594 @@
-@be)
+@be#be_v)
@@ -2596 +2596 @@
-hordered)
+hordered#order_v)
@@ -2604 +2604 @@
-And)
+And#and_conj1)
@@ -2607 +2607 @@
-so))
+so#so_adv))
@@ -2610 +2610 @@
-I))
+I#i_pron))
@@ -2612 +2612 @@
-pray)
+pray#pray_v)
@@ -2615 +2615 @@
-yow))
+yow#you_pron))
@@ -2620 +2620 @@
-my)
+my#my_adj)
@@ -2622 +2622 @@
-lord))
+lord#lord_n))
@@ -2627 +2627 @@
-to)
+to#to_prep)
@@ -2629 +2629 @@
-show)
+show#show_v)
@@ -2632 +2632 @@
-hys)
+hys#his_adj)
@@ -2634 +2634 @@
-Grace))
+Grace#grace_n))
@@ -2639 +2639 @@
-seyng)
+seyng#see_v)
@@ -2642 +2642 @@
-that)
+that#that_conj)
@@ -2646 +2646 @@
-the)
+the#the_adj)
@@ -2648 +2648 @@
-Kyng)
+Kyng#king_n)
@@ -2651 +2651 @@
-my)
+my#my_adj)
@@ -2653 +2653 @@
-howsbande))
+howsbande#husband_n))
@@ -2657 +2657 @@
-ys)
+ys#be_v)
@@ -2659 +2659 @@
-departed)
+departed#depart_v)
@@ -2662 +2662 @@
-to)
+to#to_prep)
@@ -2665 +2665 @@
-God)))
+God#NA)))
@@ -2671 +2671 @@
-of)
+of#of_prep)
@@ -2674 +2674 @@
-whos)
+whos#whose_pron)
@@ -2676 +2676 @@
-sole)))
+sole#soul_n)))
@@ -2684 +2684 @@
-God))
+God#NA))
@@ -2686 +2686 @@
-pardon)))))))
+pardon#pardon_v)))))))
@@ -2694 +2694 @@
-And)
+And#and_conj1)
@@ -2697 +2697 @@
-wher)
+wher#where_adv)
@@ -2699 +2699 @@
-as)
+as#as_adv)
@@ -2706 +2706 @@
-yow))
+yow#you_pron))
@@ -2708 +2708 @@
-a_vyse)
+a_vyse#advise_v)
@@ -2711 +2711 @@
-me))
+me#me_pron1))
@@ -2714 +2714 @@
-that)
+that#that_conj)
@@ -2718 +2718 @@
-I))
+I#i_pron))
@@ -2720 +2720 @@
-shulde)
+shulde#shall_v)
@@ -2722 +2722 @@
-macke)
+macke#make_v1)
@@ -2725 +2725 @@
-no)
+no#no_adj)
@@ -2727 +2727 @@
-promas)))))))
+promas#promise_n)))))))
@@ -2732 +2732 @@
-My)
+My#my_adj)
@@ -2734 +2734 @@
-Lord))
+Lord#lord_n))
@@ -2739 +2739 @@
-I))
+I#i_pron))
@@ -2741 +2741 @@
-trust)
+trust#trust_v)
@@ -2749 +2749 @@
-the)
+the#the_adj)
@@ -2751 +2751 @@
-Kyng))
+Kyng#king_n))
@@ -2754 +2754 @@
-my)
+my#my_adj)
@@ -2756 +2756 @@
-brother))
+brother#brother_n))
@@ -2759 +2759 @@
-and)
+and#and_conj1)
@@ -2762 +2762 @@
-yow))))
+yow#you_pron))))
@@ -2764 +2764 @@
-wole)
+wole#will_v1)
@@ -2766 +2766 @@
-not)
+not#not_adv)
@@ -2768 +2768 @@
-reken)
+reken#reckon_v)
@@ -2771 +2771 @@
-in)
+in#in_prep)
@@ -2774 +2774 @@
-me)))
+me#me_pron1)))
@@ -2777 +2777 @@
-sovche)
+sovche#such_adj)
@@ -2779 +2779 @@
-chyldhode))))
+chyldhode#childhood_n))))
@@ -2788 +2788 @@
-I))
+I#i_pron))
@@ -2790 +2790 @@
-trust)
+trust#trust_v)
@@ -2797 +2797 @@
-I))
+I#i_pron))
@@ -2799 +2799 @@
-have)
+have#have_v)
@@ -2802 +2802 @@
-so)
+so#so_adv)
@@ -2806 +2806 @@
-horderd)
+horderd#order_v)
@@ -2809 +2809 @@
-my)
+my#my_adj)
@@ -2811 +2811 @@
-selffe))
+selffe#self_pron))
@@ -2814 +2814 @@
-so)
+so#so_adv)
@@ -2819 +2819 @@
-sens)
+sens#since_adv)
@@ -2822 +2822 @@
-that)
+that#that_conj)
@@ -2826 +2826 @@
-I))
+I#i_pron))
@@ -2828 +2828 @@
-came)
+came#come_v)
@@ -2831 +2831 @@
-hether)))))
+hether#hither_adv)))))
@@ -2836 +2836 @@
-that)
+that#that_conj)
@@ -2840 +2840 @@
-I))
+I#i_pron))
@@ -2842 +2842 @@
-trust)
+trust#trust_v)
@@ -2849 +2849 @@
-yt))
+yt#it_pron))
@@ -2851 +2851 @@
-hathe)
+hathe#have_v)
@@ -2853 +2853 @@
-ben)
+ben#be_v)
@@ -2856 +2856 @@
-to)
+to#to_prep)
@@ -2859 +2859 @@
-the)
+the#the_adj)
@@ -2861 +2861 @@
-honar)
+honar#honour_n)
@@ -2864 +2864 @@
-of)
+of#of_prep)
@@ -2868 +2868 @@
-the)
+the#the_adj)
@@ -2870 +2870 @@
-Kyng)
+Kyng#king_n)
@@ -2873 +2873 @@
-my)
+my#my_adj)
@@ -2875 +2875 @@
-brother)))
+brother#brother_n)))
@@ -2878 +2878 @@
-and)
+and#and_conj1)
@@ -2881 +2881 @@
-me)))))))
+me#me_pron1)))))))
@@ -2886 +2886 @@
-sense)
+sense#since_adv)
@@ -2893 +2893 @@
-I))
+I#i_pron))
@@ -2895 +2895 @@
-come)
+come#come_v)
@@ -2898 +2898 @@
-hether)))))))))))
+hether#hither_adv)))))))))))
@@ -2906 +2906 @@
-and)
+and#and_conj1)
@@ -2909 +2909 @@
-so))
+so#so_adv))
@@ -2912 +2912 @@
-I))
+I#i_pron))
@@ -2914 +2914 @@
-trust)
+trust#trust_v)
@@ -2919 +2919 @@
-to)
+to#to_prep)
@@ -2921 +2921 @@
-contenew))
+contenew#continue_v))
@@ -2930 +2930 @@
-Yff)
+Yff#if_conj)
@@ -2937 +2937 @@
-ther))
+ther#there_adv))
@@ -2939 +2939 @@
-be)
+be#be_v)
@@ -2942 +2942 @@
-any)
+any#any_adj)
@@ -2944 +2944 @@
-thynge)
+thynge#thing_n1)
@@ -2949 +2949 @@
-that)
+that#that_conj)
@@ -2955 +2955 @@
-I))
+I#i_pron))
@@ -2957 +2957 @@
-may)
+may#may_v1)
@@ -2959 +2959 @@
-do)
+do#do_v)
@@ -2962 +2962 @@
-for)
+for#for_prep)
@@ -2965 +2965 @@
-yow)))))))))
+yow#you_pron)))))))))
@@ -2968 +2968 @@
-I))
+I#i_pron))
@@ -2970 +2970 @@
-wold)
+wold#will_v1)
@@ -2972 +2972 @@
-be)
+be#be_v)
@@ -2975 +2975 @@
-glade)
+glade#glad_adj)
@@ -2978 +2978 @@
-for)
+for#for_prep)
@@ -2980 +2980 @@
-to)
+to#to_prep)
@@ -2982 +2982 @@
-do)
+do#do_v)
@@ -2985 +2985 @@
-yt))
+yt#it_pron))
@@ -2988 +2988 @@
-in)
+in#in_prep)
@@ -2991 +2991 @@
-thys)
+thys#this_pron)
@@ -2993 +2993 @@
-partes)))))
+partes#part_n)))))
@@ -3002 +3002 @@
-I))
+I#i_pron))
@@ -3004 +3004 @@
-shal@)
+shal@#shall_v)
@@ -3006 +3006 @@
-@be)
+@be#be_v)
@@ -3009 +3009 @@
-glade)
+glade#glad_adj)
@@ -3012 +3012 @@
-to)
+to#to_prep)
@@ -3014 +3014 @@
-do)
+do#do_v)
@@ -3017 +3017 @@
-yt))
+yt#it_pron))
@@ -3020 +3020 @@
-for)
+for#for_prep)
@@ -3023 +3023 @@
-yow)))))
+yow#you_pron)))))
@@ -3033 +3033 @@
-No)
+No#no_adj)
@@ -3035 +3035 @@
-more)))
+more#more_adj)))
@@ -3038 +3038 @@
-to)
+to#to_prep)
@@ -3041 +3041 @@
-you)))
+you#you_pron)))
@@ -3044 +3044 @@
-at)
+at#at_prep)
@@ -3047 +3047 @@
-thys)
+thys#this_pron)
@@ -3049 +3049 @@
-tyme))))
+tyme#time_n))))
@@ -3055 +3055 @@
-but)
+but#but_prep)
@@ -3058 +3058 @@
-Jh~s))
+Jh~s#NA))
@@ -3060 +3060 @@
-preserve)
+preserve#preserve_v)
@@ -3063 +3063 @@
-yow))
+yow#you_pron))
@@ -3071 +3071 @@
-Wretten)
+Wretten#write_v)
@@ -3074 +3074 @@
-at)
+at#at_prep)
@@ -3077 +3077 @@
-Pares)))
+Pares#NA)))
@@ -3080 +3080 @@
-the)
+the#the_adj)
@@ -3082 +3082 @@
-x.)
+x.#ten_adj)
@@ -3084 +3084 @@
-day)
+day#day_n)
@@ -3087 +3087 @@
-of)
+of#of_prep)
@@ -3090 +3090 @@
-January)))
+January#NA)))
@@ -3092 +3092 @@
-1515))
+1515#NA))
@@ -3100 +3100 @@
-By)
+By#by_prep)
@@ -3103 +3103 @@
-yowr)
+yowr#your_pron)
@@ -3105 +3105 @@
-lowyng)
+lowyng#loving_adj)
@@ -3107 +3107 @@
-frende)
+frende#friend_n)
@@ -3110 +3110 @@
-Mary))
+Mary#NA))
@@ -3113 +3113 @@
-Quene)
+Quene#queen_n)
@@ -3116 +3116 @@
-of)
+of#of_prep)
@@ -3119 +3119 @@
-France)))))
+France#NA)))))
@@ -3127 +3127 @@
-To)
+To#to_prep)
@@ -3130 +3130 @@
-my)
+my#my_adj)
@@ -3132 +3132 @@
-Lord)
+Lord#lord_n)
@@ -3135 +3135 @@
-of)
+of#of_prep)
@@ -3138 +3138 @@
-Yorke))))
+Yorke#NA))))
