--- pettit-1625-e2-p1.psd.vert	2025-09-10 13:51:11
+++ pettit-1625-e2-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-Cosin)
+Cosin#cousin_n)
@@ -17 +17 @@
-Oxenden)))
+Oxenden#NA)))
@@ -22 +22 @@
-With)
+With#with_prep)
@@ -25 +25 @@
-the)
+the#the_adj)
@@ -27 +27 @@
-remembraunce)
+remembraunce#remembrance_n)
@@ -30 +30 @@
-of)
+of#of_prep)
@@ -33 +33 @@
-my)
+my#my_adj)
@@ -35 +35 @@
-love)))))
+love#love_n1)))))
@@ -38 +38 @@
-I))
+I#i_pron))
@@ -41 +41 @@
-hartily))
+hartily#heartily_adv))
@@ -43 +43 @@
-salute)
+salute#salute_v)
@@ -46 +46 @@
-you))
+you#you_pron))
@@ -55 +55 @@
-I))
+I#i_pron))
@@ -57 +57 @@
-have)
+have#have_v)
@@ -60 +60 @@
-nowe))
+nowe#now_adv))
@@ -62 +62 @@
-sent)
+sent#send_v1)
@@ -65 +65 @@
-my)
+my#my_adj)
@@ -67 +67 @@
-Daughter)
+Daughter#daughter_n)
@@ -70 +70 @@
-Pettit)))
+Pettit#NA)))
@@ -74 +74 @@
-a)
+a#a_adj2)
@@ -76 +76 @@
-fewe))
+fewe#few_adj))
@@ -78 +78 @@
-smale)
+smale#small_adj)
@@ -80 +80 @@
-Lopsteres)
+Lopsteres#lobster_n1)
@@ -83 +83 @@
-taken)
+taken#take_v)
@@ -86 +86 @@
-yesterday))))
+yesterday#yesterday_adv))))
@@ -94 +94 @@
-and)
+and#and_conj1)
@@ -97 +97 @@
-my)
+my#my_adj)
@@ -99 +99 @@
-Desire))
+Desire#desire_n))
@@ -101 +101 @@
-was)
+was#be_v)
@@ -104 +104 @@
-to)
+to#to_prep)
@@ -106 +106 @@
-have)
+have#have_v)
@@ -108 +108 @@
-had)
+had#have_v)
@@ -111 +111 @@
-more)
+more#more_adj)
@@ -113 +113 @@
-store)
+store#store_n)
@@ -116 +116 @@
-of)
+of#of_prep)
@@ -120 +120 @@
-this)
+this#this_pron)
@@ -122 +122 @@
-morninges))
+morninges#morning_n))
@@ -124 +124 @@
-takeinge))))
+takeinge#taking_n))))
@@ -127 +127 @@
-to)
+to#to_prep)
@@ -129 +129 @@
-have)
+have#have_v)
@@ -131 +131 @@
-sent)
+sent#send_v1)
@@ -134 +134 @@
-them))
+them#them_pron))
@@ -137 +137 @@
-alive)))
+alive#alive_adj)))
@@ -142 +142 @@
-that)
+that#that_conj)
@@ -146 +146 @@
-they))
+they#they_pron))
@@ -148 +148 @@
-might)
+might#may_v1)
@@ -150 +150 @@
-have)
+have#have_v)
@@ -154 +154 @@
-the)
+the#the_adj)
@@ -156 +156 @@
-longer)))
+longer#long_adj1)))
@@ -158 +158 @@
-beene)
+beene#be_v)
@@ -160 +160 @@
-kept)
+kept#keep_v)
@@ -163 +163 @@
-good)))))
+good#good_adj)))))
@@ -171 +171 @@
-but)
+but#but_prep)
@@ -174 +174 @@
-this)
+this#this_pron)
@@ -176 +176 @@
-northerly)
+northerly#northerly_adj)
@@ -178 +178 @@
-winde))
+winde#wind_n1))
@@ -180 +180 @@
-hath)
+hath#have_v)
@@ -182 +182 @@
-frustrateed)
+frustrateed#frustrate_v)
@@ -185 +185 @@
-my)
+my#my_adj)
@@ -187 +187 @@
-expectation)
+expectation#expectation_n)
@@ -190 +190 @@
-therein)))
+therein#therein_adv)))
@@ -199 +199 @@
-My)
+My#my_adj)
@@ -201 +201 @@
-Desire))
+Desire#desire_n))
@@ -203 +203 @@
-is)
+is#be_v)
@@ -206 +206 @@
-that)
+that#that_conj)
@@ -210 +210 @@
-these))
+these#these_pron))
@@ -212 +212 @@
-may)
+may#may_v1)
@@ -214 +214 @@
-bee)
+bee#be_v)
@@ -216 +216 @@
-accepted)
+accepted#accept_v)
@@ -219 +219 @@
-as)
+as#as_adv)
@@ -222 +222 @@
-a)
+a#a_adj2)
@@ -224 +224 @@
-Signe)
+Signe#sign_n)
@@ -227 +227 @@
-of)
+of#of_prep)
@@ -230 +230 @@
-my)
+my#my_adj)
@@ -232 +232 @@
-Love)))))))
+Love#love_n1)))))))
@@ -240 +240 @@
-and)
+and#and_conj1)
@@ -244 +244 @@
-so))
+so#so_adv))
@@ -246 +246 @@
-prayeing)
+prayeing#pray_v)
@@ -249 +249 @@
-you))
+you#you_pron))
@@ -252 +252 @@
-to)
+to#to_prep)
@@ -254 +254 @@
-remember)
+remember#remember_v)
@@ -257 +257 @@
-us))
+us#us_pron))
@@ -260 +260 @@
-to)
+to#to_prep)
@@ -264 +264 @@
-my)
+my#my_adj)
@@ -266 +266 @@
-Ladye)
+Ladye#lady_n)
@@ -271 +271 @@
-my)
+my#my_adj)
@@ -273 +273 @@
-Cozine))
+Cozine#cousin_n))
@@ -276 +276 @@
-your)
+your#your_pron)
@@ -278 +278 @@
-wife)))
+wife#wife_n)))
@@ -284 +284 @@
-my)
+my#my_adj)
@@ -286 +286 @@
-Daughter)))
+Daughter#daughter_n)))
@@ -289 +289 @@
-and)
+and#and_conj1)
@@ -292 +292 @@
-the)
+the#the_adj)
@@ -294 +294 @@
-rest)
+rest#rest_n2)
@@ -297 +297 @@
-of)
+of#of_prep)
@@ -300 +300 @@
-our)
+our#our_pron)
@@ -302 +302 @@
-freinds)))))))))
+freinds#friend_n)))))))))
@@ -307 +307 @@
-I))
+I#i_pron))
@@ -309 +309 @@
-commende)
+commende#commend_v)
@@ -312 +312 @@
-you))
+you#you_pron))
@@ -315 +315 @@
-to)
+to#to_prep)
@@ -318 +318 @@
-God))))
+God#NA))))
@@ -324 +324 @@
-and)
+and#and_conj1)
@@ -328 +328 @@
-rest)
+rest#rest_v2)
@@ -333 +333 @@
-Your)
+Your#your_pron)
@@ -335 +335 @@
-loveing)
+loveing#loving_adj)
@@ -337 +337 @@
-kinsman)
+kinsman#kinsman_n)
@@ -340 +340 @@
-Val.)
+Val.#NA)
@@ -342 +342 @@
-Pettit))))
+Pettit#NA))))
@@ -348 +348 @@
-Daundelion))
+Daundelion#NA))
@@ -354 +354 @@
-this)
+this#this_pron)
@@ -356 +356 @@
-25th)
+25th#NA)
@@ -359 +359 @@
-of)
+of#of_prep)
@@ -362 +362 @@
-Aprill)))
+Aprill#NA)))
@@ -364 +364 @@
-1625))
+1625#NA))
@@ -371 +371 @@
-At)
+At#at_prep)
@@ -374 +374 @@
-my)
+my#my_adj)
@@ -376 +376 @@
-last)
+last#last_adv)
@@ -378 +378 @@
-beeing)
+beeing#being_n)
@@ -381 +381 @@
-at)
+at#at_prep)
@@ -384 +384 @@
-Denton)))))
+Denton#NA)))))
@@ -387 +387 @@
-it))
+it#it_pron))
@@ -389 +389 @@
-was)
+was#be_v)
@@ -391 +391 @@
-desired)
+desired#desire_v)
@@ -395 +395 @@
-that)
+that#that_conj)
@@ -399 +399 @@
-I))
+I#i_pron))
@@ -401 +401 @@
-should)
+should#shall_v)
@@ -403 +403 @@
-write)
+write#write_v)
@@ -406 +406 @@
-to)
+to#to_prep)
@@ -409 +409 @@
-my)
+my#my_adj)
@@ -411 +411 @@
-son)
+son#son_n1)
@@ -414 +414 @@
-Val.))))
+Val.#NA))))
@@ -417 +417 @@
-to)
+to#to_prep)
@@ -420 +420 @@
-provide)
+provide#provide_v)
@@ -422 +422 @@
-and)
+and#and_conj1)
@@ -424 +424 @@
-sende))
+sende#send_v1))
@@ -426 +426 @@
-downe)
+downe#down_adv)
@@ -429 +429 @@
-to)
+to#to_prep)
@@ -432 +432 @@
-Sandwich)))
+Sandwich#NA)))
@@ -435 +435 @@
-a)
+a#a_adj2)
@@ -437 +437 @@
-Grave)
+Grave#grave_n1)
@@ -439 +439 @@
-Stone)
+Stone#stone_n)
@@ -442 +442 @@
-for)
+for#for_prep)
@@ -446 +446 @@
-his)
+his#his_adj)
@@ -448 +448 @@
-brother's))
+brother's#brother_n))
@@ -450 +450 @@
-Grave)))
+Grave#grave_n1)))
@@ -455 +455 @@
-with)
+with#with_prep)
@@ -458 +458 @@
-an)
+an#a_adj2)
@@ -460 +460 @@
-Inscription)
+Inscription#inscription_n)
@@ -463 +463 @@
-according)
+according#accord_v)
@@ -466 +466 @@
-to)
+to#to_prep)
@@ -469 +469 @@
-that)
+that#that_pron)
@@ -473 +473 @@
-then))
+then#then_adv))
@@ -475 +475 @@
-geiven)
+geiven#give_v)
@@ -478 +478 @@
-mee))
+mee#me_pron1))
@@ -481 +481 @@
-in)
+in#in_prep)
@@ -484 +484 @@
-writeing)))))))))))))
+writeing#writing_n)))))))))))))
@@ -489 +489 @@
-and)
+and#and_conj1)
@@ -492 +492 @@
-that)
+that#that_conj)
@@ -496 +496 @@
-his)
+his#his_adj)
@@ -498 +498 @@
-Coate_Armes))
+Coate_Armes#SEE_coat_n))
@@ -500 +500 @@
-should)
+should#shall_v)
@@ -502 +502 @@
-bee)
+bee#be_v)
@@ -505 +505 @@
-likewise))
+likewise#likewise_adv))
@@ -507 +507 @@
-ingraven)
+ingraven#engrave_v)
@@ -510 +510 @@
-one)
+one#on_prep)
@@ -513 +513 @@
-the)
+the#the_adj)
@@ -515 +515 @@
-same)))))))
+same#same_adj)))))))
@@ -521 +521 @@
-which)
+which#which_adj)
@@ -523 +523 @@
-directions))
+directions#direction_n))
@@ -531 +531 @@
-I))
+I#i_pron))
@@ -533 +533 @@
-gave)
+gave#give_v)
@@ -536 +536 @@
-him))
+him#him_pron))
@@ -539 +539 @@
-at)
+at#at_prep)
@@ -542 +542 @@
-his)
+his#his_adj)
@@ -544 +544 @@
-goeing)
+goeing#going_n)
@@ -547 +547 @@
-from)
+from#from_prep)
@@ -550 +550 @@
-hence)))))))
+hence#hence_adv)))))))
@@ -559 +559 @@
-He))
+He#he_pron))
@@ -561 +561 @@
-asked)
+asked#ask_v)
@@ -564 +564 @@
-mee))
+mee#me_pron1))
@@ -567 +567 @@
-if)
+if#if_conj)
@@ -574 +574 @@
-the)
+the#the_adj)
@@ -576 +576 @@
-Spracklinge)
+Spracklinge#NA)
@@ -578 +578 @@
-Coat))
+Coat#coat_n))
@@ -580 +580 @@
-should)
+should#shall_v)
@@ -582 +582 @@
-be)
+be#be_v)
@@ -584 +584 @@
-joyned)
+joyned#join_v1)
@@ -587 +587 @@
-therwith)))
+therwith#therewith_adv)))
@@ -592 +592 @@
-or)
+or#or_conj2)
@@ -595 +595 @@
-not)))
+not#not_adv)))
@@ -601 +601 @@
-which))
+which#which_adj))
@@ -607 +607 @@
-I))
+I#i_pron))
@@ -609 +609 @@
-could)
+could#can_v1)
@@ -611 +611 @@
-not)
+not#not_adv)
@@ -614 +614 @@
-then))
+then#then_adv))
@@ -616 +616 @@
-resolve)
+resolve#resolve_v)
@@ -619 +619 @@
-him))
+him#him_pron))
@@ -622 +622 @@
-of)
+of#of_prep)
@@ -632 +632 @@
-but)
+but#but_prep)
@@ -636 +636 @@
-promised)
+promised#promise_v)
@@ -639 +639 @@
-to)
+to#to_prep)
@@ -641 +641 @@
-write)
+write#write_v)
@@ -644 +644 @@
-to)
+to#to_prep)
@@ -647 +647 @@
-you)))
+you#you_pron)))
@@ -650 +650 @@
-to)
+to#to_prep)
@@ -652 +652 @@
-know)
+know#know_v)
@@ -655 +655 @@
-your)
+your#your_pron)
@@ -657 +657 @@
-minde)
+minde#mind_n1)
@@ -660 +660 @@
-therein)))
+therein#therein_adv)))
@@ -666 +666 @@
-which))
+which#which_adj))
@@ -672 +672 @@
-I))
+I#i_pron))
@@ -674 +674 @@
-thinke)
+thinke#think_v2)
@@ -682 +682 @@
-were)
+were#be_v)
@@ -685 +685 @@
-needelesse))
+needelesse#needless_adv))
@@ -690 +690 @@
-unlesse)
+unlesse#unless_adv)
@@ -697 +697 @@
-some)
+some#some_pron)
@@ -699 +699 @@
-mention)
+mention#mention_n)
@@ -703 +703 @@
-were)
+were#be_v)
@@ -705 +705 @@
-made)
+made#make_v1)
@@ -708 +708 @@
-in)
+in#in_prep)
@@ -711 +711 @@
-the)
+the#the_adj)
@@ -713 +713 @@
-Inscription)))
+Inscription#inscription_n)))
@@ -716 +716 @@
-of)
+of#of_prep)
@@ -719 +719 @@
-his)
+his#his_adj)
@@ -721 +721 @@
-match)
+match#match_n1)
@@ -724 +724 @@
-with)
+with#with_prep)
@@ -727 +727 @@
-one)
+one#one_adj)
@@ -730 +730 @@
-of)
+of#of_prep)
@@ -733 +733 @@
-that)
+that#that_pron)
@@ -735 +735 @@
-house)))))))))))))))))
+house#house_n1)))))))))))))))))
@@ -741 +741 @@
-and)
+and#and_conj1)
@@ -744 +744 @@
-then))
+then#then_adv))
@@ -747 +747 @@
-a)
+a#a_adj2)
@@ -749 +749 @@
-pictchere)
+pictchere#picture_n)
@@ -752 +752 @@
-of)
+of#of_prep)
@@ -755 +755 @@
-the)
+the#the_adj)
@@ -757 +757 @@
-Sprakling)
+Sprakling#NA)
@@ -759 +759 @@
-Armes))))
+Armes#arms_n))))
@@ -761 +761 @@
-wil@)
+wil@#will_v1)
@@ -763 +763 @@
-@be)
+@be#be_v)
@@ -766 +766 @@
-needefull)
+needefull#needful_adj)
@@ -769 +769 @@
-to)
+to#to_prep)
@@ -771 +771 @@
-bee)
+bee#be_v)
@@ -773 +773 @@
-sent)
+sent#send_v1)
@@ -775 +775 @@
-upp)
+upp#up_adv1)
@@ -778 +778 @@
-to)
+to#to_prep)
@@ -781 +781 @@
-him)))))
+him#him_pron)))))
@@ -790 +790 @@
-To)
+To#to_prep)
@@ -793 +793 @@
-his)
+his#his_adj)
@@ -796 +796 @@
-verie)
+verie#very_adj)
@@ -798 +798 @@
-loveing))
+loveing#loving_adj))
@@ -800 +800 @@
-cosin)
+cosin#cousin_n)
@@ -803 +803 @@
-Mr.)
+Mr.#NA)
@@ -805 +805 @@
-Richard)
+Richard#NA)
@@ -807 +807 @@
-Oxenden))))
+Oxenden#NA))))
@@ -810 +810 @@
-at)
+at#at_prep)
@@ -813 +813 @@
-his)
+his#his_adj)
@@ -815 +815 @@
-house)
+house#house_n1)
@@ -818 +818 @@
-neere)
+neere#near_adj)
@@ -821 +821 @@
-Denton)))))
+Denton#NA)))))
@@ -823 +823 @@
-geve)
+geve#give_v)
@@ -826 +826 @@
-these))
+these#these_pron))
@@ -838 +838 @@
-Cosin)
+Cosin#cousin_n)
@@ -841 +841 @@
-Oxenden)))
+Oxenden#NA)))
@@ -846 +846 @@
-With)
+With#with_prep)
@@ -849 +849 @@
-the)
+the#the_adj)
@@ -851 +851 @@
-rememberance)
+rememberance#remembrance_n)
@@ -854 +854 @@
-of)
+of#of_prep)
@@ -857 +857 @@
-my)
+my#my_adj)
@@ -859 +859 @@
-Love)))))
+Love#love_n1)))))
@@ -862 +862 @@
-I))
+I#i_pron))
@@ -865 +865 @@
-hartely))
+hartely#heartily_adv))
@@ -867 +867 @@
-salute)
+salute#salute_v)
@@ -871 +871 @@
-you))
+you#you_pron))
@@ -874 +874 @@
-and)
+and#and_conj1)
@@ -877 +877 @@
-the)
+the#the_adj)
@@ -879 +879 @@
-rest)
+rest#rest_n2)
@@ -882 +882 @@
-of)
+of#of_prep)
@@ -885 +885 @@
-our)
+our#our_pron)
@@ -887 +887 @@
-freindes)
+freindes#friend_n)
@@ -890 +890 @@
-with)
+with#with_prep)
@@ -893 +893 @@
-you))))))))
+you#you_pron))))))))
@@ -902 +902 @@
-God))
+God#NA))
@@ -904 +904 @@
-bee)
+bee#be_v)
@@ -906 +906 @@
-thanked)
+thanked#thank_v)
@@ -911 +911 @@
-for)
+for#for_prep)
@@ -915 +915 @@
-my)
+my#my_adj)
@@ -917 +917 @@
-daughter's))
+daughter's#daughter_n))
@@ -919 +919 @@
-safe)
+safe#safe_adj)
@@ -921 +921 @@
-deliveraunce))))
+deliveraunce#deliverance_n))))
@@ -927 +927 @@
-and)
+and#and_conj1)
@@ -930 +930 @@
-God))
+God#NA))
@@ -932 +932 @@
-blesse)
+blesse#bless_v1)
@@ -935 +935 @@
-that)
+that#that_pron)
@@ -937 +937 @@
-Little)
+Little#little_adj)
@@ -939 +939 @@
-one)))
+one#one_adj)))
@@ -945 +945 @@
-and)
+and#and_conj1)
@@ -949 +949 @@
-make)
+make#make_v1)
@@ -953 +953 @@
-him))
+him#him_pron))
@@ -956 +956 @@
-his)
+his#his_adj)
@@ -958 +958 @@
-Servant)))
+Servant#servant_n)))
@@ -966 +966 @@
-and)
+and#and_conj1)
@@ -969 +969 @@
-God))
+God#NA))
@@ -971 +971 @@
-sende)
+sende#send_v1)
@@ -975 +975 @@
-us))
+us#us_pron))
@@ -978 +978 @@
-and)
+and#and_conj1)
@@ -981 +981 @@
-all)
+all#all_adj)
@@ -983 +983 @@
-other)
+other#other_adj)
@@ -985 +985 @@
-his)
+his#his_adj)
@@ -987 +987 @@
-freindes))))
+freindes#friend_n))))
@@ -990 +990 @@
-much)
+much#much_adj)
@@ -992 +992 @@
-Joy)
+Joy#joy_n)
@@ -995 +995 @@
-and)
+and#and_conj1)
@@ -998 +998 @@
-Comfort)))
+Comfort#comfort_n)))
@@ -1001 +1001 @@
-of)
+of#of_prep)
@@ -1004 +1004 @@
-him))))
+him#him_pron))))
@@ -1013 +1013 @@
-I))
+I#i_pron))
@@ -1015 +1015 @@
-purpose)
+purpose#purpose_v)
@@ -1021 +1021 @@
-God))
+God#NA))
@@ -1023 +1023 @@
-willing))
+willing#will_v2))
@@ -1028 +1028 @@
-to)
+to#to_prep)
@@ -1030 +1030 @@
-bee)
+bee#be_v)
@@ -1033 +1033 @@
-with)
+with#with_prep)
@@ -1036 +1036 @@
-you)))
+you#you_pron)))
@@ -1039 +1039 @@
-one)
+one#on_prep)
@@ -1042 +1042 @@
-Sonday)
+Sonday#NA)
@@ -1045 +1045 @@
-next)))))
+next#next_adj)))))
@@ -1048 +1048 @@
-to)
+to#to_prep)
@@ -1050 +1050 @@
-perfourme)
+perfourme#perform_v)
@@ -1054 +1054 @@
-my)
+my#my_adj)
@@ -1056 +1056 @@
-Daughter's))
+Daughter's#daughter_n))
@@ -1058 +1058 @@
-Desire)))
+Desire#desire_n)))
@@ -1066 +1066 @@
-And)
+And#and_conj1)
@@ -1070 +1070 @@
-so))
+so#so_adv))
@@ -1072 +1072 @@
-comendeing)
+comendeing#commend_v)
@@ -1075 +1075 @@
-you))
+you#you_pron))
@@ -1078 +1078 @@
-to)
+to#to_prep)
@@ -1081 +1081 @@
-God))))
+God#NA))))
@@ -1084 +1084 @@
-I))
+I#i_pron))
@@ -1086 +1086 @@
-rest)
+rest#rest_v2)
@@ -1089 +1089 @@
-Your)
+Your#your_pron)
@@ -1091 +1091 @@
-Loveing)
+Loveing#loving_adj)
@@ -1093 +1093 @@
-Freinde)
+Freinde#friend_n)
@@ -1096 +1096 @@
-Val.)
+Val.#NA)
@@ -1098 +1098 @@
-Pettit))))
+Pettit#NA))))
@@ -1104 +1104 @@
-Daundelion))
+Daundelion#NA))
@@ -1110 +1110 @@
-this)
+this#this_pron)
@@ -1112 +1112 @@
-9th)
+9th#NA)
@@ -1115 +1115 @@
-of)
+of#of_prep)
@@ -1118 +1118 @@
-September)))
+September#NA)))
@@ -1120 +1120 @@
-1625))
+1625#NA))
@@ -1133 +1133 @@
-Cosin)
+Cosin#cousin_n)
@@ -1136 +1136 @@
-Oxenden)))
+Oxenden#NA)))
@@ -1141 +1141 @@
-With)
+With#with_prep)
@@ -1144 +1144 @@
-the)
+the#the_adj)
@@ -1146 +1146 @@
-rememberaunce)
+rememberaunce#remembrance_n)
@@ -1149 +1149 @@
-of)
+of#of_prep)
@@ -1152 +1152 @@
-my)
+my#my_adj)
@@ -1154 +1154 @@
-love)))))
+love#love_n1)))))
@@ -1157 +1157 @@
-I))
+I#i_pron))
@@ -1159 +1159 @@
-doe)
+doe#do_v)
@@ -1162 +1162 @@
-hartily))
+hartily#heartily_adv))
@@ -1164 +1164 @@
-salute)
+salute#salute_v)
@@ -1167 +1167 @@
-you)))
+you#you_pron)))
@@ -1173 +1173 @@
-and)
+and#and_conj1)
@@ -1176 +1176 @@
-I))
+I#i_pron))
@@ -1178 +1178 @@
-have)
+have#have_v)
@@ -1181 +1181 @@
-now))
+now#now_adv))
@@ -1184 +1184 @@
-heerewith))
+heerewith#herewith_adv))
@@ -1186 +1186 @@
-sent)
+sent#send_v1)
@@ -1189 +1189 @@
-you))
+you#you_pron))
@@ -1192 +1192 @@
-a)
+a#a_adj2)
@@ -1194 +1194 @@
-smale)
+smale#small_adj)
@@ -1196 +1196 @@
-Roulett)
+Roulett#roulette_n)
@@ -1199 +1199 @@
-of)
+of#of_prep)
@@ -1204 +1204 @@
-Northdowne)
+Northdowne#NA)
@@ -1206 +1206 @@
-Ale)))
+Ale#ale_n)))
@@ -1210 +1210 @@
-which))
+which#which_adj))
@@ -1216 +1216 @@
-I))
+I#i_pron))
@@ -1218 +1218 @@
-praye)
+praye#pray_v)
@@ -1227 +1227 @@
-you))
+you#you_pron))
@@ -1229 +1229 @@
-accept)
+accept#accept_v)
@@ -1232 +1232 @@
-as)
+as#as_adv)
@@ -1235 +1235 @@
-a)
+a#a_adj2)
@@ -1237 +1237 @@
-Signe)
+Signe#sign_n)
@@ -1240 +1240 @@
-of)
+of#of_prep)
@@ -1243 +1243 @@
-my)
+my#my_adj)
@@ -1245 +1245 @@
-Love)))
+Love#love_n1)))
@@ -1248 +1248 @@
-and)
+and#and_conj1)
@@ -1251 +1251 @@
-Token)
+Token#token_n)
@@ -1254 +1254 @@
-of)
+of#of_prep)
@@ -1257 +1257 @@
-my)
+my#my_adj)
@@ -1259 +1259 @@
-thankfulnes)
+thankfulnes#thankfulness_n)
@@ -1262 +1262 @@
-for)
+for#for_prep)
@@ -1265 +1265 @@
-manie)
+manie#many_adj)
@@ -1267 +1267 @@
-Curtesies)
+Curtesies#courtesy_n)
@@ -1270 +1270 @@
-received)
+received#receive_v)
@@ -1273 +1273 @@
-from)
+from#from_prep)
@@ -1276 +1276 @@
-you)))))))))))))))))
+you#you_pron)))))))))))))))))
@@ -1285 +1285 @@
-if)
+if#if_conj)
@@ -1292 +1292 @@
-it))
+it#it_pron))
@@ -1294 +1294 @@
-bee)
+bee#be_v)
@@ -1297 +1297 @@
-good)))))
+good#good_adj)))))
@@ -1300 +1300 @@
-I))
+I#i_pron))
@@ -1302 +1302 @@
-have)
+have#have_v)
@@ -1305 +1305 @@
-my)
+my#my_adj)
@@ -1307 +1307 @@
-Desire))
+Desire#desire_n))
@@ -1315 +1315 @@
-but)
+but#but_prep)
@@ -1318 +1318 @@
-if)
+if#if_conj)
@@ -1325 +1325 @@
-it))
+it#it_pron))
@@ -1327 +1327 @@
-prove)
+prove#prove_v)
@@ -1329 +1329 @@
-not)
+not#not_adv)
@@ -1332 +1332 @@
-well)))))
+well#well_adj)))))
@@ -1338 +1338 @@
-I))
+I#i_pron))
@@ -1340 +1340 @@
-pray)
+pray#pray_v)
@@ -1343 +1343 @@
-you)))
+you#you_pron)))
@@ -1345 +1345 @@
-blame)
+blame#blame_v)
@@ -1349 +1349 @@
-the)
+the#the_adj)
@@ -1351 +1351 @@
-Brewer))
+Brewer#brewer_n))
@@ -1354 +1354 @@
-and)
+and#and_conj1)
@@ -1357 +1357 @@
-not)
+not#not_adv)
@@ -1359 +1359 @@
-mee))))
+mee#me_pron1))))
@@ -1367 +1367 @@
-And)
+And#and_conj1)
@@ -1371 +1371 @@
-so))
+so#so_adv))
@@ -1373 +1373 @@
-prayeing)
+prayeing#pray_v)
@@ -1376 +1376 @@
-you)))
+you#you_pron)))
@@ -1379 +1379 @@
-to)
+to#to_prep)
@@ -1381 +1381 @@
-remember)
+remember#remember_v)
@@ -1384 +1384 @@
-me))
+me#me_pron1))
@@ -1387 +1387 @@
-to)
+to#to_prep)
@@ -1391 +1391 @@
-my)
+my#my_adj)
@@ -1393 +1393 @@
-Ladye)
+Ladye#lady_n)
@@ -1398 +1398 @@
-my)
+my#my_adj)
@@ -1400 +1400 @@
-Cosin))
+Cosin#cousin_n))
@@ -1403 +1403 @@
-your)
+your#your_pron)
@@ -1405 +1405 @@
-Wife)))
+Wife#wife_n)))
@@ -1411 +1411 @@
-my)
+my#my_adj)
@@ -1413 +1413 @@
-Daughter)
+Daughter#daughter_n)
@@ -1416 +1416 @@
-Pettit))))
+Pettit#NA))))
@@ -1419 +1419 @@
-and)
+and#and_conj1)
@@ -1422 +1422 @@
-the)
+the#the_adj)
@@ -1424 +1424 @@
-rest)
+rest#rest_n2)
@@ -1427 +1427 @@
-of)
+of#of_prep)
@@ -1430 +1430 @@
-our)
+our#our_pron)
@@ -1432 +1432 @@
-Freindes))))))))
+Freindes#friend_n))))))))
@@ -1437 +1437 @@
-I))
+I#i_pron))
@@ -1439 +1439 @@
-commende)
+commende#commend_v)
@@ -1442 +1442 @@
-you))
+you#you_pron))
@@ -1445 +1445 @@
-to)
+to#to_prep)
@@ -1448 +1448 @@
-God))))
+God#NA))))
@@ -1454 +1454 @@
-and)
+and#and_conj1)
@@ -1458 +1458 @@
-rest)
+rest#rest_v2)
@@ -1461 +1461 @@
-Your)
+Your#your_pron)
@@ -1463 +1463 @@
-Loveing)
+Loveing#loving_adj)
@@ -1465 +1465 @@
-Freinde)
+Freinde#friend_n)
@@ -1468 +1468 @@
-Val.)
+Val.#NA)
@@ -1470 +1470 @@
-Pettit))))
+Pettit#NA))))
@@ -1476 +1476 @@
-Daundelion))
+Daundelion#NA))
@@ -1482 +1482 @@
-the)
+the#the_adj)
@@ -1484 +1484 @@
-5th)
+5th#NA)
@@ -1487 +1487 @@
-of)
+of#of_prep)
@@ -1490 +1490 @@
-March)))
+March#NA)))
@@ -1492 +1492 @@
-1625))
+1625#NA))
