--- proud-1636-e2-p1.psd.vert	2025-09-10 13:51:11
+++ proud-1636-e2-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-Neve)
+Neve#NA)
@@ -16 +16 @@
-Oxinden))
+Oxinden#NA))
@@ -21 +21 @@
-I))
+I#i_pron))
@@ -23 +23 @@
-being)
+being#be_v)
@@ -26 +26 @@
-welleng)
+welleng#willing_adj)
@@ -29 +29 @@
-to)
+to#to_prep)
@@ -31 +31 @@
-fullfell)
+fullfell#fulfil_v)
@@ -34 +34 @@
-all)
+all#all_adj)
@@ -36 +36 @@
-your)
+your#your_pron)
@@ -38 +38 @@
-desires)
+desires#desire_n)
@@ -41 +41 @@
-to)
+to#to_prep)
@@ -43 +43 @@
-perswaid)
+perswaid#persuade_v)
@@ -46 +46 @@
-your)
+your#your_pron)
@@ -48 +48 @@
-brothar)
+brothar#brother_n)
@@ -51 +51 @@
-Rechard)
+Rechard#NA)
@@ -53 +53 @@
-Oxienden)))
+Oxienden#NA)))
@@ -56 +56 @@
-to)
+to#to_prep)
@@ -58 +58 @@
-go)
+go#go_v)
@@ -60 +60 @@
-oufer))))))
+oufer#over_adv))))))
@@ -65 +65 @@
-because)
+because#because_adv)
@@ -72 +72 @@
-hee))
+hee#he_pron))
@@ -74 +74 @@
-should)
+should#shall_v)
@@ -76 +76 @@
-not)
+not#not_adv)
@@ -78 +78 @@
-be)
+be#be_v)
@@ -81 +81 @@
-trubbelsom)
+trubbelsom#troublesome_adj)
@@ -84 +84 @@
-to)
+to#to_prep)
@@ -88 +88 @@
-you))
+you#you_pron))
@@ -91 +91 @@
-nor)
+nor#nor_conj1)
@@ -94 +94 @@
-my)
+my#my_adj)
@@ -96 +96 @@
-sester)))))))))
+sester#sister_n)))))))))
@@ -104 +104 @@
-but)
+but#but_prep)
@@ -108 +108 @@
-past)
+past#pass_v)
@@ -111 +111 @@
-my)
+my#my_adj)
@@ -113 +113 @@
-word))
+word#word_n))
@@ -116 +116 @@
-to)
+to#to_prep)
@@ -119 +119 @@
-my)
+my#my_adj)
@@ -121 +121 @@
-Cosson)
+Cosson#cousin_n)
@@ -124 +124 @@
-Pettet))))
+Pettet#NA))))
@@ -127 +127 @@
-for)
+for#for_prep)
@@ -130 +130 @@
-5)
+5#five_v)
@@ -132 +132 @@
-pounds)
+pounds#pound_n1)
@@ -141 +141 @@
-to)
+to#to_prep)
@@ -143 +143 @@
-send)
+send#send_v1)
@@ -146 +146 @@
-hem))
+hem#him_pron))
@@ -148 +148 @@
-oufer))
+oufer#over_adv))
@@ -151 +151 @@
-and)
+and#and_conj1)
@@ -156 +156 @@
-let)
+let#let_v1)
@@ -160 +160 @@
-hem))
+hem#him_pron))
@@ -162 +162 @@
-be)
+be#be_v)
@@ -165 +165 @@
-weth)
+weth#with_prep)
@@ -168 +168 @@
-me)))
+me#me_pron1)))
@@ -171 +171 @@
-when)
+when#when_adv)
@@ -178 +178 @@
-hee))
+hee#he_pron))
@@ -180 +180 @@
-plesed)))))))))))
+plesed#please_v)))))))))))
@@ -188 +188 @@
-and)
+and#and_conj1)
@@ -191 +191 @@
-for)
+for#for_prep)
@@ -194 +194 @@
-all)
+all#all_adj)
@@ -196 +196 @@
-my)
+my#my_adj)
@@ -198 +198 @@
-good)
+good#good_adj)
@@ -200 +200 @@
-well)))
+well#will_n1)))
@@ -203 +203 @@
-I))
+I#i_pron))
@@ -205 +205 @@
-find)
+find#find_v)
@@ -213 +213 @@
-hee))
+hee#he_pron))
@@ -215 +215 @@
-dus)
+dus#do_v)
@@ -217 +217 @@
-not)
+not#not_adv)
@@ -219 +219 @@
-euse)
+euse#use_v)
@@ -222 +222 @@
-me))
+me#me_pron1))
@@ -225 +225 @@
-like)
+like#like_adj)
@@ -228 +228 @@
-a)
+a#a_adj2)
@@ -230 +230 @@
-Gentelman))))
+Gentelman#gentleman_n))))
@@ -233 +233 @@
-and)
+and#and_conj1)
@@ -238 +238 @@
-well)
+well#will_v1)
@@ -240 +240 @@
-not)
+not#not_adv)
@@ -242 +242 @@
-pai)
+pai#pay_v1)
@@ -245 +245 @@
-my)
+my#my_adj)
@@ -247 +247 @@
-cosson)
+cosson#cousin_n)
@@ -250 +250 @@
-pettet)))))
+pettet#NA)))))
@@ -255 +255 @@
-all)
+all#all_adj)
@@ -257 +257 @@
-thought)
+thought#though_adv)
@@ -266 +266 @@
-hee))
+hee#he_pron))
@@ -268 +268 @@
-said)
+said#say_v1)
@@ -275 +275 @@
-hee))
+hee#he_pron))
@@ -277 +277 @@
-had)
+had#have_v)
@@ -280 +280 @@
-the)
+the#the_adj)
@@ -282 +282 @@
-monny)))))))))
+monny#money_n)))))))))
@@ -290 +290 @@
-but)
+but#but_prep)
@@ -293 +293 @@
-I))
+I#i_pron))
@@ -295 +295 @@
-ded)
+ded#do_v)
@@ -297 +297 @@
-not)
+not#not_adv)
@@ -299 +299 @@
-thenke)
+thenke#think_v2)
@@ -306 +306 @@
-I))
+I#i_pron))
@@ -308 +308 @@
-should)
+should#shall_v)
@@ -310 +310 @@
-haufe)
+haufe#have_v)
@@ -312 +312 @@
-ben)
+ben#be_v)
@@ -315 +315 @@
-so)
+so#so_adv)
@@ -317 +317 @@
-unkindly))
+unkindly#unkindly_adv))
@@ -319 +319 @@
-delt)
+delt#deal_v)
@@ -322 +322 @@
-weth_all)
+weth_all#withal_adv)
@@ -327 +327 @@
-by)
+by#by_prep)
@@ -330 +330 @@
-hem)
+hem#him_pron)
@@ -334 +334 @@
-nor)
+nor#nor_conj1)
@@ -336 +336 @@
-you)))))
+you#you_pron)))))
@@ -344 +344 @@
-for)
+for#for_prep)
@@ -347 +347 @@
-you))
+you#you_pron))
@@ -349 +349 @@
-wret)
+wret#write_v)
@@ -352 +352 @@
-to)
+to#to_prep)
@@ -355 +355 @@
-my)
+my#my_adj)
@@ -357 +357 @@
-Cosson)
+Cosson#cousin_n)
@@ -360 +360 @@
-Pettet))))
+Pettet#NA))))
@@ -363 +363 @@
-to)
+to#to_prep)
@@ -365 +365 @@
-let)
+let#let_v1)
@@ -369 +369 @@
-hem))
+hem#him_pron))
@@ -371 +371 @@
-haufe)
+haufe#have_v)
@@ -374 +374 @@
-such)
+such#such_adj)
@@ -376 +376 @@
-monnys)
+monnys#money_n)
@@ -379 +379 @@
-as)
+as#as_adv)
@@ -390 +390 @@
-hee))
+hee#he_pron))
@@ -392 +392 @@
-ded)
+ded#do_v)
@@ -394 +394 @@
-want))))))))
+want#want_v))))))))
@@ -400 +400 @@
-and)
+and#and_conj1)
@@ -403 +403 @@
-you))
+you#you_pron))
@@ -405 +405 @@
-wret)
+wret#write_v)
@@ -408 +408 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-me)))
+me#me_pron1)))
@@ -414 +414 @@
-to)
+to#to_prep)
@@ -416 +416 @@
-get)
+get#get_v)
@@ -419 +419 @@
-hem))
+hem#him_pron))
@@ -421 +421 @@
-oufar)
+oufar#over_adv)
@@ -424 +424 @@
-so)
+so#so_adv)
@@ -426 +426 @@
-sone)
+sone#soon_adv)
@@ -429 +429 @@
-as)
+as#as_adv)
@@ -440 +440 @@
-I))
+I#i_pron))
@@ -442 +442 @@
-cold)
+cold#can_v1)
@@ -449 +449 @@
-be_fore)
+be_fore#before_adv)
@@ -456 +456 @@
-his)
+his#his_adj)
@@ -458 +458 @@
-monny))
+monny#money_n))
@@ -460 +460 @@
-was)
+was#be_v)
@@ -462 +462 @@
-spent))))
+spent#spend_v1))))
@@ -468 +468 @@
-which))
+which#which_adj))
@@ -476 +476 @@
-I))
+I#i_pron))
@@ -478 +478 @@
-ded))))
+ded#do_v))))
@@ -487 +487 @@
-it))
+it#it_pron))
@@ -489 +489 @@
-well)
+well#will_v1)
@@ -491 +491 @@
-make)
+make#make_v1)
@@ -495 +495 @@
-me))
+me#me_pron1))
@@ -497 +497 @@
-haufe)
+haufe#have_v)
@@ -500 +500 @@
-care))
+care#care_n1))
@@ -503 +503 @@
-heareaftar))
+heareaftar#hereafter_adv))
@@ -507 +507 @@
-how))
+how#how_adv))
@@ -515 +515 @@
-I))
+I#i_pron))
@@ -517 +517 @@
-do)
+do#do_v)
@@ -520 +520 @@
-cortesis)
+cortesis#courtesy_n)
@@ -523 +523 @@
-for)
+for#for_prep)
@@ -526 +526 @@
-my)
+my#my_adj)
@@ -528 +528 @@
-frends)))))))
+frends#friend_n)))))))
@@ -536 +536 @@
-for)
+for#for_prep)
@@ -541 +541 @@
-what))
+what#what_pron))
@@ -549,3 +549,3 @@
-I))
-(DOD
-ded))))
+I#i_pron))
+(DOD
+ded#do_v))))
@@ -553 +553 @@
-was)
+was#be_v)
@@ -556 +556 @@
-for)
+for#for_prep)
@@ -559 +559 @@
-good)
+good#good_adj)
@@ -561 +561 @@
-well)))
+well#will_n1)))
@@ -570 +570 @@
-it))
+it#it_pron))
@@ -572 +572 @@
-may@)
+may@#may_v1)
@@ -574 +574 @@
-@be)
+@be#be_v)
@@ -581 +581 @@
-there))
+there#there_adv))
@@ -583 +583 @@
-may)
+may#may_v1)
@@ -586 +586 @@
-some))
+some#some_pron))
@@ -588 +588 @@
-haufe)
+haufe#have_v)
@@ -591 +591 @@
-a_casion))
+a_casion#occasion_n1))
@@ -594 +594 @@
-heare)
+heare#here_adv)
@@ -596 +596 @@
-aftar))))
+aftar#after_adv))))
@@ -604 +604 @@
-but)
+but#but_prep)
@@ -607 +607 @@
-I))
+I#i_pron))
@@ -609 +609 @@
-haufe)
+haufe#have_v)
@@ -611 +611 @@
-cond)
+cond#con_v1)
@@ -614 +614 @@
-my)
+my#my_adj)
@@ -616 +616 @@
-leson))
+leson#lesson_n))
@@ -625 +625 @@
-I))
+I#i_pron))
@@ -627 +627 @@
-heare)
+heare#hear_v)
@@ -635 +635 @@
-you))
+you#you_pron))
@@ -637 +637 @@
-haufe)
+haufe#have_v)
@@ -640 +640 @@
-yet))
+yet#yet_adv))
@@ -643 +643 @@
-some)
+some#some_pron)
@@ -645 +645 @@
-monnys)
+monnys#money_n)
@@ -648 +648 @@
-of)
+of#of_prep)
@@ -651 +651 @@
-hes))))
+hes#his_pron1))))
@@ -654 +654 @@
-in)
+in#in_prep)
@@ -657 +657 @@
-your)
+your#your_pron)
@@ -659 +659 @@
-hands)))))
+hands#hand_n)))))
@@ -662 +662 @@
-and)
+and#and_conj1)
@@ -665 +665 @@
-that)
+that#that_conj)
@@ -669 +669 @@
-hee))
+hee#he_pron))
@@ -671 +671 @@
-hath)
+hath#have_v)
@@ -673 +673 @@
-sent)
+sent#send_v1)
@@ -676 +676 @@
-for)
+for#for_prep)
@@ -679 +679 @@
-it)))))))
+it#it_pron)))))))
@@ -688 +688 @@
-if)
+if#if_conj)
@@ -695 +695 @@
-you))
+you#you_pron))
@@ -697 +697 @@
-well)
+well#will_v1)
@@ -699 +699 @@
-please)
+please#please_v)
@@ -702 +702 @@
-to)
+to#to_prep)
@@ -704 +704 @@
-send)
+send#send_v1)
@@ -707 +707 @@
-that)
+that#that_pron)
@@ -709 +709 @@
-monny))
+monny#money_n))
@@ -712 +712 @@
-to)
+to#to_prep)
@@ -715 +715 @@
-my)
+my#my_adj)
@@ -717 +717 @@
-Cosson)
+Cosson#cousin_n)
@@ -720 +720 @@
-Pettet))))))))
+Pettet#NA))))))))
@@ -723 +723 @@
-hee)
+hee#he_pron)
@@ -727 +727 @@
-can)
+can#can_v1)
@@ -729 +729 @@
-pai)
+pai#pay_v1)
@@ -732 +732 @@
-hemselfe))
+hemselfe#himself_pron))
@@ -741 +741 @@
-for)
+for#for_prep)
@@ -744 +744 @@
-my)
+my#my_adj)
@@ -746 +746 @@
-on)
+on#own_adj)
@@ -748 +748 @@
-part)))
+part#part_n)))
@@ -751 +751 @@
-if)
+if#if_conj)
@@ -758 +758 @@
-my)
+my#my_adj)
@@ -760 +760 @@
-Cosson)
+Cosson#cousin_n)
@@ -763 +763 @@
-Petet)))
+Petet#NA)))
@@ -765 +765 @@
-demand)
+demand#demand_v)
@@ -768 +768 @@
-his)
+his#his_adj)
@@ -770 +770 @@
-monny))
+monny#money_n))
@@ -773 +773 @@
-of)
+of#of_prep)
@@ -776 +776 @@
-me))))))
+me#me_pron1))))))
@@ -779 +779 @@
-I))
+I#i_pron))
@@ -781 +781 @@
-will)
+will#will_v1)
@@ -783 +783 @@
-put)
+put#put_v)
@@ -786 +786 @@
-it))
+it#it_pron))
@@ -788 +788 @@
-over)
+over#over_adv)
@@ -791 +791 @@
-to)
+to#to_prep)
@@ -794 +794 @@
-my)
+my#my_adj)
@@ -796 +796 @@
-landelord)
+landelord#landlord_n)
@@ -800 +800 @@
-ho))
+ho#who_pron))
@@ -807 +807 @@
-sweares)
+sweares#swear_v)
@@ -814 +814 @@
-hee))
+hee#he_pron))
@@ -816 +816 @@
-well)
+well#will_v1)
@@ -818 +818 @@
-a_rest)
+a_rest#arrest_v)
@@ -821 +821 @@
-hem))
+hem#him_pron))
@@ -824 +824 @@
-for)
+for#for_prep)
@@ -827 +827 @@
-it)))
+it#it_pron)))
@@ -832 +832 @@
-for)
+for#for_prep)
@@ -835 +835 @@
-hes)
+hes#his_adj)
@@ -837 +837 @@
-ell)
+ell#ill_adj)
@@ -839 +839 @@
-eueseng)
+eueseng#using_n)
@@ -844 +844 @@
-of)
+of#of_prep)
@@ -847 +847 @@
-hem)))))))))))
+hem#him_pron)))))))))))
@@ -855 +855 @@
-and)
+and#and_conj1)
@@ -858 +858 @@
-when)
+when#when_adv)
@@ -866 +866 @@
-hee))
+hee#he_pron))
@@ -868 +868 @@
-is)
+is#be_v)
@@ -871 +871 @@
-in)
+in#in_prep)
@@ -874 +874 @@
-preson))))
+preson#prison_n))))
@@ -877 +877 @@
-and)
+and#and_conj1)
@@ -881 +881 @@
-all)
+all#all_adj)
@@ -883 +883 @@
-hes)
+hes#his_adj)
@@ -885 +885 @@
-monny))
+monny#money_n))
@@ -887 +887 @@
-spent))))))
+spent#spend_v1))))))
@@ -890 +890 @@
-hes)
+hes#his_adj)
@@ -892 +892 @@
-frends))
+frends#friend_n))
@@ -894 +894 @@
-well)
+well#will_v1)
@@ -896 +896 @@
-wesh)
+wesh#wash_v)
@@ -903 +903 @@
-it))
+it#it_pron))
@@ -905 +905 @@
-had)
+had#have_v)
@@ -907 +907 @@
-ben)
+ben#be_v)
@@ -909 +909 @@
-paid)))
+paid#pay_v1)))
@@ -918,3 +918,3 @@
-I))
-(DOP
-do)
+I#i_pron))
+(DOP
+do#do_v)
@@ -922 +922 @@
-not)
+not#not_adv)
@@ -924 +924 @@
-care)
+care#care_v)
@@ -927 +927 @@
-for)
+for#for_prep)
@@ -930 +930 @@
-5)
+5#five_v)
@@ -932 +932 @@
-pounds)))
+pounds#pound_n1)))
@@ -936 +936 @@
-had)
+had#have_v)
@@ -939 +939 @@
-it))
+it#it_pron))
@@ -941 +941 @@
-ben)
+ben#be_v)
@@ -944 +944 @@
-to)
+to#to_prep)
@@ -946 +946 @@
-do)
+do#do_v)
@@ -949 +949 @@
-hem))
+hem#him_pron))
@@ -952 +952 @@
-good))))))
+good#good_adj))))))
@@ -958 +958 @@
-but)
+but#but_prep)
@@ -961 +961 @@
-to)
+to#to_prep)
@@ -963 +963 @@
-be)
+be#be_v)
@@ -965 +965 @@
-cheted)
+cheted#cheat_v)
@@ -968 +968 @@
-in)
+in#in_prep)
@@ -971 +971 @@
-thes)
+thes#this_pron)
@@ -973 +973 @@
-mannar))))
+mannar#manner_n))))
@@ -976 +976 @@
-I))
+I#i_pron))
@@ -978 +978 @@
-well)
+well#will_v1)
@@ -980 +980 @@
-not)
+not#not_adv)
@@ -985 +985 @@
-if)
+if#if_conj)
@@ -992 +992 @@
-heare))
+heare#here_adv))
@@ -996 +996 @@
-be)
+be#be_v)
@@ -999 +999 @@
-right))
+right#right_n))
@@ -1002 +1002 @@
-in)
+in#in_prep)
@@ -1005 +1005 @@
-London))))))
+London#NA))))))
@@ -1014 +1014 @@
-I))
+I#i_pron))
@@ -1016 +1016 @@
-prai)
+prai#pray_v)
@@ -1024 +1024 @@
-god))
+god#NA))
@@ -1026 +1026 @@
-bles)
+bles#bless_v1)
@@ -1029 +1029 @@
-hem)))
+hem#him_pron)))
@@ -1032 +1032 @@
-and)
+and#and_conj1)
@@ -1037 +1037 @@
-geufe)
+geufe#give_v)
@@ -1040 +1040 @@
-hem))
+hem#him_pron))
@@ -1043 +1043 @@
-gras)
+gras#grace_n)
@@ -1046 +1046 @@
-to)
+to#to_prep)
@@ -1048 +1048 @@
-take)
+take#take_v)
@@ -1051 +1051 @@
-some)
+some#some_pron)
@@ -1053 +1053 @@
-onnest)
+onnest#honest_adj)
@@ -1055 +1055 @@
-cores))))
+cores#course_n1))))
@@ -1058 +1058 @@
-that)
+that#that_conj)
@@ -1062 +1062 @@
-hee))
+hee#he_pron))
@@ -1064 +1064 @@
-be)
+be#be_v)
@@ -1067 +1067 @@
-no)
+no#no_adj)
@@ -1069 +1069 @@
-shame)
+shame#shame_n)
@@ -1072 +1072 @@
-to)
+to#to_prep)
@@ -1075 +1075 @@
-hes)
+hes#his_adj)
@@ -1077 +1077 @@
-frends))))))))))
+frends#friend_n))))))))))
@@ -1086 +1086 @@
-when)
+when#when_adv)
@@ -1093 +1093 @@
-hes)
+hes#his_adj)
@@ -1095 +1095 @@
-monny))
+monny#money_n))
@@ -1097 +1097 @@
-is)
+is#be_v)
@@ -1099 +1099 @@
-spent))))
+spent#spend_v1))))
@@ -1102 +1102 @@
-you))
+you#you_pron))
@@ -1104 +1104 @@
-should)
+should#shall_v)
@@ -1106 +1106 @@
-do)
+do#do_v)
@@ -1109 +1109 @@
-well))
+well#well_adv))
@@ -1112 +1112 @@
-to)
+to#to_prep)
@@ -1114 +1114 @@
-kep)
+kep#keep_v)
@@ -1117 +1117 @@
-some)
+some#some_pron)
@@ -1120 +1120 @@
-of)
+of#of_prep)
@@ -1123 +1123 @@
-hes)
+hes#his_adj)
@@ -1125 +1125 @@
-monny))))
+monny#money_n))))
@@ -1128 +1128 @@
-in)
+in#in_prep)
@@ -1131 +1131 @@
-your)
+your#your_pron)
@@ -1133 +1133 @@
-hand)))))
+hand#hand_n)))))
@@ -1139 +1139 @@
-for)
+for#for_prep)
@@ -1142 +1142 @@
-when)
+when#when_adv)
@@ -1149 +1149 @@
-hee))
+hee#he_pron))
@@ -1151 +1151 @@
-comes)
+comes#come_v)
@@ -1154 +1154 @@
-to)
+to#to_prep)
@@ -1157 +1157 @@
-want))))))
+want#want_n2))))))
@@ -1160 +1160 @@
-I))
+I#i_pron))
@@ -1162 +1162 @@
-know)
+know#know_v)
@@ -1165 +1165 @@
-no)
+no#no_adj)
@@ -1167 +1167 @@
-frend)
+frend#friend_n)
@@ -1176 +1176 @@
-hee))
+hee#he_pron))
@@ -1178 +1178 @@
-has)
+has#have_v)
@@ -1181 +1181 @@
-to)
+to#to_prep)
@@ -1183 +1183 @@
-fly)
+fly#fly_v1)
@@ -1186 +1186 @@
-to)
+to#to_prep)
@@ -1191 +1191 @@
-but)
+but#but_prep)
@@ -1194 +1194 @@
-to)
+to#to_prep)
@@ -1197 +1197 @@
-you)))))
+you#you_pron)))))
@@ -1206 +1206 @@
-my)
+my#my_adj)
@@ -1208 +1208 @@
-loufe))
+loufe#luff_n1))
@@ -1211 +1211 @@
-to)
+to#to_prep)
@@ -1214 +1214 @@
-my)
+my#my_adj)
@@ -1216 +1216 @@
-Nese)))
+Nese#niece_n)))
@@ -1219 +1219 @@
-weth)
+weth#with_prep)
@@ -1222 +1222 @@
-my)
+my#my_adj)
@@ -1224 +1224 @@
-best)
+best#best_adj)
@@ -1226 +1226 @@
-respect)
+respect#respect_n)
@@ -1229 +1229 @@
-to)
+to#to_prep)
@@ -1232 +1232 @@
-yourself)))))
+yourself#yourself_pron)))))
@@ -1241 +1241 @@
-I))
+I#i_pron))
@@ -1243 +1243 @@
-desir)
+desir#desire_v)
@@ -1246 +1246 @@
-your)
+your#your_pron)
@@ -1248 +1248 @@
-kind)
+kind#kind_adj)
@@ -1250 +1250 @@
-ansar))
+ansar#answer_n))
@@ -1259 +1259 @@
-Good)
+Good#good_adj)
@@ -1261 +1261 @@
-nevey))
+nevey#nephew_n))
@@ -1263 +1263 @@
-do)
+do#do_v)
@@ -1266 +1266 @@
-me))
+me#me_pron1))
@@ -1270 +1270 @@
-so)
+so#so_adv)
@@ -1272 +1272 @@
-mych))
+mych#much_adj))
@@ -1274 +1274 @@
-favor)
+favor#favour_n)
@@ -1277 +1277 @@
-as)
+as#as_adv)
@@ -1282 +1282 @@
-when)
+when#when_adv)
@@ -1289 +1289 @@
-you))
+you#you_pron))
@@ -1291 +1291 @@
-ar)
+ar#be_v)
@@ -1294 +1294 @@
-at)
+at#at_prep)
@@ -1297 +1297 @@
-lesure))))))
+lesure#leisure_n))))))
@@ -1299 +1299 @@
-to)
+to#to_prep)
@@ -1301 +1301 @@
-veufe)
+veufe#view_v)
@@ -1306 +1306 @@
-my)
+my#my_adj)
@@ -1308 +1308 @@
-woods)
+woods#wood_n1)
@@ -1311 +1311 @@
-at)
+at#at_prep)
@@ -1314 +1314 @@
-wolleg)))))
+wolleg#NA)))))
@@ -1317 +1317 @@
-and)
+and#and_conj1)
@@ -1322 +1322 @@
-send)
+send#send_v1)
@@ -1325 +1325 @@
-me))
+me#me_pron1))
@@ -1328 +1328 @@
-word))
+word#word_n))
@@ -1332 +1332 @@
-what)
+what#what_pron)
@@ -1340 +1340 @@
-you))
+you#you_pron))
@@ -1342 +1342 @@
-thenke)
+thenke#think_v2)
@@ -1349 +1349 @@
-they))
+they#they_pron))
@@ -1351 +1351 @@
-are)
+are#be_v)
@@ -1354 +1354 @@
-worth)
+worth#worth_adj)
@@ -1359 +1359 @@
-a_n@)
+a_n@#a_adj2)
@@ -1361 +1361 @@
-@akar)))))))))))
+@akar#acre_n)))))))))))
@@ -1370 +1370 @@
-if)
+if#if_conj)
@@ -1377 +1377 @@
-you))
+you#you_pron))
@@ -1379 +1379 @@
-well)
+well#will_v1)
@@ -1381 +1381 @@
-do)
+do#do_v)
@@ -1384 +1384 @@
-me))
+me#me_pron1))
@@ -1387 +1387 @@
-thes)
+thes#this_pron)
@@ -1389 +1389 @@
-favor)))))
+favor#favour_n)))))
@@ -1392 +1392 @@
-I))
+I#i_pron))
@@ -1396 +1396 @@
-well)
+well#will_v1)
@@ -1398 +1398 @@
-be)
+be#be_v)
@@ -1401 +1401 @@
-redy)
+redy#ready_adj)
@@ -1405 +1405 @@
-to)
+to#to_prep)
@@ -1407 +1407 @@
-do)
+do#do_v)
@@ -1410 +1410 @@
-the)
+the#the_adj)
@@ -1412 +1412 @@
-like)
+like#like_adj)
@@ -1414 +1414 @@
-cortesey))
+cortesey#courtesy_n))
@@ -1417 +1417 @@
-for)
+for#for_prep)
@@ -1421 +1421 @@
-you))
+you#you_pron))
@@ -1424 +1424 @@
-and)
+and#and_conj1)
@@ -1427 +1427 @@
-yours))))))
+yours#yours_pron))))))
@@ -1432 +1432 @@
-and)
+and#and_conj1)
@@ -1435 +1435 @@
-to)
+to#to_prep)
@@ -1437 +1437 @@
-help)
+help#help_v)
@@ -1440 +1440 @@
-me))
+me#me_pron1))
@@ -1443 +1443 @@
-to)
+to#to_prep)
@@ -1446 +1446 @@
-a)
+a#a_adj2)
@@ -1448 +1448 @@
-Chapman)))
+Chapman#chapman_n)))
@@ -1451 +1451 @@
-for)
+for#for_prep)
@@ -1454 +1454 @@
-them)))))))
+them#them_pron)))))))
@@ -1462 +1462 @@
-for)
+for#for_prep)
@@ -1465 +1465 @@
-I))
+I#i_pron))
@@ -1467 +1467 @@
-purpos)
+purpos#purpose_v)
@@ -1470 +1470 @@
-to)
+to#to_prep)
@@ -1472 +1472 @@
-cut)
+cut#cut_v)
@@ -1475 +1475 @@
-som))
+som#some_pron))
@@ -1477 +1477 @@
-down)
+down#down_adv)
@@ -1480 +1480 @@
-the)
+the#the_adj)
@@ -1482 +1482 @@
-wenttar))))
+wenttar#winter_n1))))
@@ -1488 +1488 @@
-and)
+and#and_conj1)
@@ -1491 +1491 @@
-you))
+you#you_pron))
@@ -1494 +1494 @@
-dewelleng)
+dewelleng#dwell_v)
@@ -1497 +1497 @@
-so)
+so#so_adv)
@@ -1499 +1499 @@
-ny)))
+ny#nigh_adv)))
@@ -1501 +1501 @@
-may)
+may#may_v1)
@@ -1503 +1503 @@
-do)
+do#do_v)
@@ -1506 +1506 @@
-me))
+me#me_pron1))
@@ -1509 +1509 @@
-a)
+a#a_adj2)
@@ -1511 +1511 @@
-gret)
+gret#great_adj)
@@ -1513 +1513 @@
-favor))
+favor#favour_n))
@@ -1525 +1525 @@
-To)
+To#to_prep)
@@ -1528 +1528 @@
-my)
+my#my_adj)
@@ -1530 +1530 @@
-loving)
+loving#loving_adj)
@@ -1532 +1532 @@
-Nevey)
+Nevey#nephew_n)
@@ -1535 +1535 @@
-Master)
+Master#NA)
@@ -1537 +1537 @@
-Hendry)
+Hendry#NA)
@@ -1539 +1539 @@
-Oxienden))))
+Oxienden#NA))))
@@ -1542 +1542 @@
-at)
+at#at_prep)
@@ -1545 +1545 @@
-his)
+his#his_adj)
@@ -1547 +1547 @@
-hous)
+hous#house_n1)
@@ -1550 +1550 @@
-in)
+in#in_prep)
@@ -1553 +1553 @@
-Wolleg)
+Wolleg#NA)
@@ -1555 +1555 @@
-parres)))))
+parres#parish_n)))))
@@ -1560 +1560 @@
-thes)))
+thes#these_pron)))
