--- roxinden-1626-e2-p1.psd.vert	2025-09-10 13:51:11
+++ roxinden-1626-e2-p1.lem.vert	2025-09-10 13:50:58
@@ -11 +11 @@
-Cossen)
+Cossen#cousin_n)
@@ -13 +13 @@
-Pettit))
+Pettit#NA))
@@ -18 +18 @@
-I))
+I#i_pron))
@@ -20 +20 @@
-received)
+received#receive_v)
@@ -23 +23 @@
-your)
+your#your_pron)
@@ -25 +25 @@
-letter)))
+letter#letter_n1)))
@@ -31 +31 @@
-and)
+and#and_conj1)
@@ -34 +34 @@
-my)
+my#my_adj)
@@ -36 +36 @@
-sister))
+sister#sister_n))
@@ -38 +38 @@
-hath)
+hath#have_v)
@@ -40 +40 @@
-received)
+received#receive_v)
@@ -43 +43 @@
-the)
+the#the_adj)
@@ -45 +45 @@
-lobsters)
+lobsters#lobster_n1)
@@ -56 +56 @@
-you))
+you#you_pron))
@@ -58 +58 @@
-sent)
+sent#send_v1)
@@ -61 +61 @@
-her))))
+her#her_pron1))))
@@ -67 +67 @@
-for)
+for#for_prep)
@@ -70 +70 @@
-which)))
+which#which_adj)))
@@ -76 +76 @@
-shee))
+shee#she_pron))
@@ -78 +78 @@
-gives)
+gives#give_v)
@@ -81 +81 @@
-you))
+you#you_pron))
@@ -84 +84 @@
-many)
+many#many_adj)
@@ -86 +86 @@
-thankes)
+thankes#thank_n)
@@ -96 +96 @@
-and)
+and#and_conj1)
@@ -99 +99 @@
-we))
+we#we_pron))
@@ -101 +101 @@
-both)
+both#both_pron)
@@ -103 +103 @@
-desire)
+desire#desire_v)
@@ -106 +106 @@
-that)
+that#that_conj)
@@ -110 +110 @@
-you))
+you#you_pron))
@@ -112 +112 @@
-would)
+would#will_v1)
@@ -114 +114 @@
-excuse)
+excuse#excuse_v)
@@ -117 +117 @@
-our)
+our#our_pron)
@@ -119 +119 @@
-bouldnes)
+bouldnes#boldness_n)
@@ -122 +122 @@
-in)
+in#in_prep)
@@ -125 +125 @@
-troubleing)
+troubleing#trouble_v)
@@ -128 +128 @@
-you))
+you#you_pron))
@@ -131 +131 @@
-in)
+in#in_prep)
@@ -134 +134 @@
-such)
+such#such_adj)
@@ -136 +136 @@
-bussines))))))))
+bussines#business_n))))))))
@@ -145 +145 @@
-Our)
+Our#our_pron)
@@ -147 +147 @@
-desire))
+desire#desire_n))
@@ -149 +149 @@
-was)
+was#be_v)
@@ -152 +152 @@
-when)
+when#when_adv)
@@ -159 +159 @@
-we))
+we#we_pron))
@@ -161 +161 @@
-sent))))
+sent#send_v1))))
@@ -169 +169 @@
-Since)
+Since#since_adv)
@@ -176 +176 @@
-you))
+you#you_pron))
@@ -178 +178 @@
-write)
+write#write_v)
@@ -181 +181 @@
-that)
+that#that_conj)
@@ -185 +185 @@
-you))
+you#you_pron))
@@ -187 +187 @@
-desire)
+desire#desire_v)
@@ -190 +190 @@
-to)
+to#to_prep)
@@ -192 +192 @@
-know)
+know#know_v)
@@ -195 +195 @@
-whether)
+whether#whether_pron)
@@ -201 +201 @@
-the)
+the#the_adj)
@@ -203 +203 @@
-Sprackling)
+Sprackling#NA)
@@ -205 +205 @@
-armes))
+armes#arms_n))
@@ -207 +207 @@
-should)
+should#shall_v)
@@ -209 +209 @@
-be)
+be#be_v)
@@ -211 +211 @@
-ioyned)
+ioyned#join_v1)
@@ -214 +214 @@
-with)
+with#with_prep)
@@ -218 +218 @@
-your)
+your#your_pron)
@@ -220 +220 @@
-sonnes))))
+sonnes#son_n1))))
@@ -224 +224 @@
-which))
+which#which_adj))
@@ -232 +232 @@
-my)
+my#my_adj)
@@ -234 +234 @@
-sister))
+sister#sister_n))
@@ -236 +236 @@
-refereth)
+refereth#refer_v)
@@ -238 +238 @@
-both)
+both#both_pron)
@@ -241 +241 @@
-to)
+to#to_prep)
@@ -244 +244 @@
-you)))
+you#you_pron)))
@@ -249 +249 @@
-like)
+like#like_adj)
@@ -251 +251 @@
-as)
+as#as_adv)
@@ -253 +253 @@
-also)
+also#also_adv)
@@ -257 +257 @@
-what)
+what#what_pron)
@@ -259 +259 @@
-inscription)))))))))))))
+inscription#inscription_n)))))))))))))
