--- southard-1686-e3-p1.psd.vert	2025-09-10 13:51:11
+++ southard-1686-e3-p1.lem.vert	2025-09-10 13:50:58
@@ -14 +14 @@
-For)
+For#for_prep)
@@ -17 +17 @@
-Mr)
+Mr#NA)
@@ -19 +19 @@
-John)
+John#NA)
@@ -21 +21 @@
-Pinney)))
+Pinney#NA)))
@@ -24 +24 @@
-at)
+at#at_prep)
@@ -27 +27 @@
-his)
+his#his_adj)
@@ -29 +29 @@
-Logens)
+Logens#lodging_n)
@@ -32 +32 @@
-in)
+in#in_prep)
@@ -35 +35 @@
-great)
+great#NA)
@@ -37 +37 @@
-Shep)
+Shep#NA)
@@ -39 +39 @@
-Stret)))
+Stret#NA)))
@@ -42 +42 @@
-in)
+in#in_prep)
@@ -45 +45 @@
-Dobelin)))
+Dobelin#NA)))
@@ -48 +48 @@
-in)
+in#in_prep)
@@ -51 +51 @@
-Irland))))))
+Irland#NA))))))
@@ -60 +60 @@
-June)
+June#NA)
@@ -62 +62 @@
-22=d=)
+22=d=#NA)
@@ -64 +64 @@
-1686))
+1686#NA))
@@ -71 +71 @@
-Sarah))
+Sarah#NA))
@@ -74 +74 @@
-&)
+&#and_conj1)
@@ -77 +77 @@
-her)
+her#her_pron2)
@@ -79 +79 @@
-mother)))
+mother#mother_n1)))
@@ -89 +89 @@
-Dare)
+Dare#dear_adj1)
@@ -91 +91 @@
-&)
+&#and_conj1)
@@ -93 +93 @@
-onred))
+onred#honoured_adj))
@@ -95 +95 @@
-father))
+father#father_n))
@@ -98 +98 @@
-youers))
+youers#yours_pron))
@@ -101 +101 @@
-I))
+I#i_pron))
@@ -103 +103 @@
-reseved)
+reseved#receive_v)
@@ -107 +107 @@
-for)
+for#for_prep)
@@ -110 +110 @@
-which)))
+which#which_adj)))
@@ -118 +118 @@
-I))
+I#i_pron))
@@ -120 +120 @@
-give)
+give#give_v)
@@ -125 +125 @@
-you))
+you#you_pron))
@@ -129 +129 @@
-hombell)
+hombell#humble_adj)
@@ -131 +131 @@
-&)
+&#and_conj1)
@@ -133 +133 @@
-harty))
+harty#hearty_adj))
@@ -135 +135 @@
-thanks)
+thanks#thank_n)
@@ -139 +139 @@
-that)
+that#that_conj)
@@ -143 +143 @@
-you))
+you#you_pron))
@@ -145 +145 @@
-ar)
+ar#be_v)
@@ -147 +147 @@
-plesed)
+plesed#please_v)
@@ -150 +150 @@
-to)
+to#to_prep)
@@ -152 +152 @@
-writ)
+writ#write_v)
@@ -155 +155 @@
-one)
+one#one_adj)
@@ -157 +157 @@
-woord)
+woord#word_n)
@@ -162 +162 @@
-of)
+of#of_prep)
@@ -165 +165 @@
-comfard)))))))
+comfard#comfort_n)))))))
@@ -168 +168 @@
-&)
+&#and_conj1)
@@ -171 +171 @@
-that)
+that#that_conj)
@@ -175 +175 @@
-you))
+you#you_pron))
@@ -177 +177 @@
-ar)
+ar#be_v)
@@ -180 +180 @@
-willing)
+willing#willing_adj)
@@ -183 +183 @@
-to)
+to#to_prep)
@@ -185 +185 @@
-parmet)
+parmet#permit_v)
@@ -188 +188 @@
-me))
+me#me_pron1))
@@ -191 +191 @@
-once)
+once#once_adv)
@@ -194 +194 @@
-more)))
+more#more_adj)))
@@ -197 +197 @@
-in)
+in#in_adv)
@@ -199 +199 @@
-to)
+to#to_prep)
@@ -202 +202 @@
-youer)
+youer#your_pron)
@@ -204 +204 @@
-presenc)))
+presenc#presence_n)))
@@ -207 +207 @@
-after)
+after#after_adv)
@@ -211 +211 @@
-So)
+So#so_adv)
@@ -213 +213 @@
-gret))
+gret#great_adj))
@@ -215 +215 @@
-afence)
+afence#offence_n)
@@ -218 +218 @@
-commeted)))))))))
+commeted#commit_v)))))))))
@@ -221 +221 @@
-the)
+the#the_adj)
@@ -223 +223 @@
-only)
+only#only_adj)
@@ -225 +225 @@
-thin)
+thin#thin_adj)
@@ -237 +237 @@
-I))
+I#i_pron))
@@ -239 +239 @@
-desired))
+desired#desire_v))
@@ -242 +242 @@
-&)
+&#and_conj1)
@@ -247 +247 @@
-have)
+have#have_v)
@@ -249 +249 @@
-Long)
+Long#long_v1)
@@ -252 +252 @@
-for)
+for#for_prep)
@@ -258 +258 @@
-cold)
+cold#can_v1)
@@ -261 +261 @@
-I))
+I#i_pron))
@@ -263 +263 @@
-have)
+have#have_v)
@@ -265 +265 @@
-bin)
+bin#be_v)
@@ -267 +267 @@
-permeted)
+permeted#permit_v)
@@ -270 +270 @@
-youer)
+youer#your_pron)
@@ -272 +272 @@
-writings)
+writings#writing_n)
@@ -283 +283 @@
-my)
+my#my_adj)
@@ -285 +285 @@
-mother))
+mother#mother_n1))
@@ -287 +287 @@
-have))))))))))))))
+have#have_v))))))))))))))
@@ -293 +293 @@
-&)
+&#and_conj1)
@@ -296 +296 @@
-as)
+as#as_adv)
@@ -299 +299 @@
-for)
+for#for_prep)
@@ -302 +302 @@
-the)
+the#the_adj)
@@ -304 +304 @@
-writen)
+writen#writing_n)
@@ -307 +307 @@
-of)
+of#of_prep)
@@ -310 +310 @@
-bettescom))))))
+bettescom#NA))))))
@@ -313 +313 @@
-I))
+I#i_pron))
@@ -315 +315 @@
-gave)
+gave#give_v)
@@ -318 +318 @@
-you))
+you#you_pron))
@@ -321 +321 @@
-acount)
+acount#account_n)
@@ -326 +326 @@
-hou))
+hou#how_adv))
@@ -334 +334 @@
-it))
+it#it_pron))
@@ -336 +336 @@
-was)
+was#be_v)
@@ -338 +338 @@
-don)))
+don#do_v)))
@@ -341 +341 @@
-&)
+&#and_conj1)
@@ -345 +345 @@
-what))
+what#what_pron))
@@ -353 +353 @@
-it))
+it#it_pron))
@@ -355 +355 @@
-cost)
+cost#cost_v)
@@ -358 +358 @@
-whin)
+whin#when_adv)
@@ -365 +365 @@
-it))
+it#it_pron))
@@ -367 +367 @@
-wos)
+wos#be_v)
@@ -369 +369 @@
-done))))))))))
+done#do_v))))))))))
@@ -375 +375 @@
-&)
+&#and_conj1)
@@ -378 +378 @@
-my)
+my#my_adj)
@@ -380 +380 @@
-brother)
+brother#brother_n)
@@ -383 +383 @@
-nat)))
+nat#NA)))
@@ -385 +385 @@
-nu)
+nu#know_v)
@@ -388 +388 @@
-all))
+all#all_adj))
@@ -391 +391 @@
-of)
+of#of_prep)
@@ -394 +394 @@
-it)))
+it#it_pron)))
@@ -397 +397 @@
-before)
+before#before_adv)
@@ -404 +404 @@
-thare))
+thare#there_adv))
@@ -406 +406 @@
-woose)
+woose#be_v)
@@ -409 +409 @@
-any)
+any#any_adj)
@@ -411 +411 @@
-thin))
+thin#thing_n1))
@@ -413 +413 @@
-don)
+don#do_v)
@@ -416 +416 @@
-in)
+in#in_prep)
@@ -419 +419 @@
-it)))))))
+it#it_pron)))))))
@@ -425 +425 @@
-&)
+&#and_conj1)
@@ -429 +429 @@
-consented)
+consented#consent_v)
@@ -432 +432 @@
-to)
+to#to_prep)
@@ -435 +435 @@
-it))))
+it#it_pron))))
@@ -441 +441 @@
-&)
+&#and_conj1)
@@ -444 +444 @@
-as)
+as#as_adv)
@@ -447 +447 @@
-for)
+for#for_prep)
@@ -450 +450 @@
-youer)
+youer#your_pron)
@@ -452 +452 @@
-wooch))))
+wooch#wish_n1))))
@@ -455 +455 @@
-my)
+my#my_adj)
@@ -457 +457 @@
-brother)
+brother#brother_n)
@@ -460 +460 @@
-John)))
+John#NA)))
@@ -462 +462 @@
-ose)
+ose#owe_v)
@@ -465 +465 @@
-sister)
+sister#NA)
@@ -467 +467 @@
-Rachell))
+Rachell#NA))
@@ -470 +470 @@
-for)
+for#for_prep)
@@ -473 +473 @@
-one)
+one#one_adj)
@@ -477 +477 @@
-which))
+which#which_adj))
@@ -485 +485 @@
-he))
+he#he_pron))
@@ -487 +487 @@
-gave)
+gave#give_v)
@@ -490 +490 @@
-to)
+to#to_prep)
@@ -493 +493 @@
-his)
+his#his_adj)
@@ -495 +495 @@
-mestres))))))))
+mestres#mistress_n))))))))
@@ -501 +501 @@
-&)
+&#and_conj1)
@@ -505 +505 @@
-ose)
+ose#owe_v)
@@ -508 +508 @@
-me))
+me#me_pron1))
@@ -511 +511 @@
-for)
+for#for_prep)
@@ -514 +514 @@
-this)
+this#this_pron)
@@ -518 +518 @@
-which))
+which#which_adj))
@@ -524 +524 @@
-brother)
+brother#brother_n)
@@ -527 +527 @@
-Clarke)))
+Clarke#NA)))
@@ -529 +529 @@
-is)
+is#be_v)
@@ -532 +532 @@
-witness)
+witness#witness_n)
@@ -535 +535 @@
-to)
+to#to_prep)
@@ -543 +543 @@
-for)
+for#for_prep)
@@ -546 +546 @@
-he))
+he#he_pron))
@@ -548 +548 @@
-boft)
+boft#boff_v)
@@ -551 +551 @@
-him)))
+him#him_pron)))
@@ -557 +557 @@
-&)
+&#and_conj1)
@@ -560 +560 @@
-I))
+I#i_pron))
@@ -562 +562 @@
-paid)
+paid#pay_v1)
@@ -565 +565 @@
-him))
+him#him_pron))
@@ -568 +568 @@
-for)
+for#for_prep)
@@ -571 +571 @@
-it))))
+it#it_pron))))
@@ -577 +577 @@
-&)
+&#and_conj1)
@@ -582 +582 @@
-never))
+never#never_adv))
@@ -584 +584 @@
-had)
+had#have_v)
@@ -586 +586 @@
-afarden)
+afarden#offer_v)
@@ -591 +591 @@
-for)
+for#for_prep)
@@ -594 +594 @@
-it)))
+it#it_pron)))
@@ -597 +597 @@
-againe))
+againe#again_adv))
@@ -600 +600 @@
-so)
+so#so_adv)
@@ -603 +603 @@
-that)
+that#that_conj)
@@ -607 +607 @@
-if)
+if#if_conj)
@@ -614 +614 @@
-you))
+you#you_pron))
@@ -616 +616 @@
-ar)
+ar#be_v)
@@ -618 +618 @@
-plesed)
+plesed#please_v)
@@ -621 +621 @@
-to)
+to#to_prep)
@@ -623 +623 @@
-give)
+give#give_v)
@@ -626 +626 @@
-him))
+him#him_pron))
@@ -629 +629 @@
-in)
+in#in_prep)
@@ -632 +632 @@
-gostes)
+gostes#ghost_n)
@@ -636 +636 @@
-&)
+&#and_conj1)
@@ -638 +638 @@
-oneste)))))))
+oneste#honesty_n)))))))
@@ -641 +641 @@
-it))
+it#it_pron))
@@ -643 +643 @@
-belong)
+belong#belong_v)
@@ -646 +646 @@
-to)
+to#to_prep)
@@ -649 +649 @@
-me)))))))
+me#me_pron1)))))))
@@ -655 +655 @@
-&)
+&#and_conj1)
@@ -658 +658 @@
-I))
+I#i_pron))
@@ -660 +660 @@
-hope)
+hope#hope_v)
@@ -667 +667 @@
-thofe)
+thofe#though_adv)
@@ -674 +674 @@
-I))
+I#i_pron))
@@ -676 +676 @@
-have)
+have#have_v)
@@ -678 +678 @@
-done)
+done#do_v)
@@ -681 +681 @@
-evely))
+evely#evilly_adv))
@@ -684 +684 @@
-in)
+in#in_prep)
@@ -687 +687 @@
-one)
+one#one_adj)
@@ -689 +689 @@
-thine))))))
+thine#thin_adj))))))
@@ -692 +692 @@
-you))
+you#you_pron))
@@ -694 +694 @@
-will)
+will#will_v1)
@@ -696 +696 @@
-not)
+not#not_adv)
@@ -698 +698 @@
-excemt)
+excemt#exempt_v)
@@ -701 +701 @@
-me))
+me#me_pron1))
@@ -704 +704 @@
-fro~)
+fro~#from_prep)
@@ -707 +707 @@
-bing)
+bing#be_v)
@@ -710 +710 @@
-a@)
+a@#a_adj2)
@@ -712 +712 @@
-@child))
+@child#child_n))
@@ -715 +715 @@
-as)
+as#as_adv)
@@ -717 +717 @@
-wille)
+wille#well_adv)
@@ -720 +720 @@
-as)
+as#as_adv)
@@ -723 +723 @@
-my)
+my#my_adj)
@@ -725 +725 @@
-brother)
+brother#brother_n)
@@ -729 +729 @@
-ho))
+ho#who_pron))
@@ -738 +738 @@
-never))
+never#never_adv))
@@ -740 +740 @@
-ofended)
+ofended#offend_v)
@@ -743 +743 @@
-you))
+you#you_pron))
@@ -746 +746 @@
-before))
+before#before_adv))
@@ -749 +749 @@
-to)
+to#to_prep)
@@ -752 +752 @@
-my)
+my#my_adj)
@@ -754 +754 @@
-noleg))))
+noleg#knowledge_n))))
@@ -757 +757 @@
-nor)
+nor#nor_conj1)
@@ -762 +762 @@
-wronged)
+wronged#wrong_v)
@@ -765 +765 @@
-you))
+you#you_pron))
@@ -768 +768 @@
-of)
+of#of_prep)
@@ -771 +771 @@
-one)
+one#one_adj)
@@ -773 +773 @@
-farden)
+farden#farthing_n)
@@ -778 +778 @@
-that)
+that#that_conj)
@@ -782 +782 @@
-I))
+I#i_pron))
@@ -784 +784 @@
-no)
+no#know_v)
@@ -787 +787 @@
-of)
+of#of_prep)
@@ -796 +796 @@
-thofe)
+thofe#though_adv)
@@ -803 +803 @@
-I))
+I#i_pron))
@@ -805 +805 @@
-have)
+have#have_v)
@@ -807 +807 @@
-bine)
+bine#be_v)
@@ -810 +810 @@
-moch))
+moch#much_adj))
@@ -813 +813 @@
-wrangd)
+wrangd#wrong_v)
@@ -815 +815 @@
-&)
+&#and_conj1)
@@ -817 +817 @@
-aboused)))))
+aboused#abuse_v)))))
@@ -820 +820 @@
-you))
+you#you_pron))
@@ -822 +822 @@
-will)
+will#will_v1)
@@ -824 +824 @@
-no)
+no#know_v)
@@ -831 +831 @@
-I))
+I#i_pron))
@@ -833 +833 @@
-woos)
+woos#be_v)
@@ -836 +836 @@
-never))
+never#never_adv))
@@ -839 +839 @@
-woos)
+woos#worse_adj)
@@ -842 +842 @@
-thin)
+thin#than_conj)
@@ -845 +845 @@
-my)
+my#my_adj)
@@ -847 +847 @@
-woord)))
+woord#word_n)))
@@ -850 +850 @@
-to)
+to#to_prep)
@@ -853 +853 @@
-any)
+any#any_adj)
@@ -855 +855 @@
-one))))
+one#one_adj))))
@@ -858 +858 @@
-in)
+in#in_prep)
@@ -861 +861 @@
-my)
+my#my_adj)
@@ -863 +863 @@
-Life)))))
+Life#life_n)))))
@@ -868 +868 @@
-moch)
+moch#much_adj)
@@ -870 +870 @@
-more)))
+more#more_adj)))
@@ -872 +872 @@
-woold)
+woold#will_v1)
@@ -875 +875 @@
-i))
+i#i_pron))
@@ -877 +877 @@
-not)
+not#not_adv)
@@ -882 +882 @@
-to)
+to#to_prep)
@@ -885 +885 @@
-you))))
+you#you_pron))))
@@ -888 +888 @@
-hoping)
+hoping#hope_v)
@@ -895 +895 @@
-I))
+I#i_pron))
@@ -897 +897 @@
-may)
+may#may_v1)
@@ -899 +899 @@
-find)
+find#find_v)
@@ -902 +902 @@
-others))
+others#other_adj))
@@ -905 +905 @@
-so))
+so#so_adv))
@@ -908 +908 @@
-to)
+to#to_prep)
@@ -911 +911 @@
-me)))
+me#me_pron1)))
@@ -914 +914 @@
-thof)
+thof#though_adv)
@@ -921 +921 @@
-I))
+I#i_pron))
@@ -923 +923 @@
-have)
+have#have_v)
@@ -926 +926 @@
-Letell)
+Letell#little_adj)
@@ -928 +928 @@
-Reson)
+Reson#reason_n1)
@@ -931 +931 @@
-to)
+to#to_prep)
@@ -933 +933 @@
-bost)
+bost#boast_v1)
@@ -936 +936 @@
-of)
+of#of_prep)
@@ -939 +939 @@
-it)))
+it#it_pron)))
@@ -942 +942 @@
-yat)))))))))))
+yat#yet_adv)))))))))))
@@ -948 +948 @@
-for)
+for#for_prep)
@@ -951 +951 @@
-I))
+I#i_pron))
@@ -953 +953 @@
-find)
+find#find_v)
@@ -960 +960 @@
-all))
+all#all_adj))
@@ -962 +962 @@
-ar)
+ar#be_v)
@@ -965 +965 @@
-against)
+against#against_prep)
@@ -968 +968 @@
-me)))
+me#me_pron1)))
@@ -971 +971 @@
-thofe)
+thofe#though_adv)
@@ -979 +979 @@
-I))
+I#i_pron))
@@ -981 +981 @@
-have)
+have#have_v)
@@ -985 +985 @@
-boren)
+boren#bear_v1)
@@ -988 +988 @@
-the)
+the#the_adj)
@@ -990 +990 @@
-borden)
+borden#burden_n)
@@ -993 +993 @@
-&)
+&#and_conj1)
@@ -996 +996 @@
-het)))
+het#heat_n)))
@@ -999 +999 @@
-of)
+of#of_prep)
@@ -1002 +1002 @@
-the)
+the#the_adj)
@@ -1004 +1004 @@
-day))))
+day#day_n))))
@@ -1007 +1007 @@
-for)
+for#for_prep)
@@ -1010 +1010 @@
-all))))
+all#all_adj))))
@@ -1013 +1013 @@
-&)
+&#and_conj1)
@@ -1016 +1016 @@
-ventred)
+ventred#venture_v)
@@ -1019 +1019 @@
-my)
+my#my_adj)
@@ -1021 +1021 @@
-one)
+one#one_adj)
@@ -1023 +1023 @@
-Life))
+Life#life_n))
@@ -1026 +1026 @@
-to)
+to#to_prep)
@@ -1028 +1028 @@
-save)
+save#save_v)
@@ -1031 +1031 @@
-others)))))))))))
+others#other_adj)))))))))))
@@ -1038 +1038 @@
-yot))
+yot#though_adv))
@@ -1041 +1041 @@
-now))
+now#now_adv))
@@ -1044 +1044 @@
-it))
+it#it_pron))
@@ -1046 +1046 @@
-tis)
+tis#be_v)
@@ -1049 +1049 @@
-Letell))
+Letell#little_adj))
@@ -1051 +1051 @@
-Consedred))
+Consedred#consider_v))
@@ -1058 +1058 @@
-I))
+I#i_pron))
@@ -1060 +1060 @@
-have)
+have#have_v)
@@ -1062 +1062 @@
-none)
+none#know_v)
@@ -1065 +1065 @@
-Letell)
+Letell#little_adj)
@@ -1068 +1068 @@
-elce))
+elce#else_adv))
@@ -1071 +1071 @@
-bot)
+bot#but_prep)
@@ -1074 +1074 @@
-trobell)))))
+trobell#trouble_n)))))
@@ -1081 +1081 @@
-the)
+the#the_adj)
@@ -1083 +1083 @@
-Lord))
+Lord#NA))
@@ -1085 +1085 @@
-give)
+give#give_v)
@@ -1088 +1088 @@
-me))
+me#me_pron1))
@@ -1091 +1091 @@
-sofren)
+sofren#sovereign_n)
@@ -1093 +1093 @@
-graces))
+graces#grace_n))
@@ -1096 +1096 @@
-that)
+that#that_conj)
@@ -1101 +1101 @@
-I))
+I#i_pron))
@@ -1103 +1103 @@
-may)
+may#may_v1)
@@ -1105 +1105 @@
-bare)
+bare#bear_v1)
@@ -1107 +1107 @@
-up))
+up#up_adv1))
@@ -1110 +1110 @@
-&)
+&#and_conj1)
@@ -1113 +1113 @@
-bare)
+bare#bear_v1)
@@ -1115 +1115 @@
-out)
+out#out_adv)
@@ -1118 +1118 @@
-to)
+to#to_prep)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-end))))))))
+end#end_n))))))))
@@ -1130 +1130 @@
-it))
+it#it_pron))
@@ -1132 +1132 @@
-may)
+may#may_v1)
@@ -1134 +1134 @@
-bring)
+bring#bring_v)
@@ -1137 +1137 @@
-me))
+me#me_pron1))
@@ -1140 +1140 @@
-the)
+the#the_adj)
@@ -1142 +1142 @@
-narer)
+narer#near_adj)
@@ -1145 +1145 @@
-to)
+to#to_prep)
@@ -1148 +1148 @@
-god)))))
+god#NA)))))
@@ -1154 +1154 @@
-&)
+&#and_conj1)
@@ -1157 +1157 @@
-thin))
+thin#then_adv))
@@ -1160 +1160 @@
-it_t@))
+it_t@#it_pron))
@@ -1162 +1162 @@
-@will)
+@will#will_v1)
@@ -1164 +1164 @@
-be)
+be#be_v)
@@ -1167 +1167 @@
-will))
+will#well_adj))
@@ -1170 +1170 @@
-with)
+with#with_prep)
@@ -1173 +1173 @@
-me)))
+me#me_pron1)))
@@ -1176 +1176 @@
-thof)
+thof#though_adv)
@@ -1183 +1183 @@
-I))
+I#i_pron))
@@ -1185 +1185 @@
-am)
+am#be_v)
@@ -1187 +1187 @@
-wich)
+wich#NA)
@@ -1190 +1190 @@
-never))
+never#never_adv))
@@ -1193 +1193 @@
-so))
+so#so_adv))
@@ -1196 +1196 @@
-yett))))))
+yett#yet_adv))))))
@@ -1207 +1207 @@
-the)
+the#the_adj)
@@ -1211 +1211 @@
-Lord))
+Lord#NA))
@@ -1214 +1214 @@
-bless)
+bless#bless_v1)
@@ -1216 +1216 @@
-&)
+&#and_conj1)
@@ -1218 +1218 @@
-presarve)
+presarve#preserve_v)
@@ -1220 +1220 @@
-&)
+&#and_conj1)
@@ -1222 +1222 @@
-cape))
+cape#keep_v))
@@ -1225 +1225 @@
-you)))
+you#you_pron)))
@@ -1228 +1228 @@
-&)
+&#and_conj1)
@@ -1233 +1233 @@
-give)
+give#give_v)
@@ -1236 +1236 @@
-you))
+you#you_pron))
@@ -1239 +1239 @@
-the)
+the#the_adj)
@@ -1241 +1241 @@
-oyle)
+oyle#oil_n1)
@@ -1244 +1244 @@
-of)
+of#of_prep)
@@ -1247 +1247 @@
-goy))))
+goy#joy_n))))
@@ -1250 +1250 @@
-for)
+for#for_prep)
@@ -1253 +1253 @@
-the)
+the#the_adj)
@@ -1255 +1255 @@
-speret)
+speret#spirit_n)
@@ -1258 +1258 @@
-of)
+of#of_prep)
@@ -1261 +1261 @@
-heveness))))))))))
+heveness#heaviness_n))))))))))
@@ -1263 +1263 @@
-is)
+is#be_v)
@@ -1266 +1266 @@
-the)
+the#the_adj)
@@ -1268 +1268 @@
-daile)
+daile#daily_adj)
@@ -1270 +1270 @@
-prayers)
+prayers#prayer_n1)
@@ -1273 +1273 @@
-of)
+of#of_prep)
@@ -1276 +1276 @@
-youer)
+youer#your_pron)
@@ -1278 +1278 @@
-affected)
+affected#affected_adj.III)
@@ -1280 +1280 @@
-pasecated)
+pasecated#NA)
@@ -1284 +1284 @@
-hart_broken)
+hart_broken#heartbroken_adj)
@@ -1286 +1286 @@
-degected)
+degected#dejected_adj)
@@ -1288 +1288 @@
-obedent)
+obedent#obedient_adj)
@@ -1290 +1290 @@
-faithfull)
+faithfull#faithful_adj)
@@ -1292 +1292 @@
-dutifull)
+dutifull#dutiful_adj)
@@ -1294 +1294 @@
-dafter)
+dafter#daughter_n)
@@ -1297 +1297 @@
-Sarah)
+Sarah#NA)
@@ -1299 +1299 @@
-Southard))))))
+Southard#NA))))))
