--- tunstall-1519-e1-p1.psd.vert	2025-09-10 13:51:11
+++ tunstall-1519-e1-p1.lem.vert	2025-09-10 13:50:58
@@ -19 +19 @@
-Please)
+Please#please_v)
@@ -22 +22 @@
-it))
+it#it_pron))
@@ -25 +25 @@
-your)
+your#your_pron)
@@ -27 +27 @@
-Grace))
+Grace#grace_n))
@@ -30 +30 @@
-to)
+to#to_prep)
@@ -32 +32 @@
-understand)
+understand#understand_v)
@@ -35 +35 @@
-that)
+that#that_conj)
@@ -39 +39 @@
-here))
+here#here_adv))
@@ -42 +42 @@
-in)
+in#in_prep)
@@ -45 +45 @@
-these)
+these#these_pron)
@@ -47 +47 @@
-parties)))
+parties#party_n)))
@@ -50 +50 @@
-I))
+I#i_pron))
@@ -52 +52 @@
-met)
+met#meet_v)
@@ -55 +55 @@
-with)
+with#with_prep)
@@ -58 +58 @@
-a)
+a#a_adj2)
@@ -60 +60 @@
-servant)
+servant#servant_n)
@@ -63 +63 @@
-of)
+of#of_prep)
@@ -67 +67 @@
-the)
+the#the_adj)
@@ -69 +69 @@
-Kings))))
+Kings#king_n))))
@@ -72 +72 @@
-called)
+called#call_v)
@@ -78 +78 @@
-Nicholas)
+Nicholas#NA)
@@ -80 +80 @@
-Cratzer))))
+Cratzer#NA))))
@@ -83 +83 @@
-an)
+an#a_adj2)
@@ -85 +85 @@
-Almayne))
+Almayne#NA))
@@ -90 +90 @@
-deviser)
+deviser#deviser_n)
@@ -93 +93 @@
-of)
+of#of_prep)
@@ -97 +97 @@
-the)
+the#the_adj)
@@ -99 +99 @@
-Kings))
+Kings#king_n))
@@ -101 +101 @@
-Horologies))))
+Horologies#horology_n1))))
@@ -107 +107 @@
-whiche))
+whiche#which_adj))
@@ -114 +114 @@
-shewed)
+shewed#show_v)
@@ -117 +117 @@
-me))
+me#me_pron1))
@@ -122 +122 @@
-howe))
+howe#how_adv))
@@ -130 +130 @@
-the)
+the#the_adj)
@@ -132 +132 @@
-King))
+King#king_n))
@@ -134 +134 @@
-had)
+had#have_v)
@@ -136 +136 @@
-licenced)
+licenced#license_v)
@@ -140 +140 @@
-hym))
+hym#him_pron))
@@ -142 +142 @@
-to)
+to#to_prep)
@@ -144 +144 @@
-be)
+be#be_v)
@@ -147 +147 @@
-absent)
+absent#absent_adj)
@@ -150 +150 @@
-for)
+for#for_prep)
@@ -153 +153 @@
-a)
+a#a_adj2)
@@ -155 +155 @@
-season)))))))
+season#season_n)))))))
@@ -160 +160 @@
-and)
+and#and_conj1)
@@ -163 +163 @@
-that)
+that#that_conj)
@@ -167 +167 @@
-he))
+he#he_pron))
@@ -169 +169 @@
-was)
+was#be_v)
@@ -172 +172 @@
-redy)
+redy#ready_adj)
@@ -175 +175 @@
-to)
+to#to_prep)
@@ -177 +177 @@
-retorne)
+retorne#return_v1)
@@ -180 +180 @@
-in)
+in#in_adv)
@@ -182 +182 @@
-to)
+to#to_prep)
@@ -185 +185 @@
-Englande)))))))))))
+Englande#NA)))))))))))
@@ -191 +191 @@
-Whome))
+Whome#whom_pron))
@@ -199 +199 @@
-I))
+I#i_pron))
@@ -201 +201 @@
-desired)
+desired#desire_v)
@@ -204 +204 @@
-to)
+to#to_prep)
@@ -206 +206 @@
-tary)
+tary#tarry_v)
@@ -209 +209 @@
-unto)
+unto#unto_prep)
@@ -216 +216 @@
-I))
+I#i_pron))
@@ -218 +218 @@
-might)
+might#may_v1)
@@ -220 +220 @@
-write)
+write#write_v)
@@ -223 +223 @@
-unto)
+unto#unto_prep)
@@ -227 +227 @@
-the)
+the#the_adj)
@@ -229 +229 @@
-Kings))
+Kings#king_n))
@@ -231 +231 @@
-Highnes)))
+Highnes#highness_n)))
@@ -234 +234 @@
-to)
+to#to_prep)
@@ -236 +236 @@
-knowe)
+knowe#know_v)
@@ -239 +239 @@
-his)
+his#his_adj)
@@ -241 +241 @@
-pleas=r=)
+pleas=r=#pleasure_n)
@@ -244 +244 @@
-whether)
+whether#whether_pron)
@@ -250 +250 @@
-he))
+he#he_pron))
@@ -252 +252 @@
-wolde)
+wolde#will_v1)
@@ -254 +254 @@
-suffer)
+suffer#suffer_v)
@@ -258 +258 @@
-hym))
+hym#him_pron))
@@ -260 +260 @@
-to)
+to#to_prep)
@@ -262 +262 @@
-be)
+be#be_v)
@@ -265 +265 @@
-in)
+in#in_prep)
@@ -268 +268 @@
-company)))
+company#company_n)))
@@ -271 +271 @@
-with)
+with#with_prep)
@@ -274 +274 @@
-me)))
+me#me_pron1)))
@@ -277 +277 @@
-for)
+for#for_prep)
@@ -280 +280 @@
-a)
+a#a_adj2)
@@ -282 +282 @@
-season)))
+season#season_n)))
@@ -287 +287 @@
-unto)
+unto#unto_prep)
@@ -294 +294 @@
-the)
+the#the_adj)
@@ -296 +296 @@
-Assemble)
+Assemble#assembly_n)
@@ -299 +299 @@
-of)
+of#of_prep)
@@ -302 +302 @@
-the)
+the#the_adj)
@@ -304 +304 @@
-Electors)))
+Electors#elector_n)))
@@ -308 +308 @@
-were)
+were#be_v)
@@ -310 +310 @@
-past)
+past#pass_v)
@@ -316 +316 @@
-with)
+with#with_prep)
@@ -319 +319 @@
-whom)))
+whom#whom_pron)))
@@ -325 +325 @@
-he))
+he#he_pron))
@@ -327 +327 @@
-hath)
+hath#have_v)
@@ -332 +332 @@
-as)
+as#as_adv)
@@ -343 +343 @@
-he))
+he#he_pron))
@@ -345 +345 @@
-sayeth))))
+sayeth#say_v1))))
@@ -350 +350 @@
-acqueyntance)
+acqueyntance#acquaintance_n)
@@ -353 +353 @@
-of)
+of#of_prep)
@@ -356,3 +356,3 @@
-many)
-(ADJ
-noble)
+many#many_adj)
+(ADJ
+noble#noble_adj)
@@ -360 +360 @@
-men)
+men#man_n1)
@@ -363 +363 @@
-being)
+being#be_v)
@@ -366 +366 @@
-aboute)
+aboute#about_adv)
@@ -369 +369 @@
-theym))))))
+theym#them_pron))))))
@@ -380 +380 @@
-I))
+I#i_pron))
@@ -382 +382 @@
-thinke)
+thinke#think_v2)
@@ -389 +389 @@
-if)
+if#if_conj)
@@ -397 +397 @@
-the)
+the#the_adj)
@@ -399 +399 @@
-Kings))
+Kings#king_n))
@@ -401 +401 @@
-Grace))
+Grace#grace_n))
@@ -403 +403 @@
-wolde)
+wolde#will_v1)
@@ -405 +405 @@
-com~aunde)
+com~aunde#command_v)
@@ -408 +408 @@
-hym))
+hym#him_pron))
@@ -411 +411 @@
-to)
+to#to_prep)
@@ -413 +413 @@
-tary)
+tary#tarry_v)
@@ -416 +416 @@
-for)
+for#for_prep)
@@ -419 +419 @@
-a)
+a#a_adj2)
@@ -421 +421 @@
-season)))
+season#season_n)))
@@ -424 +424 @@
-with)
+with#with_prep)
@@ -427 +427 @@
-me)))))))
+me#me_pron1)))))))
@@ -432 +432 @@
-he))
+he#he_pron))
@@ -434 +434 @@
-might)
+might#may_v1)
@@ -437 +437 @@
-at)
+at#at_prep)
@@ -440 +440 @@
-the)
+the#the_adj)
@@ -442 +442 @@
-saide)
+saide#said_adj)
@@ -444 +444 @@
-Assemble)))
+Assemble#assembly_n)))
@@ -446 +446 @@
-do)
+do#do_v)
@@ -449 +449 @@
-the)
+the#the_adj)
@@ -451 +451 @@
-King))
+King#king_n))
@@ -455 +455 @@
-as)
+as#as_adv)
@@ -457 +457 @@
-goode))
+goode#good_adj))
@@ -459 +459 @@
-service)
+service#service_n1)
@@ -462 +462 @@
-as)
+as#as_adv)
@@ -465 +465 @@
-if)
+if#if_conj)
@@ -472 +472 @@
-he))
+he#he_pron))
@@ -474 +474 @@
-were)
+were#be_v)
@@ -477 +477 @@
-in)
+in#in_prep)
@@ -480 +480 @@
-Englande)))))))
+Englande#NA)))))))
@@ -485 +485 @@
-in)
+in#in_prep)
@@ -488 +488 @@
-serching)
+serching#search_v)
@@ -492 +492 @@
-howe))
+howe#how_adv))
@@ -502 +502 @@
-the)
+the#the_adj)
@@ -504 +504 @@
-nobles)
+nobles#noble_adj)
@@ -507 +507 @@
-of)
+of#of_prep)
@@ -510 +510 @@
-Almayne))))
+Almayne#NA))))
@@ -512 +512 @@
-be)
+be#be_v)
@@ -515 +515 @@
-mynded))
+mynded#minded_adj.II-III))
@@ -518 +518 @@
-concernyng)
+concernyng#concerning_prep)
@@ -521 +521 @@
-the)
+the#the_adj)
@@ -523 +523 @@
-affaiers)
+affaiers#affair_n)
@@ -526 +526 @@
-of)
+of#of_prep)
@@ -529 +529 @@
-the)
+the#the_adj)
@@ -531 +531 @@
-Empier))))))))))))
+Empier#empire_n))))))))))))
@@ -539 +539 @@
-and)
+and#and_conj1)
@@ -542 +542 @@
-yet))
+yet#yet_adv))
@@ -545 +545 @@
-he))
+he#he_pron))
@@ -547 +547 @@
-shulde)
+shulde#shall_v)
@@ -549 +549 @@
-put)
+put#put_v)
@@ -552 +552 @@
-the)
+the#the_adj)
@@ -554 +554 @@
-King))
+King#king_n))
@@ -557 +557 @@
-to)
+to#to_prep)
@@ -560 +560 @@
-no)
+no#no_adj)
@@ -562 +562 @@
-charge)))
+charge#charge_n)))
@@ -570 +570 @@
-for)
+for#for_prep)
@@ -573 +573 @@
-he))
+he#he_pron))
@@ -575 +575 @@
-shall)
+shall#shall_v)
@@ -577 +577 @@
-have)
+have#have_v)
@@ -580 +580 @@
-mete)
+mete#meat_n)
@@ -582 +582 @@
-and)
+and#and_conj1)
@@ -584 +584 @@
-drynke))
+drynke#drink_n))
@@ -587 +587 @@
-with)
+with#with_prep)
@@ -590 +590 @@
-me)))
+me#me_pron1)))
@@ -600 +600 @@
-Wherein))
+Wherein#wherein_adv))
@@ -606 +606 @@
-I))
+I#i_pron))
@@ -608 +608 @@
-beseche)
+beseche#beseech_v)
@@ -611 +611 @@
-your)
+your#your_pron)
@@ -613 +613 @@
-Grace))
+Grace#grace_n))
@@ -616 +616 @@
-to)
+to#to_prep)
@@ -618 +618 @@
-knowe)
+knowe#know_v)
@@ -622 +622 @@
-the)
+the#the_adj)
@@ -624 +624 @@
-Kings))
+Kings#king_n))
@@ -626 +626 @@
-pleas=r=)
+pleas=r=#pleasure_n)
@@ -636 +636 @@
-for)
+for#for_prep)
@@ -639 +639 @@
-oneles@)
+oneles@#unless_adv)
@@ -646 +646 @@
-@he))
+@he#he_pron))
@@ -648 +648 @@
-have)
+have#have_v)
@@ -651 +651 @@
-com~aundement)
+com~aundement#commandment_n)
@@ -654 +654 @@
-from)
+from#from_prep)
@@ -659 +659 @@
-the)
+the#the_adj)
@@ -661 +661 @@
-Kings))
+Kings#king_n))
@@ -663 +663 @@
-Highness))
+Highness#highness_n))
@@ -668 +668 @@
-or)
+or#or_conj2)
@@ -670 +670 @@
-ells)
+ells#else_adv)
@@ -673 +673 @@
-your)
+your#your_pron)
@@ -675 +675 @@
-Grace)))))))))
+Grace#grace_n)))))))))
@@ -679 +679 @@
-neither)
+neither#neither_adv)
@@ -682 +682 @@
-he))
+he#he_pron))
@@ -684 +684 @@
-will)
+will#will_v1)
@@ -686 +686 @@
-tary))
+tary#tarry_v))
@@ -692 +692 @@
-nor)
+nor#nor_conj1)
@@ -695 +695 @@
-I))
+I#i_pron))
@@ -697 +697 @@
-will)
+will#will_v1)
@@ -699 +699 @@
-kepe)
+kepe#keep_v)
@@ -702 +702 @@
-hym))
+hym#him_pron))
@@ -705 +705 @@
-here))
+here#here_adv))
@@ -708 +708 @@
-longer))
+longer#long_adj1))
@@ -717 +717 @@
-Wherfor))
+Wherfor#wherefore_adv))
@@ -725 +725 @@
-I))
+I#i_pron))
@@ -727 +727 @@
-beseche)
+beseche#beseech_v)
@@ -730 +730 @@
-your)
+your#your_pron)
@@ -732 +732 @@
-Grace))
+Grace#grace_n))
@@ -737 +737 @@
-by)
+by#by_prep)
@@ -740 +740 @@
-the)
+the#the_adj)
@@ -742 +742 @@
-next)
+next#next_adj)
@@ -744 +744 @@
-post)))
+post#post_n2)))
@@ -750 +750 @@
-in)
+in#in_prep)
@@ -753 +753 @@
-this)
+this#this_pron)
@@ -755 +755 @@
-small)
+small#small_adj)
@@ -757 +757 @@
-matier)))
+matier#matter_n1)))
@@ -759 +759 @@
-to)
+to#to_prep)
@@ -761 +761 @@
-knowe)
+knowe#know_v)
@@ -766 +766 @@
-the)
+the#the_adj)
@@ -768 +768 @@
-Kings))
+Kings#king_n))
@@ -770 +770 @@
-pleas=r=))
+pleas=r=#pleasure_n))
@@ -773 +773 @@
-and)
+and#and_conj1)
@@ -776 +776 @@
-yo=rs=))))))
+yo=rs=#yours_pron))))))
@@ -785 +785 @@
-Other)
+Other#other_adj)
@@ -787 +787 @@
-newes)
+newes#news_n)
@@ -790 +790 @@
-then)
+then#than_conj)
@@ -793 +793 @@
-suche)
+suche#such_adj)
@@ -796 +796 @@
-as)
+as#as_adv)
@@ -807 +807 @@
-I))
+I#i_pron))
@@ -809 +809 @@
-have)
+have#have_v)
@@ -811 +811 @@
-written)
+written#write_v)
@@ -814 +814 @@
-to)
+to#to_prep)
@@ -817 +817 @@
-the)
+the#the_adj)
@@ -819 +819 @@
-Kyng)))
+Kyng#king_n)))
@@ -822 +822 @@
-at)
+at#at_prep)
@@ -825 +825 @@
-this)
+this#this_pron)
@@ -827 +827 @@
-tyme))))))))
+tyme#time_n))))))))
@@ -832 +832 @@
-there))
+there#there_adv))
@@ -834 +834 @@
-be)
+be#be_v)
@@ -836 +836 @@
-not)
+not#not_adv)
@@ -842 +842 @@
-whiche))
+whiche#which_adj))
@@ -848,3 +848,3 @@
-I))
-(MD
-shall)
+I#i_pron))
+(MD
+shall#shall_v)
@@ -852 +852 @@
-not)
+not#not_adv)
@@ -854 +854 @@
-nede)
+nede#need_v2)
@@ -859 +859 @@
-to)
+to#to_prep)
@@ -861 +861 @@
-repete))
+repete#repeat_v))
@@ -866 +866 @@
-forasmoche)
+forasmoche#forasmuch_adv)
@@ -869 +869 @@
-as)
+as#as_adv)
@@ -877 +877 @@
-the)
+the#the_adj)
@@ -879 +879 @@
-Kings))
+Kings#king_n))
@@ -881 +881 @@
-lettres))
+lettres#letter_n1))
@@ -883 +883 @@
-shall)
+shall#shall_v)
@@ -885 +885 @@
-come)
+come#come_v)
@@ -888 +888 @@
-to)
+to#to_prep)
@@ -891 +891 @@
-the)
+the#the_adj)
@@ -893 +893 @@
-handes)
+handes#hand_n)
@@ -896 +896 @@
-of)
+of#of_prep)
@@ -899 +899 @@
-your)
+your#your_pron)
@@ -901 +901 @@
-Grace)
+Grace#grace_n)
@@ -907 +907 @@
-Whiche))
+Whiche#which_adj))
@@ -915 +915 @@
-Almighty)
+Almighty#almighty_adj)
@@ -917 +917 @@
-Jh~u))
+Jh~u#NA))
@@ -919 +919 @@
-preserve)
+preserve#preserve_v)
@@ -922 +922 @@
-to)
+to#to_prep)
@@ -926 +926 @@
-his)
+his#his_adj)
@@ -928 +928 @@
-pleas=r=))
+pleas=r=#pleasure_n))
@@ -931 +931 @@
-and)
+and#and_conj1)
@@ -934 +934 @@
-yo=rs=)))))))))))))))))
+yo=rs=#yours_pron)))))))))))))))))
@@ -943 +943 @@
-ffrom)
+ffrom#from_prep)
@@ -946 +946 @@
-Luke)))
+Luke#NA)))
@@ -949 +949 @@
-the)
+the#the_adj)
@@ -951 +951 @@
-xij=th=)
+xij=th=#NA)
@@ -953 +953 @@
-daye)
+daye#day_n)
@@ -956 +956 @@
-of)
+of#of_prep)
@@ -959 +959 @@
-Octobre))))
+Octobre#NA))))
@@ -967 +967 @@
-By)
+By#by_prep)
@@ -971 +971 @@
-your)
+your#your_pron)
@@ -973 +973 @@
-Gracys))
+Gracys#grace_n))
@@ -976 +976 @@
-most)
+most#most_adj)
@@ -978 +978 @@
-humble))
+humble#humble_adj))
@@ -980 +980 @@
-bedeman)
+bedeman#beadsman_n)
@@ -983 +983 @@
-Cuthbert)
+Cuthbert#NA)
@@ -985 +985 @@
-Tunstal)))
+Tunstal#NA)))
@@ -1005 +1005 @@
-Plese)
+Plese#please_v)
@@ -1008 +1008 @@
-it))
+it#it_pron))
@@ -1011 +1011 @@
-your)
+your#your_pron)
@@ -1013 +1013 @@
-Grace))
+Grace#grace_n))
@@ -1016 +1016 @@
-to)
+to#to_prep)
@@ -1018 +1018 @@
-undirstond)
+undirstond#understand_v)
@@ -1021 +1021 @@
-that)
+that#that_conj)
@@ -1025 +1025 @@
-for)
+for#for_prep)
@@ -1028 +1028 @@
-as)
+as#as_adv)
@@ -1030 +1030 @@
-moch)
+moch#much_adj)
@@ -1033 +1033 @@
-as)
+as#as_adv)
@@ -1040 +1040 @@
-I))
+I#i_pron))
@@ -1042 +1042 @@
-have)
+have#have_v)
@@ -1045 +1045 @@
-at)
+at#at_prep)
@@ -1048 +1048 @@
-lengthe)))
+lengthe#length_n)))
@@ -1050 +1050 @@
-written)
+written#write_v)
@@ -1053 +1053 @@
-to)
+to#to_prep)
@@ -1056 +1056 @@
-the)
+the#the_adj)
@@ -1058 +1058 @@
-Kinge)))
+Kinge#king_n)))
@@ -1061 +1061 @@
-at)
+at#at_prep)
@@ -1064 +1064 @@
-this)
+this#this_pron)
@@ -1066 +1066 @@
-tyme)))
+tyme#time_n)))
@@ -1069 +1069 @@
-off)
+off#of_prep)
@@ -1072 +1072 @@
-his)
+his#his_adj)
@@ -1074 +1074 @@
-affaires)))
+affaires#affair_n)))
@@ -1077 +1077 @@
-to)
+to#to_prep)
@@ -1079 +1079 @@
-have)
+have#have_v)
@@ -1082 +1082 @@
-recourse)
+recourse#recourse_n1)
@@ -1085 +1085 @@
-to)
+to#to_prep)
@@ -1088 +1088 @@
-his)
+his#his_adj)
@@ -1090 +1090 @@
-Lettre))))
+Lettre#letter_n1))))
@@ -1093 +1093 @@
-for)
+for#for_prep)
@@ -1096 +1096 @@
-the)
+the#the_adj)
@@ -1098 +1098 @@
-knowlege)
+knowlege#knowledge_n)
@@ -1101 +1101 @@
-of)
+of#of_prep)
@@ -1104 +1104 @@
-them)))))))))))
+them#them_pron)))))))))))
@@ -1109 +1109 @@
-Your)
+Your#your_pron)
@@ -1111 +1111 @@
-Grace))
+Grace#grace_n))
@@ -1113 +1113 @@
-may)
+may#may_v1)
@@ -1115 +1115 @@
-perceyve)
+perceyve#perceive_v)
@@ -1118 +1118 @@
-by)
+by#by_prep)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-said)
+said#said_adj)
@@ -1125 +1125 @@
-lettre)))
+lettre#letter_n1)))
@@ -1128 +1128 @@
-that)
+that#that_conj)
@@ -1132 +1132 @@
-lettres)
+lettres#letter_n1)
@@ -1136 +1136 @@
-be)
+be#be_v)
@@ -1141 +1141 @@
-substantially))
+substantially#substantially_adv))
@@ -1143 +1143 @@
-to)
+to#to_prep)
@@ -1145 +1145 @@
-be)
+be#be_v)
@@ -1147 +1147 @@
-made))
+made#make_v1))
@@ -1150 +1150 @@
-and)
+and#and_conj1)
@@ -1153 +1153 @@
-to)
+to#to_prep)
@@ -1155 +1155 @@
-be)
+be#be_v)
@@ -1157 +1157 @@
-sent)
+sent#send_v1)
@@ -1160 +1160 @@
-both)
+both#both_pron)
@@ -1163 +1163 @@
-to)
+to#to_prep)
@@ -1166 +1166 @@
-the)
+the#the_adj)
@@ -1168 +1168 @@
-Kinge)
+Kinge#king_n)
@@ -1171 +1171 @@
-Catholyk)))
+Catholyk#catholic_adj)))
@@ -1174 +1174 @@
-cheffly)))
+cheffly#chiefly_adv)))
@@ -1177 +1177 @@
-and)
+and#and_conj1)
@@ -1180 +1180 @@
-as)
+as#as_adv)
@@ -1193 +1193 @@
-me))
+me#me_pron1))
@@ -1195 +1195 @@
-semeth))))
+semeth#seem_v2))))
@@ -1198 +1198 @@
-also)
+also#also_adv)
@@ -1200 +1200 @@
-to)
+to#to_prep)
@@ -1204 +1204 @@
-the)
+the#the_adj)
@@ -1206 +1206 @@
-Lord)
+Lord#lord_n)
@@ -1209 +1209 @@
-Cheuers)))
+Cheuers#NA)))
@@ -1212 +1212 @@
-and)
+and#and_conj1)
@@ -1215 +1215 @@
-the)
+the#the_adj)
@@ -1217 +1217 @@
-Chaunceler))))))))))
+Chaunceler#chancellor_n))))))))))
@@ -1220 +1220 @@
-from)
+from#from_prep)
@@ -1223 +1223 @@
-the)
+the#the_adj)
@@ -1225 +1225 @@
-Kinge))))))))
+Kinge#king_n))))))))
@@ -1234 +1234 @@
-Such)
+Such#such_adj)
@@ -1236 +1236 @@
-practise)
+practise#practice_n)
@@ -1239 +1239 @@
-as)
+as#as_adv)
@@ -1248 +1248 @@
-they))
+they#they_pron))
@@ -1250 +1250 @@
-suspecte)
+suspecte#suspect_v)
@@ -1255 +1255 @@
-to)
+to#to_prep)
@@ -1257 +1257 @@
-be)
+be#be_v)
@@ -1259 +1259 @@
-made)
+made#make_v1)
@@ -1262 +1262 @@
-again)
+again#again_adv)
@@ -1265 +1265 @@
-them))))))))
+them#them_pron))))))))
@@ -1267 +1267 @@
-wyl)
+wyl#will_v1)
@@ -1270 +1270 @@
-never))
+never#never_adv))
@@ -1272 +1272 @@
-be)
+be#be_v)
@@ -1274 +1274 @@
-brogth)
+brogth#bring_v)
@@ -1276 +1276 @@
-aboute)
+aboute#about_adv)
@@ -1282 +1282 @@
-that))
+that#that_pron))
@@ -1284 +1284 @@
-is)
+is#be_v)
@@ -1287 +1287 @@
-to)
+to#to_prep)
@@ -1289 +1289 @@
-say)
+say#say_v1)
@@ -1292 +1292 @@
-to)
+to#to_prep)
@@ -1294 +1294 @@
-remove)
+remove#remove_v)
@@ -1297 +1297 @@
-them))
+them#them_pron))
@@ -1300 +1300 @@
-out)
+out#out_adv)
@@ -1302 +1302 @@
-off)
+off#of_prep)
@@ -1307 +1307 @@
-governaunce))))))
+governaunce#governance_n))))))
@@ -1312 +1312 @@
-oonlesse)
+oonlesse#unless_adv)
@@ -1319 +1319 @@
-it))
+it#it_pron))
@@ -1321 +1321 @@
-com)
+com#come_v)
@@ -1324 +1324 @@
-off)
+off#of_prep)
@@ -1328 +1328 @@
-the)
+the#the_adj)
@@ -1330 +1330 @@
-Kinge)
+Kinge#king_n)
@@ -1333 +1333 @@
-Catholyke)))
+Catholyke#catholic_adj)))
@@ -1335 +1335 @@
-own)
+own#own_adj)
@@ -1337 +1337 @@
-mynd))))))
+mynd#mind_n1))))))
@@ -1345 +1345 @@
-for)
+for#for_prep)
@@ -1348 +1348 @@
-albeyt)
+albeyt#albeit_conj)
@@ -1355 +1355 @@
-he))
+he#he_pron))
@@ -1357 +1357 @@
-be)
+be#be_v)
@@ -1360 +1360 @@
-yonge)))))
+yonge#young_adj)))))
@@ -1365 +1365 @@
-yet))
+yet#yet_adv))
@@ -1368 +1368 @@
-he))
+he#he_pron))
@@ -1370 +1370 @@
-knoweth)
+knoweth#know_v)
@@ -1373 +1373 @@
-al_redy))
+al_redy#already_adj))
@@ -1378 +1378 @@
-how)
+how#how_adv)
@@ -1380 +1380 @@
-gret))
+gret#great_adj))
@@ -1382 +1382 @@
-a)
+a#a_adj2)
@@ -1384 +1384 @@
-man))
+man#man_n1))
@@ -1392 +1392 @@
-he))
+he#he_pron))
@@ -1394 +1394 @@
-is))))
+is#be_v))))
@@ -1400 +1400 @@
-and)
+and#and_conj1)
@@ -1403 +1403 @@
-he))
+he#he_pron))
@@ -1405 +1405 @@
-is)
+is#be_v)
@@ -1407 +1407 @@
-callyd)
+callyd#call_v)
@@ -1412 +1412 @@
-as)
+as#as_adv)
@@ -1423 +1423 @@
-I))
+I#i_pron))
@@ -1425 +1425 @@
-can)
+can#can_v1)
@@ -1427 +1427 @@
-perceyve))))
+perceyve#perceive_v))))
@@ -1435 +1435 @@
-veray)
+veray#very_adj)
@@ -1437 +1437 @@
-fast))
+fast#fast_adj))
@@ -1442 +1442 @@
-wher))
+wher#where_adv))
@@ -1450 +1450 @@
-he))
+he#he_pron))
@@ -1453 +1453 @@
-lovyth)
+lovyth#love_v1)
@@ -1455 +1455 @@
-or)
+or#or_conj2)
@@ -1457 +1457 @@
-hatyth))))))
+hatyth#hate_v))))))
@@ -1467 +1467 @@
-Wherfor))
+Wherfor#wherefore_adv))
@@ -1475 +1475 @@
-by)
+by#by_prep)
@@ -1478 +1478 @@
-force)))
+force#force_n1)))
@@ -1481 +1481 @@
-he))
+he#he_pron))
@@ -1483 +1483 @@
-wyl)
+wyl#will_v1)
@@ -1485 +1485 @@
-not)
+not#not_adv)
@@ -1487 +1487 @@
-be)
+be#be_v)
@@ -1489 +1489 @@
-masteryd)
+masteryd#master_v)
@@ -1492 +1492 @@
-onlesse)
+onlesse#unless_adv)
@@ -1499 +1499 @@
-it))
+it#it_pron))
@@ -1501 +1501 @@
-be)
+be#be_v)
@@ -1504 +1504 @@
-first))
+first#first_adj))
@@ -1507 +1507 @@
-by)
+by#by_prep)
@@ -1510 +1510 @@
-th'@)
+th'@#the_adj)
@@ -1512 +1512 @@
-@emperor)))
+@emperor#emperor_n)))
@@ -1514 +1514 @@
-persuadyd)
+persuadyd#persuade_v)
@@ -1517 +1517 @@
-to)
+to#to_prep)
@@ -1520 +1520 @@
-hym))))))
+hym#him_pron))))))
@@ -1523 +1523 @@
-yff)
+yff#if_conj)
@@ -1530 +1530 @@
-th@)
+th@#the_adj)
@@ -1532 +1532 @@
-@emperor))
+@emperor#emperor_n))
@@ -1534 +1534 @@
-wer)
+wer#be_v)
@@ -1537 +1537 @@
-so)
+so#so_adv)
@@ -1539 +1539 @@
-myndyd)))))))
+myndyd#minded_adj.II-III)))))))
@@ -1548 +1548 @@
-Your)
+Your#your_pron)
@@ -1550 +1550 @@
-Grace))
+Grace#grace_n))
@@ -1552 +1552 @@
-may)
+may#may_v1)
@@ -1554 +1554 @@
-se)
+se#see_v)
@@ -1557 +1557 @@
-mor)
+mor#more_adj)
@@ -1559 +1559 @@
-at)
+at#at_prep)
@@ -1562 +1562 @@
-large)))
+large#large_adj)))
@@ -1567 +1567 @@
-what))
+what#what_pron))
@@ -1575 +1575 @@
-I))
+I#i_pron))
@@ -1577 +1577 @@
-have)
+have#have_v)
@@ -1579 +1579 @@
-written)
+written#write_v)
@@ -1582 +1582 @@
-to)
+to#to_prep)
@@ -1585 +1585 @@
-the)
+the#the_adj)
@@ -1587 +1587 @@
-Kinge)))
+Kinge#king_n)))
@@ -1590 +1590 @@
-in)
+in#in_prep)
@@ -1593 +1593 @@
-that)
+that#that_pron)
@@ -1595 +1595 @@
-mater)))))))
+mater#matter_n1)))))))
@@ -1601 +1601 @@
-and)
+and#and_conj1)
@@ -1604 +1604 @@
-forasmoch)
+forasmoch#forasmuch_adv)
@@ -1607 +1607 @@
-as)
+as#as_adv)
@@ -1615,3 +1615,3 @@
-thes)
-(NS
-men))
+thes#these_pron)
+(NS
+men#man_n1))
@@ -1619 +1619 @@
-have)
+have#have_v)
@@ -1622 +1622 @@
-now))
+now#now_adv))
@@ -1624 +1624 @@
-detecte)
+detecte#detect_v)
@@ -1627 +1627 @@
-to)
+to#to_prep)
@@ -1629 +1629 @@
-have)
+have#have_v)
@@ -1633 +1633 @@
-our)
+our#our_pron)
@@ -1635 +1635 @@
-master))
+master#master_n1))
@@ -1638 +1638 @@
-suspecte)
+suspecte#suspect_adj)
@@ -1641 +1641 @@
-to)
+to#to_prep)
@@ -1643 +1643 @@
-practyse)
+practyse#practise_v)
@@ -1646 +1646 @@
-again)
+again#again_adv)
@@ -1649 +1649 @@
-them))))))))
+them#them_pron))))))))
@@ -1654 +1654 @@
-and)
+and#and_conj1)
@@ -1658 +1658 @@
-it))
+it#it_pron))
@@ -1660 +1660 @@
-may)
+may#may_v1)
@@ -1662 +1662 @@
-apper)
+apper#appear_v)
@@ -1665 +1665 @@
-that)
+that#that_conj)
@@ -1669 +1669 @@
-lettres)
+lettres#letter_n1)
@@ -1673 +1673 @@
-have)
+have#have_v)
@@ -1675 +1675 @@
-com~en)
+com~en#come_v)
@@ -1678 +1678 @@
-to)
+to#to_prep)
@@ -1681 +1681 @@
-ther)
+ther#their_adj)
@@ -1683 +1683 @@
-knowlege)))
+knowlege#knowledge_n)))
@@ -1686 +1686 @@
-written)
+written#write_v)
@@ -1689 +1689 @@
-off)
+off#of_prep)
@@ -1692 +1692 @@
-that)
+that#that_pron)
@@ -1694 +1694 @@
-mater))))
+mater#matter_n1))))
@@ -1700 +1700 @@
-for)
+for#for_prep)
@@ -1703 +1703 @@
-which)
+which#which_adj)
@@ -1705 +1705 @@
-cause)))
+cause#cause_n)))
@@ -1713 +1713 @@
-they))
+they#they_pron))
@@ -1715 +1715 @@
-wyl)
+wyl#will_v1)
@@ -1717 +1717 @@
-suspect)
+suspect#suspect_v)
@@ -1721 +1721 @@
-the)
+the#the_adj)
@@ -1723 +1723 @@
-Kings))
+Kings#king_n))
@@ -1725 +1725 @@
-lettres))
+lettres#letter_n1))
@@ -1729 +1729 @@
-the)
+the#the_adj)
@@ -1731 +1731 @@
-mor)))))))))))))
+mor#more_adj)))))))))))))
@@ -1738 +1738 @@
-therfor))
+therfor#therefore_adv))
@@ -1741 +1741 @@
-me))
+me#me_pron1))
@@ -1743 +1743 @@
-semyth)
+semyth#seem_v2)
@@ -1750 +1750 @@
-it))
+it#it_pron))
@@ -1752 +1752 @@
-shuld)
+shuld#shall_v)
@@ -1754 +1754 @@
-be)
+be#be_v)
@@ -1757 +1757 @@
-wel))
+wel#well_adv))
@@ -1759 +1759 @@
-doon)
+doon#do_v)
@@ -1762 +1762 @@
-to)
+to#to_prep)
@@ -1764 +1764 @@
-write)
+write#write_v)
@@ -1767 +1767 @@
-al))
+al#all_adj))
@@ -1771 +1771 @@
-to)
+to#to_prep)
@@ -1774 +1774 @@
-th@)
+th@#the_adj)
@@ -1776 +1776 @@
-@emperor)))
+@emperor#emperor_n)))
@@ -1779 +1779 @@
-or)
+or#or_conj2)
@@ -1782 +1782 @@
-to)
+to#to_prep)
@@ -1785 +1785 @@
-M.)
+M.#NA)
@@ -1787 +1787 @@
-Wyngfeld)))))
+Wyngfeld#NA)))))
@@ -1790 +1790 @@
-in)
+in#in_prep)
@@ -1793 +1793 @@
-syfer)))
+syfer#cipher_n)))
@@ -1798 +1798 @@
-and)
+and#and_conj1)
@@ -1801 +1801 @@
-lykwyse))
+lykwyse#likewise_adv))
@@ -1804 +1804 @@
-to)
+to#to_prep)
@@ -1807 +1807 @@
-me))))
+me#me_pron1))))
@@ -1810 +1810 @@
-in)
+in#in_prep)
@@ -1813 +1813 @@
-maters)
+maters#matter_n1)
@@ -1817 +1817 @@
-which))
+which#which_adj))
@@ -1823 +1823 @@
-the)
+the#the_adj)
@@ -1825 +1825 @@
-Kinge))
+Kinge#king_n))
@@ -1827 +1827 @@
-wold)
+wold#will_v1)
@@ -1829 +1829 @@
-have)
+have#have_v)
@@ -1834 +1834 @@
-kepte)
+kepte#keep_v)
@@ -1837 +1837 @@
-secret))))))))))
+secret#secret_adj))))))))))
@@ -1845 +1845 @@
-for)
+for#for_prep)
@@ -1848 +1848 @@
-ther))
+ther#there_adv))
@@ -1850 +1850 @@
-is)
+is#be_v)
@@ -1853 +1853 @@
-no)
+no#no_adj)
@@ -1855 +1855 @@
-trust))
+trust#trust_n))
@@ -1858 +1858 @@
-to)
+to#to_prep)
@@ -1860 +1860 @@
-be)
+be#be_v)
@@ -1862 +1862 @@
-put)
+put#put_v)
@@ -1865 +1865 @@
-in)
+in#in_prep)
@@ -1868 +1868 @@
-the)
+the#the_adj)
@@ -1870 +1870 @@
-Posts))))
+Posts#post_n2))))
@@ -1876 +1876 @@
-for)
+for#for_prep)
@@ -1879 +1879 @@
-which)
+which#which_adj)
@@ -1881 +1881 @@
-cause)))
+cause#cause_n)))
@@ -1889 +1889 @@
-I))
+I#i_pron))
@@ -1891 +1891 @@
-have)
+have#have_v)
@@ -1893 +1893 @@
-sent)
+sent#send_v1)
@@ -1896 +1896 @@
-your)
+your#your_pron)
@@ -1898 +1898 @@
-Grace))
+Grace#grace_n))
@@ -1901 +1901 @@
-a)
+a#a_adj2)
@@ -1903 +1903 @@
-sypher)
+sypher#cipher_n)
@@ -1907 +1907 @@
-herin))
+herin#herein_adv))
@@ -1909 +1909 @@
-inclosyd))
+inclosyd#enclose_v))
@@ -1915 +1915 @@
-wherunto))
+wherunto#whereunto_adv))
@@ -1924 +1924 @@
-I))
+I#i_pron))
@@ -1926 +1926 @@
-have)
+have#have_v)
@@ -1928 +1928 @@
-addyd)
+addyd#add_v)
@@ -1931 +1931 @@
-certain)
+certain#certain_adj)
@@ -1933 +1933 @@
-lettres)
+lettres#letter_n1)
@@ -1936 +1936 @@
-and)
+and#and_conj1)
@@ -1939 +1939 @@
-names)))
+names#name_n)))
@@ -1942 +1942 @@
-necessary))))
+necessary#necessary_adj))))
@@ -1947 +1947 @@
-and)
+and#and_conj1)
@@ -1949 +1949 @@
-also)
+also#also_adv)
@@ -1954 +1954 @@
-have)
+have#have_v)
@@ -1956 +1956 @@
-chaungyd)
+chaungyd#change_v)
@@ -1959 +1959 @@
-the)
+the#the_adj)
@@ -1961 +1961 @@
-ordre)
+ordre#order_n)
@@ -1964 +1964 @@
-off)
+off#of_prep)
@@ -1967 +1967 @@
-it)))))))))))
+it#it_pron)))))))))))
@@ -1975 +1975 @@
-for)
+for#for_prep)
@@ -1978 +1978 @@
-as)
+as#as_adv)
@@ -1981 +1981 @@
-to)
+to#to_prep)
@@ -1984 +1984 @@
-the)
+the#the_adj)
@@ -1986 +1986 @@
-odyr)
+odyr#other_adj)
@@ -1997 +1997 @@
-I))
+I#i_pron))
@@ -1999 +1999 @@
-had))))))
+had#have_v))))))
@@ -2004 +2004 @@
-it))
+it#it_pron))
@@ -2006 +2006 @@
-was)
+was#be_v)
@@ -2009 +2009 @@
-the)
+the#the_adj)
@@ -2011 +2011 @@
-same)
+same#same_adj)
@@ -2016 +2016 @@
-that)
+that#that_conj)
@@ -2022 +2022 @@
-M.)
+M.#NA)
@@ -2024 +2024 @@
-Secretary))
+Secretary#NA))
@@ -2026 +2026 @@
-had))))
+had#have_v))))
@@ -2034 +2034 @@
-and)
+and#and_conj1)
@@ -2038 +2038 @@
-lakkyd)
+lakkyd#lack_v1)
@@ -2041 +2041 @@
-many)
+many#many_adj)
@@ -2043 +2043 @@
-thyngs)
+thyngs#thing_n1)
@@ -2046 +2046 @@
-necessary)))
+necessary#necessary_adj)))
@@ -2052 +2052 @@
-which))
+which#which_adj))
@@ -2060 +2060 @@
-I))
+I#i_pron))
@@ -2062 +2062 @@
-send)
+send#send_v1)
@@ -2065 +2065 @@
-to)
+to#to_prep)
@@ -2068 +2068 @@
-th@)
+th@#the_adj)
@@ -2070 +2070 @@
-@entent)
+@entent#intent_n)
@@ -2078 +2078 @@
-your)
+your#your_pron)
@@ -2080 +2080 @@
-Grace))
+Grace#NA))
@@ -2082 +2082 @@
-may)
+may#may_v1)
@@ -2085 +2085 @@
-mor)
+mor#more_adj)
@@ -2087 +2087 @@
-surly))
+surly#surely_adv))
@@ -2089 +2089 @@
-write)
+write#write_v)
@@ -2092 +2092 @@
-such)
+such#such_adj)
@@ -2094 +2094 @@
-thynges)
+thynges#thing_n1)
@@ -2097 +2097 @@
-as)
+as#as_adv)
@@ -2107 +2107 @@
-shal)
+shal#shall_v)
@@ -2109 +2109 @@
-be)
+be#be_v)
@@ -2112 +2112 @@
-necessary)
+necessary#necessary_adj)
@@ -2115 +2115 @@
-to)
+to#to_prep)
@@ -2117 +2117 @@
-have)
+have#have_v)
@@ -2122 +2122 @@
-kept)
+kept#keep_v)
@@ -2125 +2125 @@
-secret))))))))))
+secret#secret_adj))))))))))
@@ -2130 +2130 @@
-or)
+or#or_conj2)
@@ -2134 +2134 @@
-the)
+the#the_adj)
@@ -2136 +2136 @@
-Kynge))
+Kynge#king_n))
@@ -2139 +2139 @@
-yff)
+yff#if_conj)
@@ -2146 +2146 @@
-it))
+it#it_pron))
@@ -2148 +2148 @@
-please)
+please#please_v)
@@ -2151 +2151 @@
-hym))
+hym#him_pron))
@@ -2154 +2154 @@
-to)
+to#to_prep)
@@ -2156 +2156 @@
-use)
+use#use_v)
@@ -2159 +2159 @@
-yt))))))))))))))
+yt#it_pron))))))))))))))
@@ -2168 +2168 @@
-I))
+I#i_pron))
@@ -2170 +2170 @@
-sent)
+sent#send_v1)
@@ -2173 +2173 @@
-M.)
+M.#NA)
@@ -2175 +2175 @@
-Rychmont))
+Rychmont#NA))
@@ -2178 +2178 @@
-to)
+to#to_prep)
@@ -2181 +2181 @@
-Cales)))
+Cales#NA)))
@@ -2184 +2184 @@
-with)
+with#with_prep)
@@ -2187 +2187 @@
-thys)
+thys#this_pron)
@@ -2189 +2189 @@
-boget)))
+boget#budget_n)))
@@ -2194 +2194 @@
-and)
+and#and_conj1)
@@ -2197 +2197 @@
-bycause)
+bycause#because_adv)
@@ -2200 +2200 @@
-that)
+that#that_conj)
@@ -2204 +2204 @@
-the)
+the#the_adj)
@@ -2206 +2206 @@
-sayd)
+sayd#said_adj)
@@ -2208 +2208 @@
-cypher))
+cypher#cipher_n))
@@ -2210 +2210 @@
-may)
+may#may_v1)
@@ -2213 +2213 @@
-surly))
+surly#surely_adv))
@@ -2215 +2215 @@
-come)
+come#come_v)
@@ -2218 +2218 @@
-to)
+to#to_prep)
@@ -2222 +2222 @@
-your)
+your#your_pron)
@@ -2224 +2224 @@
-Graces))
+Graces#grace_n))
@@ -2226 +2226 @@
-hands)))))))
+hands#hand_n)))))))
@@ -2235 +2235 @@
-I))
+I#i_pron))
@@ -2237 +2237 @@
-undirstond)
+undirstond#understand_v)
@@ -2240 +2240 @@
-by)
+by#by_prep)
@@ -2243 +2243 @@
-the)
+the#the_adj)
@@ -2245 +2245 @@
-Chaunceler)))
+Chaunceler#chancellor_n)))
@@ -2248 +2248 @@
-touchynge)
+touchynge#touching_prep)
@@ -2253 +2253 @@
-the)
+the#the_adj)
@@ -2255 +2255 @@
-Duke)
+Duke#duke_n)
@@ -2258 +2258 @@
-of)
+of#of_prep)
@@ -2261 +2261 @@
-Gelders)))))
+Gelders#NA)))))
@@ -2264 +2264 @@
-that)
+that#that_conj)
@@ -2268 +2268 @@
-he))
+he#he_pron))
@@ -2270 +2270 @@
-hath)
+hath#have_v)
@@ -2273 +2273 @@
-now))
+now#now_adv))
@@ -2276 +2276 @@
-late))
+late#late_adv))
@@ -2278 +2278 @@
-written)
+written#write_v)
@@ -2281 +2281 @@
-a)
+a#a_adj2)
@@ -2283 +2283 @@
-Lettre))
+Lettre#letter_n1))
@@ -2286 +2286 @@
-to)
+to#to_prep)
@@ -2289 +2289 @@
-the)
+the#the_adj)
@@ -2291 +2291 @@
-Kinge)
+Kinge#king_n)
@@ -2294 +2294 @@
-Catholyk))))
+Catholyk#catholic_adj))))
@@ -2297 +2297 @@
-off)
+off#of_prep)
@@ -2300 +2300 @@
-a)
+a#a_adj2)
@@ -2303 +2303 @@
-more)
+more#more_adj)
@@ -2305 +2305 @@
-sobre))
+sobre#sober_adj))
@@ -2307 +2307 @@
-style)
+style#style_n)
@@ -2310 +2310 @@
-then)
+then#than_conj)
@@ -2319 +2319 @@
-he))
+he#he_pron))
@@ -2321 +2321 @@
-was)
+was#be_v)
@@ -2324 +2324 @@
-wont)
+wont#wont_adj)
@@ -2329 +2329 @@
-to)
+to#to_prep)
@@ -2331 +2331 @@
-use))))))))
+use#use_v))))))))
@@ -2337 +2337 @@
-by)
+by#by_prep)
@@ -2340 +2340 @@
-which)))
+which#which_adj)))
@@ -2348 +2348 @@
-it))
+it#it_pron))
@@ -2350 +2350 @@
-may)
+may#may_v1)
@@ -2352 +2352 @@
-appere)
+appere#appear_v)
@@ -2359 +2359 @@
-he))
+he#he_pron))
@@ -2361 +2361 @@
-makyth)
+makyth#make_v1)
@@ -2364 +2364 @@
-overture)
+overture#overture_n)
@@ -2367 +2367 @@
-off)
+off#of_prep)
@@ -2370 +2370 @@
-treaty))))
+treaty#treaty_n))))
@@ -2374 +2374 @@
-whiche))
+whiche#which_adj))
@@ -2380 +2380 @@
-I))
+I#i_pron))
@@ -2382 +2382 @@
-thynke)
+thynke#think_v2)
@@ -2390 +2390 @@
-shal)
+shal#shall_v)
@@ -2392 +2392 @@
-be)
+be#be_v)
@@ -2394 +2394 @@
-herkynyd)
+herkynyd#hearken_v)
@@ -2397 +2397 @@
-unto)
+unto#unto_prep)
@@ -2408 +2408 @@
-Your)
+Your#your_pron)
@@ -2410 +2410 @@
-Grace))
+Grace#grace_n))
@@ -2412 +2412 @@
-may)
+may#may_v1)
@@ -2414 +2414 @@
-aduertise)
+aduertise#advertise_v)
@@ -2417 +2417 @@
-the)
+the#the_adj)
@@ -2419 +2419 @@
-Kinge))
+Kinge#king_n))
@@ -2422 +2422 @@
-theroff)))
+theroff#thereof_adv)))
@@ -2428 +2428 @@
-for)
+for#for_prep)
@@ -2431 +2431 @@
-it))
+it#it_pron))
@@ -2433 +2433 @@
-is)
+is#be_v)
@@ -2435 +2435 @@
-not)
+not#not_adv)
@@ -2438 +2438 @@
-in)
+in#in_prep)
@@ -2441 +2441 @@
-his)
+his#his_adj)
@@ -2443 +2443 @@
-lettre)))
+lettre#letter_n1)))
@@ -2452 +2452 @@
-I))
+I#i_pron))
@@ -2454 +2454 @@
-have)
+have#have_v)
@@ -2457 +2457 @@
-at)
+at#at_prep)
@@ -2460 +2460 @@
-diuerse)
+diuerse#diverse_adj)
@@ -2462 +2462 @@
-tymes)))
+tymes#time_n)))
@@ -2465 +2465 @@
-sene)
+sene#sin_adv)
@@ -2474 +2474 @@
-I))
+I#i_pron))
@@ -2476 +2476 @@
-come)
+come#come_v)
@@ -2479 +2479 @@
-in)
+in#in_adv)
@@ -2481 +2481 @@
-to)
+to#to_prep)
@@ -2484 +2484 @@
-thes)
+thes#these_pron)
@@ -2486 +2486 @@
-parties))))))
+parties#party_n))))))
@@ -2488 +2488 @@
-lent)
+lent#lend_v2)
@@ -2491 +2491 @@
-M.)
+M.#NA)
@@ -2493 +2493 @@
-Spinel))
+Spinel#NA))
@@ -2496 +2496 @@
-money))
+money#money_n))
@@ -2500 +2500 @@
-which))
+which#which_adj))
@@ -2507 +2507 @@
-monteth)
+monteth#mount_v)
@@ -2510 +2510 @@
-in)
+in#in_prep)
@@ -2513 +2513 @@
-al)))
+al#all_adj)))
@@ -2516 +2516 @@
-to)
+to#to_prep)
@@ -2519 +2519 @@
-thyrty@)
+thyrty@#thirty_adj)
@@ -2521 +2521 @@
-@=li=)
+@=li=#pound_n1)
@@ -2524 +2524 @@
-sterlinge)))))))
+sterlinge#sterling_n)))))))
@@ -2530 +2530 @@
-and)
+and#and_conj1)
@@ -2533 +2533 @@
-as)
+as#as_adv)
@@ -2536 +2536 @@
-yet)))
+yet#yet_adv)))
@@ -2539 +2539 @@
-I))
+I#i_pron))
@@ -2541 +2541 @@
-am)
+am#be_v)
@@ -2544 +2544 @@
-unpayd))
+unpayd#unpaid_adj))
@@ -2553 +2553 @@
-At)
+At#at_prep)
@@ -2556 +2556 @@
-thys)
+thys#this_pron)
@@ -2558 +2558 @@
-tyme)))
+tyme#time_n)))
@@ -2561 +2561 @@
-he))
+he#he_pron))
@@ -2563 +2563 @@
-writeth)
+writeth#write_v)
@@ -2568 +2568 @@
-as)
+as#as_adv)
@@ -2579 +2579 @@
-he))
+he#he_pron))
@@ -2581 +2581 @@
-shewyth)
+shewyth#show_v)
@@ -2584 +2584 @@
-me)))))
+me#me_pron1)))))
@@ -2589 +2589 @@
-to)
+to#to_prep)
@@ -2592 +2592 @@
-your)
+your#your_pron)
@@ -2594 +2594 @@
-Grace)))
+Grace#grace_n)))
@@ -2597 +2597 @@
-theroff))
+theroff#thereof_adv))
@@ -2605 +2605 @@
-and)
+and#and_conj1)
@@ -2609 +2609 @@
-hath)
+hath#have_v)
@@ -2611 +2611 @@
-gyffen)
+gyffen#give_v)
@@ -2614 +2614 @@
-me))
+me#me_pron1))
@@ -2617 +2617 @@
-a)
+a#a_adj2)
@@ -2619 +2619 @@
-byl))
+byl#bill_n3))
@@ -2622 +2622 @@
-for)
+for#for_prep)
@@ -2625 +2625 @@
-my)
+my#my_adj)
@@ -2627 +2627 @@
-stuard))
+stuard#steward_n))
@@ -2629 +2629 @@
-to)
+to#to_prep)
@@ -2631 +2631 @@
-receyve)
+receyve#receive_v)
@@ -2634 +2634 @@
-it))
+it#it_pron))
@@ -2637 +2637 @@
-off)
+off#of_prep)
@@ -2640 +2640 @@
-M.)
+M.#NA)
@@ -2642 +2642 @@
-Heron)))
+Heron#NA)))
@@ -2645 +2645 @@
-off)
+off#of_prep)
@@ -2648 +2648 @@
-his)
+his#his_adj)
@@ -2650 +2650 @@
-payment)
+payment#payment_n1)
@@ -2653 +2653 @@
-off)
+off#of_prep)
@@ -2656 +2656 @@
-his)
+his#his_adj)
@@ -2658 +2658 @@
-fee)
+fee#fee_n2)
@@ -2667 +2667 @@
-next))
+next#next_adj))
@@ -2669 +2669 @@
-to)
+to#to_prep)
@@ -2671 +2671 @@
-be)
+be#be_v)
@@ -2673 +2673 @@
-payd))))))))
+payd#pay_v1))))))))
@@ -2682 +2682 @@
-I))
+I#i_pron))
@@ -2684 +2684 @@
-besech)
+besech#beseech_v)
@@ -2687 +2687 @@
-your)
+your#your_pron)
@@ -2689 +2689 @@
-Grace))
+Grace#grace_n))
@@ -2692 +2692 @@
-to)
+to#to_prep)
@@ -2694 +2694 @@
-help)
+help#help_v)
@@ -2701 +2701 @@
-the)
+the#the_adj)
@@ -2703 +2703 @@
-said)
+said#said_adj)
@@ -2705 +2705 @@
-sume))
+sume#sum_n1))
@@ -2707 +2707 @@
-may)
+may#may_v1)
@@ -2709 +2709 @@
-be)
+be#be_v)
@@ -2711 +2711 @@
-payd)
+payd#pay_v1)
@@ -2714 +2714 @@
-to)
+to#to_prep)
@@ -2717 +2717 @@
-my)
+my#my_adj)
@@ -2719 +2719 @@
-stuard))))))
+stuard#steward_n))))))
@@ -2728 +2728 @@
-I)
+I#i_pron)
@@ -2732 +2732 @@
-owe)
+owe#owe_v)
@@ -2735 +2735 @@
-it)
+it#it_pron)
@@ -2737 +2737 @@
-my))
+my#my_adj))
@@ -2740 +2740 @@
-selff))
+selff#self_pron))
@@ -2743 +2743 @@
-in)
+in#in_prep)
@@ -2746 +2746 @@
-odyr)
+odyr#other_adj)
@@ -2748 +2748 @@
-places)))
+places#place_n.II-VII)))
@@ -2751 +2751 @@
-with)
+with#with_prep)
@@ -2754 +2754 @@
-mor)))
+mor#more_adj)))
@@ -2762 +2762 @@
-and)
+and#and_conj1)
@@ -2765 +2765 @@
-lykwyse))
+lykwyse#likewise_adv))
@@ -2768 +2768 @@
-I))
+I#i_pron))
@@ -2770 +2770 @@
-besech)
+besech#beseech_v)
@@ -2773 +2773 @@
-your)
+your#your_pron)
@@ -2775 +2775 @@
-Grace))
+Grace#grace_n))
@@ -2778 +2778 @@
-to)
+to#to_prep)
@@ -2780 +2780 @@
-be)
+be#be_v)
@@ -2783 +2783 @@
-good)
+good#good_adj)
@@ -2785 +2785 @@
-lord)
+lord#lord_n)
@@ -2788 +2788 @@
-to)
+to#to_prep)
@@ -2791 +2791 @@
-my)
+my#my_adj)
@@ -2793 +2793 @@
-por)
+por#poor_adj)
@@ -2795 +2795 @@
-frends)
+frends#friend_n)
@@ -2798 +2798 @@
-and)
+and#and_conj1)
@@ -2801 +2801 @@
-servaunts)))))
+servaunts#servant_n)))))
@@ -2804 +2804 @@
-in)
+in#in_prep)
@@ -2807 +2807 @@
-all)
+all#all_adj)
@@ -2809 +2809 @@
-ther)
+ther#their_adj)
@@ -2811 +2811 @@
-suts)
+suts#suit_n)
@@ -2819 +2819 @@
-to)
+to#to_prep)
@@ -2821 +2821 @@
-be)
+be#be_v)
@@ -2823 +2823 @@
-made)
+made#make_v1)
@@ -2826 +2826 @@
-unto)
+unto#unto_prep)
@@ -2829 +2829 @@
-your)
+your#your_pron)
@@ -2831 +2831 @@
-Grace)))
+Grace#grace_n)))
@@ -2834 +2834 @@
-in)
+in#in_prep)
@@ -2838 +2838 @@
-my)
+my#my_adj)
@@ -2840 +2840 @@
-causes))
+causes#cause_n))
@@ -2843 +2843 @@
-or)
+or#or_conj2)
@@ -2846 +2846 @@
-therys)))))))))))
+therys#their_adj)))))))))))
@@ -2854 +2854 @@
-and)
+and#and_conj1)
@@ -2857 +2857 @@
-thus))
+thus#thus_adv))
@@ -2860 +2860 @@
-yff)
+yff#if_conj)
@@ -2867 +2867 @@
-ther))
+ther#there_adv))
@@ -2869 +2869 @@
-be)
+be#be_v)
@@ -2874 +2874 @@
-any)
+any#any_adj)
@@ -2876 +2876 @@
-service)
+service#service_n1)
@@ -2881 +2881 @@
-that)
+that#that_conj)
@@ -2887 +2887 @@
-I))
+I#i_pron))
@@ -2889 +2889 @@
-may)
+may#may_v1)
@@ -2891 +2891 @@
-doo)
+doo#do_v)
@@ -2894 +2894 @@
-for)
+for#for_prep)
@@ -2897 +2897 @@
-your)
+your#your_pron)
@@ -2899 +2899 @@
-Grace)))
+Grace#grace_n)))
@@ -2902 +2902 @@
-in)
+in#in_prep)
@@ -2905 +2905 @@
-this)
+this#this_pron)
@@ -2907 +2907 @@
-parties)))))))))
+parties#party_n)))))))))
@@ -2911 +2911 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2913 +2913 @@
-pleas=r=))
+pleas=r=#pleasure_n))
@@ -2916 +2916 @@
-knowen)))
+knowen#known_adj)))
@@ -2919 +2919 @@
-I))
+I#i_pron))
@@ -2921 +2921 @@
-am)
+am#be_v)
@@ -2924 +2924 @@
-redy))
+redy#ready_adj))
@@ -2927 +2927 @@
-as)
+as#as_adv)
@@ -2937 +2937 @@
-knoweth)
+knoweth#know_v)
@@ -2940 +2940 @@
-our)
+our#our_pron)
@@ -2942 +2942 @@
-Lord)
+Lord#NA)
@@ -2948 +2948 @@
-who))
+who#who_pron))
@@ -2956 +2956 @@
-ever))
+ever#ever_adv))
@@ -2958 +2958 @@
-preserve)
+preserve#preserve_v)
@@ -2961 +2961 @@
-your)
+your#your_pron)
@@ -2963 +2963 @@
-Grace))))))))
+Grace#grace_n))))))))
@@ -2971 +2971 @@
-From)
+From#from_prep)
@@ -2974 +2974 @@
-Bruxellys)))
+Bruxellys#NA)))
@@ -2980 +2980 @@
-the)
+the#the_adj)
@@ -2982 +2982 @@
-xxv=th=)
+xxv=th=#NA)
@@ -2984 +2984 @@
-day)
+day#day_n)
@@ -2987 +2987 @@
-off)
+off#of_prep)
@@ -2990 +2990 @@
-November)))
+November#NA)))
@@ -2998 +2998 @@
-Your)
+Your#your_pron)
@@ -3001 +3001 @@
-most)
+most#most_adj)
@@ -3003 +3003 @@
-humble))
+humble#humble_adj))
@@ -3005 +3005 @@
-bedman)
+bedman#beadsman_n)
@@ -3008 +3008 @@
-Cuthbert)
+Cuthbert#NA)
@@ -3010 +3010 @@
-Tunstal))
+Tunstal#NA))
@@ -3018 +3018 @@
-To)
+To#to_prep)
@@ -3022 +3022 @@
-the)
+the#the_adj)
@@ -3025 +3025 @@
-moste)
+moste#most_adj)
@@ -3027 +3027 @@
-reverend))
+reverend#reverend_adj))
@@ -3029 +3029 @@
-fader)
+fader#father_n)
@@ -3032 +3032 @@
-in)
+in#in_prep)
@@ -3035 +3035 @@
-God))))
+God#NA))))
@@ -3038 +3038 @@
-and)
+and#and_conj1)
@@ -3041 +3041 @@
-his)
+his#his_adj)
@@ -3045 +3045 @@
-moste)
+moste#most_adj)
@@ -3047 +3047 @@
-singler))
+singler#singular_adj))
@@ -3049 +3049 @@
-good))
+good#good_adj))
@@ -3051 +3051 @@
-Lorde)
+Lorde#lord_n)
@@ -3054 +3054 @@
-Lord)
+Lord#lord_n)
@@ -3056 +3056 @@
-Cardinall)
+Cardinall#cardinal_n)
@@ -3059 +3059 @@
-of)
+of#of_prep)
@@ -3064 +3064 @@
-Yorke)))))))
+Yorke#NA)))))))
