--- anhatton-1693-e3-p2.psd.vert	2025-09-10 13:51:17
+++ anhatton-1693-e3-p2.lem.vert	2025-09-10 13:51:25
@@ -13 +13 @@
-Y=e=)
+Y=e=#the_adj)
@@ -15 +15 @@
-3=d=)
+3=d=#third_adj)
@@ -18 +18 @@
-of)
+of#of_prep)
@@ -21 +21 @@
-March)))
+March#NA)))
@@ -25 +25 @@
-1691)
+1691#NA)
@@ -36 +36 @@
-I))
+I#i_pron))
@@ -38 +38 @@
-am)
+am#be_v)
@@ -41 +41 @@
-very)
+very#very_adj)
@@ -43 +43 @@
-glad)
+glad#glad_adj)
@@ -46 +46 @@
-to)
+to#to_prep)
@@ -48 +48 @@
-heare)
+heare#hear_v)
@@ -56 +56 @@
-y=r=)
+y=r=#your_pron)
@@ -58 +58 @@
-Losp.))
+Losp.#lordship_n))
@@ -61 +61 @@
-and)
+and#and_conj1)
@@ -64 +64 @@
-all)
+all#all_adj)
@@ -66 +66 @@
-y=e=)
+y=e=#the_adj)
@@ -68 +68 @@
-company))))
+company#company_n))))
@@ -70 +70 @@
-gott)
+gott#get_v)
@@ -73 +73 @@
-so)
+so#so_adv)
@@ -75 +75 @@
-well))
+well#well_adv))
@@ -78 +78 @@
-to)
+to#to_prep)
@@ -81 +81 @@
-Maidwell)
+Maidwell#NA)
@@ -87 +87 @@
-w=ch=))
+w=ch=#which_adj))
@@ -94 +94 @@
-I))
+I#i_pron))
@@ -96 +96 @@
-was)
+was#be_v)
@@ -99 +99 @@
-so)
+so#so_adv)
@@ -101 +101 @@
-forgetfull)
+forgetfull#forgetful_adj)
@@ -104 +104 @@
-not)
+not#not_adv)
@@ -106 +106 @@
-to)
+to#to_prep)
@@ -108 +108 @@
-thinke)
+thinke#think_v2)
@@ -115 +115 @@
-you))
+you#you_pron))
@@ -117 +117 @@
-were)
+were#be_v)
@@ -120 +120 @@
-there)))))))
+there#there_adv)))))))
@@ -123 +123 @@
-but)
+but#but_prep)
@@ -128 +128 @@
-sent)
+sent#send_v1)
@@ -131 +131 @@
-my)
+my#my_adj)
@@ -133 +133 @@
-letter))
+letter#letter_n1))
@@ -136 +136 @@
-to)
+to#to_prep)
@@ -139 +139 @@
-Kirby)))
+Kirby#NA)))
@@ -145 +145 @@
-w=ch=))
+w=ch=#which_adj))
@@ -152 +152 @@
-made)
+made#make_v1)
@@ -156 +156 @@
-it))
+it#it_pron))
@@ -159 +159 @@
-longer))
+longer#long_adj1))
@@ -162 +162 @@
-before)
+before#before_adv)
@@ -169 +169 @@
-it))
+it#it_pron))
@@ -171 +171 @@
-gott)
+gott#get_v)
@@ -174 +174 @@
-to)
+to#to_prep)
@@ -177 +177 @@
-you)))))))))))))))))))
+you#you_pron)))))))))))))))))))
@@ -186 +186 @@
-My)
+My#my_adj)
@@ -188 +188 @@
-L=d=)
+L=d=#lord_n)
@@ -191 +191 @@
-Finch)))
+Finch#NA)))
@@ -193 +193 @@
-has)
+has#have_v)
@@ -195 +195 @@
-writ)
+writ#write_v)
@@ -198 +198 @@
-a)
+a#a_adj2)
@@ -200 +200 @@
-letter))
+letter#letter_n1))
@@ -203 +203 @@
-to)
+to#to_prep)
@@ -206 +206 @@
-my)
+my#my_adj)
@@ -208 +208 @@
-sister)
+sister#sister_n)
@@ -211 +211 @@
-Betty))
+Betty#NA))
@@ -217 +217 @@
-w=ch=))
+w=ch=#which_adj))
@@ -223 +223 @@
-I))
+I#i_pron))
@@ -225 +225 @@
-hope)
+hope#hope_v)
@@ -234 +234 @@
-she))
+she#she_pron))
@@ -236 +236 @@
-has)
+has#have_v)
@@ -238 +238 @@
-recievid)))))))
+recievid#receive_v)))))))
@@ -247 +247 @@
-Hear))
+Hear#here_adv))
@@ -249 +249 @@
-is)
+is#be_v)
@@ -252 +252 @@
-a)
+a#a_adj2)
@@ -255 +255 @@
-most)
+most#most_adj)
@@ -257 +257 @@
-dismal))
+dismal#dismal_n))
@@ -259 +259 @@
-story)
+story#story_n)
@@ -262 +262 @@
-of)
+of#of_prep)
@@ -266 +266 @@
-my)
+my#my_adj)
@@ -268 +268 @@
-L=d=)
+L=d=#lord_n)
@@ -271 +271 @@
-Salisbery@))
+Salisbery@#NA))
@@ -273 +273 @@
-@s))
+@s#NA))
@@ -275 +275 @@
-two)
+two#two_adj)
@@ -277 +277 @@
-brothers)
+brothers#brother_n)
@@ -281 +281 @@
-who))
+who#who_pron))
@@ -289 +289 @@
-he))
+he#he_pron))
@@ -291 +291 @@
-sent)
+sent#send_v1)
@@ -294 +294 @@
-into)
+into#into_prep)
@@ -297 +297 @@
-France))))))))
+France#NA))))))))
@@ -306 +306 @@
-They))
+They#they_pron))
@@ -308 +308 @@
-were)
+were#be_v)
@@ -311 +311 @@
-at)
+at#at_prep)
@@ -314 +314 @@
-St.)
+St.#NA)
@@ -316 +316 @@
-Germains)))
+Germains#NA)))
@@ -324 +324 @@
-and)
+and#and_conj1)
@@ -328 +328 @@
-lay)
+lay#lie_v1)
@@ -331 +331 @@
-in)
+in#in_prep)
@@ -334 +334 @@
-a)
+a#a_adj2)
@@ -336 +336 @@
-room)))
+room#room_n1)))
@@ -339 +339 @@
-together)))
+together#together_adv)))
@@ -345 +345 @@
-and)
+and#and_conj1)
@@ -349 +349 @@
-quarreld)
+quarreld#quarrel_v)
@@ -358 +358 @@
-t@))
+t@#it_pron))
@@ -360 +360 @@
-@is)
+@is#be_v)
@@ -362 +362 @@
-not)
+not#not_adv)
@@ -365 +365 @@
-yet))
+yet#yet_adv))
@@ -367 +367 @@
-known)
+known#know_v)
@@ -371 +371 @@
-for)
+for#for_prep)
@@ -374 +374 @@
-what))))
+what#what_pron))))
@@ -382 +382 @@
-but)
+but#but_prep)
@@ -386 +386 @@
-rise)
+rise#rise_v)
@@ -389 +389 @@
-out)
+out#out_adv)
@@ -391 +391 @@
-of)
+of#of_prep)
@@ -394 +394 @@
-bed))))
+bed#bed_n))))
@@ -400 +400 @@
-and)
+and#and_conj1)
@@ -404 +404 @@
-tooke)
+tooke#take_v)
@@ -407 +407 @@
-there)
+there#their_adj)
@@ -409 +409 @@
-swords))
+swords#sword_n))
@@ -417 +417 @@
-and)
+and#and_conj1)
@@ -421 +421 @@
-fought)
+fought#fight_v)
@@ -426 +426 @@
-tell)
+tell#till_prep)
@@ -433 +433 @@
-they))
+they#they_pron))
@@ -435 +435 @@
-both)
+both#both_pron)
@@ -437 +437 @@
-fell)
+fell#fall_v)
@@ -439 +439 @@
-dwone)
+dwone#down_adv)
@@ -445 +445 @@
-w=ch=))
+w=ch=#which_adj))
@@ -452 +452 @@
-was)
+was#be_v)
@@ -454 +454 @@
-heared)
+heared#hear_v)
@@ -457 +457 @@
-in)
+in#in_prep)
@@ -460 +460 @@
-the)
+the#the_adj)
@@ -464 +464 @@
-room)
+room#room_n1)
@@ -467 +467 @@
-under)
+under#under_prep)
@@ -470 +470 @@
-them))))))))))
+them#them_pron))))))))))
@@ -479 +479 @@
-They)
+They#they_pron)
@@ -484 +484 @@
-that)
+that#that_conj)
@@ -489 +489 @@
-were)
+were#be_v)
@@ -492 +492 @@
-there)))))
+there#there_adv)))))
@@ -494 +494 @@
-run)
+run#run_v)
@@ -496 +496 @@
-up)
+up#up_adv1)
@@ -499 +499 @@
-to)
+to#to_prep)
@@ -502 +502 @@
-there)
+there#their_adj)
@@ -504 +504 @@
-chamber)
+chamber#chamber_n)
@@ -506 +506 @@
-door)
+door#door_n)
@@ -512 +512 @@
-w=ch=))
+w=ch=#which_adj))
@@ -518 +518 @@
-they))
+they#they_pron))
@@ -520 +520 @@
-found)
+found#find_v)
@@ -525 +525 @@
-fastend))))))
+fastend#fasten_v))))))
@@ -533 +533 @@
-but)
+but#but_prep)
@@ -540 +540 @@
-forcing)
+forcing#force_v1)
@@ -543 +543 @@
-it))
+it#it_pron))
@@ -546 +546 @@
-open)))
+open#open_adj)))
@@ -550 +550 @@
-found)
+found#find_v)
@@ -553 +553 @@
-them))
+them#them_pron))
@@ -557 +557 @@
-weltring)
+weltring#welter_v1)
@@ -560 +560 @@
-in)
+in#in_prep)
@@ -563 +563 @@
-thier)
+thier#their_adj)
@@ -565 +565 @@
-blood))))
+blood#blood_n))))
@@ -568 +568 @@
-and)
+and#and_conj1)
@@ -571 +571 @@
-craling)
+craling#crawl_v1)
@@ -574 +574 @@
-to)
+to#to_prep)
@@ -576 +576 @@
-ask)
+ask#ask_v)
@@ -579 +579 @@
-one)
+one#one_adj)
@@ -581 +581 @@
-another))
+another#another_pron))
@@ -584 +584 @@
-pardon))))))
+pardon#pardon_n1))))))
@@ -593 +593 @@
-T@))
+T@#it_pron))
@@ -595 +595 @@
-@is)
+@is#be_v)
@@ -597 +597 @@
-sayed)
+sayed#say_v1)
@@ -604 +604 @@
-there))
+there#there_adv))
@@ -606 +606 @@
-was)
+was#be_v)
@@ -609 +609 @@
-little)
+little#little_adj)
@@ -611 +611 @@
-hopes)
+hopes#hope_n1)
@@ -614 +614 @@
-that)
+that#that_conj)
@@ -618 +618 @@
-either))
+either#either_pron))
@@ -620 +620 @@
-could)
+could#can_v1)
@@ -622 +622 @@
-recover))))))
+recover#recover_v1))))))
@@ -631 +631 @@
-Some)
+Some#some_pron)
@@ -633 +633 @@
-letters))
+letters#letter_n1))
@@ -635 +635 @@
-say)
+say#say_v1)
@@ -642 +642 @@
-the)
+the#the_adj)
@@ -644 +644 @@
-eldest))
+eldest#eldest_adj))
@@ -646 +646 @@
-was)
+was#be_v)
@@ -649 +649 @@
-already))
+already#already_adj))
@@ -652 +652 @@
-dead))
+dead#dead_adj))
@@ -657 +657 @@
-tho)
+tho#though_adv)
@@ -666 +666 @@
-wone))
+wone#one_adj))
@@ -668 +668 @@
-can@)
+can@#can_v1)
@@ -670 +670 @@
-@not)
+@not#not_adv)
@@ -672 +672 @@
-say)
+say#say_v1)
@@ -679 +679 @@
-this))
+this#this_pron))
@@ -681 +681 @@
-is)
+is#be_v)
@@ -684 +684 @@
-certain)))))))))
+certain#certain_adj)))))))))
@@ -693 +693 @@
-there))
+there#there_adv))
@@ -695 +695 @@
-are)
+are#be_v)
@@ -699 +699 @@
-so)
+so#so_adv)
@@ -701 +701 @@
-manny))
+manny#many_adj))
@@ -703 +703 @@
-letters)
+letters#letter_n1)
@@ -708 +708 @@
-that)
+that#that_conj)
@@ -713 +713 @@
-speake)
+speake#speak_v)
@@ -716 +716 @@
-of)
+of#of_prep)
@@ -719 +719 @@
-it)))))
+it#it_pron)))))
@@ -722 +722 @@
-y=t=)
+y=t=#that_conj)
@@ -726 +726 @@
-there)
+there#their_adj)
@@ -728 +728 @@
-relations))
+relations#relation_n))
@@ -730 +730 @@
-are)
+are#be_v)
@@ -733 +733 @@
-fearfull)
+fearfull#fearful_adj)
@@ -740 +740 @@
-it))
+it#it_pron))
@@ -742 +742 @@
-is)
+is#be_v)
@@ -745 +745 @@
-too)
+too#too_adv)
@@ -747 +747 @@
-true))))))))
+true#true_adj))))))))
@@ -756 +756 @@
-S=r=)
+S=r=#NA)
@@ -758 +758 @@
-Edward)
+Edward#NA)
@@ -760 +760 @@
-Villars))
+Villars#NA))
@@ -762 +762 @@
-is)
+is#be_v)
@@ -765 +765 @@
-to)
+to#to_prep)
@@ -767 +767 @@
-be)
+be#be_v)
@@ -770 +770 @@
-a)
+a#a_adj2)
@@ -772 +772 @@
-Viscount)))
+Viscount#viscount_n)))
@@ -780 +780 @@
-but)
+but#but_prep)
@@ -783 +783 @@
-wither)
+wither#whether_pron)
@@ -790 +790 @@
-English)))
+English#english_adj)))
@@ -793 +793 @@
-or)
+or#or_conj2)
@@ -796 +796 @@
-no)))))
+no#no_adv2)))))
@@ -799 +799 @@
-I))
+I#i_pron))
@@ -801 +801 @@
-ca@)
+ca@#can_v1)
@@ -803 +803 @@
-@nt)
+@nt#not_adv)
@@ -805 +805 @@
-tell)
+tell#tell_v)
@@ -814 +814 @@
-I))
+I#i_pron))
@@ -816 +816 @@
-begg)
+begg#beg_v)
@@ -819 +819 @@
-the)
+the#the_adj)
@@ -821 +821 @@
-favour)
+favour#favour_n)
@@ -824 +824 @@
-of)
+of#of_prep)
@@ -827 +827 @@
-y=r=)
+y=r=#your_pron)
@@ -829 +829 @@
-Losp.))))
+Losp.#lordship_n))))
@@ -832 +832 @@
-to)
+to#to_prep)
@@ -834 +834 @@
-give)
+give#give_v)
@@ -837 +837 @@
-my)
+my#my_adj)
@@ -839 +839 @@
-hum=ble=)
+hum=ble=#humble_adj)
@@ -841 +841 @@
-dutty))
+dutty#duty_n))
@@ -844 +844 @@
-to)
+to#to_prep)
@@ -847 +847 @@
-my)
+my#my_adj)
@@ -849 +849 @@
-Lady))))
+Lady#lady_n))))
@@ -858 +858 @@
-I))
+I#i_pron))
@@ -860 +860 @@
-am)
+am#be_v)
@@ -864 +864 @@
-y=r=)
+y=r=#your_pron)
@@ -866 +866 @@
-Losp.))
+Losp.#lordship_n))
@@ -869 +869 @@
-most)
+most#most_adj)
@@ -871 +871 @@
-obedient))
+obedient#obedient_adj))
@@ -873 +873 @@
-duttyfull)
+duttyfull#dutiful_adj)
@@ -875 +875 @@
-daughter)
+daughter#daughter_n)
@@ -880 +880 @@
-A.)
+A.#NA)
@@ -882 +882 @@
-Nottingham)))
+Nottingham#NA)))
@@ -896 +896 @@
-Y=e=)
+Y=e=#the_adj)
@@ -898 +898 @@
-31)
+31#NA)
@@ -901 +901 @@
-of)
+of#of_prep)
@@ -904 +904 @@
-De.)))
+De.#NA)))
@@ -908 +908 @@
-1694)
+1694#NA)
@@ -919 +919 @@
-My)
+My#my_adj)
@@ -921 +921 @@
-Lord))
+Lord#lord_n))
@@ -926 +926 @@
-The)
+The#the_adj)
@@ -928 +928 @@
-sad)
+sad#sad_adj)
@@ -930 +930 @@
-confirmation)
+confirmation#confirmation_n)
@@ -942 +942 @@
-last)
+last#last_adv)
@@ -944 +944 @@
-nights))
+nights#night_n))
@@ -946 +946 @@
-letters))
+letters#letter_n1))
@@ -948 +948 @@
-brought)
+brought#bring_v)
@@ -951 +951 @@
-me))))
+me#me_pron1))))
@@ -954 +954 @@
-of)
+of#of_prep)
@@ -957 +957 @@
-the)
+the#the_adj)
@@ -959 +959 @@
-death)
+death#death_n)
@@ -962 +962 @@
-of)
+of#of_prep)
@@ -965 +965 @@
-my)
+my#my_adj)
@@ -968 +968 @@
-most)
+most#most_adj)
@@ -970 +970 @@
-deare))
+deare#dear_adj1))
@@ -972 +972 @@
-mistress)
+mistress#mistress_n)
@@ -977 +977 @@
-the)
+the#the_adj)
@@ -979 +979 @@
-Queen)))))))
+Queen#queen_n)))))))
@@ -983 +983 @@
-gives)
+gives#give_v)
@@ -986 +986 @@
-me))
+me#me_pron1))
@@ -990 +990 @@
-so)
+so#so_adv)
@@ -992 +992 @@
-just))
+just#just_adj))
@@ -994 +994 @@
-an)
+an#a_adj2)
@@ -996 +996 @@
-affliction)
+affliction#affliction_n)
@@ -999 +999 @@
-that)
+that#that_conj)
@@ -1003 +1003 @@
-I))
+I#i_pron))
@@ -1005 +1005 @@
-am)
+am#be_v)
@@ -1009 +1009 @@
-but)
+but#but_prep)
@@ -1011 +1011 @@
-little))
+little#little_adj))
@@ -1013 +1013 @@
-thoughtfull)
+thoughtfull#thoughtful_adj)
@@ -1016 +1016 @@
-of)
+of#of_prep)
@@ -1019 +1019 @@
-my)
+my#my_adj)
@@ -1021 +1021 @@
-neerest)
+neerest#near_adj)
@@ -1023 +1023 @@
-concerns)))))))
+concerns#concern_n)))))))
@@ -1033 +1033 @@
-The)
+The#the_adj)
@@ -1035 +1035 @@
-want)
+want#want_n2)
@@ -1038 +1038 @@
-of)
+of#of_prep)
@@ -1041 +1041 @@
-coach)
+coach#coach_n)
@@ -1043 +1043 @@
-horses))))
+horses#horse_n))))
@@ -1045 +1045 @@
-has)
+has#have_v)
@@ -1050 +1050 @@
-since)
+since#since_adv)
@@ -1054 +1054 @@
-my)
+my#my_adj)
@@ -1056 +1056 @@
-Lords))
+Lords#lord_n))
@@ -1058 +1058 @@
-going)))
+going#going_n)))
@@ -1062 +1062 @@
-hinderd)
+hinderd#hinder_v)
@@ -1065 +1065 @@
-me))
+me#me_pron1))
@@ -1068 +1068 @@
-from)
+from#from_prep)
@@ -1071 +1071 @@
-waitting)
+waitting#wait_v1)
@@ -1074 +1074 @@
-of)
+of#of_prep)
@@ -1077 +1077 @@
-y=r=)
+y=r=#your_pron)
@@ -1079 +1079 @@
-Losp.))))))
+Losp.#lordship_n))))))
@@ -1084 +1084 @@
-and)
+and#and_conj1)
@@ -1088 +1088 @@
-the)
+the#the_adj)
@@ -1090 +1090 @@
-news)
+news#news_n)
@@ -1093 +1093 @@
-this)
+this#this_pron)
@@ -1095 +1095 @@
-last)
+last#last_adv)
@@ -1097 +1097 @@
-week))
+week#week_n))
@@ -1100 +1100 @@
-of)
+of#of_prep)
@@ -1103 +1103 @@
-her)
+her#her_pron2)
@@ -1105 +1105 @@
-illness))))
+illness#illness_n))))
@@ -1108 +1108 @@
-from)
+from#from_prep)
@@ -1111 +1111 @@
-sending)
+sending#send_v1)
@@ -1114 +1114 @@
-to)
+to#to_prep)
@@ -1116 +1116 @@
-know)
+know#know_v)
@@ -1120 +1120 @@
-how))
+how#how_adv))
@@ -1128 +1128 @@
-you))
+you#you_pron))
@@ -1130 +1130 @@
-did))))))))
+did#do_v))))))))
@@ -1139 +1139 @@
-God))
+God#NA))
@@ -1141 +1141 @@
-has)
+has#have_v)
@@ -1143 +1143 @@
-been)
+been#be_v)
@@ -1145 +1145 @@
-pleased)
+pleased#please_v)
@@ -1148 +1148 @@
-to)
+to#to_prep)
@@ -1150 +1150 @@
-shorten)
+shorten#shorten_v)
@@ -1153 +1153 @@
-her)
+her#her_pron2)
@@ -1155 +1155 @@
-days))
+days#day_n))
@@ -1158 +1158 @@
-as)
+as#as_adv)
@@ -1161 +1161 @@
-a)
+a#a_adj2)
@@ -1163 +1163 @@
-reward)
+reward#reward_n)
@@ -1166 +1166 @@
-of)
+of#of_prep)
@@ -1169 +1169 @@
-her)
+her#her_pron2)
@@ -1171 +1171 @@
-sufferings)
+sufferings#suffering_n)
@@ -1177 +1177 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1185 +1185 @@
-in)
+in#in_prep)
@@ -1188 +1188 @@
-this)
+this#this_pron)
@@ -1190 +1190 @@
-world)))
+world#world_n)))
@@ -1192 +1192 @@
-were)
+were#be_v)
@@ -1194 +1194 @@
-not)
+not#not_adv)
@@ -1197 +1197 @@
-slight))))
+slight#slight_adj))))
@@ -1203 +1203 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1213 +1213 @@
-w=th=)
+w=th=#with_prep)
@@ -1216 +1216 @@
-the)
+the#the_adj)
@@ -1218 +1218 @@
-goodness)
+goodness#goodness_n)
@@ -1221 +1221 @@
-of)
+of#of_prep)
@@ -1224 +1224 @@
-her)
+her#her_pron2)
@@ -1226 +1226 @@
-owne)
+owne#own_adj)
@@ -1228 +1228 @@
-nature)))))
+nature#nature_n)))))
@@ -1232 +1232 @@
-had)
+had#have_v)
@@ -1234 +1234 @@
-formed)
+formed#form_v1)
@@ -1237 +1237 @@
-in)
+in#in_prep)
@@ -1240 +1240 @@
-her)))
+her#her_pron1)))
@@ -1244 +1244 @@
-so)
+so#so_adv)
@@ -1246 +1246 @@
-great))
+great#great_adj))
@@ -1248 +1248 @@
-a)
+a#a_adj2)
@@ -1250 +1250 @@
-degree)
+degree#degree_n)
@@ -1253 +1253 @@
-of)
+of#of_prep)
@@ -1256 +1256 @@
-vertu)))
+vertu#virtue_n)))
@@ -1265 +1265 @@
-it))
+it#it_pron))
@@ -1267 +1267 @@
-might)
+might#may_v1)
@@ -1270 +1270 @@
-truely))
+truely#truly_adv))
@@ -1272 +1272 @@
-be)
+be#be_v)
@@ -1274 +1274 @@
-sayd)
+sayd#say_v1)
@@ -1281 +1281 @@
-the)
+the#the_adj)
@@ -1283 +1283 @@
-world))
+world#world_n))
@@ -1285 +1285 @@
-was)
+was#be_v)
@@ -1287 +1287 @@
-not)
+not#not_adv)
@@ -1290 +1290 @@
-worthy)
+worthy#worthy_adj)
@@ -1293 +1293 @@
-of)
+of#of_prep)
@@ -1296 +1296 @@
-her))))))))))))))))
+her#her_pron1))))))))))))))))
@@ -1305 +1305 @@
-A)
+A#a_adj2)
@@ -1307 +1307 @@
-poore)
+poore#poor_adj)
@@ -1309 +1309 @@
-lamentation))
+lamentation#lamentation_n))
@@ -1311 +1311 @@
-is)
+is#be_v)
@@ -1314 +1314 @@
-all)
+all#all_adj)
@@ -1316 +1316 @@
-the)
+the#the_adj)
@@ -1318 +1318 @@
-gratitude)
+gratitude#gratitude_n)
@@ -1323 +1323 @@
-that)
+that#that_conj)
@@ -1328 +1328 @@
-can)
+can#can_v1)
@@ -1330 +1330 @@
-be)
+be#be_v)
@@ -1332 +1332 @@
-payd)
+payd#pay_v1)
@@ -1335 +1335 @@
-her))
+her#her_pron1))
@@ -1338 +1338 @@
-now))
+now#now_adv))
@@ -1341 +1341 @@
-for)
+for#for_prep)
@@ -1344 +1344 @@
-that)
+that#that_pron)
@@ -1346 +1346 @@
-favor)
+favor#favour_n)
@@ -1357 +1357 @@
-so)
+so#so_adv)
@@ -1359 +1359 @@
-undeservedly))
+undeservedly#undeservedly_adv))
@@ -1362 +1362 @@
-she))
+she#she_pron))
@@ -1364 +1364 @@
-bestowed)
+bestowed#bestow_v)
@@ -1367 +1367 @@
-on)
+on#on_prep)
@@ -1370 +1370 @@
-me))))))))))
+me#me_pron1))))))))))
@@ -1380 +1380 @@
-is)
+is#be_v)
@@ -1383 +1383 @@
-all)
+all#all_adj)
@@ -1388 +1388 @@
-that)
+that#that_conj)
@@ -1394 +1394 @@
-now))
+now#now_adv))
@@ -1396 +1396 @@
-can)
+can#can_v1)
@@ -1398 +1398 @@
-be)
+be#be_v)
@@ -1400 +1400 @@
-returned)
+returned#return_v1)
@@ -1403 +1403 @@
-by)
+by#by_prep)
@@ -1408 +1408 @@
-my)
+my#my_adj)
@@ -1410 +1410 @@
-Lord))
+Lord#lord_n))
@@ -1415 +1415 @@
-Y=r=)
+Y=r=#your_pron)
@@ -1418 +1418 @@
-most)
+most#most_adj)
@@ -1420 +1420 @@
-obedient))
+obedient#obedient_adj))
@@ -1424 +1424 @@
-duttyfull)
+duttyfull#dutiful_adj)
@@ -1426 +1426 @@
-daughter)
+daughter#daughter_n)
@@ -1431 +1431 @@
-A.)
+A.#NA)
@@ -1433 +1433 @@
-Nottingham)))))))
+Nottingham#NA)))))))
@@ -1444 +1444 @@
-Twelfth)
+Twelfth#NA)
@@ -1446 +1446 @@
-Day)
+Day#day_n)
@@ -1450 +1450 @@
-1695)
+1695#NA)
@@ -1461 +1461 @@
-My)
+My#my_adj)
@@ -1463 +1463 @@
-Lord))
+Lord#lord_n))
@@ -1468 +1468 @@
-My)
+My#my_adj)
@@ -1470 +1470 @@
-Lord)
+Lord#lord_n)
@@ -1473 +1473 @@
-Nottingham))
+Nottingham#NA))
@@ -1477 +1477 @@
-has)
+has#have_v)
@@ -1481 +1481 @@
-so)
+so#so_adv)
@@ -1483 +1483 @@
-great))
+great#great_adj))
@@ -1485 +1485 @@
-a)
+a#a_adj2)
@@ -1487 +1487 @@
-share)
+share#share_n3)
@@ -1490 +1490 @@
-in)
+in#in_prep)
@@ -1493 +1493 @@
-this)
+this#this_pron)
@@ -1495 +1495 @@
-loss)))
+loss#loss_n1)))
@@ -1500 +1500 @@
-himself))
+himself#himself_pron))
@@ -1505 +1505 @@
-that)
+that#that_conj)
@@ -1509 +1509 @@
-his)
+his#his_adj)
@@ -1511 +1511 @@
-owne)
+owne#own_adj)
@@ -1513 +1513 @@
-affliction))
+affliction#affliction_n))
@@ -1515 +1515 @@
-makes)
+makes#make_v1)
@@ -1519 +1519 @@
-him))
+him#him_pron))
@@ -1521 +1521 @@
-apprehend)
+apprehend#apprehend_v)
@@ -1524 +1524 @@
-mine)))))
+mine#mine_adj)))))
@@ -1533 +1533 @@
-I))
+I#i_pron))
@@ -1535 +1535 @@
-must)
+must#must_v1)
@@ -1537 +1537 @@
-be)
+be#be_v)
@@ -1543 +1543 @@
-excesively)
+excesively#excessively_adv)
@@ -1545 +1545 @@
-ungratefull)
+ungratefull#ungrateful_adj)
@@ -1548 +1548 @@
-to)
+to#to_prep)
@@ -1551 +1551 @@
-that)
+that#that_pron)
@@ -1553 +1553 @@
-incompareable)
+incompareable#incomparable_adj)
@@ -1555 +1555 @@
-Princess)))
+Princess#princess_n)))
@@ -1560 +1560 @@
-to)
+to#to_prep)
@@ -1562 +1562 @@
-be)
+be#be_v)
@@ -1565 +1565 @@
-unconcerned)
+unconcerned#unconcerned_adj)
@@ -1568 +1568 @@
-that)
+that#that_conj)
@@ -1572 +1572 @@
-she))
+she#she_pron))
@@ -1574 +1574 @@
-is)
+is#be_v)
@@ -1576 +1576 @@
-gone))))))
+gone#go_v))))))
@@ -1581 +1581 @@
-and)
+and#and_conj1)
@@ -1585 +1585 @@
-yet)
+yet#yet_adv)
@@ -1587 +1587 @@
-more))
+more#more_adj))
@@ -1589 +1589 @@
-ungratefull)
+ungratefull#ungrateful_adj)
@@ -1592 +1592 @@
-to)
+to#to_prep)
@@ -1595 +1595 @@
-God)))
+God#NA)))
@@ -1600 +1600 @@
-to)
+to#to_prep)
@@ -1602 +1602 @@
-be)
+be#be_v)
@@ -1605 +1605 @@
-insensible)
+insensible#insensible_adj)
@@ -1608 +1608 @@
-of)
+of#of_prep)
@@ -1611 +1611 @@
-those)
+those#those_pron)
@@ -1613 +1613 @@
-many)
+many#many_adj)
@@ -1615 +1615 @@
-blessings)
+blessings#blessing_n)
@@ -1626 +1626 @@
-he))
+he#he_pron))
@@ -1629 +1629 @@
-still))
+still#still_adv))
@@ -1631 +1631 @@
-continues)
+continues#continue_v)
@@ -1634 +1634 @@
-to)
+to#to_prep)
@@ -1637 +1637 @@
-me))))))))))))
+me#me_pron1))))))))))))
@@ -1646 +1646 @@
-I))
+I#i_pron))
@@ -1648 +1648 @@
-can@)
+can@#can_v1)
@@ -1650 +1650 @@
-@not)
+@not#not_adv)
@@ -1652 +1652 @@
-express)
+express#express_v1)
@@ -1655 +1655 @@
-myself))
+myself#myself_pron))
@@ -1658 +1658 @@
-better))
+better#better_adj))
@@ -1661 +1661 @@
-upon)
+upon#upon_prep)
@@ -1664 +1664 @@
-this)
+this#this_pron)
@@ -1666 +1666 @@
-sad)
+sad#sad_adj)
@@ -1668 +1668 @@
-occasion)))
+occasion#occasion_n1)))
@@ -1671 +1671 @@
-then)
+then#than_conj)
@@ -1674 +1674 @@
-as)
+as#as_adv)
@@ -1685 +1685 @@
-the)
+the#the_adj)
@@ -1687 +1687 @@
-Dean)
+Dean#dean_n1)
@@ -1690 +1690 @@
-of)
+of#of_prep)
@@ -1693 +1693 @@
-Pauls))))
+Pauls#NA))))
@@ -1695 +1695 @@
-has)
+has#have_v)
@@ -1698 +1698 @@
-already))
+already#already_adj))
@@ -1700 +1700 @@
-done)))))
+done#do_v)))))
@@ -1710 +1710 @@
-We))
+We#we_pron))
@@ -1712 +1712 @@
-must)
+must#must_v1)
@@ -1714 +1714 @@
-be)
+be#be_v)
@@ -1717 +1717 @@
-dumb)))
+dumb#dumb_adj)))
@@ -1720 +1720 @@
-and)
+and#and_conj1)
@@ -1723 +1723 @@
-not)
+not#not_adv)
@@ -1725 +1725 @@
-open)
+open#open_v)
@@ -1728 +1728 @@
-our)
+our#our_pron)
@@ -1730 +1730 @@
-mouths))))
+mouths#mouth_n))))
@@ -1735 +1735 @@
-because)
+because#because_adv)
@@ -1742 +1742 @@
-God))
+God#NA))
@@ -1744 +1744 @@
-did)
+did#do_v)
@@ -1747 +1747 @@
-it)))))
+it#it_pron)))))
@@ -1756 +1756 @@
-I))
+I#i_pron))
@@ -1758 +1758 @@
-have)
+have#have_v)
@@ -1760 +1760 @@
-obeyed)
+obeyed#obey_v)
@@ -1764 +1764 @@
-y=r=)
+y=r=#your_pron)
@@ -1766 +1766 @@
-Losp.))
+Losp.#lordship_n))
@@ -1768 +1768 @@
-commands)
+commands#command_n)
@@ -1771 +1771 @@
-to)
+to#to_prep)
@@ -1774 +1774 @@
-the)
+the#the_adj)
@@ -1776 +1776 @@
-two)
+two#two_adj)
@@ -1778 +1778 @@
-Lady)
+Lady#NA)
@@ -1780 +1780 @@
-Marys)
+Marys#NA)
@@ -1786 +1786 @@
-who))
+who#who_pron))
@@ -1794 +1794 @@
-give)
+give#give_v)
@@ -1797 +1797 @@
-you))
+you#you_pron))
@@ -1800 +1800 @@
-their)
+their#their_adj)
@@ -1802 +1802 @@
-humble)
+humble#humble_adj)
@@ -1804 +1804 @@
-service)))
+service#service_n1)))
@@ -1807 +1807 @@
-and)
+and#and_conj1)
@@ -1812 +1812 @@
-hope)
+hope#hope_v)
@@ -1819 +1819 @@
-I))
+I#i_pron))
@@ -1821 +1821 @@
-shall)
+shall#shall_v)
@@ -1823 +1823 @@
-return)
+return#return_v1)
@@ -1826 +1826 @@
-from)
+from#from_prep)
@@ -1829 +1829 @@
-London)))
+London#NA)))
@@ -1832 +1832 @@
-so)
+so#so_adv)
@@ -1834 +1834 @@
-soon)
+soon#soon_adv)
@@ -1843 +1843 @@
-they))
+they#they_pron))
@@ -1845 +1845 @@
-may)
+may#may_v1)
@@ -1848 +1848 @@
-w=th=)
+w=th=#with_prep)
@@ -1851 +1851 @@
-me)))
+me#me_pron1)))
@@ -1853 +1853 @@
-waitt)
+waitt#wait_v1)
@@ -1856 +1856 @@
-of)
+of#of_prep)
@@ -1859 +1859 @@
-you)))))))))))))))
+you#you_pron)))))))))))))))
@@ -1868 +1868 @@
-I))
+I#i_pron))
@@ -1870 +1870 @@
-should)
+should#shall_v)
@@ -1872 +1872 @@
-have)
+have#have_v)
@@ -1874 +1874 @@
-been)
+been#be_v)
@@ -1877 +1877 @@
-very)
+very#very_adj)
@@ -1879 +1879 @@
-glad)
+glad#glad_adj)
@@ -1882 +1882 @@
-to)
+to#to_prep)
@@ -1884 +1884 @@
-have)
+have#have_v)
@@ -1886 +1886 @@
-waitted)
+waitted#wait_v1)
@@ -1889 +1889 @@
-of)
+of#of_prep)
@@ -1892 +1892 @@
-you)))
+you#you_pron)))
@@ -1895 +1895 @@
-as)
+as#as_adv)
@@ -1902 +1902 @@
-I))
+I#i_pron))
@@ -1904 +1904 @@
-went)
+went#go_v)
@@ -1906 +1906 @@
-up))))))
+up#up_adv1))))))
@@ -1914 +1914 @@
-but)
+but#but_prep)
@@ -1917 +1917 @@
-my)
+my#my_adj)
@@ -1919 +1919 @@
-coachman))
+coachman#coachman_n))
@@ -1921 +1921 @@
-tells)
+tells#tell_v)
@@ -1928 +1928 @@
-t@))
+t@#it_pron))
@@ -1930 +1930 @@
-@is)
+@is#be_v)
@@ -1933 +1933 @@
-much)
+much#much_adj)
@@ -1935 +1935 @@
-the)
+the#the_adj)
@@ -1937 +1937 @@
-worse)
+worse#worse_adj)
@@ -1939 +1939 @@
-way))))
+way#way_n1))))
@@ -1947 +1947 @@
-and)
+and#and_conj1)
@@ -1953 +1953 @@
-I))
+I#i_pron))
@@ -1955 +1955 @@
-confess))
+confess#confess_v))
@@ -1960 +1960 @@
-upon)
+upon#upon_prep)
@@ -1963 +1963 @@
-this)
+this#this_pron)
@@ -1965 +1965 @@
-occasion)))
+occasion#occasion_n1)))
@@ -1968 +1968 @@
-I))
+I#i_pron))
@@ -1970 +1970 @@
-am)
+am#be_v)
@@ -1973 +1973 @@
-desirous)
+desirous#desirous_adj)
@@ -1976 +1976 @@
-to)
+to#to_prep)
@@ -1978 +1978 @@
-have)
+have#have_v)
@@ -1982 +1982 @@
-as)
+as#as_adv)
@@ -1984 +1984 @@
-few))
+few#few_adj))
@@ -1986 +1986 @@
-delays)
+delays#delay_n)
@@ -1989 +1989 @@
-as)
+as#as_adv)
@@ -2000 +2000 @@
-I))
+I#i_pron))
@@ -2002 +2002 @@
-can)
+can#can_v1)
@@ -2007 +2007 @@
-in)
+in#in_prep)
@@ -2010 +2010 @@
-that)
+that#that_pron)
@@ -2012 +2012 @@
-performance)
+performance#performance_n)
@@ -2015 +2015 @@
-of)
+of#of_prep)
@@ -2018 +2018,3 @@
-my)
+my#my_adj)
+(ADJ
+poore#poor_adj)
@@ -2020,3 +2022 @@
-poore)
-(ADJ
-remaining)
+remaining#remaining_adj)
@@ -2024 +2024 @@
-dutty)
+dutty#duty_n)
@@ -2027 +2027 @@
-to)
+to#to_prep)
@@ -2030 +2030 @@
-my)
+my#my_adj)
@@ -2032 +2032 @@
-deare)
+deare#dear_adj1)
@@ -2034 +2034 @@
-mistress))))))))))
+mistress#mistress_n))))))))))
@@ -2043 +2043 @@
-Essex))
+Essex#NA))
@@ -2046 +2046 @@
-I))
+I#i_pron))
@@ -2048 +2048 @@
-thinke)
+thinke#think_v2)
@@ -2053 +2053 @@
-to)
+to#to_prep)
@@ -2055 +2055 @@
-take)
+take#take_v)
@@ -2058 +2058 @@
-w=th=)
+w=th=#with_prep)
@@ -2061 +2061 @@
-me))))
+me#me_pron1))))
@@ -2069 +2069 @@
-and)
+and#and_conj1)
@@ -2072 +2072 @@
-the)
+the#the_adj)
@@ -2074 +2074 @@
-others))
+others#other_adj))
@@ -2076 +2076 @@
-have)
+have#have_v)
@@ -2078 +2078 @@
-not)
+not#not_adv)
@@ -2081 +2081 @@
-yet))
+yet#yet_adv))
@@ -2083 +2083 @@
-got)
+got#get_v)
@@ -2086 +2086 @@
-there)
+there#their_adj)
@@ -2088 +2088 @@
-mourning))
+mourning#mourning_n1))
@@ -2097 +2097 @@
-At)
+At#at_prep)
@@ -2100 +2100 @@
-my)
+my#my_adj)
@@ -2102 +2102 @@
-return)))
+return#return_n)))
@@ -2105 +2105 @@
-I))
+I#i_pron))
@@ -2107 +2107 @@
-hope)
+hope#hope_v)
@@ -2114 +2114 @@
-they))
+they#they_pron))
@@ -2116 +2116 @@
-may)
+may#may_v1)
@@ -2118 +2118 @@
-waitt)
+waitt#wait_v1)
@@ -2121 +2121 @@
-of)
+of#of_prep)
@@ -2124 +2124 @@
-you)))
+you#you_pron)))
@@ -2129 +2129 @@
-w=th=)
+w=th=#with_prep)
@@ -2133 +2133 @@
-y=r=)
+y=r=#your_pron)
@@ -2135 +2135 @@
-Losp.))
+Losp.#lordship_n))
@@ -2138 +2138 @@
-most)
+most#most_adj)
@@ -2140 +2140 @@
-obedient))
+obedient#obedient_adj))
@@ -2144 +2144 @@
-duttyfull)
+duttyfull#dutiful_adj)
@@ -2146 +2146 @@
-daughter)
+daughter#daughter_n)
@@ -2151 +2151 @@
-A.)
+A.#NA)
@@ -2153 +2153 @@
-Nottingham))))))
+Nottingham#NA))))))
