--- charles-1676-e3-p2.psd.vert	2025-09-10 13:51:17
+++ charles-1676-e3-p2.lem.vert	2025-09-10 13:51:25
@@ -13 +13 @@
-CHARLES)
+CHARLES#NA)
@@ -15 +15 @@
-R.))
+R.#NA))
@@ -22 +22 @@
-As)
+As#as_adv)
@@ -25 +25 @@
-to)
+to#to_prep)
@@ -28 +28 @@
-ye)
+ye#the_adj)
@@ -30 +30 @@
-matter)
+matter#matter_n1)
@@ -33 +33 @@
-of)
+of#of_prep)
@@ -36 +36 @@
-regulating)
+regulating#regulate_v)
@@ -39 +39 @@
-Corporations)))))))
+Corporations#corporation_n)))))))
@@ -44 +44 @@
-wee))
+wee#we_pron))
@@ -46 +46 @@
-have)
+have#have_v)
@@ -48 +48 @@
-considered)
+considered#consider_v)
@@ -51 +51 @@
-of)
+of#of_prep)
@@ -54 +54 @@
-y=e=)
+y=e=#the_adj)
@@ -56 +56 @@
-principall)
+principall#principal_adj)
@@ -58 +58 @@
-points)
+points#point_n1)
@@ -61 +61 @@
-represented)
+represented#represent_v)
@@ -64 +64 @@
-by)
+by#by_prep)
@@ -67 +67 @@
-you)))
+you#you_pron)))
@@ -73 +73 @@
-to)
+to#to_prep)
@@ -75 +75 @@
-have)
+have#have_v)
@@ -77 +77 @@
-beene)
+beene#be_v)
@@ -79 +79 @@
-drawne)
+drawne#draw_v)
@@ -81 +81 @@
-up))
+up#up_adv1))
@@ -84 +84 @@
-&)
+&#and_conj1)
@@ -87 +87 @@
-reported)
+reported#report_v)
@@ -90 +90 @@
-to)
+to#to_prep)
@@ -93 +93 @@
-y=e=)
+y=e=#the_adj)
@@ -95 +95 @@
-Councell)))
+Councell#council_n)))
@@ -98 +98 @@
-by)
+by#by_prep)
@@ -101 +101 @@
-y=e=)
+y=e=#the_adj)
@@ -103 +103 @@
-Comittee)
+Comittee#committee_n.II)
@@ -106 +106 @@
-appointed)
+appointed#appoint_v)
@@ -109 +109 @@
-in)
+in#in_prep)
@@ -112 +112 @@
-that)
+that#that_pron)
@@ -114 +114 @@
-businesse))))))))))))
+businesse#business_n))))))))))))
@@ -122 +122 @@
-&)
+&#and_conj1)
@@ -126 +126 @@
-are)
+are#be_v)
@@ -129 +129 @@
-of)
+of#of_prep)
@@ -132 +132 @@
-opinion)
+opinion#opinion_n)
@@ -135 +135 @@
-&)
+&#and_conj1)
@@ -138 +138 @@
-accordingly))
+accordingly#accordingly_adv))
@@ -141 +141 @@
-Our)
+Our#our_pron)
@@ -143 +143 @@
-Pleasure))
+Pleasure#pleasure_n))
@@ -145 +145 @@
-is))
+is#be_v))
@@ -150 +150 @@
-That)
+That#that_conj)
@@ -154 +154 @@
-as)
+as#as_adv)
@@ -157 +157 @@
-to)
+to#to_prep)
@@ -160 +160 @@
-that)
+that#that_pron)
@@ -163 +163 @@
-concerning)
+concerning#concerning_prep)
@@ -166 +166 @@
-y=e=)
+y=e=#the_adj)
@@ -168 +168 @@
-Oath)
+Oath#oath_n)
@@ -171 +171 @@
-of)
+of#of_prep)
@@ -174 +174 @@
-Supremacy)))
+Supremacy#supremacy_n)))
@@ -182 +182 @@
-to)
+to#to_prep)
@@ -184 +184 @@
-be)
+be#be_v)
@@ -186 +186 @@
-taken)
+taken#take_v)
@@ -189 +189 @@
-in)
+in#in_prep)
@@ -192 +192 @@
-all)
+all#all_adj)
@@ -194 +194 @@
-Corporations))))))))))
+Corporations#corporation_n))))))))))
@@ -197 +197 @@
-it))
+it#it_pron))
@@ -199 +199 @@
-bee)
+bee#be_v)
@@ -201 +201 @@
-fixed)
+fixed#fix_v)
@@ -204 +204 @@
-as)
+as#as_adv)
@@ -207 +207 @@
-a)
+a#a_adj2)
@@ -209 +209 @@
-Generall)
+Generall#general_adj)
@@ -211 +211 @@
-rule)))
+rule#rule_n1)))
@@ -216 +216 @@
-that)
+that#that_conj)
@@ -220 +220 @@
-noe)
+noe#no_adj)
@@ -222 +222 @@
-person)
+person#person_n)
@@ -227 +227 @@
-whatsoever)))))
+whatsoever#whatsoever_pron)))))
@@ -229 +229 @@
-bee)
+bee#be_v)
@@ -231 +231 @@
-admitted)
+admitted#admit_v)
@@ -234 +234 @@
-into)
+into#into_prep)
@@ -237 +237 @@
-any)
+any#any_adj)
@@ -239 +239 @@
-Place)
+Place#place_n.II-VII)
@@ -242 +242 @@
-of)
+of#of_prep)
@@ -245 +245 @@
-Magistrature)
+Magistrature#magistrature_n)
@@ -247 +247 @@
-or)
+or#or_conj2)
@@ -249 +249 @@
-Government)))
+Government#government_n)))
@@ -252 +252 @@
-in)
+in#in_prep)
@@ -255 +255 @@
-any)
+any#any_adj)
@@ -257 +257 @@
-Corporation)
+Corporation#corporation_n)
@@ -260 +260 @@
-within)
+within#within_adv)
@@ -263 +263 @@
-that)
+that#that_pron)
@@ -265 +265 @@
-Our)
+Our#our_pron)
@@ -267 +267 @@
-Kingdome)))))))
+Kingdome#kingdom_n)))))))
@@ -270 +270 @@
-till)
+till#till_prep)
@@ -277 +277 @@
-hee))
+hee#he_pron))
@@ -279 +279 @@
-shall)
+shall#shall_v)
@@ -281 +281 @@
-have)
+have#have_v)
@@ -284 +284 @@
-first))
+first#first_adj))
@@ -286 +286 @@
-taken)
+taken#take_v)
@@ -289 +289 @@
-y=e=)
+y=e=#the_adj)
@@ -291 +291 @@
-Oath)
+Oath#oath_n)
@@ -294 +294 @@
-of)
+of#of_prep)
@@ -297 +297 @@
-Supremacy)))))))
+Supremacy#supremacy_n)))))))
@@ -302 +302 @@
-except)
+except#except_adj)
@@ -305 +305 @@
-such)
+such#such_adj)
@@ -308 +308 @@
-as)
+as#as_adv)
@@ -317 +317 @@
-Our)
+Our#our_pron)
@@ -319 +319 @@
-Lieutenant)
+Lieutenant#lieutenant_n)
@@ -322 +322 @@
-for)
+for#for_prep)
@@ -325 +325 @@
-y=e=)
+y=e=#the_adj)
@@ -327 +327 @@
-time)
+time#time_n)
@@ -330 +330 @@
-being))))
+being#be_v))))
@@ -335 +335 @@
-or)
+or#or_conj2)
@@ -338 +338 @@
-other)
+other#other_adj)
@@ -340 +340 @@
-Chiefe)
+Chiefe#chief_adj)
@@ -342 +342 @@
-Governor)
+Governor#governor_n)
@@ -345 +345 @@
-for)
+for#for_prep)
@@ -348 +348 @@
-y=e=)
+y=e=#the_adj)
@@ -350 +350 @@
-time)
+time#time_n)
@@ -353 +353 @@
-being)))))))
+being#be_v)))))))
@@ -355 +355 @@
-shall)
+shall#shall_v)
@@ -357 +357 @@
-think)
+think#think_v2)
@@ -363 +363 @@
-fitt)
+fitt#fit_adj)
@@ -370 +370 @@
-for)
+for#for_prep)
@@ -373 +373 @@
-some)
+some#some_pron)
@@ -375 +375 @@
-particular)
+particular#particular_adj)
@@ -377 +377 @@
-reasons)))
+reasons#reason_n1)))
@@ -379 +379 @@
-to)
+to#to_prep)
@@ -381 +381 @@
-dispense)
+dispense#dispense_v)
@@ -384 +384 @@
-w=th=)
+w=th=#with_prep)
@@ -389 +389 @@
-in)
+in#in_prep)
@@ -392 +392 @@
-that)
+that#that_pron)
@@ -394 +394 @@
-behalfe))))))))))))))))))
+behalfe#behalf_n))))))))))))))))))
@@ -402 +402 @@
-&)
+&#and_conj1)
@@ -405 +405 @@
-suitable)
+suitable#suitable_adj)
@@ -408 +408 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-this))))
+this#this_pron))))
@@ -414 +414 @@
-Our)
+Our#our_pron)
@@ -416 +416 @@
-Pleasure))
+Pleasure#pleasure_n))
@@ -418 +418 @@
-is)
+is#be_v)
@@ -423 +423 @@
-that)
+that#that_conj)
@@ -427 +427 @@
-you))
+you#you_pron))
@@ -429 +429 @@
-doe)
+doe#do_v)
@@ -433 +433 @@
-even)
+even#even_adj)
@@ -435 +435 @@
-forthwi=th=))
+forthwi=th=#forthwith_adv))
@@ -438 +438 @@
-&)
+&#and_conj1)
@@ -441 +441 @@
-at)
+at#at_prep)
@@ -444 +444 @@
-y=e=)
+y=e=#the_adj)
@@ -446 +446 @@
-same)
+same#same_adj)
@@ -448 +448 @@
-time)
+time#time_n)
@@ -453 +453 @@
-that)
+that#that_conj)
@@ -459 +459 @@
-you))
+you#you_pron))
@@ -461 +461 @@
-shall)
+shall#shall_v)
@@ -463 +463 @@
-execute)
+execute#execute_v)
@@ -466 +466 @@
-this)
+this#this_pron)
@@ -468 +468 @@
-rule))
+rule#rule_n1))
@@ -471 +471 @@
-in)
+in#in_prep)
@@ -474 +474 @@
-y=e=)
+y=e=#the_adj)
@@ -476 +476 @@
-generall)))
+generall#general_adj)))
@@ -479 +479 @@
-towards)
+towards#towards_prep)
@@ -482 +482 @@
-Corporac~ons)))))))))
+Corporac~ons#corporation_n)))))))))
@@ -484 +484 @@
-make)
+make#make_v1)
@@ -487 +487 @@
-use)
+use#use_n)
@@ -490 +490 @@
-of)
+of#of_prep)
@@ -493 +493 @@
-y=e=)
+y=e=#the_adj)
@@ -495 +495 @@
-Power)
+Power#power_n1)
@@ -498 +498 @@
-of)
+of#of_prep)
@@ -501 +501 @@
-dispensing)
+dispensing#dispense_v)
@@ -504 +504 @@
-in)
+in#in_prep)
@@ -507 +507 @@
-this)
+this#this_pron)
@@ -509 +509 @@
-particular)
+particular#particular_adj)
@@ -512 +512 @@
-of)
+of#of_prep)
@@ -515 +515 @@
-y=e=)
+y=e=#the_adj)
@@ -517 +517 @@
-Oaths)))))
+Oaths#oath_n)))))
@@ -520 +520 @@
-with)
+with#with_prep)
@@ -523 +523 @@
-such)
+such#such_adj)
@@ -525 +525 @@
-persons)
+persons#person_n)
@@ -528 +528 @@
-in)
+in#in_prep)
@@ -531 +531 @@
-any)
+any#any_adj)
@@ -533 +533 @@
-Corporations)))
+Corporations#corporation_n)))
@@ -536 +536 @@
-as)
+as#as_adv)
@@ -545 +545 @@
-you))
+you#you_pron))
@@ -547 +547 @@
-shall)
+shall#shall_v)
@@ -549 +549 @@
-find)
+find#find_v)
@@ -554 +554 @@
-qualified)
+qualified#qualify_v)
@@ -557 +557 @@
-for)
+for#for_prep)
@@ -560 +560 @@
-that)
+that#that_pron)
@@ -562 +562 @@
-grace)))
+grace#grace_n)))
@@ -565 +565 @@
-by)
+by#by_prep)
@@ -568 +568 @@
-their)
+their#their_adj)
@@ -570 +570 @@
-Loyalty)
+Loyalty#loyalty_n)
@@ -576 +576 @@
-Sobriety)))
+Sobriety#sobriety_n)))
@@ -582 +582 @@
-prudence)))
+prudence#prudence_n)))
@@ -588 +588 @@
-estate)
+estate#estate_n)
@@ -591 +591 @@
-in)
+in#in_prep)
@@ -594 +594 @@
-trading)))))
+trading#trading_n)))))
@@ -599 +599 @@
-or)
+or#or_conj2)
@@ -602 +602 @@
-other)
+other#other_adj)
@@ -604 +604 @@
-particular)
+particular#particular_adj)
@@ -606 +606 @@
-considerations))))))))))))))))))
+considerations#consideration_n))))))))))))))))))
@@ -614 +614 @@
-And)
+And#and_conj1)
@@ -617 +617 @@
-as)
+as#as_adv)
@@ -620 +620 @@
-to)
+to#to_prep)
@@ -623 +623 @@
-y=e=)
+y=e=#the_adj)
@@ -625 +625 @@
-encouragement)
+encouragement#encouragement_n)
@@ -633 +633 @@
-to)
+to#to_prep)
@@ -635 +635 @@
-bee)
+bee#be_v)
@@ -637 +637 @@
-given)
+given#give_v)
@@ -640 +640 @@
-to)
+to#to_prep)
@@ -643 +643 @@
-Forreigners)))))
+Forreigners#foreigner_n)))))
@@ -647 +647 @@
-to)
+to#to_prep)
@@ -649 +649 @@
-come))
+come#come_v))
@@ -652 +652 @@
-&)
+&#and_conj1)
@@ -657 +657 @@
-settle)
+settle#settle_v)
@@ -660 +660 @@
-themselves))
+themselves#themselves_pron))
@@ -663 +663 @@
-in)
+in#in_prep)
@@ -666 +666 @@
-that)
+that#that_pron)
@@ -668 +668 @@
-Our)
+Our#our_pron)
@@ -670 +670 @@
-Kingdome)))))))))
+Kingdome#kingdom_n)))))))))
@@ -675 +675 @@
-wee))
+wee#we_pron))
@@ -677 +677 @@
-think)
+think#think_v2)
@@ -681 +681 @@
-it))
+it#it_pron))
@@ -684 +684 @@
-of)
+of#of_prep)
@@ -687 +687 @@
-great)
+great#great_adj)
@@ -689 +689 @@
-advantage)))
+advantage#advantage_n)))
@@ -692 +692 @@
-to)
+to#to_prep)
@@ -695 +695 @@
-Our)
+Our#our_pron)
@@ -697 +697 @@
-said)
+said#said_adj)
@@ -699 +699 @@
-Kingdome))))
+Kingdome#kingdom_n))))
@@ -707 +707 @@
-&)
+&#and_conj1)
@@ -710 +710 @@
-accordingly))
+accordingly#accordingly_adv))
@@ -713 +713 @@
-Our)
+Our#our_pron)
@@ -715 +715 @@
-Pleasure))
+Pleasure#pleasure_n))
@@ -717 +717 @@
-is)
+is#be_v)
@@ -722 +722 @@
-That)
+That#that_conj)
@@ -726 +726 @@
-you))
+you#you_pron))
@@ -728 +728 @@
-doe)
+doe#do_v)
@@ -732 +732 @@
-take)
+take#take_v)
@@ -735 +735 @@
-care))
+care#care_n1))
@@ -738 +738 @@
-that)
+that#that_conj)
@@ -742 +742 @@
-all)
+all#all_adj)
@@ -744 +744 @@
-Forreigners)
+Forreigners#foreigner_n)
@@ -750 +750 @@
-as)
+as#as_adv)
@@ -752 +752 @@
-well)
+well#well_adv)
@@ -757 +757 @@
-Protestants))
+Protestants#NA))
@@ -760 +760 @@
-as)
+as#as_adv)
@@ -763 +763 @@
-others)))))
+others#other_adj)))))
@@ -767 +767 @@
-be)
+be#be_v)
@@ -769 +769 @@
-admitted)
+admitted#admit_v)
@@ -772 +772 @@
-into)
+into#into_prep)
@@ -775 +775 @@
-Corporac~ons)))))))
+Corporac~ons#corporation_n)))))))
@@ -783 +783 @@
-&)
+&#and_conj1)
@@ -786 +786 @@
-this))
+this#this_pron))
@@ -788 +788 @@
-is)
+is#be_v)
@@ -793 +793 @@
-what))
+what#what_pron))
@@ -799 +799 @@
-wee))
+wee#we_pron))
@@ -801 +801 @@
-think)
+think#think_v2)
@@ -807 +807 @@
-most)
+most#most_adj)
@@ -809 +809 @@
-necessary)
+necessary#necessary_adj)
@@ -812 +812 @@
-to)
+to#to_prep)
@@ -814 +814 @@
-be)
+be#be_v)
@@ -816 +816 @@
-signified)
+signified#signify_v)
@@ -819 +819 @@
-to)
+to#to_prep)
@@ -822 +822 @@
-you)))
+you#you_pron)))
@@ -825 +825 @@
-at)
+at#at_prep)
@@ -828 +828 @@
-y=e=)
+y=e=#the_adj)
@@ -830 +830 @@
-present)))
+present#present_adj)))
@@ -833 +833 @@
-in)
+in#in_prep)
@@ -836 +836 @@
-those)
+those#those_pron)
@@ -838 +838 @@
-matters)))))))))
+matters#matter_n1)))))))))
@@ -846 +846 @@
-Given)
+Given#give_v)
@@ -849 +849 @@
-at)
+at#at_prep)
@@ -852 +852 @@
-Our)
+Our#our_pron)
@@ -854 +854 @@
-Court)
+Court#court_n1)
@@ -857 +857 @@
-at)
+at#at_prep)
@@ -860 +860 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -863 +863 @@
-y=e=)
+y=e=#the_adj)
@@ -865 +865 @@
-31=st=)
+31=st=#NA)
@@ -867 +867 @@
-day)
+day#day_n)
@@ -870 +870 @@
-of)
+of#of_prep)
@@ -873 +873 @@
-August)))
+August#NA)))
@@ -877 +877 @@
-1672))
+1672#NA))
@@ -882 +882 @@
-in)
+in#in_prep)
@@ -885 +885 @@
-y=e=)
+y=e=#the_adj)
@@ -887 +887 @@
-24=th=)
+24=th=#NA)
@@ -889 +889 @@
-year)
+year#year_n)
@@ -892 +892 @@
-of)
+of#of_prep)
@@ -895 +895 @@
-Our)
+Our#our_pron)
@@ -897 +897 @@
-Reigne)))))
+Reigne#reign_n)))))
@@ -911 +911 @@
-RULES)
+RULES#rule_n1)
@@ -913 +913 @@
-AND)
+AND#and_conj1)
@@ -915 +915 @@
-INSTRUCTIONS)
+INSTRUCTIONS#instruction_n)
@@ -918 +918 @@
-FOR)
+FOR#for_prep)
@@ -921 +921 @@
-OUR)
+OUR#our_pron)
@@ -925 +925 @@
-RIGHT)
+RIGHT#right_adv)
@@ -927 +927 @@
-TRUSTY))
+TRUSTY#trusty_adj))
@@ -930 +930 @@
-AND)
+AND#and_conj1)
@@ -933 +933 @@
-RIGHT)
+RIGHT#right_adv)
@@ -935 +935 @@
-WELLBELOVED))))
+WELLBELOVED#well-beloved_adj))))
@@ -937 +937 @@
-COUSIN)
+COUSIN#cousin_n)
@@ -940 +940 @@
-AND)
+AND#and_conj1)
@@ -943 +943 @@
-COUNSELLOR)))
+COUNSELLOR#counsellor_n)))
@@ -946 +946 @@
-ARTHUR)
+ARTHUR#NA)
@@ -949 +949 @@
-EARL)
+EARL#earl_n)
@@ -952 +952 @@
-OF)
+OF#of_prep)
@@ -955 +955 @@
-ESSEX))))
+ESSEX#NA))))
@@ -958 +958 @@
-OUR)
+OUR#our_pron)
@@ -960 +960 @@
-LIEUTENANT)
+LIEUTENANT#lieutenant_n)
@@ -962 +962 @@
-GOVERNOUR)
+GOVERNOUR#governor_n)
@@ -965 +965 @@
-OF)
+OF#of_prep)
@@ -968 +968 @@
-OUR)
+OUR#our_pron)
@@ -970 +970 @@
-KINGDOM)
+KINGDOM#kingdom_n)
@@ -973 +973 @@
-OF)
+OF#of_prep)
@@ -976 +976 @@
-IRELAND))))))))))
+IRELAND#NA))))))))))
@@ -982 +982 @@
-1)
+1#NA)
@@ -989 +989 @@
-Our)
+Our#our_pron)
@@ -991 +991 @@
-pleasure))
+pleasure#pleasure_n))
@@ -993 +993 @@
-is)
+is#be_v)
@@ -998 +998 @@
-and)
+and#and_conj1)
@@ -1001 +1001 @@
-wee))
+wee#we_pron))
@@ -1003 +1003 @@
-doe)
+doe#do_v)
@@ -1005 +1005 @@
-Declare))
+Declare#declare_v))
@@ -1008 +1008 @@
-that)
+that#that_conj)
@@ -1012 +1012 @@
-we))
+we#we_pron))
@@ -1014 +1014 @@
-will)
+will#will_v1)
@@ -1016 +1016 @@
-Signe)
+Signe#sign_v1)
@@ -1019 +1019 @@
-no)
+no#no_adj)
@@ -1021 +1021 @@
-Letters)
+Letters#letter_n1)
@@ -1024 +1024 @@
-for)
+for#for_prep)
@@ -1028 +1028 @@
-the)
+the#the_adj)
@@ -1030 +1030 @@
-granting)
+granting#granting_n)
@@ -1033 +1033 @@
-of)
+of#of_prep)
@@ -1036 +1036 @@
-any)
+any#any_adj)
@@ -1038 +1038 @@
-Money)
+Money#money_n)
@@ -1041 +1041 @@
-or)
+or#or_conj2)
@@ -1044 +1044 @@
-Lands))))))
+Lands#land_n1))))))
@@ -1047 +1047 @@
-or)
+or#or_conj2)
@@ -1051 +1051 @@
-Releasing)
+Releasing#release_v1)
@@ -1053 +1053 @@
-or)
+or#or_conj2)
@@ -1055 +1055 @@
-abating))
+abating#abate_v1))
@@ -1058 +1058 @@
-any)
+any#any_adj)
@@ -1060 +1060 @@
-Rents)
+Rents#rent_n1)
@@ -1063 +1063 @@
-or)
+or#or_conj2)
@@ -1066 +1066 @@
-other)
+other#other_adj)
@@ -1068 +1068 @@
-summs)
+summs#sum_n1)
@@ -1071 +1071 @@
-of)
+of#of_prep)
@@ -1074 +1074 @@
-money)))))
+money#money_n)))))
@@ -1077 +1077 @@
-due)
+due#due_adj)
@@ -1080 +1080 @@
-to)
+to#to_prep)
@@ -1083 +1083 @@
-us)))
+us#us_pron)))
@@ -1086 +1086 @@
-in)
+in#in_prep)
@@ -1089 +1089 @@
-that)
+that#that_pron)
@@ -1091 +1091 @@
-our)
+our#our_pron)
@@ -1093 +1093 @@
-Kingdome)
+Kingdome#kingdom_n)
@@ -1096 +1096 @@
-of)
+of#of_prep)
@@ -1099 +1099 @@
-Ireland))))))))))))
+Ireland#NA))))))))))))
@@ -1104 +1104 @@
-but)
+but#but_prep)
@@ -1107 +1107 @@
-upon)
+upon#upon_prep)
@@ -1110 +1110 @@
-a)
+a#a_adj2)
@@ -1112 +1112 @@
-petition)
+petition#petition_n)
@@ -1115 +1115 @@
-presented)
+presented#present_v)
@@ -1118 +1118 @@
-to)
+to#to_prep)
@@ -1121 +1121 @@
-us)))))))))
+us#us_pron)))))))))
@@ -1125 +1125 @@
-which)
+which#which_adj)
@@ -1127 +1127 @@
-petition))
+petition#petition_n))
@@ -1135 +1135 @@
-is)
+is#be_v)
@@ -1138 +1138 @@
-to)
+to#to_prep)
@@ -1140 +1140 @@
-be)
+be#be_v)
@@ -1143 +1143 @@
-first))
+first#first_adj))
@@ -1145 +1145 @@
-either)
+either#either_pron)
@@ -1147 +1147 @@
-recommended)
+recommended#recommend_v1)
@@ -1150 +1150 @@
-by)
+by#by_prep)
@@ -1153 +1153 @@
-you)
+you#you_pron)
@@ -1156 +1156 @@
-our)
+our#our_pron)
@@ -1158 +1158 @@
-Lieutenant))))))
+Lieutenant#lieutenant_n))))))
@@ -1161 +1161 @@
-or)
+or#or_conj2)
@@ -1165 +1165 @@
-the)
+the#the_adj)
@@ -1167 +1167 @@
-same))
+same#same_adj))
@@ -1169 +1169 @@
-is)
+is#be_v)
@@ -1172 +1172 @@
-to)
+to#to_prep)
@@ -1174 +1174 @@
-be)
+be#be_v)
@@ -1176 +1176 @@
-transmitted)
+transmitted#transmit_v)
@@ -1179 +1179 @@
-unto)
+unto#unto_prep)
@@ -1182 +1182 @@
-you)))
+you#you_pron)))
@@ -1185 +1185 @@
-by)
+by#by_prep)
@@ -1188 +1188 @@
-one)
+one#one_adj)
@@ -1191 +1191 @@
-of)
+of#of_prep)
@@ -1194 +1194 @@
-our)
+our#our_pron)
@@ -1196 +1196 @@
-Principall)
+Principall#principal_adj)
@@ -1198 +1198 @@
-Secretaries)
+Secretaries#secretary_n1)
@@ -1201 +1201 @@
-of)
+of#of_prep)
@@ -1204 +1204 @@
-State))))))))))
+State#state_n))))))))))
@@ -1207 +1207 @@
-and)
+and#and_conj1)
@@ -1211 +1211 @@
-your)
+your#your_pron)
@@ -1213 +1213 @@
-sense)
+sense#sense_n)
@@ -1216 +1216 @@
-and)
+and#and_conj1)
@@ -1219 +1219 @@
-opinion))))
+opinion#opinion_n))))
@@ -1221 +1221 @@
-to)
+to#to_prep)
@@ -1223 +1223 @@
-be)
+be#be_v)
@@ -1225 +1225 @@
-had)
+had#have_v)
@@ -1228 +1228 @@
-before)
+before#before_adv)
@@ -1235 +1235 @@
-any)
+any#any_adj)
@@ -1237 +1237 @@
-order)
+order#order_n)
@@ -1240 +1240 @@
-or)
+or#or_conj2)
@@ -1243 +1243 @@
-Letter)))
+Letter#letter_n1)))
@@ -1247 +1247 @@
-be)
+be#be_v)
@@ -1249 +1249 @@
-signed)
+signed#sign_v1)
@@ -1252 +1252 @@
-thereupon)))))))))
+thereupon#thereupon_adv)))))))))
@@ -1260 +1260 @@
-2)
+2#NA)
@@ -1264 +1264 @@
-That)
+That#that_conj)
@@ -1268 +1268 @@
-afterwards))
+afterwards#afterwards_adv))
@@ -1271 +1271 @@
-such)
+such#such_adj)
@@ -1273 +1273 @@
-Peticion))
+Peticion#petition_n))
@@ -1275 +1275 @@
-shall)
+shall#shall_v)
@@ -1277 +1277 @@
-be)
+be#be_v)
@@ -1279 +1279 @@
-referred)
+referred#refer_v)
@@ -1282 +1282 @@
-to)
+to#to_prep)
@@ -1285 +1285 @@
-our)
+our#our_pron)
@@ -1287 +1287 @@
-High)
+High#high_adj)
@@ -1289 +1289 @@
-Treasurer)
+Treasurer#treasurer_n)
@@ -1292 +1292 @@
-of)
+of#of_prep)
@@ -1295 +1295 @@
-England)))
+England#NA)))
@@ -1301 +1301 @@
-who))
+who#who_pron))
@@ -1308 +1308 @@
-is)
+is#be_v)
@@ -1311 +1311 @@
-to)
+to#to_prep)
@@ -1313 +1313 @@
-be)
+be#be_v)
@@ -1315 +1315 @@
-made)
+made#make_v1)
@@ -1320 +1320 @@
-acquainted)
+acquainted#acquaint_v)
@@ -1323 +1323 @@
-with)
+with#with_prep)
@@ -1328 +1328 @@
-what))
+what#what_pron))
@@ -1336 +1336 @@
-you)
+you#you_pron)
@@ -1339 +1339 @@
-Our)
+Our#our_pron)
@@ -1341 +1341 @@
-Lieutenant)))
+Lieutenant#lieutenant_n)))
@@ -1343 +1343 @@
-shall)
+shall#shall_v)
@@ -1345 +1345 @@
-write)))))))
+write#write_v)))))))
@@ -1350 +1350 @@
-either)
+either#either_pron)
@@ -1353 +1353 @@
-in)
+in#in_prep)
@@ -1356 +1356 @@
-recommending)
+recommending#recommend_v1)
@@ -1359 +1359 @@
-such)
+such#such_adj)
@@ -1361 +1361 @@
-Peticion))))
+Peticion#petition_n))))
@@ -1366 +1366 @@
-or)
+or#or_conj2)
@@ -1369 +1369 @@
-upon)
+upon#upon_prep)
@@ -1372 +1372 @@
-transmission)
+transmission#transmission_n)
@@ -1375 +1375 @@
-of)
+of#of_prep)
@@ -1378 +1378 @@
-any)
+any#any_adj)
@@ -1380 +1380 @@
-such)
+such#such_adj)
@@ -1383 +1383 @@
-unto)
+unto#unto_prep)
@@ -1386 +1386 @@
-you))))))))))))))
+you#you_pron))))))))))))))
@@ -1394 +1394 @@
-And)
+And#and_conj1)
@@ -1398 +1398 @@
-our)
+our#our_pron)
@@ -1400 +1400 @@
-said)
+said#said_adj)
@@ -1402 +1402 @@
-High)
+High#high_adj)
@@ -1404 +1404 @@
-Treasurer's))
+Treasurer's#treasurer_n))
@@ -1406 +1406 @@
-Report)
+Report#report_n)
@@ -1410 +1410 @@
-is)
+is#be_v)
@@ -1413 +1413 @@
-to)
+to#to_prep)
@@ -1415 +1415 @@
-be)
+be#be_v)
@@ -1417 +1417 @@
-had))
+had#have_v))
@@ -1420 +1420 @@
-thereupon))
+thereupon#thereupon_adv))
@@ -1428 +1428 @@
-3)
+3#NA)
@@ -1433 +1433 @@
-You))
+You#you_pron))
@@ -1435 +1435 @@
-shall)
+shall#shall_v)
@@ -1437 +1437 @@
-give)
+give#give_v)
@@ -1440 +1440 @@
-no)
+no#no_adj)
@@ -1442 +1442 @@
-orders))
+orders#order_n))
@@ -1445 +1445 @@
-upon)
+upon#upon_prep)
@@ -1448 +1448 @@
-any)
+any#any_adj)
@@ -1450 +1450 @@
-Letters)
+Letters#letter_n1)
@@ -1453 +1453 @@
-for)
+for#for_prep)
@@ -1456 +1456 @@
-granting)
+granting#grant_v)
@@ -1459 +1459 @@
-money)
+money#money_n)
@@ -1461 +1461 @@
-Lands))))
+Lands#land_n1))))
@@ -1467 +1467 @@
-Releasing)
+Releasing#release_v1)
@@ -1469 +1469 @@
-or)
+or#or_conj2)
@@ -1471 +1471 @@
-Abating))
+Abating#abate_v1))
@@ -1474 +1474 @@
-Rents))
+Rents#rent_n1))
@@ -1477 +1477 @@
-in)
+in#in_prep)
@@ -1480 +1480 @@
-Ireland))))
+Ireland#NA))))
@@ -1487 +1487 @@
-which))
+which#which_adj))
@@ -1494 +1494 @@
-shall)
+shall#shall_v)
@@ -1496 +1496 @@
-come)
+come#come_v)
@@ -1501 +1501 @@
-from)
+from#from_prep)
@@ -1504 +1504 @@
-us)))
+us#us_pron)))
@@ -1507 +1507 @@
-in)
+in#in_prep)
@@ -1510 +1510 @@
-any)
+any#any_adj)
@@ -1512 +1512 @@
-other)
+other#other_adj)
@@ -1514 +1514 @@
-forme)
+forme#form_n)
@@ -1517 +1517 @@
-or)
+or#or_conj2)
@@ -1520 +1520 @@
-method)))))))
+method#method_n)))))))
@@ -1523 +1523 @@
-or)
+or#or_conj2)
@@ -1527 +1527 @@
-wherein))
+wherein#wherein_adv))
@@ -1535 +1535 @@
-these)
+these#these_pron)
@@ -1537 +1537 @@
-Rules))
+Rules#rule_n1))
@@ -1539 +1539 @@
-shall)
+shall#shall_v)
@@ -1541 +1541 @@
-not)
+not#not_adv)
@@ -1543 +1543 @@
-be)
+be#be_v)
@@ -1545 +1545 @@
-observed)))))))
+observed#observe_v)))))))
@@ -1553 +1553 @@
-6)
+6#NA)
@@ -1562 +1562 @@
-Whensoever))
+Whensoever#whensoever_adv))
@@ -1571 +1571 @@
-there))
+there#there_adv))
@@ -1573 +1573 @@
-shall)
+shall#shall_v)
@@ -1575 +1575 @@
-be)
+be#be_v)
@@ -1578 +1578 @@
-any)
+any#any_adj)
@@ -1580 +1580 @@
-Letters)
+Letters#letter_n1)
@@ -1583 +1583 @@
-from)
+from#from_prep)
@@ -1586 +1586 @@
-Us)))
+Us#us_pron)))
@@ -1589 +1589 @@
-for)
+for#for_prep)
@@ -1592 +1592 @@
-disposing)
+disposing#dispose_v)
@@ -1595 +1595 @@
-of)
+of#of_prep)
@@ -1598 +1598 @@
-any)
+any#any_adj)
@@ -1600 +1600 @@
-money)))
+money#money_n)))
@@ -1603 +1603 @@
-to)
+to#to_prep)
@@ -1606 +1606 @@
-any)
+any#any_adj)
@@ -1608 +1608 @@
-publick)
+publick#public_adj)
@@ -1610 +1610 @@
-uses)))))))
+uses#use_n)))))))
@@ -1615 +1615 @@
-and)
+and#and_conj1)
@@ -1619 +1619 @@
-there))
+there#there_adv))
@@ -1621 +1621 @@
-shall)
+shall#shall_v)
@@ -1623 +1623 @@
-be)
+be#be_v)
@@ -1626 +1626 @@
-other)
+other#other_adj)
@@ -1628 +1628 @@
-Letters)
+Letters#letter_n1)
@@ -1631 +1631 @@
-from)
+from#from_prep)
@@ -1634 +1634 @@
-Us))))
+Us#us_pron))))
@@ -1637 +1637 @@
-at)
+at#at_prep)
@@ -1640 +1640 @@
-the)
+the#the_adj)
@@ -1642 +1642 @@
-same)
+same#same_adj)
@@ -1644 +1644 @@
-time)))
+time#time_n)))
@@ -1647 +1647 @@
-for)
+for#for_prep)
@@ -1650 +1650 @@
-the)
+the#the_adj)
@@ -1652 +1652 @@
-payment)
+payment#payment_n1)
@@ -1655 +1655 @@
-of)
+of#of_prep)
@@ -1658 +1658 @@
-any)
+any#any_adj)
@@ -1660 +1660 @@
-money)))))
+money#money_n)))))
@@ -1663 +1663 @@
-to)
+to#to_prep)
@@ -1666 +1666 @@
-any)
+any#any_adj)
@@ -1668 +1668 @@
-particular)
+particular#particular_adj)
@@ -1670 +1670 @@
-persons))))))))
+persons#person_n))))))))
@@ -1675 +1675 @@
-in)
+in#in_prep)
@@ -1678 +1678 @@
-all)
+all#all_adj)
@@ -1680 +1680 @@
-such)
+such#such_adj)
@@ -1682 +1682 @@
-cases)))
+cases#case_n1)))
@@ -1685 +1685 @@
-the)
+the#the_adj)
@@ -1687 +1687 @@
-publique)
+publique#public_adj)
@@ -1689 +1689 @@
-Letters))
+Letters#letter_n1))
@@ -1691 +1691 @@
-shall)
+shall#shall_v)
@@ -1693 +1693 @@
-be)
+be#be_v)
@@ -1696 +1696 @@
-obeyed)
+obeyed#obey_v)
@@ -1698 +1698 @@
-and)
+and#and_conj1)
@@ -1700 +1700 @@
-preferred))
+preferred#prefer_v))
@@ -1703 +1703 @@
-before)
+before#before_adv)
@@ -1706 +1706 @@
-the)
+the#the_adj)
@@ -1708 +1708 @@
-private)))
+private#private_adj1)))
@@ -1716 +1716 @@
-7)
+7#NA)
@@ -1723 +1723 @@
-In)
+In#in_prep)
@@ -1726 +1726 @@
-case)
+case#case_n1)
@@ -1733 +1733 @@
-our)
+our#our_pron)
@@ -1735 +1735 @@
-Revenue))
+Revenue#revenue_n))
@@ -1737 +1737 @@
-shall)
+shall#shall_v)
@@ -1740 +1740 @@
-at)
+at#at_prep)
@@ -1743 +1743 @@
-any)
+any#any_adj)
@@ -1745 +1745 @@
-time)))
+time#time_n)))
@@ -1747 +1747 @@
-not)
+not#not_adv)
@@ -1749 +1749 @@
-hold)
+hold#hold_v)
@@ -1751 +1751 @@
-out)
+out#out_adv)
@@ -1754 +1754 @@
-to)
+to#to_prep)
@@ -1756 +1756 @@
-pay)
+pay#pay_v1)
@@ -1759 +1759 @@
-the)
+the#the_adj)
@@ -1761 +1761 @@
-whole)
+whole#whole_adj)
@@ -1763 +1763 @@
-Establishment)))))))
+Establishment#establishment_n)))))))
@@ -1766 +1766 @@
-the)
+the#the_adj)
@@ -1768 +1768 @@
-same))
+same#same_adj))
@@ -1770 +1770 @@
-shall)
+shall#shall_v)
@@ -1772 +1772 @@
-not)
+not#not_adv)
@@ -1774 +1774 @@
-be)
+be#be_v)
@@ -1776 +1776 @@
-applied)
+applied#apply_v)
@@ -1779 +1779 @@
-to)
+to#to_prep)
@@ -1782 +1782 @@
-the)
+the#the_adj)
@@ -1784 +1784 @@
-payment)
+payment#payment_n1)
@@ -1787 +1787 @@
-of)
+of#of_prep)
@@ -1790 +1790 @@
-any)
+any#any_adj)
@@ -1792 +1792 @@
-Pensions)))))
+Pensions#pension_n)))))
@@ -1795 +1795 @@
-untill)
+untill#until_prep)
@@ -1802 +1802 @@
-the)
+the#the_adj)
@@ -1804 +1804 @@
-rest)
+rest#rest_n2)
@@ -1807 +1807 @@
-of)
+of#of_prep)
@@ -1810 +1810 @@
-the)
+the#the_adj)
@@ -1813 +1813 @@
-Civill)
+Civill#civil_adj)
@@ -1815 +1815 @@
-and)
+and#and_conj1)
@@ -1817 +1817 @@
-Military))
+Military#military_adj))
@@ -1819 +1819 @@
-Lists))))
+Lists#list_n6))))
@@ -1821 +1821 @@
-be)
+be#be_v)
@@ -1824 +1824 @@
-first))
+first#first_adj))
@@ -1826 +1826 @@
-payd))))
+payd#pay_v1))))
@@ -1834 +1834 @@
-And)
+And#and_conj1)
@@ -1837 +1837 @@
-if)
+if#if_conj)
@@ -1844 +1844 @@
-afterwards))
+afterwards#afterwards_adv))
@@ -1847 +1847 @@
-the)
+the#the_adj)
@@ -1849 +1849 @@
-same))
+same#same_adj))
@@ -1851 +1851 @@
-will)
+will#will_v1)
@@ -1853 +1853 @@
-not)
+not#not_adv)
@@ -1855 +1855 @@
-hold)
+hold#hold_v)
@@ -1857 +1857 @@
-out)
+out#out_adv)
@@ -1860 +1860 @@
-to)
+to#to_prep)
@@ -1863 +1863 @@
-the)
+the#the_adj)
@@ -1865 +1865 @@
-payment)
+payment#payment_n1)
@@ -1868 +1868 @@
-of)
+of#of_prep)
@@ -1871 +1871 @@
-all)
+all#all_adj)
@@ -1873 +1873 @@
-Pensions))))))))
+Pensions#pension_n))))))))
@@ -1878 +1878 @@
-a)
+a#a_adj2)
@@ -1880 +1880 @@
-proportionable)
+proportionable#proportionable_adj)
@@ -1882 +1882 @@
-abatement))
+abatement#abatement_n1))
@@ -1884 +1884 @@
-shall)
+shall#shall_v)
@@ -1886 +1886 @@
-be)
+be#be_v)
@@ -1888 +1888 @@
-made)
+made#make_v1)
@@ -1891 +1891 @@
-out)
+out#out_adv)
@@ -1893 +1893 @@
-of)
+of#of_prep)
@@ -1896 +1896 @@
-each)
+each#each_adj)
@@ -1899 +1899 @@
-of)
+of#of_prep)
@@ -1902 +1902 @@
-them)))))
+them#them_pron)))))
@@ -1910 +1910 @@
-8)
+8#NA)
@@ -1917 +1917 @@
-No)
+No#no_adj)
@@ -1919 +1919 @@
-Letters)
+Letters#letter_n1)
@@ -1922 +1922 @@
-or)
+or#or_conj2)
@@ -1925 +1925 @@
-Directions)))
+Directions#direction_n)))
@@ -1928 +1928 @@
-from)
+from#from_prep)
@@ -1931 +1931 @@
-us)))
+us#us_pron)))
@@ -1934 +1934 @@
-for)
+for#for_prep)
@@ -1937 +1937 @@
-the)
+the#the_adj)
@@ -1939 +1939 @@
-payment)
+payment#payment_n1)
@@ -1942 +1942 @@
-of)
+of#of_prep)
@@ -1945 +1945 @@
-any)
+any#any_adj)
@@ -1947 +1947 @@
-money))))))
+money#money_n))))))
@@ -1949 +1949 @@
-shall)
+shall#shall_v)
@@ -1951 +1951 @@
-be)
+be#be_v)
@@ -1953 +1953 @@
-directed)
+directed#direct_v)
@@ -1956 +1956 @@
-immediately))
+immediately#immediately_adv))
@@ -1960 +1960 @@
-to)
+to#to_prep)
@@ -1963 +1963 @@
-the)
+the#the_adj)
@@ -1965 +1965 @@
-Vice)
+Vice#vice_prefix)
@@ -1967 +1967 @@
-Treasurer)
+Treasurer#treasurer_n)
@@ -1970 +1970 @@
-of)
+of#of_prep)
@@ -1973 +1973 @@
-Ireland)))))
+Ireland#NA)))))
@@ -1976 +1976 @@
-but)
+but#but_prep)
@@ -1979 +1979 @@
-to)
+to#to_prep)
@@ -1982 +1982 @@
-the)
+the#the_adj)
@@ -1984 +1984 @@
-Chief)
+Chief#chief_adj)
@@ -1986 +1986 @@
-Governor)
+Governor#governor_n)
@@ -1989 +1989 @@
-or)
+or#or_conj2)
@@ -1992 +1992 @@
-Governors)))
+Governors#governor_n)))
@@ -1995 +1995 @@
-for)
+for#for_prep)
@@ -1998 +1998 @@
-the)
+the#the_adj)
@@ -2000 +2000 @@
-time)
+time#time_n)
@@ -2003 +2003 @@
-being))))))))
+being#be_v))))))))
@@ -2011 +2011 @@
-And)
+And#and_conj1)
@@ -2014 +2014 @@
-no)
+no#no_adj)
@@ -2016 +2016 @@
-payments)
+payments#payment_n1)
@@ -2019 +2019 @@
-made)
+made#make_v1)
@@ -2022 +2022 @@
-by)
+by#by_prep)
@@ -2025 +2025 @@
-the)
+the#the_adj)
@@ -2027 +2027 @@
-Vice)
+Vice#vice_prefix)
@@ -2029 +2029 @@
-Treasurer)))
+Treasurer#treasurer_n)))
@@ -2032 +2032 @@
-upon)
+upon#upon_prep)
@@ -2035 +2035 @@
-any)
+any#any_adj)
@@ -2037 +2037 @@
-such)
+such#such_adj)
@@ -2039 +2039 @@
-Letters)
+Letters#letter_n1)
@@ -2042 +2042 @@
-or)
+or#or_conj2)
@@ -2045 +2045 @@
-Directions)))
+Directions#direction_n)))
@@ -2048 +2048 @@
-from)
+from#from_prep)
@@ -2051 +2051 @@
-Us)))))
+Us#us_pron)))))
@@ -2054 +2054 @@
-without)
+without#without_adv)
@@ -2058 +2058 @@
-the)
+the#the_adj)
@@ -2060 +2060 @@
-Chief)
+Chief#chief_adj)
@@ -2062 +2062 @@
-Governor's))
+Governor's#governor_n))
@@ -2064 +2064 @@
-Warrant)
+Warrant#warrant_n1)
@@ -2067 +2067 @@
-thereupon))))))
+thereupon#thereupon_adv))))))
@@ -2069 +2069 @@
-shall)
+shall#shall_v)
@@ -2071 +2071 @@
-be)
+be#be_v)
@@ -2073 +2073 @@
-allowed)
+allowed#allow_v)
@@ -2076 +2076 @@
-upon)
+upon#upon_prep)
@@ -2080 +2080 @@
-the)
+the#the_adj)
@@ -2082 +2082 @@
-Vice)
+Vice#vice_prefix)
@@ -2084 +2084 @@
-Treasurer's))
+Treasurer's#treasurer_n))
@@ -2086 +2086 @@
-accompts)))
+accompts#account_n)))
@@ -2094 +2094 @@
-10)
+10#NA)
@@ -2099 +2099 @@
-All)
+All#all_adj)
@@ -2101 +2101 @@
-Letters))
+Letters#letter_n1))
@@ -2103 +2103 @@
-must)
+must#must_v1)
@@ -2105 +2105 @@
-have)
+have#have_v)
@@ -2107 +2107 @@
-been)
+been#be_v)
@@ -2110 +2110 @@
-first))
+first#first_adj))
@@ -2112 +2112 @@
-entered)
+entered#enter_v)
@@ -2115 +2115 @@
-at)
+at#at_prep)
@@ -2118 +2118 @@
-the)
+the#the_adj)
@@ -2120 +2120 @@
-Signet)
+Signet#signet_n)
@@ -2122 +2122 @@
-Office)
+Office#office_n)
@@ -2125 +2125 @@
-here))
+here#here_adv))
@@ -2128 +2128 @@
-in)
+in#in_prep)
@@ -2131 +2131 @@
-England)))))
+England#NA)))))
@@ -2134 +2134 @@
-before)
+before#before_adv)
@@ -2137 +2137 @@
-being)
+being#be_v)
@@ -2139 +2139 @@
-issued)
+issued#issue_v)
@@ -2142 +2142 @@
-for)
+for#for_prep)
@@ -2145 +2145 @@
-payment)))))
+payment#payment_n1)))))
@@ -2151 +2151 @@
-Whereby))
+Whereby#whereby_adv))
@@ -2159 +2159 @@
-the)
+the#the_adj)
@@ -2161 +2161 @@
-great)
+great#great_adj)
@@ -2163 +2163 @@
-disorder)
+disorder#disorder_n)
@@ -2166 +2166 @@
-in)
+in#in_prep)
@@ -2169 +2169 @@
-procuring)
+procuring#procure_v)
@@ -2172 +2172 @@
-Our)
+Our#our_pron)
@@ -2174 +2174 @@
-grant)
+grant#grant_n1)
@@ -2177 +2177 @@
-for)
+for#for_prep)
@@ -2180 +2180 @@
-the)
+the#the_adj)
@@ -2182 +2182 @@
-same)
+same#same_adj)
@@ -2184 +2184 @@
-thing))))
+thing#thing_n1))))
@@ -2187 +2187 @@
-to)
+to#to_prep)
@@ -2190 +2190 @@
-severall)
+severall#several_adj)
@@ -2192 +2192 @@
-persons))))))
+persons#person_n))))))
@@ -2195 +2195 @@
-may)
+may#may_v1)
@@ -2197 +2197 @@
-and)
+and#and_conj1)
@@ -2199 +2199 @@
-will))
+will#will_v1))
@@ -2201 +2201 @@
-be)
+be#be_v)
@@ -2203 +2203 @@
-prevented)))
+prevented#prevent_v)))
@@ -2211 +2211 @@
-Given)
+Given#give_v)
@@ -2214 +2214 @@
-at)
+at#at_prep)
@@ -2217 +2217 @@
-Our)
+Our#our_pron)
@@ -2219 +2219 @@
-court)
+court#court_n1)
@@ -2222 +2222 @@
-at)
+at#at_prep)
@@ -2225 +2225 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -2228 +2228 @@
-the)
+the#the_adj)
@@ -2230 +2230 @@
-22=nd=)
+22=nd=#NA)
@@ -2232 +2232 @@
-Day)
+Day#day_n)
@@ -2235 +2235 @@
-of)
+of#of_prep)
@@ -2238 +2238 @@
-September)))
+September#NA)))
@@ -2240 +2240 @@
-1675))
+1675#NA))
@@ -2243 +2243 @@
-in)
+in#in_prep)
@@ -2246 +2246 @@
-the)
+the#the_adj)
@@ -2248 +2248 @@
-seven_and_twentieth)
+seven_and_twentieth#NA)
@@ -2250 +2250 @@
-year)
+year#year_n)
@@ -2253 +2253 @@
-of)
+of#of_prep)
@@ -2256 +2256 @@
-our)
+our#our_pron)
@@ -2258 +2258 @@
-Reigne)))))
+Reigne#reign_n)))))
@@ -2272 +2272 @@
-CHARLES)
+CHARLES#NA)
@@ -2274 +2274 @@
-R.)
+R.#NA)
@@ -2285 +2285 @@
-Right)
+Right#right_adv)
@@ -2287 +2287 @@
-Trusty))
+Trusty#trusty_adj))
@@ -2290 +2290 @@
-and)
+and#and_conj1)
@@ -2293 +2293 @@
-Right)
+Right#right_adv)
@@ -2295 +2295 @@
-Welbeloved))))
+Welbeloved#well-beloved_adj))))
@@ -2297 +2297 @@
-Cousin)
+Cousin#cousin_n)
@@ -2300 +2300 @@
-and)
+and#and_conj1)
@@ -2303 +2303 @@
-Counceller))))
+Counceller#counsellor_n))))
@@ -2308 +2308 @@
-Wee))
+Wee#we_pron))
@@ -2310 +2310 @@
-greet)
+greet#greet_v1)
@@ -2313 +2313 @@
-you))
+you#you_pron))
@@ -2316 +2316 @@
-well))
+well#well_adv))
@@ -2325 +2325 @@
-Whereas)
+Whereas#whereas_adv)
@@ -2334 +2334 @@
-by)
+by#by_prep)
@@ -2337 +2337 @@
-Our)
+Our#our_pron)
@@ -2339 +2339 @@
-Letters)
+Letters#letter_n1)
@@ -2342 +2342 @@
-Patents))
+Patents#patent_adj))
@@ -2345 +2345 @@
-under)
+under#under_prep)
@@ -2350 +2350 @@
-Our)
+Our#our_pron)
@@ -2352 +2352 @@
-Great)
+Great#NA)
@@ -2354 +2354 @@
-Seale)
+Seale#NA)
@@ -2357 +2357 @@
-of)
+of#of_prep)
@@ -2360 +2360 @@
-England)))))
+England#NA)))))
@@ -2365 +2365 @@
-bearing)
+bearing#bear_v1)
@@ -2368 +2368 @@
-date))
+date#date_n2))
@@ -2372 +2372 @@
-the)
+the#the_adj)
@@ -2374 +2374 @@
-first)
+first#first_adj)
@@ -2376 +2376 @@
-day)
+day#day_n)
@@ -2379 +2379 @@
-of)
+of#of_prep)
@@ -2382 +2382 @@
-December))))
+December#NA))))
@@ -2385 +2385 @@
-in)
+in#in_prep)
@@ -2388 +2388 @@
-the)
+the#the_adj)
@@ -2390 +2390 @@
-23=rd=)
+23=rd=#NA)
@@ -2392 +2392 @@
-yeare)
+yeare#year_n)
@@ -2395 +2395 @@
-of)
+of#of_prep)
@@ -2398 +2398 @@
-Our)
+Our#our_pron)
@@ -2400 +2400 @@
-Reigne)))))))))
+Reigne#reign_n)))))))))
@@ -2405 +2405 @@
-Wee))
+Wee#we_pron))
@@ -2407 +2407 @@
-have)
+have#have_v)
@@ -2410 +2410 @@
-given)
+given#give_v)
@@ -2412 +2412 @@
-and)
+and#and_conj1)
@@ -2414 +2414 @@
-granted))
+granted#grant_v))
@@ -2417 +2417 @@
-unto)
+unto#unto_prep)
@@ -2421 +2421 @@
-Our)
+Our#our_pron)
@@ -2425 +2425 @@
-Deare))
+Deare#dear_adj1))
@@ -2428 +2428 @@
-and)
+and#and_conj1)
@@ -2431 +2431 @@
-Entirely)
+Entirely#entirely_adj)
@@ -2433 +2433 @@
-Beloved))))
+Beloved#beloved_adj))))
@@ -2435 +2435 @@
-Cousin)
+Cousin#cousin_n)
@@ -2438 +2438 @@
-Prince)
+Prince#NA)
@@ -2440 +2440 @@
-Rupert)))
+Rupert#NA)))
@@ -2446 +2446 @@
-his)
+his#his_adj)
@@ -2448 +2448 @@
-Executors)
+Executors#executor_n)
@@ -2454 +2454 @@
-Administrators)))
+Administrators#administrator_n)))
@@ -2459 +2459 @@
-and)
+and#and_conj1)
@@ -2462 +2462 @@
-Assignes)))))))
+Assignes#assign_n2)))))))
@@ -2467 +2467 @@
-the)
+the#the_adj)
@@ -2469 +2469 @@
-Sole)
+Sole#sole_adj)
@@ -2471 +2471 @@
-Power)
+Power#power_n1)
@@ -2474 +2474 @@
-and)
+and#and_conj1)
@@ -2477 +2477 @@
-Priviledge)))
+Priviledge#privilege_n)))
@@ -2480 +2480 @@
-to)
+to#to_prep)
@@ -2483 +2483 @@
-use)
+use#use_v)
@@ -2485 +2485 @@
-exercise)
+exercise#exercise_v)
@@ -2487 +2487 @@
-and)
+and#and_conj1)
@@ -2489 +2489 @@
-enjoy))
+enjoy#enjoy_v))
@@ -2492 +2492 @@
-the)
+the#the_adj)
@@ -2494 +2494 @@
-new)
+new#new_adj)
@@ -2496 +2496 @@
-Arts)
+Arts#art_n)
@@ -2499 +2499 @@
-or)
+or#or_conj2)
@@ -2502 +2502 @@
-Inventions)))
+Inventions#invention_n)))
@@ -2505 +2505 @@
-of)
+of#of_prep)
@@ -2509 +2509 @@
-converting)
+converting#convert_v)
@@ -2512 +2512 @@
-into)
+into#into_prep)
@@ -2515 +2515 @@
-Steele)))
+Steele#steel_n1)))
@@ -2518 +2518 @@
-all)
+all#all_adj)
@@ -2520 +2520 @@
-manner)
+manner#manner_n)
@@ -2523 +2523 @@
-of)
+of#of_prep)
@@ -2527 +2527 @@
-edged)
+edged#edged_adj)
@@ -2529 +2529 @@
-Tooles)
+Tooles#tool_n)
@@ -2535 +2535 @@
-Files))))
+Files#file_n1))))
@@ -2540 +2540 @@
-and)
+and#and_conj1)
@@ -2543 +2543 @@
-other)
+other#other_adj)
@@ -2545 +2545 @@
-Instruments)
+Instruments#instrument_n)
@@ -2549 +2549 @@
-forged)
+forged#forge_v1)
@@ -2551 +2551 @@
-and)
+and#and_conj1)
@@ -2553 +2553 @@
-formed))
+formed#form_v1))
@@ -2556 +2556 @@
-in)
+in#in_prep)
@@ -2559 +2559 @@
-soft)
+soft#soft_adj)
@@ -2561 +2561 @@
-Iron))))))))))
+Iron#iron_n1))))))))))
@@ -2566 +2566 @@
-or)
+or#or_conj2)
@@ -2570 +2570 @@
-any)
+any#any_adj)
@@ -2572 +2572 @@
-part)
+part#part_n)
@@ -2575 +2575 @@
-of)
+of#of_prep)
@@ -2578 +2578 @@
-the)
+the#the_adj)
@@ -2580 +2580 @@
-said)
+said#said_adj)
@@ -2582 +2582 @@
-Tooles)
+Tooles#tool_n)
@@ -2588 +2588 @@
-Files)))
+Files#file_n1)))
@@ -2593 +2593 @@
-or)
+or#or_conj2)
@@ -2596 +2596 @@
-other)
+other#other_adj)
@@ -2598 +2598 @@
-Instruments))))))
+Instruments#instrument_n))))))
@@ -2601 +2601 @@
-after)
+after#after_adv)
@@ -2608 +2608 @@
-they))
+they#they_pron))
@@ -2610 +2610 @@
-are)
+are#be_v)
@@ -2613 +2613 @@
-soe))
+soe#so_adv))
@@ -2616 +2616 @@
-forged)
+forged#forge_v1)
@@ -2618 +2618 @@
-or)
+or#or_conj2)
@@ -2620 +2620 @@
-framed)))))))))))
+framed#frame_v)))))))))))
@@ -2625 +2625 @@
-As)
+As#as_adv)
@@ -2629 +2629 @@
-also)
+also#also_adv)
@@ -2631 +2631 @@
-for)
+for#for_prep)
@@ -2634 +2634 @@
-the)
+the#the_adj)
@@ -2636 +2636 @@
-like)
+like#like_adj)
@@ -2638 +2638 @@
-converting)
+converting#converting_n)
@@ -2641 +2641 @@
-of)
+of#of_prep)
@@ -2644 +2644 @@
-all)
+all#all_adj)
@@ -2646 +2646 @@
-manner)
+manner#manner_n)
@@ -2649 +2649 @@
-of)
+of#of_prep)
@@ -2652 +2652 @@
-Iron)
+Iron#iron_n1)
@@ -2654 +2654 @@
-Wier)))))
+Wier#wire_n)))))
@@ -2659 +2659 @@
-after)
+after#after_adv)
@@ -2666 +2666 @@
-it))
+it#it_pron))
@@ -2668 +2668 @@
-is)
+is#be_v)
@@ -2670 +2670 @@
-drawne))))))
+drawne#draw_v))))))
@@ -2675 +2675 @@
-and)
+and#and_conj1)
@@ -2677 +2677 @@
-alsoe)
+alsoe#also_adv)
@@ -2680 +2680 @@
-of)
+of#of_prep)
@@ -2684 +2684 @@
-preparing))
+preparing#prepare_v))
@@ -2687 +2687 @@
-and)
+and#and_conj1)
@@ -2690 +2690 @@
-softning)
+softning#soften_v)
@@ -2693 +2693 @@
-all)
+all#all_adj)
@@ -2696 +2696 @@
-Cast)
+Cast#cast_v)
@@ -2698 +2698 @@
-and)
+and#and_conj1)
@@ -2700 +2700 @@
-Melted))
+Melted#melted_adj1))
@@ -2702 +2702 @@
-Iron))))
+Iron#iron_n1))))
@@ -2707 +2707 @@
-soe)
+soe#so_adv)
@@ -2710 +2710 @@
-that)
+that#that_conj)
@@ -2714 +2714 @@
-it))
+it#it_pron))
@@ -2716 +2716 @@
-may)
+may#may_v1)
@@ -2718 +2718 @@
-be)
+be#be_v)
@@ -2721 +2721 @@
-filed)
+filed#file_v1)
@@ -2723 +2723 @@
-&)
+&#and_conj1)
@@ -2725 +2725 @@
-wrought))
+wrought#work_v))
@@ -2728 +2728 @@
-as)
+as#as_adv)
@@ -2739 +2739 @@
-forged)
+forged#forged_adj)
@@ -2741 +2741 @@
-Iron))
+Iron#iron_n1))
@@ -2743 +2743 @@
-is)
+is#be_v)
@@ -2750 +2750 @@
-and)
+and#and_conj1)
@@ -2753 +2753 @@
-of)
+of#of_prep)
@@ -2756 +2756 @@
-tincturing)
+tincturing#SEE_tincture_v)
@@ -2759 +2759 @@
-of)
+of#of_prep)
@@ -2762 +2762 @@
-Copper)))
+Copper#copper_n1)))
@@ -2765 +2765 @@
-upon)
+upon#upon_prep)
@@ -2768 +2768 @@
-Iron)))
+Iron#iron_n1)))
@@ -2771 +2771 @@
-as)
+as#as_adv)
@@ -2784 +2784 @@
-to)
+to#to_prep)
@@ -2787 +2787 @@
-him)
+him#him_pron)
@@ -2789 +2789 @@
-or)
+or#or_conj2)
@@ -2791 +2791 @@
-them)))
+them#them_pron)))
@@ -2793 +2793 @@
-shall)
+shall#shall_v)
@@ -2795 +2795 @@
-seem)
+seem#seem_v2)
@@ -2798 +2798 @@
-meet)))))))))))
+meet#meet_adj)))))))))))
@@ -2803 +2803 @@
-To)
+To#to_prep)
@@ -2806 +2806 @@
-have)
+have#have_v)
@@ -2808 +2808 @@
-and)
+and#and_conj1)
@@ -2810 +2810 @@
-enjoy))
+enjoy#enjoy_v))
@@ -2813 +2813 @@
-the)
+the#the_adj)
@@ -2815 +2815 @@
-same))
+same#same_adj))
@@ -2819 +2819 @@
-for)
+for#for_prep)
@@ -2821 +2821 @@
-and)
+and#and_conj1)
@@ -2823 +2823 @@
-during))
+during#during_prep))
@@ -2826 +2826 @@
-the)
+the#the_adj)
@@ -2828 +2828 @@
-terme)
+terme#term_n)
@@ -2831 +2831 @@
-of)
+of#of_prep)
@@ -2834 +2834 @@
-fourteen)
+fourteen#fourteen_adj)
@@ -2836 +2836 @@
-yeares)))))
+yeares#year_n)))))
@@ -2839 +2839 @@
-according)
+according#accord_v)
@@ -2842 +2842 @@
-to)
+to#to_prep)
@@ -2845 +2845 @@
-the)
+the#the_adj)
@@ -2847 +2847 @@
-Act)
+Act#act_n)
@@ -2850 +2850 @@
-of)
+of#of_prep)
@@ -2853 +2853 @@
-Parliament)))
+Parliament#NA)))
@@ -2856 +2856 @@
-in)
+in#in_prep)
@@ -2859 +2859 @@
-that)
+that#that_pron)
@@ -2861 +2861 @@
-case)
+case#case_n1)
@@ -2865 +2865 @@
-made)
+made#make_v1)
@@ -2867 +2867 @@
-and)
+and#and_conj1)
@@ -2869 +2869 @@
-provided))))))))))))
+provided#provide_v))))))))))))
@@ -2878 +2878 @@
-Our)
+Our#our_pron)
@@ -2880 +2880 @@
-will)
+will#will_n1)
@@ -2883 +2883 @@
-and)
+and#and_conj1)
@@ -2886 +2886 @@
-pleasure))))
+pleasure#pleasure_n))))
@@ -2888 +2888 @@
-is)
+is#be_v)
@@ -2893 +2893 @@
-And)
+And#and_conj1)
@@ -2896 +2896 @@
-Wee))
+Wee#we_pron))
@@ -2898 +2898 @@
-doe)
+doe#do_v)
@@ -2901 +2901 @@
-hereby))
+hereby#hereby_adv))
@@ -2904 +2904 @@
-require)
+require#require_v)
@@ -2906 +2906 @@
-and)
+and#and_conj1)
@@ -2908 +2908 @@
-Authorise))
+Authorise#authorize_v))
@@ -2911 +2911 @@
-you)))
+you#you_pron)))
@@ -2914 +2914 @@
-to)
+to#to_prep)
@@ -2916 +2916 @@
-give)
+give#give_v)
@@ -2919 +2919 @@
-Order))
+Order#order_n))
@@ -2922 +2922 @@
-for)
+for#for_prep)
@@ -2925 +2925 @@
-y=e=)
+y=e=#the_adj)
@@ -2927 +2927 @@
-passing)
+passing#passing_n)
@@ -2930 +2930 @@
-of)
+of#of_prep)
@@ -2933 +2933 @@
-Letters)
+Letters#letter_n1)
@@ -2936 +2936 @@
-Patents))
+Patents#patent_adj))
@@ -2941 +2941 @@
-under)
+under#under_prep)
@@ -2944 +2944 @@
-the)
+the#the_adj)
@@ -2946 +2946 @@
-Great)
+Great#NA)
@@ -2948 +2948 @@
-Seale)
+Seale#NA)
@@ -2951 +2951 @@
-of)
+of#of_prep)
@@ -2954 +2954 @@
-that)
+that#that_pron)
@@ -2956 +2956 @@
-Our)
+Our#our_pron)
@@ -2958 +2958 @@
-Kingdome)))))
+Kingdome#kingdom_n)))))
@@ -2963 +2963 @@
-containing)
+containing#contain_v)
@@ -2966 +2966 @@
-Our)
+Our#our_pron)
@@ -2968 +2968 @@
-Grant)
+Grant#grant_n1)
@@ -2971 +2971 @@
-unto)
+unto#unto_prep)
@@ -2974 +2974 @@
-Our)
+Our#our_pron)
@@ -2976 +2976 @@
-said)
+said#said_adj)
@@ -2979 +2979 @@
-most)
+most#most_adj)
@@ -2981 +2981 @@
-Deare))
+Deare#dear_adj1))
@@ -2983 +2983 @@
-Cousin)
+Cousin#cousin_n)
@@ -2987 +2987 @@
-Prince)
+Prince#NA)
@@ -2989 +2989 @@
-Rupert))
+Rupert#NA))
@@ -2995 +2995 @@
-his)
+his#his_adj)
@@ -2997 +2997 @@
-Executors)
+Executors#executor_n)
@@ -3003 +3003 @@
-Administrators)))
+Administrators#administrator_n)))
@@ -3008 +3008 @@
-and)
+and#and_conj1)
@@ -3011 +3011 @@
-Assignes))))))))
+Assignes#assign_n2))))))))
@@ -3014 +3014 @@
-of)
+of#of_prep)
@@ -3017 +3017 @@
-the)
+the#the_adj)
@@ -3019 +3019 @@
-said)
+said#said_adj)
@@ -3021 +3021 @@
-Exercise)
+Exercise#exercise_n)
@@ -3024 +3024 @@
-and)
+and#and_conj1)
@@ -3027 +3027 @@
-use)))
+use#use_n)))
@@ -3030 +3030 @@
-of)
+of#of_prep)
@@ -3033 +3033 @@
-the)
+the#the_adj)
@@ -3035 +3035 @@
-said)
+said#said_adj)
@@ -3037 +3037 @@
-Arts)
+Arts#art_n)
@@ -3040 +3040 @@
-and)
+and#and_conj1)
@@ -3043 +3043 @@
-Inventions)))))
+Inventions#invention_n)))))
@@ -3046 +3046 @@
-in)
+in#in_prep)
@@ -3049 +3049 @@
-that)
+that#that_pron)
@@ -3051 +3051 @@
-Our)
+Our#our_pron)
@@ -3053 +3053 @@
-Kingdome)))
+Kingdome#kingdom_n)))
@@ -3056 +3056 @@
-for)
+for#for_prep)
@@ -3059 +3059 @@
-the)
+the#the_adj)
@@ -3061 +3061 @@
-terme)
+terme#term_n)
@@ -3064 +3064 @@
-of)
+of#of_prep)
@@ -3067 +3067 @@
-fourteen)
+fourteen#fourteen_adj)
@@ -3069 +3069 @@
-yeares)))))
+yeares#year_n)))))
@@ -3073 +3073 @@
-in)
+in#in_prep)
@@ -3076 +3076 @@
-the)
+the#the_adj)
@@ -3078 +3078 @@
-same)
+same#same_adj)
@@ -3080 +3080 @@
-manner)))
+manner#manner_n)))
@@ -3085 +3085 @@
-and)
+and#and_conj1)
@@ -3088 +3088 @@
-as)
+as#as_adv)
@@ -3090 +3090 @@
-fully)
+fully#fully_adv)
@@ -3093 +3093 @@
-and)
+and#and_conj1)
@@ -3096 +3096 @@
-beneficially)))))
+beneficially#beneficially_adv)))))
@@ -3099 +3099 @@
-as)
+as#as_adv)
@@ -3109 +3109 @@
-is)
+is#be_v)
@@ -3111 +3111 @@
-contained)
+contained#contain_v)
@@ -3114 +3114 @@
-in)
+in#in_prep)
@@ -3117 +3117 @@
-Our)
+Our#our_pron)
@@ -3119 +3119 @@
-said)
+said#said_adj)
@@ -3121 +3121 @@
-Letters)
+Letters#letter_n1)
@@ -3124 +3124 @@
-Patents))
+Patents#patent_adj))
@@ -3127 +3127 @@
-under)
+under#under_prep)
@@ -3130 +3130 @@
-Our)
+Our#our_pron)
@@ -3132 +3132 @@
-Great)
+Great#NA)
@@ -3134 +3134 @@
-Seale)
+Seale#NA)
@@ -3137 +3137 @@
-of)
+of#of_prep)
@@ -3140 +3140 @@
-England))))))))))))))))))
+England#NA))))))))))))))))))
@@ -3148 +3148 @@
-And)
+And#and_conj1)
@@ -3151 +3151 @@
-for)
+for#for_prep)
@@ -3155 +3155 @@
-soe))
+soe#so_adv))
@@ -3157 +3157 @@
-doing)))
+doing#do_v)))
@@ -3160 +3160 @@
-this))
+this#this_pron))
@@ -3162 +3162 @@
-shall)
+shall#shall_v)
@@ -3164 +3164 @@
-be)
+be#be_v)
@@ -3168 +3168 @@
-unto)
+unto#unto_prep)
@@ -3171 +3171 @@
-you)))
+you#you_pron)))
@@ -3174 +3174 @@
-and)
+and#and_conj1)
@@ -3177 +3177 @@
-to)
+to#to_prep)
@@ -3180 +3180 @@
-all)
+all#all_adj)
@@ -3182 +3182 @@
-other)
+other#other_adj)
@@ -3184 +3184 @@
-Our)
+Our#our_pron)
@@ -3186 +3186 @@
-Officers)
+Officers#officer_n)
@@ -3189 +3189 @@
-and)
+and#and_conj1)
@@ -3192 +3192 @@
-Ministers)))
+Ministers#minister_n)))
@@ -3196 +3196 @@
-whom))
+whom#whom_pron))
@@ -3204 +3204 @@
-it))
+it#it_pron))
@@ -3206 +3206 @@
-may)
+may#may_v1)
@@ -3208 +3208 @@
-concerne)))))))
+concerne#concern_v)))))))
@@ -3211 +3211 @@
-a)
+a#a_adj2)
@@ -3213 +3213 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -3215 +3215 @@
-Warrant)
+Warrant#warrant_n1)
@@ -3225 +3225 @@
-And)
+And#and_conj1)
@@ -3228 +3228 @@
-soe))
+soe#so_adv))
@@ -3231 +3231 @@
-Wee))
+Wee#we_pron))
@@ -3233 +3233 @@
-bid)
+bid#bid_v)
@@ -3236 +3236 @@
-you))
+you#you_pron))
@@ -3239 +3239 @@
-heartily))
+heartily#heartily_adv))
@@ -3242 +3242 @@
-Farewell))
+Farewell#farewell_int))
@@ -3250 +3250 @@
-Given)
+Given#give_v)
@@ -3253 +3253 @@
-at)
+at#at_prep)
@@ -3256 +3256 @@
-Our)
+Our#our_pron)
@@ -3258 +3258 @@
-Court)
+Court#court_n1)
@@ -3261 +3261 @@
-at)
+at#at_prep)
@@ -3264 +3264 @@
-Windsor)))))
+Windsor#NA)))))
@@ -3267 +3267 @@
-the)
+the#the_adj)
@@ -3269 +3269 @@
-11=th=)
+11=th=#NA)
@@ -3271 +3271 @@
-day)
+day#day_n)
@@ -3274 +3274 @@
-of)
+of#of_prep)
@@ -3277 +3277 @@
-June)))
+June#NA)))
@@ -3281 +3281 @@
-1679))
+1679#NA))
@@ -3286 +3286 @@
-in)
+in#in_prep)
@@ -3289 +3289 @@
-the)
+the#the_adj)
@@ -3291 +3291 @@
-Six_and_Twentieth)
+Six_and_Twentieth#NA)
@@ -3293 +3293 @@
-yeare)
+yeare#year_n)
@@ -3296 +3296 @@
-of)
+of#of_prep)
@@ -3299 +3299 @@
-Our)
+Our#our_pron)
@@ -3301 +3301 @@
-Reigne)))))
+Reigne#reign_n)))))
