--- chatton-1691-e3-p2.psd.vert	2025-09-10 13:51:17
+++ chatton-1691-e3-p2.lem.vert	2025-09-10 13:51:25
@@ -10 +10 @@
-March)
+March#NA)
@@ -12 +12 @@
-5)
+5#five_v)
@@ -16 +16 @@
-1690/1)
+1690/1#NA)
@@ -27 +27 @@
-I))
+I#i_pron))
@@ -29 +29 @@
-have)
+have#have_v)
@@ -34 +34 @@
-my)
+my#my_adj)
@@ -36 +36 @@
-Lord))
+Lord#lord_n))
@@ -40 +40 @@
-been)
+been#be_v)
@@ -43 +43 @@
-soe)
+soe#so_adv)
@@ -45 +45 @@
-unfortunate)
+unfortunate#unfortunate_adj)
@@ -48 +48 @@
-y=t=)
+y=t=#that_conj)
@@ -52 +52 @@
-I))
+I#i_pron))
@@ -54 +54 @@
-cou'd)
+cou'd#can_v1)
@@ -56 +56 @@
-not)
+not#not_adv)
@@ -59 +59 @@
-possibly))
+possibly#possibly_adv))
@@ -61 +61 @@
-dispatch)
+dispatch#dispatch_v)
@@ -64 +64 @@
-my)
+my#my_adj)
@@ -66 +66 @@
-business))
+business#business_n))
@@ -69 +69 @@
-this)
+this#this_pron)
@@ -71 +71 @@
-weeke)))))
+weeke#week_n)))))
@@ -79 +79 @@
-but)
+but#but_prep)
@@ -82 +82 @@
-I))
+I#i_pron))
@@ -84 +84 @@
-doubt)
+doubt#doubt_v)
@@ -86 +86 @@
-not)
+not#not_adv)
@@ -89 +89 @@
-but)
+but#but_prep)
@@ -92 +92 @@
-y=e=)
+y=e=#the_adj)
@@ -94 +94 @@
-next)
+next#next_adj)
@@ -96 +96 @@
-weeke))
+weeke#week_n))
@@ -99 +99 @@
-by)
+by#by_prep)
@@ -102 +102 @@
-y=e=)
+y=e=#the_adj)
@@ -104 +104 @@
-Oundle)
+Oundle#NA)
@@ -106 +106 @@
-coach)))
+coach#coach_n)))
@@ -108 +108 @@
-to)
+to#to_prep)
@@ -110 +110 @@
-waite)
+waite#wait_v1)
@@ -113 +113 @@
-on)
+on#on_prep)
@@ -116 +116 @@
-y=r=)
+y=r=#your_pron)
@@ -118 +118 @@
-Lo=ppe=))))
+Lo=ppe=#lordship_n))))
@@ -127 +127 @@
-I))
+I#i_pron))
@@ -129 +129 @@
-wase)
+wase#be_v)
@@ -132 +132 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -135 +135 @@
-at)
+at#at_prep)
@@ -139 +139 @@
-M=r=)
+M=r=#NA)
@@ -141 +141 @@
-Ashmoles))))
+Ashmoles#NA))))
@@ -150 +150 @@
-Both)
+Both#both_pron)
@@ -153 +153 @@
-he))
+he#he_pron))
@@ -156 +156 @@
-and)
+and#and_conj1)
@@ -159 +159 @@
-M=rs=)
+M=rs=#NA)
@@ -161 +161 @@
-Ashmole))))
+Ashmole#NA))))
@@ -163 +163 @@
-are)
+are#be_v)
@@ -166 +166 @@
-very)
+very#very_adj)
@@ -168 +168 @@
-ill))
+ill#ill_adj))
@@ -177 +177 @@
-They))
+They#they_pron))
@@ -179 +179 @@
-give)
+give#give_v)
@@ -182 +182 @@
-their)
+their#their_adj)
@@ -184 +184 @@
-humble)
+humble#humble_adj)
@@ -186 +186 @@
-service))
+service#service_n1))
@@ -189 +189 @@
-to)
+to#to_prep)
@@ -192 +192 @@
-y=r=)
+y=r=#your_pron)
@@ -194 +194 @@
-Lo=ppe=)))
+Lo=ppe=#lordship_n)))
@@ -202 +202 @@
-and)
+and#and_conj1)
@@ -208 +208 @@
-had)
+had#have_v)
@@ -211 +211 @@
-any)
+any#any_adj)
@@ -214 +214 @@
-of)
+of#of_prep)
@@ -217 +217 @@
-y=e=)
+y=e=#the_adj)
@@ -219 +219 @@
-layers)
+layers#layer_n)
@@ -222 +222 @@
-of)
+of#of_prep)
@@ -225 +225 @@
-his)
+his#his_adj)
@@ -227 +227 @@
-arbutus))))))
+arbutus#arbutus_n))))))
@@ -229 +229 @@
-taken)
+taken#take_v)
@@ -232 +232 @@
-root))))
+root#root_n1))))
@@ -237 +237 @@
-I))
+I#i_pron))
@@ -239 +239 @@
-had)
+had#have_v)
@@ -241 +241 @@
-sent)
+sent#send_v1)
@@ -244 +244 @@
-you))
+you#you_pron))
@@ -247 +247 @@
-some))
+some#some_pron))
@@ -255 +255 @@
-for)
+for#for_prep)
@@ -258 +258 @@
-he))
+he#he_pron))
@@ -260 +260 @@
-ordred)
+ordred#order_v)
@@ -263 +263 @@
-his)
+his#his_adj)
@@ -265 +265 @@
-gardener))
+gardener#gardener_n))
@@ -268 +268 @@
-to)
+to#to_prep)
@@ -270 +270 @@
-dig)
+dig#dig_v)
@@ -273 +273 @@
-y=m=))
+y=m=#them_pron))
@@ -275 +275 @@
-up)
+up#up_adv1)
@@ -281 +281 @@
-w=ch=))
+w=ch=#which_adj))
@@ -289 +289 @@
-he))
+he#he_pron))
@@ -291 +291 @@
-did)
+did#do_v)
@@ -294 +294 @@
-in)
+in#in_prep)
@@ -297 +297 @@
-my)
+my#my_adj)
@@ -299 +299 @@
-presence))))))
+presence#presence_n))))))
@@ -307 +307 @@
-but)
+but#but_prep)
@@ -310 +310 @@
-noe)
+noe#no_adj)
@@ -312 +312 @@
-one)
+one#one_adj)
@@ -314 +314 @@
-layer))
+layer#layer_n))
@@ -316 +316 @@
-had)
+had#have_v)
@@ -319 +319 @@
-any)
+any#any_adj)
@@ -321 +321 @@
-root))
+root#root_n1))
@@ -330 +330 @@
-He))
+He#he_pron))
@@ -332 +332 @@
-gave)
+gave#give_v)
@@ -335 +335 @@
-me))
+me#me_pron1))
@@ -338 +338 @@
-some)
+some#some_pron)
@@ -341 +341 @@
-of)
+of#of_prep)
@@ -344 +344 @@
-y=e=)
+y=e=#the_adj)
@@ -346 +346 @@
-fruit)
+fruit#fruit_n)
@@ -349 +349 @@
-dryed))
+dryed#dry_v))
@@ -355 +355 @@
-w=ch=))
+w=ch=#which_adj))
@@ -361 +361 @@
-he))
+he#he_pron))
@@ -363 +363 @@
-sath)
+sath#say_v1)
@@ -372 +372 @@
-if)
+if#if_conj)
@@ -375 +375 @@
-sown)))
+sown#sow_v1)))
@@ -377 +377 @@
-will)
+will#will_v1)
@@ -379 +379 @@
-grow))))))))
+grow#grow_v))))))))
@@ -389 +389 @@
-I))
+I#i_pron))
@@ -391 +391 @@
-have)
+have#have_v)
@@ -393 +393 @@
-sent)
+sent#send_v1)
@@ -396 +396 @@
-y=m=))
+y=m=#them_pron))
@@ -401 +401 @@
-in)
+in#in_prep)
@@ -404 +404 @@
-a)
+a#a_adj2)
@@ -406 +406 @@
-box)
+box#box_n2)
@@ -409 +409 @@
-of)
+of#of_prep)
@@ -412 +412 @@
-toies)
+toies#toy_n)
@@ -416 +416 @@
-w=ch=))
+w=ch=#which_adj))
@@ -424 +424 @@
-my)
+my#my_adj)
@@ -426 +426 @@
-wife))
+wife#wife_n))
@@ -428 +428 @@
-sends)
+sends#send_v1)
@@ -431 +431 @@
-to)
+to#to_prep)
@@ -434 +434 @@
-my)
+my#my_adj)
@@ -436 +436 @@
-nephews)
+nephews#nephew_n)
@@ -439 +439 @@
-and)
+and#and_conj1)
@@ -442 +442 @@
-neices))))))))))))
+neices#niece_n))))))))))))
@@ -449 +449 @@
-and)
+and#and_conj1)
@@ -453 +453 @@
-w=th=)
+w=th=#with_prep)
@@ -456 +456 @@
-y=m=)))
+y=m=#them_pron)))
@@ -460 +460 @@
-some)
+some#some_pron)
@@ -463 +463 @@
-of)
+of#of_prep)
@@ -466 +466 @@
-y=e=)
+y=e=#the_adj)
@@ -468 +468 @@
-great)
+great#great_adj)
@@ -470 +470 @@
-hawes))))
+hawes#haw_n1))))
@@ -475 +475 @@
-and)
+and#and_conj1)
@@ -477 +477 @@
-also)
+also#also_adv)
@@ -480 +480 @@
-some)
+some#some_pron)
@@ -483 +483 @@
-of)
+of#of_prep)
@@ -486 +486 @@
-y=e=)
+y=e=#the_adj)
@@ -489 +489 @@
-Rhodia)
+Rhodia#NA)
@@ -491 +491 @@
-Radix))
+Radix#NA))
@@ -495 +495 @@
-w=ch=))
+w=ch=#which_adj))
@@ -502 +502 @@
-must)
+must#must_v1)
@@ -504 +504 @@
-be)
+be#be_v)
@@ -506 +506 @@
-planted)
+planted#plant_v)
@@ -509 +509 @@
-in)
+in#in_prep)
@@ -512 +512 @@
-a)
+a#a_adj2)
@@ -514 +514 @@
-pot))))))))))))
+pot#pot_n1))))))))))))
@@ -522 +522 @@
-And)
+And#and_conj1)
@@ -525 +525 @@
-I))
+I#i_pron))
@@ -527 +527 @@
-have)
+have#have_v)
@@ -530 +530 @@
-likewise))
+likewise#likewise_adv))
@@ -532 +532 @@
-sent)
+sent#send_v1)
@@ -535 +535 @@
-3)
+3#three_adj)
@@ -537 +537 @@
-little)
+little#little_adj)
@@ -539 +539 @@
-bundles)
+bundles#bundle_n)
@@ -542 +542 @@
-of)
+of#of_prep)
@@ -545 +545 @@
-grafts)
+grafts#graft_n1)
@@ -548 +548 @@
-of)
+of#of_prep)
@@ -551 +551 @@
-peare)
+peare#pear_n)
@@ -553 +553 @@
-trees)
+trees#tree_n)
@@ -559 +559 @@
-much))
+much#much_adj))
@@ -561 +561 @@
-commended)
+commended#commend_v)
@@ -564 +564 @@
-by)
+by#by_prep)
@@ -567 +567 @@
-M=r=)
+M=r=#NA)
@@ -569 +569 @@
-Ashmole)))))))))
+Ashmole#NA)))))))))
@@ -578 +578 @@
-They))
+They#they_pron))
@@ -580 +580 @@
-were)
+were#be_v)
@@ -582 +582 @@
-cut)
+cut#cut_v)
@@ -584 +584 @@
-of)
+of#off_adv)
@@ -587 +587 @@
-of)
+of#of_prep)
@@ -590 +590 @@
-standard)
+standard#standard_n)
@@ -592 +592 @@
-trees)))
+trees#tree_n)))
@@ -601 +601 @@
-No.)
+No.#number_n)
@@ -603 +603 @@
-1))
+1#one_adj))
@@ -607 +607 @@
-an)
+an#a_adj2)
@@ -609 +609 @@
-excellent)
+excellent#excellent_adj)
@@ -611 +611 @@
-peare)
+peare#pear_n)
@@ -619 +619 @@
-to)
+to#to_prep)
@@ -621 +621 @@
-eat)
+eat#eat_v)
@@ -624 +624 @@
-raw))))
+raw#raw_adj))))
@@ -633 +633 @@
-No.)
+No.#number_n)
@@ -635 +635 @@
-2))
+2#two_adj))
@@ -640 +640 @@
-a)
+a#a_adj2)
@@ -643 +643 @@
-very)
+very#very_adj)
@@ -645 +645 @@
-good))
+good#good_adj))
@@ -647 +647 @@
-baking)
+baking#baking_n)
@@ -649 +649 @@
-peare))
+peare#pear_n))
@@ -652 +652 @@
-and)
+and#and_conj1)
@@ -655 +655 @@
-a)
+a#a_adj2)
@@ -657 +657 @@
-great)
+great#great_adj)
@@ -659 +659 @@
-bearer)))
+bearer#bearer_n)))
@@ -668 +668 @@
-He))
+He#he_pron))
@@ -670 +670 @@
-calls)
+calls#call_v)
@@ -674 +674 @@
-it))
+it#it_pron))
@@ -678 +678 @@
-y=e=)
+y=e=#the_adj)
@@ -680 +680 @@
-ladies))
+ladies#lady_n))
@@ -682 +682 @@
-on)
+on#own_adj)
@@ -686 +686 @@
-thigh)))
+thigh#thigh_n)))
@@ -695 +695 @@
-No.)
+No.#number_n)
@@ -697 +697 @@
-3))
+3#three_adj))
@@ -701 +701 @@
-y=e=)
+y=e=#the_adj)
@@ -703 +703 @@
-Ashmole)
+Ashmole#NA)
@@ -705 +705 @@
-peare)
+peare#pear_n)
@@ -711 +711 @@
-some)
+some#some_pron)
@@ -714 +714 @@
-of)
+of#of_prep)
@@ -717 +717 @@
-w=ch=))))
+w=ch=#which_adj))))
@@ -724 +724 @@
-have)
+have#have_v)
@@ -726 +726 @@
-weighed)
+weighed#weigh_v1)
@@ -729 +729 @@
-20)
+20#twenty_adj)
@@ -731 +731 @@
-ounces))))
+ounces#ounce_n1))))
@@ -740 +740 @@
-It))
+It#it_pron))
@@ -742 +742 @@
-is)
+is#be_v)
@@ -745 +745 @@
-an)
+an#a_adj2)
@@ -747 +747 @@
-excellent)
+excellent#excellent_adj)
@@ -749 +749 @@
-baking)
+baking#baking_n)
@@ -751 +751 @@
-peare))
+peare#pear_n))
@@ -760 +760 @@
-W=th=)
+W=th=#with_prep)
@@ -763 +763 @@
-those)
+those#those_pron)
@@ -765 +765 @@
-grafts)))
+grafts#graft_n1)))
@@ -767 +767 @@
-are)
+are#be_v)
@@ -770 +770 @@
-2)
+2#two_adj)
@@ -772 +772 @@
-bundles)
+bundles#bundle_n)
@@ -775 +775 @@
-of)
+of#of_prep)
@@ -778 +778 @@
-grafts)
+grafts#graft_n1)
@@ -781 +781 @@
-of)
+of#of_prep)
@@ -784 +784 @@
-his)
+his#his_adj)
@@ -786 +786 @@
-famous)
+famous#famous_adj)
@@ -788 +788 @@
-great)
+great#great_adj)
@@ -790 +790 @@
-haw)
+haw#haw_n2)
@@ -792 +792 @@
-tree)))
+tree#tree_n)))
@@ -798 +798 @@
-w=ch=))
+w=ch=#which_adj))
@@ -805 +805 @@
-must)
+must#must_v1)
@@ -807 +807 @@
-be)
+be#be_v)
@@ -809 +809 @@
-grafted)
+grafted#graft_v1)
@@ -812 +812 @@
-on)
+on#on_prep)
@@ -816 +816 @@
-white)
+white#white_adj)
@@ -818 +818 @@
-or)
+or#or_conj2)
@@ -822 +822 @@
-blacke))
+blacke#black_adj))
@@ -824 +824 @@
-thorne)))))))))
+thorne#thorn_n)))))))))
@@ -830 +830 @@
-and)
+and#and_conj1)
@@ -834 +834 @@
-will)
+will#will_v1)
@@ -836 +836 @@
-doe)
+doe#do_v)
@@ -839 +839 @@
-very)
+very#very_adj)
@@ -841 +841 @@
-well))
+well#well_adv))
@@ -844,3 +844,3 @@
-to)
-(BE
-be)
+to#to_prep)
+(BE
+be#be_v)
@@ -848 +848 @@
-planted)
+planted#plant_v)
@@ -851 +851 @@
-in)
+in#in_prep)
@@ -854 +854 @@
-y=r=)
+y=r=#your_pron)
@@ -856 +856 @@
-wilderness))))
+wilderness#wilderness_n))))
@@ -864 +864 @@
-C.)
+C.#NA)
@@ -866 +866 @@
-Hatton)
+Hatton#NA)
@@ -880 +880 @@
-Pell-Mell)
+Pell-Mell#NA)
@@ -888 +888 @@
-April)
+April#NA)
@@ -890 +890 @@
-10)
+10#ten_adj)
@@ -894 +894 @@
-1692)
+1692#NA)
@@ -905 +905 @@
-The)
+The#the_adj)
@@ -907 +907 @@
-only)
+only#only_adj)
@@ -909 +909 @@
-news)
+news#news_n)
@@ -920 +920 @@
-we))
+we#we_pron))
@@ -922 +922 @@
-have)
+have#have_v)
@@ -925 +925 @@
-yet))
+yet#yet_adv))
@@ -927 +927 @@
-heard))))
+heard#hear_v))))
@@ -929 +929 @@
-is)
+is#be_v)
@@ -933 +933 @@
-y=t=)
+y=t=#that_conj)
@@ -937 +937 @@
-great)
+great#great_adj)
@@ -939 +939 @@
-preparations)
+preparations#preparation_n)
@@ -943 +943 @@
-are)
+are#be_v)
@@ -945 +945 @@
-making)
+making#make_v1)
@@ -948 +948 @@
-to)
+to#to_prep)
@@ -950 +950 @@
-send)
+send#send_v1)
@@ -953 +953 @@
-forces))
+forces#force_n1))
@@ -956 +956 @@
-to)
+to#to_prep)
@@ -959 +959 @@
-Guernsey)
+Guernsey#NA)
@@ -961 +961 @@
-and)
+and#and_conj1)
@@ -963 +963 @@
-Jersey))))))
+Jersey#NA))))))
@@ -968 +968 @@
-and)
+and#and_conj1)
@@ -971 +971 @@
-y=t=)
+y=t=#that_conj)
@@ -975,3 +975,3 @@
-engineers))
-(BEP
-are)
+engineers#engineer_n))
+(BEP
+are#be_v)
@@ -979 +979 @@
-going)
+going#go_v)
@@ -982 +982 @@
-thither))
+thither#thither_adv))
@@ -985 +985 @@
-to)
+to#to_prep)
@@ -987 +987 @@
-fortify)
+fortify#fortify_v)
@@ -990 +990 @@
-those)
+those#those_pron)
@@ -992 +992 @@
-islands)
+islands#island_n)
@@ -998 +998 @@
-whereof))
+whereof#whereof_adv))
@@ -1004 +1004 @@
-some)
+some#some_pron)
@@ -1007 +1007 @@
-of)
+of#of_prep)
@@ -1010 +1010 @@
-y=e=)
+y=e=#the_adj)
@@ -1012 +1012 @@
-cheifest)
+cheifest#chief_adj)
@@ -1014 +1014 @@
-officers)
+officers#officer_n)
@@ -1018 +1018 @@
-are)
+are#be_v)
@@ -1021 +1021 @@
-to)
+to#to_prep)
@@ -1023 +1023 @@
-be)
+be#be_v)
@@ -1025 +1025 @@
-made)
+made#make_v1)
@@ -1031 +1031 @@
-governors)))
+governors#governor_n)))
@@ -1036 +1036 @@
-as)
+as#as_adv)
@@ -1047 +1047 @@
-y=e=)
+y=e=#the_adj)
@@ -1049 +1049 @@
-common)
+common#common_adj)
@@ -1051 +1051 @@
-report)
+report#report_n)
@@ -1055 +1055 @@
-is)
+is#be_v)
@@ -1061 +1061 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1067 +1067 @@
-I))
+I#i_pron))
@@ -1069 +1069 @@
-thought)
+thought#think_v2)
@@ -1075 +1075 @@
-fit))
+fit#fit_adj))
@@ -1078 +1078 @@
-to)
+to#to_prep)
@@ -1080 +1080 @@
-hint)
+hint#hint_v)
@@ -1083 +1083 @@
-to)
+to#to_prep)
@@ -1086 +1086 @@
-y=r=)
+y=r=#your_pron)
@@ -1088 +1088 @@
-Lo=pp=)
+Lo=pp=#lordship_n)
@@ -1094 +1094 @@
-who))
+who#who_pron))
@@ -1101 +1101 @@
-will)
+will#will_v1)
@@ -1107 +1107 @@
-I))
+I#i_pron))
@@ -1109 +1109 @@
-assure)
+assure#assure_v)
@@ -1112 +1112 @@
-myself)))
+myself#myself_pron)))
@@ -1116 +1116 @@
-receive)
+receive#receive_v)
@@ -1119 +1119 @@
-a)
+a#a_adj2)
@@ -1123 +1123 @@
-more)
+more#more_adj)
@@ -1125 +1125 @@
-perfect))
+perfect#perfect_adj))
@@ -1128 +1128 @@
-and)
+and#and_conj1)
@@ -1131 +1131 @@
-fuller))))
+fuller#full_adj))))
@@ -1133 +1133 @@
-account)
+account#account_n)
@@ -1136 +1136 @@
-then)
+then#than_conj)
@@ -1147 +1147 @@
-I))
+I#i_pron))
@@ -1149 +1149 @@
-can)
+can#can_v1)
@@ -1151 +1151 @@
-give)))))))))))))))))))))))))
+give#give_v)))))))))))))))))))))))))
@@ -1160 +1160 @@
-Harrison))
+Harrison#NA))
@@ -1162 +1162 @@
-is)
+is#be_v)
@@ -1164 +1164 @@
-condemned)
+condemned#condemn_v)
@@ -1167 +1167 @@
-for)
+for#for_prep)
@@ -1170 +1170 @@
-y=e=)
+y=e=#the_adj)
@@ -1172 +1172 @@
-murder)
+murder#murder_n)
@@ -1175 +1175 @@
-of)
+of#of_prep)
@@ -1178 +1178 @@
-D=r=)
+D=r=#NA)
@@ -1180 +1180 @@
-Clinch)))))
+Clinch#NA)))))
@@ -1189 +1189 @@
-All)
+All#all_adj)
@@ -1191 +1191 @@
-provisions)
+provisions#provision_n)
@@ -1194 +1194 @@
-here)))
+here#here_adv)))
@@ -1196 +1196 @@
-are)
+are#be_v)
@@ -1199 +1199 @@
-very)
+very#very_adj)
@@ -1201 +1201 @@
-dear))
+dear#dear_adj1))
@@ -1210 +1210 @@
-A)
+A#a_adj2)
@@ -1212 +1212 @@
-loaf)
+loaf#loaf_n1)
@@ -1215 +1215 @@
-of)
+of#of_prep)
@@ -1218 +1218 @@
-bread)
+bread#bread_n)
@@ -1224 +1224 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1231 +1231 @@
-used)
+used#use_v)
@@ -1234 +1234 @@
-to)
+to#to_prep)
@@ -1236 +1236 @@
-cost)
+cost#cost_v)
@@ -1240 +1240 @@
-but)
+but#but_prep)
@@ -1242 +1242 @@
-9))
+9#nine_adj))
@@ -1244 +1244 @@
-pence))))))))
+pence#penny_n))))))))
@@ -1248 +1248 @@
-costs)
+costs#cost_v)
@@ -1251 +1251 @@
-now))
+now#now_adv))
@@ -1254 +1254 @@
-12@)
+12@#twelve_adj)
@@ -1256 +1256 @@
-@=d=))
+@=d=#penny_n))
@@ -1266 +1266 @@
-Mutton)
+Mutton#mutton_n)
@@ -1269 +1269 @@
-y=e=)
+y=e=#the_adj)
@@ -1271 +1271 @@
-best)))
+best#best_adj)))
@@ -1273 +1273 @@
-is)
+is#be_v)
@@ -1276 +1276 @@
-5@)
+5@#five_v)
@@ -1278 +1278 @@
-@=d=)
+@=d=#penny_n)
@@ -1281 +1281 @@
-y=e=)
+y=e=#the_adj)
@@ -1283 +1283 @@
-pound))))
+pound#pound_n1))))
@@ -1290 +1290 @@
-Beef))
+Beef#beef_n))
@@ -1293 +1293 @@
-3@)
+3@#three_adj)
@@ -1295 +1295 @@
-@=d=))))
+@=d=#penny_n))))
@@ -1304 +1304 @@
-This))
+This#this_pron))
@@ -1306 +1306 @@
-is)
+is#be_v)
@@ -1309 +1309 @@
-news)
+news#news_n)
@@ -1312 +1312 @@
-more)
+more#more_adj)
@@ -1314 +1314 @@
-welcome)
+welcome#welcome_n1)
@@ -1317 +1317 @@
-to)
+to#to_prep)
@@ -1320 +1320 @@
-those)
+those#those_pron)
@@ -1324 +1324 @@
-who))
+who#who_pron))
@@ -1331 +1331 @@
-live)
+live#live_v1)
@@ -1334 +1334 @@
-in)
+in#in_prep)
@@ -1337 +1337 @@
-y=e=)
+y=e=#the_adj)
@@ -1339 +1339 @@
-country)))))))
+country#country_n)))))))
@@ -1342 +1342 @@
-then)
+then#than_conj)
@@ -1345 +1345 @@
-to)
+to#to_prep)
@@ -1348 +1348 @@
-them)
+them#them_pron)
@@ -1352 +1352 @@
-who))
+who#who_pron))
@@ -1359 +1359 @@
-are)
+are#be_v)
@@ -1361 +1361 @@
-forced)
+forced#force_v1)
@@ -1364 +1364 @@
-to)
+to#to_prep)
@@ -1366 +1366 @@
-be)
+be#be_v)
@@ -1369 +1369 @@
-in)
+in#in_prep)
@@ -1372 +1372 @@
-town)))))))))))
+town#town_n)))))))))))
@@ -1380 +1380 @@
-C.)
+C.#NA)
@@ -1382 +1382 @@
-Hatton)
+Hatton#NA)
@@ -1396 +1396 @@
-May)
+May#NA)
@@ -1398 +1398 @@
-17)
+17#seventeen_adj)
@@ -1402 +1402 @@
-1692)
+1692#NA)
@@ -1413 +1413 @@
-This)
+This#this_pron)
@@ -1415 +1415 @@
-day))
+day#day_n))
@@ -1420 +1420 @@
-my)
+my#my_adj)
@@ -1422 +1422 @@
-L=d=))
+L=d=#lord_n))
@@ -1427 +1427 @@
-a)
+a#a_adj2)
@@ -1429 +1429 @@
-proclamation)
+proclamation#proclamation_n)
@@ -1433 +1433 @@
-is)
+is#be_v)
@@ -1435 +1435 @@
-come)
+come#come_v)
@@ -1438 +1438 @@
-forth))
+forth#forth_adv))
@@ -1441 +1441 @@
-signifying)
+signifying#signify_v)
@@ -1444 +1444 @@
-y=t=)
+y=t=#that_conj)
@@ -1452 +1452 @@
-y=e=)
+y=e=#the_adj)
@@ -1454 +1454 @@
-fleet))
+fleet#fleet_n1))
@@ -1456 +1456 @@
-being)
+being#be_v)
@@ -1458 +1458 @@
-out))
+out#out_adv))
@@ -1461 +1461 @@
-and)
+and#and_conj1)
@@ -1465 +1465 @@
-all)
+all#all_adj)
@@ -1467 +1467 @@
-things))
+things#thing_n1))
@@ -1470 +1470 @@
-soe)
+soe#so_adv)
@@ -1472 +1472 @@
-well))
+well#well_adv))
@@ -1474 +1474 @@
-prepared)
+prepared#prepare_v)
@@ -1477 +1477 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1480 +1480 @@
-y=e=)
+y=e=#the_adj)
@@ -1482 +1482 @@
-publick)
+publick#public_adj)
@@ -1484 +1484 @@
-enemy))))))
+enemy#enemy_n))))))
@@ -1489 +1489 @@
-it))
+it#it_pron))
@@ -1491 +1491 @@
-is)
+is#be_v)
@@ -1493 +1493 @@
-thought)
+thought#think_v2)
@@ -1499 +1499 @@
-fit))
+fit#fit_adj))
@@ -1502 +1502 @@
-to)
+to#to_prep)
@@ -1504 +1504 @@
-prorogue)
+prorogue#prorogue_v)
@@ -1507 +1507 @@
-y=e=)
+y=e=#the_adj)
@@ -1509 +1509 @@
-parliament))
+parliament#parliament_n1))
@@ -1512 +1512 @@
-to)
+to#to_prep)
@@ -1515 +1515 @@
-y=e=)
+y=e=#the_adj)
@@ -1517 +1517 @@
-14=th=)
+14=th=#NA)
@@ -1520 +1520 @@
-of)
+of#of_prep)
@@ -1523 +1523 @@
-June))))))))))
+June#NA))))))))))
@@ -1532 +1532 @@
-The)
+The#the_adj)
@@ -1534 +1534 @@
-rencounter)
+rencounter#rencounter_n)
@@ -1543 +1543 @@
-I))
+I#i_pron))
@@ -1545 +1545 @@
-told)
+told#tell_v)
@@ -1548 +1548 @@
-y=r=)
+y=r=#your_pron)
@@ -1550 +1550 @@
-Lo=ppe=))
+Lo=ppe=#lordship_n))
@@ -1553 +1553 @@
-of)
+of#of_prep)
@@ -1558 +1558 @@
-in)
+in#in_prep)
@@ -1561 +1561 @@
-my)
+my#my_adj)
@@ -1563 +1563 @@
-last)))))
+last#last_adv)))))
@@ -1566 +1566 @@
-betwixt)
+betwixt#betwixt_prep)
@@ -1570 +1570 @@
-y=e=)
+y=e=#the_adj)
@@ -1572 +1572 @@
-L=d=)
+L=d=#lord_n)
@@ -1575 +1575 @@
-Thanet)))
+Thanet#NA)))
@@ -1578 +1578 @@
-and)
+and#and_conj1)
@@ -1581 +1581 @@
-L=d=)
+L=d=#NA)
@@ -1583 +1583 @@
-Clare))))))
+Clare#NA))))))
@@ -1585 +1585 @@
-wase)
+wase#be_v)
@@ -1588 +1588 @@
-very)
+very#very_adj)
@@ -1590 +1590 @@
-true))
+true#true_adj))
@@ -1598 +1598 @@
-but)
+but#but_prep)
@@ -1601 +1601 @@
-y=e=)
+y=e=#the_adj)
@@ -1603 +1603 @@
-L=d=)
+L=d=#lord_n)
@@ -1606 +1606 @@
-Thanet)))
+Thanet#NA)))
@@ -1608 +1608 @@
-wase)
+wase#be_v)
@@ -1610 +1610 @@
-not)
+not#not_adv)
@@ -1612 +1612 @@
-hurt)
+hurt#hurt_v)
@@ -1615 +1615 @@
-in)
+in#in_prep)
@@ -1620 +1620 @@
-y=e=)
+y=e=#the_adj)
@@ -1622 +1622 @@
-thigh)
+thigh#thigh_n)
@@ -1625 +1625 @@
-but)
+but#but_prep)
@@ -1628 +1628 @@
-shoulder)))))
+shoulder#shoulder_n)))))
@@ -1636 +1636 @@
-and)
+and#and_conj1)
@@ -1639 +1639 @@
-w=th=)
+w=th=#with_prep)
@@ -1642 +1642 @@
-parrying)
+parrying#parry_v)
@@ -1645 +1645 @@
-w=th=)
+w=th=#with_prep)
@@ -1648 +1648 @@
-their)
+their#their_adj)
@@ -1650 +1650 @@
-left)
+left#left_adj1)
@@ -1652 +1652 @@
-hands)))
+hands#hand_n)))
@@ -1655 +1655 @@
-y=e=)
+y=e=#the_adj)
@@ -1657 +1657 @@
-thrusts))))
+thrusts#thrust_n))))
@@ -1660 +1660 @@
-y=e=)
+y=e=#the_adj)
@@ -1662 +1662 @@
-hands)
+hands#hand_n)
@@ -1665 +1665 @@
-of)
+of#of_prep)
@@ -1668 +1668 @@
-both)
+both#both_pron)
@@ -1670 +1670 @@
-y=e=)
+y=e=#the_adj)
@@ -1672 +1672 @@
-lords))))
+lords#lord_n))))
@@ -1674 +1674 @@
-were)
+were#be_v)
@@ -1676 +1676 @@
-cut)
+cut#cut_v)
@@ -1684 +1684 @@
-But)
+But#but_prep)
@@ -1687 +1687 @@
-it))
+it#it_pron))
@@ -1689 +1689 @@
-is)
+is#be_v)
@@ -1691 +1691 @@
-said)
+said#say_v1)
@@ -1698 +1698 @@
-my)
+my#my_adj)
@@ -1700 +1700 @@
-L=d=)
+L=d=#lord_n)
@@ -1703 +1703 @@
-Thanet)))
+Thanet#NA)))
@@ -1708 +1708 @@
-before)
+before#before_adv)
@@ -1716 +1716 @@
-company))
+company#company_n))
@@ -1718 +1718 @@
-came)
+came#come_v)
@@ -1720 +1720 @@
-in))
+in#in_adv))
@@ -1723 +1723 @@
-and)
+and#and_conj1)
@@ -1728 +1728 @@
-parted)
+parted#part_v)
@@ -1731 +1731 @@
-y=m=)))))))
+y=m=#them_pron)))))))
@@ -1735 +1735 @@
-had)
+had#have_v)
@@ -1738 +1738 @@
-his)
+his#his_adj)
@@ -1740 +1740 @@
-sword))
+sword#sword_n))
@@ -1743 +1743 @@
-ag=t=)
+ag=t=#against_prep)
@@ -1747 +1747 @@
-y=e=)
+y=e=#the_adj)
@@ -1749 +1749 @@
-L=d=)
+L=d=#lord_n)
@@ -1752 +1752 @@
-Clare)))
+Clare#NA)))
@@ -1754 +1754 @@
-breast)))))
+breast#breast_n)))))
@@ -1762 +1762 @@
-but)
+but#but_prep)
@@ -1767 +1767 @@
-very)
+very#very_adj)
@@ -1769 +1769 @@
-generously))
+generously#generously_adv))
@@ -1771 +1771 @@
-wou'd)
+wou'd#will_v1)
@@ -1773 +1773 @@
-not)
+not#not_adv)
@@ -1775 +1775 @@
-thrust)
+thrust#thrust_v)
@@ -1778 +1778 @@
-home))
+home#home_n1))
@@ -1787 +1787 @@
-L=d=)
+L=d=#NA)
@@ -1789 +1789 @@
-Yarmouth))
+Yarmouth#NA))
@@ -1791 +1791 @@
-is)
+is#be_v)
@@ -1793 +1793 @@
-sent)
+sent#send_v1)
@@ -1796 +1796 @@
-to)
+to#to_prep)
@@ -1799 +1799 @@
-y=e=)
+y=e=#the_adj)
@@ -1801 +1801 @@
-Tower))))
+Tower#NA))))
@@ -1807 +1807 @@
-and)
+and#and_conj1)
@@ -1810 +1810 @@
-his)
+his#his_adj)
@@ -1812 +1812 @@
-lady))
+lady#lady_n))
@@ -1814 +1814 @@
-is)
+is#be_v)
@@ -1817 +1817 @@
-w=th=)
+w=th=#with_prep)
@@ -1820 +1820 @@
-him)))
+him#him_pron)))
@@ -1830 +1830 @@
-The)
+The#the_adj)
@@ -1832 +1832 @@
-L=d=)
+L=d=#lord_n)
@@ -1835 +1835 @@
-Middleton)))
+Middleton#NA)))
@@ -1841 +1841 @@
-L=d=)
+L=d=#NA)
@@ -1843 +1843 @@
-Dunmore)))
+Dunmore#NA)))
@@ -1848 +1848 @@
-and)
+and#and_conj1)
@@ -1851 +1851 @@
-S=r=)
+S=r=#NA)
@@ -1853 +1853 @@
-Andrew)
+Andrew#NA)
@@ -1855 +1855 @@
-Forrester)))
+Forrester#NA)))
@@ -1860 +1860 @@
-all)
+all#all_adj)
@@ -1862 +1862 @@
-3)
+3#three_adj)
@@ -1865 +1865 @@
-in)
+in#in_prep)
@@ -1868 +1868 @@
-y=e=)
+y=e=#the_adj)
@@ -1870 +1870 @@
-proclamation)))))
+proclamation#proclamation_n)))))
@@ -1874 +1874 @@
-were)
+were#be_v)
@@ -1876 +1876 @@
-seized)
+seized#seize_v)
@@ -1879 +1879 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -1888 +1888 @@
-My)
+My#my_adj)
@@ -1890 +1890 @@
-wife)
+wife#wife_n)
@@ -1893 +1893 @@
-and)
+and#and_conj1)
@@ -1896 +1896 @@
-self))))
+self#self_pron))))
@@ -1898 +1898 @@
-are)
+are#be_v)
@@ -1901 +1901 @@
-to)
+to#to_prep)
@@ -1905 +1905 @@
-y=r=)
+y=r=#your_pron)
@@ -1907 +1907 @@
-Lo=ppe=))
+Lo=ppe=#lordship_n))
@@ -1913 +1913 @@
-Lady)
+Lady#NA)
@@ -1915 +1915 @@
-Hatton)))
+Hatton#NA)))
@@ -1920 +1920 @@
-and)
+and#and_conj1)
@@ -1923 +1923 @@
-all)
+all#all_adj)
@@ -1926 +1926 @@
-at)
+at#at_prep)
@@ -1929 +1929 @@
-Kirby)))))))
+Kirby#NA)))))))
@@ -1934 +1934 @@
-most)
+most#most_adj)
@@ -1936 +1936 @@
-faithfull))
+faithfull#faithful_adj))
@@ -1939 +1939 @@
-and)
+and#and_conj1)
@@ -1942 +1942 @@
-very)
+very#very_adj)
@@ -1944 +1944 @@
-humble))))
+humble#humble_adj))))
@@ -1946 +1946 @@
-servants))
+servants#servant_n))
@@ -1954 +1954 @@
-C.)
+C.#NA)
@@ -1956 +1956 @@
-Hatton)
+Hatton#NA)
@@ -1965 +1965 @@
-It))
+It#it_pron))
@@ -1967 +1967 @@
-will)
+will#will_v1)
@@ -1969 +1969 @@
-be)
+be#be_v)
@@ -1972 +1972 @@
-time)
+time#time_n)
@@ -1975 +1975 @@
-to)
+to#to_prep)
@@ -1977 +1977 @@
-watch)
+watch#watch_v)
@@ -1980 +1980 @@
-the)
+the#the_adj)
@@ -1982 +1982 @@
-ripening)
+ripening#ripening_n)
@@ -1985 +1985 @@
-of)
+of#of_prep)
@@ -1988 +1988 @@
-y=e=)
+y=e=#the_adj)
@@ -1990 +1990 @@
-seeds)
+seeds#seed_n)
@@ -1993 +1993 @@
-of)
+of#of_prep)
@@ -1996 +1996 @@
-y=e=)
+y=e=#the_adj)
@@ -1998 +1998 @@
-broad)
+broad#broad_adj1)
@@ -2000 +2000 @@
-leaf)
+leaf#leaf_n1)
@@ -2002 +2002 @@
-elme))))))))
+elme#elm_n))))))))
