--- eliz-1600-e2-p2.psd.vert	2025-09-10 13:51:18
+++ eliz-1600-e2-p2.lem.vert	2025-09-10 13:51:25
@@ -14 +14 @@
-Althoughe)
+Althoughe#although_conj)
@@ -21 +21 @@
-we))
+we#we_pron))
@@ -23 +23 @@
-have)
+have#have_v)
@@ -25 +25 @@
-deferred)
+deferred#defer_v1)
@@ -28 +28 @@
-longe))
+longe#long_adj1))
@@ -31 +31 @@
-to)
+to#to_prep)
@@ -33 +33 @@
-p'sent)
+p'sent#present_v)
@@ -36 +36 @@
-to)
+to#to_prep)
@@ -39 +39 @@
-yo=w=)))
+yo=w=#you_pron)))
@@ -42 +42 @@
-o=r=)
+o=r=#our_pron)
@@ -44 +44 @@
-grieved)
+grieved#grieved_adj)
@@ -46 +46 @@
-thoughtes)))
+thoughtes#thought_n)))
@@ -51 +51 @@
-because)
+because#because_adv)
@@ -58 +58 @@
-we))
+we#we_pron))
@@ -60 +60 @@
-liked)
+liked#like_v1)
@@ -63 +63 @@
-full)
+full#full_adj)
@@ -65 +65 @@
-ill))
+ill#ill_adv))
@@ -68 +68 @@
-to)
+to#to_prep)
@@ -70 +70 @@
-make)
+make#make_v1)
@@ -74 +74 @@
-yo=w=)
+yo=w=#you_pron)
@@ -79 +79 @@
-the)
+the#the_adj)
@@ -81 +81 @@
-firste)
+firste#first_adj)
@@ -83 +83 @@
-refleccon)
+refleccon#reflection_n)
@@ -86 +86 @@
-of)
+of#of_prep)
@@ -89 +89 @@
-o=r=)
+o=r=#our_pron)
@@ -91 +91 @@
-misfortunes))))
+misfortunes#misfortune_n))))
@@ -97 +97 @@
-whome))
+whome#whom_pron))
@@ -103 +103 @@
-we))
+we#we_pron))
@@ -105 +105 @@
-haue)
+haue#have_v)
@@ -108 +108 @@
-allwaies))
+allwaies#always_adv))
@@ -110 +110 @@
-soughte)
+soughte#seek_v)
@@ -116 +116 @@
-rather))
+rather#rather_adv))
@@ -118 +118 @@
-to)
+to#to_prep)
@@ -121 +121 @@
-cherishe)
+cherishe#cherish_v)
@@ -123 +123 @@
-and)
+and#and_conj1)
@@ -125 +125 @@
-comforte)))))))))))))
+comforte#comfort_v)))))))))))))
@@ -130 +130 @@
-yet))
+yet#yet_adv))
@@ -133 +133 @@
-knowinge)
+knowinge#know_v)
@@ -136 +136 @@
-that)
+that#that_conj)
@@ -140 +140 @@
-necessitie))
+necessitie#necessity_n))
@@ -142 +142 @@
-must)
+must#must_v1)
@@ -144 +144 @@
-bringe)
+bringe#bring_v)
@@ -147 +147 @@
-to)
+to#to_prep)
@@ -150 +150 @@
-yo=r=)
+yo=r=#your_pron)
@@ -152 +152 @@
-eares)))
+eares#ear_n1)))
@@ -155 +155 @@
-&)
+&#and_conj1)
@@ -158 +158 @@
-nature))
+nature#nature_n))
@@ -161 +161 @@
-consequentlye))
+consequentlye#consequently_adv))
@@ -163 +163 @@
-move))
+move#move_v))
@@ -166 +166 @@
-passions))
+passions#passion_n))
@@ -169 +169 @@
-in)
+in#in_prep)
@@ -172 +172 @@
-yo=r=)
+yo=r=#your_pron)
@@ -174 +174 @@
-harte))))))
+harte#heart_n))))))
@@ -179 +179 @@
-wee))
+wee#we_pron))
@@ -181 +181 @@
-resolved)
+resolved#resolve_v)
@@ -186 +186 @@
-noe)
+noe#no_adj)
@@ -188 +188 @@
-long=er=)))
+long=er=#long_adj1)))
@@ -190 +190 @@
-to)
+to#to_prep)
@@ -192 +192 @@
-smother)
+smother#smother_v)
@@ -195 +195 @@
-either)
+either#either_pron)
@@ -198 +198 @@
-o=r=)
+o=r=#our_pron)
@@ -200 +200 @@
-care)
+care#care_n1)
@@ -203 +203 @@
-for)
+for#for_prep)
@@ -206 +206 @@
-yo=r=)
+yo=r=#your_pron)
@@ -208 +208 @@
-sorrowe))))
+sorrowe#sorrow_n))))
@@ -211 +211 @@
-or)
+or#or_conj2)
@@ -214 +214 @@
-the)
+the#the_adj)
@@ -216 +216 @@
-sympathy)
+sympathy#sympathy_n)
@@ -219 +219 @@
-of)
+of#of_prep)
@@ -222 +222 @@
-o=r=)
+o=r=#our_pron)
@@ -224 +224 @@
-greife)))))
+greife#grief_n)))))
@@ -227 +227 @@
-for)
+for#for_prep)
@@ -230 +230 @@
-his)
+his#his_adj)
@@ -232 +232 @@
-death)
+death#death_n)
@@ -236 +236 @@
-wherein))
+wherein#wherein_adv))
@@ -244 +244 @@
-y=e=)
+y=e=#the_adj)
@@ -246 +246 @@
-societie)
+societie#society_n)
@@ -249 +249 @@
-in)
+in#in_prep)
@@ -252 +252 @@
-sorrowe))))
+sorrowe#sorrow_n))))
@@ -254 +254 @@
-may)
+may#may_v1)
@@ -256 +256 @@
-worke)
+worke#work_v)
@@ -259 +259 @@
-diminution))))))))
+diminution#diminution_n))))))))
@@ -268 +268 @@
-We))
+We#we_pron))
@@ -270 +270 @@
-doe)
+doe#do_v)
@@ -272 +272 @@
-assure)
+assure#assure_v)
@@ -275 +275 @@
-yo=w=))
+yo=w=#you_pron))
@@ -280 +280 @@
-by)
+by#by_prep)
@@ -283 +283 @@
-this)
+this#this_pron)
@@ -285 +285 @@
-true)
+true#true_adj)
@@ -287 +287 @@
-messenger)
+messenger#messenger_n)
@@ -290 +290 @@
-of)
+of#of_prep)
@@ -293 +293 @@
-o=r=)
+o=r=#our_pron)
@@ -295 +295 @@
-mynde)))))
+mynde#mind_n1)))))
@@ -298 +298 @@
-y=t=)
+y=t=#that_conj)
@@ -302 +302 @@
-nature))
+nature#NA))
@@ -304 +304 @@
-can)
+can#can_v1)
@@ -306 +306 @@
-have)
+have#have_v)
@@ -308 +308 @@
-stirred)
+stirred#stir_v)
@@ -312 +312 @@
-noe)
+noe#no_adj)
@@ -314 +314 @@
-more))
+more#more_adj))
@@ -316 +316 @@
-dolours)
+dolours#dolour_n)
@@ -319 +319 @@
-of)
+of#of_prep)
@@ -322 +322 @@
-affection)))
+affection#affection_n)))
@@ -327 +327 @@
-in)
+in#in_prep)
@@ -330 +330 @@
-yo=w=)))
+yo=w=#you_pron)))
@@ -333 +333 @@
-as)
+as#as_adv)
@@ -336 +336 @@
-a)
+a#a_adj2)
@@ -338 +338 @@
-mother)))
+mother#mother_n1)))
@@ -341 +341 @@
-for)
+for#for_prep)
@@ -344 +344 @@
-a)
+a#a_adj2)
@@ -346 +346 @@
-deare)
+deare#dear_adj1)
@@ -348 +348 @@
-sonne)))
+sonne#son_n1)))
@@ -353 +353 @@
-then)
+then#than_conj)
@@ -362 +362 @@
-the)
+the#the_adj)
@@ -364 +364 @@
-gratefullnes)
+gratefullnes#gratefulness_n)
@@ -367 +367 @@
-and)
+and#and_conj1)
@@ -370 +370 @@
-memorye)))
+memorye#memory_n)))
@@ -373 +373 @@
-of)
+of#of_prep)
@@ -376 +376 @@
-his)
+his#his_adj)
@@ -378 +378 @@
-service)
+service#service_n1)
@@ -381 +381 @@
-past)))))
+past#past_adj)))))
@@ -383 +383 @@
-hath)
+hath#have_v)
@@ -385 +385 @@
-wroughte)
+wroughte#work_v)
@@ -388 +388 @@
-in)
+in#in_prep)
@@ -391 +391 @@
-us)
+us#us_pron)
@@ -394 +394 @@
-his)
+his#his_adj)
@@ -396 +396 @@
-sou'aigne))))
+sou'aigne#sovereign_n))))
@@ -403 +403 @@
-app=r=hension)
+app=r=hension#apprehension_n)
@@ -406 +406 @@
-of)
+of#of_prep)
@@ -409 +409 @@
-o=r=)
+o=r=#our_pron)
@@ -411 +411 @@
-mysse)
+mysse#miss_n1)
@@ -414 +414 @@
-of)
+of#of_prep)
@@ -418 +418 @@
-so)
+so#so_adv)
@@ -420 +420 @@
-worthye))
+worthye#worthy_adj))
@@ -422 +422 @@
-a)
+a#a_adj2)
@@ -424 +424 @@
-servant)))))))))))
+servant#servant_n)))))))))))
@@ -432 +432 @@
-But)
+But#but_prep)
@@ -435 +435 @@
-now)
+now#now_adv)
@@ -440 +440 @@
-y=t=)
+y=t=#that_conj)
@@ -447 +447 @@
-in)
+in#in_prep)
@@ -452 +452 @@
-nature)))
+nature#nature_n)))
@@ -455 +455 @@
-his)
+his#his_adj)
@@ -457 +457 @@
-com~on)
+com~on#common_adj)
@@ -459 +459 @@
-worke))
+worke#work_n))
@@ -461 +461 @@
-is)
+is#be_v)
@@ -463 +463 @@
-done))
+done#do_v))
@@ -466 +466 @@
-and)
+and#and_conj1)
@@ -472 +472 @@
-he)
+he#he_pron)
@@ -477 +477 @@
-that)
+that#that_conj)
@@ -482 +482 @@
-was)
+was#be_v)
@@ -484 +484 @@
-borne)
+borne#bear_v1)
@@ -487 +487 @@
-to)
+to#to_prep)
@@ -489 +489 @@
-dye)))))
+dye#die_v1)))))
@@ -491 +491 @@
-hath)
+hath#have_v)
@@ -493 +493 @@
-payd)
+payd#pay_v1)
@@ -496 +496 @@
-his)
+his#his_adj)
@@ -498 +498 @@
-tribute)))))))
+tribute#tribute_n)))))))
@@ -502 +502 @@
-let)
+let#let_v1)
@@ -506 +506 @@
-that)
+that#that_pron)
@@ -508 +508 @@
-Christian)
+Christian#christian_adj)
@@ -510 +510 @@
-discretion)
+discretion#discretion_n)
@@ -514 +514 @@
-staye)
+staye#stay_v1)
@@ -517 +517 @@
-the)
+the#the_adj)
@@ -519 +519 @@
-flux)
+flux#flux_n)
@@ -522 +522 @@
-of)
+of#of_prep)
@@ -525 +525 @@
-yo=r=)
+yo=r=#your_pron)
@@ -527 +527 @@
-imoderate)
+imoderate#immoderate_adj)
@@ -531 +531 @@
-greivinge))))
+greivinge#grieving_n))))
@@ -535 +535 @@
-w=ch=))
+w=ch=#which_adj))
@@ -542 +542 @@
-hathe)
+hathe#have_v)
@@ -544 +544 @@
-instructed)
+instructed#instruct_v)
@@ -547 +547 @@
-yo=w=))
+yo=w=#you_pron))
@@ -549 +549 @@
-both)
+both#NA)
@@ -552 +552 @@
-by)
+by#by_prep)
@@ -557,3 +557,3 @@
-example)
-(CONJ
-&)
+example#example_n)
+(CONJ
+&#and_conj1)
@@ -561 +561 @@
-knowledge)))
+knowledge#knowledge_n)))
@@ -566 +566 @@
-that)
+that#that_conj)
@@ -570 +570 @@
-nothinge)
+nothinge#nothing_pron)
@@ -573 +573 @@
-of)
+of#of_prep)
@@ -576 +576 @@
-this)
+this#this_pron)
@@ -578 +578 @@
-kinde))))
+kinde#kind_n))))
@@ -580 +580 @@
-hath)
+hath#have_v)
@@ -582 +582 @@
-happened)
+happened#happen_v)
@@ -585 +585 @@
-but)
+but#but_prep)
@@ -587 +587 @@
-by)
+by#by_prep)
@@ -590 +590 @@
-gode's)
+gode's#NA)
@@ -592 +592 @@
-p'vidence))))))))
+p'vidence#providence_n))))))))
@@ -600 +600 @@
-and)
+and#and_conj1)
@@ -602 +602 @@
-let)
+let#let_v1)
@@ -606 +606 @@
-these)
+these#these_pron)
@@ -608 +608 @@
-lynes)
+lynes#line_n2)
@@ -611 +611 @@
-from)
+from#from_prep)
@@ -614 +614 @@
-yo=r=)
+yo=r=#your_pron)
@@ -617 +617 @@
-lovinge)
+lovinge#loving_adj)
@@ -619 +619 @@
-&)
+&#and_conj1)
@@ -621 +621 @@
-gracous))
+gracous#gracious_adj))
@@ -625 +625 @@
-sou'aigne))))
+sou'aigne#sovereign_n))))
@@ -627 +627 @@
-serve)
+serve#serve_v1)
@@ -630 +630 @@
-to)
+to#to_prep)
@@ -632 +632 @@
-assure)
+assure#assure_v)
@@ -635 +635 @@
-yo=w=))
+yo=w=#you_pron))
@@ -638 +638 @@
-y=t=)
+y=t=#that_conj)
@@ -642 +642 @@
-there))
+there#there_adv))
@@ -644 +644 @@
-shall)
+shall#shall_v)
@@ -647 +647 @@
-ever))
+ever#ever_adv))
@@ -649 +649 @@
-appeare)
+appeare#appear_v)
@@ -652 +652 @@
-the)
+the#the_adj)
@@ -654 +654 @@
-livelye)
+livelye#lively_adj)
@@ -656 +656 @@
-characters)
+characters#character_n)
@@ -661 +661 @@
-of)
+of#of_prep)
@@ -664 +664 @@
-yo=w=)
+yo=w=#you_pron)
@@ -666 +666 @@
-and)
+and#and_conj1)
@@ -668 +668 @@
-yo=rs=)))
+yo=rs=#yours_pron)))
@@ -673 +673 @@
-that)
+that#that_conj)
@@ -678 +678 @@
-are)
+are#be_v)
@@ -681 +681 @@
-beste)
+beste#best_adj)
@@ -684 +684 @@
-in)
+in#in_prep)
@@ -687 +687 @@
-o=r=)
+o=r=#our_pron)
@@ -690 +690 @@
-valewinge)
+valewinge#value_v)
@@ -693 +693 @@
-rightelye))
+rightelye#rightly_adv))
@@ -696 +696 @@
-all)
+all#all_adj)
@@ -698 +698 @@
-theire)
+theire#their_adj)
@@ -701 +701 @@
-faithfull)
+faithfull#faithful_adj)
@@ -703 +703 @@
-and)
+and#and_conj1)
@@ -705 +705 @@
-honest))
+honest#honest_adj))
@@ -707 +707 @@
-indevo=res=)))))))))))))
+indevo=res=#endeavour_n)))))))))))))
@@ -716 +716 @@
-Nowe))
+Nowe#now_adv))
@@ -719 +719 @@
-at)
+at#at_prep)
@@ -722 +722 @@
-this)
+this#this_pron)
@@ -724 +724 @@
-tyme)))
+tyme#time_n)))
@@ -727 +727 @@
-we))
+we#we_pron))
@@ -729 +729 @@
-will)
+will#will_v1)
@@ -731 +731 @@
-not)
+not#not_adv)
@@ -733 +733 @@
-wryte)
+wryte#write_v)
@@ -736 +736 @@
-of)
+of#of_prep)
@@ -739 +739 @@
-this)
+this#this_pron)
@@ -741 +741 @@
-unsilent)
+unsilent#unsilent_adj)
@@ -743 +743 @@
-subjecte))))
+subjecte#subject_n))))
@@ -749 +749 @@
-but)
+but#but_prep)
@@ -753 +753 @@
-haue)
+haue#have_v)
@@ -757 +757 @@
-dispatched)
+dispatched#dispatch_v)
@@ -760 +760 @@
-this)
+this#this_pron)
@@ -762 +762 @@
-gent.))
+gent.#gentleman_n))
@@ -765 +765 @@
-to)
+to#to_prep)
@@ -767 +767 @@
-visitt)
+visitt#visit_v)
@@ -770 +770 @@
-both)
+both#both_pron)
@@ -773 +773 @@
-yo=r=)
+yo=r=#your_pron)
@@ -775 +775 @@
-Lorde))
+Lorde#lord_n))
@@ -778 +778 @@
-and)
+and#and_conj1)
@@ -781 +781 @@
-yo=w=))))
+yo=w=#you_pron))))
@@ -785 +785 @@
-to)
+to#to_prep)
@@ -787 +787 @@
-condole)
+condole#condole_v)
@@ -790 +790 @@
-w=th=)
+w=th=#with_prep)
@@ -793 +793 @@
-yo=w=)))
+yo=w=#you_pron)))
@@ -796 +796 @@
-in)
+in#in_prep)
@@ -799 +799 @@
-the)
+the#the_adj)
@@ -801 +801 @@
-true)
+true#true_adj)
@@ -803 +803 @@
-sence)
+sence#sense_n)
@@ -806 +806 @@
-of)
+of#of_prep)
@@ -809 +809 @@
-yo=r=)
+yo=r=#your_pron)
@@ -811 +811 @@
-Love))))))
+Love#love_n1))))))
@@ -814 +814 @@
-and)
+and#and_conj1)
@@ -817 +817 @@
-to)
+to#to_prep)
@@ -819 +819 @@
-pray)
+pray#pray_v)
@@ -822 +822 @@
-yo=w=))
+yo=w=#you_pron))
@@ -825 +825 @@
-that)
+that#that_conj)
@@ -829 +829 @@
-the)
+the#the_adj)
@@ -831 +831 @@
-worlde))
+worlde#world_n))
@@ -833 +833 @@
-may)
+may#may_v1)
@@ -835 +835 @@
-see)
+see#see_v)
@@ -838 +838 @@
-that)
+that#that_conj)
@@ -844 +844 @@
-tyme))
+tyme#time_n))
@@ -846 +846 @@
-curethe)
+curethe#cure_v1)
@@ -849 +849 @@
-in)
+in#in_prep)
@@ -852 +852 @@
-weak)
+weak#weak_adj)
@@ -854 +854 @@
-myndes)))
+myndes#mind_n1)))
@@ -857 +857 @@
-that)
+that#that_pron)
@@ -859 +859 @@
-discretion)
+discretion#discretion_n)
@@ -862 +862 @@
-&)
+&#and_conj1)
@@ -865 +865 @@
-moderacon)))
+moderacon#moderation_n)))
@@ -876 +876 @@
-he))
+he#he_pron))
@@ -878 +878 @@
-hopeth)
+hopeth#hope_v)
@@ -881 +881 @@
-in)
+in#in_prep)
@@ -884 +884 @@
-yo=w=)))
+yo=w=#you_pron)))
@@ -887 +887 @@
-in)
+in#in_prep)
@@ -890 +890 @@
-this)
+this#this_pron)
@@ -892 +892 @@
-accident)
+accident#accident_n)
@@ -898 +898 @@
-wheare))
+wheare#where_adv))
@@ -906 +906 @@
-there))
+there#there_adv))
@@ -908 +908 @@
-is)
+is#be_v)
@@ -911 +911 @@
-the)
+the#the_adj)
@@ -913 +913 @@
-fitt)
+fitt#fit_adj)
@@ -915 +915 @@
-occasion)
+occasion#occasion_n1)
@@ -918 +918 @@
-to)
+to#to_prep)
@@ -920 +920 @@
-demonstrate)
+demonstrate#demonstrate_v)
@@ -923 +923 @@
-true)
+true#true_adj)
@@ -925 +925 @@
-patience)
+patience#patience_n.I)
@@ -928 +928 @@
-and)
+and#and_conj1)
@@ -931 +931 @@
-moderation)))))))))))))))))))))
+moderation#moderation_n)))))))))))))))))))))
