--- interview-1520-e1-p2.psd.vert	2025-09-10 13:51:18
+++ interview-1520-e1-p2.lem.vert	2025-09-10 13:51:25
@@ -14 +14 @@
-Sir))
+Sir#sir_n))
@@ -17 +17 @@
-Please)
+Please#please_v)
@@ -20 +20 @@
-it))
+it#it_pron))
@@ -23 +23 @@
-your)
+your#your_pron)
@@ -25 +25 @@
-Grace)))
+Grace#grace_n)))
@@ -31 +31 @@
-over)
+over#over_prep)
@@ -33 +33 @@
-and)
+and#and_conj1)
@@ -35 +35 @@
-besides))
+besides#besides_adv))
@@ -38 +38 @@
-the)
+the#the_adj)
@@ -40 +40 @@
-joyous)
+joyous#joyous_adj)
@@ -42 +42 @@
-personnall)
+personnall#personal_adj)
@@ -44 +44 @@
-Commyng)
+Commyng#coming_n)
@@ -47 +47 @@
-to)
+to#to_prep)
@@ -50 +50 @@
-your)
+your#your_pron)
@@ -52 +52 @@
-Highnesse))))
+Highnesse#highness_n))))
@@ -56 +56 @@
-of)
+of#of_prep)
@@ -59 +59 @@
-the)
+the#the_adj)
@@ -61 +61 @@
-Emperor)
+Emperor#emperor_n)
@@ -68 +68 @@
-within)
+within#within_adv)
@@ -71 +71 @@
-this)
+this#this_pron)
@@ -73 +73 @@
-your)
+your#your_pron)
@@ -75 +75 @@
-Reame))))
+Reame#realm_n))))
@@ -80 +80 @@
-and)
+and#and_conj1)
@@ -83 +83 @@
-of)
+of#of_prep)
@@ -86 +86 @@
-the)
+the#the_adj)
@@ -88 +88 @@
-comfortable)
+comfortable#comfortable_adj)
@@ -90 +90 @@
-mutuell)
+mutuell#mutual_adj)
@@ -92 +92 @@
-communicacion)
+communicacion#communication_n)
@@ -95 +95 @@
-and)
+and#and_conj1)
@@ -98 +98 @@
-entreteignment)))
+entreteignment#entertainment_n)))
@@ -103 +103 @@
-that)
+that#that_conj)
@@ -108 +108 @@
-was)
+was#be_v)
@@ -111 +111 @@
-betwene)
+betwene#between_prep)
@@ -114 +114 @@
-you))))))))))
+you#you_pron))))))))))
@@ -120 +120 @@
-whoys)
+whoys#whose_pron)
@@ -122 +122 @@
-commyng)
+commyng#coming_n)
@@ -125 +125 @@
-into)
+into#into_prep)
@@ -128 +128 @@
-your)
+your#your_pron)
@@ -130 +130 @@
-said)
+said#said_adj)
@@ -132 +132 @@
-Reame))))
+Reame#realm_n))))
@@ -139 +139 @@
-is)
+is#be_v)
@@ -142 +142 @@
-greatly))
+greatly#greatly_adv))
@@ -145 +145 @@
-estemed)
+estemed#esteem_v)
@@ -147 +147 @@
-and)
+and#and_conj1)
@@ -149 +149 @@
-noted))
+noted#note_v2))
@@ -152 +152 @@
-to)
+to#to_prep)
@@ -156 +156 @@
-th'@)
+th'@#the_adj)
@@ -158 +158 @@
-@inestimable)
+@inestimable#inestimable_adj)
@@ -160 +160 @@
-honor)
+honor#honour_n)
@@ -163 +163 @@
-of)
+of#of_prep)
@@ -166 +166 @@
-your)
+your#your_pron)
@@ -168 +168 @@
-Highnes))))
+Highnes#highness_n))))
@@ -171 +171 @@
-and)
+and#and_conj1)
@@ -174 +174 @@
-your)
+your#your_pron)
@@ -176 +176 @@
-said)
+said#said_adj)
@@ -178 +178 @@
-Reame)))))))
+Reame#realm_n)))))))
@@ -183 +183 @@
-we))
+we#we_pron))
@@ -185 +185 @@
-have)
+have#have_v)
@@ -187 +187 @@
-had)
+had#have_v)
@@ -190 +190 @@
-knowledge))
+knowledge#knowledge_n))
@@ -193 +193 @@
-to)
+to#to_prep)
@@ -196 +196 @@
-our)
+our#our_pron)
@@ -199 +199 @@
-most)
+most#most_adj)
@@ -201 +201 @@
-singular))
+singular#singular_adj))
@@ -203 +203 @@
-comforte)
+comforte#comfort_n)
@@ -206 +206 @@
-and)
+and#and_conj1)
@@ -209 +209 @@
-rejoysing)))))
+rejoysing#rejoicing_n)))))
@@ -215 +215 @@
-not)
+not#not_adv)
@@ -217 +217 @@
-oonly)
+oonly#only_adj)
@@ -219 +219 @@
-of)
+of#of_prep)
@@ -222 +222 @@
-your)
+your#your_pron)
@@ -225 +225 @@
-gode)
+gode#good_adj)
@@ -229 +229 @@
-prosperous)
+prosperous#prosperous_adj)
@@ -233 +233 @@
-and)
+and#and_conj1)
@@ -235 +235 @@
-fortunate))
+fortunate#fortunate_adj))
@@ -237 +237 @@
-late)
+late#late_adj1)
@@ -239 +239 @@
-passage)
+passage#passage_n)
@@ -242 +242 @@
-and)
+and#and_conj1)
@@ -245 +245 @@
-arrivall)))
+arrivall#arrival_n)))
@@ -248 +248 @@
-at)
+at#at_prep)
@@ -251 +251 @@
-your)
+your#your_pron)
@@ -253 +253 @@
-towne)
+towne#town_n)
@@ -256 +256 @@
-of)
+of#of_prep)
@@ -261 +261 @@
-Calays)))))))
+Calays#NA)))))))
@@ -266 +266 @@
-but)
+but#but_prep)
@@ -268 +268 @@
-also)
+also#also_adv)
@@ -271 +271 @@
-of)
+of#of_prep)
@@ -274 +274 @@
-the)
+the#the_adj)
@@ -277 +277 @@
-most)
+most#most_adj)
@@ -279 +279 @@
-honorable))
+honorable#honourable_adj))
@@ -281 +281 @@
-personnal)
+personnal#personal_adj)
@@ -283 +283 @@
-late)
+late#late_adj1)
@@ -285 +285 @@
-meting)
+meting#meeting_n)
@@ -288 +288 @@
-of)
+of#of_prep)
@@ -292 +292 @@
-your)
+your#your_pron)
@@ -294 +294 @@
-Grace))
+Grace#grace_n))
@@ -297 +297 @@
-and)
+and#and_conj1)
@@ -300 +300 @@
-the)
+the#the_adj)
@@ -302 +302 @@
-Frenshe)
+Frenshe#french_adj)
@@ -304 +304 @@
-King)))))
+King#king_n)))))
@@ -309 +309 @@
-with)
+with#with_prep)
@@ -312 +312 @@
-the)
+the#the_adj)
@@ -316 +316 @@
-goodliest))
+goodliest#goodly_adj))
@@ -319 +319 @@
-and)
+and#and_conj1)
@@ -322 +322 @@
-moost)
+moost#most_adj)
@@ -324 +324 @@
-commendable))))
+commendable#commendable_adj))))
@@ -326 +326 @@
-ordre)
+ordre#order_n)
@@ -330 +330 @@
-devised)
+devised#devise_v)
@@ -332 +332 @@
-and)
+and#and_conj1)
@@ -334 +334 @@
-observed))
+observed#observe_v))
@@ -337 +337 @@
-therin))
+therin#therein_adv))
@@ -340 +340 @@
-at)
+at#at_prep)
@@ -343 +343 @@
-that)
+that#that_pron)
@@ -345 +345 @@
-tyme))))
+tyme#time_n))))
@@ -351 +351 @@
-the)
+the#the_adj)
@@ -353 +353 @@
-like)
+like#like_adj)
@@ -356 +356 @@
-whereof)))
+whereof#whereof_adv)))
@@ -364 +364 @@
-heretofore))
+heretofore#heretofore_adv))
@@ -366 +366 @@
-hath)
+hath#have_v)
@@ -368 +368 @@
-nat)
+nat#nat_adv)
@@ -370 +370 @@
-been)
+been#be_v)
@@ -372 +372 @@
-seen)))))))))
+seen#see_v)))))))))
@@ -379 +379 @@
-For)
+For#for_prep)
@@ -382 +382 @@
-the)
+the#the_adj)
@@ -384 +384 @@
-which)))
+which#which_adj)))
@@ -389 +389 @@
-and)
+and#and_conj1)
@@ -392 +392 @@
-for)
+for#for_prep)
@@ -395 +395 @@
-that)
+that#that_conj)
@@ -399 +399 @@
-every)
+every#every_adj)
@@ -401 +401 @@
-thing)
+thing#thing_n1)
@@ -404 +404 @@
-hitherto))
+hitherto#hitherto_adv))
@@ -407 +407 @@
-in)
+in#in_prep)
@@ -410 +410 @@
-this)
+this#this_pron)
@@ -412 +412 @@
-your)
+your#your_pron)
@@ -414 +414 @@
-royall)
+royall#royal_adj)
@@ -416 +416 @@
-Journey))))
+Journey#journey_n))))
@@ -419 +419 @@
-so)
+so#so_adv)
@@ -421 +421 @@
-fortunately)
+fortunately#fortunately_adv)
@@ -424 +424 @@
-and)
+and#and_conj1)
@@ -427 +427 @@
-prosperously))))
+prosperously#prosperously_adv))))
@@ -429 +429 @@
-succedeth)
+succedeth#succeed_v)
@@ -433 +433 @@
-oon)
+oon#one_adj)
@@ -435 +435 @@
-thing))
+thing#thing_n1))
@@ -438 +438 @@
-after)
+after#after_adv)
@@ -441 +441 @@
-another)))))))))
+another#another_pron)))))))))
@@ -452 +452 @@
-we))
+we#we_pron))
@@ -455 +455 @@
-and)
+and#and_conj1)
@@ -458 +458 @@
-all)
+all#all_adj)
@@ -460 +460 @@
-other)
+other#other_adj)
@@ -462 +462 @@
-your)
+your#your_pron)
@@ -464 +464 @@
-subgietts))))
+subgietts#subject_n))))
@@ -466 +466 @@
-have)
+have#have_v)
@@ -469 +469 @@
-greate)
+greate#great_adj)
@@ -471 +471 @@
-cause)
+cause#cause_n)
@@ -474 +474 @@
-to)
+to#to_prep)
@@ -476 +476 @@
-geve)
+geve#give_v)
@@ -479 +479 @@
-laudes)
+laudes#laud_n1)
@@ -481 +481 @@
-and)
+and#and_conj1)
@@ -483 +483 @@
-praise))
+praise#praise_n))
@@ -486 +486 @@
-to)
+to#to_prep)
@@ -489 +489 @@
-almighty)
+almighty#almighty_adj)
@@ -491 +491 @@
-God)))))))
+God#NA)))))))
@@ -499 +499 @@
-and)
+and#and_conj1)
@@ -504 +504 @@
-so))
+so#so_adv))
@@ -507 +507 @@
-full)
+full#full_adj)
@@ -509 +509 @@
-humbly))
+humbly#humbly_adv))
@@ -511 +511 @@
-do)
+do#do_v)
@@ -517 +517 @@
-firmely))
+firmely#firmly_adv))
@@ -519 +519 @@
-trusting)
+trusting#trust_v)
@@ -526 +526 @@
-all)
+all#all_adj)
@@ -528 +528 @@
-your)
+your#your_pron)
@@ -530 +530 @@
-causes)
+causes#cause_n)
@@ -533 +533 @@
-and)
+and#and_conj1)
@@ -536 +536 @@
-matiers))))
+matiers#matter_n1))))
@@ -538 +538 @@
-shall)
+shall#shall_v)
@@ -540 +540 @@
-take)
+take#take_v)
@@ -543 +543 @@
-such)
+such#such_adj)
@@ -546 +546 @@
-effectuell)
+effectuell#effectual_adj)
@@ -548 +548 @@
-and)
+and#and_conj1)
@@ -550 +550 @@
-fortunate))
+fortunate#fortunate_adj))
@@ -552 +552 @@
-conclusion)
+conclusion#conclusion_n)
@@ -555 +555 @@
-as)
+as#as_adv)
@@ -565 +565 @@
-shal@)
+shal@#shall_v)
@@ -567 +567 @@
-@be)
+@be#be_v)
@@ -571 +571 @@
-to)
+to#to_prep)
@@ -574 +574 @@
-the)
+the#the_adj)
@@ -577 +577 @@
-perpetuall)
+perpetuall#perpetual_adj)
@@ -579 +579 @@
-and)
+and#and_conj1)
@@ -581 +581 @@
-immortall))
+immortall#immortal_adj))
@@ -583 +583 @@
-memorie)
+memorie#memory_n)
@@ -586 +586 @@
-of)
+of#of_prep)
@@ -589 +589 @@
-your)
+your#your_pron)
@@ -591 +591 @@
-said)
+said#said_adj)
@@ -593 +593 @@
-Grace)))))
+Grace#grace_n)))))
@@ -598 +598 @@
-and)
+and#and_conj1)
@@ -601 +601 @@
-to)
+to#to_prep)
@@ -604 +604 @@
-the)
+the#the_adj)
@@ -606 +606 @@
-universall)
+universall#universal_adj)
@@ -608 +608 @@
-weale)
+weale#weal_n1)
@@ -614 +614 @@
-rest)))
+rest#rest_n1)))
@@ -619 +619 @@
-and)
+and#and_conj1)
@@ -622 +622 @@
-tranquilitie)))
+tranquilitie#tranquillity_n)))
@@ -625 +625 @@
-of)
+of#of_prep)
@@ -628 +628 @@
-all)
+all#all_adj)
@@ -630 +630 @@
-Christendome))))))))))))))
+Christendome#NA))))))))))))))
@@ -638 +638 @@
-Ascertaignyng)
+Ascertaignyng#ascertain_v)
@@ -641 +641 @@
-your)
+your#your_pron)
@@ -643 +643 @@
-Highnes))
+Highnes#highness_n))
@@ -650 +650 @@
-we))
+we#we_pron))
@@ -652 +652 @@
-were)
+were#be_v)
@@ -655 +655 @@
-on)
+on#on_prep)
@@ -658 +658 @@
-Saturday)
+Saturday#NA)
@@ -662 +662 @@
-last))
+last#last_adv))
@@ -664 +664 @@
-passed))))
+passed#pass_v))))
@@ -667 +667 @@
-at)
+at#at_prep)
@@ -670 +670 @@
-your)
+your#your_pron)
@@ -672 +672 @@
-manoir)
+manoir#manor_n)
@@ -675 +675 @@
-of)
+of#of_prep)
@@ -678 +678 @@
-Richemounte)))))
+Richemounte#NA)))))
@@ -681 +681 @@
-with)
+with#with_prep)
@@ -684 +684 @@
-your)
+your#your_pron)
@@ -686 +686 @@
-dearest)
+dearest#dear_adj1)
@@ -688 +688 @@
-daughter)
+daughter#daughter_n)
@@ -691 +691 @@
-the)
+the#the_adj)
@@ -693 +693 @@
-Princesse))
+Princesse#princess_n))
@@ -699 +699 @@
-who))
+who#who_pron))
@@ -709 +709 @@
-lauded)
+lauded#laud_v)
@@ -711 +711 @@
-bee)
+bee#be_v)
@@ -714 +714 @@
-Almighty)
+Almighty#almighty_adj)
@@ -716 +716 @@
-God)))
+God#NA)))
@@ -720 +720 @@
-is)
+is#be_v)
@@ -724 +724 @@
-right)
+right#right_adv)
@@ -726 +726 @@
-mery))
+mery#merry_adj))
@@ -729 +729 @@
-and)
+and#and_conj1)
@@ -732 +732 @@
-in)
+in#in_prep)
@@ -735 +735 @@
-prosperous)
+prosperous#prosperous_adj)
@@ -737 +737 @@
-helth)
+helth#health_n)
@@ -740 +740 @@
-and)
+and#and_conj1)
@@ -743 +743 @@
-state)))))))
+state#state_n)))))))
@@ -749 +749 @@
-dailly))
+dailly#daily_adv))
@@ -751 +751 @@
-exercising)
+exercising#exercise_v)
@@ -754 +754 @@
-her)
+her#her_pron1)
@@ -756 +756 @@
-self))
+self#self_pron))
@@ -759 +759 @@
-in)
+in#in_prep)
@@ -762 +762 @@
-vertuous)
+vertuous#virtuous_adj)
@@ -764 +764 @@
-pastymes)
+pastymes#pastime_n)
@@ -767 +767 @@
-and)
+and#and_conj1)
@@ -770 +770 @@
-occupacions)))
+occupacions#occupation_n)))
@@ -776 +776 @@
-whereof))
+whereof#whereof_adv))
@@ -782 +782 @@
-we))
+we#we_pron))
@@ -784 +784 @@
-sawe)
+sawe#see_v)
@@ -787 +787 @@
-sum)
+sum#some_pron)
@@ -789 +789 @@
-experience)
+experience#experience_n)
@@ -794 +794 @@
-afore)
+afore#afore_adv)
@@ -801 +801 @@
-we))
+we#we_pron))
@@ -803 +803 @@
-departed)
+departed#depart_v)
@@ -806 +806 @@
-from)
+from#from_prep)
@@ -809 +809 @@
-her)))))))))))))))))
+her#her_pron1)))))))))))))))))
@@ -819 +819 @@
-Fynally))
+Fynally#finally_adv))
@@ -822 +822 @@
-as)
+as#as_adv)
@@ -825 +825 @@
-for)
+for#for_prep)
@@ -828 +828 @@
-th@)
+th@#the_adj)
@@ -830 +830 @@
-@astate)
+@astate#estate_n)
@@ -833 +833 @@
-of)
+of#of_prep)
@@ -836 +836 @@
-your)
+your#your_pron)
@@ -838 +838 @@
-Reame))))))
+Reame#realm_n))))))
@@ -843 +843 @@
-loved)
+loved#love_v1)
@@ -845 +845 @@
-be)
+be#be_v)
@@ -848 +848 @@
-God)))
+God#NA)))
@@ -853 +853 @@
-it))
+it#it_pron))
@@ -855 +855 @@
-is)
+is#be_v)
@@ -858 +858 @@
-in)
+in#in_prep)
@@ -861 +861 @@
-good)
+good#good_adj)
@@ -863 +863 @@
-peax)
+peax#peace_n)
@@ -869 +869 @@
-rest)))
+rest#rest_n1)))
@@ -874 +874 @@
-and)
+and#and_conj1)
@@ -877 +877 @@
-tranquilitie))))))
+tranquilitie#tranquillity_n))))))
@@ -882 +882 @@
-and)
+and#and_conj1)
@@ -886 +886 @@
-your)
+your#your_pron)
@@ -888 +888 @@
-subgietts))
+subgietts#subject_n))
@@ -890 +890 @@
-conserved)
+conserved#conserve_v)
@@ -893 +893 @@
-in)
+in#in_prep)
@@ -897 +897 @@
-good)
+good#good_adj)
@@ -899 +899 @@
-ordre))
+ordre#order_n))
@@ -902 +902 @@
-and)
+and#and_conj1)
@@ -905 +905 @@
-peasible)
+peasible#peaceable_adj)
@@ -907 +907 @@
-rule)))))))
+rule#rule_n1)))))))
@@ -915 +915 @@
-and)
+and#and_conj1)
@@ -918 +918 @@
-we))
+we#we_pron))
@@ -921 +921 @@
-ensuyng)
+ensuyng#ensue_v)
@@ -924 +924 @@
-your)
+your#your_pron)
@@ -926 +926 @@
-commandement))
+commandement#commandment_n))
@@ -929 +929 @@
-according)
+according#accord_v)
@@ -932 +932 @@
-to)
+to#to_prep)
@@ -935 +935 @@
-our)
+our#our_pron)
@@ -937 +937 @@
-duties)))))
+duties#duty_n)))))
@@ -942 +942 @@
-dailly))
+dailly#daily_adv))
@@ -945 +945 @@
-assemble)
+assemble#assemble_v1)
@@ -947 +947 @@
-and)
+and#and_conj1)
@@ -949 +949 @@
-contynue))
+contynue#continue_v))
@@ -952 +952 @@
-in)
+in#in_prep)
@@ -955 +955 @@
-your)
+your#your_pron)
@@ -957 +957 @@
-Counsaill)))
+Counsaill#council_n)))
@@ -965 +965 @@
-and)
+and#and_conj1)
@@ -969 +969 @@
-have)
+have#have_v)
@@ -971 +971 @@
-had)
+had#have_v)
@@ -974 +974 @@
-afore)
+afore#afore_adv)
@@ -977 +977 @@
-us)))
+us#us_pron)))
@@ -980 +980 @@
-sum)
+sum#some_pron)
@@ -982 +982 @@
-causes)
+causes#cause_n)
@@ -985 +985 @@
-and)
+and#and_conj1)
@@ -988 +988 @@
-matiers)))
+matiers#matter_n1)))
@@ -996 +996 @@
-to)
+to#to_prep)
@@ -998 +998 @@
-bee)
+bee#be_v)
@@ -1001 +1001 @@
-punysshed)
+punysshed#punish_v)
@@ -1003 +1003 @@
-and)
+and#and_conj1)
@@ -1005 +1005 @@
-refourmed))
+refourmed#reform_v1))
@@ -1008 +1008 @@
-after)
+after#after_adv)
@@ -1011 +1011 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1013 +1013 @@
-Lawes)))))
+Lawes#law_n1)))))
@@ -1017 +1017 @@
-as)
+as#as_adv)
@@ -1019 +1019 @@
-well)
+well#well_adv)
@@ -1024 +1024 @@
-from)
+from#from_prep)
@@ -1027 +1027 @@
-farr)
+farr#far_adj)
@@ -1029 +1029 @@
-places)))
+places#place_n.II-VII)))
@@ -1032 +1032 @@
-as)
+as#as_adv)
@@ -1037 +1037 @@
-from)
+from#from_prep)
@@ -1040 +1040 @@
-other)
+other#other_adj)
@@ -1043 +1043 @@
-nigh)
+nigh#nigh_adv)
@@ -1046 +1046 @@
-in)
+in#in_prep)
@@ -1049 +1049 @@
-these)
+these#these_pron)
@@ -1051 +1051 @@
-parties))))))))
+parties#party_n))))))))
@@ -1057 +1057 @@
-to)
+to#to_prep)
@@ -1060 +1060 @@
-th'@)
+th'@#the_adj)
@@ -1062 +1062 @@
-@ordering)
+@ordering#ordering_n)
@@ -1065 +1065 @@
-wherof))))
+wherof#whereof_adv))))
@@ -1071 +1071 @@
-we))
+we#we_pron))
@@ -1073 +1073 @@
-trust)
+trust#trust_v)
@@ -1082 +1082 @@
-we))
+we#we_pron))
@@ -1084 +1084 @@
-do)
+do#do_v)
@@ -1086 +1086 @@
-endevor)
+endevor#endeavour_v)
@@ -1089 +1089 @@
-us))
+us#us_pron))
@@ -1092 +1092 @@
-as)
+as#as_adv)
@@ -1102 +1102 @@
-shal@)
+shal@#shall_v)
@@ -1104 +1104 @@
-@be)
+@be#be_v)
@@ -1107 +1107 @@
-to)
+to#to_prep)
@@ -1110 +1110 @@
-the)
+the#the_adj)
@@ -1112 +1112 @@
-contentacion)
+contentacion#contentation_n)
@@ -1115 +1115 @@
-of)
+of#of_prep)
@@ -1118 +1118 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1120 +1120 @@
-said)
+said#said_adj)
@@ -1122 +1122 @@
-Highnesse)))))))))))))
+Highnesse#highness_n)))))))))))))
@@ -1130 +1130 @@
-At)
+At#at_prep)
@@ -1133 +1133 @@
-your)
+your#your_pron)
@@ -1135 +1135 @@
-Paloys)
+Paloys#palace_n1)
@@ -1138 +1138 @@
-of)
+of#of_prep)
@@ -1141 +1141 @@
-Westm.)))))
+Westm.#NA)))))
@@ -1147 +1147 @@
-the)
+the#the_adj)
@@ -1149 +1149 @@
-xiij=th=.)
+xiij=th=.#NA)
@@ -1151 +1151 @@
-day)
+day#day_n)
@@ -1154 +1154 @@
-of)
+of#of_prep)
@@ -1157 +1157 @@
-June)))
+June#NA)))
@@ -1165 +1165 @@
-Your)
+Your#your_pron)
@@ -1168 +1168 @@
-most)
+most#most_adj)
@@ -1170 +1170 @@
-humble))
+humble#humble_adj))
@@ -1172 +1172 @@
-subjects)
+subjects#subject_n)
@@ -1176 +1176 @@
-T.)
+T.#NA)
@@ -1178 +1178 @@
-Norfolke))
+Norfolke#NA))
@@ -1182 +1182 @@
-Ri.)
+Ri.#NA)
@@ -1184 +1184 @@
-Wynton)))
+Wynton#NA)))
@@ -1190 +1190 @@
-W.)
+W.#NA)
@@ -1192 +1192 @@
-Lincoln)))
+Lincoln#NA)))
@@ -1198 +1198 @@
-J.)
+J.#NA)
@@ -1201 +1201 @@
-Abbot)
+Abbot#abbot_n)
@@ -1204 +1204 @@
-of)
+of#of_prep)
@@ -1207 +1207 @@
-Westm.))))))
+Westm.#NA))))))
@@ -1211 +1211 @@
-John)
+John#NA)
@@ -1213 +1213 @@
-Berners)))
+Berners#NA)))
@@ -1219 +1219 @@
-Jo.)
+Jo.#NA)
@@ -1221 +1221 @@
-Fyneux)))
+Fyneux#NA)))
@@ -1227 +1227 @@
-Thomas)
+Thomas#NA)
@@ -1229 +1229 @@
-Louell)))
+Louell#NA)))
@@ -1235 +1235 @@
-Robt.)
+Robt.#NA)
@@ -1237 +1237 @@
-Brudenell)))
+Brudenell#NA)))
@@ -1243 +1243 @@
-John)
+John#NA)
@@ -1245 +1245 @@
-Cutte)))
+Cutte#NA)))
@@ -1251 +1251 @@
-C.)
+C.#NA)
@@ -1253 +1253 @@
-Wyndham)))
+Wyndham#NA)))
@@ -1259 +1259 @@
-T.)
+T.#NA)
@@ -1261 +1261 @@
-Magnus)))
+Magnus#NA)))
@@ -1267 +1267 @@
-John)
+John#NA)
@@ -1269 +1269 @@
-Fitz)
+Fitz#NA)
@@ -1271 +1271 @@
-James))))
+James#NA))))
