--- jubarring-1630-e2-p2.psd.vert	2025-09-10 13:51:18
+++ jubarring-1630-e2-p2.lem.vert	2025-09-10 13:51:25
@@ -14 +14 @@
-Madam))
+Madam#madam_n))
@@ -17 +17 @@
-The)
+The#the_adj)
@@ -19 +19 @@
-want)
+want#want_n2)
@@ -22 +22 @@
-of)
+of#of_prep)
@@ -25 +25 @@
-your)
+your#your_pron)
@@ -27 +27 @@
-presence)
+presence#presence_n)
@@ -30 +30 @@
-amongst)
+amongst#amongst_prep)
@@ -33 +33 @@
-us))))))
+us#us_pron))))))
@@ -36 +36 @@
-now))
+now#now_adv))
@@ -38 +38 @@
-is)
+is#be_v)
@@ -43 +43 @@
-so)
+so#so_adv)
@@ -45 +45 @@
-much))
+much#much_adj))
@@ -47 +47 @@
-the)
+the#the_adj)
@@ -49 +49 @@
-worss))
+worss#worse_adv))
@@ -51 +51 @@
-to)
+to#to_prep)
@@ -53 +53 @@
-be)
+be#be_v)
@@ -55 +55 @@
-borne)
+borne#bear_v1)
@@ -58 +58 @@
-because)
+because#because_adv)
@@ -65 +65 @@
-we))
+we#we_pron))
@@ -67 +67 @@
-had)
+had#have_v)
@@ -70 +70 @@
-soe)
+soe#so_adv)
@@ -72 +72 @@
-certainly))
+certainly#certainly_adv))
@@ -75 +75 @@
-now))
+now#now_adv))
@@ -77 +77 @@
-propounded)
+propounded#propound_v)
@@ -80 +80 @@
-to)
+to#to_prep)
@@ -83 +83 @@
-our)
+our#our_pron)
@@ -85 +85 @@
-selves)))
+selves#self_pron)))
@@ -88 +88 @@
-the)
+the#the_adj)
@@ -90 +90 @@
-comfort)
+comfort#comfort_n)
@@ -93 +93 @@
-of)
+of#of_prep)
@@ -96 +96 @@
-you))))))))
+you#you_pron))))))))
@@ -104 +104 @@
-but)
+but#but_prep)
@@ -107 +107 @@
-you))
+you#you_pron))
@@ -109 +109 @@
-have)
+have#have_v)
@@ -113 +113 @@
-soe)
+soe#so_adv)
@@ -115 +115 @@
-lawfull))
+lawfull#lawful_adj))
@@ -117 +117 @@
-a)
+a#a_adj2)
@@ -119 +119 @@
-cause)
+cause#cause_n)
@@ -122 +122 @@
-to)
+to#to_prep)
@@ -124 +124 @@
-detaine)
+detaine#detain_v)
@@ -127 +127 @@
-you)))
+you#you_pron)))
@@ -130 +130 @@
-that)
+that#that_conj)
@@ -134 +134 @@
-we))
+we#we_pron))
@@ -136 +136 @@
-ware)
+ware#be_v)
@@ -139 +139 @@
-unreasonable)
+unreasonable#unreasonable_adj)
@@ -142 +142 @@
-not)
+not#not_adv)
@@ -144 +144 @@
-to)
+to#to_prep)
@@ -146 +146 @@
-be)
+be#be_v)
@@ -148 +148 @@
-sattissfied)))
+sattissfied#satisfy_v)))
@@ -154 +154 @@
-especially)
+especially#especially_adv)
@@ -156 +156 @@
-sence)
+sence#since_adv)
@@ -163 +163 @@
-you))
+you#you_pron))
@@ -165 +165 @@
-are)
+are#be_v)
@@ -167 +167 @@
-pleased)
+pleased#please_v)
@@ -170 +170 @@
-to)
+to#to_prep)
@@ -172 +172 @@
-assure)
+assure#assure_v)
@@ -175 +175 @@
-us))
+us#us_pron))
@@ -178 +178 @@
-your)
+your#your_pron)
@@ -180 +180 @@
-company))
+company#company_n))
@@ -183 +183 @@
-constantly))
+constantly#constantly_adv))
@@ -186 +186 @@
-at)
+at#at_prep)
@@ -189 +189 @@
-the)
+the#the_adj)
@@ -191 +191 @@
-springe)
+springe#spring_n1)
@@ -197 +197 @@
-which))
+which#which_adj))
@@ -203 +203 @@
-all_though)
+all_though#although_conj)
@@ -210 +210 @@
-we))
+we#we_pron))
@@ -212 +212 @@
-can@)
+can@#can_v1)
@@ -214 +214 @@
-@not)
+@not#not_adv)
@@ -216 +216 @@
-but)
+but#but_prep)
@@ -218 +218 @@
-think)
+think#think_v2)
@@ -224 +224 @@
-longe))))))
+longe#long_adj1))))))
@@ -229 +229 @@
-yett))
+yett#yet_adv))
@@ -232 +232 @@
-they))
+they#they_pron))
@@ -234 +234 @@
-are)
+are#be_v)
@@ -237 +237 @@
-good)
+good#good_adj)
@@ -239 +239 @@
-hopes)
+hopes#hope_n1)
@@ -242 +242 @@
-to)
+to#to_prep)
@@ -244 +244 @@
-make)
+make#make_v1)
@@ -248 +248 @@
-us))
+us#us_pron))
@@ -250 +250 @@
-pass)
+pass#pass_v)
@@ -252 +252 @@
-over)
+over#over_adv)
@@ -255 +255 @@
-the)
+the#the_adj)
@@ -257 +257 @@
-dull)
+dull#dull_adj)
@@ -259 +259 @@
-winter)
+winter#winter_n1)
@@ -261 +261 @@
-quarter))
+quarter#quarter_n))
@@ -264 +264 @@
-the)
+the#the_adj)
@@ -266 +266 @@
-cheerfuller)))))))))))))))))
+cheerfuller#cheerfully_adv)))))))))))))))))
@@ -274 +274 @@
-And)
+And#and_conj1)
@@ -277 +277 @@
-in)
+in#in_prep)
@@ -280 +280 @@
-the)
+the#the_adj)
@@ -282 +282 @@
-mean)
+mean#mean_adj2)
@@ -284 +284 @@
-time)))
+time#time_n)))
@@ -287 +287 @@
-I))
+I#i_pron))
@@ -289 +289 @@
-shall)
+shall#shall_v)
@@ -291 +291 @@
-not)
+not#not_adv)
@@ -293 +293 @@
-omitt)
+omitt#omit_v)
@@ -296 +296 @@
-any)
+any#any_adj)
@@ -298 +298 @@
-fitt)
+fitt#fit_adj)
@@ -300 +300 @@
-opertunity)
+opertunity#opportunity_n)
@@ -303 +303 @@
-to)
+to#to_prep)
@@ -305 +305 @@
-visett)
+visett#visit_v)
@@ -310 +310 @@
-you))
+you#you_pron))
@@ -316 +316 @@
-which))
+which#which_adj))
@@ -324 +324 @@
-I))
+I#i_pron))
@@ -326 +326 @@
-intend)
+intend#intend_v)
@@ -329 +329 @@
-when)
+when#when_adv)
@@ -336 +336 @@
-I))
+I#i_pron))
@@ -338 +338 @@
-come)
+come#come_v)
@@ -341 +341 @@
-to)
+to#to_prep)
@@ -344 +344 @@
-London)))
+London#NA)))
@@ -347 +347 @@
-if)
+if#if_conj)
@@ -355 +355 @@
-I))
+I#i_pron))
@@ -357 +357 @@
-make)
+make#make_v1)
@@ -360 +360 @@
-any)
+any#any_adj)
@@ -362 +362 @@
-stay))
+stay#stay_n3))
@@ -365 +365 @@
-thear))))
+thear#there_adv))))
@@ -368 +368 @@
-and)
+and#and_conj1)
@@ -371 +371 @@
-that)
+that#that_conj)
@@ -375 +375 @@
-the)
+the#the_adj)
@@ -377 +377 @@
-weather))
+weather#weather_n))
@@ -379 +379 @@
-is)
+is#be_v)
@@ -383 +383 @@
-any)
+any#any_adj)
@@ -385 +385 @@
-thinge))
+thinge#thing_n1))
@@ -387 +387 @@
-seasonable))))))))))))))
+seasonable#seasonable_adj))))))))))))))
@@ -396 +396 @@
-I))
+I#i_pron))
@@ -398 +398 @@
-begin)
+begin#begin_v1)
@@ -401 +401 @@
-to)
+to#to_prep)
@@ -403 +403 @@
-finde)
+finde#find_v)
@@ -406 +406 @@
-an)
+an#a_adj2)
@@ -408 +408 @@
-inclination)
+inclination#inclination_n)
@@ -411 +411 @@
-to)
+to#to_prep)
@@ -414 +414 @@
-the)
+the#the_adj)
@@ -416 +416 @@
-same)
+same#same_adj)
@@ -418 +418 @@
-infermitie)
+infermitie#infirmity_n)
@@ -421 +421 @@
-of)
+of#of_prep)
@@ -424 +424 @@
-my)
+my#my_adj)
@@ -426 +426 @@
-lungs)))
+lungs#lung_n)))
@@ -431 +431 @@
-that)
+that#that_conj)
@@ -435 +435 @@
-I))
+I#i_pron))
@@ -437 +437 @@
-began)
+began#begin_v1)
@@ -440 +440 @@
-withall)
+withall#withal_adv)
@@ -445 +445 @@
-last)
+last#last_adv)
@@ -447 +447 @@
-winter))))
+winter#winter_n1))))
@@ -452 +452 @@
-that)
+that#that_conj)
@@ -457 +457 @@
-will)
+will#will_v1)
@@ -459 +459 @@
-hinder)
+hinder#hinder_v)
@@ -462 +462 @@
-my)
+my#my_adj)
@@ -464 +464 @@
-corage))
+corage#courage_n))
@@ -467 +467 @@
-from)
+from#from_prep)
@@ -470 +470 @@
-being)
+being#be_v)
@@ -473 +473 @@
-to)
+to#too_adv)
@@ -475 +475 @@
-bould)
+bould#bold_adj)
@@ -478 +478 @@
-with)
+with#with_prep)
@@ -481 +481 @@
-all)
+all#all_adj)
@@ -483 +483 @@
-weathers))))))))))))
+weathers#weather_n))))))))))))
@@ -491 +491 @@
-but)
+but#but_prep)
@@ -494 +494 @@
-in)
+in#in_prep)
@@ -497 +497 @@
-truth)))
+truth#truth_n)))
@@ -500 +500 @@
-I))
+I#i_pron))
@@ -502 +502 @@
-longe)
+longe#long_v1)
@@ -505 +505 @@
-to)
+to#to_prep)
@@ -507 +507 @@
-waite)
+waite#wait_v1)
@@ -510 +510 @@
-upon)
+upon#upon_prep)
@@ -513 +513 @@
-you))))
+you#you_pron))))
@@ -523 +523 @@
-In)
+In#in_prep)
@@ -526 +526 @@
-the)
+the#the_adj)
@@ -528 +528 @@
-mean)
+mean#mean_adj2)
@@ -530 +530 @@
-while)))
+while#while_n)))
@@ -533 +533 @@
-and)
+and#and_conj1)
@@ -536 +536 @@
-ever))))
+ever#ever_adv))))
@@ -539 +539 @@
-my)
+my#my_adj)
@@ -541 +541 @@
-best)
+best#best_adj)
@@ -543 +543 @@
-wishes)
+wishes#wish_n1)
@@ -546 +546 @@
-and)
+and#and_conj1)
@@ -549 +549 @@
-service))))
+service#service_n1))))
@@ -551 +551 @@
-attend)
+attend#attend_v)
@@ -554 +554 @@
-you)))
+you#you_pron)))
@@ -560 +560 @@
-and)
+and#and_conj1)
@@ -565 +565 @@
-so))
+so#so_adv))
@@ -567 +567 @@
-rest)
+rest#rest_v2)
@@ -570 +570 @@
-Your)
+Your#your_pron)
@@ -574 +574 @@
-most)
+most#most_adj)
@@ -576 +576 @@
-respectively))
+respectively#respectively_adv))
@@ -578 +578 @@
-loving))
+loving#loving_adj))
@@ -580 +580 @@
-daughter)
+daughter#daughter_n)
@@ -583 +583 @@
-Judith)
+Judith#NA)
@@ -585 +585 @@
-Barrinton))))
+Barrinton#NA))))
@@ -591 +591 @@
-Friday)
+Friday#NA)
@@ -596 +596 @@
-October))
+October#NA))
@@ -607 +607 @@
-To)
+To#to_prep)
@@ -610 +610 @@
-my)
+my#my_adj)
@@ -613 +613 @@
-most)
+most#most_adj)
@@ -615 +615 @@
-honored))
+honored#honoured_adj))
@@ -617 +617 @@
-mother)
+mother#mother_n1)
@@ -620 +620 @@
-the)
+the#the_adj)
@@ -622 +622 @@
-Lady)
+Lady#lady_n)
@@ -625 +625 @@
-Barrington)))))
+Barrington#NA)))))
@@ -628 +628 @@
-at)
+at#at_prep)
@@ -631 +631 @@
-Harrow)
+Harrow#NA)
@@ -633 +633 @@
-Hill)))
+Hill#NA)))
@@ -636 +636 @@
-this)))
+this#this_pron)))
@@ -649 +649 @@
-Sweet)
+Sweet#sweet_adj)
@@ -651 +651 @@
-Mrs)
+Mrs#NA)
@@ -653 +653 @@
-Necton))
+Necton#NA))
@@ -656 +656 @@
-I))
+I#i_pron))
@@ -658 +658 @@
-hope)
+hope#hope_v)
@@ -665 +665 @@
-you))
+you#you_pron))
@@ -667 +667 @@
-have)
+have#have_v)
@@ -669 +669 @@
-receaved)
+receaved#receive_v)
@@ -672 +672 @@
-my)
+my#my_adj)
@@ -674 +674 @@
-former)
+former#former_adj)
@@ -676 +676 @@
-letter)
+letter#letter_n1)
@@ -679 +679 @@
-of)
+of#of_prep)
@@ -682 +682 @@
-my)
+my#my_adj)
@@ -684 +684 @@
-intention)
+intention#intention_n)
@@ -688 +688 @@
-now))
+now#now_adv))
@@ -690 +690 @@
-to)
+to#to_prep)
@@ -692 +692 @@
-be)
+be#be_v)
@@ -695 +695 @@
-at)
+at#at_prep)
@@ -699 +699 @@
-Mr)
+Mr#NA)
@@ -701 +701 @@
-Martin's))))
+Martin's#NA))))
@@ -704 +704 @@
-if)
+if#if_conj)
@@ -711 +711 @@
-all)
+all#all_adj)
@@ -713 +713 @@
-things))
+things#thing_n1))
@@ -715 +715 @@
-may)
+may#may_v1)
@@ -717 +717 @@
-fall)
+fall#fall_v)
@@ -719 +719 @@
-out)
+out#out_adv)
@@ -722 +722 @@
-conveniently))))))))))))
+conveniently#conveniently_adv))))))))))))
@@ -728 +728 @@
-and)
+and#and_conj1)
@@ -731 +731 @@
-I))
+I#i_pron))
@@ -733 +733 @@
-have)
+have#have_v)
@@ -736 +736 @@
-now))
+now#now_adv))
@@ -738 +738 @@
-directed)
+directed#direct_v)
@@ -741 +741 @@
-thease))
+thease#these_pron))
@@ -744 +744 @@
-unto)
+unto#unto_prep)
@@ -747 +747 @@
-you)))
+you#you_pron)))
@@ -750 +750 @@
-to)
+to#to_prep)
@@ -752 +752 @@
-begg)
+begg#beg_v)
@@ -759 +759 @@
-I))
+I#i_pron))
@@ -761 +761 @@
-may)
+may#may_v1)
@@ -764 +764 @@
-now))
+now#now_adv))
@@ -766 +766 @@
-receave)
+receave#receive_v)
@@ -769 +769 @@
-an)
+an#a_adj2)
@@ -771 +771 @@
-answear))
+answear#answer_n))
@@ -776 +776 @@
-because)
+because#because_adv)
@@ -783 +783 @@
-we))
+we#we_pron))
@@ -785 +785 @@
-intend)
+intend#intend_v)
@@ -788 +788 @@
-to)
+to#to_prep)
@@ -790 +790 @@
-be)
+be#be_v)
@@ -793 +793 @@
-in)
+in#in_prep)
@@ -796 +796 @@
-London)))
+London#NA)))
@@ -799 +799 @@
-on)
+on#on_prep)
@@ -802 +802 @@
-Tuseday)
+Tuseday#NA)
@@ -804 +804 @@
-or)
+or#or_conj2)
@@ -806 +806 @@
-Wensday)
+Wensday#NA)
@@ -809 +809 @@
-next)))))))))))
+next#next_adj)))))))))))
@@ -817 +817 @@
-and)
+and#and_conj1)
@@ -821 +821 @@
-I))
+I#i_pron))
@@ -823 +823 @@
-beseech)
+beseech#beseech_v)
@@ -826 +826 @@
-you)))
+you#you_pron)))
@@ -828 +828 @@
-lett)
+lett#let_v1)
@@ -832 +832 @@
-Mr)
+Mr#NA)
@@ -834 +834 @@
-Pahanesin))
+Pahanesin#NA))
@@ -836 +836 @@
-know)
+know#know_v)
@@ -843 +843 @@
-my)
+my#my_adj)
@@ -845 +845 @@
-husband))
+husband#husband_n))
@@ -847 +847 @@
-will)
+will#will_v1)
@@ -849 +849 @@
-bringe)
+bringe#bring_v)
@@ -851 +851 @@
-up)
+up#up_adv1)
@@ -854 +854 @@
-with)
+with#with_prep)
@@ -857 +857 @@
-him)))
+him#him_pron)))
@@ -860 +860 @@
-a)
+a#a_adj2)
@@ -862 +862 @@
-fine)
+fine#fine_adj)
@@ -864 +864 @@
-gelding)
+gelding#gelding_n1)
@@ -869 +869 @@
-for)
+for#for_prep)
@@ -872 +872 @@
-his)
+his#his_adj)
@@ -874 +874 @@
-frend)))
+frend#friend_n)))
@@ -879 +879 @@
-such)
+such#such_adj)
@@ -882 +882 @@
-as)
+as#as_adv)
@@ -893,3 +893,3 @@
-a)
-(N
-frend))
+a#a_adj2)
+(N
+frend#friend_n))
@@ -897 +897 @@
-may)
+may#may_v1)
@@ -899 +899 @@
-put)
+put#put_v)
@@ -902 +902 @@
-into)
+into#into_prep)
@@ -905 +905 @@
-another's)
+another's#another_pron)
@@ -907 +907 @@
-hands))))))))))
+hands#hand_n))))))))))
@@ -915 +915 @@
-and)
+and#and_conj1)
@@ -918 +918 @@
-withall))
+withall#withal_adv))
@@ -920 +920 @@
-lett)
+lett#let_v1)
@@ -924 +924 @@
-him))
+him#him_pron))
@@ -926 +926 @@
-know)
+know#know_v)
@@ -936 +936 @@
-as)
+as#as_adv)
@@ -939 +939 @@
-the)
+the#the_adj)
@@ -941 +941 @@
-best)
+best#best_adj)
@@ -943 +943 @@
-entertainment)
+entertainment#entertainment_n)
@@ -954 +954 @@
-I))
+I#i_pron))
@@ -956 +956 @@
-can)
+can#can_v1)
@@ -958 +958 @@
-give)
+give#give_v)
@@ -961 +961 @@
-him))))))
+him#him_pron))))))
@@ -966 +966 @@
-his)
+his#his_adj)
@@ -968 +968 @@
-mistres))
+mistres#mistress_n))
@@ -970 +970 @@
-is)
+is#be_v)
@@ -973 +973 @@
-well)))
+well#well_adj)))
@@ -976 +976 @@
-and)
+and#and_conj1)
@@ -981 +981 @@
-joynes)
+joynes#join_v1)
@@ -984 +984 @@
-with)
+with#with_prep)
@@ -987 +987 @@
-me)))
+me#me_pron1)))
@@ -990 +990 @@
-in)
+in#in_prep)
@@ -993 +993 @@
-all)
+all#all_adj)
@@ -995 +995 @@
-harty)
+harty#hearty_adj)
@@ -997 +997 @@
-good)
+good#good_adj)
@@ -999 +999 @@
-wishes)))
+wishes#wish_n1)))
@@ -1002 +1002 @@
-to)
+to#to_prep)
@@ -1005 +1005 @@
-him))))))))
+him#him_pron))))))))
@@ -1013 +1013 @@
-And)
+And#and_conj1)
@@ -1016 +1016 @@
-so))
+so#so_adv))
@@ -1019 +1019 @@
-beging)
+beging#beg_v)
@@ -1022 +1022 @@
-pardon))
+pardon#pardon_n1))
@@ -1025 +1025 @@
-for)
+for#for_prep)
@@ -1028 +1028 @@
-thease)
+thease#these_pron)
@@ -1030 +1030 @@
-abrupt)
+abrupt#abrupt_adj)
@@ -1032 +1032 @@
-lines)
+lines#line_n2)
@@ -1038 +1038 @@
-which))
+which#which_adj))
@@ -1044 +1044 @@
-I))
+I#i_pron))
@@ -1046 +1046 @@
-am)
+am#be_v)
@@ -1048 +1048 @@
-constrained)
+constrained#constrain_v)
@@ -1053 +1053 @@
-to)
+to#to_prep)
@@ -1055 +1055 @@
-hasten)))))))
+hasten#hasten_v)))))))
@@ -1060 +1060 @@
-with)
+with#with_prep)
@@ -1063 +1063 @@
-my)
+my#my_adj)
@@ -1065 +1065 @@
-affectionate)
+affectionate#affectionate_adj)
@@ -1067 +1067 @@
-love)
+love#love_n1)
@@ -1070 +1070 @@
-to)
+to#to_prep)
@@ -1076 +1076 @@
-you))
+you#you_pron))
@@ -1079 +1079 @@
-and)
+and#and_conj1)
@@ -1082 +1082 @@
-your)
+your#your_pron)
@@ -1084 +1084 @@
-husband)))
+husband#husband_n)))
@@ -1087 +1087 @@
-and)
+and#and_conj1)
@@ -1090,3 +1090,3 @@
-my)
-(ADJ
-pretty)
+my#my_adj)
+(ADJ
+pretty#pretty_adj)
@@ -1094 +1094 @@
-godaughter)))))))
+godaughter#god-daughter_n)))))))
@@ -1097 +1097 @@
-I))
+I#i_pron))
@@ -1099 +1099 @@
-rest)
+rest#rest_v2)
@@ -1102 +1102 @@
-Yours))
+Yours#yours_pron))
@@ -1106 +1106 @@
-faithfully))
+faithfully#faithfully_adv))
@@ -1108 +1108 @@
-to)
+to#to_prep)
@@ -1110 +1110 @@
-serve)
+serve#serve_v1)
@@ -1113 +1113 @@
-you)))
+you#you_pron)))
@@ -1116 +1116 @@
-Judith)
+Judith#NA)
@@ -1118 +1118 @@
-Barrington)))
+Barrington#NA)))
@@ -1124 +1124 @@
-Hatfield)
+Hatfield#NA)
@@ -1132 +1132 @@
-28)
+28#NA)
@@ -1134 +1134 @@
-January)
+January#NA)
@@ -1138 +1138 @@
-1629)
+1629#NA)
@@ -1149 +1149 @@
-To)
+To#to_prep)
@@ -1152 +1152 @@
-my)
+my#my_adj)
@@ -1155 +1155 @@
-much)
+much#much_adj)
@@ -1157 +1157 @@
-esteemed))
+esteemed#esteemed_adj))
@@ -1159 +1159 @@
-frend)
+frend#friend_n)
@@ -1162 +1162 @@
-Mrs)
+Mrs#NA)
@@ -1164 +1164 @@
-Necton))
+Necton#NA))
@@ -1167 +1167 @@
-in)
+in#in_prep)
@@ -1170 +1170 @@
-Aldersgate)
+Aldersgate#NA)
@@ -1172 +1172 @@
-Street)))
+Street#NA)))
@@ -1175 +1175 @@
-neer)
+neer#near_adj)
@@ -1178 +1178 @@
-the)
+the#the_adj)
@@ -1180 +1180 @@
-Halfe)
+Halfe#NA)
@@ -1182 +1182 @@
-Moone)
+Moone#NA)
@@ -1184 +1184 @@
-tavern)))))
+tavern#NA)))))
@@ -1197 +1197 @@
-Madam))
+Madam#madam_n))
@@ -1200 +1200 @@
-I))
+I#i_pron))
@@ -1202 +1202 @@
-have)
+have#have_v)
@@ -1205 +1205 @@
-noe)
+noe#no_adj)
@@ -1207 +1207 @@
-hart)
+hart#heart_n)
@@ -1210 +1210 @@
-to)
+to#to_prep)
@@ -1212 +1212 @@
-lett)
+lett#let_v1)
@@ -1216 +1216 @@
-a)
+a#a_adj2)
@@ -1218 +1218 @@
-frend)
+frend#friend_n)
@@ -1221 +1221 @@
-soe)
+soe#so_adv)
@@ -1223 +1223 @@
-neer)
+neer#near_adj)
@@ -1226 +1226 @@
-us))))
+us#us_pron))))
@@ -1228 +1228 @@
-pass)
+pass#pass_v)
@@ -1231 +1231 @@
-to)
+to#to_prep)
@@ -1234 +1234 @@
-you)))
+you#you_pron)))
@@ -1237 +1237 @@
-without)
+without#without_adv)
@@ -1241 +1241 @@
-some)
+some#some_pron)
@@ -1243 +1243 @@
-few))
+few#few_adj))
@@ -1245 +1245 @@
-lines)
+lines#line_n2)
@@ -1248 +1248 @@
-of)
+of#of_prep)
@@ -1251 +1251 @@
-my)
+my#my_adj)
@@ -1253 +1253 @@
-respects)
+respects#respect_n)
@@ -1256 +1256 @@
-to)
+to#to_prep)
@@ -1259 +1259 @@
-you))))))))))
+you#you_pron))))))))))
@@ -1270 +1270 @@
-Whear))
+Whear#where_adv))
@@ -1278 +1278 @@
-I))
+I#i_pron))
@@ -1280 +1280 @@
-owe)
+owe#owe_v)
@@ -1284 +1284 @@
-soe)
+soe#so_adv)
@@ -1286 +1286 @@
-much))))))
+much#much_adj))))))
@@ -1291 +1291 @@
-this))
+this#this_pron))
@@ -1293 +1293 @@
-is)
+is#be_v)
@@ -1296 +1296 @@
-the)
+the#the_adj)
@@ -1298 +1298 @@
-least)
+least#least_adj)
@@ -1309 +1309 @@
-I))
+I#i_pron))
@@ -1311 +1311 @@
-can)
+can#can_v1)
@@ -1313 +1313 @@
-doe))))
+doe#do_v))))
@@ -1321 +1321 @@
-and)
+and#and_conj1)
@@ -1325 +1325 @@
-your)
+your#your_pron)
@@ -1327 +1327 @@
-lovinge)
+lovinge#loving_adj)
@@ -1329 +1329 @@
-care)
+care#care_n1)
@@ -1332 +1332 @@
-of)
+of#of_prep)
@@ -1335 +1335 @@
-my)
+my#my_adj)
@@ -1337 +1337 @@
-safety)
+safety#safety_n)
@@ -1340 +1340 @@
-in)
+in#in_prep)
@@ -1343 +1343 @@
-our)
+our#our_pron)
@@ -1345 +1345 @@
-returne)
+returne#return_n)
@@ -1348 +1348 @@
-from)
+from#from_prep)
@@ -1351 +1351 @@
-London)))))))
+London#NA)))))))
@@ -1356 +1356 @@
-with)
+with#with_prep)
@@ -1359 +1359 @@
-your)
+your#your_pron)
@@ -1361 +1361 @@
-kinde)
+kinde#kind_adj)
@@ -1363 +1363 @@
-letter))))
+letter#letter_n1))))
@@ -1369 +1369 @@
-challenged)
+challenged#challenge_v)
@@ -1372 +1372 @@
-my)
+my#my_adj)
@@ -1374 +1374 @@
-thankfulnes))
+thankfulnes#thankfulness_n))
@@ -1377 +1377 @@
-before)
+before#before_adv)
@@ -1380 +1380 @@
-now)))
+now#now_adv)))
@@ -1385 +1385 @@
-as)
+as#as_adv)
@@ -1387 +1387 @@
-well)
+well#well_adv)
@@ -1390 +1390 @@
-as)
+as#as_adv)
@@ -1393 +1393 @@
-the)
+the#the_adj)
@@ -1395 +1395 @@
-account)
+account#account_n)
@@ -1398 +1398 @@
-to)
+to#to_prep)
@@ -1400 +1400 @@
-lett)
+lett#let_v1)
@@ -1404 +1404 @@
-you))
+you#you_pron))
@@ -1406 +1406 @@
-kno)
+kno#know_v)
@@ -1413 +1413 @@
-I))
+I#i_pron))
@@ -1415 +1415 @@
-think)
+think#think_v2)
@@ -1422 +1422 @@
-we))
+we#we_pron))
@@ -1424 +1424 @@
-fared)
+fared#fare_v1)
@@ -1427 +1427 @@
-the)
+the#the_adj)
@@ -1429 +1429 @@
-better))
+better#better_adj))
@@ -1432 +1432 @@
-for)
+for#for_prep)
@@ -1435 +1435 @@
-your)
+your#your_pron)
@@ -1437 +1437 @@
-good)
+good#good_adj)
@@ -1439 +1439 @@
-wishes))))))))))))
+wishes#wish_n1))))))))))))
@@ -1447 +1447 @@
-for)
+for#for_prep)
@@ -1450 +1450 @@
-I))
+I#i_pron))
@@ -1452 +1452 @@
-see)
+see#see_v)
@@ -1459 +1459 @@
-you))
+you#you_pron))
@@ -1461 +1461 @@
-wear)
+wear#be_v)
@@ -1464 +1464 @@
-mindfull)
+mindfull#mindful_adj)
@@ -1467 +1467 @@
-of)
+of#of_prep)
@@ -1470 +1470 @@
-us))))))
+us#us_pron))))))
@@ -1478 +1478 @@
-And)
+And#and_conj1)
@@ -1481 +1481 @@
-soe))
+soe#so_adv))
@@ -1483 +1483 @@
-shall)
+shall#shall_v)
@@ -1486 +1486 @@
-I))
+I#i_pron))
@@ -1488 +1488 @@
-be)
+be#be_v)
@@ -1491 +1491 @@
-now))
+now#now_adv))
@@ -1494 +1494 @@
-to)
+to#to_prep)
@@ -1496 +1496 @@
-gett)
+gett#get_v)
@@ -1500 +1500 @@
-your)
+your#your_pron)
@@ -1502 +1502 @@
-gardin))
+gardin#garden_n))
@@ -1505 +1505 @@
-and)
+and#and_conj1)
@@ -1508 +1508 @@
-all)
+all#all_adj)
@@ -1510 +1510 @@
-things))))
+things#thing_n1))))
@@ -1513 +1513 @@
-to)
+to#to_prep)
@@ -1516 +1516 @@
-your)
+your#your_pron)
@@ -1518 +1518 @@
-contentment)))
+contentment#contentment_n)))
@@ -1521 +1521 @@
-against)
+against#against_prep)
@@ -1528 +1528 @@
-you))
+you#you_pron))
@@ -1530 +1530 @@
-please)
+please#please_v)
@@ -1533 +1533 @@
-to)
+to#to_prep)
@@ -1535 +1535 @@
-do)
+do#do_v)
@@ -1538 +1538 @@
-us))
+us#us_pron))
@@ -1541 +1541 @@
-the)
+the#the_adj)
@@ -1543 +1543 @@
-favour)
+favour#favour_n)
@@ -1554 +1554 @@
-you))
+you#you_pron))
@@ -1556 +1556 @@
-have)
+have#have_v)
@@ -1558 +1558 @@
-promised)))
+promised#promise_v)))
@@ -1564 +1564 @@
-which))
+which#which_adj))
@@ -1572 +1572 @@
-now))
+now#now_adv))
@@ -1574 +1574 @@
-draws)
+draws#draw_v)
@@ -1576 +1576 @@
-on)
+on#on_adv)
@@ -1579 +1579 @@
-soe)
+soe#so_adv)
@@ -1581 +1581 @@
-fast)
+fast#fast_adv)
@@ -1584 +1584 @@
-that)
+that#that_conj)
@@ -1588 +1588 @@
-our)
+our#our_pron)
@@ -1590 +1590 @@
-hopes))
+hopes#hope_n1))
@@ -1592 +1592 @@
-grow)
+grow#grow_v)
@@ -1595 +1595 @@
-stronge)))))))))))))
+stronge#strong_adj)))))))))))))
@@ -1604 +1604 @@
-My)
+My#my_adj)
@@ -1606 +1606 @@
-daily)
+daily#daily_adj)
@@ -1608 +1608 @@
-prayers))
+prayers#prayer_n1))
@@ -1610 +1610 @@
-attend)
+attend#attend_v)
@@ -1613 +1613 @@
-you))
+you#you_pron))
@@ -1616 +1616 @@
-that)
+that#that_conj)
@@ -1620 +1620 @@
-we))
+we#we_pron))
@@ -1622 +1622 @@
-may)
+may#may_v1)
@@ -1624 +1624 @@
-enjoye)
+enjoye#enjoy_v)
@@ -1627 +1627 @@
-with)
+with#with_prep)
@@ -1630 +1630 @@
-health)
+health#health_n)
@@ -1636 +1636 @@
-which))
+which#which_adj))
@@ -1642 +1642 @@
-I))
+I#i_pron))
@@ -1644 +1644 @@
-am)
+am#be_v)
@@ -1647 +1647 @@
-confident)
+confident#confident_adj)
@@ -1654 +1654 @@
-this)
+this#this_pron)
@@ -1656 +1656 @@
-ayre))
+ayre#air_n))
@@ -1658 +1658 @@
-will)
+will#will_v1)
@@ -1660 +1660 @@
-be)
+be#be_v)
@@ -1663 +1663 @@
-no)
+no#no_adj)
@@ -1665 +1665 @@
-enemye)
+enemye#enemy_n)
@@ -1668 +1668 @@
-to)
+to#to_prep)
@@ -1678 +1678 @@
-and)
+and#and_conj1)
@@ -1681 +1681 @@
-I))
+I#i_pron))
@@ -1683 +1683 @@
-am)
+am#be_v)
@@ -1686 +1686 @@
-sure)
+sure#sure_adj)
@@ -1693 +1693 @@
-you))
+you#you_pron))
@@ -1695 +1695 @@
-have)
+have#have_v)
@@ -1698 +1698 @@
-many)
+many#many_adj)
@@ -1700 +1700 @@
-harts)
+harts#heart_n)
@@ -1705 +1705 @@
-that)
+that#that_conj)
@@ -1710 +1710 @@
-doe)
+doe#do_v)
@@ -1712 +1712 @@
-attend)
+attend#attend_v)
@@ -1715 +1715 @@
-for)
+for#for_prep)
@@ -1718 +1718 @@
-that)
+that#that_pron)
@@ -1720 +1720 @@
-good)
+good#good_adj)
@@ -1722 +1722 @@
-daye)))))))))
+daye#day_n)))))))))
@@ -1731 +1731 @@
-Mine))
+Mine#mine_adj))
@@ -1733 +1733 @@
-must)
+must#must_v1)
@@ -1735 +1735 @@
-be)
+be#be_v)
@@ -1738 +1738 @@
-in)
+in#in_prep)
@@ -1741 +1741 @@
-the)
+the#the_adj)
@@ -1743 +1743 @@
-first)
+first#first_adj)
@@ -1745 +1745 @@
-ranke))))
+ranke#rank_n1))))
@@ -1751 +1751 @@
-and)
+and#and_conj1)
@@ -1755 +1755 @@
-shall)
+shall#shall_v)
@@ -1758 +1758 @@
-ever))
+ever#ever_adv))
@@ -1760 +1760 @@
-dispose)
+dispose#dispose_v)
@@ -1763 +1763 @@
-of)
+of#of_prep)
@@ -1767 +1767 @@
-me))
+me#me_pron1))
@@ -1770 +1770 @@
-and)
+and#and_conj1)
@@ -1773 +1773 @@
-my)
+my#my_adj)
@@ -1775 +1775 @@
-actions)))))
+actions#action_n)))))
@@ -1778 +1778 @@
-to)
+to#to_prep)
@@ -1780 +1780 @@
-render)
+render#render_v)
@@ -1784 +1784 @@
-me))
+me#me_pron1))
@@ -1787 +1787 @@
-Your)
+Your#your_pron)
@@ -1791 +1791 @@
-most)
+most#most_adj)
@@ -1793 +1793 @@
-respectively))
+respectively#respectively_adv))
@@ -1795 +1795 @@
-lovinge))
+lovinge#loving_adj))
@@ -1797 +1797 @@
-daughter)
+daughter#daughter_n)
@@ -1805 +1805 @@
-to)
+to#to_prep)
@@ -1807 +1807 @@
-command)))
+command#command_v)))
@@ -1810 +1810 @@
-Judith)
+Judith#NA)
@@ -1812 +1812 @@
-Barrington))))))
+Barrington#NA))))))
@@ -1818 +1818 @@
-Thursday))
+Thursday#NA))
@@ -1825 +1825 @@
-My)
+My#my_adj)
@@ -1827 +1827 @@
-sisters))
+sisters#sister_n))
@@ -1829 +1829 @@
-remember)
+remember#remember_v)
@@ -1832 +1832 @@
-thear)
+thear#their_adj)
@@ -1834 +1834 @@
-best)
+best#best_adj)
@@ -1836 +1836 @@
-services))
+services#service_n1))
@@ -1839 +1839 @@
-to)
+to#to_prep)
@@ -1842 +1842 @@
-you)))
+you#you_pron)))
@@ -1853 +1853 @@
-To)
+To#to_prep)
@@ -1856 +1856 @@
-my)
+my#my_adj)
@@ -1859 +1859 @@
-most)
+most#most_adj)
@@ -1861 +1861 @@
-honored))
+honored#honoured_adj))
@@ -1863 +1863 @@
-mother)
+mother#mother_n1)
@@ -1866 +1866 @@
-the)
+the#the_adj)
@@ -1868 +1868 @@
-Lady)
+Lady#lady_n)
@@ -1871 +1871 @@
-Barrington)))))
+Barrington#NA)))))
@@ -1874 +1874 @@
-at)
+at#at_prep)
@@ -1877 +1877 @@
-Harrow)))
+Harrow#NA)))
@@ -1880 +1880 @@
-this)))
+this#this_pron)))
