--- koxinden-1636-e2-p2.psd.vert	2025-09-10 13:51:18
+++ koxinden-1636-e2-p2.lem.vert	2025-09-10 13:51:25
@@ -14 +14 @@
-Sonn))
+Sonn#son_n1))
@@ -19 +19 @@
-i))
+i#i_pron))
@@ -21 +21 @@
-resceved)
+resceved#receive_v)
@@ -24 +24 @@
-your)
+your#your_pron)
@@ -26 +26 @@
-letter)))
+letter#letter_n1)))
@@ -32 +32 @@
-and)
+and#and_conj1)
@@ -36 +36 @@
-am)
+am#be_v)
@@ -39 +39 @@
-glad)
+glad#glad_adj)
@@ -42 +42 @@
-to)
+to#to_prep)
@@ -44 +44 @@
-heare)
+heare#hear_v)
@@ -47 +47 @@
-of)
+of#of_prep)
@@ -51 +51 @@
-my)
+my#my_adj)
@@ -53 +53 @@
-dafteres))
+dafteres#daughter_n))
@@ -55 +55 @@
-recoverri))))))
+recoverri#recovery_n))))))
@@ -62 +62 @@
-i))
+i#i_pron))
@@ -64 +64 @@
-sent)
+sent#send_v1)
@@ -67 +67 @@
-you))
+you#you_pron))
@@ -70 +70 @@
-a)
+a#a_adj2)
@@ -72 +72 @@
-letter))
+letter#letter_n1))
@@ -75 +75 @@
-last)
+last#last_adv)
@@ -77 +77 @@
-weeke))
+weeke#week_n))
@@ -81 +81 @@
-which)
+which#which_adj)
@@ -83 +83 @@
-letter))
+letter#letter_n1))
@@ -89 +89 @@
-it))
+it#it_pron))
@@ -91 +91 @@
-may@)
+may@#may_v1)
@@ -93 +93 @@
-@be)
+@be#be_v)
@@ -102 +102 @@
-you))
+you#you_pron))
@@ -104 +104 @@
-have)
+have#have_v)
@@ -106 +106 @@
-had)
+had#have_v)
@@ -109 +109 @@
-sence))))))
+sence#since_adv))))))
@@ -118 +118 @@
-My)
+My#my_adj)
@@ -120 +120 @@
-sonn)
+sonn#son_n1)
@@ -123 +123 @@
-Barrow)))
+Barrow#NA)))
@@ -125 +125 @@
-rememberes)
+rememberes#remember_v)
@@ -128 +128 @@
-his)
+his#his_adj)
@@ -130 +130 @@
-services))
+services#service_n1))
@@ -134 +134 @@
-unto)
+unto#unto_prep)
@@ -137 +137 @@
-you)))
+you#you_pron)))
@@ -140 +140 @@
-an)
+an#and_conj1)
@@ -143 +143 @@
-to)
+to#to_prep)
@@ -146 +146 @@
-my)
+my#my_adj)
@@ -148 +148 @@
-Dafter))))))
+Dafter#daughter_n))))))
@@ -154 +154 @@
-and)
+and#and_conj1)
@@ -158 +158 @@
-desires)
+desires#desire_v)
@@ -161 +161 @@
-to)
+to#to_prep)
@@ -163 +163 @@
-be)
+be#be_v)
@@ -165 +165 @@
-exskcuseesed)
+exskcuseesed#excuse_v)
@@ -170 +170 @@
-for)
+for#for_prep)
@@ -174 +174 @@
-not)
+not#not_adv)
@@ -176 +176 @@
-writing))
+writing#write_v))
@@ -179 +179 @@
-and)
+and#and_conj1)
@@ -182 +182 @@
-not)
+not#not_adv)
@@ -184 +184 @@
-sending)
+sending#send_v1)
@@ -187 +187 @@
-your)
+your#your_pron)
@@ -189 +189 @@
-bookes))))))))
+bookes#book_n))))))))
@@ -196 +196 @@
-his)
+his#his_adj)
@@ -198 +198 @@
-sute)
+sute#suit_n)
@@ -201 +201 @@
-with)
+with#with_prep)
@@ -204 +204 @@
-his)
+his#his_adj)
@@ -206 +206 @@
-mother))))
+mother#mother_n1))))
@@ -208 +208 @@
-is)
+is#be_v)
@@ -210 +210 @@
-not)
+not#not_adv)
@@ -213 +213 @@
-yet))
+yet#yet_adv))
@@ -215 +215 @@
-ended)
+ended#end_v1)
@@ -223 +223 @@
-But)
+But#but_prep)
@@ -226 +226 @@
-nou))
+nou#now_adv))
@@ -228 +228 @@
-is)
+is#be_v)
@@ -231 +231 @@
-the)
+the#the_adj)
@@ -233 +233 @@
-time)
+time#time_n)
@@ -236 +236 @@
-of)
+of#of_prep)
@@ -239 +239 @@
-trial))))
+trial#trial_n1))))
@@ -248 +248 @@
-My)
+My#my_adj)
@@ -250 +250 @@
-dafter)
+dafter#daughter_n)
@@ -253 +253 @@
-Barrow)))
+Barrow#NA)))
@@ -255 +255 @@
-is)
+is#be_v)
@@ -257 +257 @@
-not)
+not#not_adv)
@@ -260 +260 @@
-well)))
+well#well_adj)))
@@ -266 +266 @@
-But)
+But#but_prep)
@@ -269 +269 @@
-Bes)
+Bes#NA)
@@ -271 +271 @@
-and)
+and#and_conj1)
@@ -273 +273 @@
-Addam))
+Addam#NA))
@@ -275 +275 @@
-ar)
+ar#be_v)
@@ -278 +278 @@
-well)))
+well#well_adj)))
@@ -284 +284 @@
-an)
+an#and_conj1)
@@ -288 +288 @@
-remember)
+remember#remember_v)
@@ -291 +291 @@
-there)
+there#their_adj)
@@ -293 +293 @@
-love))
+love#love_n1))
@@ -296 +296 @@
-unto)
+unto#unto_prep)
@@ -300 +300 @@
-you))
+you#you_pron))
@@ -303 +303 @@
-an)
+an#and_conj1)
@@ -306 +306 @@
-there)
+there#their_adj)
@@ -308 +308 @@
-sister))))))
+sister#sister_n))))))
@@ -315 +315 @@
-so))
+so#so_adv))
@@ -317 +317 @@
-doth)
+doth#do_v)
@@ -320 +320 @@
-my)
+my#my_adj)
@@ -322 +322 @@
-dafter)
+dafter#daughter_n)
@@ -325 +325 @@
-Barrow))))
+Barrow#NA))))
@@ -335 +335 @@
-pray))
+pray#pray_v))
@@ -337 +337 @@
-remember)
+remember#remember_v)
@@ -340 +340 @@
-my)
+my#my_adj)
@@ -342 +342 @@
-Love))
+Love#love_n1))
@@ -345 +345 @@
-to)
+to#to_prep)
@@ -348 +348 @@
-my)
+my#my_adj)
@@ -350 +350 @@
-Dafter))))
+Dafter#daughter_n))))
@@ -353 +353 @@
-an)
+an#and_conj1)
@@ -357 +357 @@
-my)
+my#my_adj)
@@ -359 +359 @@
-servis))
+servis#service_n1))
@@ -362 +362 @@
-to)
+to#to_prep)
@@ -365 +365 @@
-Mrs.)
+Mrs.#NA)
@@ -367 +367 @@
-Kent))))))
+Kent#NA))))))
@@ -374 +374 @@
-i))
+i#i_pron))
@@ -376 +376 @@
-did)
+did#do_v)
@@ -378 +378 @@
-heeare)
+heeare#hear_v)
@@ -385 +385 @@
-Ma=s=)
+Ma=s=#NA)
@@ -387 +387 @@
-Kent))
+Kent#NA))
@@ -389 +389 @@
-was)
+was#be_v)
@@ -392 +392 @@
-in)
+in#in_prep)
@@ -395 +395 @@
-London))))))
+London#NA))))))
@@ -401 +401 @@
-but)
+but#but_prep)
@@ -404 +404 @@
-hee))
+hee#he_pron))
@@ -406 +406 @@
-was)
+was#be_v)
@@ -408 +408 @@
-not)
+not#not_adv)
@@ -411 +411 @@
-so)
+so#so_adv)
@@ -413 +413 @@
-kine)
+kine#kind_adj)
@@ -418 +418 @@
-as)
+as#as_adv)
@@ -422 +422 @@
-to)
+to#to_prep)
@@ -424 +424 @@
-com))
+com#come_v))
@@ -430 +430 @@
-see)
+see#see_v)
@@ -433 +433 @@
-mee))))))))
+mee#me_pron1))))))))
@@ -439 +439 @@
-nether)
+nether#neither_adv)
@@ -442 +442 @@
-have)
+have#have_v)
@@ -445 +445 @@
-i))
+i#i_pron))
@@ -447 +447 @@
-seene)
+seene#see_v)
@@ -450 +450 @@
-sur)
+sur#NA)
@@ -452 +452 @@
-James)
+James#NA)
@@ -454 +454 @@
-Oxsinden)))
+Oxsinden#NA)))
@@ -457 +457 @@
-nor)
+nor#nor_conj1)
@@ -460 +460 @@
-heard)
+heard#hear_v)
@@ -464 +464 @@
-so)
+so#so_adv)
@@ -466 +466 @@
-much))
+much#much_adj))
@@ -469 +469 @@
-as)
+as#as_adv)
@@ -472 +472 @@
-commondasiones))))
+commondasiones#commendation_n))))
@@ -475 +475 @@
-from)
+from#from_prep)
@@ -478 +478 @@
-anni)
+anni#any_adj)
@@ -481 +481 @@
-of)
+of#of_prep)
@@ -484 +484 @@
-them)))))))
+them#them_pron)))))))
@@ -493 +493 @@
-Sence)
+Sence#since_adv)
@@ -500 +500 @@
-you))
+you#you_pron))
@@ -502 +502 @@
-went)
+went#go_v)
@@ -505 +505 @@
-from)
+from#from_prep)
@@ -508 +508 @@
-Londan))))))
+Londan#NA))))))
@@ -511 +511 @@
-i))
+i#i_pron))
@@ -513 +513 @@
-was)
+was#be_v)
@@ -516 +516 @@
-a_most))
+a_most#almost_adv))
@@ -519 +519 @@
-in)
+in#in_prep)
@@ -522 +522 @@
-the)
+the#the_adj)
@@ -524 +524 @@
-mind)))
+mind#mind_n1)))
@@ -527 +527 @@
-that)
+that#that_conj)
@@ -531 +531 @@
-there))
+there#there_adv))
@@ -533 +533 @@
-was)
+was#be_v)
@@ -536 +536 @@
-som)
+som#some_pron)
@@ -538 +538 @@
-inpost))
+inpost#impost_n1))
@@ -542 +542 @@
-set)
+set#set_v)
@@ -545 +545 @@
-one)
+one#on_prep)
@@ -548 +548 @@
-inke)
+inke#ink_n1)
@@ -550 +550 @@
-an)
+an#and_conj1)
@@ -552 +552 @@
-paper))))))
+paper#paper_n))))))
@@ -558 +558 @@
-and)
+and#and_conj1)
@@ -561 +561 @@
-wee))
+wee#we_pron))
@@ -563 +563 @@
-had)
+had#have_v)
@@ -565 +565 @@
-not)
+not#not_adv)
@@ -567 +567 @@
-heard)
+heard#hear_v)
@@ -570 +570 @@
-of)
+of#of_prep)
@@ -573 +573 @@
-the)
+the#the_adj)
@@ -575 +575 @@
-nues)))
+nues#news_n)))
@@ -578 +578 @@
-here)))
+here#here_adv)))
@@ -584 +584 @@
-but)
+but#but_prep)
@@ -587 +587 @@
-if)
+if#if_conj)
@@ -594 +594 @@
-it))
+it#it_pron))
@@ -596 +596 @@
-be)
+be#be_v)
@@ -598 +598 @@
-not)
+not#not_adv)
@@ -601 +601 @@
-that)))))
+that#that_pron)))))
@@ -604 +604 @@
-sure))
+sure#sure_adj))
@@ -609 +609 @@
-something)
+something#something_n)
@@ -612 +612 @@
-eles)))
+eles#else_adv)))
@@ -614 +614 @@
-was)
+was#be_v)
@@ -617 +617 @@
-the)
+the#the_adj)
@@ -619 +619 @@
-case))
+case#cause_n))
@@ -628 +628 @@
-eles))
+eles#else_adv))
@@ -631 +631 @@
-i))
+i#i_pron))
@@ -633 +633 @@
-shold)
+shold#shall_v)
@@ -635 +635 @@
-heere)
+heere#hear_v)
@@ -638 +638 @@
-oftetenner))
+oftetenner#often_adv))
@@ -643 +643 @@
-out)
+out#out_adv)
@@ -645 +645 @@
-of)
+of#of_prep)
@@ -648 +648 @@
-Kente)))
+Kente#NA)))
@@ -657 +657 @@
-My)
+My#my_adj)
@@ -659 +659 @@
-sonn)
+sonn#son_n1)
@@ -662 +662 @@
-Barrow)))
+Barrow#NA)))
@@ -664 +664 @@
-desires)
+desires#desire_v)
@@ -667 +667 @@
-you))
+you#you_pron))
@@ -670 +670 @@
-to)
+to#to_prep)
@@ -672 +672 @@
-speake)
+speake#speak_v)
@@ -675 +675 @@
-to)
+to#to_prep)
@@ -678 +678 @@
-Addam)
+Addam#NA)
@@ -680 +680 @@
-Cuill)))
+Cuill#NA)))
@@ -685 +685 @@
-to)
+to#to_prep)
@@ -687 +687 @@
-send)
+send#send_v1)
@@ -690 +690 @@
-him))
+him#him_pron))
@@ -693 +693 @@
-word)
+word#word_n)
@@ -697 +697 @@
-how))
+how#how_adv))
@@ -705 +705 @@
-the)
+the#the_adj)
@@ -707 +707 @@
-bissines)
+bissines#business_n)
@@ -710 +710 @@
-at)
+at#at_prep)
@@ -713 +713 @@
-elam))))
+elam#NA))))
@@ -715 +715 @@
-stanes)))))))
+stanes#stance_v)))))))
@@ -722 +722 @@
-heare))
+heare#here_adv))
@@ -726 +726 @@
-is)
+is#be_v)
@@ -729 +729 @@
-no)
+no#no_adj)
@@ -731 +731 @@
-nues)
+nues#news_n)
@@ -734 +734 @@
-worth)
+worth#worth_adj)
@@ -737 +737 @@
-the)
+the#the_adj)
@@ -741 +741 @@
-relasion))))
+relasion#relation_n))))
@@ -744 +744 @@
-at)
+at#at_prep)
@@ -747 +747 @@
-this)
+this#this_pron)
@@ -749 +749 @@
-time)))
+time#time_n)))
@@ -758 +758 @@
-this))
+this#this_adv))
@@ -763 +763 @@
-praying)
+praying#pray_v)
@@ -766 +766 @@
-to)
+to#to_prep)
@@ -769 +769 @@
-God)))
+God#NA)))
@@ -772 +772 @@
-for)
+for#for_prep)
@@ -776 +776 @@
-your)
+your#your_pron)
@@ -778 +778 @@
-helth))
+helth#health_n))
@@ -781 +781 @@
-an)
+an#and_conj1)
@@ -784 +784 @@
-youres))))))
+youres#yours_pron))))))
@@ -789 +789 @@
-i))
+i#i_pron))
@@ -791 +791 @@
-rest)
+rest#rest_v2)
@@ -794 +794 @@
-your)
+your#your_pron)
@@ -796 +796 @@
-loveing)
+loveing#loving_adj)
@@ -798 +798 @@
-mother)
+mother#mother_n1)
@@ -801 +801 @@
-K.)
+K.#NA)
@@ -803 +803 @@
-O.))))
+O.#NA))))
@@ -809 +809 @@
-Febe)
+Febe#NA)
@@ -811 +811 @@
-the)
+the#the_adj)
@@ -813 +813 @@
-furst)
+furst#first_adj)
@@ -815 +815 @@
-1637))
+1637#NA))
@@ -822 +822 @@
-i))
+i#i_pron))
@@ -824 +824 @@
-will)
+will#will_v1)
@@ -826 +826 @@
-send)
+send#send_v1)
@@ -829 +829 @@
-your)
+your#your_pron)
@@ -831 +831 @@
-selles))
+selles#seal_n2))
@@ -834 +834 @@
-veri)
+veri#very_adj)
@@ -836 +836 @@
-shortely))
+shortely#shortly_adv))
