--- masham-1630-e2-p2.psd.vert	2025-09-10 13:51:18
+++ masham-1630-e2-p2.lem.vert	2025-09-10 13:51:25
@@ -17 +17 @@
-Deare)
+Deare#dear_adj1)
@@ -19 +19 @@
-mother))
+mother#mother_n1))
@@ -22 +22 @@
-I))
+I#i_pron))
@@ -24 +24 @@
-have)
+have#have_v)
@@ -27 +27 @@
-better))
+better#better_adj))
@@ -29 +29 @@
-considered)
+considered#consider_v)
@@ -32 +32 @@
-of)
+of#of_prep)
@@ -36 +36 @@
-my)
+my#my_adj)
@@ -38 +38 @@
-brother)
+brother#brother_n)
@@ -41 +41 @@
-Knightli@))
+Knightli@#NA))
@@ -43 +43 @@
-@'s))
+@'s#NA))
@@ -45 +45 @@
-letter)
+letter#letter_n1)
@@ -51 +51 @@
-which))
+which#which_adj))
@@ -58 +58 @@
-is)
+is#be_v)
@@ -61 +61 @@
-very)
+very#very_adj)
@@ -63 +63 @@
-hard)
+hard#hard_adj)
@@ -71 +71 @@
-to)
+to#to_prep)
@@ -73 +73 @@
-reade))))
+reade#read_v))))
@@ -79 +79 @@
-he))
+he#he_pron))
@@ -81 +81 @@
-writing)
+writing#write_v)
@@ -85 +85 @@
-so)
+so#so_adv)
@@ -87 +87 @@
-bad))
+bad#bad_adj))
@@ -89 +89 @@
-a)
+a#a_adj2)
@@ -91 +91 @@
-hand)))))))
+hand#hand_n)))))))
@@ -99 +99 @@
-but)
+but#but_prep)
@@ -102 +102 @@
-I))
+I#i_pron))
@@ -104 +104 @@
-perseve)
+perseve#perceive_v)
@@ -111 +111 @@
-ther))
+ther#there_adv))
@@ -113 +113 @@
-is)
+is#be_v)
@@ -115 +115 @@
-not)
+not#not_adv)
@@ -119 +119 @@
-soe)
+soe#so_adv)
@@ -121 +121 @@
-great))
+great#great_adj))
@@ -123 +123 @@
-a)
+a#a_adj2)
@@ -125 +125 @@
-differanc)
+differanc#difference_n1)
@@ -128 +128 @@
-between)
+between#between_prep)
@@ -132 +132 @@
-Sir)
+Sir#NA)
@@ -134 +134 @@
-Robert))
+Robert#NA))
@@ -137 +137 @@
-and)
+and#and_conj1)
@@ -140 +140 @@
-us)))))
+us#us_pron)))))
@@ -143 +143 @@
-as)
+as#as_adv)
@@ -152 +152 @@
-I))
+I#i_pron))
@@ -154 +154 @@
-thought)
+thought#think_v2)
@@ -163 +163 @@
-ther))
+ther#there_adv))
@@ -165 +165 @@
-had)
+had#have_v)
@@ -167 +167 @@
-bin)))))))))
+bin#be_v)))))))))
@@ -176 +176 @@
-Jug))
+Jug#NA))
@@ -178 +178 @@
-is)
+is#be_v)
@@ -181 +181 @@
-desirus)
+desirus#desirous_adj)
@@ -184 +184 @@
-to)
+to#to_prep)
@@ -186 +186 @@
-keep)
+keep#keep_v)
@@ -189 +189 @@
-hir)
+hir#her_pron2)
@@ -191 +191 @@
-owne)
+owne#own_adj)
@@ -193 +193 @@
-land))
+land#land_n1))
@@ -198 +198 @@
-tho)
+tho#though_adv)
@@ -205 +205 @@
-she))
+she#she_pron))
@@ -207 +207 @@
-have)
+have#have_v)
@@ -211 +211 @@
-the)
+the#the_adj)
@@ -213 +213 @@
-less)))
+less#less_adj)))
@@ -216 +216 @@
-joynter)))))))
+joynter#jointure_n)))))))
@@ -224 +224 @@
-and)
+and#and_conj1)
@@ -227 +227 @@
-I))
+I#i_pron))
@@ -229 +229 @@
-think)
+think#think_v2)
@@ -236 +236 @@
-hir)
+hir#her_pron2)
@@ -238 +238 @@
-owne)
+owne#own_adj)
@@ -240 +240 @@
-inheritanc))
+inheritanc#inheritance_n))
@@ -242 +242 @@
-will@)
+will@#will_v1)
@@ -244 +244 @@
-@be)
+@be#be_v)
@@ -247 +247 @@
-better)
+better#better_adj)
@@ -250 +250 @@
-for)
+for#for_prep)
@@ -253 +253 @@
-hir)
+hir#her_pron2)
@@ -255 +255 @@
-preferment)))
+preferment#preferment_n)))
@@ -260 +260 @@
-in)
+in#in_prep)
@@ -263 +263 @@
-a)
+a#a_adj2)
@@ -265 +265 @@
-second)
+second#second_adj)
@@ -267 +267 @@
-match)))
+match#match_n1)))
@@ -272 +272 @@
-if)
+if#if_conj)
@@ -279 +279 @@
-God))
+God#NA))
@@ -281 +281 @@
-should)
+should#shall_v)
@@ -283 +283 @@
-take)
+take#take_v)
@@ -286 +286 @@
-away))
+away#away_adv))
@@ -289 +289 @@
-hir)
+hir#her_pron2)
@@ -291 +291 @@
-husband))
+husband#husband_n))
@@ -294 +294 @@
-withowt)
+withowt#without_adv)
@@ -297 +297 @@
-iswe))))))
+iswe#issue_n))))))
@@ -302 +302 @@
-than)
+than#than_conj)
@@ -305 +305 @@
-her)
+her#her_pron2)
@@ -307 +307 @@
-joynter)))))
+joynter#jointure_n)))))
@@ -317 +317 @@
-Sir)
+Sir#NA)
@@ -319 +319 @@
-Robert))
+Robert#NA))
@@ -321 +321 @@
-ofers)
+ofers#offer_v)
@@ -325 +325 @@
-300)
+300#NA)
@@ -327 +327 @@
-maintinans))
+maintinans#maintenance_n))
@@ -330 +330 @@
-and)
+and#and_conj1)
@@ -335 +335 @@
-so)
+so#so_adv)
@@ -337 +337 @@
-much))
+much#much_adj))
@@ -339 +339 @@
-joynter))
+joynter#jointure_n))
@@ -341 +341 @@
-aded)
+aded#add_v)
@@ -344 +344 @@
-to)
+to#to_prep)
@@ -347 +347 @@
-hir)
+hir#her_pron2)
@@ -349 +349 @@
-owne))))))
+owne#own_adj))))))
@@ -352 +352 @@
-if)
+if#if_conj)
@@ -360 +360 @@
-she))
+she#she_pron))
@@ -362 +362 @@
-oute_live)
+oute_live#outlive_v)
@@ -365 +365 @@
-hir)
+hir#her_pron2)
@@ -367 +367 @@
-husband)))
+husband#husband_n)))
@@ -370 +370 @@
-and)
+and#and_conj1)
@@ -375 +375 @@
-have)
+have#have_v)
@@ -378 +378 @@
-isswe)
+isswe#issue_n)
@@ -381 +381 @@
-by)
+by#by_prep)
@@ -384 +384 @@
-him))))))))))
+him#him_pron))))))))))
@@ -391 +391 @@
-other)
+other#other_adj)
@@ -393 +393 @@
-wayes))
+wayes#way_n1))
@@ -395 +395 @@
-not)))
+not#not_adv)))
@@ -401 +401 @@
-which))
+which#which_adj))
@@ -407 +407 @@
-I))
+I#i_pron))
@@ -409 +409 @@
-se)
+se#see_v)
@@ -412 +412 @@
-no)
+no#no_adj)
@@ -414 +414 @@
-reson)
+reson#reason_n1)
@@ -417 +417 @@
-for)
+for#for_prep)
@@ -424 +424 @@
-because)
+because#because_adv)
@@ -431 +431 @@
-he))
+he#he_pron))
@@ -433 +433 @@
-shall)
+shall#shall_v)
@@ -435 +435 @@
-injoye)
+injoye#enjoy_v)
@@ -438 +438 @@
-hir)
+hir#her_pron2)
@@ -440 +440 @@
-land))
+land#land_n1))
@@ -443 +443 @@
-for)
+for#for_prep)
@@ -446 +446 @@
-his)
+his#his_adj)
@@ -448 +448 @@
-life))))))))
+life#life_n))))))))
@@ -457 +457 @@
-I))
+I#i_pron))
@@ -459 +459 @@
-dout)
+dout#doubt_v)
@@ -461 +461 @@
-not)
+not#not_adv)
@@ -464 +464 @@
-but)
+but#but_prep)
@@ -470 +470 @@
-my)
+my#my_adj)
@@ -472 +472 @@
-brother)
+brother#brother_n)
@@ -475 +475 @@
-Knightly)))
+Knightly#NA)))
@@ -477 +477 @@
-will)
+will#will_v1)
@@ -479 +479 @@
-prevaile)
+prevaile#prevail_v)
@@ -482 +482 @@
-with)
+with#with_prep)
@@ -485 +485 @@
-Sir)
+Sir#NA)
@@ -487 +487 @@
-Robert)))
+Robert#NA)))
@@ -490 +490 @@
-to)
+to#to_prep)
@@ -492 +492 @@
-yeald)
+yeald#yield_v)
@@ -495 +495 @@
-to)
+to#to_prep)
@@ -498 +498 @@
-our)
+our#our_pron)
@@ -500 +500 @@
-resonable)
+resonable#reasonable_adj)
@@ -502 +502 @@
-demaunds))))))
+demaunds#demand_n1))))))
@@ -511 +511 @@
-I))
+I#i_pron))
@@ -513 +513 @@
-have)
+have#have_v)
@@ -515 +515 @@
-writ)
+writ#write_v)
@@ -518 +518 @@
-to)
+to#to_prep)
@@ -521 +521 @@
-him))))
+him#him_pron))))
@@ -527 +527 @@
-and)
+and#and_conj1)
@@ -530 +530 @@
-when)
+when#when_adv)
@@ -537 +537 @@
-I))
+I#i_pron))
@@ -539 +539 @@
-here)
+here#hear_v)
@@ -542 +542 @@
-his)
+his#his_adj)
@@ -544 +544 @@
-answer)))))
+answer#answer_n)))))
@@ -547 +547 @@
-you))
+you#you_pron))
@@ -549 +549 @@
-shall)
+shall#shall_v)
@@ -551 +551 @@
-know)
+know#know_v)
@@ -554 +554 @@
-more))
+more#more_adj))
@@ -563 +563 @@
-I))
+I#i_pron))
@@ -565 +565 @@
-hope)
+hope#hope_v)
@@ -572 +572 @@
-God))
+God#NA))
@@ -574 +574 @@
-will)
+will#will_v1)
@@ -576 +576 @@
-dispose)
+dispose#dispose_v)
@@ -579 +579 @@
-all))
+all#all_adj))
@@ -582 +582 @@
-for)
+for#for_prep)
@@ -585 +585 @@
-the)
+the#the_adj)
@@ -587 +587 @@
-best)))))
+best#best_adj)))))
@@ -596 +596 @@
-Thus))
+Thus#thus_adv))
@@ -599 +599 @@
-with)
+with#with_prep)
@@ -602 +602 @@
-my)
+my#my_adj)
@@ -604 +604 @@
-humble)
+humble#humble_adj)
@@ -606 +606 @@
-duty)))
+duty#duty_n)))
@@ -609 +609 @@
-praying)
+praying#pray_v)
@@ -613 +613 @@
-for)
+for#for_prep)
@@ -616 +616 @@
-the)
+the#the_adj)
@@ -618 +618 @@
-continuanc)
+continuanc#continuance_n)
@@ -621 +621 @@
-of)
+of#of_prep)
@@ -624 +624 @@
-your)
+your#your_pron)
@@ -626 +626 @@
-health)))))
+health#health_n)))))
@@ -632 +632 @@
-especialy)
+especialy#especially_adv)
@@ -634 +634 @@
-for)
+for#for_prep)
@@ -637 +637 @@
-the)
+the#the_adj)
@@ -639 +639 @@
-thriving)
+thriving#thriving_n)
@@ -642 +642 @@
-of)
+of#of_prep)
@@ -645 +645 @@
-your)
+your#your_pron)
@@ -647 +647 @@
-sole))))))))
+sole#soul_n))))))))
@@ -652 +652 @@
-hoping)
+hoping#hope_v)
@@ -656 +656 @@
-ere)
+ere#ere_adv)
@@ -659 +659 @@
-long)))
+long#long_adj1)))
@@ -661 +661 @@
-to)
+to#to_prep)
@@ -663 +663 @@
-se)
+se#see_v)
@@ -666 +666 @@
-you))))
+you#you_pron))))
@@ -671 +671 @@
-I))
+I#i_pron))
@@ -673 +673 @@
-remaine)
+remaine#remain_v)
@@ -676 +676 @@
-Your)
+Your#your_pron)
@@ -678 +678 @@
-obedient)
+obedient#obedient_adj)
@@ -680 +680 @@
-daughter)
+daughter#daughter_n)
@@ -683 +683 @@
-Elizabeth)
+Elizabeth#NA)
@@ -685 +685 @@
-Masham))))
+Masham#NA))))
@@ -692 +692 @@
-I))
+I#i_pron))
@@ -694 +694 @@
-desir)
+desir#desire_v)
@@ -697 +697 @@
-to)
+to#to_prep)
@@ -699 +699 @@
-have)
+have#have_v)
@@ -703 +703 @@
-my)
+my#my_adj)
@@ -705 +705 @@
-kind)
+kind#kind_adj)
@@ -707 +707 @@
-love))
+love#love_n1))
@@ -709 +709 @@
-remembred)
+remembred#remember_v)
@@ -712 +712 @@
-to)
+to#to_prep)
@@ -716 +716 @@
-my)
+my#my_adj)
@@ -718 +718 @@
-brother)
+brother#brother_n)
@@ -721 +721 @@
-and)
+and#and_conj1)
@@ -724 +724 @@
-sisters))))
+sisters#sister_n))))
@@ -727 +727 @@
-and)
+and#and_conj1)
@@ -730 +730 @@
-the)
+the#the_adj)
@@ -732 +732 @@
-rest)
+rest#rest_n2)
@@ -735 +735 @@
-of)
+of#of_prep)
@@ -738 +738 @@
-my)
+my#my_adj)
@@ -740 +740 @@
-frinds)))))))))
+frinds#friend_n)))))))))
@@ -755 +755 @@
-To)
+To#to_prep)
@@ -758 +758 @@
-my)
+my#my_adj)
@@ -761 +761 @@
-much)
+much#much_adj)
@@ -763 +763 @@
-honoured))
+honoured#honoured_adj))
@@ -765 +765 @@
-mother)
+mother#mother_n1)
@@ -768 +768 @@
-the)
+the#the_adj)
@@ -770 +770 @@
-Lady)
+Lady#lady_n)
@@ -773 +773 @@
-Barrington)))))
+Barrington#NA)))))
@@ -776 +776 @@
-these))
+these#these_pron))
@@ -778 +778 @@
-be)
+be#be_v)
@@ -780 +780 @@
-given))
+given#give_v))
@@ -796 +796 @@
-Deare)
+Deare#dear_adj1)
@@ -798 +798 @@
-mother))
+mother#mother_n1))
@@ -801 +801 @@
-I))
+I#i_pron))
@@ -803 +803 @@
-rejoyce)
+rejoyce#rejoice_v)
@@ -806 +806 @@
-to)
+to#to_prep)
@@ -808 +808 @@
-think)
+think#think_v2)
@@ -815 +815 @@
-the)
+the#the_adj)
@@ -817 +817 @@
-time)
+time#time_n)
@@ -821 +821 @@
-drawes)
+drawes#draw_v)
@@ -824 +824 @@
-neer))
+neer#near_adj))
@@ -829 +829 @@
-that)
+that#that_conj)
@@ -835 +835 @@
-we))
+we#we_pron))
@@ -837 +837 @@
-shall)
+shall#shall_v)
@@ -839 +839 @@
-injoye)
+injoye#enjoy_v)
@@ -842 +842 @@
-you))
+you#you_pron))
@@ -845 +845 @@
-at)
+at#at_prep)
@@ -848 +848 @@
-Hatfelde)
+Hatfelde#NA)
@@ -854 +854 @@
-where))
+where#where_adv))
@@ -862 +862 @@
-you))
+you#you_pron))
@@ -864 +864 @@
-are)
+are#be_v)
@@ -867 +867 @@
-much))
+much#much_adj))
@@ -869 +869 @@
-desired))))))))))
+desired#desire_v))))))))))
@@ -878 +878 @@
-I))
+I#i_pron))
@@ -880 +880 @@
-shall@)
+shall@#shall_v)
@@ -882 +882 @@
-@be)
+@be#be_v)
@@ -885 +885 @@
-very)
+very#very_adj)
@@ -887 +887 @@
-glad)
+glad#glad_adj)
@@ -890 +890 @@
-to)
+to#to_prep)
@@ -892 +892 @@
-here)
+here#hear_v)
@@ -895 +895 @@
-that)
+that#that_conj)
@@ -899 +899 @@
-my)
+my#my_adj)
@@ -901 +901 @@
-sister))
+sister#sister_n))
@@ -903 +903 @@
-is)
+is#be_v)
@@ -906 +906 @@
-safly))
+safly#safely_adv))
@@ -908 +908 @@
-delivered))))))
+delivered#deliver_v1))))))
@@ -914 +914 @@
-and)
+and#and_conj1)
@@ -917 +917 @@
-then))
+then#then_adv))
@@ -920 +920 @@
-I))
+I#i_pron))
@@ -922 +922 @@
-hope)
+hope#hope_v)
@@ -929 +929 @@
-you))
+you#you_pron))
@@ -933 +933 @@
-will)
+will#will_v1)
@@ -935 +935 @@
-come)
+come#come_v)
@@ -938 +938 @@
-cherfully))))
+cherfully#cheerfully_adv))))
@@ -947 +947 @@
-I))
+I#i_pron))
@@ -949 +949 @@
-thoute)
+thoute#think_v2)
@@ -955 +955 @@
-good))
+good#good_adj))
@@ -958 +958 @@
-to)
+to#to_prep)
@@ -960 +960 @@
-lett)
+lett#let_v1)
@@ -964 +964 @@
-you))
+you#you_pron))
@@ -966 +966 @@
-know)
+know#know_v)
@@ -969 +969 @@
-of)
+of#of_prep)
@@ -972 +972 @@
-a)
+a#a_adj2)
@@ -974 +974 @@
-proposition)
+proposition#proposition_n)
@@ -977 +977 @@
-of)
+of#of_prep)
@@ -980 +980 @@
-a)
+a#a_adj2)
@@ -982 +982 @@
-match)
+match#match_n1)
@@ -985 +985 @@
-for)
+for#for_prep)
@@ -988 +988 @@
-Jug)))))
+Jug#NA)))))
@@ -991 +991 @@
-propownded)
+propownded#propound_v)
@@ -994 +994 @@
-by)
+by#by_prep)
@@ -997 +997 @@
-Sir)
+Sir#NA)
@@ -999 +999 @@
-Nathanell)
+Nathanell#NA)
@@ -1001 +1001 @@
-Rich)))))))))
+Rich#NA)))))))))
@@ -1011 +1011 @@
-The)
+The#the_adj)
@@ -1013 +1013 @@
-gentleman's))
+gentleman's#gentleman_n))
@@ -1015 +1015 @@
-name))
+name#name_n))
@@ -1017 +1017 @@
-is)
+is#be_v)
@@ -1020,3 +1020 @@
-Mr)
-(NPR
-Sant)
+Mr#NA)
@@ -1024 +1022,3 @@
-Johns)
+Sant#NA)
+(NPR
+Johns#NA)
@@ -1029 +1029 @@
-that)
+that#that_conj)
@@ -1034 +1034 @@
-was)
+was#be_v)
@@ -1037 +1037 @@
-lately))
+lately#lately_adv))
@@ -1040 +1040 @@
-in)
+in#in_prep)
@@ -1043 +1043 @@
-prison)))
+prison#prison_n)))
@@ -1046 +1046 @@
-in)
+in#in_prep)
@@ -1049 +1049 @@
-the)
+the#the_adj)
@@ -1051 +1051 @@
-tower))))))
+tower#NA))))))
@@ -1060 +1060 @@
-I))
+I#i_pron))
@@ -1062 +1062 @@
-here)
+here#hear_v)
@@ -1065 +1065 @@
-very)
+very#very_adj)
@@ -1067 +1067 @@
-worthyly))
+worthyly#worthily_adv))
@@ -1070 +1070 @@
-of)
+of#of_prep)
@@ -1073 +1073 @@
-the)
+the#the_adj)
@@ -1075 +1075 @@
-man)))
+man#man_n1)))
@@ -1083 +1083 @@
-but)
+but#but_prep)
@@ -1086 +1086 @@
-it))
+it#it_pron))
@@ -1088 +1088 @@
-semes)
+semes#seem_v2)
@@ -1095 +1095 @@
-his)
+his#his_adj)
@@ -1097 +1097 @@
-estate))
+estate#estate_n))
@@ -1099 +1099 @@
-is)
+is#be_v)
@@ -1103 +1103 @@
-very)
+very#very_adj)
@@ -1105 +1105 @@
-small))
+small#small_adj))
@@ -1111 +1111 @@
-not)
+not#not_adv)
@@ -1113 +1113 @@
-above)
+above#above_adv)
@@ -1116 +1116 @@
-200@)
+200@#NA)
@@ -1118 +1118 @@
-@=li=)
+@=li=#pound_n1)
@@ -1121 +1121 @@
-a)
+a#a_adj2)
@@ -1123 +1123 @@
-yere))))))))
+yere#year_n))))))))
@@ -1131 +1131 @@
-and)
+and#and_conj1)
@@ -1134 +1134 @@
-besids))
+besids#besides_adv))
@@ -1137 +1137 @@
-his)
+his#his_adj)
@@ -1139 +1139 @@
-father))
+father#father_n))
@@ -1141 +1141 @@
-was)
+was#be_v)
@@ -1144 +1144 @@
-a)
+a#a_adj2)
@@ -1146 +1146 @@
-base)
+base#base_adj)
@@ -1148 +1148 @@
-sonne)
+sonne#son_n1)
@@ -1155 +1155 @@
-as)
+as#as_adv)
@@ -1166 +1166 @@
-I))
+I#i_pron))
@@ -1168 +1168 @@
-have)
+have#have_v)
@@ -1170 +1170 @@
-hard))))
+hard#hear_v))))
@@ -1175 +1175 @@
-of)
+of#of_prep)
@@ -1178 +1178 @@
-the)
+the#the_adj)
@@ -1180 +1180 @@
-Lord)
+Lord#lord_n)
@@ -1183 +1183 @@
-Sant)
+Sant#NA)
@@ -1185 +1185 @@
-Johns))))
+Johns#NA))))
@@ -1195 +1195 @@
-Sir)
+Sir#NA)
@@ -1197 +1197 @@
-Nathanell))
+Nathanell#NA))
@@ -1200 +1200 @@
-and)
+and#and_conj1)
@@ -1203 +1203 @@
-he))))
+he#he_pron))))
@@ -1205 +1205 @@
-are)
+are#be_v)
@@ -1207 +1207 @@
-both)
+both#both_pron)
@@ -1210 +1210 @@
-at)
+at#at_prep)
@@ -1213 +1213 @@
-Hatfeld))))
+Hatfeld#NA))))
@@ -1219 +1219 @@
-and)
+and#and_conj1)
@@ -1223 +1223 @@
-meane)
+meane#mean_v1)
@@ -1226 +1226 @@
-to)
+to#to_prep)
@@ -1228 +1228 @@
-take)
+take#take_v)
@@ -1231 +1231 @@
-our)
+our#our_pron)
@@ -1233 +1233 @@
-howse))
+howse#house_n1))
@@ -1236 +1236 @@
-in)
+in#in_prep)
@@ -1239 +1239 @@
-the)
+the#the_adj)
@@ -1241 +1241 @@
-way)
+way#way_n1)
@@ -1244 +1244 @@
-to)
+to#to_prep)
@@ -1247 +1247 @@
-Pergoe))))))
+Pergoe#NA))))))
@@ -1255 +1255 @@
-and)
+and#and_conj1)
@@ -1258 +1258 @@
-when)
+when#when_adv)
@@ -1265 +1265 @@
-he))
+he#he_pron))
@@ -1267 +1267 @@
-informes)
+informes#inform_v)
@@ -1270 +1270 @@
-me))
+me#me_pron1))
@@ -1273 +1273 @@
-more)
+more#more_adj)
@@ -1275 +1275 @@
-fully)))))
+fully#fully_adv)))))
@@ -1278 +1278 @@
-then))
+then#then_adv))
@@ -1281 +1281 @@
-you))
+you#you_pron))
@@ -1283 +1283 @@
-shall)
+shall#shall_v)
@@ -1285 +1285 @@
-here)
+here#hear_v)
@@ -1293 +1293 @@
-for)
+for#for_prep)
@@ -1296 +1296 @@
-I))
+I#i_pron))
@@ -1298 +1298 @@
-will)
+will#will_v1)
@@ -1300 +1300 @@
-doe)
+doe#do_v)
@@ -1303 +1303 @@
-nothing))
+nothing#nothing_pron))
@@ -1306 +1306 @@
-withowt)
+withowt#without_adv)
@@ -1309 +1309 @@
-your)
+your#your_pron)
@@ -1311 +1311 @@
-advise)))
+advise#advice_n)))
@@ -1320 +1320 @@
-He))
+He#he_pron))
@@ -1322 +1322 @@
-is)
+is#be_v)
@@ -1326 +1326 @@
-a)
+a#a_adj2)
@@ -1328 +1328 @@
-lawyer))
+lawyer#lawyer_n))
@@ -1333 +1333 @@
-but)
+but#but_prep)
@@ -1336 +1336 @@
-young))))
+young#young_adj))))
@@ -1344 +1344 @@
-and)
+and#and_conj1)
@@ -1347 +1347 @@
-therfor))
+therfor#therefore_adv))
@@ -1350 +1350 @@
-I))
+I#i_pron))
@@ -1352 +1352 @@
-think)
+think#think_v2)
@@ -1359 +1359 @@
-his)
+his#his_adj)
@@ -1361 +1361 @@
-practis))
+practis#practice_n))
@@ -1363 +1363 @@
-is)
+is#be_v)
@@ -1366 +1366 @@
-little))))
+little#little_adj))))
@@ -1375 +1375 @@
-yet))
+yet#yet_adv))
@@ -1378 +1378 @@
-I))
+I#i_pron))
@@ -1380 +1380 @@
-beseech)
+beseech#beseech_v)
@@ -1383 +1383 @@
-you))
+you#you_pron))
@@ -1387 +1387 @@
-to)
+to#to_prep)
@@ -1389 +1389 @@
-consider)
+consider#consider_v)
@@ -1392 +1392 @@
-well))
+well#well_adv))
@@ -1395 +1395 @@
-of)
+of#of_prep)
@@ -1398 +1398 @@
-it))))
+it#it_pron))))
@@ -1401 +1401 @@
-and)
+and#and_conj1)
@@ -1406 +1406 @@
-keep)
+keep#keep_v)
@@ -1409 +1409 @@
-it))
+it#it_pron))
@@ -1412 +1412 @@
-very)
+very#very_adj)
@@ -1414 +1414 @@
-secrett)))))
+secrett#secret_adj)))))
@@ -1422 +1422 @@
-And)
+And#and_conj1)
@@ -1424 +1424 @@
-let)
+let#let_v1)
@@ -1428 +1428 @@
-me))
+me#me_pron1))
@@ -1430 +1430 @@
-crave)
+crave#crave_v)
@@ -1433 +1433 @@
-your)
+your#your_pron)
@@ -1435 +1435 @@
-prayers)
+prayers#prayer_n1)
@@ -1438 +1438 @@
-to)
+to#to_prep)
@@ -1441 +1441 @@
-God)))
+God#NA)))
@@ -1444 +1444 @@
-that)
+that#that_conj)
@@ -1448 +1448 @@
-he))
+he#he_pron))
@@ -1450 +1450 @@
-will)
+will#will_v1)
@@ -1452 +1452 @@
-be)
+be#be_v)
@@ -1454 +1454 @@
-pleased)
+pleased#please_v)
@@ -1457 +1457 @@
-to)
+to#to_prep)
@@ -1459 +1459 @@
-direct)
+direct#direct_v)
@@ -1462 +1462 @@
-all))
+all#all_adj))
@@ -1465 +1465 @@
-as)
+as#as_adv)
@@ -1475 +1475 @@
-may@)
+may@#may_v1)
@@ -1477 +1477 @@
-@be)
+@be#be_v)
@@ -1480 +1480 @@
-most))
+most#most_adj))
@@ -1483 +1483 @@
-for)
+for#for_prep)
@@ -1487 +1487 @@
-his)
+his#his_adj)
@@ -1489 +1489 @@
-glory))
+glory#glory_n))
@@ -1492 +1492 @@
-and)
+and#and_conj1)
@@ -1495 +1495 @@
-all)
+all#all_adj)
@@ -1497 +1497 @@
-our)
+our#our_pron)
@@ -1499 +1499 @@
-comforts)))))))))))))
+comforts#comfort_n)))))))))))))
@@ -1508 +1508 @@
-Thus))
+Thus#thus_adv))
@@ -1511 +1511 @@
-with)
+with#with_prep)
@@ -1514 +1514 @@
-my)
+my#my_adj)
@@ -1516 +1516 @@
-humble)
+humble#humble_adj)
@@ -1518 +1518 @@
-dutye)))
+dutye#duty_n)))
@@ -1521 +1521 @@
-praying)
+praying#pray_v)
@@ -1524 +1524 @@
-for)
+for#for_prep)
@@ -1527 +1527 @@
-your)
+your#your_pron)
@@ -1529 +1529 @@
-increse)
+increse#increase_n)
@@ -1532 +1532 @@
-of)
+of#of_prep)
@@ -1535 +1535 @@
-comfort)
+comfort#comfort_n)
@@ -1538 +1538 @@
-in)
+in#in_prep)
@@ -1541 +1541 @@
-sole)
+sole#soul_n)
@@ -1543 +1543 @@
-and)
+and#and_conj1)
@@ -1545 +1545 @@
-body))))))))
+body#body_n))))))))
@@ -1550 +1550 @@
-I))
+I#i_pron))
@@ -1552 +1552 @@
-remaine)
+remaine#remain_v)
@@ -1555 +1555 @@
-Your)
+Your#your_pron)
@@ -1557 +1557 @@
-dutyfull)
+dutyfull#dutiful_adj)
@@ -1559 +1559 @@
-daughter)
+daughter#daughter_n)
@@ -1562 +1562 @@
-Elizabeth)
+Elizabeth#NA)
@@ -1564 +1564 @@
-Masham))))
+Masham#NA))))
@@ -1571 +1571 @@
-I))
+I#i_pron))
@@ -1573 +1573 @@
-have)
+have#have_v)
@@ -1575 +1575 @@
-sent)
+sent#send_v1)
@@ -1578 +1578 @@
-you))
+you#you_pron))
@@ -1581 +1581 @@
-a)
+a#a_adj2)
@@ -1583 +1583 @@
-small)
+small#small_adj)
@@ -1585 +1585 @@
-remembranc)
+remembranc#remembrance_n)
@@ -1588 +1588 @@
-of)
+of#of_prep)
@@ -1591 +1591 @@
-my)
+my#my_adj)
@@ -1593 +1593 @@
-duty)))
+duty#duty_n)))
@@ -1598 +1598 @@
-sum)
+sum#some_pron)
@@ -1600 +1600 @@
-plaine)
+plaine#plain_adj1.A)
@@ -1602 +1602 @@
-gloves)))
+gloves#glove_n)))
@@ -1614 +1614 @@
-To)
+To#to_prep)
@@ -1617 +1617 @@
-my)
+my#my_adj)
@@ -1620 +1620 @@
-much)
+much#much_adj)
@@ -1622 +1622 @@
-honoured))
+honoured#honoured_adj))
@@ -1624 +1624 @@
-mother)
+mother#mother_n1)
@@ -1627 +1627 @@
-the)
+the#the_adj)
@@ -1629 +1629 @@
-Lady)
+Lady#lady_n)
@@ -1632 +1632 @@
-Barrington)))))
+Barrington#NA)))))
@@ -1635 +1635 @@
-these))
+these#these_pron))
@@ -1637 +1637 @@
-be)
+be#be_v)
@@ -1639 +1639 @@
-given))
+given#give_v))
@@ -1655 +1655 @@
-Deare)
+Deare#dear_adj1)
@@ -1657 +1657 @@
-mother))
+mother#mother_n1))
@@ -1660 +1660 @@
-I))
+I#i_pron))
@@ -1662 +1662 @@
-acknowledg)
+acknowledg#acknowledge_v)
@@ -1666 +1666 @@
-my)
+my#my_adj)
@@ -1668 +1668 @@
-self))
+self#self_pron))
@@ -1672 +1672 @@
-very)
+very#very_adj)
@@ -1674 +1674 @@
-much)))
+much#much_adj)))
@@ -1676 +1676 @@
-bownd)
+bownd#bind_v)
@@ -1679 +1679 @@
-to)
+to#to_prep)
@@ -1682 +1682 @@
-you)))
+you#you_pron)))
@@ -1685 +1685 @@
-for)
+for#for_prep)
@@ -1688 +1688 @@
-your)
+your#your_pron)
@@ -1690 +1690 @@
-care)
+care#care_n1)
@@ -1693 +1693 @@
-of)
+of#of_prep)
@@ -1696 +1696 @@
-Jug)
+Jug#NA)
@@ -1698 +1698 @@
-Altham)))))
+Altham#NA)))))
@@ -1701 +1701 @@
-in)
+in#in_prep)
@@ -1704 +1704 @@
-this)
+this#this_pron)
@@ -1706 +1706 @@
-watye)
+watye#weighty_adj)
@@ -1708 +1708 @@
-busines)
+busines#business_n)
@@ -1714 +1714 @@
-wherin))
+wherin#wherein_adv))
@@ -1720 +1720 @@
-I))
+I#i_pron))
@@ -1722 +1722 @@
-would)
+would#will_v1)
@@ -1724 +1724 @@
-be)
+be#be_v)
@@ -1727 +1727 @@
-loth)
+loth#loath_adj)
@@ -1730 +1730 @@
-to)
+to#to_prep)
@@ -1732 +1732 @@
-doe)
+doe#do_v)
@@ -1735 +1735 @@
-anything)
+anything#anything_pron)
@@ -1740 +1740 @@
-withowt)
+withowt#without_adv)
@@ -1743 +1743 @@
-your)
+your#your_pron)
@@ -1745 +1745 @@
-advice)
+advice#advice_n)
@@ -1748 +1748 @@
-and)
+and#and_conj1)
@@ -1751 +1751 @@
-aprobation))))))))))))
+aprobation#approbation_n))))))))))))
@@ -1760 +1760 @@
-I))
+I#i_pron))
@@ -1762 +1762 @@
-conffes)
+conffes#confess_v)
@@ -1769 +1769 @@
-the)
+the#the_adj)
@@ -1771 +1771 @@
-man))
+man#man_n1))
@@ -1773 +1773 @@
-moveth)
+moveth#mouth_v)
@@ -1776 +1776 @@
-me))
+me#me_pron1))
@@ -1779 +1779 @@
-much))
+much#much_adj))
@@ -1782 +1782 @@
-to)
+to#to_prep)
@@ -1784 +1784 @@
-aprove)
+aprove#approve_v1)
@@ -1787 +1787 @@
-of)
+of#of_prep)
@@ -1790 +1790 @@
-it))))))
+it#it_pron))))))
@@ -1798 +1798 @@
-but)
+but#but_prep)
@@ -1801 +1801 @@
-I))
+I#i_pron))
@@ -1803 +1803 @@
-know)
+know#know_v)
@@ -1810 +1810 @@
-God))
+God#NA))
@@ -1812 +1812 @@
-comandes)
+comandes#command_v)
@@ -1815 +1815 @@
-me))
+me#me_pron1))
@@ -1818 +1818 @@
-to)
+to#to_prep)
@@ -1820 +1820 @@
-have)
+have#have_v)
@@ -1823 +1823 @@
-a)
+a#a_adj2)
@@ -1825 +1825 @@
-care))
+care#care_n1))
@@ -1828 +1828 @@
-in)
+in#in_prep)
@@ -1831 +1831 @@
-the)
+the#the_adj)
@@ -1833 +1833 @@
-second)
+second#second_adj)
@@ -1835 +1835 @@
-place)))
+place#place_n.II-VII)))
@@ -1838 +1838 @@
-of)
+of#of_prep)
@@ -1841 +1841 @@
-the)
+the#the_adj)
@@ -1843 +1843 @@
-outward)
+outward#outward_adj)
@@ -1845 +1845 @@
-conveniencis))))))
+conveniencis#convenience_n))))))
@@ -1854 +1854 @@
-tho)
+tho#though_adv)
@@ -1861 +1861 @@
-I))
+I#i_pron))
@@ -1863 +1863 @@
-desier)
+desier#desire_v)
@@ -1866 +1866 @@
-to)
+to#to_prep)
@@ -1868 +1868 @@
-acsept)
+acsept#accept_v)
@@ -1871 +1871 @@
-of)
+of#of_prep)
@@ -1875 +1875 @@
-much)
+much#much_adj)
@@ -1877 +1877 @@
-less))))
+less#less_adj))))
@@ -1880 +1880 @@
-with)
+with#with_prep)
@@ -1883 +1883 @@
-such)
+such#such_adj)
@@ -1885 +1885 @@
-a)
+a#a_adj2)
@@ -1887 +1887 @@
-man)))))))
+man#man_n1)))))))
@@ -1892 +1892 @@
-yet))
+yet#yet_adv))
@@ -1895 +1895 @@
-I))
+I#i_pron))
@@ -1897 +1897 @@
-shall@)
+shall@#shall_v)
@@ -1899 +1899 @@
-@be)
+@be#be_v)
@@ -1902 +1902 @@
-much))
+much#much_adj))
@@ -1904 +1904 @@
-taxed)
+taxed#tax_v)
@@ -1907 +1907 @@
-of)
+of#of_prep)
@@ -1910 +1910 @@
-hir)
+hir#her_pron2)
@@ -1912 +1912 @@
-friends)))
+friends#friend_n)))
@@ -1915 +1915 @@
-if)
+if#if_conj)
@@ -1922 +1922 @@
-I))
+I#i_pron))
@@ -1924 +1924 @@
-look)
+look#look_v)
@@ -1926 +1926 @@
-not)
+not#not_adv)
@@ -1929 +1929 @@
-for)
+for#for_prep)
@@ -1932 +1932 @@
-a)
+a#a_adj2)
@@ -1934 +1934 @@
-compedency)
+compedency#competency_n)
@@ -1937 +1937 @@
-of)
+of#of_prep)
@@ -1940 +1940 @@
-owtward)
+owtward#outward_adj)
@@ -1942 +1942 @@
-estate))))))))
+estate#estate_n))))))))
@@ -1951 +1951 @@
-I))
+I#i_pron))
@@ -1953 +1953 @@
-think)
+think#think_v2)
@@ -1960 +1960 @@
-thar))
+thar#there_adv))
@@ -1962 +1962 @@
-may)
+may#may_v1)
@@ -1964 +1964 @@
-be)
+be#be_v)
@@ -1969 +1969 @@
-prity)
+prity#pretty_adv)
@@ -1971 +1971 @@
-well))))
+well#well_adv))))
@@ -1979 +1979 @@
-for)
+for#for_prep)
@@ -1982 +1982 @@
-thare)
+thare#their_adj)
@@ -1984 +1984 @@
-2)
+2#two_adj)
@@ -1986 +1986 @@
-lives)
+lives#life_n)
@@ -1991 +1991 @@
-his)
+his#his_adj)
@@ -1993 +1993 @@
-and)
+and#and_conj1)
@@ -1995 +1995 @@
-hirs)))
+hirs#heterocyclic_adj)))
@@ -1999 +1999 @@
-will)
+will#will_v1)
@@ -2001 +2001 @@
-make)
+make#make_v1)
@@ -2006 +2006 @@
-between)
+between#between_prep)
@@ -2010 +2010 @@
-4)
+4#NA)
@@ -2012 +2012 @@
-and)
+and#and_conj1)
@@ -2014 +2014 @@
-5_100@)))))
+5_100@#NA)))))
@@ -2016 +2016 @@
-@=li=)
+@=li=#pound_n1)
@@ -2019 +2019 @@
-yearly))
+yearly#yearly_adv))
@@ -2022 +2022 @@
-besides)
+besides#besides_adv)
@@ -2027 +2027 @@
-what))
+what#what_pron))
@@ -2035 +2035 @@
-he))
+he#he_pron))
@@ -2037 +2037 @@
-may)
+may#may_v1)
@@ -2039 +2039 @@
-get)
+get#get_v)
@@ -2042 +2042 @@
-in)
+in#in_prep)
@@ -2045 +2045 @@
-law))))))))
+law#law_n1))))))))
@@ -2053 +2053 @@
-but)
+but#but_prep)
@@ -2056 +2056 @@
-some))
+some#some_pron))
@@ -2058 +2058 @@
-say)
+say#say_v1)
@@ -2065 +2065 @@
-hi))
+hi#he_pron))
@@ -2067 +2067 @@
-can@)
+can@#can_v1)
@@ -2069 +2069 @@
-@not)
+@not#not_adv)
@@ -2071 +2071 @@
-get)
+get#get_v)
@@ -2074 +2074 @@
-much))
+much#much_adj))
@@ -2077 +2077 @@
-yet))))
+yet#yet_adv))))
@@ -2085 +2085 @@
-But)
+But#but_prep)
@@ -2088 +2088 @@
-Sir)
+Sir#NA)
@@ -2090 +2090 @@
-Nathanell)
+Nathanell#NA)
@@ -2092 +2092 @@
-Rich))
+Rich#NA))
@@ -2094 +2094 @@
-tolde)
+tolde#tell_v)
@@ -2097 +2097 @@
-me))
+me#me_pron1))
@@ -2104 +2104 @@
-he))
+he#he_pron))
@@ -2106 +2106 @@
-thought)
+thought#think_v2)
@@ -2113 +2113 @@
-Mr)
+Mr#NA)
@@ -2115 +2115 @@
-Saint)
+Saint#NA)
@@ -2117 +2117 @@
-Jhons)
+Jhons#NA)
@@ -2121 +2121 @@
-colde)
+colde#can_v1)
@@ -2123 +2123 @@
-not)
+not#not_adv)
@@ -2125 +2125 @@
-spend)
+spend#spend_v1)
@@ -2128 +2128 @@
-less)
+less#less_adj)
@@ -2131 +2131 @@
-then)
+then#than_conj)
@@ -2134 +2134 @@
-500@)
+500@#NA)
@@ -2136 +2136 @@
-@=li=)
+@=li=#pound_n1)
@@ -2139 +2139 @@
-yearly)))))
+yearly#yearly_adv)))))
@@ -2142 +2142 @@
-him)
+him#him_pron)
@@ -2144 +2144 @@
-self))
+self#self_pron))
@@ -2147 +2147 @@
-now)
+now#now_adv)
@@ -2158 +2158 @@
-he))
+he#he_pron))
@@ -2160 +2160 @@
-is)
+is#be_v)
@@ -2163 +2163 @@
-a)
+a#a_adj2)
@@ -2165 +2165 @@
-single)
+single#single_adj)
@@ -2167 +2167 @@
-man)))))))))
+man#man_n1)))))))))
@@ -2176 +2176 @@
-if)
+if#if_conj)
@@ -2183 +2183 @@
-that))
+that#that_pron))
@@ -2185 +2185 @@
-be)
+be#be_v)
@@ -2188 +2188 @@
-soe)))))
+soe#so_adv)))))
@@ -2191 +2191 @@
-then))
+then#then_adv))
@@ -2194 +2194 @@
-hir)
+hir#her_pron2)
@@ -2196 +2196 @@
-estate))
+estate#estate_n))
@@ -2198 +2198 @@
-will@)
+will@#will_v1)
@@ -2200 +2200 @@
-@be)
+@be#be_v)
@@ -2203 +2203 @@
-but)
+but#but_prep)
@@ -2205 +2205 @@
-little)
+little#little_adj)
@@ -2209 +2209 @@
-to)
+to#to_prep)
@@ -2211 +2211 @@
-pay)
+pay#pay_v1)
@@ -2214 +2214 @@
-howsrent)))
+howsrent#house-rent_n)))
@@ -2217 +2217 @@
-and)
+and#and_conj1)
@@ -2222 +2222 @@
-maintaine)
+maintaine#maintain_v)
@@ -2225 +2225 @@
-howskeeping))
+howskeeping#housekeeping_n))
@@ -2228 +2228 @@
-if)
+if#if_conj)
@@ -2235 +2235 @@
-thay))
+thay#they_pron))
@@ -2237 +2237 @@
-keep)
+keep#keep_v)
@@ -2240 +2240 @@
-any)))))))
+any#any_adj)))))))
@@ -2245 +2245 @@
-or)
+or#or_conj2)
@@ -2247 +2247 @@
-elc)
+elc#else_adv)
@@ -2250 +2250 @@
-to)
+to#to_prep)
@@ -2252 +2252 @@
-sogiorn)
+sogiorn#sojourn_v)
@@ -2255 +2255 @@
-in)
+in#in_prep)
@@ -2259 +2259 @@
-another)
+another#another_pron)
@@ -2261 +2261 @@
-bodie's))
+bodie's#body_n))
@@ -2263 +2263 @@
-howse)))))))
+howse#house_n1)))))))
@@ -2272 +2272 @@
-Thay))
+Thay#they_pron))
@@ -2274 +2274 @@
-can@)
+can@#can_v1)
@@ -2276 +2276 @@
-@not)
+@not#not_adv)
@@ -2278 +2278 @@
-gather)
+gather#gather_v)
@@ -2281 +2281 @@
-much)
+much#much_adj)
@@ -2289 +2289 @@
-to)
+to#to_prep)
@@ -2291 +2291 @@
-purchas)
+purchas#purchase_v)
@@ -2294 +2294 @@
-for)
+for#for_prep)
@@ -2297 +2297 @@
-posteryty))))))
+posteryty#posterity_n))))))
@@ -2306 +2306 @@
-I))
+I#i_pron))
@@ -2308 +2308 @@
-beseech)
+beseech#beseech_v)
@@ -2311 +2311 @@
-you))
+you#you_pron))
@@ -2315 +2315 @@
-to)
+to#to_prep)
@@ -2317 +2317 @@
-way)
+way#weigh_v1)
@@ -2320 +2320 @@
-all)
+all#all_adj)
@@ -2322 +2322 @@
-things))
+things#thing_n1))
@@ -2325 +2325 @@
-well)))
+well#well_adv)))
@@ -2328 +2328 @@
-and)
+and#and_conj1)
@@ -2333 +2333 @@
-give)
+give#give_v)
@@ -2336 +2336 @@
-me))
+me#me_pron1))
@@ -2339 +2339 @@
-your)
+your#your_pron)
@@ -2341 +2341 @@
-considerat)
+considerat#considerate_adj)
@@ -2343 +2343 @@
-advice)))))
+advice#advice_n)))))
@@ -2352 +2352 @@
-She))
+She#she_pron))
@@ -2354 +2354 @@
-hath)
+hath#have_v)
@@ -2357 +2357 @@
-good)
+good#good_adj)
@@ -2359 +2359 @@
-frinds)
+frinds#friend_n)
@@ -2362 +2362 @@
-to)
+to#to_prep)
@@ -2364 +2364 @@
-advise)
+advise#advise_v)
@@ -2367 +2367 @@
-for)
+for#for_prep)
@@ -2370 +2370 @@
-hir)))
+hir#her_pron1)))
@@ -2373 +2373 @@
-with)
+with#with_prep)
@@ -2376 +2376 @@
-you))))
+you#you_pron))))
@@ -2382 +2382 @@
-my)
+my#my_adj)
@@ -2384 +2384 @@
-brother)
+brother#brother_n)
@@ -2387 +2387 @@
-Garard)))
+Garard#NA)))
@@ -2390 +2390 @@
-and)
+and#and_conj1)
@@ -2393 +2393 @@
-my)
+my#my_adj)
@@ -2395 +2395 @@
-brother)
+brother#brother_n)
@@ -2398 +2398 @@
-Mewix)))))
+Mewix#NA)))))
@@ -2404 +2404 @@
-who))
+who#who_pron))
@@ -2410 +2410 @@
-I))
+I#i_pron))
@@ -2412 +2412 @@
-know)
+know#know_v)
@@ -2420 +2420 @@
-desire)
+desire#desire_v)
@@ -2423 +2423 @@
-to)
+to#to_prep)
@@ -2425 +2425 @@
-seek)
+seek#seek_v)
@@ -2428 +2428 @@
-God's)
+God's#NA)
@@ -2430 +2430 @@
-honor))
+honor#honour_n))
@@ -2433 +2433 @@
-in)
+in#in_prep)
@@ -2436 +2436 @@
-the)
+the#the_adj)
@@ -2438 +2438 @@
-chefe)
+chefe#chief_adj)
@@ -2440 +2440 @@
-place)))))))))
+place#place_n.II-VII)))))))))
@@ -2448 +2448 @@
-and)
+and#and_conj1)
@@ -2451 +2451 @@
-then))
+then#then_adv))
@@ -2454 +2454 @@
-God))
+God#NA))
@@ -2456 +2456 @@
-gives)
+gives#give_v)
@@ -2459 +2459 @@
-leav))
+leav#leave_n1))
@@ -2462 +2462 @@
-to)
+to#to_prep)
@@ -2464 +2464 @@
-seek)
+seek#seek_v)
@@ -2467 +2467 @@
-other)
+other#other_adj)
@@ -2469 +2469 @@
-things)
+things#thing_n1)
@@ -2472 +2472 @@
-as)
+as#as_adv)
@@ -2482 +2482 @@
-may)
+may#may_v1)
@@ -2484 +2484 @@
-be)
+be#be_v)
@@ -2487 +2487 @@
-fitt)
+fitt#fit_adj)
@@ -2490 +2490 @@
-to)
+to#to_prep)
@@ -2492 +2492 @@
-make)
+make#make_v1)
@@ -2496 +2496 @@
-our)
+our#our_pron)
@@ -2498 +2498 @@
-pasadg)
+pasadg#passage_n)
@@ -2504 +2504 @@
-the)
+the#the_adj)
@@ -2506 +2506 @@
-more))
+more#more_adj))
@@ -2508 +2508 @@
-comfortable))
+comfortable#comfortable_adj))
@@ -2511 +2511 @@
-to)
+to#to_prep)
@@ -2514 +2514 @@
-that)
+that#that_pron)
@@ -2516 +2516 @@
-place)
+place#place_n.II-VII)
@@ -2520 +2520 @@
-whare))
+whare#where_adv))
@@ -2528 +2528 @@
-we))
+we#we_pron))
@@ -2530 +2530 @@
-shall)
+shall#shall_v)
@@ -2532 +2532 @@
-have)
+have#have_v)
@@ -2535 +2535 @@
-no)
+no#no_adj)
@@ -2537 +2537 @@
-need))
+need#need_n1))
@@ -2540 +2540 @@
-of)
+of#of_prep)
@@ -2543 +2543 @@
-these)
+these#these_pron)
@@ -2545 +2545 @@
-vanitis)))))))))))))))
+vanitis#vanity_n)))))))))))))))
@@ -2554 +2554 @@
-The)
+The#the_adj)
@@ -2556 +2556 @@
-lord))
+lord#NA))
@@ -2558 +2558 @@
-fitt)
+fitt#fit_v1)
@@ -2561 +2561 @@
-us))
+us#us_pron))
@@ -2564 +2564 @@
-for)
+for#for_prep)
@@ -2568 +2568 @@
-soe)
+soe#so_adv)
@@ -2570 +2570 @@
-gloryus))
+gloryus#glorious_adj))
@@ -2572 +2572 @@
-a)
+a#a_adj2)
@@ -2574 +2574 @@
-place)))
+place#place_n.II-VII)))
@@ -2583 +2583 @@
-I))
+I#i_pron))
@@ -2585 +2585 @@
-am)
+am#be_v)
@@ -2588 +2588 @@
-very)
+very#very_adj)
@@ -2590 +2590 @@
-glad)
+glad#glad_adj)
@@ -2593 +2593 @@
-to)
+to#to_prep)
@@ -2595 +2595 @@
-here)
+here#hear_v)
@@ -2602 +2602 @@
-you))
+you#you_pron))
@@ -2604 +2604 @@
-have)
+have#have_v)
@@ -2607 +2607 @@
-your)
+your#your_pron)
@@ -2611 +2611 @@
-health))
+health#health_n))
@@ -2614 +2614 @@
-soe)
+soe#so_adv)
@@ -2616 +2616 @@
-well))))))
+well#well_adv))))))
@@ -2625 +2625 @@
-I))
+I#i_pron))
@@ -2628 +2628 @@
-much))
+much#much_adj))
@@ -2630 +2630 @@
-longe)
+longe#long_v1)
@@ -2633 +2633 @@
-to)
+to#to_prep)
@@ -2635 +2635 @@
-se)
+se#see_v)
@@ -2638 +2638 @@
-you)))
+you#you_pron)))
@@ -2647 +2647 @@
-I))
+I#i_pron))
@@ -2649 +2649 @@
-hope)
+hope#hope_v)
@@ -2656 +2656 @@
-the)
+the#the_adj)
@@ -2658 +2658 @@
-time))
+time#time_n))
@@ -2660 +2660 @@
-will)
+will#will_v1)
@@ -2662 +2662 @@
-not)
+not#not_adv)
@@ -2664 +2664 @@
-be)
+be#be_v)
@@ -2667 +2667 @@
-longe))))
+longe#long_adj1))))
@@ -2676 +2676 @@
-The)
+The#the_adj)
@@ -2678 +2678 @@
-lord))
+lord#NA))
@@ -2680 +2680 @@
-send)
+send#send_v1)
@@ -2683 +2683 @@
-my)
+my#my_adj)
@@ -2685 +2685 @@
-sister))
+sister#sister_n))
@@ -2688 +2688 @@
-a)
+a#a_adj2)
@@ -2690 +2690 @@
-safe)
+safe#safe_adj)
@@ -2692 +2692 @@
-deliveranc))
+deliveranc#deliverance_n))
@@ -2695 +2695 @@
-that)
+that#that_conj)
@@ -2699 +2699 @@
-you))
+you#you_pron))
@@ -2701 +2701 @@
-may)
+may#may_v1)
@@ -2703 +2703 @@
-come)
+come#come_v)
@@ -2706 +2706 @@
-with)
+with#with_prep)
@@ -2709 +2709 @@
-more)
+more#more_adj)
@@ -2711 +2711 @@
-joy)))))
+joy#joy_n)))))
@@ -2720 +2720 @@
-Thus))
+Thus#thus_adv))
@@ -2723 +2723 @@
-in)
+in#in_prep)
@@ -2726 +2726 @@
-hast)))
+hast#haste_n)))
@@ -2729 +2729 @@
-with)
+with#with_prep)
@@ -2732 +2732 @@
-my)
+my#my_adj)
@@ -2734 +2734 @@
-humble)
+humble#humble_adj)
@@ -2736 +2736 @@
-duty)
+duty#duty_n)
@@ -2739 +2739 @@
-and)
+and#and_conj1)
@@ -2742 +2742 @@
-harty)
+harty#hearty_adj)
@@ -2744 +2744 @@
-prayers)
+prayers#prayer_n1)
@@ -2747 +2747 @@
-for)
+for#for_prep)
@@ -2750 +2750 @@
-your)
+your#your_pron)
@@ -2752 +2752 @@
-increse)
+increse#increase_n)
@@ -2755 +2755 @@
-of)
+of#of_prep)
@@ -2758 +2758 @@
-happynes)
+happynes#happiness_n)
@@ -2762 +2762 @@
-here))
+here#here_adv))
@@ -2765 +2765 @@
-and)
+and#and_conj1)
@@ -2768 +2768 @@
-especially)
+especially#especially_adv)
@@ -2770 +2770 @@
-in)
+in#in_prep)
@@ -2773 +2773 @@
-the)
+the#the_adj)
@@ -2775 +2775 @@
-life)
+life#life_n)
@@ -2783 +2783 @@
-to)
+to#to_prep)
@@ -2785 +2785 @@
-come)))))))))))))))
+come#come_v)))))))))))))))
@@ -2790 +2790 @@
-I))
+I#i_pron))
@@ -2792 +2792 @@
-rest)
+rest#rest_v2)
@@ -2795 +2795 @@
-Your)
+Your#your_pron)
@@ -2798 +2798 @@
-ever)
+ever#ever_adv)
@@ -2800 +2800 @@
-obedient))
+obedient#obedient_adj))
@@ -2802 +2802 @@
-daughter)
+daughter#daughter_n)
@@ -2805 +2805 @@
-Elizabeth)
+Elizabeth#NA)
@@ -2807 +2807 @@
-Masham))))
+Masham#NA))))
@@ -2814 +2814 @@
-I))
+I#i_pron))
@@ -2816 +2816 @@
-desire)
+desire#desire_v)
@@ -2819 +2819 @@
-to)
+to#to_prep)
@@ -2821 +2821 @@
-be)
+be#be_v)
@@ -2823 +2823 @@
-remembred)
+remembred#remember_v)
@@ -2826 +2826 @@
-to)
+to#to_prep)
@@ -2829 +2829 @@
-all)
+all#all_adj)
@@ -2831 +2831 @@
-my)
+my#my_adj)
@@ -2833 +2833 @@
-brothers)
+brothers#brother_n)
@@ -2836 +2836 @@
-and)
+and#and_conj1)
@@ -2839 +2839 @@
-sisters))))))
+sisters#sister_n))))))
@@ -2848 +2848 @@
-I))
+I#i_pron))
@@ -2850 +2850 @@
-beseech)
+beseech#beseech_v)
@@ -2853 +2853 @@
-you))
+you#you_pron))
@@ -2857 +2857 @@
-if)
+if#if_conj)
@@ -2864 +2864 @@
-you))
+you#you_pron))
@@ -2866 +2866 @@
-think)
+think#think_v2)
@@ -2870 +2870 @@
-it))
+it#it_pron))
@@ -2872 +2872 @@
-not)
+not#not_adv)
@@ -2875 +2875 @@
-fitt))
+fitt#fit_adj))
@@ -2878 +2878 @@
-to)
+to#to_prep)
@@ -2880 +2880 @@
-proceed)
+proceed#proceed_v)
@@ -2883 +2883 @@
-in)
+in#in_prep)
@@ -2886 +2886 @@
-the)
+the#the_adj)
@@ -2888 +2888 @@
-busines))))))))
+busines#business_n))))))))
@@ -2890 +2890 @@
-that)
+that#that_conj)
@@ -2896 +2896 @@
-you))
+you#you_pron))
@@ -2898 +2898 @@
-will)
+will#will_v1)
@@ -2900 +2900 @@
-write)
+write#write_v)
@@ -2903 +2903 @@
-your)
+your#your_pron)
@@ -2905 +2905 @@
-minde))
+minde#mind_n1))
@@ -2908 +2908 @@
-to)
+to#to_prep)
@@ -2911 +2911 @@
-Jug)))))
+Jug#NA)))))
@@ -2919 +2919 @@
-for)
+for#for_prep)
@@ -2922 +2922 @@
-she))
+she#she_pron))
@@ -2924 +2924 @@
-desirs)
+desirs#desire_v)
@@ -2927 +2927 @@
-to)
+to#to_prep)
@@ -2929 +2929 @@
-be)
+be#be_v)
@@ -2931 +2931 @@
-directed)
+directed#direct_v)
@@ -2934 +2934 @@
-by)
+by#by_prep)
@@ -2937 +2937 @@
-you))))
+you#you_pron))))
@@ -2949 +2949 @@
-For)
+For#for_prep)
@@ -2952 +2952 @@
-my)
+my#my_adj)
@@ -2956 +2956 @@
-much)
+much#much_adj)
@@ -2958 +2958 @@
-honoured))
+honoured#honoured_adj))
@@ -2961 +2961 @@
-and)
+and#and_conj1)
@@ -2964 +2964 @@
-most)
+most#most_adj)
@@ -2966 +2966 @@
-deare))))
+deare#dear_adj1))))
@@ -2968 +2968 @@
-mother)
+mother#mother_n1)
@@ -2971 +2971 @@
-the)
+the#the_adj)
@@ -2973 +2973 @@
-Lady)
+Lady#lady_n)
@@ -2976 +2976 @@
-Barrington)))
+Barrington#NA)))
@@ -2979 +2979 @@
-at)
+at#at_prep)
@@ -2982 +2982 @@
-Harrow)))))
+Harrow#NA)))))
@@ -2985 +2985 @@
-these)))
+these#these_pron)))
