--- rhaddsr-1712-e3-p2.psd.vert	2025-09-10 13:51:18
+++ rhaddsr-1712-e3-p2.lem.vert	2025-09-10 13:51:26
@@ -13 +13 @@
-Clapham)
+Clapham#NA)
@@ -21 +21 @@
-28=th=)
+28=th=#NA)
@@ -23 +23 @@
-May)
+May#NA)
@@ -27 +27 @@
-1712)
+1712#NA)
@@ -35 +35 @@
-Wedensday)
+Wedensday#NA)
@@ -44 +44 @@
-Deare)
+Deare#dear_adj1)
@@ -46 +46 @@
-Grandson))
+Grandson#grandson_n))
@@ -51 +51 @@
-I))
+I#i_pron))
@@ -53 +53 @@
-came)
+came#come_v)
@@ -56 +56 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -59 +59 @@
-to)
+to#to_prep)
@@ -62 +62 @@
-this)
+this#this_pron)
@@ -64 +64 @@
-place)))
+place#place_n.II-VII)))
@@ -72 +72 @@
-and)
+and#and_conj1)
@@ -77 +77 @@
-according)
+according#accord_v)
@@ -80 +80 @@
-to)
+to#to_prep)
@@ -83 +83 @@
-my)
+my#my_adj)
@@ -85 +85 @@
-promise))))
+promise#promise_n))))
@@ -91 +91 @@
-do)
+do#do_v)
@@ -93 +93 @@
-answer)
+answer#answer_v)
@@ -96 +96 @@
-yours)
+yours#yours_pron)
@@ -99 +99 @@
-of)
+of#of_prep)
@@ -102 +102 @@
-18=th=)
+18=th=#NA)
@@ -104 +104 @@
-instant)
+instant#instant_adj)
@@ -107 +107 @@
-from)
+from#from_prep)
@@ -110 +110 @@
-Christ)
+Christ#NA)
@@ -112 +112 @@
-Colledge)
+Colledge#NA)
@@ -115 +115 @@
-in)
+in#in_prep)
@@ -118 +118 @@
-Oxford))))))))
+Oxford#NA))))))))
@@ -127 +127 @@
-It))
+It#it_pron))
@@ -129 +129 @@
-happens)
+happens#happen_v)
@@ -132 +132 @@
-to)
+to#to_prep)
@@ -134 +134 @@
-be)
+be#be_v)
@@ -137 +137 @@
-this)
+this#this_pron)
@@ -139 +139 @@
-day))
+day#day_n))
@@ -143 +143 @@
-40)
+40#forty_adj)
@@ -145 +145 @@
-years))
+years#year_n))
@@ -150 +150 @@
-that)
+that#that_conj)
@@ -154 +154 @@
-I))
+I#i_pron))
@@ -156 +156 @@
-was)
+was#be_v)
@@ -158 +158 @@
-burnt)
+burnt#burn_v1)
@@ -161 +161 @@
-out)
+out#out_adv)
@@ -163 +163 @@
-of)
+of#of_prep)
@@ -166 +166 @@
-the)
+the#the_adj)
@@ -168 +168 @@
-Royall)
+Royall#NA)
@@ -170 +170 @@
-James)))
+James#NA)))
@@ -173 +173 @@
-in)
+in#in_prep)
@@ -176 +176 @@
-the)
+the#the_adj)
@@ -178 +178 @@
-Sole)
+Sole#NA)
@@ -180 +180 @@
-Bay)
+Bay#NA)
@@ -182 +182 @@
-fight)
+fight#fight_n)
@@ -185 +185 @@
-against)
+against#against_prep)
@@ -188 +188 @@
-the)
+the#the_adj)
@@ -190 +190 @@
-Dutch))))))))
+Dutch#dutch_adj))))))))
@@ -200 +200 @@
-Am)
+Am#be_v)
@@ -203 +203 @@
-well))
+well#well_adv))
@@ -205 +205 @@
-pleased)
+pleased#please_v)
@@ -208 +208 @@
-to)
+to#to_prep)
@@ -210 +210 @@
-find)
+find#find_v)
@@ -213 +213 @@
-the)
+the#the_adj)
@@ -215 +215 @@
-efforts)
+efforts#effort_n)
@@ -226 +226 @@
-you))
+you#you_pron))
@@ -228 +228 @@
-intend)
+intend#intend_v)
@@ -231 +231 @@
-to)
+to#to_prep)
@@ -233 +233 @@
-make)
+make#make_v1)
@@ -237 +237 @@
-yourselfe))
+yourselfe#yourself_pron))
@@ -240 +240 @@
-famous)
+famous#famous_adj)
@@ -243 +243 @@
-in)
+in#in_prep)
@@ -246 +246 @@
-Westminster)
+Westminster#NA)
@@ -248 +248 @@
-Hall))))))))))
+Hall#NA))))))))))
@@ -257 +257 @@
-It))
+It#it_pron))
@@ -259 +259 @@
-is)
+is#be_v)
@@ -262 +262 @@
-like)
+like#like_adj)
@@ -265 +265 @@
-the)
+the#the_adj)
@@ -267 +267 @@
-saying)
+saying#saying_n1)
@@ -270 +270 @@
-of)
+of#of_prep)
@@ -273 +273 @@
-your)
+your#your_pron)
@@ -275 +275 @@
-Unckle)
+Unckle#uncle_n)
@@ -278 +278 @@
-Nich=s=))
+Nich=s=#NA))
@@ -284 +284 @@
-who))
+who#who_pron))
@@ -291 +291 @@
-doubted)
+doubted#doubt_v)
@@ -293 +293 @@
-not)
+not#not_adv)
@@ -296 +296 @@
-but)
+but#but_prep)
@@ -298 +298 @@
-to)
+to#to_prep)
@@ -300 +300 @@
-be)
+be#be_v)
@@ -303 +303 @@
-as)
+as#as_adv)
@@ -305 +305 @@
-great)
+great#great_adj)
@@ -308 +308 @@
-as)
+as#as_adv)
@@ -319 +319 @@
-S=r=)
+S=r=#NA)
@@ -321 +321 @@
-Cloudesly)
+Cloudesly#NA)
@@ -323 +323 @@
-Shovell))
+Shovell#NA))
@@ -325 +325 @@
-was))))))))))))
+was#be_v))))))))))))
@@ -333 +333 @@
-and)
+and#and_conj1)
@@ -336 +336 @@
-he))
+he#he_pron))
@@ -338 +338 @@
-pushes)
+pushes#push_v)
@@ -341 +341 @@
-very)
+very#very_adj)
@@ -343 +343 @@
-faire))
+faire#fair_adv))
@@ -346 +346 @@
-for)
+for#for_prep)
@@ -349 +349 @@
-it)))
+it#it_pron)))
@@ -358 +358 @@
-Your)
+Your#your_pron)
@@ -360 +360 @@
-father)
+father#father_n)
@@ -363 +363 @@
-and)
+and#and_conj1)
@@ -366 +366 @@
-family))))
+family#family_n))))
@@ -368 +368 @@
-went)
+went#go_v)
@@ -371 +371 @@
-to)
+to#to_prep)
@@ -374 +374 @@
-Wakehurst)))
+Wakehurst#NA)))
@@ -379 +379 @@
-Satuarday)
+Satuarday#NA)
@@ -382 +382 @@
-last)))
+last#last_adv)))
@@ -392 +392 @@
-tooke)
+tooke#take_v)
@@ -396 +396 @@
-Betty))
+Betty#NA))
@@ -399 +399 @@
-and)
+and#and_conj1)
@@ -402 +402 @@
-Fanny)
+Fanny#NA)
@@ -404 +404 @@
-Clark))))
+Clark#NA))))
@@ -406 +406 @@
-downe)
+downe#down_adv)
@@ -409 +409 @@
-w=th=)
+w=th=#with_prep)
@@ -412 +412 @@
-them)))
+them#them_pron)))
@@ -422 +422 @@
-gote)
+gote#get_v)
@@ -425 +425 @@
-well))
+well#well_adv))
@@ -428 +428 @@
-thither))
+thither#thither_adv))
@@ -437 +437 @@
-Yo=r=)
+Yo=r=#your_pron)
@@ -439 +439 @@
-Unckle)
+Unckle#uncle_n)
@@ -442 +442 @@
-Richard)))
+Richard#NA)))
@@ -447 +447 @@
-the)
+the#the_adj)
@@ -449 +449 @@
-weeke)
+weeke#week_n)
@@ -452 +452 @@
-past)))
+past#past_adj)))
@@ -456 +456 @@
-hath)
+hath#have_v)
@@ -458 +458 @@
-bine)
+bine#be_v)
@@ -461 +461 @@
-very)
+very#very_adj)
@@ -463 +463 @@
-ill)
+ill#ill_adj)
@@ -466 +466 @@
-w=th=)
+w=th=#with_prep)
@@ -469 +469 @@
-a)
+a#a_adj2)
@@ -471 +471 @@
-feavor)
+feavor#favour_n)
@@ -474 +474 @@
-and)
+and#and_conj1)
@@ -477 +477 @@
-ague)))
+ague#ague_n)))
@@ -483 +483 @@
-w=ch=))
+w=ch=#which_adj))
@@ -490 +490 @@
-kept)
+kept#keep_v)
@@ -493 +493 @@
-me))
+me#me_pron1))
@@ -496 +496 @@
-from)
+from#from_prep)
@@ -499 +499 @@
-hence)))
+hence#hence_adv)))
@@ -502 +502 @@
-longer)
+longer#long_adj1)
@@ -505 +505 @@
-than)
+than#than_conj)
@@ -516 +516 @@
-I))
+I#i_pron))
@@ -518 +518 @@
-designed))))))))))
+designed#design_v))))))))))
@@ -528 +528 @@
-is)
+is#be_v)
@@ -531 +531 @@
-under)
+under#under_prep)
@@ -534 +534 @@
-the)
+the#the_adj)
@@ -536 +536 @@
-advice)
+advice#advice_n)
@@ -539 +539 @@
-of)
+of#of_prep)
@@ -542 +542 @@
-Doctor)
+Doctor#NA)
@@ -544 +544 @@
-Ratclif)
+Ratclif#NA)
@@ -550 +550 @@
-who))
+who#who_pron))
@@ -557 +557 @@
-gave)
+gave#give_v)
@@ -560 +560 @@
-me))
+me#me_pron1))
@@ -563 +563 @@
-leave))
+leave#leave_n1))
@@ -566 +566 @@
-to)
+to#to_prep)
@@ -568 +568 @@
-come)
+come#come_v)
@@ -570 +570 @@
-downe)
+downe#down_adv)
@@ -573 +573 @@
-hither)))
+hither#hither_adv)))
@@ -578 +578 @@
-promiscing)
+promiscing#promise_v)
@@ -581 +581 @@
-his)
+his#his_adj)
@@ -583 +583 @@
-care)
+care#care_n1)
@@ -586 +586 @@
-of)
+of#of_prep)
@@ -589 +589 @@
-him)))))))))))
+him#him_pron)))))))))))
@@ -597 +597 @@
-and)
+and#and_conj1)
@@ -601 +601 @@
-was)
+was#be_v)
@@ -604 +604 @@
-downe)
+downe#down_prep)
@@ -607 +607 @@
-staires)))
+staires#stair_n)))
@@ -610 +610 @@
-when)
+when#when_adv)
@@ -618 +618 @@
-came)
+came#come_v)
@@ -621 +621 @@
-away)))))
+away#away_adv)))))
@@ -630 +630 @@
-With)
+With#with_prep)
@@ -633 +633 @@
-my)
+my#my_adj)
@@ -635 +635 @@
-harty)
+harty#hearty_adj)
@@ -637 +637 @@
-prayers)
+prayers#prayer_n1)
@@ -640 +640 @@
-for)
+for#for_prep)
@@ -643 +643 @@
-yo=r=)
+yo=r=#your_pron)
@@ -645 +645 @@
-health)
+health#health_n)
@@ -648 +648 @@
-and)
+and#and_conj1)
@@ -651 +651 @@
-wellfare)))))))
+wellfare#welfare_n)))))))
@@ -656 +656 @@
-I))
+I#i_pron))
@@ -658 +658 @@
-am)
+am#be_v)
@@ -661 +661 @@
-Yo=r=)
+Yo=r=#your_pron)
@@ -664 +664 @@
-most)
+most#most_adj)
@@ -666 +666 @@
-afec=t=))
+afec=t=#affect_adj))
@@ -668 +668 @@
-grandfather)
+grandfather#grandfather_n)
@@ -673 +673 @@
-R=d=.)
+R=d=.#NA)
@@ -675 +675 @@
-H.))))
+H.#NA))))
