--- tbarring-1630-e2-p2.psd.vert	2025-09-10 13:51:18
+++ tbarring-1630-e2-p2.lem.vert	2025-09-10 13:51:26
@@ -14 +14 @@
-Madame))
+Madame#madam_n))
@@ -17 +17 @@
-This)
+This#this_pron)
@@ -19 +19 @@
-short)
+short#short_adj)
@@ -21 +21 @@
-time)
+time#time_n)
@@ -24 +24 @@
-of)
+of#of_prep)
@@ -27 +27 @@
-distance)))
+distance#distance_n)))
@@ -30 +30 @@
-since)
+since#since_adv)
@@ -37 +37 @@
-I))
+I#i_pron))
@@ -39 +39 @@
-was)
+was#be_v)
@@ -42 +42 @@
-with)
+with#with_prep)
@@ -45 +45 @@
-you)))))))
+you#you_pron)))))))
@@ -47 +47 @@
-hath)
+hath#have_v)
@@ -49 +49 @@
-produced)
+produced#produce_v)
@@ -52 +52 @@
-nothing)
+nothing#nothing_pron)
@@ -55 +55 @@
-either)
+either#either_pron)
@@ -58 +58 @@
-new))
+new#new_adj))
@@ -61 +61 @@
-or)
+or#or_conj2)
@@ -64 +64 @@
-worthy)
+worthy#worthy_adj)
@@ -67 +67 @@
-of)
+of#of_prep)
@@ -70 +70 @@
-your)
+your#your_pron)
@@ -72 +72 @@
-eye)))))))
+eye#eye_n1)))))))
@@ -80 +80 @@
-only)
+only#only_adj)
@@ -83 +83 @@
-the)
+the#the_adj)
@@ -85 +85 @@
-greate)
+greate#great_adj)
@@ -87 +87 @@
-affaire)
+affaire#affair_n)
@@ -90 +90 @@
-of)
+of#of_prep)
@@ -94 +94 @@
-the)
+the#the_adj)
@@ -96 +96 @@
-King's))
+King's#king_n))
@@ -98 +98 @@
-Bench))))
+Bench#bench_n))))
@@ -100 +100 @@
-is)
+is#be_v)
@@ -103 +103 @@
-to_morrow))
+to_morrow#tomorrow_adv))
@@ -106 +106 @@
-to)
+to#to_prep)
@@ -108 +108 @@
-be)
+be#be_v)
@@ -110 +110 @@
-tryed)
+tryed#try_v)
@@ -113 +113 @@
-between)
+between#between_prep)
@@ -117 +117 @@
-the)
+the#the_adj)
@@ -119 +119 @@
-king))
+king#king_n))
@@ -122 +122 @@
-and)
+and#and_conj1)
@@ -125 +125 @@
-our)
+our#our_pron)
@@ -127 +127 @@
-parliament)
+parliament#parliament_n1)
@@ -129 +129 @@
-men))))))
+men#man_n1))))))
@@ -138 +138 @@
-The)
+The#the_adj)
@@ -140 +140 @@
-time))
+time#time_n))
@@ -142 +142 @@
-is)
+is#be_v)
@@ -144 +144 @@
-fixt)
+fixt#fix_v)
@@ -153 +153 @@
-God))
+God#NA))
@@ -155 +155 @@
-send)
+send#send_v1)
@@ -158 +158 @@
-such)
+such#such_adj)
@@ -160 +160 @@
-an)
+an#a_adj2)
@@ -162 +162 @@
-issue)
+issue#issue_n)
@@ -165 +165 @@
-as)
+as#as_adv)
@@ -176 +176 @@
-may)
+may#may_v1)
@@ -179 +179 @@
-most))
+most#most_adj))
@@ -181 +181 @@
-gloryfye)
+gloryfye#glorify_v)
@@ -184 +184 @@
-him)))
+him#him_pron)))
@@ -187 +187 @@
-and)
+and#and_conj1)
@@ -190 +190 @@
-benefitt)
+benefitt#benefit_v)
@@ -193 +193 @@
-king)
+king#king_n)
@@ -195 +195 @@
-and)
+and#and_conj1)
@@ -197 +197 @@
-subjects))))))))
+subjects#subject_n))))))))
@@ -206 +206 @@
-This)
+This#this_pron)
@@ -208 +208 @@
-day))
+day#day_n))
@@ -211 +211 @@
-my)
+my#my_adj)
@@ -213 +213 @@
-cozen)
+cozen#cousin_n)
@@ -216 +216 @@
-Dunch)))
+Dunch#NA)))
@@ -218 +218 @@
-maryes)
+maryes#marry_v)
@@ -221 +221 @@
-his)
+his#his_adj)
@@ -223 +223 @@
-brave)
+brave#brave_adj)
@@ -225 +225 @@
-mistress)
+mistress#mistress_n)
@@ -230 +230 @@
-a)
+a#a_adj2)
@@ -232 +232 @@
-gentle-woman)
+gentle-woman#gentlewoman_n)
@@ -235 +235 @@
-of)
+of#of_prep)
@@ -238 +238 @@
-portion)))))
+portion#portion_n)))))
@@ -245 +245 @@
-education)
+education#education_n)
@@ -247 +247 @@
-and)
+and#and_conj1)
@@ -249 +249 @@
-proportion)
+proportion#proportion_n)
@@ -252 +252 @@
-paralel)
+paralel#parallel_n)
@@ -255 +255 @@
-to)
+to#to_prep)
@@ -258 +258 @@
-his)
+his#his_adj)
@@ -260 +260 @@
-estate)))))
+estate#estate_n)))))
@@ -263 +263 @@
-and)
+and#and_conj1)
@@ -266 +266 @@
-credit)
+credit#credit_n)
@@ -269 +269 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -272 +272 @@
-for)
+for#for_prep)
@@ -275 +275 @@
-his)
+his#his_adj)
@@ -279 +279 @@
-long))
+long#long_adj1))
@@ -281 +281 @@
-delayed))
+delayed#delayed_adj1))
@@ -283 +283 @@
-matrimonye))))))))
+matrimonye#matrimony_n))))))))
@@ -293 +293 @@
-My)
+My#my_adj)
@@ -295 +295 @@
-wife))
+wife#wife_n))
@@ -298 +298 @@
-and)
+and#and_conj1)
@@ -301 +301 @@
-my)
+my#my_adj)
@@ -303 +303 @@
-selfe))))
+selfe#self_pron))))
@@ -305 +305 @@
-are)
+are#be_v)
@@ -308 +308 @@
-solemnely))
+solemnely#solemnly_adv))
@@ -310 +310 @@
-invited)
+invited#invite_v)
@@ -313 +313 @@
-to_morrow))
+to_morrow#tomorrow_adv))
@@ -316 +316 @@
-to)
+to#to_prep)
@@ -318 +318 @@
-be)
+be#be_v)
@@ -321 +321 @@
-thare))
+thare#there_adv))
@@ -324 +324 @@
-when)
+when#when_adv)
@@ -330 +330 @@
-is)
+is#be_v)
@@ -333 +333 @@
-the)
+the#the_adj)
@@ -335 +335 @@
-festivall))))))
+festivall#festival_adj))))))
@@ -342 +342 @@
-this)
+this#this_pron)
@@ -344 +344 @@
-day))
+day#day_n))
@@ -346 +346 @@
-not)
+not#not_adv)
@@ -348 +348 @@
-allowing)
+allowing#allow_v)
@@ -351 +351 @@
-it)))
+it#it_pron)))
@@ -356 +356 @@
-and)
+and#and_conj1)
@@ -360 +360 @@
-the)
+the#the_adj)
@@ -362 +362 @@
-joyneing)
+joyneing#joining_n)
@@ -365 +365 @@
-of)
+of#of_prep)
@@ -368 +368 @@
-hands))))
+hands#hand_n))))
@@ -370 +370 @@
-being)
+being#be_v)
@@ -373 +373 @@
-alltogether)
+alltogether#altogether_adj)
@@ -375 +375 @@
-private)))))
+private#private_adj1)))))
@@ -384 +384 @@
-Madame))
+Madame#madam_n))
@@ -387 +387 @@
-my)
+my#my_adj)
@@ -389 +389 @@
-intention)
+intention#intention_n)
@@ -392 +392 @@
-and)
+and#and_conj1)
@@ -395 +395 @@
-desyre))))
+desyre#desire_n))))
@@ -397 +397 @@
-is)
+is#be_v)
@@ -400 +400 @@
-to)
+to#to_prep)
@@ -402 +402 @@
-see)
+see#see_v)
@@ -405 +405 @@
-you))
+you#you_pron))
@@ -408 +408 @@
-againe))
+againe#again_adv))
@@ -411 +411 @@
-before)
+before#before_adv)
@@ -414 +414 @@
-my)
+my#my_adj)
@@ -416 +416 @@
-jornye)
+jornye#journey_n)
@@ -419 +419 @@
-to)
+to#to_prep)
@@ -422 +422 @@
-Hatfeild)))))
+Hatfeild#NA)))))
@@ -427 +427 @@
-if)
+if#if_conj)
@@ -434 +434 @@
-I))
+I#i_pron))
@@ -436 +436 @@
-can)
+can#can_v1)
@@ -438 +438 @@
-find)
+find#find_v)
@@ -441 +441 @@
-any)
+any#any_adj)
@@ -443 +443 @@
-time)
+time#time_n)
@@ -446 +446 @@
-from)
+from#from_prep)
@@ -449 +449 @@
-my)
+my#my_adj)
@@ -451 +451 @@
-busynes)))))))
+busynes#business_n)))))))
@@ -456 +456 @@
-haveing)
+haveing#have_v)
@@ -458 +458 @@
-had)
+had#have_v)
@@ -462 +462 @@
-so)
+so#so_adv)
@@ -464 +464 @@
-little))
+little#little_adj))
@@ -466 +466 @@
-libertye)
+libertye#liberty_n)
@@ -471 +471 @@
-by)
+by#by_prep)
@@ -474 +474 @@
-reson)
+reson#reason_n1)
@@ -477 +477 @@
-of)
+of#of_prep)
@@ -480 +480 @@
-my)
+my#my_adj)
@@ -482 +482 @@
-lord)))))
+lord#lord_n)))))
@@ -485 +485 @@
-to)
+to#to_prep)
@@ -487 +487 @@
-discourse)
+discourse#discourse_v)
@@ -490 +490 @@
-with)
+with#with_prep)
@@ -493 +493 @@
-you)))
+you#you_pron)))
@@ -496 +496 @@
-of)
+of#of_prep)
@@ -499 +499 @@
-diverse)
+diverse#diverse_adj)
@@ -501 +501 @@
-particulars)
+particulars#particular_adj)
@@ -505 +505 @@
-which))
+which#which_adj))
@@ -511 +511 @@
-I))
+I#i_pron))
@@ -513 +513 @@
-resolved)
+resolved#resolve_v)
@@ -518 +518 @@
-to)
+to#to_prep)
@@ -520 +520 @@
-impart)
+impart#impart_v)
@@ -523 +523 @@
-to)
+to#to_prep)
@@ -526 +526 @@
-you))))))
+you#you_pron))))))
@@ -532 +532 @@
-som)
+som#some_pron)
@@ -535 +535 @@
-of)
+of#of_prep)
@@ -538 +538 @@
-mirth))))
+mirth#mirth_n))))
@@ -544 +544 @@
-som)
+som#some_pron)
@@ -547 +547 @@
-for)
+for#for_prep)
@@ -550 +550 @@
-meear)
+meear#mere_adj2)
@@ -552 +552 @@
-notice)))))
+notice#notice_n)))))
@@ -558 +558 @@
-som)
+som#some_pron)
@@ -561 +561 @@
-of)
+of#of_prep)
@@ -566 +566 @@
-though)
+though#though_adv)
@@ -569 +569 @@
-small)
+small#small_adj)
@@ -573 +573 @@
-busines))))))))))))
+busines#business_n))))))))))))
@@ -582 +582 @@
-My)
+My#my_adj)
@@ -584 +584 @@
-wife))
+wife#wife_n))
@@ -586 +586 @@
-tenders)
+tenders#tender_v1)
@@ -589 +589 @@
-her)
+her#her_pron2)
@@ -591 +591 @@
-dewtye)
+dewtye#duty_n)
@@ -594 +594 @@
-and)
+and#and_conj1)
@@ -597 +597 @@
-service))))
+service#service_n1))))
@@ -600 +600 @@
-to)
+to#to_prep)
@@ -603 +603 @@
-you)))
+you#you_pron)))
@@ -608 +608 @@
-desyring)
+desyring#desire_v)
@@ -611 +611 @@
-your)
+your#your_pron)
@@ -613 +613 @@
-excuse)
+excuse#excuse_n)
@@ -616 +616 @@
-for)
+for#for_prep)
@@ -620 +620 @@
-her))
+her#her_pron1))
@@ -622 +622 @@
-not)
+not#not_adv)
@@ -624 +624 @@
-waiteing)
+waiteing#wait_v1)
@@ -627 +627 @@
-on)
+on#on_prep)
@@ -630 +630 @@
-you)))))))
+you#you_pron)))))))
@@ -634 +634 @@
-haveing)
+haveing#have_v)
@@ -637 +637 @@
-no)
+no#no_adj)
@@ -639 +639 @@
-horses)
+horses#horse_n)
@@ -642 +642 @@
-of)
+of#of_prep)
@@ -645 +645 @@
-her)
+her#her_pron2)
@@ -647 +647 @@
-owne)))))
+owne#own_adj)))))
@@ -650 +650 @@
-and)
+and#and_conj1)
@@ -653 +653 @@
-being)
+being#be_v)
@@ -656 +656 @@
-so)
+so#so_adv)
@@ -658 +658 @@
-lately))
+lately#lately_adv))
@@ -660 +660 @@
-perplext)
+perplext#perplex_v)
@@ -663 +663 @@
-with)
+with#with_prep)
@@ -666 +666 @@
-those)
+those#those_pron)
@@ -669 +669 @@
-hyred))))
+hyred#hire_v))))
@@ -674 +674 @@
-as)
+as#as_adv)
@@ -685 +685 @@
-Tobie))
+Tobie#NA))
@@ -687 +687 @@
-knowes)))))))
+knowes#know_v)))))))
@@ -695 +695 @@
-Owr)
+Owr#our_pron)
@@ -697 +697 @@
-love)
+love#love_n1)
@@ -700 +700 @@
-to)
+to#to_prep)
@@ -703 +703 @@
-all)
+all#all_adj)
@@ -705 +705 @@
-our)
+our#our_pron)
@@ -707 +707 @@
-freinds)))
+freinds#friend_n)))
@@ -715 +715 @@
-and)
+and#and_conj1)
@@ -718 +718 @@
-so))
+so#so_adv))
@@ -721 +721 @@
-I))
+I#i_pron))
@@ -723 +723 @@
-rest)
+rest#rest_v2)
@@ -726 +726 @@
-Your)
+Your#your_pron)
@@ -729 +729 @@
-most)
+most#most_adj)
@@ -731 +731 @@
-obedient)
+obedient#obedient_adj)
@@ -734 +734 @@
-and)
+and#and_conj1)
@@ -737 +737 @@
-loving))))
+loving#loving_adj))))
@@ -739 +739 @@
-sonn)
+sonn#son_n1)
@@ -742 +742 @@
-Thomas)
+Thomas#NA)
@@ -744 +744 @@
-Barrington))))
+Barrington#NA))))
@@ -753 +753 @@
-To)
+To#to_prep)
@@ -756 +756 @@
-my)
+my#my_adj)
@@ -759 +759 @@
-most)
+most#most_adj)
@@ -761 +761 @@
-honored))
+honored#honoured_adj))
@@ -763 +763 @@
-mother)
+mother#mother_n1)
@@ -766 +766 @@
-the)
+the#the_adj)
@@ -768 +768 @@
-Lady)
+Lady#lady_n)
@@ -771 +771 @@
-Barrington)))))
+Barrington#NA)))))
@@ -774 +774 @@
-at)
+at#at_prep)
@@ -777 +777 @@
-Harrow)
+Harrow#NA)
@@ -779 +779 @@
-Hill))))
+Hill#NA))))
@@ -792 +792 @@
-Madame))
+Madame#madam_n))
@@ -795 +795 @@
-This)
+This#this_pron)
@@ -797 +797 @@
-toune))
+toune#town_n))
@@ -799 +799 @@
-is)
+is#be_v)
@@ -802 +802 @@
-so)
+so#so_adv)
@@ -804 +804 @@
-barren)
+barren#barren_adj)
@@ -807 +807 @@
-of)
+of#of_prep)
@@ -810 +810 @@
-newes)))
+newes#news_n)))
@@ -815 +815 @@
-as)
+as#as_adv)
@@ -818 +818 @@
-if)
+if#if_conj)
@@ -825 +825 @@
-the)
+the#the_adj)
@@ -827 +827 @@
-frost))
+frost#frost_n))
@@ -829 +829 @@
-had)
+had#have_v)
@@ -833 +833 @@
-foreclosed)
+foreclosed#foreclose_v)
@@ -836 +836 @@
-the)
+the#the_adj)
@@ -838 +838 @@
-streame)))))))
+streame#stream_n)))))))
@@ -847 +847 @@
-The)
+The#the_adj)
@@ -849 +849 @@
-only)
+only#only_adj)
@@ -851 +851 @@
-intelligence)
+intelligence#intelligence_n)
@@ -862 +862 @@
-I))
+I#i_pron))
@@ -864 +864 @@
-can)
+can#can_v1)
@@ -866 +866 @@
-derive)
+derive#derive_v)
@@ -869 +869 @@
-to)
+to#to_prep)
@@ -872 +872 @@
-yow))))))
+yow#you_pron))))))
@@ -874 +874 @@
-is)
+is#be_v)
@@ -882 +882 @@
-the)
+the#the_adj)
@@ -884 +884 @@
-French)
+French#french_adj)
@@ -886 +886 @@
-king))
+king#king_n))
@@ -888 +888 @@
-hath)
+hath#have_v)
@@ -890 +890 @@
-advaunced)
+advaunced#advance_v)
@@ -893 +893 @@
-his)
+his#his_adj)
@@ -895 +895 @@
-foote)
+foote#foot_n)
@@ -897 +897 @@
-forces))
+forces#force_n1))
@@ -900 +900 @@
-to)
+to#to_prep)
@@ -903 +903 @@
-Piamont))))
+Piamont#NA))))
@@ -906 +906 @@
-and)
+and#and_conj1)
@@ -910 +910 @@
-the)
+the#the_adj)
@@ -912 +912 @@
-cardinall))
+cardinall#cardinal_n))
@@ -914 +914 @@
-demaunds)
+demaunds#demand_v)
@@ -917 +917 @@
-high)
+high#high_adj)
@@ -919 +919 @@
-conditions))
+conditions#condition_n))
@@ -922 +922 @@
-in)
+in#in_prep)
@@ -925 +925 @@
-his)
+his#his_adj)
@@ -927 +927 @@
-behalfe)))
+behalfe#behalf_n)))
@@ -930 +930 @@
-thare))))))
+thare#there_adv))))))
@@ -939 +939 @@
-His)
+His#his_adj)
@@ -941 +941 @@
-naval)
+naval#naval_adj)
@@ -943 +943 @@
-power))
+power#power_n1))
@@ -945 +945 @@
-is)
+is#be_v)
@@ -948 +948 @@
-in)
+in#in_prep)
@@ -951 +951 @@
-dayly)
+dayly#daily_adj)
@@ -953 +953 @@
-preparation)))
+preparation#preparation_n)))
@@ -959 +959 @@
-he))
+he#he_pron))
@@ -961 +961 @@
-haveing)
+haveing#have_v)
@@ -963 +963 @@
-dessigned)
+dessigned#design_v)
@@ -966 +966 @@
-30,000)
+30,000#NA)
@@ -968 +968 @@
-crounes)
+crounes#crown_n)
@@ -971 +971 @@
-by)
+by#by_prep)
@@ -974 +974 @@
-the)
+the#the_adj)
@@ -976 +976 @@
-yeear))))
+yeear#year_n))))
@@ -979 +979 @@
-to)
+to#to_prep)
@@ -981 +981 @@
-fortyfye)
+fortyfye#fortify_v)
@@ -984 +984 @@
-him)
+him#him_pron)
@@ -986 +986 @@
-selfe))
+selfe#self_pron))
@@ -989 +989 @@
-at)
+at#at_prep)
@@ -992 +992 @@
-sea)))))
+sea#sea_n)))))
@@ -1000 +1000 @@
-and)
+and#and_conj1)
@@ -1003 +1003 @@
-towards)
+towards#towards_prep)
@@ -1006 +1006 @@
-the)
+the#the_adj)
@@ -1008 +1008 @@
-effecting)
+effecting#effecting_n)
@@ -1011 +1011 @@
-heearof))))
+heearof#hereof_adv))))
@@ -1015 +1015 @@
-hath)
+hath#have_v)
@@ -1017 +1017 @@
-procured)
+procured#procure_v)
@@ -1021 +1021 @@
-20)
+20#twenty_adj)
@@ -1023 +1023 @@
-or)
+or#or_conj2)
@@ -1025 +1025 @@
-30))
+30#thirty_adj))
@@ -1027 +1027 @@
-familyes)
+familyes#family_n)
@@ -1030 +1030 @@
-of)
+of#of_prep)
@@ -1033 +1033 @@
-the)
+the#the_adj)
@@ -1035 +1035 @@
-best)
+best#best_adj)
@@ -1037 +1037 @@
-shipp)
+shipp#ship_n1)
@@ -1039 +1039 @@
-wrights)
+wrights#wright_n1)
@@ -1044 +1044 @@
-that)
+that#that_conj)
@@ -1052 +1052 @@
-he))
+he#he_pron))
@@ -1054 +1054 @@
-can)
+can#can_v1)
@@ -1056 +1056 @@
-obtaine)
+obtaine#obtain_v)
@@ -1059 +1059 @@
-into)
+into#into_prep)
@@ -1062 +1062 @@
-his)
+his#his_adj)
@@ -1064 +1064 @@
-countrye))))))))
+countrye#country_n))))))))
@@ -1073 +1073 @@
-Besydes)
+Besydes#besides_adv)
@@ -1076 +1076 @@
-this)))
+this#this_pron)))
@@ -1082 +1082 @@
-he))
+he#he_pron))
@@ -1084 +1084 @@
-finding)
+finding#find_v)
@@ -1087 +1087 @@
-that)
+that#that_conj)
@@ -1091 +1091 @@
-the)
+the#the_adj)
@@ -1093 +1093 @@
-only)
+only#only_adj)
@@ -1095 +1095 @@
-way)
+way#way_n1)
@@ -1098 +1098 @@
-to)
+to#to_prep)
@@ -1100 +1100 @@
-improove)
+improove#improve_v2)
@@ -1103 +1103 @@
-his)
+his#his_adj)
@@ -1105 +1105 @@
-strength)
+strength#strength_n)
@@ -1108 +1108 @@
-for)
+for#for_prep)
@@ -1111 +1111 @@
-the)
+the#the_adj)
@@ -1113 +1113 @@
-seas))))))
+seas#sea_n))))))
@@ -1115 +1115 @@
-is)
+is#be_v)
@@ -1118 +1118 @@
-to)
+to#to_prep)
@@ -1120 +1120 @@
-take)
+take#take_v)
@@ -1122 +1122 @@
-of)
+of#off_adv)
@@ -1125 +1125 @@
-that)
+that#that_pron)
@@ -1127 +1127 @@
-contempt)
+contempt#contempt_n)
@@ -1131 +1131 @@
-which))
+which#which_adj))
@@ -1137 +1137 @@
-the)
+the#the_adj)
@@ -1139 +1139 @@
-nobless)
+nobless#noble_adj)
@@ -1142 +1142 @@
-of)
+of#of_prep)
@@ -1145 +1145 @@
-Fraunce))))
+Fraunce#NA))))
@@ -1147 +1147 @@
-held)
+held#hold_v)
@@ -1150 +1150 @@
-formerly))
+formerly#formerly_adv))
@@ -1153 +1153 @@
-all)
+all#all_adj)
@@ -1155 +1155 @@
-marchandizeing))
+marchandizeing#merchandising_n))
@@ -1158 +1158 @@
-in)
+in#in_prep)
@@ -1165 +1165 @@
-that)
+that#that_conj)
@@ -1169 +1169 @@
-so))
+so#so_adv))
@@ -1172 +1172 @@
-the)
+the#the_adj)
@@ -1174 +1174 @@
-publique)
+publique#public_adj)
@@ -1177 +1177 @@
-of)
+of#of_prep)
@@ -1180 +1180 @@
-the)
+the#the_adj)
@@ -1182 +1182 @@
-better)
+better#better_adj)
@@ -1184 +1184 @@
-condition))))
+condition#condition_n))))
@@ -1187 +1187 @@
-applying)
+applying#apply_v)
@@ -1190 +1190 @@
-them)
+them#them_pron)
@@ -1192 +1192 @@
-selves))
+selves#self_pron))
@@ -1195 +1195 @@
-to)
+to#to_prep)
@@ -1198 +1198 @@
-trafique))))
+trafique#traffic_n))))
@@ -1200 +1200 @@
-might)
+might#may_v1)
@@ -1203 +1203 @@
-necessaryly))
+necessaryly#necessarily_adv))
@@ -1205 +1205 @@
-add)
+add#add_v)
@@ -1208 +1208 @@
-increase)
+increase#increase_n)
@@ -1213 +1213 @@
-to)
+to#to_prep)
@@ -1216 +1216 @@
-his)
+his#his_adj)
@@ -1218 +1218 @@
-kingdom)))
+kingdom#kingdom_n)))
@@ -1221 +1221 @@
-in)
+in#in_prep)
@@ -1224 +1224 @@
-shipping)))))))))
+shipping#shipping_n1)))))))))
@@ -1231 +1231 @@
-he)
+he#he_pron)
@@ -1235 +1235 @@
-hath)
+hath#have_v)
@@ -1238 +1238 @@
-him)
+him#him_pron)
@@ -1240 +1240 @@
-selfe))
+selfe#self_pron))
@@ -1243 +1243 @@
-immediately))
+immediately#immediately_adv))
@@ -1245 +1245 @@
-professed)
+professed#profess_v)
@@ -1248 +1248 @@
-that)
+that#that_conj)
@@ -1252 +1252 @@
-he))
+he#he_pron))
@@ -1254 +1254 @@
-will)
+will#will_v1)
@@ -1256 +1256 @@
-trade)
+trade#trade_v)
@@ -1259 +1259 @@
-at)
+at#at_prep)
@@ -1262 +1262 @@
-sea)))
+sea#sea_n)))
@@ -1265 +1265 @@
-as)
+as#as_adv)
@@ -1268 +1268 @@
-a)
+a#a_adj2)
@@ -1270 +1270 @@
-marchant))))))
+marchant#merchant_n))))))
@@ -1276 +1276 @@
-and)
+and#and_conj1)
@@ -1280 +1280 @@
-hath)
+hath#have_v)
@@ -1282 +1282 @@
-innobled)
+innobled#ennoble_v)
@@ -1285 +1285 @@
-thareby))
+thareby#thereby_adv))
@@ -1288 +1288 @@
-that)
+that#that_pron)
@@ -1290 +1290 @@
-qualytye)))
+qualytye#quality_n)))
@@ -1296 +1296 @@
-and)
+and#and_conj1)
@@ -1300 +1300 @@
-will)
+will#will_v1)
@@ -1303 +1303 @@
-no)
+no#no_adj)
@@ -1305 +1305 @@
-question))
+question#question_n))
@@ -1308 +1308 @@
-in)
+in#in_prep)
@@ -1311 +1311 @@
-probabilytye)))
+probabilytye#probability_n)))
@@ -1313 +1313 @@
-be)
+be#be_v)
@@ -1317 +1317 @@
-much)
+much#much_adj)
@@ -1319 +1319 @@
-more))
+more#more_adj))
@@ -1321 +1321 @@
-powerfull)
+powerfull#powerful_adj)
@@ -1324 +1324 @@
-at)
+at#at_prep)
@@ -1327 +1327 @@
-sea))))
+sea#sea_n))))
@@ -1333 +1333 @@
-of)
+of#of_prep)
@@ -1336 +1336 @@
-which)))
+which#which_adj)))
@@ -1344 +1344 @@
-the)
+the#the_adj)
@@ -1346 +1346 @@
-French)
+French#french_adj)
@@ -1348 +1348 @@
-embassador))
+embassador#embassador_n))
@@ -1350 +1350 @@
-knowes))))
+knowes#know_v))))
@@ -1356 +1356 @@
-for)
+for#for_prep)
@@ -1359 +1359 @@
-he))
+he#he_pron))
@@ -1362 +1362 @@
-allready))
+allready#already_adj))
@@ -1364 +1364 @@
-professes)
+professes#profess_v)
@@ -1367 +1367 @@
-against)
+against#against_prep)
@@ -1371 +1371 @@
-owr)
+owr#our_pron)
@@ -1373 +1373 @@
-king's))
+king's#king_n))
@@ -1375 +1375 @@
-sole)
+sole#sole_adj)
@@ -1377 +1377 @@
-regalytye)
+regalytye#regality_n1)
@@ -1380 +1380 @@
-on)
+on#on_prep)
@@ -1383 +1383 @@
-the)
+the#the_adj)
@@ -1385 +1385 @@
-narrow)
+narrow#narrow_adj)
@@ -1387 +1387 @@
-seas)))))
+seas#sea_n)))))
@@ -1392 +1392 @@
-or)
+or#or_adv1)
@@ -1396 +1396 @@
-his)
+his#his_adj)
@@ -1398 +1398 @@
-master)
+master#master_n1)
@@ -1400 +1400 @@
-his))
+his#NA))
@@ -1403 +1403 @@
-rayseinge)
+rayseinge#raise_v1)
@@ -1406 +1406 @@
-sayle))
+sayle#sail_n1))
@@ -1409 +1409 @@
-to)
+to#to_prep)
@@ -1413 +1413 @@
-our)
+our#our_pron)
@@ -1415 +1415 @@
-king's))
+king's#king_n))
@@ -1417 +1417 @@
-shipps))))))
+shipps#ship_n1))))))
@@ -1425 +1425 @@
-But)
+But#but_prep)
@@ -1428 +1428 @@
-his)
+his#his_adj)
@@ -1430 +1430 @@
-indevors))
+indevors#endeavour_n))
@@ -1432 +1432 @@
-have)
+have#have_v)
@@ -1434 +1434 @@
-not)
+not#not_adv)
@@ -1437 +1437 @@
-yet))
+yet#yet_adv))
@@ -1439 +1439 @@
-raysed)
+raysed#raise_v1)
@@ -1442 +1442 @@
-his)
+his#his_adj)
@@ -1444 +1444 @@
-ambition))
+ambition#ambition_n))
@@ -1447 +1447 @@
-high)
+high#high_adj)
@@ -1450 +1450 @@
-enough)))
+enough#enough_adj)))
@@ -1455 +1455 @@
-as)
+as#as_adv)
@@ -1466 +1466 @@
-all)
+all#all_adj)
@@ -1468 +1468 @@
-men))
+men#man_n1))
@@ -1470 +1470 @@
-thinke))))
+thinke#think_v2))))
@@ -1475 +1475 @@
-to)
+to#to_prep)
@@ -1477 +1477 @@
-attempt)
+attempt#attempt_v)
@@ -1480 +1480 @@
-us))
+us#us_pron))
@@ -1483 +1483 @@
-with)
+with#with_prep)
@@ -1486 +1486 @@
-a)
+a#a_adj2)
@@ -1488 +1488 @@
-bodye)
+bodye#body_n)
@@ -1491 +1491 @@
-of)
+of#of_prep)
@@ -1494 +1494 @@
-a)
+a#a_adj2)
@@ -1496 +1496 @@
-navye))))))
+navye#navy_n))))))
@@ -1505 +1505 @@
-yet))
+yet#yet_adv))
@@ -1508 +1508 @@
-God))
+God#NA))
@@ -1510 +1510 @@
-open)
+open#open_v)
@@ -1513 +1513 @@
-the)
+the#the_adj)
@@ -1515 +1515 @@
-eyes)
+eyes#eye_n1)
@@ -1518 +1518 @@
-of)
+of#of_prep)
@@ -1521 +1521 @@
-state))))
+state#state_n))))
@@ -1524 +1524 @@
-to)
+to#to_prep)
@@ -1526 +1526 @@
-foresee)
+foresee#foresee_v)
@@ -1529 +1529 @@
-the)
+the#the_adj)
@@ -1531 +1531 @@
-perill)
+perill#peril_n)
@@ -1534 +1534 @@
-of)
+of#of_prep)
@@ -1537 +1537 @@
-this)
+this#this_pron)
@@ -1539 +1539 @@
-groeing)
+groeing#growing_adj)
@@ -1541 +1541 @@
-evell)))))
+evell#evil_adj)))))
@@ -1549 +1549 @@
-And)
+And#and_conj1)
@@ -1552 +1552 @@
-so))
+so#so_adv))
@@ -1555 +1555 @@
-with)
+with#with_prep)
@@ -1559 +1559 @@
-my))
+my#my_adj))
@@ -1561 +1561 @@
-humble)
+humble#humble_adj)
@@ -1563 +1563 @@
-dewtye)
+dewtye#duty_n)
@@ -1566 +1566 @@
-and)
+and#and_conj1)
@@ -1570 +1570 @@
-my)
+my#my_adj)
@@ -1572 +1572 @@
-wive's))))))
+wive's#wife_n))))))
@@ -1575 +1575 @@
-I))
+I#i_pron))
@@ -1577 +1577 @@
-rest)
+rest#rest_v2)
@@ -1580 +1580 @@
-Your)
+Your#your_pron)
@@ -1583 +1583 @@
-most)
+most#most_adj)
@@ -1585 +1585 @@
-obedient)
+obedient#obedient_adj)
@@ -1588 +1588 @@
-and)
+and#and_conj1)
@@ -1591 +1591 @@
-loving))))
+loving#loving_adj))))
@@ -1593 +1593 @@
-sonn)
+sonn#son_n1)
@@ -1596 +1596 @@
-Thomas)
+Thomas#NA)
@@ -1598 +1598 @@
-Barrington))))
+Barrington#NA))))
@@ -1607 +1607 @@
-Madame))
+Madame#madam_n))
@@ -1612 +1612 @@
-my)
+my#my_adj)
@@ -1614 +1614 @@
-wife))
+wife#wife_n))
@@ -1616 +1616 @@
-is)
+is#be_v)
@@ -1619 +1619 @@
-so)
+so#so_adv)
@@ -1621 +1621 @@
-unwilling)
+unwilling#unwilling_adj)
@@ -1624 +1624 @@
-and)
+and#and_conj1)
@@ -1627 +1627 @@
-carefull)))
+carefull#careful_adj)))
@@ -1630 +1630 @@
-not)
+not#not_adv)
@@ -1632 +1632 @@
-to)
+to#to_prep)
@@ -1634 +1634 @@
-intermedle)
+intermedle#intermeddle_v)
@@ -1637 +1637 @@
-with)
+with#with_prep)
@@ -1640 +1640 @@
-or)
+or#or_conj2)
@@ -1642 +1642 @@
-dispose)
+dispose#dispose_v)
@@ -1644 +1644 @@
-of))
+of#of_prep))
@@ -1649 +1649 @@
-what))
+what#what_pron))
@@ -1656 +1656 @@
-is)
+is#be_v)
@@ -1659 +1659 @@
-yours)))))))
+yours#yours_pron)))))))
@@ -1662 +1662 @@
-that)
+that#that_conj)
@@ -1666 +1666 @@
-she))
+she#she_pron))
@@ -1668 +1668 @@
-will)
+will#will_v1)
@@ -1670 +1670 @@
-not)
+not#not_adv)
@@ -1672 +1672 @@
-be)
+be#be_v)
@@ -1674 +1674 @@
-perswaded)
+perswaded#persuade_v)
@@ -1677 +1677 @@
-to)
+to#to_prep)
@@ -1679 +1679 @@
-make)
+make#make_v1)
@@ -1683 +1683 @@
-your)
+your#your_pron)
@@ -1685 +1685 @@
-old)
+old#old_adj)
@@ -1687 +1687 @@
-peuter))
+peuter#pewter_n))
@@ -1690 +1690 @@
-new)))
+new#new_adj)))
@@ -1693 +1693 @@
-with_oute)
+with_oute#without_adv)
@@ -1696 +1696 @@
-your)
+your#your_pron)
@@ -1698 +1698 @@
-consent)
+consent#consent_n)
@@ -1704 +1704 @@
-which))
+which#which_adj))
@@ -1712 +1712 @@
-being)
+being#be_v)
@@ -1714 +1714 @@
-graunted))
+graunted#grant_v))
@@ -1717 +1717 @@
-it))
+it#it_pron))
@@ -1719 +1719 @@
-shall)
+shall#shall_v)
@@ -1721 +1721 @@
-be)
+be#be_v)
@@ -1724 +1724 @@
-the)
+the#the_adj)
@@ -1726 +1726 @@
-same)
+same#same_adj)
@@ -1729 +1729 @@
-in)
+in#in_prep)
@@ -1732 +1732 @@
-nomber)
+nomber#number_n)
@@ -1734 +1734 @@
-and)
+and#and_conj1)
@@ -1736 +1736 @@
-waight))))))))))))
+waight#weight_n1))))))))))))
@@ -1745 +1745 @@
-the)
+the#the_adj)
@@ -1747 +1747 @@
-charge))
+charge#charge_n))
@@ -1749 +1749 @@
-will)
+will#will_v1)
@@ -1751 +1751 @@
-be)
+be#be_v)
@@ -1754 +1754 @@
-myne))
+myne#mine_adj))
@@ -1757 +1757 @@
-as)
+as#as_adv)
@@ -1759 +1759 @@
-well)
+well#well_adv)
@@ -1762 +1762 @@
-as)
+as#as_adv)
@@ -1765 +1765 @@
-the)
+the#the_adj)
@@ -1767 +1767 @@
-comodiousnes))))
+comodiousnes#commodiousness_n))))
@@ -1772 +1772 @@
-now)
+now#now_adv)
@@ -1774 +1774 @@
-especially)
+especially#especially_adv)
@@ -1777 +1777 @@
-when)
+when#when_adv)
@@ -1784 +1784 @@
-we))
+we#we_pron))
@@ -1786 +1786 @@
-expect)
+expect#expect_v)
@@ -1789 +1789 @@
-som)
+som#some_pron)
@@ -1791 +1791 @@
-freinds)
+freinds#friend_n)
@@ -1794 +1794 @@
-of)
+of#of_prep)
@@ -1797 +1797 @@
-worth))))
+worth#worth_n1))))
@@ -1800 +1800 @@
-before)
+before#before_adv)
@@ -1803 +1803 @@
-Easter)))))))
+Easter#NA)))))))
@@ -1812 +1812 @@
-I))
+I#i_pron))
@@ -1814 +1814 @@
-desyre)
+desyre#desire_v)
@@ -1817 +1817 @@
-your)
+your#your_pron)
@@ -1819 +1819 @@
-answar)
+answar#answer_n)
@@ -1822 +1822 @@
-to)
+to#to_prep)
@@ -1825 +1825 @@
-this)
+this#this_pron)
@@ -1827 +1827 @@
-resonable)
+resonable#reasonable_adj)
@@ -1829 +1829 @@
-request))))
+request#request_n1))))
@@ -1832 +1832 @@
-for)
+for#for_prep)
@@ -1836 +1836 @@
-my)
+my#my_adj)
@@ -1838 +1838 @@
-wive's))
+wive's#wife_n))
@@ -1840 +1840 @@
-satisfaction)))
+satisfaction#satisfaction_n)))
@@ -1849 +1849 @@
-Theise))
+Theise#these_pron))
@@ -1851 +1851 @@
-be)
+be#be_v)
@@ -1853 +1853 @@
-delivered)
+delivered#deliver_v1)
@@ -1856 +1856 @@
-to)
+to#to_prep)
@@ -1859 +1859 @@
-the)
+the#the_adj)
@@ -1861 +1861 @@
-honourable)
+honourable#honourable_adj)
@@ -1864 +1864 @@
-my)
+my#my_adj)
@@ -1867 +1867 @@
-verye)
+verye#very_adj)
@@ -1869 +1869 @@
-good))
+good#good_adj))
@@ -1871 +1871 @@
-mother)
+mother#mother_n1)
@@ -1874 +1874 @@
-the)
+the#the_adj)
@@ -1876 +1876 @@
-Lady)
+Lady#lady_n)
@@ -1879 +1879 @@
-Johan)
+Johan#NA)
@@ -1881 +1881 @@
-Barrington))))))
+Barrington#NA))))))
@@ -1884 +1884 @@
-at)
+at#at_prep)
@@ -1887 +1887 @@
-Harrow))))
+Harrow#NA))))
