--- turnerherb-1562-e1-p2.psd.vert	2025-09-10 13:51:18
+++ turnerherb-1562-e1-p2.lem.vert	2025-09-10 13:51:26
@@ -15 +15 @@
-Of)
+Of#of_prep)
@@ -19 +19 @@
-the)
+the#the_adj)
@@ -21 +21 @@
-walnut))
+walnut#walnut_n1))
@@ -24 +24 @@
-and)
+and#and_conj1)
@@ -27 +27 @@
-the)
+the#the_adj)
@@ -29 +29 @@
-walnut)
+walnut#walnut_n1)
@@ -31 +31 @@
-tree))))
+tree#tree_n))))
@@ -43 +43 @@
-Nux)
+Nux#NA)
@@ -46 +46 @@
-wythout)
+wythout#without_adv)
@@ -49 +49 @@
-any)
+any#any_adj)
@@ -51 +51 @@
-farther)
+farther#farther_adv)
@@ -53 +53 @@
-addition))))
+addition#addition_n))))
@@ -55 +55 @@
-is)
+is#be_v)
@@ -57 +57 @@
-called)
+called#call_v)
@@ -64 +64 @@
-a)
+a#a_adj2)
@@ -66 +66 @@
-Walnut))
+Walnut#walnut_n1))
@@ -69 +69 @@
-or)
+or#or_conj2)
@@ -72 +72 @@
-a)
+a#a_adj2)
@@ -74 +74 @@
-walnut)
+walnut#walnut_n1)
@@ -76 +76 @@
-tree)))))
+tree#tree_n)))))
@@ -86 +86 @@
-It))
+It#it_pron))
@@ -88 +88 @@
-is)
+is#be_v)
@@ -90 +90 @@
-called)
+called#call_v)
@@ -92 +92 @@
-also)
+also#also_adv)
@@ -99 +99 @@
-Iuglans))
+Iuglans#NA))
@@ -104 +104 @@
-nux)
+nux#NA)
@@ -106 +106 @@
-persica))
+persica#NA))
@@ -111 +111 @@
-glans)
+glans#NA)
@@ -113 +113 @@
-iouis))
+iouis#NA))
@@ -118 +118 @@
-nux)
+nux#NA)
@@ -120 +120 @@
-bisilica))
+bisilica#NA))
@@ -125 +125 @@
-&)
+&#and_conj1)
@@ -128 +128 @@
-nux)
+nux#NA)
@@ -130 +130 @@
-regia))))))
+regia#NA))))))
@@ -137 +137 @@
-in)
+in#in_prep)
@@ -140 +140 @@
-Greek)))
+Greek#NA)))
@@ -147 +147 @@
-Carya)
+Carya#NA)
@@ -149 +149 @@
-basilica))))))
+basilica#NA))))))
@@ -156 +156 @@
-in)
+in#in_prep)
@@ -159 +159 @@
-Dutch)))
+Dutch#NA)))
@@ -166 +166 @@
-Eyn)
+Eyn#NA)
@@ -168 +168 @@
-nusz)
+nusz#NA)
@@ -170 +170 @@
-baum)
+baum#NA)
@@ -174 +174 @@
-oder)
+oder#NA)
@@ -176 +176 @@
-ein)
+ein#NA)
@@ -178 +178 @@
-Welshnusz))))))
+Welshnusz#NA))))))
@@ -185 +185 @@
-in)
+in#in_prep)
@@ -188 +188 @@
-French)))
+French#NA)))
@@ -195 +195 @@
-Ung)
+Ung#NA)
@@ -197 +197 @@
-noier))))))
+noier#NA))))))
@@ -207 +207 @@
-The)
+The#the_adj)
@@ -209 +209 @@
-walnut))
+walnut#walnut_n1))
@@ -212 +212 @@
-and)
+and#and_conj1)
@@ -215 +215 @@
-the)
+the#the_adj)
@@ -217 +217 @@
-walnut)
+walnut#walnut_n1)
@@ -219 +219 @@
-trees))))
+trees#tree_n))))
@@ -221 +221 @@
-are)
+are#be_v)
@@ -225 +225 @@
-so)
+so#so_adv)
@@ -227 +227 @@
-well))
+well#well_adv))
@@ -229 +229 @@
-knowen)
+knowen#known_adj)
@@ -234 +234 @@
-in)
+in#in_prep)
@@ -237 +237 @@
-all)
+all#all_adj)
@@ -239 +239 @@
-countries)))
+countries#country_n)))
@@ -242 +242 @@
-that)
+that#that_conj)
@@ -246 +246 @@
-I))
+I#i_pron))
@@ -248 +248 @@
-nede)
+nede#need_v2)
@@ -250 +250 @@
-not)
+not#not_adv)
@@ -253 +253 @@
-to)
+to#to_prep)
@@ -255 +255 @@
-describe))))
+describe#describe_v))))
@@ -261 +261 @@
-wherefore))
+wherefore#wherefore_adv))
@@ -269 +269 @@
-I))
+I#i_pron))
@@ -271 +271 @@
-entende)
+entende#intend_v)
@@ -275 +275 @@
-to)
+to#to_prep)
@@ -277 +277 @@
-leaue)
+leaue#leave_v1)
@@ -280 +280 @@
-the)
+the#the_adj)
@@ -282 +282 @@
-description)))
+description#description_n)))
@@ -285 +285 @@
-and)
+and#and_conj1)
@@ -288 +288 @@
-to)
+to#to_prep)
@@ -290 +290 @@
-go)
+go#go_v)
@@ -293 +293 @@
-to)
+to#to_prep)
@@ -296 +296 @@
-the)
+the#the_adj)
@@ -298 +298 @@
-properties)
+properties#property_n)
@@ -301 +301 @@
-of)
+of#of_prep)
@@ -304 +304 @@
-it))))))))))
+it#it_pron))))))))))
@@ -320 +320 @@
-The)
+The#the_adj)
@@ -322 +322 @@
-vertues)
+vertues#virtue_n)
@@ -325 +325 @@
-of)
+of#of_prep)
@@ -328 +328 @@
-the)
+the#the_adj)
@@ -330 +330 @@
-Vualnut)))
+Vualnut#walnut_n1)))
@@ -342 +342 @@
-Walnuttes))
+Walnuttes#walnut_n1))
@@ -344 +344 @@
-are)
+are#be_v)
@@ -348 +348 @@
-harde)
+harde#hard_adj)
@@ -351 +351 @@
-of)
+of#of_prep)
@@ -354 +354 @@
-digestion))))
+digestion#digestion_n))))
@@ -360 +360 @@
-not)
+not#not_adv)
@@ -362 +362 @@
-good)
+good#good_adj)
@@ -365 +365 @@
-for)
+for#for_prep)
@@ -368 +368 @@
-the)
+the#the_adj)
@@ -370 +370 @@
-stomake))))))
+stomake#stomach_n))))))
@@ -378 +378 @@
-and)
+and#and_conj1)
@@ -382 +382 @@
-inge~dre)
+inge~dre#engender_v)
@@ -385 +385 @@
-choler))
+choler#choler_n))
@@ -393 +393 @@
-and)
+and#and_conj1)
@@ -396 +396 @@
-they))
+they#they_pron))
@@ -398 +398 @@
-make)
+make#make_v1)
@@ -402 +402 @@
-y=e=)
+y=e=#the_adj)
@@ -404 +404 @@
-head))
+head#head_n))
@@ -406 +406 @@
-ache))
+ache#ache_v))
@@ -415 +415 @@
-They))
+They#they_pron))
@@ -417 +417 @@
-are)
+are#be_v)
@@ -420 +420 @@
-euell)
+euell#evil_adj)
@@ -423 +423 @@
-for)
+for#for_prep)
@@ -426 +426 @@
-them)
+them#them_pron)
@@ -431 +431 @@
-that)
+that#that_conj)
@@ -436 +436 @@
-haue)
+haue#have_v)
@@ -439 +439 @@
-the)
+the#the_adj)
@@ -441 +441 @@
-coughe)))))))
+coughe#cough_n)))))))
@@ -450 +450 @@
-They))
+They#they_pron))
@@ -452 +452 @@
-are)
+are#be_v)
@@ -455 +455 @@
-fit)
+fit#fit_adj)
@@ -458 +458 @@
-to)
+to#to_prep)
@@ -460 +460 @@
-be)
+be#be_v)
@@ -462 +462 @@
-taken)
+taken#take_v)
@@ -465 +465 @@
-fastinge))
+fastinge#fast_v2))
@@ -468 +468 @@
-of)
+of#of_prep)
@@ -471 +471 @@
-them)
+them#them_pron)
@@ -476 +476 @@
-that)
+that#that_conj)
@@ -481 +481 @@
-woulde)
+woulde#will_v1)
@@ -483 +483 @@
-vomite)))))))
+vomite#vomit_v)))))))
@@ -492 +492 @@
-If)
+If#if_conj)
@@ -500 +500 @@
-they))
+they#they_pron))
@@ -502 +502 @@
-be)
+be#be_v)
@@ -504 +504 @@
-taken)
+taken#take_v)
@@ -507 +507 @@
-afore)
+afore#afore_adv)
@@ -510 +510 @@
-hand)))
+hand#hand_n)))
@@ -513 +513 @@
-wyth)
+wyth#with_prep)
@@ -516 +516 @@
-rue)
+rue#rue_n2)
@@ -518 +518 @@
-and)
+and#and_conj1)
@@ -520 +520 @@
-figges))))
+figges#fig_n1))))
@@ -525 +525 @@
-and)
+and#and_conj1)
@@ -528 +528 @@
-also)
+also#also_adv)
@@ -531 +531 @@
-after)
+after#after_adv)
@@ -534 +534 @@
-meat)))
+meat#meat_n)))
@@ -537 +537 @@
-by)
+by#by_prep)
@@ -539 +539 @@
-&)
+&#and_conj1)
@@ -541 +541 @@
-by)))))))
+by#by_prep)))))))
@@ -546 +546 @@
-they))
+they#they_pron))
@@ -548 +548 @@
-wythstande)
+wythstande#withstand_v)
@@ -551 +551 @@
-poison))
+poison#poison_n))
@@ -560 +560 @@
-They))
+They#they_pron))
@@ -562 +562 @@
-do)
+do#do_v)
@@ -566 +566 @@
-no)
+no#no_adj)
@@ -568 +568 @@
-lesse)))
+lesse#less_adj)))
@@ -571 +571 @@
-if)
+if#if_conj)
@@ -578 +578 @@
-they))
+they#they_pron))
@@ -580 +580 @@
-be)
+be#be_v)
@@ -582 +582 @@
-eaten)
+eaten#eat_v)
@@ -585 +585 @@
-after)
+after#after_adv)
@@ -588 +588 @@
-that)
+that#that_conj)
@@ -592 +592 @@
-a)
+a#a_adj2)
@@ -594 +594 @@
-man))
+man#man_n1))
@@ -596 +596 @@
-hath)
+hath#have_v)
@@ -598 +598 @@
-dronke~)
+dronke~#drink_v1)
@@ -601 +601 @@
-poison))))))))
+poison#poison_n))))))))
@@ -610 +610 @@
-If)
+If#if_conj)
@@ -617 +617 @@
-they))
+they#they_pron))
@@ -619 +619 @@
-be)
+be#be_v)
@@ -621 +621 @@
-eate~)
+eate~#eat_v)
@@ -624 +624 @@
-in)
+in#in_prep)
@@ -627 +627 @@
-greate)
+greate#great_adj)
@@ -629 +629 @@
-plenty))))))
+plenty#plenty_n))))))
@@ -632 +632 @@
-they))
+they#they_pron))
@@ -634 +634 @@
-driue)
+driue#drive_v)
@@ -636 +636 @@
-out)
+out#out_adv)
@@ -639 +639 @@
-brode)
+brode#broad_adj1)
@@ -641 +641 @@
-wormes))
+wormes#worm_n))
@@ -650 +650 @@
-It))
+It#it_pron))
@@ -652 +652 @@
-is)
+is#be_v)
@@ -655 +655 @@
-good))
+good#good_adj))
@@ -658 +658 @@
-to)
+to#to_prep)
@@ -660 +660 @@
-lay)
+lay#lay_v1)
@@ -663 +663 @@
-them))
+them#them_pron))
@@ -665 +665 @@
-to)
+to#to_prep)
@@ -670 +670 @@
-wyth)
+wyth#with_prep)
@@ -674 +674 @@
-a)
+a#a_adj2)
@@ -676 +676 @@
-litle))
+litle#little_adj))
@@ -678 +678 @@
-hony)
+hony#honey_n)
@@ -681 +681 @@
-and)
+and#and_conj1)
@@ -684 +684 @@
-rue)))))
+rue#rue_n2)))))
@@ -688 +688 @@
-for)
+for#for_prep)
@@ -691 +691 @@
-the)
+the#the_adj)
@@ -693 +693 @@
-burninge)
+burninge#burning_adj)
@@ -695 +695 @@
-heate)
+heate#heat_n)
@@ -698 +698 @@
-or)
+or#or_conj2)
@@ -701 +701 @@
-inflammation)))
+inflammation#inflammation_n)))
@@ -704 +704 @@
-of)
+of#of_prep)
@@ -707 +707 @@
-the)
+the#the_adj)
@@ -709 +709 @@
-papes)))))
+papes#pap_n1)))))
@@ -715 +715 @@
-for)
+for#for_prep)
@@ -719 +719 @@
-impostemes))
+impostemes#imposthume_n))
@@ -724 +724 @@
-and)
+and#and_conj1)
@@ -727 +727 @@
-places)
+places#place_n.II-VII)
@@ -730 +730 @@
-out)
+out#out_adv)
@@ -732 +732 @@
-of)
+of#of_prep)
@@ -735 +735 @@
-ioynt))))))))))
+ioynt#joint_n))))))))))
@@ -744 +744 @@
-If)
+If#if_conj)
@@ -751 +751 @@
-they))
+they#they_pron))
@@ -753,3 +753,3 @@
-be)
-(VAN
-layde)
+be#be_v)
+(VAN
+layde#lay_v1)
@@ -757 +757 @@
-to)
+to#to_prep)
@@ -760 +760 @@
-wyth)
+wyth#with_prep)
@@ -764 +764 @@
-an)
+an#a_adj2)
@@ -766 +766 @@
-onion))
+onion#onion_n))
@@ -772 +772 @@
-salt)))
+salt#salt_n1)))
@@ -775 +775 @@
-and)
+and#and_conj1)
@@ -778 +778 @@
-hony))))))))
+hony#honey_n))))))))
@@ -783 +783 @@
-they))
+they#they_pron))
@@ -785 +785 @@
-are)
+are#be_v)
@@ -788 +788 @@
-good)
+good#good_adj)
@@ -791 +791 @@
-for)
+for#for_prep)
@@ -794 +794 @@
-the)
+the#the_adj)
@@ -796 +796 @@
-bytinges)
+bytinges#biting_n)
@@ -798 +798 @@
-both)
+both#NA)
@@ -801 +801 @@
-of)
+of#of_prep)
@@ -806 +806 @@
-men)
+men#man_n1)
@@ -808 +808 @@
-and)
+and#and_conj1)
@@ -810 +810 @@
-dogges))))))
+dogges#dog_n))))))
@@ -819 +819 @@
-If)
+If#if_conj)
@@ -827 +827 @@
-they))
+they#they_pron))
@@ -829 +829 @@
-be)
+be#be_v)
@@ -831 +831 @@
-burnt)
+burnt#burn_v1)
@@ -834 +834 @@
-wyth)
+wyth#with_prep)
@@ -837 +837 @@
-their)
+their#their_adj)
@@ -839 +839 @@
-vtter)
+vtter#utter_adj)
@@ -841 +841 @@
-huskes))))
+huskes#husk_n1))))
@@ -844 +844 @@
-and)
+and#and_conj1)
@@ -847 +847 @@
-laide)
+laide#lay_v1)
@@ -850 +850 @@
-vnto)
+vnto#unto_prep)
@@ -853 +853 @@
-the)
+the#the_adj)
@@ -855 +855 @@
-nauell))))))))
+nauell#navel_n))))))))
@@ -860 +860 @@
-they))
+they#they_pron))
@@ -862 +862 @@
-sta~che)
+sta~che#stanch_v)
@@ -865 +865 @@
-the)
+the#the_adj)
@@ -867 +867 @@
-gnawinge)
+gnawinge#gnawing_n)
@@ -870 +870 @@
-of)
+of#of_prep)
@@ -873 +873 @@
-y=e=)
+y=e=#the_adj)
@@ -875 +875 @@
-belly))))
+belly#belly_n))))
@@ -884 +884 @@
-The)
+The#the_adj)
@@ -886 +886 @@
-shell)
+shell#shell_n)
@@ -890 +890 @@
-burnt)
+burnt#burn_v1)
@@ -892 +892 @@
-and)
+and#and_conj1)
@@ -894 +894 @@
-broken))
+broken#break_v))
@@ -897 +897 @@
-in)
+in#in_prep)
@@ -900 +900 @@
-oyle)
+oyle#oil_n1)
@@ -902 +902 @@
-and)
+and#and_conj1)
@@ -904 +904 @@
-wine)))))
+wine#wine_n1)))))
@@ -908 +908 @@
-is)
+is#be_v)
@@ -911 +911 @@
-good)
+good#good_adj)
@@ -917 +917 @@
-to)
+to#to_prep)
@@ -919 +919 @@
-anoynt)
+anoynt#anoint_v)
@@ -922 +922 @@
-childers)
+childers#child_n)
@@ -924 +924 @@
-heades))
+heades#head_n))
@@ -927 +927 @@
-wythall)
+wythall#withal_adv)
@@ -932 +932 @@
-to)
+to#to_prep)
@@ -934 +934 @@
-make)
+make#make_v1)
@@ -938 +938 @@
-the)
+the#the_adj)
@@ -940 +940 @@
-here))
+here#hair_n))
@@ -942 +942 @@
-grow))))))
+grow#grow_v))))))
@@ -951 +951 @@
-the)
+the#the_adj)
@@ -953 +953 @@
-same))
+same#same_adj))
@@ -955 +955 @@
-is)
+is#be_v)
@@ -957 +957 @@
-also)
+also#also_adv)
@@ -960 +960 @@
-good)
+good#good_adj)
@@ -968 +968 @@
-to)
+to#to_prep)
@@ -970 +970 @@
-fill)
+fill#fill_v)
@@ -972 +972 @@
-vp)
+vp#up_adv1)
@@ -975 +975 @@
-the)
+the#the_adj)
@@ -977 +977 @@
-bare)
+bare#bare_adj)
@@ -979 +979 @@
-places)
+places#place_n.II-VII)
@@ -982 +982 @@
-of)
+of#of_prep)
@@ -985 +985 @@
-scalled)
+scalled#scalled_adj)
@@ -987 +987 @@
-heades)))))))
+heades#head_n)))))))
@@ -996 +996 @@
-The)
+The#the_adj)
@@ -998 +998 @@
-kernels)
+kernels#kernel_n1)
@@ -1001 +1001 @@
-burned)))
+burned#burn_v1)))
@@ -1006 +1006 @@
-if)
+if#if_conj)
@@ -1014 +1014 @@
-they))
+they#they_pron))
@@ -1016 +1016 @@
-be)
+be#be_v)
@@ -1018 +1018 @@
-broken))
+broken#break_v))
@@ -1021 +1021 @@
-and)
+and#and_conj1)
@@ -1024 +1024 @@
-laide)
+laide#lay_v1)
@@ -1026 +1026 @@
-to)
+to#to_prep)
@@ -1029 +1029 @@
-wyth)
+wyth#with_prep)
@@ -1032 +1032 @@
-wine))))))))
+wine#wine_n1))))))))
@@ -1037 +1037 @@
-they))
+they#they_pron))
@@ -1039 +1039 @@
-will)
+will#will_v1)
@@ -1041 +1041 @@
-stay)
+stay#stay_v1)
@@ -1044 +1044 @@
-the)
+the#the_adj)
@@ -1046 +1046 @@
-blody)
+blody#bloody_adj)
@@ -1048 +1048 @@
-yssew)
+yssew#issue_n)
@@ -1053 +1053 @@
-that)
+that#that_conj)
@@ -1059 +1059 @@
-som)
+som#some_pron)
@@ -1061 +1061 @@
-tyme))
+tyme#time_n))
@@ -1064 +1064 @@
-wemen))
+wemen#woman_n))
@@ -1066 +1066 @@
-haue))))
+haue#have_v))))
@@ -1075 +1075 @@
-The)
+The#the_adj)
@@ -1077 +1077 @@
-same))
+same#same_adj))
@@ -1079 +1079 @@
-are)
+are#be_v)
@@ -1082 +1082 @@
-good)
+good#good_adj)
@@ -1090 +1090 @@
-to)
+to#to_prep)
@@ -1092 +1092 @@
-laye)
+laye#lay_v1)
@@ -1095 +1095 @@
-to)
+to#to_prep)
@@ -1099 +1099 @@
-olde)
+olde#old_adj)
@@ -1101 +1101 @@
-carbuncles))
+carbuncles#carbuncle_n))
@@ -1104 +1104 @@
-and)
+and#and_conj1)
@@ -1107 +1107 @@
-crepinge)
+crepinge#creeping_adj)
@@ -1109 +1109 @@
-sores)))
+sores#sore_n1)))
@@ -1115 +1115 @@
-tetters)))
+tetters#tetter_n)))
@@ -1118 +1118 @@
-and)
+and#and_conj1)
@@ -1121 +1121 @@
-impostemes)
+impostemes#imposthume_n)
@@ -1126 +1126 @@
-that)
+that#that_conj)
@@ -1131 +1131 @@
-are)
+are#be_v)
@@ -1134 +1134 @@
-in)
+in#in_prep)
@@ -1137 +1137 @@
-the)
+the#the_adj)
@@ -1139 +1139 @@
-corners)
+corners#corner_n1)
@@ -1142 +1142 @@
-of)
+of#of_prep)
@@ -1145 +1145 @@
-the)
+the#the_adj)
@@ -1147 +1147 @@
-eye))))))))))))))
+eye#eye_n1))))))))))))))
@@ -1156 +1156 @@
-The)
+The#the_adj)
@@ -1158 +1158 @@
-same)
+same#same_adj)
@@ -1162 +1162 @@
-chowed))
+chowed#chew_v))
@@ -1165 +1165 @@
-and)
+and#and_conj1)
@@ -1168 +1168 @@
-laide)
+laide#lay_v1)
@@ -1171 +1171 @@
-vpon)
+vpon#upon_prep)
@@ -1174 +1174 @@
-the)
+the#the_adj)
@@ -1176 +1176 @@
-head)))))))
+head#head_n)))))))
@@ -1180 +1180 @@
-are)
+are#be_v)
@@ -1183 +1183 @@
-a)
+a#a_adj2)
@@ -1185 +1185 @@
-present)
+present#present_adj)
@@ -1187 +1187 @@
-remedy)
+remedy#remedy_n)
@@ -1190 +1190 @@
-for)
+for#for_prep)
@@ -1193 +1193 @@
-the)
+the#the_adj)
@@ -1195 +1195 @@
-fallyng)
+fallyng#falling_n1)
@@ -1198 +1198 @@
-of)
+of#of_prep)
@@ -1201 +1201 @@
-the)
+the#the_adj)
@@ -1203 +1203 @@
-heare))))))
+heare#hair_n))))))
@@ -1212 +1212 @@
-A)
+A#a_adj2)
@@ -1214 +1214 @@
-man))
+man#man_n1))
@@ -1216 +1216 @@
-may)
+may#may_v1)
@@ -1218 +1218 @@
-make)
+make#make_v1)
@@ -1221 +1221 @@
-oyle))
+oyle#oil_n1))
@@ -1224 +1224 @@
-of)
+of#of_prep)
@@ -1227 +1227 @@
-the)
+the#the_adj)
@@ -1229 +1229 @@
-kirnels)
+kirnels#kernel_n1)
@@ -1232 +1232 @@
-of)
+of#of_prep)
@@ -1235 +1235 @@
-walnuttes)))))
+walnuttes#walnut_n1)))))
@@ -1240 +1240 @@
-if)
+if#if_conj)
@@ -1247 +1247 @@
-he))
+he#he_pron))
@@ -1249 +1249 @@
-will)
+will#will_v1)
@@ -1251 +1251 @@
-presse)
+presse#press_v1)
@@ -1254 +1254 @@
-them))
+them#them_pron))
@@ -1259 +1259 @@
-specially)
+specially#specially_adv)
@@ -1261 +1261 @@
-when)
+when#when_adv)
@@ -1268 +1268 @@
-they))
+they#they_pron))
@@ -1270 +1270 @@
-are)
+are#be_v)
@@ -1273 +1273 @@
-olde))))))))
+olde#old_adj))))))))
@@ -1282 +1282 @@
-They)
+They#they_pron)
@@ -1287 +1287 @@
-that)
+that#that_conj)
@@ -1292 +1292 @@
-are)
+are#be_v)
@@ -1295 +1295 @@
-grene)))))
+grene#green_adj)))))
@@ -1299 +1299 @@
-are)
+are#be_v)
@@ -1301 +1301 @@
-not)
+not#not_adv)
@@ -1304 +1304 @@
-so)
+so#so_adv)
@@ -1306 +1306 @@
-ill)
+ill#ill_adj)
@@ -1309 +1309 @@
-for)
+for#for_prep)
@@ -1312 +1312 @@
-the)
+the#the_adj)
@@ -1314 +1314 @@
-stomake)))
+stomake#stomach_n)))
@@ -1317 +1317 @@
-as)
+as#as_adv)
@@ -1328 +1328 @@
-the)
+the#the_adj)
@@ -1330 +1330 @@
-old)
+old#old_adj)
@@ -1332 +1332 @@
-nuttes))
+nuttes#nut_n1))
@@ -1334 +1334 @@
-be)))))
+be#be_v)))))
@@ -1339 +1339 @@
-because)
+because#because_adv)
@@ -1346 +1346 @@
-they))
+they#they_pron))
@@ -1348 +1348 @@
-are)
+are#be_v)
@@ -1351 +1351 @@
-sweter)))))
+sweter#sweet_adj)))))
@@ -1360 +1360 @@
-If)
+If#if_conj)
@@ -1367 +1367 @@
-they))
+they#they_pron))
@@ -1369 +1369 @@
-be)
+be#be_v)
@@ -1371 +1371 @@
-mixed)
+mixed#mix_v)
@@ -1374 +1374 @@
-with)
+with#with_prep)
@@ -1379 +1379 @@
-garleke))))))
+garleke#garlic_n))))))
@@ -1384 +1384 @@
-they))
+they#they_pron))
@@ -1386 +1386 @@
-take)
+take#take_v)
@@ -1389 +1389 @@
-away))
+away#away_adv))
@@ -1392 +1392 @@
-the)
+the#the_adj)
@@ -1394 +1394 @@
-sharpnes))
+sharpnes#sharpness_n))
@@ -1402 +1402 @@
-and)
+and#and_conj1)
@@ -1405 +1405 @@
-they))
+they#they_pron))
@@ -1407 +1407 @@
-do)
+do#do_v)
@@ -1410 +1410 @@
-if)
+if#if_conj)
@@ -1417 +1417 @@
-they))
+they#they_pron))
@@ -1419 +1419 @@
-be)
+be#be_v)
@@ -1421 +1421 @@
-layde)
+layde#lay_v1)
@@ -1423 +1423 @@
-to)
+to#to_prep)
@@ -1426 +1426 @@
-emplasterwise)))))
+emplasterwise#SEE_emplaster_n)))))
@@ -1430 +1430 @@
-driue)
+driue#drive_v)
@@ -1433 +1433 @@
-awaye))
+awaye#away_adv))
@@ -1436 +1436 @@
-the)
+the#the_adj)
@@ -1438 +1438 @@
-blew)
+blew#blue_adj)
@@ -1440 +1440 @@
-markes)
+markes#mark_n1)
@@ -1445 +1445 @@
-that)
+that#that_conj)
@@ -1450 +1450 @@
-come)
+come#come_v)
@@ -1453 +1453 @@
-of)
+of#of_prep)
@@ -1456 +1456 @@
-stripes))))))
+stripes#stripe_n2))))))
@@ -1465 +1465 @@
-The)
+The#the_adj)
@@ -1467 +1467 @@
-walnut)
+walnut#walnut_n1)
@@ -1469 +1469 @@
-tree))
+tree#tree_n))
@@ -1471 +1471 @@
-both)
+both#NA)
@@ -1474 +1474 @@
-in)
+in#in_prep)
@@ -1479 +1479 @@
-his)
+his#his_adj)
@@ -1481 +1481 @@
-leaues)
+leaues#leaf_n1)
@@ -1484 +1484 @@
-and)
+and#and_conj1)
@@ -1487 +1487 @@
-buddes)))))
+buddes#bud_n1)))))
@@ -1489 +1489 @@
-hath)
+hath#have_v)
@@ -1492 +1492 @@
-a)
+a#a_adj2)
@@ -1494 +1494 @@
-certayne)
+certayne#certain_adj)
@@ -1496 +1496 @@
-bindinge))
+bindinge#binding_n))
@@ -1504 +1504 @@
-but)
+but#but_prep)
@@ -1507 +1507 @@
-the)
+the#the_adj)
@@ -1509 +1509 @@
-bindinge))
+bindinge#binding_n))
@@ -1511 +1511 @@
-is)
+is#be_v)
@@ -1514 +1514 @@
-most)
+most#most_adj)
@@ -1516 +1516 @@
-euidently))
+euidently#evidently_adv))
@@ -1518 +1518 @@
-perceyued)
+perceyued#perceive_v)
@@ -1521 +1521 @@
-in)
+in#in_prep)
@@ -1524 +1524 @@
-the)
+the#the_adj)
@@ -1526 +1526 @@
-vtter)
+vtter#utter_adj)
@@ -1528 +1528 @@
-huskes)
+huskes#husk_n1)
@@ -1533 +1533 @@
-both)
+both#both_pron)
@@ -1535 +1535 @@
-moyst)
+moyst#moist_adj)
@@ -1537 +1537 @@
-and)
+and#and_conj1)
@@ -1539 +1539 @@
-drye))))
+drye#dry_adj))))
@@ -1547 +1547 @@
-and)
+and#and_conj1)
@@ -1550 +1550 @@
-therefore))
+therefore#therefore_adv))
@@ -1553 +1553 @@
-fullers))
+fullers#fuller_n1))
@@ -1555 +1555 @@
-dorse)
+dorse#NA)
@@ -1558 +1558 @@
-them))
+them#them_pron))
@@ -1566 +1566 @@
-But)
+But#but_prep)
@@ -1569 +1569 @@
-I))
+I#i_pron))
@@ -1574 +1574 @@
-saith)
+saith#say_v1)
@@ -1577 +1577 @@
-Galen)))
+Galen#NA)))
@@ -1581 +1581 @@
-presse)
+presse#press_v1)
@@ -1583 +1583 @@
-out)
+out#out_adv)
@@ -1586 +1586 @@
-the)
+the#the_adj)
@@ -1588 +1588 @@
-iuice)
+iuice#juice_n)
@@ -1591 +1591 @@
-of)
+of#of_prep)
@@ -1594 +1594 @@
-the)
+the#the_adj)
@@ -1596 +1596 @@
-huskes))))
+huskes#husk_n1))))
@@ -1599 +1599 @@
-as)
+as#as_adv)
@@ -1610 +1610 @@
-I))
+I#i_pron))
@@ -1612 +1612 @@
-do)
+do#do_v)
@@ -1617 +1617 @@
-out)
+out#out_adv)
@@ -1619 +1619 @@
-of)
+of#of_prep)
@@ -1622 +1622 @@
-the)
+the#the_adj)
@@ -1624 +1624 @@
-Mulberies)
+Mulberies#mulberry_n)
@@ -1627 +1627 @@
-&)
+&#and_conj1)
@@ -1630 +1630 @@
-bambleberes))))))))
+bambleberes#brambleberry_n))))))))
@@ -1640 +1640 @@
-and)
+and#and_conj1)
@@ -1644 +1644 @@
-set)
+set#set_v)
@@ -1647 +1647 @@
-it))
+it#it_pron))
@@ -1650 +1650 @@
-wyth)
+wyth#with_prep)
@@ -1653 +1653 @@
-sodden)
+sodden#sodden_adj)
@@ -1655 +1655 @@
-hony)))
+hony#honey_n)))
@@ -1663 +1663 @@
-and)
+and#and_conj1)
@@ -1667 +1667 @@
-vse)
+vse#use_v)
@@ -1670 +1670 @@
-it))
+it#it_pron))
@@ -1673 +1673 @@
-in)
+in#in_prep)
@@ -1676 +1676 @@
-the)
+the#the_adj)
@@ -1678 +1678 @@
-steade)
+steade#stead_n)
@@ -1681 +1681 @@
-of)
+of#of_prep)
@@ -1684 +1684 @@
-a)
+a#a_adj2)
@@ -1686 +1686 @@
-mouth_healinge)
+mouth_healinge#cNA)
@@ -1688 +1688 @@
-medicine)))))
+medicine#medicine_n1)))))
@@ -1693 +1693 @@
-as)
+as#as_adv)
@@ -1704 +1704 @@
-I))
+I#i_pron))
@@ -1706 +1706 @@
-vse)
+vse#use_v)
@@ -1709 +1709 @@
-the)
+the#the_adj)
@@ -1711 +1711 @@
-foresaide)
+foresaide#foresaid_adj)
@@ -1713 +1713 @@
-juice)
+juice#juice_n)
@@ -1716 +1716 @@
-of)
+of#of_prep)
@@ -1719 +1719 @@
-mulberes)
+mulberes#mulberry_n)
@@ -1721 +1721 @@
-and)
+and#and_conj1)
@@ -1723 +1723 @@
-brambleberes)))))))
+brambleberes#brambleberry_n)))))))
@@ -1732 +1732 @@
-The)
+The#the_adj)
@@ -1734 +1734 @@
-kirnell)
+kirnell#kernel_n1)
@@ -1737 +1737 @@
-of)
+of#of_prep)
@@ -1740 +1740 @@
-the)
+the#the_adj)
@@ -1742 +1742 @@
-nut))))
+nut#nut_n1))))
@@ -1747 +1747 @@
-when)
+when#when_adv)
@@ -1754 +1754 @@
-it))
+it#it_pron))
@@ -1756 +1756 @@
-is)
+is#be_v)
@@ -1758 +1758 @@
-wythered))))
+wythered#wither_v2))))
@@ -1762 +1762 @@
-is)
+is#be_v)
@@ -1766 +1766 @@
-of)
+of#of_prep)
@@ -1769 +1769 @@
-subtill)
+subtill#subtile_adj)
@@ -1771 +1771 @@
-partes)))
+partes#part_n)))
@@ -1774 +1774 @@
-and)
+and#and_conj1)
@@ -1777 +1777 @@
-a)
+a#a_adj2)
@@ -1779 +1779 @@
-dryinge)
+dryinge#drying_adj)
@@ -1781 +1781 @@
-medicine)
+medicine#medicine_n1)
@@ -1784 +1784 @@
-wyth_out)
+wyth_out#without_adv)
@@ -1787 +1787 @@
-any)
+any#any_adj)
@@ -1789 +1789 @@
-bytynge))))))
+bytynge#biting_n))))))
@@ -1802 +1802 @@
-Out)
+Out#out_adv)
@@ -1804 +1804 @@
-of)
+of#of_prep)
@@ -1807 +1807 @@
-Pliny))
+Pliny#NA))
@@ -1824 +1824 @@
-The)
+The#the_adj)
@@ -1826 +1826 @@
-Gretians))
+Gretians#NA))
@@ -1828 +1828 @@
-haue)
+haue#have_v)
@@ -1830 +1830 @@
-named)
+named#name_v)
@@ -1833 +1833 @@
-the)
+the#the_adj)
@@ -1835 +1835 @@
-walnut)
+walnut#walnut_n1)
@@ -1838 +1838 @@
-of)
+of#of_prep)
@@ -1841 +1841 @@
-that)
+that#that_pron)
@@ -1846 +1846 @@
-that)
+that#that_conj)
@@ -1850 +1850 @@
-it))
+it#it_pron))
@@ -1852 +1852 @@
-bringeth)
+bringeth#bring_v)
@@ -1855 +1855 @@
-the)
+the#the_adj)
@@ -1857 +1857 @@
-heade)
+heade#head_n)
@@ -1859 +1859 @@
-ache)))))))
+ache#ache_n1)))))))
@@ -1867 +1867 @@
-for)
+for#for_prep)
@@ -1870 +1870 @@
-the)
+the#the_adj)
@@ -1872 +1872 @@
-strengthe)
+strengthe#strength_n)
@@ -1875 +1875 @@
-of)
+of#of_prep)
@@ -1879 +1879 @@
-the)
+the#the_adj)
@@ -1881 +1881 @@
-trees))
+trees#tree_n))
@@ -1884 +1884 @@
-and)
+and#and_conj1)
@@ -1887 +1887 @@
-the)
+the#the_adj)
@@ -1889 +1889 @@
-leaues))))))
+leaues#leaf_n1))))))
@@ -1891 +1891 @@
-perche)
+perche#pierce_v)
@@ -1894 +1894 @@
-into)
+into#into_prep)
@@ -1897 +1897 @@
-the)
+the#the_adj)
@@ -1899 +1899 @@
-brayne)))
+brayne#brain_n)))
@@ -1908 +1908 @@
-if)
+if#if_conj)
@@ -1915 +1915 @@
-the)
+the#the_adj)
@@ -1917 +1917 @@
-nuttes))
+nuttes#nut_n1))
@@ -1919 +1919 @@
-be)
+be#be_v)
@@ -1921 +1921 @@
-taken)
+taken#take_v)
@@ -1924 +1924 @@
-a_fore)
+a_fore#afore_adv)
@@ -1927 +1927 @@
-hand))))))
+hand#hand_n))))))
@@ -1932 +1932 @@
-they))
+they#they_pron))
@@ -1935 +1935 @@
-breake)
+breake#break_v)
@@ -1937 +1937 @@
-and)
+and#and_conj1)
@@ -1939 +1939 @@
-quenche))
+quenche#quench_v))
@@ -1942 +1942 @@
-the)
+the#the_adj)
@@ -1944 +1944 @@
-myght)
+myght#might_n)
@@ -1947 +1947 @@
-of)
+of#of_prep)
@@ -1950 +1950 @@
-poyson))))
+poyson#poison_n))))
@@ -1959 +1959 @@
-They))
+They#they_pron))
@@ -1961 +1961 @@
-are)
+are#be_v)
@@ -1964 +1964 @@
-good)
+good#good_adj)
@@ -1967 +1967 @@
-to)
+to#to_prep)
@@ -1969 +1969 @@
-be)
+be#be_v)
@@ -1971 +1971 @@
-laide)
+laide#lay_v1)
@@ -1974 +1974 @@
-to)
+to#to_prep)
@@ -1977 +1977 @@
-the)
+the#the_adj)
@@ -1979 +1979 @@
-squynsie)))
+squynsie#squinsy_n)))
@@ -1982 +1982 @@
-wyth)
+wyth#with_prep)
@@ -1985 +1985 @@
-rue)
+rue#rue_n2)
@@ -1987 +1987 @@
-&)
+&#and_conj1)
@@ -1989 +1989 @@
-oyle)))))
+oyle#oil_n1)))))
@@ -1998 +1998 @@
-Cneius)
+Cneius#NA)
@@ -2000 +2000 @@
-Pompeius))
+Pompeius#NA))
@@ -2003 +2003 @@
-when)
+when#when_adv)
@@ -2010 +2010 @@
-he))
+he#he_pron))
@@ -2012 +2012 @@
-had)
+had#have_v)
@@ -2014 +2014 @@
-ouercomed)
+ouercomed#overcome_v)
@@ -2017 +2017 @@
-the)
+the#the_adj)
@@ -2019 +2019 @@
-myghty)
+myghty#mighty_adj)
@@ -2021 +2021 @@
-king)
+king#king_n)
@@ -2024 +2024 @@
-Mythrydites))))))
+Mythrydites#NA))))))
@@ -2029 +2029 @@
-he))
+he#he_pron))
@@ -2031 +2031 @@
-founde)
+founde#find_v)
@@ -2034 +2034 @@
-in)
+in#in_prep)
@@ -2037 +2037 @@
-his)
+his#his_adj)
@@ -2040 +2040 @@
-most)
+most#most_adj)
@@ -2042 +2042 @@
-secret))
+secret#secret_adj))
@@ -2044 +2044 @@
-treserhouse)))
+treserhouse#treasure-house_n)))
@@ -2047 +2047 @@
-in)
+in#in_prep)
@@ -2050 +2050 @@
-a)
+a#a_adj2)
@@ -2052 +2052 @@
-lytle)
+lytle#little_adj)
@@ -2054 +2054 @@
-boke)
+boke#book_n)
@@ -2057 +2057 @@
-by)
+by#by_prep)
@@ -2060 +2060 @@
-it)
+it#it_pron)
@@ -2062 +2062 @@
-selffe)))))
+selffe#self_pron)))))
@@ -2065 +2065 @@
-wrytte~)
+wrytte~#write_v)
@@ -2068 +2068 @@
-wyth)
+wyth#with_prep)
@@ -2071 +2071 @@
-his)
+his#his_adj)
@@ -2073 +2073 @@
-owne)
+owne#own_adj)
@@ -2075 +2075 @@
-ha~de))))
+ha~de#hand_n))))
@@ -2078 +2078 @@
-this)
+this#this_pron)
@@ -2082 +2082 @@
-preseruatiue)
+preseruatiue#preservative_adj)
@@ -2088 +2088 @@
-the)
+the#the_adj)
@@ -2090 +2090 @@
-co~position)
+co~position#composition_n)
@@ -2093 +2093 @@
-wherof)))
+wherof#whereof_adv)))
@@ -2100 +2100 @@
-is)
+is#be_v)
@@ -2103 +2103 @@
-this)))))
+this#this_pron)))))
@@ -2113 +2113 @@
-Take)
+Take#take_v)
@@ -2117 +2117 @@
-two)
+two#two_adj)
@@ -2119 +2119 @@
-drye)
+drye#dry_adj)
@@ -2121 +2121 @@
-Walnuttes))
+Walnuttes#walnut_n1))
@@ -2124 +2124 @@
-and)
+and#and_conj1)
@@ -2127 +2127 @@
-ij.)
+ij.#two_adj)
@@ -2129 +2129 @@
-figges)))
+figges#fig_n1)))
@@ -2132 +2132 @@
-and)
+and#and_conj1)
@@ -2135 +2135 @@
-xx.)
+xx.#twenty_adj)
@@ -2137 +2137 @@
-leaues)
+leaues#leaf_n1)
@@ -2140 +2140 @@
-of)
+of#of_prep)
@@ -2143 +2143 @@
-rue)))))))
+rue#rue_n2)))))))
@@ -2149 +2149 @@
-breake)
+breake#break_v)
@@ -2152 +2152 @@
-them))
+them#them_pron))
@@ -2155 +2155 @@
-together))))
+together#together_adv))))
@@ -2158 +2158 @@
-and)
+and#and_conj1)
@@ -2161 +2161 @@
-put)
+put#put_v)
@@ -2164 +2164 @@
-a)
+a#a_adj2)
@@ -2166 +2166 @@
-corne)
+corne#corn_n1)
@@ -2169 +2169 @@
-of)
+of#of_prep)
@@ -2172 +2172 @@
-salt))))
+salt#salt_n1))))
@@ -2175 +2175 @@
-tho)
+tho#though_adv)
@@ -2178 +2178 @@
-them))))))
+them#them_pron))))))
@@ -2183 +2183 @@
-and)
+and#and_conj1)
@@ -2187 +2187 @@
-if)
+if#if_conj)
@@ -2194 +2194 @@
-yow))
+yow#you_pron))
@@ -2196 +2196 @@
-eate)
+eate#eat_v)
@@ -2199 +2199 @@
-this)
+this#this_pron)
@@ -2201 +2201 @@
-medicine))
+medicine#medicine_n1))
@@ -2204 +2204 @@
-fastinge)))))
+fastinge#fast_v2)))))
@@ -2209 +2209 @@
-ther))
+ther#there_adv))
@@ -2211 +2211 @@
-shall)
+shall#shall_v)
@@ -2214 +2214 @@
-no)
+no#no_adj)
@@ -2216 +2216 @@
-poyson))
+poyson#poison_n))
@@ -2218 +2218 @@
-hurt)
+hurt#hurt_v)
@@ -2221 +2221 @@
-yow))
+yow#you_pron))
@@ -2224 +2224 @@
-that)
+that#that_pron)
@@ -2226 +2226 @@
-daye))))
+daye#day_n))))
@@ -2235 +2235 @@
-the)
+the#the_adj)
@@ -2237 +2237 @@
-kirnels)
+kirnels#kernel_n1)
@@ -2240 +2240 @@
-of)
+of#of_prep)
@@ -2243 +2243 @@
-the)
+the#the_adj)
@@ -2245 +2245 @@
-nuttes))))
+nuttes#nut_n1))))
@@ -2248 +2248 @@
-if)
+if#if_conj)
@@ -2255 +2255 @@
-they))
+they#they_pron))
@@ -2257 +2257 @@
-be)
+be#be_v)
@@ -2259 +2259 @@
-chowed)
+chowed#chew_v)
@@ -2262 +2262 @@
-of)
+of#of_prep)
@@ -2265 +2265 @@
-a)
+a#a_adj2)
@@ -2267 +2267 @@
-man)
+man#man_n1)
@@ -2270 +2270 @@
-fastinge)))))))
+fastinge#fast_v2)))))))
@@ -2274 +2274 @@
-is)
+is#be_v)
@@ -2279 +2279 @@
-a)
+a#a_adj2)
@@ -2281 +2281 @@
-present)
+present#present_adj)
@@ -2283 +2283 @@
-remedy))
+remedy#remedy_n))
@@ -2286 +2286 @@
-if)
+if#if_conj)
@@ -2293 +2293 @@
-they))
+they#they_pron))
@@ -2295 +2295 @@
-be)
+be#be_v)
@@ -2297 +2297 @@
-layde)
+layde#lay_v1)
@@ -2300 +2300 @@
-vnto)
+vnto#unto_prep)
@@ -2303 +2303 @@
-the)
+the#the_adj)
@@ -2305 +2305 @@
-bytinges)
+bytinges#biting_n)
@@ -2308 +2308 @@
-of)
+of#of_prep)
@@ -2311 +2311 @@
-a)
+a#a_adj2)
@@ -2313 +2313 @@
-mad)
+mad#mad_adj)
@@ -2315 +2315 @@
-dogge))))))))
+dogge#dog_n))))))))
@@ -2324 +2324 @@
-Som))
+Som#some_pron))
@@ -2326 +2326 @@
-vse)
+vse#use_v)
@@ -2329 +2329 @@
-to)
+to#to_prep)
@@ -2331 +2331 @@
-make)
+make#make_v1)
@@ -2334 +2334 @@
-succat))
+succat#succate_n))
@@ -2337 +2337 @@
-wyth)
+wyth#with_prep)
@@ -2340 +2340 @@
-hony)
+hony#honey_n)
@@ -2342 +2342 @@
-or)
+or#or_conj2)
@@ -2344 +2344 @@
-sugar)))
+sugar#sugar_n)))
@@ -2347 +2347 @@
-of)
+of#of_prep)
@@ -2350 +2350 @@
-the)
+the#the_adj)
@@ -2352 +2352 @@
-yonge)
+yonge#young_adj)
@@ -2354 +2354 @@
-nuttes)
+nuttes#nut_n1)
@@ -2360 +2360 @@
-whiche))
+whiche#which_adj))
@@ -2367 +2367 @@
-are)
+are#be_v)
@@ -2369 +2369 @@
-pulled)
+pulled#pull_v)
@@ -2374 +2374 @@
-of)
+of#of_prep)
@@ -2377 +2377 @@
-the)
+the#the_adj)
@@ -2379 +2379 @@
-tree)))
+tree#tree_n)))
@@ -2382 +2382 @@
-about)
+about#about_adv)
@@ -2384 +2384 @@
-midsomer)))))))
+midsomer#midsummer_n)))))))
@@ -2393 +2393 @@
-Tragus))
+Tragus#NA))
@@ -2395 +2395 @@
-wryteth)
+wryteth#write_v)
@@ -2398 +2398 @@
-that)
+that#that_conj)
@@ -2402 +2402 @@
-the)
+the#the_adj)
@@ -2404 +2404 @@
-water)
+water#water_n)
@@ -2408 +2408 @@
-whiche))
+whiche#which_adj))
@@ -2415 +2415 @@
-is)
+is#be_v)
@@ -2417 +2417 @@
-destilled)
+destilled#distil_v)
@@ -2420 +2420 @@
-out)
+out#out_adv)
@@ -2422 +2422 @@
-of)
+of#of_prep)
@@ -2425 +2425 @@
-the)
+the#the_adj)
@@ -2427 +2427 @@
-grene)
+grene#green_adj)
@@ -2429 +2429 @@
-nuttes)
+nuttes#nut_n1)
@@ -2432 +2432 @@
-gathered)
+gathered#gather_v)
@@ -2435 +2435 @@
-at)
+at#at_prep)
@@ -2438 +2438 @@
-mitsomer)))))))))
+mitsomer#midsummer_n)))))))))
@@ -2444 +2444 @@
-is)
+is#be_v)
@@ -2447 +2447 @@
-good)
+good#good_adj)
@@ -2451 +2451 @@
-for)
+for#for_prep)
@@ -2455 +2455 @@
-the)
+the#the_adj)
@@ -2457 +2457 @@
-inflammation)
+inflammation#inflammation_n)
@@ -2460 +2460 @@
-of)
+of#of_prep)
@@ -2463 +2463 @@
-the)
+the#the_adj)
@@ -2465 +2465 @@
-pappes))))
+pappes#pap_n1))))
@@ -2471 +2471 @@
-impostumes)))))
+impostumes#imposthume_n)))))
@@ -2476 +2476 @@
-and)
+and#and_conj1)
@@ -2479 +2479 @@
-for)
+for#for_prep)
@@ -2482 +2482 @@
-places)
+places#place_n.II-VII)
@@ -2485 +2485 @@
-out)
+out#out_adv)
@@ -2487 +2487 @@
-of)
+of#of_prep)
@@ -2490 +2490 @@
-ioynt))))))))))
+ioynt#joint_n))))))))))
@@ -2499 +2499 @@
-the)
+the#the_adj)
@@ -2501 +2501 @@
-oyle)
+oyle#oil_n1)
@@ -2506 +2506 @@
-that)
+that#that_conj)
@@ -2511 +2511 @@
-is)
+is#be_v)
@@ -2513 +2513 @@
-pressed)
+pressed#press_v1)
@@ -2516 +2516 @@
-out)
+out#out_adv)
@@ -2518 +2518 @@
-of)
+of#of_prep)
@@ -2521 +2521 @@
-the)
+the#the_adj)
@@ -2523 +2523 @@
-walnuttes))))))
+walnuttes#walnut_n1))))))
@@ -2528 +2528 @@
-saieth)
+saieth#say_v1)
@@ -2531 +2531 @@
-Tragus)))
+Tragus#NA)))
@@ -2535 +2535 @@
-is)
+is#be_v)
@@ -2538 +2538 @@
-good)
+good#good_adj)
@@ -2541 +2541 @@
-for)
+for#for_prep)
@@ -2544 +2544 @@
-the)
+the#the_adj)
@@ -2546 +2546 @@
-purposes)
+purposes#purpose_n)
@@ -2550 +2550 @@
-aboue))
+aboue#above_adv))
@@ -2552 +2552 @@
-rehersed)))))
+rehersed#rehearse_v)))))
