--- brightland-1711-1.psd.vert	2025-09-10 13:52:07
+++ brightland-1711-1.lem.vert	2025-09-10 13:51:57
@@ -13 +13 @@
-Reasons)
+Reasons#reason_n1)
@@ -16 +16 @@
-for)
+for#for_prep)
@@ -19 +19 @@
-an)
+an#a_adj2)
@@ -21 +21 @@
-English)
+English#english_adj)
@@ -23 +23 @@
-Education)))
+Education#education_n)))
@@ -36 +36 @@
-New)
+New#new_adj)
@@ -38 +38 @@
-Methods))
+Methods#method_n))
@@ -40 +40 @@
-being)
+being#be_v)
@@ -43 +43 @@
-always))
+always#always_adv))
@@ -46 +46 @@
-liable)
+liable#liable_adj)
@@ -49 +49 @@
-to)
+to#to_prep)
@@ -52 +52 @@
-Censure)))))
+Censure#censure_n)))))
@@ -57 +57 @@
-'t@))
+'t@#it_pron))
@@ -59 +59 @@
-@is)
+@is#be_v)
@@ -62 +62 @@
-requisite))
+requisite#requisite_adj))
@@ -65 +65 @@
-to)
+to#to_prep)
@@ -67 +67 @@
-prefix)
+prefix#prefix_v)
@@ -70 +70 @@
-some)
+some#some_pron)
@@ -72 +72 @@
-Reasons)
+Reasons#reason_n1)
@@ -76 +76 @@
-for)
+for#for_prep)
@@ -79 +79 @@
-this)
+this#this_pron)
@@ -81 +81 @@
-Proposal)
+Proposal#proposal_n)
@@ -84 +84 @@
-of)
+of#of_prep)
@@ -87 +87 @@
-an)
+an#a_adj2)
@@ -91 +91 @@
-English)
+English#english_adj)
@@ -93 +93 @@
-Education)))))
+Education#education_n)))))
@@ -100 +100 @@
-and)
+and#and_conj1)
@@ -103 +103 @@
-for)
+for#for_prep)
@@ -106 +106 @@
-having)
+having#have_v)
@@ -110 +110 @@
-the)
+the#the_adj)
@@ -112 +112 @@
-Youth)
+Youth#youth_n)
@@ -115 +115 @@
-of)
+of#of_prep)
@@ -120 +120 @@
-Great)
+Great#great_adj)
@@ -122 +122 @@
-Britain))))
+Britain#NA))))
@@ -126 +126 @@
-taught)
+taught#teach_v)
@@ -129 +129 @@
-the)
+the#the_adj)
@@ -133 +133 @@
-Grammar)
+Grammar#grammar_n)
@@ -138 +138 @@
-and)
+and#and_conj1)
@@ -143 +143 @@
-Rhetoric)))
+Rhetoric#rhetoric_n1)))
@@ -147 +147 @@
-&c.)
+&c.#et-cetera_n)
@@ -152 +152 @@
-of)
+of#of_prep)
@@ -155 +155 @@
-their)
+their#their_adj)
@@ -157 +157 @@
-own)
+own#own_adj)
@@ -159 +159 @@
-Mother-Tongue))))
+Mother-Tongue#mother-tongue_n))))
@@ -164 +164 @@
-before)
+before#before_adv)
@@ -171 +171 @@
-they))
+they#they_pron))
@@ -173 +173 @@
-learn)
+learn#learn_v)
@@ -177 +177 @@
-Latin))
+Latin#NA))
@@ -182 +182 @@
-or)
+or#or_conj2)
@@ -185 +185 @@
-any)
+any#any_adj)
@@ -187 +187 @@
-other)
+other#other_adj)
@@ -189 +189 @@
-Language))))))))))))))
+Language#language_n))))))))))))))
@@ -198 +198 @@
-First))
+First#first_adj))
@@ -201 +201 @@
-then))
+then#then_adv))
@@ -206 +206 @@
-Can)
+Can#can_v1)
@@ -209 +209 @@
-any)
+any#any_adj)
@@ -211 +211 @@
-thing))
+thing#thing_n1))
@@ -213 +213 @@
-be)
+be#be_v)
@@ -216 +216 @@
-more)
+more#more_adj)
@@ -218 +218 @@
-Absurd)
+Absurd#absurd_adj)
@@ -221 +221 @@
-and)
+and#and_conj1)
@@ -224 +224 @@
-Barbarous)))
+Barbarous#barbarous_adj)))
@@ -229 +229 @@
-than)
+than#than_conj)
@@ -232 +232 @@
-our)
+our#our_pron)
@@ -234 +234 @@
-present)
+present#present_adj)
@@ -236 +236 @@
-Method)
+Method#method_n)
@@ -239 +239 @@
-of)
+of#of_prep)
@@ -242 +242 @@
-Disciplining)
+Disciplining#discipline_v)
@@ -245 +245 @@
-Children))
+Children#child_n))
@@ -248 +248 @@
-in)
+in#in_prep)
@@ -251 +251 @@
-the)
+the#the_adj)
@@ -255 +255 @@
-Initiatory)
+Initiatory#initiatory_adj)
@@ -259 +259 @@
-part)
+part#part_n)
@@ -262 +262 @@
-of)
+of#of_prep)
@@ -265 +265 @@
-Literature)))))))
+Literature#literature_n)))))))
@@ -271 +271 @@
-To)
+To#to_prep)
@@ -273 +273 @@
-push)
+push#push_v)
@@ -276 +276 @@
-tender)
+tender#tender_adj)
@@ -278 +278 @@
-Wits))
+Wits#wit_n))
@@ -281 +281 @@
-into)
+into#into_prep)
@@ -284 +284 @@
-the)
+the#the_adj)
@@ -286 +286 @@
-intricate)
+intricate#intricate_adj)
@@ -288 +288 @@
-Mazes)
+Mazes#maze_n1)
@@ -291 +291 @@
-of)
+of#of_prep)
@@ -295 +295 @@
-Grammar))
+Grammar#grammar_n))
@@ -300 +300 @@
-and)
+and#and_conj1)
@@ -303 +303 @@
-a)
+a#a_adj2)
@@ -305 +305 @@
-Latin)
+Latin#latin_adj)
@@ -307 +307 @@
-Grammar))))))))
+Grammar#grammar_n))))))))
@@ -313 +313 @@
-To)
+To#to_prep)
@@ -315 +315 @@
-learn)
+learn#learn_v)
@@ -318 +318 @@
-an)
+an#a_adj2)
@@ -320 +320 @@
-Unknown)
+Unknown#unknown_adj)
@@ -324 +324 @@
-Art))
+Art#art_n))
@@ -329 +329 @@
-by)
+by#by_prep)
@@ -332 +332 @@
-an)
+an#a_adj2)
@@ -334 +334 @@
-Unknown)
+Unknown#unknown_adj)
@@ -338 +338 @@
-Language)))))
+Language#language_n)))))
@@ -346 +346 @@
-To)
+To#to_prep)
@@ -348 +348 @@
-carry)
+carry#carry_v)
@@ -351 +351 @@
-them))
+them#them_pron))
@@ -354 +354 @@
-a)
+a#a_adj2)
@@ -356 +356 @@
-Dark)
+Dark#dark_adj)
@@ -358 +358 @@
-Round-about)
+Round-about#roundabout_n)
@@ -360 +360 @@
-Way))))
+Way#way_n1))))
@@ -366 +366 @@
-to)
+to#to_prep)
@@ -368 +368 @@
-let)
+let#let_v1)
@@ -371 +371 @@
-them))
+them#them_pron))
@@ -373 +373 @@
-in)
+in#in_adv)
@@ -376 +376 @@
-at)
+at#at_prep)
@@ -379 +379 @@
-a)
+a#a_adj2)
@@ -381 +381 @@
-Back-door))))))))))
+Back-door#backdoor_n))))))))))
@@ -386 +386 @@
-Whereas)
+Whereas#whereas_adv)
@@ -393 +393 @@
-by)
+by#by_prep)
@@ -396 +396 @@
-Teaching)
+Teaching#teach_v)
@@ -399 +399 @@
-them))
+them#them_pron))
@@ -402 +402 @@
-First))
+First#first_adj))
@@ -405 +405 @@
-the)
+the#the_adj)
@@ -407 +407 @@
-Grammar)
+Grammar#grammar_n)
@@ -410 +410 @@
-of)
+of#of_prep)
@@ -413 +413 @@
-their)
+their#their_adj)
@@ -415 +415 @@
-Own)
+Own#own_adj)
@@ -417 +417 @@
-Mother-Tongue)
+Mother-Tongue#mother-tongue_n)
@@ -422 +422 @@
-so)
+so#so_adv)
@@ -424 +424 @@
-easie)
+easie#easy_adj)
@@ -432 +432 @@
-to)
+to#to_prep)
@@ -434 +434 @@
-be)
+be#be_v)
@@ -436 +436 @@
-learnt)))))))))
+learnt#learn_v)))))))))
@@ -443 +443 @@
-Their)
+Their#their_adj)
@@ -445 +445 @@
-Advance)
+Advance#advance_n)
@@ -448 +448 @@
-to)
+to#to_prep)
@@ -451 +451 @@
-the)
+the#the_adj)
@@ -453 +453 @@
-Grammars)
+Grammars#grammar_n)
@@ -456 +456 @@
-of)
+of#of_prep)
@@ -459 +459 @@
-Latin)
+Latin#NA)
@@ -461 +461 @@
-and)
+and#and_conj1)
@@ -463 +463 @@
-Greek))))))
+Greek#NA))))))
@@ -465 +465 @@
-would)
+would#will_v1)
@@ -467 +467 @@
-be)
+be#be_v)
@@ -470 +470 @@
-gradual)
+gradual#gradual_adj)
@@ -472 +472 @@
-and)
+and#and_conj1)
@@ -474 +474 @@
-pleasant)))))
+pleasant#pleasant_adj)))))
@@ -482 +482 @@
-But)
+But#but_prep)
@@ -485 +485 @@
-our)
+our#our_pron)
@@ -487 +487 @@
-precipitate)
+precipitate#precipitate_adj)
@@ -489 +489 @@
-Practice)
+Practice#practice_n)
@@ -492 +492 @@
-of)
+of#of_prep)
@@ -495 +495 @@
-hurrying)
+hurrying#hurry_v)
@@ -498 +498 @@
-them))
+them#them_pron))
@@ -501 +501 @@
-over)
+over#over_prep)
@@ -504 +504 @@
-such)
+such#such_adj)
@@ -506 +506 @@
-a)
+a#a_adj2)
@@ -508 +508 @@
-Gulph)))
+Gulph#gulf_n)))
@@ -513 +513 @@
-before)
+before#before_adv)
@@ -520 +520 @@
-we))
+we#we_pron))
@@ -522 +522 @@
-have)
+have#have_v)
@@ -524 +524 @@
-built)
+built#build_v)
@@ -527 +527 @@
-'em))
+'em#them_pron))
@@ -530 +530 @@
-a)
+a#a_adj2)
@@ -532 +532 @@
-Bridge)
+Bridge#bridge_n)
@@ -535 +535 @@
-to)
+to#to_prep)
@@ -538 +538 @@
-it))))))))))
+it#it_pron))))))))))
@@ -542 +542 @@
-is)
+is#be_v)
@@ -545 +545 @@
-such)
+such#such_adj)
@@ -547 +547 @@
-a)
+a#a_adj2)
@@ -549 +549 @@
-Shock)
+Shock#shock_n3)
@@ -552 +552 @@
-to)
+to#to_prep)
@@ -555 +555 @@
-their)
+their#their_adj)
@@ -557 +557 @@
-weak)
+weak#weak_adj)
@@ -561 +561 @@
-Understandings)))
+Understandings#understanding_n)))
@@ -566 +566 @@
-that)
+that#that_conj)
@@ -570 +570 @@
-they))
+they#they_pron))
@@ -573 +573 @@
-very)
+very#very_adj)
@@ -575 +575 @@
-seldom))
+seldom#seldom_adv))
@@ -577 +577 @@
-recover)
+recover#recover_v1)
@@ -580 +580 @@
-it)))))
+it#it_pron)))))
@@ -596 +596 @@
-To)
+To#to_prep)
@@ -599 +599 @@
-Speak)
+Speak#speak_v)
@@ -601 +601 @@
-and)
+and#and_conj1)
@@ -603 +603 @@
-Write))
+Write#write_v))
@@ -606 +606 @@
-without)
+without#without_adv)
@@ -609 +609 @@
-Absurdity)))
+Absurdity#absurdity_n)))
@@ -612 +612 @@
-the)
+the#the_adj)
@@ -614 +614 @@
-Language)
+Language#language_n)
@@ -617 +617 @@
-of)
+of#of_prep)
@@ -620 +620 @@
-one's)
+one's#one_adj)
@@ -622 +622 @@
-Country)))))
+Country#country_n)))))
@@ -626 +626 @@
-is)
+is#be_v)
@@ -629 +629 @@
-certainly)
+certainly#certainly_adv)
@@ -631 +631 @@
-commendable))
+commendable#commendable_adj))
@@ -634 +634 @@
-in)
+in#in_prep)
@@ -637 +637 @@
-Persons)
+Persons#person_n)
@@ -640 +640 @@
-of)
+of#of_prep)
@@ -643 +643 @@
-all)
+all#all_adj)
@@ -645 +645 @@
-stations))))))
+stations#station_n))))))
@@ -650 +650 @@
-but)
+but#but_prep)
@@ -654 +654 @@
-to)
+to#to_prep)
@@ -657 +657 @@
-some)))
+some#some_pron)))
@@ -660 +660 @@
-indispensably)
+indispensably#indispensably_adv)
@@ -664 +664 @@
-necessary))))
+necessary#necessary_adj))))
@@ -676 +676 @@
-I))
+I#i_pron))
@@ -678 +678 @@
-appeal)
+appeal#appeal_v)
@@ -681 +681 @@
-to)
+to#to_prep)
@@ -684 +684 @@
-common)
+common#common_adj)
@@ -686 +686 @@
-Sense)))
+Sense#sense_n)))
@@ -691 +691 @@
-if)
+if#if_conj)
@@ -697 +697 @@
-the)
+the#the_adj)
@@ -700 +700 @@
-having)
+having#have_v)
@@ -704 +704 @@
-a)
+a#a_adj2)
@@ -706 +706 @@
-Grammar)
+Grammar#grammar_n)
@@ -709 +709 @@
-of)
+of#of_prep)
@@ -712 +712 @@
-our)
+our#our_pron)
@@ -714 +714 @@
-Mother-Tongue))))
+Mother-Tongue#mother-tongue_n))))
@@ -719 +719 @@
-first))
+first#first_adj))
@@ -721 +721 @@
-taught)
+taught#teach_v)
@@ -724 +724 @@
-in)
+in#in_prep)
@@ -727 +727 @@
-our)
+our#our_pron)
@@ -729 +729 @@
-Schools))))))
+Schools#school_n1))))))
@@ -733 +733 @@
-would)
+would#will_v1)
@@ -735 +735 @@
-not)
+not#not_adv)
@@ -739 +739 @@
-so)
+so#so_adv)
@@ -741 +741 @@
-far))
+far#far_adj))
@@ -745 +745 @@
-facilitate)
+facilitate#facilitate_v)
@@ -749 +749 @@
-our)
+our#our_pron)
@@ -751 +751 @@
-Youths))
+Youths#youth_n))
@@ -753 +753 @@
-learning)
+learning#learn_v)
@@ -756 +756 @@
-their)
+their#their_adj)
@@ -759 +759 @@
-Latin)
+Latin#latin_adj)
@@ -761 +761 @@
-and)
+and#and_conj1)
@@ -763 +763 @@
-Greek))
+Greek#greek_adj))
@@ -765 +765 @@
-Grammars)))
+Grammars#grammar_n)))
@@ -770 +770 @@
-as)
+as#as_adv)
@@ -773 +773 @@
-to)
+to#to_prep)
@@ -775 +775 @@
-afford)
+afford#afford_v)
@@ -778 +778 @@
-'em))
+'em#them_pron))
@@ -781 +781 @@
-spare-time)
+spare-time#SEE_spare_adj)
@@ -784 +784 @@
-for)
+for#for_prep)
@@ -788 +788 @@
-Arithmetic))
+Arithmetic#arithmetic_n))
@@ -794 +794 @@
-Astronomy)))
+Astronomy#astronomy_n)))
@@ -800 +800 @@
-Geography)))
+Geography#geography_n)))
@@ -806 +806 @@
-History)))
+History#history_n)))
@@ -811 +811 @@
-and)
+and#and_conj1)
@@ -814 +814 @@
-other)
+other#other_adj)
@@ -816 +816 @@
-useful)
+useful#useful_adj)
@@ -818 +818 @@
-Knowledges)
+Knowledges#knowledge_n)
@@ -825 +825 @@
-that)
+that#that_conj)
@@ -830 +830 @@
-would)
+would#will_v1)
@@ -832 +832 @@
-make)
+make#make_v1)
@@ -836 +836 @@
-them))
+them#them_pron))
@@ -838 +838 @@
-pass)
+pass#pass_v)
@@ -841 +841 @@
-the)
+the#the_adj)
@@ -843 +843 @@
-Spring)
+Spring#spring_n1)
@@ -846 +846 @@
-of)
+of#of_prep)
@@ -849 +849 @@
-their)
+their#their_adj)
@@ -851 +851 @@
-Life)))
+Life#life_n)))
@@ -856 +856 @@
-with)
+with#with_prep)
@@ -859 +859 @@
-Pleasure)
+Pleasure#pleasure_n)
@@ -861 +861 @@
-and)
+and#and_conj1)
@@ -863 +863 @@
-Profit)))
+Profit#profit_n)))
@@ -870 +870 @@
-that)
+that#that_conj)
@@ -875 +875 @@
-is)
+is#be_v)
@@ -878 +878 @@
-now))
+now#now_adv))
@@ -881 +881 @@
-miserably))
+miserably#miserably_adv))
@@ -883 +883 @@
-worn)
+worn#wear_v1)
@@ -885 +885 @@
-out)
+out#out_adv)
@@ -888 +888 @@
-in)
+in#in_prep)
@@ -891 +891 @@
-Grammatical)
+Grammatical#grammatical_adj)
@@ -893 +893 @@
-Perplexities)))))))))))))))))
+Perplexities#perplexity_n)))))))))))))))))
@@ -908 +908 @@
-'T@))
+'T@#it_pron))
@@ -910 +910 @@
-@is)
+@is#be_v)
@@ -916 +916 @@
-I))
+I#i_pron))
@@ -919 +919 @@
-humbly))
+humbly#humbly_adv))
@@ -921 +921 @@
-conceive))
+conceive#conceive_v))
@@ -926 +926 @@
-evident))
+evident#evident_adj))
@@ -929 +929 @@
-from)
+from#from_prep)
@@ -932 +932 @@
-the)
+the#the_adj)
@@ -934 +934 @@
-Premises)))
+Premises#premise_n)))
@@ -939 +939 @@
-that)
+that#that_conj)
@@ -943 +943 @@
-the)
+the#the_adj)
@@ -945 +945 @@
-propos'd)
+propos'd#proposed_adj)
@@ -947 +947 @@
-Method)
+Method#method_n)
@@ -950 +950 @@
-of)
+of#of_prep)
@@ -953 +953 @@
-Commencing)
+Commencing#commencing_n)
@@ -956 +956 @@
-with)
+with#with_prep)
@@ -959 +959 @@
-an)
+an#a_adj2)
@@ -963 +963 @@
-English)
+English#english_adj)
@@ -965 +965 @@
-Education))))))
+Education#education_n))))))
@@ -969 +969 @@
-would)
+would#will_v1)
@@ -971 +971 @@
-be)
+be#be_v)
@@ -974 +974 @@
-a)
+a#a_adj2)
@@ -976 +976 @@
-great)
+great#great_adj)
@@ -978 +978 @@
-Benefit)
+Benefit#benefit_n)
@@ -982 +982 @@
-to)
+to#to_prep)
@@ -985 +985 @@
-our)
+our#our_pron)
@@ -987 +987 @@
-Youth)))
+Youth#youth_n)))
@@ -994 +994 @@
-whether)
+whether#whether_pron)
@@ -999 +999 @@
-continued))
+continued#continue_v))
@@ -1004 +1004 @@
-or)
+or#or_conj2)
@@ -1007 +1007 @@
-taken)
+taken#take_v)
@@ -1009 +1009 @@
-off)
+off#off_adv)
@@ -1012 +1012 @@
-from)
+from#from_prep)
@@ -1015 +1015 @@
-School))))))))
+School#school_n1))))))))
@@ -1021 +1021 @@
-to)
+to#to_prep)
@@ -1024 +1024 @@
-Adult)
+Adult#adult_adj)
@@ -1026 +1026 @@
-Persons)
+Persons#person_n)
@@ -1033 +1033 @@
-that)
+that#that_conj)
@@ -1038 +1038 @@
-have)
+have#have_v)
@@ -1040 +1040 @@
-forgot)
+forgot#forget_v)
@@ -1043 +1043 @@
-or)
+or#or_conj2)
@@ -1046 +1046 @@
-never))
+never#never_adv))
@@ -1048 +1048 @@
-had))
+had#have_v))
@@ -1051 +1051 @@
-School-Learning))
+School-Learning#SEE_school_n1))
@@ -1054 +1054 @@
-any)
+any#any_adj)
@@ -1056 +1056 @@
-farther)
+farther#farther_adv)
@@ -1059 +1059 @@
-than)
+than#than_conj)
@@ -1062 +1062 @@
-to)
+to#to_prep)
@@ -1064 +1064 @@
-Read)))))))))
+Read#read_v)))))))))
@@ -1069 +1069 @@
-since)
+since#since_adv)
@@ -1076 +1076 @@
-by)
+by#by_prep)
@@ -1079 +1079 @@
-the)
+the#the_adj)
@@ -1081 +1081 @@
-help)
+help#help_n)
@@ -1084 +1084 @@
-of)
+of#of_prep)
@@ -1087 +1087 @@
-an)
+an#a_adj2)
@@ -1091 +1091 @@
-English)
+English#english_adj)
@@ -1093 +1093 @@
-Grammar)
+Grammar#grammar_n)
@@ -1098 +1098 @@
-and)
+and#and_conj1)
@@ -1103 +1103 @@
-Rhetoric)))))))
+Rhetoric#rhetoric_n1)))))))
@@ -1108 +1108 @@
-they))
+they#they_pron))
@@ -1110 +1110 @@
-may)
+may#may_v1)
@@ -1112 +1112 @@
-arrive)
+arrive#arrive_v)
@@ -1115 +1115 @@
-to)
+to#to_prep)
@@ -1118 +1118 @@
-no)
+no#no_adj)
@@ -1120 +1120 @@
-small)
+small#small_adj)
@@ -1122 +1122 @@
-Accomplishments)
+Accomplishments#accomplishment_n)
@@ -1124 +1124 @@
-both)
+both#NA)
@@ -1127 +1127 @@
-in)
+in#in_prep)
@@ -1132 +1132 @@
-Speaking)
+Speaking#speaking_n)
@@ -1134 +1134 @@
-and)
+and#and_conj1)
@@ -1136 +1136 @@
-Writing))))))))))
+Writing#writing_n))))))))))
@@ -1148 +1148 @@
-Now))
+Now#now_adv))
@@ -1152 +1152 @@
-the)
+the#the_adj)
@@ -1154 +1154 @@
-Reasons)
+Reasons#reason_n1)
@@ -1157 +1157 @@
-for)
+for#for_prep)
@@ -1162 +1162 @@
-and)
+and#and_conj1)
@@ -1164 +1164 @@
-good)
+good#good_adj)
@@ -1166 +1166 @@
-Consequences)
+Consequences#consequence_n)
@@ -1169 +1169 @@
-of)))
+of#of_prep)))
@@ -1174 +1174 @@
-having)
+having#have_v)
@@ -1178 +1178 @@
-our)
+our#our_pron)
@@ -1182 +1182 @@
-English)
+English#english_adj)
@@ -1184 +1184 @@
-Youth))
+Youth#youth_n))
@@ -1189 +1189 @@
-first))
+first#first_adj))
@@ -1191 +1191 @@
-taught)
+taught#teach_v)
@@ -1195 +1195 @@
-the)
+the#the_adj)
@@ -1199 +1199 @@
-Rhetoric))
+Rhetoric#rhetoric_n1))
@@ -1206 +1206 @@
-as)
+as#as_adv)
@@ -1208 +1208 @@
-well)
+well#well_adv)
@@ -1211 +1211 @@
-as)
+as#as_adv)
@@ -1216 +1216 @@
-Grammar))))
+Grammar#grammar_n))))
@@ -1223 +1223 @@
-of)
+of#of_prep)
@@ -1226 +1226 @@
-their)
+their#their_adj)
@@ -1228 +1228 @@
-own)
+own#own_adj)
@@ -1230 +1230 @@
-Language))))))))
+Language#language_n))))))))
@@ -1234 +1234 @@
-being)
+being#be_v)
@@ -1237 +1237 @@
-much)
+much#much_adj)
@@ -1239 +1239 @@
-the)
+the#the_adj)
@@ -1241 +1241 @@
-same)))
+same#same_adj)))
@@ -1248 +1248 @@
-what))
+what#what_pron))
@@ -1254 +1254 @@
-I))
+I#i_pron))
@@ -1256 +1256 @@
-have)
+have#have_v)
@@ -1261 +1261 @@
-to)
+to#to_prep)
@@ -1263 +1263 @@
-add)))))
+add#add_v)))))
@@ -1265 +1265 @@
-shall)
+shall#shall_v)
@@ -1267 +1267 @@
-be)
+be#be_v)
@@ -1270 +1270 @@
-with)
+with#with_prep)
@@ -1273 +1273 @@
-reference)
+reference#reference_n)
@@ -1276 +1276 @@
-to)
+to#to_prep)
@@ -1279 +1279 @@
-Both)))))
+Both#both_pron)))))
@@ -1288 +1288 @@
-'T@))
+'T@#it_pron))
@@ -1290 +1290 @@
-@is)
+@is#be_v)
@@ -1293 +1293 @@
-contrary)
+contrary#contrary_adj)
@@ -1296 +1296 @@
-to)
+to#to_prep)
@@ -1299 +1299 @@
-Sense)
+Sense#sense_n)
@@ -1301 +1301 @@
-and)
+and#and_conj1)
@@ -1303 +1303 @@
-Reason))))
+Reason#reason_n1))))
@@ -1308 +1308 @@
-says)
+says#say_v1)
@@ -1311 +1311 @@
-an)
+an#a_adj2)
@@ -1313 +1313 @@
-able)
+able#able_adj)
@@ -1315 +1315 @@
-Judge))
+Judge#judge_n))
@@ -1318 +1318 @@
-in)
+in#in_prep)
@@ -1321 +1321 @@
-this)
+this#this_pron)
@@ -1323 +1323 @@
-Case))))
+Case#case_n1))))
@@ -1328 +1328 @@
-to)
+to#to_prep)
@@ -1330 +1330 @@
-put)
+put#put_v)
@@ -1333 +1333 @@
-our)
+our#our_pron)
@@ -1337 +1337 @@
-English)
+English#english_adj)
@@ -1339 +1339 @@
-Youth))
+Youth#youth_n))
@@ -1344 +1344 @@
-to)
+to#to_prep)
@@ -1346 +1346 @@
-toil)
+toil#toil_v1)
@@ -1349 +1349 @@
-in)
+in#in_prep)
@@ -1352 +1352 @@
-foreign)
+foreign#foreign_adj)
@@ -1354 +1354 @@
-Languages))))
+Languages#language_n))))
@@ -1359 +1359 @@
-while)
+while#while_adv)
@@ -1366 +1366 @@
-our)
+our#our_pron)
@@ -1368 +1368 @@
-own)
+own#own_adj)
@@ -1370 +1370 @@
-Excellent)
+Excellent#excellent_adj)
@@ -1372 +1372 @@
-Language))
+Language#language_n))
@@ -1374 +1374 @@
-lies)
+lies#lie_v1)
@@ -1377 +1377 @@
-neglected))
+neglected#neglected_adj))
@@ -1382 +1382 @@
-as)
+as#as_adv)
@@ -1385 +1385 @@
-if)
+if#if_conj)
@@ -1392 +1392 @@
-it))
+it#it_pron))
@@ -1394 +1394 @@
-were)
+were#be_v)
@@ -1398 +1398 @@
-savage))
+savage#savage_adj))
@@ -1401 +1401 @@
-and)
+and#and_conj1)
@@ -1404 +1404 @@
-unfit)
+unfit#unfit_adj)
@@ -1407 +1407 @@
-to)
+to#to_prep)
@@ -1409 +1409 @@
-entertain)
+entertain#entertain_v)
@@ -1412 +1412 @@
-the)
+the#the_adj)
@@ -1414 +1414 @@
-Liberal)
+Liberal#liberal_adj)
@@ -1416 +1416 @@
-Arts)
+Arts#art_n)
@@ -1419 +1419 @@
-and)
+and#and_conj1)
@@ -1422 +1422 @@
-Sciences))))))))))))))))
+Sciences#science_n))))))))))))))))
@@ -1427 +1427 @@
-whereas)
+whereas#whereas_adv)
@@ -1436 +1436 @@
-if)
+if#if_conj)
@@ -1443 +1443 @@
-we))
+we#we_pron))
@@ -1445 +1445 @@
-would)
+would#will_v1)
@@ -1447 +1447 @@
-take)
+take#take_v)
@@ -1450 +1450 @@
-care))
+care#care_n1))
@@ -1453 +1453 @@
-to)
+to#to_prep)
@@ -1456 +1456 @@
-polish)
+polish#polish_v)
@@ -1458 +1458 @@
-and)
+and#and_conj1)
@@ -1460 +1460 @@
-adorn))
+adorn#adorn_v))
@@ -1463 +1463 @@
-our)
+our#our_pron)
@@ -1465 +1465 @@
-Language))))))
+Language#language_n))))))
@@ -1470 +1470 @@
-we))
+we#we_pron))
@@ -1472 +1472 @@
-should)
+should#shall_v)
@@ -1474 +1474 @@
-find)
+find#find_v)
@@ -1478 +1478 @@
-it))
+it#it_pron))
@@ -1481 +1481 @@
-as)
+as#as_adv)
@@ -1483 +1483 @@
-capable)
+capable#capable_adj)
@@ -1486 +1486 @@
-of)
+of#of_prep)
@@ -1489 +1489 @@
-being)
+being#be_v)
@@ -1492 +1492 @@
-the)
+the#the_adj)
@@ -1494 +1494 @@
-Receptacle)
+Receptacle#receptacle_n)
@@ -1497 +1497 @@
-and)
+and#and_conj1)
@@ -1500 +1500 @@
-Repository)))
+Repository#repository_n)))
@@ -1503 +1503 @@
-of)
+of#of_prep)
@@ -1506 +1506 @@
-Learning)))))))
+Learning#learning_n)))))))
@@ -1511 +1511 @@
-as)
+as#as_adv)
@@ -1514 +1514 @@
-those)
+those#those_pron)
@@ -1519 +1519 @@
-that)
+that#that_conj)
@@ -1524 +1524 @@
-are)
+are#be_v)
@@ -1526 +1526 @@
-call'd)
+call'd#call_v)
@@ -1532 +1532 @@
-the)
+the#the_adj)
@@ -1536 +1536 @@
-Learned)
+Learned#learned_adj)
@@ -1538 +1538 @@
-Languages)))))))))))
+Languages#language_n)))))))))))
@@ -1551 +1551 @@
-But)
+But#but_prep)
@@ -1554 +1554 @@
-after)
+after#after_adv)
@@ -1557 +1557 @@
-all)))
+all#all_adj)))
@@ -1564 +1564 @@
-because)
+because#because_adv)
@@ -1571 +1571 @@
-Arguments)
+Arguments#argument_n)
@@ -1574 +1574 @@
-from)
+from#from_prep)
@@ -1577 +1577 @@
-Interest))))
+Interest#interest_n))))
@@ -1579 +1579 @@
-are)
+are#be_v)
@@ -1582 +1582 @@
-most)
+most#most_adj)
@@ -1584 +1584 @@
-persuasive)))))
+persuasive#persuasive_adj)))))
@@ -1589 +1589 @@
-I))
+I#i_pron))
@@ -1591 +1591 @@
-would)
+would#will_v1)
@@ -1593 +1593 @@
-ask)
+ask#ask_v)
@@ -1596 +1596 @@
-those)
+those#those_pron)
@@ -1598 +1598 @@
-Parents)
+Parents#parent_n)
@@ -1604 +1604 @@
-who))
+who#who_pron))
@@ -1611 +1611 @@
-have)
+have#have_v)
@@ -1615 +1615 @@
-their)
+their#their_adj)
@@ -1617 +1617 @@
-Children))
+Children#child_n))
@@ -1619 +1619 @@
-bred)
+bred#breed_v)
@@ -1622 +1622 @@
-Scholars))
+Scholars#scholar_n))
@@ -1625 +1625 @@
-chiefly)
+chiefly#chiefly_adv)
@@ -1627 +1627 @@
-for)
+for#for_prep)
@@ -1630 +1630 @@
-a)
+a#a_adj2)
@@ -1632 +1632 @@
-Livelihood)))))))
+Livelihood#livelihood_n1)))))))
@@ -1639 +1639 @@
-In)
+In#in_prep)
@@ -1642 +1642 @@
-what)
+what#what_pron)
@@ -1644 +1644 @@
-Language)))
+Language#language_n)))
@@ -1651 +1651 @@
-is)
+is#be_v)
@@ -1654 +1654 @@
-the)
+the#the_adj)
@@ -1656 +1656 @@
-Thriving)
+Thriving#thriving_adj)
@@ -1658 +1658 @@
-Business)
+Business#business_n)
@@ -1661 +1661 @@
-of)
+of#of_prep)
@@ -1664 +1664 @@
-our)
+our#our_pron)
@@ -1666 +1666 @@
-Nation))))
+Nation#nation_n))))
@@ -1668 +1668 @@
-transacted)))
+transacted#transact_v)))
@@ -1673 +1673 @@
-And)
+And#and_conj1)
@@ -1678 +1678 @@
-Whether)
+Whether#whether_pron)
@@ -1684 +1684 @@
-a)
+a#a_adj2)
@@ -1686 +1686 @@
-voluble)
+voluble#voluble_adj)
@@ -1690 +1690 @@
-English)
+English#english_adj)
@@ -1692 +1692 @@
-Tongue)
+Tongue#tongue_n)
@@ -1699 +1699 @@
-in)
+in#in_prep)
@@ -1702 +1702 @@
-their)
+their#their_adj)
@@ -1704 +1704 @@
-Head))))
+Head#head_n))))
@@ -1706 +1706 @@
-will)
+will#will_v1)
@@ -1708 +1708 @@
-not)
+not#not_adv)
@@ -1710 +1710 @@
-carry)
+carry#carry_v)
@@ -1713 +1713 @@
-them))
+them#them_pron))
@@ -1716 +1716 @@
-farther)
+farther#farther_adv)
@@ -1721 +1721 @@
-in)
+in#in_prep)
@@ -1724 +1724 @@
-the)
+the#the_adj)
@@ -1726 +1726 @@
-ways)
+ways#way_n1)
@@ -1729 +1729 @@
-of)
+of#of_prep)
@@ -1732 +1732 @@
-Profit)
+Profit#profit_n)
@@ -1734 +1734 @@
-and)
+and#and_conj1)
@@ -1736 +1736 @@
-Preferment)))))
+Preferment#preferment_n)))))
@@ -1741 +1741 @@
-than)
+than#than_conj)
@@ -1744 +1744 @@
-all)
+all#all_adj)
@@ -1746 +1746 @@
-the)
+the#the_adj)
@@ -1750 +1750 @@
-Learned)
+Learned#learned_adj)
@@ -1752 +1752 @@
-Languages)))))))
+Languages#language_n)))))))
@@ -1766 +1766 @@
-Generous)
+Generous#generous_adj)
@@ -1768 +1768 @@
-Spirits))
+Spirits#spirit_n))
@@ -1770 +1770 @@
-will)
+will#will_v1)
@@ -1773 +1773 @@
-always))
+always#always_adv))
@@ -1775 +1775 @@
-have)
+have#have_v)
@@ -1778 +1778 @@
-a)
+a#a_adj2)
@@ -1780 +1780 @@
-Concern))
+Concern#concern_n))
@@ -1783 +1783 @@
-for)
+for#for_prep)
@@ -1786 +1786 @@
-the)
+the#the_adj)
@@ -1788 +1788 @@
-Benefit)
+Benefit#benefit_n)
@@ -1791 +1791 @@
-and)
+and#and_conj1)
@@ -1794 +1794 @@
-Credit)))
+Credit#credit_n)))
@@ -1797 +1797 @@
-of)
+of#of_prep)
@@ -1800 +1800 @@
-their)
+their#their_adj)
@@ -1802 +1802 @@
-Country)))))
+Country#country_n)))))
@@ -1811 +1811 @@
-And)
+And#and_conj1)
@@ -1816 +1816 @@
-how)
+how#how_adv)
@@ -1818 +1818 @@
-far)))
+far#far_adj)))
@@ -1826 +1826 @@
-the)
+the#the_adj)
@@ -1828 +1828 @@
-Honour)
+Honour#honour_n)
@@ -1831 +1831 @@
-and)
+and#and_conj1)
@@ -1834 +1834 @@
-Interest)))
+Interest#interest_n)))
@@ -1837 +1837 @@
-of)
+of#of_prep)
@@ -1842 +1842 @@
-Great)
+Great#great_adj)
@@ -1844 +1844 @@
-Britain))))
+Britain#NA))))
@@ -1848 +1848 @@
-are)
+are#be_v)
@@ -1850 +1850 @@
-concern'd)
+concern'd#concern_v)
@@ -1853 +1853 @@
-in)
+in#in_prep)
@@ -1856 +1856 @@
-the)
+the#the_adj)
@@ -1858 +1858 @@
-Cultivating)
+Cultivating#cultivating_n)
@@ -1861 +1861 @@
-of)
+of#of_prep)
@@ -1864 +1864 @@
-Our)
+Our#our_pron)
@@ -1866 +1866 @@
-Language)))))))
+Language#language_n)))))))
@@ -1871 +1871 @@
-I))
+I#i_pron))
@@ -1873 +1873 @@
-presume)
+presume#presume_v)
@@ -1875 +1875 @@
-not)
+not#not_adv)
@@ -1880 +1880 @@
-to)
+to#to_prep)
@@ -1882 +1882 @@
-say)))
+say#say_v1)))
@@ -1889 +1889 @@
-only)
+only#only_adj)
@@ -1893 +1893 @@
-That)
+That#that_conj)
@@ -1898 +1898 @@
-a)
+a#a_adj2)
@@ -1900 +1900 @@
-neighbouring)
+neighbouring#neighbouring_adj)
@@ -1902 +1902 @@
-Nation))
+Nation#nation_n))
@@ -1904 +1904 @@
-has)
+has#have_v)
@@ -1906 +1906 @@
-taken)
+taken#take_v)
@@ -1909 +1909 @@
-Care))
+Care#care_n1))
@@ -1912 +1912 @@
-of)
+of#of_prep)
@@ -1915 +1915 @@
-Theirs))))
+Theirs#their_adj))))
@@ -1920 +1920 @@
-and)
+and#and_conj1)
@@ -1923 +1923 @@
-found)
+found#find_v)
@@ -1926 +1926 @@
-their)
+their#their_adj)
@@ -1928 +1928 @@
-Accounts))
+Accounts#account_n))
@@ -1931 +1931 @@
-in@)
+in@#in_prep)
@@ -1934 +1934 @@
-@'t)))))))))
+@'t#it_pron)))))))))
@@ -1943 +1943 @@
-The)
+The#the_adj)
@@ -1945 +1945 @@
-'foremention'd)
+'foremention'd#aforementioned_adj)
@@ -1949 +1949 @@
-Cicero))
+Cicero#NA))
@@ -1956 +1956 @@
-when)
+when#when_adv)
@@ -1959 +1959 @@
-advanc'd)
+advanc'd#advance_v)
@@ -1962 +1962 @@
-to)
+to#to_prep)
@@ -1965 +1965 @@
-the)
+the#the_adj)
@@ -1967 +1967 @@
-Consulship)))))
+Consulship#consulship_n)))))
@@ -1971 +1971 @@
-was)
+was#be_v)
@@ -1974 +1974 @@
-so)
+so#so_adv)
@@ -1976 +1976 @@
-far)
+far#far_adj)
@@ -1979 +1979 @@
-from)
+from#from_prep)
@@ -1982 +1982 @@
-neglecting)
+neglecting#neglect_v)
@@ -1985 +1985 @@
-his)
+his#his_adj)
@@ -1987 +1987 @@
-Mother-Tongue))))
+Mother-Tongue#mother-tongue_n))))
@@ -1992 +1992 @@
-that)
+that#that_conj)
@@ -1996 +1996 @@
-in)
+in#in_prep)
@@ -1999 +1999 @@
-the)
+the#the_adj)
@@ -2001 +2001 @@
-heat)
+heat#heat_n)
@@ -2004 +2004 @@
-of)
+of#of_prep)
@@ -2007 +2007 @@
-Civil)
+Civil#civil_adj)
@@ -2009 +2009 @@
-War)
+War#war_n1)
@@ -2012 +2012 @@
-between)
+between#between_prep)
@@ -2017 +2017 @@
-Caesar)
+Caesar#NA)
@@ -2021 +2021 @@
-and)
+and#and_conj1)
@@ -2025 +2025 @@
-Pompey)))))))
+Pompey#NA)))))))
@@ -2032 +2032 @@
-when)
+when#when_adv)
@@ -2039 +2039 @@
-he))
+he#he_pron))
@@ -2041 +2041 @@
-knew)
+knew#know_v)
@@ -2043 +2043 @@
-not)
+not#not_adv)
@@ -2048 +2048 @@
-whither))
+whither#whither_adv))
@@ -2053 +2053 @@
-to)
+to#to_prep)
@@ -2055 +2055 @@
-send)
+send#send_v1)
@@ -2058 +2058 @@
-Wife)
+Wife#wife_n)
@@ -2060 +2060 @@
-or)
+or#or_conj2)
@@ -2062 +2062 @@
-Children))))
+Children#child_n))))
@@ -2067 +2067 @@
-nor)
+nor#nor_conj1)
@@ -2071 +2071 @@
-where))
+where#where_adv))
@@ -2076 +2076 @@
-to)
+to#to_prep)
@@ -2078 +2078 @@
-hide)
+hide#hide_v1)
@@ -2081 +2081 @@
-his)
+his#his_adj)
@@ -2083 +2083 @@
-own)
+own#own_adj)
@@ -2085 +2085 @@
-Head)))))))))
+Head#head_n)))))))))
@@ -2090 +2090 @@
-he))
+he#he_pron))
@@ -2092 +2092 @@
-importunes)
+importunes#importune_v)
@@ -2095 +2095 @@
-his)
+his#his_adj)
@@ -2097 +2097 @@
-Friend)
+Friend#friend_n)
@@ -2102 +2102 @@
-Atticus)))
+Atticus#NA)))
@@ -2109 +2109 @@
-in)
+in#in_prep)
@@ -2112 +2112 @@
-a)
+a#a_adj2)
@@ -2114 +2114 @@
-Letter)
+Letter#letter_n1)
@@ -2117 +2117 @@
-of)
+of#of_prep)
@@ -2120 +2120 @@
-State-matters)))))
+State-matters#cNA)))))
@@ -2125 +2125 @@
-to)
+to#to_prep)
@@ -2127 +2127 @@
-satisfie)
+satisfie#satisfy_v)
@@ -2130 +2130 @@
-him))
+him#him_pron))
@@ -2135 +2135 @@
-whether)
+whether#whether_pron)
@@ -2141 +2141 @@
-he))
+he#he_pron))
@@ -2143 +2143 @@
-ought)
+ought#ought_v)
@@ -2145 +2145 @@
-to)
+to#to_prep)
@@ -2147 +2147 @@
-write)
+write#write_v)
@@ -2154 +2154 @@
-ad)
+ad#NA)
@@ -2156 +2156 @@
-Piraeum)
+Piraeum#NA)
@@ -2160 +2160 @@
-in)
+in#NA)
@@ -2162 +2162 @@
-Piraea)
+Piraea#NA)
@@ -2166 +2166 @@
-in)
+in#NA)
@@ -2168 +2168 @@
-Piraeum)))
+Piraeum#NA)))
@@ -2175 +2175 @@
-or)
+or#or_conj2)
@@ -2181 +2181 @@
-Piraeum))
+Piraeum#NA))
@@ -2186 +2186 @@
-without)
+without#without_adv)
@@ -2189 +2189 @@
-a)
+a#a_adj2)
@@ -2191 +2191 @@
-Preposition))))))))))))
+Preposition#preposition_n))))))))))))
@@ -2202 +2202 @@
-then))
+then#then_adv))
@@ -2204 +2204 @@
-adds)
+adds#add_v)
@@ -2207 +2207 @@
-these)
+these#these_pron)
@@ -2209 +2209 @@
-Emphatical)
+Emphatical#emphatical_adj)
@@ -2211 +2211 @@
-Words))
+Words#word_n))
@@ -2222 +2222 @@
-Resolve)
+Resolve#resolve_v)
@@ -2225 +2225 @@
-me))
+me#me_pron1))
@@ -2228 +2228 @@
-this)
+this#this_pron)
@@ -2230 +2230 @@
-Doubt)))
+Doubt#doubt_n1)))
@@ -2235 +2235 @@
-and)
+and#and_conj1)
@@ -2239 +2239 @@
-you))
+you#you_pron))
@@ -2241 +2241 @@
-will)
+will#will_v1)
@@ -2243 +2243 @@
-rid)
+rid#rid_v)
@@ -2246 +2246 @@
-me))
+me#me_pron1))
@@ -2249 +2249 @@
-of)
+of#of_prep)
@@ -2252 +2252 @@
-a)
+a#a_adj2)
@@ -2254 +2254 @@
-great)
+great#great_adj)
@@ -2256 +2256 @@
-Vneasiness)))))
+Vneasiness#uneasiness_n)))))
@@ -2267 +2267 @@
-What))
+What#what_pron))
@@ -2272 +2272 @@
-This)
+This#this_pron)
@@ -2274 +2274 @@
-Prince)
+Prince#prince_n)
@@ -2277 +2277 @@
-of)
+of#of_prep)
@@ -2280 +2280 @@
-Eloquence)))
+Eloquence#eloquence_n)))
@@ -2287 +2287 @@
-at)
+at#at_prep)
@@ -2290 +2290 @@
-the)
+the#the_adj)
@@ -2292 +2292 @@
-Age)
+Age#age_n)
@@ -2295 +2295 @@
-of)
+of#of_prep)
@@ -2298 +2298 @@
-60)))))
+60#sixty_adj)))))
@@ -2303 +2303 @@
-a)
+a#a_adj2)
@@ -2305 +2305 @@
-Man)
+Man#man_n1)
@@ -2309 +2309 @@
-of)
+of#of_prep)
@@ -2312 +2312 @@
-his)
+his#his_adj)
@@ -2314 +2314 @@
-Authority)))
+Authority#authority_n)))
@@ -2320 +2320 @@
-in)
+in#in_prep)
@@ -2323 +2323 @@
-that)
+that#that_pron)
@@ -2325 +2325 @@
-Care)
+Care#care_n1)
@@ -2328 +2328 @@
-for)
+for#for_prep)
@@ -2331 +2331 @@
-his)
+his#his_adj)
@@ -2333 +2333 @@
-Country))))))
+Country#country_n))))))
@@ -2339 +2339 @@
-in)
+in#in_prep)
@@ -2342 +2342 @@
-that)
+that#that_pron)
@@ -2344 +2344 @@
-Jeopardy)
+Jeopardy#jeopardy_n)
@@ -2347 +2347 @@
-for)
+for#for_prep)
@@ -2350 +2350 @@
-Himself)))
+Himself#himself_pron)))
@@ -2355 +2355 @@
-and)
+and#and_conj1)
@@ -2358 +2358 @@
-extreme)
+extreme#extreme_adj)
@@ -2360 +2360 @@
-Necessity)
+Necessity#necessity_n)
@@ -2363 +2363 @@
-of)
+of#of_prep)
@@ -2366 +2366 @@
-his)
+his#his_adj)
@@ -2368 +2368 @@
-dearest)
+dearest#dear_adj1)
@@ -2370 +2370 @@
-Friends))))))))))
+Friends#friend_n))))))))))
@@ -2375 +2375 @@
-so)
+so#so_adv)
@@ -2377 +2377 @@
-Sollicitous)
+Sollicitous#solicitous_adj)
@@ -2380 +2380 @@
-for)
+for#for_prep)
@@ -2383 +2383 @@
-a)
+a#a_adj2)
@@ -2385 +2385 @@
-small)
+small#small_adj)
@@ -2387 +2387 @@
-Criticism)
+Criticism#criticism_n)
@@ -2390 +2390 @@
-in)
+in#in_prep)
@@ -2393 +2393 @@
-the)
+the#the_adj)
@@ -2395 +2395 @@
-Grammar)
+Grammar#grammar_n)
@@ -2398 +2398 @@
-of)
+of#of_prep)
@@ -2403 +2403 @@
-his)
+his#his_adj)
@@ -2405 +2405 @@
-Native)
+Native#native_adj)
@@ -2407 +2407 @@
-Language))))))))
+Language#language_n))))))))
@@ -2417 +2417 @@
-And)
+And#and_conj1)
@@ -2420 +2420 @@
-can)
+can#can_v1)
@@ -2423 +2423 @@
-we))
+we#we_pron))
@@ -2425 +2425 @@
-give)
+give#give_v)
@@ -2428 +2428 @@
-our)
+our#our_pron)
@@ -2432 +2432 @@
-British)
+British#british_adj)
@@ -2434 +2434 @@
-Censor))
+Censor#censor_n))
@@ -2439 +2439 @@
-just)
+just#just_adj)
@@ -2441 +2441 @@
-Cause)
+Cause#cause_n)
@@ -2444 +2444 @@
-to)
+to#to_prep)
@@ -2446 +2446 @@
-complain)
+complain#complain_v)
@@ -2449 +2449 @@
-of)
+of#of_prep)
@@ -2452 +2452 @@
-the)
+the#the_adj)
@@ -2454 +2454 @@
-continual)
+continual#continual_adj)
@@ -2456 +2456 @@
-corruption)
+corruption#corruption_n)
@@ -2459 +2459 @@
-in)
+in#in_prep)
@@ -2462 +2462 @@
-our)
+our#our_pron)
@@ -2464 +2464 @@
-Stile)))))
+Stile#style_n)))))
@@ -2469 +2469 @@
-that)
+that#that_conj)
@@ -2473 +2473 @@
-a)
+a#a_adj2)
@@ -2475 +2475 @@
-Catalogue))
+Catalogue#catalogue_n))
@@ -2477,3 +2477,3 @@
-could)
-(BE
-be)
+could#can_v1)
+(BE
+be#be_v)
@@ -2481 +2481 @@
-produc'd)
+produc'd#produce_v)
@@ -2484 +2484 @@
-of)
+of#of_prep)
@@ -2488 +2488 @@
-lately-@)
+lately-@#lately_adv)
@@ -2490 +2490 @@
-@written))
+@written#written_adj))
@@ -2494 +2494 @@
-English)
+English#english_adj)
@@ -2498 +2498 @@
-Books)
+Books#book_n)
@@ -2501 +2501 @@
-of)
+of#of_prep)
@@ -2505 +2505 @@
-100)
+100#NA)
@@ -2509 +2509 @@
-l.))
+l.#pound_n1))
@@ -2513 +2513 @@
-price)))))
+price#price_n)))))
@@ -2518 +2518 @@
-without)
+without#without_adv)
@@ -2521 +2521 @@
-ten)
+ten#ten_adj)
@@ -2523 +2523 @@
-Lines)
+Lines#line_n2)
@@ -2526 +2526 @@
-together))
+together#together_adv))
@@ -2529 +2529 @@
-of)
+of#of_prep)
@@ -2533 +2533 @@
-Common)
+Common#common_adj)
@@ -2535 +2535 @@
-Grammar))
+Grammar#grammar_n))
@@ -2540 +2540 @@
-or)
+or#or_conj2)
@@ -2543 +2543 @@
-Common)
+Common#common_adj)
@@ -2545 +2545 @@
-Sense))))))))))))
+Sense#sense_n))))))))))))
@@ -2554 +2554 @@
-Then))
+Then#then_adv))
@@ -2557 +2557 @@
-how)
+how#how_adv)
@@ -2559 +2559 @@
-Unaccountable))
+Unaccountable#unaccountable_adj))
@@ -2564 +2564 @@
-is)
+is#be_v)
@@ -2567 +2567 @@
-it))
+it#it_pron))
@@ -2573 +2573 @@
-that)
+that#that_conj)
@@ -2577 +2577 @@
-the)
+the#the_adj)
@@ -2580 +2580 @@
-Teaching)
+Teaching#teach_v)
@@ -2583 +2583 @@
-a)
+a#a_adj2)
@@ -2585 +2585 @@
-Good)
+Good#good_adj)
@@ -2589 +2589 @@
-English)
+English#english_adj)
@@ -2593 +2593 @@
-Stile))))
+Stile#style_n))))
@@ -2595 +2595 @@
-should)
+should#shall_v)
@@ -2597 +2597 @@
-be)
+be#be_v)
@@ -2600 +2600 @@
-no)
+no#no_adj)
@@ -2602 +2602 @@
-Part)
+Part#part_n)
@@ -2605 +2605 @@
-of)
+of#of_prep)
@@ -2610 +2610 @@
-English)
+English#english_adj)
@@ -2614 +2614 @@
-Disciplin))))))
+Disciplin#discipline_n))))))
@@ -2621 +2621 @@
-and)
+and#and_conj1)
@@ -2624 +2624 @@
-to)
+to#to_prep)
@@ -2626 +2626 @@
-put)
+put#put_v)
@@ -2629 +2629 @@
-our)
+our#our_pron)
@@ -2631 +2631 @@
-Youth))
+Youth#youth_n))
@@ -2634 +2634 @@
-upon)
+upon#upon_prep)
@@ -2637 +2637 @@
-the)
+the#the_adj)
@@ -2639 +2639 @@
-study)
+study#study_n)
@@ -2642 +2642 @@
-of)
+of#of_prep)
@@ -2645 +2645 @@
-Foreign)
+Foreign#foreign_adj)
@@ -2647 +2647 @@
-Languages)))))
+Languages#language_n)))))
@@ -2650 +2650 @@
-with)
+with#with_prep)
@@ -2653 +2653 @@
-the)
+the#the_adj)
@@ -2655 +2655 @@
-same)
+same#same_adj)
@@ -2657 +2657 @@
-Discretion)
+Discretion#discretion_n)
@@ -2662 +2662 @@
-that)
+that#that_conj)
@@ -2667 +2667 @@
-sends)
+sends#send_v2)
@@ -2670 +2670 @@
-them))
+them#them_pron))
@@ -2673 +2673 @@
-to)
+to#to_prep)
@@ -2675 +2675 @@
-travel)
+travel#travel_v)
@@ -2678 +2678 @@
-Foreign)
+Foreign#foreign_adj)
@@ -2680 +2680 @@
-Countries)))))))
+Countries#country_n)))))))
@@ -2685 +2685 @@
-before)
+before#before_adv)
@@ -2692 +2692 @@
-they))
+they#they_pron))
@@ -2694 +2694 @@
-know)
+know#know_v)
@@ -2697 +2697 @@
-the)
+the#the_adj)
@@ -2699 +2699 @@
-Constitution)
+Constitution#constitution_n)
@@ -2702 +2702 @@
-and)
+and#and_conj1)
@@ -2705 +2705 @@
-Customs)))
+Customs#custom_n)))
@@ -2708 +2708 @@
-of)
+of#of_prep)
@@ -2711 +2711 @@
-their)
+their#their_adj)
@@ -2713 +2713 @@
-Own)))))))))))
+Own#own_adj)))))))))))
@@ -2721 +2721 @@
-But)
+But#but_prep)
@@ -2724 +2724 @@
-here))
+here#here_adv))
@@ -2727 +2727 @@
-an)
+an#a_adj2)
@@ -2729 +2729 @@
-importunate)
+importunate#importunate_adj)
@@ -2731 +2731 @@
-Question)
+Question#question_n)
@@ -2735 +2735 @@
-will)
+will#will_v1)
@@ -2737 +2737 @@
-be)
+be#be_v)
@@ -2739 +2739 @@
-ask'd)
+ask'd#ask_v)
@@ -2746 +2746 @@
-viz.)
+viz.#viz_adv)
@@ -2751 +2751 @@
-Why))
+Why#why_adv))
@@ -2754 +2754 @@
-an)
+an#a_adj2)
@@ -2758 +2758 @@
-English)
+English#english_adj)
@@ -2762 +2762 @@
-Education)))
+Education#education_n)))
@@ -2771 +2771 @@
-Had)
+Had#have_v)
@@ -2774 +2774 @@
-we))
+we#we_pron))
@@ -2778 +2778 @@
-no)
+no#no_adj)
@@ -2782 +2782 @@
-good)
+good#good_adj)
@@ -2784 +2784 @@
-Scholars))
+Scholars#scholar_n))
@@ -2788 +2788 @@
-made)
+made#make_v1)
@@ -2791 +2791 @@
-in)
+in#in_prep)
@@ -2794 +2794 @@
-the)
+the#the_adj)
@@ -2796 +2796 @@
-Old)
+Old#old_adj)
@@ -2798 +2798 @@
-Way)))))
+Way#way_n1)))))
@@ -2806 +2806 @@
-Yes)
+Yes#yes_adv)
@@ -2810 +2810 @@
-and)
+and#and_conj1)
@@ -2815 +2815 @@
-good)
+good#good_adj)
@@ -2817 +2817 @@
-Masters)
+Masters#master_n1)
@@ -2823 +2823 @@
-of)
+of#of_prep)
@@ -2826 +2826 @@
-Stile)))
+Stile#style_n)))
@@ -2831 +2831 @@
-too))
+too#too_adv))
@@ -2837 +2837 @@
-never))
+never#never_adv))
@@ -2840 +2840 @@
-more)
+more#more_adj)
@@ -2843 +2843 @@
-than)
+than#than_conj)
@@ -2846 +2846 @@
-in)
+in#in_prep)
@@ -2849 +2849 @@
-our)
+our#our_pron)
@@ -2851 +2851 @@
-Days))))))
+Days#day_n))))))
@@ -2856 +2856 @@
-and)
+and#and_conj1)
@@ -2859 +2859 @@
-of)
+of#of_prep)
@@ -2862 +2862 @@
-several)
+several#several_adj)
@@ -2864 +2864 @@
-Professions))))
+Professions#profession_n))))
@@ -2873 +2873 @@
-especially))
+especially#especially_adv))
@@ -2876 +2876 @@
-the)
+the#the_adj)
@@ -2878 +2878 @@
-Divines)
+Divines#divine_n2)
@@ -2881 +2881 @@
-of)
+of#of_prep)
@@ -2884 +2884 @@
-Our)
+Our#our_pron)
@@ -2886 +2886 @@
-Church)))
+Church#NA)))
@@ -2892 +2892 @@
-whose)
+whose#whose_pron)
@@ -2894 +2894 @@
-Discourses))
+Discourses#discourse_n))
@@ -2904 +2904 @@
-with)
+with#with_prep)
@@ -2907 +2907 @@
-all)
+all#all_adj)
@@ -2909 +2909 @@
-the)
+the#the_adj)
@@ -2911 +2911 @@
-Charms)
+Charms#charm_n1)
@@ -2914 +2914 @@
-of)
+of#of_prep)
@@ -2917 +2917 @@
-Foreign)
+Foreign#foreign_adj)
@@ -2919 +2919 @@
-Eloquence)))))
+Eloquence#eloquence_n)))))
@@ -2923 +2923 @@
-have)
+have#have_v)
@@ -2925 +2925 @@
-kept)
+kept#keep_v)
@@ -2927 +2927 @@
-up)
+up#up_adv1)
@@ -2930 +2930 @@
-the)
+the#the_adj)
@@ -2932 +2932 @@
-Dignity)
+Dignity#dignity_n)
@@ -2935 +2935 @@
-of)
+of#of_prep)
@@ -2938 +2938 @@
-Our)
+Our#our_pron)
@@ -2940 +2940 @@
-Mother-Tongue))))
+Mother-Tongue#mother-tongue_n))))
@@ -2943 +2943 @@
-by)
+by#by_prep)
@@ -2948 +2948 @@
-The)
+The#the_adj)
@@ -2950 +2950 @@
-Comprehensive)
+Comprehensive#comprehensive_adj)
@@ -2952 +2952 @@
-English)
+English#english_adj)
@@ -2954 +2954 @@
-Energy)
+Energy#energy_n)
@@ -2961 +2961 @@
-as)
+as#as_adv)
@@ -2970 +2970 @@
-the)
+the#the_adj)
@@ -2972 +2972 @@
-Lord)
+Lord#lord_n)
@@ -2977 +2977 @@
-Roscommon)))
+Roscommon#NA)))
@@ -2981 +2981 @@
-calls)
+calls#call_v)
@@ -2987 +2987 @@
-it)))))))))))
+it#it_pron)))))))))))
@@ -2996 +2996 @@
-particularly))
+particularly#particularly_adv))
@@ -2999 +2999 @@
-that)
+that#that_pron)
@@ -3001 +3001 @@
-Oracle)
+Oracle#oracle_n)
@@ -3004 +3004 @@
-of)
+of#of_prep)
@@ -3007 +3007 @@
-English)
+English#english_adj)
@@ -3009 +3009 @@
-Stile)))
+Stile#style_n)))
@@ -3015 +3015 @@
-who))
+who#who_pron))
@@ -3026 +3026 @@
-Extensive)
+Extensive#extensive_adj)
@@ -3028 +3028 @@
-Sense))
+Sense#sense_n))
@@ -3031 +3031 @@
-still))
+still#still_adv))
@@ -3034 +3034 @@
-into)
+into#into_prep)
@@ -3037 +3037 @@
-Compass)))
+Compass#compass_n1)))
@@ -3039 +3039 @@
-drew))
+drew#draw_v))
@@ -3048 +3048 @@
-Said)
+Said#say_v1)
@@ -3053 +3053 @@
-what))
+what#what_pron))
@@ -3060 +3060 @@
-was)
+was#be_v)
@@ -3063 +3063 @@
-just))))))
+just#just_adj))))))
@@ -3068 +3068 @@
-and)
+and#and_conj1)
@@ -3072 +3072 @@
-always))
+always#always_adv))
@@ -3075 +3075 @@
-something)
+something#something_n)
@@ -3078 +3078 @@
-New))))))))))
+New#new_adj))))))))))
@@ -3089 +3089 @@
-The)
+The#the_adj)
@@ -3092 +3092 @@
-most)
+most#most_adj)
@@ -3094 +3094 @@
-Polite))
+Polite#polite_adj))
@@ -3096 +3096 @@
-Ages)
+Ages#age_n)
@@ -3099 +3099 @@
-of)
+of#of_prep)
@@ -3104 +3104 @@
-Greece)
+Greece#NA)
@@ -3108 +3108 @@
-and)
+and#and_conj1)
@@ -3112 +3112 @@
-Rome))))
+Rome#NA))))
@@ -3116 +3116 @@
-pretend)
+pretend#pretend_v)
@@ -3120 +3120 @@
-to)
+to#to_prep)
@@ -3123 +3123 @@
-but)
+but#but_prep)
@@ -3125 +3125 @@
-One)
+One#one_adj)
@@ -3127 +3127 @@
-Perfect)
+Perfect#perfect_adj)
@@ -3129 +3129 @@
-Pattern)
+Pattern#pattern_n)
@@ -3132 +3132 @@
-of)
+of#of_prep)
@@ -3135 +3135 @@
-Eloquence)))))
+Eloquence#eloquence_n)))))
@@ -3140 +3140 @@
-and)
+and#and_conj1)
@@ -3143 +3143 @@
-that)
+that#that_conj)
@@ -3147 +3147 @@
-in)
+in#in_prep)
@@ -3152 +3152 @@
-Caesar)
+Caesar#NA)
@@ -3156 +3156 @@
-only)))
+only#only_adj)))
@@ -3160 +3160 @@
-cou'd)
+cou'd#can_v1)
@@ -3163 +3163 @@
-no)
+no#no_adj)
@@ -3165 +3165 @@
-Fault))
+Fault#fault_n))
@@ -3167 +3167 @@
-be)
+be#be_v)
@@ -3169 +3169 @@
-found)))))
+found#find_v)))))
@@ -3178 +3178 @@
-Now))
+Now#now_adv))
@@ -3181 +3181 @@
-I))
+I#i_pron))
@@ -3183 +3183 @@
-think)
+think#think_v2)
@@ -3190 +3190 @@
-I))
+I#i_pron))
@@ -3192 +3192 @@
-may)
+may#may_v1)
@@ -3195 +3195 @@
-modestly))
+modestly#modestly_adv))
@@ -3197 +3197 @@
-ask)
+ask#ask_v)
@@ -3202 +3202 @@
-Whether)
+Whether#whether_pron)
@@ -3208 +3208 @@
-that)
+that#that_pron)
@@ -3210 +3210 @@
-small)
+small#small_adj)
@@ -3212 +3212 @@
-Piece)
+Piece#piece_n)
@@ -3217 +3217 @@
-call'd)
+call'd#call_v)
@@ -3225 +3225 @@
-Observations)
+Observations#observation_n)
@@ -3228 +3228 @@
-on)
+on#on_prep)
@@ -3232 +3232 @@
-Monsieur)
+Monsieur#NA)
@@ -3236 +3236 @@
-Sorbier's))
+Sorbier's#NA))
@@ -3240 +3240 @@
-Voyage)
+Voyage#voyage_n)
@@ -3243 +3243 @@
-into)
+into#into_prep)
@@ -3248 +3248 @@
-England)))))))))
+England#NA)))))))))
@@ -3252 +3252 @@
-will)
+will#will_v1)
@@ -3254 +3254 @@
-not)
+not#not_adv)
@@ -3256 +3256 @@
-bear)
+bear#bear_v1)
@@ -3259 +3259 @@
-reading))
+reading#reading_n2))
@@ -3262 +3262 @@
-after)
+after#after_adv)
@@ -3265,3 +3265,3 @@
-the)
-(ADJ
-Celebrated)
+the#the_adj)
+(ADJ
+Celebrated#celebrated_adj)
@@ -3269 +3269 @@
-Commentaries)
+Commentaries#commentary_n)
@@ -3272 +3272 @@
-of)
+of#of_prep)
@@ -3277 +3277 @@
-Caesar)))))))))
+Caesar#NA)))))))))
@@ -3287 +3287 @@
-Nor)
+Nor#nor_conj1)
@@ -3289 +3289 @@
-has)
+has#have_v)
@@ -3292 +3292 @@
-the)
+the#the_adj)
@@ -3294 +3294 @@
-Province)
+Province#province_n)
@@ -3297 +3297 @@
-of)
+of#of_prep)
@@ -3302 +3302 @@
-Poetry))))
+Poetry#poetry_n))))
@@ -3306 +3306 @@
-been)
+been#be_v)
@@ -3309 +3309 @@
-so)
+so#so_adv)
@@ -3311 +3311 @@
-ill)
+ill#ill_adv)
@@ -3315 +3315 @@
-serv'd)
+serv'd#serve_v1)
@@ -3320 +3320 @@
-as)
+as#as_adv)
@@ -3323 +3323 @@
-not)
+not#not_adv)
@@ -3325 +3325 @@
-to)
+to#to_prep)
@@ -3327 +3327 @@
-answer)
+answer#answer_v)
@@ -3330 +3330 @@
-the)
+the#the_adj)
@@ -3332 +3332 @@
-Character)
+Character#character_n)
@@ -3335 +3335 @@
-given)
+given#give_v)
@@ -3338 +3338 @@
-by)
+by#by_prep)
@@ -3341 +3341 @@
-the)
+the#the_adj)
@@ -3343 +3343 @@
-same)
+same#same_adj)
@@ -3345 +3345 @@
-sagacious)
+sagacious#sagacious_adj)
@@ -3347 +3347 @@
-noble)
+noble#noble_adj)
@@ -3349 +3349 @@
-Critic))))
+Critic#critic_n))))
@@ -3354 +3354 @@
-that)
+that#that_conj)
@@ -3362 +3362 @@
-One)
+One#one_adj)
@@ -3364 +3364 @@
-Massy)
+Massy#massy_adj)
@@ -3368 +3368 @@
-English)
+English#english_adj)
@@ -3370 +3370 @@
-Line)
+Line#line_n2)
@@ -3375 +3375 @@
-Drawn)
+Drawn#draw_v)
@@ -3378 +3378 @@
-in)
+in#in_prep)
@@ -3383 +3383 @@
-French)
+French#french_adj)
@@ -3385 +3385 @@
-Wire)))))
+Wire#wire_n)))))
@@ -3391 +3391 @@
-would)
+would#will_v1)
@@ -3394 +3394 @@
-thro')
+thro'#through_prep)
@@ -3397 +3397 @@
-whole)
+whole#whole_adj)
@@ -3401 +3401 @@
-Pages)))
+Pages#page_n2)))
@@ -3405 +3405 @@
-shine))))))
+shine#shine_v))))))
@@ -3416 +3416 @@
-What))
+What#what_pron))
@@ -3421 +3421 @@
-shall)
+shall#shall_v)
@@ -3424 +3424 @@
-we))
+we#we_pron))
@@ -3426 +3426 @@
-say)
+say#say_v1)
@@ -3429 +3429 @@
-then)))
+then#then_adv)))
@@ -3439 +3439 @@
-has)
+has#have_v)
@@ -3442 +3442 @@
-our)
+our#our_pron)
@@ -3446 +3446 @@
-Censor))
+Censor#censor_n))
@@ -3450 +3450 @@
-complain'd)
+complain'd#complain_v)
@@ -3453 +3453 @@
-without)
+without#without_adv)
@@ -3456 +3456 @@
-Cause))))
+Cause#cause_n))))
@@ -3461 +3461 @@
-and)
+and#and_conj1)
@@ -3464 +3464 @@
-given)
+given#give_v)
@@ -3467 +3467 @@
-a)
+a#a_adj2)
@@ -3469 +3469 @@
-false)
+false#false_adj)
@@ -3471 +3471 @@
-Alarm)
+Alarm#alarm_int)
@@ -3474 +3474 @@
-of)
+of#of_prep)
@@ -3477 +3477 @@
-Danger)))
+Danger#danger_n)))
@@ -3482 +3482 @@
-to)
+to#to_prep)
@@ -3485 +3485 @@
-the)
+the#the_adj)
@@ -3487 +3487 @@
-Language)
+Language#language_n)
@@ -3490 +3490 @@
-of)
+of#of_prep)
@@ -3493 +3493 @@
-our)
+our#our_pron)
@@ -3495 +3495 @@
-Country)))))
+Country#country_n)))))
@@ -3500 +3500 @@
-that)
+that#that_conj)
@@ -3504 +3504 @@
-our)
+our#our_pron)
@@ -3506 +3506 @@
-Sterling)
+Sterling#sterling_n)
@@ -3508 +3508 @@
-English))
+English#NA))
@@ -3510 +3510 @@
-is)
+is#be_v)
@@ -3512 +3512 @@
-got)
+got#get_v)
@@ -3515 +3515 @@
-into)
+into#into_prep)
@@ -3518 +3518 @@
-the)
+the#the_adj)
@@ -3520 +3520 @@
-Hands)
+Hands#hand_n)
@@ -3523 +3523 @@
-of)
+of#of_prep)
@@ -3526 +3526 @@
-Clippers)
+Clippers#clipper_n1)
@@ -3528 +3528 @@
-and)
+and#and_conj1)
@@ -3530 +3530 @@
-Coyners))))))))))
+Coyners#coiner_n))))))))))
@@ -3539 +3539 @@
-I))
+I#i_pron))
@@ -3541 +3541 @@
-refer)
+refer#refer_v)
@@ -3544 +3544 @@
-those)
+those#those_pron)
@@ -3549 +3549 @@
-that)
+that#that_conj)
@@ -3554 +3554 @@
-make)
+make#make_v1)
@@ -3557 +3557 @@
-a)
+a#a_adj2)
@@ -3559 +3559 @@
-Doubt)))))
+Doubt#doubt_n1)))))
@@ -3564 +3564 @@
-to)
+to#to_prep)
@@ -3567 +3567 @@
-his)
+his#his_adj)
@@ -3569 +3569 @@
-Remonstrance)
+Remonstrance#remonstrance_n)
@@ -3572 +3572 @@
-of)
+of#of_prep)
@@ -3575 +3575 @@
-the)
+the#the_adj)
@@ -3577 +3577 @@
-Case)))))
+Case#case_n1)))))
@@ -3586 +3586 @@
-I))
+I#i_pron))
@@ -3588 +3588 @@
-crave)
+crave#crave_v)
@@ -3591 +3591 @@
-leave))
+leave#leave_n1))
@@ -3595 +3595 @@
-farther))
+farther#farther_adv))
@@ -3597 +3597 @@
-to)
+to#to_prep)
@@ -3599 +3599 @@
-assert)
+assert#assert_v)
@@ -3604 +3604 @@
-That)
+That#that_conj)
@@ -3608 +3608 @@
-the)
+the#the_adj)
@@ -3610 +3610 @@
-chief)
+chief#chief_adj)
@@ -3612 +3612 @@
-Criminals))
+Criminals#criminal_adj))
@@ -3614 +3614 @@
-have)
+have#have_v)
@@ -3616 +3616 @@
-been)
+been#be_v)
@@ -3619 +3619 @@
-our)
+our#our_pron)
@@ -3621 +3621 @@
-late)
+late#late_adj1)
@@ -3623 +3623 @@
-Pretenders)
+Pretenders#pretender_n)
@@ -3626 +3626 @@
-to)
+to#to_prep)
@@ -3629 +3629 @@
-Wit)))
+Wit#wit_n)))
@@ -3635 +3635 @@
-to)
+to#to_prep)
@@ -3638 +3638 @@
-whom)))
+whom#whom_pron)))
@@ -3646 +3646 @@
-we))
+we#we_pron))
@@ -3648 +3648 @@
-may)
+may#may_v1)
@@ -3650 +3650 @@
-say)
+say#say_v1)
@@ -3655 +3655 @@
-as)
+as#as_adv)
@@ -3661 +3661 @@
-Petronius))
+Petronius#NA))
@@ -3666 +3666 @@
-to)
+to#to_prep)
@@ -3669 +3669 @@
-the)
+the#the_adj)
@@ -3671 +3671 @@
-Wittlings)
+Wittlings#witling_n)
@@ -3674 +3674 @@
-of)
+of#of_prep)
@@ -3677 +3677 @@
-his)
+his#his_adj)
@@ -3679 +3679 @@
-time)))))))
+time#time_n)))))))
@@ -3687 +3687 @@
-Gentlemen))
+Gentlemen#gentleman_n))
@@ -3691 +3691 @@
-give)
+give#give_v)
@@ -3694 +3694 @@
-me))
+me#me_pron1))
@@ -3697 +3697 @@
-leave))
+leave#leave_n1))
@@ -3700 +3700 @@
-to)
+to#to_prep)
@@ -3702 +3702 @@
-tell)
+tell#tell_v)
@@ -3705 +3705 @@
-you))
+you#you_pron))
@@ -3710 +3710 @@
-that)
+that#that_conj)
@@ -3714 +3714 @@
-you))
+you#you_pron))
@@ -3716 +3716 @@
-have)
+have#have_v)
@@ -3718 +3718 @@
-been)
+been#be_v)
@@ -3721 +3721 @@
-the)
+the#the_adj)
@@ -3723 +3723 @@
-Principal)
+Principal#principal_adj)
@@ -3725 +3725 @@
-Persons)
+Persons#person_n)
@@ -3730 +3730 @@
-that)
+that#that_conj)
@@ -3735 +3735 @@
-have)
+have#have_v)
@@ -3737 +3737 @@
-murder'd)
+murder'd#murder_v)
@@ -3740 +3740 @@
-Eloquence)))))))))))))))
+Eloquence#eloquence_n)))))))))))))))
@@ -3751 +3751 @@
-Our)
+Our#our_pron)
@@ -3753 +3753 @@
-Stage-Buffoons)
+Stage-Buffoons#cNA)
@@ -3759 +3759 @@
-from)
+from#from_prep)
@@ -3762 +3762 @@
-puny)
+puny#puny_n)
@@ -3764 +3764 @@
-Actors)))
+Actors#actor_n)))
@@ -3766 +3766 @@
-turn'd)
+turn'd#turn_v)
@@ -3769 +3769 @@
-Authors))))
+Authors#author_n))))
@@ -3773 +3773 @@
-have)
+have#have_v)
@@ -3776 +3776 @@
-at)
+at#at_prep)
@@ -3779 +3779 @@
-once)))
+once#once_adv)))
@@ -3781 +3781 @@
-endeavour'd)
+endeavour'd#endeavour_v)
@@ -3784 +3784 @@
-to)
+to#to_prep)
@@ -3786 +3786 @@
-debauch)
+debauch#debauch_v)
@@ -3789 +3789 @@
-our)
+our#our_pron)
@@ -3791 +3791 @@
-Morals)
+Morals#moral_n)
@@ -3794 +3794 @@
-and)
+and#and_conj1)
@@ -3797 +3797 @@
-Dialect))))
+Dialect#dialect_n))))
@@ -3802 +3802 @@
-by)
+by#by_prep)
@@ -3806 +3806 @@
-senseless)
+senseless#senseless_adj)
@@ -3808 +3808 @@
-Cant))
+Cant#cant_n3))
@@ -3813 +3813 @@
-and)
+and#and_conj1)
@@ -3816 +3816 @@
-new)
+new#new_adj)
@@ -3818 +3818 @@
-Affectations)
+Affectations#affectation_n)
@@ -3821 +3821 @@
-of)
+of#of_prep)
@@ -3824 +3824 @@
-Speech)))))
+Speech#speech_n1)))))
@@ -3831 +3831 @@
-that)
+that#that_conj)
@@ -3837 +3837 @@
-have)
+have#have_v)
@@ -3839 +3839 @@
-catch'd)
+catch'd#catch_v)
@@ -3842 +3842 @@
-the)
+the#the_adj)
@@ -3844 +3844 @@
-Town)))
+Town#town_n)))
@@ -3849 +3849 @@
-and)
+and#and_conj1)
@@ -3852 +3852 @@
-infected)
+infected#infect_v)
@@ -3855 +3855 @@
-Conversation))))))))
+Conversation#conversation_n))))))))
@@ -3860 +3860 @@
-so)
+so#so_adv)
@@ -3863 +3863 @@
-that)
+that#that_conj)
@@ -3867 +3867 @@
-it))
+it#it_pron))
@@ -3869 +3869 @@
-concern'd)
+concern'd#concern_v)
@@ -3872 +3872 @@
-the)
+the#the_adj)
@@ -3876 +3876 @@
-Censor)
+Censor#censor_n)
@@ -3879 +3879 @@
-of)
+of#of_prep)
@@ -3882 +3882 @@
-Great)
+Great#great_adj)
@@ -3884 +3884 @@
-Britain))))
+Britain#NA))))
@@ -3889 +3889 @@
-to)
+to#to_prep)
@@ -3891 +3891 @@
-take)
+take#take_v)
@@ -3894 +3894 @@
-care))
+care#care_n1))
@@ -3897 +3897 @@
-that)
+that#that_conj)
@@ -3901 +3901 @@
-the)
+the#the_adj)
@@ -3903 +3903 @@
-Commonwealth)
+Commonwealth#commonwealth_n)
@@ -3906 +3906 @@
-of)
+of#of_prep)
@@ -3909 +3909 @@
-Letters)))
+Letters#letter_n1)))
@@ -3913 +3913 @@
-come)
+come#come_v)
@@ -3916 +3916 @@
-to)
+to#to_prep)
@@ -3919 +3919 @@
-no)
+no#no_adj)
@@ -3921 +3921 @@
-Damage)))
+Damage#damage_n)))
@@ -3927 +3927 @@
-which))
+which#which_adj))
@@ -3937 +3937 @@
-like)
+like#like_adj)
@@ -3940 +3940 @@
-other)
+other#other_adj)
@@ -3942 +3942 @@
-States)))
+States#state_n)))
@@ -3947 +3947 @@
-when)
+when#when_adv)
@@ -3952 +3952 @@
-arriv'd)
+arriv'd#arrive_v)
@@ -3955 +3955 @@
-to)
+to#to_prep)
@@ -3958 +3958 @@
-Perfection)))))
+Perfection#perfection_n)))))
@@ -3962 +3962 @@
-is)
+is#be_v)
@@ -3965 +3965 @@
-then))
+then#then_adv))
@@ -3968 +3968 @@
-in)
+in#in_prep)
@@ -3971 +3971 @@
-most)
+most#most_adj)
@@ -3973 +3973 @@
-danger)
+danger#danger_n)
@@ -3976 +3976 @@
-of)
+of#of_prep)
@@ -3979 +3979 @@
-falling)
+falling#falling_n1)
@@ -3982 +3982 @@
-to)
+to#to_prep)
@@ -3985 +3985 @@
-Decay))))))))))))))))
+Decay#decay_n))))))))))))))))
@@ -3996 +3996 @@
-J.)
+J.#NA)
@@ -3998 +3998 @@
-B.))
+B.#NA))
@@ -4003 +4003 @@
-upon)
+upon#upon_prep)
@@ -4006 +4006 @@
-the)
+the#the_adj)
@@ -4008 +4008 @@
-great)
+great#great_adj)
@@ -4010 +4010 @@
-Encouragement)
+Encouragement#encouragement_n)
@@ -4013 +4013 @@
-to)
+to#to_prep)
@@ -4016 +4016 @@
-the)
+the#the_adj)
@@ -4018 +4018 @@
-First)
+First#first_adj)
@@ -4020 +4020 @@
-Impression)
+Impression#impression_n)
@@ -4023 +4023 @@
-of)
+of#of_prep)
@@ -4026 +4026 @@
-his)
+his#his_adj)
@@ -4030 +4030 @@
-Grammar)))))))
+Grammar#grammar_n)))))))
@@ -4036 +4036 @@
-has)
+has#have_v)
@@ -4039 +4039 @@
-with)
+with#with_prep)
@@ -4042 +4042 @@
-indefatigable)
+indefatigable#indefatigable_adj)
@@ -4044 +4044 @@
-Industry)))
+Industry#industry_n)))
@@ -4046 +4046 @@
-consulted)
+consulted#consult_v)
@@ -4049 +4049 @@
-all)
+all#all_adj)
@@ -4051 +4051 @@
-the)
+the#the_adj)
@@ -4054 +4054 @@
-Polite)
+Polite#polite_adj)
@@ -4056 +4056 @@
-and)
+and#and_conj1)
@@ -4058 +4058 @@
-Learned))
+Learned#learned_adj))
@@ -4060 +4060 @@
-Men)
+Men#man_n1)
@@ -4065 +4065 @@
-that)
+that#that_conj)
@@ -4072 +4072 @@
-he))
+he#he_pron))
@@ -4075 +4075 @@
-and)
+and#and_conj1)
@@ -4078 +4078 @@
-his)
+his#his_adj)
@@ -4080 +4080 @@
-Friends))))
+Friends#friend_n))))
@@ -4082 +4082 @@
-cou'd)
+cou'd#can_v1)
@@ -4084 +4084 @@
-engage)
+engage#engage_v)
@@ -4089 +4089 @@
-in)
+in#in_prep)
@@ -4092 +4092 @@
-order)
+order#order_n)
@@ -4095 +4095 @@
-to)
+to#to_prep)
@@ -4097 +4097 @@
-obtain)
+obtain#obtain_v)
@@ -4100 +4100 @@
-such)
+such#such_adj)
@@ -4102 +4102 @@
-Improvements)
+Improvements#improvement_n)
@@ -4107 +4107 @@
-as)
+as#as_adv)
@@ -4117 +4117 @@
-might)
+might#may_v1)
@@ -4119 +4119 @@
-make)
+make#make_v1)
@@ -4123 +4123 @@
-it))
+it#it_pron))
@@ -4125 +4125 @@
-approach)
+approach#approach_v)
@@ -4128 +4128 @@
-near)
+near#near_adj)
@@ -4131 +4131 @@
-to)
+to#to_prep)
@@ -4134 +4134 @@
-Perfection))))
+Perfection#perfection_n))))
@@ -4137 +4137 @@
-in)
+in#in_prep)
@@ -4140 +4140 @@
-the)
+the#the_adj)
@@ -4144 +4144 @@
-Second)
+Second#second_adj)
@@ -4146 +4146 @@
-Edition))))))))))))))
+Edition#edition_n))))))))))))))
@@ -4156 +4156 @@
-And)
+And#and_conj1)
@@ -4161 +4161 @@
-to)
+to#to_prep)
@@ -4163 +4163 @@
-render)
+render#render_v)
@@ -4167 +4167 @@
-his)
+his#his_adj)
@@ -4169 +4169 @@
-System)
+System#system_n)
@@ -4172 +4172 @@
-of)
+of#of_prep)
@@ -4175 +4175 @@
-an)
+an#a_adj2)
@@ -4177 +4177 @@
-English)
+English#english_adj)
@@ -4179 +4179 @@
-Education))))
+Education#education_n))))
@@ -4182 +4182 @@
-Compleat))))
+Compleat#complete_adj))))
@@ -4187 +4187 @@
-he))
+he#he_pron))
@@ -4189 +4189 @@
-has)
+has#have_v)
@@ -4191 +4191 @@
-procur'd)
+procur'd#procure_v)
@@ -4194 +4194 @@
-an)
+an#a_adj2)
@@ -4196 +4196 @@
-Addition)
+Addition#addition_n)
@@ -4199 +4199 @@
-of)
+of#of_prep)
@@ -4202 +4202 @@
-the)
+the#the_adj)
@@ -4204 +4204 @@
-Arts)
+Arts#art_n)
@@ -4207 +4207 @@
-of)
+of#of_prep)
@@ -4212 +4212 @@
-Poetry)
+Poetry#poetry_n)
@@ -4216 +4216 @@
-Rhetoric)
+Rhetoric#rhetoric_n1)
@@ -4220 +4220 @@
-and)
+and#and_conj1)
@@ -4224 +4224 @@
-Logic))))))
+Logic#logic_n))))))
@@ -4235 +4235 @@
-In)
+In#in_prep)
@@ -4238 +4238 @@
-the)
+the#the_adj)
@@ -4242 +4242 @@
-Poetry)))
+Poetry#poetry_n)))
@@ -4248 +4248 @@
-is)
+is#be_v)
@@ -4250 +4250 @@
-deliver'd)
+deliver'd#deliver_v1)
@@ -4254 +4254 @@
-not)
+not#not_adv)
@@ -4256,3 +4256,3 @@
-only)
-(D
-the)
+only#only_adj)
+(D
+the#the_adj)
@@ -4260 +4260 @@
-Versification)
+Versification#versification_n)
@@ -4263 +4263 @@
-and)
+and#and_conj1)
@@ -4266 +4266 @@
-Numbers)))
+Numbers#number_n)))
@@ -4269 +4269 @@
-of)
+of#of_prep)
@@ -4272 +4272 @@
-our)
+our#our_pron)
@@ -4274 +4274 @@
-Tongue))))
+Tongue#tongue_n))))
@@ -4279 +4279 @@
-but)
+but#but_prep)
@@ -4281 +4281 @@
-also)
+also#also_adv)
@@ -4284 +4284 @@
-Rules)
+Rules#rule_n1)
@@ -4287 +4287 @@
-for)
+for#for_prep)
@@ -4290 +4290 @@
-the)
+the#the_adj)
@@ -4292 +4292 @@
-several)
+several#several_adj)
@@ -4294 +4294 @@
-Provinces)
+Provinces#province_n)
@@ -4297 +4297 @@
-of)
+of#of_prep)
@@ -4300 +4300 @@
-that)
+that#that_pron)
@@ -4302 +4302 @@
-Useful)
+Useful#useful_adj)
@@ -4304 +4304 @@
-Art)))))
+Art#art_n)))))
@@ -4310 +4310 @@
-which))
+which#which_adj))
@@ -4317 +4317 @@
-will)
+will#will_v1)
@@ -4319 +4319 @@
-fix)
+fix#fix_v)
@@ -4322 +4322 @@
-in)
+in#in_prep)
@@ -4325 +4325 @@
-the)
+the#the_adj)
@@ -4327 +4327 @@
-Minds)
+Minds#mind_n1)
@@ -4330 +4330 @@
-of)
+of#of_prep)
@@ -4333 +4333 @@
-Youth)))))
+Youth#youth_n)))))
@@ -4336 +4336 @@
-a)
+a#a_adj2)
@@ -4338 +4338 @@
-true)
+true#true_adj)
@@ -4340 +4340 @@
-Judgment)
+Judgment#judgement_n)
@@ -4343 +4343 @@
-of)
+of#of_prep)
@@ -4346 +4346 @@
-Performances)
+Performances#performance_n)
@@ -4349 +4349 @@
-in)
+in#in_prep)
@@ -4352 +4352 @@
-that)
+that#that_pron)
@@ -4354 +4354 @@
-kind)))))))))))
+kind#kind_n)))))))))))
@@ -4364 +4364 @@
-The)
+The#the_adj)
@@ -4368 +4368 @@
-Rhetoric))
+Rhetoric#rhetoric_n1))
@@ -4372,3 +4372,3 @@
-shall)
-(NEG
-not)
+shall#shall_v)
+(NEG
+not#not_adv)
@@ -4376 +4376 @@
-only)
+only#only_adj)
@@ -4378 +4378 @@
-be)
+be#be_v)
@@ -4380 +4380 @@
-drawn)
+drawn#draw_v)
@@ -4384 +4384 @@
-from)
+from#from_prep)
@@ -4387 +4387 @@
-the)
+the#the_adj)
@@ -4389 +4389 @@
-Best)
+Best#best_adj)
@@ -4392 +4392 @@
-of)
+of#of_prep)
@@ -4395 +4395 @@
-the)
+the#the_adj)
@@ -4399 +4399 @@
-Ancients)
+Ancients#ancient_adj)
@@ -4404 +4404 @@
-and)
+and#and_conj1)
@@ -4409 +4409 @@
-Moderns)))))))
+Moderns#modern_adj)))))))
@@ -4416 +4416 @@
-and)
+and#and_conj1)
@@ -4419 +4419 @@
-from)
+from#from_prep)
@@ -4422 +4422 @@
-the)
+the#the_adj)
@@ -4424 +4424 @@
-Nature)
+Nature#nature_n)
@@ -4427 +4427 @@
-of)
+of#of_prep)
@@ -4430 +4430 @@
-the)
+the#the_adj)
@@ -4432 +4432 @@
-Thing))))))))
+Thing#thing_n1))))))))
@@ -4437 +4437 @@
-but)
+but#but_prep)
@@ -4440 +4440 @@
-be)
+be#be_v)
@@ -4442 +4442 @@
-deliver'd)
+deliver'd#deliver_v1)
@@ -4445 +4445 @@
-in)
+in#in_prep)
@@ -4448 +4448 @@
-such)
+such#such_adj)
@@ -4450 +4450 @@
-Terms)
+Terms#term_n)
@@ -4455 +4455 @@
-as)
+as#as_adv)
@@ -4465 +4465 @@
-may)
+may#may_v1)
@@ -4469 +4469 @@
-inform)
+inform#inform_v)
@@ -4476 +4476 @@
-not)
+not#not_adv)
@@ -4480 +4480 @@
-puzzle))
+puzzle#puzzle_v))
@@ -4487 +4487 @@
-the)
+the#the_adj)
@@ -4489 +4489 @@
-Learner)))))))))
+Learner#learner_n)))))))))
@@ -4499 +4499 @@
-The)
+The#the_adj)
@@ -4503 +4503 @@
-Logic))
+Logic#logic_n))
@@ -4507 +4507 @@
-will)
+will#will_v1)
@@ -4509 +4509 @@
-be)
+be#be_v)
@@ -4511 +4511 @@
-freed)
+freed#free_v)
@@ -4514 +4514 @@
-from)
+from#from_prep)
@@ -4517 +4517 @@
-the)
+the#the_adj)
@@ -4519 +4519 @@
-useless)
+useless#useless_adj)
@@ -4523 +4523 @@
-Jargon)
+Jargon#jargon_n1)
@@ -4528 +4528 @@
-and)
+and#and_conj1)
@@ -4533 +4533 @@
-Methods)))
+Methods#method_n)))
@@ -4538 +4538 @@
-of)
+of#of_prep)
@@ -4541 +4541 @@
-the)
+the#the_adj)
@@ -4545 +4545 @@
-Schools))))))
+Schools#school_n1))))))
@@ -4552 +4552 @@
-and)
+and#and_conj1)
@@ -4555 +4555 @@
-appear)
+appear#appear_v)
@@ -4558 +4558 @@
-a)
+a#a_adj2)
@@ -4560 +4560 @@
-Just)
+Just#just_adj)
@@ -4562 +4562 @@
-Art)
+Art#art_n)
@@ -4565 +4565 @@
-of)
+of#of_prep)
@@ -4570 +4570 @@
-Reasoning)))
+Reasoning#reasoning_n)))
@@ -4579 +4579 @@
-always))
+always#always_adv))
@@ -4581 +4581 @@
-directed)
+directed#direct_v)
@@ -4584 +4584 @@
-to)
+to#to_prep)
@@ -4587 +4587 @@
-the)
+the#the_adj)
@@ -4589 +4589 @@
-finding)
+finding#finding_n)
@@ -4591 +4591 @@
-out)
+out#out_adv)
@@ -4594 +4594 @@
-of)
+of#of_prep)
@@ -4597 +4597 @@
-Truth))))))
+Truth#truth_n))))))
@@ -4602 +4602 @@
-and)
+and#and_conj1)
@@ -4605 +4605 @@
-not)
+not#not_adv)
@@ -4607 +4607 @@
-for)
+for#for_prep)
@@ -4610 +4610 @@
-the)
+the#the_adj)
@@ -4612 +4612 @@
-support)
+support#support_n)
@@ -4615 +4615 @@
-of)
+of#of_prep)
@@ -4620 +4620 @@
-Squabbles))))))))))
+Squabbles#squabble_n))))))))))
