--- aungier-1674-e3-h.psd.vert	2025-09-10 13:50:35
+++ aungier-1674-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-Lond))
+Lond#NA))
@@ -19 +19 @@
-June)
+June#NA)
@@ -21 +21 @@
-5)
+5#five_v)
@@ -23 +23 @@
-1675))
+1675#NA))
@@ -30 +30 @@
-My)
+My#my_adj)
@@ -32 +32 @@
-Lord))
+Lord#lord_n))
@@ -38 +38 @@
-The)
+The#the_adj)
@@ -40 +40 @@
-scene))
+scene#scene_n))
@@ -42 +42 @@
-being)
+being#be_v)
@@ -45 +45 @@
-much))
+much#much_adj))
@@ -47 +47 @@
-altered)
+altered#alter_v)
@@ -50 +50 @@
-in)
+in#in_prep)
@@ -53 +53 @@
-ye)
+ye#the_adj)
@@ -55 +55 @@
-difference)
+difference#difference_n1)
@@ -58 +58 @@
-between)
+between#between_prep)
@@ -61 +61 @@
-both)
+both#both_pron)
@@ -63 +63 @@
-Houses)))))
+Houses#house_n1)))))
@@ -66 +66 @@
-since)
+since#since_adv)
@@ -69 +69 @@
-my)
+my#my_adj)
@@ -71 +71 @@
-last))))
+last#last_adv))))
@@ -76 +76 @@
-I))
+I#i_pron))
@@ -78 +78 @@
-presume)
+presume#presume_v)
@@ -81 +81 @@
-to)
+to#to_prep)
@@ -83 +83 @@
-give)
+give#give_v)
@@ -86 +86 @@
-yr)
+yr#your_pron)
@@ -88 +88 @@
-Ex=cy=))
+Ex=cy=#excellency_n))
@@ -91 +91 @@
-this)
+this#this_pron)
@@ -93 +93 @@
-short)
+short#short_adj)
@@ -95 +95 @@
-narrative)
+narrative#narrative_n)
@@ -98 +98 @@
-of)
+of#of_prep)
@@ -101 +101 @@
-it)))))
+it#it_pron)))))
@@ -110 +110 @@
-On)
+On#on_prep)
@@ -113 +113 @@
-Tuesday)
+Tuesday#NA)
@@ -116 +116 @@
-last))))
+last#last_adv))))
@@ -119 +119 @@
-the)
+the#the_adj)
@@ -121 +121 @@
-House)
+House#house_n1)
@@ -124 +124 @@
-of)
+of#of_prep)
@@ -127 +127 @@
-Commons))))
+Commons#commons_n))))
@@ -129 +129 @@
-ordered)
+ordered#order_v)
@@ -132 +132 @@
-their)
+their#their_adj)
@@ -134 +134 @@
-Sergeant))
+Sergeant#sergeant_n))
@@ -137 +137 @@
-to)
+to#to_prep)
@@ -139 +139 @@
-take)
+take#take_v)
@@ -142 +142 @@
-into)
+into#into_prep)
@@ -145 +145 @@
-Custodye)))
+Custodye#custody_n)))
@@ -149 +149 @@
-Sir)
+Sir#NA)
@@ -151 +151 @@
-Jo:)
+Jo:#NA)
@@ -153 +153 @@
-Churchill))
+Churchill#NA))
@@ -159 +159 @@
-Mr)
+Mr#NA)
@@ -161 +161 @@
-Serg=t=)
+Serg=t=#NA)
@@ -163 +163 @@
-Pemberton)))
+Pemberton#NA)))
@@ -169 +169 @@
-Mr)
+Mr#NA)
@@ -171 +171 @@
-Serg=t=)
+Serg=t=#NA)
@@ -173 +173 @@
-Pecke)))
+Pecke#NA)))
@@ -176 +176 @@
-and)
+and#and_conj1)
@@ -179 +179 @@
-Mr)
+Mr#NA)
@@ -181 +181 @@
-Charles)
+Charles#NA)
@@ -183 +183 @@
-Porter))))
+Porter#NA))))
@@ -188 +188 @@
-as)
+as#as_adv)
@@ -191 +191 @@
-persons)
+persons#person_n)
@@ -195 +195 @@
-who))
+who#who_pron))
@@ -202 +202 @@
-had)
+had#have_v)
@@ -204 +204 @@
-infringed)
+infringed#infringe_v1)
@@ -207 +207 @@
-the)
+the#the_adj)
@@ -209 +209 @@
-Previledges)
+Previledges#privilege_n)
@@ -212 +212 @@
-of)
+of#of_prep)
@@ -215 +215 @@
-ye)
+ye#the_adj)
@@ -217 +217 @@
-House))))
+House#house_n1))))
@@ -220 +220 @@
-by)
+by#by_prep)
@@ -223 +223 @@
-appearing)
+appearing#appear_v)
@@ -226 +226 @@
-at)
+at#at_prep)
@@ -230 +230 @@
-the)
+the#the_adj)
@@ -232 +232 @@
-Lords'))
+Lords'#NA))
@@ -234 +234 @@
-Barre)))
+Barre#NA)))
@@ -237 +237 @@
-in)
+in#in_prep)
@@ -240 +240 @@
-a)
+a#a_adj2)
@@ -242 +242 @@
-case)
+case#case_n1)
@@ -246 +246 @@
-wherein))
+wherein#wherein_adv))
@@ -254 +254 @@
-Mr.)
+Mr.#NA)
@@ -256 +256 @@
-Dalmahoy)
+Dalmahoy#NA)
@@ -261 +261 @@
-a)
+a#a_adj2)
@@ -263 +263 @@
-member)
+member#member_n)
@@ -266 +266 @@
-of)
+of#of_prep)
@@ -269 +269 @@
-ye)
+ye#the_adj)
@@ -271 +271 @@
-House)))))
+House#house_n1)))))
@@ -275 +275 @@
-was)
+was#be_v)
@@ -277 +277 @@
-concerned))))))))))))
+concerned#concern_v))))))))))))
@@ -285 +285 @@
-And)
+And#and_conj1)
@@ -288 +288 @@
-to)
+to#to_prep)
@@ -291 +291 @@
-the)
+the#the_adj)
@@ -293 +293 @@
-intent)
+intent#intent_n)
@@ -296 +296 @@
-that)
+that#that_conj)
@@ -300 +300 @@
-they))
+they#they_pron))
@@ -302 +302 @@
-might)
+might#may_v1)
@@ -304 +304 @@
-be)
+be#be_v)
@@ -307 +307 @@
-ye)
+ye#the_adj)
@@ -309 +309 @@
-easier))
+easier#easy_adj))
@@ -311 +311 @@
-had)))))
+had#have_v)))))
@@ -316 +316 @@
-Mr)
+Mr#mr_n)
@@ -318 +318 @@
-Speaker))
+Speaker#speaker_n))
@@ -320 +320 @@
-invited)
+invited#invite_v)
@@ -323 +323 @@
-them))
+them#them_pron))
@@ -326 +326 @@
-to)
+to#to_prep)
@@ -329 +329 @@
-dinner))))
+dinner#dinner_n))))
@@ -335 +335 @@
-and)
+and#and_conj1)
@@ -340 +340 @@
-after)
+after#after_adv)
@@ -347 +347 @@
-they))
+they#they_pron))
@@ -349 +349 @@
-had)
+had#have_v)
@@ -351 +351 @@
-made)
+made#make_v1)
@@ -354 +354 @@
-good)
+good#good_adj)
@@ -356 +356 @@
-cheere)))))
+cheere#cheer_n1)))))
@@ -358 +358 @@
-gave)
+gave#give_v)
@@ -361 +361 @@
-them))
+them#them_pron))
@@ -365 +365 @@
-very)
+very#very_adj)
@@ -367 +367 @@
-good))
+good#good_adj))
@@ -369 +369 @@
-advice)
+advice#advice_n)
@@ -372 +372 @@
-to)
+to#to_prep)
@@ -375 +375 @@
-obviate)
+obviate#obviate_v)
@@ -377 +377 @@
-and)
+and#and_conj1)
@@ -379 +379 @@
-prevent))
+prevent#prevent_v))
@@ -382 +382 @@
-any)
+any#any_adj)
@@ -384 +384 @@
-further)
+further#further_adj)
@@ -386 +386 @@
-disputes)
+disputes#dispute_n)
@@ -389 +389 @@
-between)
+between#between_prep)
@@ -392 +392 @@
-both)
+both#both_pron)
@@ -394 +394 @@
-Houses)))))
+Houses#house_n1)))))
@@ -400 +400 @@
-wch))
+wch#which_adj))
@@ -406 +406 @@
-they))
+they#they_pron))
@@ -409 +409 @@
-then))
+then#then_adv))
@@ -411 +411 @@
-seemed)
+seemed#seem_v2)
@@ -414 +414 @@
-to)
+to#to_prep)
@@ -416 +416 @@
-complye)
+complye#comply_v)
@@ -419 +419 @@
-withe)
+withe#with_prep)
@@ -430 +430 @@
-But)
+But#but_prep)
@@ -434 +434 @@
-our)
+our#our_pron)
@@ -436 +436 @@
-Serg=t=))
+Serg=t=#sergeant_n))
@@ -438 +438 @@
-letting)
+letting#let_v1)
@@ -442 +442 @@
-them))
+them#them_pron))
@@ -444 +444 @@
-have)
+have#have_v)
@@ -447 +447 @@
-their)
+their#their_adj)
@@ -449 +449 @@
-liberty))
+liberty#liberty_n))
@@ -452 +452 @@
-upon)
+upon#upon_prep)
@@ -455 +455 @@
-parroll)))))
+parroll#parole_n)))))
@@ -458 +458 @@
-they))
+they#they_pron))
@@ -460 +460 @@
-were)
+were#be_v)
@@ -463 +463 @@
-the)
+the#the_adj)
@@ -465 +465 @@
-next)
+next#next_adj)
@@ -467 +467 @@
-day))
+day#day_n))
@@ -469 +469 @@
-sent)
+sent#send_v1)
@@ -472 +472 @@
-for)
+for#for_prep)
@@ -477 +477 @@
-by)
+by#by_prep)
@@ -480 +480 @@
-ye)
+ye#the_adj)
@@ -482 +482 @@
-Lords))))
+Lords#lord_n))))
@@ -485 +485 @@
-and)
+and#and_conj1)
@@ -489 +489 @@
-att)
+att#at_prep)
@@ -492 +492 @@
-their)
+their#their_adj)
@@ -494 +494 @@
-barre)))
+barre#bar_n1)))
@@ -496 +496 @@
-interrogated)
+interrogated#interrogate_v)
@@ -499 +499 @@
-in)
+in#in_prep)
@@ -502 +502 @@
-all)
+all#all_adj)
@@ -504 +504 @@
-particulars)
+particulars#particular_adj)
@@ -507 +507 @@
-concerning)
+concerning#concerning_prep)
@@ -510 +510 @@
-their)
+their#their_adj)
@@ -512 +512 @@
-committment)))))))
+committment#commitment_n)))))))
@@ -520 +520 @@
-and)
+and#and_conj1)
@@ -524 +524 @@
-had)
+had#have_v)
@@ -527 +527 @@
-thereafter))
+thereafter#thereafter_adv))
@@ -530 +530 @@
-given)
+given#give_v)
@@ -533 +533 @@
-them))
+them#them_pron))
@@ -536 +536 @@
-the)
+the#the_adj)
@@ -538 +538 @@
-protection)
+protection#protection_n)
@@ -541 +541 @@
-of)
+of#of_prep)
@@ -544 +544 @@
-ye)
+ye#the_adj)
@@ -546 +546 @@
-House)))))
+House#NA)))))
@@ -555 +555 @@
-The)
+The#the_adj)
@@ -557 +557 @@
-same)
+same#same_adj)
@@ -559 +559 @@
-day))
+day#day_n))
@@ -562 +562 @@
-ye)
+ye#the_adj)
@@ -564 +564 @@
-Lords))
+Lords#lord_n))
@@ -566 +566 @@
-sent)
+sent#send_v1)
@@ -569 +569 @@
-a)
+a#a_adj2)
@@ -571 +571 @@
-message)
+message#message_n)
@@ -574 +574 @@
-to)
+to#to_prep)
@@ -577 +577 @@
-us)))
+us#us_pron)))
@@ -580 +580 @@
-bye)
+bye#by_prep)
@@ -584 +584 @@
-my)
+my#my_adj)
@@ -586 +586 @@
-Lord)
+Lord#lord_n)
@@ -588 +588 @@
-Chief)
+Chief#chief_adj)
@@ -590 +590 @@
-Justice)
+Justice#justice_n)
@@ -593 +593 @@
-of)
+of#of_prep)
@@ -596 +596 @@
-the)
+the#the_adj)
@@ -598 +598 @@
-Common)
+Common#NA)
@@ -600 +600 @@
-Pleas))))
+Pleas#NA))))
@@ -603 +603 @@
-and)
+and#and_conj1)
@@ -606 +606 @@
-my)
+my#my_adj)
@@ -608 +608 @@
-Lo:)
+Lo:#lord_n)
@@ -610 +610 @@
-Chiefe)
+Chiefe#chief_adj)
@@ -612 +612 @@
-Baron)))))
+Baron#baron_n)))))
@@ -615 +615 @@
-for)
+for#for_prep)
@@ -618 +618 @@
-a)
+a#a_adj2)
@@ -620 +620 @@
-present)
+present#present_adj)
@@ -622 +622 @@
-Conference)
+Conference#conference_n)
@@ -625 +625 @@
-upon)
+upon#upon_prep)
@@ -628 +628 @@
-matters)
+matters#matter_n1)
@@ -631 +631 @@
-of)
+of#of_prep)
@@ -634 +634 @@
-high)
+high#high_adj)
@@ -636 +636 @@
-importance)))
+importance#importance_n)))
@@ -640 +640 @@
-wherein))
+wherein#wherein_adv))
@@ -650 +650 @@
-his)
+his#his_adj)
@@ -652 +652 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -654 +654 @@
-honour))
+honour#honour_n))
@@ -657 +657 @@
-and)
+and#and_conj1)
@@ -660 +660 @@
-ye)
+ye#the_adj)
@@ -662 +662 @@
-safety)
+safety#safety_n)
@@ -665 +665 @@
-of)
+of#of_prep)
@@ -668 +668 @@
-ye)
+ye#the_adj)
@@ -670 +670 @@
-Kingdom))))))
+Kingdom#kingdom_n))))))
@@ -672 +672 @@
-was)
+was#be_v)
@@ -674 +674 @@
-concerned))))))))
+concerned#concern_v))))))))
@@ -682 +682 @@
-and)
+and#and_conj1)
@@ -686 +686 @@
-the)
+the#the_adj)
@@ -688 +688 @@
-Houses))
+Houses#house_n1))
@@ -690 +690 @@
-agreeing)
+agreeing#agree_v)
@@ -693 +693 @@
-to)
+to#to_prep)
@@ -696 +696 @@
-ye)
+ye#the_adj)
@@ -698 +698 @@
-present)
+present#present_adj)
@@ -700 +700 @@
-Conference))))
+Conference#conference_n))))
@@ -703 +703 @@
-the)
+the#the_adj)
@@ -705 +705 @@
-inclosed)
+inclosed#enclosed_adj)
@@ -707 +707 @@
-paper))
+paper#paper_n))
@@ -709 +709 @@
-was)
+was#be_v)
@@ -711 +711 @@
-read)
+read#read_v)
@@ -714 +714 @@
-with)
+with#with_prep)
@@ -717 +717 @@
-a)
+a#a_adj2)
@@ -720 +720 @@
-very)
+very#very_adj)
@@ -722 +722 @@
-audible))
+audible#audible_adj))
@@ -724 +724 @@
-voice)))
+voice#voice_n)))
@@ -727 +727 @@
-by)
+by#by_prep)
@@ -730 +730 @@
-my)
+my#my_adj)
@@ -732 +732 @@
-Lord)
+Lord#lord_n)
@@ -734 +734 @@
-Privy)
+Privy#NA)
@@ -736 +736 @@
-Seale)))
+Seale#NA)))
@@ -745 +745 @@
-On)
+On#on_prep)
@@ -748 +748 @@
-Thursday)))
+Thursday#NA)))
@@ -751 +751 @@
-the)
+the#the_adj)
@@ -753 +753 @@
-House))
+House#NA))
@@ -755 +755 @@
-sent)
+sent#send_v1)
@@ -758 +758 @@
-their)
+their#their_adj)
@@ -760 +760 @@
-Sergeant))
+Sergeant#sergeant_n))
@@ -763 +763 @@
-to)
+to#to_prep)
@@ -766 +766 @@
-ye)
+ye#the_adj)
@@ -768 +768 @@
-tower)))
+tower#NA)))
@@ -771 +771 @@
-for)
+for#for_prep)
@@ -774 +774 @@
-neglect)
+neglect#neglect_n)
@@ -779 +779 @@
-of)
+of#of_prep)
@@ -782 +782 @@
-his)
+his#his_adj)
@@ -784 +784 @@
-duty)))))
+duty#duty_n)))))
@@ -787 +787 @@
-in)
+in#in_prep)
@@ -790 +790 @@
-suffering)
+suffering#suffer_v)
@@ -793 +793 @@
-his)
+his#his_adj)
@@ -795 +795 @@
-prisoners))
+prisoners#prisoner_n2))
@@ -798 +798 @@
-to)
+to#to_prep)
@@ -800 +800 @@
-escape)))))
+escape#escape_v)))))
@@ -806 +806 @@
-and)
+and#and_conj1)
@@ -810 +810 @@
-ordered)
+ordered#order_v)
@@ -813 +813 @@
-Mr)
+Mr#mr_n)
@@ -815 +815 @@
-Secretary))
+Secretary#secretary_n1))
@@ -819 +819 @@
-in)
+in#in_prep)
@@ -822 +822 @@
-their)
+their#their_adj)
@@ -824 +824 @@
-name)))
+name#name_n)))
@@ -826 +826 @@
-to)
+to#to_prep)
@@ -828 +828 @@
-beseech)
+beseech#beseech_v)
@@ -831 +831 @@
-his)
+his#his_adj)
@@ -833 +833 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -836 +836 @@
-that)
+that#that_conj)
@@ -840 +840 @@
-wee))
+wee#we_pron))
@@ -842 +842 @@
-might)
+might#may_v1)
@@ -844 +844 @@
-have)
+have#have_v)
@@ -848 +848 @@
-another)
+another#another_pron)
@@ -850 +850 @@
-Sergeant))
+Sergeant#sergeant_n))
@@ -852 +852 @@
-appointed)
+appointed#appoint_v)
@@ -855 +855 @@
-to)
+to#to_prep)
@@ -857 +857 @@
-attend)
+attend#attend_v)
@@ -860 +860 @@
-them)))))))
+them#them_pron)))))))
@@ -868 +868 @@
-and)
+and#and_conj1)
@@ -871 +871 @@
-accordingly))
+accordingly#accordingly_adv))
@@ -874 +874 @@
-my)
+my#my_adj)
@@ -876 +876 @@
-Lord)
+Lord#lord_n)
@@ -878 +878 @@
-Chamberlaine))
+Chamberlaine#chamberlain_n))
@@ -880 +880 @@
-appoynted)
+appoynted#appoint_v)
@@ -883 +883 @@
-one)
+one#one_adj)
@@ -885 +885 @@
-Topham)
+Topham#NA)
@@ -888 +888 @@
-a)
+a#a_adj2)
@@ -891 +891 @@
-very)
+very#very_adj)
@@ -893 +893 @@
-stout))
+stout#stout_adj))
@@ -895 +895 @@
-fellow)))
+fellow#fellow_n)))
@@ -898 +898 @@
-to)
+to#to_prep)
@@ -900 +900 @@
-attend)
+attend#attend_v)
@@ -903 +903 @@
-ye)
+ye#the_adj)
@@ -905 +905 @@
-House)))
+House#NA)))
@@ -914 +914 @@
-Yesterday)
+Yesterday#yesterday_adv)
@@ -916 +916 @@
-morning))
+morning#morning_n))
@@ -919 +919 @@
-as)
+as#as_adv)
@@ -926 +926 @@
-Mr)
+Mr#mr_n)
@@ -928 +928 @@
-Speaker))
+Speaker#speaker_n))
@@ -930 +930 @@
-was)
+was#be_v)
@@ -932 +932 @@
-coming)
+coming#come_v)
@@ -935 +935 @@
-through)
+through#through_prep)
@@ -938 +938 @@
-Westminster)
+Westminster#NA)
@@ -940 +940 @@
-Hall)))
+Hall#NA)))
@@ -943 +943 @@
-to)
+to#to_prep)
@@ -946 +946 @@
-ye)
+ye#the_adj)
@@ -948 +948 @@
-House))))))
+House#NA))))))
@@ -951 +951 @@
-he))
+he#he_pron))
@@ -953 +953 @@
-mett)
+mett#meet_v)
@@ -956 +956 @@
-Mr)
+Mr#NA)
@@ -958 +958 @@
-Serg=t=)
+Serg=t=#NA)
@@ -960 +960 @@
-Pemberton)
+Pemberton#NA)
@@ -964 +964 @@
-whom))
+whom#whom_pron))
@@ -970 +970 @@
-he))
+he#he_pron))
@@ -972 +972 @@
-commanded)
+commanded#command_v)
@@ -975 +975 @@
-his)
+his#his_adj)
@@ -977 +977 @@
-officers)
+officers#officer_n)
@@ -981 +981 @@
-then))
+then#then_adv))
@@ -983 +983 @@
-attending)))
+attending#attend_v)))
@@ -988 +988 @@
-to)
+to#to_prep)
@@ -991 +991 @@
-apprehend)
+apprehend#apprehend_v)
@@ -993 +993 @@
-and)
+and#and_conj1)
@@ -995 +995 @@
-secure))
+secure#secure_v))
@@ -998 +998 @@
-in)
+in#in_prep)
@@ -1002 +1002 @@
-ye)
+ye#the_adj)
@@ -1004 +1004 @@
-Speaker's))
+Speaker's#speaker_n))
@@ -1006 +1006 @@
-Chamber)))))))
+Chamber#chamber_n)))))))
@@ -1012 +1012 @@
-of)
+of#of_prep)
@@ -1015 +1015 @@
-wch)))
+wch#which_adj)))
@@ -1021 +1021 @@
-he))
+he#he_pron))
@@ -1023 +1023 @@
-gave)
+gave#give_v)
@@ -1026 +1026 @@
-ye)
+ye#the_adj)
@@ -1028 +1028 @@
-House))
+House#NA))
@@ -1031 +1031 @@
-an)
+an#a_adj2)
@@ -1033 +1033 @@
-acc=t=)
+acc=t=#account_n)
@@ -1038 +1038 @@
-as)
+as#as_adv)
@@ -1040 +1040 @@
-soone)
+soone#soon_adv)
@@ -1043 +1043 @@
-as)
+as#as_adv)
@@ -1054 +1054 @@
-they))
+they#they_pron))
@@ -1056 +1056 @@
-sate)))))))
+sate#sit_v)))))))
@@ -1064 +1064 @@
-and)
+and#and_conj1)
@@ -1068 +1068 @@
-receaved)
+receaved#receive_v)
@@ -1071 +1071 @@
-the)
+the#the_adj)
@@ -1073 +1073 @@
-thanks)
+thanks#thank_n)
@@ -1076 +1076 @@
-of)
+of#of_prep)
@@ -1079 +1079 @@
-ye)
+ye#the_adj)
@@ -1081 +1081 @@
-House))))
+House#NA))))
@@ -1084 +1084 @@
-for)
+for#for_prep)
@@ -1087 +1087 @@
-his)
+his#his_adj)
@@ -1089 +1089 @@
-zeale)
+zeale#zeal_n)
@@ -1092 +1092 @@
-and)
+and#and_conj1)
@@ -1095 +1095 @@
-care)))
+care#care_n1)))
@@ -1098 +1098 @@
-in)
+in#in_prep)
@@ -1102 +1102 @@
-thus))
+thus#thus_adv))
@@ -1104 +1104 @@
-asserting)
+asserting#assert_v)
@@ -1107 +1107 @@
-their)
+their#their_adj)
@@ -1109 +1109 @@
-previledges))))))
+previledges#privilege_n))))))
@@ -1117 +1117 @@
-and)
+and#and_conj1)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-House))
+House#NA))
@@ -1125 +1125 @@
-being)
+being#be_v)
@@ -1128 +1128 @@
-then))
+then#then_adv))
@@ -1130 +1130 @@
-alsoe)
+alsoe#also_adv)
@@ -1132 +1132 @@
-informed)
+informed#inform_v)
@@ -1135 +1135 @@
-that)
+that#that_conj)
@@ -1140 +1140 @@
-Sir)
+Sir#NA)
@@ -1142 +1142 @@
-John)
+John#NA)
@@ -1144 +1144 @@
-Churchill))
+Churchill#NA))
@@ -1148 +1148 @@
-Mr)
+Mr#NA)
@@ -1150 +1150 @@
-Serg=t=)
+Serg=t=#NA)
@@ -1152 +1152 @@
-Peeke)))
+Peeke#NA)))
@@ -1155 +1155 @@
-and)
+and#and_conj1)
@@ -1158 +1158 @@
-Mr.)
+Mr.#NA)
@@ -1160 +1160 @@
-Charley)
+Charley#NA)
@@ -1162 +1162 @@
-Porter))))
+Porter#NA))))
@@ -1164 +1164 @@
-were)
+were#be_v)
@@ -1167 +1167 @@
-belowe))
+belowe#below_adv))
@@ -1170 +1170 @@
-in)
+in#in_prep)
@@ -1173 +1173 @@
-ye)
+ye#the_adj)
@@ -1175 +1175 @@
-Hall))))))
+Hall#hall_n1))))))
@@ -1180 +1180 @@
-they))
+they#they_pron))
@@ -1182 +1182 @@
-commanded)
+commanded#command_v)
@@ -1185 +1185 @@
-instantly))
+instantly#instantly_adv))
@@ -1188 +1188 @@
-their)
+their#their_adj)
@@ -1190 +1190 @@
-Serg=t=))
+Serg=t=#sergeant_n))
@@ -1194 +1194 @@
-to)
+to#to_prep)
@@ -1196 +1196 @@
-carry)
+carry#carry_v)
@@ -1199 +1199 @@
-his)
+his#his_adj)
@@ -1201 +1201 @@
-men))
+men#man_n1))
@@ -1204 +1204 @@
-with)
+with#with_prep)
@@ -1207 +1207 @@
-him)))
+him#him_pron)))
@@ -1210 +1210 @@
-downe)
+downe#down_adv)
@@ -1212 +1212 @@
-into)
+into#into_prep)
@@ -1215 +1215 @@
-ye)
+ye#the_adj)
@@ -1217 +1217 @@
-Hall))))
+Hall#hall_n1))))
@@ -1220 +1220 @@
-and)
+and#and_conj1)
@@ -1223 +1223 @@
-to)
+to#to_prep)
@@ -1225 +1225 @@
-apprehend)
+apprehend#apprehend_v)
@@ -1228 +1228 @@
-ye)
+ye#the_adj)
@@ -1230 +1230 @@
-sayd)
+sayd#said_adj)
@@ -1232 +1232 @@
-persons))
+persons#person_n))
@@ -1235 +1235 @@
-though)
+though#though_adv)
@@ -1242 +1242 @@
-they))
+they#they_pron))
@@ -1244 +1244 @@
-were)
+were#be_v)
@@ -1246 +1246 @@
-pleading)
+pleading#plead_v)
@@ -1249 +1249 @@
-at)
+at#at_prep)
@@ -1252 +1252 @@
-ye)
+ye#the_adj)
@@ -1254 +1254 @@
-Barres)
+Barres#bar_n1)
@@ -1257 +1257 @@
-of)
+of#of_prep)
@@ -1260 +1260 @@
-any)
+any#any_adj)
@@ -1263 +1263 @@
-of)
+of#of_prep)
@@ -1266 +1266 @@
-ye)
+ye#the_adj)
@@ -1268 +1268 @@
-Courtes))))))))))))
+Courtes#court_n1))))))))))))
@@ -1273 +1273 @@
-and)
+and#and_conj1)
@@ -1276 +1276 @@
-to)
+to#to_prep)
@@ -1278 +1278 @@
-goe)
+goe#go_v)
@@ -1281 +1281 @@
-well)
+well#well_adv)
@@ -1283 +1283 @@
-attended))
+attended#attended_adj))
@@ -1286 +1286 @@
-that)
+that#that_conj)
@@ -1290 +1290 @@
-he))
+he#he_pron))
@@ -1292 +1292 @@
-might)
+might#may_v1)
@@ -1294 +1294 @@
-not)
+not#not_adv)
@@ -1296 +1296 @@
-receave)
+receave#receive_v)
@@ -1299 +1299 @@
-an)
+an#a_adj2)
@@ -1301 +1301 @@
-affront)
+affront#affront_n)
@@ -1304 +1304 @@
-from)
+from#from_prep)
@@ -1307 +1307 @@
-ye)
+ye#the_adj)
@@ -1309 +1309 @@
-Black-Rodde)
+Black-Rodde#black-rod_n)
@@ -1313 +1313 @@
-whom))
+whom#whom_pron))
@@ -1319 +1319 @@
-they))
+they#they_pron))
@@ -1321 +1321 @@
-understood)
+understood#understand_v)
@@ -1329 +1329 @@
-was)
+was#be_v)
@@ -1332 +1332 @@
-in)
+in#in_prep)
@@ -1335 +1335 @@
-ye)
+ye#the_adj)
@@ -1337 +1337 @@
-Court)
+Court#NA)
@@ -1340 +1340 @@
-of)
+of#of_prep)
@@ -1343 +1343 @@
-Requests)))))
+Requests#NA)))))
@@ -1345 +1345 @@
-armed)
+armed#arm_v1)
@@ -1348 +1348 @@
-with)
+with#with_prep)
@@ -1351 +1351 @@
-an)
+an#a_adj2)
@@ -1353 +1353 @@
-order)
+order#order_n)
@@ -1356 +1356 @@
-from)
+from#from_prep)
@@ -1359 +1359 @@
-ye)
+ye#the_adj)
@@ -1361 +1361 @@
-House)
+House#house_n1)
@@ -1364 +1364 @@
-of)
+of#of_prep)
@@ -1367 +1367 @@
-Lords)))))
+Lords#lord_n)))))
@@ -1370 +1370 @@
-to)
+to#to_prep)
@@ -1372 +1372 @@
-rescue)
+rescue#rescue_v)
@@ -1375 +1375 @@
-the)
+the#the_adj)
@@ -1377 +1377 @@
-Prisoners)))))))))))))))))
+Prisoners#prisoner_n2)))))))))))))))))
@@ -1386 +1386 @@
-This)
+This#this_pron)
@@ -1388 +1388 @@
-order)
+order#order_n)
@@ -1391 +1391 @@
-of)
+of#of_prep)
@@ -1394 +1394 @@
-the)
+the#the_adj)
@@ -1396 +1396 @@
-House)
+House#house_n1)
@@ -1399 +1399 @@
-of)
+of#of_prep)
@@ -1402 +1402 @@
-Commons))))))
+Commons#commons_n))))))
@@ -1404 +1404 @@
-was)
+was#be_v)
@@ -1407 +1407 @@
-very)
+very#very_adj)
@@ -1409 +1409 @@
-briskly))
+briskly#briskly_adv))
@@ -1411 +1411 @@
-executed)
+executed#execute_v)
@@ -1416 +1416 @@
-the)
+the#the_adj)
@@ -1418 +1418 @@
-said)
+said#said_adj)
@@ -1420 +1420 @@
-persons))
+persons#person_n))
@@ -1422 +1422 @@
-being)
+being#be_v)
@@ -1424 +1424 @@
-taken)
+taken#take_v)
@@ -1427 +1427 @@
-from)
+from#from_prep)
@@ -1430 +1430 @@
-ye)
+ye#the_adj)
@@ -1432 +1432 @@
-severall)
+severall#several_adj)
@@ -1434 +1434 @@
-Barrs)
+Barrs#bar_n1)
@@ -1437 +1437 @@
-of)
+of#of_prep)
@@ -1440 +1440 @@
-ye)
+ye#the_adj)
@@ -1442 +1442 @@
-Chancery)
+Chancery#NA)
@@ -1448 +1448 @@
-King's)
+King's#NA)
@@ -1450 +1450 @@
-bench)))
+bench#NA)))
@@ -1453 +1453 @@
-and)
+and#and_conj1)
@@ -1456 +1456 @@
-Common)
+Common#NA)
@@ -1458 +1458 @@
-Pleas))))))))
+Pleas#NA))))))))
@@ -1463 +1463 @@
-and)
+and#and_conj1)
@@ -1466 +1466 @@
-brought)
+brought#bring_v)
@@ -1469 +1469 @@
-safe))
+safe#safe_adj))
@@ -1472 +1472 @@
-into)
+into#into_prep)
@@ -1476 +1476 @@
-ye)
+ye#the_adj)
@@ -1478 +1478 @@
-Speakers))
+Speakers#speaker_n))
@@ -1480 +1480 @@
-Chamber)))
+Chamber#chamber_n)))
@@ -1483 +1483 @@
-without)
+without#without_adv)
@@ -1486 +1486 @@
-opposition))))))
+opposition#opposition_n))))))
@@ -1494 +1494 @@
-But)
+But#but_prep)
@@ -1498 +1498 @@
-ye)
+ye#the_adj)
@@ -1500 +1500 @@
-House)
+House#house_n1)
@@ -1503 +1503 @@
-of)
+of#of_prep)
@@ -1506 +1506 @@
-Lords))))
+Lords#lord_n))))
@@ -1508 +1508 @@
-being)
+being#be_v)
@@ -1510 +1510 @@
-informed)
+informed#inform_v)
@@ -1513 +1513 @@
-of)
+of#of_prep)
@@ -1516 +1516 @@
-these)
+these#these_pron)
@@ -1518 +1518 @@
-proceedings))))
+proceedings#proceeding_n))))
@@ -1521 +1521 @@
-they))
+they#they_pron))
@@ -1524 +1524 @@
-immediately))
+immediately#immediately_adv))
@@ -1526 +1526 @@
-order)
+order#order_v)
@@ -1529 +1529 @@
-their)
+their#their_adj)
@@ -1531 +1531 @@
-Black)
+Black#black_adj)
@@ -1533 +1533 @@
-Rodde))
+Rodde#rod_n1))
@@ -1536 +1536 @@
-to)
+to#to_prep)
@@ -1538 +1538 @@
-fetch)
+fetch#fetch_v)
@@ -1540 +1540 @@
-out)
+out#out_adv)
@@ -1543 +1543 @@
-ye)
+ye#the_adj)
@@ -1545 +1545 @@
-sayd)
+sayd#said_adj)
@@ -1547 +1547 @@
-prisoners))
+prisoners#prisoner_n2))
@@ -1552 +1552 @@
-wherever))
+wherever#wherever_adv))
@@ -1558 +1558 @@
-they))
+they#they_pron))
@@ -1560 +1560 @@
-should)
+should#shall_v)
@@ -1562 +1562 @@
-find)
+find#find_v)
@@ -1568 +1568 @@
-them))
+them#them_pron))
@@ -1570 +1570 @@
-committed))))))
+committed#commit_v))))))
@@ -1579 +1579 @@
-they))
+they#they_pron))
@@ -1582 +1582 @@
-likewise))
+likewise#likewise_adv))
@@ -1584 +1584 @@
-order)
+order#order_v)
@@ -1587 +1587 @@
-their)
+their#their_adj)
@@ -1589 +1589 @@
-black)
+black#black_adj)
@@ -1591 +1591 @@
-Rod))
+Rod#rod_n1))
@@ -1594 +1594 @@
-to)
+to#to_prep)
@@ -1596 +1596 @@
-apprehend)
+apprehend#apprehend_v)
@@ -1599 +1599 @@
-ye)
+ye#the_adj)
@@ -1601 +1601 @@
-Sergeant)
+Sergeant#sergeant_n)
@@ -1604 +1604 @@
-of)
+of#of_prep)
@@ -1607 +1607 @@
-ye)
+ye#the_adj)
@@ -1609 +1609 @@
-House)
+House#house_n1)
@@ -1612 +1612 @@
-of)
+of#of_prep)
@@ -1615 +1615 @@
-Commons)))))))
+Commons#commons_n)))))))
@@ -1623 +1623 @@
-and)
+and#and_conj1)
@@ -1627 +1627 @@
-addresse)
+addresse#address_v)
@@ -1630 +1630 @@
-to)
+to#to_prep)
@@ -1633 +1633 @@
-his)
+his#his_adj)
@@ -1635 +1635 @@
-Ma=tie=)))
+Ma=tie=#majesty_n)))
@@ -1638 +1638 @@
-by)
+by#by_prep)
@@ -1641 +1641 @@
-word)
+word#word_n)
@@ -1644 +1644 @@
-of)
+of#of_prep)
@@ -1647 +1647 @@
-ye)
+ye#the_adj)
@@ -1649 +1649 @@
-white-staves)))))
+white-staves#white-staff_n)))))
@@ -1652 +1652 @@
-that)
+that#that_conj)
@@ -1656 +1656 @@
-another)
+another#another_pron)
@@ -1658 +1658 @@
-Sergeant))
+Sergeant#sergeant_n))
@@ -1660 +1660 @@
-might)
+might#may_v1)
@@ -1662 +1662 @@
-be)
+be#be_v)
@@ -1664 +1664 @@
-appoynted)
+appoynted#appoint_v)
@@ -1667 +1667 @@
-to)
+to#to_prep)
@@ -1669 +1669 @@
-attend)
+attend#attend_v)
@@ -1672 +1672 @@
-ye)
+ye#the_adj)
@@ -1674 +1674 @@
-House)
+House#house_n1)
@@ -1677 +1677 @@
-of)
+of#of_prep)
@@ -1680 +1680 @@
-Com=ns=)))))))
+Com=ns=#commons_n)))))))
@@ -1689 +1689 @@
-The)
+The#the_adj)
@@ -1691 +1691 @@
-House)
+House#house_n1)
@@ -1694 +1694 @@
-of)
+of#of_prep)
@@ -1697 +1697 @@
-Com=ns=))))
+Com=ns=#commons_n))))
@@ -1700 +1700 @@
-after)
+after#after_adv)
@@ -1703 +1703 @@
-a)
+a#a_adj2)
@@ -1705 +1705 @@
-long)
+long#long_adj1)
@@ -1707 +1707 @@
-debate)))
+debate#debate_n1)))
@@ -1709 +1709 @@
-vote)
+vote#vote_v)
@@ -1713 +1713 @@
-the)
+the#the_adj)
@@ -1715 +1715 @@
-said)
+said#said_adj)
@@ -1717 +1717 @@
-persons))
+persons#person_n))
@@ -1719 +1719 @@
-to)
+to#to_prep)
@@ -1721 +1721 @@
-be)
+be#be_v)
@@ -1723 +1723 @@
-sent)
+sent#send_v1)
@@ -1726 +1726 @@
-to)
+to#to_prep)
@@ -1729 +1729 @@
-ye)
+ye#the_adj)
@@ -1731 +1731 @@
-Tower)))))
+Tower#NA)))))
@@ -1737 +1737 @@
-and)
+and#and_conj1)
@@ -1740 +1740 @@
-Sir)
+Sir#NA)
@@ -1742 +1742 @@
-John)
+John#NA)
@@ -1744 +1744 @@
-Robinson)
+Robinson#NA)
@@ -1747 +1747 @@
-Ser=t=)
+Ser=t=#servant_n)
@@ -1750 +1750 @@
-of)
+of#of_prep)
@@ -1753 +1753 @@
-ye)
+ye#the_adj)
@@ -1755 +1755 @@
-Tower)))))
+Tower#NA)))))
@@ -1758 +1758 @@
-being)
+being#be_v)
@@ -1761 +1761 @@
-then))
+then#then_adv))
@@ -1764 +1764 @@
-in)
+in#in_prep)
@@ -1767 +1767 @@
-ye)
+ye#the_adj)
@@ -1769 +1769 @@
-House))))
+House#NA))))
@@ -1771 +1771 @@
-had)
+had#have_v)
@@ -1774 +1774 @@
-directions)
+directions#direction_n)
@@ -1777 +1777 @@
-not)
+not#not_adv)
@@ -1779 +1779 @@
-to)
+to#to_prep)
@@ -1781 +1781 @@
-sett)
+sett#set_v)
@@ -1784 +1784 @@
-them))
+them#them_pron))
@@ -1787 +1787 @@
-at)
+at#at_prep)
@@ -1790 +1790 @@
-liberty)))
+liberty#liberty_n)))
@@ -1793 +1793 @@
-without)
+without#without_adv)
@@ -1796 +1796 @@
-an)
+an#a_adj2)
@@ -1798 +1798 @@
-order)
+order#order_n)
@@ -1801 +1801 @@
-from)
+from#from_prep)
@@ -1804 +1804 @@
-ye)
+ye#the_adj)
@@ -1806 +1806 @@
-House)))))))
+House#NA)))))))
@@ -1815 +1815 @@
-By)
+By#by_prep)
@@ -1818 +1818 @@
-this)
+this#this_pron)
@@ -1820 +1820 @@
-time)))
+time#time_n)))
@@ -1823 +1823 @@
-it))
+it#it_pron))
@@ -1825 +1825 @@
-was)
+was#be_v)
@@ -1828 +1828 @@
-twoe)
+twoe#two_adj)
@@ -1831 +1831 @@
-of)
+of#of_prep)
@@ -1834 +1834 @@
-ye)
+ye#the_adj)
@@ -1836 +1836 @@
-clock)))))
+clock#clock_n1)))))
@@ -1842 +1842 @@
-and)
+and#and_conj1)
@@ -1845 +1845 @@
-ye)
+ye#the_adj)
@@ -1847 +1847 @@
-House)
+House#house_n1)
@@ -1850 +1850 @@
-of)
+of#of_prep)
@@ -1853 +1853 @@
-Lords)))
+Lords#lord_n)))
@@ -1859 +1859 @@
-had)
+had#have_v)
@@ -1861 +1861 @@
-adjourned)
+adjourned#adjourn_v)
@@ -1864 +1864 @@
-for)
+for#for_prep)
@@ -1868 +1868 @@
-an)
+an#a_adj2)
@@ -1870 +1870 @@
-hour))
+hour#hour_n))
@@ -1873 +1873 @@
-or)
+or#or_conj2)
@@ -1876 +1876 @@
-two)))))
+two#two_adj)))))
@@ -1880 +1880 @@
-whose)
+whose#whose_pron)
@@ -1882 +1882 @@
-example)
+example#example_n)
@@ -1885 +1885 @@
-in)
+in#in_prep)
@@ -1888 +1888 @@
-yt)
+yt#that_pron)
@@ -1890 +1890 @@
-particular))))
+particular#particular_adj))))
@@ -1896 +1896 @@
-ye)
+ye#the_adj)
@@ -1898 +1898 @@
-House)
+House#house_n1)
@@ -1901 +1901 @@
-of)
+of#of_prep)
@@ -1904 +1904 @@
-Commons))))
+Commons#commons_n))))
@@ -1906 +1906 @@
-thought)
+thought#think_v2)
@@ -1912 +1912 @@
-fitt)
+fitt#fit_adj)
@@ -1920 +1920 @@
-to)
+to#to_prep)
@@ -1922 +1922 @@
-follow)))))))
+follow#follow_v)))))))
@@ -1930 +1930 @@
-And)
+And#and_conj1)
@@ -1933 +1933 @@
-while)
+while#while_adv)
@@ -1940 +1940 @@
-ye)
+ye#the_adj)
@@ -1942 +1942 @@
-Speaker))
+Speaker#speaker_n))
@@ -1944 +1944 @@
-was)
+was#be_v)
@@ -1947 +1947 @@
-at)
+at#at_prep)
@@ -1950 +1950 @@
-dinner))))))
+dinner#dinner_n))))))
@@ -1953 +1953 @@
-the)
+the#the_adj)
@@ -1955 +1955 @@
-Sergeant))
+Sergeant#sergeant_n))
@@ -1957 +1957 @@
-hired)
+hired#hire_v)
@@ -1960 +1960 @@
-three)
+three#three_adj)
@@ -1962 +1962 @@
-coaches))
+coaches#coach_n))
@@ -1965 +1965 @@
-at)
+at#at_prep)
@@ -1968 +1968 @@
-Westminster)
+Westminster#NA)
@@ -1970 +1970 @@
-Hall)
+Hall#NA)
@@ -1972 +1972 @@
-Gate)))
+Gate#NA)))
@@ -1975 +1975 @@
-pretending)
+pretending#pretend_v)
@@ -1978 +1978 @@
-to)
+to#to_prep)
@@ -1980 +1980 @@
-carrye)
+carrye#carry_v)
@@ -1983 +1983 @@
-his)
+his#his_adj)
@@ -1985 +1985 @@
-prisoners))
+prisoners#prisoner_n2))
@@ -1988 +1988 @@
-through)
+through#through_prep)
@@ -1991 +1991 @@
-ye)
+ye#the_adj)
@@ -1993 +1993 @@
-Citty)))
+Citty#NA)))
@@ -1996 +1996 @@
-to)
+to#to_prep)
@@ -1999 +1999 @@
-ye)
+ye#the_adj)
@@ -2001 +2001 @@
-tower)))))
+tower#NA)))))
@@ -2006 +2006 @@
-being)
+being#be_v)
@@ -2009 +2009 @@
-then))
+then#then_adv))
@@ -2011 +2011 @@
-informed)
+informed#inform_v)
@@ -2014 +2014 @@
-that)
+that#that_conj)
@@ -2018 +2018 @@
-ye)
+ye#the_adj)
@@ -2020 +2020 @@
-black)
+black#black_adj)
@@ -2022 +2022 @@
-rod))
+rod#rod_n1))
@@ -2024 +2024 @@
-lay)
+lay#lie_v1)
@@ -2027 +2027 @@
-in)
+in#in_prep)
@@ -2030 +2030 @@
-waite)))
+waite#wait_n)))
@@ -2033 +2033 @@
-for)
+for#for_prep)
@@ -2036 +2036 @@
-him)))
+him#him_pron)))
@@ -2039 +2039 @@
-in)
+in#in_prep)
@@ -2042 +2042 @@
-a)
+a#a_adj2)
@@ -2044 +2044 @@
-house)
+house#house_n1)
@@ -2047 +2047 @@
-neare)
+neare#near_adj)
@@ -2050 +2050 @@
-ye)
+ye#the_adj)
@@ -2052 +2052 @@
-Pallace)
+Pallace#NA)
@@ -2054 +2054 @@
-Yard)
+Yard#NA)
@@ -2056 +2056 @@
-gate))))))))
+gate#gate_n1))))))))
@@ -2064 +2064 @@
-But)
+But#but_prep)
@@ -2067 +2067 @@
-he))
+he#he_pron))
@@ -2069 +2069 @@
-went)
+went#go_v)
@@ -2072 +2072 @@
-discreetlyer))
+discreetlyer#discreetly_adv))
@@ -2075 +2075 @@
-to)
+to#to_prep)
@@ -2078 +2078 @@
-worke)))
+worke#work_n)))
@@ -2086 +2086 @@
-for)
+for#for_prep)
@@ -2089 +2089 @@
-he))
+he#he_pron))
@@ -2091 +2091 @@
-carryed)
+carryed#carry_v)
@@ -2094 +2094 @@
-his)
+his#his_adj)
@@ -2096 +2096 @@
-prisoners))
+prisoners#prisoner_n2))
@@ -2099 +2099 @@
-through)
+through#through_prep)
@@ -2103 +2103 @@
-Sir)
+Sir#NA)
@@ -2105 +2105 @@
-John)
+John#NA)
@@ -2107 +2107 @@
-Cotton's))
+Cotton's#NA))
@@ -2109 +2109 @@
-house))))
+house#house_n1))))
@@ -2115 +2115 @@
-and)
+and#and_conj1)
@@ -2119 +2119 @@
-tooke)
+tooke#take_v)
@@ -2122 +2122 @@
-bote))
+bote#boat_n))
@@ -2125 +2125 @@
-at)
+at#at_prep)
@@ -2128 +2128 @@
-his)
+his#his_adj)
@@ -2130 +2130 @@
-garden)
+garden#garden_n)
@@ -2132 +2132 @@
-stairs))))
+stairs#stair_n))))
@@ -2138 +2138 @@
-and)
+and#and_conj1)
@@ -2143 +2143 @@
-by)
+by#by_prep)
@@ -2146 +2146 @@
-water)))
+water#water_n)))
@@ -2148 +2148 @@
-delivered)
+delivered#deliver_v1)
@@ -2151 +2151 @@
-them))
+them#them_pron))
@@ -2154 +2154 @@
-safe))
+safe#safe_adj))
@@ -2157 +2157 @@
-into)
+into#into_prep)
@@ -2161 +2161 @@
-Sir)
+Sir#NA)
@@ -2163 +2163 @@
-John)
+John#NA)
@@ -2165 +2165 @@
-Robinson's)
+Robinson's#NA)
@@ -2169 +2169 @@
-hands)
+hands#hand_n)
@@ -2175 +2175 @@
-to)
+to#to_prep)
@@ -2178 +2178 @@
-whom)))
+whom#whom_pron)))
@@ -2186 +2186 @@
-in)
+in#in_prep)
@@ -2189 +2189 @@
-three)
+three#three_adj)
@@ -2191 +2191 @@
-minutes)))
+minutes#minute_n1)))
@@ -2194 +2194 @@
-the)
+the#the_adj)
@@ -2196 +2196 @@
-black-rod))
+black-rod#black-rod_n))
@@ -2198 +2198 @@
-came)
+came#come_v)
@@ -2201 +2201 @@
-to)
+to#to_prep)
@@ -2203 +2203 @@
-demand)
+demand#demand_v)
@@ -2206 +2206 @@
-the)
+the#the_adj)
@@ -2208 +2208 @@
-Prisoners)))))))
+Prisoners#prisoner_n2)))))))
@@ -2216 +2216 @@
-but)
+but#but_prep)
@@ -2219 +2219 @@
-the)
+the#the_adj)
@@ -2221 +2221 @@
-Lieut.))
+Lieut.#lieutenant_n))
@@ -2224 +2224 @@
-positively))
+positively#positively_adv))
@@ -2226 +2226 @@
-refused)
+refused#refuse_v)
@@ -2229 +2229 @@
-to)
+to#to_prep)
@@ -2231 +2231 @@
-deliver)
+deliver#deliver_v1)
@@ -2234 +2234 @@
-them)))
+them#them_pron)))
@@ -2243 +2243 @@
-As)
+As#as_adv)
@@ -2245 +2245 @@
-soone)
+soone#soon_adv)
@@ -2248 +2248 @@
-as)
+as#as_adv)
@@ -2259 +2259 @@
-the)
+the#the_adj)
@@ -2261 +2261 @@
-House))
+House#NA))
@@ -2263 +2263 @@
-mett)))))
+mett#meet_v)))))
@@ -2266 +2266 @@
-they))
+they#they_pron))
@@ -2268 +2268 @@
-agreed)
+agreed#agree_v)
@@ -2271 +2271 @@
-upon)
+upon#upon_prep)
@@ -2274 +2274 @@
-the)
+the#the_adj)
@@ -2276 +2276 @@
-enclosed)
+enclosed#enclosed_adj)
@@ -2278 +2278 @@
-Reasons)))
+Reasons#reason_n1)))
@@ -2281 +2281 @@
-in)
+in#in_prep)
@@ -2284 +2284 @@
-answere)
+answere#answer_n)
@@ -2287 +2287 @@
-to)
+to#to_prep)
@@ -2290 +2290 @@
-ye)
+ye#the_adj)
@@ -2292 +2292 @@
-Paper)
+Paper#paper_n)
@@ -2295 +2295 @@
-delivered)
+delivered#deliver_v1)
@@ -2298 +2298 @@
-ye)
+ye#the_adj)
@@ -2300 +2300 @@
-day)
+day#day_n)
@@ -2303 +2303 @@
-before)))
+before#before_adv)))
@@ -2306 +2306 @@
-by)
+by#by_prep)
@@ -2309 +2309 @@
-ye)
+ye#the_adj)
@@ -2311 +2311 @@
-Lords)))
+Lords#lord_n)))
@@ -2314 +2314 @@
-at)
+at#at_prep)
@@ -2317 +2317 @@
-a)
+a#a_adj2)
@@ -2319 +2319 @@
-Conference)))))))))
+Conference#conference_n)))))))))
@@ -2325 +2325 @@
-and)
+and#and_conj1)
@@ -2329 +2329 @@
-sent)
+sent#send_v1)
@@ -2331 +2331 @@
-up)
+up#up_adv1)
@@ -2334 +2334 @@
-a)
+a#a_adj2)
@@ -2336 +2336 @@
-message)
+message#message_n)
@@ -2339 +2339 @@
-for)
+for#for_prep)
@@ -2342 +2342 @@
-a)
+a#a_adj2)
@@ -2344 +2344 @@
-Conference)
+Conference#conference_n)
@@ -2347 +2347 @@
-on)
+on#on_prep)
@@ -2350 +2350 @@
-ye)
+ye#the_adj)
@@ -2352 +2352 @@
-subject)
+subject#subject_n)
@@ -2354 +2354 @@
-matter)
+matter#matter_n1)
@@ -2357 +2357 @@
-of)
+of#of_prep)
@@ -2360 +2360 @@
-ye)
+ye#the_adj)
@@ -2362 +2362 @@
-late)
+late#late_adj1)
@@ -2364 +2364 @@
-Conference)))))))
+Conference#conference_n)))))))
@@ -2370 +2370 @@
-to)
+to#to_prep)
@@ -2373 +2373 @@
-wch)))
+wch#which_adj)))
@@ -2381 +2381 @@
-ye)
+ye#the_adj)
@@ -2383 +2383 @@
-Lords))
+Lords#lord_n))
@@ -2385 +2385 @@
-replyed)
+replyed#reply_v)
@@ -2388 +2388 @@
-that)
+that#that_conj)
@@ -2392 +2392 @@
-they))
+they#they_pron))
@@ -2394 +2394 @@
-would)
+would#will_v1)
@@ -2396 +2396 @@
-send)
+send#send_v1)
@@ -2398 +2398 @@
-up)
+up#up_adv1)
@@ -2401 +2401 @@
-an)
+an#a_adj2)
@@ -2403 +2403 @@
-answere))
+answere#answer_n))
@@ -2406 +2406 @@
-by)
+by#by_prep)
@@ -2409 +2409 @@
-messengers)
+messengers#messenger_n)
@@ -2412 +2412 @@
-of)
+of#of_prep)
@@ -2415 +2415 @@
-their)
+their#their_adj)
@@ -2417 +2417 @@
-owne))))))))))
+owne#own_adj))))))))))
@@ -2425 +2425 @@
-But)
+But#but_prep)
@@ -2430 +2430 @@
-their)
+their#their_adj)
@@ -2432 +2432 @@
-black)
+black#black_adj)
@@ -2434 +2434 @@
-rodd))
+rodd#rod_n1))
@@ -2436 +2436 @@
-being)
+being#be_v)
@@ -2438 +2438 @@
-returned)
+returned#return_v1)
@@ -2441 +2441 @@
-from)
+from#from_prep)
@@ -2444 +2444 @@
-ye)
+ye#the_adj)
@@ -2446 +2446 @@
-tower))))
+tower#NA))))
@@ -2449 +2449 @@
-and)
+and#and_conj1)
@@ -2454 +2454 @@
-giving)
+giving#give_v)
@@ -2457 +2457 @@
-them))
+them#them_pron))
@@ -2460 +2460 @@
-an)
+an#a_adj2)
@@ -2462 +2462 @@
-acct)
+acct#act_n)
@@ -2465 +2465 @@
-that)
+that#that_conj)
@@ -2469 +2469 @@
-ye)
+ye#the_adj)
@@ -2471 +2471 @@
-Lieu=t=)
+Lieu=t=#lieutenant_n)
@@ -2474 +2474 @@
-of)
+of#of_prep)
@@ -2477 +2477 @@
-ye)
+ye#the_adj)
@@ -2479 +2479 @@
-tower))))
+tower#NA))))
@@ -2481 +2481 @@
-refused)
+refused#refuse_v)
@@ -2484 +2484 @@
-to)
+to#to_prep)
@@ -2486 +2486 @@
-deliver)
+deliver#deliver_v1)
@@ -2489 +2489 @@
-ye)
+ye#the_adj)
@@ -2491 +2491 @@
-Prisoners)))))))))
+Prisoners#prisoner_n2)))))))))
@@ -2496 +2496 @@
-they))
+they#they_pron))
@@ -2498 +2498 @@
-voted)
+voted#vote_v)
@@ -2501 +2501 @@
-a)
+a#a_adj2)
@@ -2503 +2503 @@
-second)
+second#second_adj)
@@ -2505 +2505 @@
-addresse)
+addresse#address_n)
@@ -2508 +2508 @@
-to)
+to#to_prep)
@@ -2511 +2511 @@
-his)
+his#his_adj)
@@ -2513 +2513 @@
-Ma=tie=)))
+Ma=tie=#majesty_n)))
@@ -2516 +2516 @@
-by)
+by#by_prep)
@@ -2519 +2519 @@
-ye)
+ye#the_adj)
@@ -2521 +2521 @@
-white-staves)))
+white-staves#white-staff_n)))
@@ -2524 +2524 @@
-that)
+that#that_conj)
@@ -2529 +2529 @@
-ye)
+ye#the_adj)
@@ -2531 +2531 @@
-Lieu=tnt=)
+Lieu=tnt=#lieutenant_n)
@@ -2534 +2534 @@
-of)
+of#of_prep)
@@ -2537 +2537 @@
-ye)
+ye#the_adj)
@@ -2539 +2539 @@
-tower))))
+tower#NA))))
@@ -2541 +2541 @@
-should)
+should#shall_v)
@@ -2543 +2543 @@
-be)
+be#be_v)
@@ -2545 +2545 @@
-removed))
+removed#remove_v))
@@ -2548 +2548 @@
-and)
+and#and_conj1)
@@ -2552 +2552 @@
-a)
+a#a_adj2)
@@ -2554 +2554 @@
-new)
+new#new_adj)
@@ -2556 +2556 @@
-one))
+one#one_adj))
@@ -2558 +2558 @@
-appoynted)
+appoynted#appoint_v)
@@ -2561 +2561 @@
-in)
+in#in_prep)
@@ -2564 +2564 @@
-his)
+his#his_adj)
@@ -2566 +2566 @@
-stead)))))))
+stead#stead_n)))))))
@@ -2572 +2572 @@
-to)
+to#to_prep)
@@ -2575 +2575 @@
-wch)))
+wch#which_adj)))
@@ -2583 +2583 @@
-his)
+his#his_adj)
@@ -2585 +2585 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -2588 +2588 @@
-then))
+then#then_adv))
@@ -2590 +2590 @@
-answered)
+answered#answer_v)
@@ -2593 +2593 @@
-that)
+that#that_conj)
@@ -2598 +2598 @@
-he))
+he#he_pron))
@@ -2600 +2600 @@
-would)
+would#will_v1)
@@ -2602 +2602 @@
-consider)
+consider#consider_v)
@@ -2605 +2605 @@
-of)
+of#of_prep)
@@ -2608 +2608 @@
-it))))
+it#it_pron))))
@@ -2611 +2611 @@
-and)
+and#and_conj1)
@@ -2614 +2614 @@
-give)
+give#give_v)
@@ -2617 +2617 @@
-them))
+them#them_pron))
@@ -2620 +2620 @@
-an)
+an#a_adj2)
@@ -2622 +2622 @@
-answere))
+answere#answer_n))
@@ -2625 +2625 @@
-this)
+this#this_pron)
@@ -2627 +2627 @@
-day))
+day#day_n))
@@ -2630 +2630 @@
-by)
+by#by_prep)
@@ -2633 +2633 @@
-5)
+5#five_v)
@@ -2636 +2636 @@
-in)
+in#in_prep)
@@ -2639 +2639 @@
-ye)
+ye#the_adj)
@@ -2641 +2641 @@
-afternoone))))))))))))
+afternoone#afternoon_n))))))))))))
@@ -2650 +2650 @@
-When)
+When#when_adv)
@@ -2657 +2657 @@
-the)
+the#the_adj)
@@ -2659 +2659 @@
-House)
+House#house_n1)
@@ -2662 +2662 @@
-of)
+of#of_prep)
@@ -2665 +2665 @@
-Commons))))
+Commons#commons_n))))
@@ -2667 +2667 @@
-met)
+met#meet_v)
@@ -2670 +2670 @@
-this)
+this#this_pron)
@@ -2672 +2672 @@
-morning)))))
+morning#morning_n)))))
@@ -2677 +2677 @@
-they))
+they#they_pron))
@@ -2679 +2679 @@
-found)
+found#find_v)
@@ -2686 +2686 @@
-their)
+their#their_adj)
@@ -2688 +2688 @@
-Sergent))
+Sergent#sergeant_n))
@@ -2690 +2690 @@
-was)
+was#be_v)
@@ -2692 +2692 @@
-changed)))
+changed#change_v)))
@@ -2700 +2700 @@
-and)
+and#and_conj1)
@@ -2703 +2703 @@
-while)
+while#while_adv)
@@ -2710 +2710 @@
-they))
+they#they_pron))
@@ -2712 +2712 @@
-were)
+were#be_v)
@@ -2714 +2714 @@
-debating)
+debating#debate_v1)
@@ -2717 +2717 @@
-on)
+on#on_prep)
@@ -2720 +2720 @@
-it)))
+it#it_pron)))
@@ -2723 +2723 @@
-with)
+with#with_prep)
@@ -2726 +2726 @@
-some)
+some#some_pron)
@@ -2728 +2728 @@
-warmth))))))
+warmth#warmth_n))))))
@@ -2731 +2731 @@
-Mr)
+Mr#NA)
@@ -2733 +2733 @@
-Secretary)
+Secretary#NA)
@@ -2735 +2735 @@
-Coventry))
+Coventry#NA))
@@ -2737 +2737 @@
-came)
+came#come_v)
@@ -2739 +2739 @@
-in))
+in#in_adv))
@@ -2745 +2745 @@
-and)
+and#and_conj1)
@@ -2750 +2750 @@
-from)
+from#from_prep)
@@ -2753 +2753 @@
-his)
+his#his_adj)
@@ -2755 +2755 @@
-Ma=tie=)))
+Ma=tie=#majesty_n)))
@@ -2757 +2757 @@
-told)
+told#tell_v)
@@ -2760 +2760 @@
-us))
+us#us_pron))
@@ -2764 +2764 @@
-that)
+that#that_conj)
@@ -2768 +2768 @@
-it))
+it#it_pron))
@@ -2770 +2770 @@
-was)
+was#be_v)
@@ -2774 +2774 @@
-his)
+his#his_adj)
@@ -2776 +2776 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -2778 +2778 @@
-pleasure))
+pleasure#pleasure_n))
@@ -2785 +2785 @@
-wee))
+wee#we_pron))
@@ -2787 +2787 @@
-should)
+should#shall_v)
@@ -2790 +2790 @@
-immediately))
+immediately#immediately_adv))
@@ -2792 +2792 @@
-adjourne)
+adjourne#adjourn_v)
@@ -2795 +2795 @@
-till)
+till#till_prep)
@@ -2798 +2798 @@
-4)
+4#four_adj)
@@ -2801 +2801 @@
-of)
+of#of_prep)
@@ -2804 +2804 @@
-ye)
+ye#the_adj)
@@ -2806 +2806 @@
-clock)))
+clock#clock_n1)))
@@ -2809 +2809 @@
-in)
+in#in_prep)
@@ -2812 +2812 @@
-ye)
+ye#the_adj)
@@ -2814 +2814 @@
-afternoon)))
+afternoon#afternoon_n)))
@@ -2820 +2820 @@
-when))
+when#when_adv))
@@ -2826 +2826 @@
-it))
+it#it_pron))
@@ -2828 +2828 @@
-was)
+was#be_v)
@@ -2831 +2831 @@
-his)
+his#his_adj)
@@ -2833 +2833 @@
-pleasure))
+pleasure#pleasure_n))
@@ -2842 +2842 @@
-we))
+we#we_pron))
@@ -2844 +2844 @@
-should)
+should#shall_v)
@@ -2846 +2846 @@
-attend)
+attend#attend_v)
@@ -2849 +2849 @@
-him))
+him#him_pron))
@@ -2852 +2852 @@
-in)
+in#in_prep)
@@ -2855 +2855 @@
-ye)
+ye#the_adj)
@@ -2857 +2857 @@
-bankquetting)
+bankquetting#banqueting_n)
@@ -2859 +2859 @@
-House)))))))))))))
+House#house_n1)))))))))))))
@@ -2864 +2864 @@
-and)
+and#and_conj1)
@@ -2867 +2867 @@
-that)
+that#that_conj)
@@ -2871 +2871 @@
-his)
+his#his_adj)
@@ -2873 +2873 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -2875 +2875 @@
-had)
+had#have_v)
@@ -2877 +2877 @@
-sent)
+sent#send_v1)
@@ -2880 +2880 @@
-the)
+the#the_adj)
@@ -2882 +2882 @@
-same)
+same#same_adj)
@@ -2884 +2884 @@
-message))
+message#message_n))
@@ -2887 +2887 @@
-to)
+to#to_prep)
@@ -2890 +2890 @@
-ye)
+ye#the_adj)
@@ -2892 +2892 @@
-Lords)))))))
+Lords#lord_n)))))))
@@ -2901 +2901 @@
-Accordingly))
+Accordingly#accordingly_adv))
@@ -2904 +2904 @@
-we))
+we#we_pron))
@@ -2906 +2906 @@
-mett)
+mett#meet_v)
@@ -2909 +2909 @@
-when)
+when#when_adv)
@@ -2916 +2916 @@
-his)
+his#his_adj)
@@ -2918 +2918 @@
-Mat=ie=))
+Mat=ie=#majesty_n))
@@ -2920 +2920 @@
-made)
+made#make_v1)
@@ -2923 +2923 @@
-us))
+us#us_pron))
@@ -2926 +2926 @@
-ye)
+ye#the_adj)
@@ -2928 +2928 @@
-enclosed)
+enclosed#enclosed_adj)
@@ -2930 +2930 @@
-gracious)
+gracious#gracious_adj)
@@ -2932 +2932 @@
-speech)
+speech#speech_n1)
@@ -2936 +2936 @@
-after)
+after#after_adv)
@@ -2939 +2939 @@
-wch)))
+wch#which_adj)))
@@ -2949 +2949 @@
-the)
+the#the_adj)
@@ -2951 +2951 @@
-Speaker))
+Speaker#speaker_n))
@@ -2953 +2953 @@
-returning)
+returning#return_v1)
@@ -2956 +2956 @@
-to)
+to#to_prep)
@@ -2959 +2959 @@
-ye)
+ye#the_adj)
@@ -2961 +2961 @@
-House))))
+House#NA))))
@@ -2964 +2964 @@
-we))
+we#we_pron))
@@ -2966 +2966 @@
-voted)
+voted#vote_v)
@@ -2969 +2969 @@
-ye)
+ye#the_adj)
@@ -2971 +2971 @@
-enclosed)
+enclosed#enclosed_adj)
@@ -2973 +2973 @@
-votes)))
+votes#vote_n)))
@@ -2976 +2976 @@
-and)
+and#and_conj1)
@@ -2981 +2981 @@
-adjourned)
+adjourned#adjourn_v)
@@ -2984 +2984 @@
-ye)
+ye#the_adj)
@@ -2986 +2986 @@
-debate)
+debate#debate_n1)
@@ -2989 +2989 @@
-of)
+of#of_prep)
@@ -2992 +2992 @@
-our)
+our#our_pron)
@@ -2994 +2994 @@
-Sergeant))))
+Sergeant#sergeant_n))))
@@ -2997 +2997 @@
-till)
+till#till_prep)
@@ -3000 +3000 @@
-Munday)
+Munday#NA)
@@ -3002 +3002 @@
-morning)))))))))))
+morning#morning_n)))))))))))
@@ -3011 +3011 @@
-The)
+The#the_adj)
@@ -3013 +3013 @@
-Lords))
+Lords#lord_n))
@@ -3016 +3016 @@
-after))
+after#after_adv))
@@ -3018 +3018 @@
-mett))
+mett#meet_v))
@@ -3024 +3024 @@
-and)
+and#and_conj1)
@@ -3027 +3027 @@
-the)
+the#the_adj)
@@ -3029 +3029 @@
-white-staves))
+white-staves#white-staff_n))
@@ -3031 +3031 @@
-reporting)
+reporting#report_v)
@@ -3034 +3034 @@
-to)
+to#to_prep)
@@ -3037 +3037 @@
-their)
+their#their_adj)
@@ -3039 +3039 @@
-lordships)))
+lordships#lordship_n)))
@@ -3042 +3042 @@
-that)
+that#that_conj)
@@ -3047 +3047 @@
-his)
+his#his_adj)
@@ -3049 +3049 @@
-Mat=ie=))
+Mat=ie=#majesty_n))
@@ -3051 +3051 @@
-had)
+had#have_v)
@@ -3053 +3053 @@
-considered)
+considered#consider_v)
@@ -3056 +3056 @@
-of)
+of#of_prep)
@@ -3059 +3059 @@
-their)
+their#their_adj)
@@ -3061 +3061 @@
-addresse)
+addresse#address_n)
@@ -3064 +3064 @@
-concerning)
+concerning#concerning_prep)
@@ -3067 +3067 @@
-ye)
+ye#the_adj)
@@ -3069 +3069 @@
-removal)
+removal#removal_n)
@@ -3072 +3072 @@
-of)
+of#of_prep)
@@ -3075 +3075 @@
-ye)
+ye#the_adj)
@@ -3077 +3077 @@
-Lieu=t=)
+Lieu=t=#lieutenant_n)
@@ -3080 +3080 @@
-of)
+of#of_prep)
@@ -3083 +3083 @@
-ye)
+ye#the_adj)
@@ -3085 +3085 @@
-tower))))))))))
+tower#NA))))))))))
@@ -3090 +3090 @@
-but)
+but#but_prep)
@@ -3095 +3095 @@
-sawe)
+sawe#see_v)
@@ -3098 +3098 @@
-no)
+no#no_adj)
@@ -3100 +3100 @@
-reason)
+reason#reason_n1)
@@ -3103 +3103 @@
-for)
+for#for_prep)
@@ -3106 +3106 @@
-it))))))))
+it#it_pron))))))))
@@ -3112 +3112 @@
-Upon)
+Upon#upon_prep)
@@ -3115 +3115 @@
-wch)))
+wch#which_adj)))
@@ -3123 +3123 @@
-their)
+their#their_adj)
@@ -3125 +3125 @@
-Lord=sps=))
+Lord=sps=#lordship_n))
@@ -3127 +3127 @@
-adjourned)
+adjourned#adjourn_v)
@@ -3130 +3130 @@
-till)
+till#till_prep)
@@ -3133 +3133 @@
-Munday)))))
+Munday#NA)))))
@@ -3142 +3142 @@
-My)
+My#my_adj)
@@ -3144 +3144 @@
-paper))
+paper#paper_n))
@@ -3146 +3146 @@
-only)
+only#NA)
@@ -3149 +3149 @@
-now))
+now#now_adv))
@@ -3153 +3153 @@
-allowes)
+allowes#allow_v)
@@ -3156 +3156 @@
-mee))
+mee#me_pron1))
@@ -3161 +3161 @@
-roome)
+roome#room_n1)
@@ -3164 +3164 @@
-to)
+to#to_prep)
@@ -3166 +3166 @@
-beg)
+beg#beg_v)
@@ -3170 +3170 @@
-your)
+your#your_pron)
@@ -3172 +3172 @@
-Exc=eyes=))
+Exc=eyes=#excellency_n))
@@ -3174 +3174 @@
-pardon)
+pardon#pardon_n1)
@@ -3177 +3177 @@
-for)
+for#for_prep)
@@ -3180 +3180 @@
-this)
+this#this_pron)
@@ -3182 +3182 @@
-tedious)
+tedious#tedious_adj)
@@ -3184 +3184 @@
-narrative)
+narrative#narrative_n)
@@ -3188 +3188 @@
-wch))
+wch#which_adj))
@@ -3195 +3195 @@
-is)
+is#be_v)
@@ -3198 +3198 @@
-ye)
+ye#the_adj)
@@ -3200 +3200 @@
-true)
+true#true_adj)
@@ -3202 +3202 @@
-state)
+state#state_n)
@@ -3205 +3205 @@
-of)
+of#of_prep)
@@ -3208 +3208 @@
-matters)
+matters#matter_n1)
@@ -3211 +3211 @@
-of)
+of#of_prep)
@@ -3214 +3214 @@
-fact)))
+fact#fact_n)))
@@ -3217 +3217 @@
-betweene)
+betweene#between_prep)
@@ -3220 +3220 @@
-both)
+both#both_pron)
@@ -3222 +3222 @@
-Houses)))))))))))))
+Houses#house_n1)))))))))))))
@@ -3231 +3231 @@
-I))
+I#i_pron))
@@ -3233 +3233 @@
-am)
+am#be_v)
@@ -3236 +3236 @@
-my)
+my#my_adj)
@@ -3238 +3238 @@
-Lord))
+Lord#lord_n))
@@ -3242 +3242 @@
-Yr)
+Yr#your_pron)
@@ -3244 +3244 @@
-Excy=s=))
+Excy=s=#excellency_n))
@@ -3247 +3247 @@
-most)
+most#most_adj)
@@ -3249 +3249 @@
-humble))
+humble#humble_adj))
@@ -3251 +3251 @@
-servant)
+servant#servant_n)
@@ -3254 +3254 @@
-Fran)
+Fran#NA)
@@ -3256 +3256 @@
-Aungier))))
+Aungier#NA))))
