--- bedyll-1537-e1-h.psd.vert	2025-09-10 13:50:35
+++ bedyll-1537-e1-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-My)
+My#my_adj)
@@ -17 +17 @@
-very)
+very#very_adj)
@@ -19 +19 @@
-good))
+good#good_adj))
@@ -21 +21 @@
-Lord))
+Lord#lord_n))
@@ -26 +26 @@
-after)
+after#after_adv)
@@ -29 +29 @@
-my)
+my#my_adj)
@@ -32 +32 @@
-moost)
+moost#most_adj)
@@ -34 +34 @@
-hertie))
+hertie#hearty_adj))
@@ -36 +36 @@
-commendations)))
+commendations#commendation_n)))
@@ -39 +39 @@
-it))
+it#it_pron))
@@ -41 +41 @@
-shall)
+shall#shall_v)
@@ -43 +43 @@
-please)
+please#please_v)
@@ -46 +46 @@
-yo=r=)
+yo=r=#your_pron)
@@ -48 +48 @@
-Lordship))
+Lordship#lordship_n))
@@ -51 +51 @@
-to)
+to#to_prep)
@@ -53 +53 @@
-understand)
+understand#understand_v)
@@ -56 +56 @@
-that)
+that#that_conj)
@@ -60 +60 @@
-the)
+the#the_adj)
@@ -62 +62 @@
-monks)
+monks#monk_n)
@@ -65 +65 @@
-of)
+of#of_prep)
@@ -68 +68 @@
-the)
+the#the_adj)
@@ -70 +70 @@
-Charterhouse)
+Charterhouse#NA)
@@ -73 +73 @@
-here))
+here#here_adv))
@@ -76 +76 @@
-at)
+at#at_prep)
@@ -79 +79 @@
-London)))))
+London#NA)))))
@@ -85 +85 @@
-whiche))
+whiche#which_adj))
@@ -92 +92 @@
-wer)
+wer#be_v)
@@ -94 +94 @@
-committed)
+committed#commit_v)
@@ -97 +97 @@
-to)
+to#to_prep)
@@ -100 +100 @@
-Newgate)))
+Newgate#NA)))
@@ -103 +103 @@
-for)
+for#for_prep)
@@ -106 +106 @@
-thair)
+thair#their_adj)
@@ -108 +108 @@
-traitorus)
+traitorus#traitorous_adj)
@@ -110 +110 @@
-behavor)
+behavor#behaviour_n)
@@ -114 +114 @@
-long)
+long#long_adj1)
@@ -116 +116 @@
-tyme))
+tyme#time_n))
@@ -118 +118 @@
-continued)
+continued#continue_v)
@@ -121 +121 @@
-against)
+against#against_prep)
@@ -125 +125 @@
-the)
+the#the_adj)
@@ -127 +127 @@
-Kings))
+Kings#king_n))
@@ -129 +129 @@
-Grace)))))))))
+Grace#grace_n)))))))))
@@ -133 +133 @@
-be)
+be#be_v)
@@ -136 +136 @@
-almoost))
+almoost#almost_adv))
@@ -138 +138 @@
-dispeched)
+dispeched#despeche_v)
@@ -141 +141 @@
-by)
+by#by_prep)
@@ -144 +144 @@
-th)
+th#the_adj)
@@ -146 +146 @@
-'and)
+'and#hand_n)
@@ -149 +149 @@
-of)
+of#of_prep)
@@ -152 +152 @@
-God)))))
+God#NA)))))
@@ -157 +157 @@
-as)
+as#as_adv)
@@ -168 +168 @@
-it))
+it#it_pron))
@@ -170 +170 @@
-may)
+may#may_v1)
@@ -172 +172 @@
-apper)
+apper#appear_v)
@@ -175 +175 @@
-to)
+to#to_prep)
@@ -178 +178 @@
-you)))
+you#you_pron)))
@@ -181 +181 @@
-by)
+by#by_prep)
@@ -184 +184 @@
-this)
+this#this_pron)
@@ -186 +186 @@
-byll)
+byll#bill_n3)
@@ -189 +189 @@
-inclosed))))))))
+inclosed#enclose_v))))))))
@@ -195 +195 @@
-Wherof))
+Wherof#whereof_adv))
@@ -202 +202 @@
-considering)
+considering#consider_v)
@@ -208 +208 @@
-thair)
+thair#their_adj)
@@ -210 +210 @@
-behavor))
+behavor#behaviour_n))
@@ -213 +213 @@
-and)
+and#and_conj1)
@@ -216 +216 @@
-the)
+the#the_adj)
@@ -218 +218 @@
-hole)
+hole#whole_adj)
@@ -220 +220 @@
-mater)))))
+mater#matter_n1)))))
@@ -225 +225 @@
-I))
+I#i_pron))
@@ -227 +227 @@
-am)
+am#be_v)
@@ -229 +229 @@
-not)
+not#not_adv)
@@ -232 +232 @@
-sory)
+sory#sorry_adj)
@@ -239 +239 @@
-but)
+but#but_prep)
@@ -244 +244 @@
-wold)
+wold#will_v1)
@@ -247 +247 @@
-that)
+that#that_conj)
@@ -251 +251 @@
-al)
+al#all_adj)
@@ -253 +253 @@
-suche)
+suche#such_adj)
@@ -256 +256 @@
-as)
+as#as_adv)
@@ -266 +266 @@
-love)
+love#love_v1)
@@ -268 +268 @@
-not)
+not#not_adv)
@@ -273 +273 @@
-the)
+the#the_adj)
@@ -275 +275 @@
-Kings))
+Kings#king_n))
@@ -277 +277 @@
-Highnes))
+Highnes#highness_n))
@@ -280 +280 @@
-and)
+and#and_conj1)
@@ -283 +283 @@
-his)
+his#his_adj)
@@ -285 +285 @@
-wordly)
+wordly#worldly_adj)
@@ -287 +287 @@
-honor)))))))
+honor#honour_n)))))))
@@ -289 +289 @@
-wer)
+wer#be_v)
@@ -292 +292 @@
-in)
+in#in_prep)
@@ -295 +295 @@
-like)
+like#like_adj)
@@ -297 +297 @@
-caas))))))))))))
+caas#case_n1))))))))))))
@@ -306 +306 @@
-My)
+My#my_adj)
@@ -308 +308 @@
-Lord))
+Lord#lord_n))
@@ -313 +313 @@
-as)
+as#as_adv)
@@ -324 +324 @@
-ye))
+ye#ye_pron))
@@ -326 +326 @@
-may)
+may#may_v1)
@@ -333 +333 @@
-I))
+I#i_pron))
@@ -335 +335 @@
-desir)
+desir#desire_v)
@@ -338 +338 @@
-you))
+you#you_pron))
@@ -342 +342 @@
-in)
+in#in_prep)
@@ -345 +345 @@
-the)
+the#the_adj)
@@ -347 +347 @@
-wey)
+wey#way_n1)
@@ -350 +350 @@
-of)
+of#of_prep)
@@ -353 +353 @@
-charite)))))
+charite#charity_n)))))
@@ -358 +358 @@
-and)
+and#and_conj1)
@@ -361 +361 @@
-none)
+none#none_pron)
@@ -363 +363 @@
-other)
+other#other_adj)
@@ -365 +365 @@
-wise))))
+wise#wise_n1))))
@@ -372 +372 @@
-to)
+to#to_prep)
@@ -374 +374 @@
-be)
+be#be_v)
@@ -377 +377 @@
-good)
+good#good_adj)
@@ -379 +379 @@
-lord)
+lord#lord_n)
@@ -382 +382 @@
-to)
+to#to_prep)
@@ -385 +385 @@
-the)
+the#the_adj)
@@ -387 +387 @@
-Prior)
+Prior#prior_n1)
@@ -390 +390 @@
-of)
+of#of_prep)
@@ -393 +393 @@
-the)
+the#the_adj)
@@ -395 +395 @@
-said)
+said#said_adj)
@@ -397 +397 @@
-Charterhouse)))
+Charterhouse#NA)))
@@ -403 +403 @@
-which))
+which#which_adj))
@@ -411 +411 @@
-is)
+is#be_v)
@@ -415 +415 @@
-as)
+as#as_adv)
@@ -417 +417 @@
-honest))
+honest#honest_adj))
@@ -419 +419 @@
-a)
+a#a_adj2)
@@ -421 +421 @@
-man)
+man#man_n1)
@@ -424 +424 @@
-as)
+as#as_adv)
@@ -435 +435 @@
-ever))
+ever#ever_adv))
@@ -437 +437 @@
-was)
+was#be_v)
@@ -440 +440 @@
-in)
+in#in_prep)
@@ -443 +443 @@
-that)
+that#that_pron)
@@ -445 +445 @@
-habite)))))))
+habite#habit_n)))))))
@@ -450 +450 @@
-or)
+or#or_conj2)
@@ -452 +452 @@
-els)
+els#else_adv)
@@ -455 +455 @@
-I))
+I#i_pron))
@@ -457 +457 @@
-am)
+am#be_v)
@@ -460 +460 @@
-much))
+much#much_adj))
@@ -462 +462 @@
-deceyved)))
+deceyved#deceive_v)))
@@ -469 +469 @@
-and)
+and#and_conj1)
@@ -474 +474 @@
-is)
+is#be_v)
@@ -477 +477 @@
-one)
+one#one_adj)
@@ -481 +481 @@
-whiche))
+whiche#which_adj))
@@ -490 +490 @@
-never))
+never#never_adv))
@@ -492 +492 @@
-offended)
+offended#offend_v)
@@ -496 +496 @@
-the)
+the#the_adj)
@@ -498 +498 @@
-Kings))
+Kings#king_n))
@@ -500 +500 @@
-grace))
+grace#grace_n))
@@ -503 +503 @@
-by)
+by#by_prep)
@@ -506 +506 @@
-disobedience)
+disobedience#disobedience_n)
@@ -509 +509 @@
-of)
+of#of_prep)
@@ -512 +512 @@
-his)
+his#his_adj)
@@ -514 +514 @@
-Lawes))))))
+Lawes#law_n1))))))
@@ -519 +519 @@
-but)
+but#but_prep)
@@ -524 +524 @@
-hath)
+hath#have_v)
@@ -526 +526 @@
-labored)
+labored#labour_v)
@@ -529 +529 @@
-very)
+very#very_adj)
@@ -531 +531 @@
-sore))
+sore#sore_adv))
@@ -534 +534 @@
-continually))
+continually#continually_adv))
@@ -537 +537 @@
-for)
+for#for_prep)
@@ -540 +540 @@
-the)
+the#the_adj)
@@ -542 +542 @@
-reformation)
+reformation#reformation_n)
@@ -545 +545 @@
-of)
+of#of_prep)
@@ -548 +548 @@
-his)
+his#his_adj)
@@ -550 +550 @@
-brethern))))))))))))
+brethern#brother_n))))))))))))
@@ -555 +555 @@
-and)
+and#and_conj1)
@@ -561 +561 @@
-now))
+now#now_adv))
@@ -564 +564 @@
-at)
+at#at_prep)
@@ -567 +567 @@
-the)
+the#the_adj)
@@ -569 +569 @@
-last)))
+last#last_adv)))
@@ -574 +574 @@
-at)
+at#at_prep)
@@ -577 +577 @@
-myn)
+myn#mine_adj)
@@ -579 +579 @@
-exhortation)
+exhortation#exhortation_n)
@@ -582 +582 @@
-and)
+and#and_conj1)
@@ -585 +585 @@
-instigation)))))
+instigation#instigation_n)))))
@@ -590 +590 @@
-constantly))
+constantly#constantly_adv))
@@ -592 +592 @@
-moved)
+moved#move_v)
@@ -595 +595 @@
-and)
+and#and_conj1)
@@ -598 +598 @@
-finally))
+finally#finally_adv))
@@ -600 +600 @@
-persuaded))
+persuaded#persuade_v))
@@ -603 +603 @@
-his)
+his#his_adj)
@@ -605 +605 @@
-brethern))
+brethern#brother_n))
@@ -609 +609 @@
-to)
+to#to_prep)
@@ -611 +611 @@
-surrender)
+surrender#surrender_v)
@@ -614 +614 @@
-thair)
+thair#their_adj)
@@ -616 +616 @@
-house)
+house#house_n1)
@@ -622 +622 @@
-lands)))
+lands#land_n1)))
@@ -627 +627 @@
-and)
+and#and_conj1)
@@ -630 +630 @@
-goods))))
+goods#good_adj))))
@@ -635 +635 @@
-into)
+into#into_prep)
@@ -639 +639 @@
-the)
+the#the_adj)
@@ -641 +641 @@
-Kings))
+Kings#king_n))
@@ -643 +643 @@
-hands))))
+hands#hand_n))))
@@ -648 +648 @@
-and)
+and#and_conj1)
@@ -651 +651 @@
-to)
+to#to_prep)
@@ -653 +653 @@
-trust)
+trust#trust_v)
@@ -656 +656 @@
-only)
+only#only_adj)
@@ -658 +658 @@
-to)
+to#to_prep)
@@ -661 +661 @@
-his)
+his#his_adj)
@@ -663 +663 @@
-mercy)
+mercy#mercy_n)
@@ -666 +666 @@
-and)
+and#and_conj1)
@@ -669 +669 @@
-grace))))))))))))))))
+grace#grace_n))))))))))))))))
@@ -678 +678 @@
-I))
+I#i_pron))
@@ -680 +680 @@
-beseche)
+beseche#beseech_v)
@@ -683 +683 @@
-you))
+you#you_pron))
@@ -688 +688 @@
-my)
+my#my_adj)
@@ -690 +690 @@
-Lord))
+Lord#lord_n))
@@ -693 +693 @@
-that)
+that#that_conj)
@@ -697 +697 @@
-the)
+the#the_adj)
@@ -699 +699 @@
-said)
+said#said_adj)
@@ -701 +701 @@
-Prior))
+Prior#prior_n1))
@@ -703 +703 @@
-may)
+may#may_v1)
@@ -705 +705 @@
-be)
+be#be_v)
@@ -708 +708 @@
-so)
+so#so_adv)
@@ -712 +712 @@
-entreated)
+entreated#entreat_v)
@@ -715 +715 @@
-by)
+by#by_prep)
@@ -718 +718 @@
-your)
+your#your_pron)
@@ -720 +720 @@
-help)))
+help#help_n)))
@@ -725 +725 @@
-that)
+that#that_conj)
@@ -730 +730 @@
-he))
+he#he_pron))
@@ -732 +732 @@
-be)
+be#be_v)
@@ -734 +734 @@
-not)
+not#not_adv)
@@ -737 +737 @@
-sory)))
+sory#sorry_adj)))
@@ -742 +742 @@
-and)
+and#and_conj1)
@@ -745 +745 @@
-repent)
+repent#repent_v)
@@ -748 +748 @@
-that)
+that#that_conj)
@@ -752 +752 @@
-he))
+he#he_pron))
@@ -754 +754 @@
-hath)
+hath#have_v)
@@ -757 +757 @@
-fered)
+fered#fear_v)
@@ -759 +759 @@
-and)
+and#and_conj1)
@@ -761 +761 @@
-folowed))
+folowed#follow_v))
@@ -765 +765 @@
-your)
+your#your_pron)
@@ -767 +767 @@
-sore)
+sore#sore_adj1)
@@ -769 +769 @@
-words))
+words#word_n))
@@ -772 +772 @@
-and)
+and#and_conj1)
@@ -775 +775 @@
-my)
+my#my_adj)
@@ -777 +777 @@
-gentil)
+gentil#gentle_adj)
@@ -779 +779 @@
-exhortation)))
+exhortation#exhortation_n)))
@@ -782 +782 @@
-made)
+made#make_v1)
@@ -785 +785 @@
-unto)
+unto#unto_prep)
@@ -788 +788 @@
-him))))
+him#him_pron))))
@@ -791 +791 @@
-to)
+to#to_prep)
@@ -793 +793 @@
-surrender)
+surrender#surrender_v)
@@ -796 +796 @@
-his)
+his#his_adj)
@@ -798 +798 @@
-said)
+said#said_adj)
@@ -800 +800 @@
-house))))))))
+house#house_n1))))))))
@@ -805 +805 @@
-and)
+and#and_conj1)
@@ -808 +808 @@
-think)
+think#think_v2)
@@ -811 +811 @@
-that)
+that#that_conj)
@@ -815 +815 @@
-he))
+he#he_pron))
@@ -817 +817 @@
-myght)
+myght#may_v1)
@@ -819 +819 @@
-have)
+have#have_v)
@@ -821 +821 @@
-kept)
+kept#keep_v)
@@ -824 +824 @@
-the)
+the#the_adj)
@@ -826 +826 @@
-same))
+same#same_adj))
@@ -831 +831 @@
-if)
+if#if_conj)
@@ -839 +839 @@
-yo=r=)
+yo=r=#your_pron)
@@ -841 +841 @@
-Lordshyp))
+Lordshyp#lordship_n))
@@ -844 +844 @@
-and)
+and#and_conj1)
@@ -847 +847 @@
-I))))
+I#i_pron))))
@@ -849 +849 @@
-had)
+had#have_v)
@@ -851 +851 @@
-not)
+not#not_adv)
@@ -853 +853 @@
-led)
+led#lead_v1)
@@ -856 +856 @@
-him))
+him#him_pron))
@@ -859 +859 @@
-to)
+to#to_prep)
@@ -862 +862 @@
-the)
+the#the_adj)
@@ -864 +864 @@
-said)
+said#said_adj)
@@ -866 +866 @@
-surrender))))))))))))))
+surrender#surrender_n))))))))))))))
@@ -874 +874 @@
-But)
+But#but_prep)
@@ -877 +877 @@
-suerly))
+suerly#surely_adv))
@@ -882 +882 @@
-I))
+I#i_pron))
@@ -884 +884 @@
-beleve)
+beleve#believe_v)
@@ -889 +889 @@
-that)
+that#that_conj)
@@ -893 +893 @@
-I))
+I#i_pron))
@@ -895 +895 @@
-knowe)
+knowe#know_v)
@@ -898 +898 @@
-the)
+the#the_adj)
@@ -900 +900 @@
-man))
+man#man_n1))
@@ -903 +903 @@
-so)
+so#so_adv)
@@ -905 +905 @@
-well)
+well#well_adv)
@@ -908 +908 @@
-that)
+that#that_conj)
@@ -914 +914 @@
-how)
+how#how_adv)
@@ -916 +916 @@
-soever))
+soever#soever_adv))
@@ -924 +924 @@
-he))
+he#he_pron))
@@ -926 +926 @@
-be)
+be#be_v)
@@ -928 +928 @@
-order))))
+order#order_v))))
@@ -931 +931 @@
-he))
+he#he_pron))
@@ -933 +933 @@
-wol@)
+wol@#will_v1)
@@ -935 +935 @@
-@be)
+@be#be_v)
@@ -938 +938 @@
-contented))
+contented#contented_adj))
@@ -941 +941 @@
-without)
+without#without_adv)
@@ -944 +944 @@
-grudge))))))))
+grudge#grudge_n))))))))
@@ -953 +953 @@
-He))
+He#he_pron))
@@ -955 +955 @@
-is)
+is#be_v)
@@ -958 +958 @@
-a)
+a#a_adj2)
@@ -960 +960 @@
-man)
+man#man_n1)
@@ -963 +963 @@
-of)
+of#of_prep)
@@ -966 +966 @@
-suche)
+suche#such_adj)
@@ -968 +968 @@
-charite)
+charite#charity_n)
@@ -971 +971 @@
-as)
+as#as_adv)
@@ -978 +978 @@
-I))
+I#i_pron))
@@ -980 +980 @@
-have)
+have#have_v)
@@ -982 +982 @@
-not)
+not#not_adv)
@@ -984 +984 @@
-seen)
+seen#see_v)
@@ -987 +987 @@
-the)
+the#the_adj)
@@ -989 +989 @@
-like))))))))
+like#like_adj))))))))
@@ -998 +998 @@
-As)
+As#as_adv)
@@ -1001 +1001 @@
-towching)
+towching#touching_prep)
@@ -1004 +1004 @@
-the)
+the#the_adj)
@@ -1006 +1006 @@
-house)
+house#house_n1)
@@ -1009 +1009 @@
-of)
+of#of_prep)
@@ -1012 +1012 @@
-the)
+the#the_adj)
@@ -1014 +1014 @@
-Charterhouse))))))
+Charterhouse#NA))))))
@@ -1017 +1017 @@
-I))
+I#i_pron))
@@ -1019 +1019 @@
-pray)
+pray#pray_v)
@@ -1022 +1022 @@
-Good))
+Good#NA))
@@ -1026 +1026 @@
-if)
+if#if_conj)
@@ -1033 +1033 @@
-it))
+it#it_pron))
@@ -1035 +1035 @@
-shall)
+shall#shall_v)
@@ -1037 +1037 @@
-please)
+please#please_v)
@@ -1040 +1040 @@
-the)
+the#the_adj)
@@ -1042 +1042 @@
-King))
+King#king_n))
@@ -1045 +1045 @@
-to)
+to#to_prep)
@@ -1047 +1047 @@
-alter)
+alter#alter_v)
@@ -1050 +1050 @@
-it))))))
+it#it_pron))))))
@@ -1054 +1054 @@
-that)
+that#that_conj)
@@ -1060 +1060 @@
-it))
+it#it_pron))
@@ -1062 +1062 @@
-may)
+may#may_v1)
@@ -1064 +1064 @@
-be)
+be#be_v)
@@ -1066 +1066 @@
-turned)
+turned#turn_v)
@@ -1069 +1069 @@
-into)
+into#into_prep)
@@ -1072 +1072 @@
-a)
+a#a_adj2)
@@ -1074 +1074 @@
-better)
+better#better_adj)
@@ -1076 +1076 @@
-use)))
+use#use_n)))
@@ -1081 +1081 @@
-seing)
+seing#see_v)
@@ -1089 +1089 @@
-it))
+it#it_pron))
@@ -1091 +1091 @@
-is)
+is#be_v)
@@ -1094 +1094 @@
-in)
+in#in_prep)
@@ -1097 +1097 @@
-the)
+the#the_adj)
@@ -1099 +1099 @@
-face)
+face#face_n)
@@ -1102 +1102 @@
-of)
+of#of_prep)
@@ -1105 +1105 @@
-the)
+the#the_adj)
@@ -1107 +1107 @@
-world))))))
+world#world_n))))))
@@ -1112 +1112 @@
-and)
+and#and_conj1)
@@ -1118 +1118 @@
-muche)
+muche#much_adj)
@@ -1120 +1120 @@
-communication)
+communication#communication_n)
@@ -1124 +1124 @@
-wol)
+wol#will_v1)
@@ -1126 +1126 @@
-run)
+run#run_v)
@@ -1129 +1129 @@
-thereof))
+thereof#thereof_adv))
@@ -1132 +1132 @@
-throughout)
+throughout#throughout_prep)
@@ -1135 +1135 @@
-this)
+this#this_pron)
@@ -1137 +1137 @@
-realme))))))))))
+realme#realm_n))))))))))
@@ -1145 +1145 @@
-for)
+for#for_prep)
@@ -1148 +1148 @@
-London))
+London#NA))
@@ -1150 +1150 @@
-is)
+is#be_v)
@@ -1153 +1153 @@
-the)
+the#the_adj)
@@ -1155 +1155 @@
-common)
+common#common_adj)
@@ -1157 +1157 @@
-countrey)
+countrey#country_n)
@@ -1160 +1160 @@
-of)
+of#of_prep)
@@ -1163 +1163 @@
-al)
+al#all_adj)
@@ -1165 +1165 @@
-England)))
+England#NA)))
@@ -1171 +1171 @@
-from)
+from#from_prep)
@@ -1174 +1174 @@
-which)))
+which#which_adj)))
@@ -1183 +1183 @@
-is)
+is#be_v)
@@ -1185 +1185 @@
-derived)
+derived#derive_v)
@@ -1188 +1188 @@
-to)
+to#to_prep)
@@ -1191 +1191 @@
-al)
+al#all_adj)
@@ -1193 +1193 @@
-parts)
+parts#part_n)
@@ -1196 +1196 @@
-of)
+of#of_prep)
@@ -1199 +1199 @@
-this)
+this#this_pron)
@@ -1201 +1201 @@
-realme)))))
+realme#realm_n)))))
@@ -1204 +1204 @@
-al)
+al#all_adj)
@@ -1206 +1206 @@
-good)
+good#good_adj)
@@ -1209 +1209 @@
-and)
+and#and_conj1)
@@ -1212 +1212 @@
-yll)))
+yll#ill_adj)))
@@ -1215 +1215 @@
-occurrent)
+occurrent#occurrent_adj)
@@ -1218 +1218 @@
-here)))))))
+here#here_adv)))))))
@@ -1226 +1226 @@
-From)
+From#from_prep)
@@ -1229 +1229 @@
-London)))
+London#NA)))
@@ -1235 +1235 @@
-the)
+the#the_adj)
@@ -1237 +1237 @@
-xiiij=th=.)
+xiiij=th=.#NA)
@@ -1239 +1239 @@
-day)
+day#day_n)
@@ -1242 +1242 @@
-of)
+of#of_prep)
@@ -1245 +1245 @@
-Juny)))
+Juny#NA)))
@@ -1253 +1253 @@
-By)
+By#by_prep)
@@ -1257 +1257 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1259 +1259 @@
-Lordships))
+Lordships#lordship_n))
@@ -1262 +1262 @@
-at)
+at#at_prep)
@@ -1265 +1265 @@
-commaundement)))
+commaundement#commandment_n)))
@@ -1268 +1268 @@
-Thomas)
+Thomas#NA)
@@ -1270 +1270 @@
-Bedyll))))
+Bedyll#NA))))
@@ -1279 +1279 @@
-Ther))
+Ther#there_adv))
@@ -1281 +1281 @@
-be)
+be#be_v)
@@ -1283 +1283 @@
-departed)
+departed#depart_v)
@@ -1291 +1291 @@
-Brother)
+Brother#NA)
@@ -1293 +1293 @@
-William)
+William#NA)
@@ -1295 +1295 @@
-Greenewode))
+Greenewode#NA))
@@ -1301 +1301 @@
-Dane)
+Dane#NA)
@@ -1303 +1303 @@
-John)
+John#NA)
@@ -1305 +1305 @@
-Davye)))
+Davye#NA)))
@@ -1311 +1311 @@
-Brother)
+Brother#NA)
@@ -1313 +1313 @@
-Robert)
+Robert#NA)
@@ -1315 +1315 @@
-Salt)))
+Salt#NA)))
@@ -1321 +1321 @@
-Brother)
+Brother#NA)
@@ -1323 +1323 @@
-Water)
+Water#NA)
@@ -1325 +1325 @@
-Peereson)))
+Peereson#NA)))
@@ -1331 +1331 @@
-Dane)
+Dane#NA)
@@ -1333 +1333 @@
-Thomas)
+Thomas#NA)
@@ -1335 +1335 @@
-Greene))))
+Greene#NA))))
@@ -1346 +1346 @@
-Ther))
+Ther#there_adv))
@@ -1348 +1348 @@
-be)
+be#be_v)
@@ -1351 +1351 @@
-even)
+even#even_adj)
@@ -1353 +1353 @@
-at)
+at#at_prep)
@@ -1356 +1356 @@
-the)
+the#the_adj)
@@ -1358 +1358 @@
-poynt)
+poynt#point_n1)
@@ -1361 +1361 @@
-of)
+of#of_prep)
@@ -1364 +1364 @@
-dethe)))))
+dethe#death_n)))))
@@ -1372 +1372 @@
-Brother)
+Brother#NA)
@@ -1374 +1374 @@
-Thomas)
+Thomas#NA)
@@ -1376 +1376 @@
-Scryven))
+Scryven#NA))
@@ -1382 +1382 @@
-Brother)
+Brother#NA)
@@ -1384 +1384 @@
-Thomas)
+Thomas#NA)
@@ -1386 +1386 @@
-Reedyng))))
+Reedyng#NA))))
@@ -1397 +1397 @@
-Ther))
+Ther#there_adv))
@@ -1399 +1399 @@
-be)
+be#be_v)
@@ -1402 +1402 @@
-sycke))
+sycke#sick_adj))
@@ -1410 +1410 @@
-Dane)
+Dane#NA)
@@ -1412 +1412 @@
-Thomas)
+Thomas#NA)
@@ -1414 +1414 @@
-Jonson))
+Jonson#NA))
@@ -1420 +1420 @@
-Brother)
+Brother#NA)
@@ -1422 +1422 @@
-William)
+William#NA)
@@ -1424 +1424 @@
-Hore))))
+Hore#NA))))
@@ -1435 +1435 @@
-One)
+One#one_adj)
@@ -1439 +1439 @@
-is)
+is#be_v)
@@ -1442 +1442 @@
-hole))
+hole#whole_adj))
@@ -1449 +1449 @@
-Dane)
+Dane#NA)
@@ -1451 +1451 @@
-Bird))
+Bird#NA))
