--- capel-1676-e3-h.psd.vert	2025-09-10 13:50:35
+++ capel-1676-e3-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-May)
+May#may_v1)
@@ -17 +17 @@
-it))
+it#it_pron))
@@ -19 +19 @@
-please)
+please#please_v)
@@ -22 +22 @@
-yr)
+yr#your_pron)
@@ -24 +24 @@
-Ma=ty=)))
+Ma=ty=#majesty_n)))
@@ -27 +27 @@
-Having)
+Having#have_v)
@@ -29 +29 @@
-sent)
+sent#send_v1)
@@ -31 +31 @@
-over)
+over#over_adv)
@@ -34 +34 @@
-Sir)
+Sir#NA)
@@ -36 +36 @@
-Cyrill)
+Cyrill#NA)
@@ -38 +38 @@
-Wyche)
+Wyche#NA)
@@ -43 +43 @@
-to)
+to#to_prep)
@@ -45 +45 @@
-attend)
+attend#attend_v)
@@ -49 +49 @@
-yr)
+yr#your_pron)
@@ -51 +51 @@
-Majesties))
+Majesties#majesty_n))
@@ -53 +53 @@
-service)
+service#service_n1)
@@ -56 +56 @@
-in)
+in#in_prep)
@@ -59 +59 @@
-Parlm=t=)))))
+Parlm=t=#parliament_n1)))))
@@ -63 +63 @@
-who))
+who#who_pron))
@@ -71 +71 @@
-being)
+being#be_v)
@@ -74 +74 @@
-sufficiently))
+sufficiently#sufficiently_adv))
@@ -76 +76 @@
-instructed)
+instructed#instruct_v)
@@ -79 +79 @@
-to)
+to#to_prep)
@@ -81 +81 @@
-give)
+give#give_v)
@@ -84 +84 @@
-yr)
+yr#your_pron)
@@ -86 +86 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -89 +89 @@
-a)
+a#a_adj2)
@@ -91 +91 @@
-full)
+full#full_adj)
@@ -93 +93 @@
-account)
+account#account_n)
@@ -96 +96 @@
-of)
+of#of_prep)
@@ -99 +99 @@
-all)
+all#all_adj)
@@ -101 +101 @@
-ye)
+ye#the_adj)
@@ -103 +103 @@
-affairs)
+affairs#affair_n)
@@ -106 +106 @@
-in)
+in#in_prep)
@@ -109 +109 @@
-this)
+this#this_pron)
@@ -111 +111 @@
-Kingdom))))))))
+Kingdom#kingdom_n))))))))
@@ -114 +114 @@
-T'@))
+T'@#it_pron))
@@ -116 +116 @@
-@will)
+@will#will_v1)
@@ -118 +118 @@
-be)
+be#be_v)
@@ -121 +121 @@
-unnecessary))
+unnecessary#unnecessary_adj))
@@ -125 +125 @@
-by)
+by#by_prep)
@@ -128 +128 @@
-this)
+this#this_pron)
@@ -130 +130 @@
-Letter)))
+Letter#letter_n1)))
@@ -132 +132 @@
-to)
+to#to_prep)
@@ -134 +134 @@
-trouble)
+trouble#trouble_v)
@@ -137 +137 @@
-yr)
+yr#your_pron)
@@ -139 +139 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -142 +142 @@
-with)
+with#with_prep)
@@ -145 +145 @@
-ye)
+ye#the_adj)
@@ -147 +147 @@
-particulars)))))))
+particulars#particular_adj)))))))
@@ -152 +152 @@
-Two)
+Two#two_adj)
@@ -154 +154 @@
-things)
+things#thing_n1)
@@ -158 +158 @@
-have)
+have#have_v)
@@ -161 +161 @@
-principally))
+principally#principally_adv))
@@ -163 +163 @@
-employd)
+employd#employ_v)
@@ -166 +166 @@
-my)
+my#my_adj)
@@ -168 +168 @@
-time)
+time#time_n)
@@ -171 +171 @@
-here))
+here#here_adv))
@@ -174 +174 @@
-since)
+since#since_adv)
@@ -177 +177 @@
-my)
+my#my_adj)
@@ -179 +179 @@
-last)
+last#last_adv)
@@ -182 +182 @@
-kissing)
+kissing#kiss_v)
@@ -186 +186 @@
-yr)
+yr#your_pron)
@@ -188 +188 @@
-Mat=y's=))
+Mat=y's=#majesty_n))
@@ -190 +190 @@
-hands))))))
+hands#hand_n))))))
@@ -196 +196 @@
-The)
+The#the_adj)
@@ -201 +201 @@
-ascertaining)
+ascertaining#ascertain_v)
@@ -204 +204 @@
-ye)
+ye#the_adj)
@@ -206 +206 @@
-Quit)
+Quit#quit_adj)
@@ -208 +208 @@
-rents))
+rents#rent_n1))
@@ -211 +211 @@
-by)
+by#by_prep)
@@ -214 +214 @@
-reducing)
+reducing#reduce_v)
@@ -217 +217 @@
-those)
+those#those_pron)
@@ -221 +221 @@
-wch))
+wch#which_adj))
@@ -228 +228 @@
-are)
+are#be_v)
@@ -231 +231 @@
-upon)
+upon#upon_prep)
@@ -234 +234 @@
-barren)
+barren#barren_adj)
@@ -236 +236 @@
-lands))))))))))
+lands#land_n1))))))))))
@@ -241 +241 @@
-and)
+and#and_conj1)
@@ -244 +244 @@
-the)
+the#the_adj)
@@ -249 +249 @@
-settling)
+settling#settle_v)
@@ -252 +252 @@
-ye)
+ye#the_adj)
@@ -254 +254 @@
-province)
+province#province_n)
@@ -257 +257 @@
-of)
+of#of_prep)
@@ -260 +260 @@
-Conaght)))))))
+Conaght#NA)))))))
@@ -266 +266 @@
-both)
+both#both_pron)
@@ -268 +268 @@
-wch))
+wch#which_adj))
@@ -275 +275 @@
-will)
+will#will_v1)
@@ -279 +279 @@
-I))
+I#i_pron))
@@ -281 +281 @@
-doubt)
+doubt#doubt_v)
@@ -283 +283 @@
-not))
+not#not_adv))
@@ -285 +285 @@
-turne)
+turne#turn_v)
@@ -288 +288 @@
-to)
+to#to_prep)
@@ -291 +291 @@
-acct)))
+acct#act_n)))
@@ -294 +294 @@
-for)
+for#for_prep)
@@ -298 +298 @@
-your)
+your#your_pron)
@@ -300 +300 @@
-Mat=y's=))
+Mat=y's=#majesty_n))
@@ -302 +302 @@
-service))))))
+service#service_n1))))))
@@ -310 +310 @@
-For)
+For#for_prep)
@@ -313 +313 @@
-ye)
+ye#the_adj)
@@ -315 +315 @@
-first)
+first#first_adj)
@@ -318 +318 @@
-of)
+of#of_prep)
@@ -321 +321 @@
-them))))
+them#them_pron))))
@@ -323 +323 @@
-will)
+will#will_v1)
@@ -325 +325 @@
-render)
+render#render_v)
@@ -329 +329 @@
-that)
+that#that_pron)
@@ -331 +331 @@
-Revenue))
+Revenue#revenue_n))
@@ -335 +335 @@
-for)
+for#for_prep)
@@ -338 +338 @@
-ever)
+ever#ever_adv)
@@ -341 +341 @@
-after))))
+after#after_adv))))
@@ -343 +343 @@
-solvent)))
+solvent#solvent_adj)))
@@ -351 +351 @@
-And)
+And#and_conj1)
@@ -354 +354 @@
-ye)
+ye#the_adj)
@@ -356 +356 @@
-latter))
+latter#latter_adj))
@@ -359 +359 @@
-as)
+as#as_adv)
@@ -371 +371 @@
-it))
+it#it_pron))
@@ -373 +373 @@
-is)
+is#be_v)
@@ -376 +376 @@
-a)
+a#a_adj2)
@@ -378 +378 @@
-work)
+work#work_n)
@@ -381 +381 @@
-of)
+of#of_prep)
@@ -384 +384 @@
-great)
+great#great_adj)
@@ -386 +386 @@
-Justice)
+Justice#justice_n)
@@ -389 +389 @@
-and)
+and#and_conj1)
@@ -392 +392 @@
-Charity)))))))
+Charity#charity_n)))))))
@@ -397 +397 @@
-and)
+and#and_conj1)
@@ -404 +404 @@
-will)
+will#will_v1)
@@ -406 +406 @@
-relieve)
+relieve#relieve_v)
@@ -409 +409 @@
-many)
+many#many_adj)
@@ -411 +411 @@
-poor)
+poor#poor_adj)
@@ -413 +413 @@
-familys)
+familys#family_n)
@@ -419 +419 @@
-who))
+who#who_pron))
@@ -426 +426 @@
-have)
+have#have_v)
@@ -429 +429 @@
-for)
+for#for_prep)
@@ -432 +432 @@
-divers)
+divers#divers_adj)
@@ -434 +434 @@
-years)))
+years#year_n)))
@@ -436 +436 @@
-bin)
+bin#be_v)
@@ -438 +438 @@
-kept)
+kept#keep_v)
@@ -441 +441 @@
-out)
+out#out_adv)
@@ -443 +443 @@
-of)
+of#of_prep)
@@ -446 +446 @@
-their)
+their#their_adj)
@@ -448 +448 @@
-estate)))))))))))
+estate#estate_n)))))))))))
@@ -453 +453 @@
-so))
+so#so_adv))
@@ -455 +455 @@
-will)
+will#will_v1)
@@ -458 +458 @@
-it))
+it#it_pron))
@@ -460 +460 @@
-also)
+also#also_adv)
@@ -463 +463 @@
-in)
+in#in_prep)
@@ -466 +466 @@
-some)
+some#some_pron)
@@ -468 +468 @@
-proportion)))
+proportion#proportion_n)))
@@ -470 +470 @@
-improve)
+improve#improve_v2)
@@ -474 +474 @@
-yr)
+yr#your_pron)
@@ -476 +476 @@
-Mat=ey's=))
+Mat=ey's=#majesty_n))
@@ -478 +478 @@
-Revenue))
+Revenue#revenue_n))
@@ -481 +481 @@
-by)
+by#by_prep)
@@ -484 +484 @@
-bringing)
+bringing#bring_v)
@@ -487 +487 @@
-many)
+many#many_adj)
@@ -489 +489 @@
-Quitt)
+Quitt#quit_adj)
@@ -491 +491 @@
-Rents))
+Rents#rent_n1))
@@ -494 +494 @@
-into)
+into#into_prep)
@@ -497 +497 @@
-charge)))
+charge#charge_n)))
@@ -503 +503 @@
-wch))
+wch#which_adj))
@@ -510 +510 @@
-could)
+could#can_v1)
@@ -512 +512 @@
-not)
+not#not_adv)
@@ -514 +514 @@
-be)
+be#be_v)
@@ -516 +516 @@
-done)
+done#do_v)
@@ -519 +519 @@
-till)
+till#till_prep)
@@ -526 +526 @@
-ye)
+ye#the_adj)
@@ -528 +528 @@
-lands))
+lands#land_n1))
@@ -530 +530 @@
-were)
+were#be_v)
@@ -532 +532 @@
-passt)
+passt#pass_v)
@@ -535 +535 @@
-in)
+in#in_prep)
@@ -538 +538 @@
-patent))))))))))
+patent#patent_n))))))))))
@@ -547 +547 @@
-In)
+In#in_prep)
@@ -550 +550 @@
-both)
+both#both_pron)
@@ -553 +553 @@
-these))
+these#these_pron))
@@ -556 +556 @@
-as)
+as#as_adv)
@@ -559 +559 @@
-also)
+also#also_adv)
@@ -561 +561 @@
-in)
+in#in_prep)
@@ -564 +564 @@
-all)
+all#all_adj)
@@ -566 +566 @@
-other)
+other#other_adj)
@@ -568 +568 @@
-matters))))))
+matters#matter_n1))))))
@@ -571 +571 @@
-Sir)
+Sir#NA)
@@ -573 +573 @@
-Cyrill)
+Cyrill#NA)
@@ -575 +575 @@
-Wyche))
+Wyche#NA))
@@ -580 +580 @@
-when)
+when#when_adv)
@@ -587 +587 @@
-yr)
+yr#your_pron)
@@ -589 +589 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -591 +591 @@
-shall)
+shall#shall_v)
@@ -593 +593 @@
-please)
+please#please_v)
@@ -596 +596 @@
-to)
+to#to_prep)
@@ -598 +598 @@
-allow)
+allow#allow_v)
@@ -601 +601 @@
-him))
+him#him_pron))
@@ -604 +604 @@
-ye)
+ye#the_adj)
@@ -606 +606 @@
-opportunity))))))
+opportunity#opportunity_n))))))
@@ -608 +608 @@
-will)
+will#will_v1)
@@ -610 +610 @@
-give)
+give#give_v)
@@ -613 +613 @@
-yr)
+yr#your_pron)
@@ -615 +615 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -618 +618 @@
-a)
+a#a_adj2)
@@ -620 +620 @@
-relacon)
+relacon#relation_n)
@@ -623 +623 @@
-of)
+of#of_prep)
@@ -627 +627 @@
-ye)
+ye#the_adj)
@@ -629 +629 @@
-methods)
+methods#method_n)
@@ -632 +632 @@
-of)
+of#of_prep)
@@ -635 +635 @@
-my)
+my#my_adj)
@@ -637 +637 @@
-proceedings))))
+proceedings#proceeding_n))))
@@ -642 +642 @@
-and)
+and#and_conj1)
@@ -645 +645 @@
-ye)
+ye#the_adj)
@@ -647 +647 @@
-measures)
+measures#measure_n)
@@ -658 +658 @@
-I))
+I#i_pron))
@@ -660 +660 @@
-take)
+take#take_v)
@@ -663 +663 @@
-in)
+in#in_prep)
@@ -666 +666 @@
-all)))))))))
+all#all_adj)))))))))
@@ -672 +672 @@
-wherein))
+wherein#wherein_adv))
@@ -678 +678 @@
-I))
+I#i_pron))
@@ -680 +680 @@
-hope)
+hope#hope_v)
@@ -687 +687 @@
-yr)
+yr#your_pron)
@@ -689 +689 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -691 +691 @@
-will)
+will#will_v1)
@@ -693 +693 @@
-receive)
+receive#receive_v)
@@ -696 +696 @@
-that)
+that#that_pron)
@@ -698 +698 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -703 +703 @@
-as)
+as#as_adv)
@@ -713 +713 @@
-may)
+may#may_v1)
@@ -715 +715 @@
-bee)
+bee#be_v)
@@ -718 +718 @@
-a)
+a#a_adj2)
@@ -720 +720 @@
-further)
+further#further_adj)
@@ -722 +722 @@
-confirmation)
+confirmation#confirmation_n)
@@ -725 +725 @@
-of)
+of#of_prep)
@@ -728 +728 @@
-ye)
+ye#the_adj)
@@ -730 +730 @@
-truth)
+truth#truth_n)
@@ -733 +733 @@
-and)
+and#and_conj1)
@@ -736 +736 @@
-reallity)))
+reallity#reality_n)))
@@ -742 +742 @@
-wherein))
+wherein#wherein_adv))
@@ -750 +750 @@
-I))
+I#i_pron))
@@ -752 +752 @@
-desire)
+desire#desire_v)
@@ -755 +755 @@
-to)
+to#to_prep)
@@ -757 +757 @@
-appear)
+appear#appear_v)
@@ -760 +760 @@
-in)
+in#in_prep)
@@ -763 +763 @@
-all)
+all#all_adj)
@@ -765 +765 @@
-humility)))
+humility#humility_n)))
@@ -769 +769 @@
-Yr)
+Yr#your_pron)
@@ -771 +771 @@
-Mat=ys=))
+Mat=ys=#majesty_n))
@@ -775 +775 @@
-etc.)
+etc.#et-cetera_n)
@@ -779 +779 @@
-etc.))))))))))))))))))
+etc.#et-cetera_n))))))))))))))))))
@@ -792 +792 @@
-May)
+May#may_v1)
@@ -795 +795 @@
-it))
+it#it_pron))
@@ -797 +797 @@
-please)
+please#please_v)
@@ -800 +800 @@
-yr)
+yr#your_pron)
@@ -802 +802 @@
-Mat=y=)))
+Mat=y=#majesty_n)))
@@ -807 +807 @@
-I))
+I#i_pron))
@@ -809 +809 @@
-doe)
+doe#do_v)
@@ -812 +812 @@
-with)
+with#with_prep)
@@ -815 +815 @@
-all)
+all#all_adj)
@@ -817 +817 @@
-humility)))
+humility#humility_n)))
@@ -819 +819 @@
-acknowledge)
+acknowledge#acknowledge_v)
@@ -823 +823 @@
-ye)
+ye#the_adj)
@@ -825 +825 @@
-favour)
+favour#favour_n)
@@ -834 +834 @@
-yr)
+yr#your_pron)
@@ -836 +836 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -838 +838 @@
-was)
+was#be_v)
@@ -840 +840 @@
-pleased)
+pleased#please_v)
@@ -845 +845 @@
-to)
+to#to_prep)
@@ -847 +847 @@
-doe)
+doe#do_v)
@@ -850 +850 @@
-mee))
+mee#me_pron1))
@@ -853 +853 @@
-in)
+in#in_prep)
@@ -856 +856 @@
-signifying)
+signifying#signify_v)
@@ -859 +859 @@
-by)
+by#by_prep)
@@ -862 +862 @@
-a)
+a#a_adj2)
@@ -864 +864 @@
-letter)
+letter#letter_n1)
@@ -867 +867 @@
-written)
+written#write_v)
@@ -870 +870 @@
-with)
+with#with_prep)
@@ -873 +873 @@
-yr)
+yr#your_pron)
@@ -875 +875 @@
-owne)
+owne#own_adj)
@@ -877 +877 @@
-hand))))))
+hand#hand_n))))))
@@ -880 +880 @@
-yr)
+yr#your_pron)
@@ -882 +882 @@
-intention)
+intention#intention_n)
@@ -885 +885 @@
-of)
+of#of_prep)
@@ -888 +888 @@
-my)
+my#my_adj)
@@ -891 +891 @@
-returning)
+returning#return_v1)
@@ -894 +894 @@
-home))))))))))))
+home#home_n1))))))))))))
@@ -899 +899 @@
-and)
+and#and_conj1)
@@ -902 +902 @@
-that)
+that#that_conj)
@@ -906 +906 @@
-yr)
+yr#your_pron)
@@ -908 +908 @@
-Mat=ey=))
+Mat=ey=#majesty_n))
@@ -910 +910 @@
-has)
+has#have_v)
@@ -912 +912 @@
-also)
+also#also_adv)
@@ -915 +915 @@
-therein))
+therein#therein_adv))
@@ -917 +917 @@
-given)
+given#give_v)
@@ -920 +920 @@
-mee))
+mee#me_pron1))
@@ -923 +923 @@
-such)
+such#such_adj)
@@ -925 +925 @@
-an)
+an#a_adj2)
@@ -927 +927 @@
-undeniable)
+undeniable#undeniable_adj)
@@ -929 +929 @@
-testimony)
+testimony#testimony_n)
@@ -932 +932 @@
-of)
+of#of_prep)
@@ -935 +935 @@
-yr)
+yr#your_pron)
@@ -937 +937 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -940 +940 @@
-in)
+in#in_prep)
@@ -943 +943 @@
-my)
+my#my_adj)
@@ -945 +945 @@
-management)
+management#management_n)
@@ -948 +948 @@
-of)
+of#of_prep)
@@ -951 +951 @@
-those)
+those#those_pron)
@@ -953 +953 @@
-affairs)
+affairs#affair_n)
@@ -957 +957 @@
-wch))
+wch#which_adj))
@@ -965 +965 @@
-yr)
+yr#your_pron)
@@ -967 +967 @@
-Mat=ey=))
+Mat=ey=#majesty_n))
@@ -969 +969 @@
-hath)
+hath#have_v)
@@ -972 +972 @@
-for)
+for#for_prep)
@@ -976 +976 @@
-almost)
+almost#almost_adv)
@@ -978 +978 @@
-five))
+five#five_v))
@@ -980 +980 @@
-years)))
+years#year_n)))
@@ -982 +982 @@
-committed)
+committed#commit_v)
@@ -985 +985 @@
-to)
+to#to_prep)
@@ -988 +988 @@
-my)
+my#my_adj)
@@ -990 +990 @@
-care)))))))))))
+care#care_n1)))))))))))
@@ -993 +993 @@
-wth)
+wth#with_prep)
@@ -996 +996 @@
-ye)
+ye#the_adj)
@@ -998 +998 @@
-assurance)
+assurance#assurance_n)
@@ -1001 +1001 @@
-yt)
+yt#that_conj)
@@ -1005 +1005 @@
-you))
+you#you_pron))
@@ -1007 +1007 @@
-will)
+will#will_v1)
@@ -1009 +1009 @@
-be)
+be#be_v)
@@ -1011 +1011 @@
-pleased)
+pleased#please_v)
@@ -1014 +1014 @@
-to)
+to#to_prep)
@@ -1016 +1016 @@
-retain)
+retain#retain_v)
@@ -1019 +1019 @@
-in)
+in#in_prep)
@@ -1022 +1022 @@
-yr)
+yr#your_pron)
@@ -1024 +1024 @@
-princely)
+princely#princely_adj)
@@ -1026 +1026 @@
-memory)))
+memory#memory_n)))
@@ -1029 +1029 @@
-ye)
+ye#the_adj)
@@ -1031 +1031 @@
-endeavors)
+endeavors#endeavour_n)
@@ -1042 +1042 @@
-I))
+I#i_pron))
@@ -1044 +1044 @@
-have)
+have#have_v)
@@ -1046 +1046 @@
-used)
+used#use_v)
@@ -1050 +1050 @@
-justly)
+justly#justly_adv)
@@ -1052 +1052 @@
-and)
+and#and_conj1)
@@ -1054 +1054 @@
-honestly))
+honestly#honestly_adv))
@@ -1056 +1056 @@
-to)
+to#to_prep)
@@ -1058 +1058 @@
-discharge)
+discharge#discharge_v)
@@ -1061 +1061 @@
-my)
+my#my_adj)
@@ -1063 +1063 @@
-Trust)
+Trust#trust_n)
@@ -1067 +1067 @@
-wch))
+wch#which_adj))
@@ -1073 +1073 @@
-truly))
+truly#truly_adv))
@@ -1076 +1076 @@
-I))
+I#i_pron))
@@ -1078 +1078 @@
-can)
+can#can_v1)
@@ -1080 +1080 @@
-justify)
+justify#justify_v)
@@ -1083 +1083 @@
-to)
+to#to_prep)
@@ -1086 +1086 @@
-all)
+all#all_adj)
@@ -1088 +1088 @@
-ye)
+ye#the_adj)
@@ -1090 +1090 @@
-world)))
+world#world_n)))
@@ -1099 +1099 @@
-I))
+I#i_pron))
@@ -1101 +1101 @@
-have)
+have#have_v)
@@ -1103 +1103 @@
-performed)
+performed#perform_v)
@@ -1106 +1106 @@
-with)
+with#with_prep)
@@ -1109 +1109 @@
-an)
+an#a_adj2)
@@ -1111 +1111 @@
-upright)
+upright#upright_adj)
@@ -1113 +1113 @@
-conscience)))))))))))))))))))))))
+conscience#conscience_n)))))))))))))))))))))))
@@ -1119 +1119 @@
-and)
+and#and_conj1)
@@ -1122 +1122 @@
-if)
+if#if_conj)
@@ -1129 +1129 @@
-any)
+any#any_adj)
@@ -1131 +1131 @@
-cavills)
+cavills#cavil_n)
@@ -1135 +1135 @@
-have)
+have#have_v)
@@ -1137 +1137 @@
-arisen)
+arisen#arise_v)
@@ -1140 +1140 @@
-agnst)
+agnst#against_prep)
@@ -1143 +1143 @@
-my)
+my#my_adj)
@@ -1145 +1145 @@
-proceedings)
+proceedings#proceeding_n)
@@ -1148 +1148 @@
-here)))))))
+here#here_adv)))))))
@@ -1151 +1151 @@
-it))
+it#it_pron))
@@ -1153 +1153 @@
-may)
+may#may_v1)
@@ -1155 +1155 @@
-bee)
+bee#be_v)
@@ -1162 +1162 @@
-they))
+they#they_pron))
@@ -1164 +1164 @@
-have)
+have#have_v)
@@ -1166 +1166 @@
-bin)
+bin#be_v)
@@ -1168 +1168 @@
-founded)
+founded#found_v2)
@@ -1171 +1171 @@
-upon)
+upon#upon_prep)
@@ -1174 +1174 @@
-my)
+my#my_adj)
@@ -1178 +1178 @@
-insisting)
+insisting#insist_v)
@@ -1181 +1181 @@
-strictly))
+strictly#strictly_adv))
@@ -1184 +1184 @@
-on)
+on#on_prep)
@@ -1188 +1188 @@
-yr)
+yr#your_pron)
@@ -1190 +1190 @@
-Mat=y's=))
+Mat=y's=#majesty_n))
@@ -1192 +1192 @@
-Right))))
+Right#right_n))))
@@ -1195 +1195 @@
-and)
+and#and_conj1)
@@ -1198 +1198 @@
-not)
+not#not_adv)
@@ -1200 +1200 @@
-admitting)
+admitting#admit_v)
@@ -1203 +1203 @@
-some)
+some#some_pron)
@@ -1205 +1205 @@
-persons))
+persons#person_n))
@@ -1208 +1208 @@
-to)
+to#to_prep)
@@ -1210 +1210 @@
-take)
+take#take_v)
@@ -1213 +1213 @@
-that)
+that#that_pron)
@@ -1218 +1218 @@
-as)
+as#as_adv)
@@ -1221 +1221 @@
-due)))
+due#due_adj)))
@@ -1225 +1225 @@
-wch))
+wch#which_adj))
@@ -1233 +1233 @@
-if)
+if#if_conj)
@@ -1236 +1236 @@
-allowed)))
+allowed#allow_v)))
@@ -1239 +1239 @@
-they))
+they#they_pron))
@@ -1241 +1241 @@
-ought)
+ought#ought_v)
@@ -1243 +1243 @@
-to)
+to#to_prep)
@@ -1245 +1245 @@
-accept)
+accept#accept_v)
@@ -1248 +1248 @@
-as)
+as#as_adv)
@@ -1251 +1251 @@
-Grace)))
+Grace#grace_n)))
@@ -1254 +1254 @@
-from)
+from#from_prep)
@@ -1257 +1257 @@
-yr)
+yr#your_pron)
@@ -1259 +1259 @@
-Majestie)))))))))))))
+Majestie#majesty_n)))))))))))))
@@ -1267 +1267 @@
-And)
+And#and_conj1)
@@ -1270 +1270 @@
-as)
+as#as_adv)
@@ -1281 +1281 @@
-it))
+it#it_pron))
@@ -1283 +1283 @@
-hath)
+hath#have_v)
@@ -1286 +1286 @@
-ever))
+ever#ever_adv))
@@ -1288 +1288 @@
-bin)
+bin#be_v)
@@ -1291 +1291 @@
-my)
+my#my_adj)
@@ -1293 +1293 @@
-practice)
+practice#practice_n)
@@ -1296 +1296 @@
-in)
+in#in_prep)
@@ -1299 +1299 @@
-these)
+these#these_pron)
@@ -1301 +1301 @@
-Employments)
+Employments#employment_n)
@@ -1305 +1305 @@
-wherewith))
+wherewith#wherewith_adv))
@@ -1311 +1311 @@
-yr)
+yr#your_pron)
@@ -1313 +1313 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -1315 +1315 @@
-hath)
+hath#have_v)
@@ -1317 +1317 @@
-thought)
+thought#think_v2)
@@ -1323 +1323 @@
-fitt))
+fitt#fit_adj))
@@ -1328 +1328 @@
-to)
+to#to_prep)
@@ -1330 +1330 @@
-honour)
+honour#honour_v)
@@ -1333 +1333 @@
-mee)))))))))
+mee#me_pron1)))))))))
@@ -1337 +1337 @@
-strenuously))
+strenuously#strenuously_adv))
@@ -1339 +1339 @@
-to)
+to#to_prep)
@@ -1342 +1342 @@
-assert)
+assert#assert_v)
@@ -1344 +1344 @@
-and)
+and#and_conj1)
@@ -1346 +1346 @@
-maintaine))
+maintaine#maintain_v))
@@ -1349 +1349 @@
-ye)
+ye#the_adj)
@@ -1351 +1351 @@
-prerogative)
+prerogative#prerogative_n)
@@ -1355 +1355 @@
-to)
+to#to_prep)
@@ -1358 +1358 @@
-wch)))
+wch#which_adj)))
@@ -1366 +1366 @@
-you))
+you#you_pron))
@@ -1368 +1368 @@
-were)
+were#be_v)
@@ -1370 +1370 @@
-borne))))))))
+borne#bear_v1))))))))
@@ -1373 +1373 @@
-so))
+so#so_adv))
@@ -1376 +1376 @@
-I))
+I#i_pron))
@@ -1378 +1378 @@
-hope)
+hope#hope_v)
@@ -1385 +1385 @@
-yr)
+yr#your_pron)
@@ -1387 +1387 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -1389 +1389 @@
-will)
+will#will_v1)
@@ -1391 +1391 @@
-be)
+be#be_v)
@@ -1393 +1393 @@
-pleased)
+pleased#please_v)
@@ -1396 +1396 @@
-to)
+to#to_prep)
@@ -1398 +1398 @@
-believe)
+believe#believe_v)
@@ -1401 +1401 @@
-yt)
+yt#that_conj)
@@ -1405 +1405 @@
-in)
+in#in_prep)
@@ -1408 +1408 @@
-all)
+all#all_adj)
@@ -1410 +1410 @@
-capacitys)
+capacitys#capacity_n)
@@ -1421 +1421 @@
-yr)
+yr#your_pron)
@@ -1423 +1423 @@
-Mat=y=))
+Mat=y=#majesty_n))
@@ -1425 +1425 @@
-shall)
+shall#shall_v)
@@ -1428 +1428 @@
-at)
+at#at_prep)
@@ -1431 +1431 @@
-any)
+any#any_adj)
@@ -1433 +1433 @@
-time)))
+time#time_n)))
@@ -1435 +1435 @@
-place)
+place#place_v)
@@ -1438 +1438 @@
-mee))))))
+mee#me_pron1))))))
@@ -1441 +1441 @@
-I))
+I#i_pron))
@@ -1443 +1443 @@
-will)
+will#will_v1)
@@ -1445 +1445 @@
-continue)
+continue#continue_v)
@@ -1448 +1448 @@
-steady)
+steady#steady_adj)
@@ -1450 +1450 @@
-and)
+and#and_conj1)
@@ -1452 +1452 @@
-firme)
+firme#firm_adj)
@@ -1455 +1455 @@
-to)
+to#to_prep)
@@ -1458 +1458 @@
-ye)
+ye#the_adj)
@@ -1460 +1460 @@
-same)
+same#same_adj)
@@ -1462 +1462 @@
-cause))))
+cause#cause_n))))
@@ -1465 +1465 @@
-as)
+as#as_adv)
@@ -1468 +1468 @@
-being)
+being#be_v)
@@ -1471 +1471 @@
-with)
+with#with_prep)
@@ -1474 +1474 @@
-all)
+all#all_adj)
@@ -1476 +1476 @@
-devotion)))
+devotion#devotion_n)))
@@ -1481 +1481 @@
-etc.)
+etc.#et-cetera_n)
@@ -1485 +1485 @@
-etc.))))))))))
+etc.#et-cetera_n))))))))))
@@ -1491 +1491 @@
-To)
+To#to_prep)
@@ -1494 +1494 @@
-Ye)
+Ye#the_adj)
@@ -1496 +1496 @@
-King)))
+King#king_n)))
@@ -1502 +1502 @@
-Dublin)
+Dublin#NA)
@@ -1504 +1504 @@
-Castle))
+Castle#NA))
@@ -1510 +1510 @@
-April)
+April#NA)
@@ -1512 +1512 @@
-28)
+28#NA)
@@ -1516 +1516 @@
-1677)
+1677#NA)
