--- charles-1676-e3-h.psd.vert	2025-09-10 13:50:35
+++ charles-1676-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-Charles)
+Charles#NA)
@@ -16 +16 @@
-R.)))
+R.#rex_n2)))
@@ -25 +25 @@
-Right)
+Right#right_adv)
@@ -27 +27 @@
-Trusty))
+Trusty#trusty_adj))
@@ -30 +30 @@
-and)
+and#and_conj1)
@@ -33 +33 @@
-Right)
+Right#right_adv)
@@ -35 +35 @@
-Wellbeloved))))
+Wellbeloved#well-beloved_adj))))
@@ -37 +37 @@
-Cousin)
+Cousin#cousin_n)
@@ -40 +40 @@
-&)
+&#and_conj1)
@@ -43 +43 @@
-Councellour))))
+Councellour#counsellor_n))))
@@ -48 +48 @@
-Wee))
+Wee#we_pron))
@@ -50 +50 @@
-Greet)
+Greet#greet_v1)
@@ -53 +53 @@
-you))
+you#you_pron))
@@ -56 +56 @@
-Well))
+Well#well_adv))
@@ -65 +65 @@
-Whereas)
+Whereas#whereas_adv)
@@ -72 +72 @@
-Wee))
+Wee#we_pron))
@@ -74 +74 @@
-are)
+are#be_v)
@@ -76 +76 @@
-given)
+given#give_v)
@@ -79 +79 @@
-to)
+to#to_prep)
@@ -81 +81 @@
-understand)
+understand#understand_v)
@@ -85 +85 @@
-that)
+that#that_conj)
@@ -89 +89 @@
-the)
+the#the_adj)
@@ -91 +91 @@
-Earle)
+Earle#earl_n)
@@ -94 +94 @@
-of)
+of#of_prep)
@@ -97 +97 @@
-Orrery))))
+Orrery#NA))))
@@ -99 +99 @@
-hath)
+hath#have_v)
@@ -101 +101 @@
-applyed)
+applyed#apply_v)
@@ -104 +104 @@
-to)
+to#to_prep)
@@ -107 +107 @@
-you)))
+you#you_pron)))
@@ -110 +110 @@
-for)
+for#for_prep)
@@ -113 +113 @@
-your)
+your#your_pron)
@@ -115 +115 @@
-Lycence)
+Lycence#licence_n)
@@ -118 +118 @@
-&)
+&#and_conj1)
@@ -121 +121 @@
-permission)))
+permission#permission_n)))
@@ -124 +124 @@
-to)
+to#to_prep)
@@ -126 +126 @@
-plant)
+plant#plant_v)
@@ -129 +129 @@
-certaine)
+certaine#certain_adj)
@@ -131 +131 @@
-greate)
+greate#great_adj)
@@ -133 +133 @@
-Guns))
+Guns#gun_n))
@@ -136 +136 @@
-in)
+in#in_prep)
@@ -139 +139 @@
-his)
+his#his_adj)
@@ -141 +141 @@
-Castle)
+Castle#castle_n)
@@ -144 +144 @@
-or)
+or#or_conj2)
@@ -147 +147 @@
-House)))
+House#house_n1)))
@@ -150 +150 @@
-of)
+of#of_prep)
@@ -153 +153 @@
-Ballymartin)))))
+Ballymartin#NA)))))
@@ -156 +156 @@
-within)
+within#within_adv)
@@ -159 +159 @@
-that)
+that#that_pron)
@@ -161 +161 @@
-Our)
+Our#our_pron)
@@ -163 +163 @@
-Kingdome))))))))
+Kingdome#kingdom_n))))))))
@@ -168 +168 @@
-and)
+and#and_conj1)
@@ -171 +171 @@
-that)
+that#that_conj)
@@ -175 +175 @@
-hee))
+hee#he_pron))
@@ -177 +177 @@
-doth)
+doth#do_v)
@@ -180 +180 @@
-further))
+further#further_adj))
@@ -182 +182 @@
-pretend)
+pretend#pretend_v)
@@ -185 +185 @@
-to)
+to#to_prep)
@@ -188 +188 @@
-Power)
+Power#power_n1)
@@ -193 +193 @@
-under)
+under#under_prep)
@@ -196 +196 @@
-the)
+the#the_adj)
@@ -198 +198 @@
-Greate)
+Greate#NA)
@@ -200 +200 @@
-Seale)
+Seale#NA)
@@ -203 +203 @@
-of)
+of#of_prep)
@@ -206 +206 @@
-that)
+that#that_pron)
@@ -208 +208 @@
-Our)
+Our#our_pron)
@@ -210 +210 @@
-Kingdome)))))
+Kingdome#kingdom_n)))))
@@ -214 +214 @@
-to)
+to#to_prep)
@@ -216 +216 @@
-fortify)
+fortify#fortify_v)
@@ -219 +219 @@
-one)
+one#one_adj)
@@ -221 +221 @@
-other)
+other#other_adj)
@@ -224 +224 @@
-of)
+of#of_prep)
@@ -227 +227 @@
-his)
+his#his_adj)
@@ -229 +229 @@
-Houses)
+Houses#house_n1)
@@ -232 +232 @@
-called)
+called#call_v)
@@ -238 +238 @@
-Charle)
+Charle#NA)
@@ -240 +240 @@
-Ville)))))))
+Ville#NA)))))))
@@ -243 +243 @@
-in)
+in#in_prep)
@@ -246 +246 @@
-Our)
+Our#our_pron)
@@ -248 +248 @@
-said)
+said#said_adj)
@@ -250 +250 @@
-Kingdome)))
+Kingdome#kingdom_n)))
@@ -253 +253 @@
-with)
+with#with_prep)
@@ -256 +256 @@
-Forts)
+Forts#fort_n1)
@@ -258 +258 @@
-and)
+and#and_conj1)
@@ -260 +260 @@
-Bullwarkes))))
+Bullwarkes#bulwark_n))))
@@ -265 +265 @@
-and)
+and#and_conj1)
@@ -268 +268 @@
-to)
+to#to_prep)
@@ -271 +271 @@
-Mount)
+Mount#mount_v)
@@ -273 +273 @@
-and)
+and#and_conj1)
@@ -275 +275 @@
-use))
+use#use_v))
@@ -278 +278 @@
-Greate)
+Greate#great_adj)
@@ -280 +280 @@
-Guns))
+Guns#gun_n))
@@ -283 +283 @@
-on)
+on#on_prep)
@@ -286 +286 @@
-the)
+the#the_adj)
@@ -288 +288 @@
-said)
+said#said_adj)
@@ -290 +290 @@
-Fortifications))))))))))))))
+Fortifications#fortification_n))))))))))))))
@@ -295 +295 @@
-Wee))
+Wee#we_pron))
@@ -297 +297 @@
-can@)
+can@#can_v1)
@@ -299 +299 @@
-@not)
+@not#not_adv)
@@ -301 +301 @@
-but)
+but#but_prep)
@@ -303 +303 @@
-take)
+take#take_v)
@@ -306 +306 @@
-notice))
+notice#notice_n))
@@ -309 +309 @@
-to)
+to#to_prep)
@@ -312 +312 @@
-you)))
+you#you_pron)))
@@ -316 +316 @@
-how)
+how#how_adv)
@@ -318 +318 @@
-unfitt))
+unfitt#unfit_adj))
@@ -324 +324 @@
-Wee))
+Wee#we_pron))
@@ -326 +326 @@
-thinke)
+thinke#think_v2)
@@ -332 +332 @@
-it))
+it#it_pron))
@@ -335 +335 @@
-That)
+That#that_conj)
@@ -339 +339 @@
-any)
+any#any_adj)
@@ -341 +341 @@
-such)
+such#such_adj)
@@ -343 +343 @@
-Lycences))
+Lycences#licence_n))
@@ -345 +345 @@
-should)
+should#shall_v)
@@ -347 +347 @@
-be)
+be#be_v)
@@ -349 +349 @@
-granted))))))
+granted#grant_v))))))
@@ -360 +360 @@
-And)
+And#and_conj1)
@@ -363 +363 @@
-therefore))
+therefore#therefore_adv))
@@ -366 +366 @@
-Wee))
+Wee#we_pron))
@@ -368 +368 @@
-doe)
+doe#do_v)
@@ -370 +370 @@
-not)
+not#not_adv)
@@ -372 +372 @@
-onely)
+onely#only_adj)
@@ -375 +375 @@
-well))
+well#well_adv))
@@ -377 +377 @@
-approve)
+approve#approve_v1)
@@ -380 +380 @@
-yo=r=)
+yo=r=#your_pron)
@@ -382 +382 @@
-refusall)
+refusall#refusal_n)
@@ -385 +385 @@
-of)
+of#of_prep)
@@ -388 +388 @@
-the)
+the#the_adj)
@@ -390 +390 @@
-desire)
+desire#desire_n)
@@ -401 +401 @@
-he))
+he#he_pron))
@@ -403 +403 @@
-made)
+made#make_v1)
@@ -406 +406 @@
-as)
+as#as_adv)
@@ -409 +409 @@
-to)
+to#to_prep)
@@ -412 +412 @@
-Ballymartin)))))))))
+Ballymartin#NA)))))))))
@@ -420 +420 @@
-But)
+But#but_prep)
@@ -423 +423 @@
-further))
+further#further_adj))
@@ -426 +426 @@
-Wee))
+Wee#we_pron))
@@ -428 +428 @@
-will)
+will#will_v1)
@@ -433 +433 @@
-and)
+and#and_conj1)
@@ -436 +436 @@
-accordingly))
+accordingly#accordingly_adv))
@@ -439 +439 @@
-Wee))
+Wee#we_pron))
@@ -441 +441 @@
-doe)
+doe#do_v)
@@ -444 +444 @@
-hereby)))
+hereby#hereby_adv)))
@@ -447 +447 @@
-sufficiently))
+sufficiently#sufficiently_adv))
@@ -450 +450 @@
-authorise)
+authorise#authorize_v)
@@ -452 +452 @@
-and)
+and#and_conj1)
@@ -454 +454 @@
-require))
+require#require_v))
@@ -457 +457 @@
-you))
+you#you_pron))
@@ -460 +460 @@
-to)
+to#to_prep)
@@ -462 +462 @@
-give)
+give#give_v)
@@ -465 +465 @@
-order))
+order#order_n))
@@ -468 +468 @@
-for)
+for#for_prep)
@@ -471 +471 @@
-the)
+the#the_adj)
@@ -477 +477 @@
-recalling)
+recalling#recall_v)
@@ -479 +479 @@
-and)
+and#and_conj1)
@@ -481 +481 @@
-revokeing))
+revokeing#revoke_v))
@@ -484 +484 @@
-in)
+in#in_prep)
@@ -487 +487 @@
-such)
+such#such_adj)
@@ -489 +489 @@
-way)
+way#way_n1)
@@ -492 +492 @@
-and)
+and#and_conj1)
@@ -495 +495 @@
-forme)))
+forme#form_n)))
@@ -498 +498 @@
-as)
+as#as_adv)
@@ -508 +508 @@
-shall)
+shall#shall_v)
@@ -510 +510 @@
-bee)
+bee#be_v)
@@ -512 +512 @@
-found)
+found#find_v)
@@ -518 +518 @@
-necessary))))))))
+necessary#necessary_adj))))))))
@@ -521 +521 @@
-the)
+the#the_adj)
@@ -523 +523 @@
-said)
+said#said_adj)
@@ -525 +525 @@
-Power)
+Power#power_n1)
@@ -528 +528 @@
-and)
+and#and_conj1)
@@ -531 +531 @@
-Lycence)))
+Lycence#licence_n)))
@@ -535 +535 @@
-soe))
+soe#so_adv))
@@ -537 +537 @@
-pretended)
+pretended#pretend_v)
@@ -542 +542 @@
-to)
+to#to_prep)
@@ -544 +544 @@
-be)
+be#be_v)
@@ -546 +546 @@
-granted)
+granted#grant_v)
@@ -549 +549 @@
-to)
+to#to_prep)
@@ -552 +552 @@
-him)))
+him#him_pron)))
@@ -555 +555 @@
-to)
+to#to_prep)
@@ -557 +557 @@
-fortify)
+fortify#fortify_v)
@@ -560 +560 @@
-his)
+his#his_adj)
@@ -562 +562 @@
-House)
+House#house_n1)
@@ -565 +565 @@
-of)
+of#of_prep)
@@ -568 +568 @@
-Charle)
+Charle#NA)
@@ -570 +570 @@
-Ville)))
+Ville#NA)))
@@ -573 +573 @@
-as)
+as#as_adv)
@@ -576 +576 @@
-aforesaid))))))))
+aforesaid#aforesaid_adj))))))))
@@ -581 +581 @@
-in)
+in#in_prep)
@@ -584 +584 @@
-case)
+case#case_n1)
@@ -591 +591 @@
-upon)
+upon#upon_prep)
@@ -594 +594 @@
-enquiry)))
+enquiry#inquiry_n)))
@@ -597 +597 @@
-you))
+you#you_pron))
@@ -599 +599 @@
-doe)
+doe#do_v)
@@ -601 +601 @@
-finde)
+finde#find_v)
@@ -604 +604 @@
-that)
+that#that_conj)
@@ -608 +608 @@
-any)
+any#any_adj)
@@ -610 +610 @@
-such)
+such#such_adj)
@@ -612 +612 @@
-Lycence)
+Lycence#licence_n)
@@ -615 +615 @@
-or)
+or#or_conj2)
@@ -618 +618 @@
-Authority))))
+Authority#authority_n))))
@@ -620 +620 @@
-has)
+has#have_v)
@@ -622 +622 @@
-been)
+been#be_v)
@@ -624 +624 @@
-granted)
+granted#grant_v)
@@ -627 +627 @@
-to)
+to#to_prep)
@@ -630 +630 @@
-him)))
+him#him_pron)))
@@ -636 +636 @@
-which))
+which#which_adj))
@@ -642 +642 @@
-Wee))
+Wee#we_pron))
@@ -644 +644 @@
-can@)
+can@#can_v1)
@@ -646 +646 @@
-@not)
+@not#not_adv)
@@ -648 +648 @@
-but)
+but#but_prep)
@@ -650 +650 @@
-Looke)
+Looke#look_v)
@@ -653 +653 @@
-upon)
+upon#upon_prep)
@@ -658 +658 @@
-as)
+as#as_adv)
@@ -661 +661 @@
-a)
+a#a_adj2)
@@ -663 +663 @@
-surprise)
+surprise#surprise_n)
@@ -666 +666 @@
-of)
+of#of_prep)
@@ -669 +669 @@
-a)
+a#a_adj2)
@@ -671 +671 @@
-dangerous)
+dangerous#dangerous_adj)
@@ -673 +673 @@
-nature)))
+nature#nature_n)))
@@ -676 +676 @@
-upon)
+upon#upon_prep)
@@ -679 +679 @@
-us)))))
+us#us_pron)))))
@@ -684 +684 @@
-if)
+if#if_conj)
@@ -691 +691 @@
-indeed))
+indeed#indeed_adv))
@@ -694 +694 @@
-Wee))
+Wee#we_pron))
@@ -696 +696 @@
-have)
+have#have_v)
@@ -699 +699 @@
-att)
+att#at_prep)
@@ -702 +702 @@
-any)
+any#any_adj)
@@ -704 +704 @@
-time)
+time#time_n)
@@ -707 +707 @@
-heretofore))))
+heretofore#heretofore_adv))))
@@ -709 +709 @@
-made)
+made#make_v1)
@@ -712 +712 @@
-him))
+him#him_pron))
@@ -715 +715 @@
-such)
+such#such_adj)
@@ -717 +717 @@
-a)
+a#a_adj2)
@@ -719 +719 @@
-Grant))))))))))))))))
+Grant#grant_n1))))))))))))))))
@@ -725 +725 @@
-For)
+For#for_prep)
@@ -728 +728 @@
-which)))
+which#which_adj)))
@@ -734 +734 @@
-these)
+these#these_pron)
@@ -736 +736 @@
-Our)
+Our#our_pron)
@@ -738 +738 @@
-Letters))
+Letters#letter_n1))
@@ -740 +740 @@
-shall)
+shall#shall_v)
@@ -742 +742 @@
-bee)
+bee#be_v)
@@ -745 +745 @@
-your)
+your#your_pron)
@@ -747 +747 @@
-Warrant)
+Warrant#warrant_n1)
@@ -757 +757 @@
-And)
+And#and_conj1)
@@ -760 +760 @@
-soe))
+soe#so_adv))
@@ -763 +763 @@
-Wee))
+Wee#we_pron))
@@ -765 +765 @@
-bid)
+bid#bid_v)
@@ -768 +768 @@
-you))
+you#you_pron))
@@ -771 +771 @@
-heartily))
+heartily#heartily_adv))
@@ -774 +774 @@
-farewell))
+farewell#farewell_int))
@@ -782 +782 @@
-Given)
+Given#give_v)
@@ -785 +785 @@
-att)
+att#at_prep)
@@ -788 +788 @@
-Our)
+Our#our_pron)
@@ -790 +790 @@
-Court)
+Court#court_n1)
@@ -793 +793 @@
-att)
+att#at_prep)
@@ -796 +796 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -799 +799 @@
-the)
+the#the_adj)
@@ -801 +801 @@
-7th)
+7th#NA)
@@ -803 +803 @@
-day)
+day#day_n)
@@ -806 +806 @@
-of)
+of#of_prep)
@@ -809 +809 @@
-September)))
+September#NA)))
@@ -812 +812 @@
-in)
+in#in_prep)
@@ -815 +815 @@
-the)
+the#the_adj)
@@ -817 +817 @@
-24th)
+24th#NA)
@@ -819 +819 @@
-yeare)
+yeare#year_n)
@@ -822 +822 @@
-of)
+of#of_prep)
@@ -825 +825 @@
-Our)
+Our#our_pron)
@@ -827 +827 @@
-Reigne)))))
+Reigne#reign_n)))))
@@ -829 +829 @@
-1672))
+1672#NA))
@@ -843 +843 @@
-Charles)
+Charles#NA)
@@ -846 +846 @@
-R.)))
+R.#rex_n2)))
@@ -855 +855 @@
-Right)
+Right#right_adv)
@@ -857 +857 @@
-Trusty))
+Trusty#trusty_adj))
@@ -860 +860 @@
-and)
+and#and_conj1)
@@ -863 +863 @@
-Right)
+Right#right_adv)
@@ -865 +865 @@
-Welbeloved))))
+Welbeloved#well-beloved_adj))))
@@ -867 +867 @@
-Cousin)
+Cousin#cousin_n)
@@ -870 +870 @@
-&)
+&#and_conj1)
@@ -873 +873 @@
-Counsellor))))
+Counsellor#counsellor_n))))
@@ -878 +878 @@
-Wee))
+Wee#we_pron))
@@ -880 +880 @@
-greet)
+greet#greet_v1)
@@ -883 +883 @@
-you))
+you#you_pron))
@@ -886 +886 @@
-well))
+well#well_adv))
@@ -896 +896 @@
-Whereas)
+Whereas#whereas_adv)
@@ -903 +903 @@
-Wee))
+Wee#we_pron))
@@ -905 +905 @@
-were)
+were#be_v)
@@ -907 +907 @@
-pleased)
+pleased#please_v)
@@ -910 +910 @@
-to)
+to#to_prep)
@@ -912 +912 @@
-referre)
+referre#refer_v)
@@ -915 +915 @@
-the)
+the#the_adj)
@@ -917 +917 @@
-examination)
+examination#examination_n)
@@ -920 +920 @@
-of)
+of#of_prep)
@@ -923 +923 @@
-the)
+the#the_adj)
@@ -925 +925 @@
-late)
+late#late_adj1)
@@ -927 +927 @@
-difference)
+difference#difference_n1)
@@ -930 +930 @@
-arisen)
+arisen#arise_v)
@@ -933 +933 @@
-in)
+in#in_prep)
@@ -936 +936 @@
-the)
+the#the_adj)
@@ -938 +938 @@
-Corporae~on)
+Corporae~on#corporation_n)
@@ -943 +943 @@
-of)
+of#of_prep)
@@ -946 +946 @@
-Our)
+Our#our_pron)
@@ -948 +948 @@
-City)
+City#city_n)
@@ -953 +953 @@
-of)
+of#of_prep)
@@ -956 +956 @@
-Dublin)
+Dublin#NA)
@@ -959 +959 @@
-in)
+in#in_prep)
@@ -962 +962 @@
-that)
+that#that_pron)
@@ -964 +964 @@
-Our)
+Our#our_pron)
@@ -966 +966 @@
-Kingdome)))))))))
+Kingdome#kingdom_n)))))))))
@@ -971 +971 @@
-about)
+about#about_adv)
@@ -974 +974 @@
-the)
+the#the_adj)
@@ -976 +976 @@
-Election)
+Election#election_n)
@@ -979 +979 @@
-of)
+of#of_prep)
@@ -982 +982 @@
-Com~on)
+Com~on#common_adj)
@@ -984 +984 @@
-Councell)
+Councell#council_n)
@@ -986 +986 @@
-men)))
+men#man_n1)))
@@ -989 +989 @@
-out)
+out#out_adv)
@@ -991 +991 @@
-of)
+of#of_prep)
@@ -994 +994 @@
-the)
+the#the_adj)
@@ -996 +996 @@
-severall)
+severall#several_adj)
@@ -998 +998 @@
-Companies)))
+Companies#company_n)))
@@ -1003 +1003 @@
-to)
+to#to_prep)
@@ -1006 +1006 @@
-the)
+the#the_adj)
@@ -1008 +1008 @@
-Com~ittee)
+Com~ittee#committee_n.II)
@@ -1011 +1011 @@
-of)
+of#of_prep)
@@ -1014 +1014 @@
-Our)
+Our#our_pron)
@@ -1016 +1016 @@
-Privy)
+Privy#NA)
@@ -1018 +1018 @@
-Councill)
+Councill#NA)
@@ -1021 +1021 @@
-for)
+for#for_prep)
@@ -1024 +1024 @@
-the)
+the#the_adj)
@@ -1026 +1026 @@
-affaires)
+affaires#affair_n)
@@ -1029 +1029 @@
-of)
+of#of_prep)
@@ -1032 +1032 @@
-Ireland)))))))))))))))))))
+Ireland#NA)))))))))))))))))))
@@ -1037 +1037 @@
-and)
+and#and_conj1)
@@ -1042 +1042 @@
-they))
+they#they_pron))
@@ -1044 +1044 @@
-having)
+having#have_v)
@@ -1046 +1046 @@
-reported)
+reported#report_v)
@@ -1049 +1049 @@
-their)
+their#their_adj)
@@ -1051 +1051 @@
-opinion)
+opinion#opinion_n)
@@ -1054 +1054 @@
-thereupon)))
+thereupon#thereupon_adv)))
@@ -1057 +1057 @@
-to)
+to#to_prep)
@@ -1060 +1060 @@
-us))))
+us#us_pron))))
@@ -1065 +1065 @@
-and)
+and#and_conj1)
@@ -1069 +1069 @@
-Wee))
+Wee#we_pron))
@@ -1072 +1072 @@
-considered)
+considered#consider_v)
@@ -1074 +1074 @@
-&)
+&#and_conj1)
@@ -1076 +1076 @@
-approved))
+approved#approve_v1))
@@ -1079 +1079 @@
-of)
+of#of_prep)
@@ -1082 +1082 @@
-the)
+the#the_adj)
@@ -1084 +1084 @@
-same))))))))
+same#same_adj))))))))
@@ -1089 +1089 @@
-Wee))
+Wee#we_pron))
@@ -1091 +1091 @@
-have)
+have#have_v)
@@ -1093 +1093 @@
-thought)
+thought#think_v2)
@@ -1099 +1099 @@
-fit))
+fit#fit_adj))
@@ -1102 +1102 @@
-in)
+in#in_prep)
@@ -1105 +1105 @@
-pursuance)
+pursuance#pursuance_n)
@@ -1108 +1108 @@
-thereof))))
+thereof#thereof_adv))))
@@ -1111 +1111 @@
-to)
+to#to_prep)
@@ -1113 +1113 @@
-signify)
+signify#signify_v)
@@ -1116 +1116 @@
-to)
+to#to_prep)
@@ -1119 +1119 @@
-you)))
+you#you_pron)))
@@ -1122 +1122 @@
-Our)
+Our#our_pron)
@@ -1124 +1124 @@
-Pleasure)
+Pleasure#pleasure_n)
@@ -1127 +1127 @@
-that)
+that#that_conj)
@@ -1132 +1132 @@
-the)
+the#the_adj)
@@ -1134 +1134 @@
-last)
+last#last_adv)
@@ -1136 +1136 @@
-choice)
+choice#choice_n)
@@ -1139 +1139 @@
-made)
+made#make_v1)
@@ -1142 +1142 @@
-by)
+by#by_prep)
@@ -1145 +1145 @@
-the)
+the#the_adj)
@@ -1147 +1147 @@
-Lord)
+Lord#lord_n)
@@ -1149 +1149 @@
-Mayor)
+Mayor#mayor_n)
@@ -1152 +1152 @@
-of)
+of#of_prep)
@@ -1155 +1155 @@
-that)
+that#that_pron)
@@ -1157 +1157 @@
-Our)
+Our#our_pron)
@@ -1159 +1159 @@
-Citty)))))
+Citty#city_n)))))
@@ -1162 +1162 @@
-of)
+of#of_prep)
@@ -1165 +1165 @@
-the)
+the#the_adj)
@@ -1167 +1167 @@
-compleate)
+compleate#complete_adj)
@@ -1169 +1169 @@
-number)
+number#number_n)
@@ -1172 +1172 @@
-of)
+of#of_prep)
@@ -1175 +1175 @@
-Com~on)
+Com~on#common_adj)
@@ -1177 +1177 @@
-Counsell)
+Counsell#council_n)
@@ -1179 +1179 @@
-men)))))
+men#man_n1)))))
@@ -1182 +1182 @@
-all)
+all#all_adj)
@@ -1184 +1184 @@
-at)
+at#at_prep)
@@ -1187 +1187 @@
-once))))
+once#once_adv))))
@@ -1192 +1192 @@
-leaving)
+leaving#leave_v1)
@@ -1194 +1194 @@
-out)
+out#out_adv)
@@ -1197 +1197 @@
-the)
+the#the_adj)
@@ -1200 +1200 @@
-ten)
+ten#ten_adj)
@@ -1202 +1202 @@
-or)
+or#or_conj2)
@@ -1204 +1204 @@
-eleven))
+eleven#eleven_adj))
@@ -1206 +1206 @@
-Roman)
+Roman#NA)
@@ -1208 +1208 @@
-Catholiques)
+Catholiques#NA)
@@ -1213 +1213 @@
-that)
+that#that_conj)
@@ -1218 +1218 @@
-were)
+were#be_v)
@@ -1220 +1220 @@
-chosen)
+chosen#choose_v)
@@ -1223 +1223 @@
-at)
+at#at_prep)
@@ -1226 +1226 @@
-the)
+the#the_adj)
@@ -1228 +1228 @@
-first)
+first#first_adj)
@@ -1230 +1230 @@
-choice))))))))
+choice#choice_n))))))))
@@ -1234 +1234 @@
-shall)
+shall#shall_v)
@@ -1236 +1236 @@
-stand))
+stand#stand_v))
@@ -1239 +1239 @@
-and)
+and#and_conj1)
@@ -1242 +1242 @@
-be)
+be#be_v)
@@ -1244 +1244 @@
-confirmed)))))))))
+confirmed#confirm_v)))))))))
@@ -1250 +1250 @@
-and)
+and#and_conj1)
@@ -1253 +1253 @@
-Wee))
+Wee#we_pron))
@@ -1255 +1255 @@
-doe)
+doe#do_v)
@@ -1258 +1258 @@
-hereby))
+hereby#hereby_adv))
@@ -1261 +1261 @@
-require)
+require#require_v)
@@ -1263 +1263 @@
-and)
+and#and_conj1)
@@ -1265 +1265 @@
-direct))
+direct#direct_v))
@@ -1268 +1268 @@
-you))
+you#you_pron))
@@ -1271 +1271 @@
-to)
+to#to_prep)
@@ -1273 +1273 @@
-doe)
+doe#do_v)
@@ -1276 +1276 @@
-all)
+all#all_adj)
@@ -1278 +1278 @@
-things))
+things#thing_n1))
@@ -1283 +1283 @@
-and)
+and#and_conj1)
@@ -1285 +1285 @@
-give)
+give#give_v)
@@ -1288 +1288 @@
-all)
+all#all_adj)
@@ -1290 +1290 @@
-necessary)
+necessary#necessary_adj)
@@ -1292 +1292 @@
-Orders)))
+Orders#order_n)))
@@ -1295 +1295 @@
-for)
+for#for_prep)
@@ -1298 +1298 @@
-the)
+the#the_adj)
@@ -1300 +1300 @@
-speedy)
+speedy#speedy_adj)
@@ -1302 +1302 @@
-settling)
+settling#settling_n)
@@ -1305 +1305 @@
-of)
+of#of_prep)
@@ -1308 +1308 @@
-this)
+this#this_pron)
@@ -1310 +1310 @@
-affaire)))
+affaire#affair_n)))
@@ -1313 +1313 @@
-accordingly)))))
+accordingly#accordingly_adv)))))
@@ -1321 +1321 @@
-And)
+And#and_conj1)
@@ -1324 +1324 @@
-for)
+for#for_prep)
@@ -1328 +1328 @@
-soe))
+soe#so_adv))
@@ -1330 +1330 @@
-doing)))
+doing#do_v)))
@@ -1333 +1333 @@
-this))
+this#this_pron))
@@ -1335 +1335 @@
-shall)
+shall#shall_v)
@@ -1337 +1337 @@
-be)
+be#be_v)
@@ -1340 +1340 @@
-your)
+your#your_pron)
@@ -1342 +1342 @@
-Warrant)
+Warrant#warrant_n1)
@@ -1352 +1352 @@
-Given)
+Given#give_v)
@@ -1355 +1355 @@
-at)
+at#at_prep)
@@ -1358 +1358 @@
-Our)
+Our#our_pron)
@@ -1360 +1360 @@
-Court)
+Court#court_n1)
@@ -1363 +1363 @@
-at)
+at#at_prep)
@@ -1366 +1366 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -1371 +1371 @@
-the)
+the#the_adj)
@@ -1375 +1375 @@
-day)
+day#day_n)
@@ -1378 +1378 @@
-of)
+of#of_prep)
@@ -1381 +1381 @@
-August)))
+August#NA)))
@@ -1385 +1385 @@
-1673)
+1673#NA)
@@ -1390 +1390 @@
-in)
+in#in_prep)
@@ -1393 +1393 @@
-the)
+the#the_adj)
@@ -1395 +1395 @@
-Five_and_Twentieth)
+Five_and_Twentieth#NA)
@@ -1397 +1397 @@
-yeare)
+yeare#year_n)
@@ -1400 +1400 @@
-of)
+of#of_prep)
@@ -1403 +1403 @@
-Our)
+Our#our_pron)
@@ -1405 +1405 @@
-Reigne))))))
+Reigne#reign_n))))))
@@ -1419 +1419 @@
-Charles)
+Charles#NA)
@@ -1422 +1422 @@
-R.)))
+R.#rex_n2)))
@@ -1431 +1431 @@
-Right)
+Right#right_adv)
@@ -1433 +1433 @@
-Trusty))
+Trusty#trusty_adj))
@@ -1436 +1436 @@
-and)
+and#and_conj1)
@@ -1439 +1439 @@
-Right)
+Right#right_adv)
@@ -1441 +1441 @@
-Welbeloved))))
+Welbeloved#well-beloved_adj))))
@@ -1443 +1443 @@
-Cousin)
+Cousin#cousin_n)
@@ -1446 +1446 @@
-and)
+and#and_conj1)
@@ -1449 +1449 @@
-Councellour))))
+Councellour#counsellor_n))))
@@ -1454 +1454 @@
-Wee))
+Wee#we_pron))
@@ -1456 +1456 @@
-greet)
+greet#greet_v1)
@@ -1459 +1459 @@
-you))
+you#you_pron))
@@ -1462 +1462 @@
-well))
+well#well_adv))
@@ -1472 +1472 @@
-Whereas)
+Whereas#whereas_adv)
@@ -1479 +1479 @@
-Wee))
+Wee#we_pron))
@@ -1481 +1481 @@
-have)
+have#have_v)
@@ -1483 +1483 @@
-been)
+been#be_v)
@@ -1485 +1485 @@
-informed)
+informed#inform_v)
@@ -1488 +1488 @@
-of)
+of#of_prep)
@@ -1491 +1491 @@
-certaine)
+certaine#certain_adj)
@@ -1493 +1493 @@
-disorders)
+disorders#disorder_n)
@@ -1496 +1496 @@
-fomenting)
+fomenting#foment_v)
@@ -1499 +1499 @@
-in)
+in#in_prep)
@@ -1502 +1502 @@
-Our)
+Our#our_pron)
@@ -1504 +1504 @@
-Citty)
+Citty#city_n)
@@ -1507 +1507 @@
-of)
+of#of_prep)
@@ -1510 +1510 @@
-Dublin)))))
+Dublin#NA)))))
@@ -1513 +1513 @@
-to)
+to#to_prep)
@@ -1515 +1515 @@
-create)
+create#create_v)
@@ -1518 +1518 @@
-disturbances))
+disturbances#disturbance_n))
@@ -1521 +1521 @@
-upon)
+upon#upon_prep)
@@ -1524 +1524 @@
-y=e=)
+y=e=#the_adj)
@@ -1526 +1526 @@
-account)
+account#account_n)
@@ -1529 +1529 @@
-of)
+of#of_prep)
@@ -1532 +1532 @@
-y=e=)
+y=e=#the_adj)
@@ -1534 +1534 @@
-late)
+late#late_adj1)
@@ -1536 +1536 @@
-Rules))
+Rules#rule_n1))
@@ -1539 +1539 @@
-for)
+for#for_prep)
@@ -1542 +1542 @@
-regulating)
+regulating#regulate_v)
@@ -1546 +1546 @@
-that))
+that#that_pron))
@@ -1549 +1549 @@
-and)
+and#and_conj1)
@@ -1552 +1552 @@
-the)
+the#the_adj)
@@ -1554 +1554 @@
-other)
+other#other_adj)
@@ -1558 +1558 @@
-Corporations)
+Corporations#corporation_n)
@@ -1561 +1561 @@
-of)
+of#of_prep)
@@ -1564 +1564 @@
-Our)
+Our#our_pron)
@@ -1566 +1566 @@
-Kingdome)
+Kingdome#kingdom_n)
@@ -1569 +1569 @@
-of)
+of#of_prep)
@@ -1572 +1572 @@
-Ireland))))))))))))))))))))
+Ireland#NA))))))))))))))))))))
@@ -1577 +1577 @@
-And)
+And#and_conj1)
@@ -1580 +1580 @@
-Whereas)
+Whereas#whereas_adv)
@@ -1587 +1587 @@
-the)
+the#the_adj)
@@ -1589 +1589 @@
-procuring)
+procuring#procuring_n)
@@ -1592 +1592 @@
-of)
+of#of_prep)
@@ -1595 +1595 @@
-hands)))
+hands#hand_n)))
@@ -1598 +1598 @@
-to)
+to#to_prep)
@@ -1601 +1601 @@
-a)
+a#a_adj2)
@@ -1603 +1603 @@
-Petition)
+Petition#petition_n)
@@ -1606 +1606 @@
-against)
+against#against_prep)
@@ -1609 +1609 @@
-a)
+a#a_adj2)
@@ -1611 +1611 @@
-Law))))))
+Law#law_n1))))))
@@ -1613 +1613 @@
-is)
+is#be_v)
@@ -1616 +1616 @@
-in)
+in#in_prep)
@@ -1619 +1619 @@
-it)
+it#it_pron)
@@ -1621 +1621 @@
-self)))
+self#self_pron)))
@@ -1624 +1624 @@
-an)
+an#a_adj2)
@@ -1626 +1626 @@
-Act)
+Act#act_n)
@@ -1629 +1629 @@
-of)
+of#of_prep)
@@ -1632 +1632 @@
-Sedition))))
+Sedition#sedition_n))))
@@ -1637 +1637 @@
-especially)
+especially#especially_adv)
@@ -1639 +1639 @@
-when)
+when#when_adv)
@@ -1642 +1642 @@
-prepared)
+prepared#prepare_v)
@@ -1645 +1645 @@
-to)
+to#to_prep)
@@ -1647 +1647 @@
-be)
+be#be_v)
@@ -1649 +1649 @@
-addressed)
+addressed#address_v)
@@ -1652 +1652 @@
-to)
+to#to_prep)
@@ -1655 +1655 @@
-such)
+such#such_adj)
@@ -1657 +1657 @@
-a)
+a#a_adj2)
@@ -1659 +1659 @@
-Power)
+Power#power_n1)
@@ -1662 +1662 @@
-as)
+as#as_adv)
@@ -1672 +1672 @@
-can@)
+can@#can_v1)
@@ -1674 +1674 @@
-@not)
+@not#not_adv)
@@ -1676 +1676 @@
-change)
+change#change_v)
@@ -1679 +1679 @@
-the)
+the#the_adj)
@@ -1681 +1681 @@
-Law)
+Law#law_n1)
@@ -1686 +1686 @@
-though)
+though#though_adv)
@@ -1689 +1689 @@
-never)
+never#never_adv)
@@ -1691 +1691 @@
-soe)
+soe#so_adv)
@@ -1693 +1693 @@
-greivous)))))))))))))))))
+greivous#grievous_adj)))))))))))))))))
@@ -1699 +1699 @@
-Wee))
+Wee#we_pron))
@@ -1701 +1701 @@
-having)
+having#have_v)
@@ -1703 +1703 @@
-taken)
+taken#take_v)
@@ -1707 +1707 @@
-the)
+the#the_adj)
@@ -1709 +1709 @@
-same))
+same#same_adj))
@@ -1714 +1714 @@
-into)
+into#into_prep)
@@ -1717 +1717 @@
-Our)
+Our#our_pron)
@@ -1719 +1719 @@
-consideration)))
+consideration#consideration_n)))
@@ -1724 +1724 @@
-and)
+and#and_conj1)
@@ -1728 +1728 @@
-that)
+that#that_conj)
@@ -1732 +1732 @@
-the)
+the#the_adj)
@@ -1734 +1734 @@
-said)
+said#said_adj)
@@ -1736 +1736 @@
-Rules))
+Rules#rule_n1))
@@ -1742 +1742 @@
-as)
+as#as_adv)
@@ -1744 +1744 @@
-farre))
+farre#far_adj))
@@ -1747 +1747 @@
-as)
+as#as_adv)
@@ -1758 +1758 @@
-wee))
+wee#we_pron))
@@ -1760 +1760 @@
-can)
+can#can_v1)
@@ -1762 +1762 @@
-take)
+take#take_v)
@@ -1765 +1765 @@
-notice))
+notice#notice_n))
@@ -1768 +1768 @@
-of)
+of#of_prep)
@@ -1771 +1771 @@
-them)))))))
+them#them_pron)))))))
@@ -1775 +1775 @@
-are)
+are#be_v)
@@ -1778 +1778 @@
-extreamely)
+extreamely#extremely_adv)
@@ -1780 +1780 @@
-usefull)
+usefull#useful_adj)
@@ -1783 +1783 @@
-to)
+to#to_prep)
@@ -1786 +1786 @@
-Government)))
+Government#government_n)))
@@ -1789 +1789 @@
-in)
+in#in_prep)
@@ -1792 +1792 @@
-many)
+many#many_adj)
@@ -1794 +1794 @@
-respects))))))
+respects#respect_n))))))
@@ -1799 +1799 @@
-and)
+and#and_conj1)
@@ -1802 +1802 @@
-that)
+that#that_conj)
@@ -1806 +1806 @@
-if)
+if#if_conj)
@@ -1813 +1813 @@
-Wee))
+Wee#we_pron))
@@ -1815 +1815 @@
-should)
+should#shall_v)
@@ -1817 +1817 @@
-think)
+think#think_v2)
@@ -1823 +1823 @@
-fit))
+fit#fit_adj))
@@ -1826 +1826 @@
-to)
+to#to_prep)
@@ -1828 +1828 @@
-give)
+give#give_v)
@@ -1831 +1831 @@
-way))
+way#way_n1))
@@ -1834 +1834 @@
-against)
+against#against_prep)
@@ -1837 +1837 @@
-any)
+any#any_adj)
@@ -1839 +1839 @@
-complaints)))
+complaints#complaint_n)))
@@ -1844 +1844 @@
-though)
+though#though_adv)
@@ -1851,3 +1851,3 @@
-they))
-(MD
-may)
+they#they_pron))
+(MD
+may#may_v1)
@@ -1855 +1855 @@
-seem)
+seem#seem_v2)
@@ -1858 +1858 @@
-very)
+very#very_adj)
@@ -1860 +1860 @@
-specious))))))))))
+specious#specious_adj))))))))))
@@ -1866 +1866 @@
-might)
+might#may_v1)
@@ -1869 +1869 @@
-yet))
+yet#yet_adv))
@@ -1872 +1872 @@
-in)
+in#in_prep)
@@ -1875 +1875 @@
-the)
+the#the_adj)
@@ -1877 +1877 @@
-consequence)))
+consequence#consequence_n)))
@@ -1879 +1879 @@
-disturbe)
+disturbe#disturb_v)
@@ -1882 +1882 @@
-Our)
+Our#our_pron)
@@ -1884 +1884 @@
-whole)
+whole#whole_adj)
@@ -1886 +1886 @@
-Government)
+Government#government_n)
@@ -1889 +1889 @@
-there))))))
+there#there_adv))))))
@@ -1894 +1894 @@
-And)
+And#and_conj1)
@@ -1897 +1897 @@
-that)
+that#that_conj)
@@ -1901 +1901 @@
-the)
+the#the_adj)
@@ -1903 +1903 @@
-said)
+said#said_adj)
@@ -1905 +1905 @@
-Rules))
+Rules#rule_n1))
@@ -1907 +1907 @@
-are)
+are#be_v)
@@ -1910 +1910 @@
-of)
+of#of_prep)
@@ -1913 +1913 @@
-equall)
+equall#equal_adj)
@@ -1915 +1915 @@
-credit)
+credit#credit_n)
@@ -1918 +1918 @@
-and)
+and#and_conj1)
@@ -1921 +1921 @@
-Authority)))
+Authority#authority_n)))
@@ -1924 +1924 @@
-with)
+with#with_prep)
@@ -1927 +1927 @@
-the)
+the#the_adj)
@@ -1929 +1929 @@
-Act)
+Act#NA)
@@ -1932 +1932 @@
-of)
+of#of_prep)
@@ -1935 +1935 @@
-Settlement)
+Settlement#NA)
@@ -1941 +1941 @@
-the)
+the#the_adj)
@@ -1943 +1943 @@
-attempt)
+attempt#attempt_n)
@@ -1946 +1946 @@
-of)
+of#of_prep)
@@ -1949 +1949 @@
-breaking)
+breaking#break_v)
@@ -1952 +1952 @@
-any)
+any#any_adj)
@@ -1954 +1954 @@
-part)
+part#part_n)
@@ -1957 +1957 @@
-whereof))))))
+whereof#whereof_adv))))))
@@ -1963 +1963 @@
-Wee))
+Wee#we_pron))
@@ -1965 +1965 @@
-look)
+look#look_v)
@@ -1968 +1968 @@
-upon)
+upon#upon_prep)
@@ -1973 +1973 @@
-as)
+as#as_adv)
@@ -1976 +1976 @@
-very)
+very#very_adj)
@@ -1978 +1978 @@
-dangerous)
+dangerous#dangerous_adj)
@@ -1981 +1981 @@
-and)
+and#and_conj1)
@@ -1984 +1984 @@
-prejudiciall)))
+prejudiciall#prejudicial_adj1)))
@@ -1987 +1987 @@
-to)
+to#to_prep)
@@ -1990 +1990 @@
-Our)
+Our#our_pron)
@@ -1992 +1992 @@
-Service)))))))))))))))))))
+Service#service_n1)))))))))))))))))))
@@ -1997 +1997 @@
-Wee))
+Wee#we_pron))
@@ -1999 +1999 @@
-have)
+have#have_v)
@@ -2004 +2004 @@
-therefore))
+therefore#therefore_adv))
@@ -2008 +2008 @@
-thought)
+thought#think_v2)
@@ -2014 +2014 @@
-fit))
+fit#fit_adj))
@@ -2017 +2017 @@
-to)
+to#to_prep)
@@ -2020 +2020 @@
-direct)
+direct#direct_v)
@@ -2022 +2022 @@
-and)
+and#and_conj1)
@@ -2024 +2024 @@
-require))
+require#require_v))
@@ -2027 +2027 @@
-you))
+you#you_pron))
@@ -2032 +2032 @@
-as)
+as#as_adv)
@@ -2043 +2043 @@
-Wee))
+Wee#we_pron))
@@ -2045 +2045 @@
-doe)
+doe#do_v)
@@ -2050 +2050 @@
-by)
+by#by_prep)
@@ -2053 +2053 @@
-these)
+these#these_pron)
@@ -2055 +2055 @@
-Presents))))))
+Presents#present_n1))))))
@@ -2061 +2061 @@
-to)
+to#to_prep)
@@ -2063 +2063 @@
-use)
+use#use_v)
@@ -2066 +2066 @@
-such)
+such#such_adj)
@@ -2068 +2068 @@
-means)
+means#mean_n2)
@@ -2071 +2071 @@
-as)
+as#as_adv)
@@ -2080 +2080 @@
-you))
+you#you_pron))
@@ -2082 +2082 @@
-shall)
+shall#shall_v)
@@ -2084 +2084 @@
-think)
+think#think_v2)
@@ -2090 +2090 @@
-fit)
+fit#fit_adj)
@@ -2093 +2093 @@
-for)
+for#for_prep)
@@ -2096 +2096 @@
-the)
+the#the_adj)
@@ -2098 +2098 @@
-effectuall)
+effectuall#effectual_adj)
@@ -2103 +2103 @@
-suppressing)
+suppressing#suppress_v)
@@ -2106 +2106 @@
-all)
+all#all_adj)
@@ -2108 +2108 @@
-Preparations)
+Preparations#preparation_n)
@@ -2111 +2111 @@
-to)
+to#to_prep)
@@ -2114 +2114 @@
-such)
+such#such_adj)
@@ -2116 +2116 @@
-a)
+a#a_adj2)
@@ -2118 +2118 @@
-Disorder)))))))))))))
+Disorder#disorder_n)))))))))))))
@@ -2121 +2121 @@
-in)
+in#in_prep)
@@ -2124 +2124 @@
-the)
+the#the_adj)
@@ -2126 +2126 @@
-same)
+same#same_adj)
@@ -2128 +2128 @@
-manner)
+manner#manner_n)
@@ -2131 +2131 @@
-as)
+as#as_adv)
@@ -2142 +2142 @@
-you))
+you#you_pron))
@@ -2144 +2144 @@
-would)
+would#will_v1)
@@ -2146 +2146 @@
-doe)
+doe#do_v)
@@ -2149 +2149 @@
-any)
+any#any_adj)
@@ -2151 +2151 @@
-other)
+other#other_adj)
@@ -2153 +2153 @@
-Sedition))))))))
+Sedition#sedition_n))))))))
@@ -2158 +2158 @@
-and)
+and#and_conj1)
@@ -2161 +2161 @@
-that)
+that#that_conj)
@@ -2165 +2165 @@
-you))
+you#you_pron))
@@ -2167 +2167 @@
-give)
+give#give_v)
@@ -2170 +2170 @@
-Order))
+Order#order_n))
@@ -2173 +2173 @@
-to)
+to#to_prep)
@@ -2176 +2176 @@
-Our)
+Our#our_pron)
@@ -2178 +2178 @@
-Atturney)
+Atturney#attorney_n)
@@ -2181 +2181 @@
-or)
+or#or_conj2)
@@ -2184 +2184 @@
-Sollicitor)))
+Sollicitor#solicitor_n)))
@@ -2187 +2187 @@
-Generall))
+Generall#general_adj))
@@ -2190 +2190 @@
-there))))
+there#there_adv))))
@@ -2193 +2193 @@
-to)
+to#to_prep)
@@ -2195 +2195 @@
-proceed)
+proceed#proceed_v)
@@ -2198 +2198 @@
-by)
+by#by_prep)
@@ -2201 +2201 @@
-information)))
+information#information_n)))
@@ -2204 +2204 @@
-against)
+against#against_prep)
@@ -2207 +2207 @@
-the)
+the#the_adj)
@@ -2209 +2209 @@
-Principall)
+Principall#principal_adj)
@@ -2211 +2211 @@
-movers)
+movers#mover_n1)
@@ -2214 +2214 @@
-in)
+in#in_prep)
@@ -2217 +2217 @@
-this)
+this#this_pron)
@@ -2219 +2219 @@
-businesse))))))
+businesse#business_n))))))
@@ -2224 +2224 @@
-if)
+if#if_conj)
@@ -2231 +2231 @@
-you))
+you#you_pron))
@@ -2233 +2233 @@
-should)
+should#shall_v)
@@ -2235 +2235 @@
-think)
+think#think_v2)
@@ -2238 +2238 @@
-it))
+it#it_pron))
@@ -2241 +2241 @@
-for)
+for#for_prep)
@@ -2244 +2244 @@
-Our)
+Our#our_pron)
@@ -2246 +2246 @@
-Service))))))))))))
+Service#service_n1))))))))))))
@@ -2255 +2255 @@
-Neverthelesse))
+Neverthelesse#nevertheless_adv))
@@ -2260 +2260 @@
-Our)
+Our#our_pron)
@@ -2262 +2262 @@
-pleasure))
+pleasure#pleasure_n))
@@ -2264 +2264 @@
-is)
+is#be_v)
@@ -2269 +2269 @@
-that)
+that#that_conj)
@@ -2273 +2273 @@
-you))
+you#you_pron))
@@ -2276 +2276 @@
-enquire)
+enquire#inquire_v)
@@ -2278 +2278 @@
-and)
+and#and_conj1)
@@ -2280 +2280 @@
-certify))
+certify#certify_v))
@@ -2283 +2283 @@
-unto)
+unto#unto_prep)
@@ -2286 +2286 @@
-Us)))
+Us#us_pron)))
@@ -2293 +2293 @@
-with)
+with#with_prep)
@@ -2296 +2296 @@
-what)
+what#what_pron)
@@ -2298 +2298 @@
-convenient)
+convenient#convenient_adj)
@@ -2300 +2300 @@
-speed)))
+speed#speed_n)))
@@ -2308 +2308 @@
-you))
+you#you_pron))
@@ -2310 +2310 @@
-may)
+may#may_v1)
@@ -2318 +2318 @@
-what)
+what#what_pron)
@@ -2320 +2320 @@
-Numbers)
+Numbers#number_n)
@@ -2330 +2330 @@
-there))
+there#there_adv))
@@ -2332 +2332 @@
-are)
+are#be_v)
@@ -2335 +2335 @@
-of)
+of#of_prep)
@@ -2338 +2338 @@
-persons)
+persons#person_n)
@@ -2341 +2341 @@
-living)
+living#live_v1)
@@ -2344 +2344 @@
-in)
+in#in_prep)
@@ -2347 +2347 @@
-any)
+any#any_adj)
@@ -2350 +2350 @@
-of)
+of#of_prep)
@@ -2353 +2353 @@
-the)
+the#the_adj)
@@ -2355 +2355 @@
-Corporations)
+Corporations#corporation_n)
@@ -2358 +2358 @@
-of)
+of#of_prep)
@@ -2361 +2361 @@
-that)
+that#that_pron)
@@ -2363 +2363 @@
-Our)
+Our#our_pron)
@@ -2365 +2365 @@
-Kingdome)))))))
+Kingdome#kingdom_n)))))))
@@ -2368 +2368 @@
-within)
+within#within_adv)
@@ -2371 +2371 @@
-the)
+the#the_adj)
@@ -2373 +2373 @@
-benefit)
+benefit#benefit_n)
@@ -2376 +2376 @@
-of)
+of#of_prep)
@@ -2379 +2379 @@
-the)
+the#the_adj)
@@ -2381 +2381 @@
-words)
+words#word_n)
@@ -2386 +2386 @@
-and)
+and#and_conj1)
@@ -2388 +2388 @@
-others))
+others#other_adj))
@@ -2393 +2393 @@
-in)
+in#in_prep)
@@ -2396 +2396 @@
-the)
+the#the_adj)
@@ -2398 +2398 @@
-second)
+second#second_adj)
@@ -2400 +2400 @@
-Rule))))))))
+Rule#rule_n1))))))))
@@ -2407 +2407 @@
-that)
+that#that_conj)
@@ -2412 +2412 @@
-may)
+may#may_v1)
@@ -2414 +2414 @@
-give)
+give#give_v)
@@ -2417 +2417 @@
-occasion))
+occasion#occasion_n1))
@@ -2420 +2420 @@
-of)
+of#of_prep)
@@ -2423 +2423 @@
-offence)))))))))
+offence#offence_n)))))))))
@@ -2428 +2428 @@
-to)
+to#to_prep)
@@ -2431 +2431 @@
-the)
+the#the_adj)
@@ -2433 +2433 @@
-End)
+End#end_n)
@@ -2436 +2436 @@
-that)
+that#that_conj)
@@ -2440 +2440 @@
-Wee))
+Wee#we_pron))
@@ -2442 +2442 @@
-may)
+may#may_v1)
@@ -2444 +2444 @@
-be)
+be#be_v)
@@ -2446 +2446 @@
-informed)
+informed#inform_v)
@@ -2449 +2449 @@
-whether)
+whether#whether_pron)
@@ -2455 +2455 @@
-there))
+there#there_adv))
@@ -2457 +2457 @@
-may)
+may#may_v1)
@@ -2459 +2459 @@
-be)
+be#be_v)
@@ -2462 +2462 @@
-any)
+any#any_adj)
@@ -2464 +2464 @@
-seeming)
+seeming#seeming_adj)
@@ -2466 +2466 @@
-grounds)
+grounds#ground_n)
@@ -2469 +2469 @@
-for)
+for#for_prep)
@@ -2472 +2472 @@
-these)
+these#these_pron)
@@ -2474 +2474 @@
-complaints))))))))))))
+complaints#complaint_n))))))))))))
@@ -2482 +2482 @@
-And)
+And#and_conj1)
@@ -2485 +2485 @@
-for)
+for#for_prep)
@@ -2489 +2489 @@
-soe))
+soe#so_adv))
@@ -2491 +2491 @@
-doing)))
+doing#do_v)))
@@ -2494 +2494 @@
-this))
+this#this_pron))
@@ -2496 +2496 @@
-shall)
+shall#shall_v)
@@ -2498 +2498 @@
-be)
+be#be_v)
@@ -2501 +2501 @@
-your)
+your#your_pron)
@@ -2503 +2503 @@
-Warrant)
+Warrant#warrant_n1)
@@ -2513 +2513 @@
-And)
+And#and_conj1)
@@ -2516 +2516 @@
-soe))
+soe#so_adv))
@@ -2519 +2519 @@
-Wee))
+Wee#we_pron))
@@ -2521 +2521 @@
-bid)
+bid#bid_v)
@@ -2524 +2524 @@
-you))
+you#you_pron))
@@ -2527 +2527 @@
-heartily))
+heartily#heartily_adv))
@@ -2530 +2530 @@
-farewell))
+farewell#farewell_int))
@@ -2538 +2538 @@
-Given)
+Given#give_v)
@@ -2541 +2541 @@
-at)
+at#at_prep)
@@ -2544 +2544 @@
-Our)
+Our#our_pron)
@@ -2546 +2546 @@
-Court)
+Court#court_n1)
@@ -2549 +2549 @@
-at)
+at#at_prep)
@@ -2552 +2552 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -2555 +2555 @@
-the)
+the#the_adj)
@@ -2557 +2557 @@
-31=st=)
+31=st=#NA)
@@ -2559 +2559 @@
-day)
+day#day_n)
@@ -2562 +2562 @@
-of)
+of#of_prep)
@@ -2565 +2565 @@
-March)))
+March#NA)))
@@ -2569 +2569 @@
-1674)
+1674#NA)
@@ -2574 +2574 @@
-in)
+in#in_prep)
@@ -2577 +2577 @@
-the)
+the#the_adj)
@@ -2579 +2579 @@
-Six_&_Twentieth)
+Six_&_Twentieth#NA)
@@ -2581 +2581 @@
-yeare)
+yeare#year_n)
@@ -2584 +2584 @@
-of)
+of#of_prep)
@@ -2587 +2587 @@
-Our)
+Our#our_pron)
@@ -2589 +2589 @@
-Reigne))))))
+Reigne#reign_n))))))
