--- conway-1623-e2-h.psd.vert	2025-09-10 13:50:35
+++ conway-1623-e2-h.lem.vert	2025-09-10 13:50:45
@@ -15 +15 @@
-Most)
+Most#most_adj)
@@ -17 +17 @@
-gratious))
+gratious#gracious_adj))
@@ -19 +19 @@
-patron))
+patron#patron_n))
@@ -24 +24 @@
-I))
+I#i_pron))
@@ -26 +26 @@
-am)
+am#be_v)
@@ -28 +28 @@
-commaunded)
+commaunded#command_v)
@@ -31 +31 @@
-by)
+by#by_prep)
@@ -34 +34 @@
-his)
+his#his_adj)
@@ -36 +36 @@
-Majestie)))
+Majestie#majesty_n)))
@@ -39 +39 @@
-for)
+for#for_prep)
@@ -42 +42 @@
-the)
+the#the_adj)
@@ -44 +44 @@
-shortening)
+shortening#shortening_n)
@@ -47 +47 @@
-of)
+of#of_prep)
@@ -50 +50 @@
-his)
+his#his_adj)
@@ -52 +52 @@
-labour)))))
+labour#labour_n)))))
@@ -57 +57 @@
-to)
+to#to_prep)
@@ -59 +59 @@
-signifie)
+signifie#signify_v)
@@ -62 +62 @@
-unto)
+unto#unto_prep)
@@ -65 +65 @@
-your)
+your#your_pron)
@@ -67 +67 @@
-Grace)))
+Grace#grace_n)))
@@ -70 +70 @@
-that)
+that#that_conj)
@@ -74 +74 @@
-his)
+his#his_adj)
@@ -76 +76 @@
-Highnes))
+Highnes#highness_n))
@@ -78 +78 @@
-may)
+may#may_v1)
@@ -81 +81 @@
-soe))
+soe#so_adv))
@@ -83 +83 @@
-be)
+be#be_v)
@@ -85 +85 @@
-informed)
+informed#inform_v)
@@ -88 +88 @@
-of)
+of#of_prep)
@@ -91 +91 @@
-the)
+the#the_adj)
@@ -93 +93 @@
-great)
+great#great_adj)
@@ -95 +95 @@
-streyght)
+streyght#strait_adj)
@@ -98 +98 @@
-and)
+and#and_conj1)
@@ -101 +101 @@
-distress)))
+distress#distress_n)))
@@ -110 +110 @@
-his)
+his#his_adj)
@@ -112 +112 @@
-Majestie))
+Majestie#majesty_n))
@@ -114 +114 @@
-was)
+was#be_v)
@@ -117 +117 @@
-in)
+in#in_prep)
@@ -122 +122 @@
-upon)
+upon#upon_prep)
@@ -125 +125 @@
-the)
+the#the_adj)
@@ -127 +127 @@
-receipt)
+receipt#receipt_n)
@@ -130 +130 @@
-and)
+and#and_conj1)
@@ -133 +133 @@
-consideracon)))
+consideracon#consideration_n)))
@@ -136 +136 @@
-of)
+of#of_prep)
@@ -139 +139 @@
-the)
+the#the_adj)
@@ -141 +141 @@
-Articles)
+Articles#article_n)
@@ -144 +144 @@
-brought)
+brought#bring_v)
@@ -147 +147 @@
-by)
+by#by_prep)
@@ -150 +150 @@
-M=r=.)
+M=r=.#NA)
@@ -152 +152 @@
-Killigrey))))))))
+Killigrey#NA))))))))
@@ -157 +157 @@
-upon)
+upon#upon_prep)
@@ -160 +160 @@
-three)
+three#three_adj)
@@ -162 +162 @@
-poynts))))))))))
+poynts#point_n1))))))))))
@@ -170 +170 @@
-1)
+1#one_adj)
@@ -175 +175 @@
-The)
+The#the_adj)
@@ -177 +177 @@
-perpetuitie)
+perpetuitie#perpetuity_n)
@@ -180 +180 @@
-of)
+of#of_prep)
@@ -183 +183 @@
-time)))
+time#time_n)))
@@ -186 +186 @@
-for)
+for#for_prep)
@@ -189 +189 @@
-the)
+the#the_adj)
@@ -191 +191 @@
-abrogation)
+abrogation#abrogation_n)
@@ -194 +194 @@
-of)
+of#of_prep)
@@ -197 +197 @@
-all)
+all#all_adj)
@@ -199 +199 @@
-Lawes)
+Lawes#law_n1)
@@ -202 +202 @@
-concerning)
+concerning#concerning_prep)
@@ -205 +205 @@
-the)
+the#the_adj)
@@ -207 +207 @@
-Roman)
+Roman#NA)
@@ -209 +209 @@
-Catholiques)))
+Catholiques#NA)))
@@ -220 +220 @@
-in)
+in#in_prep)
@@ -223 +223 @@
-noe)
+noe#no_adj)
@@ -225 +225 @@
-time)))
+time#time_n)))
@@ -227 +227 @@
-to)
+to#to_prep)
@@ -229 +229 @@
-bee)
+bee#be_v)
@@ -231 +231 @@
-renewed)
+renewed#renew_v1)
@@ -234 +234 @@
-against)
+against#against_prep)
@@ -237 +237 @@
-them))))))))))
+them#them_pron))))))))))
@@ -242 +242 @@
-nor)
+nor#nor_conj1)
@@ -245 +245 @@
-any)
+any#any_adj)
@@ -247 +247 @@
-other)
+other#other_adj)
@@ -255 +255 @@
-to)
+to#to_prep)
@@ -257 +257 @@
-bee)
+bee#be_v)
@@ -259 +259 @@
-raised)
+raised#raise_v1)
@@ -262 +262 @@
-in)
+in#in_prep)
@@ -265 +265 @@
-their)
+their#their_adj)
@@ -267 +267 @@
-place)))
+place#place_n.II-VII)))
@@ -270 +270 @@
-upon)
+upon#upon_prep)
@@ -273 +273 @@
-any)
+any#any_adj)
@@ -275 +275 @@
-occasion)))))))
+occasion#occasion_n1)))))))
@@ -283 +283 @@
-2)
+2#two_adj)
@@ -288 +288 @@
-His)
+His#his_adj)
@@ -290 +290 @@
-Majestie))
+Majestie#majesty_n))
@@ -292 +292 @@
-must)
+must#must_v1)
@@ -294 +294 @@
-do)
+do#do_v)
@@ -297 +297 @@
-his)
+his#his_adj)
@@ -299 +299 @@
-best))
+best#best_adj))
@@ -302 +302 @@
-that)
+that#that_conj)
@@ -307 +307 @@
-the)
+the#the_adj)
@@ -309 +309 @@
-Articles)
+Articles#article_n)
@@ -312 +312 @@
-in)
+in#in_prep)
@@ -315 +315 @@
-favour)
+favour#favour_n)
@@ -318 +318 @@
-of)
+of#of_prep)
@@ -321 +321 @@
-the)
+the#the_adj)
@@ -323 +323 @@
-Roman)
+Roman#NA)
@@ -325 +325 @@
-Catholiques))))))
+Catholiques#NA))))))
@@ -327 +327 @@
-must)
+must#must_v1)
@@ -329 +329 @@
-bee)
+bee#be_v)
@@ -331 +331 @@
-confirmed)
+confirmed#confirm_v)
@@ -334 +334 @@
-by)
+by#by_prep)
@@ -337 +337 @@
-Parliament)))
+Parliament#NA)))
@@ -340 +340 @@
-within)
+within#within_adv)
@@ -343 +343 @@
-three)
+three#three_adj)
@@ -345 +345 @@
-yeares)))
+yeares#year_n)))
@@ -348 +348 @@
-infallibly)))
+infallibly#infallibly_adv)))
@@ -353 +353 @@
-and)
+and#and_conj1)
@@ -357 +357 @@
-sooner))
+sooner#soon_adv))
@@ -362 +362 @@
-if)
+if#if_conj)
@@ -369 +369 @@
-the)
+the#the_adj)
@@ -371 +371 @@
-constitution)
+constitution#constitution_n)
@@ -374 +374 @@
-of)
+of#of_prep)
@@ -377 +377 @@
-affairs))))
+affairs#affair_n))))
@@ -379 +379 @@
-would)
+would#will_v1)
@@ -381 +381 @@
-permit)
+permit#permit_v)
@@ -384 +384 @@
-it)))))))))
+it#it_pron)))))))))
@@ -395 +395 @@
-3)
+3#three_adj)
@@ -400 +400 @@
-The)
+The#the_adj)
@@ -402 +402 @@
-Councells))
+Councells#council_n))
@@ -404 +404 @@
-oath)
+oath#oath_n)
@@ -413 +413 @@
-In)
+In#in_prep)
@@ -416 +416 @@
-the)
+the#the_adj)
@@ -418 +418 @@
-first)))
+first#first_adj)))
@@ -423 +423 @@
-his)
+his#his_adj)
@@ -425 +425 @@
-Majestie))
+Majestie#majesty_n))
@@ -427 +427 @@
-foresawe)
+foresawe#foresee_v)
@@ -431 +431 @@
-an)
+an#a_adj2)
@@ -433 +433 @@
-infinite)
+infinite#infinite_adj)
@@ -435 +435 @@
-liberty)
+liberty#liberty_n)
@@ -440 +440 @@
-a)
+a#a_adj2)
@@ -442 +442 @@
-perpetuall)
+perpetuall#perpetual_adj)
@@ -444 +444 @@
-emunity))
+emunity#immunity_n))
@@ -448 +448 @@
-graunted)
+graunted#grant_v)
@@ -451 +451 @@
-to)
+to#to_prep)
@@ -454 +454 @@
-the)
+the#the_adj)
@@ -456 +456 @@
-Roman)
+Roman#NA)
@@ -458 +458 @@
-Catholiques)))
+Catholiques#NA)))
@@ -464 +464 @@
-which))
+which#which_adj))
@@ -471 +471 @@
-if)
+if#if_conj)
@@ -479 +479 @@
-it))
+it#it_pron))
@@ -481 +481 @@
-should)
+should#shall_v)
@@ -483 +483 @@
-bring)
+bring#bring_v)
@@ -486 +486 @@
-them))
+them#them_pron))
@@ -489 +489 @@
-to)
+to#to_prep)
@@ -492 +492 @@
-daungerous)
+daungerous#dangerous_adj)
@@ -494 +494 @@
-encrease))))
+encrease#increase_n))))
@@ -499 +499 @@
-or)
+or#or_conj2)
@@ -502 +502 @@
-incourage)
+incourage#encourage_v)
@@ -505 +505 @@
-them))
+them#them_pron))
@@ -508 +508 @@
-to)
+to#to_prep)
@@ -511 +511 @@
-the)
+the#the_adj)
@@ -513 +513 @@
-acting)
+acting#acting_n)
@@ -516 +516 @@
-of)
+of#of_prep)
@@ -519 +519 @@
-insolencies))))))))))
+insolencies#insolency_n))))))))))
@@ -524 +524 @@
-his)
+his#his_adj)
@@ -526 +526 @@
-conscience))
+conscience#conscience_n))
@@ -528 +528 @@
-opposeth)
+opposeth#oppose_v)
@@ -531 +531 @@
-his)
+his#his_adj)
@@ -533 +533 @@
-wisedome)
+wisedome#wisdom_n)
@@ -536 +536 @@
-of)
+of#of_prep)
@@ -539 +539 @@
-Gouerment)))))
+Gouerment#government_n)))))
@@ -544 +544 @@
-and)
+and#and_conj1)
@@ -548 +548 @@
-his)
+his#his_adj)
@@ -550 +550 @@
-Soueraignity))
+Soueraignity#sovereignity_n))
@@ -554 +554 @@
-runs)
+runs#run_v)
@@ -557 +557 @@
-a)
+a#a_adj2)
@@ -559 +559 @@
-daunger)))))))
+daunger#danger_n)))))))
@@ -568 +568 @@
-Touching)
+Touching#touching_prep)
@@ -571 +571 @@
-the)
+the#the_adj)
@@ -573 +573 @@
-Parliament)))
+Parliament#parliament_n1)))
@@ -578 +578 @@
-his)
+his#his_adj)
@@ -580 +580 @@
-Majestie))
+Majestie#majesty_n))
@@ -582 +582 @@
-sawe)
+sawe#see_v)
@@ -586 +586 @@
-it))
+it#it_pron))
@@ -589 +589 @@
-impossible)
+impossible#impossible_adj)
@@ -597 +597 @@
-for)
+for#for_prep)
@@ -600 +600 @@
-him))
+him#him_pron))
@@ -602 +602 @@
-to)
+to#to_prep)
@@ -604 +604 @@
-effect)))))
+effect#effect_v)))))
@@ -612 +612 @@
-neither)
+neither#neither_adv)
@@ -614 +614 @@
-did)
+did#do_v)
@@ -617 +617 @@
-his)
+his#his_adj)
@@ -619 +619 @@
-affection)
+affection#affection_n)
@@ -622 +622 @@
-and)
+and#and_conj1)
@@ -625 +625 @@
-reason))))
+reason#reason_n1))))
@@ -627 +627 @@
-incline)
+incline#incline_v)
@@ -630 +630 @@
-to)
+to#to_prep)
@@ -632 +632 @@
-exercise)
+exercise#exercise_v)
@@ -635 +635 @@
-his)
+his#his_adj)
@@ -637 +637 @@
-power))
+power#power_n1))
@@ -640 +640 @@
-that)
+that#that_pron)
@@ -642 +642 @@
-way))
+way#way_n1))
@@ -645 +645 @@
-if)
+if#if_conj)
@@ -652 +652 @@
-it))
+it#it_pron))
@@ -654 +654 @@
-were)
+were#be_v)
@@ -657 +657 @@
-in)
+in#in_prep)
@@ -660 +660 @@
-his)
+his#his_adj)
@@ -662 +662 @@
-hand)))))))
+hand#hand_n)))))))
@@ -670 +670 @@
-And)
+And#and_conj1)
@@ -673 +673 @@
-for)
+for#for_prep)
@@ -677 +677 @@
-the)
+the#the_adj)
@@ -679 +679 @@
-Councells))
+Councells#council_n))
@@ -681 +681 @@
-oath)))
+oath#oath_n)))
@@ -686 +686 @@
-as)
+as#as_adv)
@@ -697 +697 @@
-his)
+his#his_adj)
@@ -699 +699 @@
-actes))
+actes#act_n))
@@ -701 +701 @@
-had)
+had#have_v)
@@ -703 +703 @@
-not)
+not#not_adv)
@@ -706 +706 @@
-need))
+need#need_n1))
@@ -709 +709 @@
-of)
+of#of_prep)
@@ -712 +712 @@
-theire)
+theire#their_adj)
@@ -714 +714 @@
-fortification))))))
+fortification#fortification_n))))))
@@ -719 +719 @@
-soe))
+soe#so_adv))
@@ -721 +721 @@
-might)
+might#may_v1)
@@ -725 +725 @@
-his)
+his#his_adj)
@@ -727 +727 @@
-honor))
+honor#honour_n))
@@ -730 +730 @@
-and)
+and#and_conj1)
@@ -733 +733 @@
-the)
+the#the_adj)
@@ -735 +735 @@
-cause))))
+cause#cause_n))))
@@ -737 +737 @@
-recieve)
+recieve#receive_v)
@@ -740 +740 @@
-prejudice))
+prejudice#prejudice_n))
@@ -743 +743 @@
-by)
+by#by_prep)
@@ -746 +746 @@
-their)
+their#their_adj)
@@ -748 +748 @@
-refusall)))
+refusall#refusal_n)))
@@ -756 +756 @@
-But)
+But#but_prep)
@@ -759 +759 @@
-that)
+that#that_pron)
@@ -763 +763 @@
-which))
+which#which_adj))
@@ -771 +771 @@
-pinch't)
+pinch't#pinch_v)
@@ -773 +773 @@
-and)
+and#and_conj1)
@@ -775 +775 @@
-perplex't))
+perplex't#perplex_v))
@@ -778 +778 @@
-most)))))
+most#most_adj)))))
@@ -782 +782 @@
-was)
+was#be_v)
@@ -788 +788 @@
-that)
+that#that_conj)
@@ -792 +792 @@
-this))
+this#this_pron))
@@ -794 +794 @@
-was)
+was#be_v)
@@ -796 +796 @@
-not)
+not#not_adv)
@@ -799 +799 @@
-now))
+now#now_adv))
@@ -802 +802 @@
-a)
+a#a_adj2)
@@ -805 +805 @@
-free)
+free#free_adj)
@@ -807 +807 @@
-and)
+and#and_conj1)
@@ -809 +809 @@
-entire))
+entire#entire_adj))
@@ -811 +811 @@
-cause)))))
+cause#cause_n)))))
@@ -819 +819 @@
-But)
+But#but_prep)
@@ -823 +823 @@
-his)
+his#his_adj)
@@ -825 +825 @@
-Majesties))
+Majesties#majesty_n))
@@ -827 +827 @@
-power))
+power#power_n1))
@@ -829 +829 @@
-was)
+was#be_v)
@@ -831 +831 @@
-given)
+given#give_v)
@@ -834 +834 @@
-to)
+to#to_prep)
@@ -837 +837 @@
-the)
+the#the_adj)
@@ -839 +839 @@
-Prince)))
+Prince#prince_n)))
@@ -845 +845 @@
-according)
+according#accord_v)
@@ -848 +848 @@
-to)
+to#to_prep)
@@ -851 +851 @@
-which)
+which#which_adj)
@@ -853 +853 @@
-power))))
+power#power_n1))))
@@ -863 +863 @@
-his)
+his#his_adj)
@@ -865 +865 @@
-Highness))
+Highness#highness_n))
@@ -867 +867 @@
-had)
+had#have_v)
@@ -869 +869 @@
-concluded)
+concluded#conclude_v)
@@ -872 +872 @@
-theise)
+theise#these_pron)
@@ -874 +874 @@
-Articles))
+Articles#article_n))
@@ -879 +879 @@
-soe)
+soe#so_adv)
@@ -882 +882 @@
-as)
+as#as_adv)
@@ -890 +890 @@
-now))
+now#now_adv))
@@ -893 +893 @@
-it))
+it#it_pron))
@@ -895 +895 @@
-went)
+went#go_v)
@@ -898 +898 @@
-upon)
+upon#upon_prep)
@@ -901 +901 @@
-the)
+the#the_adj)
@@ -903 +903 @@
-honor)
+honor#honour_n)
@@ -906 +906 @@
-of)
+of#of_prep)
@@ -910 +910 @@
-his)
+his#his_adj)
@@ -912 +912 @@
-Majestie))
+Majestie#majesty_n))
@@ -915 +915 @@
-and)
+and#and_conj1)
@@ -918 +918 @@
-the)
+the#the_adj)
@@ -920 +920 @@
-Prince))))))))
+Prince#prince_n))))))))
@@ -925 +925 @@
-and)
+and#and_conj1)
@@ -929 +929 @@
-perhaps))
+perhaps#perhaps_adv))
@@ -932 +932 @@
-upon)
+upon#upon_prep)
@@ -935 +935 @@
-the)
+the#the_adj)
@@ -937 +937 @@
-liberty)
+liberty#liberty_n)
@@ -940 +940 @@
-and)
+and#and_conj1)
@@ -943 +943 @@
-power)))
+power#power_n1)))
@@ -946 +946 @@
-of)
+of#of_prep)
@@ -950 +950 @@
-his)
+his#his_adj)
@@ -952 +952 @@
-Highness))
+Highness#highness_n))
@@ -954 +954 @@
-returne)
+returne#return_n)
@@ -957 +957 @@
-and)
+and#and_conj1)
@@ -960 +960 @@
-safety)
+safety#safety_n)
@@ -963 +963 @@
-of)
+of#of_prep)
@@ -966 +966 @@
-his)
+his#his_adj)
@@ -968 +968 @@
-person)))))))))))))))))
+person#person_n)))))))))))))))))
@@ -977 +977 @@
-These)
+These#these_pron)
@@ -979 +979 @@
-tender)
+tender#tender_adj)
@@ -981 +981 @@
-considerations)
+considerations#consideration_n)
@@ -984 +984 @@
-of)
+of#of_prep)
@@ -988 +988 @@
-honor))
+honor#honour_n))
@@ -994 +994 @@
-suerty)
+suerty#surety_n)
@@ -997 +997 @@
-of)
+of#of_prep)
@@ -1000 +1000 @@
-his)
+his#his_adj)
@@ -1002 +1002 @@
-estate)))))
+estate#estate_n)))))
@@ -1008 +1008 @@
-fatherly)
+fatherly#fatherly_adj)
@@ -1010 +1010 @@
-loue)))
+loue#love_n1)))
@@ -1015 +1015 @@
-and)
+and#and_conj1)
@@ -1018 +1018 @@
-conscience))))))
+conscience#conscience_n))))))
@@ -1023 +1023 @@
-his)
+his#his_adj)
@@ -1025 +1025 @@
-Majestie))
+Majestie#majesty_n))
@@ -1027 +1027 @@
-debated)
+debated#debate_v1)
@@ -1030 +1030 @@
-some)
+some#some_pron)
@@ -1032 +1032 @@
-dayes))
+dayes#day_n))
@@ -1037 +1037 @@
-with)
+with#with_prep)
@@ -1041 +1041 @@
-as)
+as#as_adv)
@@ -1043 +1043 @@
-much))
+much#much_adj))
@@ -1045 +1045 @@
-wisedome)
+wisedome#wisdom_n)
@@ -1051 +1051 @@
-naturall)
+naturall#natural_adj)
@@ -1053 +1053 @@
-affection)))
+affection#affection_n)))
@@ -1059 +1059 @@
-courage)))
+courage#courage_n)))
@@ -1064 +1064 @@
-and)
+and#and_conj1)
@@ -1067 +1067 @@
-piety)))
+piety#piety_n)))
@@ -1072 +1072 @@
-as)
+as#as_adv)
@@ -1082 +1082 @@
-became)
+became#become_v)
@@ -1085 +1085 @@
-a)
+a#a_adj2)
@@ -1087 +1087 @@
-greate)
+greate#great_adj)
@@ -1091 +1091 @@
-wise)
+wise#wise_adj)
@@ -1095 +1095 @@
-religious)
+religious#religious_adj)
@@ -1097 +1097 @@
-Kinge)
+Kinge#king_n)
@@ -1102 +1102 @@
-and)
+and#and_conj1)
@@ -1105 +1105 @@
-tender)
+tender#tender_adj)
@@ -1107 +1107 @@
-louing)
+louing#loving_adj)
@@ -1109 +1109 @@
-Father)))))))))
+Father#father_n)))))))))
@@ -1117 +1117 @@
-And)
+And#and_conj1)
@@ -1120 +1120 @@
-in)
+in#in_prep)
@@ -1123 +1123 @@
-conclusion)))
+conclusion#conclusion_n)))
@@ -1126 +1126 @@
-having)
+having#have_v)
@@ -1129 +1129 @@
-often))
+often#often_adv))
@@ -1131 +1131 @@
-honored)
+honored#honour_v)
@@ -1135 +1135 @@
-Secretary)
+Secretary#NA)
@@ -1137 +1137 @@
-Calvert))
+Calvert#NA))
@@ -1143 +1143 @@
-Cottington)))
+Cottington#NA)))
@@ -1148 +1148 @@
-and)
+and#and_conj1)
@@ -1151 +1151 @@
-myself))))
+myself#myself_pron))))
@@ -1154 +1154 @@
-with)
+with#with_prep)
@@ -1157 +1157 @@
-the)
+the#the_adj)
@@ -1159 +1159 @@
-hearing)
+hearing#hearing_n)
@@ -1162 +1162 @@
-of)
+of#of_prep)
@@ -1167 +1167 @@
-his)
+his#his_adj)
@@ -1169 +1169 @@
-wise)
+wise#wise_adj)
@@ -1171 +1171 @@
-apprehensions)
+apprehensions#apprehension_n)
@@ -1174 +1174 @@
-and)
+and#and_conj1)
@@ -1177 +1177 @@
-deliberations)))
+deliberations#deliberation_n1)))
@@ -1183 +1183 @@
-in)
+in#in_prep)
@@ -1185 +1185 @@
-and)
+and#and_conj1)
@@ -1187 +1187 @@
-about))
+about#about_adv))
@@ -1190 +1190 @@
-his)
+his#his_adj)
@@ -1192 +1192 @@
-greate)
+greate#great_adj)
@@ -1194 +1194 @@
-perplexity))))))))
+perplexity#perplexity_n))))))))
@@ -1199 +1199 @@
-his)
+his#his_adj)
@@ -1201 +1201 @@
-Majesty))
+Majesty#majesty_n))
@@ -1203 +1203 @@
-resolved)
+resolved#resolve_v)
@@ -1206 +1206 @@
-to)
+to#to_prep)
@@ -1208 +1208 @@
-call)
+call#call_v)
@@ -1211 +1211 @@
-to)
+to#to_prep)
@@ -1214 +1214 @@
-him)))
+him#him_pron)))
@@ -1217 +1217 @@
-some)
+some#some_pron)
@@ -1220 +1220 @@
-of)
+of#of_prep)
@@ -1223 +1223 @@
-the)
+the#the_adj)
@@ -1226 +1226 @@
-most)
+most#most_adj)
@@ -1228 +1228 @@
-eminent))
+eminent#eminent_adj))
@@ -1231 +1231 @@
-of)
+of#of_prep)
@@ -1234 +1234 @@
-his)
+his#his_adj)
@@ -1236 +1236 @@
-Councell)))))
+Councell#council_n)))))
@@ -1242 +1242 @@
-whose)
+whose#whose_pron)
@@ -1244 +1244 @@
-names))
+names#name_n))
@@ -1251 +1251 @@
-are)
+are#be_v)
@@ -1254 +1254 @@
-here))
+here#here_adv))
@@ -1256 +1256 @@
-inclosed))))
+inclosed#enclose_v))))
@@ -1261 +1261 @@
-that)
+that#that_conj)
@@ -1266 +1266 @@
-hee))
+hee#he_pron))
@@ -1268 +1268 @@
-might)
+might#may_v1)
@@ -1270 +1270 @@
-open)
+open#open_v)
@@ -1273 +1273 @@
-himself))
+himself#himself_pron))
@@ -1276 +1276 @@
-to)
+to#to_prep)
@@ -1279 +1279 @@
-them))))
+them#them_pron))))
@@ -1285 +1285 @@
-recieve)
+recieve#receive_v)
@@ -1288 +1288 @@
-advice)
+advice#advice_n)
@@ -1290 +1290 @@
-or)
+or#or_conj2)
@@ -1292 +1292 @@
-confirmacion))))
+confirmacion#confirmation_n))))
@@ -1297 +1297 @@
-or)
+or#or_conj2)
@@ -1301 +1301 @@
-at)
+at#at_prep)
@@ -1304 +1304 @@
-least)))
+least#least_adj)))
@@ -1306 +1306 @@
-take)
+take#take_v)
@@ -1311 +1311 @@
-by)
+by#by_prep)
@@ -1314 +1314 @@
-those)
+those#those_pron)
@@ -1316 +1316 @@
-leading)
+leading#leading_adj)
@@ -1318 +1318 @@
-voyces)))
+voyces#voice_n)))
@@ -1323 +1323 @@
-a)
+a#a_adj2)
@@ -1325 +1325 @@
-measure)
+measure#measure_n)
@@ -1328 +1328 @@
-of)
+of#of_prep)
@@ -1331 +1331 @@
-the)
+the#the_adj)
@@ -1333 +1333 @@
-iudgments)
+iudgments#judgement_n)
@@ -1336 +1336 @@
-and)
+and#and_conj1)
@@ -1339 +1339 @@
-affections)))
+affections#affection_n)))
@@ -1342 +1342 @@
-of)
+of#of_prep)
@@ -1345 +1345 @@
-the)
+the#the_adj)
@@ -1347 +1347 @@
-rest)))))))))))
+rest#rest_n2)))))))))))
@@ -1356 +1356 @@
-Without)
+Without#without_adv)
@@ -1359 +1359 @@
-flattery)))
+flattery#flattery_n)))
@@ -1363 +1363 @@
-I))
+I#i_pron))
@@ -1365 +1365 @@
-speak)
+speak#speak_v)
@@ -1368 +1368 @@
-the)
+the#the_adj)
@@ -1370 +1370 @@
-truthe))
+truthe#truth_n))
@@ -1373 +1373 @@
-before)
+before#before_adv)
@@ -1376 +1376 @@
-God))))
+God#NA))))
@@ -1381 +1381 @@
-those)
+those#those_pron)
@@ -1383 +1383 @@
-Councellors))
+Councellors#counsellor_n))
@@ -1385 +1385 @@
-assembled)
+assembled#assemble_v1)
@@ -1388 +1388 @@
-on)
+on#on_prep)
@@ -1391 +1391 @@
-Sunday)))
+Sunday#NA)))
@@ -1394 +1394 @@
-at)
+at#at_prep)
@@ -1397 +1397 @@
-Wanstid)))
+Wanstid#NA)))
@@ -1406 +1406 @@
-His)
+His#his_adj)
@@ -1408 +1408 @@
-Majestie))
+Majestie#majesty_n))
@@ -1410 +1410 @@
-made)
+made#make_v1)
@@ -1414 +1414 @@
-the)
+the#the_adj)
@@ -1417 +1417 @@
-most)
+most#most_adj)
@@ -1419 +1419 @@
-serious)))
+serious#serious_adj)))
@@ -1427 +1427 @@
-I))
+I#i_pron))
@@ -1429 +1429 @@
-may)
+may#may_v1)
@@ -1431 +1431 @@
-say))
+say#say_v1))
@@ -1437 +1437 @@
-the)
+the#the_adj)
@@ -1440 +1440 @@
-most)
+most#most_adj)
@@ -1442 +1442 @@
-sadd)
+sadd#sad_adj)
@@ -1448 +1448 @@
-fatherly)))
+fatherly#fatherly_adj)))
@@ -1454 +1454 @@
-kinde)))
+kinde#kind_adj)))
@@ -1460 +1460 @@
-kingly)))
+kingly#kingly_adj)))
@@ -1466 +1466 @@
-wise)))
+wise#wise_adj)))
@@ -1471 +1471 @@
-and)
+and#and_conj1)
@@ -1474 +1474 @@
-pious)))
+pious#pious_adj)))
@@ -1480 +1480 @@
-manly)))
+manly#manly_adj)))
@@ -1486 +1486 @@
-stout))))
+stout#stout_adj))))
@@ -1488 +1488 @@
-speech)))
+speech#speech_n1)))
@@ -1493 +1493 @@
-that)
+that#that_conj)
@@ -1499 +1499 @@
-ever))
+ever#ever_adv))
@@ -1502 +1502 @@
-I))
+I#i_pron))
@@ -1504 +1504 @@
-heard)))
+heard#hear_v)))
@@ -1510 +1510 @@
-which))
+which#which_adj))
@@ -1518 +1518 @@
-noe)
+noe#no_adj)
@@ -1520 +1520 @@
-man))
+man#man_n1))
@@ -1522 +1522 @@
-can)
+can#can_v1)
@@ -1525 +1525 @@
-repeate)
+repeate#repeat_v)
@@ -1527 +1527 @@
-or)
+or#or_conj2)
@@ -1529 +1529 @@
-relate))
+relate#relate_v))
@@ -1534 +1534 @@
-without)
+without#without_adv)
@@ -1537 +1537 @@
-blemishing)
+blemishing#blemish_v)
@@ -1542 +1542 @@
-but)
+but#but_prep)
@@ -1544 +1544 @@
-himself)))))))
+himself#himself_pron)))))))
@@ -1552 +1552 @@
-But)
+But#but_prep)
@@ -1555 +1555 @@
-this)
+this#this_pron)
@@ -1557 +1557 @@
-effect)
+effect#effect_n)
@@ -1562 +1562 @@
-it))
+it#it_pron))
@@ -1564 +1564 @@
-wrought)
+wrought#work_v)
@@ -1573 +1573 @@
-all)
+all#all_adj)
@@ -1575 +1575 @@
-the)
+the#the_adj)
@@ -1577 +1577 @@
-Lords))
+Lords#lord_n))
@@ -1579 +1579 @@
-were)
+were#be_v)
@@ -1582 +1582 @@
-of)
+of#of_prep)
@@ -1585 +1585 @@
-opinion)
+opinion#opinion_n)
@@ -1589 +1589 @@
-that)
+that#that_conj)
@@ -1594 +1594 @@
-his)
+his#his_adj)
@@ -1596 +1596 @@
-Highness))
+Highness#highness_n))
@@ -1598 +1598 @@
-wordes)
+wordes#word_n)
@@ -1601 +1601 @@
-and)
+and#and_conj1)
@@ -1604 +1604 @@
-Articles))))
+Articles#article_n))))
@@ -1606 +1606 @@
-must)
+must#must_v1)
@@ -1608 +1608 @@
-bee)
+bee#be_v)
@@ -1610 +1610 @@
-made)
+made#make_v1)
@@ -1616 +1616 @@
-good)))))
+good#good_adj)))))
@@ -1622 +1622 @@
-that)
+that#that_conj)
@@ -1626 +1626 @@
-the)
+the#the_adj)
@@ -1628 +1628 @@
-oath))
+oath#oath_n))
@@ -1631 +1631 @@
-by)
+by#by_prep)
@@ -1634 +1634 @@
-the)
+the#the_adj)
@@ -1636 +1636 @@
-Councell)))
+Councell#council_n)))
@@ -1638 +1638 @@
-must)
+must#must_v1)
@@ -1640 +1640 @@
-bee)
+bee#be_v)
@@ -1642 +1642 @@
-taken)))))))))
+taken#take_v)))))))))
@@ -1650 +1650 @@
-and)
+and#and_conj1)
@@ -1653 +1653 @@
-with)
+with#with_prep)
@@ -1656 +1656 @@
-one)
+one#one_adj)
@@ -1658 +1658 @@
-voyce)))
+voyce#voice_n)))
@@ -1662 +1662 @@
-gave)
+gave#give_v)
@@ -1665 +1665 @@
-counsell))
+counsell#counsel_n))
@@ -1670 +1670 @@
-as)
+as#as_adv)
@@ -1674 +1674 @@
-without)
+without#without_adv)
@@ -1677 +1677 @@
-which)))
+which#which_adj)))
@@ -1685 +1685 @@
-nothing))
+nothing#nothing_pron))
@@ -1687 +1687 @@
-could)
+could#can_v1)
@@ -1689 +1689 @@
-bee)
+bee#be_v)
@@ -1692 +1692 @@
-well))))
+well#well_adj))))
@@ -1697 +1697 @@
-that)
+that#that_conj)
@@ -1702 +1702 @@
-the)
+the#the_adj)
@@ -1704 +1704 @@
-Prince))
+Prince#prince_n))
@@ -1706 +1706 @@
-must)
+must#must_v1)
@@ -1708 +1708 @@
-marry)
+marry#marry_v)
@@ -1711 +1711 @@
-and)
+and#and_conj1)
@@ -1713 +1713 @@
-bringe)
+bringe#bring_v)
@@ -1716 +1716 @@
-his)
+his#his_adj)
@@ -1718 +1718 @@
-Lady))
+Lady#lady_n))
@@ -1721 +1721 @@
-away))
+away#away_adv))
@@ -1724 +1724 @@
-with)
+with#with_prep)
@@ -1727 +1727 @@
-him))))
+him#him_pron))))
@@ -1730 +1730 @@
-this)
+this#this_pron)
@@ -1732 +1732 @@
-yeare)
+yeare#year_n)
@@ -1737,3 +1737,3 @@
-this)
-(ADJ
-old)
+this#this_pron)
+(ADJ
+old#old_adj)
@@ -1741 +1741 @@
-yeare))))
+yeare#year_n))))
@@ -1746 +1746 @@
-or)
+or#or_conj2)
@@ -1748 +1748 @@
-ells)
+ells#else_adv)
@@ -1754 +1754 @@
-the)
+the#the_adj)
@@ -1756 +1756 @@
-Prince))
+Prince#prince_n))
@@ -1759 +1759 @@
-presently))
+presently#presently_adv))
@@ -1761 +1761 @@
-to)
+to#to_prep)
@@ -1763 +1763 @@
-returne)
+returne#return_v1)
@@ -1766 +1766 @@
-without)
+without#without_adv)
@@ -1769 +1769 @@
-marriage)
+marriage#marriage_n)
@@ -1771 +1771 @@
-or)
+or#or_conj2)
@@ -1773 +1773 @@
-contract)))
+contract#contract_n1)))
@@ -1778 +1778 @@
-leaving)
+leaving#leave_v1)
@@ -1782 +1782 @@
-both)
+both#both_pron)
@@ -1784 +1784 @@
-those))
+those#those_pron))
@@ -1786 +1786 @@
-to)
+to#to_prep)
@@ -1788 +1788 @@
-bee)
+bee#be_v)
@@ -1790 +1790 @@
-accomplished)
+accomplished#accomplish_v)
@@ -1793 +1793 @@
-by)
+by#by_prep)
@@ -1796 +1796 @@
-the)
+the#the_adj)
@@ -1798 +1798 @@
-usuall)
+usuall#usual_adj)
@@ -1800 +1800 @@
-formes)))))))))
+formes#form_n)))))))))
@@ -1810 +1810 @@
-My)
+My#my_adj)
@@ -1812 +1812 @@
-Maister))
+Maister#master_n1))
@@ -1814 +1814 @@
-having)
+having#have_v)
@@ -1816 +1816 @@
-honored)
+honored#honour_v)
@@ -1819 +1819 @@
-mee))
+mee#me_pron1))
@@ -1822 +1822 @@
-with)
+with#with_prep)
@@ -1825 +1825 @@
-his)
+his#his_adj)
@@ -1827 +1827 @@
-commaundements)
+commaundements#commandment_n)
@@ -1830 +1830 @@
-in)
+in#in_prep)
@@ -1833 +1833 @@
-the)
+the#the_adj)
@@ -1835 +1835 @@
-trayne)
+trayne#train_n1)
@@ -1838 +1838 @@
-and)
+and#and_conj1)
@@ -1841 +1841 @@
-shadow)))
+shadow#shadow_n)))
@@ -1844 +1844 @@
-of)
+of#of_prep)
@@ -1847 +1847 @@
-that))))))))
+that#that_pron))))))))
@@ -1852 +1852 @@
-I))
+I#i_pron))
@@ -1854 +1854 @@
-presume)
+presume#presume_v)
@@ -1857 +1857 @@
-to)
+to#to_prep)
@@ -1859 +1859 @@
-convey)
+convey#convey_v1)
@@ -1862 +1862 @@
-by)
+by#by_prep)
@@ -1865 +1865 @@
-your)
+your#your_pron)
@@ -1867 +1867 @@
-Grace)))
+Grace#grace_n)))
@@ -1872 +1872 @@
-to)
+to#to_prep)
@@ -1875 +1875 @@
-his)
+his#his_adj)
@@ -1877 +1877 @@
-Highnes)))
+Highnes#highness_n)))
@@ -1883 +1883 @@
-as)
+as#as_adv)
@@ -1885 +1885 @@
-humble))
+humble#humble_adj))
@@ -1887 +1887 @@
-thankfulnes)
+thankfulnes#thankfulness_n)
@@ -1890 +1890 @@
-as)
+as#as_adv)
@@ -1899 +1899 @@
-my)
+my#my_adj)
@@ -1901 +1901 @@
-poore)
+poore#poor_adj)
@@ -1903 +1903 @@
-heart))
+heart#heart_n))
@@ -1905 +1905 @@
-is)
+is#be_v)
@@ -1908 +1908 @@
-capable)
+capable#capable_adj)
@@ -1911 +1911 @@
-of)
+of#of_prep)
@@ -1918 +1918 @@
-for)
+for#for_prep)
@@ -1921 +1921 @@
-the)
+the#the_adj)
@@ -1923 +1923 @@
-signe)
+signe#sign_n)
@@ -1926 +1926 @@
-of)
+of#of_prep)
@@ -1930 +1930 @@
-his)
+his#his_adj)
@@ -1932 +1932 @@
-Highnes))
+Highnes#highness_n))
@@ -1934 +1934 @@
-memory)
+memory#memory_n)
@@ -1937 +1937 @@
-of)
+of#of_prep)
@@ -1940 +1940 @@
-me)))))
+me#me_pron1)))))
@@ -1943 +1943 @@
-and)
+and#and_conj1)
@@ -1946 +1946 @@
-favour)
+favour#favour_n)
@@ -1949 +1949 @@
-to)
+to#to_prep)
@@ -1952 +1952 @@
-me)))
+me#me_pron1)))
@@ -1955 +1955 @@
-received)
+received#receive_v)
@@ -1958 +1958 @@
-by)
+by#by_prep)
@@ -1961 +1961 @@
-M=r=)
+M=r=#NA)
@@ -1963 +1963 @@
-Killigrey))))))))))
+Killigrey#NA))))))))))
@@ -1971 +1971 @@
-And)
+And#and_conj1)
@@ -1974 +1974 @@
-I))
+I#i_pron))
@@ -1976 +1976 @@
-beseech)
+beseech#beseech_v)
@@ -1979 +1979 @@
-your)
+your#your_pron)
@@ -1981 +1981 @@
-Grace))
+Grace#grace_n))
@@ -1984 +1984 @@
-to)
+to#to_prep)
@@ -1986 +1986 @@
-promise)
+promise#promise_v)
@@ -1989 +1989 @@
-to)
+to#to_prep)
@@ -1992 +1992 @@
-his)
+his#his_adj)
@@ -1994 +1994 @@
-Highnes)))
+Highnes#highness_n)))
@@ -1997 +1997 @@
-for)
+for#for_prep)
@@ -2000 +2000 @@
-mee)))
+mee#me_pron1)))
@@ -2003 +2003 @@
-that)
+that#that_conj)
@@ -2008 +2008 @@
-I))
+I#i_pron))
@@ -2010 +2010 @@
-will)
+will#will_v1)
@@ -2014 +2014 @@
-not)
+not#not_adv)
@@ -2016 +2016 @@
-onely)
+onely#only_adj)
@@ -2018 +2018 @@
-fill)
+fill#fill_v)
@@ -2021 +2021 @@
-my)
+my#my_adj)
@@ -2023 +2023 @@
-pocketts))
+pocketts#pocket_n))
@@ -2026 +2026 @@
-with)
+with#with_prep)
@@ -2029 +2029 @@
-papers)))
+papers#paper_n)))
@@ -2032 +2032 @@
-to)
+to#to_prep)
@@ -2034 +2034 @@
-attend)
+attend#attend_v)
@@ -2037 +2037 @@
-his)
+his#his_adj)
@@ -2039 +2039 @@
-service))))
+service#service_n1))))
@@ -2044 +2044 @@
-but)
+but#but_prep)
@@ -2048 +2048 @@
-bleeding))
+bleeding#bleed_v))
@@ -2052 +2052 @@
-emptie)
+emptie#empty_v)
@@ -2055 +2055 @@
-my)
+my#my_adj)
@@ -2057 +2057 @@
-vaynes))
+vaynes#vein_n))
@@ -2060 +2060 @@
-for)
+for#for_prep)
@@ -2063 +2063 @@
-it))))))))
+it#it_pron))))))))
@@ -2072 +2072 @@
-It))
+It#it_pron))
@@ -2074 +2074 @@
-is)
+is#be_v)
@@ -2077 +2077 @@
-your)
+your#your_pron)
@@ -2079 +2079 @@
-free)
+free#free_adj)
@@ -2081 +2081 @@
-goodnes)
+goodnes#goodness_n)
@@ -2084 +2084 @@
-and)
+and#and_conj1)
@@ -2087 +2087 @@
-favour)))
+favour#favour_n)))
@@ -2090 +2090 @@
-to)
+to#to_prep)
@@ -2093 +2093 @@
-mee)))
+mee#me_pron1)))
@@ -2097 +2097 @@
-which))
+which#which_adj))
@@ -2105 +2105 @@
-open's)
+open's#open_v)
@@ -2108 +2108 @@
-the)
+the#the_adj)
@@ -2110 +2110 @@
-way)))
+way#way_n1)))
@@ -2115 +2115 @@
-and)
+and#and_conj1)
@@ -2120 +2120 @@
-hath)
+hath#have_v)
@@ -2122 +2122 @@
-ledd)
+ledd#lead_v1)
@@ -2125 +2125 @@
-mee))
+mee#me_pron1))
@@ -2128 +2128 @@
-to)
+to#to_prep)
@@ -2131 +2131 @@
-that)
+that#that_pron)
@@ -2133 +2133 @@
-condition)
+condition#condition_n)
@@ -2136 +2136 @@
-of)
+of#of_prep)
@@ -2139 +2139 @@
-favour)))
+favour#favour_n)))
@@ -2150 +2150 @@
-I))
+I#i_pron))
@@ -2152 +2152 @@
-have))))))))))
+have#have_v))))))))))
@@ -2160 +2160 @@
-Preserve)
+Preserve#preserve_v)
@@ -2163 +2163 @@
-your)
+your#your_pron)
@@ -2165 +2165 @@
-creature))
+creature#creature_n))
@@ -2171 +2171 @@
-as)
+as#as_adv)
@@ -2173 +2173 @@
-farr))
+farr#far_adj))
@@ -2176 +2176 @@
-as)
+as#as_adv)
@@ -2187 +2187 @@
-you))
+you#you_pron))
@@ -2189 +2189 @@
-iudge)
+iudge#judge_v)
@@ -2196 +2196 @@
-my)
+my#my_adj)
@@ -2198 +2198 @@
-faith)
+faith#faith_n)
@@ -2201 +2201 @@
-and)
+and#and_conj1)
@@ -2204 +2204 @@
-thankfulnes)))
+thankfulnes#thankfulness_n)))
@@ -2208 +2208 @@
-will)
+will#will_v1)
@@ -2210 +2210 @@
-warrant)
+warrant#warrant_v)
@@ -2213 +2213 @@
-you))
+you#you_pron))
@@ -2219 +2219 @@
-which))
+which#which_adj))
@@ -2227 +2227 @@
-I))
+I#i_pron))
@@ -2229 +2229 @@
-dare)
+dare#dare_v1)
@@ -2231 +2231 @@
-not)
+not#not_adv)
@@ -2233 +2233 @@
-discribe)
+discribe#describe_v)
@@ -2238 +2238 @@
-knoweing)
+knoweing#know_v)
@@ -2246 +2246 @@
-my)
+my#my_adj)
@@ -2248 +2248 @@
-thoughts))
+thoughts#thought_n))
@@ -2250 +2250 @@
-exceede)
+exceede#exceed_v)
@@ -2253 +2253 @@
-my)
+my#my_adj)
@@ -2255 +2255 @@
-words)))
+words#word_n)))
@@ -2260 +2260 @@
-and)
+and#and_conj1)
@@ -2266 +2266 @@
-may)
+may#may_v1)
@@ -2268 +2268 @@
-outgoe)
+outgoe#outgo_v)
@@ -2271 +2271 @@
-the)
+the#the_adj)
@@ -2273 +2273 @@
-opinion)
+opinion#opinion_n)
@@ -2276 +2276 @@
-of)
+of#of_prep)
@@ -2279 +2279 @@
-others)))))
+others#other_adj)))))
@@ -2284 +2284 @@
-and)
+and#and_conj1)
@@ -2287 +2287 @@
-fall)
+fall#fall_v)
@@ -2290 +2290 @@
-short)
+short#short_adj)
@@ -2293 +2293 @@
-of)
+of#of_prep)
@@ -2296 +2296 @@
-my)
+my#my_adj)
@@ -2298 +2298 @@
-owne)
+owne#own_adj)
@@ -2300 +2300 @@
-knowledge)
+knowledge#knowledge_n)
@@ -2306 +2306 @@
-a)
+a#a_adj2)
@@ -2308 +2308 @@
-part)
+part#part_n)
@@ -2311 +2311 @@
-of)
+of#of_prep)
@@ -2314 +2314 @@
-which))))
+which#which_adj))))
@@ -2321 +2321 @@
-is)
+is#be_v)
@@ -2325 +2325 @@
-that)
+that#that_conj)
@@ -2329 +2329 @@
-I))
+I#i_pron))
@@ -2331 +2331 @@
-will)
+will#will_v1)
@@ -2334 +2334 @@
-live)
+live#live_v1)
@@ -2336 +2336 @@
-and)
+and#and_conj1)
@@ -2338 +2338 @@
-dye))
+dye#die_v1))
@@ -2342 +2342 @@
-Your)
+Your#your_pron)
@@ -2344 +2344 @@
-Graces))
+Graces#grace_n))
@@ -2347 +2347 @@
-Most)
+Most#most_adj)
@@ -2349 +2349 @@
-humble))
+humble#humble_adj))
@@ -2351 +2351 @@
-servant)
+servant#servant_n)
@@ -2354 +2354 @@
-Edw.)
+Edw.#NA)
@@ -2356 +2356 @@
-Conwey))))))))))))))))))))))))))
+Conwey#NA))))))))))))))))))))))))))
@@ -2364 +2364 @@
-Theobalds))
+Theobalds#NA))
@@ -2370 +2370 @@
-17=th=)
+17=th=#NA)
@@ -2373 +2373 @@
-of)
+of#of_prep)
@@ -2376 +2376 @@
-July)))
+July#NA)))
@@ -2378 +2378 @@
-1623)
+1623#NA)
