--- counc-1688-e3-h.psd.vert	2025-09-10 13:50:35
+++ counc-1688-e3-h.lem.vert	2025-09-10 13:50:45
@@ -17 +17 @@
-After)
+After#after_adv)
@@ -20 +20 @@
-our)
+our#our_pron)
@@ -23 +23 @@
-very)
+very#very_adj)
@@ -25 +25 @@
-hearty))
+hearty#hearty_adj))
@@ -27 +27 @@
-Commendations)
+Commendations#commendation_n)
@@ -30 +30 @@
-to)
+to#to_prep)
@@ -33 +33 @@
-your)
+your#your_pron)
@@ -35 +35 @@
-Lordship)))))
+Lordship#lordship_n)))))
@@ -41 +41 @@
-it))
+it#it_pron))
@@ -43 +43 @@
-having)
+having#have_v)
@@ -45 +45 @@
-pleased)
+pleased#please_v)
@@ -48 +48 @@
-Almighty)
+Almighty#almighty_adj)
@@ -50 +50 @@
-God))
+God#NA))
@@ -54 +54 @@
-about)
+about#about_adv)
@@ -56 +56 @@
-ten))
+ten#ten_adj))
@@ -59 +59 @@
-of)
+of#of_prep)
@@ -62 +62 @@
-the)
+the#the_adj)
@@ -64 +64 @@
-Clock)))
+Clock#clock_n1)))
@@ -67 +67 @@
-this)
+this#this_pron)
@@ -69 +69 @@
-morning)))
+morning#morning_n)))
@@ -73 +73 @@
-to)
+to#to_prep)
@@ -75 +75 @@
-blesse)
+blesse#bless_v1)
@@ -79 +79 @@
-His)
+His#his_adj)
@@ -81 +81 @@
-Majesty))
+Majesty#majesty_n))
@@ -84 +84 @@
-and)
+and#and_conj1)
@@ -87 +87 @@
-his)
+his#his_adj)
@@ -89 +89 @@
-royall)
+royall#royal_adj)
@@ -91 +91 @@
-Consort)
+Consort#consort_n1)
@@ -94 +94 @@
-the)
+the#the_adj)
@@ -96 +96 @@
-Queene)))))
+Queene#queen_n)))))
@@ -99 +99 @@
-with)
+with#with_prep)
@@ -102 +102 @@
-the)
+the#the_adj)
@@ -104 +104 @@
-birth)
+birth#birth_n1)
@@ -107 +107 @@
-of)
+of#of_prep)
@@ -110 +110 @@
-a)
+a#a_adj2)
@@ -112 +112 @@
-hopefull)
+hopefull#hopeful_adj)
@@ -114 +114 @@
-Son))))))
+Son#son_n1))))))
@@ -119 +119 @@
-and)
+and#and_conj1)
@@ -124 +124 @@
-his)
+his#his_adj)
@@ -126 +126 @@
-Majesty's))
+Majesty's#majesty_n))
@@ -128 +128 @@
-Kingdomes)
+Kingdomes#kingdom_n)
@@ -131 +131 @@
-and)
+and#and_conj1)
@@ -134 +134 @@
-Dominions))))
+Dominions#dominion_n))))
@@ -137 +137 @@
-with)
+with#with_prep)
@@ -140 +140 @@
-a)
+a#a_adj2)
@@ -142 +142 @@
-Prince)))))))
+Prince#prince_n)))))))
@@ -147 +147 @@
-Wee))
+Wee#we_pron))
@@ -149 +149 @@
-doe)
+doe#do_v)
@@ -152 +152 @@
-by)
+by#by_prep)
@@ -156 +156 @@
-His)
+His#his_adj)
@@ -158 +158 @@
-Majesty's))
+Majesty's#majesty_n))
@@ -160 +160 @@
-command)))
+command#command_n)))
@@ -163 +163 @@
-hereby))
+hereby#hereby_adv))
@@ -165 +165 @@
-signify)
+signify#signify_v)
@@ -168 +168 @@
-the)
+the#the_adj)
@@ -170 +170 @@
-same))
+same#same_adj))
@@ -173 +173 @@
-unto)
+unto#unto_prep)
@@ -176 +176 @@
-your)
+your#your_pron)
@@ -178 +178 @@
-Lordship)))
+Lordship#lordship_n)))
@@ -183 +183 @@
-desiring)
+desiring#desire_v)
@@ -186 +186 @@
-that)
+that#that_conj)
@@ -190 +190 @@
-it))
+it#it_pron))
@@ -192 +192 @@
-be)
+be#be_v)
@@ -195 +195 @@
-likewise))
+likewise#likewise_adv))
@@ -198 +198 @@
-forthwith))
+forthwith#forthwith_adv))
@@ -200 +200 @@
-communicated)
+communicated#communicate_v)
@@ -203 +203 @@
-by)
+by#by_prep)
@@ -206 +206 @@
-You)))
+You#you_pron)))
@@ -209 +209 @@
-to)
+to#to_prep)
@@ -215 +215 @@
-your)
+your#your_pron)
@@ -217 +217 @@
-Deputy)
+Deputy#deputy_n)
@@ -219 +219 @@
-Lieutenants))
+Lieutenants#lieutenant_n))
@@ -225 +225 @@
-the)
+the#the_adj)
@@ -227 +227 @@
-Justices)
+Justices#justice_n)
@@ -230 +230 @@
-of)
+of#of_prep)
@@ -233 +233 @@
-Peace)))))
+Peace#peace_n)))))
@@ -238 +238 @@
-and)
+and#and_conj1)
@@ -241 +241 @@
-the)
+the#the_adj)
@@ -243 +243 @@
-severall)
+severall#several_adj)
@@ -245 +245 @@
-Corporations)
+Corporations#corporation_n)
@@ -248 +248 @@
-within)
+within#within_adv)
@@ -251 +251 @@
-your)
+your#your_pron)
@@ -253 +253 @@
-Lieutenancy)))))))
+Lieutenancy#lieutenancy_n)))))))
@@ -258 +258 @@
-to)
+to#to_prep)
@@ -261 +261 @@
-the)
+the#the_adj)
@@ -263 +263 @@
-end)
+end#end_n)
@@ -270 +270 @@
-they))
+they#they_pron))
@@ -272 +272 @@
-may)
+may#may_v1)
@@ -274 +274 @@
-all)
+all#all_adj)
@@ -276 +276 @@
-joyne)
+joyne#join_v1)
@@ -281 +281 @@
-at)
+at#at_prep)
@@ -284 +284 @@
-such)
+such#such_adj)
@@ -286 +286 @@
-time)
+time#time_n)
@@ -289 +289 @@
-as)
+as#as_adv)
@@ -298 +298 @@
-his)
+his#his_adj)
@@ -300 +300 @@
-Majesty))
+Majesty#majesty_n))
@@ -302 +302 @@
-shall)
+shall#shall_v)
@@ -304 +304 @@
-please)
+please#please_v)
@@ -309 +309 @@
-to)
+to#to_prep)
@@ -311 +311 @@
-appoint)
+appoint#appoint_v)
@@ -314 +314 @@
-by)
+by#by_prep)
@@ -317 +317 @@
-his)
+his#his_adj)
@@ -319 +319 @@
-royall)
+royall#royal_adj)
@@ -321 +321 @@
-Proclamation)))
+Proclamation#proclamation_n)))
@@ -324 +324 @@
-for)
+for#for_prep)
@@ -327 +327 @@
-that)
+that#that_pron)
@@ -329 +329 @@
-purpose)))))))))
+purpose#purpose_n)))))))))
@@ -335 +335 @@
-as)
+as#as_adv)
@@ -337 +337 @@
-well)
+well#well_adv)
@@ -342 +342 @@
-in)
+in#in_prep)
@@ -345 +345 @@
-solemne)
+solemne#solemn_adj)
@@ -347 +347 @@
-Thanksgiving)
+Thanksgiving#thanksgiving_n)
@@ -350 +350 @@
-to)
+to#to_prep)
@@ -353 +353 @@
-Almighty)
+Almighty#almighty_adj)
@@ -355 +355 @@
-God)))
+God#NA)))
@@ -360 +360 @@
-for)
+for#for_prep)
@@ -364 +364 @@
-so)
+so#so_adv)
@@ -366 +366 @@
-inestimable))
+inestimable#inestimable_adj))
@@ -368 +368 @@
-a)
+a#a_adj2)
@@ -370 +370 @@
-blessing)))))
+blessing#blessing_n)))))
@@ -375 +375 @@
-as)
+as#as_adv)
@@ -378 +378 @@
-in)
+in#in_prep)
@@ -381 +381 @@
-such)
+such#such_adj)
@@ -383 +383 @@
-other)
+other#other_adj)
@@ -385 +385 @@
-expressions)
+expressions#expression_n)
@@ -388 +388 @@
-of)
+of#of_prep)
@@ -391 +391 @@
-publique)
+publique#public_adj)
@@ -393 +393 @@
-rejoyceing)))
+rejoyceing#rejoicing_n)))
@@ -398 +398 @@
-as)
+as#as_adv)
@@ -408 +408 @@
-are)
+are#be_v)
@@ -411 +411 @@
-suitable)
+suitable#suitable_adj)
@@ -413 +413 @@
-and)
+and#and_conj1)
@@ -415 +415 @@
-accustomed))
+accustomed#accustomed_adj))
@@ -418 +418 @@
-on)
+on#on_prep)
@@ -422 +422 @@
-so)
+so#so_adv)
@@ -424 +424 @@
-great))
+great#great_adj))
@@ -426 +426 @@
-an)
+an#a_adj2)
@@ -428 +428 @@
-occasion)))))))))))))))))
+occasion#occasion_n1)))))))))))))))))
@@ -436 +436 @@
-And)
+And#and_conj1)
@@ -439 +439 @@
-so))
+so#so_adv))
@@ -442 +442 @@
-wee))
+wee#we_pron))
@@ -444 +444 @@
-bid)
+bid#bid_v)
@@ -447 +447 @@
-your)
+your#your_pron)
@@ -449 +449 @@
-Lordship))
+Lordship#lordship_n))
@@ -452 +452 @@
-very)
+very#very_adj)
@@ -454 +454 @@
-heartely))
+heartely#heartily_adv))
@@ -457 +457 @@
-farewell))
+farewell#farewell_int))
@@ -465 +465 @@
-From)
+From#from_prep)
@@ -468 +468 @@
-the)
+the#the_adj)
@@ -470 +470 @@
-Council)
+Council#council_n)
@@ -472 +472 @@
-Chamber)
+Chamber#chamber_n)
@@ -475 +475 @@
-in)
+in#in_prep)
@@ -478 +478 @@
-Whitehall)))))
+Whitehall#NA)))))
@@ -484 +484 @@
-this)
+this#this_pron)
@@ -486 +486 @@
-10=th=)
+10=th=#NA)
@@ -489 +489 @@
-of)
+of#of_prep)
@@ -492 +492 @@
-June)))
+June#NA)))
@@ -494 +494 @@
-1688)
+1688#NA)
@@ -503 +503 @@
-Yo=r=)
+Yo=r=#your_pron)
@@ -505 +505 @@
-Lo=ps=))
+Lo=ps=#lordship_n))
@@ -508 +508 @@
-very)
+very#very_adj)
@@ -510 +510 @@
-loving))
+loving#loving_adj))
@@ -512 +512 @@
-friends)
+friends#friend_n)
@@ -516 +516 @@
-Jeffreys))
+Jeffreys#NA))
@@ -522 +522 @@
-C.)
+C.#NA)
@@ -524 +524 @@
-Sunderland)))
+Sunderland#NA)))
@@ -528 +528 @@
-P.)
+P.#NA)
@@ -530 +530 @@
-Bathe)))
+Bathe#NA)))
@@ -536 +536 @@
-Powis)))
+Powis#NA)))
@@ -542 +542 @@
-Middleton)))
+Middleton#NA)))
@@ -548 +548 @@
-Crauen)))
+Crauen#NA)))
@@ -554 +554 @@
-Castlemaine)))
+Castlemaine#NA)))
@@ -560 +560 @@
-Douer)))
+Douer#NA)))
@@ -566 +566 @@
-Dartmouth)))
+Dartmouth#NA)))
@@ -572 +572 @@
-J.)
+J.#NA)
@@ -574 +574 @@
-Ernle)))
+Ernle#NA)))
@@ -580 +580 @@
-John)
+John#NA)
@@ -582 +582 @@
-Nicholas))))
+Nicholas#NA))))
@@ -590 +590 @@
-To)
+To#to_prep)
@@ -593 +593 @@
-our)
+our#our_pron)
@@ -596 +596 @@
-very)
+very#very_adj)
@@ -598 +598 @@
-good))
+good#good_adj))
@@ -600 +600 @@
-Lord)
+Lord#lord_n)
@@ -603 +603 @@
-Laurence)
+Laurence#NA)
@@ -606 +606 @@
-Earle)
+Earle#earl_n)
@@ -609 +609 @@
-of)
+of#of_prep)
@@ -612 +612 @@
-Rochester))))
+Rochester#NA))))
@@ -615 +615 @@
-Lord)
+Lord#lord_n)
@@ -617 +617 @@
-Lieutenant)
+Lieutenant#lieutenant_n)
@@ -620 +620 @@
-of)
+of#of_prep)
@@ -623 +623 @@
-the)
+the#the_adj)
@@ -625 +625 @@
-County)
+County#county_n1)
@@ -628 +628 @@
-of)
+of#of_prep)
@@ -631 +631 @@
-Hartford))))))))
+Hartford#NA))))))))
