--- cromwell-1534-e1-h.psd.vert	2025-09-10 13:50:35
+++ cromwell-1534-e1-h.lem.vert	2025-09-10 13:50:45
@@ -16 +16 @@
-After)
+After#after_adv)
@@ -19 +19 @@
-my)
+my#my_adj)
@@ -22 +22 @@
-right)
+right#right_adv)
@@ -24 +24 @@
-hartie))
+hartie#hearty_adj))
@@ -26 +26 @@
-commendations))
+commendations#commendation_n))
@@ -35 +35 @@
-Whereas)
+Whereas#whereas_adv)
@@ -43 +43 @@
-the)
+the#the_adj)
@@ -45 +45 @@
-Kings))
+Kings#king_n))
@@ -47 +47 @@
-Maiestie))
+Maiestie#majesty_n))
@@ -53 +53 @@
-about)
+about#about_adv)
@@ -55 +55 @@
-a)
+a#a_adj2)
@@ -57 +57 @@
-twelfmoneth))
+twelfmoneth#twelvemonth_n))
@@ -59 +59 @@
-past))
+past#past_prep))
@@ -63 +63 @@
-gave)
+gave#give_v)
@@ -66 +66 @@
-a)
+a#a_adj2)
@@ -68 +68 @@
-pardonne)
+pardonne#pardon_n1)
@@ -71 +71 @@
-to)
+to#to_prep)
@@ -74 +74 @@
-a)
+a#a_adj2)
@@ -76 +76 @@
-company)
+company#company_n)
@@ -79 +79 @@
-of)
+of#of_prep)
@@ -82 +82 @@
-lewde)
+lewde#lewd_adj)
@@ -84 +84 @@
-personnes)
+personnes#person_n)
@@ -87 +87 @@
-within)
+within#within_adv)
@@ -90 +90 @@
-this)
+this#this_pron)
@@ -92 +92 @@
-realme)))
+realme#realm_n)))
@@ -95 +95 @@
-calling)
+calling#call_v)
@@ -99 +99 @@
-themselves))
+themselves#themselves_pron))
@@ -102 +102 @@
-Gipcyans))))))))
+Gipcyans#NA))))))))
@@ -107 +107 @@
-for)
+for#for_prep)
@@ -110 +110 @@
-a)
+a#a_adj2)
@@ -113 +113 @@
-most)
+most#most_adj)
@@ -115 +115 @@
-shamfull)
+shamfull#shameful_adj)
@@ -118 +118 @@
-and)
+and#and_conj1)
@@ -121 +121 @@
-detestable))))
+detestable#detestable_adj))))
@@ -123 +123 @@
-murder)
+murder#murder_n)
@@ -126 +126 @@
-commytted)
+commytted#commit_v)
@@ -129 +129 @@
-amongs)
+amongs#amongst_prep)
@@ -132 +132 @@
-them))))))
+them#them_pron))))))
@@ -137 +137 @@
-with)
+with#with_prep)
@@ -140 +140 @@
-a)
+a#a_adj2)
@@ -142 +142 @@
-speceall)
+speceall#special_adj)
@@ -144 +144 @@
-proviso)
+proviso#proviso_conj)
@@ -147 +147 @@
-inserted)
+inserted#insert_v)
@@ -150 +150 @@
-by)
+by#by_prep)
@@ -153 +153 @@
-their)
+their#their_adj)
@@ -155 +155 @@
-owne)
+owne#own_adj)
@@ -157 +157 @@
-consents))))
+consents#consent_n))))
@@ -162 +162 @@
-that)
+that#that_conj)
@@ -166 +166 @@
-onles)
+onles#unless_adv)
@@ -173 +173 @@
-they))
+they#they_pron))
@@ -175 +175 @@
-shuld)
+shuld#shall_v)
@@ -177 +177 @@
-all)
+all#all_adj)
@@ -179 +179 @@
-avoyde)
+avoyde#avoid_v)
@@ -182 +182 @@
-this)
+this#this_pron)
@@ -185 +185 @@
-his)
+his#his_adj)
@@ -187 +187 @@
-Graces))
+Graces#grace_n))
@@ -189 +189 @@
-realme))
+realme#realm_n))
@@ -192 +192 @@
-by)
+by#by_prep)
@@ -195 +195 @@
-a)
+a#a_adj2)
@@ -197 +197 @@
-certeyn)
+certeyn#certain_adj)
@@ -199 +199 @@
-daye)
+daye#day_n)
@@ -203 +203 @@
-sythens))
+sythens#sithence_adv))
@@ -205 +205 @@
-expired)))))))
+expired#expire_v)))))))
@@ -210 +210 @@
-yt))
+yt#it_pron))
@@ -214 +214 @@
-shuld)
+shuld#shall_v)
@@ -216 +216 @@
-be)
+be#be_v)
@@ -219 +219 @@
-lawfull)
+lawfull#lawful_adj)
@@ -222 +222 @@
-to)
+to#to_prep)
@@ -225 +225 @@
-all)
+all#all_adj)
@@ -228 +228 @@
-his)
+his#his_adj)
@@ -230 +230 @@
-Graces))
+Graces#grace_n))
@@ -232 +232 @@
-offycers))))
+offycers#officer_n))))
@@ -235 +235 @@
-to)
+to#to_prep)
@@ -237 +237 @@
-hang)
+hang#hang_v)
@@ -240 +240 @@
-them))
+them#them_pron))
@@ -243 +243 @@
-in)
+in#in_prep)
@@ -246 +246 @@
-all)
+all#all_adj)
@@ -248 +248 @@
-places)
+places#place_n.II-VII)
@@ -251 +251 @@
-of)
+of#of_prep)
@@ -254 +254 @@
-his)
+his#his_adj)
@@ -256 +256 @@
-realme)))
+realme#realm_n)))
@@ -262 +262 @@
-where))
+where#where_adv))
@@ -270 +270 @@
-they))
+they#they_pron))
@@ -272 +272 @@
-myght)
+myght#may_v1)
@@ -274 +274 @@
-be)
+be#be_v)
@@ -276 +276 @@
-apprehended)))))
+apprehended#apprehend_v)))))
@@ -281 +281 @@
-without)
+without#without_adv)
@@ -284 +284 @@
-any)
+any#any_adj)
@@ -286 +286 @@
-further)
+further#further_adj)
@@ -288 +288 @@
-examynacion)
+examynacion#examination_n)
@@ -291 +291 @@
-or)
+or#or_conj2)
@@ -294 +294 @@
-tryal)))
+tryal#trial_n1)))
@@ -297 +297 @@
-after)
+after#after_adv)
@@ -300 +300 @@
-forme)
+forme#form_n)
@@ -303 +303 @@
-of)
+of#of_prep)
@@ -306 +306 @@
-the)
+the#the_adj)
@@ -308 +308 @@
-lawe)))))))
+lawe#law_n1)))))))
@@ -313 +313 @@
-as)
+as#as_adv)
@@ -324 +324 @@
-in)
+in#in_prep)
@@ -327 +327 @@
-their)
+their#their_adj)
@@ -329 +329 @@
-letter)
+letter#letter_n1)
@@ -332 +332 @@
-patents))
+patents#patent_adj))
@@ -335 +335 @@
-of)
+of#of_prep)
@@ -338 +338 @@
-the)
+the#the_adj)
@@ -340 +340 @@
-said)
+said#said_adj)
@@ -342 +342 @@
-pardon)))))
+pardon#pardon_n1)))))
@@ -344 +344 @@
-is)
+is#be_v)
@@ -346 +346 @@
-expressed)))))))))))))
+expressed#express_v1)))))))))))))
@@ -351 +351 @@
-His)
+His#his_adj)
@@ -353 +353 @@
-Grace))
+Grace#grace_n))
@@ -358 +358 @@
-hering)
+hering#hear_v)
@@ -363 +363 @@
-tell)
+tell#tell_v)
@@ -367 +367 @@
-that)
+that#that_conj)
@@ -371 +371 @@
-they))
+they#they_pron))
@@ -373 +373 @@
-doo)
+doo#do_v)
@@ -376 +376 @@
-yet))
+yet#yet_adv))
@@ -378 +378 @@
-lynger)
+lynger#linger_v)
@@ -381 +381 @@
-here))
+here#here_adv))
@@ -384 +384 @@
-within)
+within#within_adv)
@@ -387 +387 @@
-his)
+his#his_adj)
@@ -389 +389 @@
-realme)))
+realme#realm_n)))
@@ -394 +394 @@
-not)
+not#not_adv)
@@ -396 +396 @@
-avoyding)
+avoyding#avoid_v)
@@ -399 +399 @@
-the)
+the#the_adj)
@@ -401 +401 @@
-same))
+same#same_adj))
@@ -404 +404 @@
-according)
+according#accord_v)
@@ -407 +407 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-his)
+his#his_adj)
@@ -413 +413 @@
-commaundement))
+commaundement#commandment_n))
@@ -416 +416 @@
-and)
+and#and_conj1)
@@ -419 +419 @@
-their)
+their#their_adj)
@@ -421 +421 @@
-owne)
+owne#own_adj)
@@ -423 +423 @@
-promes)))))))))
+promes#promise_n)))))))))
@@ -428 +428 @@
-and)
+and#and_conj1)
@@ -431 +431 @@
-that)
+that#that_conj)
@@ -435 +435 @@
-albeit)
+albeit#albeit_conj)
@@ -442 +442 @@
-his)
+his#his_adj)
@@ -444 +444 @@
-poore)
+poore#poor_adj)
@@ -446 +446 @@
-subjectes))
+subjectes#subject_n))
@@ -448 +448 @@
-be)
+be#be_v)
@@ -451 +451 @@
-dayly))
+dayly#daily_adv))
@@ -454 +454 @@
-spoyled)
+spoyled#spoil_v1)
@@ -458 +458 @@
-robbed)
+robbed#rob_v)
@@ -462 +462 @@
-and)
+and#and_conj1)
@@ -464 +464 @@
-deceyved))
+deceyved#deceive_v))
@@ -467 +467 @@
-by)
+by#by_prep)
@@ -470 +470 @@
-them))))))
+them#them_pron))))))
@@ -475 +475 @@
-yet))
+yet#yet_adv))
@@ -479 +479 @@
-his)
+his#his_adj)
@@ -481 +481 @@
-Highnes))
+Highnes#highness_n))
@@ -483 +483 @@
-officers)
+officers#officer_n)
@@ -486 +486 @@
-and)
+and#and_conj1)
@@ -489 +489 @@
-Ministres))))
+Ministres#minister_n))))
@@ -493 +493 @@
-lytle))
+lytle#little_adj))
@@ -495 +495 @@
-regarding)
+regarding#regard_v)
@@ -498 +498 @@
-their)
+their#their_adj)
@@ -500 +500 @@
-dieuties)
+dieuties#duty_n)
@@ -503 +503 @@
-towards)
+towards#towards_prep)
@@ -506 +506 @@
-his)
+his#his_adj)
@@ -508 +508 @@
-Majestye)))))
+Majestye#majesty_n)))))
@@ -512 +512 @@
-do)
+do#do_v)
@@ -514 +514 @@
-permyt)
+permyt#permit_v)
@@ -517 +517 @@
-them))
+them#them_pron))
@@ -521 +521 @@
-to)
+to#to_prep)
@@ -524 +524 @@
-lynger)
+lynger#linger_v)
@@ -526 +526 @@
-and)
+and#and_conj1)
@@ -528 +528 @@
-loyter))
+loyter#loiter_v))
@@ -531 +531 @@
-in)
+in#in_prep)
@@ -534 +534 @@
-all)
+all#all_adj)
@@ -536 +536 @@
-partys))))
+partys#party_n))))
@@ -541 +541 @@
-and)
+and#and_conj1)
@@ -544 +544 @@
-to)
+to#to_prep)
@@ -546 +546 @@
-exercise)
+exercise#exercise_v)
@@ -549 +549 @@
-all)
+all#all_adj)
@@ -551 +551 @@
-their)
+their#their_adj)
@@ -553 +553 @@
-falshods)
+falshods#falsehood_n)
@@ -559 +559 @@
-felonyes)))
+felonyes#felony_n1)))
@@ -564 +564 @@
-and)
+and#and_conj1)
@@ -567 +567 @@
-treasons))))
+treasons#treason_n))))
@@ -570 +570 @@
-unpunished)))))))))))
+unpunished#unpunished_adj)))))))))))
@@ -574 +574 @@
-hathe)
+hathe#have_v)
@@ -576 +576 @@
-commaunded)
+commaunded#command_v)
@@ -579 +579 @@
-me))
+me#me_pron1))
@@ -582 +582 @@
-to)
+to#to_prep)
@@ -584 +584 @@
-sygnifye)
+sygnifye#signify_v)
@@ -587 +587 @@
-unto)
+unto#unto_prep)
@@ -591 +591 @@
-youe))
+youe#you_pron))
@@ -596 +596 @@
-and)
+and#and_conj1)
@@ -599 +599 @@
-the)
+the#the_adj)
@@ -601 +601 @@
-Shires)
+Shires#shire_n)
@@ -605 +605 @@
-next))
+next#next_adj))
@@ -607 +607 @@
-adjoynyng))))))
+adjoynyng#adjoin_v))))))
@@ -612 +612 @@
-whether)
+whether#whether_pron)
@@ -618 +618 @@
-any)
+any#any_adj)
@@ -621 +621 @@
-of)
+of#of_prep)
@@ -624 +624 @@
-the)
+the#the_adj)
@@ -626 +626 @@
-sayd)
+sayd#said_adj)
@@ -628 +628 @@
-personnes)
+personnes#person_n)
@@ -632 +632 @@
-calling)
+calling#call_v)
@@ -636 +636 @@
-themselfes))
+themselfes#themselves_pron))
@@ -639 +639 @@
-Egipcyans))))
+Egipcyans#NA))))
@@ -644 +644 @@
-or)
+or#or_conj2)
@@ -649 +649 @@
-that)
+that#that_conj)
@@ -654 +654 @@
-hathe)
+hathe#have_v)
@@ -657 +657 @@
-heretofore))
+heretofore#heretofore_adv))
@@ -659 +659 @@
-called)
+called#call_v)
@@ -663 +663 @@
-themselfes))
+themselfes#themselves_pron))
@@ -666 +666 @@
-Egipcyans))))))))))
+Egipcyans#NA))))))))))
@@ -670 +670 @@
-shall)
+shall#shall_v)
@@ -672 +672 @@
-fortune)
+fortune#fortune_v)
@@ -675 +675 @@
-to)
+to#to_prep)
@@ -678 +678 @@
-enter)
+enter#enter_v)
@@ -680 +680 @@
-or)
+or#or_conj2)
@@ -682 +682 @@
-travayle))
+travayle#travel_v))
@@ -685 +685 @@
-in)
+in#in_prep)
@@ -688 +688 @@
-the)
+the#the_adj)
@@ -690 +690 @@
-same)))))))
+same#same_adj)))))))
@@ -698 +698 @@
-And)
+And#and_conj1)
@@ -701 +701 @@
-in)
+in#in_prep)
@@ -704 +704 @@
-cace)
+cace#case_n1)
@@ -711 +711 @@
-youe))
+youe#you_pron))
@@ -713 +713 @@
-shall)
+shall#shall_v)
@@ -716 +716 @@
-here)
+here#hear_v)
@@ -718 +718 @@
-or)
+or#or_conj2)
@@ -720 +720 @@
-knowe))
+knowe#know_v))
@@ -723 +723 @@
-of)
+of#of_prep)
@@ -726 +726 @@
-any)
+any#any_adj)
@@ -728 +728 @@
-suche)))
+suche#such_adj)))
@@ -734 +734 @@
-be)
+be#be_v)
@@ -737 +737 @@
-they))
+they#they_pron))
@@ -740 +740 @@
-men)
+men#man_n1)
@@ -742 +742 @@
-or)
+or#or_conj2)
@@ -744 +744 @@
-women))))))))
+women#woman_n))))))))
@@ -749 +749 @@
-that)
+that#that_conj)
@@ -753 +753 @@
-ye))
+ye#ye_pron))
@@ -755 +755 @@
-shall)
+shall#shall_v)
@@ -757 +757 @@
-compell)
+compell#compel_v)
@@ -760 +760 @@
-them))
+them#them_pron))
@@ -764 +764 @@
-to)
+to#to_prep)
@@ -766 +766 @@
-depart)
+depart#depart_v)
@@ -769 +769 @@
-to)
+to#to_prep)
@@ -772 +772 @@
-the)
+the#the_adj)
@@ -774 +774 @@
-next)
+next#next_adj)
@@ -776 +776 @@
-porte)
+porte#port_n1)
@@ -779 +779 @@
-of)
+of#of_prep)
@@ -782 +782 @@
-the)
+the#the_adj)
@@ -784 +784 @@
-See)))
+See#sea_n)))
@@ -787 +787 @@
-to)
+to#to_prep)
@@ -790 +790 @@
-the)
+the#the_adj)
@@ -792 +792 @@
-place)
+place#place_n.II-VII)
@@ -796 +796 @@
-where))
+where#where_adv))
@@ -804 +804 @@
-they))
+they#they_pron))
@@ -806 +806 @@
-shal@)
+shal@#shall_v)
@@ -808 +808 @@
-@be)
+@be#be_v)
@@ -810 +810 @@
-taken))))))))
+taken#take_v))))))))
@@ -815 +815 @@
-and)
+and#and_conj1)
@@ -818 +818 @@
-eyther)
+eyther#either_pron)
@@ -823 +823 @@
-wythout)
+wythout#without_adv)
@@ -826 +826 @@
-delaye)))
+delaye#delay_n)))
@@ -829 +829 @@
-uppon)
+uppon#upon_prep)
@@ -832 +832 @@
-the)
+the#the_adj)
@@ -834 +834 @@
-first)
+first#first_adj)
@@ -836 +836 @@
-wynde)
+wynde#wind_n1)
@@ -841 +841 @@
-that)
+that#that_conj)
@@ -846 +846 @@
-may)
+may#may_v1)
@@ -848 +848 @@
-conveye)
+conveye#convey_v1)
@@ -851 +851 @@
-them))
+them#them_pron))
@@ -854 +854 @@
-into)
+into#into_prep)
@@ -857 +857 @@
-any)
+any#any_adj)
@@ -859 +859 @@
-parte)
+parte#part_n)
@@ -862 +862 @@
-of)
+of#of_prep)
@@ -865 +865 @@
-beyond)
+beyond#beyond_adv)
@@ -868 +868 @@
-the)
+the#the_adj)
@@ -870 +870 @@
-Sees))))))))))
+Sees#sea_n))))))))))
@@ -874 +874 @@
-to)
+to#to_prep)
@@ -876 +876 @@
-take)
+take#take_v)
@@ -879 +879 @@
-shipping)))
+shipping#shipping_n1)))
@@ -882 +882 @@
-and)
+and#and_conj1)
@@ -885 +885 @@
-to)
+to#to_prep)
@@ -887 +887 @@
-passe)
+passe#pass_v)
@@ -890 +890 @@
-in_to)
+in_to#into_prep)
@@ -893 +893 @@
-owtward)
+owtward#outward_adj)
@@ -895 +895 @@
-partyes))))))
+partyes#party_n))))))
@@ -900 +900 @@
-or)
+or#or_conj2)
@@ -904 +904 @@
-if)
+if#if_conj)
@@ -911 +911 @@
-they))
+they#they_pron))
@@ -913 +913 @@
-shall)
+shall#shall_v)
@@ -916 +916 @@
-in)
+in#in_prep)
@@ -919 +919 @@
-any)
+any#any_adj)
@@ -921 +921 @@
-wise)))
+wise#wise_n1)))
@@ -923 +923 @@
-breke)
+breke#break_v)
@@ -926 +926 @@
-that)
+that#that_pron)
@@ -928 +928 @@
-commaundement)))))
+commaundement#commandment_n)))))
@@ -933 +933 @@
-without)
+without#without_adv)
@@ -936 +936 @@
-any)
+any#any_adj)
@@ -938 +938 @@
-tract)))
+tract#tract_n3)))
@@ -940 +940 @@
-to)
+to#to_prep)
@@ -942 +942 @@
-see)
+see#see_v)
@@ -946 +946 @@
-them))
+them#them_pron))
@@ -948 +948 @@
-executed)
+executed#execute_v)
@@ -951 +951 @@
-according)
+according#accord_v)
@@ -954 +954 @@
-to)
+to#to_prep)
@@ -959 +959 @@
-the)
+the#the_adj)
@@ -961 +961 @@
-Kings))
+Kings#king_n))
@@ -963 +963 @@
-Hieghnes))
+Hieghnes#highness_n))
@@ -965 +965 @@
-sayd)
+sayd#said_adj)
@@ -967 +967 @@
-Lettres)
+Lettres#letter_n1)
@@ -970 +970 @@
-patents))
+patents#patent_adj))
@@ -973 +973 @@
-remaynyng)
+remaynyng#remain_v)
@@ -976 +976 @@
-of)
+of#of_prep)
@@ -979 +979 @@
-Recorde)))
+Recorde#record_n)))
@@ -982 +982 @@
-in)
+in#in_prep)
@@ -985 +985 @@
-his)
+his#his_adj)
@@ -987 +987 @@
-Chauncery))))
+Chauncery#chancery_n))))
@@ -991 +991 @@
-which))
+which#which_adj))
@@ -1001 +1001 @@
-with)
+with#with_prep)
@@ -1004 +1004 @@
-these)))
+these#these_pron)))
@@ -1008 +1008 @@
-shal@)
+shal@#shall_v)
@@ -1010 +1010 @@
-@be)
+@be#be_v)
@@ -1015 +1015 @@
-your)
+your#your_pron)
@@ -1017 +1017 @@
-discharge)
+discharge#discharge_n)
@@ -1020 +1020 @@
-in)
+in#in_prep)
@@ -1023 +1023 @@
-that)
+that#that_pron)
@@ -1025 +1025 @@
-behaulf)))))))))))))))
+behaulf#behalf_n)))))))))))))))
@@ -1030 +1030 @@
-not)
+not#not_adv)
@@ -1032 +1032 @@
-fayling)
+fayling#fail_v)
@@ -1035 +1035 @@
-t'@)
+t'@#to_prep)
@@ -1037 +1037 @@
-@accomplishe)
+@accomplishe#accomplish_v)
@@ -1040 +1040 @@
-the)
+the#the_adj)
@@ -1042 +1042 @@
-tenor)
+tenor#tenor_n1)
@@ -1045 +1045 @@
-hereof)))
+hereof#hereof_adv)))
@@ -1048 +1048 @@
-with)
+with#with_prep)
@@ -1051 +1051 @@
-all)
+all#all_adj)
@@ -1053 +1053 @@
-effect)
+effect#effect_n)
@@ -1056 +1056 @@
-and)
+and#and_conj1)
@@ -1059 +1059 @@
-diligence)))))
+diligence#diligence_n1)))))
@@ -1064 +1064 @@
-without)
+without#without_adv)
@@ -1067 +1067 @@
-sparing)
+sparing#spare_v1)
@@ -1070 +1070 @@
-uppon)
+uppon#upon_prep)
@@ -1073 +1073 @@
-any)
+any#any_adj)
@@ -1075 +1075 @@
-Commyssion)
+Commyssion#commission_n1)
@@ -1081 +1081 @@
-Licence)))
+Licence#licence_n)))
@@ -1086 +1086 @@
-or)
+or#or_conj2)
@@ -1089 +1089 @@
-Placarde)))
+Placarde#placard_n)))
@@ -1094 +1094 @@
-that)
+that#that_conj)
@@ -1100 +1100 @@
-they))
+they#they_pron))
@@ -1102 +1102 @@
-may)
+may#may_v1)
@@ -1105 +1105 @@
-shewe)
+shewe#show_v)
@@ -1107 +1107 @@
-or)
+or#or_conj2)
@@ -1109 +1109 @@
-aledge))
+aledge#allege_v2))
@@ -1112 +1112 @@
-for)
+for#for_prep)
@@ -1115 +1115 @@
-themselfes)))
+themselfes#themselves_pron)))
@@ -1118 +1118 @@
-to)
+to#to_prep)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-contrary)))))))))
+contrary#contrary_adj)))))))))
@@ -1128 +1128 @@
-as)
+as#as_adv)
@@ -1139 +1139 @@
-ye))
+ye#ye_pron))
@@ -1141 +1141 @@
-tender)
+tender#tender_v2)
@@ -1145,3 +1145,3 @@
-his)
-(N$
-Graces))
+his#his_adj)
+(N$
+Graces#grace_n))
@@ -1149 +1149 @@
-pleasor)
+pleasor#pleasure_n)
@@ -1153 +1153 @@
-which))
+which#which_adj))
@@ -1160 +1160 @@
-also)
+also#also_adv)
@@ -1162 +1162 @@
-ys)
+ys#be_v)
@@ -1166 +1166 @@
-that)
+that#that_conj)
@@ -1170 +1170 @@
-youe))
+youe#you_pron))
@@ -1172 +1172 @@
-shall)
+shall#shall_v)
@@ -1174 +1174 @@
-gyve)
+gyve#give_v)
@@ -1177 +1177 @@
-notyce))
+notyce#notice_n))
@@ -1180 +1180 @@
-to)
+to#to_prep)
@@ -1183 +1183 @@
-all)
+all#all_adj)
@@ -1185 +1185 @@
-the)
+the#the_adj)
@@ -1187 +1187 @@
-Justices)
+Justices#justice_n)
@@ -1190 +1190 @@
-of)
+of#of_prep)
@@ -1193 +1193 @@
-Peax)))
+Peax#peace_n)))
@@ -1196 +1196 @@
-in)
+in#in_prep)
@@ -1200 +1200 @@
-that)
+that#that_pron)
@@ -1202 +1202 @@
-Countye)
+Countye#county_n1)
@@ -1206 +1206 @@
-where))
+where#where_adv))
@@ -1214 +1214 @@
-youe))
+youe#you_pron))
@@ -1216 +1216 @@
-resyde))))
+resyde#reside_v1))))
@@ -1221 +1221 @@
-and)
+and#and_conj1)
@@ -1224 +1224 @@
-the)
+the#the_adj)
@@ -1226 +1226 @@
-Shires)
+Shires#shire_n)
@@ -1229 +1229 @@
-adjoynant))))))))
+adjoynant#adjoinant_adj))))))))
@@ -1234 +1234 @@
-that)
+that#that_conj)
@@ -1238 +1238 @@
-they))
+they#they_pron))
@@ -1240 +1240 @@
-may)
+may#may_v1)
@@ -1242 +1242 @@
-accomplishe)
+accomplishe#accomplish_v)
@@ -1245 +1245 @@
-the)
+the#the_adj)
@@ -1247 +1247 @@
-tenor)
+tenor#tenor_n1)
@@ -1250 +1250 @@
-hereof)))
+hereof#hereof_adv)))
@@ -1253 +1253 @@
-accordingly)))))))))))))))))
+accordingly#accordingly_adv)))))))))))))))))
@@ -1262 +1262 @@
-Thus))
+Thus#thus_adv))
@@ -1264 +1264 @@
-fare)
+fare#fare_v1)
@@ -1267 +1267 @@
-ye))
+ye#ye_pron))
@@ -1270 +1270 @@
-hertely)
+hertely#heartily_adv)
@@ -1272 +1272 @@
-wel))
+wel#well_adv))
@@ -1280 +1280 @@
-From)
+From#from_prep)
@@ -1283 +1283 @@
-the)
+the#the_adj)
@@ -1285 +1285 @@
-Neate)))
+Neate#NA)))
@@ -1291 +1291 @@
-the)
+the#the_adj)
@@ -1293 +1293 @@
-v=th=.)
+v=th=.#NA)
@@ -1295 +1295 @@
-day)
+day#day_n)
@@ -1298 +1298 @@
-of)
+of#of_prep)
@@ -1301 +1301 @@
-December))))
+December#NA))))
@@ -1307 +1307 @@
-the)
+the#the_adj)
@@ -1309 +1309 @@
-xxix=th=)
+xxix=th=#NA)
@@ -1311 +1311 @@
-yer)
+yer#year_n)
@@ -1314 +1314 @@
-of)
+of#of_prep)
@@ -1318 +1318 @@
-his)
+his#his_adj)
@@ -1320 +1320 @@
-Ma=ties=))
+Ma=ties=#majesty_n))
@@ -1323 +1323 @@
-most)
+most#most_adj)
@@ -1325 +1325 @@
-noble))
+noble#noble_adj))
@@ -1327 +1327 @@
-Regne))))
+Regne#reign_n))))
@@ -1333 +1333 @@
-Yo=r=)
+Yo=r=#your_pron)
@@ -1335 +1335 @@
-louyng)
+louyng#loving_adj)
@@ -1337 +1337 @@
-ffreend)
+ffreend#friend_n)
@@ -1340 +1340 @@
-Thomas)
+Thomas#NA)
@@ -1342 +1342 @@
-Crumwell))
+Crumwell#NA))
@@ -1350 +1350 @@
-To)
+To#to_prep)
@@ -1353 +1353 @@
-my)
+my#my_adj)
@@ -1356 +1356 @@
-verye)
+verye#very_adj)
@@ -1358 +1358 @@
-good))
+good#good_adj))
@@ -1360 +1360 @@
-Lorde)
+Lorde#lord_n)
@@ -1363 +1363 @@
-my)
+my#my_adj)
@@ -1365 +1365 @@
-Lorde)
+Lorde#lord_n)
@@ -1368 +1368 @@
-of)
+of#of_prep)
@@ -1371 +1371 @@
-Chestre))))
+Chestre#NA))))
@@ -1374 +1374 @@
-President)
+President#president_n)
@@ -1377 +1377 @@
-of)
+of#of_prep)
@@ -1380 +1380 @@
-the)
+the#the_adj)
@@ -1382 +1382 @@
-Marches)
+Marches#march_n3)
@@ -1385 +1385 @@
-of)
+of#of_prep)
@@ -1388 +1388 @@
-Wales)))))))
+Wales#NA)))))))
