--- ebeaum-1510-e1-h.psd.vert	2025-09-10 13:50:35
+++ ebeaum-1510-e1-h.lem.vert	2025-09-10 13:50:45
@@ -15 +15 @@
-Right)
+Right#right_adv)
@@ -17 +17 @@
-wyrschipfull))
+wyrschipfull#worshipful_adj))
@@ -19 +19 @@
-Ser))
+Ser#sir_n))
@@ -24 +24 @@
-In)
+In#in_prep)
@@ -27 +27 @@
-my)
+my#my_adj)
@@ -29 +29 @@
-best)
+best#best_adj)
@@ -31 +31 @@
-maner)
+maner#manner_n)
@@ -36 +36 @@
-that)
+that#that_conj)
@@ -42 +42 @@
-I))
+I#i_pron))
@@ -44 +44 @@
-cane)
+cane#can_v1)
@@ -49 +49 @@
-I))
+I#i_pron))
@@ -51 +51 @@
-recomend)
+recomend#recommend_v1)
@@ -54 +54 @@
-me))
+me#me_pron1))
@@ -57 +57 @@
-to)
+to#to_prep)
@@ -60 +60 @@
-you)))
+you#you_pron)))
@@ -65 +65 @@
-desyryng)
+desyryng#desire_v)
@@ -68 +68 @@
-hertly))
+hertly#heartily_adv))
@@ -71 +71 @@
-to)
+to#to_prep)
@@ -73 +73 @@
-here)
+here#hear_v)
@@ -76 +76 @@
-of)
+of#of_prep)
@@ -79 +79 @@
-your)
+your#your_pron)
@@ -81 +81 @@
-welefar)))))
+welefar#welfare_n)))))
@@ -90 +90 @@
-Ser))
+Ser#sir_n))
@@ -95 +95 @@
-I))
+I#i_pron))
@@ -97 +97 @@
-hafe)
+hafe#have_v)
@@ -99 +99 @@
-resayvyd)
+resayvyd#receive_v)
@@ -102 +102 @@
-your)
+your#your_pron)
@@ -104 +104 @@
-wrytyngs))
+wrytyngs#writing_n))
@@ -112 +112 @@
-and)
+and#and_conj1)
@@ -116 +116 @@
-persayvys)
+persayvys#perceive_v)
@@ -119 +119 @@
-ham))
+ham#them_pron))
@@ -122 +122 @@
-verey)
+verey#very_adj)
@@ -124 +124 @@
-wele))
+wele#well_adv))
@@ -132 +132 @@
-and)
+and#and_conj1)
@@ -134 +134 @@
-also)
+also#also_adv)
@@ -137 +137 @@
-I))
+I#i_pron))
@@ -139 +139 @@
-send)
+send#send_v1)
@@ -142 +142 @@
-George)
+George#NA)
@@ -145 +145 @@
-of)
+of#of_prep)
@@ -148 +148 @@
-Mytton))))
+Mytton#NA))))
@@ -151 +151 @@
-to)
+to#to_prep)
@@ -154 +154 @@
-the)
+the#the_adj)
@@ -156 +156 @@
-person)))
+person#person_n)))
@@ -159 +159 @@
-on)
+on#on_prep)
@@ -162 +162 @@
-the)
+the#the_adj)
@@ -164 +164 @@
-Thurseday)
+Thurseday#NA)
@@ -167 +167 @@
-afor)
+afor#afore_adv)
@@ -170 +170 @@
-All)
+All#NA)
@@ -172 +172 @@
-Halo)
+Halo#NA)
@@ -174 +174 @@
-day)))))
+day#NA)))))
@@ -182 +182 @@
-and)
+and#and_conj1)
@@ -185 +185 @@
-ther))
+ther#there_adv))
@@ -188 +188 @@
-the)
+the#the_adj)
@@ -192 +192 @@
-person))
+person#person_n))
@@ -194 +194 @@
-hase)
+hase#have_v)
@@ -196 +196 @@
-grauntyd)
+grauntyd#grant_v)
@@ -199 +199 @@
-that)
+that#that_conj)
@@ -203 +203 @@
-I))
+I#i_pron))
@@ -205 +205 @@
-sall)
+sall#shall_v)
@@ -207 +207 @@
-hafe)
+hafe#have_v)
@@ -210 +210 @@
-a)
+a#a_adj2)
@@ -214 +214 @@
-porcion)
+porcion#portion_n)
@@ -217 +217 @@
-both)
+both#both_pron)
@@ -220 +220 @@
-Eytys))
+Eytys#NA))
@@ -225 +225 @@
-and)
+and#and_conj1)
@@ -228 +228 @@
-the)
+the#the_adj)
@@ -230 +230 @@
-rentall)
+rentall#rental_n)
@@ -233 +233 @@
-of)
+of#of_prep)
@@ -236 +236 @@
-Hornby)))))))))
+Hornby#NA)))))))))
@@ -244 +244 @@
-Also)
+Also#also_adv)
@@ -247 +247 @@
-he))
+he#he_pron))
@@ -249 +249 @@
-has)
+has#have_v)
@@ -251 +251 @@
-poyntyd)
+poyntyd#point_v2)
@@ -254 +254 @@
-me))
+me#me_pron1))
@@ -257 +257 @@
-that)
+that#that_conj)
@@ -261,3 +261,3 @@
-I))
-(MD
-sall)
+I#i_pron))
+(MD
+sall#shall_v)
@@ -265 +265 @@
-not)
+not#not_adv)
@@ -267 +267 @@
-com)
+com#come_v)
@@ -270 +270 @@
-to)
+to#to_prep)
@@ -273 +273 @@
-hym)))
+hym#him_pron)))
@@ -276 +276 @@
-or)
+or#or_adv1)
@@ -279 +279 @@
-the)
+the#the_adj)
@@ -281 +281 @@
-morue)
+morue#morrow_n)
@@ -284 +284 @@
-after)
+after#after_adv)
@@ -287 +287 @@
-Martynmes)
+Martynmes#NA)
@@ -289 +289 @@
-day)))))))
+day#NA)))))))
@@ -297 +297 @@
-and)
+and#and_conj1)
@@ -300 +300 @@
-I))
+I#i_pron))
@@ -302 +302 @@
-cowde)
+cowde#can_v1)
@@ -304 +304 @@
-not)
+not#not_adv)
@@ -306 +306 @@
-cause)
+cause#cause_v1)
@@ -309 +309 @@
-hym))
+hym#him_pron))
@@ -312 +312 @@
-to)
+to#to_prep)
@@ -314 +314 @@
-poynt)
+poynt#point_v2)
@@ -317 +317 @@
-no)
+no#no_adj)
@@ -319 +319 @@
-soner)))
+soner#soon_adv)))
@@ -327 +327 @@
-for)
+for#for_prep)
@@ -330 +330 @@
-he))
+he#he_pron))
@@ -332 +332 @@
-sayd)
+sayd#say_v1)
@@ -339 +339 @@
-he))
+he#he_pron))
@@ -341 +341 @@
-cowde)
+cowde#can_v1)
@@ -343 +343 @@
-not)
+not#not_adv)
@@ -345 +345 @@
-geyt)
+geyt#get_v)
@@ -348 +348 @@
-hyt))
+hyt#it_pron))
@@ -351 +351 @@
-or)
+or#or_adv1)
@@ -354 +354 @@
-then)))))
+then#then_adv)))))
@@ -363 +363 @@
-and)
+and#and_conj1)
@@ -366 +366 @@
-I))
+I#i_pron))
@@ -368 +368 @@
-sall)
+sall#shall_v)
@@ -370 +370 @@
-kepe)
+kepe#keep_v)
@@ -373 +373 @@
-that)
+that#that_pron)
@@ -375 +375 @@
-day))
+day#day_n))
@@ -378 +378 @@
-with)
+with#with_prep)
@@ -381 +381 @@
-the)
+the#the_adj)
@@ -383 +383 @@
-grace)
+grace#grace_n)
@@ -386 +386 @@
-of)
+of#of_prep)
@@ -389 +389 @@
-God))))))
+God#NA))))))
@@ -392 +392 @@
-and)
+and#and_conj1)
@@ -395 +395 @@
-geyt)
+geyt#get_v)
@@ -398 +398 @@
-of)
+of#of_prep)
@@ -401 +401 @@
-hym)))
+hym#him_pron)))
@@ -404 +404 @@
-all)
+all#all_adj)
@@ -409 +409 @@
-that)
+that#that_conj)
@@ -415 +415 @@
-I))
+I#i_pron))
@@ -417 +417 @@
-cane)
+cane#can_v1)
@@ -428 +428 @@
-Ser))
+Ser#sir_n))
@@ -433 +433 @@
-I))
+I#i_pron))
@@ -435 +435 @@
-wold)
+wold#will_v1)
@@ -437 +437 @@
-avyse)
+avyse#advise_v)
@@ -440 +440 @@
-you))
+you#you_pron))
@@ -445 +445 @@
-and)
+and#and_conj1)
@@ -453 +453 @@
-my)
+my#my_adj)
@@ -455 +455 @@
-cosyn)
+cosyn#cousin_n)
@@ -458 +458 @@
-John)
+John#NA)
@@ -460 +460 @@
-Herryngton)))
+Herryngton#NA)))
@@ -462 +462 @@
-man))
+man#man_n1))
@@ -464 +464 @@
-be)
+be#be_v)
@@ -467 +467 @@
-in)
+in#in_prep)
@@ -470 +470 @@
-no)
+no#no_adj)
@@ -472 +472 @@
-ruporte)
+ruporte#report_n)
@@ -475 +475 @@
-of)
+of#of_prep)
@@ -478 +478 @@
-sekenes))))))))
+sekenes#sickness_n))))))))
@@ -483 +483 @@
-to)
+to#to_prep)
@@ -485 +485 @@
-geyt)
+geyt#get_v)
@@ -488 +488 @@
-all)
+all#all_adj)
@@ -490 +490 @@
-the)
+the#the_adj)
@@ -492 +492 @@
-evydens)
+evydens#evidence_n)
@@ -497 +497 @@
-of)
+of#of_prep)
@@ -500 +500 @@
-hym)))
+hym#him_pron)))
@@ -505 +505 @@
-that)
+that#that_conj)
@@ -511 +511 @@
-ze))
+ze#ye_pron))
@@ -513 +513 @@
-cane)
+cane#can_v1)
@@ -518 +518 @@
-or)
+or#or_adv1)
@@ -525 +525 @@
-Ser)
+Ser#NA)
@@ -527 +527 @@
-James))
+James#NA))
@@ -529 +529 @@
-com)
+com#come_v)
@@ -531 +531 @@
-up)))))
+up#up_adv1)))))
@@ -539 +539 @@
-for)
+for#for_prep)
@@ -542 +542 @@
-he))
+he#he_pron))
@@ -544 +544 @@
-is)
+is#be_v)
@@ -546 +546 @@
-purpast)
+purpast#purpose_v)
@@ -549 +549 @@
-to)
+to#to_prep)
@@ -551 +551 @@
-com)
+com#come_v)
@@ -554 +554 @@
-hastly)))
+hastly#hastly_adv)))
@@ -562 +562 @@
-Also)
+Also#also_adv)
@@ -566 +566 @@
-Ser)
+Ser#NA)
@@ -568 +568 @@
-James))
+James#NA))
@@ -571 +571 @@
-and)
+and#and_conj1)
@@ -574 +574 @@
-the)
+the#the_adj)
@@ -576 +576 @@
-Person)
+Person#parson_n)
@@ -579 +579 @@
-of)
+of#of_prep)
@@ -582 +582 @@
-Sladeborne))))))
+Sladeborne#NA))))))
@@ -584 +584 @@
-thynk)
+thynk#think_v2)
@@ -588 +588 @@
-that)
+that#that_conj)
@@ -592 +592 @@
-my)
+my#my_adj)
@@ -594 +594 @@
-cosyn)
+cosyn#cousin_n)
@@ -597 +597 @@
-John)))
+John#NA)))
@@ -599 +599 @@
-was)
+was#be_v)
@@ -601 +601 @@
-puseynd)))
+puseynd#poison_v)))
@@ -606 +606 @@
-and)
+and#and_conj1)
@@ -609 +609 @@
-that)
+that#that_conj)
@@ -613 +613 @@
-his)
+his#his_adj)
@@ -615 +615 @@
-servant))
+servant#servant_n))
@@ -617 +617 @@
-was)
+was#be_v)
@@ -619 +619 @@
-hyryd)
+hyryd#hire_v)
@@ -622 +622 @@
-to)
+to#to_prep)
@@ -624 +624 @@
-do)
+do#do_v)
@@ -627 +627 @@
-hit))
+hit#it_pron))
@@ -630 +630 @@
-by)
+by#by_prep)
@@ -633 +633 @@
-my)
+my#my_adj)
@@ -635 +635 @@
-broder)
+broder#brother_n)
@@ -638 +638 @@
-Sir)
+Sir#NA)
@@ -640 +640 @@
-Edward))))))))
+Edward#NA))))))))
@@ -645 +645 @@
-and)
+and#and_conj1)
@@ -652 +652 @@
-yf)
+yf#if_conj)
@@ -659 +659 @@
-it))
+it#it_pron))
@@ -662 +662 @@
-so))
+so#so_adv))
@@ -664 +664 @@
-be))))
+be#be_v))))
@@ -667 +667 @@
-then))
+then#then_adv))
@@ -670 +670 @@
-he))
+he#he_pron))
@@ -672 +672 @@
-forfets)
+forfets#forfeit_v)
@@ -675 +675 @@
-all))))))
+all#all_adj))))))
@@ -684 +684 @@
-More_over))
+More_over#moreover_adv))
@@ -687 +687 @@
-I))
+I#i_pron))
@@ -689 +689 @@
-send)
+send#send_v1)
@@ -692 +692 @@
-Netylton))
+Netylton#NA))
@@ -695 +695 @@
-for)
+for#for_prep)
@@ -698 +698 @@
-Ser)
+Ser#NA)
@@ -700 +700 @@
-James)))
+James#NA)))
@@ -704 +704 @@
-to)
+to#to_prep)
@@ -706 +706 @@
-mete)
+mete#meet_v)
@@ -709 +709 @@
-me)))
+me#me_pron1)))
@@ -712 +712 @@
-and)
+and#and_conj1)
@@ -717 +717 @@
-speke)
+speke#speak_v)
@@ -720 +720 @@
-with)
+with#with_prep)
@@ -723 +723 @@
-me))))))
+me#me_pron1))))))
@@ -731 +731 @@
-and)
+and#and_conj1)
@@ -734 +734 @@
-he))
+he#he_pron))
@@ -736 +736 @@
-said)
+said#say_v1)
@@ -739 +739 @@
-that)
+that#that_conj)
@@ -743 +743 @@
-he))
+he#he_pron))
@@ -745 +745 @@
-wold)
+wold#will_v1)
@@ -747 +747 @@
-com)
+com#come_v)
@@ -750 +750 @@
-home))
+home#home_n1))
@@ -753 +753 @@
-to)
+to#to_prep)
@@ -756 +756 @@
-me)))))
+me#me_pron1)))))
@@ -764 +764 @@
-and)
+and#and_conj1)
@@ -767 +767 @@
-yf)
+yf#if_conj)
@@ -774 +774 @@
-he))
+he#he_pron))
@@ -777 +777 @@
-so))
+so#so_adv))
@@ -779 +779 @@
-do))))
+do#do_v))))
@@ -782 +782 @@
-os)
+os#as_adv)
@@ -793 +793 @@
-we))
+we#we_pron))
@@ -795 +795 @@
-leyfe))))
+leyfe#live_v1))))
@@ -798 +798 @@
-ze))
+ze#ye_pron))
@@ -800 +800 @@
-sal)
+sal#shall_v)
@@ -802 +802 @@
-hafe)
+hafe#have_v)
@@ -805 +805 @@
-woord))
+woord#word_n))
@@ -813 +813 @@
-And)
+And#and_conj1)
@@ -816 +816 @@
-as)
+as#as_adv)
@@ -819 +819 @@
-for)
+for#for_prep)
@@ -823 +823 @@
-John)
+John#NA)
@@ -825 +825 @@
-Heton))
+Heton#NA))
@@ -828 +828 @@
-and)
+and#and_conj1)
@@ -831 +831 @@
-Roger)
+Roger#NA)
@@ -833 +833 @@
-Leyner))))))
+Leyner#NA))))))
@@ -836 +836 @@
-they))
+they#they_pron))
@@ -838 +838 @@
-come)
+come#come_v)
@@ -840 +840 @@
-not)
+not#not_adv)
@@ -843 +843 @@
-here))
+here#here_adv))
@@ -852 +852 @@
-zet))
+zet#yet_adv))
@@ -855 +855 @@
-I))
+I#i_pron))
@@ -857 +857 @@
-send)
+send#send_v1)
@@ -860 +860 @@
-ham))
+ham#them_pron))
@@ -863 +863 @@
-a)
+a#a_adj2)
@@ -865 +865 @@
-letter))
+letter#letter_n1))
@@ -868 +868 @@
-by)
+by#by_prep)
@@ -871 +871 @@
-Thomas)
+Thomas#NA)
@@ -873 +873 @@
-Orscha))))
+Orscha#NA))))
@@ -879 +879 @@
-and)
+and#and_conj1)
@@ -882 +882 @@
-I))
+I#i_pron))
@@ -884 +884 @@
-had)
+had#have_v)
@@ -887 +887 @@
-no)
+no#no_adj)
@@ -889 +889 @@
-word))
+word#word_n))
@@ -898 +898 @@
-Agayn)
+Agayn#again_adv)
@@ -900 +900 @@
-also))
+also#also_adv))
@@ -905 +905 @@
-Ser))
+Ser#sir_n))
@@ -910 +910 @@
-ze))
+ze#ye_pron))
@@ -912 +912 @@
-wryte)
+wryte#write_v)
@@ -915 +915 @@
-to)
+to#to_prep)
@@ -918 +918 @@
-me)))
+me#me_pron1)))
@@ -921 +921 @@
-for)
+for#for_prep)
@@ -924 +924 @@
-mone)))
+mone#money_n)))
@@ -932 +932 @@
-and)
+and#and_conj1)
@@ -935 +935 @@
-ze))
+ze#ye_pron))
@@ -937 +937 @@
-knawe)
+knawe#know_v)
@@ -940 +940 @@
-that)
+that#that_conj)
@@ -944 +944 @@
-I))
+I#i_pron))
@@ -946 +946 @@
-cane)
+cane#can_v1)
@@ -948 +948 @@
-make)
+make#make_v1)
@@ -951 +951 @@
-no)
+no#no_adj)
@@ -953 +953 @@
-schift))
+schift#shift_n))
@@ -956 +956 @@
-or)
+or#or_adv1)
@@ -959 +959 @@
-Candylmes)))))
+Candylmes#NA)))))
@@ -967 +967 @@
-bot)
+bot#but_prep)
@@ -971 +971 @@
-yf)
+yf#if_conj)
@@ -978 +978 @@
-ze))
+ze#ye_pron))
@@ -980 +980 @@
-think)
+think#think_v2)
@@ -987 +987 @@
-I))
+I#i_pron))
@@ -989 +989 @@
-sall)
+sall#shall_v)
@@ -991 +991 @@
-go)
+go#go_v)
@@ -994 +994 @@
-boro)
+boro#borrow_v1)
@@ -997 +997 @@
-hit))
+hit#it_pron))
@@ -1000 +1000 @@
-of)
+of#of_prep)
@@ -1003 +1003 @@
-my)
+my#my_adj)
@@ -1005 +1005 @@
-lord)
+lord#lord_n)
@@ -1008 +1008 @@
-Archbyschop))))))))))
+Archbyschop#archbishop_n))))))))))
@@ -1013 +1013 @@
-and)
+and#and_conj1)
@@ -1016 +1016 @@
-yf)
+yf#if_conj)
@@ -1023 +1023 @@
-ze))
+ze#ye_pron))
@@ -1025 +1025 @@
-will)
+will#will_v2)
@@ -1028 +1028 @@
-that)
+that#that_conj)
@@ -1032 +1032 @@
-I))
+I#i_pron))
@@ -1035 +1035 @@
-so))
+so#so_adv))
@@ -1037 +1037 @@
-do))))))))
+do#do_v))))))))
@@ -1041 +1041 @@
-send)
+send#send_v1)
@@ -1044 +1044 @@
-me))
+me#me_pron1))
@@ -1047 +1047 @@
-a)
+a#a_adj2)
@@ -1049 +1049 @@
-byll))
+byll#bill_n3))
@@ -1052 +1052 @@
-by)
+by#by_prep)
@@ -1055 +1055 @@
-the)
+the#the_adj)
@@ -1057 +1057 @@
-next)
+next#next_adj)
@@ -1062 +1062 @@
-that)
+that#that_conj)
@@ -1068 +1068 @@
-ze))
+ze#ye_pron))
@@ -1070 +1070 @@
-cane)
+cane#can_v1)
@@ -1081 +1081 @@
-Ser))
+Ser#sir_n))
@@ -1086 +1086 @@
-I))
+I#i_pron))
@@ -1088 +1088 @@
-sall)
+sall#shall_v)
@@ -1090 +1090 @@
-send)
+send#send_v1)
@@ -1093 +1093 @@
-Netylton))
+Netylton#NA))
@@ -1096 +1096 @@
-to)
+to#to_prep)
@@ -1099 +1099 @@
-zou)))
+zou#you_pron)))
@@ -1102 +1102 @@
-als)
+als#as_adv)
@@ -1104 +1104 @@
-hastly)
+hastly#hastly_adv)
@@ -1107 +1107 @@
-os)
+os#as_adv)
@@ -1118 +1118 @@
-I))
+I#i_pron))
@@ -1120 +1120 @@
-cane)
+cane#can_v1)
@@ -1130 +1130 @@
-Also)
+Also#also_adv)
@@ -1135 +1135 @@
-Ser))
+Ser#sir_n))
@@ -1140 +1140 @@
-on)
+on#on_prep)
@@ -1143 +1143 @@
-the)
+the#the_adj)
@@ -1145 +1145 @@
-Frydday)
+Frydday#NA)
@@ -1148 +1148 @@
-after)
+after#after_adv)
@@ -1155 +1155 @@
-ze))
+ze#ye_pron))
@@ -1157 +1157 @@
-departyd))))))
+departyd#depart_v))))))
@@ -1159 +1159 @@
-come)
+come#come_v)
@@ -1163 +1163 @@
-John)
+John#NA)
@@ -1165 +1165 @@
-Sayville))
+Sayville#NA))
@@ -1168 +1168 @@
-and)
+and#and_conj1)
@@ -1171 +1171 @@
-Ser)
+Ser#NA)
@@ -1173 +1173 @@
-Edward)
+Edward#NA)
@@ -1177 +1177 @@
-Thirrell)))
+Thirrell#NA)))
@@ -1184 +1184 @@
-and)
+and#and_conj1)
@@ -1187 +1187 @@
-Ser)
+Ser#NA)
@@ -1189 +1189 @@
-William)
+William#NA)
@@ -1191 +1191 @@
-Wilkynson)))
+Wilkynson#NA)))
@@ -1197 +1197 @@
-Ser)
+Ser#NA)
@@ -1199 +1199 @@
-Robert)
+Robert#NA)
@@ -1201 +1201 @@
-Audley)))
+Audley#NA)))
@@ -1206 +1206 @@
-and)
+and#and_conj1)
@@ -1209 +1209 @@
-Ric.)
+Ric.#NA)
@@ -1211 +1211 @@
-Gledylle))))
+Gledylle#NA))))
@@ -1214 +1214 @@
-with)
+with#with_prep)
@@ -1217 +1217 @@
-Ric.)))
+Ric.#NA)))
@@ -1225 +1225 @@
-and)
+and#and_conj1)
@@ -1229 +1229 @@
-hafe)
+hafe#have_v)
@@ -1231 +1231 @@
-dyscharge)
+dyscharge#discharge_v)
@@ -1234 +1234 @@
-hym))
+hym#him_pron))
@@ -1242 +1242 @@
-and)
+and#and_conj1)
@@ -1245 +1245 @@
-he))
+he#he_pron))
@@ -1247 +1247 @@
-wold)
+wold#will_v1)
@@ -1249 +1249 @@
-take)
+take#take_v)
@@ -1252 +1252 @@
-none))
+none#none_pron))
@@ -1258 +1258 @@
-hym)))
+hym#him_pron)))
@@ -1266 +1266 @@
-Also)
+Also#also_adv)
@@ -1271 +1271 @@
-Ser))
+Ser#sir_n))
@@ -1276 +1276 @@
-I))
+I#i_pron))
@@ -1278 +1278 @@
-pray)
+pray#pray_v)
@@ -1281 +1281 @@
-zou))
+zou#you_pron))
@@ -1285 +1285 @@
-to)
+to#to_prep)
@@ -1287 +1287 @@
-kepe)
+kepe#keep_v)
@@ -1290 +1290 @@
-zou))
+zou#you_pron))
@@ -1293 +1293 @@
-out)
+out#out_adv)
@@ -1295 +1295 @@
-of)
+of#of_prep)
@@ -1298 +1298 @@
-all)
+all#all_adj)
@@ -1300 +1300 @@
-jopertese))))
+jopertese#jeopardy_n))))
@@ -1305 +1305 @@
-and)
+and#and_conj1)
@@ -1308 +1308 @@
-to)
+to#to_prep)
@@ -1310 +1310 @@
-make)
+make#make_v1)
@@ -1313 +1313 @@
-myche)
+myche#much_adj)
@@ -1316 +1316 @@
-of)
+of#of_prep)
@@ -1319 +1319 @@
-zour)
+zour#your_pron)
@@ -1321 +1321 @@
-selfe)))))))
+selfe#self_pron)))))))
@@ -1329 +1329 @@
-and)
+and#and_conj1)
@@ -1332 +1332 @@
-the)
+the#the_adj)
@@ -1334 +1334 @@
-Holy)
+Holy#holy_adj)
@@ -1336 +1336 @@
-Trenete))
+Trenete#NA))
@@ -1338 +1338 @@
-hafe)
+hafe#have_v)
@@ -1341 +1341 @@
-zou))
+zou#you_pron))
@@ -1344 +1344 @@
-in)
+in#in_prep)
@@ -1347 +1347 @@
-hys)
+hys#his_adj)
@@ -1349 +1349 @@
-blysyd)
+blysyd#blessed_adj)
@@ -1351 +1351 @@
-kepeyng)))
+kepeyng#keeping_n)))
@@ -1359 +1359 @@
-Your)
+Your#your_pron)
@@ -1361 +1361 @@
-wyfe)
+wyfe#wife_n)
@@ -1366 +1366 @@
-Elezabeth)
+Elezabeth#NA)
@@ -1368 +1368 @@
-Beaumount))
+Beaumount#NA))
