--- ecumberl-1545-e1-h.psd.vert	2025-09-10 13:50:35
+++ ecumberl-1545-e1-h.lem.vert	2025-09-10 13:50:45
@@ -17 +17 @@
-Jhesus))
+Jhesus#NA))
@@ -20 +20 @@
-Dere)
+Dere#dear_adj1)
@@ -22 +22 @@
-hart))
+hart#heart_n))
@@ -27 +27 @@
-after)
+after#after_adv)
@@ -30 +30 @@
-my)
+my#my_adj)
@@ -33 +33 @@
-moste)
+moste#most_adj)
@@ -35 +35 @@
-hartye))
+hartye#hearty_adj))
@@ -37 +37 @@
-commendatyons)))
+commendatyons#commendation_n)))
@@ -42 +42 @@
-thys))
+thys#this_pron))
@@ -44 +44 @@
-shal@)
+shal@#shall_v)
@@ -46 +46 @@
-@be)
+@be#be_v)
@@ -49 +49 @@
-to)
+to#to_prep)
@@ -51 +51 @@
-sertify)
+sertify#certify_v)
@@ -54 +54 @@
-yow))
+yow#you_pron))
@@ -57 +57 @@
-that)
+that#that_conj)
@@ -62 +62 @@
-sense)
+sense#since_adv)
@@ -65 +65 @@
-yowr)
+yowr#your_pron)
@@ -67 +67 @@
-departure)
+departure#departure_n)
@@ -70 +70 @@
-frome)
+frome#from_prep)
@@ -73 +73 @@
-me)))))
+me#me_pron1)))))
@@ -78 +78 @@
-I))
+I#i_pron))
@@ -80 +80 @@
-have)
+have#have_v)
@@ -82 +82 @@
-byn)
+byn#be_v)
@@ -85 +85 @@
-very)
+very#very_adj)
@@ -87 +87 @@
-seke)))
+seke#sick_adj)))
@@ -90 +90 @@
-&)
+&#and_conj1)
@@ -94 +94 @@
-att)
+att#at_prep)
@@ -97 +97 @@
-thys)
+thys#this_pron)
@@ -99 +99 @@
-present)))
+present#present_adj)))
@@ -102 +102 @@
-my)
+my#my_adj)
@@ -104 +104 @@
-watter))
+watter#water_n))
@@ -106 +106 @@
-ys)
+ys#be_v)
@@ -109 +109 @@
-very)
+very#very_adj)
@@ -111 +111 @@
-redd))
+redd#red_adj))
@@ -117 +117 @@
-wherby))
+wherby#whereby_adv))
@@ -125 +125 @@
-I))
+I#i_pron))
@@ -127 +127 @@
-suppos)
+suppos#suppose_v)
@@ -134 +134 @@
-I))
+I#i_pron))
@@ -136 +136 @@
-have)
+have#have_v)
@@ -140 +140 @@
-the)
+the#the_adj)
@@ -142 +142 @@
-jaundes))
+jaundes#jaundice_n))
@@ -145 +145 @@
-&)
+&#and_conj1)
@@ -148 +148 @@
-the)
+the#the_adj)
@@ -150 +150 @@
-aygew)))
+aygew#ague_n)))
@@ -152 +152 @@
-both)))))))))))
+both#both_pron)))))))))))
@@ -160 +160 @@
-for)
+for#for_prep)
@@ -163 +163 @@
-I))
+I#i_pron))
@@ -165 +165 @@
-have)
+have#have_v)
@@ -168 +168 @@
-none)
+none#none_pron)
@@ -170 +170 @@
-abyde)
+abyde#abide_n)
@@ -173 +173 @@
-to)
+to#to_prep)
@@ -176 +176 @@
-meate)))))
+meate#meat_n)))))
@@ -182 +182 @@
-&)
+&#and_conj1)
@@ -185 +185 @@
-I))
+I#i_pron))
@@ -187 +187 @@
-have)
+have#have_v)
@@ -190 +190 @@
-suche)
+suche#such_adj)
@@ -192 +192 @@
-payns)
+payns#pain_n1)
@@ -196 +196 @@
-in)
+in#in_prep)
@@ -199 +199 @@
-my)
+my#my_adj)
@@ -201 +201 @@
-syde)))
+syde#side_n1)))
@@ -204 +204 @@
-&)
+&#and_conj1)
@@ -207 +207 @@
-towardes)
+towardes#towards_prep)
@@ -210 +210 @@
-my)
+my#my_adj)
@@ -212 +212 @@
-bak)))))
+bak#back_n1)))))
@@ -215 +215 @@
-as)
+as#as_adv)
@@ -226 +226 @@
-I))
+I#i_pron))
@@ -228 +228 @@
-had)
+had#have_v)
@@ -231 +231 @@
-att)
+att#at_prep)
@@ -234 +234 @@
-Brauham)
+Brauham#NA)
@@ -240 +240 @@
-wher))
+wher#where_adv))
@@ -248 +248 @@
-ytt))
+ytt#it_pron))
@@ -250 +250 @@
-be_gane)
+be_gane#begin_v1)
@@ -253 +253 @@
-with)
+with#with_prep)
@@ -256 +256 @@
-me)))
+me#me_pron1)))
@@ -259 +259 @@
-furst))))))))))
+furst#first_adj))))))))))
@@ -265 +265 @@
-Wher)
+Wher#where_adv)
@@ -267 +267 @@
-for))
+for#for_prep))
@@ -275 +275 @@
-I))
+I#i_pron))
@@ -277 +277 @@
-desyre)
+desyre#desire_v)
@@ -280 +280 @@
-yow))
+yow#you_pron))
@@ -284 +284 @@
-to)
+to#to_prep)
@@ -286 +286 @@
-help)
+help#help_v)
@@ -289 +289 @@
-me))
+me#me_pron1))
@@ -292 +292 @@
-to)
+to#to_prep)
@@ -295 +295 @@
-a)
+a#a_adj2)
@@ -297 +297 @@
-physyssyon))))
+physyssyon#physician_n))))
@@ -300 +300 @@
-&)
+&#and_conj1)
@@ -303 +303 @@
-that)
+that#that_conj)
@@ -307 +307 @@
-thys)
+thys#this_pron)
@@ -309 +309 @@
-berer))
+berer#bearer_n))
@@ -311 +311 @@
-may)
+may#may_v1)
@@ -313 +313 @@
-brynge)
+brynge#bring_v)
@@ -316 +316 @@
-hym))
+hym#him_pron))
@@ -319 +319 @@
-with)
+with#with_prep)
@@ -322 +322 @@
-hym)))))))))
+hym#him_pron)))))))))
@@ -330 +330 @@
-for)
+for#for_prep)
@@ -333 +333 @@
-now))
+now#now_adv))
@@ -336 +336 @@
-in)
+in#in_prep)
@@ -339 +339 @@
-the)
+the#the_adj)
@@ -341 +341 @@
-begynning)))
+begynning#beginning_n)))
@@ -344 +344 @@
-I))
+I#i_pron))
@@ -346 +346 @@
-trust)
+trust#trust_v)
@@ -353 +353 @@
-I))
+I#i_pron))
@@ -355 +355 @@
-may)
+may#may_v1)
@@ -357 +357 @@
-have)
+have#have_v)
@@ -360 +360 @@
-gud)
+gud#good_adj)
@@ -362 +362 @@
-remedy))))
+remedy#remedy_n))))
@@ -370 +370 @@
-&)
+&#and_conj1)
@@ -374 +374 @@
-the)
+the#the_adj)
@@ -376 +376 @@
-longer)
+longer#long_adj1)
@@ -387 +387 @@
-ytt))
+ytt#it_pron))
@@ -389 +389 @@
-ys)
+ys#be_v)
@@ -391 +391 @@
-delayed)))))
+delayed#delay_v1)))))
@@ -394 +394 @@
-the)
+the#the_adj)
@@ -396 +396 @@
-worse))
+worse#worse_adj))
@@ -399 +399 @@
-ytt))
+ytt#it_pron))
@@ -401 +401 @@
-wyl@)
+wyl@#will_v1)
@@ -403 +403 @@
-@be)
+@be#be_v)
@@ -411 +411 @@
-Also)
+Also#also_adv)
@@ -414 +414 @@
-my)
+my#my_adj)
@@ -416 +416 @@
-sister)
+sister#sister_n)
@@ -419 +419 @@
-Powys)))
+Powys#NA)))
@@ -421 +421 @@
-ys)
+ys#be_v)
@@ -423 +423 @@
-comyd)
+comyd#come_v)
@@ -426 +426 @@
-to)
+to#to_prep)
@@ -429 +429 @@
-me))))
+me#me_pron1))))
@@ -435 +435 @@
-&)
+&#and_conj1)
@@ -439 +439 @@
-ys)
+ys#be_v)
@@ -442 +442 @@
-very)
+very#very_adj)
@@ -444 +444 @@
-desyrous)
+desyrous#desirous_adj)
@@ -447 +447 @@
-to)
+to#to_prep)
@@ -449 +449 @@
-se)
+se#see_v)
@@ -452 +452 @@
-yow))
+yow#you_pron))
@@ -458 +458 @@
-whiche))
+whiche#which_adj))
@@ -464 +464 @@
-I))
+I#i_pron))
@@ -466 +466 @@
-trust)
+trust#trust_v)
@@ -474 +474 @@
-shal@)
+shal@#shall_v)
@@ -476 +476 @@
-@be)
+@be#be_v)
@@ -479 +479 @@
-the)
+the#the_adj)
@@ -481 +481 @@
-sooner))
+sooner#soon_adv))
@@ -484 +484 @@
-at)
+at#at_prep)
@@ -487 +487 @@
-this)
+this#this_pron)
@@ -489 +489 @@
-tyme))))))))))
+tyme#time_n))))))))))
@@ -495 +495 @@
-&)
+&#and_conj1)
@@ -498 +498 @@
-thus))
+thus#thus_adv))
@@ -501 +501 @@
-Jhesu))
+Jhesu#NA))
@@ -503 +503 @@
-send)
+send#send_v1)
@@ -506 +506 @@
-hus)
+hus#us_pron)
@@ -509 +509 @@
-both)))
+both#both_pron)))
@@ -512 +512 @@
-healthe))
+healthe#health_n))
@@ -520 +520 @@
-Att)
+Att#at_prep)
@@ -523 +523 @@
-my)
+my#my_adj)
@@ -525 +525 @@
-lodge)
+lodge#lodge_n)
@@ -528 +528 @@
-of)
+of#of_prep)
@@ -531 +531 @@
-Carleton))))
+Carleton#NA))))
@@ -539 +539 @@
-the)
+the#the_adj)
@@ -541 +541 @@
-xiiij=th=)
+xiiij=th=#NA)
@@ -543 +543 @@
-day)
+day#day_n)
@@ -546 +546 @@
-of)
+of#of_prep)
@@ -549 +549 @@
-February)))
+February#NA)))
@@ -557 +557 @@
-And)
+And#and_conj1)
@@ -562 +562 @@
-dere)
+dere#dear_adj1)
@@ -564 +564 @@
-hart))
+hart#heart_n))
@@ -570 +570 @@
-I))
+I#i_pron))
@@ -572 +572 @@
-pray)
+pray#pray_v)
@@ -575 +575 @@
-yow)))
+yow#you_pron)))
@@ -577 +577 @@
-send)
+send#send_v1)
@@ -580 +580 @@
-for)
+for#for_prep)
@@ -583 +583 @@
-Doctor)
+Doctor#NA)
@@ -585 +585 @@
-Stephyns)))
+Stephyns#NA)))
@@ -593 +593 @@
-for)
+for#for_prep)
@@ -596 +596 @@
-he))
+he#he_pron))
@@ -598 +598 @@
-knowyth)
+knowyth#know_v)
@@ -601 +601 @@
-best))
+best#best_adj))
@@ -604 +604 @@
-my)
+my#my_adj)
@@ -606 +606 @@
-complexon)
+complexon#complexion_n)
@@ -609 +609 @@
-for)
+for#for_prep)
@@ -612 +612 @@
-such)
+such#such_adj)
@@ -614 +614 @@
-cawsys))))
+cawsys#cause_n))))
@@ -622 +622 @@
-By)
+By#by_prep)
@@ -625 +625 @@
-yowr)
+yowr#your_pron)
@@ -627 +627 @@
-assuryd)
+assuryd#assured_adj)
@@ -629 +629 @@
-loufyng)
+loufyng#loving_adj)
@@ -631 +631 @@
-wyff)
+wyff#wife_n)
@@ -636 +636 @@
-Elenor)
+Elenor#NA)
@@ -638 +638 @@
-Cumbarland)))
+Cumbarland#NA)))
