--- edmondes-1599-e2-h.psd.vert	2025-09-10 13:50:35
+++ edmondes-1599-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-It))
+It#it_pron))
@@ -16 +16 @@
-maie)
+maie#may_v1)
@@ -18 +18 @@
-please)
+please#please_v)
@@ -21 +21 @@
-yor)
+yor#your_pron)
@@ -23 +23 @@
-honor))
+honor#honour_n))
@@ -32 +32 @@
-I))
+I#i_pron))
@@ -34 +34 @@
-haue)
+haue#have_v)
@@ -37 +37 @@
-according)
+according#accord_v)
@@ -40 +40 @@
-to)
+to#to_prep)
@@ -44 +44 @@
-yor)
+yor#your_pron)
@@ -46 +46 @@
-honors))
+honors#honour_n))
@@ -48 +48 @@
-Comandement))))
+Comandement#commandment_n))))
@@ -50 +50 @@
-used)
+used#use_v)
@@ -53 +53 @@
-all)
+all#all_adj)
@@ -55 +55 @@
-endevor)
+endevor#endeavour_n)
@@ -58 +58 @@
-and)
+and#and_conj1)
@@ -61 +61 @@
-industrie))))
+industrie#industry_n))))
@@ -64 +64 @@
-to)
+to#to_prep)
@@ -66 +66 @@
-fynd)
+fynd#find_v)
@@ -68 +68 @@
-out)
+out#out_adv)
@@ -71 +71 @@
-a)
+a#a_adj2)
@@ -73 +73 @@
-meanes)
+meanes#mean_n2)
@@ -76 +76 @@
-for)
+for#for_prep)
@@ -79 +79 @@
-yor)
+yor#your_pron)
@@ -81 +81 @@
-honor))
+honor#honour_n))
@@ -83 +83 @@
-to)
+to#to_prep)
@@ -85 +85 @@
-receiue)
+receiue#receive_v)
@@ -88 +88 @@
-aduertisementes))
+aduertisementes#advertisement_n))
@@ -91 +91 @@
-out)
+out#out_adv)
@@ -93 +93 @@
-of)
+of#of_prep)
@@ -96 +96 @@
-Spaine))))))
+Spaine#NA))))))
@@ -104 +104 @@
-but)
+but#but_prep)
@@ -109 +109 @@
-being)
+being#be_v)
@@ -112 +112 @@
-verie)
+verie#very_adj)
@@ -114 +114 @@
-hard))
+hard#hard_adj))
@@ -117 +117 @@
-to)
+to#to_prep)
@@ -119 +119 @@
-meet)
+meet#meet_v)
@@ -122 +122 @@
-wth)
+wth#with_prep)
@@ -125 +125 @@
-one)
+one#one_adj)
@@ -133 +133 @@
-to)
+to#to_prep)
@@ -135 +135 @@
-send)
+send#send_v1)
@@ -138 +138 @@
-exp~slie))
+exp~slie#expressly_adv))
@@ -141 +141 @@
-thither))))
+thither#thither_adv))))
@@ -148 +148 @@
-that)
+that#that_conj)
@@ -154 +154 @@
-is)
+is#be_v)
@@ -157 +157 @@
-faithfull)
+faithfull#faithful_adj)
@@ -159 +159 @@
-and)
+and#and_conj1)
@@ -161 +161 @@
-sufficient)))
+sufficient#sufficient_adj)))
@@ -166 +166 @@
-and)
+and#and_conj1)
@@ -171 +171 @@
-can)
+can#can_v1)
@@ -173 +173 @@
-be)
+be#be_v)
@@ -176 +176 @@
-able)
+able#able_adj)
@@ -179 +179 @@
-to)
+to#to_prep)
@@ -181 +181 @@
-serue)
+serue#serve_v1)
@@ -184 +184 @@
-yor)
+yor#your_pron)
@@ -186 +186 @@
-honor))
+honor#honour_n))
@@ -189 +189 @@
-as)
+as#as_adv)
@@ -199 +199 @@
-were)
+were#be_v)
@@ -202 +202 @@
-fitt)))))))))))))))
+fitt#fit_adj)))))))))))))))
@@ -207 +207 @@
-I))
+I#i_pron))
@@ -209 +209 @@
-haue)
+haue#have_v)
@@ -212 +212 @@
-therefore))
+therefore#therefore_adv))
@@ -214 +214 @@
-aduised)
+aduised#advise_v)
@@ -218 +218 @@
-howe))
+howe#how_adv))
@@ -223 +223 @@
-to)
+to#to_prep)
@@ -225 +225 @@
-wourke)
+wourke#work_v)
@@ -228 +228 @@
-the)
+the#the_adj)
@@ -230 +230 @@
-same))
+same#same_adj))
@@ -233 +233 @@
-by)
+by#by_prep)
@@ -236 +236 @@
-those)
+those#those_pron)
@@ -239 +239 @@
-of)
+of#of_prep)
@@ -242 +242 @@
-the)
+the#the_adj)
@@ -244 +244 @@
-best)
+best#best_adj)
@@ -246 +246 @@
-sorte)))
+sorte#sort_n2)))
@@ -249 +249 @@
-here))
+here#here_adv))
@@ -254 +254 @@
-that)
+that#that_conj)
@@ -259 +259 @@
-haue)
+haue#have_v)
@@ -262 +262 @@
-correspondencie)
+correspondencie#correspondency_n)
@@ -265 +265 @@
-in)
+in#in_prep)
@@ -268 +268 @@
-Spayne))))))))))
+Spayne#NA))))))))))
@@ -276 +276 @@
-And)
+And#and_conj1)
@@ -280 +280 @@
-aboue)
+aboue#above_adv)
@@ -283 +283 @@
-all)
+all#all_adj)
@@ -285 +285 @@
-others)))
+others#other_adj)))
@@ -288 +288 @@
-one)
+one#one_adj)
@@ -290 +290 @@
-Monsr)
+Monsr#NA)
@@ -292 +292 @@
-Anthoine)
+Anthoine#NA)
@@ -294 +294 @@
-Balbany)
+Balbany#NA)
@@ -299 +299 @@
-an)
+an#a_adj2)
@@ -301 +301 @@
-Itallian)
+Itallian#italian_adj)
@@ -303 +303 @@
-marchant)
+marchant#merchant_n)
@@ -306 +306 @@
-of)
+of#of_prep)
@@ -309 +309 @@
-the)
+the#the_adj)
@@ -311 +311 @@
-Principall)
+Principall#principal_adj)
@@ -314 +314 @@
-of)
+of#of_prep)
@@ -317 +317 @@
-this)
+this#this_pron)
@@ -319 +319 @@
-Towne)))))))
+Towne#town_n)))))))
@@ -323 +323 @@
-hauing)
+hauing#have_v)
@@ -326 +326 @@
-the)
+the#the_adj)
@@ -330 +330 @@
-most)
+most#most_adj)
@@ -332 +332 @@
-frequent))
+frequent#frequent_adj))
@@ -335 +335 @@
-&)
+&#and_conj1)
@@ -338 +338 @@
-best))))
+best#best_adj))))
@@ -340 +340 @@
-aduertisementes)
+aduertisementes#advertisement_n)
@@ -343 +343 @@
-from)
+from#from_prep)
@@ -346 +346 @@
-thence))))
+thence#thence_adv))))
@@ -349 +349 @@
-because)
+because#because_adv)
@@ -352 +352 @@
-of)
+of#of_prep)
@@ -355 +355 @@
-his)
+his#his_adj)
@@ -357 +357 @@
-great)
+great#great_adj)
@@ -359 +359 @@
-trade)
+trade#trade_n)
@@ -363 +363 @@
-there))
+there#there_adv))
@@ -368 +368 @@
-and)
+and#and_conj1)
@@ -371 +371 @@
-wth)
+wth#with_prep)
@@ -374 +374 @@
-the)
+the#the_adj)
@@ -376 +376 @@
-chief)
+chief#chief_adj)
@@ -378 +378 @@
-p~sone)
+p~sone#person_n)
@@ -381 +381 @@
-of)
+of#of_prep)
@@ -384 +384 @@
-that)
+that#that_pron)
@@ -386 +386 @@
-Countrie)))))))))))
+Countrie#country_n)))))))))))
@@ -389 +389 @@
-I))
+I#i_pron))
@@ -391 +391 @@
-sought)
+sought#seek_v)
@@ -394 +394 @@
-to)
+to#to_prep)
@@ -396 +396 @@
-haue)
+haue#have_v)
@@ -400 +400 @@
-it))
+it#it_pron))
@@ -402 +402 @@
-p~pounded)
+p~pounded#propound_v)
@@ -405 +405 @@
-to)
+to#to_prep)
@@ -408 +408 @@
-him)))
+him#him_pron)))
@@ -413 +413 @@
-whether)
+whether#whether_pron)
@@ -419 +419 @@
-he))
+he#he_pron))
@@ -421 +421 @@
-would)
+would#will_v1)
@@ -423 +423 @@
-be)
+be#be_v)
@@ -426 +426 @@
-content)
+content#content_adj)
@@ -430 +430 @@
-to)
+to#to_prep)
@@ -432 +432 @@
-emploie)
+emploie#employ_v)
@@ -435 +435 @@
-his)
+his#his_adj)
@@ -437 +437 @@
-frends))
+frends#friend_n))
@@ -440 +440 @@
-to)
+to#to_prep)
@@ -442 +442 @@
-aduertise)
+aduertise#advertise_v)
@@ -445 +445 @@
-him))
+him#him_pron))
@@ -448 +448 @@
-of)
+of#of_prep)
@@ -451 +451 @@
-that)
+that#that_pron)
@@ -455 +455 @@
-wch))
+wch#which_adj))
@@ -462 +462 @@
-should)
+should#shall_v)
@@ -464 +464 @@
-passe)
+passe#pass_v)
@@ -467 +467 @@
-there))))))))
+there#there_adv))))))))
@@ -472 +472 @@
-and)
+and#and_conj1)
@@ -475 +475 @@
-that)
+that#that_conj)
@@ -479 +479 @@
-yor)
+yor#your_pron)
@@ -481 +481 @@
-honor))
+honor#honour_n))
@@ -483 +483 @@
-might)
+might#may_v1)
@@ -485 +485 @@
-receiue)
+receiue#receive_v)
@@ -488 +488 @@
-information)
+information#information_n)
@@ -491 +491 @@
-thereof))))))
+thereof#thereof_adv))))))
@@ -497 +497 @@
-wch)
+wch#which_adj)
@@ -499 +499 @@
-curtesie))
+curtesie#courtesy_n))
@@ -507 +507 @@
-yor)
+yor#your_pron)
@@ -509 +509 @@
-honor))
+honor#honour_n))
@@ -511 +511 @@
-would)
+would#will_v1)
@@ -514 +514 @@
-alwaies))
+alwaies#always_adv))
@@ -517 +517 @@
-kindlie))
+kindlie#kindly_adv))
@@ -519 +519 @@
-acknoweledge)
+acknoweledge#acknowledge_v)
@@ -522 +522 @@
-towardes)
+towardes#towards_prep)
@@ -526 +526 @@
-himselfe))
+himselfe#himself_pron))
@@ -529 +529 @@
-&)
+&#and_conj1)
@@ -532 +532 @@
-anie)
+anie#any_adj)
@@ -534 +534 @@
-his)
+his#his_adj)
@@ -536 +536 @@
-frendes)))))
+frendes#friend_n)))))
@@ -539 +539 @@
-as)
+as#as_adv)
@@ -546 +546 @@
-they))
+they#they_pron))
@@ -548 +548 @@
-should)
+should#shall_v)
@@ -550 +550 @@
-haue)
+haue#have_v)
@@ -553 +553 @@
-anie)
+anie#any_adj)
@@ -555 +555 @@
-neede))
+neede#need_n1))
@@ -558 +558 @@
-to)
+to#to_prep)
@@ -560 +560 @@
-use)
+use#use_v)
@@ -564 +564 @@
-yor)
+yor#your_pron)
@@ -566 +566 @@
-honors))
+honors#honour_n))
@@ -568 +568 @@
-fauor))))))))))))))
+fauor#favour_n))))))))))))))
@@ -577 +577 @@
-There))
+There#there_adv))
@@ -579 +579 @@
-was)
+was#be_v)
@@ -582 +582 @@
-some)
+some#some_pron)
@@ -584 +584 @@
-difficultie)
+difficultie#difficulty_n)
@@ -588 +588 @@
-made)
+made#make_v1)
@@ -591 +591 @@
-at)
+at#at_prep)
@@ -594 +594 @@
-the)
+the#the_adj)
@@ -596 +596 @@
-first)))
+first#first_adj)))
@@ -599 +599 @@
-to)
+to#to_prep)
@@ -601 +601 @@
-deale)
+deale#deal_v)
@@ -604 +604 @@
-wth)
+wth#with_prep)
@@ -607 +607 @@
-him)))
+him#him_pron)))
@@ -610 +610 @@
-therin)))
+therin#therein_adv)))
@@ -615 +615 @@
-because)
+because#because_adv)
@@ -622 +622 @@
-being)
+being#be_v)
@@ -626 +626 @@
-a)
+a#a_adj2)
@@ -628 +628 @@
-sup~stitious)
+sup~stitious#superstitious_adj)
@@ -630 +630 @@
-papist))
+papist#NA))
@@ -635 +635 @@
-and)
+and#and_conj1)
@@ -638 +638 @@
-not)
+not#not_adv)
@@ -640 +640 @@
-to)
+to#to_prep)
@@ -642 +642 @@
-be)
+be#be_v)
@@ -644 +644 @@
-swayed)
+swayed#sway_v)
@@ -647 +647 @@
-by)
+by#by_prep)
@@ -650 +650 @@
-the)
+the#the_adj)
@@ -652 +652 @@
-consideration)
+consideration#consideration_n)
@@ -655 +655 @@
-of)
+of#of_prep)
@@ -658 +658 @@
-benefittes)))))))))
+benefittes#benefit_n)))))))))
@@ -663 +663 @@
-in)
+in#in_prep)
@@ -666 +666 @@
-respect)
+respect#respect_n)
@@ -669 +669 @@
-of)
+of#of_prep)
@@ -672 +672 @@
-the)
+the#the_adj)
@@ -674 +674 @@
-greatnes)
+greatnes#greatness_n)
@@ -677 +677 @@
-of)
+of#of_prep)
@@ -680 +680 @@
-his)
+his#his_adj)
@@ -682 +682 @@
-estate)))))))
+estate#estate_n)))))))
@@ -687 +687 @@
-it))
+it#it_pron))
@@ -689 +689 @@
-was)
+was#be_v)
@@ -691 +691 @@
-doubted)
+doubted#doubt_v)
@@ -698 +698 @@
-the)
+the#the_adj)
@@ -700 +700 @@
-ouuerture))
+ouuerture#overture_n))
@@ -702 +702 @@
-would)
+would#will_v1)
@@ -704 +704 @@
-not)
+not#not_adv)
@@ -706 +706 @@
-be)
+be#be_v)
@@ -709 +709 @@
-agreable)
+agreable#agreeable_adj)
@@ -712 +712 @@
-to)
+to#to_prep)
@@ -715 +715 @@
-him))))
+him#him_pron))))
@@ -720 +720 @@
-but)
+but#but_prep)
@@ -723 +723 @@
-for)
+for#for_prep)
@@ -726 +726 @@
-that)
+that#that_conj)
@@ -730 +730 @@
-he)
+he#he_pron)
@@ -733 +733 @@
-wth)
+wth#with_prep)
@@ -736 +736 @@
-Monsr)
+Monsr#NA)
@@ -738 +738 @@
-Zamatt))))
+Zamatt#NA))))
@@ -740 +740 @@
-doe)
+doe#do_v)
@@ -742 +742 @@
-shippe)
+shippe#ship_v)
@@ -745 +745 @@
-great)
+great#great_adj)
@@ -747 +747 @@
-quantetie)
+quantetie#quantity_n)
@@ -750 +750 @@
-of)
+of#of_prep)
@@ -753 +753 @@
-Corne))))
+Corne#corn_n1))))
@@ -756 +756 @@
-to)
+to#to_prep)
@@ -758 +758 @@
-transport)
+transport#transport_v)
@@ -761 +761 @@
-out)
+out#out_adv)
@@ -763 +763 @@
-of)
+of#of_prep)
@@ -766 +766 @@
-the)
+the#the_adj)
@@ -768 +768 @@
-Realme)))
+Realme#realm_n)))
@@ -771 +771 @@
-into)
+into#into_prep)
@@ -774 +774 @@
-Spayne)))))))))))))
+Spayne#NA)))))))))))))
@@ -782 +782 @@
-and)
+and#and_conj1)
@@ -785 +785 @@
-arguying)
+arguying#argue_v)
@@ -788 +788 @@
-wth)
+wth#with_prep)
@@ -791 +791 @@
-me)))
+me#me_pron1)))
@@ -796 +796 @@
-whether)
+whether#whether_pron)
@@ -802 +802 @@
-the)
+the#the_adj)
@@ -804 +804 @@
-same))
+same#same_adj))
@@ -806 +806 @@
-be)
+be#be_v)
@@ -809 +809 @@
-in)
+in#in_prep)
@@ -812 +812 @@
-danger)
+danger#danger_n)
@@ -815 +815 @@
-to)
+to#to_prep)
@@ -817 +817 @@
-be)
+be#be_v)
@@ -819 +819 @@
-staied)
+staied#stay_v1)
@@ -822 +822 @@
-if)
+if#if_conj)
@@ -829 +829 @@
-the)
+the#the_adj)
@@ -831 +831 @@
-said)
+said#said_adj)
@@ -833 +833 @@
-vessells))
+vessells#vessel_n1))
@@ -835 +835 @@
-be)
+be#be_v)
@@ -837 +837 @@
-mett)
+mett#meet_v)
@@ -840 +840 @@
-by)
+by#by_prep)
@@ -843 +843 @@
-anie)
+anie#any_adj)
@@ -845 +845 @@
-englishe)
+englishe#english_adj)
@@ -847 +847 @@
-shippes))))))))))))
+shippes#ship_n1))))))))))))
@@ -850 +850 @@
-I))
+I#i_pron))
@@ -852 +852 @@
-did)
+did#do_v)
@@ -855 +855 @@
-thereof))
+thereof#thereof_adv))
@@ -857 +857 @@
-assure)
+assure#assure_v)
@@ -860 +860 @@
-him))
+him#him_pron))
@@ -869 +869 @@
-yett))
+yett#yet_adv))
@@ -872 +872 @@
-afterwardes))
+afterwardes#afterwards_adv))
@@ -875 +875 @@
-I))
+I#i_pron))
@@ -877 +877 @@
-did)
+did#do_v)
@@ -879 +879 @@
-cause)
+cause#cause_v1)
@@ -883 +883 @@
-it))
+it#it_pron))
@@ -885 +885 @@
-to)
+to#to_prep)
@@ -887 +887 @@
-be)
+be#be_v)
@@ -889 +889 @@
-signified)
+signified#signify_v)
@@ -892 +892 @@
-to)
+to#to_prep)
@@ -895 +895 @@
-him)))
+him#him_pron)))
@@ -900 +900 @@
-that)
+that#that_conj)
@@ -904 +904 @@
-if)
+if#if_conj)
@@ -911 +911 @@
-for)
+for#for_prep)
@@ -914 +914 @@
-his)
+his#his_adj)
@@ -916 +916 @@
-better)
+better#better_adj)
@@ -918 +918 @@
-assurance)))
+assurance#assurance_n)))
@@ -921 +921 @@
-he))
+he#he_pron))
@@ -923 +923 @@
-would)
+would#will_v1)
@@ -925 +925 @@
-be)
+be#be_v)
@@ -927 +927 @@
-strengthned)
+strengthned#strengthen_v)
@@ -930 +930 @@
-wth)
+wth#with_prep)
@@ -933 +933 @@
-anie)
+anie#any_adj)
@@ -935 +935 @@
-pasportes)
+pasportes#passport_n1)
@@ -938 +938 @@
-from)
+from#from_prep)
@@ -941 +941 @@
-my)
+my#my_adj)
@@ -943 +943 @@
-lord)
+lord#lord_n)
@@ -945 +945 @@
-Admyrall))))))))
+Admyrall#admiral_n))))))))
@@ -947 +947 @@
-that)
+that#that_conj)
@@ -953 +953 @@
-I))
+I#i_pron))
@@ -955 +955 @@
-durst)
+durst#dare_v1)
@@ -957 +957 @@
-p~sume)
+p~sume#presume_v)
@@ -964 +964 @@
-yor)
+yor#your_pron)
@@ -966 +966 @@
-honor))
+honor#honour_n))
@@ -968 +968 @@
-would)
+would#will_v1)
@@ -970 +970 @@
-gratifie)
+gratifie#gratify_v)
@@ -973 +973 @@
-him))
+him#him_pron))
@@ -976 +976 @@
-wth)
+wth#with_prep)
@@ -979 +979 @@
-anie)
+anie#any_adj)
@@ -984 +984 @@
-that)
+that#that_conj)
@@ -990 +990 @@
-he))
+he#he_pron))
@@ -992 +992 @@
-should)
+should#shall_v)
@@ -994 +994 @@
-neede)
+neede#need_v2)
@@ -997 +997 @@
-for)
+for#for_prep)
@@ -1000 +1000 @@
-his)
+his#his_adj)
@@ -1002 +1002 @@
-p~ticuler)))))))
+p~ticuler#particular_adj)))))))
@@ -1005 +1005 @@
-and)
+and#and_conj1)
@@ -1007 +1007 @@
-fauor)
+fauor#favour_v)
@@ -1010 +1010 @@
-him))
+him#him_pron))
@@ -1013 +1013 @@
-in)
+in#in_prep)
@@ -1016 +1016 @@
-all)
+all#all_adj)
@@ -1018 +1018 @@
-other)
+other#other_adj)
@@ -1020 +1020 @@
-his)
+his#his_adj)
@@ -1022 +1022 @@
-occasions))))
+occasions#occasion_n1))))
@@ -1025 +1025 @@
-so)
+so#so_adv)
@@ -1028 +1028 @@
-as)
+as#as_adv)
@@ -1035 +1035 @@
-he))
+he#he_pron))
@@ -1037 +1037 @@
-would)
+would#will_v1)
@@ -1039 +1039 @@
-requyte)
+requyte#requite_v)
@@ -1042 +1042 @@
-it))
+it#it_pron))
@@ -1045 +1045 @@
-in)
+in#in_prep)
@@ -1048 +1048 @@
-the)
+the#the_adj)
@@ -1050 +1050 @@
-other)
+other#other_adj)
@@ -1052 +1052 @@
-curtesie)
+curtesie#courtesy_n)
@@ -1055 +1055 @@
-to)
+to#to_prep)
@@ -1057 +1057 @@
-p~cure)
+p~cure#procure_v)
@@ -1060 +1060 @@
-yor)
+yor#your_pron)
@@ -1062 +1062 @@
-honor))
+honor#honour_n))
@@ -1065 +1065 @@
-to)
+to#to_prep)
@@ -1067 +1067 @@
-receiue)
+receiue#receive_v)
@@ -1070 +1070 @@
-information)
+information#information_n)
@@ -1073 +1073 @@
-of)
+of#of_prep)
@@ -1076 +1076 @@
-that)
+that#that_pron)
@@ -1080 +1080 @@
-which))
+which#which_adj))
@@ -1087 +1087 @@
-shall)
+shall#shall_v)
@@ -1089 +1089 @@
-passe)
+passe#pass_v)
@@ -1092 +1092 @@
-in)
+in#in_prep)
@@ -1095 +1095 @@
-Spayne))))))))))))))))))))))
+Spayne#NA))))))))))))))))))))))
@@ -1104 +1104 @@
-He))
+He#he_pron))
@@ -1106 +1106 @@
-hearkned)
+hearkned#hearken_v)
@@ -1109 +1109 @@
-to)
+to#to_prep)
@@ -1112 +1112 @@
-this)
+this#this_pron)
@@ -1114 +1114 @@
-offer)))
+offer#offer_n)))
@@ -1122 +1122 @@
-and)
+and#and_conj1)
@@ -1126 +1126 @@
-assured)
+assured#assure_v)
@@ -1129 +1129 @@
-me))
+me#me_pron1))
@@ -1132 +1132 @@
-that)
+that#that_conj)
@@ -1136 +1136 @@
-he))
+he#he_pron))
@@ -1138 +1138 @@
-will)
+will#will_v1)
@@ -1140 +1140 @@
-geiue)
+geiue#give_v)
@@ -1143 +1143 @@
-yor)
+yor#your_pron)
@@ -1145 +1145 @@
-honor))
+honor#honour_n))
@@ -1148 +1148 @@
-all)
+all#all_adj)
@@ -1150 +1150 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -1153 +1153 @@
-in)
+in#in_prep)
@@ -1156 +1156 @@
-yor)
+yor#your_pron)
@@ -1158 +1158 @@
-desier))))))
+desier#desire_n))))))
@@ -1166 +1166 @@
-and)
+and#and_conj1)
@@ -1171 +1171 @@
-hereuppon))
+hereuppon#hereupon_adv))
@@ -1173 +1173 @@
-prayed)
+prayed#pray_v)
@@ -1176 +1176 @@
-me))
+me#me_pron1))
@@ -1179 +1179 @@
-to)
+to#to_prep)
@@ -1181 +1181 @@
-send)
+send#send_v1)
@@ -1184 +1184 @@
-in)
+in#in_prep)
@@ -1187 +1187 @@
-dilligence)))
+dilligence#diligence_n1)))
@@ -1192 +1192 @@
-to)
+to#to_prep)
@@ -1195 +1195 @@
-yor)
+yor#your_pron)
@@ -1197 +1197 @@
-honor)))
+honor#honour_n)))
@@ -1200 +1200 @@
-to)
+to#to_prep)
@@ -1202 +1202 @@
-beseech)
+beseech#beseech_v)
@@ -1205 +1205 @@
-you))
+you#you_pron))
@@ -1208 +1208 @@
-to)
+to#to_prep)
@@ -1210 +1210 @@
-be)
+be#be_v)
@@ -1212 +1212 @@
-pleased)
+pleased#please_v)
@@ -1215 +1215 @@
-to)
+to#to_prep)
@@ -1217 +1217 @@
-assist)
+assist#assist_v)
@@ -1220 +1220 @@
-him))
+him#him_pron))
@@ -1223 +1223 @@
-with)
+with#with_prep)
@@ -1226 +1226 @@
-the)
+the#the_adj)
@@ -1228 +1228 @@
-said)
+said#said_adj)
@@ -1230 +1230 @@
-pasportes)))))))
+pasportes#passport_n1)))))))
@@ -1238 +1238 @@
-but)
+but#but_prep)
@@ -1241 +1241 @@
-for)
+for#for_prep)
@@ -1244 +1244 @@
-that)
+that#that_conj)
@@ -1248 +1248 @@
-the)
+the#the_adj)
@@ -1250 +1250 @@
-nombr)
+nombr#number_n)
@@ -1254 +1254 @@
-wch))
+wch#which_adj))
@@ -1262 +1262 @@
-he))
+he#he_pron))
@@ -1264 +1264 @@
-demandeth))))
+demandeth#demand_v))))
@@ -1266 +1266 @@
-is)
+is#be_v)
@@ -1269 +1269 @@
-great))
+great#great_adj))
@@ -1274 +1274 @@
-being)
+being#be_v)
@@ -1277 +1277 @@
-twelue))))))
+twelue#twelve_adj))))))
@@ -1282 +1282 @@
-I))
+I#i_pron))
@@ -1284 +1284 @@
-feare)
+feare#fear_v)
@@ -1291 +1291 @@
-it))
+it#it_pron))
@@ -1293 +1293 @@
-wil@)
+wil@#will_v1)
@@ -1295 +1295 @@
-@be)
+@be#be_v)
@@ -1298 +1298 @@
-an)
+an#a_adj2)
@@ -1300 +1300 @@
-unpleasant)
+unpleasant#unpleasant_adj)
@@ -1302 +1302 @@
-motion)
+motion#motion_n)
@@ -1305 +1305 @@
-to)
+to#to_prep)
@@ -1308 +1308 @@
-yor)
+yor#your_pron)
@@ -1310 +1310 @@
-honor)))))))
+honor#honour_n)))))))
@@ -1317 +1317 @@
-yett))
+yett#yet_adv))
@@ -1320 +1320 @@
-I))
+I#i_pron))
@@ -1322 +1322 @@
-thought)
+thought#think_v2)
@@ -1328 +1328 @@
-good))
+good#good_adj))
@@ -1331 +1331 @@
-to)
+to#to_prep)
@@ -1333 +1333 @@
-p~sent)
+p~sent#present_v)
@@ -1336 +1336 @@
-it))
+it#it_pron))
@@ -1339 +1339 @@
-to)
+to#to_prep)
@@ -1343 +1343 @@
-yor)
+yor#your_pron)
@@ -1345 +1345 @@
-honors))
+honors#honour_n))
@@ -1347 +1347 @@
-consideration)))))
+consideration#consideration_n)))))
@@ -1351 +1351 @@
-first)
+first#first_adj)
@@ -1353 +1353 @@
-for)
+for#for_prep)
@@ -1356 +1356 @@
-that)
+that#that_conj)
@@ -1360 +1360 @@
-it))
+it#it_pron))
@@ -1362 +1362 @@
-is)
+is#be_v)
@@ -1365 +1365 @@
-a)
+a#a_adj2)
@@ -1367 +1367 @@
-spetiall)
+spetiall#special_adj)
@@ -1369 +1369 @@
-meanes)
+meanes#mean_n2)
@@ -1372 +1372 @@
-to)
+to#to_prep)
@@ -1374 +1374 @@
-p~cure)
+p~cure#procure_v)
@@ -1377 +1377 @@
-yor)
+yor#your_pron)
@@ -1379 +1379 @@
-honor))
+honor#honour_n))
@@ -1382 +1382 @@
-to)
+to#to_prep)
@@ -1384 +1384 @@
-receiue)
+receiue#receive_v)
@@ -1388 +1388 @@
-verie)
+verie#very_adj)
@@ -1390 +1390 @@
-goode)
+goode#good_adj)
@@ -1393 +1393 @@
-&)
+&#and_conj1)
@@ -1396 +1396 @@
-frequent))))
+frequent#frequent_adj))))
@@ -1398 +1398 @@
-aduertisements))
+aduertisements#advertisement_n))
@@ -1403 +1403 @@
-and)
+and#and_conj1)
@@ -1406 +1406 @@
-wthout)
+wthout#without_adv)
@@ -1409 +1409 @@
-anie)
+anie#any_adj)
@@ -1411 +1411 @@
-charge)
+charge#charge_n)
@@ -1414 +1414 @@
-to)
+to#to_prep)
@@ -1417 +1417 @@
-you))))))))))))
+you#you_pron))))))))))))
@@ -1422 +1422 @@
-and)
+and#and_conj1)
@@ -1425 +1425 @@
-secondlie)
+secondlie#secondly_adv)
@@ -1427 +1427 @@
-for)
+for#for_prep)
@@ -1430 +1430 @@
-that)
+that#that_conj)
@@ -1434 +1434 @@
-I))
+I#i_pron))
@@ -1436 +1436 @@
-fynd)
+fynd#find_v)
@@ -1444 +1444 @@
-these)
+these#these_pron)
@@ -1446 +1446 @@
-people))
+people#people_n))
@@ -1448 +1448 @@
-are)
+are#be_v)
@@ -1450 +1450 @@
-resolued)
+resolued#resolve_v)
@@ -1453 +1453 @@
-to)
+to#to_prep)
@@ -1455 +1455 @@
-mainteyne)
+mainteyne#maintain_v)
@@ -1458 +1458 @@
-that)
+that#that_pron)
@@ -1460 +1460 @@
-librtie)
+librtie#liberty_n)
@@ -1463 +1463 @@
-to)
+to#to_prep)
@@ -1465 +1465 @@
-Carrie)
+Carrie#carry_v)
@@ -1468 +1468 @@
-Corne))
+Corne#corn_n1))
@@ -1471 +1471 @@
-into)
+into#into_prep)
@@ -1474 +1474 @@
-Spayne)))))))
+Spayne#NA)))))))
@@ -1479 +1479 @@
-and)
+and#and_conj1)
@@ -1485,3 +1485,3 @@
-thereuppon))
-(HVP
-haue)
+thereuppon#thereupon_adv))
+(HVP
+haue#have_v)
@@ -1489 +1489 @@
-geiuen)
+geiuen#give_v)
@@ -1492 +1492 @@
-assurance)
+assurance#assurance_n)
@@ -1495 +1495 @@
-to)
+to#to_prep)
@@ -1499 +1499 @@
-Zamett))
+Zamett#NA))
@@ -1502 +1502 @@
-&)
+&#and_conj1)
@@ -1505 +1505 @@
-the)
+the#the_adj)
@@ -1507 +1507 @@
-rest)))))
+rest#rest_n2)))))
@@ -1510 +1510 @@
-to)
+to#to_prep)
@@ -1512 +1512 @@
-staie)
+staie#stay_v1)
@@ -1516 +1516 @@
-all)
+all#all_adj)
@@ -1518 +1518 @@
-englishe)
+englishe#english_adj)
@@ -1520 +1520 @@
-mens))
+mens#man_n1))
@@ -1522 +1522 @@
-goodes))
+goodes#good_adj))
@@ -1525 +1525 @@
-for)
+for#for_prep)
@@ -1528 +1528 @@
-their)
+their#their_adj)
@@ -1530 +1530 @@
-recompence)))
+recompence#recompense_n)))
@@ -1535 +1535 @@
-if)
+if#if_conj)
@@ -1542 +1542 @@
-staie)
+staie#stay_n3)
@@ -1546 +1546 @@
-be)
+be#be_v)
@@ -1548 +1548 @@
-made)
+made#make_v1)
@@ -1551 +1551 @@
-of)
+of#of_prep)
@@ -1554 +1554 @@
-their)
+their#their_adj)
@@ -1556 +1556 @@
-shippes)))
+shippes#ship_n1)))
@@ -1559 +1559 @@
-by)
+by#by_prep)
@@ -1563 +1563 @@
-her)
+her#her_pron2)
@@ -1565 +1565 @@
-maties))
+maties#matesy_n))
@@ -1567 +1567 @@
-subiectes)))))))))))))))))
+subiectes#subject_n)))))))))))))))))
@@ -1576 +1576 @@
-To)
+To#to_prep)
@@ -1578 +1578 @@
-auoyde)
+auoyde#avoid_v)
@@ -1581 +1581 @@
-the)
+the#the_adj)
@@ -1583 +1583 @@
-sendinge)
+sendinge#sending_n)
@@ -1586 +1586 @@
-of)
+of#of_prep)
@@ -1590 +1590 @@
-so)
+so#so_adv)
@@ -1592 +1592 @@
-greate))
+greate#great_adj))
@@ -1594 +1594 @@
-nombr)
+nombr#number_n)
@@ -1597 +1597 @@
-of)
+of#of_prep)
@@ -1600 +1600 @@
-pasportes)))
+pasportes#passport_n1)))
@@ -1605 +1605 @@
-as)
+as#as_adv)
@@ -1616 +1616 @@
-since))
+since#since_adv))
@@ -1619 +1619 @@
-he))
+he#he_pron))
@@ -1621 +1621 @@
-reqeyred)
+reqeyred#require_v)
@@ -1624 +1624 @@
-of)
+of#of_prep)
@@ -1627 +1627 @@
-me))))))))))
+me#me_pron1))))))))))
@@ -1632 +1632 @@
-I))
+I#i_pron))
@@ -1634 +1634 @@
-undertook)
+undertook#undertake_v)
@@ -1637 +1637 @@
-that)
+that#that_conj)
@@ -1641 +1641 @@
-yor)
+yor#your_pron)
@@ -1643 +1643 @@
-honor))
+honor#honour_n))
@@ -1645 +1645 @@
-would)
+would#will_v1)
@@ -1647 +1647 @@
-be)
+be#be_v)
@@ -1649 +1649 @@
-pleased)
+pleased#please_v)
@@ -1652 +1652 @@
-to)
+to#to_prep)
@@ -1654 +1654 @@
-p~cure)
+p~cure#procure_v)
@@ -1657 +1657 @@
-the)
+the#the_adj)
@@ -1659 +1659 @@
-release)
+release#release_n)
@@ -1662 +1662 @@
-of)
+of#of_prep)
@@ -1665 +1665 @@
-anie)
+anie#any_adj)
@@ -1668 +1668 @@
-of)
+of#of_prep)
@@ -1671 +1671 @@
-his)
+his#his_adj)
@@ -1673 +1673 @@
-shippes)))
+shippes#ship_n1)))
@@ -1678 +1678 @@
-that)
+that#that_conj)
@@ -1683 +1683 @@
-should)
+should#shall_v)
@@ -1685 +1685 @@
-be)
+be#be_v)
@@ -1687 +1687 @@
-staied))))))
+staied#stay_v1))))))
@@ -1693 +1693 @@
-wch))
+wch#which_adj))
@@ -1699 +1699 @@
-although)
+although#although_conj)
@@ -1706 +1706 @@
-he))
+he#he_pron))
@@ -1708 +1708 @@
-acknowledged)
+acknowledged#acknowledge_v)
@@ -1713 +1713 @@
-to)
+to#to_prep)
@@ -1715 +1715 @@
-be)
+be#be_v)
@@ -1718 +1718 @@
-a)
+a#a_adj2)
@@ -1720 +1720 @@
-fauor))))))
+fauor#favour_n))))))
@@ -1723 +1723 @@
-yett))
+yett#yet_adv))
@@ -1726 +1726 @@
-it))
+it#it_pron))
@@ -1728 +1728 @@
-did)
+did#do_v)
@@ -1730 +1730 @@
-not)
+not#not_adv)
@@ -1732 +1732 @@
-content)
+content#content_v)
@@ -1735 +1735 @@
-him))
+him#him_pron))
@@ -1740 +1740 @@
-for)
+for#for_prep)
@@ -1743 +1743 @@
-that)
+that#that_conj)
@@ -1747 +1747 @@
-he))
+he#he_pron))
@@ -1749 +1749 @@
-feareth)
+feareth#fear_v)
@@ -1752 +1752 @@
-that)
+that#that_conj)
@@ -1756 +1756 @@
-being)
+being#be_v)
@@ -1759 +1759 @@
-subiect)
+subiect#subject_adj)
@@ -1762 +1762 @@
-to)
+to#to_prep)
@@ -1765 +1765 @@
-that)
+that#that_pron)
@@ -1767 +1767 @@
-staie)))))
+staie#stay_n3)))))
@@ -1772 +1772 @@
-it))
+it#it_pron))
@@ -1774 +1774 @@
-will)
+will#will_v1)
@@ -1776 +1776 @@
-make)
+make#make_v1)
@@ -1781 +1781 @@
-him))
+him#him_pron))
@@ -1783 +1783 @@
-to)
+to#to_prep)
@@ -1785 +1785 @@
-loose)
+loose#lose_v1)
@@ -1788 +1788 @@
-the)
+the#the_adj)
@@ -1790 +1790 @@
-benefitt)
+benefitt#benefit_n)
@@ -1793 +1793 @@
-of)
+of#of_prep)
@@ -1796 +1796 @@
-his)
+his#his_adj)
@@ -1798 +1798 @@
-markett)))))
+markett#market_n)))))
@@ -1801 +1801 @@
-&)
+&#and_conj1)
@@ -1806 +1806 @@
-hazard)
+hazard#hazard_v)
@@ -1809 +1809 @@
-the)
+the#the_adj)
@@ -1811 +1811 @@
-p~ishing)
+p~ishing#perishing_n)
@@ -1814 +1814 @@
-of)
+of#of_prep)
@@ -1817 +1817 @@
-the)
+the#the_adj)
@@ -1819 +1819 @@
-Corne)))))))))))))))))
+Corne#corn_n1)))))))))))))))))
@@ -1828 +1828 @@
-He))
+He#he_pron))
@@ -1830 +1830 @@
-hath)
+hath#have_v)
@@ -1833 +1833 @@
-alreadie))
+alreadie#already_adj))
@@ -1835 +1835 @@
-sent)
+sent#send_v1)
@@ -1838 +1838 @@
-awaie))
+awaie#away_adv))
@@ -1841 +1841 @@
-some)
+some#some_pron)
@@ -1843 +1843 @@
-shippes)))
+shippes#ship_n1)))
@@ -1849 +1849 @@
-&)
+&#and_conj1)
@@ -1853 +1853 @@
-desireth)
+desireth#desire_v)
@@ -1856 +1856 @@
-these)
+these#these_pron)
@@ -1858 +1858 @@
-pasportes))
+pasportes#passport_n1))
@@ -1861 +1861 @@
-for)
+for#for_prep)
@@ -1864 +1864 @@
-those)
+those#those_pron)
@@ -1866 +1866 @@
-vessells)
+vessells#vessel_n1)
@@ -1870 +1870 @@
-wch))
+wch#which_adj))
@@ -1877 +1877 @@
-are)
+are#be_v)
@@ -1880 +1880 @@
-yett))
+yett#yet_adv))
@@ -1883 +1883 @@
-behind))))
+behind#behind_adv))))
@@ -1889 +1889 @@
-wch))
+wch#which_adj))
@@ -1895 +1895 @@
-he))
+he#he_pron))
@@ -1897 +1897 @@
-saieth)
+saieth#say_v1)
@@ -1905 +1905 @@
-are)
+are#be_v)
@@ -1907 +1907 @@
-not)
+not#not_adv)
@@ -1910 +1910 @@
-aboue)
+aboue#above_adv)
@@ -1913 +1913 @@
-the)
+the#the_adj)
@@ -1915 +1915 @@
-burthen)
+burthen#burden_n)
@@ -1918 +1918 @@
-of)
+of#of_prep)
@@ -1921 +1921 @@
-threescoore)
+threescoore#NA)
@@ -1923 +1923 @@
-tonnes))))))))))))
+tonnes#ton_n1))))))))))))
@@ -1929 +1929 @@
-but)
+but#but_prep)
@@ -1932 +1932 @@
-thereof))
+thereof#thereof_adv))
@@ -1935 +1935 @@
-I))
+I#i_pron))
@@ -1937 +1937 @@
-can@)
+can@#can_v1)
@@ -1939 +1939 @@
-@not)
+@not#not_adv)
@@ -1941 +1941 @@
-geiue)
+geiue#give_v)
@@ -1944 +1944 @@
-assurance))
+assurance#assurance_n))
@@ -1952 +1952 @@
-and)
+and#and_conj1)
@@ -1956 +1956 @@
-doe)
+doe#do_v)
@@ -1959 +1959 @@
-humbly))
+humbly#humbly_adv))
@@ -1961 +1961 @@
-submitt)
+submitt#submit_v)
@@ -1964 +1964 @@
-the)
+the#the_adj)
@@ -1966 +1966 @@
-iudgement)
+iudgement#judgement_n)
@@ -1969 +1969 @@
-of)
+of#of_prep)
@@ -1972 +1972 @@
-the)
+the#the_adj)
@@ -1974 +1974 @@
-conveniencie)
+conveniencie#conveniency_n)
@@ -1979 +1979 @@
-or)
+or#or_conj2)
@@ -1981 +1981 @@
-not))
+not#not_adv))
@@ -1986 +1986 @@
-of)
+of#of_prep)
@@ -1989 +1989 @@
-his)
+his#his_adj)
@@ -1991 +1991 @@
-request))))))
+request#request_n1))))))
@@ -1996 +1996 @@
-to)
+to#to_prep)
@@ -2000 +2000 @@
-yor)
+yor#your_pron)
@@ -2002 +2002 @@
-honors))
+honors#honour_n))
@@ -2004 +2004 @@
-best)
+best#best_adj)
@@ -2006 +2006 @@
-consideration)))
+consideration#consideration_n)))
@@ -2014 +2014 @@
-And)
+And#and_conj1)
@@ -2019 +2019 @@
-where))
+where#where_adv))
@@ -2027 +2027 @@
-happelie))
+happelie#happily_adv))
@@ -2030 +2030 @@
-it))
+it#it_pron))
@@ -2032 +2032 @@
-may)
+may#may_v1)
@@ -2034 +2034 @@
-be)
+be#be_v)
@@ -2036 +2036 @@
-thought)
+thought#think_v2)
@@ -2039 +2039 @@
-that)
+that#that_conj)
@@ -2043 +2043 @@
-I))
+I#i_pron))
@@ -2045 +2045 @@
-seeke)
+seeke#seek_v)
@@ -2049 +2049 @@
-thereby))
+thereby#thereby_adv))
@@ -2051 +2051 @@
-to)
+to#to_prep)
@@ -2053 +2053 @@
-p~cure)
+p~cure#procure_v)
@@ -2056 +2056 @@
-some)
+some#some_pron)
@@ -2058 +2058 @@
-p~fitt))
+p~fitt#profit_n))
@@ -2061 +2061 @@
-to)
+to#to_prep)
@@ -2064 +2064 @@
-myself)))))))))
+myself#myself_pron)))))))))
@@ -2067 +2067 @@
-I))
+I#i_pron))
@@ -2069 +2069 @@
-beseech)
+beseech#beseech_v)
@@ -2072 +2072 @@
-yor)
+yor#your_pron)
@@ -2074 +2074 @@
-honor))
+honor#honour_n))
@@ -2077 +2077 @@
-to)
+to#to_prep)
@@ -2079 +2079 @@
-beleeue)
+beleeue#believe_v)
@@ -2082 +2082 @@
-my)
+my#my_adj)
@@ -2084 +2084 @@
-poor)
+poor#poor_adj)
@@ -2086 +2086 @@
-p~testation)
+p~testation#protestation_n)
@@ -2090 +2090 @@
-wch))
+wch#which_adj))
@@ -2098 +2098 @@
-I))
+I#i_pron))
@@ -2100 +2100 @@
-will)
+will#will_v1)
@@ -2102 +2102 @@
-not)
+not#not_adv)
@@ -2104 +2104 @@
-falsifie)
+falsifie#falsify_v)
@@ -2107 +2107 @@
-to)
+to#to_prep)
@@ -2110 +2110 @@
-you)))))
+you#you_pron)))))
@@ -2115 +2115 @@
-that)
+that#that_conj)
@@ -2119 +2119 @@
-I))
+I#i_pron))
@@ -2121 +2121 @@
-haue)
+haue#have_v)
@@ -2123 +2123 @@
-not)
+not#not_adv)
@@ -2126 +2126 @@
-therein))
+therein#therein_adv))
@@ -2132 +2132 @@
-anie)
+anie#any_adj)
@@ -2134 +2134 @@
-desseing)
+desseing#design_n)
@@ -2137 +2137 @@
-or)
+or#or_conj2)
@@ -2140 +2140 @@
-interest)))
+interest#interest_n)))
@@ -2143 +2143 @@
-of)
+of#of_prep)
@@ -2146 +2146 @@
-privatt)
+privatt#private_adj1)
@@ -2148 +2148 @@
-benefitt))))
+benefitt#benefit_n))))
@@ -2153 +2153 @@
-but)
+but#but_prep)
@@ -2156 +2156 @@
-onlie)
+onlie#only_adj)
@@ -2158 +2158 @@
-a)
+a#a_adj2)
@@ -2160 +2160 @@
-superstious)
+superstious#superstitious_adj)
@@ -2162 +2162 @@
-desire)
+desire#desire_n)
@@ -2165 +2165 @@
-to)
+to#to_prep)
@@ -2167 +2167 @@
-serue)
+serue#serve_v1)
@@ -2170 +2170 @@
-yor)
+yor#your_pron)
@@ -2172 +2172 @@
-honor))
+honor#honour_n))
@@ -2175 +2175 @@
-in)
+in#in_prep)
@@ -2178 +2178 @@
-that)
+that#that_pron)
@@ -2182 +2182 @@
-wch))
+wch#which_adj))
@@ -2191 +2191 @@
-you))
+you#you_pron))
@@ -2193 +2193 @@
-haue)
+haue#have_v)
@@ -2197 +2197 @@
-so)
+so#so_adv)
@@ -2199 +2199 @@
-much)))
+much#much_adj)))
@@ -2202 +2202 @@
-in)
+in#in_prep)
@@ -2205 +2205 @@
-recommendacion))))
+recommendacion#recommendation_n))))
@@ -2210 +2210 @@
-and)
+and#and_conj1)
@@ -2215 +2215 @@
-is)
+is#be_v)
@@ -2218 +2218 @@
-of)
+of#of_prep)
@@ -2222 +2222 @@
-so)
+so#so_adv)
@@ -2224 +2224 @@
-great))
+great#great_adj))
@@ -2226 +2226 @@
-importance)))))))))))))))))
+importance#importance_n)))))))))))))))))
@@ -2235 +2235 @@
-It))
+It#it_pron))
@@ -2237 +2237 @@
-is)
+is#be_v)
@@ -2239 +2239 @@
-desired)
+desired#desire_v)
@@ -2242 +2242 @@
-that)
+that#that_conj)
@@ -2247 +2247 @@
-the)
+the#the_adj)
@@ -2249 +2249 @@
-pasportes))
+pasportes#passport_n1))
@@ -2251 +2251 @@
-may)
+may#may_v1)
@@ -2253 +2253 @@
-runne)
+runne#run_v)
@@ -2256 +2256 @@
-in)
+in#in_prep)
@@ -2259 +2259 @@
-the)
+the#the_adj)
@@ -2261 +2261 @@
-name)
+name#name_n)
@@ -2265 +2265 @@
-Peter)
+Peter#NA)
@@ -2267 +2267 @@
-Bernardin)
+Bernardin#NA)
@@ -2269 +2269 @@
-Massey))
+Massey#NA))
@@ -2272 +2272 @@
-and)
+and#and_conj1)
@@ -2275 +2275 @@
-Anthonie)
+Anthonie#NA)
@@ -2277 +2277 @@
-Balbany)))))))
+Balbany#NA)))))))
@@ -2280 +2280 @@
-and)
+and#and_conj1)
@@ -2284 +2284 @@
-blanckes))
+blanckes#blank_n))
@@ -2286 +2286 @@
-to)
+to#to_prep)
@@ -2288 +2288 @@
-be)
+be#be_v)
@@ -2290 +2290 @@
-left)
+left#leave_v1)
@@ -2293 +2293 @@
-for)
+for#for_prep)
@@ -2296 +2296 @@
-the)
+the#the_adj)
@@ -2298 +2298 @@
-names)
+names#name_n)
@@ -2302 +2302 @@
-of)
+of#of_prep)
@@ -2305 +2305 @@
-the)
+the#the_adj)
@@ -2307 +2307 @@
-shippes)))
+shippes#ship_n1)))
@@ -2312 +2312 @@
-and)
+and#and_conj1)
@@ -2315 +2315 @@
-of)
+of#of_prep)
@@ -2318 +2318 @@
-the)
+the#the_adj)
@@ -2320 +2320 @@
-Masters)))))))))))
+Masters#master_n1)))))))))))
@@ -2329 +2329 @@
-I))
+I#i_pron))
@@ -2331 +2331 @@
-hope)
+hope#hope_v)
@@ -2333 +2333 @@
-also)
+also#also_adv)
@@ -2336 +2336 @@
-to)
+to#to_prep)
@@ -2338 +2338 @@
-p~cure)
+p~cure#procure_v)
@@ -2341 +2341 @@
-that)
+that#that_conj)
@@ -2346 +2346 @@
-the)
+the#the_adj)
@@ -2348 +2348 @@
-Ambr)
+Ambr#ambassador_n)
@@ -2351 +2351 @@
-of)
+of#of_prep)
@@ -2354 +2354 @@
-Venice))))
+Venice#NA))))
@@ -2356 +2356 @@
-shall)
+shall#shall_v)
@@ -2358 +2358 @@
-drawe)
+drawe#draw_v)
@@ -2361 +2361 @@
-often))
+often#often_adv))
@@ -2364 +2364 @@
-aduertisementes))
+aduertisementes#advertisement_n))
@@ -2367 +2367 @@
-from)
+from#from_prep)
@@ -2370 +2370 @@
-their)
+their#their_adj)
@@ -2372 +2372 @@
-Ambr.)
+Ambr.#ambassador_n)
@@ -2375 +2375 @@
-in)
+in#in_prep)
@@ -2378 +2378 @@
-Spayne))))))
+Spayne#NA))))))
@@ -2381 +2381 @@
-and)
+and#and_conj1)
@@ -2384 +2384 @@
-send)
+send#send_v1)
@@ -2387 +2387 @@
-them))
+them#them_pron))
@@ -2391 +2391 @@
-from)
+from#from_prep)
@@ -2394 +2394 @@
-tyme)))
+tyme#time_n)))
@@ -2397 +2397 @@
-to)
+to#to_prep)
@@ -2400 +2400 @@
-tyme))))
+tyme#time_n))))
@@ -2403 +2403 @@
-to)
+to#to_prep)
@@ -2406 +2406 @@
-yor)
+yor#your_pron)
@@ -2408 +2408 @@
-honor))))))))
+honor#honour_n))))))))
@@ -2416 +2416 @@
-and)
+and#and_conj1)
@@ -2420 +2420 @@
-will)
+will#will_v1)
@@ -2422 +2422 @@
-emploie)
+emploie#employ_v)
@@ -2425 +2425 @@
-all)
+all#all_adj)
@@ -2427 +2427 @@
-other)
+other#other_adj)
@@ -2429 +2429 @@
-meanes)
+meanes#mean_n2)
@@ -2432 +2432 @@
-possible)))
+possible#possible_adj)))
@@ -2437 +2437 @@
-that)
+that#that_conj)
@@ -2442 +2442 @@
-by)
+by#by_prep)
@@ -2445 +2445 @@
-the)
+the#the_adj)
@@ -2447 +2447 @@
-plurallitie)
+plurallitie#plurality_n)
@@ -2450 +2450 @@
-of)
+of#of_prep)
@@ -2453 +2453 @@
-advices)))))
+advices#advice_n)))))
@@ -2456 +2456 @@
-yor)
+yor#your_pron)
@@ -2458 +2458 @@
-honor))
+honor#honour_n))
@@ -2460 +2460 @@
-maie)
+maie#may_v1)
@@ -2462 +2462 @@
-be)
+be#be_v)
@@ -2467 +2467 @@
-the)
+the#the_adj)
@@ -2469 +2469 @@
-better))
+better#better_adj))
@@ -2471 +2471 @@
-serued))
+serued#serve_v1))
@@ -2474 +2474 @@
-and)
+and#and_conj1)
@@ -2477 +2477 @@
-make)
+make#make_v1)
@@ -2480 +2480 @@
-the)
+the#the_adj)
@@ -2482 +2482 @@
-truer)
+truer#true_adj)
@@ -2484 +2484 @@
-iudgement)
+iudgement#judgement_n)
@@ -2487 +2487 @@
-of)
+of#of_prep)
@@ -2490 +2490 @@
-thinges))))))))
+thinges#thing_n1))))))))
@@ -2499 +2499 @@
-I))
+I#i_pron))
@@ -2501 +2501 @@
-doe)
+doe#do_v)
@@ -2504 +2504 @@
-contynuallie))
+contynuallie#continually_adv))
@@ -2506 +2506 @@
-sollicitt)
+sollicitt#solicit_v)
@@ -2509 +2509 @@
-the)
+the#the_adj)
@@ -2511 +2511 @@
-paiement)
+paiement#payment_n1)
@@ -2514 +2514 @@
-of)
+of#of_prep)
@@ -2517 +2517 @@
-the)
+the#the_adj)
@@ -2519 +2519 @@
-20000)
+20000#NA)
@@ -2521 +2521 @@
-Crownes))))
+Crownes#crown_n))))
@@ -2524 +2524 @@
-as)
+as#as_adv)
@@ -2535 +2535 @@
-yor)
+yor#your_pron)
@@ -2537 +2537 @@
-honor))
+honor#honour_n))
@@ -2539 +2539 @@
-demandeth)
+demandeth#demand_v)
@@ -2542 +2542 @@
-me)))))
+me#me_pron1)))))
@@ -2550 +2550 @@
-and)
+and#and_conj1)
@@ -2554 +2554 @@
-am)
+am#be_v)
@@ -2557 +2557 @@
-still))
+still#still_adv))
@@ -2560 +2560 @@
-confidentlie))
+confidentlie#confidently_adv))
@@ -2562 +2562 @@
-assured)
+assured#assure_v)
@@ -2565 +2565 @@
-by)
+by#by_prep)
@@ -2568 +2568 @@
-Monsr)
+Monsr#NA)
@@ -2570 +2570 @@
-de)
+de#NA)
@@ -2572 +2572 @@
-Rhosny)
+Rhosny#NA)
@@ -2576 +2576 @@
-who))
+who#who_pron))
@@ -2583 +2583 @@
-is)
+is#be_v)
@@ -2586 +2586 @@
-the)
+the#the_adj)
@@ -2589 +2589 @@
-fac)
+fac#NA)
@@ -2591 +2591 @@
-totum))
+totum#NA))
@@ -2594 +2594 @@
-here))
+here#here_adv))
@@ -2597 +2597 @@
-for)
+for#for_prep)
@@ -2600 +2600 @@
-monie)
+monie#money_n)
@@ -2602 +2602 @@
-matters))))))))
+matters#matter_n1))))))))
@@ -2606 +2606 @@
-that)
+that#that_conj)
@@ -2610 +2610 @@
-wthout)
+wthout#without_adv)
@@ -2613 +2613 @@
-faile)))
+faile#fail_n2)))
@@ -2616 +2616 @@
-it))
+it#it_pron))
@@ -2618 +2618 @@
-shal@)
+shal@#shall_v)
@@ -2620 +2620 @@
-@be)
+@be#be_v)
@@ -2622 +2622 @@
-answeared)
+answeared#answer_v)
@@ -2625 +2625 @@
-out)
+out#out_adv)
@@ -2627 +2627 @@
-of)
+of#of_prep)
@@ -2631 +2631 @@
-this)
+this#this_pron)
@@ -2633 +2633 @@
-yeares))
+yeares#year_n))
@@ -2635 +2635 @@
-receipt)))))
+receipt#receipt_n)))))
@@ -2640 +2640 @@
-but)
+but#but_prep)
@@ -2643 +2643 @@
-that)
+that#that_conj)
@@ -2647 +2647 @@
-it))
+it#it_pron))
@@ -2649 +2649 @@
-can@)
+can@#can_v1)
@@ -2651 +2651 @@
-@not)
+@not#not_adv)
@@ -2653 +2653 @@
-be)
+be#be_v)
@@ -2656 +2656 @@
-soone))
+soone#soon_adv))
@@ -2658 +2658 @@
-leuyed)))))
+leuyed#levy_v)))))
@@ -2667 +2667 @@
-He))
+He#he_pron))
@@ -2669 +2669 @@
-toulde)
+toulde#tell_v)
@@ -2672 +2672 @@
-me))
+me#me_pron1))
@@ -2675 +2675 @@
-that)
+that#that_conj)
@@ -2679 +2679 @@
-he))
+he#he_pron))
@@ -2681 +2681 @@
-understoode)
+understoode#understand_v)
@@ -2688 +2688 @@
-her)
+her#her_pron2)
@@ -2690 +2690 @@
-matie))
+matie#majesty_n))
@@ -2692 +2692 @@
-had)
+had#have_v)
@@ -2694 +2694 @@
-geiuen)
+geiuen#give_v)
@@ -2697 +2697 @@
-the)
+the#the_adj)
@@ -2699 +2699 @@
-said)
+said#said_adj)
@@ -2703 +2703 @@
-monie))
+monie#money_n))
@@ -2706 +2706 @@
-unto)
+unto#unto_prep)
@@ -2709 +2709 @@
-some)
+some#some_pron)
@@ -2712 +2712 @@
-aboute)
+aboute#about_adv)
@@ -2715 +2715 @@
-her))))))))))
+her#her_pron1))))))))))
@@ -2721 +2721 @@
-but)
+but#but_prep)
@@ -2724 +2724 @@
-I))
+I#i_pron))
@@ -2726 +2726 @@
-assured)
+assured#assure_v)
@@ -2729 +2729 @@
-him))
+him#him_pron))
@@ -2734 +2734 @@
-that)
+that#that_conj)
@@ -2738 +2738 @@
-it))
+it#it_pron))
@@ -2740 +2740 @@
-is)
+is#be_v)
@@ -2743 +2743 @@
-a)
+a#a_adj2)
@@ -2745 +2745 @@
-false)
+false#false_adj)
@@ -2747 +2747 @@
-information))))
+information#information_n))))
@@ -2756 +2756 @@
-Monsr)
+Monsr#NA)
@@ -2758 +2758 @@
-de)
+de#NA)
@@ -2760 +2760 @@
-Villeroy))
+Villeroy#NA))
@@ -2762 +2762 @@
-did)
+did#do_v)
@@ -2765 +2765 @@
-verie)
+verie#very_adj)
@@ -2767 +2767 @@
-kindlie))
+kindlie#kindly_adv))
@@ -2769 +2769 @@
-take)
+take#take_v)
@@ -2773 +2773 @@
-yor)
+yor#your_pron)
@@ -2775 +2775 @@
-honors))
+honors#honour_n))
@@ -2777 +2777 @@
-letter)))
+letter#letter_n1)))
@@ -2783 +2783 @@
-and)
+and#and_conj1)
@@ -2787 +2787 @@
-will)
+will#will_v1)
@@ -2790 +2790 @@
-by)
+by#by_prep)
@@ -2793 +2793 @@
-the)
+the#the_adj)
@@ -2795 +2795 @@
-Ambr.)))
+Ambr.#ambassador_n)))
@@ -2797 +2797 @@
-retourne)
+retourne#return_v1)
@@ -2800 +2800 @@
-yor)
+yor#your_pron)
@@ -2802 +2802 @@
-honor))
+honor#honour_n))
@@ -2805 +2805 @@
-answeare)
+answeare#answer_n)
@@ -2808 +2808 @@
-thereunto)))
+thereunto#thereunto_adv)))
@@ -2817 +2817 @@
-I))
+I#i_pron))
@@ -2819 +2819 @@
-did)
+did#do_v)
@@ -2821 +2821 @@
-also)
+also#also_adv)
@@ -2823 +2823 @@
-acquaynte)
+acquaynte#acquaint_v)
@@ -2826 +2826 @@
-him))
+him#him_pron))
@@ -2831 +2831 @@
-in)
+in#in_prep)
@@ -2834 +2834 @@
-other)
+other#other_adj)
@@ -2836 +2836 @@
-discourse)))
+discourse#discourse_n)))
@@ -2841 +2841 @@
-that)
+that#that_conj)
@@ -2845 +2845 @@
-the)
+the#the_adj)
@@ -2847 +2847 @@
-Ambr))
+Ambr#ambassador_n))
@@ -2849 +2849 @@
-had)
+had#have_v)
@@ -2851 +2851 @@
-not)
+not#not_adv)
@@ -2854 +2854 @@
-yett))
+yett#yet_adv))
@@ -2856 +2856 @@
-made)
+made#make_v1)
@@ -2859 +2859 @@
-anie)
+anie#any_adj)
@@ -2861 +2861 @@
-privatt)
+privatt#private_adj1)
@@ -2863 +2863 @@
-acquaintance)
+acquaintance#acquaintance_n)
@@ -2866 +2866 @@
-wth)
+wth#with_prep)
@@ -2869 +2869 @@
-yor)
+yor#your_pron)
@@ -2871 +2871 @@
-honor))))
+honor#honour_n))))
@@ -2877 +2877 @@
-touching)
+touching#touching_prep)
@@ -2880 +2880 @@
-the)
+the#the_adj)
@@ -2882 +2882 @@
-wch)))
+wch#which_adj)))
@@ -2890 +2890 @@
-he))
+he#he_pron))
@@ -2892 +2892 @@
-sayd)
+sayd#say_v1)
@@ -2895 +2895 @@
-that)
+that#that_conj)
@@ -2899 +2899 @@
-he))
+he#he_pron))
@@ -2901 +2901 @@
-would)
+would#will_v1)
@@ -2903 +2903 @@
-wryte)
+wryte#write_v)
@@ -2906 +2906 @@
-unto)
+unto#unto_prep)
@@ -2909 +2909 @@
-him)))))))
+him#him_pron)))))))
@@ -2914 +2914 @@
-being)
+being#be_v)
@@ -2917 +2917 @@
-a)
+a#a_adj2)
@@ -2919 +2919 @@
-faulte)
+faulte#fault_n)
@@ -2923 +2923 @@
-wch))
+wch#which_adj))
@@ -2931 +2931 @@
-his)
+his#his_adj)
@@ -2933 +2933 @@
-frendes)
+frendes#friend_n)
@@ -2936 +2936 @@
-here)))
+here#here_adv)))
@@ -2938 +2938 @@
-doe)
+doe#do_v)
@@ -2940 +2940 @@
-accuse)
+accuse#accuse_v)
@@ -2943 +2943 @@
-in)
+in#in_prep)
@@ -2946 +2946 @@
-him)))))
+him#him_pron)))))
@@ -2951 +2951 @@
-that)
+that#that_conj)
@@ -2955 +2955 @@
-he))
+he#he_pron))
@@ -2957 +2957 @@
-doth)
+doth#do_v)
@@ -2960 +2960 @@
-too)
+too#too_adv)
@@ -2962 +2962 @@
-formallie))
+formallie#formally_adv))
@@ -2964 +2964 @@
-hould)
+hould#hold_v)
@@ -2967 +2967 @@
-himself))
+himself#himself_pron))
@@ -2970 +2970 @@
-on)
+on#on_prep)
@@ -2973 +2973 @@
-his)
+his#his_adj)
@@ -2975 +2975 @@
-grauitie)))))))
+grauitie#gravity_n)))))))
@@ -2980 +2980 @@
-but)
+but#but_prep)
@@ -2982 +2982 @@
-being)
+being#be_v)
@@ -2985 +2985 @@
-otherwise))
+otherwise#otherwise_n))
@@ -2989 +2989 @@
-a)
+a#a_adj2)
@@ -2992 +2992 @@
-verie)
+verie#very_adj)
@@ -2994 +2994 @@
-honest))
+honest#honest_adj))
@@ -2996 +2996 @@
-gent))
+gent#gent_n))
@@ -3001 +3001 @@
-and)
+and#and_conj1)
@@ -3004 +3004 @@
-of)
+of#of_prep)
@@ -3007 +3007 @@
-great)
+great#great_adj)
@@ -3009 +3009 @@
-integritie))))))))
+integritie#integrity_n))))))))
@@ -3018 +3018 @@
-He))
+He#he_pron))
@@ -3020 +3020 @@
-doth)
+doth#do_v)
@@ -3022 +3022 @@
-complaine)
+complaine#complain_v)
@@ -3025 +3025 @@
-greatlie))
+greatlie#greatly_adv))
@@ -3028 +3028 @@
-in)
+in#in_prep)
@@ -3031 +3031 @@
-all)
+all#all_adj)
@@ -3033 +3033 @@
-his)
+his#his_adj)
@@ -3035 +3035 @@
-dispatches)))
+dispatches#dispatch_n)))
@@ -3039 +3039 @@
-that)
+that#that_conj)
@@ -3043 +3043 @@
-he))
+he#he_pron))
@@ -3045 +3045 @@
-can@)
+can@#can_v1)
@@ -3047 +3047 @@
-@not)
+@not#not_adv)
@@ -3049 +3049 @@
-receiue)
+receiue#receive_v)
@@ -3052 +3052 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -3055 +3055 @@
-from)
+from#from_prep)
@@ -3058 +3058 @@
-my)
+my#my_adj)
@@ -3060 +3060 @@
-lord)
+lord#lord_n)
@@ -3063 +3063 @@
-Admyrall))))
+Admyrall#admiral_n))))
@@ -3066 +3066 @@
-in)
+in#in_prep)
@@ -3069 +3069 @@
-matters))))
+matters#matter_n1))))
@@ -3072 +3072 @@
-of)
+of#of_prep)
@@ -3075 +3075 @@
-iustice)))))
+iustice#justice_n)))))
@@ -3081 +3081 @@
-That)
+That#that_conj)
@@ -3085 +3085 @@
-yor)
+yor#your_pron)
@@ -3087 +3087 @@
-honor))
+honor#honour_n))
@@ -3089 +3089 @@
-doth)
+doth#do_v)
@@ -3092 +3092 @@
-order)
+order#order_v)
@@ -3094 +3094 @@
-&)
+&#and_conj1)
@@ -3096 +3096 @@
-direct))
+direct#direct_v))
@@ -3099 +3099 @@
-verie)
+verie#very_adj)
@@ -3101 +3101 @@
-well)))))
+well#well_adv)))))
@@ -3106 +3106 @@
-but)
+but#but_prep)
@@ -3109 +3109 @@
-that)
+that#that_conj)
@@ -3113 +3113 @@
-his)
+his#his_adj)
@@ -3115 +3115 @@
-l.))
+l.#lordship_n))
@@ -3117 +3117 @@
-doth)
+doth#do_v)
@@ -3119 +3119 @@
-not)
+not#not_adv)
@@ -3121 +3121 @@
-answeare)
+answeare#answer_v)
@@ -3124 +3124 @@
-it))
+it#it_pron))
@@ -3127 +3127 @@
-in)
+in#in_prep)
@@ -3130 +3130 @@
-p~fourmance)
+p~fourmance#performance_n)
@@ -3132 +3132 @@
-&)
+&#and_conj1)
@@ -3134 +3134 @@
-execution))))))))
+execution#execution_n))))))))
@@ -3141 +3141 @@
-I))
+I#i_pron))
@@ -3143 +3143 @@
-assure)
+assure#assure_v)
@@ -3146 +3146 @@
-yor)
+yor#your_pron)
@@ -3148 +3148 @@
-honor))
+honor#honour_n))
@@ -3155 +3155 @@
-if)
+if#if_conj)
@@ -3162 +3162 @@
-it))
+it#it_pron))
@@ -3164 +3164 @@
-be)
+be#be_v)
@@ -3166 +3166 @@
-not)
+not#not_adv)
@@ -3168 +3168 @@
-p~vyded)
+p~vyded#provide_v)
@@ -3171 +3171 @@
-by)
+by#by_prep)
@@ -3174 +3174 @@
-some)
+some#some_pron)
@@ -3176 +3176 @@
-good)
+good#good_adj)
@@ -3178 +3178 @@
-establishement)))
+establishement#establishment_n)))
@@ -3181 +3181 @@
-to)
+to#to_prep)
@@ -3183 +3183 @@
-redresse)
+redresse#redress_v1)
@@ -3186 +3186 @@
-the)
+the#the_adj)
@@ -3188 +3188 @@
-disorders)
+disorders#disorder_n)
@@ -3191 +3191 @@
-of)
+of#of_prep)
@@ -3194 +3194 @@
-both)
+both#both_pron)
@@ -3196 +3196 @@
-sides))))))))
+sides#side_n1))))))))
@@ -3201 +3201 @@
-our)
+our#our_pron)
@@ -3203 +3203 @@
-poore)
+poore#poor_adj)
@@ -3205 +3205 @@
-marchantes))
+marchantes#merchant_n))
@@ -3207 +3207 @@
-wil@)
+wil@#will_v1)
@@ -3209 +3209 @@
-@be)
+@be#be_v)
@@ -3211 +3211 @@
-undonne)
+undonne#undo_v)
@@ -3214 +3214 @@
-by)
+by#by_prep)
@@ -3217 +3217 @@
-these)
+these#these_pron)
@@ -3219 +3219 @@
-differences)))))
+differences#difference_n1)))))
@@ -3227 +3227 @@
-and)
+and#and_conj1)
@@ -3230 +3230 @@
-yor)
+yor#your_pron)
@@ -3232 +3232 @@
-honor))
+honor#honour_n))
@@ -3234 +3234 @@
-will)
+will#will_v1)
@@ -3237 +3237 @@
-forthwth))
+forthwth#forthwith_adv))
@@ -3239 +3239 @@
-see)
+see#see_v)
@@ -3242 +3242 @@
-the)
+the#the_adj)
@@ -3244 +3244 @@
-effectes)
+effectes#effect_n)
@@ -3247 +3247 @@
-thereof)))
+thereof#thereof_adv)))
@@ -3256 +3256 @@
-I))
+I#i_pron))
@@ -3258 +3258 @@
-haue)
+haue#have_v)
@@ -3260 +3260 @@
-comunicated)
+comunicated#communicate_v)
@@ -3263 +3263 @@
-wth)
+wth#with_prep)
@@ -3266 +3266 @@
-one)
+one#one_adj)
@@ -3268 +3268 @@
-Smyth)))
+Smyth#NA)))
@@ -3271 +3271 @@
-the)
+the#the_adj)
@@ -3273 +3273 @@
-answeare)
+answeare#answer_n)
@@ -3277 +3277 @@
-wch))
+wch#which_adj))
@@ -3285 +3285 @@
-yor)
+yor#your_pron)
@@ -3287 +3287 @@
-honor))
+honor#honour_n))
@@ -3289 +3289 @@
-sent)
+sent#send_v1)
@@ -3292 +3292 @@
-me))
+me#me_pron1))
@@ -3295 +3295 @@
-from)
+from#from_prep)
@@ -3298 +3298 @@
-M=r=.)
+M=r=.#NA)
@@ -3300 +3300 @@
-Kelley)))))
+Kelley#NA)))))
@@ -3304 +3304 @@
-touching)
+touching#touching_prep)
@@ -3307 +3307 @@
-the)
+the#the_adj)
@@ -3309 +3309 @@
-wch)))
+wch#which_adj)))
@@ -3317 +3317 @@
-he))
+he#he_pron))
@@ -3319 +3319 @@
-hath)
+hath#have_v)
@@ -3321 +3321 @@
-thought)
+thought#think_v2)
@@ -3327 +3327 @@
-good))
+good#good_adj))
@@ -3330 +3330 @@
-to)
+to#to_prep)
@@ -3332 +3332 @@
-wryte)
+wryte#write_v)
@@ -3335 +3335 @@
-againe))
+againe#again_adv))
@@ -3338 +3338 @@
-to)
+to#to_prep)
@@ -3341 +3341 @@
-yor)
+yor#your_pron)
@@ -3343 +3343 @@
-honor))))))))
+honor#honour_n))))))))
@@ -3352 +3352 @@
-There))
+There#there_adv))
@@ -3354 +3354 @@
-is)
+is#be_v)
@@ -3357 +3357 @@
-no)
+no#no_adj)
@@ -3359 +3359 @@
-likelyhood)
+likelyhood#likelihood_n)
@@ -3362 +3362 @@
-to)
+to#to_prep)
@@ -3364 +3364 @@
-compound)
+compound#compound_v)
@@ -3368 +3368 @@
-so)
+so#so_adv)
@@ -3370 +3370 @@
-discordant))
+discordant#discordant_adj))
@@ -3372 +3372 @@
-allegations))))
+allegations#allegation_n))))
@@ -3377 +3377 @@
-unlesse)
+unlesse#unless_adv)
@@ -3385 +3385 @@
-they))
+they#they_pron))
@@ -3387 +3387 @@
-were)
+were#be_v)
@@ -3390 +3390 @@
-together))))
+together#together_adv))))
@@ -3395 +3395 @@
-or)
+or#or_conj2)
@@ -3398 +3398 @@
-that)
+that#that_conj)
@@ -3402 +3402 @@
-they))
+they#they_pron))
@@ -3404 +3404 @@
-will)
+will#will_v1)
@@ -3406 +3406 @@
-agree)
+agree#agree_v)
@@ -3410 +3410 @@
-to)
+to#to_prep)
@@ -3413 +3413 @@
-whom)))
+whom#whom_pron)))
@@ -3418 +3418 @@
-to)
+to#to_prep)
@@ -3420 +3420 @@
-referre)
+referre#refer_v)
@@ -3423 +3423 @@
-their)
+their#their_adj)
@@ -3425 +3425 @@
-differences)))))))))
+differences#difference_n1)))))))))
@@ -3433 +3433 @@
-and)
+and#and_conj1)
@@ -3436 +3436 @@
-these)
+these#these_pron)
@@ -3438 +3438 @@
-expostulations))
+expostulations#expostulation_n))
@@ -3441 +3441 @@
-in)
+in#in_prep)
@@ -3444 +3444 @@
-the)
+the#the_adj)
@@ -3446 +3446 @@
-mean)
+mean#mean_adj2)
@@ -3448 +3448 @@
-tyme)))
+tyme#time_n)))
@@ -3450 +3450 @@
-serve)
+serve#serve_v1)
@@ -3453 +3453 @@
-to)
+to#to_prep)
@@ -3456 +3456 @@
-litle)
+litle#little_adj)
@@ -3458 +3458 @@
-pourpose)))
+pourpose#purpose_n)))
@@ -3467 +3467 @@
-It))
+It#it_pron))
@@ -3469 +3469 @@
-is)
+is#be_v)
@@ -3472 +3472 @@
-verie)
+verie#very_adj)
@@ -3474 +3474 @@
-earnestlie))
+earnestlie#earnestly_adv))
@@ -3476 +3476 @@
-desired)
+desired#desire_v)
@@ -3479 +3479 @@
-that)
+that#that_conj)
@@ -3483 +3483 @@
-it))
+it#it_pron))
@@ -3485 +3485 @@
-will)
+will#will_v1)
@@ -3487 +3487 @@
-please)
+please#please_v)
@@ -3490 +3490 @@
-yor)
+yor#your_pron)
@@ -3492 +3492 @@
-honor))
+honor#honour_n))
@@ -3495 +3495 @@
-either)
+either#either_pron)
@@ -3498 +3498 @@
-to)
+to#to_prep)
@@ -3500 +3500 @@
-send)
+send#send_v1)
@@ -3503 +3503 @@
-the)
+the#the_adj)
@@ -3505 +3505 @@
-pasportes))
+pasportes#passport_n1))
@@ -3508 +3508 @@
-wth)
+wth#with_prep)
@@ -3511 +3511 @@
-expedition))))
+expedition#expedition_n))))
@@ -3514 +3514 @@
-or)
+or#or_conj2)
@@ -3516 +3516 @@
-ells)
+ells#else_adv)
@@ -3521 +3521 @@
-answeare)
+answeare#answer_v)
@@ -3524 +3524 @@
-that)
+that#that_conj)
@@ -3528 +3528 @@
-they))
+they#they_pron))
@@ -3530 +3530 @@
-can@)
+can@#can_v1)
@@ -3532 +3532 @@
-@not)
+@not#not_adv)
@@ -3534 +3534 @@
-be)
+be#be_v)
@@ -3536 +3536 @@
-getten))))))))
+getten#get_v))))))))
@@ -3539 +3539 @@
-for)
+for#for_prep)
@@ -3542 +3542 @@
-that)
+that#that_conj)
@@ -3546 +3546 @@
-they))
+they#they_pron))
@@ -3548 +3548 @@
-stay)
+stay#stay_v1)
@@ -3551 +3551 @@
-the)
+the#the_adj)
@@ -3553 +3553 @@
-dep~ture)
+dep~ture#departure_n)
@@ -3556 +3556 @@
-of)
+of#of_prep)
@@ -3559 +3559 @@
-their)
+their#their_adj)
@@ -3561 +3561 @@
-shippes))))
+shippes#ship_n1))))
@@ -3564 +3564 @@
-thereuppon)))))
+thereuppon#thereupon_adv)))))
@@ -3572 +3572 @@
-and)
+and#and_conj1)
@@ -3575 +3575 @@
-therefore))
+therefore#therefore_adv))
@@ -3578 +3578 @@
-yor)
+yor#your_pron)
@@ -3580 +3580 @@
-honor))
+honor#honour_n))
@@ -3582 +3582 @@
-shall)
+shall#shall_v)
@@ -3585 +3585 @@
-greatlie))
+greatlie#greatly_adv))
@@ -3587 +3587 @@
-satisfie)
+satisfie#satisfy_v)
@@ -3590 +3590 @@
-them))
+them#them_pron))
@@ -3593 +3593 @@
-to)
+to#to_prep)
@@ -3595 +3595 @@
-geiue)
+geiue#give_v)
@@ -3598 +3598 @@
-a)
+a#a_adj2)
@@ -3600 +3600 @@
-speedie)
+speedie#speedy_adj)
@@ -3602 +3602 @@
-resolution))
+resolution#resolution_n))
@@ -3606 +3606 @@
-wch))
+wch#which_adj))
@@ -3612 +3612 @@
-I))
+I#i_pron))
@@ -3614 +3614 @@
-beseech)
+beseech#beseech_v)
@@ -3617 +3617 @@
-yor)
+yor#your_pron)
@@ -3619 +3619 @@
-honor))
+honor#honour_n))
@@ -3622 +3622 @@
-to)
+to#to_prep)
@@ -3624 +3624 @@
-be)
+be#be_v)
@@ -3626 +3626 @@
-pleased)
+pleased#please_v)
@@ -3631 +3631 @@
-to)
+to#to_prep)
@@ -3633 +3633 @@
-doe))))))
+doe#do_v))))))
@@ -3641 +3641 @@
-And)
+And#and_conj1)
@@ -3644 +3644 @@
-craving)
+craving#crave_v)
@@ -3646 +3646 @@
-also)
+also#also_adv)
@@ -3649 +3649 @@
-p~don)
+p~don#pardon_n1)
@@ -3652 +3652 @@
-for)
+for#for_prep)
@@ -3655 +3655 @@
-hauing)
+hauing#have_v)
@@ -3657 +3657 @@
-troubled)
+troubled#trouble_v)
@@ -3660 +3660 @@
-yor)
+yor#your_pron)
@@ -3662 +3662 @@
-honor))
+honor#honour_n))
@@ -3665 +3665 @@
-wth)
+wth#with_prep)
@@ -3669 +3669 @@
-so)
+so#so_adv)
@@ -3671 +3671 @@
-tedious))
+tedious#tedious_adj))
@@ -3673 +3673 @@
-a)
+a#a_adj2)
@@ -3675 +3675 @@
-letter)))))))
+letter#letter_n1)))))))
@@ -3678 +3678 @@
-I))
+I#i_pron))
@@ -3681 +3681 @@
-most)
+most#most_adj)
@@ -3683 +3683 @@
-humbly))
+humbly#humbly_adv))
@@ -3685 +3685 @@
-take)
+take#take_v)
@@ -3688 +3688 @@
-my)
+my#my_adj)
@@ -3690 +3690 @@
-leaue))
+leaue#leave_n1))
@@ -3693 +3693 @@
-ffrom)
+ffrom#from_prep)
@@ -3696 +3696 @@
-Paris)))
+Paris#NA)))
@@ -3699 +3699 @@
-the)
+the#the_adj)
@@ -3701 +3701 @@
-second)
+second#second_adj)
@@ -3704 +3704 @@
-of)
+of#of_prep)
@@ -3707 +3707 @@
-Januarie)))
+Januarie#NA)))
@@ -3709 +3709 @@
-1598))
+1598#NA))
@@ -3718 +3718 @@
-I))
+I#i_pron))
@@ -3720 +3720 @@
-send)
+send#send_v1)
@@ -3723 +3723 @@
-yor)
+yor#your_pron)
@@ -3725 +3725 @@
-honer))
+honer#honour_n))
@@ -3728 +3728 @@
-the)
+the#the_adj)
@@ -3730 +3730 @@
-memoriall)
+memoriall#memorial_adj)
@@ -3733 +3733 @@
-delyuered)
+delyuered#deliver_v1)
@@ -3736 +3736 @@
-me))
+me#me_pron1))
@@ -3739 +3739 @@
-for)
+for#for_prep)
@@ -3742 +3742 @@
-the)
+the#the_adj)
@@ -3744 +3744 @@
-said)
+said#said_adj)
@@ -3746 +3746 @@
-pasportes)))))
+pasportes#passport_n1)))))
@@ -3755 +3755 @@
-Yor)
+Yor#your_pron)
@@ -3757 +3757 @@
-honors))
+honors#honour_n))
@@ -3761 +3761 @@
-most)
+most#most_adj)
@@ -3763 +3763 @@
-humblie))
+humblie#humbly_adv))
@@ -3765 +3765 @@
-bounden))
+bounden#bounden_adj))
@@ -3767 +3767 @@
-Tho:)
+Tho:#NA)
@@ -3769 +3769 @@
-Edmondes)
+Edmondes#NA)
@@ -3777 +3777 @@
-1598)
+1598#NA)
@@ -3779 +3779 @@
-2)
+2#two_adj)
@@ -3781 +3781 @@
-Jan)
+Jan#NA)
@@ -3790 +3790 @@
-M=r=)
+M=r=#NA)
@@ -3792 +3792 @@
-Edmondes))
+Edmondes#NA))
@@ -3795 +3795 @@
-to)
+to#to_prep)
@@ -3798 +3798 @@
-my)
+my#my_adj)
@@ -3800 +3800 @@
-M=r=)))
+M=r=#master_n1)))
@@ -3808 +3808 @@
-His)
+His#his_adj)
@@ -3810 +3810 @@
-endevor)
+endevor#endeavour_n)
@@ -3813 +3813 @@
-to)
+to#to_prep)
@@ -3815 +3815 @@
-setle)
+setle#settle_v)
@@ -3818 +3818 @@
-some)
+some#some_pron)
@@ -3820 +3820 @@
-intelligence)
+intelligence#intelligence_n)
@@ -3823 +3823 @@
-wth)
+wth#with_prep)
@@ -3826 +3826 @@
-Spaine))))))
+Spaine#NA))))))
@@ -3836 +3836 @@
-Twelue)
+Twelue#twelve_adj)
@@ -3838 +3838 @@
-paspor=t=~)
+paspor=t=~#passport_n1)
@@ -3842 +3842 @@
-to)
+to#to_prep)
@@ -3844 +3844 @@
-be)
+be#be_v)
@@ -3846 +3846 @@
-procured)
+procured#procure_v)
@@ -3849 +3849 @@
-from)
+from#from_prep)
@@ -3852 +3852 @@
-y=e=)
+y=e=#the_adj)
@@ -3854 +3854 @@
-l.)
+l.#lord_n)
@@ -3856 +3856 @@
-Adm=ll=)))
+Adm=ll=#admiral_n)))
@@ -3859 +3859 @@
-for)
+for#for_prep)
@@ -3862 +3862 @@
-ffrenche)
+ffrenche#french_adj)
@@ -3864 +3864 @@
-merchan=t=~))
+merchan=t=~#merchant_n))
@@ -3866 +3866 @@
-to)
+to#to_prep)
@@ -3868 +3868 @@
-carry)
+carry#carry_v)
@@ -3871 +3871 @@
-Corne))
+Corne#corn_n1))
@@ -3874 +3874 @@
-for)
+for#for_prep)
@@ -3877 +3877 @@
-Spaine))))
+Spaine#NA))))
@@ -3885 +3885 @@
-Rc.)
+Rc.#receive_v)
@@ -3888 +3888 @@
-the)
+the#the_adj)
@@ -3890 +3890 @@
-ixth))
+ixth#NA))
@@ -3893 +3893 @@
-at)
+at#at_prep)
@@ -3896 +3896 @@
-Whithall)))
+Whithall#NA)))
@@ -3904 +3904 @@
-To)
+To#to_prep)
@@ -3907 +3907 @@
-the)
+the#the_adj)
@@ -3910 +3910 @@
-right)
+right#right_adv)
@@ -3912 +3912 @@
-honorable))
+honorable#honourable_adj))
@@ -3914 +3914 @@
-S=r=.)
+S=r=.#NA)
@@ -3916 +3916 @@
-Robrt)
+Robrt#NA)
@@ -3918 +3918 @@
-Cecyll)
+Cecyll#NA)
@@ -3921 +3921 @@
-knight))
+knight#knight_n))
@@ -3925 +3925 @@
-Principall)
+Principall#principal_adj)
@@ -3927 +3927 @@
-Secretarie)
+Secretarie#secretary_n1)
@@ -3930 +3930 @@
-to)
+to#to_prep)
@@ -3934 +3934 @@
-the)
+the#the_adj)
@@ -3936 +3936 @@
-Queenes))
+Queenes#queen_n))
@@ -3938 +3938 @@
-Mat=ie=))))
+Mat=ie=#majesty_n))))
@@ -3941 +3941 @@
-and)
+and#and_conj1)
@@ -3944 +3944 @@
-chancelor)
+chancelor#chancellor_n)
@@ -3947 +3947 @@
-of)
+of#of_prep)
@@ -3950 +3950 @@
-the)
+the#the_adj)
@@ -3952 +3952 @@
-Duchee)
+Duchee#duchy_n)
@@ -3955 +3955 @@
-of)
+of#of_prep)
@@ -3958 +3958 @@
-lancaster)))))))))
+lancaster#NA)))))))))
