--- ehatton-1666-e3-h.psd.vert	2025-09-10 13:50:35
+++ ehatton-1666-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-Sept.)
+Sept.#NA)
@@ -15 +15 @@
-22)
+22#NA)
@@ -17 +17 @@
-1666)
+1666#NA)
@@ -28 +28 @@
-My)
+My#my_adj)
@@ -30 +30 @@
-dearst)
+dearst#dear_adj1)
@@ -32 +32 @@
-Dear))
+Dear#dear_adj1))
@@ -37 +37 @@
-Since)
+Since#since_adv)
@@ -44 +44 @@
-y=r=)
+y=r=#your_pron)
@@ -46 +46 @@
-father))
+father#father_n))
@@ -48 +48 @@
-tells)
+tells#tell_v)
@@ -51 +51 @@
-me))
+me#me_pron1))
@@ -54 +54 @@
-y=t=)
+y=t=#that_conj)
@@ -58 +58 @@
-your)
+your#your_pron)
@@ -60 +60 @@
-stay)
+stay#stay_n3)
@@ -63 +63 @@
-att)
+att#at_prep)
@@ -66 +66 @@
-London))))
+London#NA))))
@@ -68 +68 @@
-is)
+is#be_v)
@@ -71 +71 @@
-by)
+by#by_prep)
@@ -74 +74 @@
-my)
+my#my_adj)
@@ -76 +76 @@
-L=ad=)
+L=ad=#lady_n)
@@ -79 +79 @@
-Thenit)
+Thenit#NA)
@@ -81 +81 @@
-and)
+and#and_conj1)
@@ -83 +83 @@
-Sicelea))))
+Sicelea#NA))))
@@ -85 +85 @@
-command)
+command#command_v)
@@ -88 +88 @@
-to)
+to#to_prep)
@@ -90 +90 @@
-wait)
+wait#wait_v1)
@@ -93 +93 @@
-one)
+one#on_prep)
@@ -96 +96 @@
-y=m=)))
+y=m=#them_pron)))
@@ -99 +99 @@
-into)
+into#into_prep)
@@ -102 +102 @@
-y=e=)
+y=e=#the_adj)
@@ -104 +104 @@
-contry)))))))))
+contry#country_n)))))))))
@@ -109 +109 @@
-I))
+I#i_pron))
@@ -111 +111 @@
-must)
+must#must_v1)
@@ -113 +113 @@
-not)
+not#not_adv)
@@ -115 +115 @@
-chid)
+chid#chide_v)
@@ -118 +118 @@
-you))
+you#you_pron))
@@ -123 +123 @@
-though)
+though#though_adv)
@@ -130 +130 @@
-I))
+I#i_pron))
@@ -132 +132 @@
-can@)
+can@#can_v1)
@@ -134 +134 @@
-@not)
+@not#not_adv)
@@ -136 +136 @@
-but)
+but#but_prep)
@@ -138 +138 @@
-tell)
+tell#tell_v)
@@ -141 +141 @@
-you))
+you#you_pron))
@@ -146 +146 @@
-and)
+and#and_conj1)
@@ -149 +149 @@
-y=t=))
+y=t=#that_pron))
@@ -152 +152 @@
-truly)))
+truly#truly_adv)))
@@ -157 +157 @@
-that)
+that#that_conj)
@@ -162 +162 @@
-I))
+I#i_pron))
@@ -164 +164 @@
-am)
+am#be_v)
@@ -168 +168 @@
-very)
+very#very_adj)
@@ -170 +170 @@
-much)))
+much#much_adj)))
@@ -172 +172 @@
-troubled)
+troubled#trouble_v)
@@ -175 +175 @@
-y=t=)
+y=t=#that_conj)
@@ -179 +179 @@
-I))
+I#i_pron))
@@ -181 +181 @@
-doe)
+doe#do_v)
@@ -183 +183 @@
-not)
+not#not_adv)
@@ -185 +185 @@
-see)
+see#see_v)
@@ -188 +188 @@
-you)))))
+you#you_pron)))))
@@ -193 +193 @@
-and)
+and#and_conj1)
@@ -198 +198 @@
-the)
+the#the_adj)
@@ -200 +200 @@
-more)))
+more#more_adj)))
@@ -203 +203 @@
-because)
+because#because_adv)
@@ -210 +210 @@
-I))
+I#i_pron))
@@ -212 +212 @@
-fear)
+fear#fear_v)
@@ -219 +219 @@
-y=r=)
+y=r=#your_pron)
@@ -221 +221 @@
-father))
+father#father_n))
@@ -223 +223 @@
-will)
+will#will_v1)
@@ -225 +225 @@
-not)
+not#not_adv)
@@ -227 +227 @@
-give)
+give#give_v)
@@ -230 +230 @@
-me))
+me#me_pron1))
@@ -233 +233 @@
-leave))
+leave#leave_n1))
@@ -236 +236 @@
-to)
+to#to_prep)
@@ -238 +238 @@
-goe)
+goe#go_v)
@@ -240 +240 @@
-up)
+up#up_adv1)
@@ -243 +243 @@
-w=th=)
+w=th=#with_prep)
@@ -246 +246 @@
-him)))
+him#him_pron)))
@@ -249 +249 @@
-w=n=)
+w=n=#when_adv)
@@ -256 +256 @@
-he))
+he#he_pron))
@@ -258 +258 @@
-goes)))))))))))))))))
+goes#go_v)))))))))))))))))
@@ -267 +267 @@
-I))
+I#i_pron))
@@ -269 +269 @@
-presume)
+presume#presume_v)
@@ -276 +276 @@
-you))
+you#you_pron))
@@ -278 +278 @@
-were)
+were#be_v)
@@ -280 +280 @@
-shewed)
+shewed#show_v)
@@ -283 +283 @@
-y=e=)
+y=e=#the_adj)
@@ -285 +285 @@
-fine)
+fine#fine_adj)
@@ -287 +287 @@
-things)
+things#thing_n1)
@@ -298 +298 @@
-y=r=)
+y=r=#your_pron)
@@ -300 +300 @@
-father))
+father#father_n))
@@ -302 +302 @@
-brought)
+brought#bring_v)
@@ -305 +305 @@
-me))))
+me#me_pron1))))
@@ -311 +311 @@
-farrender)
+farrender#farandine_n)
@@ -314 +314 @@
-for)
+for#for_prep)
@@ -317 +317 @@
-a)
+a#a_adj2)
@@ -319 +319 @@
-gowne))))
+gowne#gown_n))))
@@ -324 +324 @@
-and)
+and#and_conj1)
@@ -327 +327 @@
-6)
+6#six_adj)
@@ -329 +329 @@
-pair)
+pair#pair_n1)
@@ -332 +332 @@
-of)
+of#of_prep)
@@ -335 +335 @@
-gloves)))))
+gloves#glove_n)))))
@@ -340 +340 @@
-and)
+and#and_conj1)
@@ -343 +343 @@
-a)
+a#a_adj2)
@@ -345 +345 @@
-paire)
+paire#pair_n1)
@@ -348 +348 @@
-of)
+of#of_prep)
@@ -351 +351 @@
-stockens))))))
+stockens#stocking_n2))))))
@@ -357 +357 @@
-w=ch=))
+w=ch=#which_adj))
@@ -364 +364 @@
-is)
+is#be_v)
@@ -367 +367 @@
-more)
+more#more_adj)
@@ -370 +370 @@
-y=n=)
+y=n=#than_conj)
@@ -379 +379 @@
-I))
+I#i_pron))
@@ -381 +381 @@
-hoped)
+hoped#hope_v)
@@ -384 +384 @@
-for)
+for#for_prep)
@@ -394 +394 @@
-and)
+and#and_conj1)
@@ -397 +397 @@
-so)
+so#so_adv)
@@ -399 +399 @@
-sensible)
+sensible#sensible_adj)
@@ -406 +406 @@
-I))
+I#i_pron))
@@ -408 +408 @@
-am)
+am#be_v)
@@ -411 +411 @@
-of)
+of#of_prep)
@@ -414 +414 @@
-y=e=)
+y=e=#the_adj)
@@ -416 +416 @@
-kindnes)))
+kindnes#kindness_n)))
@@ -419 +419 @@
-y=t=)
+y=t=#that_conj)
@@ -423 +423 @@
-I))
+I#i_pron))
@@ -425 +425 @@
-desir)
+desir#desire_v)
@@ -428 +428 @@
-you))
+you#you_pron))
@@ -431 +431 @@
-to)
+to#to_prep)
@@ -433 +433 @@
-help)
+help#help_v)
@@ -436 +436 @@
-me))
+me#me_pron1))
@@ -439 +439 @@
-to)
+to#to_prep)
@@ -441 +441 @@
-thank)
+thank#thank_v)
@@ -444 +444 @@
-him))
+him#him_pron))
@@ -447 +447 @@
-for)
+for#for_prep)
@@ -450 +450 @@
-it)))))))
+it#it_pron)))))))
@@ -459 +459 @@
-He))
+He#he_pron))
@@ -461 +461 @@
-is)
+is#be_v)
@@ -463 +463 @@
-pleased)
+pleased#please_v)
@@ -466 +466 @@
-to)
+to#to_prep)
@@ -468 +468 @@
-speak)
+speak#speak_v)
@@ -471 +471 @@
-kindly))
+kindly#kindly_adv))
@@ -474 +474 @@
-to)
+to#to_prep)
@@ -477 +477 @@
-me))))
+me#me_pron1))))
@@ -485 +485 @@
-and)
+and#and_conj1)
@@ -489 +489 @@
-is)
+is#be_v)
@@ -492 +492 @@
-more)
+more#more_adj)
@@ -494 +494 @@
-cheerfull)
+cheerfull#cheerful_adj)
@@ -497 +497 @@
-y=n=)
+y=n=#than_conj)
@@ -508 +508 @@
-he))
+he#he_pron))
@@ -510 +510 @@
-was)
+was#be_v)
@@ -513 +513 @@
-when)
+when#when_adv)
@@ -520 +520 @@
-he))
+he#he_pron))
@@ -522 +522 @@
-was)
+was#be_v)
@@ -525 +525 @@
-last))
+last#last_adv))
@@ -528 +528 @@
-in)
+in#in_prep)
@@ -531 +531 @@
-the)
+the#the_adj)
@@ -533 +533 @@
-contry))))))))))
+contry#country_n))))))))))
@@ -541 +541 @@
-and)
+and#and_conj1)
@@ -544 +544 @@
-I))
+I#i_pron))
@@ -546 +546 @@
-shall)
+shall#shall_v)
@@ -548 +548 @@
-be)
+be#be_v)
@@ -551 +551 @@
-as)
+as#as_adv)
@@ -553 +553 @@
-carfull)
+carfull#careful_adj)
@@ -556 +556 @@
-not)
+not#not_adv)
@@ -558 +558 @@
-to)
+to#to_prep)
@@ -560 +560 @@
-say)
+say#say_v1)
@@ -563 +563 @@
-any)
+any#any_adj)
@@ -565 +565 @@
-thing)
+thing#thing_n1)
@@ -570 +570 @@
-y=t=)
+y=t=#that_conj)
@@ -575 +575 @@
-may)
+may#may_v1)
@@ -577 +577 @@
-displease)
+displease#displease_v)
@@ -580 +580 @@
-him)))))))
+him#him_pron)))))))
@@ -586 +586 @@
-w=ch=))
+w=ch=#which_adj))
@@ -593 +593 @@
-puts)
+puts#put_v)
@@ -596 +596 @@
-me))
+me#me_pron1))
@@ -599 +599 @@
-to)
+to#to_prep)
@@ -602 +602 @@
-a)
+a#a_adj2)
@@ -604 +604 @@
-great)
+great#great_adj)
@@ -606 +606 @@
-stand)))
+stand#stand_n1)))
@@ -609 +609 @@
-in)
+in#in_prep)
@@ -612 +612 @@
-respect)
+respect#respect_n)
@@ -615 +615 @@
-of)
+of#of_prep)
@@ -618 +618 @@
-other)
+other#other_adj)
@@ -620 +620 @@
-necessarys)))))
+necessarys#necessary_adj)))))
@@ -624 +624 @@
-both)
+both#NA)
@@ -627 +627 @@
-for)
+for#for_prep)
@@ -633 +633 @@
-myself))
+myself#myself_pron))
@@ -636 +636 @@
-and)
+and#and_conj1)
@@ -639 +639 @@
-y=r=)
+y=r=#your_pron)
@@ -641 +641 @@
-poor)
+poor#poor_adj)
@@ -643 +643 @@
-sister)
+sister#sister_n)
@@ -646 +646 @@
-Mary))
+Mary#NA))
@@ -652 +652 @@
-who))
+who#who_pron))
@@ -659 +659 @@
-he))
+he#he_pron))
@@ -661 +661 @@
-has)
+has#have_v)
@@ -663 +663 @@
-not)
+not#not_adv)
@@ -665 +665 @@
-given)
+given#give_v)
@@ -668 +668 @@
-y=e=)
+y=e=#the_adj)
@@ -670 +670 @@
-worth)
+worth#worth_n1)
@@ -673 +673 @@
-of)
+of#of_prep)
@@ -676 +676 @@
-one)
+one#one_adj)
@@ -678 +678 @@
-penny))))
+penny#penny_n))))
@@ -681 +681 @@
-to)
+to#to_prep)
@@ -688 +688 @@
-nor)
+nor#nor_conj1)
@@ -692 +692 @@
-till)
+till#till_prep)
@@ -695 +695 @@
-to_day)))
+to_day#today_adv)))
@@ -699 +699 @@
-has)
+has#have_v)
@@ -701 +701 @@
-not)
+not#not_adv)
@@ -703 +703 @@
-spoak)
+spoak#speak_v)
@@ -706 +706 @@
-one)
+one#one_adj)
@@ -708 +708 @@
-word))
+word#word_n))
@@ -711 +711 @@
-to)
+to#to_prep)
@@ -714 +714 @@
-her)))))
+her#her_pron1)))))
@@ -720 +720 @@
-w=ch=))
+w=ch=#which_adj))
@@ -727 +727 @@
-is)
+is#be_v)
@@ -730 +730 @@
-a)
+a#a_adj2)
@@ -733 +733 @@
-very)
+very#very_adj)
@@ -735 +735 @@
-great))
+great#great_adj))
@@ -737 +737 @@
-trouble)
+trouble#trouble_n)
@@ -739 +739 @@
-both)
+both#NA)
@@ -742 +742 @@
-to)
+to#to_prep)
@@ -747 +747 @@
-her)
+her#her_pron1)
@@ -749 +749 @@
-and)
+and#and_conj1)
@@ -751 +751 @@
-me))))))))))))))
+me#me_pron1))))))))))))))
@@ -760 +760 @@
-I))
+I#i_pron))
@@ -762 +762 @@
-thank)
+thank#thank_v)
@@ -765 +765 @@
-you))
+you#you_pron))
@@ -768 +768 @@
-for)
+for#for_prep)
@@ -771 +771 @@
-y=r=)
+y=r=#your_pron)
@@ -773 +773 @@
-letter)))
+letter#letter_n1)))
@@ -776 +776 @@
-by)
+by#by_prep)
@@ -779 +779 @@
-y=e=)
+y=e=#the_adj)
@@ -781 +781 @@
-carrier)))
+carrier#carrier_n)))
@@ -784 +784 @@
-last)
+last#last_adv)
@@ -786 +786 @@
-week))
+week#week_n))
@@ -795 +795 @@
-I))
+I#i_pron))
@@ -797 +797 @@
-did)
+did#do_v)
@@ -799 +799 @@
-not)
+not#not_adv)
@@ -801 +801 @@
-writ)
+writ#write_v)
@@ -804 +804 @@
-again))
+again#again_adv))
@@ -807 +807 @@
-by)
+by#by_prep)
@@ -810 +810 @@
-him)))
+him#him_pron)))
@@ -815 +815 @@
-because)
+because#because_adv)
@@ -822 +822 @@
-I))
+I#i_pron))
@@ -824 +824 @@
-had)
+had#have_v)
@@ -826 +826 @@
-writ)
+writ#write_v)
@@ -829 +829 @@
-so)
+so#so_adv)
@@ -831 +831 @@
-lately))
+lately#lately_adv))
@@ -834 +834 @@
-by)
+by#by_prep)
@@ -837 +837 @@
-y=e=)
+y=e=#the_adj)
@@ -839 +839 @@
-boy)
+boy#boy_n1)
@@ -844 +844 @@
-y=t=)
+y=t=#that_conj)
@@ -849 +849 @@
-carried)
+carried#carry_v)
@@ -851 +851 @@
-up)
+up#up_adv1)
@@ -855 +855 @@
-y=r=)
+y=r=#your_pron)
@@ -857 +857 @@
-father's))
+father's#father_n))
@@ -859 +859 @@
-horses)))))))))
+horses#horse_n)))))))))
@@ -867 +867 @@
-My)
+My#my_adj)
@@ -869 +869 @@
-humble)
+humble#humble_adj)
@@ -871 +871 @@
-service)
+service#service_n1)
@@ -874 +874 @@
-to)
+to#to_prep)
@@ -878 +878 @@
-y=e=)
+y=e=#the_adj)
@@ -880 +880 @@
-Lady))
+Lady#lady_n))
@@ -883 +883 @@
-Thanet))
+Thanet#NA))
@@ -886 +886 @@
-and)
+and#and_conj1)
@@ -889 +889 @@
-my)
+my#my_adj)
@@ -891 +891 @@
-Lady)
+Lady#lady_n)
@@ -894 +894 @@
-Cicelea))))))
+Cicelea#NA))))))
@@ -903 +903 @@
-Y=r=)
+Y=r=#your_pron)
@@ -905 +905 @@
-sister))
+sister#sister_n))
@@ -907 +907 @@
-disir)
+disir#desire_v)
@@ -910 +910 @@
-y=e=)
+y=e=#the_adj)
@@ -912 +912 @@
-same))
+same#same_adj))
@@ -916 +916 @@
-to)
+to#to_prep)
@@ -919 +919 @@
-you)))
+you#you_pron)))
@@ -922 +922 @@
-and)
+and#and_conj1)
@@ -925 +925 @@
-to)
+to#to_prep)
@@ -928 +928 @@
-the)
+the#the_adj)
@@ -930 +930 @@
-L=a=)
+L=a=#lady_n)
@@ -933 +933 @@
-Cicelea))))))
+Cicelea#NA))))))
@@ -942 +942 @@
-She))
+She#she_pron))
@@ -944 +944 @@
-is)
+is#be_v)
@@ -947 +947 @@
-so)
+so#so_adv)
@@ -949 +949 @@
-troubled)
+troubled#troubled_adj)
@@ -952 +952 @@
-att)
+att#at_prep)
@@ -956 +956 @@
-y=r=)
+y=r=#your_pron)
@@ -958 +958 @@
-fathers))
+fathers#father_n))
@@ -961 +961 @@
-not)
+not#not_adv)
@@ -963 +963 @@
-looking)
+looking#look_v)
@@ -966 +966 @@
-one)
+one#on_prep)
@@ -969 +969 @@
-her)))
+her#her_pron1)))
@@ -972 +972 @@
-as)
+as#as_adv)
@@ -975 +975 @@
-upon)
+upon#upon_prep)
@@ -978 +978 @@
-others)))))))
+others#other_adj)))))))
@@ -983 +983 @@
-that)
+that#that_conj)
@@ -987 +987 @@
-really))
+really#really_adv2))
@@ -990 +990 @@
-you))
+you#you_pron))
@@ -992 +992 @@
-must)
+must#must_v1)
@@ -994 +994 @@
-excuse)
+excuse#excuse_v)
@@ -997 +997 @@
-her))
+her#her_pron1))
@@ -1000 +1000 @@
-not)
+not#not_adv)
@@ -1002 +1002 @@
-writting)
+writting#write_v)
@@ -1005 +1005 @@
-to)
+to#to_prep)
@@ -1008 +1008 @@
-you)))))))
+you#you_pron)))))))
@@ -1017 +1017 @@
-My)
+My#my_adj)
@@ -1019 +1019 @@
-dear))
+dear#dear_adj1))
@@ -1024 +1024 @@
-though)
+though#though_adv)
@@ -1031 +1031 @@
-I))
+I#i_pron))
@@ -1033 +1033 @@
-doe)
+doe#do_v)
@@ -1035 +1035 @@
-not)
+not#not_adv)
@@ -1037 +1037 @@
-see)
+see#see_v)
@@ -1040 +1040 @@
-you)))))
+you#you_pron)))))
@@ -1045 +1045 @@
-I))
+I#i_pron))
@@ -1047 +1047 @@
-hope)
+hope#hope_v)
@@ -1054 +1054 @@
-you))
+you#you_pron))
@@ -1056 +1056 @@
-think)
+think#think_v2)
@@ -1059 +1059 @@
-often))
+often#often_adv))
@@ -1062 +1062 @@
-of)
+of#of_prep)
@@ -1065 +1065 @@
-me)))))
+me#me_pron1)))))
@@ -1074 +1074 @@
-I))
+I#i_pron))
@@ -1076 +1076 @@
-assure)
+assure#assure_v)
@@ -1079 +1079 @@
-you))
+you#you_pron))
@@ -1086 +1086 @@
-I))
+I#i_pron))
@@ -1088 +1088 @@
-do)
+do#do_v)
@@ -1093 +1093 @@
-of)
+of#of_prep)
@@ -1096 +1096 @@
-you)))
+you#you_pron)))
@@ -1099 +1099 @@
-w=th=)
+w=th=#with_prep)
@@ -1103 +1103 @@
-as)
+as#as_adv)
@@ -1105 +1105 @@
-great))
+great#great_adj))
@@ -1107 +1107 @@
-kindnes)
+kindnes#kindness_n)
@@ -1110 +1110 @@
-as)
+as#as_adv)
@@ -1121 +1121 @@
-any)
+any#any_adj)
@@ -1123 +1123 @@
-mother))
+mother#mother_n1))
@@ -1125 +1125 @@
-can)
+can#can_v1)
@@ -1127 +1127 @@
-doe)
+doe#do_v)
@@ -1130 +1130 @@
-for)
+for#for_prep)
@@ -1133 +1133 @@
-a)
+a#a_adj2)
@@ -1135 +1135 @@
-childe))))))))))
+childe#child_n))))))))))
@@ -1143 +1143 @@
-and)
+and#and_conj1)
@@ -1147 +1147 @@
-trust)
+trust#trust_v)
@@ -1154 +1154 @@
-God))
+God#NA))
@@ -1156 +1156 @@
-will)
+will#will_v1)
@@ -1158 +1158 @@
-hear)
+hear#hear_v)
@@ -1161 +1161 @@
-the)
+the#the_adj)
@@ -1163 +1163 @@
-prayers)
+prayers#prayer_n1)
@@ -1168 +1168 @@
-y=t=)
+y=t=#that_conj)
@@ -1173 +1173 @@
-are)
+are#be_v)
@@ -1175 +1175 @@
-made)
+made#make_v1)
@@ -1178 +1178 @@
-day)
+day#day_n)
@@ -1180 +1180 @@
-and)
+and#and_conj1)
@@ -1182 +1182 @@
-night))
+night#night_n))
@@ -1185 +1185 @@
-for)
+for#for_prep)
@@ -1188 +1188 @@
-you)))
+you#you_pron)))
@@ -1191 +1191 @@
-by)
+by#by_prep)
@@ -1194 +1194 @@
-Y=r=)
+Y=r=#your_pron)
@@ -1198 +1198 @@
-most)
+most#most_adj)
@@ -1200 +1200 @@
-truly))
+truly#truly_adv))
@@ -1202 +1202 @@
-affectionat))
+affectionat#affectionate_adj))
@@ -1204 +1204 @@
-mother)
+mother#mother_n1)
@@ -1209 +1209 @@
-y=e=)
+y=e=#the_adj)
@@ -1211 +1211 @@
-afflicted)
+afflicted#afflicted_adj)
@@ -1213 +1213 @@
-Elizabeth)
+Elizabeth#NA)
@@ -1217 +1217 @@
-Hatton))))))))))
+Hatton#NA))))))))))
