--- eliz-1598-e2-h.psd.vert	2025-09-10 13:50:35
+++ eliz-1598-e2-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-By)
+By#by_prep)
@@ -16 +16 @@
-the)
+the#the_adj)
@@ -18 +18 @@
-Queene)
+Queene#queen_n)
@@ -23 +23 @@
-Elizabeth))))
+Elizabeth#NA))))
@@ -31 +31 @@
-R.)
+R.#right_adv)
@@ -33 +33 @@
-Trusty)
+Trusty#trusty_adj)
@@ -36 +36 @@
-and)
+and#and_conj1)
@@ -39 +39 @@
-welbeloved)))))
+welbeloved#well-beloved_adj)))))
@@ -44 +44 @@
-wee))
+wee#we_pron))
@@ -46 +46 @@
-greet)
+greet#greet_v1)
@@ -49 +49 @@
-you))
+you#you_pron))
@@ -52 +52 @@
-well))
+well#well_adv))
@@ -61 +61 @@
-wee))
+wee#we_pron))
@@ -63 +63 @@
-haue)
+haue#have_v)
@@ -65 +65 @@
-seen)
+seen#see_v)
@@ -69 +69 @@
-y=r=.)
+y=r=.#your_pron)
@@ -71 +71 @@
-l~re)
+l~re#letter_n1)
@@ -76 +76 @@
-to)
+to#to_prep)
@@ -79 +79 @@
-our)
+our#our_pron)
@@ -81 +81 @@
-Secratary))))
+Secratary#secretary_n1))))
@@ -86 +86 @@
-and)
+and#and_conj1)
@@ -89 +89 @@
-th'@)
+th'@#the_adj)
@@ -91 +91 @@
-@enclosed)
+@enclosed#enclosed_adj)
@@ -94 +94 @@
-from)
+from#from_prep)
@@ -97 +97 @@
-the)
+the#the_adj)
@@ -99 +99 @@
-Master)
+Master#master_n1)
@@ -102 +102 @@
-of)
+of#of_prep)
@@ -105 +105 @@
-Gray)))))))
+Gray#NA)))))))
@@ -111 +111 @@
-w=ch=))
+w=ch=#which_adj))
@@ -118 +118 @@
-bringeth)
+bringeth#bring_v)
@@ -121 +121 @@
-to)
+to#to_prep)
@@ -124 +124 @@
-vs)))
+vs#us_pron)))
@@ -129 +129 @@
-at)
+at#at_prep)
@@ -132 +132 @@
-one)
+one#one_adj)
@@ -134 +134 @@
-instant)))
+instant#instant_n)))
@@ -139 +139 @@
-two)
+two#two_adj)
@@ -141 +141 @@
-strange)
+strange#strange_adj)
@@ -145 +145 @@
-wonders)
+wonders#wonder_n)
@@ -152 +152 @@
-first))
+first#first_adj))
@@ -154 +154 @@
-that)
+that#that_conj)
@@ -158 +158 @@
-he)
+he#he_pron)
@@ -162 +162 @@
-is)
+is#be_v)
@@ -164 +164 @@
-resuscited)
+resuscited#resuscite_v)
@@ -170 +170 @@
-whome))
+whome#whom_pron))
@@ -176 +176 @@
-we))
+we#we_pron))
@@ -178 +178 @@
-held)
+held#hold_v)
@@ -184 +184 @@
-by)
+by#by_prep)
@@ -187 +187 @@
-this)
+this#this_pron)
@@ -189 +189 @@
-tyme)))
+tyme#time_n)))
@@ -191 +191 @@
-intombed))))))
+intombed#entomb_v))))))
@@ -197 +197 @@
-Secondly)
+Secondly#secondly_adv)
@@ -201 +201 @@
-that)
+that#that_conj)
@@ -207 +207 @@
-being)
+being#be_v)
@@ -210 +210 @@
-all)
+all#all_adj)
@@ -212 +212 @@
-this)
+this#this_pron)
@@ -214 +214 @@
-while))
+while#while_n))
@@ -217 +217 @@
-vnburied)))
+vnburied#unburied_adj)))
@@ -224 +224 @@
-he))
+he#he_pron))
@@ -226 +226 @@
-could)
+could#can_v1)
@@ -228 +228 @@
-passe)
+passe#pass_v)
@@ -230 +230 @@
-ouer)
+ouer#over_adv)
@@ -234 +234 @@
-so)
+so#so_adv)
@@ -236 +236 @@
-many))
+many#many_adj))
@@ -238 +238 @@
-yeares))
+yeares#year_n))
@@ -241 +241 @@
-in)
+in#in_prep)
@@ -244 +244 @@
-silence)))))))))))
+silence#silence_n)))))))))))
@@ -252 +252 @@
-But)
+But#but_prep)
@@ -255 +255 @@
-seing)
+seing#see_v)
@@ -262 +262 @@
-wee))
+wee#we_pron))
@@ -264 +264 @@
-do)
+do#do_v)
@@ -266 +266 @@
-find)
+find#find_v)
@@ -271 +271 @@
-that)
+that#that_conj)
@@ -275 +275 @@
-his)
+his#his_adj)
@@ -277 +277 @@
-retired)
+retired#retired_adj)
@@ -279 +279 @@
-affection)
+affection#affection_n)
@@ -282 +282 @@
-towards)
+towards#towards_prep)
@@ -285 +285 @@
-vs))))
+vs#us_pron))))
@@ -289 +289 @@
-returnes)
+returnes#return_v1)
@@ -292 +292 @@
-to)
+to#to_prep)
@@ -295 +295 @@
-his)
+his#his_adj)
@@ -297 +297 @@
-former)
+former#former_adj)
@@ -299 +299 @@
-place)
+place#place_n.II-VII)
@@ -302 +302 @@
-of)
+of#of_prep)
@@ -305 +305 @@
-confidence))))))))))
+confidence#confidence_n))))))))))
@@ -310 +310 @@
-you))
+you#you_pron))
@@ -312 +312 @@
-may)
+may#may_v1)
@@ -314 +314 @@
-do)
+do#do_v)
@@ -317 +317 @@
-well))
+well#well_adv))
@@ -322 +322 @@
-to)
+to#to_prep)
@@ -324 +324 @@
-assure)
+assure#assure_v)
@@ -327 +327 @@
-him))
+him#him_pron))
@@ -330 +330 @@
-hereby))
+hereby#hereby_adv))
@@ -335 +335 @@
-that)
+that#that_conj)
@@ -341 +341 @@
-whatsoever))
+whatsoever#whatsoever_pron))
@@ -349 +349 @@
-he))
+he#he_pron))
@@ -351 +351 @@
-shall)
+shall#shall_v)
@@ -353 +353 @@
-do)
+do#do_v)
@@ -358 +358 @@
-for)
+for#for_prep)
@@ -361 +361 @@
-our)
+our#our_pron)
@@ -363 +363 @@
-seruice)))
+seruice#service_n1)))
@@ -368 +368 @@
-in)
+in#in_prep)
@@ -371 +371 @@
-any)
+any#any_adj)
@@ -373 +373 @@
-thing))))))
+thing#thing_n1))))))
@@ -377 +377 @@
-shal@)
+shal@#shall_v)
@@ -379 +379 @@
-@be)
+@be#be_v)
@@ -381 +381 @@
-performed)
+performed#perform_v)
@@ -384 +384 @@
-to)
+to#to_prep)
@@ -387 +387 @@
-those)
+those#those_pron)
@@ -393 +393 @@
-who~))
+who~#whom_pron))
@@ -399 +399 @@
-he))
+he#he_pron))
@@ -401 +401 @@
-knowes)
+knowes#know_v)
@@ -404 +404 @@
-well))
+well#well_adv))
@@ -411 +411 @@
-to)
+to#to_prep)
@@ -413 +413 @@
-haue)
+haue#have_v)
@@ -416 +416 @@
-ever))
+ever#ever_adv))
@@ -418 +418 @@
-resolved)
+resolved#resolve_v)
@@ -421 +421 @@
-of)
+of#of_prep)
@@ -424 +424 @@
-this)
+this#this_pron)
@@ -426 +426 @@
-position)))))))))))
+position#position_n)))))))))))
@@ -434 +434 @@
-Si)
+Si#NA)
@@ -436 +436 @@
-ingratam)
+ingratam#NA)
@@ -438 +438 @@
-dixeris)
+dixeris#NA)
@@ -442 +442 @@
-omnia)
+omnia#NA)
@@ -444 +444 @@
-dixeris)
+dixeris#NA)
@@ -453 +453 @@
-For)
+For#for_prep)
@@ -456 +456 @@
-the)
+the#the_adj)
@@ -458 +458 @@
-other)
+other#other_adj)
@@ -460 +460 @@
-parte)
+parte#part_n)
@@ -463 +463 @@
-of)
+of#of_prep)
@@ -466 +466 @@
-his)
+his#his_adj)
@@ -470 +470 @@
-motion)))
+motion#motion_n)))
@@ -475 +475 @@
-that)
+that#that_conj)
@@ -479 +479 @@
-wee))
+wee#we_pron))
@@ -481 +481 @@
-would)
+would#will_v1)
@@ -483 +483 @@
-write)
+write#write_v)
@@ -486 +486 @@
-some)
+some#some_pron)
@@ -488 +488 @@
-l~re))
+l~re#letter_n1))
@@ -491 +491 @@
-to)
+to#to_prep)
@@ -494 +494 @@
-his)
+his#his_adj)
@@ -496 +496 @@
-frend)
+frend#friend_n)
@@ -502 +502 @@
-for)
+for#for_prep)
@@ -505 +505 @@
-whome)))
+whome#whom_pron)))
@@ -513 +513 @@
-he))
+he#he_pron))
@@ -515 +515 @@
-vndertaketh)))))))))
+vndertaketh#undertake_v)))))))))
@@ -520 +520 @@
-it))
+it#it_pron))
@@ -522 +522 @@
-is)
+is#be_v)
@@ -525 +525 @@
-a)
+a#a_adj2)
@@ -527 +527 @@
-Course)
+Course#course_n1)
@@ -534 +534 @@
-that)
+that#that_conj)
@@ -540 +540 @@
-wee))
+wee#we_pron))
@@ -542 +542 @@
-haue)
+haue#have_v)
@@ -545 +545 @@
-never))
+never#never_adv))
@@ -547 +547 @@
-vsed)
+vsed#use_v)
@@ -550 +550 @@
-to)
+to#to_prep)
@@ -553 +553 @@
-any)
+any#any_adj)
@@ -558 +558 @@
-but)
+but#but_prep)
@@ -561 +561 @@
-such)
+such#such_adj)
@@ -567 +567 @@
-whose)
+whose#whose_pron)
@@ -569 +569 @@
-meritt))
+meritt#merit_n))
@@ -576 +576 @@
-hath)
+hath#have_v)
@@ -579 +579 @@
-first))
+first#first_adj))
@@ -581 +581 @@
-purchased)
+purchased#purchase_v)
@@ -584 +584 @@
-trust)
+trust#trust_n)
@@ -586 +586 @@
-and)
+and#and_conj1)
@@ -588 +588 @@
-favour)))))))))))
+favour#favour_n)))))))))))
@@ -596 +596 @@
-Lett)
+Lett#let_v1)
@@ -600 +600 @@
-him))
+him#him_pron))
@@ -603 +603 @@
-therefore))
+therefore#therefore_adv))
@@ -605 +605 @@
-know)
+know#know_v)
@@ -608 +608 @@
-this)
+this#this_pron)
@@ -613 +613 @@
-that)
+that#that_conj)
@@ -617 +617 @@
-as)
+as#as_adv)
@@ -628 +628 @@
-wee))
+wee#we_pron))
@@ -630 +630 @@
-do)
+do#do_v)
@@ -632 +632 @@
-thanck)
+thanck#thank_v)
@@ -637 +637 @@
-both)
+both#both_pron)
@@ -640 +640 @@
-him))
+him#him_pron))
@@ -645 +645 @@
-and)
+and#and_conj1)
@@ -648 +648 @@
-th'@)
+th'@#the_adj)
@@ -650 +650 @@
-@other)
+@other#other_adj)
@@ -659 +659 @@
-whosoever))
+whosoever#whosoever_pron))
@@ -667 +667 @@
-he))
+he#he_pron))
@@ -669 +669 @@
-be)))))))
+be#be_v)))))))
@@ -676 +676 @@
-for)
+for#for_prep)
@@ -679 +679 @@
-his)
+his#his_adj)
@@ -681 +681 @@
-good)
+good#good_adj)
@@ -683 +683 @@
-disposition))))))
+disposition#disposition_n))))))
@@ -688 +688 @@
-so))
+so#so_adv))
@@ -691 +691 @@
-when)
+when#when_adv)
@@ -698 +698 @@
-any)
+any#any_adj)
@@ -700 +700 @@
-effect))
+effect#effect_n))
@@ -702 +702 @@
-shall)
+shall#shall_v)
@@ -704 +704 @@
-followe)
+followe#follow_v)
@@ -707 +707 @@
-of)
+of#of_prep)
@@ -710 +710 @@
-his)
+his#his_adj)
@@ -712 +712 @@
-honest)
+honest#honest_adj)
@@ -714 +714 @@
-purposes))))))
+purposes#purpose_n))))))
@@ -719 +719 @@
-our)
+our#our_pron)
@@ -721 +721 @@
-requytall))
+requytall#requital_n))
@@ -728 +728 @@
-in)
+in#in_prep)
@@ -733 +733 @@
-what))
+what#what_pron))
@@ -740 +740 @@
-is)
+is#be_v)
@@ -743 +743 @@
-fytt)
+fytt#fit_adj)
@@ -746 +746 @@
-for)
+for#for_prep)
@@ -749 +749 @@
-a)
+a#a_adj2)
@@ -751 +751 @@
-Prince))))))))
+Prince#prince_n))))))))
@@ -757 +757 @@
-shall)
+shall#shall_v)
@@ -759 +759 @@
-not)
+not#not_adv)
@@ -761 +761 @@
-be)
+be#be_v)
@@ -765 +765 @@
-long))
+long#long_adj1))
@@ -767 +767 @@
-behinde))))))
+behinde#behind_adv))))))
@@ -776 +776 @@
-Some)
+Some#some_pron)
@@ -778 +778 @@
-other)
+other#other_adj)
@@ -780 +780 @@
-things)
+things#thing_n1)
@@ -785 +785 @@
-we))
+we#we_pron))
@@ -787 +787 @@
-haue)
+haue#have_v)
@@ -789 +789 @@
-imparted)
+imparted#impart_v)
@@ -792 +792 @@
-to)
+to#to_prep)
@@ -795 +795 @@
-our)
+our#our_pron)
@@ -797 +797 @@
-Secretary)))
+Secretary#secretary_n1)))
@@ -803 +803 @@
-wherein))
+wherein#wherein_adv))
@@ -811 +811 @@
-you))
+you#you_pron))
@@ -813 +813 @@
-may)
+may#may_v1)
@@ -815 +815 @@
-beleeue)
+beleeue#believe_v)
@@ -818 +818 @@
-him))))
+him#him_pron))))
@@ -826 +826 @@
-From)
+From#from_prep)
@@ -829 +829 @@
-our)
+our#our_pron)
@@ -831 +831 @@
-Palace)
+Palace#palace_n1)
@@ -834 +834 @@
-of)
+of#of_prep)
@@ -837 +837 @@
-Westminster))))
+Westminster#NA))))
@@ -845 +845 @@
-the)
+the#the_adj)
@@ -847 +847 @@
-4=th=)
+4=th=#NA)
@@ -850 +850 @@
-of)
+of#of_prep)
@@ -853 +853 @@
-Decembr.)))
+Decembr.#NA)))
@@ -855 +855 @@
-1598)
+1598#NA)
@@ -864 +864 @@
-M=r=)
+M=r=#NA)
@@ -866 +866 @@
-Edmunds))
+Edmunds#NA))
@@ -869 +869 @@
-4=th=)
+4=th=#NA)
@@ -871 +871 @@
-December)
+December#NA)
@@ -873 +873 @@
-1598))
+1598#NA))
@@ -876 +876 @@
-ffrom)
+ffrom#from_prep)
@@ -879 +879 @@
-her)
+her#her_pron2)
@@ -881 +881 @@
-ma=tie=)))
+ma=tie=#majesty_n)))
@@ -889 +889 @@
-To)
+To#to_prep)
@@ -892 +892 @@
-our)
+our#our_pron)
@@ -895 +895 @@
-trusty)
+trusty#trusty_adj)
@@ -897 +897 @@
-and)
+and#and_conj1)
@@ -899 +899 @@
-welbeloued))
+welbeloued#well-beloved_adj))
@@ -901 +901 @@
-servant)
+servant#servant_n)
@@ -906 +906 @@
-Thomas)
+Thomas#NA)
@@ -908 +908 @@
-Edmunds))
+Edmunds#NA))
@@ -913 +913 @@
-our)
+our#our_pron)
@@ -915 +915 @@
-Agent)
+Agent#agent_n)
@@ -918 +918 @@
-w=th=)
+w=th=#with_prep)
@@ -921 +921 @@
-the)
+the#the_adj)
@@ -923 +923 @@
-French)
+French#french_adj)
@@ -925 +925 @@
-kinge)))))
+kinge#king_n)))))
@@ -939 +939 @@
-Elizabeth))
+Elizabeth#NA))
@@ -947 +947 @@
-R.)
+R.#right_adv)
@@ -949 +949 @@
-Trusty)
+Trusty#trusty_adj)
@@ -952 +952 @@
-and)
+and#and_conj1)
@@ -955 +955 @@
-welbeloued)))))
+welbeloued#well-beloved_adj)))))
@@ -960 +960 @@
-Wee))
+Wee#we_pron))
@@ -962 +962 @@
-receaued)
+receaued#receive_v)
@@ -965 +965 @@
-by)
+by#by_prep)
@@ -968 +968 @@
-the)
+the#the_adj)
@@ -970 +970 @@
-french)
+french#french_adj)
@@ -972 +972 @@
-Embassadour)))
+Embassadour#embassador_n)))
@@ -979 +979 @@
-not)
+not#not_adv)
@@ -981 +981 @@
-many))
+many#many_adj))
@@ -983 +983 @@
-daies))
+daies#day_n))
@@ -985 +985 @@
-past))
+past#past_prep))
@@ -990 +990 @@
-a)
+a#a_adj2)
@@ -993 +993 @@
-kind)
+kind#kind_adj)
@@ -995 +995 @@
-and)
+and#and_conj1)
@@ -997 +997 @@
-carefull))
+carefull#careful_adj))
@@ -999 +999 @@
-advertisement)
+advertisement#advertisement_n)
@@ -1002 +1002 @@
-and)
+and#and_conj1)
@@ -1005 +1005 @@
-advise)))
+advise#advice_n)))
@@ -1008 +1008 @@
-from)
+from#from_prep)
@@ -1011 +1011 @@
-our)
+our#our_pron)
@@ -1013 +1013 @@
-good)
+good#good_adj)
@@ -1015 +1015 @@
-Brother)
+Brother#brother_n)
@@ -1018 +1018 @@
-the)
+the#the_adj)
@@ -1020 +1020 @@
-french)
+french#french_adj)
@@ -1022 +1022 @@
-king))))
+king#king_n))))
@@ -1028 +1028 @@
-wherein))
+wherein#wherein_adv))
@@ -1037 +1037 @@
-he))
+he#he_pron))
@@ -1039 +1039 @@
-hath)
+hath#have_v)
@@ -1041 +1041 @@
-lett)
+lett#let_v1)
@@ -1045 +1045 @@
-vs))
+vs#us_pron))
@@ -1047 +1047 @@
-know)
+know#know_v)
@@ -1053 +1053 @@
-what)
+what#what_pron)
@@ -1063 +1063 @@
-he))
+he#he_pron))
@@ -1065 +1065 @@
-hath)
+hath#have_v)
@@ -1067 +1067 @@
-heard)
+heard#hear_v)
@@ -1072 +1072 @@
-of)
+of#of_prep)
@@ -1075 +1075 @@
-great)
+great#great_adj)
@@ -1077 +1077 @@
-preparations)
+preparations#preparation_n)
@@ -1082 +1082 @@
-by)
+by#by_prep)
@@ -1085 +1085 @@
-sea)))
+sea#sea_n)))
@@ -1090 +1090 @@
-in)
+in#in_prep)
@@ -1093 +1093 @@
-Spayne)))))))))
+Spayne#NA)))))))))
@@ -1098 +1098 @@
-and)
+and#and_conj1)
@@ -1105 +1105 @@
-wished)
+wished#wish_v)
@@ -1108 +1108 @@
-vs))
+vs#us_pron))
@@ -1112 +1112 @@
-carefully))
+carefully#carefully_adv))
@@ -1114 +1114 @@
-to)
+to#to_prep)
@@ -1116 +1116 @@
-stopp)
+stopp#stop_v)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-transportation)
+transportation#transportation_n)
@@ -1126 +1126 @@
-of)
+of#of_prep)
@@ -1129 +1129 @@
-warlick)
+warlick#warlike_adj)
@@ -1131 +1131 @@
-provisions)))))))
+provisions#provision_n)))))))
@@ -1136 +1136 @@
-affirming)
+affirming#affirm_v)
@@ -1141 +1141 @@
-to)
+to#to_prep)
@@ -1143 +1143 @@
-haue)
+haue#have_v)
@@ -1145 +1145 @@
-geuen)
+geuen#give_v)
@@ -1148 +1148 @@
-the)
+the#the_adj)
@@ -1150 +1150 @@
-lyke)
+lyke#like_adj)
@@ -1152 +1152 @@
-caution))
+caution#caution_n))
@@ -1157 +1157 @@
-to)
+to#to_prep)
@@ -1160 +1160 @@
-those)
+those#those_pron)
@@ -1163 +1163 @@
-of)
+of#of_prep)
@@ -1166 +1166 @@
-Holland)))))
+Holland#NA)))))
@@ -1171 +1171 @@
-by)
+by#by_prep)
@@ -1174 +1174 @@
-his)
+his#his_adj)
@@ -1176 +1176 @@
-Agent)
+Agent#agent_n)
@@ -1179 +1179 @@
-there))))))))
+there#there_adv))))))))
@@ -1184 +1184 @@
-An)
+An#a_adj2)
@@ -1186 +1186 @@
-Ouverture)
+Ouverture#overture_n)
@@ -1189 +1189 @@
-very)
+very#very_adj)
@@ -1191 +1191 @@
-wellcom)
+wellcom#welcome_n1)
@@ -1194 +1194 @@
-to)
+to#to_prep)
@@ -1197 +1197 @@
-vs)))
+vs#us_pron)))
@@ -1202 +1202 @@
-because)
+because#because_adv)
@@ -1210 +1210 @@
-it))
+it#it_pron))
@@ -1212 +1212 @@
-confirmed)
+confirmed#confirm_v)
@@ -1216 +1216 @@
-his)
+his#his_adj)
@@ -1220 +1220 @@
-constant)
+constant#constant_adj)
@@ -1222 +1222 @@
-frendship)
+frendship#friendship_n)
@@ -1228 +1228 @@
-to)
+to#to_prep)
@@ -1231 +1231 @@
-whome)))
+whome#whom_pron)))
@@ -1237 +1237 @@
-wee))
+wee#we_pron))
@@ -1239 +1239 @@
-had)
+had#have_v)
@@ -1241 +1241 @@
-Leaft)
+Leaft#leave_v1)
@@ -1244 +1244 @@
-no)
+no#no_adj)
@@ -1246 +1246 @@
-good)
+good#good_adj)
@@ -1248 +1248 @@
-office)
+office#office_n)
@@ -1253 +1253 @@
-vnperformed))))))
+vnperformed#unperformed_adj))))))
@@ -1258 +1258 @@
-and)
+and#and_conj1)
@@ -1263 +1263 @@
-assured)
+assured#assure_v)
@@ -1266 +1266 @@
-vs))
+vs#us_pron))
@@ -1269 +1269 @@
-further))
+further#further_adj))
@@ -1277 +1277 @@
-howsoe`uer))
+howsoe`uer#howsoever_adv))
@@ -1285 +1285 @@
-the)
+the#the_adj)
@@ -1287 +1287 @@
-courss)
+courss#course_n1)
@@ -1290 +1290 @@
-and)
+and#and_conj1)
@@ -1293 +1293 @@
-revolution)))
+revolution#revolution_n)))
@@ -1296 +1296 @@
-of)
+of#of_prep)
@@ -1299 +1299 @@
-humane)
+humane#human_adj)
@@ -1301 +1301 @@
-things))))
+things#thing_n1))))
@@ -1305 +1305 @@
-haue)
+haue#have_v)
@@ -1307 +1307 @@
-altered)
+altered#alter_v)
@@ -1310 +1310 @@
-the)
+the#the_adj)
@@ -1312 +1312 @@
-face)
+face#face_n)
@@ -1315 +1315 @@
-of)
+of#of_prep)
@@ -1318 +1318 @@
-his)
+his#his_adj)
@@ -1320 +1320 @@
-affaires)))))))
+affaires#affair_n)))))))
@@ -1324 +1324 @@
-that)
+that#that_conj)
@@ -1330 +1330 @@
-he))
+he#he_pron))
@@ -1332 +1332 @@
-can)
+can#can_v1)
@@ -1334 +1334 @@
-not)
+not#not_adv)
@@ -1336 +1336 @@
-but)
+but#but_prep)
@@ -1338 +1338 @@
-Judge)
+Judge#judge_v)
@@ -1343 +1343 @@
-That)
+That#that_conj)
@@ -1347 +1347 @@
-our)
+our#our_pron)
@@ -1349 +1349 @@
-prejudice))
+prejudice#prejudice_n))
@@ -1353 +1353 @@
-may)
+may#may_v1)
@@ -1358 +1358 @@
-in)
+in#in_prep)
@@ -1361 +1361 @@
-consequence)))
+consequence#consequence_n)))
@@ -1365 +1365 @@
-proue)
+proue#prove_v)
@@ -1368 +1368 @@
-his)
+his#his_adj)
@@ -1370 +1370 @@
-danger))))))))))))))
+danger#danger_n))))))))))))))
@@ -1379 +1379 @@
-When)
+When#when_adv)
@@ -1386 +1386 @@
-wee))
+wee#we_pron))
@@ -1389 +1389 @@
-further))
+further#further_adj))
@@ -1391 +1391 @@
-replied)
+replied#reply_v)
@@ -1396 +1396 @@
-that)
+that#that_conj)
@@ -1400 +1400 @@
-wee))
+wee#we_pron))
@@ -1402 +1402 @@
-ded)
+ded#do_v)
@@ -1404 +1404 @@
-heare)
+heare#hear_v)
@@ -1409 +1409 @@
-of)
+of#of_prep)
@@ -1412 +1412 @@
-diuers)
+diuers#divers_adj)
@@ -1414 +1414 @@
-Ships)
+Ships#ship_n1)
@@ -1417 +1417 @@
-in)
+in#in_prep)
@@ -1420 +1420 @@
-france)))
+france#NA)))
@@ -1426 +1426 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1433 +1433 @@
-were)
+were#be_v)
@@ -1435 +1435 @@
-destined)
+destined#destine_v)
@@ -1438 +1438 @@
-to)
+to#to_prep)
@@ -1441 +1441 @@
-Spayne)))))
+Spayne#NA)))))
@@ -1447 +1447 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1453 +1453 @@
-wee))
+wee#we_pron))
@@ -1457 +1457 @@
-could)
+could#can_v1)
@@ -1459 +1459 @@
-not)
+not#not_adv)
@@ -1464 +1464 @@
-w=th=out)
+w=th=out#without_adv)
@@ -1467 +1467 @@
-great)
+great#great_adj)
@@ -1469 +1469 @@
-Inconvenience)))
+Inconvenience#inconvenience_n)))
@@ -1473 +1473 @@
-lett)
+lett#let_v1)
@@ -1478 +1478 @@
-pass)))))))))))
+pass#pass_v)))))))))))
@@ -1483 +1483 @@
-he))
+he#he_pron))
@@ -1485 +1485 @@
-ded)
+ded#do_v)
@@ -1487 +1487 @@
-assure)
+assure#assure_v)
@@ -1490 +1490 @@
-vs))
+vs#us_pron))
@@ -1495 +1495 @@
-that)
+that#that_conj)
@@ -1499 +1499 @@
-he))
+he#he_pron))
@@ -1501 +1501 @@
-would)
+would#will_v1)
@@ -1503 +1503 @@
-write)
+write#write_v)
@@ -1506 +1506 @@
-to)
+to#to_prep)
@@ -1509 +1509 @@
-the)
+the#the_adj)
@@ -1511 +1511 @@
-king)))))
+king#king_n)))))
@@ -1519 +1519 @@
-and)
+and#and_conj1)
@@ -1524 +1524 @@
-w=th=in)
+w=th=in#within_adv)
@@ -1527 +1527 @@
-few)
+few#few_adj)
@@ -1529 +1529 @@
-dayes)
+dayes#day_n)
@@ -1532 +1532 @@
-after))))
+after#after_adv))))
@@ -1536 +1536 @@
-ded)
+ded#do_v)
@@ -1538 +1538 @@
-acquaint)
+acquaint#acquaint_v)
@@ -1541 +1541 @@
-vs))
+vs#us_pron))
@@ -1546 +1546 @@
-that)
+that#that_conj)
@@ -1551 +1551 @@
-the)
+the#the_adj)
@@ -1553 +1553 @@
-king)
+king#king_n)
@@ -1556 +1556 @@
-his)
+his#his_adj)
@@ -1558 +1558 @@
-Master)))
+Master#master_n1)))
@@ -1560 +1560 @@
-would)
+would#will_v1)
@@ -1562 +1562 @@
-haue)
+haue#have_v)
@@ -1565 +1565 @@
-an)
+an#a_adj2)
@@ -1567 +1567 @@
-honorable)
+honorable#honourable_adj)
@@ -1569 +1569 @@
-care))
+care#care_n1))
@@ -1574 +1574 @@
-at)
+at#at_prep)
@@ -1577 +1577 @@
-this)
+this#this_pron)
@@ -1579 +1579 @@
-tyme))))
+tyme#time_n))))
@@ -1584 +1584 @@
-and)
+and#and_conj1)
@@ -1589 +1589 @@
-had)
+had#have_v)
@@ -1591 +1591 @@
-sent)
+sent#send_v1)
@@ -1594 +1594 @@
-into)
+into#into_prep)
@@ -1597 +1597 @@
-france)))
+france#NA)))
@@ -1600 +1600 @@
-to)
+to#to_prep)
@@ -1602 +1602 @@
-discover)
+discover#discover_v)
@@ -1606 +1606 @@
-in)
+in#in_prep)
@@ -1609 +1609 @@
-what)
+what#what_pron)
@@ -1611 +1611 @@
-Ports)))
+Ports#port_n1)))
@@ -1621 +1621 @@
-these)
+these#these_pron)
@@ -1623 +1623 @@
-preparations))
+preparations#preparation_n))
@@ -1625 +1625 @@
-were))))))))
+were#be_v))))))))
@@ -1633 +1633 @@
-But)
+But#but_prep)
@@ -1636 +1636 @@
-now)
+now#now_adv)
@@ -1639 +1639 @@
-of)
+of#of_prep)
@@ -1642 +1642 @@
-late))))
+late#late_adj1))))
@@ -1647 +1647 @@
-when)
+when#when_adv)
@@ -1654 +1654 @@
-he))
+he#he_pron))
@@ -1656 +1656 @@
-pursued)
+pursued#pursue_v)
@@ -1659 +1659 @@
-a)
+a#a_adj2)
@@ -1661 +1661 @@
-Remonstrance)
+Remonstrance#remonstrance_n)
@@ -1666 +1666 @@
-concerning)
+concerning#concerning_prep)
@@ -1669 +1669 @@
-the)
+the#the_adj)
@@ -1671 +1671 @@
-trafficq)))))))
+trafficq#traffic_n)))))))
@@ -1676 +1676 @@
-he))
+he#he_pron))
@@ -1678 +1678 @@
-handled)
+handled#handle_v1)
@@ -1681 +1681 @@
-two)
+two#two_adj)
@@ -1683 +1683 @@
-principall)
+principall#principal_adj)
@@ -1685 +1685 @@
-poincts))
+poincts#point_n1))
@@ -1688 +1688 @@
-w=th=)
+w=th=#with_prep)
@@ -1691 +1691 @@
-our)
+our#our_pron)
@@ -1693 +1693 @@
-Counsaill)))
+Counsaill#council_n)))
@@ -1702 +1702 @@
-the)
+the#the_adj)
@@ -1704 +1704 @@
-one))
+one#one_adj))
@@ -1709 +1709 @@
-To)
+To#to_prep)
@@ -1711 +1711 @@
-haue)
+haue#have_v)
@@ -1714 +1714 @@
-Justice)
+Justice#justice_n)
@@ -1717 +1717 @@
-for)
+for#for_prep)
@@ -1720 +1720 @@
-things)
+things#thing_n1)
@@ -1723 +1723 @@
-past))))))
+past#past_adj))))))
@@ -1732 +1732 @@
-The)
+The#the_adj)
@@ -1734 +1734 @@
-other))
+other#other_adj))
@@ -1739 +1739 @@
-To)
+To#to_prep)
@@ -1741 +1741 @@
-haue)
+haue#have_v)
@@ -1745 +1745 @@
-an)
+an#a_adj2)
@@ -1747 +1747 @@
-Order))
+Order#order_n))
@@ -1749 +1749 @@
-established)
+established#establish_v)
@@ -1754 +1754 @@
-for)
+for#for_prep)
@@ -1757 +1757 @@
-prevention)
+prevention#prevention_n)
@@ -1760 +1760 @@
-of)
+of#of_prep)
@@ -1763 +1763 @@
-lyke)
+lyke#like_adj)
@@ -1765 +1765 @@
-supposed)
+supposed#supposed_adj)
@@ -1767 +1767 @@
-spoiles)
+spoiles#spoil_n)
@@ -1770 +1770 @@
-in)
+in#in_prep)
@@ -1773 +1773 @@
-future)))))))))
+future#future_adj)))))))))
@@ -1784 +1784 @@
-How)
+How#how_adv)
@@ -1786 +1786 @@
-properly))
+properly#properly_adv))
@@ -1794 +1794 @@
-these))
+these#these_pron))
@@ -1796 +1796 @@
-ded)
+ded#do_v)
@@ -1798 +1798 @@
-fall)
+fall#fall_v)
@@ -1801 +1801 @@
-into)
+into#into_prep)
@@ -1804 +1804 @@
-the)
+the#the_adj)
@@ -1806 +1806 @@
-Center)
+Center#centre_n1)
@@ -1809 +1809 @@
-of)
+of#of_prep)
@@ -1812 +1812 @@
-our)
+our#our_pron)
@@ -1814 +1814 @@
-desires)))))
+desires#desire_n)))))
@@ -1819 +1819 @@
-as)
+as#as_adv)
@@ -1822 +1822 @@
-being)
+being#be_v)
@@ -1824 +1824 @@
-both)
+both#both_pron)
@@ -1827 +1827 @@
-essentiall)
+essentiall#essential_adj)
@@ -1829 +1829 @@
-parts)
+parts#part_n)
@@ -1832 +1832 @@
-of)
+of#of_prep)
@@ -1835 +1835 @@
-all)
+all#all_adj)
@@ -1837 +1837 @@
-kingly)
+kingly#kingly_adj)
@@ -1839 +1839 @@
-gouuernement))))
+gouuernement#government_n))))
@@ -1846 +1846 @@
-the)
+the#the_adj)
@@ -1848 +1848 @@
-one))
+one#one_adj))
@@ -1853 +1853 @@
-for)
+for#for_prep)
@@ -1856 +1856 @@
-administration)
+administration#administration_n)
@@ -1859 +1859 @@
-of)
+of#of_prep)
@@ -1862 +1862 @@
-Justice))))))
+Justice#justice_n))))))
@@ -1867 +1867 @@
-the)
+the#the_adj)
@@ -1869 +1869 @@
-other))
+other#other_adj))
@@ -1874 +1874 @@
-to)
+to#to_prep)
@@ -1877 +1877 @@
-conserve)
+conserve#conserve_v)
@@ -1879 +1879 @@
-and)
+and#and_conj1)
@@ -1881 +1881 @@
-augment))
+augment#augment_v))
@@ -1884 +1884 @@
-Liberties)
+Liberties#liberty_n)
@@ -1886 +1886 @@
-and)
+and#and_conj1)
@@ -1888 +1888 @@
-fortunes)
+fortunes#fortune_n)
@@ -1891 +1891 @@
-of)
+of#of_prep)
@@ -1894 +1894 @@
-our)
+our#our_pron)
@@ -1896 +1896 @@
-subiects))))
+subiects#subject_n))))
@@ -1902 +1902 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1909 +1909 @@
-draweth)
+draweth#draw_v)
@@ -1912 +1912 @@
-their)
+their#their_adj)
@@ -1914 +1914 @@
-loue)
+loue#love_n1)
@@ -1917 +1917 @@
-and)
+and#and_conj1)
@@ -1920 +1920 @@
-obedience)))))))))))))))
+obedience#obedience_n)))))))))))))))
@@ -1924 +1924 @@
-may)
+may#may_v1)
@@ -1926 +1926 @@
-appeare)
+appeare#appear_v)
@@ -1929 +1929 @@
-if)
+if#if_conj)
@@ -1936 +1936 @@
-we))
+we#we_pron))
@@ -1938 +1938 @@
-be)
+be#be_v)
@@ -1941 +1941 @@
-rightly))
+rightly#rightly_adv))
@@ -1944 +1944 @@
-reported)
+reported#report_v)
@@ -1948 +1948 @@
-and)
+and#and_conj1)
@@ -1950 +1950 @@
-censured)))))
+censured#censure_v)))))
@@ -1958 +1958 @@
-For)
+For#for_prep)
@@ -1961 +1961 @@
-seing)
+seing#see_v)
@@ -1968 +1968 @@
-Princes))
+Princes#prince_n))
@@ -1971 +1971 @@
-by)
+by#by_prep)
@@ -1974 +1974 @@
-Institution)))
+Institution#institution_n)))
@@ -1976 +1976 @@
-must)
+must#must_v1)
@@ -1978 +1978 @@
-vse)
+vse#use_v)
@@ -1981 +1981 @@
-the)
+the#the_adj)
@@ -1983 +1983 @@
-Judgements)
+Judgements#judgement_n)
@@ -1986 +1986 @@
-of)
+of#of_prep)
@@ -1989 +1989 @@
-subaltern)
+subaltern#subaltern_n)
@@ -1991 +1991 @@
-Ministers))))
+Ministers#minister_n))))
@@ -1996 +1996 @@
-in)
+in#in_prep)
@@ -1999 +1999 @@
-distribution)
+distribution#distribution_n)
@@ -2002 +2002 @@
-of)
+of#of_prep)
@@ -2005 +2005 @@
-gouuernement))))))))
+gouuernement#government_n))))))))
@@ -2010 +2010 @@
-wee))
+wee#we_pron))
@@ -2012 +2012 @@
-ded)
+ded#do_v)
@@ -2015 +2015 @@
-immediately))
+immediately#immediately_adv))
@@ -2017 +2017 @@
-enjoine)
+enjoine#enjoin_v)
@@ -2020 +2020 @@
-all)
+all#all_adj)
@@ -2022 +2022 @@
-those)
+those#those_pron)
@@ -2026 +2026 @@
-to)
+to#to_prep)
@@ -2029 +2029 @@
-whome)))
+whome#whom_pron)))
@@ -2037 +2037 @@
-such)
+such#such_adj)
@@ -2039 +2039 @@
-aucthority)
+aucthority#authority_n)
@@ -2042 +2042 @@
-and)
+and#and_conj1)
@@ -2045 +2045 @@
-execution))))
+execution#execution_n))))
@@ -2047 +2047 @@
-belongeth))))
+belongeth#belong_v))))
@@ -2053 +2053 @@
-to)
+to#to_prep)
@@ -2055 +2055 @@
-receaue)
+receaue#receive_v)
@@ -2058 +2058 @@
-w=th=out)
+w=th=out#without_adv)
@@ -2061 +2061 @@
-delay)))
+delay#delay_n)))
@@ -2066 +2066 @@
-all)
+all#all_adj)
@@ -2068 +2068 @@
-Cahiers)
+Cahiers#cahier_n)
@@ -2071 +2071 @@
-of)
+of#of_prep)
@@ -2074 +2074 @@
-complaints)))))
+complaints#complaint_n)))))
@@ -2079 +2079 @@
-and)
+and#and_conj1)
@@ -2082 +2082 @@
-to)
+to#to_prep)
@@ -2084 +2084 @@
-bethinke)
+bethinke#bethink_v)
@@ -2091 +2091 @@
-how))
+how#how_adv))
@@ -2099 +2099 @@
-wee))
+wee#we_pron))
@@ -2101 +2101 @@
-might)
+might#may_v1)
@@ -2104 +2104 @@
-speedily))
+speedily#speedily_adv))
@@ -2106 +2106 @@
-reforme)
+reforme#reform_v1)
@@ -2109 +2109 @@
-all)
+all#all_adj)
@@ -2111 +2111 @@
-past)
+past#past_adj)
@@ -2113 +2113 @@
-disorders)
+disorders#disorder_n)
@@ -2120 +2120 @@
-that)
+that#that_conj)
@@ -2125 +2125 @@
-should)
+should#shall_v)
@@ -2127 +2127 @@
-fall)
+fall#fall_v)
@@ -2129 +2129 @@
-out)
+out#out_adv)
@@ -2132 +2132 @@
-to)
+to#to_prep)
@@ -2134 +2134 @@
-be)
+be#be_v)
@@ -2136 +2136 @@
-proved)))))))
+proved#prove_v)))))))
@@ -2141 +2141 @@
-and)
+and#and_conj1)
@@ -2145 +2145 @@
-how))
+how#how_adv))
@@ -2153 +2153 @@
-wee))
+wee#we_pron))
@@ -2155 +2155 @@
-might)
+might#may_v1)
@@ -2157 +2157 @@
-settle)
+settle#settle_v)
@@ -2160 +2160 @@
-a)
+a#a_adj2)
@@ -2162 +2162 @@
-course)
+course#course_n1)
@@ -2165 +2165 @@
-of)
+of#of_prep)
@@ -2168 +2168 @@
-quiet)
+quiet#quiet_adj)
@@ -2170 +2170 @@
-commerce))))
+commerce#commerce_n))))
@@ -2173 +2173 @@
-hereafter)))))))))
+hereafter#hereafter_adv)))))))))
@@ -2182 +2182 @@
-Herein))
+Herein#herein_adv))
@@ -2187 +2187 @@
-what))
+what#what_pron))
@@ -2194 +2194 @@
-hath)
+hath#have_v)
@@ -2196 +2196 @@
-followed))))
+followed#follow_v))))
@@ -2200 +2200 @@
-shall)
+shall#shall_v)
@@ -2202 +2202 @@
-appeare)
+appeare#appear_v)
@@ -2205 +2205 @@
-by)
+by#by_prep)
@@ -2208 +2208 @@
-this)
+this#this_pron)
@@ -2210 +2210 @@
-Memoriall)
+Memoriall#memorial_adj)
@@ -2216 +2216 @@
-whereof))
+whereof#whereof_adv))
@@ -2222 +2222 @@
-wee))
+wee#we_pron))
@@ -2224 +2224 @@
-haue)
+haue#have_v)
@@ -2226 +2226 @@
-geuen)
+geuen#give_v)
@@ -2229 +2229 @@
-his)
+his#his_adj)
@@ -2231 +2231 @@
-Embass=r=))
+Embass=r=#embassador_n))
@@ -2234 +2234 @@
-a)
+a#a_adj2)
@@ -2236 +2236 @@
-double)
+double#double_n)
@@ -2244 +2244 @@
-to)
+to#to_prep)
@@ -2247 +2247 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -2255 +2255 @@
-wee))
+wee#we_pron))
@@ -2257 +2257 @@
-referre)
+referre#refer_v)
@@ -2260 +2260 @@
-you)))))))
+you#you_pron)))))))
@@ -2265 +2265 @@
-as)
+as#as_adv)
@@ -2268 +2268 @@
-being)
+being#be_v)
@@ -2271 +2271 @@
-the)
+the#the_adj)
@@ -2273 +2273 @@
-substance)
+substance#substance_n)
@@ -2276 +2276 @@
-of)
+of#of_prep)
@@ -2279 +2279 @@
-diuers)
+diuers#divers_adj)
@@ -2281 +2281 @@
-Consultations)
+Consultations#consultation_n)
@@ -2286 +2286 @@
-between)
+between#between_prep)
@@ -2290 +2290 @@
-the)
+the#the_adj)
@@ -2292 +2292 @@
-Embass=r=))
+Embass=r=#embassador_n))
@@ -2295 +2295 @@
-and)
+and#and_conj1)
@@ -2298 +2298 @@
-our)
+our#our_pron)
@@ -2300 +2300 @@
-Counsaill)))
+Counsaill#council_n)))
@@ -2305 +2305 @@
-and)
+and#and_conj1)
@@ -2308 +2308 @@
-other)
+other#other_adj)
@@ -2310 +2310 @@
-personnes)
+personnes#person_n)
@@ -2313 +2313 @@
-of)
+of#of_prep)
@@ -2316 +2316 @@
-quality)
+quality#quality_n)
@@ -2318 +2318 @@
-and)
+and#and_conj1)
@@ -2320 +2320 @@
-vnderstanding)))
+vnderstanding#understanding_n)))
@@ -2325 +2325 @@
-appointed)
+appointed#appoint_v)
@@ -2328 +2328 @@
-to)
+to#to_prep)
@@ -2331 +2331 @@
-attend)
+attend#attend_v)
@@ -2333 +2333 @@
-and)
+and#and_conj1)
@@ -2335 +2335 @@
-assist))
+assist#assist_v))
@@ -2338 +2338 @@
-them))
+them#them_pron))
@@ -2343 +2343 @@
-in)
+in#in_prep)
@@ -2346 +2346 @@
-matters)
+matters#matter_n1)
@@ -2349 +2349 @@
-incident)
+incident#incident_adj1)
@@ -2352 +2352 @@
-to)
+to#to_prep)
@@ -2355 +2355 @@
-questions)
+questions#question_n)
@@ -2358 +2358 @@
-of)
+of#of_prep)
@@ -2361 +2361 @@
-this)
+this#this_pron)
@@ -2363 +2363 @@
-Nature)))))))))))))))))))))
+Nature#nature_n)))))))))))))))))))))
@@ -2369 +2369 @@
-wherein))
+wherein#wherein_adv))
@@ -2377 +2377 @@
-although)
+although#although_conj)
@@ -2384 +2384 @@
-it))
+it#it_pron))
@@ -2386 +2386 @@
-shall)
+shall#shall_v)
@@ -2388 +2388 @@
-appeare)
+appeare#appear_v)
@@ -2393 +2393 @@
-by)
+by#by_prep)
@@ -2396 +2396 @@
-the)
+the#the_adj)
@@ -2398 +2398 @@
-aunswer)
+aunswer#answer_n)
@@ -2401 +2401 @@
-in)
+in#in_prep)
@@ -2404 +2404 @@
-writing)))))
+writing#writing_n)))))
@@ -2409 +2409 @@
-that)
+that#that_conj)
@@ -2413 +2413 @@
-wee))
+wee#we_pron))
@@ -2415 +2415 @@
-insist)
+insist#insist_v)
@@ -2418 +2418 @@
-vpon)
+vpon#upon_prep)
@@ -2421 +2421 @@
-nothing)
+nothing#nothing_pron)
@@ -2426 +2426 @@
-but)
+but#but_prep)
@@ -2429 +2429 @@
-that)
+that#that_pron)
@@ -2435 +2435 @@
-w=ch=))
+w=ch=#which_adj))
@@ -2444 +2444 @@
-the)
+the#the_adj)
@@ -2446 +2446 @@
-Law)
+Law#law_n1)
@@ -2449 +2449 @@
-of)
+of#of_prep)
@@ -2452 +2452 @@
-Nations))))
+Nations#nation_n))))
@@ -2457 +2457 @@
-and)
+and#and_conj1)
@@ -2460 +2460 @@
-former)
+former#former_adj)
@@ -2462 +2462 @@
-Treaties)
+Treaties#treaty_n)
@@ -2465 +2465 @@
-between)
+between#between_prep)
@@ -2468 +2468 @@
-Princes))))))
+Princes#prince_n))))))
@@ -2472 +2472 @@
-may)
+may#may_v1)
@@ -2474 +2474 @@
-warrant))))))))))))
+warrant#warrant_v))))))))))))
@@ -2479 +2479 @@
-yet))
+yet#yet_adv))
@@ -2482 +2482 @@
-wee))
+wee#we_pron))
@@ -2484 +2484 @@
-command)
+command#command_v)
@@ -2487 +2487 @@
-you))
+you#you_pron))
@@ -2492 +2492 @@
-to)
+to#to_prep)
@@ -2494 +2494 @@
-tell)
+tell#tell_v)
@@ -2497 +2497 @@
-him))
+him#him_pron))
@@ -2500 +2500 @@
-plainely))
+plainely#plainly_adv1))
@@ -2505 +2505 @@
-that)
+that#that_conj)
@@ -2509 +2509 @@
-wee))
+wee#we_pron))
@@ -2511 +2511 @@
-could)
+could#can_v1)
@@ -2514 +2514 @@
-never))
+never#never_adv))
@@ -2516 +2516 @@
-haue)
+haue#have_v)
@@ -2518 +2518 @@
-expected)
+expected#expect_v)
@@ -2523 +2523 @@
-at)
+at#at_prep)
@@ -2526 +2526 @@
-this)
+this#this_pron)
@@ -2528 +2528 @@
-tyme)
+tyme#time_n)
@@ -2531 +2531 @@
-when)
+when#when_adv)
@@ -2538 +2538 @@
-wee))
+wee#we_pron))
@@ -2540 +2540 @@
-are)
+are#be_v)
@@ -2543 +2543 @@
-so)
+so#so_adv)
@@ -2545 +2545 @@
-full)
+full#full_adj)
@@ -2548 +2548 @@
-of)
+of#of_prep)
@@ -2552 +2552 @@
-trouble))
+trouble#trouble_n))
@@ -2555 +2555 @@
-and)
+and#and_conj1)
@@ -2558 +2558 @@
-expectations)
+expectations#expectation_n)
@@ -2561 +2561 @@
-of)
+of#of_prep)
@@ -2564 +2564 @@
-danger)
+danger#danger_n)
@@ -2567 +2567 @@
-to)
+to#to_prep)
@@ -2570 +2570 @@
-our)
+our#our_pron)
@@ -2572 +2572 @@
-state)))))))))))))))
+state#state_n)))))))))))))))
@@ -2581 +2581 @@
-wee))
+wee#we_pron))
@@ -2583 +2583 @@
-should)
+should#shall_v)
@@ -2585 +2585 @@
-haue)
+haue#have_v)
@@ -2587 +2587 @@
-ben)
+ben#be_v)
@@ -2590 +2590 @@
-by)
+by#by_prep)
@@ -2593 +2593 @@
-him)))
+him#him_pron)))
@@ -2595 +2595 @@
-denyed)
+denyed#deny_v)
@@ -2598 +2598 @@
-any)
+any#any_adj)
@@ -2600 +2600 @@
-such)
+such#such_adj)
@@ -2602 +2602 @@
-matter))
+matter#matter_n1))
@@ -2607 +2607 @@
-though)
+though#though_adv)
@@ -2614 +2614 @@
-it))
+it#it_pron))
@@ -2616 +2616 @@
-were)
+were#be_v)
@@ -2619 +2619 @@
-more)
+more#more_adj)
@@ -2621 +2621 @@
-vnproffitable)
+vnproffitable#unprofitable_adj)
@@ -2624 +2624 @@
-to)
+to#to_prep)
@@ -2627 +2627 @@
-his)
+his#his_adj)
@@ -2631 +2631 @@
-subiects)))))))))
+subiects#subject_n)))))))))
@@ -2636 +2636 @@
-considering)
+considering#consider_v)
@@ -2642 +2642 @@
-how)
+how#how_adv)
@@ -2644 +2644 @@
-many)
+many#many_adj)
@@ -2647 +2647 @@
-of)
+of#of_prep)
@@ -2650 +2650 @@
-ours)))))
+ours#ours_pron)))))
@@ -2660 +2660 @@
-wee))
+wee#we_pron))
@@ -2662 +2662 @@
-haue)
+haue#have_v)
@@ -2664 +2664 @@
-sacrificed)
+sacrificed#sacrifice_v)
@@ -2667 +2667 @@
-for)
+for#for_prep)
@@ -2670 +2670 @@
-his)
+his#his_adj)
@@ -2672 +2672 @@
-sake)))))
+sake#sake_n1)))))
@@ -2677 +2677 @@
-and)
+and#and_conj1)
@@ -2682 +2682 @@
-how)
+how#how_adv)
@@ -2684 +2684 @@
-little)))
+little#little_adj)))
@@ -2692 +2692 @@
-wee))
+wee#we_pron))
@@ -2694 +2694 @@
-haue)
+haue#have_v)
@@ -2696 +2696 @@
-weighed)
+weighed#weigh_v1)
@@ -2699 +2699 @@
-Vtilety))
+Vtilety#utility_n))
@@ -2704 +2704 @@
-when)
+when#when_adv)
@@ -2711 +2711 @@
-there))
+there#there_adv))
@@ -2713 +2713 @@
-hath)
+hath#have_v)
@@ -2715 +2715 @@
-ben)
+ben#be_v)
@@ -2718 +2718 @@
-question)
+question#question_n)
@@ -2721 +2721 @@
-of)
+of#of_prep)
@@ -2724 +2724 @@
-his)
+his#his_adj)
@@ -2726 +2726 @@
-safety)))))))))))))))))
+safety#safety_n)))))))))))))))))
@@ -2735 +2735 @@
-This))
+This#this_pron))
@@ -2737 +2737 @@
-do)
+do#do_v)
@@ -2740 +2740 @@
-wee))
+wee#we_pron))
@@ -2742 +2742 @@
-command)
+command#command_v)
@@ -2745 +2745 @@
-you))
+you#you_pron))
@@ -2752 +2752 @@
-to)
+to#to_prep)
@@ -2754 +2754 @@
-deliure)
+deliure#deliver_v1)
@@ -2757 +2757 @@
-vnto)
+vnto#unto_prep)
@@ -2760 +2760 @@
-him))))
+him#him_pron))))
@@ -2766 +2766 @@
-not)
+not#not_adv)
@@ -2768 +2768 @@
-as)
+as#as_adv)
@@ -2771 +2771 @@
-vpbraiding)
+vpbraiding#upbraid_v)
@@ -2774 +2774 @@
-our)
+our#our_pron)
@@ -2776 +2776 @@
-former)
+former#former_adj)
@@ -2778 +2778 @@
-helps))))
+helps#help_n))))
@@ -2783 +2783 @@
-or)
+or#or_conj2)
@@ -2786 +2786 @@
-as)
+as#as_adv)
@@ -2789 +2789 @@
-sorry)
+sorry#sorry_adj)
@@ -2792 +2792 @@
-for)
+for#for_prep)
@@ -2795 +2795 @@
-their)
+their#their_adj)
@@ -2797 +2797 @@
-effects)
+effects#effect_n)
@@ -2800 +2800 @@
-in)
+in#in_prep)
@@ -2803 +2803 @@
-his)
+his#his_adj)
@@ -2805 +2805 @@
-fortune))))))))
+fortune#fortune_n))))))))
@@ -2810 +2810 @@
-but)
+but#but_prep)
@@ -2813 +2813 @@
-as)
+as#as_adv)
@@ -2816 +2816 @@
-a)
+a#a_adj2)
@@ -2818 +2818 @@
-Prince)
+Prince#prince_n)
@@ -2823 +2823 @@
-that)
+that#that_conj)
@@ -2828 +2828 @@
-is)
+is#be_v)
@@ -2831 +2831 @@
-sensible)
+sensible#sensible_adj)
@@ -2834 +2834 @@
-of)
+of#of_prep)
@@ -2837,3 +2837,3 @@
-the)
-(ADJ
-strang)
+the#the_adj)
+(ADJ
+strang#strange_adj)
@@ -2841 +2841 @@
-proposition)
+proposition#proposition_n)
@@ -2844 +2844 @@
-of)
+of#of_prep)
@@ -2847 +2847 @@
-his)
+his#his_adj)
@@ -2849 +2849 @@
-Embassad=r=)))
+Embassad=r=#embassador_n)))
@@ -2853 +2853 @@
-w=ch=))
+w=ch=#which_adj))
@@ -2859 +2859 @@
-wee))
+wee#we_pron))
@@ -2861 +2861 @@
-can)
+can#can_v1)
@@ -2863 +2863 @@
-not)
+not#not_adv)
@@ -2865 +2865 @@
-devise)
+devise#devise_v)
@@ -2869 +2869 @@
-how))
+how#how_adv))
@@ -2876 +2876 @@
-to)
+to#to_prep)
@@ -2878 +2878 @@
-match)
+match#match_v1)
@@ -2881 +2881 @@
-w=th=)
+w=th=#with_prep)
@@ -2884 +2884 @@
-his)
+his#his_adj)
@@ -2886 +2886 @@
-first)
+first#first_adj)
@@ -2888 +2888 @@
-Introductions)
+Introductions#introduction_n)
@@ -2891 +2891 @@
-into)
+into#into_prep)
@@ -2894 +2894 @@
-this)
+this#this_pron)
@@ -2896 +2896 @@
-cause))))))))))))))))))
+cause#cause_n))))))))))))))))))
@@ -2904 +2904 @@
-for)
+for#for_prep)
@@ -2907 +2907 @@
-as)
+as#as_adv)
@@ -2918 +2918 @@
-wee))
+wee#we_pron))
@@ -2920 +2920 @@
-haue)
+haue#have_v)
@@ -2923 +2923 @@
-for)
+for#for_prep)
@@ -2926 +2926 @@
-settling)
+settling#settling_n)
@@ -2929 +2929 @@
-of)
+of#of_prep)
@@ -2932 +2932 @@
-Justice)))))
+Justice#justice_n)))))
@@ -2934 +2934 @@
-agreed)
+agreed#agree_v)
@@ -2937 +2937 @@
-of)
+of#of_prep)
@@ -2940 +2940 @@
-the)
+the#the_adj)
@@ -2942 +2942 @@
-forme)
+forme#form_n)
@@ -2945 +2945 @@
-of)
+of#of_prep)
@@ -2948 +2948 @@
-Tryalls)))))
+Tryalls#trial_n1)))))
@@ -2953 +2953 @@
-by)
+by#by_prep)
@@ -2956 +2956 @@
-accomodating)
+accomodating#accommodate_v)
@@ -2959 +2959 @@
-ourselfes))
+ourselfes#ourself_pron))
@@ -2964 +2964 @@
-in)
+in#in_prep)
@@ -2967 +2967 @@
-any)
+any#any_adj)
@@ -2969 +2969 @@
-course)
+course#course_n1)
@@ -2976 +2976 @@
-that)
+that#that_conj)
@@ -2981 +2981 @@
-may)
+may#may_v1)
@@ -2983 +2983 @@
-not)
+not#not_adv)
@@ -2985 +2985 @@
-make)
+make#make_v1)
@@ -2988 +2988 @@
-a)
+a#a_adj2)
@@ -2990 +2990 @@
-Renverse)
+Renverse#reverse_n)
@@ -2993 +2993 @@
-of)
+of#of_prep)
@@ -2996 +2996 @@
-all)
+all#all_adj)
@@ -2998 +2998 @@
-our)
+our#our_pron)
@@ -3000 +3000 @@
-formes)
+formes#form_n)
@@ -3003 +3003 @@
-of)
+of#of_prep)
@@ -3006 +3006 @@
-Justice)))))))))))))))
+Justice#justice_n)))))))))))))))
@@ -3011 +3011 @@
-wee))
+wee#we_pron))
@@ -3013 +3013 @@
-must)
+must#must_v1)
@@ -3016 +3016 @@
-needs))
+needs#needs_adv))
@@ -3018 +3018 @@
-say)
+say#say_v1)
@@ -3021 +3021 @@
-this)
+this#this_pron)
@@ -3026 +3026 @@
-That)
+That#that_conj)
@@ -3030 +3030 @@
-now)
+now#now_adv)
@@ -3036 +3036 @@
-that)
+that#that_conj)
@@ -3042 +3042 @@
-he))
+he#he_pron))
@@ -3044 +3044 @@
-is)
+is#be_v)
@@ -3046 +3046 @@
-informed)
+informed#inform_v)
@@ -3051 +3051 @@
-that)
+that#that_conj)
@@ -3055 +3055 @@
-an)
+an#a_adj2)
@@ -3057 +3057 @@
-Army))
+Army#army_n))
@@ -3059 +3059 @@
-is)
+is#be_v)
@@ -3061 +3061 @@
-making)))))
+making#make_v1)))))
@@ -3066 +3066 @@
-and)
+and#and_conj1)
@@ -3071 +3071 @@
-that)
+that#that_conj)
@@ -3077 +3077 @@
-it))
+it#it_pron))
@@ -3079 +3079 @@
-is)
+is#be_v)
@@ -3082 +3082 @@
-evident))
+evident#evident_adj))
@@ -3087 +3087 @@
-that)
+that#that_conj)
@@ -3091 +3091 @@
-a)
+a#a_adj2)
@@ -3093 +3093 @@
-particuler)
+particuler#particular_adj)
@@ -3095 +3095 @@
-portion)
+portion#portion_n)
@@ -3098 +3098 @@
-of)
+of#of_prep)
@@ -3101 +3101 @@
-Corne))))
+Corne#corn_n1))))
@@ -3103 +3103 @@
-is)
+is#be_v)
@@ -3105 +3105 @@
-expected)
+expected#expect_v)
@@ -3108 +3108 @@
-from)
+from#from_prep)
@@ -3111 +3111 @@
-Nantes)))
+Nantes#NA)))
@@ -3116 +3116 @@
-to)
+to#to_prep)
@@ -3118 +3118 @@
-furnish)
+furnish#furnish_v)
@@ -3121 +3121 @@
-that)
+that#that_pron)
@@ -3123 +3123 @@
-expedition)
+expedition#expedition_n)
@@ -3126 +3126 @@
-into)
+into#into_prep)
@@ -3129 +3129 @@
-Ireland))))))))))))
+Ireland#NA))))))))))))
@@ -3134 +3134 @@
-If)
+If#if_conj)
@@ -3141 +3141 @@
-he))
+he#he_pron))
@@ -3143 +3143 @@
-should)
+should#shall_v)
@@ -3145 +3145 @@
-yeald)
+yeald#yield_v)
@@ -3148 +3148 @@
-no)
+no#no_adj)
@@ -3150 +3150 @@
-further)
+further#further_adj)
@@ -3155 +3155 @@
-then)
+then#than_conj)
@@ -3166 +3166 @@
-ordinary)
+ordinary#ordinary_adj)
@@ -3168 +3168 @@
-rules))
+rules#rule_n1))
@@ -3170 +3170 @@
-would)
+would#will_v1)
@@ -3172 +3172 @@
-tye)
+tye#tie_v)
@@ -3175 +3175 @@
-him)))))))))
+him#him_pron)))))))))
@@ -3180 +3180 @@
-although)
+although#although_conj)
@@ -3187 +3187 @@
-we))
+we#we_pron))
@@ -3189 +3189 @@
-had)
+had#have_v)
@@ -3191 +3191 @@
-not)
+not#not_adv)
@@ -3194 +3194 @@
-plainely))
+plainely#plainly_adv1))
@@ -3196 +3196 @@
-shewed)
+shewed#show_v)
@@ -3199 +3199 @@
-the)
+the#the_adj)
@@ -3201 +3201 @@
-contrary))
+contrary#contrary_adj))
@@ -3204 +3204 @@
-to)
+to#to_prep)
@@ -3207 +3207 @@
-his)
+his#his_adj)
@@ -3209 +3209 @@
-Embassad=r=)))
+Embassad=r=#embassador_n)))
@@ -3212 +3212 @@
-as)
+as#as_adv)
@@ -3223 +3223 @@
-wee))
+wee#we_pron))
@@ -3225 +3225 @@
-haue)
+haue#have_v)
@@ -3227 +3227 @@
-don)))))))
+don#do_v)))))))
@@ -3232 +3232 @@
-yet))
+yet#yet_adv))
@@ -3235 +3235 @@
-wee))
+wee#we_pron))
@@ -3237 +3237 @@
-must)
+must#must_v1)
@@ -3240 +3240 @@
-surely))
+surely#surely_adv))
@@ -3242 +3242 @@
-conclude)
+conclude#conclude_v)
@@ -3247 +3247 @@
-that)
+that#that_conj)
@@ -3251 +3251 @@
-we))
+we#we_pron))
@@ -3253 +3253 @@
-are)
+are#be_v)
@@ -3255 +3255 @@
-requited)
+requited#requite_v)
@@ -3258 +3258 @@
-w=th=)
+w=th=#with_prep)
@@ -3261 +3261 @@
-vneven)
+vneven#uneven_adj)
@@ -3263 +3263 @@
-measure)))))))))
+measure#measure_n)))))))))
@@ -3269 +3269 @@
-and)
+and#and_conj1)
@@ -3273 +3273 @@
-must)
+must#must_v1)
@@ -3275 +3275 @@
-leaue)
+leaue#leave_v1)
@@ -3278 +3278 @@
-it))
+it#it_pron))
@@ -3281 +3281 @@
-to)
+to#to_prep)
@@ -3284 +3284 @@
-himself)))
+himself#himself_pron)))
@@ -3289 +3289 @@
-if)
+if#if_conj)
@@ -3296 +3296 @@
-his)
+his#his_adj)
@@ -3298 +3298 @@
-case))
+case#case_n1))
@@ -3300 +3300 @@
-were)
+were#be_v)
@@ -3303 +3303 @@
-ours)))))
+ours#ours_pron)))))
@@ -3309 +3309 @@
-what))
+what#what_pron))
@@ -3317 +3317 @@
-he))
+he#he_pron))
@@ -3319 +3319 @@
-would)
+would#will_v1)
@@ -3321 +3321 @@
-think)
+think#think_v2)
@@ -3326 +3326 @@
-when)
+when#when_adv)
@@ -3333 +3333 @@
-in)
+in#in_prep)
@@ -3336 +3336 @@
-such)
+such#such_adj)
@@ -3338 +3338 @@
-an)
+an#a_adj2)
@@ -3340 +3340 @@
-instant)
+instant#instant_n)
@@ -3343 +3343 @@
-of)
+of#of_prep)
@@ -3346 +3346 @@
-preparations)))))
+preparations#preparation_n)))))
@@ -3351 +3351 @@
-our)
+our#our_pron)
@@ -3353 +3353 @@
-best)
+best#best_adj)
@@ -3355 +3355 @@
-frends))
+frends#friend_n))
@@ -3357 +3357 @@
-releeue)
+releeue#relieve_v)
@@ -3360 +3360 @@
-our)
+our#our_pron)
@@ -3362 +3362 @@
-adverse)
+adverse#adverse_adj)
@@ -3364 +3364 @@
-party))
+party#party_n))
@@ -3367 +3367 @@
-in)
+in#in_prep)
@@ -3370 +3370 @@
-their)
+their#their_adj)
@@ -3372 +3372 @@
-necessities))))))))
+necessities#necessity_n))))))))
@@ -3381 +3381 @@
-For)
+For#for_prep)
@@ -3384 +3384 @@
-the)
+the#the_adj)
@@ -3386 +3386 @@
-other)
+other#other_adj)
@@ -3388 +3388 @@
-poinct)
+poinct#point_n1)
@@ -3392 +3392 @@
-concerning)
+concerning#concerning_prep)
@@ -3395 +3395 @@
-the)
+the#the_adj)
@@ -3397 +3397 @@
-prohibition)
+prohibition#prohibition_n)
@@ -3400 +3400 @@
-intended)
+intended#intend_v)
@@ -3403 +3403 @@
-against)
+against#against_prep)
@@ -3406 +3406 @@
-our)
+our#our_pron)
@@ -3408 +3408 @@
-Cloath))))))
+Cloath#cloth_n))))))
@@ -3413 +3413 @@
-and)
+and#and_conj1)
@@ -3416 +3416 @@
-stayed)
+stayed#stay_v1)
@@ -3419 +3419 @@
-vpon)
+vpon#upon_prep)
@@ -3422 +3422 @@
-y=r=)
+y=r=#your_pron)
@@ -3424 +3424 @@
-remonstrance)))
+remonstrance#remonstrance_n)))
@@ -3429 +3429 @@
-for)
+for#for_prep)
@@ -3432 +3432 @@
-so))
+so#so_adv))
@@ -3435 +3435 @@
-his)
+his#his_adj)
@@ -3437 +3437 @@
-Embassad=r=))
+Embassad=r=#embassador_n))
@@ -3439 +3439 @@
-confesseth)))))))
+confesseth#confess_v)))))))
@@ -3444 +3444 @@
-we))
+we#we_pron))
@@ -3446 +3446 @@
-can)
+can#can_v1)
@@ -3448 +3448 @@
-not)
+not#not_adv)
@@ -3450 +3450 @@
-but)
+but#but_prep)
@@ -3453 +3453 @@
-well))
+well#well_adv))
@@ -3455 +3455 @@
-interprete)
+interprete#interpret_v)
@@ -3458 +3458 @@
-his)
+his#his_adj)
@@ -3460 +3460 @@
-owne)
+owne#own_adj)
@@ -3462 +3462 @@
-particuler)
+particuler#particular_adj)
@@ -3464 +3464 @@
-proceeding))
+proceeding#proceeding_n))
@@ -3469 +3469 @@
-having)
+having#have_v)
@@ -3471 +3471 @@
-vnderstood)
+vnderstood#understand_v)
@@ -3476 +3476 @@
-that)
+that#that_conj)
@@ -3480 +3480 @@
-it))
+it#it_pron))
@@ -3482 +3482 @@
-was)
+was#be_v)
@@ -3485 +3485 @@
-rather)
+rather#rather_adv)
@@ -3489 +3489 @@
-carryed)
+carryed#carry_v)
@@ -3491 +3491 @@
-on)
+on#on_adv)
@@ -3495 +3495 @@
-so)
+so#so_adv)
@@ -3497 +3497 @@
-farr)))
+farr#far_adj)))
@@ -3500 +3500 @@
-by)
+by#by_prep)
@@ -3503 +3503 @@
-his)
+his#his_adj)
@@ -3505 +3505 @@
-ministers)))
+ministers#minister_n)))
@@ -3510 +3510 @@
-then)
+then#than_conj)
@@ -3513 +3513 @@
-resolued)
+resolued#resolve_v)
@@ -3516 +3516 @@
-by)
+by#by_prep)
@@ -3519 +3519 @@
-his)
+his#his_adj)
@@ -3521 +3521 @@
-direction))))))))
+direction#direction_n))))))))
@@ -3529 +3529 @@
-But)
+But#but_prep)
@@ -3532 +3532 @@
-yet))
+yet#yet_adv))
@@ -3535 +3535 @@
-wee))
+wee#we_pron))
@@ -3537 +3537 @@
-must)
+must#must_v1)
@@ -3539 +3539 @@
-say)
+say#say_v1)
@@ -3542 +3542 @@
-plainely))
+plainely#plainly_adv1))
@@ -3547 +3547 @@
-That)
+That#that_conj)
@@ -3551 +3551 @@
-it))
+it#it_pron))
@@ -3553 +3553 @@
-appeareth)
+appeareth#appear_v)
@@ -3556 +3556 @@
-strange))
+strange#strange_adj))
@@ -3559 +3559 @@
-vnto)
+vnto#unto_prep)
@@ -3562 +3562 @@
-vs)))
+vs#us_pron)))
@@ -3567 +3567 @@
-when)
+when#when_adv)
@@ -3574 +3574 @@
-wee))
+wee#we_pron))
@@ -3576 +3576 @@
-were)
+were#be_v)
@@ -3579 +3579 @@
-able)
+able#able_adj)
@@ -3582 +3582 @@
-to)
+to#to_prep)
@@ -3584 +3584 @@
-shew)
+shew#show_v)
@@ -3587 +3587 @@
-it)))))))
+it#it_pron)))))))
@@ -3593 +3593 @@
-that)
+that#that_conj)
@@ -3597 +3597 @@
-since)
+since#since_adv)
@@ -3600 +3600 @@
-the)
+the#the_adj)
@@ -3602 +3602 @@
-comming)
+comming#coming_n)
@@ -3605 +3605 @@
-to)
+to#to_prep)
@@ -3608 +3608 @@
-our)
+our#our_pron)
@@ -3610 +3610 @@
-Crowne)))))
+Crowne#NA)))))
@@ -3615 +3615 @@
-it))
+it#it_pron))
@@ -3617 +3617 @@
-was)
+was#be_v)
@@ -3620 +3620 @@
-never))
+never#never_adv))
@@ -3622 +3622 @@
-denyed)
+denyed#deny_v)
@@ -3625 +3625 @@
-vs))
+vs#us_pron))
@@ -3628 +3628 @@
-by)
+by#by_prep)
@@ -3631 +3631 @@
-any)
+any#any_adj)
@@ -3634 +3634 @@
-of)
+of#of_prep)
@@ -3637 +3637 @@
-his)
+his#his_adj)
@@ -3639 +3639 @@
-predecessors)))))))
+predecessors#predecessor_n)))))))
@@ -3644 +3644 @@
-and)
+and#and_conj1)
@@ -3647 +3647 @@
-that)
+that#that_conj)
@@ -3651 +3651 @@
-in)
+in#in_prep)
@@ -3654 +3654 @@
-tyme)
+tyme#time_n)
@@ -3657 +3657 @@
-of)
+of#of_prep)
@@ -3660 +3660 @@
-Charles)
+Charles#NA)
@@ -3663 +3663 @@
-the)
+the#the_adj)
@@ -3665 +3665 @@
-ix=th=))))))
+ix=th=#NA))))))
@@ -3668 +3668 @@
-a)
+a#a_adj2)
@@ -3670 +3670 @@
-direct)
+direct#direct_adj)
@@ -3672 +3672 @@
-Treaty))
+Treaty#treaty_n))
@@ -3674 +3674 @@
-was)
+was#be_v)
@@ -3676 +3676 @@
-concluded)
+concluded#conclude_v)
@@ -3679 +3679 @@
-for)
+for#for_prep)
@@ -3682 +3682 @@
-it)))
+it#it_pron)))
@@ -3685 +3685 @@
-in)
+in#in_prep)
@@ -3688 +3688 @@
-1572)))))))))
+1572#NA)))))))))
@@ -3697 +3697 @@
-his)
+his#his_adj)
@@ -3699 +3699 @@
-Embassad=r=))
+Embassad=r=#embassador_n))
@@ -3701 +3701 @@
-should)
+should#shall_v)
@@ -3703 +3703 @@
-make)
+make#make_v1)
@@ -3706 +3706 @@
-this)
+this#this_pron)
@@ -3708 +3708 @@
-replication)
+replication#replication_n)
@@ -3714 +3714 @@
-That)
+That#that_conj)
@@ -3718 +3718 @@
-in)
+in#in_prep)
@@ -3722 +3722 @@
-the)
+the#the_adj)
@@ -3724 +3724 @@
-former)
+former#former_adj)
@@ -3726 +3726 @@
-kings))
+kings#king_n))
@@ -3728 +3728 @@
-tymes)))
+tymes#time_n)))
@@ -3733 +3733 @@
-it))
+it#it_pron))
@@ -3735 +3735 @@
-was)
+was#be_v)
@@ -3738 +3738 @@
-but)
+but#but_prep)
@@ -3740 +3740 @@
-a)
+a#a_adj2)
@@ -3742 +3742 @@
-Tolleration))))
+Tolleration#toleration_n))))
@@ -3747 +3747 @@
-and)
+and#and_conj1)
@@ -3750 +3750 @@
-that)
+that#that_conj)
@@ -3754 +3754 @@
-when)
+when#when_adv)
@@ -3761 +3761 @@
-that)
+that#that_pron)
@@ -3763 +3763 @@
-Treaty))
+Treaty#treaty_n))
@@ -3765 +3765 @@
-was)
+was#be_v)
@@ -3767 +3767 @@
-made))))
+made#make_v1))))
@@ -3772 +3772 @@
-it))
+it#it_pron))
@@ -3774 +3774 @@
-was)
+was#be_v)
@@ -3777 +3777 @@
-never))
+never#never_adv))
@@ -3780 +3780 @@
-fully))
+fully#fully_adv))
@@ -3782 +3782 @@
-executed))))))
+executed#execute_v))))))
@@ -3791 +3791 @@
-This))
+This#this_pron))
@@ -3793 +3793 @@
-seemeth)
+seemeth#seem_v2)
@@ -3796 +3796 @@
-to)
+to#to_prep)
@@ -3799 +3799 @@
-vs)))
+vs#us_pron)))
@@ -3802 +3802 @@
-surely))
+surely#surely_adv))
@@ -3805 +3805 @@
-a)
+a#a_adj2)
@@ -3808 +3808 @@
-very)
+very#very_adj)
@@ -3810 +3810 @@
-hard))
+hard#hard_adj))
@@ -3812 +3812 @@
-proceeding))
+proceeding#proceeding_n))
@@ -3817 +3817 @@
-when)
+when#when_adv)
@@ -3825 +3825 @@
-if)
+if#if_conj)
@@ -3832 +3832 @@
-other)
+other#other_adj)
@@ -3834 +3834 @@
-kings)
+kings#king_n)
@@ -3838 +3838 @@
-had)
+had#have_v)
@@ -3840 +3840 @@
-suffred)
+suffred#suffer_v)
@@ -3843 +3843 @@
-it))
+it#it_pron))
@@ -3849 +3849 @@
-betweene)
+betweene#between_prep)
@@ -3852 +3852 @@
-whome)))
+whome#whom_pron)))
@@ -3860 +3860 @@
-there))
+there#there_adv))
@@ -3862 +3862 @@
-had)
+had#have_v)
@@ -3864 +3864 @@
-passed)
+passed#pass_v)
@@ -3867 +3867 @@
-often))
+often#often_adv))
@@ -3870 +3870 @@
-breaches)))))))
+breaches#breach_n)))))))
@@ -3875 +3875 @@
-wee))
+wee#we_pron))
@@ -3877 +3877 @@
-might)
+might#may_v1)
@@ -3879 +3879 @@
-not)
+not#not_adv)
@@ -3881 +3881 @@
-haue)
+haue#have_v)
@@ -3883 +3883 @@
-ben)
+ben#be_v)
@@ -3886 +3886 @@
-secure)
+secure#secure_adj)
@@ -3889 +3889 @@
-of)
+of#of_prep)
@@ -3892 +3892 @@
-him)
+him#him_pron)
@@ -3898 +3898 @@
-betweene)
+betweene#between_prep)
@@ -3901 +3901 @@
-whome)
+whome#whom_pron)
@@ -3904 +3904 @@
-and)
+and#and_conj1)
@@ -3906 +3906 @@
-vs))))
+vs#us_pron))))
@@ -3917 +3917 @@
-had)
+had#have_v)
@@ -3919 +3919 @@
-passed)
+passed#pass_v)
@@ -3923 +3923 @@
-so)
+so#so_adv)
@@ -3925 +3925 @@
-straight))
+straight#strait_adj))
@@ -3927 +3927 @@
-obligations)))))))))
+obligations#obligation_n)))))))))
@@ -3932 +3932 @@
-or)
+or#or_conj2)
@@ -3936 +3936 @@
-now)
+now#now_adv)
@@ -3941 +3941 @@
-that)
+that#that_conj)
@@ -3947 +3947 @@
-it))
+it#it_pron))
@@ -3949 +3949 @@
-is)
+is#be_v)
@@ -3951 +3951 @@
-knowen)
+knowen#know_v)
@@ -3956 +3956 @@
-that)
+that#that_conj)
@@ -3960 +3960 @@
-by)
+by#by_prep)
@@ -3963 +3963 @@
-a)
+a#a_adj2)
@@ -3965 +3965 @@
-Treaty)))
+Treaty#treaty_n)))
@@ -3970 +3970 @@
-it))
+it#it_pron))
@@ -3972 +3972 @@
-was)
+was#be_v)
@@ -3974 +3974 @@
-accorded))))))
+accorded#accord_v))))))
@@ -3978 +3978 @@
-that)
+that#that_conj)
@@ -3984 +3984 @@
-it))
+it#it_pron))
@@ -3986 +3986 @@
-should)
+should#shall_v)
@@ -3988 +3988 @@
-be)
+be#be_v)
@@ -3990 +3990 @@
-sought)
+sought#seek_v)
@@ -3993 +3993 @@
-to)
+to#to_prep)
@@ -3995 +3995 @@
-be)
+be#be_v)
@@ -3997 +3997 @@
-altered))
+altered#alter_v))
@@ -4002 +4002 @@
-because)
+because#because_adv)
@@ -4009 +4009 @@
-the)
+the#the_adj)
@@ -4011 +4011 @@
-execution))
+execution#execution_n))
@@ -4013 +4013 @@
-was)
+was#be_v)
@@ -4015 +4015 @@
-interrupted))))
+interrupted#interrupt_v))))
@@ -4018 +4018 @@
-when)
+when#when_adv)
@@ -4025 +4025 @@
-it))
+it#it_pron))
@@ -4027 +4027 @@
-is)
+is#be_v)
@@ -4030 +4030 @@
-well))
+well#well_adv))
@@ -4032 +4032 @@
-knowen)
+knowen#know_v)
@@ -4037 +4037 @@
-that)
+that#that_conj)
@@ -4043 +4043 @@
-the)
+the#the_adj)
@@ -4045 +4045 @@
-Massacre)
+Massacre#massacre_n)
@@ -4048 +4048 @@
-of)
+of#of_prep)
@@ -4051 +4051 @@
-France))))
+France#NA))))
@@ -4053 +4053 @@
-following))
+following#follow_v))
@@ -4058 +4058 @@
-and)
+and#and_conj1)
@@ -4062 +4062 @@
-so))
+so#so_adv))
@@ -4065 +4065 @@
-dayely)
+dayely#daily_adj)
@@ -4067 +4067 @@
-troubles))
+troubles#trouble_n))
@@ -4069 +4069 @@
-arrising))))
+arrising#arise_v))))
@@ -4074 +4074 @@
-it))
+it#it_pron))
@@ -4076 +4076 @@
-was)
+was#be_v)
@@ -4078 +4078 @@
-not)
+not#not_adv)
@@ -4081 +4081 @@
-our)
+our#our_pron)
@@ -4083 +4083 @@
-default))
+default#default_n))
@@ -4088 +4088 @@
-that)
+that#that_conj)
@@ -4092 +4092 @@
-it))
+it#it_pron))
@@ -4094 +4094 @@
-was)
+was#be_v)
@@ -4096 +4096 @@
-not)
+not#not_adv)
@@ -4099 +4099 @@
-in)
+in#in_prep)
@@ -4102 +4102 @@
-all)
+all#all_adj)
@@ -4104 +4104 @@
-circumstances)))
+circumstances#circumstance_n)))
@@ -4106 +4106 @@
-executed)))))))))))))
+executed#execute_v)))))))))))))
@@ -4115 +4115 @@
-This)
+This#this_pron)
@@ -4117 +4117 @@
-injurious)
+injurious#injurious_adj)
@@ -4119 +4119 @@
-Cours))
+Cours#course_n1))
@@ -4124 +4124 @@
-we))
+we#we_pron))
@@ -4126 +4126 @@
-can)
+can#can_v1)
@@ -4129 +4129 @@
-never))
+never#never_adv))
@@ -4131 +4131 @@
-imagine)
+imagine#imagine_v)
@@ -4136 +4136 @@
-to)
+to#to_prep)
@@ -4138 +4138 @@
-proceede)
+proceede#proceed_v)
@@ -4142 +4142 @@
-from)
+from#from_prep)
@@ -4145 +4145 @@
-y=e=)
+y=e=#the_adj)
@@ -4147 +4147 @@
-king)
+king#king_n)
@@ -4150 +4150 @@
-our)
+our#our_pron)
@@ -4152 +4152 @@
-brother))))
+brother#brother_n))))
@@ -4157 +4157 @@
-but)
+but#but_prep)
@@ -4160 +4160 @@
-from)
+from#from_prep)
@@ -4163 +4163 @@
-some)
+some#some_pron)
@@ -4165 +4165 @@
-other)
+other#other_adj)
@@ -4167 +4167 @@
-omissions)
+omissions#omission_n)
@@ -4170 +4170 @@
-of)
+of#of_prep)
@@ -4173 +4173 @@
-his)
+his#his_adj)
@@ -4175 +4175 @@
-Ministers))))))))
+Ministers#minister_n))))))))
@@ -4183 +4183 @@
-for)
+for#for_prep)
@@ -4186 +4186 @@
-what))
+what#what_pron))
@@ -4191 +4191 @@
-could)
+could#can_v1)
@@ -4194 +4194 @@
-more)
+more#more_adj)
@@ -4198 +4198 @@
-haue)
+haue#have_v)
@@ -4200 +4200 @@
-foretould)
+foretould#foretell_v)
@@ -4205 +4205 @@
-the)
+the#the_adj)
@@ -4207 +4207 @@
-world))
+world#world_n))
@@ -4210 +4210 @@
-of)
+of#of_prep)
@@ -4213 +4213 @@
-alteration)
+alteration#alteration_n)
@@ -4216 +4216 @@
-of)
+of#of_prep)
@@ -4219 +4219 @@
-kindnes)))))
+kindnes#kindness_n)))))
@@ -4224 +4224 @@
-if)
+if#if_conj)
@@ -4232 +4232 @@
-you))
+you#you_pron))
@@ -4234 +4234 @@
-had)
+had#have_v)
@@ -4236 +4236 @@
-not)
+not#not_adv)
@@ -4238 +4238 @@
-remembered)
+remembered#remember_v)
@@ -4241 +4241 @@
-it)))
+it#it_pron)))
@@ -4246 +4246 @@
-and)
+and#and_conj1)
@@ -4250 +4250 @@
-the)
+the#the_adj)
@@ -4252 +4252 @@
-king))
+king#king_n))
@@ -4254 +4254 @@
-suspended)
+suspended#suspend_v)
@@ -4257 +4257 @@
-it)))))))
+it#it_pron)))))))
@@ -4262 +4262 @@
-then)
+then#than_conj)
@@ -4266 +4266 @@
-so)
+so#so_adv)
@@ -4268 +4268 @@
-apert))
+apert#apert_adj))
@@ -4270 +4270 @@
-a)
+a#a_adj2)
@@ -4272 +4272 @@
-declaration)
+declaration#declaration_n)
@@ -4275 +4275 @@
-of)
+of#of_prep)
@@ -4278 +4278 @@
-hard)
+hard#hard_adj)
@@ -4280 +4280 @@
-measure)))
+measure#measure_n)))
@@ -4283 +4283 @@
-to)
+to#to_prep)
@@ -4286 +4286 @@
-our)
+our#our_pron)
@@ -4288 +4288 @@
-people))))))
+people#people_n))))))
@@ -4298 +4298 @@
-Thus)
+Thus#thus_adv)
@@ -4300 +4300 @@
-farr))
+farr#far_adj))
@@ -4303 +4303 @@
-and)
+and#and_conj1)
@@ -4306 +4306 @@
-w=th=)
+w=th=#with_prep)
@@ -4309 +4309 @@
-this)
+this#this_pron)
@@ -4311 +4311 @@
-rondeur)))))
+rondeur#rondure_n)))))
@@ -4313 +4313 @@
-haue)
+haue#have_v)
@@ -4316 +4316 @@
-wee))
+wee#we_pron))
@@ -4318 +4318 @@
-proceeded)
+proceeded#proceed_v)
@@ -4321 +4321 @@
-w=th=)
+w=th=#with_prep)
@@ -4324 +4324 @@
-a)
+a#a_adj2)
@@ -4326 +4326 @@
-desire)
+desire#desire_n)
@@ -4329 +4329 @@
-to)
+to#to_prep)
@@ -4331 +4331 @@
-cleare)
+cleare#clear_v)
@@ -4334 +4334 @@
-all)
+all#all_adj)
@@ -4336 +4336 @@
-manner)
+manner#manner_n)
@@ -4339 +4339 @@
-of)
+of#of_prep)
@@ -4342 +4342 @@
-scruples)
+scruples#scruple_n2)
@@ -4348 +4348 @@
-w=ch=))
+w=ch=#which_adj))
@@ -4355 +4355 @@
-might)
+might#may_v1)
@@ -4357 +4357 @@
-weacken)
+weacken#weaken_v)
@@ -4360 +4360 @@
-ether)
+ether#either_pron)
@@ -4362 +4362 @@
-the)
+the#the_adj)
@@ -4364 +4364 @@
-reputation)
+reputation#reputation_n)
@@ -4369 +4369 @@
-or)
+or#or_conj2)
@@ -4372 +4372 @@
-essence)))
+essence#essence_n)))
@@ -4375 +4375 @@
-of)
+of#of_prep)
@@ -4378,3 +4378,3 @@
-our)
-(ADJ
-perfect)
+our#our_pron)
+(ADJ
+perfect#perfect_adj)
@@ -4382 +4382 @@
-frendship)
+frendship#friendship_n)
@@ -4388 +4388 @@
-w=ch=))
+w=ch=#which_adj))
@@ -4394 +4394 @@
-wee))
+wee#we_pron))
@@ -4396 +4396 @@
-were)
+were#be_v)
@@ -4399 +4399 @@
-desirous)
+desirous#desirous_adj)
@@ -4404 +4404 @@
-to)
+to#to_prep)
@@ -4406 +4406 @@
-intimate)
+intimate#intimate_v)
@@ -4409 +4409 @@
-to)
+to#to_prep)
@@ -4412 +4412 @@
-himself)
+himself#himself_pron)
@@ -4418 +4418 @@
-from)
+from#from_prep)
@@ -4421 +4421 @@
-whome)))
+whome#whom_pron)))
@@ -4429 +4429 @@
-wee))
+wee#we_pron))
@@ -4431 +4431 @@
-promise)
+promise#promise_v)
@@ -4434 +4434 @@
-our)
+our#our_pron)
@@ -4436 +4436 @@
-self))
+self#self_pron))
@@ -4441 +4441 @@
-all)
+all#all_adj)
@@ -4443 +4443 @@
-such)
+such#such_adj)
@@ -4445 +4445 @@
-effects)
+effects#effect_n)
@@ -4450 +4450 @@
-as)
+as#as_adv)
@@ -4460 +4460 @@
-are)
+are#be_v)
@@ -4463 +4463 @@
-agreeable)
+agreeable#agreeable_adj)
@@ -4466 +4466 @@
-to)
+to#to_prep)
@@ -4469 +4469 @@
-Princely)
+Princely#princely_adj)
@@ -4471 +4471 @@
-Judgement)
+Judgement#judgement_n)
@@ -4474 +4474 @@
-and)
+and#and_conj1)
@@ -4477 +4477 @@
-Moderation))))))
+Moderation#moderation_n))))))
@@ -4483 +4483 @@
-both))
+both#both_pron))
@@ -4485 +4485 @@
-being)
+being#be_v)
@@ -4488 +4488 @@
-vertues)
+vertues#virtue_n)
@@ -4492 +4492 @@
-properly))
+properly#properly_adv))
@@ -4494 +4494 @@
-annexed)
+annexed#annex_v)
@@ -4497 +4497 @@
-to)
+to#to_prep)
@@ -4500 +4500 @@
-the)
+the#the_adj)
@@ -4502 +4502 @@
-ranks)
+ranks#rank_n1)
@@ -4508 +4508 @@
-w=ch=))
+w=ch=#which_adj))
@@ -4516 +4516 @@
-Princes)
+Princes#prince_n)
@@ -4520 +4520 @@
-hold)
+hold#hold_v)
@@ -4523 +4523 @@
-vnder)
+vnder#under_prep)
@@ -4526 +4526 @@
-God)))
+God#NA)))
@@ -4532 +4532 @@
-whome))
+whome#whom_pron))
@@ -4540 +4540 @@
-he))
+he#he_pron))
@@ -4542 +4542 @@
-hath)
+hath#have_v)
@@ -4544 +4544 @@
-chosen)
+chosen#choose_v)
@@ -4549 +4549 @@
-to)
+to#to_prep)
@@ -4551 +4551 @@
-rectify)
+rectify#rectify_v)
@@ -4554 +4554 @@
-the)
+the#the_adj)
@@ -4556 +4556 @@
-errors)
+errors#error_n)
@@ -4559 +4559 @@
-and)
+and#and_conj1)
@@ -4562 +4562 @@
-partialities)))
+partialities#partiality_n)))
@@ -4565 +4565 @@
-of)
+of#of_prep)
@@ -4568 +4568 @@
-Inferiors)))))
+Inferiors#inferior_adj)))))
@@ -4575 +4575 @@
-howsoeuer))
+howsoeuer#howsoever_adv))
@@ -4583 +4583 @@
-wee))
+wee#we_pron))
@@ -4585 +4585 @@
-may)
+may#may_v1)
@@ -4587 +4587 @@
-be)
+be#be_v)
@@ -4589 +4589 @@
-pressed)
+pressed#press_v1)
@@ -4592 +4592 @@
-dayly))
+dayly#daily_adv))
@@ -4595 +4595 @@
-w=th=)
+w=th=#with_prep)
@@ -4598 +4598 @@
-Importunities)
+Importunities#importunity_n)
@@ -4604 +4604 @@
-to)
+to#to_prep)
@@ -4607 +4607 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -4613 +4613 @@
-vulgar)
+vulgar#vulgar_adj)
@@ -4615 +4615 @@
-spiritts))
+spiritts#spirit_n))
@@ -4617 +4617 @@
-are)
+are#be_v)
@@ -4620 +4620 @@
-subiect)
+subiect#subject_adj)
@@ -4631 +4631 @@
-You))
+You#you_pron))
@@ -4633 +4633 @@
-shall)
+shall#shall_v)
@@ -4636 +4636 @@
-in)
+in#in_prep)
@@ -4639 +4639 @@
-the)
+the#the_adj)
@@ -4641 +4641 @@
-end)))
+end#end_n)))
@@ -4643 +4643 @@
-assure)
+assure#assure_v)
@@ -4646 +4646 @@
-him))
+him#him_pron))
@@ -4651 +4651 @@
-that)
+that#that_conj)
@@ -4655 +4655 @@
-as)
+as#as_adv)
@@ -4666 +4666 @@
-wee))
+wee#we_pron))
@@ -4668 +4668 @@
-may)
+may#may_v1)
@@ -4670 +4670 @@
-not)
+not#not_adv)
@@ -4672 +4672 @@
-condemne)
+condemne#condemn_v)
@@ -4675 +4675 @@
-our)
+our#our_pron)
@@ -4677 +4677 @@
-Subiects))
+Subiects#subject_n))
@@ -4682 +4682 @@
-onely)
+onely#only_adj)
@@ -4684 +4684 @@
-because)
+because#because_adv)
@@ -4691 +4691 @@
-they))
+they#they_pron))
@@ -4693 +4693 @@
-are)
+are#be_v)
@@ -4695 +4695 @@
-accused)))))))
+accused#accuse_v)))))))
@@ -4700 +4700 @@
-so))
+so#so_adv))
@@ -4705 +4705 @@
-whosoeuer))
+whosoeuer#whosoever_pron))
@@ -4712 +4712 @@
-can)
+can#can_v1)
@@ -4714 +4714 @@
-be)
+be#be_v)
@@ -4716 +4716 @@
-convicted))))
+convicted#convict_v))))
@@ -4720 +4720 @@
-shall)
+shall#shall_v)
@@ -4723 +4723 @@
-duely))
+duely#duly_adv))
@@ -4725 +4725 @@
-paye)
+paye#pay_v1)
@@ -4728 +4728 @@
-the)
+the#the_adj)
@@ -4730 +4730 @@
-price)
+price#price_n)
@@ -4733 +4733 @@
-of)
+of#of_prep)
@@ -4736 +4736 @@
-his)
+his#his_adj)
@@ -4738 +4738 @@
-cryme))))))
+cryme#crime_n))))))
@@ -4747 +4747 @@
-Wee))
+Wee#we_pron))
@@ -4749 +4749 @@
-haue)
+haue#have_v)
@@ -4751 +4751 @@
-commanded)
+commanded#command_v)
@@ -4754 +4754 @@
-our)
+our#our_pron)
@@ -4756 +4756 @@
-secretary))
+secretary#secretary_n1))
@@ -4761 +4761 @@
-to)
+to#to_prep)
@@ -4763 +4763 @@
-acquaint)
+acquaint#acquaint_v)
@@ -4766 +4766 @@
-you))
+you#you_pron))
@@ -4772 +4772 @@
-what))
+what#what_pron))
@@ -4779 +4779 @@
-hath)
+hath#have_v)
@@ -4781 +4781 @@
-passed)
+passed#pass_v)
@@ -4784 +4784 @@
-between)
+between#between_prep)
@@ -4788 +4788 @@
-the)
+the#the_adj)
@@ -4790 +4790 @@
-french)
+french#french_adj)
@@ -4792 +4792 @@
-Ambassad=r=)
+Ambassad=r=#ambassador_n)
@@ -4797 +4797 @@
-and)
+and#and_conj1)
@@ -4800 +4800 @@
-him)))
+him#him_pron)))
@@ -4806 +4806 @@
-in)
+in#in_prep)
@@ -4809 +4809 @@
-whome)))
+whome#whom_pron)))
@@ -4817 +4817 @@
-wee))
+wee#we_pron))
@@ -4819 +4819 @@
-find)
+find#find_v)
@@ -4822 +4822 @@
-more)
+more#more_adj)
@@ -4824 +4824 @@
-acrimony)
+acrimony#acrimony_n)
@@ -4829 +4829 @@
-then)
+then#than_conj)
@@ -4840 +4840 @@
-wee))
+wee#we_pron))
@@ -4842 +4842 @@
-expected))))
+expected#expect_v))))
@@ -4847 +4847 @@
-especially)
+especially#especially_adv)
@@ -4849 +4849 @@
-in)
+in#in_prep)
@@ -4852 +4852 @@
-this)
+this#this_pron)
@@ -4858 +4858 @@
-wherein))
+wherein#wherein_adv))
@@ -4866 +4866 @@
-he))
+he#he_pron))
@@ -4868 +4868 @@
-sheweth)
+sheweth#show_v)
@@ -4871 +4871 @@
-merely))
+merely#merely_adv2))
@@ -4874 +4874 @@
-a)
+a#a_adj2)
@@ -4876 +4876 @@
-spiritt)
+spiritt#spirit_n)
@@ -4879 +4879 @@
-of)
+of#of_prep)
@@ -4882 +4882 @@
-Choller)
+Choller#choler_n)
@@ -4884 +4884 @@
-and)
+and#and_conj1)
@@ -4886 +4886 @@
-pride))))
+pride#pride_n1))))
@@ -4891 +4891 @@
-when)
+when#when_adv)
@@ -4898 +4898 @@
-he)
+he#he_pron)
@@ -4902 +4902 @@
-gaue)
+gaue#give_v)
@@ -4905 +4905 @@
-the)
+the#the_adj)
@@ -4907 +4907 @@
-first)
+first#first_adj)
@@ -4909 +4909 @@
-cause))
+cause#cause_n))
@@ -4912 +4912 @@
-himself)))))))))))))))))
+himself#himself_pron)))))))))))))))))
@@ -4920 +4920 @@
-obserue)
+obserue#observe_v)
@@ -4924 +4924 @@
-what)
+what#what_pron)
@@ -4926 +4926 @@
-impressions))
+impressions#impression_n))
@@ -4932 +4932 @@
-he))
+he#he_pron))
@@ -4934 +4934 @@
-seeketh)
+seeketh#seek_v)
@@ -4939 +4939 @@
-to)
+to#to_prep)
@@ -4941 +4941 @@
-worke)
+worke#work_v)
@@ -4944 +4944 @@
-there)))))
+there#there_adv)))))
@@ -4952 +4952 @@
-and)
+and#and_conj1)
@@ -4954 +4954 @@
-obviate)
+obviate#obviate_v)
@@ -4957 +4957 @@
-them))
+them#them_pron))
@@ -4962 +4962 @@
-according)
+according#accord_v)
@@ -4965 +4965 @@
-to)
+to#to_prep)
@@ -4968 +4968 @@
-y=r=)
+y=r=#your_pron)
@@ -4970 +4970 @@
-discretion))))
+discretion#discretion_n))))
@@ -4978 +4978 @@
-and)
+and#and_conj1)
@@ -4980 +4980 @@
-acquaint)
+acquaint#acquaint_v)
@@ -4983 +4983 @@
-vs))
+vs#us_pron))
@@ -4986 +4986 @@
-by)
+by#by_prep)
@@ -4989 +4989 @@
-y=r=)
+y=r=#your_pron)
@@ -4991 +4991 @@
-letter)
+letter#letter_n1)
@@ -4994 +4994 @@
-to)
+to#to_prep)
@@ -4997 +4997 @@
-our)
+our#our_pron)
@@ -4999 +4999 @@
-Secretary)))))
+Secretary#secretary_n1)))))
@@ -5002 +5002 @@
-of)
+of#of_prep)
@@ -5005 +5005 @@
-all)
+all#all_adj)
@@ -5007 +5007 @@
-particulars)))
+particulars#particular_adj)))
@@ -5015 +5015 @@
-and)
+and#and_conj1)
@@ -5017 +5017 @@
-take)
+take#take_v)
@@ -5020 +5020 @@
-notice))
+notice#notice_n))
@@ -5023 +5023 @@
-from)
+from#from_prep)
@@ -5026 +5026 @@
-vs)))
+vs#us_pron)))
@@ -5029 +5029 @@
-hereby))
+hereby#hereby_adv))
@@ -5034 +5034 @@
-that)
+that#that_conj)
@@ -5039 +5039 @@
-as)
+as#as_adv)
@@ -5041 +5041 @@
-well)
+well#well_adv)
@@ -5046 +5046 @@
-by)
+by#by_prep)
@@ -5049 +5049 @@
-y=r=)
+y=r=#your_pron)
@@ -5051 +5051 @@
-letters)))
+letters#letter_n1)))
@@ -5056 +5056 @@
-as)
+as#as_adv)
@@ -5059 +5059 @@
-other)
+other#other_adj)
@@ -5061 +5061 @@
-testimonies)
+testimonies#testimony_n)
@@ -5064 +5064 @@
-geuen)
+geuen#give_v)
@@ -5067 +5067 @@
-vs))
+vs#us_pron))
@@ -5070 +5070 @@
-by)
+by#by_prep)
@@ -5073 +5073 @@
-our)
+our#our_pron)
@@ -5075 +5075 @@
-Secretary)))
+Secretary#secretary_n1)))
@@ -5078 +5078 @@
-of)
+of#of_prep)
@@ -5081 +5081 @@
-y=r=)
+y=r=#your_pron)
@@ -5083 +5083 @@
-diligence)))))))
+diligence#diligence_n1)))))))
@@ -5085 +5085 @@
-that)
+that#that_conj)
@@ -5091 +5091 @@
-wee))
+wee#we_pron))
@@ -5093 +5093 @@
-do)
+do#do_v)
@@ -5096 +5096 @@
-very)
+very#very_adj)
@@ -5098 +5098 @@
-gratiously))
+gratiously#graciously_adv))
@@ -5100 +5100 @@
-accept)
+accept#accept_v)
@@ -5103 +5103 @@
-y=r=)
+y=r=#your_pron)
@@ -5106 +5106 @@
-honest)
+honest#honest_adj)
@@ -5108 +5108 @@
-and)
+and#and_conj1)
@@ -5110 +5110 @@
-faithfull))
+faithfull#faithful_adj))
@@ -5112 +5112 @@
-endevoirs)))))
+endevoirs#endeavour_n)))))
@@ -5120 +5120 @@
-Returne)
+Returne#return_v1)
@@ -5123 +5123 @@
-thanks))
+thanks#thank_n))
@@ -5126 +5126 @@
-from)
+from#from_prep)
@@ -5129 +5129 @@
-vs)))
+vs#us_pron)))
@@ -5132 +5132 @@
-to)
+to#to_prep)
@@ -5135 +5135 @@
-the)
+the#the_adj)
@@ -5137 +5137 @@
-Duke)
+Duke#duke_n)
@@ -5140 +5140 @@
-of)
+of#of_prep)
@@ -5143 +5143 @@
-Buillon)))
+Buillon#NA)))
@@ -5149 +5149 @@
-to)
+to#to_prep)
@@ -5152 +5152 @@
-whome)))
+whome#whom_pron)))
@@ -5160 +5160 @@
-wee))
+wee#we_pron))
@@ -5162 +5162 @@
-would)
+would#will_v1)
@@ -5164 +5164 @@
-haue)
+haue#have_v)
@@ -5166 +5166 @@
-written)
+written#write_v)
@@ -5169 +5169 @@
-thanks)
+thanks#thank_n)
@@ -5172 +5172 @@
-for)
+for#for_prep)
@@ -5175 +5175 @@
-his)
+his#his_adj)
@@ -5177 +5177 @@
-kind)
+kind#kind_adj)
@@ -5179 +5179 @@
-Letter)))
+Letter#letter_n1)))
@@ -5186 +5186 @@
-but)
+but#but_prep)
@@ -5189 +5189 @@
-that)
+that#that_conj)
@@ -5193 +5193 @@
-a)
+a#a_adj2)
@@ -5195 +5195 @@
-Flux)
+Flux#flux_n)
@@ -5198 +5198 @@
-in)
+in#in_prep)
@@ -5201 +5201 @@
-our)
+our#our_pron)
@@ -5203 +5203 @@
-Eye))))
+Eye#eye_n1))))
@@ -5205 +5205 @@
-hindered)
+hindered#hinder_v)
@@ -5208 +5208 @@
-vs)))))
+vs#us_pron)))))
@@ -5213 +5213 @@
-from)
+from#from_prep)
@@ -5218 +5218 @@
-writing)
+writing#write_v)
@@ -5221 +5221 @@
-to)
+to#to_prep)
@@ -5224 +5224 @@
-the)
+the#the_adj)
@@ -5226 +5226 @@
-french)
+french#french_adj)
@@ -5228 +5228 @@
-king)))
+king#king_n)))
@@ -5233 +5233 @@
-to)
+to#to_prep)
@@ -5235 +5235 @@
-haue)
+haue#have_v)
@@ -5237 +5237 @@
-prepared)
+prepared#prepare_v)
@@ -5240 +5240 @@
-his)
+his#his_adj)
@@ -5242 +5242 @@
-minde))
+minde#mind_n1))
@@ -5245 +5245 @@
-not)
+not#not_adv)
@@ -5247 +5247 @@
-to)
+to#to_prep)
@@ -5249 +5249 @@
-mistake)
+mistake#mistake_v)
@@ -5254 +5254 @@
-vpon)
+vpon#upon_prep)
@@ -5257 +5257 @@
-an)
+an#a_adj2)
@@ -5259 +5259 @@
-vntrue)
+vntrue#untrue_adj)
@@ -5261 +5261 @@
-reporte)))
+reporte#report_n)))
@@ -5266 +5266 @@
-the)
+the#the_adj)
@@ -5268 +5268 @@
-good)
+good#good_adj)
@@ -5270 +5270 @@
-will)
+will#will_n1)
@@ -5273 +5273 @@
-of)
+of#of_prep)
@@ -5277 +5277 @@
-our)
+our#our_pron)
@@ -5281 +5281 @@
-Tablettes)
+Tablettes#tablet_n)
@@ -5287 +5287 @@
-who))
+who#who_pron))
@@ -5294 +5294 @@
-hath)
+hath#have_v)
@@ -5297 +5297 @@
-euer))
+euer#ever_adv))
@@ -5299 +5299 @@
-ben)
+ben#be_v)
@@ -5302 +5302 @@
-ready)
+ready#ready_adj)
@@ -5305 +5305 @@
-to)
+to#to_prep)
@@ -5307 +5307 @@
-do)
+do#do_v)
@@ -5310 +5310 @@
-him))
+him#him_pron))
@@ -5313 +5313 @@
-good)
+good#good_adj)
@@ -5315 +5315 @@
-offices)))))))))))))))))
+offices#office_n)))))))))))))))))
@@ -5323 +5323 @@
-At)
+At#at_prep)
@@ -5326 +5326 @@
-our)
+our#our_pron)
@@ -5328 +5328 @@
-Palace)
+Palace#palace_n1)
@@ -5331 +5331 @@
-of)
+of#of_prep)
@@ -5334 +5334 @@
-Westminster))))
+Westminster#NA))))
@@ -5342 +5342 @@
-this)
+this#this_pron)
@@ -5344 +5344 @@
-20=th=)
+20=th=#NA)
@@ -5346 +5346 @@
-day)
+day#day_n)
@@ -5349 +5349 @@
-of)
+of#of_prep)
@@ -5352 +5352 @@
-Januar.)))
+Januar.#NA)))
@@ -5354 +5354 @@
-1598)
+1598#NA)
@@ -5358 +5358 @@
-of)
+of#of_prep)
@@ -5361 +5361 @@
-our)
+our#our_pron)
@@ -5363 +5363 @@
-Raigne)))
+Raigne#reign_n)))
@@ -5365 +5365 @@
-the)
+the#the_adj)
@@ -5367 +5367 @@
-41=th=)))
+41=th=#NA)))
