--- fhatton-1677-e3-h.psd.vert	2025-09-10 13:50:35
+++ fhatton-1677-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-Kirby)
+Kirby#NA)
@@ -21 +21 @@
-March)
+March#NA)
@@ -23 +23 @@
-the)
+the#the_adj)
@@ -25 +25 @@
-31=th=)
+31=th=#NA)
@@ -29 +29 @@
-1677)
+1677#NA)
@@ -40 +40 @@
-My)
+My#my_adj)
@@ -42 +42 @@
-deare)
+deare#dear_adj1)
@@ -44 +44 @@
-lord))
+lord#lord_n))
@@ -49 +49 @@
-I))
+I#i_pron))
@@ -51 +51 @@
-hope)
+hope#hope_v)
@@ -58 +58 @@
-you))
+you#you_pron))
@@ -60 +60 @@
-have)
+have#have_v)
@@ -62 +62 @@
-received)
+received#receive_v)
@@ -65 +65 @@
-all)
+all#all_adj)
@@ -67 +67 @@
-my)
+my#my_adj)
@@ -69 +69 @@
-letters))
+letters#letter_n1))
@@ -72 +72 @@
-this)
+this#this_pron)
@@ -74 +74 @@
-week))))
+week#week_n))))
@@ -82 +82 @@
-for)
+for#for_prep)
@@ -85 +85 @@
-I))
+I#i_pron))
@@ -88 +88 @@
-never))
+never#never_adv))
@@ -90 +90 @@
-missed)
+missed#miss_v1)
@@ -93 +93 @@
-any)
+any#any_adj)
@@ -95 +95 @@
-opportunity))
+opportunity#opportunity_n))
@@ -100 +100 @@
-only)
+only#only_adj)
@@ -102 +102 @@
-by)
+by#by_prep)
@@ -105 +105 @@
-the)
+the#the_adj)
@@ -107 +107 @@
-caryer)))
+caryer#carrier_n)))
@@ -116 +116 @@
-I))
+I#i_pron))
@@ -118 +118 @@
-am)
+am#be_v)
@@ -121 +121 @@
-glad)
+glad#glad_adj)
@@ -124 +124 @@
-that)
+that#that_conj)
@@ -128 +128 @@
-you))
+you#you_pron))
@@ -130 +130 @@
-design)
+design#design_v)
@@ -133 +133 @@
-to)
+to#to_prep)
@@ -135 +135 @@
-com)))))
+com#come_v)))))
@@ -144 +144 @@
-Your)
+Your#your_pron)
@@ -146 +146 @@
-coach))
+coach#coach_n))
@@ -148 +148 @@
-shall)
+shall#shall_v)
@@ -150 +150 @@
-be)
+be#be_v)
@@ -153 +153 @@
-sure)
+sure#sure_adj)
@@ -156 +156 @@
-to)
+to#to_prep)
@@ -158 +158 @@
-meet)
+meet#meet_v)
@@ -161 +161 @@
-you))
+you#you_pron))
@@ -164 +164 @@
-at)
+at#at_prep)
@@ -167 +167 @@
-Baldock)))))
+Baldock#NA)))))
@@ -175 +175 @@
-but)
+but#but_prep)
@@ -178 +178 @@
-you))
+you#you_pron))
@@ -180 +180 @@
-must)
+must#must_v1)
@@ -182 +182 @@
-be)
+be#be_v)
@@ -185 +185 @@
-sure)
+sure#sure_adj)
@@ -188 +188 @@
-to)
+to#to_prep)
@@ -190 +190 @@
-send)
+send#send_v1)
@@ -193 +193 @@
-me))
+me#me_pron1))
@@ -196 +196 @@
-word))
+word#word_n))
@@ -201 +201 @@
-what)
+what#what_pron)
@@ -203 +203 @@
-horsemen))
+horsemen#horseman_n))
@@ -211 +211 @@
-you))
+you#you_pron))
@@ -213 +213 @@
-would)
+would#will_v1)
@@ -215 +215 @@
-have)))
+have#have_v)))
@@ -220 +220 @@
-and)
+and#and_conj1)
@@ -224 +224 @@
-at)
+at#at_prep)
@@ -227 +227 @@
-what)
+what#what_pron)
@@ -229 +229 @@
-time)))
+time#time_n)))
@@ -237 +237 @@
-the)
+the#the_adj)
@@ -239 +239 @@
-coach))
+coach#coach_n))
@@ -241 +241 @@
-must)
+must#must_v1)
@@ -243 +243 @@
-be)
+be#be_v)
@@ -246 +246 @@
-there))))))))
+there#there_adv))))))))
@@ -255 +255 @@
-I))
+I#i_pron))
@@ -257 +257 @@
-find)
+find#find_v)
@@ -264 +264 @@
-you))
+you#you_pron))
@@ -266 +266 @@
-intend)
+intend#intend_v)
@@ -269 +269 @@
-to)
+to#to_prep)
@@ -271 +271 @@
-be)
+be#be_v)
@@ -274 +274 @@
-at)
+at#at_prep)
@@ -277 +277 @@
-Kimbolton)))
+Kimbolton#NA)))
@@ -280 +280 @@
-that)
+that#that_pron)
@@ -282 +282 @@
-night)))))
+night#night_n)))))
@@ -293 +293 @@
-Pray))
+Pray#pray_v))
@@ -295 +295 @@
-be)
+be#be_v)
@@ -298 +298 @@
-sure)
+sure#sure_adj)
@@ -301 +301 @@
-to)
+to#to_prep)
@@ -303 +303 @@
-send)
+send#send_v1)
@@ -306 +306 @@
-me))
+me#me_pron1))
@@ -309 +309 @@
-word))
+word#word_n))
@@ -312 +312 @@
-wheathere)
+wheathere#whether_pron)
@@ -319 +319 @@
-you))
+you#you_pron))
@@ -321 +321 @@
-will)
+will#will_v1)
@@ -323 +323 @@
-be)
+be#be_v)
@@ -326 +326 @@
-here))
+here#here_adv))
@@ -329 +329 @@
-by)
+by#by_prep)
@@ -332 +332 @@
-dinner))))
+dinner#dinner_n))))
@@ -335 +335 @@
-or)
+or#or_conj2)
@@ -338 +338 @@
-not)))))))
+not#not_adv)))))))
@@ -347 +347 @@
-I))
+I#i_pron))
@@ -349 +349 @@
-believe)
+believe#believe_v)
@@ -356 +356 @@
-you))
+you#you_pron))
@@ -358 +358 @@
-can@)
+can@#can_v1)
@@ -360 +360 @@
-@not)
+@not#not_adv)
@@ -370 +370 @@
-but)
+but#but_prep)
@@ -372 +372 @@
-be)
+be#be_v)
@@ -375 +375 @@
-sure)
+sure#sure_adj)
@@ -382 +382 @@
-you))
+you#you_pron))
@@ -384 +384 @@
-let)
+let#let_v1)
@@ -388 +388 @@
-me))
+me#me_pron1))
@@ -390 +390 @@
-know)))))
+know#know_v)))))
@@ -399 +399 @@
-One)
+One#one_adj)
@@ -402 +402 @@
-of)
+of#of_prep)
@@ -405 +405 @@
-your)
+your#your_pron)
@@ -407 +407 @@
-best)
+best#best_adj)
@@ -409 +409 @@
-coach)
+coach#coach_n)
@@ -411 +411 @@
-horses))))
+horses#horse_n))))
@@ -413 +413 @@
-has)
+has#have_v)
@@ -415 +415 @@
-been)
+been#be_v)
@@ -418 +418 @@
-like)
+like#like_adj)
@@ -421 +421 @@
-to)
+to#to_prep)
@@ -423 +423 @@
-dye)))
+dye#die_v1)))
@@ -431 +431 @@
-but)
+but#but_prep)
@@ -434 +434 @@
-wee))
+wee#we_pron))
@@ -436 +436 @@
-hope)
+hope#hope_v)
@@ -443 +443 @@
-the)
+the#the_adj)
@@ -445 +445 @@
-danger))
+danger#danger_n))
@@ -447 +447 @@
-is)
+is#be_v)
@@ -449 +449 @@
-past)))
+past#pass_v)))
@@ -458 +458 @@
-I))
+I#i_pron))
@@ -460 +460 @@
-had)
+had#have_v)
@@ -464 +464 @@
-Shefeld))
+Shefeld#NA))
@@ -469 +469 @@
-with)
+with#with_prep)
@@ -472 +472 @@
-him)))
+him#him_pron)))
@@ -477 +477 @@
-and)
+and#and_conj1)
@@ -480 +480 @@
-all)
+all#all_adj)
@@ -482 +482 @@
-the)
+the#the_adj)
@@ -484 +484 @@
-care)
+care#care_n1)
@@ -489 +489 @@
-that)
+that#that_conj)
@@ -494 +494 @@
-can)
+can#can_v1)
@@ -496 +496 @@
-bee)))))
+bee#be_v)))))
@@ -505 +505 @@
-I))
+I#i_pron))
@@ -507 +507 @@
-hope)
+hope#hope_v)
@@ -514 +514 @@
-he))
+he#he_pron))
@@ -516 +516 @@
-will)
+will#will_v1)
@@ -520 +520 @@
-doe)
+doe#do_v)
@@ -523 +523 @@
-well))))
+well#well_adv))))
@@ -531 +531 @@
-but)
+but#but_prep)
@@ -534 +534 @@
-they))
+they#they_pron))
@@ -536 +536 @@
-say)
+say#say_v1)
@@ -543 +543 @@
-I))
+I#i_pron))
@@ -545 +545 @@
-must)
+must#must_v1)
@@ -547 +547 @@
-not)
+not#not_adv)
@@ -549 +549 @@
-venter)
+venter#venture_v)
@@ -552 +552 @@
-him))
+him#him_pron))
@@ -555 +555 @@
-in)
+in#in_prep)
@@ -558 +558 @@
-the)
+the#the_adj)
@@ -560 +560 @@
-coach)))
+coach#coach_n)))
@@ -564 +564 @@
-so)
+so#so_adv)
@@ -566 +566 @@
-farr)))
+farr#far_adj)))
@@ -569 +569 @@
-a)
+a#a_adj2)
@@ -571 +571 @@
-great)
+great#great_adj)
@@ -573 +573 @@
-whill))))
+whill#while_n))))
@@ -581 +581 @@
-But)
+But#but_prep)
@@ -584 +584 @@
-one)
+one#one_adj)
@@ -587 +587 @@
-of)
+of#of_prep)
@@ -590 +590 @@
-the)
+the#the_adj)
@@ -592 +592 @@
-cart)
+cart#cart_n)
@@ -594 +594 @@
-horsses))))
+horsses#horse_n))))
@@ -596 +596 @@
-will)
+will#will_v1)
@@ -598 +598 @@
-doe)
+doe#do_v)
@@ -601 +601 @@
-in)
+in#in_prep)
@@ -604 +604 @@
-the)
+the#the_adj)
@@ -606 +606 @@
-coach)))
+coach#coach_n)))
@@ -609 +609 @@
-very)
+very#very_adj)
@@ -611 +611 @@
-well))
+well#well_adv))
@@ -620 +620 @@
-Poor)
+Poor#poor_adj)
@@ -622 +622 @@
-little)
+little#little_adj)
@@ -624 +624 @@
-Susana))
+Susana#NA))
@@ -626 +626 @@
-is)
+is#be_v)
@@ -629 +629 @@
-very)
+very#very_adj)
@@ -631 +631 @@
-ill)
+ill#ill_adj)
@@ -634 +634 @@
-about)
+about#about_adv)
@@ -637 +637 @@
-her)
+her#her_pron2)
@@ -639 +639 @@
-teeth))))
+teeth#tooth_n))))
@@ -648 +648 @@
-I))
+I#i_pron))
@@ -650 +650 @@
-hope)
+hope#hope_v)
@@ -653 +653 @@
-in)
+in#in_prep)
@@ -656 +656 @@
-God)))
+God#NA)))
@@ -663 +663 @@
-they))
+they#they_pron))
@@ -665 +665 @@
-will)
+will#will_v1)
@@ -667 +667 @@
-not)
+not#not_adv)
@@ -669 +669 @@
-be)
+be#be_v)
@@ -672 +672 @@
-long))
+long#long_adj1))
@@ -675 +675 @@
-before)
+before#before_adv)
@@ -682 +682 @@
-they))
+they#they_pron))
@@ -684 +684 @@
-be)
+be#be_v)
@@ -686 +686 @@
-cut))))))
+cut#cut_v))))))
@@ -695 +695 @@
-Shee))
+Shee#she_pron))
@@ -697 +697 @@
-bares)
+bares#bear_v1)
@@ -700 +700 @@
-it))
+it#it_pron))
@@ -703 +703 @@
-with)
+with#with_prep)
@@ -706 +706 @@
-a)
+a#a_adj2)
@@ -708 +708 @@
-great)
+great#great_adj)
@@ -710 +710 @@
-deal)
+deal#deal_n1)
@@ -713 +713 @@
-of)
+of#of_prep)
@@ -716 +716 @@
-patience)))))
+patience#patience_n.I)))))
@@ -725 +725 @@
-My)
+My#my_adj)
@@ -727 +727 @@
-Lady)
+Lady#lady_n)
@@ -730 +730 @@
-Rockingam)))
+Rockingam#NA)))
@@ -732 +732 @@
-has)
+has#have_v)
@@ -734 +734 @@
-been)
+been#be_v)
@@ -737 +737 @@
-with)
+with#with_prep)
@@ -740 +740 @@
-me)))
+me#me_pron1)))
@@ -743 +743 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -752 +752 @@
-I))
+I#i_pron))
@@ -754 +754 @@
-thought)
+thought#think_v2)
@@ -761 +761 @@
-there))
+there#there_adv))
@@ -763 +763 @@
-had)
+had#have_v)
@@ -765 +765 @@
-been)
+been#be_v)
@@ -768 +768 @@
-wine)
+wine#wine_n1)
@@ -771 +771 @@
-enough)))
+enough#enough_adj)))
@@ -774 +774 @@
-in)
+in#in_prep)
@@ -777 +777 @@
-the)
+the#the_adj)
@@ -779 +779 @@
-house)))))
+house#house_n1)))))
@@ -787 +787 @@
-but)
+but#but_prep)
@@ -790 +790 @@
-there))
+there#there_adv))
@@ -792 +792 @@
-is)
+is#be_v)
@@ -795 +795 @@
-none))
+none#none_pron))
@@ -797 +797 @@
-left)
+left#leave_v1)
@@ -805 +805 @@
-But)
+But#but_prep)
@@ -808 +808 @@
-there))
+there#there_adv))
@@ -810 +810 @@
-is)
+is#be_v)
@@ -814 +814 @@
-forteen)
+forteen#fourteen_adj)
@@ -816 +816 @@
-botles)
+botles#bottle_n2)
@@ -819 +819 @@
-of)
+of#of_prep)
@@ -822 +822 @@
-Renish))))
+Renish#rhenish_adj))))
@@ -825 +825 @@
-and)
+and#and_conj1)
@@ -828 +828 @@
-all)
+all#all_adj)
@@ -830 +830 @@
-the)
+the#the_adj)
@@ -832 +832 @@
-sherie)
+sherie#sherry_n1)
@@ -834 +834 @@
-sack)
+sack#sack_n3)
@@ -839 +839 @@
-that)
+that#that_conj)
@@ -844 +844 @@
-was)))))
+was#be_v)))))
@@ -849 +849 @@
-but)
+but#but_prep)
@@ -852 +852 @@
-noe)
+noe#no_adj)
@@ -854 +854 @@
-other))))
+other#other_adj))))
@@ -863 +863 @@
-so))
+so#so_adv))
@@ -866 +866 @@
-now))
+now#now_adv))
@@ -869 +869 @@
-you))
+you#you_pron))
@@ -871 +871 @@
-may)
+may#may_v1)
@@ -874 +874 @@
-the)
+the#the_adj)
@@ -876 +876 @@
-better))
+better#better_adj))
@@ -878 +878 @@
-gess)
+gess#guess_v)
@@ -882 +882 @@
-what))
+what#what_pron))
@@ -890 +890 @@
-you))
+you#you_pron))
@@ -892 +892 @@
-must)
+must#must_v1)
@@ -894 +894 @@
-send)
+send#send_v1)
@@ -896 +896 @@
-down)))
+down#down_adv)))
@@ -905 +905 @@
-Some)
+Some#some_pron)
@@ -907 +907 @@
-sack))
+sack#sack_n3))
@@ -910 +910 @@
-you))
+you#you_pron))
@@ -912 +912 @@
-will)
+will#will_v1)
@@ -914 +914 @@
-need)
+need#need_v2)
@@ -918 +918 @@
-I))
+I#i_pron))
@@ -920 +920 @@
-am)
+am#be_v)
@@ -923 +923 @@
-sure)))
+sure#sure_adj)))
@@ -932 +932 @@
-My)
+My#my_adj)
@@ -934 +934 @@
-deare)
+deare#dear_adj1)
@@ -936 +936 @@
-Lord))
+Lord#lord_n))
@@ -941 +941 @@
-I))
+I#i_pron))
@@ -943 +943 @@
-should)
+should#shall_v)
@@ -945 +945 @@
-be)
+be#be_v)
@@ -948 +948 @@
-very)
+very#very_adj)
@@ -950 +950 @@
-glad)
+glad#glad_adj)
@@ -957 +957 @@
-you))
+you#you_pron))
@@ -959 +959 @@
-would)
+would#will_v1)
@@ -961 +961 @@
-bring)
+bring#bring_v)
@@ -964 +964 @@
-some)
+some#some_pron)
@@ -966 +966 @@
-chocolate))
+chocolate#chocolate_n))
@@ -969 +969 @@
-along)
+along#along_adj2)
@@ -971 +971 @@
-with)
+with#with_prep)
@@ -974 +974 @@
-you))))))
+you#you_pron))))))
@@ -983 +983 @@
-I))
+I#i_pron))
@@ -985 +985 @@
-hope)
+hope#hope_v)
@@ -992 +992 @@
-I))
+I#i_pron))
@@ -994 +994 @@
-shall)
+shall#shall_v)
@@ -996 +996 @@
-receive)
+receive#receive_v)
@@ -999 +999 @@
-a)
+a#a_adj2)
@@ -1001 +1001 @@
-good)
+good#good_adj)
@@ -1003 +1003 @@
-acount)
+acount#account_n)
@@ -1006 +1006 @@
-of)
+of#of_prep)
@@ -1009 +1009 @@
-all)
+all#all_adj)
@@ -1011 +1011 @@
-your)
+your#your_pron)
@@ -1013 +1013 @@
-business))))))
+business#business_n))))))
@@ -1021 +1021 @@
-for)
+for#for_prep)
@@ -1024 +1024 @@
-I))
+I#i_pron))
@@ -1026 +1026 @@
-long)
+long#long_v1)
@@ -1029 +1029 @@
-to)
+to#to_prep)
@@ -1031 +1031 @@
-know))
+know#know_v))
@@ -1040 +1040 @@
-My)
+My#my_adj)
@@ -1042 +1042 @@
-daughter)
+daughter#daughter_n)
@@ -1045 +1045 @@
-Nany)))
+Nany#NA)))
@@ -1047 +1047 @@
-is)
+is#be_v)
@@ -1050 +1050 @@
-very)
+very#very_adj)
@@ -1052 +1052 @@
-well))
+well#well_adj))
@@ -1060 +1060 @@
-and)
+and#and_conj1)
@@ -1064 +1064 @@
-was)
+was#be_v)
@@ -1067 +1067 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -1070 +1070 @@
-at)
+at#at_prep)
@@ -1074 +1074 @@
-my)
+my#my_adj)
@@ -1076 +1076 @@
-Lord)
+Lord#lord_n)
@@ -1079 +1079 @@
-Brudnal@))
+Brudnal@#NA))
@@ -1081 +1081 @@
-@'s))))
+@'s#NA))))
@@ -1090 +1090 @@
-I))
+I#i_pron))
@@ -1092 +1092 @@
-believe)
+believe#believe_v)
@@ -1099 +1099 @@
-I))
+I#i_pron))
@@ -1101 +1101 @@
-shall)
+shall#shall_v)
@@ -1103 +1103 @@
-like)
+like#like_v1)
@@ -1106 +1106 @@
-your)
+your#your_pron)
@@ -1108 +1108 @@
-cook))
+cook#cook_n1))
@@ -1111 +1111 @@
-very)
+very#very_adj)
@@ -1113 +1113 @@
-well))))
+well#well_adv))))
@@ -1124 +1124 @@
-Pray))
+Pray#pray_v))
@@ -1129 +1129 @@
-deare))
+deare#dear_adj1))
@@ -1133 +1133 @@
-let)
+let#let_v1)
@@ -1137 +1137 @@
-Smith))
+Smith#NA))
@@ -1139 +1139 @@
-buy)
+buy#buy_v)
@@ -1143 +1143 @@
-a)
+a#a_adj2)
@@ -1145 +1145 @@
-Wesfaily)
+Wesfaily#NA)
@@ -1147 +1147 @@
-ham))
+ham#ham_n1))
@@ -1150 +1150 @@
-and)
+and#and_conj1)
@@ -1154 +1154 @@
-two)
+two#two_adj)
@@ -1156 +1156 @@
-or)
+or#or_conj2)
@@ -1158 +1158 @@
-three))
+three#three_adj))
@@ -1160 +1160 @@
-neats)
+neats#neat_n)
@@ -1162 +1162 @@
-tongues)))))
+tongues#tongue_n)))))
@@ -1171 +1171 @@
-I))
+I#i_pron))
@@ -1173 +1173 @@
-would)
+would#will_v1)
@@ -1176 +1176 @@
-fain))
+fain#fain_adj))
@@ -1178 +1178 @@
-get)
+get#get_v)
@@ -1182 +1182 @@
-every)
+every#every_adj)
@@ -1184 +1184 @@
-thing))
+thing#thing_n1))
@@ -1187 +1187 @@
-pretty)
+pretty#pretty_adv)
@@ -1189 +1189 @@
-handsome)))
+handsome#handsome_adj)))
@@ -1192 +1192 @@
-against)
+against#against_prep)
@@ -1199 +1199 @@
-my)
+my#my_adj)
@@ -1201 +1201 @@
-Lord)
+Lord#lord_n)
@@ -1204 +1204 @@
-Manchester)))
+Manchester#NA)))
@@ -1206 +1206 @@
-comes))))
+comes#come_v))))
@@ -1215 +1215 @@
-I))
+I#i_pron))
@@ -1217 +1217 @@
-hope)
+hope#hope_v)
@@ -1224 +1224 @@
-I))
+I#i_pron))
@@ -1226 +1226 @@
-shall)
+shall#shall_v)
@@ -1228 +1228 @@
-know)
+know#know_v)
@@ -1231 +1231 @@
-the)
+the#the_adj)
@@ -1233 +1233 @@
-time))))
+time#time_n))))
@@ -1242 +1242 @@
-I))
+I#i_pron))
@@ -1244 +1244 @@
-am)
+am#be_v)
@@ -1249 +1249 @@
-my)
+my#my_adj)
@@ -1251 +1251 @@
-deare)
+deare#dear_adj1)
@@ -1253 +1253 @@
-Lord))
+Lord#lord_n))
@@ -1258 +1258 @@
-overjoyed)
+overjoyed#overjoyed_adj)
@@ -1261 +1261 @@
-to)
+to#to_prep)
@@ -1263 +1263 @@
-hear)
+hear#hear_v)
@@ -1266 +1266 @@
-that)
+that#that_conj)
@@ -1270 +1270 @@
-your)
+your#your_pron)
@@ -1272 +1272 @@
-business))
+business#business_n))
@@ -1274 +1274 @@
-goes)
+goes#go_v)
@@ -1276 +1276 @@
-on)
+on#on_adv)
@@ -1279 +1279 @@
-so)
+so#so_adv)
@@ -1281 +1281 @@
-well))
+well#well_adv))
@@ -1284 +1284 @@
-with)
+with#with_prep)
@@ -1287 +1287 @@
-the)
+the#the_adj)
@@ -1289 +1289 @@
-Bishop)))))))
+Bishop#bishop_n)))))))
@@ -1298 +1298 @@
-I))
+I#i_pron))
@@ -1300 +1300 @@
-long)
+long#long_v1)
@@ -1303 +1303 @@
-till)
+till#till_prep)
@@ -1310 +1310 @@
-the)
+the#the_adj)
@@ -1312 +1312 @@
-Act)
+Act#act_n)
@@ -1315 +1315 @@
-of)
+of#of_prep)
@@ -1318 +1318 @@
-Parliment))))
+Parliment#parliament_n1))))
@@ -1320 +1320 @@
-be)
+be#be_v)
@@ -1322 +1322 @@
-passed))))
+passed#pass_v))))
@@ -1331 +1331 @@
-I))
+I#i_pron))
@@ -1333 +1333 @@
-received)
+received#receive_v)
@@ -1337 +1337 @@
-D=r=)
+D=r=#NA)
@@ -1339 +1339 @@
-Kings))
+Kings#NA))
@@ -1341 +1341 @@
-letter))
+letter#letter_n1))
@@ -1349 +1349 @@
-but)
+but#but_prep)
@@ -1352 +1352 @@
-I))
+I#i_pron))
@@ -1354 +1354 @@
-shall)
+shall#shall_v)
@@ -1356 +1356 @@
-not)
+not#not_adv)
@@ -1358 +1358 @@
-need)
+need#need_v2)
@@ -1361 +1361 @@
-much)
+much#much_adj)
@@ -1364 +1364 @@
-of)
+of#of_prep)
@@ -1367 +1367 @@
-his)
+his#his_adj)
@@ -1369 +1369 @@
-phiseck))))
+phiseck#physic_n))))
@@ -1377 +1377 @@
-for)
+for#for_prep)
@@ -1380 +1380 @@
-I))
+I#i_pron))
@@ -1382 +1382 @@
-thank)
+thank#thank_v)
@@ -1385 +1385 @@
-God))
+God#NA))
@@ -1392 +1392 @@
-I))
+I#i_pron))
@@ -1394 +1394 @@
-am)
+am#be_v)
@@ -1397 +1397 @@
-much)
+much#much_adj)
@@ -1399 +1399 @@
-better))))
+better#better_adj))))
@@ -1408 +1408 @@
-Smith))
+Smith#NA))
@@ -1410 +1410 @@
-forgot)
+forgot#forget_v)
@@ -1413 +1413 @@
-to)
+to#to_prep)
@@ -1415 +1415 @@
-send)
+send#send_v1)
@@ -1418 +1418 @@
-the)
+the#the_adj)
@@ -1420 +1420 @@
-pickales))
+pickales#pickle_n1))
@@ -1422 +1422 @@
-down))
+down#down_adv))
@@ -1430 +1430 @@
-but)
+but#but_prep)
@@ -1433 +1433 @@
-I))
+I#i_pron))
@@ -1435 +1435 @@
-beg)
+beg#beg_v)
@@ -1442 +1442 @@
-they))
+they#they_pron))
@@ -1444 +1444 @@
-may)
+may#may_v1)
@@ -1446 +1446 @@
-not)
+not#not_adv)
@@ -1448 +1448 @@
-faile)
+faile#fail_v)
@@ -1451 +1451 @@
-this)
+this#this_pron)
@@ -1453 +1453 @@
-next)
+next#next_adj)
@@ -1455 +1455 @@
-week))))
+week#week_n))))
@@ -1463 +1463 @@
-and)
+and#and_conj1)
@@ -1468 +1468 @@
-with)
+with#with_prep)
@@ -1471 +1471 @@
-them)))
+them#them_pron)))
@@ -1476 +1476 @@
-some)
+some#some_pron)
@@ -1478 +1478 @@
-oyle))
+oyle#oil_n1))
@@ -1487 +1487 @@
-about)
+about#about_adv)
@@ -1489 +1489 @@
-a)
+a#a_adj2)
@@ -1491 +1491 @@
-pint))
+pint#pint_n))
@@ -1493 +1493 @@
-will)
+will#will_v1)
@@ -1495 +1495 @@
-be)
+be#be_v)
@@ -1498 +1498 @@
-enough))
+enough#enough_adj))
@@ -1503 +1503 @@
-because)
+because#because_adv)
@@ -1510 +1510 @@
-wee))
+wee#we_pron))
@@ -1512 +1512 @@
-shall)
+shall#shall_v)
@@ -1514 +1514 @@
-stay)
+stay#stay_v1)
@@ -1517 +1517 @@
-noe)
+noe#no_adj)
@@ -1519 +1519 @@
-great)
+great#great_adj)
@@ -1521 +1521 @@
-whill))
+whill#while_n))
@@ -1524 +1524 @@
-there)))))
+there#there_adv)))))
@@ -1533 +1533 @@
-I))
+I#i_pron))
@@ -1535 +1535 @@
-return)
+return#return_v1)
@@ -1538 +1538 @@
-you))
+you#you_pron))
@@ -1541 +1541 @@
-many)
+many#many_adj)
@@ -1543 +1543 @@
-thanks)
+thanks#thank_n)
@@ -1546 +1546 @@
-for)
+for#for_prep)
@@ -1549 +1549 @@
-the)
+the#the_adj)
@@ -1551 +1551 @@
-oysters))))
+oysters#oyster_n))))
@@ -1560 +1560 @@
-I))
+I#i_pron))
@@ -1562 +1562 @@
-can)
+can#can_v1)
@@ -1564 +1564 @@
-write)
+write#write_v)
@@ -1568 +1568 @@
-noe)
+noe#no_adj)
@@ -1570 +1570 @@
-more)))
+more#more_adj)))
@@ -1576 +1576 @@
-to_morow))
+to_morow#tomorrow_adv))
@@ -1578 +1578 @@
-being)
+being#be_v)
@@ -1581 +1581 @@
-sacrament)
+sacrament#NA)
@@ -1583 +1583 @@
-day)))
+day#NA)))
@@ -1591 +1591 @@
-But)
+But#but_prep)
@@ -1596 +1596 @@
-for)
+for#for_prep)
@@ -1599 +1599 @@
-God)
+God#NA)
@@ -1601 +1601 @@
-sake)))
+sake#sake_n1)))
@@ -1605 +1605 @@
-make)
+make#make_v1)
@@ -1608 +1608 @@
-hast))
+hast#haste_n))
@@ -1610 +1610 @@
-down)
+down#down_adv)
@@ -1618 +1618 @@
-for)
+for#for_prep)
@@ -1621 +1621 @@
-I))
+I#i_pron))
@@ -1623 +1623 @@
-am)
+am#be_v)
@@ -1626 +1626 @@
-weary)
+weary#weary_adj)
@@ -1629 +1629 @@
-of)
+of#of_prep)
@@ -1632 +1632 @@
-my)
+my#my_adj)
@@ -1634 +1634 @@
-life))))
+life#life_n))))
