--- gawdy-1588-e2-h.psd.vert	2025-09-10 13:50:35
+++ gawdy-1588-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Sir))
+Sir#sir_n))
@@ -19 +19 @@
-I))
+I#i_pron))
@@ -21 +21 @@
-receyved)
+receyved#receive_v)
@@ -24 +24 @@
-a)
+a#a_adj2)
@@ -26 +26 @@
-note)
+note#note_n2)
@@ -29 +29 @@
-from)
+from#from_prep)
@@ -32 +32 @@
-you))))
+you#you_pron))))
@@ -35 +35 @@
-by)
+by#by_prep)
@@ -38 +38 @@
-my)
+my#my_adj)
@@ -40 +40 @@
-cosin)
+cosin#cousin_n)
@@ -43 +43 @@
-Brampton))))
+Brampton#NA))))
@@ -46 +46 @@
-meeting)
+meeting#meet_v)
@@ -49 +49 @@
-him))
+him#him_pron))
@@ -52 +52 @@
-by)
+by#by_prep)
@@ -55 +55 @@
-chaunce)))
+chaunce#chance_n)))
@@ -58 +58 @@
-vppon)
+vppon#upon_prep)
@@ -61 +61 @@
-Wednesdaye)
+Wednesdaye#NA)
@@ -64 +64 @@
-at)
+at#at_prep)
@@ -67 +67 @@
-night)))
+night#night_n)))
@@ -71 +71 @@
-laste))
+laste#last_adv))
@@ -73 +73 @@
-past)))))
+past#pass_v)))))
@@ -78 +78 @@
-when)
+when#when_adv)
@@ -85 +85 @@
-I))
+I#i_pron))
@@ -87 +87 @@
-vnderstood)
+vnderstood#understand_v)
@@ -90 +90 @@
-that)
+that#that_conj)
@@ -94 +94 @@
-yow))
+yow#you_pron))
@@ -96 +96 @@
-wold)
+wold#will_v1)
@@ -98 +98 @@
-have)
+have#have_v)
@@ -102 +102 @@
-certeyne)
+certeyne#certain_adj)
@@ -104 +104 @@
-trees))
+trees#tree_n))
@@ -106 +106 @@
-to)
+to#to_prep)
@@ -108 +108 @@
-be)
+be#be_v)
@@ -110 +110 @@
-sent)
+sent#send_v1)
@@ -112 +112 @@
-downe)))))))
+downe#down_adv)))))))
@@ -121 +121 @@
-The)
+The#the_adj)
@@ -123 +123 @@
-tyme))
+tyme#time_n))
@@ -125 +125 @@
-was)
+was#be_v)
@@ -128 +128 @@
-very)
+very#very_adj)
@@ -130 +130 @@
-short)
+short#short_adj)
@@ -133 +133 @@
-to)
+to#to_prep)
@@ -135 +135 @@
-have)
+have#have_v)
@@ -137 +137 @@
-sent)
+sent#send_v1)
@@ -140 +140 @@
-any))
+any#any_adj))
@@ -142 +142 @@
-downe)
+downe#down_adv)
@@ -145 +145 @@
-by)
+by#by_prep)
@@ -148 +148 @@
-theise)
+theise#these_pron)
@@ -150 +150 @@
-carryars)))))
+carryars#carrier_n)))))
@@ -159 +159 @@
-besides))
+besides#besides_adv))
@@ -162 +162 @@
-the)
+the#the_adj)
@@ -164 +164 @@
-weather))
+weather#weather_n))
@@ -166 +166 @@
-was)
+was#be_v)
@@ -168 +168 @@
-not)
+not#not_adv)
@@ -171 +171 @@
-apt)
+apt#apt_adj)
@@ -174 +174 @@
-for)
+for#for_prep)
@@ -177 +177 @@
-removing)
+removing#removing_n)
@@ -180 +180 @@
-of)
+of#of_prep)
@@ -183 +183 @@
-trees))))))
+trees#tree_n))))))
@@ -187 +187 @@
-the)
+the#the_adj)
@@ -189 +189 @@
-frost))
+frost#frost_n))
@@ -191 +191 @@
-falling)
+falling#fall_v)
@@ -193 +193 @@
-out)
+out#out_adv)
@@ -196 +196 @@
-to)
+to#to_prep)
@@ -198 +198 @@
-be)
+be#be_v)
@@ -201 +201 @@
-so)
+so#so_adv)
@@ -203 +203 @@
-great))))
+great#great_adj))))
@@ -212 +212 @@
-My)
+My#my_adj)
@@ -214 +214 @@
-skill))
+skill#skill_n1))
@@ -216 +216 @@
-is)
+is#be_v)
@@ -219 +219 @@
-small))
+small#small_adj))
@@ -222 +222 @@
-in)
+in#in_prep)
@@ -225 +225 @@
-making)
+making#make_v1)
@@ -228 +228 @@
-choyse)
+choyse#choice_n)
@@ -231 +231 @@
-of)
+of#of_prep)
@@ -234 +234 @@
-suche)
+suche#such_adj)
@@ -236 +236 @@
-thinges)))))))
+thinges#thing_n1)))))))
@@ -243 +243 @@
-neverthelesse))
+neverthelesse#nevertheless_adv))
@@ -246 +246 @@
-I))
+I#i_pron))
@@ -248 +248 @@
-will)
+will#will_v1)
@@ -250 +250 @@
-vse)
+vse#use_v)
@@ -253 +253 @@
-all)
+all#all_adj)
@@ -255 +255 @@
-the)
+the#the_adj)
@@ -257 +257 @@
-cunning)
+cunning#cunning_n)
@@ -268 +268 @@
-I))
+I#i_pron))
@@ -270 +270 @@
-have)
+have#have_v)
@@ -273 +273 @@
-to)
+to#to_prep)
@@ -275 +275 @@
-performe)
+performe#perform_v)
@@ -280 +280 @@
-the)
+the#the_adj)
@@ -283 +283 @@
-most)
+most#most_adj)
@@ -285 +285 @@
-dutifull))
+dutifull#dutiful_adj))
@@ -287 +287 @@
-service)
+service#service_n1)
@@ -298 +298 @@
-I))
+I#i_pron))
@@ -300 +300 @@
-maye)
+maye#may_v1)
@@ -311 +311 @@
-My)
+My#my_adj)
@@ -313 +313 @@
-Cosin)
+Cosin#cousin_n)
@@ -316 +316 @@
-Bramptyn)))
+Bramptyn#NA)))
@@ -318 +318 @@
-hathe)
+hathe#have_v)
@@ -320 +320 @@
-directed)
+directed#direct_v)
@@ -323 +323 @@
-mee))
+mee#me_pron1))
@@ -326 +326 @@
-to)
+to#to_prep)
@@ -328 +328 @@
-take)
+take#take_v)
@@ -331 +331 @@
-the)
+the#the_adj)
@@ -333 +333 @@
-advise)
+advise#advice_n)
@@ -336 +336 @@
-of)
+of#of_prep)
@@ -341 +341 @@
-one)
+one#one_adj)
@@ -346 +346 @@
-that)
+that#that_conj)
@@ -351 +351 @@
-hathe)
+hathe#have_v)
@@ -355 +355 @@
-very)
+very#very_adj)
@@ -357 +357 @@
-great))
+great#great_adj))
@@ -359 +359 @@
-good)
+good#good_adj)
@@ -361 +361 @@
-skill))))
+skill#skill_n1))))
@@ -366 +366 @@
-besides)
+besides#besides_adv)
@@ -369 +369 @@
-some)
+some#some_pron)
@@ -371 +371 @@
-other)
+other#other_adj)
@@ -376 +376 @@
-that)
+that#that_conj)
@@ -380 +380 @@
-I))
+I#i_pron))
@@ -382 +382 @@
-do)
+do#do_v)
@@ -384 +384 @@
-meane)
+meane#mean_v1)
@@ -387 +387 @@
-to)
+to#to_prep)
@@ -389 +389 @@
-make)
+make#make_v1)
@@ -394 +394 @@
-acquaynted)
+acquaynted#acquaint_v)
@@ -397 +397 @@
-with)
+with#with_prep)
@@ -400 +400 @@
-the)
+the#the_adj)
@@ -402 +402 @@
-choyse)
+choyse#choice_n)
@@ -405 +405 @@
-of)
+of#of_prep)
@@ -408 +408 @@
-the)
+the#the_adj)
@@ -410 +410 @@
-best)))))))))))))))
+best#best_adj)))))))))))))))
@@ -419 +419 @@
-I))
+I#i_pron))
@@ -421 +421 @@
-do)
+do#do_v)
@@ -423 +423 @@
-hope)
+hope#hope_v)
@@ -426 +426 @@
-to)
+to#to_prep)
@@ -429 +429 @@
-provide)
+provide#provide_v)
@@ -431 +431 @@
-and)
+and#and_conj1)
@@ -433 +433 @@
-make))
+make#make_v1))
@@ -437 +437 @@
-them))
+them#them_pron))
@@ -440 +440 @@
-ready))
+ready#ready_adj))
@@ -443 +443 @@
-in)
+in#in_prep)
@@ -446 +446 @@
-good)
+good#good_adj)
@@ -448 +448 @@
-order)))
+order#order_n)))
@@ -451 +451 @@
-agaynst)
+agaynst#against_prep)
@@ -454 +454 @@
-next)
+next#next_adj)
@@ -456 +456 @@
-Thursday)))))
+Thursday#NA)))))
@@ -465 +465 @@
-I))
+I#i_pron))
@@ -467 +467 @@
-have)
+have#have_v)
@@ -470 +470 @@
-allready))
+allready#already_adj))
@@ -472 +472 @@
-sent)
+sent#send_v1)
@@ -475 +475 @@
-yow))
+yow#you_pron))
@@ -477 +477 @@
-downe)
+downe#down_adv)
@@ -480 +480 @@
-your)
+your#your_pron)
@@ -482 +482 @@
-foote)
+foote#foot_n)
@@ -484 +484 @@
-clothe))
+clothe#cloth_n))
@@ -493 +493 @@
-Bourne)
+Bourne#NA)
@@ -497 +497 @@
-S=r=)
+S=r=#NA)
@@ -499 +499 @@
-Edward)
+Edward#NA)
@@ -501 +501 @@
-Cleres))
+Cleres#NA))
@@ -503 +503 @@
-man)))
+man#man_n1)))
@@ -505 +505 @@
-had)
+had#have_v)
@@ -508 +508 @@
-charg)
+charg#charge_n)
@@ -511 +511 @@
-of)
+of#of_prep)
@@ -514 +514 @@
-it))))
+it#it_pron))))
@@ -523 +523 @@
-I))
+I#i_pron))
@@ -525 +525 @@
-have)
+have#have_v)
@@ -528 +528 @@
-nowe))
+nowe#now_adv))
@@ -530 +530 @@
-sent)
+sent#send_v1)
@@ -533 +533 @@
-you))
+you#you_pron))
@@ -535 +535 @@
-downe)
+downe#down_adv)
@@ -539 +539 @@
-a)
+a#a_adj2)
@@ -541 +541 @@
-cocke))
+cocke#cock_n1))
@@ -547 +547 @@
-ij)
+ij#two_adj)
@@ -549 +549 @@
-payre)
+payre#pair_n1)
@@ -552 +552 @@
-of)
+of#of_prep)
@@ -555 +555 @@
-gloves)))))
+gloves#glove_n)))))
@@ -561 +561 @@
-ij)
+ij#two_adj)
@@ -563 +563 @@
-dosen)
+dosen#dozen_n)
@@ -566 +566 @@
-of)
+of#of_prep)
@@ -569 +569 @@
-poyntes)))))
+poyntes#point_n3)))))
@@ -575 +575 @@
-ij)
+ij#two_adj)
@@ -577 +577 @@
-small)
+small#small_adj)
@@ -579 +579 @@
-bookes)
+bookes#book_n)
@@ -582 +582 @@
-for)
+for#for_prep)
@@ -585 +585 @@
-a)
+a#a_adj2)
@@ -587 +587 @@
-token))))))
+token#token_n))))))
@@ -596 +596 @@
-the)
+the#the_adj)
@@ -598 +598 @@
-one)
+one#one_adj)
@@ -601 +601 @@
-of)
+of#of_prep)
@@ -604 +604 @@
-them))))
+them#them_pron))))
@@ -606 +606 @@
-was)
+was#be_v)
@@ -608 +608 @@
-gyven)
+gyven#give_v)
@@ -611 +611 @@
-me))
+me#me_pron1))
@@ -614 +614 @@
-that)
+that#that_pron)
@@ -616 +616 @@
-day)
+day#day_n)
@@ -621 +621 @@
-that)
+that#that_conj)
@@ -627 +627 @@
-they))
+they#they_pron))
@@ -629 +629 @@
-rann)
+rann#run_v)
@@ -632 +632 @@
-at)
+at#at_prep)
@@ -635 +635 @@
-tilt))))))
+tilt#tilt_n2))))))
@@ -642 +642 @@
-Divers)
+Divers#divers_adj)
@@ -645 +645 @@
-of)
+of#of_prep)
@@ -648 +648 @@
-them))))
+them#them_pron))))
@@ -650 +650 @@
-being)
+being#be_v)
@@ -652 +652 @@
-gyven)
+gyven#give_v)
@@ -655 +655 @@
-to)
+to#to_prep)
@@ -658 +658 @@
-most)
+most#most_adj)
@@ -661 +661 @@
-of)
+of#of_prep)
@@ -664 +664 @@
-the)
+the#the_adj)
@@ -666 +666 @@
-lordes)
+lordes#lord_n)
@@ -671 +671 @@
-and)
+and#and_conj1)
@@ -674 +674 @@
-gentlemen)
+gentlemen#gentleman_n)
@@ -677 +677 @@
-about)
+about#about_adv)
@@ -680 +680 @@
-the)
+the#the_adj)
@@ -682 +682 @@
-Court))))))))))
+Court#court_n1))))))))))
@@ -687 +687 @@
-and)
+and#and_conj1)
@@ -691 +691 @@
-one))
+one#one_adj))
@@ -694 +694 @@
-especially))
+especially#especially_adv))
@@ -697 +697 @@
-to)
+to#to_prep)
@@ -700 +700 @@
-the)
+the#the_adj)
@@ -702 +702 @@
-Quene))))))
+Quene#queen_n))))))
@@ -711 +711 @@
-I))
+I#i_pron))
@@ -713 +713 @@
-was)
+was#be_v)
@@ -715 +715 @@
-commanded)
+commanded#command_v)
@@ -718 +718 @@
-w=th=)
+w=th=#with_prep)
@@ -721 +721 @@
-some)
+some#some_pron)
@@ -724 +724 @@
-ij)
+ij#two_adj)
@@ -726 +726 @@
-or)
+or#or_conj2)
@@ -728 +728 @@
-iij))
+iij#three_adj))
@@ -731 +731 @@
-more))))
+more#more_adj))))
@@ -734 +734 @@
-to)
+to#to_prep)
@@ -736 +736 @@
-wayte)
+wayte#wait_v1)
@@ -739 +739 @@
-vnderneathe)
+vnderneathe#underneath_prep)
@@ -742 +742 @@
-the)
+the#the_adj)
@@ -744 +744 @@
-Quene)))
+Quene#queen_n)))
@@ -747 +747 @@
-to)
+to#to_prep)
@@ -749 +749 @@
-be)
+be#be_v)
@@ -751 +751 @@
-employed)
+employed#employ_v)
@@ -754 +754 @@
-in)
+in#in_prep)
@@ -757 +757 @@
-messages)))))
+messages#message_n)))))
@@ -763 +763 @@
-wher))
+wher#where_adv))
@@ -771 +771 @@
-it))
+it#it_pron))
@@ -773 +773 @@
-pleased)
+pleased#please_v)
@@ -776 +776 @@
-her)
+her#her_pron2)
@@ -778 +778 @@
-Ma=te=))
+Ma=te=#majesty_n))
@@ -781 +781 @@
-to)
+to#to_prep)
@@ -783 +783 @@
-vse)
+vse#use_v)
@@ -786 +786 @@
-me))
+me#me_pron1))
@@ -789 +789 @@
-more)
+more#more_adj)
@@ -791 +791 @@
-gracyously))
+gracyously#graciously_adv))
@@ -794 +794 @@
-then)
+then#than_conj)
@@ -805 +805 @@
-I))
+I#i_pron))
@@ -807 +807 @@
-deserved)))))))
+deserved#deserve_v)))))))
@@ -816 +816 @@
-My)
+My#my_adj)
@@ -818 +818 @@
-oncle))
+oncle#uncle_n))
@@ -820 +820 @@
-told)
+told#tell_v)
@@ -823 +823 @@
-me))
+me#me_pron1))
@@ -826 +826 @@
-of)
+of#of_prep)
@@ -829 +829 @@
-it)))
+it#it_pron)))
@@ -832 +832 @@
-that)
+that#that_pron)
@@ -834 +834 @@
-night))
+night#night_n))
@@ -837 +837 @@
-comming)
+comming#come_v)
@@ -840 +840 @@
-to)
+to#to_prep)
@@ -843 +843 @@
-him))))
+him#him_pron))))
@@ -851 +851 @@
-for)
+for#for_prep)
@@ -855 +855 @@
-ij)
+ij#two_adj)
@@ -857 +857 @@
-or)
+or#or_conj2)
@@ -859 +859 @@
-iij))
+iij#three_adj))
@@ -862 +862 @@
-of)
+of#of_prep)
@@ -865 +865 @@
-his)
+his#his_adj)
@@ -867 +867 @@
-men))))
+men#man_n1))))
@@ -869 +869 @@
-stood)
+stood#stand_v)
@@ -872 +872 @@
-hard)
+hard#hard_adv)
@@ -874 +874 @@
-by))
+by#by_prep))
@@ -877 +877 @@
-in)
+in#in_prep)
@@ -880 +880 @@
-a)
+a#a_adj2)
@@ -882 +882 @@
-standing)))
+standing#standing_n)))
@@ -891 +891 @@
-He))
+He#he_pron))
@@ -893 +893 @@
-was)
+was#be_v)
@@ -896 +896 @@
-wonderfully)
+wonderfully#wonderfully_adv)
@@ -898 +898 @@
-well))
+well#well_adv))
@@ -900 +900 @@
-pleased)
+pleased#please_v)
@@ -903 +903 @@
-therwith))
+therwith#therewith_adv))
@@ -909 +909 @@
-w=ch=))
+w=ch=#which_adj))
@@ -916 +916 @@
-cold)
+cold#can_v1)
@@ -918 +918 @@
-not)
+not#not_adv)
@@ -920 +920 @@
-be)
+be#be_v)
@@ -923 +923 @@
-displeasing)
+displeasing#displeasing_adj)
@@ -926 +926 @@
-to)
+to#to_prep)
@@ -929 +929 @@
-me))))
+me#me_pron1))))
@@ -932 +932 @@
-to)
+to#to_prep)
@@ -934 +934 @@
-se)
+se#see_v)
@@ -938 +938 @@
-him))
+him#him_pron))
@@ -941 +941 @@
-any)
+any#any_adj)
@@ -943 +943 @@
-waye))
+waye#way_n1))
@@ -946 +946 @@
-contented))))))
+contented#contented_adj))))))
@@ -955 +955 @@
-It))
+It#it_pron))
@@ -957 +957 @@
-may)
+may#may_v1)
@@ -959 +959 @@
-be)
+be#be_v)
@@ -966 +966 @@
-you))
+you#you_pron))
@@ -968 +968 @@
-may)
+may#may_v1)
@@ -970 +970 @@
-heare)
+heare#hear_v)
@@ -973 +973 @@
-of)
+of#of_prep)
@@ -976 +976 @@
-it)))
+it#it_pron)))
@@ -979 +979 @@
-by)
+by#by_prep)
@@ -982 +982 @@
-some)
+some#some_pron)
@@ -984 +984 @@
-other)))))
+other#other_adj)))))
@@ -992 +992 @@
-for)
+for#for_prep)
@@ -995 +995 @@
-it))
+it#it_pron))
@@ -997 +997 @@
-was)
+was#be_v)
@@ -999 +999 @@
-not)
+not#not_adv)
@@ -1002 +1002 @@
-private))
+private#private_adj1))
@@ -1011 +1011 @@
-It))
+It#it_pron))
@@ -1013 +1013 @@
-was)
+was#be_v)
@@ -1016 +1016 @@
-ffriday)
+ffriday#NA)
@@ -1019 +1019 @@
-at)
+at#at_prep)
@@ -1022 +1022 @@
-night)))
+night#night_n)))
@@ -1025 +1025 @@
-before)
+before#before_adv)
@@ -1032 +1032 @@
-the)
+the#the_adj)
@@ -1034 +1034 @@
-Quene))
+Quene#queen_n))
@@ -1036 +1036 @@
-came)
+came#come_v)
@@ -1039 +1039 @@
-to)
+to#to_prep)
@@ -1043 +1043 @@
-my)
+my#my_adj)
@@ -1045 +1045 @@
-Lo.)
+Lo.#lord_n)
@@ -1047 +1047 @@
-Admyralles))))))))
+Admyralles#admiral_n))))))))
@@ -1056 +1056 @@
-Vppon)
+Vppon#upon_prep)
@@ -1059 +1059 @@
-Saterday)))
+Saterday#NA)))
@@ -1061 +1061 @@
-was)
+was#be_v)
@@ -1064 +1064 @@
-the)
+the#the_adj)
@@ -1066 +1066 @@
-running)
+running#running_n)
@@ -1069 +1069 @@
-at)
+at#at_prep)
@@ -1072 +1072 @@
-the)
+the#the_adj)
@@ -1074 +1074 @@
-tilt))))
+tilt#tilt_n2))))
@@ -1077 +1077 @@
-very)
+very#very_adj)
@@ -1079 +1079 @@
-well))
+well#well_adv))
@@ -1081 +1081 @@
-performed)
+performed#perform_v)
@@ -1086 +1086 @@
-thoughe)
+thoughe#though_adv)
@@ -1090 +1090 @@
-not)
+not#not_adv)
@@ -1092 +1092 @@
-so)
+so#so_adv)
@@ -1094 +1094 @@
-full)
+full#full_adj)
@@ -1097 +1097 @@
-of)
+of#of_prep)
@@ -1100 +1100 @@
-devises))))
+devises#device_n))))
@@ -1103 +1103 @@
-and)
+and#and_conj1)
@@ -1106 +1106 @@
-so)
+so#so_adv)
@@ -1108 +1108 @@
-riche)))
+riche#rich_adj)))
@@ -1111 +1111 @@
-as)
+as#as_adv)
@@ -1122 +1122 @@
-I))
+I#i_pron))
@@ -1124 +1124 @@
-have)
+have#have_v)
@@ -1126 +1126 @@
-seene))))))
+seene#see_v))))))
@@ -1136 +1136 @@
-My)
+My#my_adj)
@@ -1138 +1138 @@
-L.)
+L.#lord_n)
@@ -1141 +1141 @@
-of)
+of#of_prep)
@@ -1144 +1144 @@
-Essex))))
+Essex#NA))))
@@ -1147 +1147 @@
-and)
+and#and_conj1)
@@ -1150 +1150 @@
-my)
+my#my_adj)
@@ -1152 +1152 @@
-L.)
+L.#lord_n)
@@ -1155 +1155 @@
-of)
+of#of_prep)
@@ -1158 +1158 @@
-Cumberland))))))
+Cumberland#NA))))))
@@ -1160 +1160 @@
-wer)
+wer#be_v)
@@ -1163 +1163 @@
-the)
+the#the_adj)
@@ -1165 +1165 @@
-cheife)
+cheife#chief_adj)
@@ -1170 +1170 @@
-that)
+that#that_conj)
@@ -1175 +1175 @@
-ranne))))
+ranne#run_v))))
@@ -1184 +1184 @@
-M=r=)
+M=r=#NA)
@@ -1186 +1186 @@
-Gresham))
+Gresham#NA))
@@ -1188 +1188 @@
-was)
+was#be_v)
@@ -1191 +1191 @@
-well))
+well#well_adv))
@@ -1193 +1193 @@
-commended)
+commended#commend_v)
@@ -1196 +1196 @@
-for)
+for#for_prep)
@@ -1199 +1199 @@
-the)
+the#the_adj)
@@ -1201 +1201 @@
-credit)
+credit#credit_n)
@@ -1204 +1204 @@
-of)
+of#of_prep)
@@ -1207 +1207 @@
-Norffolk)))))
+Norffolk#NA)))))
@@ -1219 +1219 @@
-Vppon)
+Vppon#upon_prep)
@@ -1222 +1222 @@
-Saterday)
+Saterday#NA)
@@ -1225 +1225 @@
-at)
+at#at_prep)
@@ -1228 +1228 @@
-night)))))
+night#night_n)))))
@@ -1231 +1231 @@
-her)
+her#her_pron2)
@@ -1233 +1233 @@
-Ma=ie=))
+Ma=ie=#majesty_n))
@@ -1235 +1235 @@
-lay)
+lay#lie_v1)
@@ -1238 +1238 @@
-at)
+at#at_prep)
@@ -1242 +1242 @@
-my)
+my#my_adj)
@@ -1244 +1244 @@
-L.)
+L.#lord_n)
@@ -1246 +1246 @@
-Admiralles)))))
+Admiralles#admiral_n)))))
@@ -1251 +1251 @@
-and)
+and#and_conj1)
@@ -1255 +1255 @@
-all)
+all#all_adj)
@@ -1257 +1257 @@
-Sondaye))
+Sondaye#NA))
@@ -1260 +1260 @@
-at)
+at#at_prep)
@@ -1263 +1263 @@
-his)
+his#his_adj)
@@ -1265 +1265 @@
-charge)))))
+charge#charge_n)))))
@@ -1274 +1274 @@
-Vppon)
+Vppon#upon_prep)
@@ -1277 +1277 @@
-Mondaye)))
+Mondaye#NA)))
@@ -1280 +1280 @@
-she))
+she#she_pron))
@@ -1282 +1282 @@
-went)
+went#go_v)
@@ -1285 +1285 @@
-to)
+to#to_prep)
@@ -1289 +1289 @@
-M=r=)
+M=r=#NA)
@@ -1291 +1291 @@
-Seacreatoryes))))
+Seacreatoryes#NA))))
@@ -1294 +1294 @@
-to)
+to#to_prep)
@@ -1297 +1297 @@
-dinner)))
+dinner#dinner_n)))
@@ -1300 +1300 @@
-when)
+when#when_adv)
@@ -1307 +1307 @@
-in)
+in#in_prep)
@@ -1310 +1310 @@
-the)
+the#the_adj)
@@ -1312 +1312 @@
-gallory)))
+gallory#gallery_n)))
@@ -1315 +1315 @@
-before)
+before#before_adv)
@@ -1318 +1318 @@
-her)
+her#her_pron2)
@@ -1320 +1320 @@
-going)))
+going#going_n)))
@@ -1323 +1323 @@
-she))
+she#she_pron))
@@ -1325 +1325 @@
-knighted)
+knighted#knight_v)
@@ -1328 +1328 @@
-M=r=)
+M=r=#NA)
@@ -1330 +1330 @@
-Jhon)
+Jhon#NA)
@@ -1332 +1332 @@
-Pagington)))))
+Pagington#NA)))))
@@ -1341 +1341 @@
-That)
+That#that_pron)
@@ -1343 +1343 @@
-night))
+night#night_n))
@@ -1346 +1346 @@
-she))
+she#she_pron))
@@ -1348 +1348 @@
-cam)
+cam#come_v)
@@ -1351 +1351 @@
-to)
+to#to_prep)
@@ -1355 +1355 @@
-my)
+my#my_adj)
@@ -1357 +1357 @@
-Lo.)
+Lo.#lord_n)
@@ -1359 +1359 @@
-Admiralls))))
+Admiralls#admiral_n))))
@@ -1362 +1362 @@
-agayne))
+agayne#again_adv))
@@ -1370 +1370 @@
-and)
+and#and_conj1)
@@ -1374 +1374 @@
-dyned)
+dyned#dine_v)
@@ -1377 +1377 @@
-ther))
+ther#there_adv))
@@ -1380 +1380 @@
-vppon)
+vppon#upon_prep)
@@ -1383 +1383 @@
-Tuesday)))
+Tuesday#NA)))
@@ -1392 +1392 @@
-Vppon)
+Vppon#upon_prep)
@@ -1395 +1395 @@
-Tuesday)
+Tuesday#NA)
@@ -1398 +1398 @@
-at)
+at#at_prep)
@@ -1401 +1401 @@
-night)))))
+night#night_n)))))
@@ -1404 +1404 @@
-she))
+she#she_pron))
@@ -1406 +1406 @@
-came)
+came#come_v)
@@ -1409 +1409 @@
-to)
+to#to_prep)
@@ -1413 +1413 @@
-my)
+my#my_adj)
@@ -1415 +1415 @@
-L._Chaunclers))
+L._Chaunclers#lord-chancellor_n))
@@ -1419 +1419 @@
-wher))
+wher#where_adv))
@@ -1427 +1427 @@
-she))
+she#she_pron))
@@ -1429 +1429 @@
-hathe)
+hathe#have_v)
@@ -1431 +1431 @@
-bene)
+bene#be_v)
@@ -1434 +1434 @@
-ever)
+ever#ever_adv)
@@ -1436 +1436 @@
-since))))))
+since#since_adv))))))
@@ -1445 +1445 @@
-Her)
+Her#her_pron2)
@@ -1447 +1447 @@
-entertaynment)
+entertaynment#entertainment_n)
@@ -1453 +1453 @@
-hathe)
+hathe#have_v)
@@ -1455 +1455 @@
-bene)
+bene#be_v)
@@ -1458 +1458 @@
-very)
+very#very_adj)
@@ -1460 +1460 @@
-great))
+great#great_adj))
@@ -1463 +1463 @@
-ther))
+ther#there_adv))
@@ -1466 +1466 @@
-bothe)
+bothe#NA)
@@ -1468 +1468 @@
-for)
+for#for_prep)
@@ -1474 +1474 @@
-her)
+her#her_pron1)
@@ -1476 +1476 @@
-self))
+self#self_pron))
@@ -1479 +1479 @@
-and)
+and#and_conj1)
@@ -1482 +1482 @@
-all)
+all#all_adj)
@@ -1484 +1484 @@
-her)
+her#her_pron2)
@@ -1486 +1486 @@
-trayne)))))
+trayne#train_n1)))))
@@ -1492 +1492 @@
-w=ch=))
+w=ch=#which_adj))
@@ -1498 +1498 @@
-a)
+a#a_adj2)
@@ -1500 +1500 @@
-number)
+number#number_n)
@@ -1503 +1503 @@
-of)
+of#of_prep)
@@ -1506 +1506 @@
-vs))))
+vs#us_pron))))
@@ -1508 +1508 @@
-hathe)
+hathe#have_v)
@@ -1511 +1511 @@
-very)
+very#very_adj)
@@ -1513 +1513 @@
-well))
+well#well_adv))
@@ -1515 +1515 @@
-tasted)
+tasted#taste_v)
@@ -1518 +1518 @@
-of)
+of#of_prep)
@@ -1529 +1529 @@
-It))
+It#it_pron))
@@ -1531 +1531 @@
-is)
+is#be_v)
@@ -1533 +1533 @@
-thought)
+thought#think_v2)
@@ -1541 +1541 @@
-her)
+her#her_pron2)
@@ -1543 +1543 @@
-Ma=tie=))
+Ma=tie=#majesty_n))
@@ -1545 +1545 @@
-will)
+will#will_v1)
@@ -1547 +1547 @@
-tarry)
+tarry#tarry_v)
@@ -1552 +1552 @@
-ther))
+ther#there_adv))
@@ -1555 +1555 @@
-yet))
+yet#yet_adv))
@@ -1558 +1558 @@
-this)
+this#this_pron)
@@ -1560 +1560 @@
-Sevenight)))
+Sevenight#sennight_n)))
@@ -1563 +1563 @@
-and)
+and#and_conj1)
@@ -1567 +1567 @@
-hearafter))
+hearafter#hereafter_adv))
@@ -1570 +1570 @@
-at)
+at#at_prep)
@@ -1573 +1573 @@
-Whitehall)))))))
+Whitehall#NA)))))))
@@ -1582 +1582 @@
-I))
+I#i_pron))
@@ -1584 +1584 @@
-have)
+have#have_v)
@@ -1586 +1586 @@
-bene)
+bene#be_v)
@@ -1590 +1590 @@
-very)
+very#very_adj)
@@ -1592 +1592 @@
-little)))
+little#little_adj)))
@@ -1595 +1595 @@
-from)
+from#from_prep)
@@ -1598 +1598 @@
-the)
+the#the_adj)
@@ -1600 +1600 @@
-court)))
+court#court_n1)))
@@ -1603 +1603 @@
-all)
+all#all_adj)
@@ -1605 +1605 @@
-this)
+this#this_pron)
@@ -1607 +1607 @@
-while))
+while#while_n))
@@ -1616 +1616 @@
-yet))
+yet#yet_adv))
@@ -1619 +1619 @@
-once)
+once#once_adv)
@@ -1622 +1622 @@
-a)
+a#a_adj2)
@@ -1624 +1624 @@
-day)))
+day#day_n)))
@@ -1627 +1627 @@
-I))
+I#i_pron))
@@ -1629 +1629 @@
-do)
+do#do_v)
@@ -1631 +1631 @@
-visitt)
+visitt#visit_v)
@@ -1634 +1634 @@
-Chauncery)
+Chauncery#NA)
@@ -1636 +1636 @@
-lane)
+lane#NA)
@@ -1640 +1640 @@
-wher))
+wher#where_adv))
@@ -1648 +1648 @@
-I))
+I#i_pron))
@@ -1650 +1650 @@
-report)
+report#report_v)
@@ -1653 +1653 @@
-to)
+to#to_prep)
@@ -1656 +1656 @@
-him)))
+him#him_pron)))
@@ -1659 +1659 @@
-according)
+according#accord_v)
@@ -1662 +1662 @@
-to)
+to#to_prep)
@@ -1665 +1665 @@
-his)
+his#his_adj)
@@ -1667 +1667 @@
-request))))
+request#request_n1))))
@@ -1670 +1670 @@
-suche)
+suche#such_adj)
@@ -1672 +1672 @@
-newes)
+newes#news_n)
@@ -1675 +1675 @@
-as)
+as#as_adv)
@@ -1686 +1686 @@
-the)
+the#the_adj)
@@ -1688 +1688 @@
-court))
+court#court_n1))
@@ -1690 +1690 @@
-affordethe))))))))
+affordethe#afford_v))))))))
@@ -1699 +1699 @@
-Amongst)
+Amongst#amongst_prep)
@@ -1702 +1702 @@
-the)
+the#the_adj)
@@ -1704 +1704 @@
-rest)
+rest#rest_n2)
@@ -1707 +1707 @@
-of)
+of#of_prep)
@@ -1710 +1710 @@
-our)
+our#our_pron)
@@ -1712 +1712 @@
-talke)))))
+talke#talk_n)))))
@@ -1715 +1715 @@
-he))
+he#he_pron))
@@ -1717 +1717 @@
-knew)
+knew#know_v)
@@ -1720 +1720 @@
-my)
+my#my_adj)
@@ -1722 +1722 @@
-writing))
+writing#writing_n))
@@ -1725 +1725 @@
-at)
+at#at_prep)
@@ -1728 +1728 @@
-this)
+this#this_pron)
@@ -1730 +1730 @@
-present)))
+present#present_adj)))
@@ -1738 +1738 @@
-and)
+and#and_conj1)
@@ -1742 +1742 @@
-willed)
+willed#will_v2)
@@ -1745 +1745 @@
-me))
+me#me_pron1))
@@ -1747 +1747 @@
-not)
+not#not_adv)
@@ -1750 +1750 @@
-to)
+to#to_prep)
@@ -1752 +1752 @@
-forgett)
+forgett#forget_v)
@@ -1755 +1755 @@
-his)
+his#his_adj)
@@ -1758 +1758 @@
-very)
+very#very_adj)
@@ -1760 +1760 @@
-harty))
+harty#hearty_adj))
@@ -1762 +1762 @@
-commendacions)
+commendacions#commendation_n)
@@ -1765 +1765 @@
-to)
+to#to_prep)
@@ -1768 +1768 @@
-your)
+your#your_pron)
@@ -1770 +1770 @@
-good)
+good#good_adj)
@@ -1772 +1772 @@
-self)))))
+self#self_pron)))))
@@ -1781 +1781 @@
-Many)
+Many#many_adj)
@@ -1783 +1783 @@
-reasons))
+reasons#reason_n1))
@@ -1785 +1785 @@
-doth)
+doth#do_v)
@@ -1787 +1787 @@
-leade)
+leade#lead_v1)
@@ -1790 +1790 @@
-me))
+me#me_pron1))
@@ -1793 +1793 @@
-to)
+to#to_prep)
@@ -1795 +1795 @@
-follow)
+follow#follow_v)
@@ -1798 +1798 @@
-him)))
+him#him_pron)))
@@ -1806 +1806 @@
-and)
+and#and_conj1)
@@ -1809 +1809 @@
-the)
+the#the_adj)
@@ -1811 +1811 @@
-least)
+least#least_adj)
@@ -1813 +1813 @@
-favor)
+favor#favour_n)
@@ -1816 +1816 @@
-and)
+and#and_conj1)
@@ -1819 +1819 @@
-kyndnes))))
+kyndnes#kindness_n))))
@@ -1821 +1821 @@
-dothe)
+dothe#do_v)
@@ -1823 +1823 @@
-encorage)
+encorage#encourage_v)
@@ -1826 +1826 @@
-me))
+me#me_pron1))
@@ -1829 +1829 @@
-to)
+to#to_prep)
@@ -1831 +1831 @@
-do)
+do#do_v)
@@ -1834 +1834 @@
-him))
+him#him_pron))
@@ -1837 +1837 @@
-all)
+all#all_adj)
@@ -1839 +1839 @@
-manner)
+manner#manner_n)
@@ -1842 +1842 @@
-of)
+of#of_prep)
@@ -1845 +1845 @@
-service)))))
+service#service_n1)))))
@@ -1854 +1854 @@
-Ther))
+Ther#there_adv))
@@ -1856 +1856 @@
-be)
+be#be_v)
@@ -1859 +1859 @@
-no)
+no#no_adj)
@@ -1861 +1861 @@
-shreifes)
+shreifes#sheriff_n)
@@ -1864 +1864 @@
-as)
+as#as_adv)
@@ -1867 +1867 @@
-yet))))
+yet#yet_adv))))
@@ -1869 +1869 @@
-knowen))
+knowen#know_v))
@@ -1875 +1875 @@
-nor)
+nor#nor_conj1)
@@ -1879 +1879 @@
-wer)
+wer#be_v)
@@ -1881 +1881 @@
-not)
+not#not_adv)
@@ -1884 +1884 @@
-as)
+as#as_adv)
@@ -1887 +1887 @@
-yesternight)
+yesternight#yesternight_adv)
@@ -1890 +1890 @@
-at)
+at#at_prep)
@@ -1893 +1893 @@
-eleven)
+eleven#eleven_adj)
@@ -1896 +1896 @@
-of)
+of#of_prep)
@@ -1899 +1899 @@
-the)
+the#the_adj)
@@ -1901 +1901 @@
-clocke))))
+clocke#clock_n2))))
@@ -1904 +1904 @@
-at)
+at#at_prep)
@@ -1907 +1907 @@
-night))))))
+night#night_n))))))
@@ -1915 +1915 @@
-and)
+and#and_conj1)
@@ -1918 +1918 @@
-comming)
+comming#come_v)
@@ -1921 +1921 @@
-home))
+home#home_n1))
@@ -1924 +1924 @@
-late)))
+late#late_adv)))
@@ -1926 +1926 @@
-was)
+was#be_v)
@@ -1929 +1929 @@
-the)
+the#the_adj)
@@ -1931 +1931 @@
-cause)
+cause#cause_n)
@@ -1936 +1936 @@
-that)
+that#that_conj)
@@ -1942 +1942 @@
-my)
+my#my_adj)
@@ -1944 +1944 @@
-oncle)
+oncle#uncle_n)
@@ -1947 +1947 @@
-Anthony)
+Anthony#NA)
@@ -1949 +1949 @@
-Gavdy)))
+Gavdy#NA)))
@@ -1951 +1951 @@
-fayled)
+fayled#fail_v)
@@ -1954 +1954 @@
-of)
+of#of_prep)
@@ -1957 +1957 @@
-a)
+a#a_adj2)
@@ -1959 +1959 @@
-letter))))))
+letter#letter_n1))))))
@@ -1967 +1967 @@
-Neither)
+Neither#neither_adv)
@@ -1969 +1969 @@
-have)
+have#have_v)
@@ -1972 +1972 @@
-I))
+I#i_pron))
@@ -1975 +1975 @@
-as)
+as#as_adv)
@@ -1978 +1978 @@
-yet)))
+yet#yet_adv)))
@@ -1980 +1980 @@
-found)
+found#find_v)
@@ -1983 +1983 @@
-any)
+any#any_adj)
@@ -1985 +1985 @@
-messinger)
+messinger#messenger_n)
@@ -1988 +1988 @@
-but)
+but#but_prep)
@@ -1991 +1991 @@
-the)
+the#the_adj)
@@ -1993 +1993 @@
-hedgehogg))))
+hedgehogg#hedgehog_n))))
@@ -2001 +2001 @@
-and)
+and#and_conj1)
@@ -2004 +2004 @@
-yet))
+yet#yet_adv))
@@ -2007 +2007 @@
-I))
+I#i_pron))
@@ -2009 +2009 @@
-have)
+have#have_v)
@@ -2011 +2011 @@
-heard)
+heard#hear_v)
@@ -2014 +2014 @@
-that)
+that#that_conj)
@@ -2018 +2018 @@
-the)
+the#the_adj)
@@ -2020 +2020 @@
-hedgehogg))
+hedgehogg#hedgehog_n))
@@ -2022 +2022 @@
-overran)
+overran#overrun_v)
@@ -2025 +2025 @@
-the)
+the#the_adj)
@@ -2027 +2027 @@
-hare))))
+hare#hare_n))))
@@ -2035 +2035 @@
-and)
+and#and_conj1)
@@ -2038 +2038 @@
-so))
+so#so_adv))
@@ -2041 +2041 @@
-the)
+the#the_adj)
@@ -2043 +2043 @@
-carryar))
+carryar#carrier_n))
@@ -2045 +2045 @@
-may)
+may#may_v1)
@@ -2047 +2047 @@
-overrun)
+overrun#overrun_v)
@@ -2050 +2050 @@
-the)
+the#the_adj)
@@ -2052 +2052 @@
-post))
+post#post_n2))
@@ -2061 +2061 @@
-I))
+I#i_pron))
@@ -2063 +2063 @@
-know)
+know#know_v)
@@ -2065 +2065 @@
-not)
+not#not_adv)
@@ -2068 +2068 @@
-whether)
+whether#whether_pron)
@@ -2074 +2074 @@
-stale)
+stale#stale_adj1)
@@ -2076 +2076 @@
-Newes))
+Newes#news_n))
@@ -2078 +2078 @@
-may)
+may#may_v1)
@@ -2080 +2080 @@
-offend)
+offend#offend_v)
@@ -2083 +2083 @@
-his)
+his#his_adj)
@@ -2085 +2085 @@
-eares))
+eares#ear_n1))
@@ -2088 +2088 @@
-being)
+being#be_v)
@@ -2092 +2092 @@
-so)
+so#so_adv)
@@ -2094 +2094 @@
-long)))
+long#long_adj1)))
@@ -2097 +2097 @@
-a)
+a#a_prep1)
@@ -2100 +2100 @@
-drawing)
+drawing#drawing_n)
@@ -2103 +2103 @@
-towardes)
+towardes#towards_prep)
@@ -2106 +2106 @@
-him))))))))
+him#him_pron))))))))
@@ -2115 +2115 @@
-In)
+In#in_prep)
@@ -2118 +2118 @@
-the)
+the#the_adj)
@@ -2120 +2120 @@
-meane)
+meane#mean_adj2)
@@ -2122 +2122 @@
-tyme)))
+tyme#time_n)))
@@ -2125 +2125 @@
-I))
+I#i_pron))
@@ -2127 +2127 @@
-wold)
+wold#will_v1)
@@ -2129 +2129 @@
-have)
+have#have_v)
@@ -2133 +2133 @@
-him))
+him#him_pron))
@@ -2135 +2135 @@
-know)
+know#know_v)
@@ -2138 +2138 @@
-this)
+this#this_pron)
@@ -2143 +2143 @@
-that)
+that#that_conj)
@@ -2147 +2147 @@
-his)
+his#his_adj)
@@ -2149 +2149 @@
-Mrs.))
+Mrs.#mrs_n1))
@@ -2151 +2151 @@
-is)
+is#be_v)
@@ -2155 +2155 @@
-at)
+at#at_prep)
@@ -2158 +2158 @@
-the)
+the#the_adj)
@@ -2160 +2160 @@
-court)))
+court#court_n1)))
@@ -2163 +2163 @@
-and)
+and#and_conj1)
@@ -2166 +2166 @@
-sending)
+sending#send_v1)
@@ -2169 +2169 @@
-his)
+his#his_adj)
@@ -2171 +2171 @@
-commendacions)))))
+commendacions#commendation_n)))))
@@ -2176 +2176 @@
-with)
+with#with_prep)
@@ -2180 +2180 @@
-yowr)
+yowr#your_pron)
@@ -2182 +2182 @@
-self))
+self#self_pron))
@@ -2185 +2185 @@
-and)
+and#and_conj1)
@@ -2189 +2189 @@
-my)
+my#my_adj)
@@ -2191 +2191 @@
-good)
+good#good_adj)
@@ -2193 +2193 @@
-Brother))
+Brother#brother_n))
@@ -2196 +2196 @@
-and)
+and#and_conj1)
@@ -2199 +2199 @@
-his)
+his#his_adj)
@@ -2201 +2201 @@
-wyfe)))))))))))
+wyfe#wife_n)))))))))))
@@ -2210 +2210 @@
-She))
+She#she_pron))
@@ -2212 +2212 @@
-hathe)
+hathe#have_v)
@@ -2214 +2214 @@
-heard)
+heard#hear_v)
@@ -2217 +2217 @@
-nothing)
+nothing#nothing_pron)
@@ -2222 +2222 @@
-from)
+from#from_prep)
@@ -2225 +2225 @@
-him)))
+him#him_pron)))
@@ -2229 +2229 @@
-she))
+she#she_pron))
@@ -2231 +2231 @@
-saythe))
+saythe#say_v1))
@@ -2234 +2234 @@
-but)
+but#but_prep)
@@ -2237 +2237 @@
-only)
+only#NA)
@@ -2239 +2239 @@
-touching)
+touching#touching_prep)
@@ -2242 +2242 @@
-musters))))
+musters#muster_n1))))
@@ -2251 +2251 @@
-she))
+she#she_pron))
@@ -2253 +2253 @@
-dothe)
+dothe#do_v)
@@ -2256 +2256 @@
-not)
+not#not_adv)
@@ -2258 +2258 @@
-a)
+a#a_adj2)
@@ -2260 +2260 @@
-little))
+little#little_adj))
@@ -2262 +2262 @@
-marvell)
+marvell#marvel_v)
@@ -2265 +2265 @@
-at)
+at#at_prep)
@@ -2268 +2268 @@
-it)))
+it#it_pron)))
@@ -2277 +2277 @@
-I))
+I#i_pron))
@@ -2279 +2279 @@
-excused)
+excused#excuse_v)
@@ -2282 +2282 @@
-him))
+him#him_pron))
@@ -2290 +2290 @@
-and)
+and#and_conj1)
@@ -2294 +2294 @@
-layde)
+layde#lay_v1)
@@ -2297 +2297 @@
-all)
+all#all_adj)
@@ -2299 +2299 @@
-the)
+the#the_adj)
@@ -2301 +2301 @@
-fault))
+fault#fault_n))
@@ -2304 +2304 @@
-in)
+in#in_prep)
@@ -2307 +2307 @@
-his)
+his#his_adj)
@@ -2309 +2309 @@
-secretary))))
+secretary#secretary_n1))))
@@ -2315 +2315 @@
-for)
+for#for_prep)
@@ -2318 +2318 @@
-thoughe)
+thoughe#though_adv)
@@ -2325 +2325 @@
-he))
+he#he_pron))
@@ -2327 +2327 @@
-be)
+be#be_v)
@@ -2330 +2330 @@
-suer)))))
+suer#sure_adj)))))
@@ -2335 +2335 @@
-yet))
+yet#yet_adv))
@@ -2338 +2338 @@
-he))
+he#he_pron))
@@ -2340 +2340 @@
-is)
+is#be_v)
@@ -2343 +2343 @@
-very)
+very#very_adj)
@@ -2345 +2345 @@
-slow))
+slow#slow_adj))
@@ -2353 +2353 @@
-For)
+For#for_prep)
@@ -2356 +2356 @@
-suche)
+suche#such_adj)
@@ -2358 +2358 @@
-thinges)
+thinges#thing_n1)
@@ -2361 +2361 @@
-as)
+as#as_adv)
@@ -2370 +2370 @@
-my)
+my#my_adj)
@@ -2372 +2372 @@
-good)
+good#good_adj)
@@ -2374 +2374 @@
-sister))
+sister#sister_n))
@@ -2376 +2376 @@
-wryteth)
+wryteth#write_v)
@@ -2379 +2379 @@
-for)
+for#for_prep)
@@ -2384 +2384 @@
-she))
+she#she_pron))
@@ -2386 +2386 @@
-shall)
+shall#shall_v)
@@ -2388 +2388 @@
-receyve)
+receyve#receive_v)
@@ -2391 +2391 @@
-them)
+them#them_pron)
@@ -2394 +2394 @@
-all)))
+all#all_adj)))
@@ -2397 +2397 @@
-together))
+together#together_adv))
@@ -2400 +2400 @@
-with)
+with#with_prep)
@@ -2403 +2403 @@
-a)
+a#a_adj2)
@@ -2405 +2405 @@
-letter)))
+letter#letter_n1)))
@@ -2408 +2408 @@
-the)
+the#the_adj)
@@ -2410 +2410 @@
-next)
+next#next_adj)
@@ -2412 +2412 @@
-tyme))
+tyme#time_n))
@@ -2421 +2421 @@
-I))
+I#i_pron))
@@ -2423 +2423 @@
-have)
+have#have_v)
@@ -2425 +2425 @@
-written)
+written#write_v)
@@ -2428 +2428 @@
-to)
+to#to_prep)
@@ -2431 +2431 @@
-her)))
+her#her_pron1)))
@@ -2434 +2434 @@
-before))
+before#before_adv))
@@ -2437 +2437 @@
-touching)
+touching#touching_prep)
@@ -2441 +2441 @@
-Sir)
+Sir#NA)
@@ -2443 +2443 @@
-Charles))
+Charles#NA))
@@ -2448 +2448 @@
-and)
+and#and_conj1)
@@ -2451 +2451 @@
-my)
+my#my_adj)
@@ -2453 +2453 @@
-Lady)))
+Lady#lady_n)))
@@ -2458 +2458 @@
-and)
+and#and_conj1)
@@ -2462 +2462 @@
-divers)
+divers#divers_adj)
@@ -2464 +2464 @@
-others))
+others#other_adj))
@@ -2466 +2466 @@
-letters)))))
+letters#letter_n1)))))
@@ -2475 +2475 @@
-I))
+I#i_pron))
@@ -2477 +2477 @@
-know)
+know#know_v)
@@ -2479 +2479 @@
-not)
+not#not_adv)
@@ -2483 +2483 @@
-how))
+how#how_adv))
@@ -2491 +2491 @@
-they))
+they#they_pron))
@@ -2493 +2493 @@
-be)
+be#be_v)
@@ -2495 +2495 @@
-delivered)))
+delivered#deliver_v1)))
@@ -2504 +2504 @@
-Luke)
+Luke#NA)
@@ -2506 +2506 @@
-Woolmer))
+Woolmer#NA))
@@ -2508 +2508 @@
-hathe)
+hathe#have_v)
@@ -2510 +2510 @@
-sent)
+sent#send_v1)
@@ -2512 +2512 @@
-downe)
+downe#down_adv)
@@ -2515 +2515 @@
-all)
+all#all_adj)
@@ -2517 +2517 @@
-her)
+her#her_pron2)
@@ -2519 +2519 @@
-things))
+things#thing_n1))
@@ -2522 +2522 @@
-by)
+by#by_prep)
@@ -2525 +2525 @@
-Mrs.)
+Mrs.#NA)
@@ -2527 +2527 @@
-Tylney)))
+Tylney#NA)))
@@ -2536 +2536 @@
-I))
+I#i_pron))
@@ -2538 +2538 @@
-will)
+will#will_v1)
@@ -2540 +2540 @@
-send)
+send#send_v1)
@@ -2542 +2542 @@
-downe)
+downe#down_adv)
@@ -2546 +2546 @@
-her)
+her#her_pron2)
@@ -2548 +2548 @@
-pipes))
+pipes#pipe_n2))
@@ -2551 +2551 @@
-and)
+and#and_conj1)
@@ -2554 +2554 @@
-the)
+the#the_adj)
@@ -2556 +2556 @@
-rest))))
+rest#rest_n2))))
@@ -2559 +2559 @@
-as)
+as#as_adv)
@@ -2561 +2561 @@
-soon)
+soon#soon_adv)
@@ -2564 +2564 @@
-as)
+as#as_adv)
@@ -2575 +2575 @@
-I))
+I#i_pron))
@@ -2577 +2577 @@
-maye)
+maye#may_v1)
@@ -2588 +2588 @@
-M=r=)
+M=r=#NA)
@@ -2590 +2590 @@
-Jermy))
+Jermy#NA))
@@ -2592 +2592 @@
-is)
+is#be_v)
@@ -2594 +2594 @@
-travelled)
+travelled#travel_v)
@@ -2597 +2597 @@
-into)
+into#into_prep)
@@ -2600 +2600 @@
-a)
+a#a_adj2)
@@ -2602 +2602 @@
-Damaske)
+Damaske#damask_n)
@@ -2604 +2604 @@
-gowne)))
+gowne#gown_n)))
@@ -2612 +2612 @@
-and)
+and#and_conj1)
@@ -2616 +2616 @@
-followeth)
+followeth#follow_v)
@@ -2619 +2619 @@
-the)
+the#the_adj)
@@ -2621 +2621 @@
-courte))
+courte#court_n1))
@@ -2624 +2624 @@
-day)
+day#day_n)
@@ -2626 +2626 @@
-and)
+and#and_conj1)
@@ -2628 +2628 @@
-night))
+night#night_n))
@@ -2637 +2637 @@
-I))
+I#i_pron))
@@ -2639 +2639 @@
-have)
+have#have_v)
@@ -2641 +2641 @@
-forgott)
+forgott#forget_v)
@@ -2645 +2645 @@
-what))
+what#what_pron))
@@ -2651 +2651 @@
-he))
+he#he_pron))
@@ -2653 +2653 @@
-called)
+called#call_v)
@@ -2659 +2659 @@
-my)
+my#my_adj)
@@ -2661 +2661 @@
-oncle)
+oncle#uncle_n)
@@ -2664 +2664 @@
-Anthony))))
+Anthony#NA))))
@@ -2667 +2667 @@
-in)
+in#in_prep)
@@ -2670 +2670 @@
-Greeke))))))
+Greeke#NA))))))
@@ -2676 +2676 @@
-but)
+but#but_prep)
@@ -2679 +2679 @@
-I))
+I#i_pron))
@@ -2681 +2681 @@
-thinke)
+thinke#think_v2)
@@ -2688 +2688 @@
-he))
+he#he_pron))
@@ -2690 +2690 @@
-will)
+will#will_v1)
@@ -2693 +2693 @@
-harddly))
+harddly#hardly_adv))
@@ -2695 +2695 @@
-prove)
+prove#prove_v)
@@ -2699 +2699 @@
-him))
+him#him_pron))
@@ -2702 +2702 @@
-shreife)))
+shreife#sheriff_n)))
@@ -2705 +2705 @@
-in)
+in#in_prep)
@@ -2708 +2708 @@
-English)))))
+English#NA)))))
@@ -2717 +2717 @@
-I))
+I#i_pron))
@@ -2719 +2719 @@
-thanke)
+thanke#thank_v)
@@ -2722 +2722 @@
-his)
+his#his_adj)
@@ -2724 +2724 @@
-worshipp)))
+worshipp#worship_n)))
@@ -2731 +2731 @@
-I))
+I#i_pron))
@@ -2733 +2733 @@
-fynde)
+fynde#find_v)
@@ -2737 +2737 @@
-him))
+him#him_pron))
@@ -2740 +2740 @@
-very)
+very#very_adj)
@@ -2742 +2742 @@
-kynde)
+kynde#kind_adj)
@@ -2745 +2745 @@
-to)
+to#to_prep)
@@ -2748 +2748 @@
-me)))))
+me#me_pron1)))))
@@ -2756 +2756 @@
-and)
+and#and_conj1)
@@ -2759 +2759 @@
-he@))
+he@#he_pron))
@@ -2761 +2761 @@
-@s)
+@s#be_v)
@@ -2766 +2766 @@
-continually))
+continually#continually_adv))
@@ -2769 +2769 @@
-inquisityve)
+inquisityve#inquisitive_adj)
@@ -2772 +2772 @@
-of)
+of#of_prep)
@@ -2775 +2775 @@
-me)))
+me#me_pron1)))
@@ -2779 +2779 @@
-what))
+what#what_pron))
@@ -2787 +2787 @@
-I))
+I#i_pron))
@@ -2789 +2789 @@
-heare)
+heare#hear_v)
@@ -2792 +2792 @@
-of)
+of#of_prep)
@@ -2795 +2795 @@
-the)
+the#the_adj)
@@ -2797 +2797 @@
-pricking)
+pricking#pricking_n)
@@ -2800 +2800 @@
-of)
+of#of_prep)
@@ -2803 +2803 @@
-the)
+the#the_adj)
@@ -2805 +2805 @@
-shreifes))))))))
+shreifes#sheriff_n))))))))
@@ -2814 +2814 @@
-He))
+He#he_pron))
@@ -2816 +2816 @@
-travelled)
+travelled#travel_v)
@@ -2819 +2819 @@
-to)
+to#to_prep)
@@ -2822 +2822 @@
-the)
+the#the_adj)
@@ -2824 +2824 @@
-sea))))
+sea#sea_n))))
@@ -2830 +2830 @@
-and)
+and#and_conj1)
@@ -2834 +2834 @@
-lyes)
+lyes#lie_v1)
@@ -2837 +2837 @@
-alltogether))
+alltogether#altogether_adj))
@@ -2840 +2840 @@
-at)
+at#at_prep)
@@ -2843 +2843 @@
-the)
+the#the_adj)
@@ -2845 +2845 @@
-ankor)))
+ankor#anchor_n1)))
@@ -2854 +2854 @@
-if)
+if#if_conj)
@@ -2861 +2861 @@
-his)
+his#his_adj)
@@ -2863 +2863 @@
-kable))
+kable#cable_n))
@@ -2865 +2865 @@
-be)
+be#be_v)
@@ -2868 +2868 @@
-suer)))))
+suer#sure_adj)))))
@@ -2873 +2873 @@
-he))
+he#he_pron))
@@ -2875 +2875 @@
-is)
+is#be_v)
@@ -2878 +2878 @@
-like)
+like#like_adj)
@@ -2881 +2881 @@
-to)
+to#to_prep)
@@ -2883 +2883 @@
-do)
+do#do_v)
@@ -2886 +2886 @@
-the)
+the#the_adj)
@@ -2888 +2888 @@
-better))))
+better#better_adj))))
@@ -2897 +2897 @@
-My)
+My#my_adj)
@@ -2899 +2899 @@
-Lo.)
+Lo.#lord_n)
@@ -2902 +2902 @@
-of)
+of#of_prep)
@@ -2905 +2905 @@
-Leycester))))
+Leycester#NA))))
@@ -2907 +2907 @@
-is)
+is#be_v)
@@ -2909 +2909 @@
-expected)
+expected#expect_v)
@@ -2912 +2912 @@
-for)
+for#for_prep)
@@ -2922 +2922 @@
-and)
+and#and_conj1)
@@ -2925 +2925 @@
-some)
+some#some_pron)
@@ -2927 +2927 @@
-newes))
+newes#news_n))
@@ -2929 +2929 @@
-cam)
+cam#come_v)
@@ -2932 +2932 @@
-out)
+out#out_adv)
@@ -2934 +2934 @@
-of)
+of#of_prep)
@@ -2937 +2937 @@
-fflaunders)))
+fflaunders#NA)))
@@ -2941 +2941 @@
-very)
+very#very_adj)
@@ -2943 +2943 @@
-latly)
+latly#lately_adv)
@@ -2946 +2946 @@
-yesternight)))
+yesternight#yesternight_adv)))
@@ -2951 +2951 @@
-but)
+but#but_prep)
@@ -2954 +2954 @@
-vnknowen)
+vnknowen#unknown_adj)
@@ -2957 +2957 @@
-to)
+to#to_prep)
@@ -2960 +2960 @@
-any)
+any#any_adj)
@@ -2963 +2963 @@
-but)
+but#but_prep)
@@ -2966 +2966 @@
-the)
+the#the_adj)
@@ -2968 +2968 @@
-best))))))))
+best#best_adj))))))))
@@ -2977 +2977 @@
-Some)
+Some#some_pron)
@@ -2979 +2979 @@
-letters))
+letters#letter_n1))
@@ -2981 +2981 @@
-hathe)
+hathe#have_v)
@@ -2983 +2983 @@
-bene)
+bene#be_v)
@@ -2985 +2985 @@
-intercepted)
+intercepted#intercept_v)
@@ -2990 +2990 @@
-of)
+of#of_prep)
@@ -2993 +2993 @@
-late)))
+late#late_adj1)))
@@ -3002 +3002 @@
-The)
+The#the_adj)
@@ -3004 +3004 @@
-prince)
+prince#prince_n)
@@ -3007 +3007 @@
-of)
+of#of_prep)
@@ -3010 +3010 @@
-Parma))))
+Parma#NA))))
@@ -3012 +3012 @@
-is)
+is#be_v)
@@ -3014 +3014 @@
-come)
+come#come_v)
@@ -3017 +3017 @@
-to)
+to#to_prep)
@@ -3020 +3020 @@
-Dunkerke))))
+Dunkerke#NA))))
@@ -3026 +3026 @@
-but)
+but#but_prep)
@@ -3029 +3029 @@
-vnknowen))
+vnknowen#unknown_adj))
@@ -3033 +3033 @@
-for)
+for#for_prep)
@@ -3036 +3036 @@
-what)
+what#what_pron)
@@ -3038 +3038 @@
-cause))))
+cause#cause_n))))
@@ -3046 +3046 @@
-suspected)
+suspected#suspect_v)
@@ -3049 +3049 @@
-for)
+for#for_prep)
@@ -3052 +3052 @@
-no)
+no#no_adj)
@@ -3054 +3054 @@
-good)
+good#good_adj)
@@ -3057 +3057 @@
-to)
+to#to_prep)
@@ -3060 +3060 @@
-vs)))))
+vs#us_pron)))))
@@ -3069 +3069 @@
-As)
+As#as_adv)
@@ -3072 +3072 @@
-yesterday)))
+yesterday#yesterday_adv)))
@@ -3075 +3075 @@
-the)
+the#the_adj)
@@ -3077 +3077 @@
-Quene))
+Quene#queen_n))
@@ -3080 +3080 @@
-going)
+going#go_v)
@@ -3083 +3083 @@
-into)
+into#into_prep)
@@ -3086 +3086 @@
-the)
+the#the_adj)
@@ -3088 +3088 @@
-gallery))))
+gallery#gallery_n))))
@@ -3090 +3090 @@
-made)
+made#make_v1)
@@ -3093 +3093 @@
-an)
+an#a_adj2)
@@ -3095 +3095 @@
-old)
+old#old_adj)
@@ -3097 +3097 @@
-knight)
+knight#knight_n)
@@ -3102 +3102 @@
-S=r=)
+S=r=#NA)
@@ -3104 +3104 @@
-Lewes)
+Lewes#NA)
@@ -3106 +3106 @@
-Dyves)))
+Dyves#NA)))
@@ -3114 +3114 @@
-And)
+And#and_conj1)
@@ -3117 +3117 @@
-thus))
+thus#thus_adv))
@@ -3120 +3120 @@
-with)
+with#with_prep)
@@ -3123 +3123 @@
-my)
+my#my_adj)
@@ -3126 +3126 @@
-very)
+very#very_adj)
@@ -3128 +3128 @@
-humble))
+humble#humble_adj))
@@ -3130 +3130 @@
-duty)
+duty#duty_n)
@@ -3134 +3134 @@
-etc.))))
+etc.#et-cetera_n))))
@@ -3140 +3140 @@
-London))
+London#NA))
@@ -3146 +3146 @@
-this)
+this#this_pron)
@@ -3148 +3148 @@
-xxiiij=th=)
+xxiiij=th=#NA)
@@ -3151 +3151 @@
-of)
+of#of_prep)
@@ -3154 +3154 @@
-November)))
+November#NA)))
