--- gcromw-1534-e1-h.psd.vert	2025-09-10 13:50:35
+++ gcromw-1534-e1-h.lem.vert	2025-09-10 13:50:45
@@ -15 +15 @@
-Most)
+Most#most_adj)
@@ -17 +17 @@
-dere))
+dere#dear_adj1))
@@ -19 +19 @@
-father))
+father#father_n))
@@ -24 +24 @@
-I))
+I#i_pron))
@@ -27 +27 @@
-humbly))
+humbly#humbly_adv))
@@ -29 +29 @@
-recomend)
+recomend#recommend_v1)
@@ -32 +32 @@
-me))
+me#me_pron1))
@@ -35 +35 @@
-unto)
+unto#unto_prep)
@@ -38 +38 @@
-you)))
+you#you_pron)))
@@ -46 +46 @@
-and)
+and#and_conj1)
@@ -51 +51 @@
-hertily))
+hertily#heartily_adv))
@@ -53 +53 @@
-beseche)
+beseche#beseech_v)
@@ -56 +56 @@
-you))
+you#you_pron))
@@ -59 +59 @@
-of)
+of#of_prep)
@@ -62 +62 @@
-yowr)
+yowr#your_pron)
@@ -64 +64 @@
-dayly)
+dayly#daily_adj)
@@ -66 +66 @@
-blessyng)
+blessyng#blessing_n)
@@ -74 +74 @@
-naturally))
+naturally#naturally_adv))
@@ -76 +76 @@
-bownden)
+bownden#bind_v)
@@ -79 +79 @@
-thayreunto)))
+thayreunto#thereunto_adv)))
@@ -85 +85 @@
-for)
+for#for_prep)
@@ -89 +89 @@
-the)
+the#the_adj)
@@ -91 +91 @@
-wiche))
+wiche#which_adj))
@@ -96 +96 @@
-and)
+and#and_conj1)
@@ -99 +99 @@
-other)
+other#other_adj)
@@ -101 +101 @@
-yowr)
+yowr#your_pron)
@@ -103 +103 @@
-manifowld)
+manifowld#moanful_adj)
@@ -105 +105 @@
-benefittes)
+benefittes#benefit_n)
@@ -109 +109 @@
-to)
+to#to_prep)
@@ -112 +112 @@
-me)))
+me#me_pron1)))
@@ -114 +114 @@
-colatyt))))))
+colatyt#collate_v))))))
@@ -124 +124 @@
-I))
+I#i_pron))
@@ -126 +126 @@
-am)
+am#be_v)
@@ -129 +129 @@
-and)
+and#and_conj1)
@@ -131 +131 @@
-schal@)
+schal@#shall_v)
@@ -133 +133 @@
-@be))
+@be#be_v))
@@ -136 +136 @@
-yowr)
+yowr#your_pron)
@@ -138 +138 @@
-daly)
+daly#daily_adj)
@@ -140 +140 @@
-bedman))
+bedman#beadsman_n))
@@ -146 +146 @@
-interely))
+interely#entirely_adj))
@@ -148 +148 @@
-desyryng)
+desyryng#desire_v)
@@ -151 +151 @@
-the)
+the#the_adj)
@@ -153 +153 @@
-contineweans)
+contineweans#continuance_n)
@@ -156 +156 @@
-of)
+of#of_prep)
@@ -159 +159 @@
-the)
+the#the_adj)
@@ -161 +161 @@
-same)))))
+same#same_adj)))))
@@ -166 +166 @@
-trustyng)
+trustyng#trust_v)
@@ -169 +169 @@
-soo))
+soo#so_adv))
@@ -172 +172 @@
-to)
+to#to_prep)
@@ -175 +175 @@
-accomplysse)
+accomplysse#accomplish_v)
@@ -177 +177 @@
-and)
+and#and_conj1)
@@ -179 +179 @@
-fulfyll))
+fulfyll#fulfil_v))
@@ -182 +182 @@
-yowr)
+yowr#your_pron)
@@ -184 +184 @@
-parentall)
+parentall#parental_adj)
@@ -186 +186 @@
-com~andments)
+com~andments#commandment_n)
@@ -189 +189 @@
-in)
+in#in_prep)
@@ -192 +192 @@
-the)
+the#the_adj)
@@ -194 +194 @@
-passage)
+passage#passage_n)
@@ -197 +197 @@
-of)
+of#of_prep)
@@ -200 +200 @@
-myne)
+myne#mine_adj)
@@ -202 +202 @@
-erudicion))))))))
+erudicion#erudition_n))))))))
@@ -207 +207 @@
-that)
+that#that_conj)
@@ -211 +211 @@
-yow))
+yow#you_pron))
@@ -216 +216 @@
-my)
+my#my_adj)
@@ -218 +218 @@
-good)
+good#good_adj)
@@ -220 +220 @@
-father))
+father#father_n))
@@ -224 +224 @@
-schall)
+schall#shall_v)
@@ -227 +227 @@
-tharewith))
+tharewith#therewith_adv))
@@ -229 +229 @@
-be)
+be#be_v)
@@ -232 +232 @@
-ryght)
+ryght#right_adv)
@@ -234 +234 @@
-welcontentyd)
+welcontentyd#well-contented_adj)
@@ -239 +239 @@
-by)
+by#by_prep)
@@ -242 +242 @@
-Gods)
+Gods#NA)
@@ -244 +244 @@
-helpe)
+helpe#help_n)
@@ -250 +250 @@
-the)
+the#the_adj)
@@ -252 +252 @@
-wiche))
+wiche#which_adj))
@@ -260 +260 @@
-with)
+with#with_prep)
@@ -263 +263 @@
-hys)
+hys#his_adj)
@@ -265 +265 @@
-grace)))
+grace#grace_n)))
@@ -268 +268 @@
-hee))
+hee#he_pron))
@@ -270 +270 @@
-send)
+send#send_v1)
@@ -273 +273 @@
-hus))))))))))
+hus#us_pron))))))))))
@@ -281 +281 @@
-Amen)
+Amen#amen_int)
@@ -289 +289 @@
-Frome)
+Frome#from_prep)
@@ -292 +292 @@
-the)
+the#the_adj)
@@ -294 +294 @@
-howse)
+howse#house_n1)
@@ -297 +297 @@
-of)
+of#of_prep)
@@ -300 +300 @@
-yowr)
+yowr#your_pron)
@@ -302 +302 @@
-bedman)
+bedman#beadsman_n)
@@ -305 +305 @@
-Mr.)
+Mr.#NA)
@@ -307 +307 @@
-Doctor)
+Doctor#NA)
@@ -309 +309 @@
-Lee))))))
+Lee#NA))))))
@@ -315 +315 @@
-thys)
+thys#this_pron)
@@ -317 +317 @@
-Ester)
+Ester#NA)
@@ -319 +319 @@
-day)
+day#NA)
@@ -322 +322 @@
-in)
+in#in_prep)
@@ -325 +325 @@
-the)
+the#the_adj)
@@ -327 +327 @@
-mornyng)))
+mornyng#morning_n)))
@@ -335 +335 @@
-By)
+By#by_prep)
@@ -338 +338 @@
-yowr)
+yowr#NA)
@@ -340 +340 @@
-vigelante)
+vigelante#vigilant_adj)
@@ -342 +342 @@
-sone)
+sone#son_n1)
@@ -345 +345 @@
-Gregori)
+Gregori#NA)
@@ -347 +347 @@
-Cromewell)))
+Cromewell#NA)))
@@ -363 +363 @@
-Right)
+Right#right_adv)
@@ -365 +365 @@
-worschypfull))
+worschypfull#worshipful_adj))
@@ -367 +367 @@
-father))
+father#father_n))
@@ -372 +372 @@
-I))
+I#i_pron))
@@ -374 +374 @@
-co~mend)
+co~mend#commend_v)
@@ -377 +377 @@
-me))
+me#me_pron1))
@@ -380 +380 @@
-un_to)
+un_to#unto_prep)
@@ -383 +383 @@
-you)))
+you#you_pron)))
@@ -388 +388 @@
-desyryng)
+desyryng#desire_v)
@@ -391 +391 @@
-you))
+you#you_pron))
@@ -394 +394 @@
-of)
+of#of_prep)
@@ -397 +397 @@
-youre)
+youre#your_pron)
@@ -399 +399 @@
-dayly)
+dayly#daily_adj)
@@ -401 +401 @@
-blessyng))))
+blessyng#blessing_n))))
@@ -406 +406 @@
-sartyfying)
+sartyfying#certify_v)
@@ -409 +409 @@
-you))
+you#you_pron))
@@ -412 +412 @@
-that)
+that#that_conj)
@@ -416 +416 @@
-I))
+I#i_pron))
@@ -418 +418 @@
-am)
+am#be_v)
@@ -421 +421 @@
-in)
+in#in_prep)
@@ -424 +424 @@
-good)
+good#good_adj)
@@ -426 +426 @@
-helth)))
+helth#health_n)))
@@ -431 +431 @@
-wyth)
+wyth#with_prep)
@@ -434 +434 @@
-my)
+my#my_adj)
@@ -436 +436 @@
-cosens)
+cosens#cousin_n)
@@ -439 +439 @@
-Bersfourd)
+Bersfourd#NA)
@@ -441 +441 @@
-and)
+and#and_conj1)
@@ -443 +443 @@
-Wellyfyd)))))
+Wellyfyd#NA)))))
@@ -449 +449 @@
-thanks))
+thanks#thank_n))
@@ -451 +451 @@
-be)
+be#be_v)
@@ -454 +454 @@
-unto)
+unto#unto_prep)
@@ -457 +457 @@
-God)
+God#NA)
@@ -460 +460 @@
-omnipotente)))))))
+omnipotente#omnipotent_adj)))))))
@@ -468 +468 @@
-and)
+and#and_conj1)
@@ -472 +472 @@
-apply)
+apply#apply_v)
@@ -475 +475 @@
-owre)
+owre#our_pron)
@@ -477 +477 @@
-boks))
+boks#book_n))
@@ -480 +480 @@
-dylygently))
+dylygently#diligently_adv))
@@ -485 +485 @@
-as)
+as#as_adv)
@@ -495 +495 @@
-shall)
+shall#shall_v)
@@ -497 +497 @@
-appere)
+appere#appear_v)
@@ -501 +501 @@
-I))
+I#i_pron))
@@ -503 +503 @@
-trust))
+trust#trust_v))
@@ -506 +506 @@
-to)
+to#to_prep)
@@ -510 +510 @@
-youre)
+youre#your_pron)
@@ -512 +512 @@
-worschyp))
+worschyp#worship_n))
@@ -515 +515 @@
-and)
+and#and_conj1)
@@ -518 +518 @@
-owre)
+owre#our_pron)
@@ -520 +520 @@
-proffyts))))))))
+proffyts#profit_n))))))))
@@ -529 +529 @@
-Father))
+Father#father_n))
@@ -534 +534 @@
-I))
+I#i_pron))
@@ -536 +536 @@
-besetch)
+besetch#beseech_v)
@@ -539 +539 @@
-you))
+you#you_pron))
@@ -543 +543 @@
-whan)
+whan#when_adv)
@@ -550 +550 @@
-ye))
+ye#ye_pron))
@@ -552 +552 @@
-mett)
+mett#meet_v)
@@ -555 +555 @@
-wyth)
+wyth#with_prep)
@@ -558 +558 @@
-the)
+the#the_adj)
@@ -561 +561 @@
-ryght)
+ryght#right_adv)
@@ -563 +563 @@
-honorable))
+honorable#honourable_adj))
@@ -565 +565 @@
-lorde)
+lorde#lord_n)
@@ -568 +568 @@
-of)
+of#of_prep)
@@ -571 +571 @@
-Oxforth))))))))
+Oxforth#NA))))))))
@@ -575 +575 @@
-to)
+to#to_prep)
@@ -577 +577 @@
-geue)
+geue#give_v)
@@ -580 +580 @@
-thanks))
+thanks#thank_n))
@@ -583 +583 @@
-un_to)
+un_to#unto_prep)
@@ -586 +586 @@
-hys)
+hys#his_adj)
@@ -588 +588 @@
-Lorchyp))))
+Lorchyp#lordship_n))))
@@ -596 +596 @@
-for)
+for#for_prep)
@@ -599 +599 @@
-whan)
+whan#when_adv)
@@ -606 +606 @@
-he))
+he#he_pron))
@@ -608 +608 @@
-came)
+came#come_v)
@@ -611 +611 @@
-to)
+to#to_prep)
@@ -614 +614 @@
-a)
+a#a_adj2)
@@ -616 +616 @@
-towne)
+towne#town_n)
@@ -619 +619 @@
-callyd)
+callyd#call_v)
@@ -625 +625 @@
-Yeldam))))))
+Yeldam#NA))))))
@@ -630 +630 @@
-to)
+to#to_prep)
@@ -634 +634 @@
-the)
+the#the_adj)
@@ -636 +636 @@
-parsons))))
+parsons#parson_n))))
@@ -640 +640 @@
-there)
+there#there_adv)
@@ -642 +642 @@
-of))
+of#of_prep))
@@ -644 +644 @@
-to)
+to#to_prep)
@@ -646 +646 @@
-hunte)
+hunte#hunt_v)
@@ -649 +649 @@
-the)
+the#the_adj)
@@ -651 +651 @@
-foxe))))))
+foxe#fox_n))))))
@@ -656 +656 @@
-he))
+he#he_pron))
@@ -658 +658 @@
-sente)
+sente#send_v1)
@@ -661 +661 @@
-for)
+for#for_prep)
@@ -665 +665 @@
-me))
+me#me_pron1))
@@ -668 +668 @@
-and)
+and#and_conj1)
@@ -671 +671 @@
-my)
+my#my_adj)
@@ -673 +673 @@
-cossyns)))))
+cossyns#cousin_n)))))
@@ -681 +681 @@
-and)
+and#and_conj1)
@@ -685 +685 @@
-mad)
+mad#make_v1)
@@ -688 +688 @@
-us))
+us#us_pron))
@@ -691 +691 @@
-good)
+good#good_adj)
@@ -693 +693 @@
-schere))
+schere#cheer_n1))
@@ -701 +701 @@
-and)
+and#and_conj1)
@@ -705 +705 @@
-lett)
+lett#let_v1)
@@ -709 +709 @@
-us))
+us#us_pron))
@@ -711 +711 @@
-see)
+see#see_v)
@@ -714 +714 @@
-schuch)
+schuch#such_adj)
@@ -716 +716 @@
-game)
+game#game_n)
@@ -719 +719 @@
-and)
+and#and_conj1)
@@ -722 +722 @@
-plesure)))
+plesure#pleasure_n)))
@@ -725 +725 @@
-as)
+as#as_adv)
@@ -736 +736 @@
-I))
+I#i_pron))
@@ -739 +739 @@
-never))
+never#never_adv))
@@ -741 +741 @@
-sawe)
+sawe#see_v)
@@ -746 +746 @@
-in)
+in#in_prep)
@@ -749 +749 @@
-my)
+my#my_adj)
@@ -751 +751 @@
-lyfe))))))))
+lyfe#life_n))))))))
@@ -760 +760 @@
-more)
+more#more_adj)
@@ -762 +762 @@
-over))
+over#over_adv))
@@ -765 +765 @@
-father))
+father#father_n))
@@ -770 +770 @@
-I))
+I#i_pron))
@@ -772 +772 @@
-besetch)
+besetch#beseech_v)
@@ -775 +775 @@
-you))
+you#you_pron))
@@ -778 +778 @@
-to)
+to#to_prep)
@@ -780 +780 @@
-geve)
+geve#give_v)
@@ -783 +783 @@
-thanks))
+thanks#thank_n))
@@ -786 +786 @@
-to)
+to#to_prep)
@@ -789 +789 @@
-the)
+the#the_adj)
@@ -791 +791 @@
-for_sayde)
+for_sayde#foresaid_adj)
@@ -793 +793 @@
-parson)
+parson#parson_n)
@@ -796 +796 @@
-of)
+of#of_prep)
@@ -799 +799 @@
-Yeldam)))
+Yeldam#NA)))
@@ -805 +805 @@
-which))
+which#which_adj))
@@ -814 +814 @@
-sens)
+sens#since_adv)
@@ -821 +821 @@
-I))
+I#i_pron))
@@ -823 +823 @@
-came)
+came#come_v)
@@ -826 +826 @@
-in)
+in#in_prep)
@@ -829 +829 @@
-to)
+to#to_prep)
@@ -832 +832 @@
-the)
+the#the_adj)
@@ -834 +834 @@
-cuntry)))))))
+cuntry#country_n)))))))
@@ -836 +836 @@
-hath)
+hath#have_v)
@@ -839 +839 @@
-dyvers)
+dyvers#divers_adj)
@@ -841 +841 @@
-tymys))
+tymys#time_n))
@@ -843 +843 @@
-sente)
+sente#send_v1)
@@ -847 +847 @@
-for)
+for#for_prep)
@@ -850 +850 @@
-me)))
+me#me_pron1)))
@@ -853 +853 @@
-and)
+and#and_conj1)
@@ -856 +856 @@
-for)
+for#for_prep)
@@ -859 +859 @@
-my)
+my#my_adj)
@@ -861 +861 @@
-cossyns))))))
+cossyns#cousin_n))))))
@@ -864 +864 @@
-and)
+and#and_conj1)
@@ -869 +869 @@
-mad)
+mad#make_v1)
@@ -872 +872 @@
-us))
+us#us_pron))
@@ -875 +875 @@
-hygh)
+hygh#high_adj)
@@ -877 +877 @@
-schere))))
+schere#cheer_n1))))
@@ -882 +882 @@
-and)
+and#and_conj1)
@@ -887 +887 @@
-schewyd)
+schewyd#show_v)
@@ -890 +890 @@
-us))
+us#us_pron))
@@ -893 +893 @@
-gret)
+gret#great_adj)
@@ -895 +895 @@
-plesure)))))))))
+plesure#pleasure_n)))))))))
@@ -904 +904 @@
-For)
+For#for_prep)
@@ -907 +907 @@
-all)
+all#all_adj)
@@ -909 +909 @@
-other)
+other#other_adj)
@@ -911 +911 @@
-thyngs)
+thyngs#thing_n1)
@@ -914 +914 @@
-consarnyng)
+consarnyng#concerning_prep)
@@ -917 +917 @@
-my)
+my#my_adj)
@@ -919 +919 @@
-rayment)))))
+rayment#raiment_n)))))
@@ -925 +925 @@
-I))
+I#i_pron))
@@ -927 +927 @@
-beseche)
+beseche#beseech_v)
@@ -930 +930 @@
-you)))
+you#you_pron)))
@@ -932 +932 @@
-geve)
+geve#give_v)
@@ -935 +935 @@
-credens))
+credens#credence_n))
@@ -938 +938 @@
-to)
+to#to_prep)
@@ -941 +941 @@
-my)
+my#my_adj)
@@ -944 +944 @@
-synguler)
+synguler#singular_adj)
@@ -946 +946 @@
-good))
+good#good_adj))
@@ -948 +948 @@
-frende)
+frende#friend_n)
@@ -951 +951 @@
-Mayster)
+Mayster#NA)
@@ -953 +953 @@
-Doctor)
+Doctor#NA)
@@ -955 +955 @@
-Lee))))
+Lee#NA))))
@@ -964 +964 @@
-Thus))
+Thus#thus_adv))
@@ -967 +967 @@
-Jhesu))
+Jhesu#NA))
@@ -969 +969 @@
-have)
+have#have_v)
@@ -972 +972 @@
-you))
+you#you_pron))
@@ -975 +975 @@
-in)
+in#in_prep)
@@ -978 +978 @@
-hys)
+hys#his_adj)
@@ -980 +980 @@
-kepyng)))
+kepyng#keeping_n)))
@@ -988 +988 @@
-From)
+From#from_prep)
@@ -991 +991 @@
-Topsfyld)))
+Topsfyld#NA)))
@@ -997 +997 @@
-the)
+the#the_adj)
@@ -999 +999 @@
-xvii)
+xvii#NA)
@@ -1001 +1001 @@
-day)
+day#day_n)
@@ -1004 +1004 @@
-of)
+of#of_prep)
@@ -1007 +1007 @@
-October)))
+October#NA)))
@@ -1015 +1015 @@
-By)
+By#by_prep)
@@ -1018 +1018 @@
-your)
+your#your_pron)
@@ -1020 +1020 @@
-lowly)
+lowly#lowly_adj)
@@ -1022 +1022 @@
-sone)
+sone#son_n1)
@@ -1025 +1025 @@
-Gregory)
+Gregory#NA)
@@ -1027 +1027 @@
-Crumwell)))
+Crumwell#NA)))
@@ -1043 +1043 @@
-Ryght)
+Ryght#right_adv)
@@ -1045 +1045 @@
-worshypfulle))
+worshypfulle#worshipful_adj))
@@ -1047 +1047 @@
-father))
+father#father_n))
@@ -1052 +1052 @@
-as)
+as#as_adv)
@@ -1064 +1064 @@
-harte))
+harte#heart_n))
@@ -1066 +1066 @@
-canne)
+canne#can_v1)
@@ -1068 +1068 @@
-thynke))
+thynke#think_v2))
@@ -1071 +1071 @@
-or)
+or#or_conj2)
@@ -1077 +1077 @@
-tong))
+tong#tongue_n))
@@ -1079 +1079 @@
-canne)
+canne#can_v1)
@@ -1081 +1081 @@
-tell))))))
+tell#tell_v))))))
@@ -1086 +1086 @@
-I))
+I#i_pron))
@@ -1089 +1089 @@
-hartely))
+hartely#heartily_adv))
@@ -1091 +1091 @@
-co~mende)
+co~mende#commend_v)
@@ -1094 +1094 @@
-me))
+me#me_pron1))
@@ -1097 +1097 @@
-vn_to)
+vn_to#unto_prep)
@@ -1100 +1100 @@
-you)))
+you#you_pron)))
@@ -1105 +1105 @@
-beyng)
+beyng#be_v)
@@ -1108 +1108 @@
-always))
+always#always_adv))
@@ -1111 +1111 @@
-desyrus)
+desyrus#desirous_adj)
@@ -1114 +1114 @@
-to)
+to#to_prep)
@@ -1116 +1116 @@
-here)
+here#hear_v)
@@ -1119 +1119 @@
-of)
+of#of_prep)
@@ -1122 +1122 @@
-your)
+your#your_pron)
@@ -1124 +1124 @@
-prosperus)
+prosperus#prosperous_adj)
@@ -1126 +1126 @@
-helth)
+helth#health_n)
@@ -1129 +1129 @@
-and)
+and#and_conj1)
@@ -1132 +1132 @@
-welfare)))
+welfare#welfare_n)))
@@ -1138 +1138 @@
-whych))
+whych#which_adj))
@@ -1144 +1144 @@
-I))
+I#i_pron))
@@ -1146 +1146 @@
-pray)
+pray#pray_v)
@@ -1155 +1155 @@
-Jhsu))
+Jhsu#NA))
@@ -1158 +1158 @@
-long))
+long#long_adj1))
@@ -1160 +1160 @@
-contynwe)
+contynwe#continue_v)
@@ -1164 +1164 @@
-to)
+to#to_prep)
@@ -1167 +1167 @@
-his)
+his#his_adj)
@@ -1169 +1169 @@
-pleasure)))
+pleasure#pleasure_n)))
@@ -1174 +1174 @@
-and)
+and#and_conj1)
@@ -1177 +1177 @@
-to)
+to#to_prep)
@@ -1181 +1181 @@
-your)
+your#your_pron)
@@ -1184 +1184 @@
-moste)
+moste#most_adj)
@@ -1186 +1186 @@
-gentillyste))
+gentillyste#gentle_adj))
@@ -1188 +1188 @@
-hartis))
+hartis#heart_n))
@@ -1190 +1190 @@
-desyre))))))))))))))
+desyre#desire_n))))))))))))))
@@ -1194 +1194 @@
-&c.))
+&c.#et-cetera_n))
@@ -1201 +1201 @@
-The)
+The#the_adj)
@@ -1203 +1203 @@
-cause)
+cause#cause_n)
@@ -1206 +1206 @@
-of)
+of#of_prep)
@@ -1209 +1209 @@
-my)
+my#my_adj)
@@ -1212 +1212 @@
-wrytynge)
+wrytynge#write_v)
@@ -1215 +1215 @@
-vn_to)
+vn_to#unto_prep)
@@ -1218 +1218 @@
-you)))
+you#you_pron)))
@@ -1221 +1221 @@
-at)
+at#at_prep)
@@ -1224 +1224 @@
-thys)
+thys#this_pron)
@@ -1226 +1226 @@
-tyme)))))))
+tyme#time_n)))))))
@@ -1228 +1228 @@
-ys)
+ys#be_v)
@@ -1234 +1234 @@
-to)
+to#to_prep)
@@ -1236 +1236 @@
-desyre)
+desyre#desire_v)
@@ -1239 +1239 @@
-you))
+you#you_pron))
@@ -1242 +1242 @@
-to)
+to#to_prep)
@@ -1244 +1244 @@
-send)
+send#send_v1)
@@ -1247 +1247 @@
-me))
+me#me_pron1))
@@ -1250 +1250 @@
-your)
+your#your_pron)
@@ -1252 +1252 @@
-blessyng)
+blessyng#blessing_n)
@@ -1258 +1258 @@
-which))
+which#which_adj))
@@ -1265 +1265 @@
-ys)
+ys#be_v)
@@ -1268 +1268 @@
-more)
+more#more_adj)
@@ -1270 +1270 @@
-treasure)
+treasure#treasure_n)
@@ -1273 +1273 @@
-unto)
+unto#unto_prep)
@@ -1276 +1276 @@
-me)))
+me#me_pron1)))
@@ -1279 +1279 @@
-then)
+then#than_conj)
@@ -1282 +1282 @@
-all)
+all#all_adj)
@@ -1284 +1284 @@
-the)
+the#the_adj)
@@ -1286 +1286 @@
-abundance)
+abundance#abundance_n)
@@ -1289 +1289 @@
-of)
+of#of_prep)
@@ -1292 +1292 @@
-worldly)
+worldly#worldly_adj)
@@ -1294 +1294 @@
-goods))))))))))))
+goods#good_adj))))))))))))
@@ -1299 +1299 @@
-sertyfyinge)
+sertyfyinge#certify_v)
@@ -1302 +1302 @@
-you))
+you#you_pron))
@@ -1305 +1305 @@
-that)
+that#that_conj)
@@ -1310 +1310 @@
-I))
+I#i_pron))
@@ -1312 +1312 @@
-was)
+was#be_v)
@@ -1315 +1315 @@
-in)
+in#in_prep)
@@ -1318 +1318 @@
-good)
+good#good_adj)
@@ -1320 +1320 @@
-helth)))
+helth#health_n)))
@@ -1323 +1323 @@
-at)
+at#at_prep)
@@ -1326 +1326 @@
-the)
+the#the_adj)
@@ -1328 +1328 @@
-makynge)
+makynge#making_n1)
@@ -1331 +1331 @@
-of)
+of#of_prep)
@@ -1334 +1334 @@
-thys)
+thys#this_pron)
@@ -1336 +1336 @@
-letter)))))
+letter#letter_n1)))))
@@ -1341 +1341 @@
-thankyd)
+thankyd#thank_v)
@@ -1343 +1343 @@
-be)
+be#be_v)
@@ -1346 +1346 @@
-God)
+God#NA)
@@ -1349 +1349 @@
-omnipotent)))))
+omnipotent#omnipotent_adj)))))
@@ -1354 +1354 @@
-and)
+and#and_conj1)
@@ -1359 +1359 @@
-doo)
+doo#do_v)
@@ -1361 +1361 @@
-apply)
+apply#apply_v)
@@ -1364 +1364 @@
-my)
+my#my_adj)
@@ -1366 +1366 @@
-boke))
+boke#book_n))
@@ -1369 +1369 @@
-deligently))
+deligently#diligently_adv))
@@ -1374 +1374 @@
-as)
+as#as_adv)
@@ -1386 +1386 @@
-I))
+I#i_pron))
@@ -1388 +1388 @@
-truste)
+truste#trust_v)
@@ -1391 +1391 @@
-in)
+in#in_prep)
@@ -1394 +1394 @@
-God))))
+God#NA))))
@@ -1396 +1396 @@
-shall)
+shall#shall_v)
@@ -1399 +1399 @@
-in)
+in#in_prep)
@@ -1402 +1402 @@
-process)
+process#process_n)
@@ -1405 +1405 @@
-of)
+of#of_prep)
@@ -1408 +1408 @@
-tyme)))))
+tyme#time_n)))))
@@ -1410 +1410 @@
-apere)
+apere#appear_v)
@@ -1414 +1414 @@
-to)
+to#to_prep)
@@ -1417 +1417 @@
-my)
+my#my_adj)
@@ -1419 +1419 @@
-proffyte)))
+proffyte#profit_n)))
@@ -1424 +1424 @@
-and)
+and#and_conj1)
@@ -1427 +1427 @@
-to)
+to#to_prep)
@@ -1430 +1430 @@
-your)
+your#your_pron)
@@ -1432 +1432 @@
-contentacion)
+contentacion#contentation_n)
@@ -1435 +1435 @@
-and)
+and#and_conj1)
@@ -1438 +1438 @@
-worship)))))))))))))))
+worship#worship_n)))))))))))))))
@@ -1447 +1447 @@
-I))
+I#i_pron))
@@ -1449 +1449 @@
-have)
+have#have_v)
@@ -1451 +1451 @@
-recevid)
+recevid#receive_v)
@@ -1454 +1454 @@
-the)
+the#the_adj)
@@ -1456 +1456 @@
-tokyne)
+tokyne#token_n)
@@ -1461 +1461 @@
-that)
+that#that_conj)
@@ -1467 +1467 @@
-you))
+you#you_pron))
@@ -1469 +1469 @@
-sent)
+sent#send_v1)
@@ -1472 +1472 @@
-by)
+by#by_prep)
@@ -1475 +1475 @@
-Master)
+Master#NA)
@@ -1477 +1477 @@
-doctor)
+doctor#NA)
@@ -1479 +1479 @@
-Bekynsall)
+Bekynsall#NA)
@@ -1485 +1485 @@
-unto)
+unto#unto_prep)
@@ -1488 +1488 @@
-whome)))
+whome#whom_pron)))
@@ -1494 +1494 @@
-I))
+I#i_pron))
@@ -1496 +1496 @@
-pray)
+pray#pray_v)
@@ -1505 +1505 @@
-you))
+you#you_pron))
@@ -1507 +1507 @@
-gyve)
+gyve#give_v)
@@ -1510 +1510 @@
-thanks)))))))))))
+thanks#thank_n)))))))))))
@@ -1518 +1518 @@
-for)
+for#for_prep)
@@ -1523 +1523 @@
-at)
+at#at_prep)
@@ -1526 +1526 @@
-his)
+his#his_adj)
@@ -1529 +1529 @@
-beyng)
+beyng#be_v)
@@ -1532 +1532 @@
-now))
+now#now_adv))
@@ -1535 +1535 @@
-at)
+at#at_prep)
@@ -1538 +1538 @@
-Topsfylde))))))
+Topsfylde#NA))))))
@@ -1540 +1540 @@
-bothe)
+bothe#both_pron)
@@ -1542 +1542 @@
-mad)
+mad#make_v1)
@@ -1546 +1546 @@
-me))
+me#me_pron1))
@@ -1551 +1551 @@
-gret)
+gret#great_adj)
@@ -1553 +1553 @@
-cheare))
+cheare#cheer_n1))
@@ -1556 +1556 @@
-and)
+and#and_conj1)
@@ -1559 +1559 @@
-all)
+all#all_adj)
@@ -1561 +1561 @@
-my)
+my#my_adj)
@@ -1563 +1563 @@
-fellos)))
+fellos#fellow_n)))
@@ -1571 +1571 @@
-and)
+and#and_conj1)
@@ -1575 +1575 @@
-gave)
+gave#give_v)
@@ -1578 +1578 @@
-me))
+me#me_pron1))
@@ -1581 +1581 @@
-a)
+a#a_adj2)
@@ -1583 +1583 @@
-crone)
+crone#crown_n)
@@ -1591 +1591 @@
-to)
+to#to_prep)
@@ -1593 +1593 @@
-spende))))
+spende#spend_v1))))
@@ -1599 +1599 @@
-Wherfore))
+Wherfore#wherefore_adv))
@@ -1608 +1608 @@
-I))
+I#i_pron))
@@ -1610 +1610 @@
-beseche)
+beseche#beseech_v)
@@ -1613 +1613 @@
-you)))
+you#you_pron)))
@@ -1616 +1616 @@
-father))
+father#father_n))
@@ -1620 +1620 @@
-have)
+have#have_v)
@@ -1623 +1623 @@
-hym))
+hym#him_pron))
@@ -1626 +1626 @@
-in)
+in#in_prep)
@@ -1629 +1629 @@
-your)
+your#your_pron)
@@ -1631 +1631 @@
-remembrance)))))
+remembrance#remembrance_n)))))
@@ -1640 +1640 @@
-Thus))
+Thus#thus_adv))
@@ -1643 +1643 @@
-Jhesu))
+Jhesu#NA))
@@ -1645 +1645 @@
-preserve)
+preserve#preserve_v)
@@ -1648 +1648 @@
-you))
+you#you_pron))
@@ -1651 +1651 @@
-in)
+in#in_prep)
@@ -1654 +1654 @@
-his)
+his#his_adj)
@@ -1656 +1656 @@
-goodnes)))
+goodnes#goodness_n)))
@@ -1664 +1664 @@
-Frome)
+Frome#from_prep)
@@ -1667 +1667 @@
-Topsfylde)))
+Topsfylde#NA)))
@@ -1673 +1673 @@
-the)
+the#the_adj)
@@ -1675 +1675 @@
-xxv)
+xxv#NA)
@@ -1677 +1677 @@
-day)
+day#day_n)
@@ -1680 +1680 @@
-of)
+of#of_prep)
@@ -1683 +1683 @@
-October)))
+October#NA)))
@@ -1691 +1691 @@
-By)
+By#by_prep)
@@ -1694 +1694 @@
-your)
+your#your_pron)
@@ -1696 +1696 @@
-lowly)
+lowly#lowly_adj)
@@ -1698 +1698 @@
-sone)
+sone#son_n1)
@@ -1701 +1701 @@
-Gregori)
+Gregori#NA)
@@ -1703 +1703 @@
-Crumwell)))
+Crumwell#NA)))
