--- iplumpt-1507-e1-h.psd.vert	2025-09-10 13:50:35
+++ iplumpt-1507-e1-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-To)
+To#to_prep)
@@ -17 +17 @@
-Sir)
+Sir#NA)
@@ -19 +19 @@
-Robart)
+Robart#NA)
@@ -21 +21 @@
-Plompton)
+Plompton#NA)
@@ -26 +26 @@
-kt.))))
+kt.#knight_n))))
@@ -28 +28 @@
-be)
+be#be_v)
@@ -31 +31 @@
-thes)
+thes#this_pron)
@@ -33 +33 @@
-letter))
+letter#letter_n1))
@@ -35 +35 @@
-delivered)
+delivered#deliver_v1)
@@ -44 +44 @@
-Sir))
+Sir#sir_n))
@@ -49 +49 @@
-in)
+in#in_prep)
@@ -52 +52 @@
-the)
+the#the_adj)
@@ -55 +55 @@
-most)
+most#most_adj)
@@ -57 +57 @@
-hartyest))
+hartyest#hearty_adj))
@@ -59 +59 @@
-wyse)
+wyse#wise_n1)
@@ -64 +64 @@
-that)
+that#that_conj)
@@ -70 +70 @@
-I))
+I#i_pron))
@@ -72 +72 @@
-can)
+can#can_v1)
@@ -79 +79 @@
-I))
+I#i_pron))
@@ -81 +81 @@
-recomend)
+recomend#recommend_v1)
@@ -84 +84 @@
-me))
+me#me_pron1))
@@ -87 +87 @@
-unto)
+unto#unto_prep)
@@ -90 +90 @@
-you)))
+you#you_pron)))
@@ -99 +99 @@
-Sir))
+Sir#sir_n))
@@ -104 +104 @@
-I))
+I#i_pron))
@@ -106 +106 @@
-have)
+have#have_v)
@@ -108 +108 @@
-sent)
+sent#send_v1)
@@ -111 +111 @@
-to)
+to#to_prep)
@@ -114 +114 @@
-Wright)
+Wright#NA)
@@ -117 +117 @@
-of)
+of#of_prep)
@@ -120 +120 @@
-Idell)))))
+Idell#NA)))))
@@ -123 +123 @@
-for)
+for#for_prep)
@@ -126 +126 @@
-the)
+the#the_adj)
@@ -128 +128 @@
-money)
+money#money_n)
@@ -133 +133 @@
-that)
+that#that_conj)
@@ -139 +139 @@
-he))
+he#he_pron))
@@ -141 +141 @@
-promyst)
+promyst#promise_v)
@@ -144 +144 @@
-you))))))
+you#you_pron))))))
@@ -152 +152 @@
-and)
+and#and_conj1)
@@ -155 +155 @@
-he))
+he#he_pron))
@@ -157 +157 @@
-saith)
+saith#say_v1)
@@ -164 +164 @@
-he))
+he#he_pron))
@@ -166 +166 @@
-hath)
+hath#have_v)
@@ -169 +169 @@
-it))
+it#it_pron))
@@ -171 +171 @@
-not)
+not#not_adv)
@@ -174 +174 @@
-to)
+to#to_prep)
@@ -176 +176 @@
-len))))
+len#lend_v2))))
@@ -184 +184 @@
-and)
+and#and_conj1)
@@ -188 +188 @@
-makes)
+makes#make_v1)
@@ -191 +191 @@
-choses)))
+choses#chose_n)))
@@ -197 +197 @@
-and)
+and#and_conj1)
@@ -200 +200 @@
-so))
+so#so_adv))
@@ -203 +203 @@
-I))
+I#i_pron))
@@ -205 +205 @@
-can)
+can#can_v1)
@@ -207 +207 @@
-get)
+get#get_v)
@@ -210 +210 @@
-none))
+none#none_pron))
@@ -213 +213 @@
-nowhere))
+nowhere#nowhere_adv))
@@ -221 +221 @@
-And)
+And#and_conj1)
@@ -224 +224 @@
-as)
+as#as_adv)
@@ -227 +227 @@
-for)
+for#for_prep)
@@ -230 +230 @@
-wood))))
+wood#wood_n1))))
@@ -235 +235 @@
-ther))
+ther#there_adv))
@@ -237 +237 @@
-is)
+is#be_v)
@@ -240 +240 @@
-none)
+none#none_pron)
@@ -245 +245 @@
-that)
+that#that_conj)
@@ -250 +250 @@
-will)
+will#will_v1)
@@ -252 +252 @@
-bey))))
+bey#buy_v))))
@@ -260 +260 @@
-for)
+for#for_prep)
@@ -263 +263 @@
-they))
+they#they_pron))
@@ -265 +265 @@
-know)
+know#know_v)
@@ -272 +272 @@
-ye))
+ye#ye_pron))
@@ -274 +274 @@
-want)
+want#want_v)
@@ -277 +277 @@
-money))))
+money#money_n))))
@@ -285 +285 @@
-and)
+and#and_conj1)
@@ -288 +288 @@
-without)
+without#without_adv)
@@ -295 +295 @@
-they))
+they#they_pron))
@@ -297 +297 @@
-myght)
+myght#may_v1)
@@ -299 +299 @@
-have)
+have#have_v)
@@ -302 +302 @@
-it))
+it#it_pron))
@@ -305 +305 @@
-halfe)
+halfe#half_adj)
@@ -307 +307 @@
-for)
+for#for_prep)
@@ -310 +310 @@
-nought))))))
+nought#nought_pron))))))
@@ -315 +315 @@
-they))
+they#they_pron))
@@ -317 +317 @@
-will)
+will#will_v1)
@@ -319 +319 @@
-bey)
+bey#buy_v)
@@ -322 +322 @@
-none))
+none#none_pron))
@@ -330 +330 @@
-for)
+for#for_prep)
@@ -334 +334 @@
-your)
+your#your_pron)
@@ -336 +336 @@
-son)
+son#son_n1)
@@ -341 +341 @@
-William)
+William#NA)
@@ -343 +343 @@
-Plompton)))
+Plompton#NA)))
@@ -348 +348 @@
-and)
+and#and_conj1)
@@ -351 +351 @@
-Thomas)
+Thomas#NA)
@@ -353 +353 @@
-Bickerdyke))))
+Bickerdyke#NA))))
@@ -355 +355 @@
-hath)
+hath#have_v)
@@ -357 +357 @@
-bene)
+bene#be_v)
@@ -360 +360 @@
-every)
+every#every_adj)
@@ -362 +362 @@
-day))
+day#day_n))
@@ -365 +365 @@
-at)
+at#at_prep)
@@ -368 +368 @@
-wood)))
+wood#wood_n1)))
@@ -371 +371 @@
-sence)
+sence#since_adv)
@@ -378 +378 @@
-ye))
+ye#ye_pron))
@@ -380 +380 @@
-went))))
+went#go_v))))
@@ -388 +388 @@
-and)
+and#and_conj1)
@@ -391 +391 @@
-they))
+they#they_pron))
@@ -393 +393 @@
-can)
+can#can_v1)
@@ -395 +395 @@
-get)
+get#get_v)
@@ -398 +398 @@
-no)
+no#no_adj)
@@ -400 +400 @@
-money)
+money#money_n)
@@ -403 +403 @@
-for)
+for#for_prep)
@@ -406 +406 @@
-nothing))))
+nothing#nothing_pron))))
@@ -416 +416 @@
-for)
+for#for_prep)
@@ -419 +419 @@
-tha))
+tha#they_pron))
@@ -421 +421 @@
-will)
+will#will_v1)
@@ -423 +423 @@
-bey)
+bey#buy_v)
@@ -426 +426 @@
-none))
+none#none_pron))
@@ -429 +429 @@
-without)
+without#without_adv)
@@ -436 +436 @@
-they))
+they#they_pron))
@@ -438 +438 @@
-have)
+have#have_v)
@@ -441 +441 @@
-tymmer)
+tymmer#timber_n1)
@@ -443 +443 @@
-tres)))))
+tres#tree_n)))))
@@ -451 +451 @@
-and)
+and#and_conj1)
@@ -455 +455 @@
-will)
+will#will_v1)
@@ -457 +457 @@
-give)
+give#give_v)
@@ -460 +460 @@
-nothinge))
+nothinge#nothing_pron))
@@ -463 +463 @@
-for)
+for#for_prep)
@@ -466 +466 @@
-them)))
+them#them_pron)))
@@ -475 +475 @@
-and)
+and#and_conj1)
@@ -478 +478 @@
-so))
+so#so_adv))
@@ -480 +480 @@
-shall)
+shall#shall_v)
@@ -483 +483 @@
-your)
+your#your_pron)
@@ -485 +485 @@
-wood))
+wood#wood_n1))
@@ -487 +487 @@
-be)
+be#be_v)
@@ -489 +489 @@
-distroyed))
+distroyed#destroy_v))
@@ -492 +492 @@
-and)
+and#and_conj1)
@@ -497 +497 @@
-get)
+get#get_v)
@@ -500 +500 @@
-nought))
+nought#nought_pron))
@@ -503 +503 @@
-for)
+for#for_prep)
@@ -506 +506 @@
-it)))))
+it#it_pron)))))
@@ -515 +515 @@
-Sir))
+Sir#sir_n))
@@ -520 +520 @@
-I))
+I#i_pron))
@@ -522 +522 @@
-told)
+told#tell_v)
@@ -525 +525 @@
-you))
+you#you_pron))
@@ -528 +528 @@
-this))
+this#this_pron))
@@ -531 +531 @@
-or)
+or#or_adv1)
@@ -538 +538 @@
-ye))
+ye#ye_pron))
@@ -540 +540 @@
-went))))
+went#go_v))))
@@ -548 +548 @@
-but)
+but#but_prep)
@@ -551 +551 @@
-ye))
+ye#ye_pron))
@@ -553 +553 @@
-wold)
+wold#will_v1)
@@ -555 +555 @@
-not)
+not#not_adv)
@@ -557 +557 @@
-beleve)
+beleve#believe_v)
@@ -560 +560 @@
-me))
+me#me_pron1))
@@ -569 +569 @@
-Sir))
+Sir#sir_n))
@@ -574 +574 @@
-I))
+I#i_pron))
@@ -576 +576 @@
-have)
+have#have_v)
@@ -578 +578 @@
-taken)
+taken#take_v)
@@ -581 +581 @@
-of)
+of#of_prep)
@@ -584 +584 @@
-your)
+your#your_pron)
@@ -586 +586 @@
-tymmer)))
+tymmer#timber_n1)))
@@ -590 +590 @@
-as)
+as#as_adv)
@@ -592 +592 @@
-much))
+much#much_adj))
@@ -595 +595 @@
-as)
+as#as_adv)
@@ -604 +604 @@
-I))
+I#i_pron))
@@ -606 +606 @@
-can)
+can#can_v1)
@@ -608 +608 @@
-get)
+get#get_v)
@@ -611 +611 @@
-of)
+of#of_prep)
@@ -618 +618 @@
-or)
+or#or_adv1)
@@ -621 +621 @@
-Whitsonday)
+Whitsonday#NA)
@@ -623 +623 @@
-farme)))
+farme#farm_n2)))
@@ -626 +626 @@
-forehand))
+forehand#forehand_adj))
@@ -634 +634 @@
-and)
+and#and_conj1)
@@ -637 +637 @@
-that))
+that#that_pron))
@@ -639 +639 @@
-is)
+is#be_v)
@@ -642 +642 @@
-but)
+but#but_prep)
@@ -644 +644 @@
-litle)
+litle#little_adj)
@@ -652 +652 @@
-to)
+to#to_prep)
@@ -654 +654 @@
-do)
+do#do_v)
@@ -657 +657 @@
-you))
+you#you_pron))
@@ -660 +660 @@
-any)
+any#any_adj)
@@ -662 +662 @@
-good)))))
+good#good_adj)))))
@@ -670 +670 @@
-for)
+for#for_prep)
@@ -673 +673 @@
-ther))
+ther#there_adv))
@@ -675 +675 @@
-is)
+is#be_v)
@@ -678 +678 @@
-but)
+but#but_prep)
@@ -680 +680 @@
-some)
+some#some_pron)
@@ -685 +685 @@
-that)
+that#that_conj)
@@ -690 +690 @@
-will)
+will#will_v1)
@@ -692 +692 @@
-len)
+len#lend_v2)
@@ -696 +696 @@
-so)
+so#so_adv)
@@ -698 +698 @@
-long)))
+long#long_adj1)))
@@ -701 +701 @@
-afor)
+afor#afore_adv)
@@ -704 +704 @@
-the)
+the#the_adj)
@@ -706 +706 @@
-tyme))))))
+tyme#time_n))))))
@@ -714 +714 @@
-And)
+And#and_conj1)
@@ -717 +717 @@
-your)
+your#your_pron)
@@ -719 +719 @@
-Lenten)
+Lenten#lenten_n)
@@ -721 +721 @@
-stoufe))
+stoufe#stuff_n1))
@@ -723 +723 @@
-is)
+is#be_v)
@@ -731 +731 @@
-to)
+to#to_prep)
@@ -733 +733 @@
-bey)))
+bey#buy_v)))
@@ -741 +741 @@
-and)
+and#and_conj1)
@@ -744 +744 @@
-I))
+I#i_pron))
@@ -746 +746 @@
-wote)
+wote#wit_v1)
@@ -748 +748 @@
-not)
+not#not_adv)
@@ -752 +752 @@
-what))
+what#what_pron))
@@ -759 +759 @@
-to)
+to#to_prep)
@@ -761 +761 @@
-do)))
+do#do_v)))
@@ -767 +767 @@
-God))
+God#NA))
@@ -769 +769 @@
-wote))
+wote#wit_v1))
@@ -777 +777 @@
-for)
+for#for_prep)
@@ -780 +780 @@
-I))
+I#i_pron))
@@ -782 +782 @@
-am)
+am#be_v)
@@ -785 +785 @@
-ever))
+ever#ever_adv))
@@ -787 +787 @@
-left)
+left#leave_v1)
@@ -790 +790 @@
-of)
+of#of_prep)
@@ -793 +793 @@
-thes)
+thes#this_pron)
@@ -795 +795 @@
-fachion)))
+fachion#fashion_n)))
@@ -804 +804 @@
-Sir))
+Sir#sir_n))
@@ -809 +809 @@
-ther))
+ther#there_adv))
@@ -811 +811 @@
-is)
+is#be_v)
@@ -814 +814 @@
-land)
+land#land_n1)
@@ -817 +817 @@
-in)
+in#in_prep)
@@ -820 +820 @@
-Rybston)
+Rybston#NA)
@@ -822 +822 @@
-feild)))
+feild#NA)))
@@ -829 +829 @@
-that)
+that#that_conj)
@@ -835 +835 @@
-Christofer)
+Christofer#NA)
@@ -837 +837 @@
-Chambers))
+Chambers#NA))
@@ -839 +839 @@
-wold)
+wold#will_v1)
@@ -841 +841 @@
-bey)
+bey#buy_v)
@@ -846 +846 @@
-if)
+if#if_conj)
@@ -853 +853 @@
-ye))
+ye#ye_pron))
@@ -855 +855 @@
-will)
+will#will_v1)
@@ -857 +857 @@
-sel)
+sel#sell_v)
@@ -860 +860 @@
-it))))))))
+it#it_pron))))))))
@@ -868 +868 @@
-but)
+but#but_prep)
@@ -871 +871 @@
-I))
+I#i_pron))
@@ -873 +873 @@
-am)
+am#be_v)
@@ -875 +875 @@
-not)
+not#not_adv)
@@ -878 +878 @@
-in)
+in#in_prep)
@@ -881 +881 @@
-a)
+a#a_adj2)
@@ -883 +883 @@
-suerty)
+suerty#surety_n)
@@ -887 +887 @@
-what))
+what#what_pron))
@@ -895 +895 @@
-he))
+he#he_pron))
@@ -899 +899 @@
-will)
+will#will_v1)
@@ -901 +901 @@
-give)
+give#give_v)
@@ -904 +904 @@
-for)
+for#for_prep)
@@ -907 +907 @@
-it)))))))
+it#it_pron)))))))
@@ -915 +915 @@
-But)
+But#but_prep)
@@ -918 +918 @@
-if)
+if#if_conj)
@@ -925 +925 @@
-ye))
+ye#ye_pron))
@@ -927 +927 @@
-will)
+will#will_v1)
@@ -929 +929 @@
-sel)
+sel#sell_v)
@@ -932 +932 @@
-it)))))
+it#it_pron)))))
@@ -936 +936 @@
-send)
+send#send_v1)
@@ -939 +939 @@
-word))
+word#word_n))
@@ -942 +942 @@
-to)
+to#to_prep)
@@ -945 +945 @@
-your)
+your#your_pron)
@@ -947 +947 @@
-son)))
+son#son_n1)))
@@ -951 +951 @@
-what))
+what#what_pron))
@@ -959 +959 @@
-ye))
+ye#ye_pron))
@@ -961 +961 @@
-will)
+will#will_v1)
@@ -963 +963 @@
-doe)))
+doe#do_v)))
@@ -971 +971 @@
-for)
+for#for_prep)
@@ -974 +974 @@
-I))
+I#i_pron))
@@ -976 +976 @@
-know)
+know#know_v)
@@ -979 +979 @@
-nothing)
+nothing#nothing_pron)
@@ -982 +982 @@
-els))
+els#else_adv))
@@ -986 +986 @@
-wherwith))
+wherwith#wherewith_adv))
@@ -989 +989 @@
-to)
+to#to_prep)
@@ -991 +991 @@
-help)
+help#help_v)
@@ -994 +994 @@
-you))
+you#you_pron))
@@ -997 +997 @@
-with)
+with#with_prep)
@@ -1008 +1008 @@
-Sir))
+Sir#sir_n))
@@ -1013 +1013 @@
-for)
+for#for_prep)
@@ -1016 +1016 @@
-God)
+God#NA)
@@ -1018 +1018 @@
-sake)))
+sake#sake_n1)))
@@ -1020 +1020 @@
-take)
+take#take_v)
@@ -1023 +1023 @@
-an)
+an#a_adj2)
@@ -1025 +1025 @@
-end))
+end#end_n))
@@ -1033 +1033 @@
-for)
+for#for_prep)
@@ -1036 +1036 @@
-we))
+we#we_pron))
@@ -1038 +1038 @@
-are)
+are#be_v)
@@ -1040 +1040 @@
-brought)
+brought#bring_v)
@@ -1043 +1043 @@
-to)
+to#to_prep)
@@ -1046 +1046 @@
-begger)
+begger#beggar_n)
@@ -1048 +1048 @@
-staffe)))
+staffe#staff_n1)))
@@ -1056 +1056 @@
-for)
+for#for_prep)
@@ -1059 +1059 @@
-ye))
+ye#ye_pron))
@@ -1061 +1061 @@
-have)
+have#have_v)
@@ -1063 +1063 @@
-not)
+not#not_adv)
@@ -1070 +1070 @@
-to)
+to#to_prep)
@@ -1072 +1072 @@
-defend)
+defend#defend_v)
@@ -1075 +1075 @@
-them))
+them#them_pron))
@@ -1078 +1078 @@
-withall)
+withall#withal_adv)
@@ -1089 +1089 @@
-Sir))
+Sir#sir_n))
@@ -1094 +1094 @@
-I))
+I#i_pron))
@@ -1096 +1096 @@
-send)
+send#send_v1)
@@ -1099 +1099 @@
-you))
+you#you_pron))
@@ -1103 +1103 @@
-my)
+my#my_adj)
@@ -1105 +1105 @@
-mare))
+mare#mare_n1))
@@ -1110 +1110 @@
-and)
+and#and_conj1)
@@ -1114 +1114 @@
-iij)
+iij#three_adj)
@@ -1116 +1116 @@
-=s=))
+=s=#shilling_n))
@@ -1120 +1120 @@
-iiij)
+iiij#four_adj)
@@ -1122 +1122 @@
-=d=))))))
+=d=#penny_n))))))
@@ -1125 +1125 @@
-by)
+by#by_prep)
@@ -1128 +1128 @@
-the)
+the#the_adj)
@@ -1130 +1130 @@
-bearer)
+bearer#bearer_n)
@@ -1133 +1133 @@
-herof))))
+herof#hereof_adv))))
@@ -1141 +1141 @@
-and)
+and#and_conj1)
@@ -1145 +1145 @@
-I))
+I#i_pron))
@@ -1147 +1147 @@
-pray)
+pray#pray_v)
@@ -1150 +1150 @@
-you)))
+you#you_pron)))
@@ -1152 +1152 @@
-send)
+send#send_v1)
@@ -1155 +1155 @@
-me))
+me#me_pron1))
@@ -1158 +1158 @@
-word))
+word#word_n))
@@ -1161 +1161 @@
-as)
+as#as_adv)
@@ -1163 +1163 @@
-sone)
+sone#soon_adv)
@@ -1166 +1166 @@
-as)
+as#as_adv)
@@ -1177 +1177 @@
-ye))
+ye#ye_pron))
@@ -1179 +1179 @@
-may)
+may#may_v1)
@@ -1191 +1191 @@
-No)
+No#no_adj)
@@ -1193 +1193 @@
-more)))
+more#more_adj)))
@@ -1196 +1196 @@
-at)
+at#at_prep)
@@ -1199 +1199 @@
-this)
+this#this_pron)
@@ -1201 +1201 @@
-tyme)))
+tyme#time_n)))
@@ -1209 +1209 @@
-but)
+but#but_prep)
@@ -1212 +1212 @@
-the)
+the#the_adj)
@@ -1214 +1214 @@
-Holy)
+Holy#holy_adj)
@@ -1216 +1216 @@
-Trenyttie))
+Trenyttie#NA))
@@ -1218 +1218 @@
-send)
+send#send_v1)
@@ -1221 +1221 @@
-you))
+you#you_pron))
@@ -1224 +1224 @@
-good)
+good#good_adj)
@@ -1226 +1226 @@
-speed)
+speed#speed_n)
@@ -1229 +1229 @@
-in)
+in#in_prep)
@@ -1232 +1232 @@
-all)
+all#all_adj)
@@ -1234 +1234 @@
-your)
+your#your_pron)
@@ -1236 +1236 @@
-matters))))
+matters#matter_n1))))
@@ -1244 +1244 @@
-and)
+and#and_conj1)
@@ -1248 +1248 @@
-send)
+send#send_v1)
@@ -1251 +1251 @@
-you))
+you#you_pron))
@@ -1254 +1254 @@
-sone))
+sone#soon_adv))
@@ -1257 +1257 @@
-home))
+home#home_n1))
@@ -1266 +1266 @@
-Sir))
+Sir#sir_n))
@@ -1270 +1270 @@
-remember)
+remember#remember_v)
@@ -1274 +1274 @@
-your)
+your#your_pron)
@@ -1276 +1276 @@
-chillder))
+chillder#child_n))
@@ -1278 +1278 @@
-bookes))
+bookes#book_n))
@@ -1286 +1286 @@
-Be)
+Be#by_prep)
@@ -1289 +1289 @@
-your)
+your#your_pron)
@@ -1291 +1291 @@
-bedfellow)
+bedfellow#bedfellow_n)
@@ -1296 +1296 @@
-Isabell)
+Isabell#NA)
@@ -1298 +1298 @@
-Plompton)))
+Plompton#NA)))
