--- jbarring-1629-e2-h.psd.vert	2025-09-10 13:50:35
+++ jbarring-1629-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Deare)
+Deare#dear_adj1)
@@ -16 +16 @@
-mother))
+mother#mother_n1))
@@ -19 +19 @@
-I))
+I#i_pron))
@@ -21 +21 @@
-am)
+am#be_v)
@@ -24 +24 @@
-exceading)
+exceading#exceeding_adj)
@@ -26 +26 @@
-sorrowfull)
+sorrowfull#sorrowful_adj)
@@ -29 +29 @@
-that)
+that#that_conj)
@@ -33 +33 @@
-I))
+I#i_pron))
@@ -35 +35 @@
-have)
+have#have_v)
@@ -37 +37 @@
-been)
+been#be_v)
@@ -39 +39 @@
-forced)
+forced#force_v1)
@@ -42 +42 @@
-by)
+by#by_prep)
@@ -45 +45 @@
-reason)
+reason#reason_n1)
@@ -48 +48 @@
-of)
+of#of_prep)
@@ -51 +51 @@
-my)
+my#my_adj)
@@ -53 +53 @@
-longe)
+longe#long_adj1)
@@ -55 +55 @@
-sicknes)))))
+sicknes#sickness_n)))))
@@ -58 +58 @@
-to)
+to#to_prep)
@@ -60 +60 @@
-be)
+be#be_v)
@@ -63 +63 @@
-so)
+so#so_adv)
@@ -65 +65 @@
-often))
+often#often_adv))
@@ -68 +68 @@
-importunate)
+importunate#importunate_adj)
@@ -71 +71 @@
-to)
+to#to_prep)
@@ -73 +73 @@
-request)
+request#request_v1)
@@ -76 +76 @@
-your)
+your#your_pron)
@@ -78 +78 @@
-favour))
+favour#favour_n))
@@ -81 +81 @@
-in)
+in#in_prep)
@@ -84 +84 @@
-the)
+the#the_adj)
@@ -86 +86 @@
-releafe)
+releafe#relief_n2)
@@ -89 +89 @@
-of)
+of#of_prep)
@@ -92 +92 @@
-my)
+my#my_adj)
@@ -94 +94 @@
-greate)
+greate#great_adj)
@@ -96 +96 @@
-necessities)))))))))))
+necessities#necessity_n)))))))))))
@@ -102 +102 @@
-wherby))
+wherby#whereby_adv))
@@ -108 +108 @@
-I))
+I#i_pron))
@@ -110 +110 @@
-know)
+know#know_v)
@@ -119 +119 @@
-I))
+I#i_pron))
@@ -121 +121 @@
-have)
+have#have_v)
@@ -123 +123 @@
-beene)
+beene#be_v)
@@ -126 +126 @@
-very)
+very#very_adj)
@@ -128 +128 @@
-chargeable)
+chargeable#chargeable_adj)
@@ -131 +131 @@
-unto)
+unto#unto_prep)
@@ -134 +134 @@
-yow))))))))
+yow#you_pron))))))))
@@ -140 +140 @@
-without)
+without#without_adv)
@@ -143 +143 @@
-which)
+which#which_adj)
@@ -145 +145 @@
-love)
+love#love_n1)
@@ -148 +148 @@
-of)
+of#of_prep)
@@ -151 +151 @@
-yours)))
+yours#yours_pron)))
@@ -155 +155 @@
-so)
+so#so_adv)
@@ -157 +157 @@
-highlie))
+highlie#highly_adv))
@@ -159 +159 @@
-shewed)
+shewed#show_v)
@@ -162 +162 @@
-to)
+to#to_prep)
@@ -165 +165 @@
-mee))))))
+mee#me_pron1))))))
@@ -173 +173 @@
-I))
+I#i_pron))
@@ -175 +175 @@
-could)
+could#can_v1)
@@ -177 +177 @@
-not)
+not#not_adv)
@@ -179 +179 @@
-have)
+have#have_v)
@@ -181 +181 @@
-imagined)
+imagined#imagine_v)
@@ -185 +185 @@
-what))
+what#what_pron))
@@ -193 +193 @@
-I))
+I#i_pron))
@@ -195 +195 @@
-should)
+should#shall_v)
@@ -197 +197 @@
-have)
+have#have_v)
@@ -199 +199 @@
-done)
+done#do_v)
@@ -202 +202 @@
-for)
+for#for_prep)
@@ -205 +205 @@
-my)
+my#my_adj)
@@ -207 +207 @@
-releafe)))
+releafe#relief_n2)))
@@ -210 +210 @@
-all)
+all#all_adj)
@@ -212 +212 @@
-this)
+this#this_pron)
@@ -214 +214 @@
-time))))))
+time#time_n))))))
@@ -223 +223 @@
-I))
+I#i_pron))
@@ -225 +225 @@
-beseach)
+beseach#beseech_v)
@@ -228 +228 @@
-yow))
+yow#you_pron))
@@ -231 +231 @@
-not)
+not#not_adv)
@@ -233 +233 @@
-to)
+to#to_prep)
@@ -235 +235 @@
-esteame)
+esteame#esteem_v)
@@ -239 +239 @@
-your)
+your#your_pron)
@@ -241 +241 @@
-love))
+love#love_n1))
@@ -244 +244 @@
-ill))
+ill#ill_adv))
@@ -246 +246 @@
-bestowed)
+bestowed#bestow_v)
@@ -249 +249 @@
-upon)
+upon#upon_prep)
@@ -252 +252 @@
-mee)))
+mee#me_pron1)))
@@ -255 +255 @@
-hearin))))
+hearin#herein_adv))))
@@ -264 +264 @@
-I))
+I#i_pron))
@@ -266 +266 @@
-trust)
+trust#trust_v)
@@ -269 +269 @@
-in)
+in#in_prep)
@@ -272 +272 @@
-God)))
+God#NA)))
@@ -280 +280 @@
-I))
+I#i_pron))
@@ -282 +282 @@
-shall)
+shall#shall_v)
@@ -285 +285 @@
-never))
+never#never_adv))
@@ -287 +287 @@
-give)
+give#give_v)
@@ -290 +290 @@
-yow))
+yow#you_pron))
@@ -293 +293 @@
-cause)
+cause#cause_n)
@@ -297 +297 @@
-so))
+so#so_adv))
@@ -299 +299 @@
-to)
+to#to_prep)
@@ -301 +301 @@
-doe))))
+doe#do_v))))
@@ -306 +306 @@
-but)
+but#but_prep)
@@ -310 +310 @@
-yow))
+yow#you_pron))
@@ -312 +312 @@
-shall)
+shall#shall_v)
@@ -318 +318 @@
-God))
+God#NA))
@@ -320 +320 @@
-willing))
+willing#will_v2))
@@ -325 +325 @@
-ever))
+ever#ever_adv))
@@ -327 +327 @@
-finde)
+finde#find_v)
@@ -331 +331 @@
-mee))
+mee#me_pron1))
@@ -334 +334 @@
-willing)
+willing#willing_adj)
@@ -337 +337 @@
-to)
+to#to_prep)
@@ -339 +339 @@
-strive)
+strive#strive_v)
@@ -342 +342 @@
-to)
+to#to_prep)
@@ -344 +344 @@
-desearve)
+desearve#disserve_v)
@@ -347 +347 @@
-your)
+your#your_pron)
@@ -349 +349 @@
-love))
+love#love_n1))
@@ -352 +352 @@
-to)
+to#to_prep)
@@ -355 +355 @@
-my)
+my#my_adj)
@@ -357 +357 @@
-uttermost)
+uttermost#uttermost_adj)
@@ -359 +359 @@
-power)
+power#power_n1)
@@ -362 +362 @@
-and)
+and#and_conj1)
@@ -365 +365 @@
-indeavour)))))))))))))
+indeavour#endeavour_n)))))))))))))
@@ -374 +374 @@
-I))
+I#i_pron))
@@ -376 +376 @@
-must)
+must#must_v1)
@@ -379 +379 @@
-indeade))
+indeade#indeed_adv))
@@ -381 +381 @@
-confess)
+confess#confess_v)
@@ -388 +388 @@
-I))
+I#i_pron))
@@ -390 +390 @@
-was)
+was#be_v)
@@ -394 +394 @@
-more)
+more#more_adj)
@@ -396 +396 @@
-then)
+then#than_conj)
@@ -398 +398 @@
-ordinarie))
+ordinarie#ordinary_adj))
@@ -400 +400 @@
-chargeable)
+chargeable#chargeable_adj)
@@ -403 +403 @@
-unto)
+unto#unto_prep)
@@ -406 +406 @@
-yow))))
+yow#you_pron))))
@@ -409 +409 @@
-latelie))
+latelie#lately_adv))
@@ -414 +414 @@
-being)
+being#be_v)
@@ -416 +416 @@
-constrained)
+constrained#constrain_v)
@@ -419 +419 @@
-therunto)))
+therunto#thereunto_adv)))
@@ -423 +423 @@
-which))
+which#which_adj))
@@ -431 +431 @@
-I))
+I#i_pron))
@@ -433 +433 @@
-could)
+could#can_v1)
@@ -435 +435 @@
-not)
+not#not_adv)
@@ -437 +437 @@
-avoide)))
+avoide#avoid_v)))
@@ -445 +445 @@
-in)
+in#in_prep)
@@ -448 +448 @@
-which)
+which#which_adj)
@@ -450 +450 @@
-respect)))
+respect#respect_n)))
@@ -456 +456 @@
-I))
+I#i_pron))
@@ -458 +458 @@
-do)
+do#do_v)
@@ -460 +460 @@
-understand)
+understand#understand_v)
@@ -469 +469 @@
-yow))
+yow#you_pron))
@@ -471 +471 @@
-are)
+are#be_v)
@@ -474 +474 @@
-displeased)
+displeased#displeased_adj)
@@ -477 +477 @@
-with)
+with#with_prep)
@@ -480 +480 @@
-mee))))
+mee#me_pron1))))
@@ -486 +486 @@
-which))
+which#which_adj))
@@ -493 +493 @@
-hath)
+hath#have_v)
@@ -496 +496 @@
-much))
+much#much_adj))
@@ -498 +498 @@
-troubled)
+troubled#trouble_v)
@@ -501 +501 @@
-mee))))))))))
+mee#me_pron1))))))))))
@@ -509 +509 @@
-but)
+but#but_prep)
@@ -512 +512 @@
-I))
+I#i_pron))
@@ -514 +514 @@
-do)
+do#do_v)
@@ -516 +516 @@
-intreate)
+intreate#entreat_v)
@@ -519 +519 @@
-yow))
+yow#you_pron))
@@ -522 +522 @@
-if)
+if#if_conj)
@@ -529 +529 @@
-yow))
+yow#you_pron))
@@ -531 +531 @@
-have)
+have#have_v)
@@ -533 +533 @@
-conceaved)
+conceaved#conceive_v)
@@ -536 +536 @@
-any)
+any#any_adj)
@@ -538 +538 @@
-distast)
+distast#distaste_n)
@@ -541 +541 @@
-against)
+against#against_prep)
@@ -544 +544 @@
-mee)))
+mee#me_pron1)))
@@ -547 +547 @@
-hearin))))))
+hearin#herein_adv))))))
@@ -550 +550 @@
-that)
+that#that_conj)
@@ -554 +554 @@
-yow))
+yow#you_pron))
@@ -556 +556 @@
-would)
+would#will_v1)
@@ -558 +558 @@
-be)
+be#be_v)
@@ -560 +560 @@
-pleased)
+pleased#please_v)
@@ -563 +563 @@
-to)
+to#to_prep)
@@ -565 +565 @@
-remitt)
+remitt#remit_v)
@@ -568 +568 @@
-it)))))
+it#it_pron)))))
@@ -573 +573 @@
-assuring)
+assuring#assure_v)
@@ -576 +576 @@
-yow))
+yow#you_pron))
@@ -583 +583 @@
-I))
+I#i_pron))
@@ -585 +585 @@
-will)
+will#will_v1)
@@ -588 +588 @@
-carefully))
+carefully#carefully_adv))
@@ -590 +590 @@
-avoyde)
+avoyde#avoid_v)
@@ -593 +593 @@
-any)
+any#any_adj)
@@ -595 +595 @@
-thinge)
+thinge#thing_n1)
@@ -599 +599 @@
-which))
+which#which_adj))
@@ -605 +605 @@
-I))
+I#i_pron))
@@ -607 +607 @@
-can)
+can#can_v1)
@@ -609 +609 @@
-thinke)
+thinke#think_v2)
@@ -614 +614 @@
-to)
+to#to_prep)
@@ -616 +616 @@
-be)
+be#be_v)
@@ -619 +619 @@
-displeasing)
+displeasing#displeasing_adj)
@@ -622 +622 @@
-to)
+to#to_prep)
@@ -625 +625 @@
-yow)))))))))))
+yow#you_pron)))))))))))
@@ -634 +634 @@
-I))
+I#i_pron))
@@ -636 +636 @@
-do)
+do#do_v)
@@ -638 +638 @@
-understand)
+understand#understand_v)
@@ -645 +645 @@
-it))
+it#it_pron))
@@ -647 +647 @@
-is)
+is#be_v)
@@ -650 +650 @@
-your)
+your#your_pron)
@@ -652 +652 @@
-pleasure))
+pleasure#pleasure_n))
@@ -655 +655 @@
-to)
+to#to_prep)
@@ -657 +657 @@
-know)
+know#know_v)
@@ -660 +660 @@
-of)
+of#of_prep)
@@ -663 +663 @@
-mee)))
+mee#me_pron1)))
@@ -667 +667 @@
-what)
+what#what_pron)
@@ -669 +669 @@
-course))
+course#course_n1))
@@ -675 +675 @@
-I))
+I#i_pron))
@@ -677 +677 @@
-do)
+do#do_v)
@@ -679 +679 @@
-purpose)
+purpose#purpose_v)
@@ -684 +684 @@
-to)
+to#to_prep)
@@ -686 +686 @@
-take)
+take#take_v)
@@ -689 +689 @@
-when)
+when#when_adv)
@@ -696 +696 @@
-God))
+God#NA))
@@ -698 +698 @@
-shal@)
+shal@#shall_v)
@@ -700 +700 @@
-@be)
+@be#be_v)
@@ -702 +702 @@
-pleased)
+pleased#please_v)
@@ -705 +705 @@
-to)
+to#to_prep)
@@ -707 +707 @@
-inable)
+inable#enable_v)
@@ -710 +710 @@
-mee))
+mee#me_pron1))
@@ -713 +713 @@
-with)
+with#with_prep)
@@ -716 +716 @@
-strength)
+strength#strength_n)
@@ -719 +719 @@
-to)
+to#to_prep)
@@ -721 +721 @@
-goe)
+goe#go_v)
@@ -724 +724 @@
-abroode)))))))))))))))
+abroode#abroad_adv)))))))))))))))
@@ -733 +733 @@
-I))
+I#i_pron))
@@ -735 +735 @@
-beseach)
+beseach#beseech_v)
@@ -738 +738 @@
-you))
+you#you_pron))
@@ -741 +741 @@
-not)
+not#not_adv)
@@ -743 +743 @@
-to)
+to#to_prep)
@@ -745 +745 @@
-harbour)
+harbour#harbour_v)
@@ -748 +748 @@
-any)
+any#any_adj)
@@ -750 +750 @@
-other)
+other#other_adj)
@@ -752 +752 @@
-conceite)
+conceite#conceit_n)
@@ -755 +755 @@
-then)
+then#than_conj)
@@ -758 +758 @@
-that)
+that#that_conj)
@@ -762 +762 @@
-I))
+I#i_pron))
@@ -764 +764 @@
-am)
+am#be_v)
@@ -767 +767 @@
-resolved)
+resolved#resolved_adj)
@@ -770 +770 @@
-to)
+to#to_prep)
@@ -772 +772 @@
-take)
+take#take_v)
@@ -775 +775 @@
-such)
+such#such_adj)
@@ -777 +777 @@
-a)
+a#a_adj2)
@@ -779 +779 @@
-lawfull)
+lawfull#lawful_adj)
@@ -781 +781 @@
-course)
+course#course_n1)
@@ -784 +784 @@
-as)
+as#as_adv)
@@ -794 +794 @@
-shall)
+shall#shall_v)
@@ -796 +796 @@
-be)
+be#be_v)
@@ -799 +799 @@
-both)
+both#both_pron)
@@ -802 +802 @@
-pleasing)
+pleasing#pleasing_adj)
@@ -805 +805 @@
-to)
+to#to_prep)
@@ -808 +808 @@
-God))))
+God#NA))))
@@ -811 +811 @@
-and)
+and#and_conj1)
@@ -814 +814 @@
-liking)
+liking#liking_adj1)
@@ -817 +817 @@
-to)
+to#to_prep)
@@ -820 +820 @@
-your)
+your#your_pron)
@@ -822 +822 @@
-sealfe)))))))))))))))))
+sealfe#self_pron)))))))))))))))))
@@ -831 +831 @@
-this))
+this#this_pron))
@@ -834 +834 @@
-my)
+my#my_adj)
@@ -836 +836 @@
-desire))
+desire#desire_n))
@@ -844 +844 @@
-for)
+for#for_prep)
@@ -847 +847 @@
-I))
+I#i_pron))
@@ -849 +849 @@
-delight)
+delight#delight_v)
@@ -851 +851 @@
-not)
+not#not_adv)
@@ -854 +854 @@
-in)
+in#in_prep)
@@ -857 +857 @@
-sullenes)))
+sullenes#sullenness_n)))
@@ -865 +865 @@
-But)
+But#but_prep)
@@ -868 +868 @@
-as)
+as#as_adv)
@@ -871 +871 @@
-yet)))
+yet#yet_adv)))
@@ -874 +874 @@
-I))
+I#i_pron))
@@ -876 +876 @@
-must)
+must#must_v1)
@@ -879 +879 @@
-with)
+with#with_prep)
@@ -882 +882 @@
-patience)))
+patience#patience_n.I)))
@@ -884 +884 @@
-waight)
+waight#wait_v1)
@@ -888 +888 @@
-God)
+God#NA)
@@ -890 +890 @@
-his))
+his#NA))
@@ -892 +892 @@
-pleasure))
+pleasure#pleasure_n))
@@ -895 +895 @@
-for)
+for#for_prep)
@@ -898 +898 @@
-my)
+my#my_adj)
@@ -900 +900 @@
-former)
+former#former_adj)
@@ -902 +902 @@
-strength)))
+strength#strength_n)))
@@ -910 +910 @@
-for)
+for#for_prep)
@@ -913 +913 @@
-as)
+as#as_adv)
@@ -916 +916 @@
-yet)))
+yet#yet_adv)))
@@ -919 +919 @@
-I))
+I#i_pron))
@@ -921 +921 @@
-have)
+have#have_v)
@@ -923 +923 @@
-not))
+not#not_adv))
@@ -929 +929 @@
-nore)
+nore#nor_conj1)
@@ -933 +933 @@
-dare)
+dare#dare_v1)
@@ -935 +935 @@
-beare)
+beare#bear_v1)
@@ -938 +938 @@
-any)
+any#any_adj)
@@ -940 +940 @@
-waight))
+waight#weight_n1))
@@ -943 +943 @@
-upon)
+upon#upon_prep)
@@ -946 +946 @@
-my)
+my#my_adj)
@@ -948 +948 @@
-hurte)
+hurte#hurt_adj)
@@ -950 +950 @@
-legg)))
+legg#leg_n)))
@@ -953 +953 @@
-because)
+because#because_adv)
@@ -960 +960 @@
-the)
+the#the_adj)
@@ -962 +962 @@
-bone)
+bone#bone_n1)
@@ -966 +966 @@
-is)
+is#be_v)
@@ -968 +968 @@
-not)
+not#not_adv)
@@ -970 +970 @@
-all)
+all#all_adj)
@@ -972 +972 @@
-come)
+come#come_v)
@@ -974 +974 @@
-of)
+of#off_adv)
@@ -979 +979 @@
-that)
+that#that_conj)
@@ -984 +984 @@
-is)
+is#be_v)
@@ -987 +987 @@
-to)
+to#to_prep)
@@ -989 +989 @@
-scale)
+scale#scale_v2)
@@ -991 +991 @@
-of)))))))
+of#off_adv)))))))
@@ -1000 +1000 @@
-I))
+I#i_pron))
@@ -1003 +1003 @@
-only)
+only#only_adj)
@@ -1005 +1005 @@
-as)
+as#as_adv)
@@ -1008 +1008 @@
-yet)))
+yet#yet_adv)))
@@ -1010 +1010 @@
-goe)
+goe#go_v)
@@ -1013 +1013 @@
-with)
+with#with_prep)
@@ -1016 +1016 @@
-cruches)))
+cruches#crutch_n)))
@@ -1019 +1019 @@
-now)
+now#now_adv)
@@ -1021 +1021 @@
-and)
+and#and_conj1)
@@ -1023 +1023 @@
-then))
+then#then_adv))
@@ -1026 +1026 @@
-about)
+about#about_adv)
@@ -1029 +1029 @@
-my)
+my#my_adj)
@@ -1033 +1033 @@
-chamber)))
+chamber#chamber_n)))
@@ -1042 +1042 @@
-With)
+With#with_prep)
@@ -1045 +1045 @@
-the)
+the#the_adj)
@@ -1047 +1047 @@
-remembrance)
+remembrance#remembrance_n)
@@ -1050 +1050 @@
-of)
+of#of_prep)
@@ -1053 +1053 @@
-my)
+my#my_adj)
@@ -1055 +1055 @@
-humble)
+humble#humble_adj)
@@ -1057 +1057 @@
-dutie)))))
+dutie#duty_n)))))
@@ -1060 +1060 @@
-I))
+I#i_pron))
@@ -1062 +1062 @@
-commend)
+commend#commend_v)
@@ -1065 +1065 @@
-yow))
+yow#you_pron))
@@ -1068 +1068 @@
-to)
+to#to_prep)
@@ -1072 +1072 @@
-God)
+God#NA)
@@ -1074 +1074 @@
-his))
+his#NA))
@@ -1076 +1076 @@
-protection)))
+protection#protection_n)))
@@ -1079 +1079 @@
-remayning)
+remayning#remain_v)
@@ -1082 +1082 @@
-Your)
+Your#your_pron)
@@ -1084 +1084 @@
-obedient)
+obedient#obedient_adj)
@@ -1086 +1086 @@
-sonn)
+sonn#son_n1)
@@ -1089 +1089 @@
-John)
+John#NA)
@@ -1091 +1091 @@
-Barrington)))))
+Barrington#NA)))))
@@ -1097 +1097 @@
-London)
+London#NA)
@@ -1105 +1105 @@
-18)
+18#eighteen_adj)
@@ -1107 +1107 @@
-July)
+July#NA)
@@ -1111 +1111 @@
-1629)
+1629#NA)
@@ -1119 +1119 @@
-To)
+To#to_prep)
@@ -1122 +1122 @@
-my)
+my#my_adj)
@@ -1124 +1124 @@
-honourable)
+honourable#honourable_adj)
@@ -1126 +1126 @@
-mother)
+mother#mother_n1)
@@ -1129 +1129 @@
-the)
+the#the_adj)
@@ -1131 +1131 @@
-lady)
+lady#lady_n)
@@ -1134 +1134 @@
-Johanna)
+Johanna#NA)
@@ -1136 +1136 @@
-Barrington)))))
+Barrington#NA)))))
@@ -1149 +1149 @@
-Deare)
+Deare#dear_adj1)
@@ -1151 +1151 @@
-mother))
+mother#mother_n1))
@@ -1154 +1154 @@
-May)
+May#may_v1)
@@ -1157 +1157 @@
-it))
+it#it_pron))
@@ -1159 +1159 @@
-please)
+please#please_v)
@@ -1162 +1162 @@
-yow)))
+yow#you_pron)))
@@ -1167 +1167 @@
-since)
+since#since_adv)
@@ -1170 +1170 @@
-my)
+my#my_adj)
@@ -1172 +1172 @@
-last)
+last#last_adv)
@@ -1174 +1174 @@
-letter)
+letter#letter_n1)
@@ -1178 +1178 @@
-which))
+which#which_adj))
@@ -1184 +1184 @@
-I))
+I#i_pron))
@@ -1186 +1186 @@
-desired)
+desired#desire_v)
@@ -1189 +1189 @@
-Sir)
+Sir#NA)
@@ -1191 +1191 @@
-Francis)
+Francis#NA)
@@ -1193 +1193 @@
-Harris))
+Harris#NA))
@@ -1198 +1198 @@
-to)
+to#to_prep)
@@ -1200 +1200 @@
-deliver)
+deliver#deliver_v1)
@@ -1203 +1203 @@
-unto)
+unto#unto_prep)
@@ -1206 +1206 @@
-yow))))))))
+yow#you_pron))))))))
@@ -1209 +1209 @@
-my)
+my#my_adj)
@@ -1211 +1211 @@
-bone)
+bone#bone_n1)
@@ -1215 +1215 @@
-which))
+which#which_adj))
@@ -1222 +1222 @@
-was)
+was#be_v)
@@ -1225 +1225 @@
-to)
+to#to_prep)
@@ -1227 +1227 @@
-scale)))))
+scale#scale_v2)))))
@@ -1229 +1229 @@
-is)
+is#be_v)
@@ -1235 +1235 @@
-I))
+I#i_pron))
@@ -1237 +1237 @@
-thanck)
+thanck#thank_v)
@@ -1240 +1240 @@
-God)))
+God#NA)))
@@ -1245 +1245 @@
-quite))
+quite#quite_adv))
@@ -1247 +1247 @@
-of)
+of#off_adv)
@@ -1250 +1250 @@
-and)
+and#and_conj1)
@@ -1253 +1253 @@
-the)
+the#the_adj)
@@ -1255 +1255 @@
-wound))
+wound#wound_n))
@@ -1258 +1258 @@
-allmost))
+allmost#almost_adv))
@@ -1260 +1260 @@
-dried)
+dried#dry_v)
@@ -1262 +1262 @@
-upp))
+upp#up_adv1))
@@ -1267 +1267 @@
-soe)
+soe#so_adv)
@@ -1270 +1270 @@
-that)
+that#that_conj)
@@ -1274 +1274 @@
-I))
+I#i_pron))
@@ -1276 +1276 @@
-hope)
+hope#hope_v)
@@ -1283 +1283 @@
-I))
+I#i_pron))
@@ -1285 +1285 @@
-shall)
+shall#shall_v)
@@ -1288 +1288 @@
-now))
+now#now_adv))
@@ -1290 +1290 @@
-be)
+be#be_v)
@@ -1293 +1293 @@
-quickly))
+quickly#quickly_adv))
@@ -1296 +1296 @@
-able)
+able#able_adj)
@@ -1299 +1299 @@
-to)
+to#to_prep)
@@ -1301 +1301 @@
-goe)
+goe#go_v)
@@ -1304 +1304 @@
-abroade)))))))))
+abroade#abroad_adv)))))))))
@@ -1313 +1313 @@
-I))
+I#i_pron))
@@ -1315 +1315 @@
-do)
+do#do_v)
@@ -1317 +1317 @@
-hope)
+hope#hope_v)
@@ -1323 +1323 @@
-God))
+God#NA))
@@ -1325 +1325 @@
-willing))
+willing#will_v2))
@@ -1331 +1331 @@
-to)
+to#to_prep)
@@ -1333 +1333 @@
-gow)
+gow#go_v)
@@ -1336 +1336 @@
-for)
+for#for_prep)
@@ -1339 +1339 @@
-Swede))))
+Swede#NA))))
@@ -1342 +1342 @@
-and)
+and#and_conj1)
@@ -1345 +1345 @@
-to)
+to#to_prep)
@@ -1347 +1347 @@
-have)
+have#have_v)
@@ -1350 +1350 @@
-a)
+a#a_adj2)
@@ -1352 +1352 @@
-company))
+company#company_n))
@@ -1355 +1355 @@
-for)
+for#for_prep)
@@ -1358 +1358 @@
-that)
+that#that_pron)
@@ -1360 +1360 @@
-service))))))
+service#service_n1))))))
@@ -1368 +1368 @@
-but)
+but#but_prep)
@@ -1371 +1371 @@
-as)
+as#as_adv)
@@ -1374 +1374 @@
-yet)))
+yet#yet_adv)))
@@ -1377 +1377 @@
-wee))
+wee#we_pron))
@@ -1379 +1379 @@
-have)
+have#have_v)
@@ -1382 +1382 @@
-noe)
+noe#no_adj)
@@ -1384 +1384 @@
-absolute)
+absolute#absolute_adj)
@@ -1386 +1386 @@
-answeare)
+answeare#answer_n)
@@ -1389 +1389 @@
-of)
+of#of_prep)
@@ -1398 +1398 @@
-those)
+those#those_pron)
@@ -1400 +1400 @@
-condicions)
+condicions#condition_n)
@@ -1404 +1404 @@
-are)
+are#be_v)
@@ -1406 +1406 @@
-sent)
+sent#send_v1)
@@ -1409 +1409 @@
-to)
+to#to_prep)
@@ -1412 +1412 @@
-the)
+the#the_adj)
@@ -1414 +1414 @@
-king)))
+king#king_n)))
@@ -1420 +1420 @@
-which))
+which#which_adj))
@@ -1427 +1427 @@
-are)
+are#be_v)
@@ -1429 +1429 @@
-carried)
+carried#carry_v)
@@ -1432 +1432 @@
-thither))
+thither#thither_adv))
@@ -1435 +1435 @@
-by)
+by#by_prep)
@@ -1438 +1438 @@
-a)
+a#a_adj2)
@@ -1440 +1440 @@
-Dutchman)
+Dutchman#NA)
@@ -1444 +1444 @@
-who))
+who#who_pron))
@@ -1451 +1451 @@
-is)
+is#be_v)
@@ -1454 +1454 @@
-to)
+to#to_prep)
@@ -1456 +1456 @@
-be)
+be#be_v)
@@ -1459 +1459 @@
-our)
+our#our_pron)
@@ -1461 +1461 @@
-collonell))
+collonell#colonel_n))
@@ -1466 +1466 @@
-if)
+if#if_conj)
@@ -1473 +1473 @@
-wee))
+wee#we_pron))
@@ -1475 +1475 @@
-are)
+are#be_v)
@@ -1478 +1478 @@
-agreed)
+agreed#agreed_adj)
@@ -1481 +1481 @@
-upon)
+upon#upon_prep)
@@ -1484 +1484 @@
-our)
+our#our_pron)
@@ -1486 +1486 @@
-condicions))))))))))))))))))
+condicions#condition_n))))))))))))))))))
@@ -1497 +1497 @@
-I))
+I#i_pron))
@@ -1499 +1499 @@
-was)
+was#be_v)
@@ -1501 +1501 @@
-demanded)
+demanded#demand_v)
@@ -1504 +1504 @@
-by)
+by#by_prep)
@@ -1507 +1507 @@
-a)
+a#a_adj2)
@@ -1509 +1509 @@
-speciall)
+speciall#special_adj)
@@ -1511 +1511 @@
-frind)))
+frind#friend_n)))
@@ -1514 +1514 @@
-if)
+if#if_conj)
@@ -1521 +1521 @@
-I))
+I#i_pron))
@@ -1523 +1523 @@
-would)
+would#will_v1)
@@ -1525 +1525 @@
-be)
+be#be_v)
@@ -1528 +1528 @@
-one)
+one#one_adj)
@@ -1531 +1531 @@
-of)
+of#of_prep)
@@ -1534 +1534 @@
-the)
+the#the_adj)
@@ -1536 +1536 @@
-regiment)))))))
+regiment#regiment_n)))))))
@@ -1542 +1542 @@
-to)
+to#to_prep)
@@ -1545 +1545 @@
-which)))
+which#which_adj)))
@@ -1553 +1553 @@
-I))
+I#i_pron))
@@ -1555 +1555 @@
-have)
+have#have_v)
@@ -1558 +1558 @@
-willingly))
+willingly#willingly_adv))
@@ -1560 +1560 @@
-agreed)))
+agreed#agree_v)))
@@ -1569 +1569 @@
-As)
+As#as_adv)
@@ -1572 +1572 @@
-yet)))
+yet#yet_adv)))
@@ -1575 +1575 @@
-wee))
+wee#we_pron))
@@ -1577 +1577 @@
-have)
+have#have_v)
@@ -1580 +1580 @@
-no)
+no#no_adj)
@@ -1582 +1582 @@
-certaintie))
+certaintie#certainty_n))
@@ -1585 +1585 @@
-before)
+before#before_adv)
@@ -1592 +1592 @@
-wee))
+wee#we_pron))
@@ -1594 +1594 @@
-heare)
+heare#hear_v)
@@ -1597 +1597 @@
-from)
+from#from_prep)
@@ -1600 +1600 @@
-thence))))))
+thence#thence_adv))))))
@@ -1606 +1606 @@
-which))
+which#which_adj))
@@ -1614 +1614 @@
-wee))
+wee#we_pron))
@@ -1617 +1617 @@
-earnestlie))
+earnestlie#earnestly_adv))
@@ -1619 +1619 @@
-expect)))
+expect#expect_v)))
@@ -1628 +1628 @@
-The)
+The#the_adj)
@@ -1630 +1630 @@
-chiefe)
+chiefe#chief_adj)
@@ -1632 +1632 @@
-points)
+points#point_n1)
@@ -1641 +1641 @@
-wee))
+wee#we_pron))
@@ -1643 +1643 @@
-stand)
+stand#stand_v)
@@ -1646 +1646 @@
-upon)
+upon#upon_prep)
@@ -1650 +1650 @@
-are)
+are#be_v)
@@ -1653 +1653 @@
-for)
+for#for_prep)
@@ -1656 +1656 @@
-monie))
+monie#money_n))
@@ -1658 +1658 @@
-to)
+to#to_prep)
@@ -1660 +1660 @@
-be)
+be#be_v)
@@ -1662 +1662 @@
-paid)
+paid#pay_v1)
@@ -1665 +1665 @@
-us))
+us#us_pron))
@@ -1668 +1668 @@
-heare))
+heare#here_adv))
@@ -1671 +1671 @@
-to)
+to#to_prep)
@@ -1673 +1673 @@
-rayse)
+rayse#raise_v1)
@@ -1676 +1676 @@
-our)
+our#our_pron)
@@ -1678 +1678 @@
-men))))
+men#man_n1))))
@@ -1686 +1686 @@
-and)
+and#and_conj1)
@@ -1689 +1689 @@
-for)
+for#for_prep)
@@ -1692 +1692 @@
-the)
+the#the_adj)
@@ -1694 +1694 @@
-time)
+time#time_n)
@@ -1699 +1699 @@
-how)
+how#how_adv)
@@ -1701 +1701 @@
-long)))
+long#long_adj1)))
@@ -1709 +1709 @@
-they))
+they#they_pron))
@@ -1711 +1711 @@
-will)
+will#will_v1)
@@ -1713 +1713 @@
-entertaine)
+entertaine#entertain_v)
@@ -1716 +1716 @@
-us))))))
+us#us_pron))))))
@@ -1721 +1721 @@
-wee))
+wee#we_pron))
@@ -1723 +1723 @@
-desire)
+desire#desire_v)
@@ -1726 +1726 @@
-to)
+to#to_prep)
@@ -1728 +1728 @@
-make)
+make#make_v1)
@@ -1731 +1731 @@
-in)
+in#in_prep)
@@ -1734 +1734 @@
-our)
+our#our_pron)
@@ -1736 +1736 @@
-condicions)))
+condicions#condition_n)))
@@ -1739 +1739 @@
-that)
+that#that_conj)
@@ -1743 +1743 @@
-the)
+the#the_adj)
@@ -1745 +1745 @@
-king))
+king#king_n))
@@ -1747 +1747 @@
-shal@)
+shal@#shall_v)
@@ -1749 +1749 @@
-@be)
+@be#be_v)
@@ -1751 +1751 @@
-bound)
+bound#bind_v)
@@ -1754 +1754 @@
-to)
+to#to_prep)
@@ -1756 +1756 @@
-keepe)
+keepe#keep_v)
@@ -1759 +1759 @@
-us))
+us#us_pron))
@@ -1762 +1762 @@
-in)
+in#in_prep)
@@ -1765 +1765 @@
-pay)))
+pay#pay_n)))
@@ -1769 +1769 @@
-three)
+three#three_adj)
@@ -1773 +1773 @@
-yeares)
+yeares#year_n)
@@ -1776 +1776 @@
-at)
+at#at_prep)
@@ -1779 +1779 @@
-least))))))))
+least#least_adj))))))))
@@ -1789 +1789 @@
-They))
+They#they_pron))
@@ -1791 +1791 @@
-would)
+would#will_v1)
@@ -1793 +1793 @@
-pay)
+pay#pay_v1)
@@ -1796 +1796 @@
-us))
+us#us_pron))
@@ -1802 +1802 @@
-as)
+as#as_adv)
@@ -1805 +1805 @@
-to)
+to#to_prep)
@@ -1808 +1808 @@
-some))))
+some#some_pron))))
@@ -1811 +1811 @@
-before)))
+before#before_adv)))
@@ -1816 +1816 @@
-halfe)
+halfe#half_adj)
@@ -1818 +1818 @@
-our)
+our#our_pron)
@@ -1820 +1820 @@
-monie))
+monie#money_n))
@@ -1823 +1823 @@
-heare)))
+heare#here_adv)))
@@ -1826 +1826 @@
-and)
+and#and_conj1)
@@ -1830 +1830 @@
-the)
+the#the_adj)
@@ -1832 +1832 @@
-rest))
+rest#rest_n2))
@@ -1836 +1836 @@
-a)
+a#a_adj2)
@@ -1838 +1838 @@
-month))
+month#month_n1))
@@ -1840 +1840 @@
-after)
+after#after_adv)
@@ -1843 +1843 @@
-our)
+our#our_pron)
@@ -1845 +1845 @@
-arrivall)
+arrivall#arrival_n)
@@ -1848 +1848 @@
-theare))))
+theare#there_adv))))
@@ -1853 +1853 @@
-for)
+for#for_prep)
@@ -1856 +1856 @@
-raysing)
+raysing#raise_v1)
@@ -1859 +1859 @@
-our)
+our#our_pron)
@@ -1861 +1861 @@
-men))))))
+men#man_n1))))))
@@ -1869 +1869 @@
-but)
+but#but_prep)
@@ -1872 +1872 @@
-wee))
+wee#we_pron))
@@ -1874 +1874 @@
-hope)
+hope#hope_v)
@@ -1877 +1877 @@
-to)
+to#to_prep)
@@ -1879 +1879 @@
-have)
+have#have_v)
@@ -1882 +1882 @@
-all)
+all#all_adj)
@@ -1884 +1884 @@
-our)
+our#our_pron)
@@ -1886 +1886 @@
-monie))
+monie#money_n))
@@ -1889 +1889 @@
-heare))
+heare#here_adv))
@@ -1892 +1892 @@
-to)
+to#to_prep)
@@ -1895 +1895 @@
-rayse)
+rayse#raise_v1)
@@ -1897 +1897 @@
-and)
+and#and_conj1)
@@ -1899 +1899 @@
-transport))
+transport#transport_v))
@@ -1902 +1902 @@
-our)
+our#our_pron)
@@ -1904 +1904 @@
-men))))
+men#man_n1))))
@@ -1910 +1910 @@
-which))
+which#which_adj))
@@ -1917 +1917 @@
-is)
+is#be_v)
@@ -1920 +1920 @@
-300@)
+300@#NA)
@@ -1922 +1922 @@
-@=li=))
+@=li=#pound_n1))
@@ -1925 +1925 @@
-to)
+to#to_prep)
@@ -1928 +1928 @@
-each)
+each#each_adj)
@@ -1930 +1930 @@
-captain)))))
+captain#captain_n)))))
@@ -1940 +1940 @@
-A)
+A#a_adj2)
@@ -1942 +1942 @@
-captain's))
+captain's#captain_n))
@@ -1944 +1944 @@
-meanes))
+meanes#mean_n2))
@@ -1946 +1946 @@
-is)
+is#be_v)
@@ -1949 +1949 @@
-good))
+good#good_adj))
@@ -1952 +1952 @@
-theare))
+theare#there_adv))
@@ -1956 +1956 @@
-which))
+which#which_adj))
@@ -1964 +1964 @@
-is)
+is#be_v)
@@ -1967 +1967 @@
-twentie_five)
+twentie_five#NA)
@@ -1969 +1969 @@
-pounds)
+pounds#pound_n1)
@@ -1972 +1972 @@
-a)
+a#a_adj2)
@@ -1974 +1974 @@
-month))))
+month#month_n1))))
@@ -1981 +1981 @@
-ten))
+ten#ten_adj))
@@ -1984 +1984 @@
-to)
+to#to_prep)
@@ -1987 +1987 @@
-a)
+a#a_adj2)
@@ -1989 +1989 @@
-lieutenant)))))
+lieutenant#lieutenant_n)))))
@@ -1997 +1997 @@
-as)
+as#as_adv)
@@ -1999 +1999 @@
-much)))
+much#much_adj)))
@@ -2002 +2002 @@
-to)
+to#to_prep)
@@ -2005 +2005 @@
-an)
+an#a_adj2)
@@ -2007 +2007 @@
-ensigne)))))))
+ensigne#ensign_n)))))))
@@ -2016 +2016 @@
-I))
+I#i_pron))
@@ -2018 +2018 @@
-shall)
+shall#shall_v)
@@ -2020 +2020 @@
-be)
+be#be_v)
@@ -2023 +2023 @@
-wondrous)
+wondrous#wondrous_adj)
@@ -2025 +2025 @@
-willing)
+willing#willing_adj)
@@ -2028 +2028 @@
-to)
+to#to_prep)
@@ -2030 +2030 @@
-imbrace)
+imbrace#embrace_v2)
@@ -2033 +2033 @@
-this)
+this#this_pron)
@@ -2035 +2035 @@
-imployment))
+imployment#employment_n))
@@ -2038 +2038 @@
-if)
+if#if_conj)
@@ -2045 +2045 @@
-wee))
+wee#we_pron))
@@ -2047 +2047 @@
-can)
+can#can_v1)
@@ -2049 +2049 @@
-agree)
+agree#agree_v)
@@ -2052 +2052 @@
-to)
+to#to_prep)
@@ -2054 +2054 @@
-have)
+have#have_v)
@@ -2057,3 +2057,3 @@
-all)
-(PRO$
-our)
+all#all_adj)
+(PRO$
+our#our_pron)
@@ -2061 +2061 @@
-monie))
+monie#money_n))
@@ -2064 +2064 @@
-heare))
+heare#here_adv))
@@ -2067 +2067 @@
-to)
+to#to_prep)
@@ -2070 +2070 @@
-rayse)
+rayse#raise_v1)
@@ -2072 +2072 @@
-and)
+and#and_conj1)
@@ -2074 +2074 @@
-transport))
+transport#transport_v))
@@ -2077 +2077 @@
-our)
+our#our_pron)
@@ -2079 +2079 @@
-men)))))))))
+men#man_n1)))))))))
@@ -2088 +2088 @@
-else))
+else#else_adv))
@@ -2091 +2091 @@
-wee))
+wee#we_pron))
@@ -2093 +2093 @@
-can@)
+can@#can_v1)
@@ -2095 +2095 @@
-@not)
+@not#not_adv)
@@ -2097 +2097 @@
-rayse)
+rayse#raise_v1)
@@ -2100 +2100 @@
-them))
+them#them_pron))
@@ -2103 +2103 @@
-for)
+for#for_prep)
@@ -2106 +2106 @@
-want)
+want#want_n2)
@@ -2109 +2109 @@
-of)
+of#of_prep)
@@ -2112 +2112 @@
-monie)))))
+monie#money_n)))))
@@ -2122 +2122 @@
-I))
+I#i_pron))
@@ -2124 +2124 @@
-beseach)
+beseach#beseech_v)
@@ -2127 +2127 @@
-yow)))
+yow#you_pron)))
@@ -2129 +2129 @@
-excuse)
+excuse#excuse_v)
@@ -2132 +2132 @@
-my)
+my#my_adj)
@@ -2134 +2134 @@
-tediousness))
+tediousness#tediousness_n))
@@ -2143 +2143 @@
-with)
+with#with_prep)
@@ -2147 +2147 @@
-my)
+my#my_adj)
@@ -2149 +2149 @@
-humble)
+humble#humble_adj)
@@ -2151 +2151 @@
-duty))
+duty#duty_n))
@@ -2153 +2153 @@
-remembred)))
+remembred#remember_v)))
@@ -2156 +2156 @@
-I))
+I#i_pron))
@@ -2158 +2158 @@
-commend)
+commend#commend_v)
@@ -2161 +2161 @@
-yow))
+yow#you_pron))
@@ -2164 +2164 @@
-to)
+to#to_prep)
@@ -2167 +2167 @@
-the)
+the#the_adj)
@@ -2169 +2169 @@
-protection)
+protection#protection_n)
@@ -2172 +2172 @@
-of)
+of#of_prep)
@@ -2175 +2175 @@
-the)
+the#the_adj)
@@ -2177 +2177 @@
-allmightie)
+allmightie#almighty_adj)
@@ -2181 +2181 @@
-who))
+who#who_pron))
@@ -2189 +2189 @@
-I))
+I#i_pron))
@@ -2191 +2191 @@
-beseach)
+beseach#beseech_v)
@@ -2194 +2194 @@
-to)
+to#to_prep)
@@ -2196 +2196 @@
-prosper)
+prosper#prosper_v)
@@ -2199 +2199 @@
-all)
+all#all_adj)
@@ -2201 +2201 @@
-your)
+your#your_pron)
@@ -2203 +2203 @@
-affaires)))))))))
+affaires#affair_n)))))))))
@@ -2211 +2211 @@
-and)
+and#and_conj1)
@@ -2215 +2215 @@
-will)
+will#will_v1)
@@ -2218 +2218 @@
-ever))
+ever#ever_adv))
@@ -2220 +2220 @@
-rest)
+rest#rest_v2)
@@ -2223 +2223 @@
-Your)
+Your#your_pron)
@@ -2225 +2225 @@
-obedient)
+obedient#obedient_adj)
@@ -2227 +2227 @@
-sonn)
+sonn#son_n1)
@@ -2230 +2230 @@
-John)
+John#NA)
@@ -2232 +2232 @@
-Barrington))))
+Barrington#NA))))
@@ -2245 +2245 @@
-Deare)
+Deare#dear_adj1)
@@ -2247 +2247 @@
-mother))
+mother#mother_n1))
@@ -2250 +2250 @@
-May)
+May#may_v1)
@@ -2253 +2253 @@
-it))
+it#it_pron))
@@ -2255 +2255 @@
-please)
+please#please_v)
@@ -2258 +2258 @@
-yow)))
+yow#you_pron)))
@@ -2263 +2263 @@
-since)
+since#since_adv)
@@ -2270 +2270 @@
-I))
+I#i_pron))
@@ -2273 +2273 @@
-last))
+last#last_adv))
@@ -2275 +2275 @@
-wroghte)
+wroghte#write_v)
@@ -2278 +2278 @@
-yow)))))
+yow#you_pron)))))
@@ -2284 +2284 @@
-I))
+I#i_pron))
@@ -2286 +2286 @@
-thancke)
+thancke#thank_v)
@@ -2289 +2289 @@
-God)))
+God#NA)))
@@ -2294 +2294 @@
-my)
+my#my_adj)
@@ -2296 +2296 @@
-legg))
+legg#leg_n))
@@ -2298 +2298 @@
-is)
+is#be_v)
@@ -2301 +2301 @@
-much)
+much#much_adj)
@@ -2303 +2303 @@
-stronger)))
+stronger#strong_adj)))
@@ -2309 +2309 @@
-and)
+and#and_conj1)
@@ -2313 +2313 @@
-doth)
+doth#do_v)
@@ -2316 +2316 @@
-daylie))
+daylie#daily_adv))
@@ -2318 +2318 @@
-increase)
+increase#increase_v)
@@ -2321 +2321 @@
-in)
+in#in_prep)
@@ -2324 +2324 @@
-strength)))
+strength#strength_n)))
@@ -2329 +2329 @@
-insomuch)
+insomuch#insomuch_adv)
@@ -2332 +2332 @@
-that)
+that#that_conj)
@@ -2336 +2336 @@
-I))
+I#i_pron))
@@ -2338 +2338 @@
-am)
+am#be_v)
@@ -2341 +2341 @@
-able)
+able#able_adj)
@@ -2344 +2344 @@
-to)
+to#to_prep)
@@ -2346 +2346 @@
-walke)
+walke#walk_v)
@@ -2349 +2349 @@
-a)
+a#a_adj2)
@@ -2351 +2351 @@
-staff))
+staff#staff_n1))
@@ -2354 +2354 @@
-reasonable)
+reasonable#reasonable_adj)
@@ -2358 +2358 @@
-well)))))))
+well#well_adv)))))))
@@ -2367 +2367 @@
-only)
+only#only_adj)
@@ -2369 +2369 @@
-my)
+my#my_adj)
@@ -2371 +2371 @@
-knee))
+knee#knee_n))
@@ -2373 +2373 @@
-is)
+is#be_v)
@@ -2375 +2375 @@
-not)
+not#not_adv)
@@ -2378 +2378 @@
-yet))
+yet#yet_adv))
@@ -2380 +2380 @@
-recovred)
+recovred#recover_v1)
@@ -2388 +2388 @@
-but)
+but#but_prep)
@@ -2391 +2391 @@
-I))
+I#i_pron))
@@ -2393 +2393 @@
-hope)
+hope#hope_v)
@@ -2400 +2400 @@
-as)
+as#as_adv)
@@ -2407 +2407 @@
-strength))
+strength#strength_n))
@@ -2409 +2409 @@
-doth)
+doth#do_v)
@@ -2411 +2411 @@
-come))))
+come#come_v))))
@@ -2414 +2414 @@
-that))
+that#that_pron))
@@ -2416 +2416 @@
-will)
+will#will_v1)
@@ -2418 +2418 @@
-mende)))
+mende#mend_v)))
@@ -2427 +2427 @@
-As)
+As#as_adv)
@@ -2430 +2430 @@
-yet)))
+yet#yet_adv)))
@@ -2433 +2433 @@
-wee))
+wee#we_pron))
@@ -2435 +2435 @@
-have)
+have#have_v)
@@ -2437 +2437 @@
-receaved)
+receaved#receive_v)
@@ -2440 +2440 @@
-no)
+no#no_adj)
@@ -2442 +2442 @@
-newes)
+newes#news_n)
@@ -2445 +2445 @@
-of)
+of#of_prep)
@@ -2448 +2448 @@
-our)
+our#our_pron)
@@ -2450 +2450 @@
-bussines)
+bussines#business_n)
@@ -2453 +2453 @@
-in)
+in#in_prep)
@@ -2456 +2456 @@
-Swede)))))))
+Swede#NA)))))))
@@ -2462 +2462 @@
-but)
+but#but_prep)
@@ -2465 +2465 @@
-wee))
+wee#we_pron))
@@ -2467 +2467 @@
-do)
+do#do_v)
@@ -2470 +2470 @@
-dayly))
+dayly#daily_adv))
@@ -2472 +2472 @@
-expect)
+expect#expect_v)
@@ -2476 +2476 @@
-the)
+the#the_adj)
@@ -2478 +2478 @@
-comming)
+comming#coming_n)
@@ -2481 +2481 @@
-of)
+of#of_prep)
@@ -2484 +2484 @@
-Collonel)
+Collonel#NA)
@@ -2486 +2486 @@
-Knephusen)))
+Knephusen#NA)))
@@ -2489 +2489 @@
-from)
+from#from_prep)
@@ -2492 +2492 @@
-theance))))
+theance#thence_adv))))
@@ -2495 +2495 @@
-or)
+or#or_conj2)
@@ -2497 +2497 @@
-else)
+else#else_adv)
@@ -2500 +2500 @@
-his)
+his#his_adj)
@@ -2503 +2503 @@
-sending)
+sending#send_v1)
@@ -2506 +2506 @@
-hither))))))
+hither#hither_adv))))))
@@ -2515 +2515 @@
-I))
+I#i_pron))
@@ -2517 +2517 @@
-hope)
+hope#hope_v)
@@ -2524 +2524 @@
-wee))
+wee#we_pron))
@@ -2526 +2526 @@
-shall)
+shall#shall_v)
@@ -2528 +2528 @@
-have)
+have#have_v)
@@ -2531 +2531 @@
-such)
+such#such_adj)
@@ -2533 +2533 @@
-contentment)
+contentment#contentment_n)
@@ -2536 +2536 @@
-that)
+that#that_conj)
@@ -2540 +2540 @@
-wee))
+wee#we_pron))
@@ -2542 +2542 @@
-may)
+may#may_v1)
@@ -2544 +2544 @@
-goe)
+goe#go_v)
@@ -2546 +2546 @@
-over)
+over#over_adv)
@@ -2549 +2549 @@
-to)
+to#to_prep)
@@ -2551 +2551 @@
-searve)
+searve#serve_v1)
@@ -2554 +2554 @@
-that)
+that#that_pron)
@@ -2556 +2556 @@
-kinge))
+kinge#king_n))
@@ -2559 +2559 @@
-next)
+next#next_adj)
@@ -2561 +2561 @@
-springe)))
+springe#spring_n1)))
@@ -2567 +2567 @@
-which))
+which#which_adj))
@@ -2575 +2575 @@
-I))
+I#i_pron))
@@ -2577 +2577 @@
-doe)
+doe#do_v)
@@ -2579 +2579 @@
-desire))))))))
+desire#desire_v))))))))
@@ -2587 +2587 @@
-and)
+and#and_conj1)
@@ -2591 +2591 @@
-shall)
+shall#shall_v)
@@ -2594 +2594 @@
-willingly))
+willingly#willingly_adv))
@@ -2596 +2596 @@
-imbrace)
+imbrace#embrace_v2)
@@ -2599 +2599 @@
-any)
+any#any_adj)
@@ -2601 +2601 @@
-reasonable)
+reasonable#reasonable_adj)
@@ -2603 +2603 @@
-condicions))
+condicions#condition_n))
@@ -2606 +2606 @@
-for)
+for#for_prep)
@@ -2609 +2609 @@
-my)
+my#my_adj)
@@ -2611 +2611 @@
-parte)))
+parte#part_n)))
@@ -2620 +2620 @@
-I))
+I#i_pron))
@@ -2622 +2622 @@
-doe)
+doe#do_v)
@@ -2624 +2624 @@
-not)
+not#not_adv)
@@ -2626 +2626 @@
-heare)
+heare#hear_v)
@@ -2629 +2629 @@
-of)
+of#of_prep)
@@ -2632 +2632 @@
-any)
+any#any_adj)
@@ -2634 +2634 @@
-newes)
+newes#news_n)
@@ -2637 +2637 @@
-worth)
+worth#worth_adj)
@@ -2640 +2640 @@
-the)
+the#the_adj)
@@ -2642 +2642 @@
-wrighting)
+wrighting#writing_n)
@@ -2645 +2645 @@
-at)
+at#at_prep)
@@ -2648 +2648 @@
-this)
+this#this_pron)
@@ -2650 +2650 @@
-time)))))))
+time#time_n)))))))
@@ -2659 +2659 @@
-Thus))
+Thus#thus_adv))
@@ -2662 +2662 @@
-with)
+with#with_prep)
@@ -2666 +2666 @@
-my)
+my#my_adj)
@@ -2668 +2668 @@
-humble)
+humble#humble_adj)
@@ -2670 +2670 @@
-dutie))
+dutie#duty_n))
@@ -2672 +2672 @@
-remembred)))
+remembred#remember_v)))
@@ -2677 +2677 @@
-I))
+I#i_pron))
@@ -2679 +2679 @@
-commend)
+commend#commend_v)
@@ -2682 +2682 @@
-yow))
+yow#you_pron))
@@ -2685 +2685 @@
-to)
+to#to_prep)
@@ -2688 +2688 @@
-the)
+the#the_adj)
@@ -2690 +2690 @@
-tuition)
+tuition#tuition_n)
@@ -2693 +2693 @@
-of)
+of#of_prep)
@@ -2696 +2696 @@
-the)
+the#the_adj)
@@ -2698 +2698 @@
-allmightie)))))
+allmightie#almighty_adj)))))
@@ -2703 +2703 @@
-resting)
+resting#rest_v2)
@@ -2706 +2706 @@
-Your)
+Your#your_pron)
@@ -2708 +2708 @@
-obedient)
+obedient#obedient_adj)
@@ -2710 +2710 @@
-son)
+son#son_n1)
@@ -2713 +2713 @@
-John)
+John#NA)
@@ -2715 +2715 @@
-Barrington)))))
+Barrington#NA)))))
@@ -2721 +2721 @@
-Westminster)
+Westminster#NA)
@@ -2729 +2729 @@
-13th)
+13th#NA)
@@ -2731 +2731 @@
-October)
+October#NA)
@@ -2735 +2735 @@
-1629)
+1629#NA)
@@ -2744 +2744 @@
-To)
+To#to_prep)
@@ -2747 +2747 @@
-my)
+my#my_adj)
@@ -2750 +2750 @@
-very)
+very#very_adj)
@@ -2752 +2752 @@
-good))
+good#good_adj))
@@ -2754 +2754 @@
-freind)
+freind#friend_n)
@@ -2757 +2757 @@
-Mr)
+Mr#NA)
@@ -2759 +2759 @@
-Issac)
+Issac#NA)
@@ -2761 +2761 @@
-Ewer))
+Ewer#NA))
@@ -2766 +2766 @@
-servant)
+servant#servant_n)
@@ -2769 +2769 @@
-to)
+to#to_prep)
@@ -2772 +2772 @@
-the)
+the#the_adj)
@@ -2774 +2774 @@
-ladie)
+ladie#lady_n)
@@ -2777 +2777 @@
-Barrington)))))))
+Barrington#NA)))))))
@@ -2782 +2782 @@
-at)
+at#at_prep)
@@ -2786 +2786 @@
-Sir)
+Sir#NA)
@@ -2788 +2788 @@
-Gilbert)
+Gilbert#NA)
@@ -2790 +2790 @@
-Gerrard's))))
+Gerrard's#NA))))
@@ -2793 +2793 @@
-in)
+in#in_prep)
@@ -2796 +2796 @@
-Harrow)
+Harrow#NA)
@@ -2798 +2798 @@
-Hill))))
+Hill#NA))))
