--- jopinney-1688-e3-h.psd.vert	2025-09-10 13:50:35
+++ jopinney-1688-e3-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Daughter)
+Daughter#daughter_n)
@@ -16 +16 @@
-Hester))
+Hester#NA))
@@ -19 +19 @@
-Yours))
+Yours#yours_pron))
@@ -22 +22 @@
-I))
+I#i_pron))
@@ -24 +24 @@
-received))
+received#receive_v))
@@ -30 +30 @@
-and)
+and#and_conj1)
@@ -34 +34 @@
-have)
+have#have_v)
@@ -37 +37 @@
-such)
+such#such_adj)
@@ -39 +39 @@
-a)
+a#a_adj2)
@@ -41 +41 @@
-Narrative)
+Narrative#narrative_n)
@@ -44 +44 @@
-of)
+of#of_prep)
@@ -47 +47 @@
-the)
+the#the_adj)
@@ -49 +49 @@
-Scot)))
+Scot#NA)))
@@ -54 +54 @@
-from)
+from#from_prep)
@@ -57 +57 @@
-your)
+your#your_pron)
@@ -59 +59 @@
-Mother)))
+Mother#mother_n1)))
@@ -62 +62 @@
-that)
+that#that_conj)
@@ -66 +66 @@
-I))
+I#i_pron))
@@ -68 +68 @@
-would)
+would#will_v1)
@@ -70 +70 @@
-have)
+have#have_v)
@@ -73 +73 @@
-neither)
+neither#neither_adv)
@@ -76 +76 @@
-of)
+of#of_prep)
@@ -79 +79 @@
-you))))
+you#you_pron))))
@@ -82 +82 @@
-to)
+to#to_prep)
@@ -84 +84 @@
-have)
+have#have_v)
@@ -87 +87 @@
-to)
+to#to_prep)
@@ -89 +89 @@
-doe)
+doe#do_v)
@@ -92 +92 @@
-with)
+with#with_prep)
@@ -95 +95 @@
-her)))
+her#her_pron1)))
@@ -98 +98 @@
-at)
+at#at_prep)
@@ -101 +101 @@
-all)))))))
+all#all_adj)))))))
@@ -110 +110 @@
-She))
+She#she_pron))
@@ -112 +112 @@
-said)
+said#say_v1)
@@ -121 +121 @@
-she))
+she#she_pron))
@@ -123 +123 @@
-is)
+is#be_v)
@@ -125 +125 @@
-come)
+come#come_v)
@@ -127 +127 @@
-in)
+in#in_adv)
@@ -130 +130 @@
-among)
+among#among_prep)
@@ -133 +133 @@
-a)
+a#a_adj2)
@@ -135 +135 @@
-parcell)
+parcell#parcel_n)
@@ -138 +138 @@
-of)
+of#of_prep)
@@ -141 +141 @@
-beggars)))))))
+beggars#beggar_n)))))))
@@ -149 +149 @@
-and)
+and#and_conj1)
@@ -152 +152 @@
-if)
+if#if_conj)
@@ -155 +155 @@
-so)))
+so#so_adv)))
@@ -160 +160 @@
-t@))
+t@#it_pron))
@@ -162 +162 @@
-@is)
+@is#be_v)
@@ -166 +166 @@
-but)
+but#but_prep)
@@ -168 +168 @@
-a)
+a#a_adj2)
@@ -170 +170 @@
-thiefe))
+thiefe#thief_n))
@@ -173 +173 @@
-and)
+and#and_conj1)
@@ -176 +176 @@
-a)
+a#a_adj2)
@@ -178 +178 @@
-beggar)))
+beggar#beggar_n)))
@@ -181 +181 @@
-come)
+come#come_v)
@@ -184 +184 @@
-unhappily))
+unhappily#unhappily_adv))
@@ -187 +187 @@
-among)
+among#among_prep)
@@ -190 +190 @@
-beggars))))))
+beggars#beggar_n))))))
@@ -198 +198 @@
-for)
+for#for_prep)
@@ -201 +201 @@
-her)
+her#her_pron2)
@@ -203 +203 @@
-mother))
+mother#mother_n1))
@@ -205 +205 @@
-was)
+was#be_v)
@@ -207 +207 @@
-maintained)
+maintained#maintain_v)
@@ -210 +210 @@
-here))
+here#here_adv))
@@ -213 +213 @@
-by)
+by#by_prep)
@@ -216 +216 @@
-public)
+public#public_adj)
@@ -218 +218 @@
-almes)))
+almes#alms_n)))
@@ -229 +229 @@
-Her)
+Her#her_pron2)
@@ -231 +231 @@
-imperious)
+imperious#imperious_adj)
@@ -233 +233 @@
-pride)
+pride#pride_n1)
@@ -236 +236 @@
-and)
+and#and_conj1)
@@ -239 +239 @@
-falsnes))))
+falsnes#falseness_n))))
@@ -241 +241 @@
-make)
+make#make_v1)
@@ -245 +245 @@
-her))
+her#her_pron1))
@@ -248 +248 @@
-incapable)
+incapable#incapable_adj)
@@ -251 +251 @@
-of)
+of#of_prep)
@@ -254 +254 @@
-your)
+your#your_pron)
@@ -256 +256 @@
-commerce)
+commerce#commerce_n)
@@ -259 +259 @@
-with)
+with#with_prep)
@@ -262 +262 @@
-her)))))))
+her#her_pron1)))))))
@@ -271 +271 @@
-I))
+I#i_pron))
@@ -273 +273 @@
-hear)
+hear#hear_v)
@@ -280 +280 @@
-she))
+she#she_pron))
@@ -282 +282 @@
-did)
+did#do_v)
@@ -284 +284 @@
-borrow)
+borrow#borrow_v1)
@@ -287 +287 @@
-money))
+money#money_n))
@@ -290 +290 @@
-of)
+of#of_prep)
@@ -293 +293 @@
-Mr.)
+Mr.#NA)
@@ -295 +295 @@
-Busbie)))
+Busbie#NA)))
@@ -298 +298 @@
-being)
+being#be_v)
@@ -301 +301 @@
-lately))
+lately#lately_adv))
@@ -304 +304 @@
-in)
+in#in_prep)
@@ -307 +307 @@
-London))))))
+London#NA))))))
@@ -316 +316 @@
-if)
+if#if_conj)
@@ -323 +323 @@
-she))
+she#she_pron))
@@ -325 +325 @@
-thinkes)
+thinkes#think_v2)
@@ -332 +332 @@
-I))
+I#i_pron))
@@ -334 +334 @@
-shall)
+shall#shall_v)
@@ -336 +336 @@
-pay)
+pay#pay_v1)
@@ -339 +339 @@
-it)))))))
+it#it_pron)))))))
@@ -344 +344 @@
-she))
+she#she_pron))
@@ -346 +346 @@
-is)
+is#be_v)
@@ -348 +348 @@
-deceived)
+deceived#deceive_v)
@@ -356 +356 @@
-Let)
+Let#let_v1)
@@ -361 +361 @@
-her))
+her#her_pron1))
@@ -363 +363 @@
-get)
+get#get_v)
@@ -366 +366 @@
-her)
+her#her_pron2)
@@ -368 +368 @@
-owne)
+owne#own_adj)
@@ -370 +370 @@
-bread))
+bread#bread_n))
@@ -373 +373 @@
-by)
+by#by_prep)
@@ -376 +376 @@
-the)
+the#the_adj)
@@ -378 +378 @@
-Sweat)
+Sweat#sweat_n)
@@ -381 +381 @@
-of)
+of#of_prep)
@@ -384 +384 @@
-her)
+her#her_pron2)
@@ -386 +386 @@
-browes)))))
+browes#brow_n1)))))
@@ -389 +389 @@
-as)
+as#as_adv)
@@ -400 +400 @@
-I))
+I#i_pron))
@@ -402 +402 @@
-have)
+have#have_v)
@@ -407 +407 @@
-and)
+and#and_conj1)
@@ -412 +412 @@
-pay)
+pay#pay_v1)
@@ -415 +415 @@
-her)
+her#her_pron2)
@@ -417 +417 @@
-own)
+own#own_adj)
@@ -419 +419 @@
-debts)))))
+debts#debt_n)))))
@@ -428 +428 @@
-Her)
+Her#her_pron2)
@@ -430 +430 @@
-ill)
+ill#ill_adj)
@@ -432 +432 @@
-character)
+character#character_n)
@@ -435 +435 @@
-and)
+and#and_conj1)
@@ -438 +438 @@
-carriage))))
+carriage#carriage_n))))
@@ -440 +440 @@
-hath)
+hath#have_v)
@@ -442 +442 @@
-disobliged)
+disobliged#disoblige_v)
@@ -445 +445 @@
-me))
+me#me_pron1))
@@ -448 +448 @@
-to)
+to#to_prep)
@@ -451 +451 @@
-her)))
+her#her_pron1)))
@@ -454 +454 @@
-for)
+for#for_prep)
@@ -457 +457 @@
-Ever)))
+Ever#ever_adv)))
@@ -465 +465 @@
-And)
+And#and_conj1)
@@ -468 +468 @@
-if)
+if#if_conj)
@@ -475 +475 @@
-service)
+service#service_n1)
@@ -479 +479 @@
-be)
+be#be_v)
@@ -482 +482 @@
-to)
+to#too_adv)
@@ -484 +484 @@
-mean)
+mean#mean_adj1)
@@ -487 +487 @@
-for)
+for#for_prep)
@@ -490 +490 @@
-her))))
+her#her_pron1))))
@@ -493 +493 @@
-now))
+now#now_adv))
@@ -501 +501 @@
-by)
+by#by_prep)
@@ -504 +504 @@
-which)))
+which#which_adj)))
@@ -512 +512 @@
-she))
+she#she_pron))
@@ -515 +515 @@
-alwayes))
+alwayes#always_adv))
@@ -517 +517 @@
-lived)
+lived#live_v1)
@@ -520 +520 @@
-here)))))))
+here#here_adv)))))))
@@ -524 +524 @@
-let)
+let#let_v1)
@@ -528 +528 @@
-her))
+her#her_pron1))
@@ -530 +530 @@
-hunger)
+hunger#hunger_v)
@@ -533 +533 @@
-for)
+for#for_prep)
@@ -536 +536 @@
-me))))
+me#me_pron1))))
@@ -545 +545 @@
-I))
+I#i_pron))
@@ -547 +547 @@
-have)
+have#have_v)
@@ -550 +550 @@
-enough)
+enough#enough_adj)
@@ -553 +553 @@
-of)
+of#of_prep)
@@ -556 +556 @@
-her))))
+her#her_pron1))))
@@ -559 +559 @@
-already))
+already#already_adj))
@@ -568 +568 @@
-You))
+You#you_pron))
@@ -570 +570 @@
-write)
+write#write_v)
@@ -573 +573 @@
-of)
+of#of_prep)
@@ -576 +576 @@
-returning)
+returning#return_v1)
@@ -579 +579 @@
-your)
+your#your_pron)
@@ -581 +581 @@
-money))
+money#money_n))
@@ -584 +584 @@
-to)
+to#to_prep)
@@ -587 +587 @@
-Nath.))))))
+Nath.#NA))))))
@@ -594 +594 @@
-I))
+I#i_pron))
@@ -596 +596 @@
-would)
+would#will_v1)
@@ -598 +598 @@
-have)
+have#have_v)
@@ -601 +601 @@
-you))
+you#you_pron))
@@ -604 +604 @@
-to)
+to#to_prep)
@@ -606 +606 @@
-have)
+have#have_v)
@@ -609 +609 @@
-security))
+security#security_n))
@@ -612 +612 @@
-of)
+of#of_prep)
@@ -615 +615 @@
-him))))
+him#him_pron))))
@@ -626 +626 @@
-for)
+for#for_prep)
@@ -629 +629 @@
-though)
+though#though_adv)
@@ -636 +636 @@
-your)
+your#your_pron)
@@ -638 +638 @@
-mother))
+mother#mother_n1))
@@ -640 +640 @@
-have)
+have#have_v)
@@ -642 +642 @@
-trusted)
+trusted#trust_v)
@@ -645 +645 @@
-much))
+much#much_adj))
@@ -648 +648 @@
-in)
+in#in_prep)
@@ -651 +651 @@
-his)
+his#his_adj)
@@ -653 +653 @@
-handes))))))
+handes#hand_n))))))
@@ -656 +656 @@
-yet))
+yet#yet_adv))
@@ -659 +659 @@
-she))
+she#she_pron))
@@ -661 +661 @@
-hath)
+hath#have_v)
@@ -664 +664 @@
-something)
+something#something_n)
@@ -667 +667 @@
-left)))
+left#leave_v1)))
@@ -670 +670 @@
-If)
+If#if_conj)
@@ -677 +677 @@
-that))
+that#that_pron))
@@ -679 +679 @@
-should)
+should#shall_v)
@@ -681 +681 @@
-miscarry))))
+miscarry#miscarry_v))))
@@ -691 +691 @@
-For)
+For#for_prep)
@@ -694 +694 @@
-Rachel)))
+Rachel#NA)))
@@ -697 +697 @@
-I))
+I#i_pron))
@@ -699 +699 @@
-can@)
+can@#can_v1)
@@ -701 +701 @@
-@not)
+@not#not_adv)
@@ -704 +704 @@
-so)
+so#so_adv)
@@ -706 +706 @@
-well))
+well#well_adv))
@@ -708 +708 @@
-advise)
+advise#advise_v)
@@ -711 +711 @@
-her)))
+her#her_pron1)))
@@ -714 +714 @@
-nor)
+nor#nor_conj1)
@@ -717 +717 @@
-know)
+know#know_v)
@@ -721 +721 @@
-in)
+in#in_prep)
@@ -724 +724 @@
-what)
+what#what_pron)
@@ -726 +726 @@
-Capacity)))
+Capacity#capacity_n)))
@@ -734 +734 @@
-she))
+she#she_pron))
@@ -736 +736 @@
-can)
+can#can_v1)
@@ -738 +738 @@
-enter)
+enter#enter_v)
@@ -741 +741 @@
-againe))
+againe#again_adv))
@@ -744 +744 @@
-upon)
+upon#upon_prep)
@@ -747 +747 @@
-the)
+the#the_adj)
@@ -749 +749 @@
-Exchange)))))))
+Exchange#exchange_n)))))))
@@ -758 +758 @@
-To)
+To#to_prep)
@@ -760 +760 @@
-doe)
+doe#do_v)
@@ -763 +763 @@
-it))
+it#it_pron))
@@ -766 +766 @@
-contemptibly)))
+contemptibly#contemptibly_adv)))
@@ -769 +769 @@
-I))
+I#i_pron))
@@ -771 +771 @@
-would)
+would#will_v1)
@@ -773 +773 @@
-not)
+not#not_adv)
@@ -775 +775 @@
-advise)
+advise#advise_v)
@@ -778 +778 @@
-her))
+her#her_pron1))
@@ -786 +786 @@
-but)
+but#but_prep)
@@ -789 +789 @@
-if)
+if#if_conj)
@@ -792 +792 @@
-with)
+with#with_prep)
@@ -795 +795 @@
-Credit))))
+Credit#credit_n))))
@@ -798 +798 @@
-I))
+I#i_pron))
@@ -800 +800 @@
-shall)
+shall#shall_v)
@@ -802 +802 @@
-not)
+not#not_adv)
@@ -804 +804 @@
-be)
+be#be_v)
@@ -807 +807 @@
-against)
+against#against_prep)
@@ -810 +810 @@
-it)))
+it#it_pron)))
@@ -818 +818 @@
-But)
+But#but_prep)
@@ -821 +821 @@
-not)
+not#not_adv)
@@ -823 +823 @@
-to)
+to#to_prep)
@@ -825 +825 @@
-medle)
+medle#meddle_v)
@@ -828 +828 @@
-with)
+with#with_prep)
@@ -831 +831 @@
-the)
+the#the_adj)
@@ -833 +833 @@
-Scot))))
+Scot#NA))))
@@ -842 +842 @@
-I))
+I#i_pron))
@@ -844 +844 @@
-will)
+will#will_v1)
@@ -847 +847 @@
-rather))
+rather#rather_adv))
@@ -849 +849 @@
-maintaine)
+maintaine#maintain_v)
@@ -852 +852 @@
-her))
+her#her_pron1))
@@ -857 +857 @@
-though)
+though#though_adv)
@@ -864 +864 @@
-she))
+she#she_pron))
@@ -866 +866 @@
-hath)
+hath#have_v)
@@ -868 +868 @@
-grieved)
+grieved#grieve_v)
@@ -871 +871 @@
-me)))))
+me#me_pron1)))))
@@ -882 +882 @@
-I))
+I#i_pron))
@@ -884 +884 @@
-shall)
+shall#shall_v)
@@ -886 +886 @@
-not)
+not#not_adv)
@@ -888 +888 @@
-be)
+be#be_v)
@@ -891 +891 @@
-able)
+able#able_adj)
@@ -894 +894 @@
-to)
+to#to_prep)
@@ -896 +896 @@
-get)
+get#get_v)
@@ -898 +898 @@
-off)
+off#off_adv)
@@ -901 +901 @@
-here))
+here#here_adv))
@@ -904 +904 @@
-as)
+as#as_adv)
@@ -907 +907 @@
-yet)))))
+yet#yet_adv)))))
@@ -912 +912 @@
-though)
+though#though_adv)
@@ -919 +919 @@
-I))
+I#i_pron))
@@ -921 +921 @@
-desire)
+desire#desire_v)
@@ -924 +924 @@
-to)
+to#to_prep)
@@ -926 +926 @@
-doe)
+doe#do_v)
@@ -929 +929 @@
-it))))))
+it#it_pron))))))
@@ -938 +938 @@
-if)
+if#if_conj)
@@ -945 +945 @@
-any)
+any#any_adj)
@@ -948 +948 @@
-of)
+of#of_prep)
@@ -951 +951 @@
-you))))
+you#you_pron))))
@@ -953 +953 @@
-will)
+will#will_v1)
@@ -955 +955 @@
-come)
+come#come_v)
@@ -958 +958 @@
-to)
+to#to_prep)
@@ -961 +961 @@
-me))))))
+me#me_pron1))))))
@@ -964 +964 @@
-you))
+you#you_pron))
@@ -966 +966 @@
-shall)
+shall#shall_v)
@@ -968 +968 @@
-be)
+be#be_v)
@@ -971 +971 @@
-Welcome))
+Welcome#welcome_n1))
@@ -980 +980 @@
-The)
+The#the_adj)
@@ -982 +982 @@
-rescuing)
+rescuing#rescuing_n)
@@ -985 +985 @@
-of)
+of#of_prep)
@@ -988 +988 @@
-us)))
+us#us_pron)))
@@ -991 +991 @@
-from)
+from#from_prep)
@@ -994 +994 @@
-the)
+the#the_adj)
@@ -996 +996 @@
-housbreaker))))
+housbreaker#housebreaker_n))))
@@ -998 +998 @@
-is)
+is#be_v)
@@ -1001 +1001 @@
-a)
+a#a_adj2)
@@ -1003 +1003 @@
-great)
+great#great_adj)
@@ -1005 +1005 @@
-providence)
+providence#providence_n)
@@ -1008 +1008 @@
-of)
+of#of_prep)
@@ -1011 +1011 @@
-God))))
+God#NA))))
@@ -1020 +1020 @@
-His)
+His#his_adj)
@@ -1022 +1022 @@
-end))
+end#end_n))
@@ -1024 +1024 @@
-was)
+was#be_v)
@@ -1027 +1027 @@
-to)
+to#to_prep)
@@ -1029 +1029 @@
-have)
+have#have_v)
@@ -1031 +1031 @@
-ruined)
+ruined#ruin_v)
@@ -1034 +1034 @@
-us))))
+us#us_pron))))
@@ -1040 +1040 @@
-and)
+and#and_conj1)
@@ -1043 +1043 @@
-he))
+he#he_pron))
@@ -1045 +1045 @@
-had)
+had#have_v)
@@ -1047 +1047 @@
-don)
+don#do_v)
@@ -1050 +1050 @@
-it))
+it#it_pron))
@@ -1055 +1055 @@
-if)
+if#if_conj)
@@ -1062 +1062 @@
-I))
+I#i_pron))
@@ -1064 +1064 @@
-had)
+had#have_v)
@@ -1066 +1066 @@
-not)
+not#not_adv)
@@ -1069 +1069 @@
-strictly))
+strictly#strictly_adv))
@@ -1071 +1071 @@
-written)
+written#write_v)
@@ -1074 +1074 @@
-to)
+to#to_prep)
@@ -1077 +1077 @@
-your)
+your#your_pron)
@@ -1079 +1079 @@
-mother)))
+mother#mother_n1)))
@@ -1082 +1082 @@
-from)
+from#from_prep)
@@ -1085 +1085 @@
-hence)))
+hence#hence_adv)))
@@ -1088 +1088 @@
-that)
+that#that_conj)
@@ -1092 +1092 @@
-she))
+she#she_pron))
@@ -1094 +1094 @@
-should)
+should#shall_v)
@@ -1096 +1096 @@
-send)
+send#send_v1)
@@ -1099 +1099 @@
-out)
+out#out_adv)
@@ -1101 +1101 @@
-of)
+of#of_prep)
@@ -1104 +1104 @@
-the)
+the#the_adj)
@@ -1106 +1106 @@
-house)))
+house#house_n1)))
@@ -1109 +1109 @@
-all)
+all#all_adj)
@@ -1111 +1111 @@
-my)
+my#my_adj)
@@ -1113 +1113 @@
-writings)
+writings#writing_n)
@@ -1116 +1116 @@
-Especially)
+Especially#especially_adv)
@@ -1118 +1118 @@
-such)
+such#such_adj)
@@ -1121 +1121 @@
-as)
+as#as_adv)
@@ -1131 +1131 @@
-concerned)
+concerned#concern_v)
@@ -1134 +1134 @@
-the)
+the#the_adj)
@@ -1136 +1136 @@
-Rogue))))))))))))
+Rogue#rogue_n))))))))))))
@@ -1146 +1146 @@
-Had)
+Had#have_v)
@@ -1149 +1149 @@
-he))
+he#he_pron))
@@ -1151 +1151 @@
-gotten)
+gotten#get_v)
@@ -1154 +1154 @@
-them))
+them#them_pron))
@@ -1160 +1160 @@
-which))
+which#which_adj))
@@ -1167 +1167 @@
-was)
+was#be_v)
@@ -1170 +1170 @@
-his)
+his#his_adj)
@@ -1172 +1172 @@
-aime))))))
+aime#aim_n))))))
@@ -1177 +1177 @@
-he))
+he#he_pron))
@@ -1179 +1179 @@
-had)
+had#have_v)
@@ -1181 +1181 @@
-don)
+don#do_v)
@@ -1184 +1184 @@
-his)
+his#his_adj)
@@ -1186 +1186 @@
-purpose))
+purpose#purpose_n))
@@ -1195 +1195 @@
-God))
+God#NA))
@@ -1197 +1197 @@
-hath)
+hath#have_v)
@@ -1199 +1199 @@
-saved)
+saved#save_v)
@@ -1202 +1202 @@
-us)))
+us#us_pron)))
@@ -1208 +1208 @@
-and)
+and#and_conj1)
@@ -1211 +1211 @@
-I))
+I#i_pron))
@@ -1213 +1213 @@
-hope)
+hope#hope_v)
@@ -1216 +1216 @@
-to)
+to#to_prep)
@@ -1219 +1219 @@
-no)
+no#no_adj)
@@ -1221 +1221 @@
-great)
+great#great_adj)
@@ -1223 +1223 @@
-loss)))
+loss#loss_n1)))
@@ -1226 +1226 @@
-unless)
+unless#unless_adv)
@@ -1229 +1229 @@
-the)
+the#the_adj)
@@ -1231 +1231 @@
-Wast)
+Wast#waste_n)
@@ -1242 +1242 @@
-they))
+they#they_pron))
@@ -1244 +1244 @@
-made)
+made#make_v1)
@@ -1247 +1247 @@
-on)
+on#on_prep)
@@ -1250 +1250 @@
-your)
+your#your_pron)
@@ -1252 +1252 @@
-sister)
+sister#sister_n)
@@ -1255 +1255 @@
-Hoare))))))))
+Hoare#NA))))))))
@@ -1263 +1263 @@
-Remember)
+Remember#remember_v)
@@ -1266 +1266 @@
-me))
+me#me_pron1))
@@ -1269 +1269 @@
-to)
+to#to_prep)
@@ -1272 +1272 @@
-Rachel)))
+Rachel#NA)))
@@ -1281 +1281 @@
-God)
+God#NA)
@@ -1284 +1284 @@
-almighty)))
+almighty#almighty_adj)))
@@ -1287 +1287 @@
-bless)
+bless#bless_v1)
@@ -1289 +1289 @@
-and)
+and#and_conj1)
@@ -1291 +1291 @@
-keep))
+keep#keep_v))
@@ -1294 +1294 @@
-you))
+you#you_pron))
@@ -1298 +1298 @@
-which))
+which#which_adj))
@@ -1305 +1305 @@
-is)
+is#be_v)
@@ -1308 +1308 @@
-the)
+the#the_adj)
@@ -1310 +1310 @@
-dayly)
+dayly#daily_adj)
@@ -1312 +1312 @@
-prayer)
+prayer#prayer_n1)
@@ -1315 +1315 @@
-of)
+of#of_prep)
@@ -1318 +1318 @@
-Your)
+Your#your_pron)
@@ -1320 +1320 @@
-Aged)
+Aged#aged_adj)
@@ -1322 +1322 @@
-father)
+father#father_n)
@@ -1325 +1325 @@
-Jo.)
+Jo.#NA)
@@ -1327 +1327 @@
-Pinney))))))))
+Pinney#NA))))))))
@@ -1333 +1333 @@
-Cut)
+Cut#cut_v)
@@ -1336 +1336 @@
-this))
+this#this_pron))
@@ -1338 +1338 @@
-off))
+off#off_adv))
@@ -1344 +1344 @@
-and)
+and#and_conj1)
@@ -1346 +1346 @@
-send)
+send#send_v1)
@@ -1349 +1349 @@
-it))
+it#it_pron))
@@ -1352 +1352 @@
-to)
+to#to_prep)
@@ -1355 +1355 @@
-the)
+the#the_adj)
@@ -1357 +1357 @@
-Scott)))
+Scott#NA)))
@@ -1360 +1360 @@
-sealed)
+sealed#seal_v1)
@@ -1362 +1362 @@
-up))
+up#up_adv1))
