--- jpinney-1686-e3-h.psd.vert	2025-09-10 13:50:35
+++ jpinney-1686-e3-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-my)
+my#my_adj)
@@ -16 +16 @@
-dare)
+dare#dear_adj1)
@@ -18 +18 @@
-childe))
+childe#child_n))
@@ -21 +21 @@
-I))
+I#i_pron))
@@ -23 +23 @@
-hade)
+hade#have_v)
@@ -26 +26 @@
-great)
+great#great_adj)
@@ -28 +28 @@
-hopes)
+hopes#hope_n1)
@@ -31 +31 @@
-to)
+to#to_prep)
@@ -33 +33 @@
-have)
+have#have_v)
@@ -35 +35 @@
-seen)
+seen#see_v)
@@ -38 +38 @@
-yo=u=))
+yo=u=#you_pron))
@@ -41 +41 @@
-heare))
+heare#here_adv))
@@ -44 +44 @@
-eare)
+eare#ere_adv)
@@ -47 +47 @@
-this)
+this#this_pron)
@@ -49 +49 @@
-time))))))
+time#time_n))))))
@@ -55 +55 @@
-for)
+for#for_prep)
@@ -58 +58 @@
-I))
+I#i_pron))
@@ -60 +60 @@
-longe)
+longe#long_v1)
@@ -63 +63 @@
-gretly))
+gretly#great_adj))
@@ -66 +66 @@
-to)
+to#to_prep)
@@ -68 +68 @@
-see)
+see#see_v)
@@ -71 +71 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -81 +81 @@
-I))
+I#i_pron))
@@ -83 +83 @@
-long)
+long#long_v1)
@@ -86 +86 @@
-to)
+to#to_prep)
@@ -88 +88 @@
-see)
+see#see_v)
@@ -91 +91 @@
-yo=u=)
+yo=u=#you_pron)
@@ -94 +94 @@
-all)))))
+all#all_adj)))))
@@ -97 +97 @@
-but)
+but#but_prep)
@@ -101 +101 @@
-espesphilly))
+espesphilly#especially_adv))
@@ -104 +104 @@
-yo=u=))))
+yo=u=#you_pron))))
@@ -112 +112 @@
-for)
+for#for_prep)
@@ -115 +115 @@
-I))
+I#i_pron))
@@ -117 +117 @@
-am)
+am#be_v)
@@ -119 +119 @@
-trobelled)
+trobelled#trouble_v)
@@ -122 +122 @@
-gretly))
+gretly#great_adj))
@@ -128 +128 @@
-what))
+what#what_pron))
@@ -136 +136 @@
-yo=u=))
+yo=u=#you_pron))
@@ -138 +138 @@
-can)
+can#can_v1)
@@ -140 +140 @@
-doe)
+doe#do_v)
@@ -143 +143 @@
-wth)
+wth#with_prep)
@@ -146 +146 @@
-the)
+the#the_adj)
@@ -148 +148 @@
-workkers)))
+workkers#worker_n)))
@@ -151 +151 @@
-thes)
+thes#these_pron)
@@ -153 +153 @@
-times))))
+times#time_n))))
@@ -161 +161 @@
-for)
+for#for_prep)
@@ -164 +164 @@
-heare))
+heare#here_adv))
@@ -168 +168 @@
-is)
+is#be_v)
@@ -171 +171 @@
-no)
+no#no_adj)
@@ -173 +173 @@
-trade))
+trade#trade_n))
@@ -181 +181 @@
-and)
+and#and_conj1)
@@ -184 +184 @@
-I))
+I#i_pron))
@@ -187 +187 @@
-greatly))
+greatly#greatly_adv))
@@ -189 +189 @@
-fare)
+fare#fare_v1)
@@ -192 +192 @@
-that)
+that#that_conj)
@@ -196 +196 @@
-it))
+it#it_pron))
@@ -198 +198 @@
-is)
+is#be_v)
@@ -201 +201 @@
-as)
+as#as_adv)
@@ -203 +203 @@
-bad))
+bad#bad_adj))
@@ -206 +206 @@
-theare)))))
+theare#there_adv)))))
@@ -212 +212 @@
-and)
+and#and_conj1)
@@ -215 +215 @@
-I))
+I#i_pron))
@@ -217 +217 @@
-have)
+have#have_v)
@@ -219 +219 @@
-not)
+not#not_adv)
@@ -221 +221 @@
-harde)
+harde#hear_v)
@@ -224 +224 @@
-from)
+from#from_prep)
@@ -227 +227 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -230 +230 @@
-a@)
+a@#a_adj2)
@@ -232 +232 @@
-@long)
+@long#long_adj1)
@@ -234 +234 @@
-time))
+time#time_n))
@@ -239 +239 @@
-so)
+so#so_adv)
@@ -242 +242 @@
-that)
+that#that_conj)
@@ -246 +246 @@
-I))
+I#i_pron))
@@ -248 +248 @@
-am)
+am#be_v)
@@ -251 +251 @@
-in)
+in#in_prep)
@@ -254 +254 @@
-great)
+great#great_adj)
@@ -256 +256 @@
-fare)
+fare#fear_n)
@@ -259 +259 @@
-that)
+that#that_conj)
@@ -263 +263 @@
-yo=u=))
+yo=u=#you_pron))
@@ -265 +265 @@
-are)
+are#be_v)
@@ -267 +267 @@
-not)
+not#not_adv)
@@ -270 +270 @@
-well)))))))))
+well#well_adj)))))))))
@@ -279 +279 @@
-what))
+what#what_pron))
@@ -284 +284 @@
-to)
+to#to_prep)
@@ -286 +286 @@
-write)))
+write#write_v)))
@@ -292 +292 @@
-nothinge))
+nothinge#nothing_pron))
@@ -300 +300 @@
-how))
+how#how_adv))
@@ -308 +308 @@
-it))
+it#it_pron))
@@ -310 +310 @@
-is)
+is#be_v)
@@ -313 +313 @@
-wth)
+wth#with_prep)
@@ -316 +316 @@
-yo=u=)))))
+yo=u=#you_pron)))))
@@ -322 +322 @@
-whare)
+whare#whether_pron)
@@ -329 +329 @@
-yo=u=))
+yo=u=#you_pron))
@@ -331 +331 @@
-have)
+have#have_v)
@@ -334 +334 @@
-yo=r=)
+yo=r=#your_pron)
@@ -336 +336 @@
-helth))
+helth#health_n))
@@ -339 +339 @@
-after)
+after#after_adv)
@@ -342 +342 @@
-yo=r=)
+yo=r=#your_pron)
@@ -344 +344 @@
-fever))))
+fever#fever_n1))))
@@ -349 +349 @@
-or)
+or#or_conj2)
@@ -352 +352 @@
-no))))))
+no#no_adv2))))))
@@ -361 +361 @@
-I))
+I#i_pron))
@@ -363 +363 @@
-long)
+long#long_v1)
@@ -366 +366 @@
-to)
+to#to_prep)
@@ -368 +368 @@
-hear)
+hear#hear_v)
@@ -373 +373 @@
-how))
+how#how_adv))
@@ -381 +381 @@
-all)
+all#all_adj)
@@ -383 +383 @@
-things))
+things#thing_n1))
@@ -385 +385 @@
-goe)
+goe#go_v)
@@ -388 +388 @@
-thare))))
+thare#there_adv))))
@@ -394 +394 @@
-whear)
+whear#whether_pron)
@@ -400 +400 @@
-yo=u=))
+yo=u=#you_pron))
@@ -402 +402 @@
-are)
+are#be_v)
@@ -404 +404 @@
-all)
+all#all_adj)
@@ -407 +407 @@
-in)
+in#in_prep)
@@ -410 +410 @@
-helth))))))
+helth#health_n))))))
@@ -413 +413 @@
-and)
+and#and_conj1)
@@ -417 +417 @@
-how))
+how#how_adv))
@@ -426 +426 @@
-Azea))
+Azea#NA))
@@ -431 +431 @@
-and)
+and#and_conj1)
@@ -434 +434 @@
-his)
+his#his_adj)
@@ -436 +436 @@
-sister))))
+sister#sister_n))))
@@ -438 +438 @@
-doe)
+doe#do_v)
@@ -440 +440 @@
-accord))))
+accord#accord_v))))
@@ -443 +443 @@
-and)
+and#and_conj1)
@@ -447 +447 @@
-what))
+what#what_pron))
@@ -454 +454 @@
-is)
+is#be_v)
@@ -456 +456 @@
-becom)
+becom#become_v)
@@ -459 +459 @@
-of)
+of#of_prep)
@@ -462 +462 @@
-nubery))))))
+nubery#NA))))))
@@ -465 +465 @@
-and)
+and#and_conj1)
@@ -469 +469 @@
-how))
+how#how_adv))
@@ -477 +477 @@
-it))
+it#it_pron))
@@ -479 +479 @@
-goes)
+goes#go_v)
@@ -482 +482 @@
-w=th=)
+w=th=#with_prep)
@@ -486 +486 @@
-every)
+every#every_adj)
@@ -488 +488 @@
-one)
+one#one_adj)
@@ -491 +491 @@
-of)
+of#of_prep)
@@ -494 +494 @@
-yo=u=))))
+yo=u=#you_pron))))
@@ -500 +500 @@
-the)
+the#the_adj)
@@ -502 +502 @@
-childern))
+childern#child_n))
@@ -504 +504 @@
-also)))))))))
+also#also_adv)))))))))
@@ -513 +513 @@
-if)
+if#if_conj)
@@ -520 +520 @@
-the)
+the#the_adj)
@@ -522 +522 @@
-trade))
+trade#trade_n))
@@ -524 +524 @@
-be)
+be#be_v)
@@ -527 +527 @@
-as)
+as#as_adv)
@@ -529 +529 @@
-bad)
+bad#bad_adj)
@@ -534 +534 @@
-thare))
+thare#there_adv))
@@ -537 +537 @@
-as)
+as#as_adv)
@@ -540 +540 @@
-heare))))))
+heare#here_adv))))))
@@ -545 +545 @@
-I))
+I#i_pron))
@@ -547 +547 @@
-should)
+should#shall_v)
@@ -549 +549 @@
-ad_vise)
+ad_vise#advise_v)
@@ -552 +552 @@
-yo=u=))
+yo=u=#you_pron))
@@ -556 +556 @@
-to)
+to#to_prep)
@@ -558 +558 @@
-be)
+be#be_v)
@@ -561 +561 @@
-red)
+red#rid_v)
@@ -564 +564 @@
-of)
+of#of_prep)
@@ -567 +567 @@
-sum)
+sum#some_pron)
@@ -570 +570 @@
-of)
+of#of_prep)
@@ -573 +573 @@
-the)
+the#the_adj)
@@ -575 +575 @@
-workkers)))))))
+workkers#worker_n)))))))
@@ -581 +581 @@
-not)
+not#not_adv)
@@ -583 +583 @@
-to)
+to#to_prep)
@@ -585 +585 @@
-trune)
+trune#turn_v)
@@ -588 +588 @@
-thim))
+thim#them_pron))
@@ -590 +590 @@
-of)))
+of#off_adv)))
@@ -595 +595 @@
-but)
+but#but_prep)
@@ -598 +598 @@
-to)
+to#to_prep)
@@ -600 +600 @@
-bad)
+bad#bid_v)
@@ -603 +603 @@
-thim))
+thim#them_pron))
@@ -606 +606 @@
-away)))))
+away#away_adv)))))
@@ -615,3 +615,3 @@
-I))
-(HVP
-have)
+I#i_pron))
+(HVP
+have#have_v)
@@ -619 +619 @@
-delaid)
+delaid#delay_v1)
@@ -622 +622 @@
-writtinge))
+writtinge#writing_n))
@@ -625 +625 @@
-ever)
+ever#ever_adv)
@@ -627 +627 @@
-sence)
+sence#since_adv)
@@ -634 +634 @@
-the)
+the#the_adj)
@@ -636 +636 @@
-kinge))
+kinge#king_n))
@@ -638 +638 @@
-was)
+was#be_v)
@@ -640 +640 @@
-cround)
+cround#crown_v1)
@@ -645 +645 @@
-when)
+when#when_adv)
@@ -652 +652 @@
-I))
+I#i_pron))
@@ -654 +654 @@
-under_stande)
+under_stande#understand_v)
@@ -657 +657 @@
-that)
+that#that_conj)
@@ -661 +661 @@
-thay))
+thay#they_pron))
@@ -663 +663 @@
-did)
+did#do_v)
@@ -665 +665 @@
-not)
+not#not_adv)
@@ -667 +667 @@
-goe)
+goe#go_v)
@@ -670 +670 @@
-out)
+out#out_adv)
@@ -672 +672 @@
-of)
+of#of_prep)
@@ -675 +675 @@
-moringe)))))))))))
+moringe#moring_n)))))))))))
@@ -683 +683 @@
-for)
+for#for_prep)
@@ -686 +686 @@
-they))
+they#they_pron))
@@ -688 +688 @@
-say)
+say#say_v1)
@@ -691 +691 @@
-heare))
+heare#here_adv))
@@ -694 +694 @@
-that)
+that#that_conj)
@@ -698 +698 @@
-it))
+it#it_pron))
@@ -700 +700 @@
-will)
+will#will_v1)
@@ -702 +702 @@
-last)
+last#last_v1)
@@ -705 +705 @@
-a@)
+a@#a_adj2)
@@ -707 +707 @@
-@hole)
+@hole#whole_adj)
@@ -709 +709 @@
-yeare))))
+yeare#year_n))))
@@ -717 +717 @@
-and)
+and#and_conj1)
@@ -720 +720 @@
-if)
+if#if_conj)
@@ -723 +723 @@
-so)))
+so#so_adv)))
@@ -728 +728 @@
-it))
+it#it_pron))
@@ -730 +730 @@
-will)
+will#will_v1)
@@ -732 +732 @@
-break)
+break#break_v)
@@ -735 +735 @@
-many)
+many#many_adj)
@@ -737 +737 @@
-trade-)
+trade-#trade_n)
@@ -739 +739 @@
-peppill)))
+peppill#people_n)))
@@ -745 +745 @@
-write)
+write#write_v)
@@ -748 +748 @@
-me))
+me#me_pron1))
@@ -752 +752 @@
-weare)
+weare#whether_pron)
@@ -758 +758 @@
-yo=u=))
+yo=u=#you_pron))
@@ -760 +760 @@
-doe)
+doe#do_v)
@@ -762 +762 @@
-intend)
+intend#intend_v)
@@ -766 +766 @@
-to)
+to#to_prep)
@@ -768 +768 @@
-com)
+com#come_v)
@@ -771 +771 @@
-this)
+this#this_pron)
@@ -773 +773 @@
-summer)))
+summer#summer_n1)))
@@ -776 +776 @@
-or)
+or#or_conj2)
@@ -779 +779 @@
-not))))))
+not#not_adv))))))
@@ -784 +784 @@
-and)
+and#and_conj1)
@@ -787 +787 @@
-whare)
+whare#whether_pron)
@@ -793 +793 @@
-yo=r=)
+yo=r=#your_pron)
@@ -795 +795 @@
-brother))
+brother#brother_n))
@@ -797 +797 @@
-doe)
+doe#do_v)
@@ -799 +799 @@
-desine)
+desine#design_v)
@@ -802 +802 @@
-to)
+to#to_prep)
@@ -804 +804 @@
-com))))))
+com#come_v))))))
@@ -813 +813 @@
-his)
+his#his_adj)
@@ -815 +815 @@
-wife))
+wife#wife_n))
@@ -817 +817 @@
-is)
+is#be_v)
@@ -820 +820 @@
-heare))
+heare#here_adv))
@@ -823 +823 @@
-ofton))
+ofton#often_adv))
@@ -831 +831 @@
-till)
+till#tell_v)
@@ -834 +834 @@
-him))
+him#him_pron))
@@ -837 +837 @@
-that)
+that#that_conj)
@@ -841 +841 @@
-shee))
+shee#she_pron))
@@ -843 +843 @@
-doe)
+doe#do_v)
@@ -845 +845 @@
-desine)
+desine#design_v)
@@ -849 +849 @@
-to)
+to#to_prep)
@@ -851 +851 @@
-ride)
+ride#ride_v)
@@ -854 +854 @@
-to)
+to#to_prep)
@@ -857 +857 @@
-passones)))
+passones#passion_n)))
@@ -860 +860 @@
-the)
+the#the_adj)
@@ -862 +862 @@
-next)
+next#next_adj)
@@ -864 +864 @@
-weeke)))
+weeke#week_n)))
@@ -869 +869 @@
-and)
+and#and_conj1)
@@ -874 +874 @@
-dun)
+dun#dun_v3)
@@ -877 +877 @@
-him))
+him#him_pron))
@@ -880 +880 @@
-about)
+about#about_adv)
@@ -883 +883 @@
-the)
+the#the_adj)
@@ -885 +885 @@
-mony)))))))))
+mony#money_n)))))))))
@@ -891 +891 @@
-for)
+for#for_prep)
@@ -894 +894 @@
-now))
+now#now_adv))
@@ -897 +897 @@
-the)
+the#the_adj)
@@ -899 +899 @@
-band))
+band#band_n1))
@@ -901 +901 @@
-is)
+is#be_v)
@@ -904 +904 @@
-due))
+due#due_adj))
@@ -913 +913 @@
-wee))
+wee#we_pron))
@@ -915 +915 @@
-have)
+have#have_v)
@@ -917 +917 @@
-sold)
+sold#sell_v)
@@ -922 +922 @@
-about)
+about#about_adv)
@@ -924 +924 @@
-44@))
+44@#NA))
@@ -926 +926 @@
-@=l=))
+@=l=#pound_n1))
@@ -928 +928 @@
-worth)
+worth#worth_n1)
@@ -931 +931 @@
-of)
+of#of_prep)
@@ -934 +934 @@
-the)
+the#the_adj)
@@ -936 +936 @@
-lase)))))
+lase#lease_n1)))))
@@ -942 +942 @@
-but)
+but#but_prep)
@@ -946 +946 @@
-have)
+have#have_v)
@@ -948 +948 @@
-not)
+not#not_adv)
@@ -950 +950 @@
-received)
+received#receive_v)
@@ -954 +954 @@
-above)
+above#above_adv)
@@ -956 +956 @@
-12.)
+12.#twelve_adj)
@@ -959 +959 @@
-or)
+or#or_conj2)
@@ -962 +962 @@
-14.))))
+14.#fourteen_adj))))
@@ -965 +965 @@
-of)
+of#of_prep)
@@ -968 +968 @@
-it)))))
+it#it_pron)))))
@@ -974 +974 @@
-but)
+but#but_prep)
@@ -977 +977 @@
-in)
+in#in_prep)
@@ -980 +980 @@
-a@)
+a@#a_adj2)
@@ -982 +982 @@
-@mounth)
+@mounth#month_n1)
@@ -985 +985 @@
-more))))
+more#more_adj))))
@@ -988 +988 @@
-thare))
+thare#there_adv))
@@ -990 +990 @@
-will)
+will#will_v1)
@@ -992 +992 @@
-be)
+be#be_v)
@@ -995 +995 @@
-.30@)
+.30@#thirty_adj)
@@ -997 +997 @@
-@=l=)
+@=l=#pound_n1)
@@ -1002 +1002 @@
-due))
+due#due_adj))
@@ -1005 +1005 @@
-more))
+more#more_adj))
@@ -1013 +1013 @@
-and)
+and#and_conj1)
@@ -1016 +1016 @@
-then))
+then#then_adv))
@@ -1019 +1019 @@
-as)
+as#as_adv)
@@ -1021 +1021 @@
-soone)
+soone#soon_adv)
@@ -1024 +1024 @@
-as)
+as#as_adv)
@@ -1035 +1035 @@
-it))
+it#it_pron))
@@ -1037 +1037 @@
-is)
+is#be_v)
@@ -1039 +1039 @@
-received)))))
+received#receive_v)))))
@@ -1044 +1044 @@
-it))
+it#it_pron))
@@ -1046 +1046 @@
-shall)
+shall#shall_v)
@@ -1048 +1048 @@
-be)
+be#be_v)
@@ -1050 +1050 @@
-sent)
+sent#send_v1)
@@ -1060 +1060 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1062 +1062 @@
-brothers))
+brothers#brother_n))
@@ -1064 +1064 @@
-wife))
+wife#wife_n))
@@ -1066 +1066 @@
-have)
+have#have_v)
@@ -1068 +1068 @@
-broug)
+broug#brog_v)
@@ -1071 +1071 @@
-me))
+me#me_pron1))
@@ -1075 +1075 @@
-all_most)
+all_most#almost_adv)
@@ -1077 +1077 @@
-50@))
+50@#fifty_adj))
@@ -1079 +1079 @@
-@=l=)))
+@=l=#pound_n1)))
@@ -1085 +1085 @@
-and)
+and#and_conj1)
@@ -1088 +1088 @@
-shee))
+shee#she_pron))
@@ -1090 +1090 @@
-will)
+will#will_v1)
@@ -1092 +1092 @@
-make)
+make#make_v1)
@@ -1095 +1095 @@
-it))
+it#it_pron))
@@ -1097 +1097 @@
-up)
+up#up_adv1)
@@ -1100 +1100 @@
-50@)
+50@#fifty_adj)
@@ -1102 +1102 @@
-@=l=))
+@=l=#pound_n1))
@@ -1105 +1105 @@
-when)
+when#when_adv)
@@ -1112 +1112 @@
-shee))
+shee#she_pron))
@@ -1114 +1114 @@
-can)
+can#can_v1)
@@ -1116 +1116 @@
-get)
+get#get_v)
@@ -1118 +1118 @@
-in)
+in#in_adv)
@@ -1121 +1121 @@
-the)
+the#the_adj)
@@ -1123 +1123 @@
-mony)))))
+mony#money_n)))))
@@ -1131 +1131 @@
-ask)
+ask#ask_v)
@@ -1134 +1134 @@
-him))
+him#him_pron))
@@ -1138 +1138 @@
-what))
+what#what_pron))
@@ -1144 +1144 @@
-hee))
+hee#he_pron))
@@ -1146 +1146 @@
-will)
+will#will_v1)
@@ -1148 +1148 @@
-have)
+have#have_v)
@@ -1153 +1153 @@
-doun)
+doun#do_v)
@@ -1156 +1156 @@
-wth)
+wth#with_prep)
@@ -1159 +1159 @@
-it)))))))
+it#it_pron)))))))
@@ -1167 +1167 @@
-I))
+I#i_pron))
@@ -1169 +1169 @@
-bles)
+bles#bless_v1)
@@ -1172 +1172 @@
-god)))
+god#NA)))
@@ -1175 +1175 @@
-wee))
+wee#we_pron))
@@ -1177 +1177 @@
-are)
+are#be_v)
@@ -1180 +1180 @@
-in)
+in#in_prep)
@@ -1183 +1183 @@
-helth)))
+helth#health_n)))
@@ -1186 +1186 @@
-heare)))
+heare#here_adv)))
@@ -1192 +1192 @@
-and)
+and#and_conj1)
@@ -1196 +1196 @@
-hope)
+hope#hope_v)
@@ -1199 +1199 @@
-the)
+the#the_adj)
@@ -1201 +1201 @@
-like))
+like#like_adj))
@@ -1204 +1204 @@
-of)
+of#of_prep)
@@ -1207 +1207 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -1216 +1216 @@
-I))
+I#i_pron))
@@ -1218 +1218 @@
-obsarved)
+obsarved#observe_v)
@@ -1221 +1221 @@
-a@)
+a@#a_adj2)
@@ -1223 +1223 @@
-@passhin)
+@passhin#passion_n)
@@ -1226 +1226 @@
-in)
+in#in_prep)
@@ -1229 +1229 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1231 +1231 @@
-letter)))
+letter#letter_n1)))
@@ -1236 +1236 @@
-that)
+that#that_conj)
@@ -1240 +1240 @@
-fare))
+fare#fear_n))
@@ -1242 +1242 @@
-had)
+had#have_v)
@@ -1244 +1244 @@
-seade)
+seade#say_v1)
@@ -1247 +1247 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1249 +1249 @@
-spirrit))))
+spirrit#spirit_n))))
@@ -1255 +1255 @@
-wich))
+wich#which_adj))
@@ -1263 +1263 @@
-indeade))
+indeade#indeed_adv))
@@ -1265 +1265 @@
-ase)
+ase#have_v)
@@ -1267 +1267 @@
-ben)
+ben#be_v)
@@ -1270 +1270 @@
-sum)
+sum#some_pron)
@@ -1272 +1272 @@
-trobell))
+trobell#trouble_n))
@@ -1275 +1275 @@
-to)
+to#to_prep)
@@ -1278 +1278 @@
-me))))))
+me#me_pron1))))))
@@ -1286 +1286 @@
-trust)
+trust#trust_v)
@@ -1289 +1289 @@
-god))
+god#NA))
@@ -1292 +1292 @@
-in)
+in#in_prep)
@@ -1295 +1295 @@
-his)
+his#his_adj)
@@ -1297 +1297 @@
-promisses)))
+promisses#promise_n)))
@@ -1305 +1305 @@
-for)
+for#for_prep)
@@ -1308 +1308 @@
-hee))
+hee#he_pron))
@@ -1310 +1310 @@
-have)
+have#have_v)
@@ -1312 +1312 @@
-saide)
+saide#say_v1)
@@ -1319 +1319 @@
-I))
+I#i_pron))
@@ -1321 +1321 @@
-will)
+will#will_v1)
@@ -1324 +1324 @@
-never))
+never#never_adv))
@@ -1326 +1326 @@
-lave)
+lave#leave_v1)
@@ -1329 +1329 @@
-thee)))
+thee#thee_pron)))
@@ -1332 +1332 @@
-nor)
+nor#nor_conj1)
@@ -1335 +1335 @@
-for_sake)
+for_sake#forsake_v)
@@ -1338 +1338 @@
-thee)))))
+thee#thee_pron)))))
@@ -1347 +1347 @@
-I))
+I#i_pron))
@@ -1349 +1349 @@
-bless)
+bless#bless_v1)
@@ -1352 +1352 @@
-god))
+god#NA))
@@ -1359 +1359 @@
-I))
+I#i_pron))
@@ -1361 +1361 @@
-am)
+am#be_v)
@@ -1364 +1364 @@
-suldom))
+suldom#seldom_adv))
@@ -1366 +1366 @@
-trobbelled)
+trobbelled#trouble_v)
@@ -1369 +1369 @@
-at)
+at#at_prep)
@@ -1372 +1372 @@
-any)
+any#any_adj)
@@ -1374 +1374 @@
-thinge)
+thinge#thing_n1)
@@ -1377 +1377 @@
-of)
+of#of_prep)
@@ -1380 +1380 @@
-this)
+this#this_pron)
@@ -1382 +1382 @@
-worlde)))
+worlde#world_n)))
@@ -1385 +1385 @@
-but)
+but#but_prep)
@@ -1389 +1389 @@
-for)
+for#for_prep)
@@ -1392 +1392 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -1397 +1397 @@
-and)
+and#and_conj1)
@@ -1400 +1400 @@
-that)
+that#that_conj)
@@ -1404 +1404 @@
-I))
+I#i_pron))
@@ -1406 +1406 @@
-am)
+am#be_v)
@@ -1409 +1409 @@
-at)
+at#at_prep)
@@ -1412 +1412 @@
-such)
+such#such_adj)
@@ -1414 +1414 @@
-a@)
+a@#a_adj2)
@@ -1416 +1416 @@
-@distance)
+@distance#distance_n)
@@ -1419 +1419 @@
-from)
+from#from_prep)
@@ -1422 +1422 @@
-my)
+my#my_adj)
@@ -1426 +1426 @@
-famaly))))))))))))))
+famaly#family_n))))))))))))))
@@ -1434 +1434 @@
-but)
+but#but_prep)
@@ -1437 +1437 @@
-the)
+the#the_adj)
@@ -1439 +1439 @@
-great)
+great#great_adj)
@@ -1441 +1441 @@
-god))
+god#NA))
@@ -1443 +1443 @@
-hath)
+hath#have_v)
@@ -1445 +1445 @@
-saide)
+saide#say_v1)
@@ -1450 +1450 @@
-that)
+that#that_conj)
@@ -1454 +1454 @@
-all)
+all#all_adj)
@@ -1456 +1456 @@
-thinges))
+thinges#thing_n1))
@@ -1458 +1458 @@
-shall)
+shall#shall_v)
@@ -1460 +1460 @@
-worke)
+worke#work_v)
@@ -1463 +1463 @@
-together))
+together#together_adv))
@@ -1466 +1466 @@
-for)
+for#for_prep)
@@ -1469 +1469 @@
-good)))
+good#good_adj)))
@@ -1474 +1474 @@
-to)
+to#to_prep)
@@ -1477 +1477 @@
-thim)
+thim#them_pron)
@@ -1482 +1482 @@
-that)
+that#that_conj)
@@ -1488 +1488 @@
-truly))
+truly#truly_adv))
@@ -1490 +1490 @@
-fare)
+fare#fear_v)
@@ -1493 +1493 @@
-him))))))))
+him#him_pron))))))))
@@ -1502 +1502 @@
-the)
+the#the_adj)
@@ -1504 +1504 @@
-lord))
+lord#NA))
@@ -1506 +1506 @@
-to)
+to#to_prep)
@@ -1508 +1508 @@
-rais)
+rais#raise_v1)
@@ -1510 +1510 @@
-up)
+up#up_adv1)
@@ -1513 +1513 @@
-all)
+all#all_adj)
@@ -1515 +1515 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1517 +1517 @@
-spirrits))
+spirrits#spirit_n))
@@ -1521 +1521 @@
-to)
+to#to_prep)
@@ -1523 +1523 @@
-trost)
+trost#trust_v)
@@ -1526 +1526 @@
-in)
+in#in_prep)
@@ -1529 +1529 @@
-him))))
+him#him_pron))))
@@ -1534 +1534 @@
-and)
+and#and_conj1)
@@ -1539 +1539 @@
-sarve)
+sarve#serve_v1)
@@ -1542 +1542 @@
-him))))
+him#him_pron))))
@@ -1547 +1547 @@
-and)
+and#and_conj1)
@@ -1552 +1552 @@
-live)
+live#live_v1)
@@ -1555 +1555 @@
-to)
+to#to_prep)
@@ -1558 +1558 @@
-him))))))
+him#him_pron))))))
@@ -1564 +1564 @@
-which))
+which#which_adj))
@@ -1571 +1571 @@
-is)
+is#be_v)
@@ -1574 +1574 @@
-the)
+the#the_adj)
@@ -1576 +1576 @@
-dailly)
+dailly#daily_adj)
@@ -1578 +1578 @@
-prayere)
+prayere#prayer_n1)
@@ -1581 +1581 @@
-of)
+of#of_prep)
@@ -1584 +1584 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1587 +1587 @@
-ever)
+ever#ever_adv)
@@ -1589 +1589 @@
-Lov=g=))
+Lov=g=#loving_adj))
@@ -1591 +1591 @@
-mother)
+mother#mother_n1)
@@ -1594 +1594 @@
-Jane)
+Jane#NA)
@@ -1596 +1596 @@
-Pinny))))))))
+Pinny#NA))))))))
@@ -1609 +1609 @@
-My)
+My#my_adj)
@@ -1611 +1611 @@
-dare))
+dare#dear_adj1))
@@ -1614 +1614 @@
-heare))
+heare#here_adv))
@@ -1617 +1617 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1619 +1619 @@
-see)
+see#see_v)
@@ -1623 +1623 @@
-what))
+what#what_pron))
@@ -1631 +1631 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1633 +1633 @@
-dafter))
+dafter#daughter_n))
@@ -1635 +1635 @@
-doe)
+doe#do_v)
@@ -1637 +1637 @@
-write)
+write#write_v)
@@ -1640 +1640 @@
-that)
+that#that_conj)
@@ -1644 +1644 @@
-shee))
+shee#she_pron))
@@ -1646 +1646 @@
-have)
+have#have_v)
@@ -1651 +1651 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1653 +1653 @@
-writetinges))))))
+writetinges#writing_n))))))
@@ -1662 +1662 @@
-that))
+that#that_pron))
@@ -1664 +1664 @@
-is)
+is#be_v)
@@ -1667 +1667 @@
-true)))
+true#true_adj)))
@@ -1673 +1673 @@
-but)
+but#but_prep)
@@ -1676 +1676 @@
-what))
+what#what_pron))
@@ -1681 +1681 @@
-doe)
+doe#do_v)
@@ -1684 +1684 @@
-thay))
+thay#they_pron))
@@ -1686 +1686 @@
-sicknifie)
+sicknifie#signify_v)
@@ -1689 +1689 @@
-unles)
+unles#unless_adv)
@@ -1696 +1696 @@
-old)
+old#old_adj)
@@ -1698 +1698 @@
-pimer))
+pimer#NA))
@@ -1700 +1700 @@
-had)
+had#have_v)
@@ -1702 +1702 @@
-given)
+given#give_v)
@@ -1705 +1705 @@
-him))
+him#him_pron))
@@ -1708 +1708 @@
-mony))
+mony#many_adj))
@@ -1711 +1711 @@
-in)
+in#in_prep)
@@ -1714 +1714 @@
-consederation)
+consederation#consideration_n)
@@ -1717 +1717 @@
-of)
+of#of_prep)
@@ -1720 +1720 @@
-it))))))))))
+it#it_pron))))))))))
@@ -1728 +1728 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1730 +1730 @@
-most)
+most#must_v1)
@@ -1732 +1732 @@
-com))
+com#come_v))
@@ -1735 +1735 @@
-and)
+and#and_conj1)
@@ -1738 +1738 @@
-take)
+take#take_v)
@@ -1741 +1741 @@
-bands))
+bands#band_n1))
@@ -1744 +1744 @@
-of)
+of#of_prep)
@@ -1748 +1748 @@
-her))
+her#her_pron1))
@@ -1751 +1751 @@
-and)
+and#and_conj1)
@@ -1754 +1754 @@
-her)
+her#her_pron2)
@@ -1756 +1756 @@
-housban)))))
+housban#husband_n)))))
@@ -1759 +1759 @@
-if)
+if#if_conj)
@@ -1766 +1766 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1768 +1768 @@
-can)
+can#can_v1)
@@ -1770 +1770 @@
-get)
+get#get_v)
@@ -1773 +1773 @@
-thim))
+thim#them_pron))
@@ -1776 +1776 @@
-to)
+to#to_prep)
@@ -1778 +1778 @@
-give)
+give#give_v)
@@ -1781 +1781 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1784 +1784 @@
-bands))))))
+bands#band_n1))))))
@@ -1787 +1787 @@
-be_fore)
+be_fore#before_adv)
@@ -1794 +1794 @@
-it))
+it#it_pron))
@@ -1796 +1796 @@
-runes)
+runes#run_v)
@@ -1800 +1800 @@
-to)
+to#too_adv)
@@ -1802 +1802 @@
-fare))))))))
+fare#far_adj))))))))
@@ -1810 +1810 @@
-for)
+for#for_prep)
@@ -1813 +1813 @@
-shee))
+shee#she_pron))
@@ -1815 +1815 @@
-is)
+is#be_v)
@@ -1818 +1818 @@
-wth)
+wth#with_prep)
@@ -1821 +1821 @@
-childe))))
+childe#child_n))))
@@ -1827 +1827 @@
-and)
+and#and_conj1)
@@ -1830 +1830 @@
-how))
+how#how_adv))
@@ -1835 +1835 @@
-can)
+can#can_v1)
@@ -1838 +1838 @@
-you))
+you#you_pron))
@@ -1840 +1840 @@
-neglecke)
+neglecke#neglect_v)
@@ -1843 +1843 @@
-such)
+such#such_adj)
@@ -1845 +1845 @@
-athinge)
+athinge#othing_n)
@@ -1848 +1848 @@
-as)
+as#as_adv)
@@ -1851 +1851 @@
-this))))))
+this#this_pron))))))
@@ -1858 +1858 @@
-what))
+what#what_pron))
@@ -1863 +1863 @@
-doe)
+doe#do_v)
@@ -1866 +1866 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1868 +1868 @@
-kno~)
+kno~#know_v)
@@ -1872 +1872 @@
-how))
+how#how_adv))
@@ -1880 +1880 @@
-god))
+god#NA))
@@ -1882 +1882 @@
-may)
+may#may_v1)
@@ -1884 +1884 @@
-deale)
+deale#deal_v)
@@ -1887 +1887 @@
-wth)
+wth#with_prep)
@@ -1890 +1890 @@
-her)
+her#her_pron1)
@@ -1894 +1894 @@
-such)
+such#such_adj)
@@ -1896 +1896 @@
-a@)
+a@#a_adj2)
@@ -1898 +1898 @@
-@littill)
+@littill#little_adj)
@@ -1900 +1900 @@
-cretuare))
+cretuare#creature_n))
@@ -1903 +1903 @@
-and)
+and#and_conj1)
@@ -1906 +1906 @@
-old))
+old#old_adj))
@@ -1909 +1909 @@
-to)))))))))
+to#too_adv)))))))))
@@ -1917 +1917 @@
-and)
+and#and_conj1)
@@ -1920 +1920 @@
-then))
+then#then_adv))
@@ -1923 +1923 @@
-her)
+her#her_pron2)
@@ -1925 +1925 @@
-husband))
+husband#husband_n))
@@ -1927 +1927 @@
-shall)
+shall#shall_v)
@@ -1929 +1929 @@
-arest)
+arest#arrest_v)
@@ -1932 +1932 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1935 +1935 @@
-for)
+for#for_prep)
@@ -1938 +1938 @@
-all)
+all#all_adj)
@@ -1947 +1947 @@
-yo=u=))
+yo=u=#you_pron))
@@ -1949 +1949 @@
-are)
+are#be_v)
@@ -1952 +1952 @@
-worth)
+worth#worth_adj)
@@ -1962 +1962 @@
-and)
+and#and_conj1)
@@ -1964 +1964 @@
-see)
+see#see_v)
@@ -1968 +1968 @@
-wth)
+wth#with_prep)
@@ -1971 +1971 @@
-what)
+what#what_pron)
@@ -1973 +1973 @@
-viger)))
+viger#vigour_n)))
@@ -1981 +1981 @@
-thay))
+thay#they_pron))
@@ -1983 +1983 @@
-will)
+will#will_v1)
@@ -1985 +1985 @@
-deale)
+deale#deal_v)
@@ -1988 +1988 @@
-w=th=)
+w=th=#with_prep)
@@ -1991 +1991 @@
-yo=u=))))))
+yo=u=#you_pron))))))
@@ -1997 +1997 @@
-for)
+for#for_prep)
@@ -2000 +2000 @@
-hee))
+hee#he_pron))
@@ -2002 +2002 @@
-have)
+have#have_v)
@@ -2005 +2005 @@
-a)
+a#a_adj2)
@@ -2007 +2007 @@
-pistilinte)
+pistilinte#pestilent_adj)
@@ -2009 +2009 @@
-mother)
+mother#mother_n1)
@@ -2012 +2012 @@
-worse)
+worse#worse_adj)
@@ -2015 +2015 @@
-of)
+of#of_prep)
@@ -2018 +2018 @@
-the)
+the#the_adj)
@@ -2020 +2020 @@
-to)))
+to#two_adj)))
@@ -2023 +2023 @@
-thin)
+thin#than_conj)
@@ -2026 +2026 @@
-his)
+his#his_adj)
@@ -2028 +2028 @@
-father)))))
+father#father_n)))))
@@ -2036 +2036 @@
-but)
+but#but_prep)
@@ -2039 +2039 @@
-carnall)
+carnall#carnal_adj)
@@ -2041 +2041 @@
-morrole)
+morrole#moral_adj)
@@ -2043 +2043 @@
-pappill))
+pappill#people_n))
@@ -2046 +2046 @@
-all)
+all#all_adj)
@@ -2049 +2049 @@
-of)
+of#of_prep)
@@ -2052 +2052 @@
-thim))))
+thim#them_pron))))
@@ -2060 +2060 @@
-and)
+and#and_conj1)
@@ -2063 +2063 @@
-yet))
+yet#yet_adv))
@@ -2066 +2066 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2068 +2068 @@
-will)
+will#will_v1)
@@ -2070 +2070 @@
-horkkin)
+horkkin#hearken_v)
@@ -2073 +2073 @@
-to)
+to#to_prep)
@@ -2076 +2076 @@
-thim)
+thim#them_pron)
@@ -2081 +2081 @@
-that)
+that#that_conj)
@@ -2086 +2086 @@
-doe)
+doe#do_v)
@@ -2088 +2088 @@
-wish)
+wish#wish_v)
@@ -2091 +2091 @@
-nether)
+nether#neither_adv)
@@ -2093 +2093 @@
-yo=u=)
+yo=u=#you_pron)
@@ -2095 +2095 @@
-nor)
+nor#nor_conj1)
@@ -2097 +2097 @@
-yo=r=)
+yo=r=#your_pron)
@@ -2102 +2102 @@
-any)
+any#any_adj)
@@ -2104 +2104 @@
-good))
+good#good_adj))
@@ -2107 +2107 @@
-but)
+but#but_prep)
@@ -2110 +2110 @@
-thim)
+thim#them_pron)
@@ -2112 +2112 @@
-silfes)))))))
+silfes#self_pron)))))))
@@ -2121 +2121 @@
-in)
+in#in_prep)
@@ -2124 +2124 @@
-henderinge)
+henderinge#hinder_v)
@@ -2127 +2127 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2130 +2130 @@
-to)
+to#to_prep)
@@ -2132 +2132 @@
-com))))
+com#come_v))))
@@ -2137 +2137 @@
-why))
+why#why_adv))
@@ -2142 +2142 @@
-can@)
+can@#can_v1)
@@ -2144 +2144 @@
-@not)
+@not#not_adv)
@@ -2147 +2147 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2149 +2149 @@
-com)
+com#come_v)
@@ -2152 +2152 @@
-as)
+as#as_adv)
@@ -2154 +2154 @@
-will)
+will#well_adv)
@@ -2157 +2157 @@
-as)
+as#as_adv)
@@ -2160 +2160 @@
-mr)
+mr#NA)
@@ -2162 +2162 @@
-dadge))))))
+dadge#NA))))))
@@ -2169 +2169 @@
-hee))
+hee#he_pron))
@@ -2171 +2171 @@
-could)
+could#can_v1)
@@ -2173 +2173 @@
-be)
+be#be_v)
@@ -2175 +2175 @@
-wanttinge)
+wanttinge#want_v)
@@ -2180 +2180 @@
-to)
+to#two_adj)
@@ -2182 +2182 @@
-mounthes)))
+mounthes#month_n1)))
@@ -2188 +2188 @@
-and)
+and#and_conj1)
@@ -2191 +2191 @@
-why))
+why#why_adv))
@@ -2196 +2196 @@
-can@)
+can@#can_v1)
@@ -2198 +2198 @@
-@not)
+@not#not_adv)
@@ -2201 +2201 @@
-yo=u=)
+yo=u=#you_pron)
@@ -2206 +2206 @@
-that)
+that#that_conj)
@@ -2211 +2211 @@
-have)
+have#have_v)
@@ -2214 +2214 @@
-one)
+one#one_adj)
@@ -2222 +2222 @@
-to)
+to#to_prep)
@@ -2224 +2224 @@
-helpe)
+helpe#help_v)
@@ -2227 +2227 @@
-yo=u=))))))))
+yo=u=#you_pron))))))))
@@ -2238 +2238 @@
-I))
+I#i_pron))
@@ -2240 +2240 @@
-will)
+will#will_v1)
@@ -2242 +2242 @@
-say)
+say#say_v1)
@@ -2246 +2246 @@
-noe)
+noe#no_adj)
@@ -2248 +2248 @@
-more))))
+more#more_adj))))
@@ -2255 +2255 @@
-the)
+the#the_adj)
@@ -2257 +2257 @@
-lorde))
+lorde#NA))
@@ -2259 +2259 @@
-dericke)
+dericke#derrick_v)
@@ -2262 +2262 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2265 +2265 @@
-for)
+for#for_prep)
@@ -2268 +2268 @@
-the)
+the#the_adj)
@@ -2270 +2270 @@
-best)))
+best#best_adj)))
@@ -2278 +2278 @@
-but)
+but#but_prep)
@@ -2281 +2281 @@
-I))
+I#i_pron))
@@ -2283 +2283 @@
-most)
+most#must_v1)
@@ -2285 +2285 @@
-tell)
+tell#tell_v)
@@ -2288 +2288 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2295 +2295 @@
-it))
+it#it_pron))
@@ -2297 +2297 @@
-is)
+is#be_v)
@@ -2300 +2300 @@
-so)
+so#so_adv)
@@ -2302 +2302 @@
-bad)
+bad#bad_adj)
@@ -2307 +2307 @@
-heare))
+heare#here_adv))
@@ -2316 +2316 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2318 +2318 @@
-can@)
+can@#can_v1)
@@ -2320 +2320 @@
-@not)
+@not#not_adv)
@@ -2322 +2322 @@
-be@)
+be@#be_v)
@@ -2325 +2325 @@
-@sensabell))
+@sensabell#sensible_adj))
@@ -2328 +2328 @@
-unles)
+unles#unless_adv)
@@ -2335 +2335 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2337 +2337 @@
-ware)
+ware#be_v)
@@ -2340 +2340 @@
-hare))
+hare#here_adv))
@@ -2343 +2343 @@
-to)
+to#to_prep)
@@ -2346 +2346 @@
-see)
+see#see_v)
@@ -2348 +2348 @@
-and)
+and#and_conj1)
@@ -2350 +2350 @@
-heare))))))))))
+heare#hear_v))))))))))
@@ -2358 +2358 @@
-and)
+and#and_conj1)
@@ -2363 +2363 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2365 +2365 @@
-may)
+may#may_v1)
@@ -2367 +2367 @@
-travill)
+travill#travel_v)
@@ -2370 +2370 @@
-as)
+as#as_adv)
@@ -2372 +2372 @@
-safe)
+safe#safe_adj)
@@ -2377 +2377 @@
-heare))
+heare#here_adv))
@@ -2380 +2380 @@
-as)
+as#as_adv)
@@ -2383 +2383 @@
-thare)))
+thare#there_adv)))
@@ -2388 +2388 @@
-till)
+till#till_prep)
@@ -2395 +2395 @@
-the)
+the#the_adj)
@@ -2397 +2397 @@
-sogers))
+sogers#soldier_n))
@@ -2399 +2399 @@
-doe)
+doe#do_v)
@@ -2401 +2401 @@
-com)
+com#come_v)
@@ -2404 +2404 @@
-backe))
+backe#back_adv))
@@ -2407 +2407 @@
-from)
+from#from_prep)
@@ -2410 +2410 @@
-the)
+the#the_adj)
@@ -2412 +2412 @@
-campe)))
+campe#camp_n2)))
@@ -2416 +2416 @@
-wich))
+wich#which_adj))
@@ -2423 +2423 @@
-will)
+will#will_v1)
@@ -2425 +2425 @@
-not)
+not#not_adv)
@@ -2428 +2428 @@
-yet))
+yet#yet_adv))
@@ -2430 +2430 @@
-be)
+be#be_v)
@@ -2433 +2433 @@
-thes)
+thes#these_pron)
@@ -2436 +2436 @@
-to)
+to#two_adj)
@@ -2438 +2438 @@
-or)
+or#or_conj2)
@@ -2442 +2442 @@
-3))
+3#three_adj))
@@ -2444 +2444 @@
-monts))))))))
+monts#month_n1))))))))
@@ -2450 +2450 @@
-but)
+but#but_prep)
@@ -2453 +2453 @@
-I))
+I#i_pron))
@@ -2455 +2455 @@
-came)
+came#come_v)
@@ -2458 +2458 @@
-when)
+when#when_adv)
@@ -2465 +2465 @@
-the)
+the#the_adj)
@@ -2467 +2467 @@
-contry))
+contry#country_n))
@@ -2469 +2469 @@
-was)
+was#be_v)
@@ -2471 +2471 @@
-all)
+all#all_adj)
@@ -2474 +2474 @@
-fall))))))
+fall#full_adj))))))
@@ -2480 +2480 @@
-and)
+and#and_conj1)
@@ -2483 +2483 @@
-I))
+I#i_pron))
@@ -2485 +2485 @@
-see)
+see#see_v)
@@ -2488 +2488 @@
-no)
+no#no_adj)
@@ -2490 +2490 @@
-canppe))
+canppe#camp_n2))
@@ -2499 +2499 @@
-I))
+I#i_pron))
@@ -2501 +2501 @@
-shall)
+shall#shall_v)
@@ -2503 +2503 @@
-have)
+have#have_v)
@@ -2506 +2506 @@
-one))
+one#one_adj))
@@ -2511 +2511 @@
-to))
+to#too_adv))
@@ -2514 +2514 @@
-by)
+by#by_prep)
@@ -2517 +2517 @@
-wenser)
+wenser#NA)
@@ -2519 +2519 @@
-a@)
+a@#or_conj2)
@@ -2521 +2521 @@
-@thusday)))
+@thusday#NA)))
@@ -2530 +2530 @@
-I))
+I#i_pron))
@@ -2532 +2532 @@
-would)
+would#will_v1)
@@ -2539 +2539 @@
-I))
+I#i_pron))
@@ -2541 +2541 @@
-knowe)
+knowe#know_v)
@@ -2545 +2545 @@
-how))
+how#how_adv))
@@ -2553 +2553 @@
-you))
+you#you_pron))
@@ -2555 +2555 @@
-would)
+would#will_v1)
@@ -2557 +2557 @@
-sill)
+sill#sell_v)
@@ -2560 +2560 @@
-it)))))))
+it#it_pron)))))))
@@ -2567 +2567 @@
-I))
+I#i_pron))
@@ -2569 +2569 @@
-have)
+have#have_v)
@@ -2572 +2572 @@
-ahonderde)
+ahonderde#NA)
@@ -2574 +2574 @@
-thinges)
+thinges#thing_n1)
@@ -2582 +2582 @@
-to)
+to#to_prep)
@@ -2584 +2584 @@
-write)))))
+write#write_v)))))
@@ -2590 +2590 @@
-but)
+but#but_prep)
@@ -2593 +2593 @@
-what))
+what#what_pron))
@@ -2598 +2598 @@
-is)
+is#be_v)
@@ -2601 +2601 @@
-the)
+the#the_adj)
@@ -2603 +2603 @@
-...)))
+...#NA)))
@@ -2611 +2611 @@
-for)
+for#for_prep)
@@ -2614 +2614 @@
-writeinge))
+writeinge#writing_n))
@@ -2616 +2616 @@
-can@)
+can@#can_v1)
@@ -2618 +2618 @@
-@not)
+@not#not_adv)
@@ -2620 +2620 @@
-containe)
+containe#contain_v)
@@ -2623 +2623 @@
-it)))
+it#it_pron)))
@@ -2629 +2629 @@
-and)
+and#and_conj1)
@@ -2632 +2632 @@
-if)
+if#if_conj)
@@ -2639 +2639 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2641 +2641 @@
-com)
+com#come_v)
@@ -2643 +2643 @@
-not))))
+not#not_adv))))
@@ -2646 +2646 @@
-how))
+how#how_adv))
@@ -2651 +2651 @@
-will)
+will#will_v1)
@@ -2654 +2654 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2656 +2656 @@
-ashore)
+ashore#assure_v)
@@ -2659 +2659 @@
-her))
+her#her_pron1))
@@ -2662 +2662 @@
-of)
+of#of_prep)
@@ -2665 +2665 @@
-any)
+any#any_adj)
@@ -2667 +2667 @@
-thinge)
+thinge#thing_n1)
@@ -2678 +2678 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2680 +2680 @@
-give)
+give#give_v)
@@ -2683 +2683 @@
-her)))))))
+her#her_pron1)))))))
@@ -2691 +2691 @@
-and)
+and#and_conj1)
@@ -2694 +2694 @@
-if)
+if#if_conj)
@@ -2701 +2701 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2703 +2703 @@
-com)
+com#come_v)
@@ -2705 +2705 @@
-not))))
+not#not_adv))))
@@ -2708 +2708 @@
-I))
+I#i_pron))
@@ -2710 +2710 @@
-belive)
+belive#believe_v)
@@ -2717 +2717 @@
-I))
+I#i_pron))
@@ -2719 +2719 @@
-shall)
+shall#shall_v)
@@ -2721 +2721 @@
-not)
+not#not_adv)
@@ -2723 +2723 @@
-be)
+be#be_v)
@@ -2726 +2726 @@
-abell)
+abell#able_adj)
@@ -2729 +2729 @@
-to)
+to#to_prep)
@@ -2731 +2731 @@
-com)
+com#come_v)
@@ -2734 +2734 @@
-to)
+to#to_prep)
@@ -2737 +2737 @@
-yo=u=)))))
+yo=u=#you_pron)))))
@@ -2740 +2740 @@
-this)
+this#this_pron)
@@ -2742 +2742 @@
-halfe)
+halfe#half_adj)
@@ -2744 +2744 @@
-yare))))
+yare#year_n))))
@@ -2753 +2753 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2755 +2755 @@
-write)
+write#write_v)
@@ -2758 +2758 @@
-of)
+of#of_prep)
@@ -2761 +2761 @@
-lawe))))
+lawe#law_n1))))
@@ -2769 +2769 @@
-ware)
+ware#be_v)
@@ -2771 +2771 @@
-not)
+not#not_adv)
@@ -2774 +2774 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2777 +2777 @@
-better))
+better#better_adj))
@@ -2779 +2779 @@
-com)
+com#come_v)
@@ -2781 +2781 @@
-over))
+over#over_adv))
@@ -2786 +2786 @@
-and)
+and#and_conj1)
@@ -2789 +2789 @@
-see)
+see#see_v)
@@ -2792 +2792 @@
-to)
+to#to_prep)
@@ -2794 +2794 @@
-have)
+have#have_v)
@@ -2797 +2797 @@
-an@)
+an@#a_adj2)
@@ -2799 +2799 @@
-@end)
+@end#end_n)
@@ -2802 +2802 @@
-w=th=_out)
+w=th=_out#without_adv)
@@ -2805 +2805 @@
-lawe)))))))))
+lawe#law_n1)))))))))
@@ -2812,3 +2812,3 @@
-she))
-(MD
-would)
+she#she_pron))
+(MD
+would#will_v1)
@@ -2816 +2816 @@
-have)
+have#have_v)
@@ -2818 +2818 @@
-com)
+com#come_v)
@@ -2821 +2821 @@
-to)
+to#to_prep)
@@ -2824 +2824 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -2827 +2827 @@
-longe)
+longe#long_adj1)
@@ -2829 +2829 @@
-or)
+or#or_adv1)
@@ -2832 +2832 @@
-this)
+this#this_pron)
@@ -2834 +2834 @@
-tim)))
+tim#time_n)))
@@ -2837 +2837 @@
-if)
+if#if_conj)
@@ -2844 +2844 @@
-I))
+I#i_pron))
@@ -2846 +2846 @@
-would)
+would#will_v1)
@@ -2848 +2848 @@
-have)
+have#have_v)
@@ -2850 +2850 @@
-let)
+let#let_v1)
@@ -2854 +2854 @@
-her))
+her#her_pron1))
@@ -2856 +2856 @@
-com))))))
+com#come_v))))))
@@ -2862 +2862 @@
-but)
+but#but_prep)
@@ -2865 +2865 @@
-what))
+what#what_pron))
@@ -2870 +2870 @@
-doe)
+doe#do_v)
@@ -2873 +2873 @@
-that))
+that#that_pron))
@@ -2875 +2875 @@
-sigenifie))
+sigenifie#signify_v))
@@ -2884 +2884 @@
-I))
+I#i_pron))
@@ -2886 +2886 @@
-am)
+am#be_v)
@@ -2889 +2889 @@
-at)
+at#at_prep)
@@ -2892 +2892 @@
-a@)
+a@#a_adj2)
@@ -2894 +2894 @@
-@los)
+@los#loss_n1)
@@ -2898 +2898 @@
-what))
+what#what_pron))
@@ -2905 +2905 @@
-to)
+to#to_prep)
@@ -2907 +2907 @@
-doe)
+doe#do_v)
@@ -2910 +2910 @@
-w=th=_out)
+w=th=_out#without_adv)
@@ -2913 +2913 @@
-yo=u=)))))))
+yo=u=#you_pron)))))))
@@ -2922 +2922 @@
-so))
+so#so_adv))
@@ -2925 +2925 @@
-w=th=)
+w=th=#with_prep)
@@ -2928 +2928 @@
-my)
+my#my_adj)
@@ -2930 +2930 @@
-best)
+best#best_adj)
@@ -2932 +2932 @@
-wikes)
+wikes#wish_n1)
@@ -2937 +2937 @@
-to)
+to#to_prep)
@@ -2940 +2940 @@
-yo=u=)))))
+yo=u=#you_pron)))))
@@ -2945 +2945 @@
-deseringe)
+deseringe#desire_v)
@@ -2950 +2950 @@
-god))
+god#NA))
@@ -2954 +2954 @@
-to)
+to#to_prep)
@@ -2957 +2957 @@
-bles)
+bles#bless_v1)
@@ -2959 +2959 @@
-and)
+and#and_conj1)
@@ -2961 +2961 @@
-presarve))
+presarve#preserve_v))
@@ -2964 +2964 @@
-yo=u=)))
+yo=u=#you_pron)))
@@ -2967 +2967 @@
-and)
+and#and_conj1)
@@ -2972 +2972 @@
-keepe)
+keepe#keep_v)
@@ -2975 +2975 @@
-yo=u=))
+yo=u=#you_pron))
@@ -2978 +2978 @@
-in)
+in#in_prep)
@@ -2981 +2981 @@
-helthe)
+helthe#health_n)
@@ -2983 +2983 @@
-and)
+and#and_conj1)
@@ -2985 +2985 @@
-saftie)))))))
+saftie#safety_n)))))))
@@ -2988 +2988 @@
-in)
+in#in_prep)
@@ -2991 +2991 @@
-hast)))
+hast#haste_n)))
@@ -2994 +2994 @@
-I))
+I#i_pron))
@@ -2996 +2996 @@
-rest)
+rest#rest_v2)
@@ -2999 +2999 @@
-and)
+and#and_conj1)
@@ -3002 +3002 @@
-still))
+still#still_adv))
@@ -3004 +3004 @@
-remaine))
+remaine#remain_v))
@@ -3007 +3007 @@
-yo=r=)
+yo=r=#your_pron)
@@ -3010 +3010 @@
-ever)
+ever#ever_adv)
@@ -3012 +3012 @@
-Lo))
+Lo#loving_adj))
@@ -3014 +3014 @@
-wife)
+wife#wife_n)
@@ -3017 +3017 @@
-Jane)
+Jane#NA)
@@ -3019 +3019 @@
-Pinny))))
+Pinny#NA))))
