--- koxinden-1636-e2-h.psd.vert	2025-09-10 13:50:35
+++ koxinden-1636-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Harri))
+Harri#NA))
@@ -19 +19 @@
-I))
+I#i_pron))
@@ -21 +21 @@
-wold)
+wold#will_v1)
@@ -23 +23 @@
-have)
+have#have_v)
@@ -27 +27 @@
-you))
+you#you_pron))
@@ -29 +29 @@
-com)
+com#come_v)
@@ -32 +32 @@
-too)
+too#to_prep)
@@ -35 +35 @@
-mee)))
+mee#me_pron1)))
@@ -38 +38 @@
-too_nigth)
+too_nigth#tonight_adv)
@@ -40 +40 @@
-or)
+or#or_conj2)
@@ -42 +42 @@
-tomorrow))
+tomorrow#tomorrow_adv))
@@ -45 +45 @@
-be_times))))
+be_times#betimes_adv))))
@@ -51 +51 @@
-for)
+for#for_prep)
@@ -54 +54 @@
-i))
+i#i_pron))
@@ -56 +56 @@
-heare)
+heare#hear_v)
@@ -59 +59 @@
-by)
+by#by_prep)
@@ -62 +62 @@
-your)
+your#your_pron)
@@ -64 +64 @@
-ante)
+ante#aunt_n)
@@ -67 +67 @@
-proude))))
+proude#NA))))
@@ -70 +70 @@
-that)
+that#that_conj)
@@ -74 +74 @@
-James))
+James#NA))
@@ -76 +76 @@
-is)
+is#be_v)
@@ -78 +78 @@
-com)
+com#come_v)
@@ -81 +81 @@
-to)
+to#to_prep)
@@ -84 +84 @@
-Lundon))
+Lundon#NA))
@@ -87 +87 @@
-too)
+too#to_prep)
@@ -90 +90 @@
-Dick))))))
+Dick#NA))))))
@@ -99 +99 @@
-whether)
+whether#whether_pron)
@@ -106 +106 @@
-hee))
+hee#he_pron))
@@ -108 +108 @@
-mene)
+mene#mean_v1)
@@ -111 +111 @@
-to)
+to#to_prep)
@@ -113 +113 @@
-go)
+go#go_v)
@@ -115 +115 @@
-over)))
+over#over_adv)))
@@ -118 +118 @@
-or)
+or#or_conj2)
@@ -121 +121 @@
-no)))))
+no#no_adv2)))))
@@ -124 +124 @@
-i))
+i#i_pron))
@@ -126 +126 @@
-know)
+know#know_v)
@@ -128 +128 @@
-not))
+not#not_adv))
@@ -135 +135 @@
-hee))
+hee#he_pron))
@@ -137 +137 @@
-sent)
+sent#send_v1)
@@ -140 +140 @@
-to)
+to#to_prep)
@@ -143 +143 @@
-Keate)))
+Keate#NA)))
@@ -146 +146 @@
-for)
+for#for_prep)
@@ -149 +149 @@
-30@)
+30@#thirty_adj)
@@ -151 +151 @@
-@s)
+@s#shilling_n)
@@ -155 +155 @@
-which))
+which#which_adj))
@@ -161 +161 @@
-hee))
+hee#he_pron))
@@ -163 +163 @@
-saide)
+saide#say_v1)
@@ -172 +172 @@
-hee))
+hee#he_pron))
@@ -174 +174 @@
-will)
+will#will_v1)
@@ -176 +176 @@
-paie)
+paie#pay_v1)
@@ -179 +179 @@
-in)
+in#in_prep)
@@ -182 +182 @@
-agust)))
+agust#NA)))
@@ -186 +186 @@
-be_case)
+be_case#because_adv)
@@ -193 +193 @@
-hee))
+hee#he_pron))
@@ -195 +195 @@
-saies)
+saies#say_v1)
@@ -202 +202 @@
-you))
+you#you_pron))
@@ -204 +204 @@
-will)
+will#will_v1)
@@ -206 +206 @@
-send)
+send#send_v1)
@@ -209 +209 @@
-hem))
+hem#him_pron))
@@ -212 +212 @@
-none)))))))
+none#none_pron)))))))
@@ -215 +215 @@
-an)
+an#and_conj1)
@@ -218 +218 @@
-becase)
+becase#because_adv)
@@ -225 +225 @@
-i))
+i#i_pron))
@@ -227 +227 @@
-knowe)
+knowe#know_v)
@@ -229 +229 @@
-not)
+not#not_adv)
@@ -232 +232 @@
-whether)
+whether#whether_pron)
@@ -239 +239 @@
-he))
+he#he_pron))
@@ -241 +241 @@
-had)
+had#have_v)
@@ -243 +243 @@
-resented)
+resented#resent_v)
@@ -246 +246 @@
-youres)))
+youres#yours_pron)))
@@ -249 +249 @@
-or)
+or#or_conj2)
@@ -252 +252 @@
-no))))))))))))))))
+no#no_adv2))))))))))))))))
@@ -261 +261 @@
-writing)
+writing#write_v)
@@ -264 +264 @@
-pervestly))
+pervestly#previously_adv))
@@ -267 +267 @@
-to)
+to#to_prep)
@@ -270 +270 @@
-her)))
+her#her_pron1)))
@@ -273 +273 @@
-for)
+for#for_prep)
@@ -276 +276 @@
-it))))
+it#it_pron))))
@@ -279 +279 @@
-I))
+I#i_pron))
@@ -281 +281 @@
-bid)
+bid#bid_v)
@@ -285 +285 @@
-her))
+her#her_pron1))
@@ -287 +287 @@
-send)
+send#send_v1)
@@ -290 +290 @@
-it))
+it#it_pron))
@@ -293 +293 @@
-hem)))
+hem#him_pron)))
@@ -303 +303 @@
-She))
+She#she_pron))
@@ -305 +305 @@
-haveng)
+haveng#have_v)
@@ -307 +307 @@
-writ)
+writ#write_v)
@@ -310 +310 @@
-her)
+her#her_pron2)
@@ -312 +312 @@
-letter)))
+letter#letter_n1)))
@@ -314 +314 @@
-an)
+an#and_conj1)
@@ -317 +317 @@
-all_reddi))
+all_reddi#already_adj))
@@ -320 +320 @@
-a)
+a#a_adj2)
@@ -322 +322 @@
-letter)
+letter#letter_n1)
@@ -326 +326 @@
-came)
+came#come_v)
@@ -329 +329 @@
-from)
+from#from_prep)
@@ -332 +332 @@
-your)
+your#your_pron)
@@ -334 +334 @@
-ant)
+ant#aunt_n)
@@ -337 +337 @@
-proude))))
+proude#NA))))
@@ -340 +340 @@
-that)
+that#that_conj)
@@ -344 +344 @@
-some)
+some#some_pron)
@@ -346 +346 @@
-spidie)
+spidie#speedy_adj)
@@ -348 +348 @@
-cores))
+cores#course_n1))
@@ -350 +350 @@
-mith)
+mith#may_v1)
@@ -352 +352 @@
-bee)
+bee#be_v)
@@ -354 +354 @@
-taken)
+taken#take_v)
@@ -358 +358 @@
-for)
+for#for_prep)
@@ -361 +361 @@
-which)
+which#which_adj)
@@ -363 +363 @@
-cores)))
+cores#course_n1)))
@@ -371 +371 @@
-i))
+i#i_pron))
@@ -373 +373 @@
-wold)
+wold#will_v1)
@@ -376 +376 @@
-faine))
+faine#fain_adj))
@@ -378 +378 @@
-speake)
+speake#speak_v)
@@ -381 +381 @@
-with)
+with#with_prep)
@@ -384 +384 @@
-you))))))))
+you#you_pron))))))))
@@ -390 +390 @@
-for)
+for#for_prep)
@@ -393 +393 @@
-i))
+i#i_pron))
@@ -395 +395 @@
-wold)
+wold#will_v1)
@@ -397 +397 @@
-not)
+not#not_adv)
@@ -399 +399 @@
-have)
+have#have_v)
@@ -403 +403 @@
-you))
+you#you_pron))
@@ -405 +405 @@
-goe)
+goe#go_v)
@@ -408 +408 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-Lundon))))
+Lundon#NA))))
@@ -420 +420 @@
-This))
+This#this_pron))
@@ -423 +423 @@
-in)
+in#in_prep)
@@ -426 +426 @@
-hast)))
+hast#haste_n)))
@@ -429 +429 @@
-not)
+not#not_adv)
@@ -431 +431 @@
-knowing)
+knowing#know_v)
@@ -434 +434 @@
-well))
+well#well_adv))
@@ -438 +438 @@
-what))
+what#what_pron))
@@ -446 +446 @@
-i))
+i#i_pron))
@@ -448 +448 @@
-have)
+have#have_v)
@@ -450 +450 @@
-writ))))
+writ#write_v))))
@@ -458 +458 @@
-But)
+But#but_prep)
@@ -461 +461 @@
-when)
+when#when_adv)
@@ -468 +468 @@
-you))
+you#you_pron))
@@ -470 +470 @@
-com))))
+com#come_v))))
@@ -473 +473 @@
-i))
+i#i_pron))
@@ -475 +475 @@
-will)
+will#will_v1)
@@ -477 +477 @@
-tel)
+tel#tell_v)
@@ -480 +480 @@
-you))
+you#you_pron))
@@ -491 +491 @@
-prey))
+prey#pray_v))
@@ -493 +493 @@
-do)
+do#do_v)
@@ -495 +495 @@
-not)
+not#not_adv)
@@ -497 +497 @@
-defer)
+defer#defer_v1)
@@ -500 +500 @@
-time))
+time#time_n))
@@ -509 +509 @@
-This))
+This#this_adv))
@@ -512 +512 @@
-with)
+with#with_prep)
@@ -515 +515 @@
-my)
+my#my_adj)
@@ -517 +517 @@
-love)
+love#love_n1)
@@ -520 +520 @@
-to)
+to#to_prep)
@@ -524 +524 @@
-yourselfe))
+yourselfe#yourself_pron))
@@ -527 +527 @@
-an)
+an#and_conj1)
@@ -530 +530 @@
-my)
+my#my_adj)
@@ -532 +532 @@
-dafter)))))))
+dafter#daughter_n)))))))
@@ -535 +535 @@
-I))
+I#i_pron))
@@ -537 +537 @@
-rest)
+rest#rest_v2)
@@ -540 +540 @@
-Your)
+Your#your_pron)
@@ -542 +542 @@
-loveing)
+loveing#loving_adj)
@@ -544 +544 @@
-mother)
+mother#mother_n1)
@@ -547 +547 @@
-Kathern)
+Kathern#NA)
@@ -549 +549 @@
-Oxinden))))
+Oxinden#NA))))
@@ -557 +557 @@
-May)
+May#NA)
@@ -559 +559 @@
-23)
+23#NA)
@@ -561 +561 @@
-1634)
+1634#NA)
