--- kscrope-1536-e1-h.psd.vert	2025-09-10 13:50:35
+++ kscrope-1536-e1-h.lem.vert	2025-09-10 13:50:45
@@ -18 +18 @@
-My)
+My#my_adj)
@@ -20 +20 @@
-dewty))
+dewty#duty_n))
@@ -22 +22 @@
-promysed)
+promysed#promise_v)
@@ -25 +25 @@
-unto)
+unto#unto_prep)
@@ -28 +28 @@
-your)
+your#your_pron)
@@ -30 +30 @@
-Lordship)))
+Lordship#lordship_n)))
@@ -33 +33 @@
-in)
+in#in_prep)
@@ -36 +36 @@
-my)
+my#my_adj)
@@ -39 +39 @@
-most)
+most#most_adj)
@@ -41 +41 @@
-humbliest))
+humbliest#humble_adj))
@@ -43 +43 @@
-maner))))
+maner#manner_n))))
@@ -48 +48 @@
-advertysyng)
+advertysyng#advertise_v)
@@ -51 +51 @@
-the)
+the#the_adj)
@@ -53 +53 @@
-same))
+same#same_adj))
@@ -56 +56 @@
-that)
+that#that_conj)
@@ -60 +60 @@
-yesterday))
+yesterday#yesterday_adv))
@@ -63 +63 @@
-the)
+the#the_adj)
@@ -65 +65 @@
-commons)
+commons#commons_n)
@@ -68 +68 @@
-off)
+off#of_prep)
@@ -71 +71 @@
-Richmontshir))))
+Richmontshir#NA))))
@@ -73 +73 @@
-did)
+did#do_v)
@@ -75 +75 @@
-meat)
+meat#meet_v)
@@ -78 +78 @@
-at)
+at#at_prep)
@@ -81 +81 @@
-Richmond)
+Richmond#NA)
@@ -87 +87 @@
-wher))
+wher#where_adv))
@@ -95 +95 @@
-undoubtedly))
+undoubtedly#undoubtedly_adv))
@@ -98 +98 @@
-they))
+they#they_pron))
@@ -100 +100 @@
-dewydet)
+dewydet#divide_v)
@@ -103 +103 @@
-them))
+them#them_pron))
@@ -106 +106 @@
-in)
+in#in_prep)
@@ -109 +109 @@
-thre)
+thre#three_adj)
@@ -111 +111 @@
-partyes)
+partyes#party_n)
@@ -117 +117 @@
-wheroff))
+wheroff#whereof_adv))
@@ -123 +123 @@
-one)
+one#one_adj)
@@ -125 +125 @@
-company)
+company#company_n)
@@ -128 +128 @@
-there))
+there#there_adv))
@@ -132 +132 @@
-was)
+was#be_v)
@@ -134 +134 @@
-commandet)
+commandet#command_v)
@@ -138 +138 @@
-to)
+to#to_prep)
@@ -140 +140 @@
-come)
+come#come_v)
@@ -143 +143 @@
-this)
+this#this_pron)
@@ -145 +145 @@
-day))
+day#day_n))
@@ -148 +148 @@
-for)
+for#for_prep)
@@ -153 +153 @@
-my)
+my#my_adj)
@@ -155 +155 @@
-Lord)
+Lord#lord_n)
@@ -158 +158 @@
-my)
+my#my_adj)
@@ -160 +160 @@
-bedfelowe)))
+bedfelowe#bedfellow_n)))
@@ -163 +163 @@
-ore)
+ore#or_conj2)
@@ -167 +167 @@
-his)
+his#his_adj)
@@ -169 +169 @@
-litell)
+litell#little_adj)
@@ -171 +171 @@
-boy))
+boy#boy_n1))
@@ -174 +174 @@
-&)
+&#and_conj1)
@@ -177 +177 @@
-myn))))))
+myn#mine_adj))))))
@@ -184 +184 @@
-Sir)
+Sir#NA)
@@ -186 +186 @@
-James)
+James#NA)
@@ -188 +188 @@
-Metcalff))
+Metcalff#NA))
@@ -191 +191 @@
-ore)
+ore#or_conj2)
@@ -194 +194 @@
-his)
+his#his_adj)
@@ -196 +196 @@
-sone)
+sone#son_n1)
@@ -199 +199 @@
-Cristofer))))))
+Cristofer#NA))))))
@@ -204 +204 @@
-&)
+&#and_conj1)
@@ -207 +207 @@
-Richard)
+Richard#NA)
@@ -209 +209 @@
-Sigiswik))))))
+Sigiswik#NA))))))
@@ -214 +214 @@
-and)
+and#and_conj1)
@@ -217 +217 @@
-to)
+to#to_prep)
@@ -219 +219 @@
-brynge)
+brynge#bring_v)
@@ -222 +222 @@
-them))
+them#them_pron))
@@ -225 +225 @@
-with)
+with#with_prep)
@@ -228 +228 @@
-them)))))
+them#them_pron)))))
@@ -231 +231 @@
-or)
+or#or_conj2)
@@ -233 +233 @@
-elles)
+elles#else_adv)
@@ -237 +237 @@
-to)
+to#to_prep)
@@ -239 +239 @@
-pull)
+pull#pull_v)
@@ -241 +241 @@
-downe)
+downe#down_adv)
@@ -244 +244 @@
-their)
+their#their_adj)
@@ -246 +246 @@
-housses)))
+housses#house_n1)))
@@ -249 +249 @@
-and)
+and#and_conj1)
@@ -254 +254 @@
-spoill)
+spoill#spoil_v1)
@@ -257 +257 @@
-them))
+them#them_pron))
@@ -260 +260 @@
-off)
+off#of_prep)
@@ -263 +263 @@
-their)
+their#their_adj)
@@ -265 +265 @@
-goodes)))))))))))))))))))
+goodes#good_adj)))))))))))))))))))
@@ -274 +274 @@
-A)
+A#a_adj2)
@@ -276 +276 @@
-nother)
+nother#other_adj)
@@ -278 +278 @@
-company)
+company#company_n)
@@ -282 +282 @@
-goth)
+goth#go_v)
@@ -285 +285 @@
-fore)
+fore#for_prep)
@@ -289 +289 @@
-my)
+my#my_adj)
@@ -291 +291 @@
-Lord)
+Lord#lord_n)
@@ -294 +294 @@
-Latymer)))
+Latymer#NA)))
@@ -297 +297 @@
-ore)
+ore#or_conj2)
@@ -300 +300 @@
-his)
+his#his_adj)
@@ -302 +302 @@
-sone)))))
+sone#son_n1)))))
@@ -307 +307 @@
-Mr.)
+Mr.#NA)
@@ -309 +309 @@
-Danby)
+Danby#NA)
@@ -314 +314 @@
-with)
+with#with_prep)
@@ -317 +317 @@
-other)
+other#other_adj)
@@ -320 +320 @@
-in)
+in#in_prep)
@@ -323 +323 @@
-thosse)
+thosse#those_pron)
@@ -325 +325 @@
-quarters))))))
+quarters#quarter_n))))))
@@ -333 +333 @@
-And)
+And#and_conj1)
@@ -336 +336 @@
-the)
+the#the_adj)
@@ -338 +338 @@
-third)
+third#third_adj)
@@ -340 +340 @@
-company))
+company#company_n))
@@ -342 +342 @@
-goth)
+goth#go_v)
@@ -345 +345 @@
-to)
+to#to_prep)
@@ -348 +348 @@
-Barnard)
+Barnard#NA)
@@ -350 +350 @@
-Castell)))
+Castell#NA)))
@@ -353 +353 @@
-to)
+to#to_prep)
@@ -355 +355 @@
-bryng)
+bryng#bring_v)
@@ -358 +358 @@
-to)
+to#to_prep)
@@ -361 +361 @@
-them)))
+them#them_pron)))
@@ -364 +364 @@
-my)
+my#my_adj)
@@ -366 +366 @@
-cousyn)
+cousyn#cousin_n)
@@ -370 +370 @@
-George)
+George#NA)
@@ -372 +372 @@
-Bowes))
+Bowes#NA))
@@ -375 +375 @@
-&)
+&#and_conj1)
@@ -378 +378 @@
-his)
+his#his_adj)
@@ -380 +380 @@
-two)
+two#two_adj)
@@ -382 +382 @@
-uncles))))))
+uncles#uncle_n))))))
@@ -391 +391 @@
-My)
+My#my_adj)
@@ -393 +393 @@
-Lord)
+Lord#lord_n)
@@ -396 +396 @@
-my)
+my#my_adj)
@@ -398 +398 @@
-bedfelow)))
+bedfelow#bedfellow_n)))
@@ -400 +400 @@
-is)
+is#be_v)
@@ -403 +403 @@
-this)
+this#this_pron)
@@ -405 +405 @@
-nyght))
+nyght#night_n))
@@ -408 +408 @@
-at)
+at#at_prep)
@@ -411 +411 @@
-Helbek)
+Helbek#NA)
@@ -413 +413 @@
-Hall))))
+Hall#NA))))
@@ -419 +419 @@
-&)
+&#and_conj1)
@@ -423 +423 @@
-wul@)
+wul@#will_v1)
@@ -425 +425 @@
-@be)
+@be#be_v)
@@ -428 +428 @@
-with)
+with#with_prep)
@@ -431 +431 @@
-your)
+your#your_pron)
@@ -433 +433 @@
-Lordshipe)))
+Lordshipe#lordship_n)))
@@ -436 +436 @@
-at)
+at#at_prep)
@@ -439 +439 @@
-Skypton)))
+Skypton#NA)))
@@ -442 +442 @@
-in)
+in#in_prep)
@@ -446 +446 @@
-as)
+as#as_adv)
@@ -448 +448 @@
-convenyent))
+convenyent#convenient_adj))
@@ -450 +450 @@
-spead)
+spead#speed_n)
@@ -453 +453 @@
-as)
+as#as_adv)
@@ -464 +464 @@
-he))
+he#he_pron))
@@ -466 +466 @@
-can)
+can#can_v1)
@@ -468 +468 @@
-maik))))))
+maik#make_v1))))))
@@ -473 +473 @@
-to)
+to#to_prep)
@@ -475 +475 @@
-tak)
+tak#take_v)
@@ -478 +478 @@
-suche)
+suche#such_adj)
@@ -480 +480 @@
-parte)
+parte#part_n)
@@ -483 +483 @@
-as)
+as#as_adv)
@@ -494 +494 @@
-your)
+your#your_pron)
@@ -496 +496 @@
-Lordshipe))
+Lordshipe#lordship_n))
@@ -498 +498 @@
-dothe))))))
+dothe#do_v))))))
@@ -507 +507 @@
-And)
+And#and_conj1)
@@ -510 +510 @@
-I))
+I#i_pron))
@@ -512 +512 @@
-wull)
+wull#will_v1)
@@ -514 +514 @@
-come)
+come#come_v)
@@ -517 +517 @@
-this)
+this#this_pron)
@@ -519 +519 @@
-mornyng))
+mornyng#morning_n))
@@ -522 +522 @@
-towardes)
+towardes#towards_prep)
@@ -525 +525 @@
-Katelwell))))
+Katelwell#NA))))
@@ -528 +528 @@
-&)
+&#and_conj1)
@@ -531 +531 @@
-tary)
+tary#tarry_v)
@@ -534 +534 @@
-there))
+there#there_adv))
@@ -537 +537 @@
-off)
+off#of_prep)
@@ -540 +540 @@
-my)
+my#my_adj)
@@ -542 +542 @@
-bedfelowe)))))
+bedfelowe#bedfellow_n)))))
@@ -550 +550 @@
-and)
+and#and_conj1)
@@ -554 +554 @@
-wold)
+wold#will_v1)
@@ -556 +556 @@
-come)
+come#come_v)
@@ -559 +559 @@
-with)
+with#with_prep)
@@ -562 +562 @@
-hym)))
+hym#him_pron)))
@@ -565 +565 @@
-to)
+to#to_prep)
@@ -568 +568 @@
-Skipton)))
+Skipton#NA)))
@@ -573 +573 @@
-iff)
+iff#if_conj)
@@ -580 +580 @@
-ye))
+ye#ye_pron))
@@ -582 +582 @@
-thynk)
+thynk#think_v2)
@@ -586 +586 @@
-it))
+it#it_pron))
@@ -589 +589 @@
-good))))))
+good#good_adj))))))
@@ -597 +597 @@
-And)
+And#and_conj1)
@@ -600 +600 @@
-this)
+this#this_pron)
@@ -602 +602 @@
-nyght))
+nyght#night_n))
@@ -605 +605 @@
-I))
+I#i_pron))
@@ -607 +607 @@
-have)
+have#have_v)
@@ -609 +609 @@
-sent)
+sent#send_v1)
@@ -612 +612 @@
-my)
+my#my_adj)
@@ -614 +614 @@
-litell)
+litell#little_adj)
@@ -616 +616 @@
-boy))
+boy#boy_n1))
@@ -619 +619 @@
-with)
+with#with_prep)
@@ -622 +622 @@
-his)
+his#his_adj)
@@ -624 +624 @@
-nursse)))
+nursse#nurse_n1)))
@@ -627 +627 @@
-unto)
+unto#unto_prep)
@@ -631 +631 @@
-one)
+one#one_adj)
@@ -633 +633 @@
-poore)
+poore#poor_adj)
@@ -635 +635 @@
-mans))
+mans#man_n1))
@@ -637 +637 @@
-housse)))
+housse#house_n1)))
@@ -642 +642 @@
-to)
+to#to_prep)
@@ -644 +644 @@
-be)
+be#be_v)
@@ -646 +646 @@
-kept)
+kept#keep_v)
@@ -649 +649 @@
-privy))
+privy#privy_adj))
@@ -652 +652 @@
-there))
+there#there_adv))
@@ -655 +655 @@
-to)
+to#to_prep)
@@ -662 +662 @@
-we))
+we#we_pron))
@@ -664 +664 @@
-knowe)
+knowe#know_v)
@@ -667 +667 @@
-forther))))))
+forther#further_adj))))))
@@ -675 +675 @@
-And)
+And#and_conj1)
@@ -680 +680 @@
-what))
+what#what_pron))
@@ -689 +689 @@
-your)
+your#your_pron)
@@ -691 +691 @@
-Lordshipes))
+Lordshipes#lordship_n))
@@ -693 +693 @@
-mynd))
+mynd#mind_n1))
@@ -695 +695 @@
-is)
+is#be_v)
@@ -698 +698 @@
-in)
+in#in_prep)
@@ -701 +701 @@
-the)
+the#the_adj)
@@ -703 +703 @@
-premysses))))))
+premysses#premise_n))))))
@@ -706 +706 @@
-I))
+I#i_pron))
@@ -708 +708 @@
-wull)
+wull#will_v1)
@@ -711 +711 @@
-hertly))
+hertly#heartily_adv))
@@ -713 +713 @@
-besuche)
+besuche#beseech_v)
@@ -716 +716 @@
-you))
+you#you_pron))
@@ -719 +719 @@
-to)
+to#to_prep)
@@ -721 +721 @@
-send)
+send#send_v1)
@@ -724 +724 @@
-it))
+it#it_pron))
@@ -727 +727 @@
-to)
+to#to_prep)
@@ -730 +730 @@
-Catelwell)))
+Catelwell#NA)))
@@ -733 +733 @@
-with)
+with#with_prep)
@@ -736 +736 @@
-this)
+this#this_pron)
@@ -738 +738 @@
-berere))))
+berere#bearer_n))))
@@ -747 +747 @@
-Thus))
+Thus#thus_adv))
@@ -750 +750 @@
-th@)
+th@#the_adj)
@@ -752 +752 @@
-@oly)
+@oly#NA)
@@ -754 +754 @@
-Gost))
+Gost#NA))
@@ -756 +756 @@
-preserve)
+preserve#preserve_v)
@@ -759 +759 @@
-youre)
+youre#your_pron)
@@ -761 +761 @@
-good)
+good#good_adj)
@@ -763 +763 @@
-Lordship)
+Lordship#lordship_n)
@@ -766 +766 @@
-with)
+with#with_prep)
@@ -769 +769 @@
-my)
+my#my_adj)
@@ -771 +771 @@
-Lady)
+Lady#lady_n)
@@ -774 +774 @@
-my)
+my#my_adj)
@@ -776 +776 @@
-mother))))
+mother#mother_n1))))
@@ -779 +779 @@
-&)
+&#and_conj1)
@@ -782 +782 @@
-all)
+all#all_adj)
@@ -784 +784 @@
-youres))))
+youres#yours_pron))))
@@ -787 +787 @@
-in)
+in#in_prep)
@@ -790 +790 @@
-comfort)))
+comfort#comfort_n)))
@@ -798 +798 @@
-At)
+At#at_prep)
@@ -801 +801 @@
-Bolton))
+Bolton#NA))
@@ -809 +809 @@
-this)
+this#this_pron)
@@ -811 +811 @@
-Setterday)
+Setterday#NA)
@@ -814 +814 @@
-before)
+before#before_adv)
@@ -817 +817 @@
-day)))
+day#day_n)))
@@ -825 +825 @@
-Your)
+Your#your_pron)
@@ -827 +827 @@
-humbliest)
+humbliest#humble_adj)
@@ -829 +829 @@
-doughter)
+doughter#daughter_n)
@@ -834 +834 @@
-Kateryn)
+Kateryn#NA)
@@ -836 +836 @@
-Scrope))
+Scrope#NA))
