--- lords-1539-e1-h.psd.vert	2025-09-10 13:50:35
+++ lords-1539-e1-h.lem.vert	2025-09-10 13:50:45
@@ -17 +17 @@
-Our)
+Our#our_pron)
@@ -20 +20 @@
-most)
+most#most_adj)
@@ -22 +22 @@
-bounden))
+bounden#bounden_adj))
@@ -24 +24 @@
-dueties))
+dueties#duty_n))
@@ -27 +27 @@
-right)
+right#right_adv)
@@ -29 +29 @@
-humbly))
+humbly#humbly_adv))
@@ -31 +31 @@
-remembred)
+remembred#remember_v)
@@ -34 +34 @@
-to)
+to#to_prep)
@@ -37 +37 @@
-your)
+your#your_pron)
@@ -40 +40 @@
-most)
+most#most_adj)
@@ -42 +42 @@
-excellent))
+excellent#excellent_adj))
@@ -44 +44 @@
-Maieste)))
+Maieste#majesty_n)))
@@ -52 +52 @@
-Please)
+Please#please_v)
@@ -55 +55 @@
-your)
+your#your_pron)
@@ -58 +58 @@
-most)
+most#most_adj)
@@ -60 +60 @@
-noble))
+noble#noble_adj))
@@ -62 +62 @@
-Grace))
+Grace#grace_n))
@@ -65 +65 @@
-to)
+to#to_prep)
@@ -67 +67 @@
-be)
+be#be_v)
@@ -69 +69 @@
-advertised)
+advertised#advertise_v)
@@ -72 +72 @@
-that)
+that#that_conj)
@@ -76 +76 @@
-We)
+We#we_pron)
@@ -79 +79 @@
-your)
+your#your_pron)
@@ -82 +82 @@
-most)
+most#most_adj)
@@ -84 +84 @@
-humble))
+humble#humble_adj))
@@ -86 +86 @@
-subjects)
+subjects#subject_n)
@@ -89 +89 @@
-and)
+and#and_conj1)
@@ -92 +92 @@
-obedient)
+obedient#obedient_adj)
@@ -94 +94 @@
-servaunts)))))
+servaunts#servant_n)))))
@@ -96 +96 @@
-have)
+have#have_v)
@@ -99 +99 @@
-this)
+this#this_pron)
@@ -101 +101 @@
-present)
+present#present_adj)
@@ -103 +103 @@
-daye))
+daye#day_n))
@@ -105 +105 @@
-employed)
+employed#employ_v)
@@ -108 +108 @@
-all)
+all#all_adj)
@@ -110 +110 @@
-our)
+our#our_pron)
@@ -112 +112 @@
-most)
+most#most_adj)
@@ -114 +114 @@
-diligence)
+diligence#diligence_n1)
@@ -120 +120 @@
-industrie)))
+industrie#industry_n)))
@@ -125 +125 @@
-and)
+and#and_conj1)
@@ -128 +128 @@
-activite))))
+activite#activity_n))))
@@ -131 +131 @@
-to)
+to#to_prep)
@@ -133 +133 @@
-trye)
+trye#try_v)
@@ -135 +135 @@
-oute)
+oute#out_adv)
@@ -138 +138 @@
-the)
+the#the_adj)
@@ -140 +140 @@
-veray)
+veray#very_adj)
@@ -142 +142 @@
-botom)
+botom#bottom_n)
@@ -145 +145 @@
-and)
+and#and_conj1)
@@ -148 +148 @@
-pith)))
+pith#pith_n)))
@@ -151 +151 @@
-of)
+of#of_prep)
@@ -154 +154 @@
-suche)
+suche#such_adj)
@@ -156 +156 @@
-things)
+things#thing_n1)
@@ -159 +159 @@
-as)
+as#as_adv)
@@ -168 +168 @@
-the)
+the#the_adj)
@@ -170 +170 @@
-Lorde)
+Lorde#lord_n)
@@ -173 +173 @@
-Lawarre)))
+Lawarre#NA)))
@@ -175 +175 @@
-hath)
+hath#have_v)
@@ -177 +177 @@
-ben)
+ben#be_v)
@@ -179 +179 @@
-detected)
+detected#detect_v)
@@ -184 +184 @@
-to)
+to#to_prep)
@@ -186 +186 @@
-have)
+have#have_v)
@@ -188 +188 @@
-offended)
+offended#offend_v)
@@ -191 +191 @@
-your)
+your#your_pron)
@@ -193 +193 @@
-Majeste)))))))))))))
+Majeste#NA)))))))))))))
@@ -201 +201 @@
-But)
+But#but_prep)
@@ -204 +204 @@
-as)
+as#as_adv)
@@ -207 +207 @@
-yet)))
+yet#yet_adv)))
@@ -210 +210 @@
-we))
+we#we_pron))
@@ -212 +212 @@
-can)
+can#can_v1)
@@ -214 +214 @@
-fynde)
+fynde#find_v)
@@ -217 +217 @@
-no)
+no#no_adj)
@@ -219 +219 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -221 +221 @@
-grounde)
+grounde#ground_n)
@@ -224 +224 @@
-to)
+to#to_prep)
@@ -226 +226 @@
-committe)
+committe#commit_v)
@@ -229 +229 @@
-hym))
+hym#him_pron))
@@ -232 +232 @@
-to)
+to#to_prep)
@@ -235 +235 @@
-prison)))
+prison#prison_n)))
@@ -238 +238 @@
-into)
+into#into_prep)
@@ -242 +242 @@
-your)
+your#your_pron)
@@ -244 +244 @@
-Graces))
+Graces#grace_n))
@@ -246 +246 @@
-Towr)))))
+Towr#NA)))))
@@ -254 +254 @@
-And)
+And#and_conj1)
@@ -257 +257 @@
-for)
+for#for_prep)
@@ -260 +260 @@
-thies)
+thies#these_pron)
@@ -263 +263 @@
-two)
+two#two_adj)
@@ -265 +265 @@
-or)
+or#or_conj2)
@@ -267 +267 @@
-thre))
+thre#three_adj))
@@ -269 +269 @@
-dayis)))
+dayis#day_n)))
@@ -272 +272 @@
-th'@)
+th'@#the_adj)
@@ -274 +274 @@
-@affaires))
+@affaires#affair_n))
@@ -279 +279 @@
-as)
+as#as_adv)
@@ -290 +290 @@
-your)
+your#your_pron)
@@ -292 +292 @@
-Maieste))
+Maieste#majesty_n))
@@ -294 +294 @@
-knoweth))))
+knoweth#know_v))))
@@ -298 +298 @@
-be)
+be#be_v)
@@ -301 +301 @@
-suche)
+suche#such_adj)
@@ -304 +304 @@
-that)
+that#that_conj)
@@ -308 +308 @@
-we))
+we#we_pron))
@@ -310 +310 @@
-have)
+have#have_v)
@@ -312 +312 @@
-differred)
+differred#differ_v)
@@ -315 +315 @@
-tyll)
+tyll#till_prep)
@@ -322 +322 @@
-the)
+the#the_adj)
@@ -324 +324 @@
-same))
+same#same_adj))
@@ -326 +326 @@
-be)
+be#be_v)
@@ -328 +328 @@
-passed))))
+passed#pass_v))))
@@ -331 +331 @@
-the)
+the#the_adj)
@@ -333 +333 @@
-further)
+further#further_adj)
@@ -335 +335 @@
-enserching))
+enserching#ensearching_n))
@@ -340 +340 @@
-with)
+with#with_prep)
@@ -343 +343 @@
-all)
+all#all_adj)
@@ -345 +345 @@
-meanes)
+meanes#mean_n2)
@@ -348 +348 @@
-possible))
+possible#possible_adj))
@@ -351 +351 @@
-to)
+to#to_prep)
@@ -353 +353 @@
-trye)
+trye#try_v)
@@ -356 +356 @@
-the)
+the#the_adj)
@@ -358 +358 @@
-very)
+very#very_adj)
@@ -360 +360 @@
-effect)
+effect#effect_n)
@@ -363 +363 @@
-of)
+of#of_prep)
@@ -366 +366 @@
-his)
+his#his_adj)
@@ -368 +368 @@
-detection))))))))))
+detection#detection_n))))))))))
@@ -378 +378 @@
-In)
+In#in_prep)
@@ -381 +381 @@
-the)
+the#the_adj)
@@ -383 +383 @@
-meane)
+meane#mean_adj2)
@@ -385 +385 @@
-tyme)))
+tyme#time_n)))
@@ -388 +388 @@
-we))
+we#we_pron))
@@ -390 +390 @@
-have)
+have#have_v)
@@ -393 +393 @@
-in)
+in#in_prep)
@@ -397 +397 @@
-your)
+your#your_pron)
@@ -399 +399 @@
-Maiesties))
+Maiesties#majesty_n))
@@ -401 +401 @@
-name)))
+name#name_n)))
@@ -403 +403 @@
-commaunded)
+commaunded#command_v)
@@ -406 +406 @@
-hym))
+hym#him_pron))
@@ -409 +409 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-write)
+write#write_v)
@@ -414 +414 @@
-all)
+all#all_adj)
@@ -416 +416 @@
-suche)
+suche#such_adj)
@@ -418 +418 @@
-things)
+things#thing_n1)
@@ -422 +422 @@
-as)
+as#as_adv)
@@ -433 +433 @@
-he))
+he#he_pron))
@@ -435 +435 @@
-hath)
+hath#have_v)
@@ -438 +438 @@
-allredy))
+allredy#already_adj))
@@ -440 +440 @@
-confessed))))
+confessed#confess_v))))
@@ -445 +445 @@
-and)
+and#and_conj1)
@@ -450 +450 @@
-that)
+that#that_conj)
@@ -455 +455 @@
-can)
+can#can_v1)
@@ -457 +457 @@
-come)
+come#come_v)
@@ -460 +460 @@
-to)
+to#to_prep)
@@ -463 +463 @@
-his)
+his#his_adj)
@@ -465 +465 @@
-mynd))))))))))
+mynd#mind_n1))))))))))
@@ -470 +470 @@
-And)
+And#and_conj1)
@@ -474 +474 @@
-further))
+further#further_adj))
@@ -479 +479 @@
-that)
+that#that_conj)
@@ -483 +483 @@
-upon)
+upon#upon_prep)
@@ -486 +486 @@
-payne)
+payne#pain_n1)
@@ -489 +489 @@
-of)
+of#of_prep)
@@ -492 +492 @@
-his)
+his#his_adj)
@@ -494 +494 @@
-allegeaunce)))))
+allegeaunce#allegiance_n)))))
@@ -497 +497 @@
-he))
+he#he_pron))
@@ -499 +499 @@
-shal)
+shal#shall_v)
@@ -501 +501 @@
-kepe)
+kepe#keep_v)
@@ -504 +504 @@
-his)
+his#his_adj)
@@ -506 +506 @@
-house))
+house#house_n1))
@@ -511 +511 @@
-and)
+and#and_conj1)
@@ -513 +513 @@
-commone)
+commone#common_v)
@@ -516 +516 @@
-with)
+with#with_prep)
@@ -519 +519 @@
-no)
+no#no_adj)
@@ -521 +521 @@
-maner)
+maner#manner_n)
@@ -524 +524 @@
-suspecte)
+suspecte#suspect_adj)
@@ -526 +526 @@
-persone)))))
+persone#person_n)))))
@@ -529 +529 @@
-tyll)
+tyll#till_prep)
@@ -536 +536 @@
-we))
+we#we_pron))
@@ -538 +538 @@
-shall)
+shall#shall_v)
@@ -541 +541 @@
-further))
+further#further_adj))
@@ -543 +543 @@
-declare)
+declare#declare_v)
@@ -546 +546 @@
-unto)
+unto#unto_prep)
@@ -549 +549 @@
-hym)))
+hym#him_pron)))
@@ -552 +552 @@
-your)
+your#your_pron)
@@ -554 +554 @@
-graciouse)
+graciouse#gracious_adj)
@@ -556 +556 @@
-pleasur)))))))))
+pleasur#pleasure_n)))))))))
@@ -565 +565 @@
-Beseching)
+Beseching#beseech_v)
@@ -568 +568 @@
-your)
+your#your_pron)
@@ -571 +571 @@
-most)
+most#most_adj)
@@ -573 +573 @@
-noble)
+noble#noble_adj)
@@ -576 +576 @@
-and)
+and#and_conj1)
@@ -579 +579 @@
-benigne))))
+benigne#benign_adj))))
@@ -581 +581 @@
-Grace))
+Grace#grace_n))
@@ -585 +585 @@
-that)
+that#that_conj)
@@ -591 +591 @@
-seen)
+seen#see_v)
@@ -594 +594 @@
-upon)
+upon#upon_prep)
@@ -597 +597 @@
-consyderacion))))
+consyderacion#consideration_n))))
@@ -599 +599 @@
-that)
+that#that_conj)
@@ -605 +605 @@
-we))
+we#we_pron))
@@ -607 +607 @@
-fynde)
+fynde#find_v)
@@ -610 +610 @@
-as)
+as#as_adv)
@@ -613 +613 @@
-yet)))
+yet#yet_adv)))
@@ -616 +616 @@
-no)
+no#no_adj)
@@ -618 +618 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -620 +620 @@
-mater)
+mater#matter_n1)
@@ -623 +623 @@
-agenst)
+agenst#against_prep)
@@ -626 +626 @@
-hym)))))))
+hym#him_pron)))))))
@@ -631 +631 @@
-and)
+and#and_conj1)
@@ -634 +634 @@
-that)
+that#that_conj)
@@ -638 +638 @@
-having)
+having#have_v)
@@ -641 +641 @@
-respect))
+respect#respect_n))
@@ -645 +645 @@
-as@)
+as@#as_adv)
@@ -647 +647 @@
-@well)
+@well#well_adv)
@@ -652 +652 @@
-to)
+to#to_prep)
@@ -655 +655 @@
-your)
+your#your_pron)
@@ -657 +657 @@
-mercyfull)
+mercyfull#merciful_adj)
@@ -659 +659 @@
-clemencye)))
+clemencye#clemency_n)))
@@ -664 +664 @@
-as)
+as#as_adv)
@@ -666 +666 @@
-also)
+also#also_adv)
@@ -669 +669 @@
-to)
+to#to_prep)
@@ -673 +673 @@
-your)
+your#your_pron)
@@ -675 +675 @@
-Graces))
+Graces#grace_n))
@@ -677 +677 @@
-honor))))))
+honor#honour_n))))))
@@ -681 +681 @@
-that)
+that#that_conj)
@@ -688 +688 @@
-wold)
+wold#will_v1)
@@ -690 +690 @@
-not)
+not#not_adv)
@@ -692 +692 @@
-have)
+have#have_v)
@@ -696 +696 @@
-hym))
+hym#him_pron))
@@ -699 +699 @@
-upon)
+upon#upon_prep)
@@ -702 +702 @@
-a)
+a#a_adj2)
@@ -704 +704 @@
-weak)
+weak#weak_adj)
@@ -706 +706 @@
-grounde)
+grounde#ground_n)
@@ -712 +712 @@
-wherof))
+wherof#whereof_adv))
@@ -720 +720 @@
-he))
+he#he_pron))
@@ -722 +722 @@
-myght)
+myght#may_v1)
@@ -724 +724 @@
-clere)
+clere#clear_v)
@@ -727 +727 @@
-hym)
+hym#him_pron)
@@ -729 +729 @@
-self))
+self#self_pron))
@@ -732 +732 @@
-afterwarde))))))
+afterwarde#afterward_adv))))))
@@ -736 +736 @@
-to)
+to#to_prep)
@@ -738 +738 @@
-be)
+be#be_v)
@@ -741 +741 @@
-extremely))
+extremely#extremely_adv))
@@ -743 +743 @@
-handeled))))))))
+handeled#handle_v1))))))))
@@ -748 +748 @@
-we))
+we#we_pron))
@@ -750 +750 @@
-have)
+have#have_v)
@@ -752 +752 @@
-respyted)
+respyted#respite_v)
@@ -755 +755 @@
-his)
+his#his_adj)
@@ -757 +757 @@
-Emprisonement))
+Emprisonement#imprisonment_n))
@@ -766 +766 @@
-It))
+It#it_pron))
@@ -768 +768 @@
-may)
+may#may_v1)
@@ -770 +770 @@
-please)
+please#please_v)
@@ -775 +775 @@
-your)
+your#your_pron)
@@ -777 +777 @@
-Highnes))
+Highnes#highness_n))
@@ -781 +781 @@
-not)
+not#not_adv)
@@ -783 +783 @@
-to)
+to#to_prep)
@@ -785 +785 @@
-be)
+be#be_v)
@@ -787 +787 @@
-offended)
+offended#offend_v)
@@ -790 +790 @@
-therewith)))
+therewith#therewith_adv)))
@@ -795 +795 @@
-but)
+but#but_prep)
@@ -798 +798 @@
-to)
+to#to_prep)
@@ -800 +800 @@
-pardone)
+pardone#pardon_v)
@@ -803 +803 @@
-us))
+us#us_pron))
@@ -806 +806 @@
-as)
+as#as_adv)
@@ -815 +815 @@
-we))
+we#we_pron))
@@ -817 +817 @@
-trust)
+trust#trust_v)
@@ -826 +826 @@
-your)
+your#your_pron)
@@ -828 +828 @@
-Highnes))
+Highnes#highness_n))
@@ -831 +831 @@
-of)
+of#of_prep)
@@ -834 +834 @@
-your)
+your#your_pron)
@@ -837 +837 @@
-most)
+most#most_adj)
@@ -839 +839 @@
-gracious))
+gracious#gracious_adj))
@@ -841 +841 @@
-disposition)))
+disposition#disposition_n)))
@@ -843 +843 @@
-woll)
+woll#will_v1)
@@ -854 +854 @@
-Assuredly))
+Assuredly#assuredly_adv))
@@ -857 +857 @@
-if)
+if#if_conj)
@@ -864 +864 @@
-we))
+we#we_pron))
@@ -866 +866 @@
-shuld)
+shuld#shall_v)
@@ -868 +868 @@
-have)
+have#have_v)
@@ -870 +870 @@
-committed)
+committed#commit_v)
@@ -873 +873 @@
-hym))
+hym#him_pron))
@@ -876 +876 @@
-to)
+to#to_prep)
@@ -879 +879 @@
-the)
+the#the_adj)
@@ -881 +881 @@
-Towr))))))
+Towr#NA))))))
@@ -888 +888 @@
-howesoever))
+howesoever#howsoever_adv))
@@ -896 +896 @@
-the)
+the#the_adj)
@@ -898 +898 @@
-matier))
+matier#matter_n1))
@@ -900 +900 @@
-shuld)
+shuld#shall_v)
@@ -902 +902 @@
-waye))))
+waye#weigh_v1))))
@@ -907 +907 @@
-it))
+it#it_pron))
@@ -909 +909 @@
-shuld)
+shuld#shall_v)
@@ -913 +913 @@
-so)
+so#so_adv)
@@ -915 +915 @@
-moch))
+moch#much_adj))
@@ -919 +919 @@
-touch)
+touch#touch_v)
@@ -922 +922 @@
-his)
+his#his_adj)
@@ -924 +924 @@
-honeste))
+honeste#honesty_n))
@@ -929 +929 @@
-and)
+and#and_conj1)
@@ -932 +932 @@
-he))
+he#he_pron))
@@ -935 +935 @@
-by)
+by#by_prep)
@@ -938 +938 @@
-the)
+the#the_adj)
@@ -940 +940 @@
-same)))
+same#same_adj)))
@@ -942 +942 @@
-shuld)
+shuld#shall_v)
@@ -944 +944 @@
-be)
+be#be_v)
@@ -946 +946 @@
-put)
+put#put_v)
@@ -949 +949 @@
-to)
+to#to_prep)
@@ -952 +952 @@
-such)
+such#such_adj)
@@ -954 +954 @@
-a)
+a#a_adj2)
@@ -956 +956 @@
-rebuke))))
+rebuke#rebuke_n))))
@@ -961 +961 @@
-that)
+that#that_conj)
@@ -965 +965 @@
-he))
+he#he_pron))
@@ -967 +967 @@
-shuld)
+shuld#shall_v)
@@ -970 +970 @@
-never))
+never#never_adv))
@@ -972 +972 @@
-be)
+be#be_v)
@@ -975 +975 @@
-hable)
+hable#able_adj)
@@ -978 +978 @@
-to)
+to#to_prep)
@@ -980 +980 @@
-recover)
+recover#recover_v1)
@@ -983 +983 @@
-it))))))
+it#it_pron))))))
@@ -992 +992 @@
-Therefore))
+Therefore#therefore_adv))
@@ -995 +995 @@
-agayn))
+agayn#again_adv))
@@ -1001 +1001 @@
-most)
+most#most_adj)
@@ -1003 +1003 @@
-humbly))
+humbly#humbly_adv))
@@ -1005 +1005 @@
-prostrate))
+prostrate#prostrate_adj))
@@ -1008 +1008 @@
-at)
+at#at_prep)
@@ -1012 +1012 @@
-your)
+your#your_pron)
@@ -1014 +1014 @@
-Maiesties))
+Maiesties#majesty_n))
@@ -1016 +1016 @@
-fete)))
+fete#foot_n)))
@@ -1021 +1021 @@
-we))
+we#we_pron))
@@ -1023 +1023 @@
-beseche)
+beseche#beseech_v)
@@ -1026 +1026 @@
-the)
+the#the_adj)
@@ -1028 +1028 @@
-same))
+same#same_adj))
@@ -1031 +1031 @@
-to)
+to#to_prep)
@@ -1033 +1033 @@
-pardone)
+pardone#pardon_v)
@@ -1036 +1036 @@
-us)))
+us#us_pron)))
@@ -1041 +1041 @@
-not)
+not#not_adv)
@@ -1043 +1043 @@
-doubting)
+doubting#doubt_v)
@@ -1046 +1046 @@
-but)
+but#but_prep)
@@ -1052 +1052 @@
-in)
+in#in_prep)
@@ -1056 +1056 @@
-the)
+the#the_adj)
@@ -1058 +1058 @@
-same))
+same#same_adj))
@@ -1061 +1061 @@
-and)
+and#and_conj1)
@@ -1064 +1064 @@
-all)
+all#all_adj)
@@ -1066 +1066 @@
-other)
+other#other_adj)
@@ -1069 +1069 @@
-your)
+your#your_pron)
@@ -1071 +1071 @@
-Highnes))
+Highnes#highness_n))
@@ -1073 +1073 @@
-maters)))))
+maters#matter_n1)))))
@@ -1076 +1076 @@
-we))
+we#we_pron))
@@ -1078 +1078 @@
-shal)
+shal#shall_v)
@@ -1080 +1080 @@
-not)
+not#not_adv)
@@ -1082 +1082 @@
-faile)
+faile#fail_v)
@@ -1085 +1085 @@
-to)
+to#to_prep)
@@ -1087 +1087 @@
-endevoyre)
+endevoyre#endeavour_v)
@@ -1090 +1090 @@
-our)
+our#our_pron)
@@ -1092 +1092 @@
-selfs))
+selfs#self_pron))
@@ -1095 +1095 @@
-according)
+according#accord_v)
@@ -1098 +1098 @@
-to)
+to#to_prep)
@@ -1101 +1101 @@
-our)
+our#our_pron)
@@ -1104 +1104 @@
-most)
+most#most_adj)
@@ -1106 +1106 @@
-bounden))
+bounden#bounden_adj))
@@ -1108 +1108 @@
-dueties))))
+dueties#duty_n))))
@@ -1111 +1111 @@
-as)
+as#as_adv)
@@ -1121 +1121 @@
-shal@)
+shal@#shall_v)
@@ -1123 +1123 @@
-@be)
+@be#be_v)
@@ -1129 +1129 @@
-we))
+we#we_pron))
@@ -1131 +1131 @@
-hoope))
+hoope#hope_v))
@@ -1136 +1136 @@
-to)
+to#to_prep)
@@ -1140 +1140 @@
-your)
+your#your_pron)
@@ -1142 +1142 @@
-Graces))
+Graces#grace_n))
@@ -1144 +1144 @@
-satisfaction)
+satisfaction#satisfaction_n)
@@ -1147 +1147 @@
-and)
+and#and_conj1)
@@ -1150 +1150 @@
-contentement))))))))))))
+contentement#contentment_n))))))))))))
@@ -1158 +1158 @@
-Prayeng)
+Prayeng#pray_v)
@@ -1161 +1161 @@
-Allmyghty)
+Allmyghty#almighty_adj)
@@ -1163 +1163 @@
-God))
+God#NA))
@@ -1166 +1166 @@
-to)
+to#to_prep)
@@ -1168 +1168 @@
-maynteyne)
+maynteyne#maintain_v)
@@ -1172 +1172 @@
-your)
+your#your_pron)
@@ -1174 +1174 @@
-Ma=ties=.))
+Ma=ties=.#majesty_n))
@@ -1176 +1176 @@
-prosperouse)
+prosperouse#prosperous_adj)
@@ -1178 +1178 @@
-regne)
+regne#reign_n)
@@ -1184 +1184 @@
-honor)))
+honor#honour_n)))
@@ -1189 +1189 @@
-and)
+and#and_conj1)
@@ -1192 +1192 @@
-lif))))
+lif#life_n))))
@@ -1197 +1197 @@
-to)
+to#to_prep)
@@ -1200 +1200 @@
-our)
+our#our_pron)
@@ -1202 +1202 @@
-fruition)))
+fruition#fruition_n)))
@@ -1206 +1206 @@
-long))
+long#long_adj1))
@@ -1208 +1208 @@
-to)
+to#to_prep)
@@ -1210 +1210 @@
-endure)))
+endure#endure_v)))
@@ -1218 +1218 @@
-Writen)
+Writen#write_v)
@@ -1221 +1221 @@
-at)
+at#at_prep)
@@ -1225 +1225 @@
-your)
+your#your_pron)
@@ -1227 +1227 @@
-Graces))
+Graces#grace_n))
@@ -1229 +1229 @@
-Cite)
+Cite#city_n)
@@ -1232 +1232 @@
-of)
+of#of_prep)
@@ -1235 +1235 @@
-London)))))
+London#NA)))))
@@ -1240 +1240 @@
-the)
+the#the_adj)
@@ -1242 +1242 @@
-first)
+first#first_adj)
@@ -1244 +1244 @@
-daye)
+daye#day_n)
@@ -1247 +1247 @@
-of)
+of#of_prep)
@@ -1250 +1250 @@
-Decembr))))
+Decembr#NA))))
@@ -1253 +1253 @@
-the)
+the#the_adj)
@@ -1255 +1255 @@
-xxx=th=)
+xxx=th=#NA)
@@ -1257 +1257 @@
-yere)
+yere#year_n)
@@ -1260 +1260 @@
-of)
+of#of_prep)
@@ -1263 +1263 @@
-yo=r=)
+yo=r=#your_pron)
@@ -1266 +1266 @@
-most)
+most#most_adj)
@@ -1268 +1268 @@
-prosperouse)
+prosperouse#prosperous_adj)
@@ -1271 +1271 @@
-and)
+and#and_conj1)
@@ -1274 +1274 @@
-noble))))
+noble#noble_adj))))
@@ -1276 +1276 @@
-regne))))
+regne#reign_n))))
@@ -1285 +1285 @@
-Your)
+Your#your_pron)
@@ -1287 +1287 @@
-Maiesties))
+Maiesties#majesty_n))
@@ -1290 +1290 @@
-most)
+most#most_adj)
@@ -1292 +1292 @@
-bounden)
+bounden#bounden_adj)
@@ -1296 +1296 @@
-feithfull)))
+feithfull#faithful_adj)))
@@ -1299 +1299 @@
-and)
+and#and_conj1)
@@ -1302 +1302 @@
-humble))))
+humble#humble_adj))))
@@ -1304 +1304 @@
-subjects)
+subjects#subject_n)
@@ -1308 +1308 @@
-servaunts)))
+servaunts#servant_n)))
@@ -1311 +1311 @@
-and)
+and#and_conj1)
@@ -1314 +1314 @@
-beadesmen)))
+beadesmen#beadsman_n)))
@@ -1318 +1318 @@
-Thomas)
+Thomas#NA)
@@ -1320 +1320 @@
-Audeley))
+Audeley#NA))
@@ -1326 +1326 @@
-T.)
+T.#NA)
@@ -1328 +1328 @@
-Norfolk)))
+Norfolk#NA)))
@@ -1334 +1334 @@
-Charlys)
+Charlys#NA)
@@ -1336 +1336 @@
-Suffolk)))
+Suffolk#NA)))
@@ -1342 +1342 @@
-Thomas)
+Thomas#NA)
@@ -1344 +1344 @@
-Crumwell)))
+Crumwell#NA)))
@@ -1350 +1350 @@
-Robt.)
+Robt.#NA)
@@ -1352 +1352 @@
-Sussex)))
+Sussex#NA)))
@@ -1358 +1358 @@
-E.)
+E.#NA)
@@ -1360 +1360 @@
-Hertford)))
+Hertford#NA)))
@@ -1366 +1366 @@
-W.)
+W.#NA)
@@ -1368 +1368 @@
-Southampton)))))
+Southampton#NA)))))
@@ -1376 +1376 @@
-To)
+To#to_prep)
@@ -1380 +1380 @@
-the)
+the#the_adj)
@@ -1382 +1382 @@
-Kings))
+Kings#king_n))
@@ -1385 +1385 @@
-moost)
+moost#most_adj)
@@ -1387 +1387 @@
-Royall))
+Royall#royal_adj))
@@ -1389 +1389 @@
-Maieste))
+Maieste#majesty_n))
