--- morelet-1529-e1-h.psd.vert	2025-09-10 13:50:35
+++ morelet-1529-e1-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Maystres)
+Maystres#NA)
@@ -16 +16 @@
-Alyce))
+Alyce#NA))
@@ -21 +21 @@
-in)
+in#in_prep)
@@ -24 +24 @@
-my)
+my#my_adj)
@@ -27 +27 @@
-moste)
+moste#most_adj)
@@ -29 +29 @@
-hartie))
+hartie#hearty_adj))
@@ -31 +31 @@
-wyse)))
+wyse#wise_n1)))
@@ -34 +34 @@
-I))
+I#i_pron))
@@ -36 +36 @@
-recommende)
+recommende#recommend_v1)
@@ -39 +39 @@
-me))
+me#me_pron1))
@@ -42 +42 @@
-to)
+to#to_prep)
@@ -45 +45 @@
-you)))
+you#you_pron)))
@@ -55 +55 @@
-And)
+And#and_conj1)
@@ -58 +58 @@
-where)
+where#where_adv)
@@ -60 +60 @@
-as)
+as#as_adv)
@@ -67 +67 @@
-I))
+I#i_pron))
@@ -69 +69 @@
-am)
+am#be_v)
@@ -71 +71 @@
-enformed)
+enformed#inform_v)
@@ -74 +74 @@
-by)
+by#by_prep)
@@ -77 +77 @@
-my)
+my#my_adj)
@@ -79 +79 @@
-sone)
+sone#son_n1)
@@ -82 +82 @@
-Heron))))
+Heron#NA))))
@@ -85 +85 @@
-of)
+of#of_prep)
@@ -88 +88 @@
-the)
+the#the_adj)
@@ -90 +90 @@
-losse)
+losse#loss_n1)
@@ -93 +93 @@
-of)
+of#of_prep)
@@ -97 +97 @@
-our)
+our#our_pron)
@@ -101 +101 @@
-barns)
+barns#barn_n)
@@ -104 +104 @@
-and)
+and#and_conj1)
@@ -107 +107 @@
-our)
+our#our_pron)
@@ -109 +109 @@
-neighbours)
+neighbours#neighbour_n)
@@ -111 +111 @@
-also)))
+also#also_adv)))
@@ -114 +114 @@
-with)
+with#with_prep)
@@ -117 +117 @@
-all)
+all#all_adj)
@@ -119 +119 @@
-the)
+the#the_adj)
@@ -121 +121 @@
-corne)
+corne#corn_n1)
@@ -126 +126 @@
-that)
+that#that_conj)
@@ -131 +131 @@
-was)
+was#be_v)
@@ -134 +134 @@
-therin)))))))))))))
+therin#therein_adv)))))))))))))
@@ -139 +139 @@
-albeit)
+albeit#albeit_conj)
@@ -148 +148 @@
-savyng)
+savyng#saving_prep)
@@ -151 +151 @@
-Goddis)
+Goddis#NA)
@@ -153 +153 @@
-pleasuer)))
+pleasuer#pleasure_n)))
@@ -158 +158 @@
-it))
+it#it_pron))
@@ -160 +160 @@
-wer)
+wer#be_v)
@@ -163 +163 @@
-greate)
+greate#great_adj)
@@ -165 +165 @@
-pytie)
+pytie#pity_n)
@@ -168 +168 @@
-of)
+of#of_prep)
@@ -173 +173 @@
-so)
+so#so_adv)
@@ -175 +175 @@
-myche))
+myche#much_adj))
@@ -179 +179 @@
-good)
+good#good_adj)
@@ -181 +181 @@
-corne))
+corne#corn_n1))
@@ -183 +183 @@
-loste)))))))
+loste#lose_v1)))))))
@@ -186 +186 @@
-yet))
+yet#yet_adv))
@@ -189 +189 @@
-sythe)
+sythe#sith_adv)
@@ -196 +196 @@
-it))
+it#it_pron))
@@ -198 +198 @@
-hathe)
+hathe#have_v)
@@ -200 +200 @@
-lyked)
+lyked#like_v1)
@@ -203 +203 @@
-hym))
+hym#him_pron))
@@ -206 +206 @@
-to)
+to#to_prep)
@@ -208 +208 @@
-sende)
+sende#send_v1)
@@ -211 +211 @@
-vs))
+vs#us_pron))
@@ -214 +214 @@
-suche)
+suche#such_adj)
@@ -216 +216 @@
-a)
+a#a_adj2)
@@ -218 +218 @@
-chaunce))))))
+chaunce#chance_n))))))
@@ -223 +223 @@
-we))
+we#we_pron))
@@ -226 +226 @@
-muste)
+muste#must_v1)
@@ -228 +228 @@
-and))
+and#and_conj1))
@@ -230 +230 @@
-ar)
+ar#be_v)
@@ -232 +232 @@
-bounden)
+bounden#bind_v)
@@ -236 +236 @@
-not)
+not#not_adv)
@@ -238 +238 @@
-onely)
+onely#only_adj)
@@ -240 +240 @@
-to)
+to#to_prep)
@@ -242 +242 @@
-be)
+be#be_v)
@@ -245 +245 @@
-content)))
+content#content_adj)))
@@ -248 +248 @@
-but)
+but#but_prep)
@@ -250 +250 @@
-also)
+also#also_adv)
@@ -253 +253 @@
-to)
+to#to_prep)
@@ -255 +255 @@
-be)
+be#be_v)
@@ -258 +258 @@
-glade)
+glade#glad_adj)
@@ -261 +261 @@
-of)
+of#of_prep)
@@ -264 +264 @@
-his)
+his#his_adj)
@@ -266 +266 @@
-visitacion)))))))
+visitacion#visitation_n)))))))
@@ -275 +275 @@
-He))
+He#he_pron))
@@ -277 +277 @@
-sent)
+sent#send_v1)
@@ -280 +280 @@
-vs))
+vs#us_pron))
@@ -283 +283 @@
-all)
+all#all_adj)
@@ -288 +288 @@
-that)
+that#that_conj)
@@ -294 +294 @@
-we))
+we#we_pron))
@@ -296 +296 @@
-haue)
+haue#have_v)
@@ -298 +298 @@
-loste)))))
+loste#lose_v1)))))
@@ -304 +304 @@
-and)
+and#and_conj1)
@@ -307 +307 @@
-sythe)
+sythe#sith_adv)
@@ -314 +314 @@
-he))
+he#he_pron))
@@ -316 +316 @@
-hathe)
+hathe#have_v)
@@ -319 +319 @@
-by)
+by#by_prep)
@@ -322 +322 @@
-syche)
+syche#such_adj)
@@ -324 +324 @@
-a)
+a#a_adj2)
@@ -326 +326 @@
-chaunce)))
+chaunce#chance_n)))
@@ -328 +328 @@
-taken)
+taken#take_v)
@@ -331 +331 @@
-yt))
+yt#it_pron))
@@ -334 +334 @@
-away))
+away#away_adv))
@@ -337 +337 @@
-ageyne)))))
+ageyne#again_adv)))))
@@ -340 +340 @@
-his)
+his#his_adj)
@@ -342 +342 @@
-pleasuer))
+pleasuer#pleasure_n))
@@ -344 +344 @@
-be)
+be#be_v)
@@ -346 +346 @@
-fulfylled)
+fulfylled#fulfil_v)
@@ -354 +354 @@
-let)
+let#let_v1)
@@ -359 +359 @@
-vs))
+vs#us_pron))
@@ -362 +362 @@
-never))
+never#never_adv))
@@ -364 +364 @@
-gruge)
+gruge#grudge_v)
@@ -367 +367 @@
-therat)))
+therat#thereat_adv)))
@@ -370 +370 @@
-but)
+but#but_prep)
@@ -375 +375 @@
-take)
+take#take_v)
@@ -378 +378 @@
-in)
+in#in_prep)
@@ -381 +381 @@
-good)
+good#good_adj)
@@ -383 +383 @@
-worth)))))
+worth#worth_n1)))))
@@ -386 +386 @@
-and)
+and#and_conj1)
@@ -392 +392 @@
-hartely))
+hartely#heartily_adv))
@@ -394 +394 @@
-thanke)
+thanke#thank_v)
@@ -397 +397 @@
-hym))
+hym#him_pron))
@@ -401 +401 @@
-as)
+as#as_adv)
@@ -403 +403 @@
-well)
+well#well_adv)
@@ -408 +408 @@
-for)
+for#for_prep)
@@ -411 +411 @@
-aduersytie)))
+aduersytie#adversity_n)))
@@ -414 +414 @@
-as)
+as#as_adv)
@@ -417 +417 @@
-for)
+for#for_prep)
@@ -420 +420 @@
-prosperytie)))))))))
+prosperytie#prosperity_n)))))))))
@@ -426 +426 @@
-and)
+and#and_conj1)
@@ -429 +429 @@
-peraduenture))
+peraduenture#peradventure_adv))
@@ -434 +434 @@
-we))
+we#we_pron))
@@ -436 +436 @@
-haue)
+haue#have_v)
@@ -439 +439 @@
-more)
+more#more_adj)
@@ -441 +441 @@
-cause)
+cause#cause_n)
@@ -444 +444 @@
-to)
+to#to_prep)
@@ -446 +446 @@
-thanke)
+thanke#thank_v)
@@ -449 +449 @@
-hym))
+hym#him_pron))
@@ -452 +452 @@
-for)
+for#for_prep)
@@ -455 +455 @@
-our)
+our#our_pron)
@@ -457 +457 @@
-losse)))
+losse#loss_n1)))
@@ -460 +460 @@
-then)
+then#than_conj)
@@ -463 +463 @@
-for)
+for#for_prep)
@@ -466 +466 @@
-our)
+our#our_pron)
@@ -468 +468 @@
-wynnyng))))))
+wynnyng#winning_n1))))))
@@ -476 +476 @@
-for)
+for#for_prep)
@@ -479 +479 @@
-his)
+his#his_adj)
@@ -481 +481 @@
-wysedome))
+wysedome#wisdom_n))
@@ -484 +484 @@
-better)
+better#better_adj)
@@ -488 +488 @@
-seethe)
+seethe#see_v)
@@ -492 +492 @@
-what))
+what#what_pron))
@@ -499 +499 @@
-ys)
+ys#be_v)
@@ -502 +502 @@
-good))
+good#good_adj))
@@ -505 +505 @@
-for)
+for#for_prep)
@@ -508 +508 @@
-vs)))))
+vs#us_pron)))))
@@ -511 +511 @@
-then)
+then#than_conj)
@@ -522 +522 @@
-we)
+we#we_pron)
@@ -526 +526 @@
-do)
+do#do_v)
@@ -531 +531 @@
-ourselves)))))
+ourselves#ourself_pron)))))
@@ -540 +540 @@
-Therfore))
+Therfore#therefore_adv))
@@ -544 +544 @@
-I))
+I#i_pron))
@@ -546 +546 @@
-pray)
+pray#pray_v)
@@ -549 +549 @@
-you)))
+you#you_pron)))
@@ -551 +551 @@
-be)
+be#be_v)
@@ -554 +554 @@
-of)
+of#of_prep)
@@ -557 +557 @@
-good)
+good#good_adj)
@@ -559 +559 @@
-chere))))
+chere#cheer_n1))))
@@ -565 +565 @@
-and)
+and#and_conj1)
@@ -567 +567 @@
-take)
+take#take_v)
@@ -570 +570 @@
-all)
+all#all_adj)
@@ -572 +572 @@
-the)
+the#the_adj)
@@ -574 +574 @@
-howshold))
+howshold#household_n))
@@ -577 +577 @@
-with)
+with#with_prep)
@@ -580 +580 @@
-you)))
+you#you_pron)))
@@ -583 +583 @@
-to)
+to#to_prep)
@@ -586 +586 @@
-chyrche))))
+chyrche#church_n))))
@@ -592 +592 @@
-and)
+and#and_conj1)
@@ -595 +595 @@
-ther))
+ther#there_adv))
@@ -597 +597 @@
-thanke)
+thanke#thank_v)
@@ -600 +600 @@
-God))
+God#NA))
@@ -603 +603 @@
-bothe)
+bothe#both_pron)
@@ -606 +606 @@
-for)
+for#for_prep)
@@ -609 +609 @@
-that)
+that#that_pron)
@@ -620 +620 @@
-he))
+he#he_pron))
@@ -622 +622 @@
-hathe)
+hathe#have_v)
@@ -624 +624 @@
-geven)
+geven#give_v)
@@ -627 +627 @@
-vs))))))
+vs#us_pron))))))
@@ -630 +630 @@
-and)
+and#and_conj1)
@@ -633 +633 @@
-for)
+for#for_prep)
@@ -636 +636 @@
-that)
+that#that_pron)
@@ -647 +647 @@
-he))
+he#he_pron))
@@ -649 +649 @@
-hathe)
+hathe#have_v)
@@ -651 +651 @@
-taken)
+taken#take_v)
@@ -654 +654 @@
-from)
+from#from_prep)
@@ -657 +657 @@
-vs))))))))
+vs#us_pron))))))))
@@ -660 +660 @@
-and)
+and#and_conj1)
@@ -663 +663 @@
-for)
+for#for_prep)
@@ -666 +666 @@
-that)
+that#that_pron)
@@ -677 +677 @@
-he))
+he#he_pron))
@@ -679 +679 @@
-hathe)
+hathe#have_v)
@@ -681 +681 @@
-lefte)
+lefte#leave_v1)
@@ -684 +684 @@
-vs))))
+vs#us_pron))))
@@ -690 +690 @@
-which))
+which#which_adj))
@@ -698 +698 @@
-yf)
+yf#if_conj)
@@ -705 +705 @@
-yt))
+yt#it_pron))
@@ -707 +707 @@
-please)
+please#please_v)
@@ -710 +710 @@
-hym)))))
+hym#him_pron)))))
@@ -713 +713 @@
-he))
+he#he_pron))
@@ -715 +715 @@
-can)
+can#can_v1)
@@ -717 +717 @@
-increase)
+increase#increase_v)
@@ -724 +724 @@
-when))
+when#when_adv))
@@ -732 +732 @@
-he))
+he#he_pron))
@@ -734 +734 @@
-wyll)
+wyll#will_v1)
@@ -742 +742 @@
-and)
+and#and_conj1)
@@ -745 +745 @@
-yf)
+yf#if_conj)
@@ -752 +752 @@
-it))
+it#it_pron))
@@ -754 +754 @@
-please)
+please#please_v)
@@ -757 +757 @@
-hym))
+hym#him_pron))
@@ -760 +760 @@
-to)
+to#to_prep)
@@ -762 +762 @@
-leve)
+leve#leave_v1)
@@ -765 +765 @@
-vs))
+vs#us_pron))
@@ -769 +769 @@
-yet)
+yet#yet_adv)
@@ -771 +771 @@
-lesse)))))))
+lesse#less_adj)))))))
@@ -776 +776 @@
-at)
+at#at_prep)
@@ -779 +779 @@
-his)
+his#his_adj)
@@ -781 +781 @@
-pleasuer)))
+pleasuer#pleasure_n)))
@@ -783 +783 @@
-be)
+be#be_v)
@@ -786 +786 @@
-yt))
+yt#it_pron))
@@ -795 +795 @@
-I))
+I#i_pron))
@@ -797 +797 @@
-pray)
+pray#pray_v)
@@ -800 +800 @@
-you))
+you#you_pron))
@@ -804 +804 @@
-to)
+to#to_prep)
@@ -806 +806 @@
-make)
+make#make_v1)
@@ -809 +809 @@
-some)
+some#some_pron)
@@ -811 +811 @@
-good)
+good#good_adj)
@@ -813 +813 @@
-enserche))
+enserche#search_n))
@@ -817 +817 @@
-what))
+what#what_pron))
@@ -825 +825 @@
-my)
+my#my_adj)
@@ -827 +827 @@
-poore)
+poore#poor_adj)
@@ -829 +829 @@
-neyghebors))
+neyghebors#neighbour_n))
@@ -831 +831 @@
-haue)
+haue#have_v)
@@ -833 +833 @@
-loste))))
+loste#lose_v1))))
@@ -836 +836 @@
-and)
+and#and_conj1)
@@ -841 +841 @@
-byd)
+byd#bid_v)
@@ -845 +845 @@
-them))
+them#them_pron))
@@ -847 +847 @@
-take)
+take#take_v)
@@ -850 +850 @@
-no)
+no#no_adj)
@@ -852 +852 @@
-thought)
+thought#thought_n)
@@ -855 +855 @@
-therfore)))))))
+therfore#therefore_adv)))))))
@@ -863 +863 @@
-for)
+for#for_prep)
@@ -866 +866 @@
-and)
+and#and_conj1)
@@ -873 +873 @@
-I))
+I#i_pron))
@@ -875 +875 @@
-shuld)
+shuld#shall_v)
@@ -877 +877 @@
-not)
+not#not_adv)
@@ -879 +879 @@
-leve)
+leve#leave_v1)
@@ -882 +882 @@
-my)
+my#my_adj)
@@ -884 +884 @@
-selff))
+selff#self_pron))
@@ -887 +887 @@
-a)
+a#a_adj2)
@@ -889 +889 @@
-spone)))))
+spone#spoon_n)))))
@@ -892 +892 @@
-there))
+there#there_adv))
@@ -894 +894 @@
-shall)
+shall#shall_v)
@@ -897 +897 @@
-no)
+no#no_adj)
@@ -899 +899 @@
-poore)
+poore#poor_adj)
@@ -901 +901 @@
-neghebore)
+neghebore#neighbour_n)
@@ -904 +904 @@
-of)
+of#of_prep)
@@ -907 +907 @@
-myne))))
+myne#mine_adj))))
@@ -909 +909 @@
-berre)
+berre#bere_v)
@@ -912 +912 @@
-no)
+no#no_adj)
@@ -914 +914 @@
-losse))
+losse#loss_n1))
@@ -917 +917 @@
-by)
+by#by_prep)
@@ -920 +920 @@
-eny)
+eny#any_adj)
@@ -922 +922 @@
-chaunce)
+chaunce#chance_n)
@@ -932 +932 @@
-hapned)
+hapned#happen_v)
@@ -935 +935 @@
-in)
+in#in_prep)
@@ -938 +938 @@
-my)
+my#my_adj)
@@ -940 +940 @@
-howse)))))))
+howse#house_n1)))))))
@@ -950 +950 @@
-I))
+I#i_pron))
@@ -952 +952 @@
-pray)
+pray#pray_v)
@@ -955 +955 @@
-you)))
+you#you_pron)))
@@ -957 +957 @@
-be)
+be#be_v)
@@ -960 +960 @@
-wyth)
+wyth#with_prep)
@@ -964 +964 @@
-my)
+my#my_adj)
@@ -966 +966 @@
-children))
+children#child_n))
@@ -969 +969 @@
-and)
+and#and_conj1)
@@ -972 +972 @@
-your)
+your#your_pron)
@@ -974 +974 @@
-howsholde)))))
+howsholde#household_n)))))
@@ -977 +977 @@
-mery)
+mery#merry_adj)
@@ -980 +980 @@
-in)
+in#in_prep)
@@ -983 +983 @@
-God)))))
+God#NA)))))
@@ -989 +989 @@
-and)
+and#and_conj1)
@@ -991 +991 @@
-devyse)
+devyse#devise_v)
@@ -994 +994 @@
-somewhat))
+somewhat#somewhat_n))
@@ -997 +997 @@
-wythe)
+wythe#with_prep)
@@ -1000 +1000 @@
-your)
+your#your_pron)
@@ -1002 +1002 @@
-frendys)))
+frendys#friend_n)))
@@ -1007 +1007 @@
-what)
+what#what_pron)
@@ -1009 +1009 @@
-way)
+way#way_n1)
@@ -1018 +1018 @@
-were)
+were#be_v)
@@ -1021 +1021 @@
-best)
+best#best_adj)
@@ -1029 +1029 @@
-to)
+to#to_prep)
@@ -1031 +1031 @@
-take))))
+take#take_v))))
@@ -1034 +1034 @@
-for)
+for#for_prep)
@@ -1037 +1037 @@
-provysyon)
+provysyon#provision_n)
@@ -1041 +1041 @@
-to)
+to#to_prep)
@@ -1043 +1043 @@
-be)
+be#be_v)
@@ -1047 +1047 @@
-made)
+made#make_v1)
@@ -1051 +1051 @@
-for)
+for#for_prep)
@@ -1054 +1054 @@
-corne)
+corne#corn_n1)
@@ -1057 +1057 @@
-for)
+for#for_prep)
@@ -1060 +1060 @@
-our)
+our#our_pron)
@@ -1062 +1062 @@
-houshold)))))
+houshold#household_n)))))
@@ -1065 +1065 @@
-and)
+and#and_conj1)
@@ -1068 +1068 @@
-for)
+for#for_prep)
@@ -1071 +1071 @@
-seede)))))
+seede#seed_n)))))
@@ -1074 +1074 @@
-thys)
+thys#this_pron)
@@ -1076 +1076 @@
-yere)
+yere#year_n)
@@ -1079 +1079 @@
-commyng))))))
+commyng#come_v))))))
@@ -1085 +1085 @@
-yff)
+yff#if_conj)
@@ -1094 +1094 @@
-ye))
+ye#ye_pron))
@@ -1098 +1098 @@
-thyncke)
+thyncke#think_v2)
@@ -1102 +1102 @@
-yt))
+yt#it_pron))
@@ -1105 +1105 @@
-good))
+good#good_adj))
@@ -1108 +1108 @@
-that)
+that#that_conj)
@@ -1112 +1112 @@
-we))
+we#we_pron))
@@ -1114 +1114 @@
-kepe)
+kepe#keep_v)
@@ -1117 +1117 @@
-the)
+the#the_adj)
@@ -1119 +1119 @@
-grounde))
+grounde#ground_n))
@@ -1122 +1122 @@
-stylle))
+stylle#still_adv))
@@ -1125 +1125 @@
-in)
+in#in_prep)
@@ -1130 +1130 @@
-our)
+our#our_pron)
@@ -1134 +1134 @@
-handys))))))))))
+handys#hand_n))))))))))
@@ -1141 +1141 @@
-and)
+and#and_conj1)
@@ -1144 +1144 @@
-whether)
+whether#whether_pron)
@@ -1150 +1150 @@
-ye))
+ye#ye_pron))
@@ -1152 +1152 @@
-thyncke)
+thyncke#think_v2)
@@ -1156 +1156 @@
-yt))
+yt#it_pron))
@@ -1159 +1159 @@
-good))
+good#good_adj))
@@ -1162 +1162 @@
-that)
+that#that_conj)
@@ -1167 +1167 @@
-we))
+we#we_pron))
@@ -1170 +1170 @@
-so))
+so#so_adv))
@@ -1172 +1172 @@
-shall)
+shall#shall_v)
@@ -1176 +1176 @@
-do))
+do#do_v))
@@ -1179 +1179 @@
-or)
+or#or_conj2)
@@ -1182 +1182 @@
-not))))))))))
+not#not_adv))))))))))
@@ -1190 +1190 @@
-yet)
+yet#yet_adv)
@@ -1193 +1193 @@
-I))
+I#i_pron))
@@ -1195 +1195 @@
-thyncke)
+thyncke#think_v2)
@@ -1202 +1202 @@
-it))
+it#it_pron))
@@ -1204 +1204 @@
-were)
+were#be_v)
@@ -1206 +1206 @@
-not)
+not#not_adv)
@@ -1209 +1209 @@
-best))
+best#best_adj))
@@ -1214 +1214 @@
-sodenly))
+sodenly#suddenly_adv))
@@ -1217 +1217 @@
-thus))
+thus#thus_adv))
@@ -1221 +1221 @@
-to)
+to#to_prep)
@@ -1223 +1223 @@
-leve)
+leve#leave_v1)
@@ -1226 +1226 @@
-yt)
+yt#it_pron)
@@ -1229 +1229 @@
-all)))
+all#all_adj)))
@@ -1231 +1231 @@
-vp))
+vp#up_adv1))
@@ -1234 +1234 @@
-and)
+and#and_conj1)
@@ -1237 +1237 @@
-to)
+to#to_prep)
@@ -1239 +1239 @@
-put)
+put#put_v)
@@ -1242 +1242 @@
-away))
+away#away_adv))
@@ -1245 +1245 @@
-our)
+our#our_pron)
@@ -1247 +1247 @@
-folke))
+folke#folk_n))
@@ -1250 +1250 @@
-of)
+of#of_prep)
@@ -1253 +1253 @@
-our)
+our#our_pron)
@@ -1255 +1255 @@
-ferm))))))
+ferm#farm_n2))))))
@@ -1260 +1260 @@
-till)
+till#till_prep)
@@ -1269 +1269 @@
-we))
+we#we_pron))
@@ -1271 +1271 @@
-haue)
+haue#have_v)
@@ -1274 +1274 @@
-somewhat))
+somewhat#somewhat_n))
@@ -1276 +1276 @@
-aduysed)
+aduysed#advise_v)
@@ -1279 +1279 @@
-vs))
+vs#us_pron))
@@ -1282 +1282 @@
-theron)))))))
+theron#thereon_adv)))))))
@@ -1287 +1287 @@
-how_be_it)
+how_be_it#howbeit_adv)
@@ -1294 +1294 @@
-yff)
+yff#if_conj)
@@ -1301 +1301 @@
-we))
+we#we_pron))
@@ -1303 +1303 @@
-haue)
+haue#have_v)
@@ -1306 +1306 @@
-more)
+more#more_adj)
@@ -1311 +1311 @@
-now))
+now#now_adv))
@@ -1315 +1315 @@
-then)
+then#than_conj)
@@ -1326 +1326 @@
-ye))
+ye#ye_pron))
@@ -1328 +1328 @@
-shall)
+shall#shall_v)
@@ -1330 +1330 @@
-nede))))
+nede#need_v2))))
@@ -1333 +1333 @@
-and)
+and#and_conj1)
@@ -1337 +1337 @@
-which))
+which#which_adj))
@@ -1344 +1344 @@
-can)
+can#can_v1)
@@ -1346 +1346 @@
-gett)
+gett#get_v)
@@ -1349 +1349 @@
-them))
+them#them_pron))
@@ -1352 +1352 @@
-other)
+other#other_adj)
@@ -1354 +1354 @@
-maysters)))))))))
+maysters#master_n1)))))))))
@@ -1357 +1357 @@
-ye))
+ye#ye_pron))
@@ -1359 +1359 @@
-may)
+may#may_v1)
@@ -1362 +1362 @@
-then))
+then#then_adv))
@@ -1364 +1364 @@
-dyscharge)
+dyscharge#discharge_v)
@@ -1367 +1367 @@
-vs))
+vs#us_pron))
@@ -1370 +1370 @@
-of)
+of#of_prep)
@@ -1373 +1373 @@
-them)))))))
+them#them_pron)))))))
@@ -1379 +1379 @@
-but)
+but#but_prep)
@@ -1382 +1382 @@
-I))
+I#i_pron))
@@ -1384 +1384 @@
-wolde)
+wolde#will_v1)
@@ -1386 +1386 @@
-not)
+not#not_adv)
@@ -1389 +1389 @@
-that)
+that#that_conj)
@@ -1393 +1393 @@
-eny)
+eny#any_adj)
@@ -1395 +1395 @@
-man))
+man#man_n1))
@@ -1397 +1397 @@
-were)
+were#be_v)
@@ -1400 +1400 @@
-sodenly))
+sodenly#suddenly_adv))
@@ -1402 +1402 @@
-sent)
+sent#send_v1)
@@ -1405 +1405 @@
-away))
+away#away_adv))
@@ -1410 +1410 @@
-he))
+he#he_pron))
@@ -1412 +1412 @@
-wote)
+wote#wit_v1)
@@ -1415 +1415 @@
-nere))
+nere#neer_adv))
@@ -1419 +1419 @@
-whyther)))))))
+whyther#whither_adv)))))))
@@ -1428 +1428 @@
-At)
+At#at_prep)
@@ -1431 +1431 @@
-my)
+my#my_adj)
@@ -1433 +1433 @@
-commyng)
+commyng#coming_n)
@@ -1436 +1436 @@
-hether))))
+hether#hither_adv))))
@@ -1439 +1439 @@
-I))
+I#i_pron))
@@ -1441 +1441 @@
-perceved)
+perceved#perceive_v)
@@ -1444 +1444 @@
-none)
+none#none_pron)
@@ -1446 +1446 @@
-other)
+other#other_adj)
@@ -1449 +1449 @@
-but)
+but#but_prep)
@@ -1452 +1452 @@
-that)
+that#that_conj)
@@ -1456 +1456 @@
-I))
+I#i_pron))
@@ -1458 +1458 @@
-shulde)
+shulde#shall_v)
@@ -1460 +1460 @@
-tary)
+tary#tarry_v)
@@ -1463 +1463 @@
-styll))
+styll#still_adv))
@@ -1466 +1466 @@
-with)
+with#with_prep)
@@ -1470 +1470 @@
-the)
+the#the_adj)
@@ -1472 +1472 @@
-Kyngis))
+Kyngis#king_n))
@@ -1474 +1474 @@
-Grace))))))))
+Grace#grace_n))))))))
@@ -1480 +1480 @@
-but)
+but#but_prep)
@@ -1483 +1483 @@
-now))
+now#now_adv))
@@ -1486 +1486 @@
-I))
+I#i_pron))
@@ -1488 +1488 @@
-shall)
+shall#shall_v)
@@ -1494 +1494 @@
-I))
+I#i_pron))
@@ -1496 +1496 @@
-thynke))
+thynke#think_v2))
@@ -1501 +1501 @@
-by)
+by#by_prep)
@@ -1504 +1504 @@
-cause)
+cause#cause_n)
@@ -1507 +1507 @@
-of)
+of#of_prep)
@@ -1510 +1510 @@
-thys)
+thys#this_pron)
@@ -1512 +1512 @@
-chaunce)))))
+chaunce#chance_n)))))
@@ -1514 +1514 @@
-gete)
+gete#get_v)
@@ -1517 +1517 @@
-leve)
+leve#leave_n1)
@@ -1522 +1522 @@
-this)
+this#this_pron)
@@ -1524 +1524 @@
-next)
+next#next_adj)
@@ -1526 +1526 @@
-weke))
+weke#week_n))
@@ -1530 +1530 @@
-to)
+to#to_prep)
@@ -1532 +1532 @@
-come)
+come#come_v)
@@ -1535 +1535 @@
-home)))
+home#home_n1)))
@@ -1538 +1538 @@
-and)
+and#and_conj1)
@@ -1543 +1543 @@
-se)
+se#see_v)
@@ -1546 +1546 @@
-you)))))
+you#you_pron)))))
@@ -1554 +1554 @@
-an)
+an#and_conj1)
@@ -1557 +1557 @@
-then))
+then#then_adv))
@@ -1559 +1559 @@
-shall)
+shall#shall_v)
@@ -1562 +1562 @@
-we))
+we#we_pron))
@@ -1565 +1565 @@
-ferther))
+ferther#farther_adv))
@@ -1567 +1567 @@
-devyse)
+devyse#devise_v)
@@ -1570 +1570 @@
-together))
+together#together_adv))
@@ -1573 +1573 @@
-vpon)
+vpon#upon_prep)
@@ -1576 +1576 @@
-all)
+all#all_adj)
@@ -1578 +1578 @@
-thyngis)))
+thyngis#thing_n1)))
@@ -1582 +1582 @@
-what)
+what#what_pron)
@@ -1584 +1584 @@
-order))
+order#order_n))
@@ -1591 +1591 @@
-shall)
+shall#shall_v)
@@ -1593 +1593 @@
-be)
+be#be_v)
@@ -1596 +1596 @@
-best)
+best#best_adj)
@@ -1604 +1604 @@
-to)
+to#to_prep)
@@ -1606 +1606 @@
-take))))))
+take#take_v))))))
@@ -1614 +1614 @@
-And)
+And#and_conj1)
@@ -1617 +1617 @@
-thus))
+thus#thus_adv))
@@ -1620 +1620 @@
-as)
+as#as_adv)
@@ -1622 +1622 @@
-hartely)
+hartely#heartily_adv)
@@ -1626 +1626 @@
-fare)
+fare#fare_v1)
@@ -1629 +1629 @@
-you))
+you#you_pron))
@@ -1632 +1632 @@
-well))
+well#well_adv))
@@ -1635 +1635 @@
-wythe)
+wythe#with_prep)
@@ -1638 +1638 @@
-all)
+all#all_adj)
@@ -1640 +1640 @@
-our)
+our#our_pron)
@@ -1642 +1642 @@
-chyldren)))
+chyldren#child_n)))
@@ -1645 +1645 @@
-as)
+as#as_adv)
@@ -1656 +1656 @@
-ye))
+ye#ye_pron))
@@ -1658 +1658 @@
-can)
+can#can_v1)
@@ -1660 +1660 @@
-wyshe))))
+wyshe#wish_v))))
@@ -1665 +1665 @@
-at)
+at#at_prep)
@@ -1668 +1668 @@
-Woodestokke)))
+Woodestokke#NA)))
@@ -1671 +1671 @@
-the)
+the#the_adj)
@@ -1673 +1673 @@
-iii=de=)
+iii=de=#third_adj)
@@ -1675 +1675 @@
-daye)
+daye#day_n)
@@ -1678 +1678 @@
-of)
+of#of_prep)
@@ -1681 +1681 @@
-September))))
+September#NA))))
@@ -1684 +1684 @@
-by)
+by#by_prep)
@@ -1687 +1687 @@
-the)
+the#the_adj)
@@ -1689 +1689 @@
-hand)
+hand#hand_n)
@@ -1692 +1692 @@
-of)
+of#of_prep)
@@ -1695 +1695 @@
-Your)
+Your#your_pron)
@@ -1697 +1697 @@
-lovyng)
+lovyng#loving_adj)
@@ -1699 +1699 @@
-husbond)
+husbond#husband_n)
@@ -1704 +1704 @@
-Thomas)
+Thomas#NA)
@@ -1706 +1706 @@
-More)
+More#NA)
@@ -1709 +1709 @@
-Kg)))))))
+Kg#knight_n)))))))
