--- nferrar-1629-e2-h.psd.vert	2025-09-10 13:50:35
+++ nferrar-1629-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-My)
+My#my_adj)
@@ -17 +17 @@
-most)
+most#most_adj)
@@ -19 +19 @@
-deare))
+deare#dear_adj1))
@@ -21 +21 @@
-mother))
+mother#mother_n1))
@@ -24 +24 @@
-Your)
+Your#your_pron)
@@ -26 +26 @@
-good)
+good#good_adj)
@@ -28 +28 @@
-arrivall)
+arrivall#arrival_n)
@@ -31 +31 @@
-at)
+at#at_prep)
@@ -34 +34 @@
-Gidding)))
+Gidding#NA)))
@@ -39 +39 @@
-I))
+I#i_pron))
@@ -41 +41 @@
-doubte)
+doubte#doubt_v)
@@ -43 +43 @@
-not)
+not#not_adv)
@@ -49 +49 @@
-which))
+which#which_adj))
@@ -55 +55 @@
-I))
+I#i_pron))
@@ -58 +58 @@
-humbly))
+humbly#humbly_adv))
@@ -60 +60 @@
-beseech)
+beseech#beseech_v)
@@ -63 +63 @@
-Almighty)
+Almighty#almighty_adj)
@@ -65 +65 @@
-God))
+God#NA))
@@ -73 +73 @@
-may)
+may#may_v1)
@@ -75 +75 @@
-bee)
+bee#be_v)
@@ -77 +77 @@
-continued)
+continued#continue_v)
@@ -80 +80 @@
-w=th=)
+w=th=#with_prep)
@@ -83 +83 @@
-all)
+all#all_adj)
@@ -85 +85 @@
-manner)
+manner#manner_n)
@@ -88 +88 @@
-of)
+of#of_prep)
@@ -91 +91 @@
-prosperity))))))))))
+prosperity#prosperity_n))))))))))
@@ -98 +98 @@
-this)
+this#this_pron)
@@ -100 +100 @@
-prayer))
+prayer#prayer_n1))
@@ -103 +103 @@
-all)
+all#all_adj)
@@ -105 +105 @@
-our)
+our#our_pron)
@@ -107 +107 @@
-freindes)
+freindes#friend_n)
@@ -110 +110 @@
-heere)))
+heere#here_adv)))
@@ -112 +112 @@
-meke)
+meke#meek_v)
@@ -115 +115 @@
-with)
+with#with_prep)
@@ -118 +118 @@
-mee))))
+mee#me_pron1))))
@@ -124 +124 @@
-and)
+and#and_conj1)
@@ -128 +128 @@
-remember)
+remember#remember_v)
@@ -131 +131 @@
-there)
+there#their_adj)
@@ -133 +133 @@
-best)
+best#best_adj)
@@ -135 +135 @@
-loues))
+loues#love_n1))
@@ -138 +138 @@
-respectiuely))
+respectiuely#respectively_adv))
@@ -141 +141 @@
-unto)
+unto#unto_prep)
@@ -145 +145 @@
-your)
+your#your_pron)
@@ -147 +147 @@
-selfe))
+selfe#self_pron))
@@ -150 +150 @@
-and)
+and#and_conj1)
@@ -153 +153 @@
-all)
+all#all_adj)
@@ -155 +155 @@
-youre)))))
+youre#your_pron)))))
@@ -163 +163 @@
-and)
+and#and_conj1)
@@ -166 +166 @@
-this))
+this#this_pron))
@@ -168 +168 @@
-is)
+is#be_v)
@@ -171 +171 @@
-all)
+all#all_adj)
@@ -176 +176 @@
-y=t=)
+y=t=#that_conj)
@@ -180 +180 @@
-I))
+I#i_pron))
@@ -182 +182 @@
-haue)
+haue#have_v)
@@ -185 +185 @@
-occasion))
+occasion#occasion_n1))
@@ -190 +190 @@
-to)
+to#to_prep)
@@ -192 +192 @@
-write)
+write#write_v)
@@ -195 +195 @@
-unto)
+unto#unto_prep)
@@ -198 +198 @@
-you))))))
+you#you_pron))))))
@@ -201 +201 @@
-sauing)
+sauing#saving_prep)
@@ -205 +205 @@
-farther))
+farther#farther_adv))
@@ -207 +207 @@
-to)
+to#to_prep)
@@ -209 +209 @@
-lett)
+lett#let_v1)
@@ -213 +213 @@
-you))
+you#you_pron))
@@ -215 +215 @@
-know)
+know#know_v)
@@ -218 +218 @@
-y=t=)
+y=t=#that_conj)
@@ -222 +222 @@
-none)
+none#none_pron)
@@ -225 +225 @@
-of)
+of#of_prep)
@@ -228 +228 @@
-the)
+the#the_adj)
@@ -230 +230 @@
-Oundell)
+Oundell#NA)
@@ -232 +232 @@
-Waggoners))))
+Waggoners#NA))))
@@ -234 +234 @@
-are)
+are#be_v)
@@ -237 +237 @@
-yet))
+yet#yet_adv))
@@ -239 +239 @@
-com)
+com#come_v)
@@ -242 +242 @@
-to)
+to#to_prep)
@@ -245 +245 @@
-towne))))))))))
+towne#town_n))))))))))
@@ -251 +251 @@
-nor)
+nor#nor_conj1)
@@ -255 +255 @@
-is)
+is#be_v)
@@ -258 +258 @@
-it))
+it#it_pron))
@@ -261 +261 @@
-certaine))
+certaine#certain_adj))
@@ -265 +265 @@
-when))
+when#when_adv))
@@ -273 +273 @@
-they))
+they#they_pron))
@@ -275 +275 @@
-will)
+will#will_v1)
@@ -277 +277 @@
-be)
+be#be_v)
@@ -282 +282 @@
-at)
+at#at_prep)
@@ -285 +285 @@
-Beuersly)
+Beuersly#NA)
@@ -287 +287 @@
-fayre))))))
+fayre#fair_n1))))))
@@ -294 +294 @@
-yett))
+yett#yet_adv))
@@ -297 +297 @@
-there))
+there#there_adv))
@@ -299 +299 @@
-is)
+is#be_v)
@@ -302 +302 @@
-some)
+some#some_pron)
@@ -304 +304 @@
-Expectation)
+Expectation#expectation_n)
@@ -307 +307 @@
-of)
+of#of_prep)
@@ -310 +310 @@
-thim))))
+thim#them_pron))))
@@ -313 +313 @@
-this)
+this#this_pron)
@@ -315 +315 @@
-present)
+present#present_adj)
@@ -317 +317 @@
-day))
+day#day_n))
@@ -325 +325 @@
-And)
+And#and_conj1)
@@ -328 +328 @@
-soe))
+soe#so_adv))
@@ -331 +331 @@
-agayne))
+agayne#again_adv))
@@ -334 +334 @@
-w=th=)
+w=th=#with_prep)
@@ -337 +337 @@
-my)
+my#my_adj)
@@ -340 +340 @@
-most)
+most#most_adj)
@@ -342 +342 @@
-humblest))
+humblest#humble_adj))
@@ -344 +344 @@
-prayers)
+prayers#prayer_n1)
@@ -347 +347 @@
-to)
+to#to_prep)
@@ -350 +350 @@
-Almighty)
+Almighty#almighty_adj)
@@ -352 +352 @@
-god)))
+god#NA)))
@@ -355 +355 @@
-for)
+for#for_prep)
@@ -358 +358 @@
-y=r=)
+y=r=#your_pron)
@@ -360 +360 @@
-happines)))))
+happines#happiness_n)))))
@@ -363 +363 @@
-I))
+I#i_pron))
@@ -365 +365 @@
-rest)
+rest#rest_v2)
@@ -368 +368 @@
-Your)
+Your#your_pron)
@@ -371 +371 @@
-most)
+most#most_adj)
@@ -373 +373 @@
-bounden)
+bounden#bounden_adj)
@@ -376 +376 @@
-and)
+and#and_conj1)
@@ -379 +379 @@
-obedient))))
+obedient#obedient_adj))))
@@ -381 +381 @@
-son)
+son#son_n1)
@@ -384 +384 @@
-Nicholas)
+Nicholas#NA)
@@ -386 +386 @@
-Ferrar))))
+Ferrar#NA))))
@@ -392 +392 @@
-Iune)
+Iune#NA)
@@ -394 +394 @@
-viij)
+viij#eight_adj)
@@ -398 +398 @@
-1626))
+1626#NA))
