--- peyton-1632-e2-h.psd.vert	2025-09-10 13:50:35
+++ peyton-1632-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Doughter)
+Doughter#daughter_n)
@@ -16 +16 @@
-Oxinden))
+Oxinden#NA))
@@ -22 +22 @@
-Which)
+Which#which_adj)
@@ -24 +24 @@
-title))
+title#title_n))
@@ -32 +32 @@
-I))
+I#i_pron))
@@ -34 +34 @@
-must)
+must#must_v1)
@@ -37 +37 @@
-now))
+now#now_adv))
@@ -39 +39 @@
-give)
+give#give_v)
@@ -42 +42 @@
-you))))
+you#you_pron))))
@@ -47 +47 @@
-your)
+your#your_pron)
@@ -49 +49 @@
-brother))
+brother#brother_n))
@@ -51 +51 @@
-hath)
+hath#have_v)
@@ -54 +54 @@
-so)
+so#so_adv)
@@ -56 +56 @@
-weall)
+weall#well_adv)
@@ -60 +60 @@
-satisfied)
+satisfied#satisfy_v)
@@ -63 +63 @@
-me))
+me#me_pron1))
@@ -66 +66 @@
-in)
+in#in_prep)
@@ -69 +69 @@
-your)
+your#your_pron)
@@ -71 +71 @@
-match)))
+match#match_n1)))
@@ -74 +74 @@
-that)
+that#that_conj)
@@ -78 +78 @@
-I))
+I#i_pron))
@@ -80 +80 @@
-wish)
+wish#wish_v)
@@ -83 +83 @@
-you))
+you#you_pron))
@@ -86 +86 @@
-much)
+much#much_adj)
@@ -88 +88 @@
-joy)
+joy#joy_n)
@@ -91 +91 @@
-and)
+and#and_conj1)
@@ -94 +94 @@
-hapines))))))
+hapines#happiness_n))))))
@@ -102 +102 @@
-and)
+and#and_conj1)
@@ -105 +105 @@
-withall))
+withall#withal_adv))
@@ -107 +107 @@
-be)
+be#be_v)
@@ -110 +110 @@
-carful)
+carful#careful_adj)
@@ -113 +113 @@
-that)
+that#that_conj)
@@ -121 +121 @@
-whatsoever))
+whatsoever#whatsoever_pron))
@@ -129 +129 @@
-you))
+you#you_pron))
@@ -131 +131 @@
-doe))))
+doe#do_v))))
@@ -135 +135 @@
-to)
+to#to_prep)
@@ -138 +138 @@
-love)
+love#love_v1)
@@ -140 +140 @@
-honer)
+honer#honour_v)
@@ -142 +142 @@
-and)
+and#and_conj1)
@@ -144 +144 @@
-obey))
+obey#obey_v))
@@ -147 +147 @@
-your)
+your#your_pron)
@@ -149 +149 @@
-housband))
+housband#husband_n))
@@ -152 +152 @@
-in)
+in#in_prep)
@@ -155 +155 @@
-all)
+all#all_adj)
@@ -157 +157 @@
-things)
+things#thing_n1)
@@ -162 +162 @@
-that)
+that#that_conj)
@@ -167 +167 @@
-is)
+is#be_v)
@@ -169 +169 @@
-fitting)
+fitting#fit_v1)
@@ -172 +172 @@
-for)
+for#for_prep)
@@ -175 +175 @@
-a)
+a#a_adj2)
@@ -177 +177 @@
-resonable)
+resonable#reasonable_adj)
@@ -179 +179 @@
-creture))))))))))
+creture#creature_n))))))))))
@@ -188 +188 @@
-I))
+I#i_pron))
@@ -192 +192 @@
-will)
+will#will_v1)
@@ -194 +194 @@
-desir)
+desir#desire_v)
@@ -197 +197 @@
-nothing)
+nothing#nothing_pron)
@@ -202 +202 @@
-that)
+that#that_conj)
@@ -207 +207 @@
-is)
+is#be_v)
@@ -210 +210 @@
-unresonable)))))
+unresonable#unreasonable_adj)))))
@@ -219 +219 @@
-you))
+you#you_pron))
@@ -221 +221 @@
-know)
+know#know_v)
@@ -225 +225 @@
-what))
+what#what_pron))
@@ -233 +233 @@
-I))
+I#i_pron))
@@ -235 +235 @@
-have)
+have#have_v)
@@ -237 +237 @@
-sufferd))))
+sufferd#suffer_v))))
@@ -244 +244 @@
-yet))
+yet#yet_adv))
@@ -247 +247 @@
-God))
+God#NA))
@@ -249 +249 @@
-hath)
+hath#have_v)
@@ -251 +251 @@
-delivered)
+delivered#deliver_v1)
@@ -256 +256 @@
-mee))
+mee#me_pron1))
@@ -259 +259 @@
-out)
+out#out_adv)
@@ -261 +261 @@
-of)
+of#of_prep)
@@ -264 +264 @@
-it)))
+it#it_pron)))
@@ -269 +269 @@
-though)
+though#though_adv)
@@ -272 +272 @@
-with)
+with#with_prep)
@@ -275 +275 @@
-infinit)
+infinit#infinite_adj)
@@ -277 +277 @@
-afliction)
+afliction#affliction_n)
@@ -280 +280 @@
-for)
+for#for_prep)
@@ -283 +283 @@
-the)
+the#the_adj)
@@ -285 +285 @@
-time))))))
+time#time_n))))))
@@ -294 +294 @@
-I))
+I#i_pron))
@@ -296 +296 @@
-have)
+have#have_v)
@@ -298 +298 @@
-had)
+had#have_v)
@@ -302 +302 @@
-so)
+so#so_adv)
@@ -304 +304 @@
-much))
+much#much_adj))
@@ -308 +308 @@
-spech)
+spech#speech_n1)
@@ -311 +311 @@
-with)
+with#with_prep)
@@ -314 +314 @@
-your)
+your#your_pron)
@@ -316 +316 @@
-brother)))
+brother#brother_n)))
@@ -319 +319 @@
-conserning)
+conserning#concerning_prep)
@@ -324 +324 @@
-your)
+your#your_pron)
@@ -326 +326 @@
-father's))
+father's#father_n))
@@ -328 +328 @@
-wille))
+wille#will_n1))
@@ -331 +331 @@
-and)
+and#and_conj1)
@@ -334 +334 @@
-your)
+your#your_pron)
@@ -336 +336 @@
-portion)
+portion#portion_n)
@@ -342 +342 @@
-which))
+which#which_adj))
@@ -350 +350 @@
-he))
+he#he_pron))
@@ -352 +352 @@
-would)
+would#will_v1)
@@ -354 +354 @@
-not)
+not#not_adv)
@@ -356 +356 @@
-beleeve)
+beleeve#believe_v)
@@ -359 +359 @@
-till)
+till#till_prep)
@@ -366 +366 @@
-I))
+I#i_pron))
@@ -368 +368 @@
-showed)
+showed#show_v)
@@ -371 +371 @@
-him))
+him#him_pron))
@@ -374 +374 @@
-the)
+the#the_adj)
@@ -376 +376 @@
-will))))))))))))
+will#will_n1))))))))))))
@@ -385 +385 @@
-he))
+he#he_pron))
@@ -387 +387 @@
-is)
+is#be_v)
@@ -390 +390 @@
-now))
+now#now_adv))
@@ -392 +392 @@
-confirmed)
+confirmed#confirm_v)
@@ -395 +395 @@
-in)
+in#in_prep)
@@ -398 +398 @@
-it))))
+it#it_pron))))
@@ -404 +404 @@
-and)
+and#and_conj1)
@@ -408 +408 @@
-says)
+says#say_v1)
@@ -415 +415 @@
-you))
+you#you_pron))
@@ -417 +417 @@
-shall)
+shall#shall_v)
@@ -419 +419 @@
-have)
+have#have_v)
@@ -422 +422 @@
-your)
+your#your_pron)
@@ -424 +424 @@
-dewe))
+dewe#due_n))
@@ -427 +427 @@
-as)
+as#as_adv)
@@ -429 +429 @@
-soune)
+soune#soon_adv)
@@ -432 +432 @@
-as)
+as#as_adv)
@@ -443 +443 @@
-he))
+he#he_pron))
@@ -445 +445 @@
-can)
+can#can_v1)
@@ -456 +456 @@
-be)
+be#be_v)
@@ -459 +459 @@
-sur)
+sur#sure_adj)
@@ -462 +462 @@
-of)
+of#of_prep)
@@ -465 +465 @@
-this)))))
+this#this_pron)))))
@@ -468 +468 @@
-you))
+you#you_pron))
@@ -470 +470 @@
-shall)
+shall#shall_v)
@@ -472 +472 @@
-have)
+have#have_v)
@@ -475 +475 @@
-it))
+it#it_pron))
@@ -480 +480 @@
-though)
+though#though_adv)
@@ -487 +487 @@
-you))
+you#you_pron))
@@ -489 +489 @@
-stay)
+stay#stay_v1)
@@ -492 +492 @@
-som)
+som#some_pron)
@@ -494 +494 @@
-tim))
+tim#time_n))
@@ -497 +497 @@
-for)
+for#for_prep)
@@ -500 +500 @@
-it))))))
+it#it_pron))))))
@@ -510 +510 @@
-in)
+in#in_prep)
@@ -513 +513 @@
-the)
+the#the_adj)
@@ -515 +515 @@
-meantime)))
+meantime#meantime_n)))
@@ -517 +517 @@
-let)
+let#let_v1)
@@ -521 +521 @@
-no)
+no#no_adj)
@@ -523 +523 @@
-respect))
+respect#respect_n))
@@ -525 +525 @@
-be)
+be#be_v)
@@ -527 +527 @@
-wanting)
+wanting#want_v)
@@ -530 +530 @@
-to)
+to#to_prep)
@@ -534 +534 @@
-your)
+your#your_pron)
@@ -536 +536 @@
-housband))
+housband#husband_n))
@@ -539 +539 @@
-and)
+and#and_conj1)
@@ -542 +542 @@
-his)
+his#his_adj)
@@ -544 +544 @@
-mother)))
+mother#mother_n1)))
@@ -549 +549 @@
-with)
+with#with_prep)
@@ -552 +552 @@
-the)
+the#the_adj)
@@ -554 +554 @@
-rest)
+rest#rest_n2)
@@ -557 +557 @@
-of)
+of#of_prep)
@@ -560 +560 @@
-his)
+his#his_adj)
@@ -562 +562 @@
-frends)))))))))
+frends#friend_n)))))))))
@@ -570 +570 @@
-in)
+in#in_prep)
@@ -573 +573 @@
-this)))
+this#this_pron)))
@@ -576 +576 @@
-you))
+you#you_pron))
@@ -578 +578 @@
-shall)
+shall#shall_v)
@@ -580 +580 @@
-gain)
+gain#gain_v2)
@@ -583 +583 @@
-yourself))
+yourself#yourself_pron))
@@ -586 +586 @@
-a)
+a#a_adj2)
@@ -588 +588 @@
-good)
+good#good_adj)
@@ -590 +590 @@
-reput)))
+reput#repute_n)))
@@ -593 +593 @@
-and)
+and#and_conj1)
@@ -596 +596 @@
-shew)
+shew#show_v)
@@ -600 +600 @@
-yourself))
+yourself#yourself_pron))
@@ -603 +603 @@
-a)
+a#a_adj2)
@@ -605 +605 @@
-vertuous)
+vertuous#virtuous_adj)
@@ -607 +607 @@
-wife)
+wife#wife_n)
@@ -611 +611 @@
-whoes)
+whoes#whose_pron)
@@ -613 +613 @@
-pris))
+pris#price_n))
@@ -620 +620 @@
-is)
+is#be_v)
@@ -622 +622 @@
-not)
+not#not_adv)
@@ -625 +625 @@
-to)
+to#to_prep)
@@ -627 +627 @@
-be)
+be#be_v)
@@ -629 +629 @@
-valued))))))))))
+valued#value_v))))))))))
@@ -638 +638 @@
-as)
+as#as_adv)
@@ -641 +641 @@
-for)
+for#for_prep)
@@ -644 +644 @@
-the)
+the#the_adj)
@@ -646 +646 @@
-bisines)
+bisines#business_n)
@@ -655 +655 @@
-you))
+you#you_pron))
@@ -657 +657 @@
-writ)
+writ#write_v)
@@ -660 +660 @@
-to)
+to#to_prep)
@@ -663 +663 @@
-me)))
+me#me_pron1)))
@@ -666 +666 @@
-about)
+about#about_adv)
@@ -673 +673 @@
-I))
+I#i_pron))
@@ -675 +675 @@
-am)
+am#be_v)
@@ -678 +678 @@
-ashamed)
+ashamed#ashamed_adj)
@@ -685 +685 @@
-I))
+I#i_pron))
@@ -687 +687 @@
-can@)
+can@#can_v1)
@@ -689 +689 @@
-@not)
+@not#not_adv)
@@ -691 +691 @@
-doe)
+doe#do_v)
@@ -694 +694 @@
-it))
+it#it_pron))
@@ -697 +697 @@
-for)
+for#for_prep)
@@ -700 +700 @@
-you))))))
+you#you_pron))))))
@@ -709 +709 @@
-my)
+my#my_adj)
@@ -711 +711 @@
-housband))
+housband#husband_n))
@@ -713 +713 @@
-siems)
+siems#seem_v2)
@@ -716 +716 @@
-to)
+to#to_prep)
@@ -718 +718 @@
-give)
+give#give_v)
@@ -721 +721 @@
-me))
+me#me_pron1))
@@ -724 +724 @@
-pour))
+pour#power_n1))
@@ -727 +727 @@
-though)
+though#though_adv)
@@ -734 +734 @@
-I))
+I#i_pron))
@@ -736 +736 @@
-have)
+have#have_v)
@@ -739 +739 @@
-no)
+no#no_adj)
@@ -741 +741 @@
-pour))))))
+pour#power_n1))))))
@@ -750 +750 @@
-he))
+he#he_pron))
@@ -752 +752 @@
-hath)
+hath#have_v)
@@ -754 +754 @@
-been)
+been#be_v)
@@ -758 +758 @@
-so)
+so#so_adv)
@@ -760 +760 @@
-ill))
+ill#ill_adj))
@@ -762 +762 @@
-a)
+a#a_adj2)
@@ -764 +764 @@
-housband)
+housband#husband_n)
@@ -769 +769 @@
-of)
+of#of_prep)
@@ -772 +772 @@
-lat)))
+lat#late_adv)))
@@ -775 +775 @@
-that)
+that#that_conj)
@@ -779 +779 @@
-I))
+I#i_pron))
@@ -782 +782 @@
-never))
+never#never_adv))
@@ -784 +784 @@
-was)
+was#be_v)
@@ -787 +787 @@
-so)
+so#so_adv)
@@ -791 +791 @@
-put)
+put#put_v)
@@ -794 +794 @@
-to)
+to#to_prep)
@@ -797 +797 @@
-it)))
+it#it_pron)))
@@ -800 +800 @@
-to)
+to#to_prep)
@@ -802 +802 @@
-bring)
+bring#bring_v)
@@ -805 +805 @@
-the)
+the#the_adj)
@@ -807 +807 @@
-wourld))
+wourld#world_n))
@@ -809 +809 @@
-about))
+about#about_adv))
@@ -812 +812 @@
-as)
+as#as_adv)
@@ -823 +823 @@
-now))
+now#now_adv))
@@ -826 +826 @@
-I))
+I#i_pron))
@@ -828 +828 @@
-am)
+am#be_v)
@@ -839 +839 @@
-besids))
+besids#besides_adv))
@@ -842 +842 @@
-he))
+he#he_pron))
@@ -844 +844 @@
-is)
+is#be_v)
@@ -847 +847 @@
-to)
+to#to_prep)
@@ -849 +849 @@
-pay)
+pay#pay_v1)
@@ -852 +852 @@
-a)
+a#a_adj2)
@@ -854 +854 @@
-gret)
+gret#great_adj)
@@ -856 +856 @@
-deale)
+deale#deal_n1)
@@ -859 +859 @@
-of)
+of#of_prep)
@@ -862 +862 @@
-mony))))
+mony#many_adj))))
@@ -865 +865 @@
-this)
+this#this_pron)
@@ -867 +867 @@
-next)
+next#next_adj)
@@ -869 +869 @@
-terme)))
+terme#term_n)))
@@ -878 +878 @@
-wher)
+wher#whether_pron)
@@ -884 +884 @@
-he))
+he#he_pron))
@@ -886 +886 @@
-will)
+will#will_v1)
@@ -888 +888 @@
-have)
+have#have_v)
@@ -891 +891 @@
-it))))
+it#it_pron))))
@@ -894 +894 @@
-I))
+I#i_pron))
@@ -896 +896 @@
-know)
+know#know_v)
@@ -898 +898 @@
-not)
+not#not_adv)
@@ -907 +907 @@
-only)
+only#only_adj)
@@ -909 +909 @@
-this)
+this#this_pron)
@@ -914 +914 @@
-I))
+I#i_pron))
@@ -916 +916 @@
-am)
+am#be_v)
@@ -919 +919 @@
-sur)
+sur#sure_adj)
@@ -928 +928 @@
-he))
+he#he_pron))
@@ -930 +930 @@
-will)
+will#will_v1)
@@ -932 +932 @@
-suffer)))
+suffer#suffer_v)))
@@ -941 +941 @@
-his)
+his#his_adj)
@@ -943 +943 @@
-father))
+father#father_n))
@@ -945 +945 @@
-delt)
+delt#deal_v)
@@ -948 +948 @@
-most)
+most#most_adj)
@@ -950 +950 @@
-unworthylie))
+unworthylie#unworthily_adv))
@@ -953 +953 @@
-with)
+with#with_prep)
@@ -956 +956 @@
-us)))
+us#us_pron)))
@@ -962 +962 @@
-which))
+which#which_adj))
@@ -969 +969 @@
-inforces)
+inforces#enforce_v)
@@ -974 +974 @@
-me))
+me#me_pron1))
@@ -977 +977 @@
-to)
+to#to_prep)
@@ -979 +979 @@
-doe)
+doe#do_v)
@@ -984 +984 @@
-what))
+what#what_pron))
@@ -992 +992 @@
-I))
+I#i_pron))
@@ -994 +994 @@
-would)
+would#will_v1)
@@ -996 +996 @@
-not)
+not#not_adv)
@@ -1007 +1007 @@
-I))
+I#i_pron))
@@ -1009 +1009 @@
-think)
+think#think_v2)
@@ -1012 +1012 @@
-to)
+to#to_prep)
@@ -1014 +1014 @@
-be)
+be#be_v)
@@ -1017 +1017 @@
-at)
+at#at_prep)
@@ -1020 +1020 @@
-London)
+London#NA)
@@ -1025 +1025 @@
-this)
+this#this_pron)
@@ -1027 +1027 @@
-next)
+next#next_adj)
@@ -1029 +1029 @@
-terme))
+terme#term_n))
@@ -1035 +1035 @@
-wher))
+wher#where_adv))
@@ -1041 +1041 @@
-I))
+I#i_pron))
@@ -1043 +1043 @@
-shal@)
+shal@#shall_v)
@@ -1045 +1045 @@
-@be)
+@be#be_v)
@@ -1048 +1048 @@
-glad)
+glad#glad_adj)
@@ -1053 +1053 @@
-to)
+to#to_prep)
@@ -1055 +1055 @@
-meete)
+meete#meet_v)
@@ -1058 +1058 @@
-you)))))))
+you#you_pron)))))))
@@ -1067 +1067 @@
-if)
+if#if_conj)
@@ -1070 +1070 @@
-not)))
+not#not_adv)))
@@ -1073 +1073 @@
-I))
+I#i_pron))
@@ -1075 +1075 @@
-desier)
+desier#desire_v)
@@ -1078 +1078 @@
-to)
+to#to_prep)
@@ -1080 +1080 @@
-see)
+see#see_v)
@@ -1083 +1083 @@
-you))
+you#you_pron))
@@ -1086 +1086 @@
-hier)))
+hier#here_adv)))
@@ -1095 +1095 @@
-Your)
+Your#your_pron)
@@ -1097 +1097 @@
-brother)
+brother#brother_n)
@@ -1100 +1100 @@
-houmfery)))
+houmfery#NA)))
@@ -1102 +1102 @@
-remembers)
+remembers#remember_v)
@@ -1105 +1105 @@
-his)
+his#his_adj)
@@ -1107 +1107 @@
-service)
+service#service_n1)
@@ -1110 +1110 @@
-to)
+to#to_prep)
@@ -1113 +1113 @@
-you))))
+you#you_pron))))
@@ -1122 +1122 @@
-So))
+So#so_adv))
@@ -1125 +1125 @@
-with)
+with#with_prep)
@@ -1128 +1128 @@
-my)
+my#my_adj)
@@ -1130 +1130 @@
-prayers)
+prayers#prayer_n1)
@@ -1133 +1133 @@
-to)
+to#to_prep)
@@ -1136 +1136 @@
-God)))
+God#NA)))
@@ -1139 +1139 @@
-to)
+to#to_prep)
@@ -1141 +1141 @@
-bless)
+bless#bless_v1)
@@ -1144 +1144 @@
-you)))))
+you#you_pron)))))
@@ -1149 +1149 @@
-I))
+I#i_pron))
@@ -1151 +1151 @@
-rest)
+rest#rest_v2)
@@ -1154 +1154 @@
-as)
+as#as_adv)
@@ -1157 +1157 @@
-ever)))
+ever#ever_adv)))
@@ -1160 +1160 @@
-Your)
+Your#your_pron)
@@ -1162 +1162 @@
-asured)
+asured#assured_adj)
@@ -1164 +1164 @@
-loving)
+loving#loving_adj)
@@ -1166 +1166 @@
-mother)
+mother#mother_n1)
@@ -1169 +1169 @@
-Mary)
+Mary#NA)
@@ -1171 +1171 @@
-Peyton))))
+Peyton#NA))))
@@ -1179 +1179 @@
-February)
+February#NA)
@@ -1181 +1181 @@
-the)
+the#the_adj)
@@ -1183 +1183 @@
-19th)
+19th#NA)
@@ -1185 +1185 @@
-1632)
+1632#NA)
