--- phenry-1685-e3-h.psd.vert	2025-09-10 13:50:35
+++ phenry-1685-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-July)
+July#NA)
@@ -17 +17 @@
-8)
+8#eight_adj)
@@ -21 +21 @@
-1685)
+1685#NA)
@@ -30 +30 @@
-Dear)
+Dear#dear_adj1)
@@ -32 +32 @@
-heart))
+heart#heart_n))
@@ -37 +37 @@
-I))
+I#i_pron))
@@ -39 +39 @@
-continue)
+continue#continue_v)
@@ -42 +42 @@
-very)
+very#very_adj)
@@ -44 +44 @@
-well))
+well#well_adj))
@@ -49 +49 @@
-at)
+at#at_prep)
@@ -52 +52 @@
-present)))
+present#present_adj)))
@@ -58 +58 @@
-thankes))
+thankes#thank_n))
@@ -60 +60 @@
-bee)
+bee#be_v)
@@ -63 +63 @@
-to)
+to#to_prep)
@@ -66 +66 @@
-God))))
+God#NA))))
@@ -74 +74 @@
-and)
+and#and_conj1)
@@ -78 +78 @@
-feel)
+feel#feel_v)
@@ -81 +81 @@
-nothing)
+nothing#nothing_pron)
@@ -86 +86 @@
-yet))
+yet#yet_adv))
@@ -89 +89 @@
-of)
+of#of_prep)
@@ -92 +92 @@
-the)
+the#the_adj)
@@ -94 +94 @@
-Inconveniences)
+Inconveniences#inconvenience_n)
@@ -97 +97 @@
-of)
+of#of_prep)
@@ -100 +100 @@
-a)
+a#a_adj2)
@@ -102 +102 @@
-prison)))))
+prison#prison_n)))))
@@ -111 +111 @@
-we))
+we#we_pron))
@@ -113 +113 @@
-are)
+are#be_v)
@@ -116 +116 @@
-better)
+better#better_adj)
@@ -120 +120 @@
-accommodated)
+accommodated#accommodate_v)
@@ -125 +125 @@
-as)
+as#as_adv)
@@ -136 +136 @@
-I))
+I#i_pron))
@@ -138 +138 @@
-acquainted)
+acquainted#acquaint_v)
@@ -141 +141 @@
-you))
+you#you_pron))
@@ -144 +144 @@
-in)
+in#in_prep)
@@ -147 +147 @@
-my)
+my#my_adj)
@@ -149 +149 @@
-last))))))
+last#last_adv))))))
@@ -154 +154 @@
-then)
+then#than_conj)
@@ -165 +165 @@
-wee))
+wee#we_pron))
@@ -167 +167 @@
-could)
+could#can_v1)
@@ -169 +169 @@
-have)
+have#have_v)
@@ -171 +171 @@
-expected))))
+expected#expect_v))))
@@ -176 +176 @@
-though)
+though#though_adv)
@@ -183 +183 @@
-wee))
+wee#we_pron))
@@ -185 +185 @@
-must)
+must#must_v1)
@@ -187 +187 @@
-pay)
+pay#pay_v1)
@@ -190 +190 @@
-for)
+for#for_prep)
@@ -193 +193 @@
-it))))))
+it#it_pron))))))
@@ -202 +202 @@
-Just)
+Just#just_adv)
@@ -204 +204 @@
-now))
+now#now_adv))
@@ -207 +207 @@
-six)
+six#six_adj)
@@ -209 +209 @@
-Min=rs=)
+Min=rs=#minister_n)
@@ -212 +212 @@
-N.)
+N.#NA)
@@ -214 +214 @@
-C.))
+C.#NA))
@@ -218 +218 @@
-are)
+are#be_v)
@@ -220 +220 @@
-brought)
+brought#bring_v)
@@ -222 +222 @@
-in)
+in#in_adv)
@@ -225 +225 @@
-hither))
+hither#hither_adv))
@@ -228 +228 @@
-from)
+from#from_prep)
@@ -231 +231 @@
-lancashire)))
+lancashire#NA)))
@@ -237 +237 @@
-more)
+more#more_adj)
@@ -240 +240 @@
-than)
+than#than_conj)
@@ -253 +253 @@
-before)))))))
+before#before_adv)))))))
@@ -259 +259 @@
-so)
+so#so_adv)
@@ -261 +261 @@
-far)
+far#far_adj)
@@ -265 +265 @@
-are)
+are#be_v)
@@ -268 +268 @@
-wee))
+wee#we_pron))
@@ -271 +271 @@
-from)
+from#from_prep)
@@ -274 +274 @@
-enlargm=t=))))
+enlargm=t=#enlargement_n))))
@@ -282 +282 @@
-but)
+but#but_prep)
@@ -285 +285 @@
-our)
+our#our_pron)
@@ -287 +287 @@
-times))
+times#time_n))
@@ -289 +289 @@
-are)
+are#be_v)
@@ -292 +292 @@
-in)
+in#in_prep)
@@ -296 +296 @@
-God's)
+God's#NA)
@@ -300 +300 @@
-hand)
+hand#hand_n)
@@ -306 +306 @@
-who))
+who#who_pron))
@@ -313 +313 @@
-hath)
+hath#have_v)
@@ -315 +315 @@
-sent)
+sent#send_v1)
@@ -318 +318 @@
-us))
+us#us_pron))
@@ -321 +321 @@
-hither))
+hither#hither_adv))
@@ -327 +327 @@
-I))
+I#i_pron))
@@ -329 +329 @@
-am)
+am#be_v)
@@ -332 +332 @@
-confident)))
+confident#confident_adj)))
@@ -337 +337 @@
-for)
+for#for_prep)
@@ -340 +340 @@
-good)))
+good#good_adj)))
@@ -345 +345 @@
-though)
+though#though_adv)
@@ -354 +354 @@
-how))
+how#how_adv))
@@ -357 +357 @@
-or)
+or#or_conj2)
@@ -360 +360 @@
-which)
+which#which_adj)
@@ -362 +362 @@
-way)))
+way#way_n1)))
@@ -365 +365 @@
-or)
+or#or_conj2)
@@ -368 +368 @@
-wherein)))))
+wherein#wherein_adv)))))
@@ -371 +371 @@
-I))
+I#i_pron))
@@ -373 +373 @@
-know)
+know#know_v)
@@ -375 +375 @@
-not))))))))
+not#not_adv))))))))
@@ -383 +383 @@
-but)
+but#but_prep)
@@ -386 +386 @@
-hee)
+hee#he_pron)
@@ -390 +390 @@
-is)
+is#be_v)
@@ -393 +393 @@
-faithful))
+faithful#faithful_adj))
@@ -398 +398 @@
-that)
+that#that_conj)
@@ -403 +403 @@
-hath)
+hath#have_v)
@@ -405 +405 @@
-promis'd)))
+promis'd#promise_v)))
@@ -415 +415 @@
-My)
+My#my_adj)
@@ -417 +417 @@
-Chamber)
+Chamber#chamber_n)
@@ -419 +419 @@
-fellows))
+fellows#fellow_n))
@@ -422 +422 @@
-&)
+&#and_conj1)
@@ -425 +425 @@
-I))))
+I#i_pron))))
@@ -427 +427 @@
-differ)
+differ#differ_v)
@@ -430 +430 @@
-someth))
+someth#something_n))
@@ -433 +433 @@
-in)
+in#in_prep)
@@ -436 +436 @@
-our)
+our#our_pron)
@@ -438 +438 @@
-apprehensions)
+apprehensions#apprehension_n)
@@ -441 +441 @@
-of)
+of#of_prep)
@@ -444 +444 @@
-things)
+things#thing_n1)
@@ -447 +447 @@
-past))))))
+past#past_adj))))))
@@ -453 +453 @@
-which))
+which#which_adj))
@@ -460 +460 @@
-wil)
+wil#will_v1)
@@ -462 +462 @@
-not)
+not#not_adv)
@@ -464 +464 @@
-bee)
+bee#be_v)
@@ -466 +466 @@
-helpt)))
+helpt#help_v)))
@@ -474 +474 @@
-but)
+but#but_prep)
@@ -477 +477 @@
-for)
+for#for_prep)
@@ -480 +480 @@
-the)
+the#the_adj)
@@ -482 +482 @@
-unseen)
+unseen#unseen_adj)
@@ -484 +484 @@
-th.)
+th.#thing_n1)
@@ -489 +489 @@
-that)
+that#that_conj)
@@ -494 +494 @@
-are)
+are#be_v)
@@ -497 +497 @@
-to)
+to#to_prep)
@@ -499 +499 @@
-come))))
+come#come_v))))
@@ -504 +504 @@
-that)
+that#that_conj)
@@ -509 +509 @@
-are)
+are#be_v)
@@ -512 +512 @@
-Eternal))))))
+Eternal#eternal_adj))))))
@@ -517 +517 @@
-wee))
+wee#we_pron))
@@ -519 +519 @@
-are)
+are#be_v)
@@ -521 +521 @@
-all)
+all#all_adj)
@@ -524 +524 @@
-one))
+one#one_adj))
@@ -533 +533 @@
-Our)
+Our#our_pron)
@@ -535 +535 @@
-Afternoones))
+Afternoones#afternoon_n))
@@ -540 +540 @@
-til)
+til#till_prep)
@@ -543 +543 @@
-late)))
+late#late_adv)))
@@ -547 +547 @@
-are)
+are#be_v)
@@ -549 +549 @@
-fild)
+fild#fill_v)
@@ -552 +552 @@
-with)
+with#with_prep)
@@ -555 +555 @@
-visitants)
+visitants#visitant_n)
@@ -561 +561 @@
-who))
+who#who_pron))
@@ -569 +569 @@
-love)
+love#love_v1)
@@ -572 +572 @@
-us)))
+us#us_pron)))
@@ -575 +575 @@
-&)
+&#and_conj1)
@@ -580 +580 @@
-wish)
+wish#wish_v)
@@ -583 +583 @@
-us))
+us#us_pron))
@@ -586 +586 @@
-wel))))
+wel#well_adv))))
@@ -589 +589 @@
-&)
+&#and_conj1)
@@ -594 +594 @@
-are)
+are#be_v)
@@ -597 +597 @@
-kind)
+kind#kind_adj)
@@ -600 +600 @@
-to)
+to#to_prep)
@@ -603 +603 @@
-us))))))))))
+us#us_pron))))))))))
@@ -611 +611 @@
-but)
+but#but_prep)
@@ -614 +614 @@
-wee))
+wee#we_pron))
@@ -616,3 +616,3 @@
-can@)
-(NEG
-@not)
+can@#can_v1)
+(NEG
+@not#not_adv)
@@ -620 +620 @@
-doe)
+doe#do_v)
@@ -623 +623 @@
-with)
+with#with_prep)
@@ -626 +626 @@
-them)))
+them#them_pron)))
@@ -631 +631 @@
-what))
+what#what_pron))
@@ -639 +639 @@
-wee))
+wee#we_pron))
@@ -641 +641 @@
-would)
+would#will_v1)
@@ -652 +652 @@
-Cos.)
+Cos.#NA)
@@ -654 +654 @@
-Crue))
+Crue#NA))
@@ -656 +656 @@
-hath)
+hath#have_v)
@@ -658 +658 @@
-been)
+been#be_v)
@@ -661 +661 @@
-an)
+an#a_adj2)
@@ -663 +663 @@
-hour))
+hour#hour_n))
@@ -666 +666 @@
-with)
+with#with_prep)
@@ -669 +669 @@
-mee)))
+mee#me_pron1)))
@@ -672 +672 @@
-this)
+this#this_pron)
@@ -674 +674 @@
-morning)))
+morning#morning_n)))
@@ -681 +681 @@
-shee))
+shee#she_pron))
@@ -683 +683 @@
-brought)
+brought#bring_v)
@@ -686 +686 @@
-mee))
+mee#me_pron1))
@@ -689 +689 @@
-a)
+a#a_adj2)
@@ -691 +691 @@
-quart)
+quart#quart_n2)
@@ -694 +694 @@
-of)
+of#of_prep)
@@ -698 +698 @@
-Aqu.)
+Aqu.#NA)
@@ -700 +700 @@
-Mirab.))
+Mirab.#NA))
@@ -704 +704 @@
-which))
+which#which_adj))
@@ -710 +710 @@
-I))
+I#i_pron))
@@ -712 +712 @@
-would)
+would#will_v1)
@@ -714 +714 @@
-have)
+have#have_v)
@@ -716 +716 @@
-had)
+had#have_v)
@@ -722 +722 @@
-her))
+her#her_pron1))
@@ -724 +724 @@
-take)
+take#take_v)
@@ -727 +727 @@
-back))
+back#back_adv))
@@ -730 +730 @@
-again))
+again#again_adv))
@@ -735 +735 @@
-til)
+til#till_prep)
@@ -738 +738 @@
-more)
+more#more_adj)
@@ -740 +740 @@
-need)))))))))
+need#need_n1)))))))))
@@ -748 +748 @@
-but)
+but#but_prep)
@@ -751 +751 @@
-shee))
+shee#she_pron))
@@ -753 +753 @@
-would)
+would#will_v1)
@@ -755 +755 @@
-not)
+not#not_adv)
@@ -766 +766 @@
-I))
+I#i_pron))
@@ -768 +768 @@
-have)
+have#have_v)
@@ -770 +770 @@
-not)
+not#not_adv)
@@ -773 +773 @@
-yet))
+yet#yet_adv))
@@ -775 +775 @@
-opened)
+opened#open_v)
@@ -778 +778 @@
-y=t=)
+y=t=#that_pron)
@@ -780 +780 @@
-little)
+little#little_adj)
@@ -782 +782 @@
-bottle)
+bottle#bottle_n2)
@@ -793 +793 @@
-I))
+I#i_pron))
@@ -795 +795 @@
-brought)
+brought#bring_v)
@@ -798 +798 @@
-with)
+with#with_prep)
@@ -801 +801 @@
-mee))))))
+mee#me_pron1))))))
@@ -806 +806 @@
-since)
+since#since_adv)
@@ -813 +813 @@
-I))
+I#i_pron))
@@ -815 +815 @@
-came))))
+came#come_v))))
@@ -821 +821 @@
-not)
+not#not_adv)
@@ -823 +823 @@
-wanting)
+wanting#want_v)
@@ -826 +826 @@
-it)))
+it#it_pron)))
@@ -831 +831 @@
-&)
+&#and_conj1)
@@ -834 +834 @@
-being)
+being#be_v)
@@ -837 +837 @@
-more)
+more#more_adj)
@@ -839 +839 @@
-afrayd)
+afrayd#afraid_adj)
@@ -842 +842 @@
-of)
+of#of_prep)
@@ -847 +847 @@
-what))
+what#what_pron))
@@ -854 +854 @@
-might)
+might#may_v1)
@@ -856 +856 @@
-heat)
+heat#heat_v)
@@ -861 +861 @@
-mee)))))))
+mee#me_pron1)))))))
@@ -866 +866 @@
-in)
+in#in_prep)
@@ -869 +869 @@
-regard)
+regard#regard_n)
@@ -876 +876 @@
-wee))
+wee#we_pron))
@@ -878 +878 @@
-have)
+have#have_v)
@@ -881 +881 @@
-no)
+no#no_adj)
@@ -883 +883 @@
-drink)
+drink#drink_n)
@@ -886 +886 @@
-but)
+but#but_prep)
@@ -889 +889 @@
-strong))))
+strong#strong_adj))))
@@ -894 +894 @@
-unless)
+unless#unless_adv)
@@ -897 +897 @@
-very)
+very#very_adj)
@@ -899 +899 @@
-seldom)))
+seldom#seldom_adv)))
@@ -904 +904 @@
-neither)
+neither#neither_adv)
@@ -906 +906 @@
-morning)
+morning#morning_n)
@@ -910 +910 @@
-noon)
+noon#noon_n)
@@ -914 +914 @@
-nor)
+nor#nor_conj1)
@@ -916 +916 @@
-night))))))
+night#night_n))))))
@@ -922 +922 @@
-w=ch=))
+w=ch=#which_adj))
@@ -929 +929 @@
-may)
+may#may_v1)
@@ -931 +931 @@
-turn)
+turn#turn_v)
@@ -934 +934 @@
-to)
+to#to_prep)
@@ -937 +937 @@
-Feavorish)
+Feavorish#feverish_adj)
@@ -939 +939 @@
-distempers)))
+distempers#distemper_n1)))
@@ -944 +944 @@
-wanting)
+wanting#want_v)
@@ -947 +947 @@
-exercise)
+exercise#exercise_n)
@@ -955 +955 @@
-I))
+I#i_pron))
@@ -957 +957 @@
-have)
+have#have_v)
@@ -959 +959 @@
-not)
+not#not_adv)
@@ -961 +961 @@
-trodden)
+trodden#tread_v)
@@ -964 +964 @@
-on)
+on#on_prep)
@@ -967 +967 @@
-the)
+the#the_adj)
@@ -969 +969 @@
-ground)
+ground#ground_n)
@@ -972 +972 @@
-since)
+since#since_adv)
@@ -975 +975 @@
-Sat.))))))
+Sat.#NA))))))
@@ -979 +979 @@
-which))
+which#which_adj))
@@ -985 +985 @@
-using)
+using#use_v)
@@ -988 +988 @@
-mys.))
+mys.#NA))
@@ -991 +991 @@
-to)
+to#to_prep)
@@ -996 +996 @@
-in)
+in#in_prep)
@@ -999 +999 @@
-mornings))))
+mornings#morning_n))))
@@ -1002 +1002 @@
-I))
+I#i_pron))
@@ -1004 +1004 @@
-thought)
+thought#think_v2)
@@ -1011 +1011 @@
-y=e=)
+y=e=#the_adj)
@@ -1013 +1013 @@
-want)
+want#want_n2)
@@ -1016 +1016 @@
-of)
+of#of_prep)
@@ -1019 +1019 @@
-it))))
+it#it_pron))))
@@ -1021 +1021 @@
-might)
+might#may_v1)
@@ -1023 +1023 @@
-bee)
+bee#be_v)
@@ -1026 +1026 @@
-prejudicial))))))))))))
+prejudicial#prejudicial_adj1))))))))))))
@@ -1034 +1034 @@
-but)
+but#but_prep)
@@ -1037 +1037 @@
-hitherto))
+hitherto#hitherto_adv))
@@ -1040 +1040 @@
-it))
+it#it_pron))
@@ -1042 +1042 @@
-is)
+is#be_v)
@@ -1044 +1044 @@
-not)
+not#not_adv)
@@ -1055 +1055 @@
-I))
+I#i_pron))
@@ -1057 +1057 @@
-have)
+have#have_v)
@@ -1059 +1059 @@
-not)
+not#not_adv)
@@ -1061 +1061 @@
-tasted)
+tasted#taste_v)
@@ -1064 +1064 @@
-Butter))
+Butter#butter_n1))
@@ -1067 +1067 @@
-yet))
+yet#yet_adv))
@@ -1070 +1070 @@
-with)
+with#with_prep)
@@ -1073 +1073 @@
-bread)))
+bread#bread_n)))
@@ -1076 +1076 @@
-since)
+since#since_adv)
@@ -1083 +1083 @@
-I))
+I#i_pron))
@@ -1085 +1085 @@
-came)
+came#come_v)
@@ -1088 +1088 @@
-from)
+from#from_prep)
@@ -1091 +1091 @@
-home))))))
+home#home_n1))))))
@@ -1100 +1100 @@
-This)
+This#this_pron)
@@ -1102 +1102 @@
-dinner))
+dinner#dinner_n))
@@ -1105 +1105 @@
-wee))
+wee#we_pron))
@@ -1107 +1107 @@
-had)
+had#have_v)
@@ -1110 +1110 @@
-Beanes)
+Beanes#bean_n)
@@ -1112 +1112 @@
-and)
+and#and_conj1)
@@ -1114 +1114 @@
-Bacon)
+Bacon#bacon_n)
@@ -1118 +1118 @@
-Sammon)
+Sammon#salmon_n1)
@@ -1120 +1120 @@
-&c.)))
+&c.#et-cetera_n)))
@@ -1126 +1126 @@
-but)
+but#but_prep)
@@ -1129 +1129 @@
-I))
+I#i_pron))
@@ -1131 +1131 @@
-am)
+am#be_v)
@@ -1134 +1134 @@
-careful)
+careful#careful_adj)
@@ -1138 +1138 @@
-w=t=))
+w=t=#what_pron))
@@ -1146 +1146 @@
-I))
+I#i_pron))
@@ -1148 +1148 @@
-eat)
+eat#eat_v)
@@ -1153 +1153 @@
-not)
+not#not_adv)
@@ -1155 +1155 @@
-Fishes)
+Fishes#fish_n1)
@@ -1157 +1157 @@
-&)
+&#and_conj1)
@@ -1159 +1159 @@
-Fleshes)))))
+Fleshes#flesh_n)))))
@@ -1168 +1168 @@
-Mrs.)
+Mrs.#NA)
@@ -1170 +1170 @@
-Wenlock))
+Wenlock#NA))
@@ -1172 +1172 @@
-was)
+was#be_v)
@@ -1175 +1175 @@
-to)
+to#to_prep)
@@ -1177 +1177 @@
-see)
+see#see_v)
@@ -1180 +1180 @@
-mee))
+mee#me_pron1))
@@ -1183 +1183 @@
-yesterday))))
+yesterday#yesterday_adv))))
@@ -1189 +1189 @@
-&)
+&#and_conj1)
@@ -1193 +1193 @@
-brought)
+brought#bring_v)
@@ -1196 +1196 @@
-mee))
+mee#me_pron1))
@@ -1199 +1199 @@
-a)
+a#a_adj2)
@@ -1201 +1201 @@
-bottle)
+bottle#bottle_n2)
@@ -1204 +1204 @@
-of)
+of#of_prep)
@@ -1207 +1207 @@
-wine))))
+wine#wine_n1))))
@@ -1216 +1216 @@
-I))
+I#i_pron))
@@ -1218 +1218 @@
-bestow)
+bestow#bestow_v)
@@ -1221 +1221 @@
-all)
+all#all_adj)
@@ -1224 +1224 @@
-of)
+of#of_prep)
@@ -1227 +1227 @@
-that)
+that#that_pron)
@@ -1229 +1229 @@
-kind))))
+kind#kind_n))))
@@ -1232 +1232 @@
-in)
+in#in_prep)
@@ -1235 +1235 @@
-Common)))
+Common#common_adj)))
@@ -1241 +1241 @@
-my)
+my#my_adj)
@@ -1243 +1243 @@
-Companions))
+Companions#companion_n1))
@@ -1246 +1246 @@
-strangers)
+strangers#stranger_n)
@@ -1249 +1249 @@
-here))))
+here#here_adv))))
@@ -1258 +1258 @@
-This)
+This#this_pron)
@@ -1260 +1260 @@
-Aftern.))
+Aftern.#afternoon_n))
@@ -1263 +1263 @@
-y=e=)
+y=e=#the_adj)
@@ -1265 +1265 @@
-Citizens))
+Citizens#citizen_n))
@@ -1267 +1267 @@
-meet)
+meet#meet_v)
@@ -1270 +1270 @@
-on)
+on#on_prep)
@@ -1273 +1273 @@
-Rood-dee)))
+Rood-dee#NA)))
@@ -1276 +1276 @@
-with)
+with#with_prep)
@@ -1279 +1279 @@
-Halberts)))
+Halberts#NA)))
@@ -1288 +1288 @@
-they))
+they#they_pron))
@@ -1290 +1290 @@
-think)
+think#think_v2)
@@ -1293 +1293 @@
-to)
+to#to_prep)
@@ -1295 +1295 @@
-take)
+take#take_v)
@@ -1298 +1298 @@
-the)
+the#the_adj)
@@ -1300 +1300 @@
-Oath)))
+Oath#oath_n)))
@@ -1309 +1309 @@
-Mrs.)
+Mrs.#NA)
@@ -1311 +1311 @@
-Clive))
+Clive#NA))
@@ -1313 +1313 @@
-is)
+is#be_v)
@@ -1316 +1316 @@
-here))
+here#here_adv))
@@ -1325 +1325 @@
-Mr.)
+Mr.#NA)
@@ -1327 +1327 @@
-Woodh.))
+Woodh.#NA))
@@ -1330 +1330 @@
-I))
+I#i_pron))
@@ -1332 +1332 @@
-hope)
+hope#hope_v)
@@ -1340 +1340 @@
-recovers)))
+recovers#recover_v1)))
@@ -1348 +1348 @@
-&)
+&#and_conj1)
@@ -1353 +1353 @@
-w=n=)
+w=n=#when_adv)
@@ -1356 +1356 @@
-wel)))
+wel#well_adj)))
@@ -1358 +1358 @@
-comes)
+comes#come_v)
@@ -1360 +1360 @@
-in)
+in#in_adv)
@@ -1363 +1363 @@
-again))
+again#again_adv))
@@ -1372 +1372 @@
-I))
+I#i_pron))
@@ -1374 +1374 @@
-have)
+have#have_v)
@@ -1376 +1376 @@
-seen)
+seen#see_v)
@@ -1379 +1379 @@
-Mr.)
+Mr.#NA)
@@ -1381 +1381 @@
-H.)))
+H.#NA)))
@@ -1387 +1387 @@
-but)
+but#but_prep)
@@ -1390 +1390 @@
-'t@))
+'t@#it_pron))
@@ -1392 +1392 @@
-@is)
+@is#be_v)
@@ -1395 +1395 @@
-at)
+at#at_prep)
@@ -1398 +1398 @@
-distance)))
+distance#distance_n)))
@@ -1406 +1406 @@
-Let)
+Let#let_v1)
@@ -1410 +1410 @@
-mee))
+mee#me_pron1))
@@ -1412 +1412 @@
-hear)
+hear#hear_v)
@@ -1415 +1415 @@
-from)
+from#from_prep)
@@ -1418 +1418 @@
-you)))
+you#you_pron)))
@@ -1424 +1424 @@
-how))
+how#how_adv))
@@ -1433 +1433 @@
-you))
+you#you_pron))
@@ -1435 +1435 @@
-doe))
+doe#do_v))
@@ -1438 +1438 @@
-&)
+&#and_conj1)
@@ -1442 +1442 @@
-the)
+the#the_adj)
@@ -1444 +1444 @@
-children)
+children#child_n)
@@ -1446 +1446 @@
-&c.))))))
+&c.#et-cetera_n))))))
@@ -1449 +1449 @@
-as)
+as#as_adv)
@@ -1451 +1451 @@
-oft)
+oft#oft_adv)
@@ -1454 +1454 @@
-as)
+as#as_adv)
@@ -1465 +1465 @@
-you))
+you#you_pron))
@@ -1467 +1467 @@
-can)
+can#can_v1)
@@ -1477 +1477 @@
-Shew)
+Shew#show_v)
@@ -1479 +1479 @@
-not)
+not#not_adv)
@@ -1482 +1482 @@
-my)
+my#my_adj)
@@ -1484 +1484 @@
-letters))
+letters#letter_n1))
@@ -1493 +1493 @@
-When)
+When#when_adv)
@@ -1500 +1500 @@
-you))
+you#you_pron))
@@ -1502 +1502 @@
-think)
+think#think_v2)
@@ -1508 +1508 @@
-fit))
+fit#fit_adj))
@@ -1511 +1511 @@
-to)
+to#to_prep)
@@ -1513 +1513 @@
-send)
+send#send_v1)
@@ -1516 +1516 @@
-for)
+for#for_prep)
@@ -1519 +1519 @@
-them)))
+them#them_pron)))
@@ -1522 +1522 @@
-from)
+from#from_prep)
@@ -1525 +1525 @@
-Salop))))))))
+Salop#NA))))))))
@@ -1529 +1529 @@
-Send)
+Send#send_v1)
@@ -1532 +1532 @@
-love))
+love#love_n1))
@@ -1535 +1535 @@
-to)
+to#to_prep)
@@ -1538 +1538 @@
-Matth.))))
+Matth.#NA))))
@@ -1545 +1545 @@
-Our)
+Our#our_pron)
@@ -1547 +1547 @@
-Gards))
+Gards#guard_n))
@@ -1549 +1549 @@
-change)
+change#change_v)
@@ -1552 +1552 @@
-every)
+every#every_adj)
@@ -1554 +1554 @@
-howr))
+howr#hour_n))
@@ -1560 +1560 @@
-which))
+which#which_adj))
@@ -1567 +1567 @@
-makes)
+makes#make_v1)
@@ -1571 +1571 @@
-it))
+it#it_pron))
@@ -1575 +1575 @@
-so)
+so#so_adv)
@@ -1577 +1577 @@
-very))
+very#very_adj))
@@ -1579 +1579 @@
-hard))
+hard#hard_adj))
@@ -1582 +1582 @@
-to)
+to#to_prep)
@@ -1584 +1584 @@
-come)
+come#come_v)
@@ -1587 +1587 @@
-to)
+to#to_prep)
@@ -1590 +1590 @@
-us)))))))
+us#us_pron)))))))
@@ -1599 +1599 @@
-else))
+else#else_adv))
@@ -1602 +1602 @@
-wee))
+wee#we_pron))
@@ -1604 +1604 @@
-might)
+might#may_v1)
@@ -1606 +1606 @@
-oblige)
+oblige#oblige_v)
@@ -1609 +1609 @@
-them))
+them#them_pron))
@@ -1618 +1618 @@
-I))
+I#i_pron))
@@ -1620 +1620 @@
-would)
+would#will_v1)
@@ -1623 +1623 @@
-gladly))
+gladly#gladly_adv))
@@ -1625 +1625 @@
-see)
+see#see_v)
@@ -1628 +1628 @@
-him))
+him#him_pron))
@@ -1636 +1636 @@
-but)
+but#but_prep)
@@ -1641 +1641 @@
-when))
+when#when_adv))
@@ -1644 +1644 @@
-or)
+or#or_conj2)
@@ -1647 +1647 @@
-how)))))
+how#how_adv)))))
@@ -1650 +1650 @@
-I))
+I#i_pron))
@@ -1652 +1652 @@
-know)
+know#know_v)
@@ -1654 +1654 @@
-not)
+not#not_adv)
@@ -1663 +1663 @@
-I))
+I#i_pron))
@@ -1665 +1665 @@
-think)
+think#think_v2)
@@ -1672 +1672 @@
-there@))
+there@#there_adv))
@@ -1674 +1674 @@
-@'s)
+@'s#be_v)
@@ -1677 +1677 @@
-little)
+little#little_adj)
@@ -1679 +1679 @@
-danger)
+danger#danger_n)
@@ -1682 +1682 @@
-of)
+of#of_prep)
@@ -1685 +1685 @@
-any)
+any#any_adj)
@@ -1687 +1687 @@
-harm)
+harm#harm_n)
@@ -1690 +1690 @@
-to)
+to#to_prep)
@@ -1693 +1693 @@
-him))))))
+him#him_pron))))))
@@ -1696 +1696 @@
-here))))
+here#here_adv))))
@@ -1701 +1701 @@
-if)
+if#if_conj)
@@ -1708 +1708 @@
-there))
+there#there_adv))
@@ -1710 +1710 @@
-bee)
+bee#be_v)
@@ -1713 +1713 @@
-none)
+none#none_pron)
@@ -1716 +1716 @@
-at)
+at#at_prep)
@@ -1719 +1719 @@
-home)))
+home#home_n1)))
@@ -1724 +1724 @@
-at)
+at#at_prep)
@@ -1727 +1727 @@
-his)
+his#his_adj)
@@ -1729 +1729 @@
-return)))))))
+return#return_n)))))))
@@ -1738 +1738 @@
-Love))
+Love#love_n1))
@@ -1742 +1742 @@
-to)
+to#to_prep)
@@ -1745 +1745 @@
-Sarah)
+Sarah#NA)
@@ -1747 +1747 @@
-&)
+&#and_conj1)
@@ -1749 +1749 @@
-El'nor)))
+El'nor#NA)))
@@ -1754 +1754 @@
-&)
+&#and_conj1)
@@ -1757 +1757 @@
-to)
+to#to_prep)
@@ -1760 +1760 @@
-Ann)
+Ann#NA)
@@ -1762 +1762 @@
-D.))))
+D.#NA))))
@@ -1765 +1765 @@
-&)
+&#and_conj1)
@@ -1768 +1768 @@
-to)
+to#to_prep)
@@ -1771 +1771 @@
-all)
+all#all_adj)
@@ -1773 +1773 @@
-y=e=)
+y=e=#the_adj)
@@ -1775 +1775 @@
-rest)
+rest#rest_n2)
@@ -1778 +1778 @@
-of)
+of#of_prep)
@@ -1781 +1781 @@
-you)))))))
+you#you_pron)))))))
@@ -1789 +1789 @@
-doe)
+doe#do_v)
@@ -1794 +1794 @@
-w=t=))
+w=t=#what_pron))
@@ -1802 +1802 @@
-you))
+you#you_pron))
@@ -1804 +1804 @@
-can)
+can#can_v1)
@@ -1810 +1810 @@
-to)
+to#to_prep)
@@ -1812 +1812 @@
-get)
+get#get_v)
@@ -1815 +1815 @@
-to)
+to#to_prep)
@@ -1818 +1818 @@
-heaven)))
+heaven#NA)))
@@ -1821 +1821 @@
-your)
+your#your_pron)
@@ -1823 +1823 @@
-selves)))
+selves#self_pron)))
@@ -1826 +1826 @@
-&)
+&#and_conj1)
@@ -1829 +1829 @@
-to)
+to#to_prep)
@@ -1831 +1831 @@
-help)
+help#help_v)
@@ -1834 +1834 @@
-one)
+one#one_adj)
@@ -1836 +1836 @@
-another))
+another#another_pron))
@@ -1839 +1839 @@
-thither))))))))
+thither#thither_adv))))))))
@@ -1847 +1847 @@
-Prepare)
+Prepare#prepare_v)
@@ -1850 +1850 @@
-for)
+for#for_prep)
@@ -1853 +1853 @@
-further)
+further#further_adj)
@@ -1855 +1855 @@
-Sufferings)
+Sufferings#suffering_n)
@@ -1861 +1861 @@
-to)
+to#to_prep)
@@ -1864 +1864 @@
-which)))
+which#which_adj)))
@@ -1870 +1870 @@
-it))
+it#it_pron))
@@ -1872 +1872 @@
-may)
+may#may_v1)
@@ -1874 +1874 @@
-bee)
+bee#be_v)
@@ -1881 +1881 @@
-these)
+these#these_pron)
@@ -1883 +1883 @@
-th.))
+th.#thing_n1))
@@ -1885 +1885 @@
-are)
+are#be_v)
@@ -1887 +1887 @@
-but)
+but#but_prep)
@@ -1890 +1890 @@
-the)
+the#the_adj)
@@ -1892 +1892 @@
-Preamble)
+Preamble#preamble_n)
@@ -1900 +1900 @@
-but)
+but#but_prep)
@@ -1903 +1903 @@
-all)
+all#all_adj)
@@ -1907 +1907 @@
-is)
+is#be_v)
@@ -1910 +1910 @@
-well))
+well#well_adv))
@@ -1915 +1915 @@
-that)
+that#that_conj)
@@ -1920 +1920 @@
-ends)
+ends#end_v1)
@@ -1923 +1923 @@
-everlastingly)
+everlastingly#everlastingly_adv)
@@ -1925 +1925 @@
-well))))
+well#well_adv))))
@@ -1933 +1933 @@
-Thankes)
+Thankes#thank_n)
@@ -1936 +1936 @@
-for)
+for#for_prep)
@@ -1939 +1939 @@
-all)
+all#all_adj)
@@ -1941 +1941 @@
-y=r=)
+y=r=#your_pron)
@@ -1943 +1943 @@
-love)
+love#love_n1)
@@ -1946 +1946 @@
-&)
+&#and_conj1)
@@ -1949 +1949 @@
-faithfulnes)))
+faithfulnes#faithfulness_n)))
@@ -1952 +1952 @@
-to)
+to#to_prep)
@@ -1955 +1955 @@
-mee)))
+mee#me_pron1)))
@@ -1960 +1960 @@
-&)
+&#and_conj1)
@@ -1963 +1963 @@
-Patience)
+Patience#patience_n.I)
@@ -1966 +1966 @@
-with)
+with#with_prep)
@@ -1969 +1969 @@
-mee)))))))
+mee#me_pron1)))))))
@@ -1978 +1978 @@
-the)
+the#the_adj)
@@ -1980 +1980 @@
-lord))
+lord#NA))
@@ -1982 +1982 @@
-wil)
+wil#will_v1)
@@ -1984 +1984 @@
-reward)
+reward#reward_v)
@@ -1987 +1987 @@
-it))
+it#it_pron))
@@ -1996 +1996 @@
-One)
+One#one_adj)
@@ -1999 +1999 @@
-of)
+of#of_prep)
@@ -2002 +2002 @@
-my)
+my#my_adj)
@@ -2004 +2004 @@
-Fellow)
+Fellow#fellow_n)
@@ -2006 +2006 @@
-Prisoners))))
+Prisoners#prisoner_n2))))
@@ -2009 +2009 @@
-last)
+last#last_adv)
@@ -2011 +2011 @@
-night))
+night#night_n))
@@ -2013 +2013 @@
-receiv'd)
+receiv'd#receive_v)
@@ -2016 +2016 @@
-a)
+a#a_adj2)
@@ -2018 +2018 @@
-letter)
+letter#letter_n1)
@@ -2021 +2021 @@
-from)
+from#from_prep)
@@ -2024 +2024 @@
-his)
+his#his_adj)
@@ -2026 +2026 @@
-wife)))
+wife#wife_n)))
@@ -2029 +2029 @@
-subscrib'd)
+subscrib'd#subscribe_v)
@@ -2035 +2035 @@
-so))
+so#so_adv))
@@ -2038 +2038 @@
-I))
+I#i_pron))
@@ -2040 +2040 @@
-rest)
+rest#rest_v2)
@@ -2043 +2043 @@
-dear)
+dear#dear_adj1)
@@ -2045 +2045 @@
-Husband))
+Husband#husband_n))
@@ -2048 +2048 @@
-in)
+in#in_prep)
@@ -2051 +2051 @@
-all)
+all#all_adj)
@@ -2053 +2053 @@
-Duty)
+Duty#duty_n)
@@ -2056 +2056 @@
-&)
+&#and_conj1)
@@ -2059 +2059 @@
-Obedience)))))
+Obedience#obedience_n)))))
@@ -2062 +2062 @@
-your)
+your#your_pron)
@@ -2064 +2064 @@
-Obedient)
+Obedient#obedient_adj)
@@ -2066 +2066 @@
-wife)))))
+wife#wife_n)))))
@@ -2075 +2075 @@
-Such))
+Such#such_adj))
@@ -2077 +2077 @@
-is)
+is#be_v)
@@ -2080 +2080 @@
-lancashire)
+lancashire#NA)
@@ -2082 +2082 @@
-kindness))
+kindness#kindness_n))
@@ -2090 +2090 @@
-but)
+but#but_prep)
@@ -2093 +2093 @@
-deeds))
+deeds#deed_n))
@@ -2095 +2095 @@
-exceed)
+exceed#exceed_v)
@@ -2098 +2098 @@
-word))
+word#word_n))
@@ -2107 +2107 @@
-I))
+I#i_pron))
@@ -2109 +2109 @@
-am)
+am#be_v)
@@ -2112 +2112 @@
-in)
+in#in_prep)
@@ -2115 +2115 @@
-short)))
+short#short_adj)))
@@ -2120 +2120 @@
-most)
+most#most_adj)
@@ -2122 +2122 @@
-intirely))
+intirely#entirely_adj))
@@ -2125 +2125 @@
-&)
+&#and_conj1)
@@ -2128 +2128 @@
-most)
+most#most_adj)
@@ -2130 +2130 @@
-affectionately))))
+affectionately#affectionately_adv))))
@@ -2132 +2132 @@
-thine)
+thine#thine_adj)
@@ -2137 +2137 @@
-P.)
+P.#NA)
@@ -2139 +2139 @@
-H.))))
+H.#NA))))
