--- rferrar-1633-e2-h.psd.vert	2025-09-10 13:50:35
+++ rferrar-1633-e2-h.lem.vert	2025-09-10 13:50:45
@@ -15 +15 @@
-Most)
+Most#most_adj)
@@ -17 +17 @@
-Deare))
+Deare#dear_adj1))
@@ -19 +19 @@
-Mother))
+Mother#mother_n1))
@@ -24 +24 @@
-beinge)
+beinge#be_v)
@@ -27 +27 @@
-In)
+In#in_prep)
@@ -30 +30 @@
-that)
+that#that_pron)
@@ -32 +32 @@
-extremety)
+extremety#extremity_n)
@@ -35 +35 @@
-of)
+of#of_prep)
@@ -38 +38 @@
-missery)))
+missery#misery_n)))
@@ -43 +43 @@
-that)
+that#that_conj)
@@ -47 +47 @@
-I))
+I#i_pron))
@@ -50 +50 @@
-nowe))
+nowe#now_adv))
@@ -52 +52 @@
-am)
+am#be_v)
@@ -54 +54 @@
-falen)
+falen#fall_v)
@@ -57 +57 @@
-Into)
+Into#into_prep)
@@ -64 +64 @@
-whether)
+whether#whither_adv)
@@ -71 +71 @@
-should)
+should#shall_v)
@@ -74 +74 @@
-I))
+I#i_pron))
@@ -76 +76 @@
-fly)
+fly#fly_v1)
@@ -79 +79 @@
-for)
+for#for_prep)
@@ -82 +82 @@
-succor))))
+succor#succour_n))))
@@ -85 +85 @@
-but)
+but#but_prep)
@@ -88 +88 @@
-to)
+to#to_prep)
@@ -91 +91 @@
-the)
+the#the_adj)
@@ -93 +93 @@
-wombe)
+wombe#womb_n)
@@ -98 +98 @@
-that)
+that#that_conj)
@@ -103 +103 @@
-bare)
+bare#bear_v1)
@@ -106 +106 @@
-me)))))))
+me#me_pron1)))))))
@@ -115 +115 @@
-I))
+I#i_pron))
@@ -117 +117 @@
-doe)
+doe#do_v)
@@ -119 +119 @@
-desier)
+desier#desire_v)
@@ -122 +122 @@
-you))
+you#you_pron))
@@ -125 +125 @@
-for)
+for#for_prep)
@@ -129 +129 @@
-Christ)
+Christ#NA)
@@ -131 +131 @@
-Iesus))
+Iesus#NA))
@@ -133 +133 @@
-sake)))
+sake#sake_n1)))
@@ -136 +136 @@
-not)
+not#not_adv)
@@ -138 +138 @@
-to)
+to#to_prep)
@@ -140 +140 @@
-lett)
+lett#let_v1)
@@ -144 +144 @@
-your)
+your#your_pron)
@@ -146 +146 @@
-pore)
+pore#poor_adj)
@@ -148 +148 @@
-Sonne))
+Sonne#son_n1))
@@ -150 +150 @@
-to)
+to#to_prep)
@@ -152 +152 @@
-goe)
+goe#go_v)
@@ -155 +155 @@
-to)
+to#to_prep)
@@ -158 +158 @@
-a)
+a#a_adj2)
@@ -160 +160 @@
-Iayle)))
+Iayle#jail_n)))
@@ -166 +166 @@
-there))
+there#there_adv))
@@ -168 +168 @@
-to)
+to#to_prep)
@@ -171 +171 @@
-ly)
+ly#lie_v1)
@@ -175 +175 @@
-dy)
+dy#die_v1)
@@ -177 +177 @@
-and)
+and#and_conj1)
@@ -179 +179 @@
-Rott)))))
+Rott#rot_v)))))
@@ -188 +188 @@
-Even)
+Even#even_adj)
@@ -190 +190 @@
-for)
+for#for_prep)
@@ -194 +194 @@
-my)
+my#my_adj)
@@ -196 +196 @@
-deare)
+deare#dear_adj1)
@@ -198 +198 @@
-fathers))
+fathers#father_n))
@@ -200 +200 @@
-sake)))
+sake#sake_n1)))
@@ -203 +203 @@
-I))
+I#i_pron))
@@ -205 +205 @@
-alsoe)
+alsoe#also_adv)
@@ -208 +208 @@
-humbly))
+humbly#humbly_adv))
@@ -210 +210 @@
-crave)
+crave#crave_v)
@@ -214 +214 @@
-yours))
+yours#yours_pron))
@@ -217 +217 @@
-and)
+and#and_conj1)
@@ -221 +221 @@
-my)
+my#my_adj)
@@ -223 +223 @@
-brothers))
+brothers#brother_n))
@@ -225 +225 @@
-helpe))))
+helpe#help_n))))
@@ -228 +228 @@
-at)
+at#at_prep)
@@ -231 +231 @@
-this)
+this#this_pron)
@@ -233 +233 @@
-tyme))))
+tyme#time_n))))
@@ -239 +239 @@
-or)
+or#or_conj2)
@@ -241 +241 @@
-else)
+else#else_adv)
@@ -244 +244 @@
-I))
+I#i_pron))
@@ -246 +246 @@
-am)
+am#be_v)
@@ -249 +249 @@
-for)
+for#for_prep)
@@ -252 +252 @@
-euer)))
+euer#ever_adv)))
@@ -254 +254 @@
-lost)
+lost#lose_v1)
@@ -263 +263 @@
-truly))
+truly#truly_adv))
@@ -266 +266 @@
-I))
+I#i_pron))
@@ -268 +268 @@
-am)
+am#be_v)
@@ -271 +271 @@
-soe)
+soe#so_adv)
@@ -273 +273 @@
-possessed)
+possessed#possessed_adj)
@@ -276 +276 @@
-w=th=)
+w=th=#with_prep)
@@ -279 +279 @@
-extremity)
+extremity#extremity_n)
@@ -282 +282 @@
-of)
+of#of_prep)
@@ -285 +285 @@
-greife)))))
+greife#grief_n)))))
@@ -288 +288 @@
-y=t=)
+y=t=#that_conj)
@@ -292 +292 @@
-I))
+I#i_pron))
@@ -294 +294 @@
-knowe)
+knowe#know_v)
@@ -296 +296 @@
-not)
+not#not_adv)
@@ -302 +302 @@
-what))
+what#what_pron))
@@ -309 +309 @@
-to)
+to#to_prep)
@@ -311 +311 @@
-write))))))
+write#write_v))))))
@@ -319 +319 @@
-but)
+but#but_prep)
@@ -322 +322 @@
-my)
+my#my_adj)
@@ -324 +324 @@
-missery))
+missery#misery_n))
@@ -326 +326 @@
-makes)
+makes#make_v1)
@@ -330 +330 @@
-me))
+me#me_pron1))
@@ -333 +333 @@
-capable)
+capable#capable_adj)
@@ -336 +336 @@
-to)
+to#to_prep)
@@ -338 +338 @@
-desier)
+desier#desire_v)
@@ -341 +341 @@
-helpe)
+helpe#help_n)
@@ -343 +343 @@
-and)
+and#and_conj1)
@@ -345 +345 @@
-Realese)))))
+Realese#release_n)))))
@@ -354 +354 @@
-this))
+this#this_pron))
@@ -356 +356 @@
-is)
+is#be_v)
@@ -359 +359 @@
-the)
+the#the_adj)
@@ -361 +361 @@
-first)
+first#first_adj)
@@ -363 +363 @@
-tyme)
+tyme#time_n)
@@ -368 +368 @@
-that)
+that#that_conj)
@@ -374 +374 @@
-I))
+I#i_pron))
@@ -376 +376 @@
-am)
+am#be_v)
@@ -379 +379 @@
-vnfortunately))
+vnfortunately#unfortunately_adv))
@@ -381 +381 @@
-falen)
+falen#fall_v)
@@ -384 +384 @@
-Into)
+Into#into_prep)
@@ -387 +387 @@
-the)
+the#the_adj)
@@ -389 +389 @@
-handes)
+handes#hand_n)
@@ -392 +392 @@
-of)
+of#of_prep)
@@ -395 +395 @@
-a)
+a#a_adj2)
@@ -397 +397 @@
-Credittor)
+Credittor#creditor_n)
@@ -400 +400 @@
-One)
+One#one_adj)
@@ -402 +402 @@
-Thomas)
+Thomas#NA)
@@ -404 +404 @@
-Sampson))))))))))
+Sampson#NA))))))))))
@@ -411 +411 @@
-the)
+the#the_adj)
@@ -413 +413 @@
-debt))
+debt#debt_n))
@@ -415 +415 @@
-is)
+is#be_v)
@@ -419 +419 @@
-about)
+about#about_adv)
@@ -421 +421 @@
-fifty))
+fifty#fifty_adj))
@@ -423 +423 @@
-poundes))
+poundes#pound_n1))
@@ -431 +431 @@
-and)
+and#and_conj1)
@@ -434 +434 @@
-I))
+I#i_pron))
@@ -436 +436 @@
-can@)
+can@#can_v1)
@@ -438 +438 @@
-@not)
+@not#not_adv)
@@ -440 +440 @@
-but)
+but#but_prep)
@@ -442 +442 @@
-hope)
+hope#hope_v)
@@ -445 +445 @@
-that)
+that#that_conj)
@@ -449 +449 @@
-hee))
+hee#he_pron))
@@ -451 +451 @@
-wil@)
+wil@#will_v1)
@@ -453 +453 @@
-@be)
+@be#be_v)
@@ -456 +456 @@
-Reasonable))))
+Reasonable#reasonable_adj))))
@@ -465 +465 @@
-The)
+The#the_adj)
@@ -467 +467 @@
-baylifs))
+baylifs#bailiff_n))
@@ -470 +470 @@
-swere)
+swere#swear_v)
@@ -472 +472 @@
-and)
+and#and_conj1)
@@ -474 +474 @@
-damne))
+damne#damn_v))
@@ -477 +477 @@
-themselves))
+themselves#themselves_pron))
@@ -480 +480 @@
-y=t=)
+y=t=#that_conj)
@@ -484 +484 @@
-yf)
+yf#if_conj)
@@ -491 +491 @@
-I))
+I#i_pron))
@@ -493 +493 @@
-cann)
+cann#can_v1)
@@ -495 +495 @@
-satisfy)
+satisfy#satisfy_v)
@@ -498 +498 @@
-him)))))
+him#him_pron)))))
@@ -501 +501 @@
-noe)
+noe#no_adj)
@@ -503 +503 @@
-other)
+other#other_adj)
@@ -505 +505 @@
-acction))
+acction#action_n))
@@ -507 +507 @@
-shall)
+shall#shall_v)
@@ -509 +509 @@
-come)
+come#come_v)
@@ -512 +512 @@
-against)
+against#against_prep)
@@ -515 +515 @@
-me))))))
+me#me_pron1))))))
@@ -521 +521 @@
-but)
+but#but_prep)
@@ -524 +524 @@
-I))
+I#i_pron))
@@ -526 +526 @@
-shall)
+shall#shall_v)
@@ -528 +528 @@
-bee)
+bee#be_v)
@@ -530 +530 @@
-delivered)
+delivered#deliver_v1)
@@ -533 +533 @@
-safly))
+safly#safely_adv))
@@ -536 +536 @@
-at)
+at#at_prep)
@@ -539 +539 @@
-my)
+my#my_adj)
@@ -541 +541 @@
-lodging)))
+lodging#lodging_n)))
@@ -551 +551 @@
-I))
+I#i_pron))
@@ -553 +553 @@
-doe)
+doe#do_v)
@@ -555 +555 @@
-beseech)
+beseech#beseech_v)
@@ -558 +558 @@
-you)))
+you#you_pron)))
@@ -561 +561 @@
-Deare)
+Deare#dear_adj1)
@@ -563 +563 @@
-Mother))
+Mother#mother_n1))
@@ -565 +565 @@
-lett)
+lett#let_v1)
@@ -570 +570 @@
-me))
+me#me_pron1))
@@ -572 +572 @@
-not)
+not#not_adv)
@@ -574 +574 @@
-bee)
+bee#be_v)
@@ -577 +577 @@
-a)
+a#a_adj2)
@@ -579 +579 @@
-Newgate)
+Newgate#NA)
@@ -581 +581 @@
-bird)))
+bird#bird_n)))
@@ -584 +584 @@
-and)
+and#and_conj1)
@@ -588 +588 @@
-soe))
+soe#so_adv))
@@ -591 +591 @@
-for)
+for#for_prep)
@@ -594 +594 @@
-ever)))
+ever#ever_adv)))
@@ -596 +596 @@
-to)
+to#to_prep)
@@ -598 +598 @@
-loose)
+loose#lose_v1)
@@ -601 +601 @@
-my)
+my#my_adj)
@@ -603 +603 @@
-liberty)
+liberty#liberty_n)
@@ -606 +606 @@
-and)
+and#and_conj1)
@@ -609 +609 @@
-fortunes)))))))
+fortunes#fortune_n)))))))
@@ -617 +617 @@
-Your)
+Your#your_pron)
@@ -619 +619 @@
-distresfull)
+distresfull#distressful_adj)
@@ -621 +621 @@
-pore)
+pore#poor_adj)
@@ -623 +623 @@
-Sonne)
+Sonne#son_n1)
@@ -626 +626 @@
-Richard)
+Richard#NA)
@@ -628 +628 @@
-Farrar)))
+Farrar#NA)))
