--- rhaddsr-1673-e3-h.psd.vert	2025-09-10 13:50:35
+++ rhaddsr-1673-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-R.)
+R.#NA)
@@ -15 +15 @@
-James)
+James#NA)
@@ -23 +23 @@
-in)
+in#in_prep)
@@ -26 +26 @@
-Southold)
+Southold#NA)
@@ -28 +28 @@
-Bay))
+Bay#NA))
@@ -36 +36 @@
-this)
+this#this_pron)
@@ -38 +38 @@
-25th)
+25th#NA)
@@ -40 +40 @@
-May)
+May#NA)
@@ -44 +44 @@
-1672)
+1672#NA)
@@ -52 +52 @@
-Saturday)
+Saturday#NA)
@@ -54 +54 @@
-evening)
+evening#evening_n1)
@@ -63 +63 @@
-My)
+My#my_adj)
@@ -65 +65 @@
-dearest)
+dearest#dear_adj1)
@@ -67 +67 @@
-love))
+love#love_n1))
@@ -72 +72 @@
-These))
+These#these_pron))
@@ -75 +75 @@
-I))
+I#i_pron))
@@ -77 +77 @@
-send)
+send#send_v1)
@@ -80 +80 @@
-by)
+by#by_prep)
@@ -83 +83 @@
-Capt.)
+Capt.#NA)
@@ -85 +85 @@
-Poole)
+Poole#NA)
@@ -91 +91 @@
-who))
+who#who_pron))
@@ -99 +99 @@
-w=th=)
+w=th=#with_prep)
@@ -102 +102 @@
-the)
+the#the_adj)
@@ -104 +104 @@
-Garland)
+Garland#NA)
@@ -106 +106 @@
-frig=tt=)
+frig=tt=#NA)
@@ -109 +109 @@
-in)
+in#in_prep)
@@ -112 +112 @@
-comp=a=)))))
+comp=a=#comparison_n)))))
@@ -116 +116 @@
-hath)
+hath#have_v)
@@ -119 +119 @@
-leave))
+leave#leave_n1))
@@ -122 +122 @@
-to)
+to#to_prep)
@@ -124 +124 @@
-goe)
+goe#go_v)
@@ -127 +127 @@
-his)
+his#his_adj)
@@ -129 +129 @@
-former)
+former#former_adj)
@@ -131 +131 @@
-intended)
+intended#intended_adj)
@@ -133 +133 @@
-voaidge)
+voaidge#voyage_n)
@@ -136 +136 @@
-for)
+for#for_prep)
@@ -139 +139 @@
-Barbados)))))))))
+Barbados#NA)))))))))
@@ -150 +150 @@
-Pray))
+Pray#pray_v))
@@ -152 +152 @@
-advize)
+advize#advise_v)
@@ -155 +155 @@
-my)
+my#my_adj)
@@ -157 +157 @@
-bro=r=)
+bro=r=#brother_n)
@@ -160 +160 @@
-Bradenham)))
+Bradenham#NA)))
@@ -163 +163 @@
-of)
+of#of_prep)
@@ -166 +166 @@
-itt)))
+itt#it_pron)))
@@ -175 +175 @@
-They))
+They#they_pron))
@@ -177 +177 @@
-will)
+will#will_v1)
@@ -179 +179 @@
-be)
+be#be_v)
@@ -182 +182 @@
-good)
+good#good_adj)
@@ -184 +184 @@
-convoy)
+convoy#convoy_n)
@@ -187 +187 @@
-for)
+for#for_prep)
@@ -191 +191 @@
-Mr.)
+Mr.#NA)
@@ -193 +193 @@
-Naufan's))
+Naufan's#NA))
@@ -195 +195 @@
-ship)
+ship#ship_n1)
@@ -198 +198 @@
-w=th=)
+w=th=#with_prep)
@@ -201 +201 @@
-masts)
+masts#mast_n1)
@@ -205 +205 @@
-&c.)))
+&c.#et-cetera_n)))
@@ -211 +211 @@
-w=ch=))
+w=ch=#which_adj))
@@ -218 +218 @@
-lyes)
+lyes#lie_v1)
@@ -221 +221 @@
-at)
+at#at_prep)
@@ -224 +224 @@
-Gravesend)))
+Gravesend#NA)))
@@ -229 +229 @@
-to)
+to#to_prep)
@@ -231 +231 @@
-goe)
+goe#go_v)
@@ -234 +234 @@
-thorough)
+thorough#through_prep)
@@ -237 +237 @@
-the)
+the#the_adj)
@@ -239 +239 @@
-Channell)))
+Channell#NA)))
@@ -242 +242 @@
-w=th=)
+w=th=#with_prep)
@@ -245 +245 @@
-them)))))))))
+them#them_pron)))))))))
@@ -257 +257 @@
-This)
+This#this_pron)
@@ -259 +259 @@
-day))
+day#day_n))
@@ -262 +262 @@
-I))
+I#i_pron))
@@ -264 +264 @@
-gote)
+gote#get_v)
@@ -267 +267 @@
-two)
+two#two_adj)
@@ -269 +269 @@
-protections)
+protections#protection_n)
@@ -272 +272 @@
-from)
+from#from_prep)
@@ -275 +275 @@
-the)
+the#the_adj)
@@ -277 +277 @@
-Duke)))
+Duke#duke_n)))
@@ -283 +283 @@
-one)
+one#one_adj)
@@ -286 +286 @@
-for)
+for#for_prep)
@@ -290 +290 @@
-Mr.)
+Mr.#NA)
@@ -292 +292 @@
-Naufan's))
+Naufan's#NA))
@@ -294 +294 @@
-ship)
+ship#ship_n1)
@@ -297 +297 @@
-at)
+at#at_prep)
@@ -300 +300 @@
-Lancaster)))
+Lancaster#NA)))
@@ -303 +303 @@
-for)
+for#for_prep)
@@ -306 +306 @@
-50)
+50#fifty_adj)
@@ -308 +308 @@
-men))))))
+men#man_n1))))))
@@ -314 +314 @@
-y=e=)
+y=e=#the_adj)
@@ -316 +316 @@
-other)
+other#other_adj)
@@ -319 +319 @@
-for)
+for#for_prep)
@@ -322 +322 @@
-y=e=)
+y=e=#the_adj)
@@ -324 +324 @@
-ship)
+ship#ship_n1)
@@ -327 +327 @@
-w=th=)
+w=th=#with_prep)
@@ -330 +330 @@
-stores)
+stores#store_n)
@@ -333 +333 @@
-for)
+for#for_prep)
@@ -336 +336 @@
-20)
+20#twenty_adj)
@@ -338 +338 @@
-men))))))))))
+men#man_n1))))))))))
@@ -344 +344 @@
-w=ch=))
+w=ch=#which_adj))
@@ -351 +351 @@
-is)
+is#be_v)
@@ -354 +354 @@
-speciall)
+speciall#special_adj)
@@ -356 +356 @@
-protections)
+protections#protection_n)
@@ -361 +361 @@
-y=t=)
+y=t=#that_conj)
@@ -365 +365 @@
-the)
+the#the_adj)
@@ -367 +367 @@
-men))
+men#man_n1))
@@ -369 +369 @@
-will)
+will#will_v1)
@@ -371 +371 @@
-not)
+not#not_adv)
@@ -373 +373 @@
-be)
+be#be_v)
@@ -375 +375 @@
-molested)))))))
+molested#molest_v)))))))
@@ -383 +383 @@
-Tell)
+Tell#tell_v)
@@ -386 +386 @@
-my)
+my#my_adj)
@@ -388 +388 @@
-Bro=r=)
+Bro=r=#brother_n)
@@ -391 +391 @@
-Bradenham)))
+Bradenham#NA)))
@@ -398 +398 @@
-I))
+I#i_pron))
@@ -400 +400 @@
-have)
+have#have_v)
@@ -402 +402 @@
-given)
+given#give_v)
@@ -405 +405 @@
-them))
+them#them_pron))
@@ -408 +408 @@
-to)
+to#to_prep)
@@ -411 +411 @@
-Mr.)
+Mr.#NA)
@@ -413 +413 @@
-Sam.)
+Sam.#NA)
@@ -415 +415 @@
-Hawkes)
+Hawkes#NA)
@@ -421 +421 @@
-who))
+who#who_pron))
@@ -428 +428 @@
-is)
+is#be_v)
@@ -430 +430 @@
-comeing)
+comeing#come_v)
@@ -433 +433 @@
-w=th=)
+w=th=#with_prep)
@@ -436 +436 @@
-them)))))))))
+them#them_pron)))))))))
@@ -445 +445 @@
-I))
+I#i_pron))
@@ -447 +447 @@
-am)
+am#be_v)
@@ -450 +450 @@
-sory)
+sory#sorry_adj)
@@ -453 +453 @@
-to)
+to#to_prep)
@@ -455 +455 @@
-heare)
+heare#hear_v)
@@ -463 +463 @@
-poore)
+poore#poor_adj)
@@ -465 +465 @@
-Sam)
+Sam#NA)
@@ -467 +467 @@
-Lane))
+Lane#NA))
@@ -469 +469 @@
-was)
+was#be_v)
@@ -471 +471 @@
-prest)
+prest#press_v2)
@@ -474 +474 @@
-into)
+into#into_prep)
@@ -477 +477 @@
-y=e=)
+y=e=#the_adj)
@@ -479 +479 @@
-French)
+French#french_adj)
@@ -481 +481 @@
-Victory))))
+Victory#victory_n))))
@@ -486 +486 @@
-and)
+and#and_conj1)
@@ -490 +490 @@
-since))
+since#since_adv))
@@ -492 +492 @@
-caryed)
+caryed#carry_v)
@@ -495 +495 @@
-into)
+into#into_prep)
@@ -498 +498 @@
-Holland)))))))))
+Holland#NA)))))))))
@@ -507 +507 @@
-I))
+I#i_pron))
@@ -509 +509 @@
-pitty)
+pitty#pity_v)
@@ -512 +512 @@
-the)
+the#the_adj)
@@ -514 +514 @@
-losse)
+losse#loss_n1)
@@ -517 +517 @@
-of)
+of#of_prep)
@@ -520 +520 @@
-the)
+the#the_adj)
@@ -522 +522 @@
-men)
+men#man_n1)
@@ -525 +525 @@
-in)
+in#in_prep)
@@ -528 +528 @@
-her))))))
+her#her_pron1))))))
@@ -536 +536 @@
-but)
+but#but_prep)
@@ -539 +539 @@
-y=e=)
+y=e=#the_adj)
@@ -541 +541 @@
-Capt.))
+Capt.#captain_n))
@@ -543 +543 @@
-will)
+will#will_v1)
@@ -545 +545 @@
-have)
+have#have_v)
@@ -548 +548 @@
-his)
+his#his_adj)
@@ -550 +550 @@
-reward))
+reward#reward_n))
@@ -553 +553 @@
-for)
+for#for_prep)
@@ -556 +556 @@
-looseing)
+looseing#lose_v1)
@@ -559 +559 @@
-her))
+her#her_pron1))
@@ -562 +562 @@
-soe)
+soe#so_adv)
@@ -564 +564 @@
-basely))))
+basely#basely_adv))))
@@ -573 +573 @@
-The)
+The#the_adj)
@@ -575 +575 @@
-Dutch)
+Dutch#dutch_adj)
@@ -577 +577 @@
-fleet))
+fleet#fleet_n1))
@@ -579 +579 @@
-lye)
+lye#lie_v1)
@@ -582 +582 @@
-now))
+now#now_adv))
@@ -585 +585 @@
-neare)
+neare#near_adj)
@@ -588 +588 @@
-the)
+the#the_adj)
@@ -590 +590 @@
-Gallaper)))
+Gallaper#NA)))
@@ -593 +593 @@
-in)
+in#in_prep)
@@ -596 +596 @@
-expectation)
+expectation#expectation_n)
@@ -599 +599 @@
-of)
+of#of_prep)
@@ -602 +602 @@
-us)))))
+us#us_pron)))))
@@ -611 +611 @@
-we))
+we#we_pron))
@@ -613 +613 @@
-are)
+are#be_v)
@@ -616 +616 @@
-very)
+very#very_adj)
@@ -618 +618 @@
-neere))
+neere#near_adj))
@@ -623 +623 @@
-ready)
+ready#ready_adj)
@@ -626 +626 @@
-to)
+to#to_prep)
@@ -628 +628 @@
-waite)
+waite#wait_v1)
@@ -631 +631 @@
-on)
+on#on_prep)
@@ -634 +634 @@
-them)))))
+them#them_pron)))))
@@ -644 +644 @@
-2)
+2#two_adj)
@@ -646 +646 @@
-or)
+or#or_conj2)
@@ -648 +648 @@
-3))
+3#three_adj))
@@ -650 +650 @@
-days))
+days#day_n))
@@ -652 +652 @@
-must)
+must#must_v1)
@@ -654 +654 @@
-not)
+not#not_adv)
@@ -656 +656 @@
-breake)
+breake#break_v)
@@ -659 +659 @@
-square))
+square#square_n))
@@ -662 +662 @@
-w=th=)
+w=th=#with_prep)
@@ -665 +665 @@
-us)))
+us#us_pron)))
@@ -673 +673 @@
-but)
+but#but_prep)
@@ -676 +676 @@
-they))
+they#they_pron))
@@ -678 +678 @@
-are)
+are#be_v)
@@ -680 +680 @@
-deceaved)
+deceaved#deceive_v)
@@ -683 +683 @@
-to)
+to#to_prep)
@@ -685 +685 @@
-think)
+think#think_v2)
@@ -692 +692 @@
-we))
+we#we_pron))
@@ -694 +694 @@
-intend)
+intend#intend_v)
@@ -697 +697 @@
-to)
+to#to_prep)
@@ -699 +699 @@
-fight)
+fight#fight_v)
@@ -702 +702 @@
-them))
+them#them_pron))
@@ -705 +705 @@
-amongst)
+amongst#amongst_prep)
@@ -708 +708 @@
-the)
+the#the_adj)
@@ -710 +710 @@
-sands)))))))
+sands#sand_n2)))))))
@@ -719 +719 @@
-I))
+I#i_pron))
@@ -721 +721 @@
-supose)
+supose#suppose_v)
@@ -728 +728 @@
-our)
+our#our_pron)
@@ -730 +730 @@
-martch))
+martch#march_n4))
@@ -732 +732 @@
-wil@)
+wil@#will_v1)
@@ -734 +734 @@
-@be)
+@be#be_v)
@@ -736 +736 @@
-over)
+over#over_adv)
@@ -739 +739 @@
-for)
+for#for_prep)
@@ -742 +742 @@
-y=e=)
+y=e=#the_adj)
@@ -744 +744 @@
-coast)
+coast#coast_n)
@@ -747 +747 @@
-of)
+of#of_prep)
@@ -750 +750 @@
-Holland)))))
+Holland#NA)))))
@@ -753 +753 @@
-into)
+into#into_prep)
@@ -757 +757 @@
-sea)
+sea#sea_n)
@@ -759 +759 @@
-roome))
+roome#room_n1))
@@ -762 +762 @@
-and)
+and#and_conj1)
@@ -765 +765 @@
-deepe)
+deepe#deep_adj)
@@ -767 +767 @@
-watter)))))))
+watter#water_n)))))))
@@ -776 +776 @@
-We))
+We#we_pron))
@@ -778 +778 @@
-are)
+are#be_v)
@@ -783 +783 @@
-notwithstanding)
+notwithstanding#notwithstanding_prep)
@@ -788 +788 @@
-Capt.)
+Capt.#NA)
@@ -790 +790 @@
-Poole))
+Poole#NA))
@@ -793 +793 @@
-and)
+and#and_conj1)
@@ -796 +796 @@
-his)
+his#his_adj)
@@ -798 +798 @@
-consarts))))
+consarts#consort_n1))))
@@ -800 +800 @@
-leaveing)
+leaveing#leave_v1)
@@ -803 +803 @@
-us))))
+us#us_pron))))
@@ -809 +809 @@
-90)
+90#ninety_adj)
@@ -811 +811 @@
-men)
+men#man_n1)
@@ -814 +814 @@
-of)
+of#of_prep)
@@ -817 +817 @@
-warr))))
+warr#war_n1))))
@@ -823 +823 @@
-26)
+26#NA)
@@ -825 +825 @@
-fire)
+fire#fire_n)
@@ -827 +827 @@
-ships)))
+ships#ship_n1)))
@@ -833 +833 @@
-many)
+many#many_adj)
@@ -835 +835 @@
-small)
+small#small_adj)
@@ -837 +837 @@
-vessells))))
+vessells#vessel_n1))))
@@ -846 +846 @@
-I))
+I#i_pron))
@@ -848 +848 @@
-supose)
+supose#suppose_v)
@@ -855 +855 @@
-the)
+the#the_adj)
@@ -857 +857 @@
-Dutch))
+Dutch#dutch_adj))
@@ -860 +860 @@
-daylie))
+daylie#daily_adv))
@@ -862 +862 @@
-add)
+add#add_v)
@@ -865 +865 @@
-to)
+to#to_prep)
@@ -868 +868 @@
-ther)
+ther#their_adj)
@@ -870 +870 @@
-strength)))
+strength#strength_n)))
@@ -873 +873 @@
-as)
+as#as_adv)
@@ -875 +875 @@
-well)
+well#well_adv)
@@ -878 +878 @@
-as)
+as#as_adv)
@@ -881 +881 @@
-wee))))))
+wee#we_pron))))))
@@ -890 +890 @@
-God)
+God#NA)
@@ -893 +893 @@
-Allmighty)))
+Allmighty#almighty_adj)))
@@ -895 +895 @@
-be)
+be#be_v)
@@ -898 +898 @@
-our)
+our#our_pron)
@@ -900 +900 @@
-defence))
+defence#defence_n))
@@ -909 +909 @@
-My)
+My#my_adj)
@@ -911 +911 @@
-deare))
+deare#dear_adj1))
@@ -916 +916 @@
-I))
+I#i_pron))
@@ -918 +918 @@
-am)
+am#be_v)
@@ -921 +921 @@
-sory)
+sory#sorry_adj)
@@ -924 +924 @@
-that)
+that#that_conj)
@@ -928 +928 @@
-my)
+my#my_adj)
@@ -930 +930 @@
-first)
+first#first_adj)
@@ -932 +932 @@
-letter)
+letter#letter_n1)
@@ -935 +935 @@
-from)
+from#from_prep)
@@ -938 +938 @@
-Southold)))
+Southold#NA)))
@@ -944 +944 @@
-w=ch=))
+w=ch=#which_adj))
@@ -951 +951 @@
-went)
+went#go_v)
@@ -954 +954 @@
-by)
+by#by_prep)
@@ -957 +957 @@
-land))))))
+land#land_n1))))))
@@ -961 +961 @@
-advized)
+advized#advise_v)
@@ -964 +964 @@
-the))
+the#thee_pron))
@@ -967 +967 @@
-of)
+of#of_prep)
@@ -970 +970 @@
-our)
+our#our_pron)
@@ -973 +973 @@
-2)
+2#two_adj)
@@ -975 +975 @@
-days))
+days#day_n))
@@ -977 +977 @@
-stay)))
+stay#stay_n3)))
@@ -982 +982 @@
-whereas)
+whereas#whereas_adv)
@@ -990 +990 @@
-we))
+we#we_pron))
@@ -992 +992 @@
-have)
+have#have_v)
@@ -994 +994 @@
-bine)
+bine#be_v)
@@ -997 +997 @@
-heere))
+heere#here_adv))
@@ -1000 +1000 @@
-4)
+4#four_adj)
@@ -1002 +1002 @@
-days)))
+days#day_n)))
@@ -1007 +1007 @@
-and)
+and#and_conj1)
@@ -1012 +1012 @@
-shall)
+shall#shall_v)
@@ -1014 +1014 @@
-stay)
+stay#stay_v1)
@@ -1020 +1020 @@
-3)
+3#three_adj)
@@ -1022 +1022 @@
-or)
+or#or_conj2)
@@ -1024 +1024 @@
-4)))
+4#four_adj)))
@@ -1026 +1026 @@
-longer)))))))))))
+longer#long_adj1)))))))))))
@@ -1036 +1036 @@
-Then))
+Then#then_adv))
@@ -1039 +1039 @@
-we))
+we#we_pron))
@@ -1041 +1041 @@
-shall)
+shall#shall_v)
@@ -1043 +1043 @@
-have)
+have#have_v)
@@ -1045 +1045 @@
-wattered)
+wattered#water_v)
@@ -1048 +1048 @@
-our)
+our#our_pron)
@@ -1050 +1050 @@
-whole)
+whole#whole_adj)
@@ -1052 +1052 @@
-fleet))
+fleet#fleet_n1))
@@ -1055 +1055 @@
-for)
+for#for_prep)
@@ -1058 +1058 @@
-one)
+one#one_adj)
@@ -1060 +1060 @@
-month))))
+month#month_n1))))
@@ -1065 +1065 @@
-and)
+and#and_conj1)
@@ -1068 +1068 @@
-victualld)
+victualld#victual_v)
@@ -1071 +1071 @@
-compleat))
+compleat#complete_adj))
@@ -1074 +1074 @@
-for)
+for#for_prep)
@@ -1077 +1077 @@
-2)
+2#two_adj)
@@ -1079 +1079 @@
-months)))))
+months#month_n1)))))
@@ -1084 +1084 @@
-and)
+and#and_conj1)
@@ -1087 +1087 @@
-mand)
+mand#man_v)
@@ -1091 +1091 @@
-I))
+I#i_pron))
@@ -1093 +1093 @@
-beleive))
+beleive#believe_v))
@@ -1096 +1096 @@
-thoroughout)
+thoroughout#throughout_prep)
@@ -1099 +1099 @@
-y=e=)
+y=e=#the_adj)
@@ -1101 +1101 @@
-fleet)))
+fleet#fleet_n1)))
@@ -1107 +1107 @@
-not)
+not#not_adv)
@@ -1109 +1109 @@
-500))
+500#NA))
@@ -1111 +1111 @@
-wanting))))
+wanting#want_v))))
@@ -1120 +1120 @@
-I))
+I#i_pron))
@@ -1123 +1123 @@
-rather))
+rather#rather_adv))
@@ -1125 +1125 @@
-think)
+think#think_v2)
@@ -1136 +1136 @@
-in)
+in#in_prep)
@@ -1140 +1140 @@
-a)
+a#a_adj2)
@@ -1142 +1142 @@
-day))
+day#day_n))
@@ -1145 +1145 @@
-or)
+or#or_conj2)
@@ -1148 +1148 @@
-two)))))
+two#two_adj)))))
@@ -1150 +1150 @@
-longer)))
+longer#long_adj1)))
@@ -1155 +1155 @@
-we))
+we#we_pron))
@@ -1157 +1157 @@
-may)
+may#may_v1)
@@ -1159 +1159 @@
-have)
+have#have_v)
@@ -1162 +1162 @@
-1000)
+1000#NA)
@@ -1164 +1164 @@
-supernumeraryes))))
+supernumeraryes#supernumerary_adj))))
@@ -1172 +1172 @@
-A)
+A#a_adj2)
@@ -1175 +1175 @@
-very)
+very#very_adj)
@@ -1177 +1177 @@
-worthy))
+worthy#worthy_adj))
@@ -1179 +1179 @@
-brave)
+brave#brave_adj)
@@ -1181 +1181 @@
-fleet)
+fleet#fleet_n1)
@@ -1187 +1187 @@
-I))
+I#i_pron))
@@ -1189 +1189 @@
-think))
+think#think_v2))
@@ -1194 +1194 @@
-as)
+as#as_adv)
@@ -1205 +1205 @@
-ever))
+ever#ever_adv))
@@ -1207 +1207 @@
-were)
+were#be_v)
@@ -1210 +1210 @@
-together)))))
+together#together_adv)))))
@@ -1219 +1219 @@
-God))
+God#NA))
@@ -1221 +1221 @@
-give)
+give#give_v)
@@ -1224 +1224 @@
-us))
+us#us_pron))
@@ -1227 +1227 @@
-couragious)
+couragious#courageous_adj)
@@ -1229 +1229 @@
-hearts))
+hearts#heart_n))
@@ -1237 +1237 @@
-and)
+and#and_conj1)
@@ -1240 +1240 @@
-then))
+then#then_adv))
@@ -1243 +1243 @@
-I))
+I#i_pron))
@@ -1245 +1245 @@
-beleive)
+beleive#believe_v)
@@ -1252 +1252 @@
-they))
+they#they_pron))
@@ -1254 +1254 @@
-may)
+may#may_v1)
@@ -1256 +1256 @@
-be)
+be#be_v)
@@ -1258 +1258 @@
-ventured)))
+ventured#venture_v)))
@@ -1267 +1267 @@
-I))
+I#i_pron))
@@ -1269 +1269 @@
-hope)
+hope#hope_v)
@@ -1276 +1276 @@
-all)
+all#all_adj)
@@ -1278 +1278 @@
-my)
+my#my_adj)
@@ -1280 +1280 @@
-loveing)
+loveing#loving_adj)
@@ -1282 +1282 @@
-relations)
+relations#relation_n)
@@ -1285 +1285 @@
-at)
+at#at_prep)
@@ -1288 +1288 @@
-Rederif)
+Rederif#NA)
@@ -1290 +1290 @@
-and)
+and#and_conj1)
@@ -1292 +1292 @@
-Wapping)))
+Wapping#NA)))
@@ -1296 +1296 @@
-are)
+are#be_v)
@@ -1299 +1299 @@
-in)
+in#in_prep)
@@ -1302 +1302 @@
-health)))
+health#health_n)))
@@ -1308 +1308 @@
-to)
+to#to_prep)
@@ -1311 +1311 @@
-whome)))
+whome#whom_pron)))
@@ -1316 +1316 @@
-present)
+present#present_v)
@@ -1319 +1319 @@
-my)
+my#my_adj)
@@ -1321 +1321 @@
-love)
+love#love_n1)
@@ -1324 +1324 @@
-and)
+and#and_conj1)
@@ -1327 +1327 @@
-saluts))))))))
+saluts#salute_n1))))))))
@@ -1336 +1336 @@
-I))
+I#i_pron))
@@ -1338 +1338 @@
-expect)
+expect#expect_v)
@@ -1341 +1341 @@
-to)
+to#to_prep)
@@ -1343 +1343 @@
-heare)
+heare#hear_v)
@@ -1346 +1346 @@
-from)
+from#from_prep)
@@ -1349 +1349 @@
-the)))
+the#thee_pron)))
@@ -1352 +1352 @@
-by)
+by#by_prep)
@@ -1355 +1355 @@
-the)
+the#the_adj)
@@ -1357 +1357 @@
-yacht)
+yacht#yacht_n)
@@ -1362 +1362 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -1368 +1368 @@
-I))
+I#i_pron))
@@ -1370 +1370 @@
-sent)
+sent#send_v1)
@@ -1373 +1373 @@
-my)
+my#my_adj)
@@ -1375 +1375 @@
-last)
+last#last_adv)
@@ -1377 +1377 @@
-letter))
+letter#letter_n1))
@@ -1380 +1380 @@
-by)
+by#by_prep)
@@ -1388 +1388 @@
-Captain)
+Captain#NA)
@@ -1390 +1390 @@
-Burstow))
+Burstow#NA))
@@ -1395 +1395 @@
-Comander))))))))
+Comander#commander_n))))))))
@@ -1404 +1404 @@
-I))
+I#i_pron))
@@ -1406 +1406 @@
-blesse)
+blesse#bless_v1)
@@ -1409 +1409 @@
-God))
+God#NA))
@@ -1416 +1416 @@
-I))
+I#i_pron))
@@ -1418 +1418 @@
-am)
+am#be_v)
@@ -1421 +1421 @@
-now))
+now#now_adv))
@@ -1424 +1424 @@
-in)
+in#in_prep)
@@ -1427 +1427 @@
-good)
+good#good_adj)
@@ -1429 +1429 @@
-health)))
+health#health_n)))
@@ -1434 +1434 @@
-though)
+though#though_adv)
@@ -1444 +1444 @@
-5)
+5#five_v)
@@ -1446 +1446 @@
-or)
+or#or_conj2)
@@ -1448 +1448 @@
-6))
+6#six_adj))
@@ -1450 +1450 @@
-days))
+days#day_n))
@@ -1452 +1452 @@
-since))
+since#since_adv))
@@ -1457 +1457 @@
-and)
+and#and_conj1)
@@ -1460 +1460 @@
-when)
+when#when_adv)
@@ -1467 +1467 @@
-we))
+we#we_pron))
@@ -1469 +1469 @@
-were)
+were#be_v)
@@ -1471 +1471 @@
-going)
+going#go_v)
@@ -1474 +1474 @@
-to)
+to#to_prep)
@@ -1476 +1476 @@
-fight)
+fight#fight_v)
@@ -1479 +1479 @@
-the)
+the#the_adj)
@@ -1481 +1481 @@
-Dutch))))))))
+Dutch#dutch_adj))))))))
@@ -1486 +1486 @@
-I))
+I#i_pron))
@@ -1488 +1488 @@
-had)
+had#have_v)
@@ -1491 +1491 @@
-such)
+such#such_adj)
@@ -1493 +1493 @@
-a)
+a#a_adj2)
@@ -1495 +1495 @@
-paine)
+paine#pain_n1)
@@ -1498 +1498 @@
-in)
+in#in_prep)
@@ -1501 +1501 @@
-my)
+my#my_adj)
@@ -1503 +1503 @@
-right)
+right#right_adj)
@@ -1505 +1505 @@
-arme)))
+arme#arm_n1)))
@@ -1508 +1508 @@
-that)
+that#that_conj)
@@ -1513 +1513 @@
-could)
+could#can_v1)
@@ -1515 +1515 @@
-not)
+not#not_adv)
@@ -1517 +1517 @@
-use)
+use#use_v)
@@ -1520 +1520 @@
-it))
+it#it_pron))
@@ -1522 +1522 @@
-but)
+but#but_prep)
@@ -1526 +1526 @@
-very)
+very#very_adj)
@@ -1528 +1528 @@
-litle)))))))))))
+litle#little_adj)))))))))))
@@ -1536 +1536 @@
-but)
+but#but_prep)
@@ -1541 +1541 @@
-now))
+now#now_adv))
@@ -1548 +1548 @@
-thanke)
+thanke#thank_v)
@@ -1551 +1551 @@
-God)))
+God#NA)))
@@ -1555 +1555 @@
-am)
+am#be_v)
@@ -1558 +1558 @@
-very)
+very#very_adj)
@@ -1560 +1560 @@
-well))
+well#well_adj))
@@ -1569 +1569 @@
-My)
+My#my_adj)
@@ -1571 +1571 @@
-deare)
+deare#dear_adj1)
@@ -1573 +1573 @@
-Betty))
+Betty#NA))
@@ -1578 +1578 @@
-I))
+I#i_pron))
@@ -1580 +1580 @@
-have)
+have#have_v)
@@ -1582 +1582 @@
-only)
+only#only_adj)
@@ -1585 +1585 @@
-to)
+to#to_prep)
@@ -1589 +1589 @@
-add)
+add#add_v)
@@ -1592 +1592 @@
-my)
+my#my_adj)
@@ -1594 +1594 @@
-saluts)
+saluts#salute_n1)
@@ -1597 +1597 @@
-to)
+to#to_prep)
@@ -1600 +1600 @@
-thyselfe)
+thyselfe#thyself_pron)
@@ -1603 +1603 @@
-w=th=)
+w=th=#with_prep)
@@ -1606 +1606 @@
-my)
+my#my_adj)
@@ -1608 +1608 @@
-daughter)))))))
+daughter#daughter_n)))))))
@@ -1618 +1618 @@
-doe)
+doe#do_v)
@@ -1620 +1620 @@
-remaine)
+remaine#remain_v)
@@ -1623 +1623 @@
-thyne)
+thyne#thine_adj)
@@ -1628 +1628 @@
-till)
+till#till_prep)
@@ -1635 +1635 @@
-death))
+death#death_n))
@@ -1638 +1638 @@
-us))
+us#us_pron))
@@ -1640 +1640 @@
-part))))
+part#part_v))))
@@ -1645 +1645 @@
-Richard)
+Richard#NA)
@@ -1647 +1647 @@
-Haddock)))
+Haddock#NA)))
@@ -1654 +1654 @@
-This))
+This#this_pron))
@@ -1657 +1657 @@
-I))
+I#i_pron))
@@ -1659 +1659 @@
-intended)
+intended#intend_v)
@@ -1662 +1662 @@
-thee))
+thee#thee_pron))
@@ -1665 +1665 @@
-by)
+by#by_prep)
@@ -1668 +1668 @@
-Capt.)
+Capt.#NA)
@@ -1670 +1670 @@
-Poole)))
+Poole#NA)))
@@ -1678 +1678 @@
-but)
+but#but_prep)
@@ -1682 +1682 @@
-was)
+was#be_v)
@@ -1684 +1684 @@
-gone)
+gone#go_v)
@@ -1687 +1687 @@
-ere)
+ere#ere_adv)
@@ -1694 +1694 @@
-I))
+I#i_pron))
@@ -1696 +1696 @@
-could)
+could#can_v1)
@@ -1698 +1698 @@
-put)
+put#put_v)
@@ -1701 +1701 @@
-it))
+it#it_pron))
@@ -1704 +1704 @@
-on)
+on#on_prep)
@@ -1707 +1707 @@
-bord)
+bord#board_n)
@@ -1710 +1710 @@
-him)))))))
+him#him_pron)))))))
@@ -1719 +1719 @@
-therfore))
+therfore#therefore_adv))
@@ -1723 +1723 @@
-doe)
+doe#do_v)
@@ -1725 +1725 @@
-send)
+send#send_v1)
@@ -1728 +1728 @@
-it))
+it#it_pron))
@@ -1731 +1731 @@
-by)
+by#by_prep)
@@ -1734 +1734 @@
-the)
+the#the_adj)
@@ -1736 +1736 @@
-post)))
+post#post_n2)))
@@ -1745 +1745 @@
-I))
+I#i_pron))
@@ -1747 +1747 @@
-rec=d=)
+rec=d=#receive_v)
@@ -1750 +1750 @@
-last)
+last#last_adv)
@@ -1752 +1752 @@
-night))
+night#night_n))
@@ -1756 +1756 @@
-bro=r=)
+bro=r=#NA)
@@ -1758 +1758 @@
-Thornbrugh's))
+Thornbrugh's#NA))
@@ -1760 +1760 @@
-letter)
+letter#letter_n1)
@@ -1765 +1765 @@
-23)
+23#NA)
@@ -1767 +1767 @@
-instant)))
+instant#instant_adj)))
@@ -1772 +1772 @@
-by)
+by#by_prep)
@@ -1776 +1776 @@
-y=e=)
+y=e=#the_adj)
@@ -1778 +1778 @@
-Dreadnought's))
+Dreadnought's#NA))
@@ -1780 +1780 @@
-Leiveten=t=)))
+Leiveten=t=#lieutenant_n)))
@@ -1789 +1789 @@
-He))
+He#he_pron))
@@ -1791 +1791 @@
-wrights)
+wrights#write_v)
@@ -1794 +1794 @@
-me))
+me#me_pron1))
@@ -1798 +1798 @@
-of)
+of#of_prep)
@@ -1801 +1801 @@
-thy)
+thy#thy_adj)
@@ -1803 +1803 @@
-health)))
+health#health_n)))
@@ -1808 +1808 @@
-and)
+and#and_conj1)
@@ -1811 +1811 @@
-y=t=)
+y=t=#that_conj)
@@ -1815 +1815 @@
-I))
+I#i_pron))
@@ -1817 +1817 @@
-shall)
+shall#shall_v)
@@ -1819 +1819 @@
-rec=e=)
+rec=e=#receive_v)
@@ -1822 +1822 @@
-a)
+a#a_adj2)
@@ -1824 +1824 @@
-letter)
+letter#letter_n1)
@@ -1827 +1827 @@
-from)
+from#from_prep)
@@ -1830 +1830 @@
-the))))
+the#thee_pron))))
@@ -1833 +1833 @@
-by)
+by#by_prep)
@@ -1836 +1836 @@
-y=e=)
+y=e=#the_adj)
@@ -1838 +1838 @@
-Hatton)
+Hatton#NA)
@@ -1840 +1840 @@
-ketch)))))))
+ketch#ketch_n1)))))))
@@ -1849 +1849 @@
-I)
+I#i_pron)
@@ -1853 +1853 @@
-am)
+am#be_v)
@@ -1856 +1856 @@
-thyne))
+thyne#thine_adj))
@@ -1861 +1861 @@
-R.)
+R.#NA)
@@ -1863 +1863 @@
-Haddock))
+Haddock#NA))
@@ -1871 +1871 @@
-26)
+26#NA)
@@ -1873 +1873 @@
-May)
+May#NA)
@@ -1877 +1877 @@
-'72)
+'72#NA)
@@ -1892 +1892 @@
-Yet))
+Yet#yet_adv))
@@ -1895 +1895 @@
-on)
+on#on_prep)
@@ -1898 +1898 @@
-bord)
+bord#board_n)
@@ -1901 +1901 @@
-y=e=)
+y=e=#the_adj)
@@ -1903 +1903 @@
-R.)
+R.#NA)
@@ -1905 +1905 @@
-Charles))))
+Charles#NA))))
@@ -1913 +1913 @@
-this)
+this#this_pron)
@@ -1915 +1915 @@
-31)
+31#NA)
@@ -1917 +1917 @@
-May)
+May#NA)
@@ -1921 +1921 @@
-1673)
+1673#NA)
@@ -1930 +1930 @@
-My)
+My#my_adj)
@@ -1932 +1932 @@
-deare)
+deare#dear_adj1)
@@ -1934 +1934 @@
-Betty))
+Betty#NA))
@@ -1939 +1939 @@
-I))
+I#i_pron))
@@ -1941 +1941 @@
-wrote)
+wrote#write_v)
@@ -1944 +1944 @@
-the))
+the#thee_pron))
@@ -1948 +1948 @@
-two)
+two#two_adj)
@@ -1950 +1950 @@
-dayes))
+dayes#day_n))
@@ -1952 +1952 @@
-since))
+since#since_adv))
@@ -1955 +1955 @@
-of)
+of#of_prep)
@@ -1958 +1958 @@
-God's)
+God's#NA)
@@ -1960 +1960 @@
-goodnes)
+goodnes#goodness_n)
@@ -1963 +1963 @@
-to)
+to#to_prep)
@@ -1966 +1966 @@
-mee)))
+mee#me_pron1)))
@@ -1969 +1969 @@
-in)
+in#in_prep)
@@ -1972,3 +1972,3 @@
-o=r=)
-(ADJ
-late)
+o=r=#our_pron)
+(ADJ
+late#late_adj1)
@@ -1976 +1976 @@
-bataile)))))
+bataile#battle_n)))))
@@ -1985 +1985 @@
-I))
+I#i_pron))
@@ -1987 +1987 @@
-gave)
+gave#give_v)
@@ -1990 +1990 @@
-the))
+the#thee_pron))
@@ -1993 +1993 @@
-acc=t=)
+acc=t=#account_n)
@@ -1996 +1996 @@
-of)
+of#of_prep)
@@ -2000 +2000 @@
-Capt.)
+Capt.#NA)
@@ -2002 +2002 @@
-Trevanions))
+Trevanions#NA))
@@ -2004 +2004 @@
-suposed)
+suposed#suppose_v)
@@ -2009 +2009 @@
-to)
+to#to_prep)
@@ -2011 +2011 @@
-be)
+be#be_v)
@@ -2013 +2013 @@
-killed)))))
+killed#kill_v)))))
@@ -2021 +2021 @@
-but)
+but#but_prep)
@@ -2024 +2024 @@
-he))
+he#he_pron))
@@ -2026 +2026 @@
-is)
+is#be_v)
@@ -2029 +2029 @@
-well))
+well#well_adj))
@@ -2037 +2037 @@
-and)
+and#and_conj1)
@@ -2039 +2039 @@
-allso)
+allso#also_adv)
@@ -2042 +2042 @@
-Capt.)
+Capt.#NA)
@@ -2044 +2044 @@
-Courtney)
+Courtney#NA)
@@ -2050 +2050 @@
-w=ch=))
+w=ch=#which_adj))
@@ -2057 +2057 @@
-was)
+was#be_v)
@@ -2059 +2059 @@
-reported)
+reported#report_v)
@@ -2064 +2064 @@
-to)
+to#to_prep)
@@ -2066 +2066 @@
-be)
+be#be_v)
@@ -2068 +2068 @@
-killed)))))
+killed#kill_v)))))
@@ -2072 +2072 @@
-is)
+is#be_v)
@@ -2076 +2076 @@
-alive))
+alive#alive_adj))
@@ -2079 +2079 @@
-and)
+and#and_conj1)
@@ -2082 +2082 @@
-well))))
+well#well_adj))))
@@ -2091 +2091 @@
-Capt.)
+Capt.#NA)
@@ -2093 +2093 @@
-Worden))
+Worden#NA))
@@ -2095 +2095 @@
-is)
+is#be_v)
@@ -2098 +2098 @@
-since))
+since#since_adv))
@@ -2101 +2101 @@
-dead)
+dead#dead_adj)
@@ -2104 +2104 @@
-of)
+of#of_prep)
@@ -2107 +2107 @@
-his)
+his#his_adj)
@@ -2109 +2109 @@
-wounds))))
+wounds#wound_n))))
@@ -2118 +2118 @@
-We))
+We#we_pron))
@@ -2120 +2120 @@
-are)
+are#be_v)
@@ -2123 +2123 @@
-now))
+now#now_adv))
@@ -2125 +2125 @@
-shifting)
+shifting#shift_v)
@@ -2128 +2128 @@
-ships))
+ships#ship_n1))
@@ -2133 +2133 @@
-goeing)
+goeing#go_v)
@@ -2136 +2136 @@
-on)
+on#on_prep)
@@ -2139 +2139 @@
-bord)
+bord#board_n)
@@ -2142 +2142 @@
-the)
+the#the_adj)
@@ -2144 +2144 @@
-London)))))
+London#NA)))))
@@ -2153 +2153 @@
-the)
+the#the_adj)
@@ -2155 +2155 @@
-reason))
+reason#reason_n1))
@@ -2158 +2158 @@
-I))
+I#i_pron))
@@ -2160 +2160 @@
-gave)
+gave#give_v)
@@ -2163 +2163 @@
-in)
+in#in_prep)
@@ -2166 +2166 @@
-my)
+my#my_adj)
@@ -2168 +2168 @@
-last)))
+last#last_adv)))
@@ -2178 +2178 @@
-S=r=)
+S=r=#NA)
@@ -2180 +2180 @@
-Jn=o=)
+Jn=o=#NA)
@@ -2182 +2182 @@
-Harman))
+Harman#NA))
@@ -2184 +2184 @@
-goes)
+goes#go_v)
@@ -2187 +2187 @@
-from)
+from#from_prep)
@@ -2190 +2190 @@
-the)
+the#the_adj)
@@ -2192 +2192 @@
-London)))
+London#NA)))
@@ -2195 +2195 @@
-into)
+into#into_prep)
@@ -2198 +2198 @@
-the)
+the#the_adj)
@@ -2200 +2200 @@
-Sovraigne))))
+Sovraigne#NA))))
@@ -2205 +2205 @@
-and)
+and#and_conj1)
@@ -2209 +2209 @@
-Capt.)
+Capt.#NA)
@@ -2211 +2211 @@
-Hayward))
+Hayward#NA))
@@ -2214 +2214 @@
-out)
+out#out_adv)
@@ -2217 +2217 @@
-y=e=)
+y=e=#the_adj)
@@ -2219 +2219 @@
-Sovraigne)))
+Sovraigne#sovereign_n)))
@@ -2222 +2222 @@
-into)
+into#into_prep)
@@ -2225 +2225 @@
-this)
+this#this_pron)
@@ -2227 +2227 @@
-ship)))))
+ship#ship_n1)))))
@@ -2236 +2236 @@
-It))
+It#it_pron))
@@ -2238 +2238 @@
-is)
+is#be_v)
@@ -2241 +2241 @@
-no)
+no#no_adj)
@@ -2243 +2243 @@
-smal)
+smal#small_adj)
@@ -2245 +2245 @@
-trouble)
+trouble#trouble_n)
@@ -2248 +2248 @@
-to)
+to#to_prep)
@@ -2251 +2251 @@
-me)))
+me#me_pron1)))
@@ -2254 +2254 @@
-to)
+to#to_prep)
@@ -2256 +2256 @@
-part)
+part#part_v)
@@ -2259 +2259 @@
-from)
+from#from_prep)
@@ -2262 +2262 @@
-this)
+this#this_pron)
@@ -2264 +2264 @@
-brave)
+brave#brave_adj)
@@ -2266 +2266 @@
-ship)))))
+ship#ship_n1)))))
@@ -2276 +2276 @@
-her)
+her#her_pron2)
@@ -2278 +2278 @@
-only)
+only#only_adj)
@@ -2280 +2280 @@
-fault))
+fault#fault_n))
@@ -2282 +2282 @@
-is)
+is#be_v)
@@ -2289 +2289 @@
-she))
+she#she_pron))
@@ -2291 +2291 @@
-is)
+is#be_v)
@@ -2294 +2294 @@
-tender_sided)))))
+tender_sided#SEE_tender_adj)))))
@@ -2301 +2301 @@
-in)
+in#in_prep)
@@ -2304 +2304 @@
-all)
+all#all_adj)
@@ -2306 +2306 @@
-respects)
+respects#respect_n)
@@ -2309 +2309 @@
-otherwayes))))
+otherwayes#otherways_adv))))
@@ -2312 +2312 @@
-the)
+the#the_adj)
@@ -2314 +2314 @@
-best)
+best#best_adj)
@@ -2316 +2316 @@
-ship)
+ship#ship_n1)
@@ -2319 +2319 @@
-in)
+in#in_prep)
@@ -2322 +2322 @@
-the)
+the#the_adj)
@@ -2324 +2324 @@
-world))))))
+world#world_n))))))
@@ -2334 +2334 @@
-My)
+My#my_adj)
@@ -2336 +2336 @@
-deare))
+deare#dear_adj1))
@@ -2341 +2341 @@
-I))
+I#i_pron))
@@ -2343 +2343 @@
-am)
+am#be_v)
@@ -2346 +2346 @@
-very)
+very#very_adj)
@@ -2348 +2348 @@
-well)))
+well#well_adv)))
@@ -2355 +2355 @@
-My)
+My#my_adj)
@@ -2357 +2357 @@
-bro=r=)
+bro=r=#brother_n)
@@ -2360 +2360 @@
-Joseph))
+Joseph#NA))
@@ -2363 +2363 @@
-and)
+and#and_conj1)
@@ -2366 +2366 @@
-unckle)
+unckle#uncle_n)
@@ -2369 +2369 @@
-Richard)))))
+Richard#NA)))))
@@ -2372 +2372 @@
-likewise))))
+likewise#likewise_adv))))
@@ -2381 +2381 @@
-The)
+The#the_adj)
@@ -2383 +2383 @@
-Prince))
+Prince#prince_n))
@@ -2386 +2386 @@
-in)
+in#in_prep)
@@ -2389 +2389 @@
-good)
+good#good_adj)
@@ -2391 +2391 @@
-health)))
+health#health_n)))
@@ -2399 +2399 @@
-and)
+and#and_conj1)
@@ -2402 +2402 @@
-our)
+our#our_pron)
@@ -2404 +2404 @@
-fleet))
+fleet#fleet_n1))
@@ -2406 +2406 @@
-prepareing)
+prepareing#prepare_v)
@@ -2409 +2409 @@
-for)
+for#for_prep)
@@ -2412 +2412 @@
-another)
+another#another_pron)
@@ -2414 +2414 @@
-incounter)))
+incounter#encounter_n)))
@@ -2419 +2419 @@
-if)
+if#if_conj)
@@ -2426 +2426 @@
-the)
+the#the_adj)
@@ -2428 +2428 @@
-Dutch))
+Dutch#NA))
@@ -2430 +2430 @@
-comes)
+comes#come_v)
@@ -2432 +2432 @@
-out))))
+out#out_adv))))
@@ -2441 +2441 @@
-My)
+My#my_adj)
@@ -2443 +2443 @@
-deare))
+deare#dear_adj1))
@@ -2448 +2448 @@
-I))
+I#i_pron))
@@ -2450 +2450 @@
-am)
+am#be_v)
@@ -2453 +2453 @@
-thyne)
+thyne#thine_adj)
@@ -2458 +2458 @@
-till)
+till#till_prep)
@@ -2461 +2461 @@
-death)))
+death#death_n)))
@@ -2466 +2466 @@
-Richard)
+Richard#NA)
@@ -2468 +2468 @@
-Haddock))
+Haddock#NA))
@@ -2479 +2479 @@
-Pray))
+Pray#pray_v))
@@ -2484 +2484 @@
-if)
+if#if_conj)
@@ -2491 +2491 @@
-Com=r=)
+Com=r=#NA)
@@ -2493 +2493 @@
-Deane))
+Deane#NA))
@@ -2495 +2495 @@
-be)
+be#be_v)
@@ -2497 +2497 @@
-not)
+not#not_adv)
@@ -2500 +2500 @@
-in)
+in#in_prep)
@@ -2503 +2503 @@
-towne))))))
+towne#town_n))))))
@@ -2507 +2507 @@
-send)
+send#send_v1)
@@ -2510 +2510 @@
-forward))
+forward#forward_adj))
@@ -2513 +2513 @@
-the)
+the#the_adj)
@@ -2515 +2515 @@
-inclosed))
+inclosed#enclosed_adj))
@@ -2518 +2518 @@
-to)
+to#to_prep)
@@ -2521 +2521 @@
-Portsm=o=)))
+Portsm=o=#NA)))
