--- rhaddjr-1692-e3-h.psd.vert	2025-09-10 13:50:35
+++ rhaddjr-1692-e3-h.lem.vert	2025-09-10 13:50:45
@@ -13 +13 @@
-Aprill)
+Aprill#NA)
@@ -15 +15 @@
-y=e=)
+y=e=#the_adj)
@@ -17 +17 @@
-23=d=)
+23=d=#NA)
@@ -21 +21 @@
-Munday)
+Munday#NA)
@@ -29 +29 @@
-in)
+in#in_prep)
@@ -32 +32 @@
-y=e=)
+y=e=#the_adj)
@@ -34 +34 @@
-Hooke))
+Hooke#NA))
@@ -43 +43 @@
-Honourd)
+Honourd#honoured_adj)
@@ -45 +45 @@
-S=r=))
+S=r=#sir_n))
@@ -50 +50 @@
-This))
+This#this_pron))
@@ -52 +52 @@
-is)
+is#be_v)
@@ -55 +55 @@
-to)
+to#to_prep)
@@ -57 +57 @@
-acquaint)
+acquaint#acquaint_v)
@@ -61 +61 @@
-of)
+of#of_prep)
@@ -64 +64 @@
-our)
+our#our_pron)
@@ -67 +67 @@
-ingaging)
+ingaging#engage_v)
@@ -70 +70 @@
-w=th=)
+w=th=#with_prep)
@@ -73 +73 @@
-y=e=)
+y=e=#the_adj)
@@ -75 +75 @@
-French))))))
+French#french_adj))))))
@@ -78 +78 @@
-and)
+and#and_conj1)
@@ -81 +81 @@
-of)
+of#of_prep)
@@ -84 +84 @@
-our)
+our#our_pron)
@@ -87 +87 @@
-haveing)
+haveing#have_v)
@@ -89 +89 @@
-gott)
+gott#get_v)
@@ -92 +92 @@
-y=e=)
+y=e=#the_adj)
@@ -94 +94 @@
-victory))))))))
+victory#victory_n))))))))
@@ -103 +103 @@
-Wee))
+Wee#we_pron))
@@ -105 +105 @@
-mett)
+mett#meet_v)
@@ -108 +108 @@
-y=m=))
+y=m=#them_pron))
@@ -111 +111 @@
-of)
+of#of_prep)
@@ -114 +114 @@
-sea)))
+sea#sea_n)))
@@ -119 +119 @@
-May)
+May#NA)
@@ -121 +121 @@
-19))
+19#nineteen_adj))
@@ -130 +130 @@
-There))
+There#there_adv))
@@ -132 +132 @@
-was)
+was#be_v)
@@ -136 +136 @@
-about)
+about#about_adv)
@@ -138 +138 @@
-60))
+60#sixty_adj))
@@ -140 +140 @@
-saile))
+saile#sail_n1))
@@ -149 +149 @@
-Wee))
+Wee#we_pron))
@@ -151 +151 @@
-fought)
+fought#fight_v)
@@ -154 +154 @@
-y=m=))
+y=m=#them_pron))
@@ -158 +158 @@
-from)
+from#from_prep)
@@ -161 +161 @@
-11)))
+11#eleven_adj)))
@@ -164 +164 @@
-to)
+to#to_prep)
@@ -167 +167 @@
-9)
+9#nine_adj)
@@ -170 +170 @@
-att)
+att#at_prep)
@@ -173 +173 @@
-night)))))
+night#night_n)))))
@@ -179 +179 @@
-since)
+since#since_adv)
@@ -182 +182 @@
-w=ch=)))
+w=ch=#which_adj)))
@@ -193 +193 @@
-have)
+have#have_v)
@@ -195 +195 @@
-been)
+been#be_v)
@@ -198 +198 @@
-in)
+in#in_prep)
@@ -201 +201 @@
-pursuit)
+pursuit#pursuit_n)
@@ -204 +204 @@
-of)
+of#of_prep)
@@ -207 +207 @@
-y=m=))))))))
+y=m=#them_pron))))))))
@@ -216 +216 @@
-There))
+There#there_adv))
@@ -218 +218 @@
-is)
+is#be_v)
@@ -220 +220 @@
-run)
+run#run_v)
@@ -223 +223 @@
-ashoare))
+ashoare#ashore_adv))
@@ -228 +228 @@
-in)
+in#in_prep)
@@ -231 +231 @@
-Sherbrook)
+Sherbrook#NA)
@@ -233 +233 @@
-bay)))
+bay#NA)))
@@ -238 +238 @@
-Torveil)
+Torveil#NA)
@@ -241 +241 @@
-w=th=)
+w=th=#with_prep)
@@ -245 +245 @@
-3)
+3#three_adj)
@@ -247 +247 @@
-more))
+more#more_adj))
@@ -249 +249 @@
-capitall)
+capitall#capital_adj)
@@ -251 +251 @@
-ships)
+ships#ship_n1)
@@ -257 +257 @@
-w=ch=))
+w=ch=#which_adj))
@@ -264 +264 @@
-are)
+are#be_v)
@@ -267 +267 @@
-now))
+now#now_adv))
@@ -269 +269 @@
-burned))))))
+burned#burn_v1))))))
@@ -278 +278 @@
-Cozen)
+Cozen#NA)
@@ -280 +280 @@
-Tom)
+Tom#NA)
@@ -282 +282 @@
-Heath))
+Heath#NA))
@@ -284 +284 @@
-burnt)
+burnt#burn_v1)
@@ -287 +287 @@
-Torveil))
+Torveil#NA))
@@ -295 +295 @@
-and)
+and#and_conj1)
@@ -299 +299 @@
-have)
+have#have_v)
@@ -301 +301 @@
-chased)
+chased#chase_v1)
@@ -304 +304 @@
-14)
+14#fourteen_adj)
@@ -306 +306 @@
-saile)
+saile#sail_n1)
@@ -309 +309 @@
-more)))
+more#more_adj)))
@@ -312 +312 @@
-in)
+in#in_prep)
@@ -315 +315 @@
-y=e=)
+y=e=#the_adj)
@@ -317 +317 @@
-Hooke)
+Hooke#NA)
@@ -323 +323 @@
-where))
+where#where_adv))
@@ -331 +331 @@
-wee))
+wee#we_pron))
@@ -334 +334 @@
-now))
+now#now_adv))
@@ -336 +336 @@
-are)))))
+are#be_v)))))
@@ -345 +345 @@
-S=r=)
+S=r=#NA)
@@ -347 +347 @@
-Cloudsly)
+Cloudsly#NA)
@@ -349 +349 @@
-Shovel))
+Shovel#NA))
@@ -351 +351 @@
-is)
+is#be_v)
@@ -353 +353 @@
-goeing)
+goeing#go_v)
@@ -355 +355 @@
-in)
+in#in_adv)
@@ -358 +358 @@
-w=th=)
+w=th=#with_prep)
@@ -361 +361 @@
-y=e=)
+y=e=#the_adj)
@@ -363 +363 @@
-3=d=)
+3=d=#third_adj)
@@ -365 +365 @@
-rates)
+rates#rate_n1)
@@ -368 +368 @@
-and)
+and#and_conj1)
@@ -371 +371 @@
-fire-ships)))))
+fire-ships#fireship_n)))))
@@ -374 +374 @@
-to)
+to#to_prep)
@@ -376 +376 @@
-destroy)
+destroy#destroy_v)
@@ -379 +379 @@
-y=m=)))
+y=m=#them_pron)))
@@ -388 +388 @@
-Wee))
+Wee#we_pron))
@@ -390 +390 @@
-have)
+have#have_v)
@@ -392 +392 @@
-been)
+been#be_v)
@@ -395 +395 @@
-soe)
+soe#so_adv)
@@ -397 +397 @@
-unfortunate)
+unfortunate#unfortunate_adj)
@@ -402 +402 @@
-as)
+as#as_adv)
@@ -405 +405 @@
-to)
+to#to_prep)
@@ -409 +409 @@
-lose)
+lose#lose_v1)
@@ -412 +412 @@
-Rear)
+Rear#NA)
@@ -414 +414 @@
-Adm=ll=)
+Adm=ll=#NA)
@@ -416 +416 @@
-Carter))
+Carter#NA))
@@ -419 +419 @@
-in)
+in#in_prep)
@@ -422 +422 @@
-y=e=)
+y=e=#the_adj)
@@ -424 +424 @@
-fight))))))
+fight#fight_n))))))
@@ -433 +433 @@
-I))
+I#i_pron))
@@ -435 +435 @@
-am)
+am#be_v)
@@ -438 +438 @@
-very)
+very#very_adj)
@@ -440 +440 @@
-well)))
+well#well_adj)))
@@ -446 +446 @@
-and)
+and#and_conj1)
@@ -450 +450 @@
-have)
+have#have_v)
@@ -452 +452 @@
-received)
+received#receive_v)
@@ -455 +455 @@
-no)
+no#no_adj)
@@ -457 +457 @@
-wound))
+wound#wound_n))
@@ -466 +466 @@
-only)
+only#only_adj)
@@ -468 +468 @@
-a)
+a#a_adj2)
@@ -470 +470 @@
-small)
+small#small_adj)
@@ -472 +472 @@
-splinter))
+splinter#splinter_n))
@@ -474 +474 @@
-hitt)
+hitt#hit_v)
@@ -477 +477 @@
-mee))
+mee#me_pron1))
@@ -480 +480 @@
-on)
+on#on_prep)
@@ -483 +483 @@
-y=e=)
+y=e=#the_adj)
@@ -485 +485 @@
-thigh)))
+thigh#thigh_n)))
@@ -493 +493 @@
-but)
+but#but_prep)
@@ -497 +497 @@
-did)
+did#do_v)
@@ -500 +500 @@
-no)
+no#no_adj)
@@ -502 +502 @@
-damage))
+damage#damage_n))
@@ -510 +510 @@
-only)
+only#only_adj)
@@ -514 +514 @@
-made)
+made#make_v1)
@@ -518 +518 @@
-itt))
+itt#it_pron))
@@ -521 +521 @@
-black)
+black#black_adj)
@@ -523 +523 @@
-and)
+and#and_conj1)
@@ -525 +525 @@
-blew)))
+blew#blue_adj)))
@@ -534 +534 @@
-I))
+I#i_pron))
@@ -536 +536 @@
-would)
+would#will_v1)
@@ -538 +538 @@
-write)
+write#write_v)
@@ -541 +541 @@
-more)
+more#more_adj)
@@ -543 +543 @@
-particularly))
+particularly#particularly_adv))
@@ -551 +551 @@
-but)
+but#but_prep)
@@ -554 +554 @@
-y=e=)
+y=e=#the_adj)
@@ -556 +556 @@
-vessell))
+vessell#vessel_n1))
@@ -560 +560 @@
-I))
+I#i_pron))
@@ -562 +562 @@
-heare))
+heare#hear_v))
@@ -564 +564 @@
-is)
+is#be_v)
@@ -566 +566 @@
-goeing)
+goeing#go_v)
@@ -569 +569 @@
-away))
+away#away_adv))
@@ -572 +572 @@
-presently))
+presently#presently_adv))
@@ -581 +581 @@
-soe))
+soe#so_adv))
@@ -586 +586 @@
-haveing)
+haveing#have_v)
@@ -590 +590 @@
-no)
+no#no_adj)
@@ -592 +592 @@
-more))
+more#more_adj))
@@ -597 +597 @@
-att)
+att#at_prep)
@@ -600 +600 @@
-present)))
+present#present_adj)))
@@ -605 +605 @@
-butt)
+butt#but_prep)
@@ -608 +608 @@
-duty)
+duty#duty_n)
@@ -611 +611 @@
-to)
+to#to_prep)
@@ -615 +615 @@
-your)
+your#your_pron)
@@ -617 +617 @@
-self))
+self#self_pron))
@@ -620 +620 @@
-and)
+and#and_conj1)
@@ -623 +623 @@
-my)
+my#my_adj)
@@ -625 +625 @@
-mother))))))))
+mother#mother_n1))))))))
@@ -630 +630 @@
-I))
+I#i_pron))
@@ -632 +632 @@
-remain)
+remain#remain_v)
@@ -635 +635 @@
-your)
+your#your_pron)
@@ -637 +637 @@
-dutyfull)
+dutyfull#dutiful_adj)
@@ -639 +639 @@
-Son)
+Son#son_n1)
@@ -644 +644 @@
-Rich=d=)
+Rich=d=#NA)
@@ -646 +646 @@
-Haddock)))
+Haddock#NA)))
@@ -655 +655 @@
-Cozen)
+Cozen#cousin_n)
@@ -657 +657 @@
-Ruffin))
+Ruffin#NA))
@@ -659 +659 @@
-is)
+is#be_v)
@@ -663 +663 @@
-alive))
+alive#alive_adj))
@@ -666 +666 @@
-and)
+and#and_conj1)
@@ -669 +669 @@
-very)
+very#very_adj)
@@ -671 +671 @@
-well))))
+well#well_adj))))
@@ -680 +680 @@
-I)
+I#i_pron)
@@ -684 +684 @@
-will)
+will#will_v1)
@@ -686 +686 @@
-write)
+write#write_v)
@@ -689 +689 @@
-y=e=)
+y=e=#the_adj)
@@ -691 +691 @@
-particulars)
+particulars#particular_adj)
@@ -694 +694 @@
-of)
+of#of_prep)
@@ -697 +697 @@
-our)
+our#our_pron)
@@ -699 +699 @@
-fight))))
+fight#fight_n))))
@@ -702 +702 @@
-as)
+as#as_adv)
@@ -704 +704 @@
-soon)
+soon#soon_adv)
@@ -707 +707 @@
-as)
+as#as_adv)
@@ -718 +718 @@
-wee))
+wee#we_pron))
@@ -720 +720 @@
-come)
+come#come_v)
@@ -723 +723 @@
-into)
+into#into_prep)
@@ -726 +726 @@
-any)
+any#any_adj)
@@ -728 +728 @@
-port)))))))
+port#port_n1)))))))
@@ -733 +733 @@
-R=d=)
+R=d=#NA)
@@ -735 +735 @@
-H.)))
+H.#NA)))
