--- roxinden-1626-e2-h.psd.vert	2025-09-10 13:50:35
+++ roxinden-1626-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-I))
+I#i_pron))
@@ -16 +16 @@
-have)
+have#have_v)
@@ -18 +18 @@
-sent)
+sent#send_v1)
@@ -21 +21 @@
-v@)
+v@#five_v)
@@ -23 +23 @@
-@=ll=)
+@=ll=#pound_n1)
@@ -28 +28 @@
-at)
+at#at_prep)
@@ -31 +31 @@
-this)
+this#this_pron)
@@ -33 +33 @@
-time)))
+time#time_n)))
@@ -36 +36 @@
-unto)
+unto#unto_prep)
@@ -39 +39 @@
-you)))
+you#you_pron)))
@@ -43 +43 @@
-which))
+which#which_adj))
@@ -49 +49 @@
-I))
+I#i_pron))
@@ -51 +51 @@
-thinke)
+thinke#think_v2)
@@ -59 +59 @@
-will)
+will#will_v1)
@@ -61 +61 @@
-be)
+be#be_v)
@@ -64 +64 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -67 +67 @@
-to)
+to#to_prep)
@@ -69 +69 @@
-discharge)
+discharge#discharge_v)
@@ -72 +72 @@
-all)
+all#all_adj)
@@ -74 +74 @@
-your)
+your#your_pron)
@@ -76 +76 @@
-expence))))))))
+expence#expense_n))))))))
@@ -85 +85 @@
-You))
+You#you_pron))
@@ -87 +87 @@
-write)
+write#write_v)
@@ -90 +90 @@
-unto)
+unto#unto_prep)
@@ -93 +93 @@
-mee)))
+mee#me_pron1)))
@@ -97 +97 @@
-that)
+that#that_conj)
@@ -101 +101 @@
-you))
+you#you_pron))
@@ -103 +103 @@
-doe)
+doe#do_v)
@@ -105 +105 @@
-intende)
+intende#intend_v)
@@ -108 +108 @@
-to)
+to#to_prep)
@@ -110 +110 @@
-come)
+come#come_v)
@@ -112 +112 @@
-downe)
+downe#down_adv)
@@ -115 +115 @@
-presently)))))
+presently#presently_adv)))))
@@ -120 +120 @@
-and)
+and#and_conj1)
@@ -123 +123 @@
-that)
+that#that_conj)
@@ -127 +127 @@
-your)
+your#your_pron)
@@ -129 +129 @@
-tutor)
+tutor#tutor_n)
@@ -133 +133 @@
-will)
+will#will_v1)
@@ -135 +135 @@
-come)
+come#come_v)
@@ -137 +137 @@
-downe)
+downe#down_adv)
@@ -140 +140 @@
-with)
+with#with_prep)
@@ -143 +143 @@
-you)))
+you#you_pron)))
@@ -149 +149 @@
-whome))
+whome#whom_pron))
@@ -155 +155 @@
-I))
+I#i_pron))
@@ -157 +157 @@
-should)
+should#shall_v)
@@ -159 +159 @@
-bee)
+bee#be_v)
@@ -162 +162 @@
-very)
+very#very_adj)
@@ -164 +164 @@
-glad)
+glad#glad_adj)
@@ -169 +169 @@
-to)
+to#to_prep)
@@ -171 +171 @@
-see)
+see#see_v)
@@ -174 +174 @@
-heare))))))))))
+heare#here_adv))))))))))
@@ -182 +182 @@
-and)
+and#and_conj1)
@@ -185 +185 @@
-I))
+I#i_pron))
@@ -187 +187 @@
-will)
+will#will_v1)
@@ -189 +189 @@
-have)
+have#have_v)
@@ -193 +193 @@
-you))
+you#you_pron))
@@ -195 +195 @@
-to)
+to#to_prep)
@@ -197 +197 @@
-tell)
+tell#tell_v)
@@ -200 +200 @@
-him))
+him#him_pron))
@@ -203 +203 @@
-from)
+from#from_prep)
@@ -206 +206 @@
-me)))
+me#me_pron1)))
@@ -209 +209 @@
-that)
+that#that_conj)
@@ -213 +213 @@
-if)
+if#if_conj)
@@ -220 +220 @@
-it))
+it#it_pron))
@@ -222 +222 @@
-please)
+please#please_v)
@@ -225 +225 @@
-him))
+him#him_pron))
@@ -228 +228 @@
-to)
+to#to_prep)
@@ -230 +230 @@
-take)
+take#take_v)
@@ -235 +235 @@
-the)
+the#the_adj)
@@ -237 +237 @@
-paynes))
+paynes#pain_n1))
@@ -242 +242 @@
-to)
+to#to_prep)
@@ -244 +244 @@
-come)
+come#come_v)
@@ -247 +247 @@
-to)
+to#to_prep)
@@ -250 +250 @@
-us))))))))
+us#us_pron))))))))
@@ -255 +255 @@
-he))
+he#he_pron))
@@ -257 +257 @@
-shall)
+shall#shall_v)
@@ -259 +259 @@
-be)
+be#be_v)
@@ -263 +263 @@
-as)
+as#as_adv)
@@ -265 +265 @@
-hartely))
+hartely#heartily_adv))
@@ -267 +267 @@
-wellcome)
+wellcome#welcome_n1)
@@ -270 +270 @@
-as)
+as#as_adv)
@@ -273 +273 @@
-any)
+any#any_adj)
@@ -275 +275 @@
-frend)
+frend#friend_n)
@@ -286 +286 @@
-we))
+we#we_pron))
@@ -288 +288 @@
-have)))
+have#have_v)))
@@ -291 +291 @@
-livinge))))))))
+livinge#live_v1))))))))
@@ -300 +300 @@
-I))
+I#i_pron))
@@ -302 +302 @@
-had)
+had#have_v)
@@ -304 +304 @@
-thought)
+thought#think_v2)
@@ -307 +307 @@
-to)
+to#to_prep)
@@ -309 +309 @@
-have)
+have#have_v)
@@ -311 +311 @@
-written)
+written#write_v)
@@ -314 +314 @@
-unto)
+unto#unto_prep)
@@ -317 +317 @@
-him)))
+him#him_pron)))
@@ -320 +320 @@
-aboute)
+aboute#about_adv)
@@ -323 +323 @@
-it))))
+it#it_pron))))
@@ -331 +331 @@
-but)
+but#but_prep)
@@ -334 +334 @@
-finding)
+finding#find_v)
@@ -338 +338 @@
-myself))
+myself#myself_pron))
@@ -341 +341 @@
-an)
+an#a_adj2)
@@ -343 +343 @@
-ill)
+ill#ill_adj)
@@ -345 +345 @@
-scribe))))
+scribe#scribe_n1))))
@@ -348 +348 @@
-I))
+I#i_pron))
@@ -350 +350 @@
-have)
+have#have_v)
@@ -352 +352 @@
-left)
+left#leave_v1)
@@ -355 +355 @@
-it))
+it#it_pron))
@@ -358 +358 @@
-undonne))
+undonne#undone_adj1))
@@ -363 +363 @@
-hopinge)
+hopinge#hope_v)
@@ -366 +366 @@
-that)
+that#that_conj)
@@ -370 +370 @@
-you))
+you#you_pron))
@@ -372 +372 @@
-have)
+have#have_v)
@@ -375 +375 @@
-soe))
+soe#so_adv))
@@ -377 +377 @@
-carried)
+carried#carry_v)
@@ -380 +380 @@
-youre)
+youre#your_pron)
@@ -382 +382 @@
-selfe))
+selfe#self_pron))
@@ -385 +385 @@
-towards)
+towards#towards_prep)
@@ -388 +388 @@
-him)))))))
+him#him_pron)))))))
@@ -394 +394 @@
-but)
+but#but_prep)
@@ -397 +397 @@
-you))
+you#you_pron))
@@ -399 +399 @@
-can)
+can#can_v1)
@@ -401 +401 @@
-persuade)
+persuade#persuade_v)
@@ -404 +404 @@
-more))
+more#more_adj))
@@ -407 +407 @@
-with)
+with#with_prep)
@@ -410 +410 @@
-him)))
+him#him_pron)))
@@ -413 +413 @@
-than)
+than#than_conj)
@@ -424 +424 @@
-my)
+my#my_adj)
@@ -426 +426 @@
-letter))
+letter#letter_n1))
@@ -428 +428 @@
-could)
+could#can_v1)
@@ -439 +439 @@
-I))
+I#i_pron))
@@ -441 +441 @@
-shall)
+shall#shall_v)
@@ -443 +443 @@
-expect)
+expect#expect_v)
@@ -446 +446 @@
-youre)
+youre#your_pron)
@@ -448 +448 @@
-comminge))
+comminge#coming_n))
@@ -451 +451 @@
-according)
+according#accord_v)
@@ -454 +454 @@
-to)
+to#to_prep)
@@ -457 +457 @@
-youre)
+youre#your_pron)
@@ -459 +459 @@
-writinge))))
+writinge#writing_n))))
@@ -468 +468 @@
-You))
+You#you_pron))
@@ -470 +470 @@
-may)
+may#may_v1)
@@ -473 +473 @@
-very)
+very#very_adj)
@@ -475 +475 @@
-easily))
+easily#easily_adv))
@@ -477 +477 @@
-come)
+come#come_v)
@@ -479 +479 @@
-down)
+down#down_adv)
@@ -482 +482 @@
-in)
+in#in_prep)
@@ -485 +485 @@
-too)
+too#two_adj)
@@ -487 +487 @@
-dayes)))
+dayes#day_n)))
@@ -496 +496 @@
-Youre)
+Youre#your_pron)
@@ -498 +498 @@
-best)
+best#best_adj)
@@ -500 +500 @@
-way))
+way#way_n1))
@@ -502 +502 @@
-is)
+is#be_v)
@@ -505 +505 @@
-to)
+to#to_prep)
@@ -507 +507 @@
-come)
+come#come_v)
@@ -511 +511 @@
-from)
+from#from_prep)
@@ -514 +514 @@
-London)))
+London#NA)))
@@ -517 +517 @@
-to)
+to#to_prep)
@@ -520 +520 @@
-Gravesend))))
+Gravesend#NA))))
@@ -523 +523 @@
-by)
+by#by_prep)
@@ -526 +526 @@
-water))))
+water#water_n))))
@@ -534 +534 @@
-and)
+and#and_conj1)
@@ -537 +537 @@
-from)
+from#from_prep)
@@ -540 +540 @@
-Gravesend)))
+Gravesend#NA)))
@@ -543 +543 @@
-you))
+you#you_pron))
@@ -545 +545 @@
-may)
+may#may_v1)
@@ -548 +548 @@
-easily))
+easily#easily_adv))
@@ -550 +550 @@
-come)
+come#come_v)
@@ -553 +553 @@
-to)
+to#to_prep)
@@ -556 +556 @@
-my)
+my#my_adj)
@@ -558 +558 @@
-house)))
+house#house_n1)))
@@ -561 +561 @@
-in)
+in#in_prep)
@@ -564 +564 @@
-halfe)
+halfe#half_adj)
@@ -566 +566 @@
-a)
+a#a_adj2)
@@ -568 +568 @@
-day)))
+day#day_n)))
@@ -577 +577 @@
-Thus))
+Thus#thus_adv))
@@ -580 +580 @@
-wishing)
+wishing#wish_v)
@@ -583 +583 @@
-you))
+you#you_pron))
@@ -586 +586 @@
-to)
+to#to_prep)
@@ -588 +588 @@
-remember)
+remember#remember_v)
@@ -591 +591 @@
-me))
+me#me_pron1))
@@ -594 +594 @@
-unto)
+unto#unto_prep)
@@ -598 +598 @@
-your)
+your#your_pron)
@@ -600 +600 @@
-tutor))
+tutor#tutor_n))
@@ -603 +603 @@
-and)
+and#and_conj1)
@@ -606 +606 @@
-the)
+the#the_adj)
@@ -608 +608 @@
-rest)
+rest#rest_n2)
@@ -611 +611 @@
-of)
+of#of_prep)
@@ -614 +614 @@
-our)
+our#our_pron)
@@ -616 +616 @@
-frends)))))))))
+frends#friend_n)))))))))
@@ -621 +621 @@
-I))
+I#i_pron))
@@ -623 +623 @@
-leave)
+leave#leave_v1)
@@ -626 +626 @@
-you))
+you#you_pron))
@@ -629 +629 @@
-to)
+to#to_prep)
@@ -632 +632 @@
-the)
+the#the_adj)
@@ -634 +634 @@
-protection)
+protection#protection_n)
@@ -637 +637 @@
-of)
+of#of_prep)
@@ -640 +640 @@
-the)
+the#the_adj)
@@ -642 +642 @@
-Almightie)))))
+Almightie#almighty_adj)))))
@@ -650 +650 @@
-Your)
+Your#your_pron)
@@ -652 +652 @@
-lovinge)
+lovinge#loving_adj)
@@ -654 +654 @@
-father)
+father#father_n)
@@ -657 +657 @@
-Richard)
+Richard#NA)
@@ -659 +659 @@
-Oxinden))
+Oxinden#NA))
@@ -667 +667 @@
-From)
+From#from_prep)
@@ -670 +670 @@
-Barham)))
+Barham#NA)))
@@ -676 +676 @@
-this)
+this#this_pron)
@@ -678 +678 @@
-14th)
+14th#NA)
@@ -681 +681 @@
-of)
+of#of_prep)
@@ -684 +684 @@
-June)))
+June#NA)))
@@ -686 +686 @@
-1626)
+1626#NA)
@@ -695 +695 @@
-To)
+To#to_prep)
@@ -698 +698 @@
-my)
+my#my_adj)
@@ -701 +701 @@
-very)
+very#very_adj)
@@ -703 +703 @@
-lovinge))
+lovinge#loving_adj))
@@ -705 +705 @@
-sonne)
+sonne#son_n1)
@@ -710,3 +710 @@
-Mr.)
-(NPR
-Henry)
+Mr.#NA)
@@ -714 +712,3 @@
-Oxinden))))
+Henry#NA)
+(NPR
+Oxinden#NA))))
@@ -719 +719 @@
-at)
+at#at_prep)
@@ -722 +722 @@
-Corpus)
+Corpus#NA)
@@ -724 +724 @@
-Christi)
+Christi#NA)
@@ -726 +726 @@
-Collidge)
+Collidge#NA)
@@ -729 +729 @@
-in)
+in#in_prep)
@@ -732 +732 @@
-Oxfoord)))))
+Oxfoord#NA)))))
@@ -736 +736 @@
-give)
+give#give_v)
@@ -739 +739 @@
-this))
+this#this_pron))
@@ -754 +754 @@
-Sonne))
+Sonne#son_n1))
@@ -759 +759 @@
-I))
+I#i_pron))
@@ -761 +761 @@
-am)
+am#be_v)
@@ -764 +764 @@
-very)
+very#very_adj)
@@ -766 +766 @@
-sorry)
+sorry#sorry_adj)
@@ -769 +769 @@
-to)
+to#to_prep)
@@ -771 +771 @@
-heare)
+heare#hear_v)
@@ -774 +774 @@
-that)
+that#that_conj)
@@ -778 +778 @@
-you))
+you#you_pron))
@@ -780 +780 @@
-are)
+are#be_v)
@@ -783 +783 @@
-sicke)))))))
+sicke#sick_adj)))))))
@@ -789 +789 @@
-but)
+but#but_prep)
@@ -792 +792 @@
-I))
+I#i_pron))
@@ -794 +794 @@
-trust)
+trust#trust_v)
@@ -797 +797 @@
-God))
+God#NA))
@@ -802 +802 @@
-that)
+that#that_conj)
@@ -806 +806 @@
-he))
+he#he_pron))
@@ -808 +808 @@
-will)
+will#will_v1)
@@ -810 +810 @@
-restore)
+restore#restore_v)
@@ -813 +813 @@
-you))
+you#you_pron))
@@ -816 +816 @@
-to)
+to#to_prep)
@@ -821 +821 @@
-your)
+your#your_pron)
@@ -823 +823 @@
-former)
+former#former_adj)
@@ -825 +825 @@
-health)))))
+health#health_n)))))
@@ -834 +834 @@
-how)
+how#how_adv)
@@ -838 +838 @@
-soe)
+soe#so_adv)
@@ -840 +840 @@
-ever))
+ever#ever_adv))
@@ -843 +843 @@
-I))
+I#i_pron))
@@ -846 +846 @@
-most)
+most#most_adj)
@@ -850 +850 @@
-earnestly))
+earnestly#earnestly_adv))
@@ -852 +852 @@
-desire)
+desire#desire_v)
@@ -855 +855 @@
-that)
+that#that_conj)
@@ -859 +859 @@
-you))
+you#you_pron))
@@ -861 +861 @@
-will)
+will#will_v1)
@@ -863 +863 @@
-take)
+take#take_v)
@@ -866 +866 @@
-the)
+the#the_adj)
@@ -868 +868 @@
-visitation))
+visitation#visitation_n))
@@ -871 +871 @@
-patiently))
+patiently#patiently_adv))
@@ -876 +876 @@
-submitting)
+submitting#submit_v)
@@ -879 +879 @@
-youre)
+youre#your_pron)
@@ -881 +881 @@
-will))
+will#will_n1))
@@ -884 +884 @@
-to)
+to#to_prep)
@@ -887 +887 @@
-his)
+his#his_adj)
@@ -892 +892 @@
-that)
+that#that_conj)
@@ -897 +897 @@
-hath)
+hath#have_v)
@@ -899 +899 @@
-sent)
+sent#send_v1)
@@ -902 +902 @@
-it)))))))))
+it#it_pron)))))))))
@@ -911 +911 @@
-I))
+I#i_pron))
@@ -913 +913 @@
-do)
+do#do_v)
@@ -917 +917 @@
-assure)
+assure#assure_v)
@@ -920 +920 @@
-you))
+you#you_pron))
@@ -923 +923 @@
-that)
+that#that_conj)
@@ -927 +927 @@
-it))
+it#it_pron))
@@ -929 +929 @@
-shall)
+shall#shall_v)
@@ -931 +931 @@
-be)
+be#be_v)
@@ -934 +934 @@
-the)
+the#the_adj)
@@ -936 +936 @@
-greatest)
+greatest#great_adj)
@@ -938 +938 @@
-comfort)
+comfort#comfort_n)
@@ -943 +943 @@
-unto)
+unto#unto_prep)
@@ -946 +946 @@
-me)))
+me#me_pron1)))
@@ -951 +951 @@
-that)
+that#that_conj)
@@ -956 +956 @@
-may)
+may#may_v1)
@@ -958 +958 @@
-be)))
+be#be_v)))
@@ -961 +961 @@
-to)
+to#to_prep)
@@ -963 +963 @@
-heare)
+heare#hear_v)
@@ -966 +966 @@
-that)
+that#that_conj)
@@ -970 +970 @@
-you))
+you#you_pron))
@@ -972 +972 @@
-doe)
+doe#do_v)
@@ -975 +975 @@
-patiently)
+patiently#patiently_adv)
@@ -977 +977 @@
-and)
+and#and_conj1)
@@ -979 +979 @@
-cheerefully))
+cheerefully#cheerfully_adv))
@@ -981 +981 @@
-undergoe)
+undergoe#undergo_v)
@@ -984 +984 @@
-this)
+this#this_pron)
@@ -989 +989 @@
-that)
+that#that_conj)
@@ -995 +995 @@
-God))
+God#NA))
@@ -997 +997 @@
-hath)
+hath#have_v)
@@ -999 +999 @@
-layd)
+layd#lay_v1)
@@ -1002 +1002 @@
-upone)
+upone#upon_prep)
@@ -1005 +1005 @@
-you)))))))))))
+you#you_pron)))))))))))
@@ -1014 +1014 @@
-Sonne))
+Sonne#son_n1))
@@ -1019 +1019 @@
-I))
+I#i_pron))
@@ -1021 +1021 @@
-woold)
+woold#will_v1)
@@ -1024 +1024 @@
-with)
+with#with_prep)
@@ -1027 +1027 @@
-all)
+all#all_adj)
@@ -1029 +1029 @@
-my)
+my#my_adj)
@@ -1031 +1031 @@
-heart)))
+heart#heart_n)))
@@ -1033 +1033 @@
-have)
+have#have_v)
@@ -1035 +1035 @@
-come)
+come#come_v)
@@ -1038 +1038 @@
-unto)
+unto#unto_prep)
@@ -1041 +1041 @@
-you)))
+you#you_pron)))
@@ -1046 +1046 @@
-if)
+if#if_conj)
@@ -1053 +1053 @@
-I))
+I#i_pron))
@@ -1055 +1055 @@
-did)
+did#do_v)
@@ -1057 +1057 @@
-thinke)
+thinke#think_v2)
@@ -1064 +1064 @@
-I))
+I#i_pron))
@@ -1066 +1066 @@
-could)
+could#can_v1)
@@ -1068 +1068 @@
-have)
+have#have_v)
@@ -1070 +1070 @@
-donne)
+donne#do_v)
@@ -1073 +1073 @@
-you))
+you#you_pron))
@@ -1076 +1076 @@
-any)
+any#any_adj)
@@ -1078 +1078 @@
-good)))))))
+good#good_adj)))))))
@@ -1086 +1086 @@
-but)
+but#but_prep)
@@ -1089 +1089 @@
-I))
+I#i_pron))
@@ -1091 +1091 @@
-am)
+am#be_v)
@@ -1094 +1094 @@
-well))
+well#well_adv))
@@ -1096 +1096 @@
-assured)
+assured#assure_v)
@@ -1099 +1099 @@
-that)
+that#that_conj)
@@ -1103 +1103 @@
-youre)
+youre#your_pron)
@@ -1105 +1105 @@
-tutor))
+tutor#tutor_n))
@@ -1107 +1107 @@
-will)
+will#will_v1)
@@ -1109 +1109 @@
-doe)
+doe#do_v)
@@ -1113 +1113 @@
-as)
+as#as_adv)
@@ -1115 +1115 @@
-much)))
+much#much_adj)))
@@ -1118 +1118 @@
-for)
+for#for_prep)
@@ -1121 +1121 @@
-you)))
+you#you_pron)))
@@ -1124 +1124 @@
-as)
+as#as_adv)
@@ -1135 +1135 @@
-I))
+I#i_pron))
@@ -1137 +1137 @@
-could)
+could#can_v1)
@@ -1142 +1142 @@
-if)
+if#if_conj)
@@ -1149 +1149 @@
-I))
+I#i_pron))
@@ -1151 +1151 @@
-weare)
+weare#be_v)
@@ -1154 +1154 @@
-with)
+with#with_prep)
@@ -1157 +1157 @@
-you)))))))))))
+you#you_pron)))))))))))
@@ -1166 +1166 @@
-I))
+I#i_pron))
@@ -1168 +1168 @@
-have)
+have#have_v)
@@ -1170 +1170 @@
-sent)
+sent#send_v1)
@@ -1173 +1173 @@
-unto)
+unto#unto_prep)
@@ -1176 +1176 @@
-you)))
+you#you_pron)))
@@ -1179 +1179 @@
-Goodman)
+Goodman#NA)
@@ -1181 +1181 @@
-Cooper)
+Cooper#NA)
@@ -1186 +1186 @@
-one)
+one#one_adj)
@@ -1190 +1190 @@
-hoome))
+hoome#whom_pron))
@@ -1196 +1196 @@
-I))
+I#i_pron))
@@ -1198 +1198 @@
-thinke)
+thinke#think_v2)
@@ -1205 +1205 @@
-you))
+you#you_pron))
@@ -1207 +1207 @@
-will)
+will#will_v1)
@@ -1210 +1210 @@
-well))
+well#well_adv))
@@ -1212 +1212 @@
-like)
+like#like_v1)
@@ -1215 +1215 @@
-of)
+of#of_prep)
@@ -1225 +1225 @@
-and)
+and#and_conj1)
@@ -1228 +1228 @@
-when)
+when#when_adv)
@@ -1235 +1235 @@
-he))
+he#he_pron))
@@ -1237 +1237 @@
-doth)
+doth#do_v)
@@ -1239 +1239 @@
-returne))))
+returne#return_v1))))
@@ -1244 +1244 @@
-if)
+if#if_conj)
@@ -1247 +1247 @@
-that)
+that#that_conj)
@@ -1251 +1251 @@
-you))
+you#you_pron))
@@ -1253 +1253 @@
-doe)
+doe#do_v)
@@ -1255 +1255 @@
-desire)
+desire#desire_v)
@@ -1258 +1258 @@
-that)
+that#that_conj)
@@ -1262 +1262 @@
-I))
+I#i_pron))
@@ -1264 +1264 @@
-should)
+should#shall_v)
@@ -1267 +1267 @@
-then))
+then#then_adv))
@@ -1269 +1269 @@
-come)
+come#come_v)
@@ -1272 +1272 @@
-unto)
+unto#unto_prep)
@@ -1275 +1275 @@
-you))))))))
+you#you_pron))))))))
@@ -1280 +1280 @@
-I))
+I#i_pron))
@@ -1282 +1282 @@
-will)
+will#will_v1)
@@ -1284 +1284 @@
-doe)
+doe#do_v)
@@ -1287 +1287 @@
-it))
+it#it_pron))
@@ -1290 +1290 @@
-with)
+with#with_prep)
@@ -1293 +1293 @@
-all)
+all#all_adj)
@@ -1295 +1295 @@
-possible)
+possible#possible_adj)
@@ -1297 +1297 @@
-speed)))
+speed#speed_n)))
@@ -1306 +1306 @@
-I))
+I#i_pron))
@@ -1308 +1308 @@
-have)
+have#have_v)
@@ -1310 +1310 @@
-sent)
+sent#send_v1)
@@ -1313 +1313 @@
-some)
+some#some_pron)
@@ -1315 +1315 @@
-money))
+money#money_n))
@@ -1318 +1318 @@
-unto)
+unto#unto_prep)
@@ -1321 +1321 @@
-you)))
+you#you_pron)))
@@ -1324 +1324 @@
-by)
+by#by_prep)
@@ -1327 +1327 @@
-him)))
+him#him_pron)))
@@ -1335 +1335 @@
-and)
+and#and_conj1)
@@ -1338 +1338 @@
-your)
+your#your_pron)
@@ -1340 +1340 @@
-grandmother))
+grandmother#grandmother_n))
@@ -1342 +1342 @@
-hath)
+hath#have_v)
@@ -1344 +1344 @@
-sent)
+sent#send_v1)
@@ -1347 +1347 @@
-you))
+you#you_pron))
@@ -1350 +1350 @@
-a)
+a#a_adj2)
@@ -1352 +1352 @@
-token))
+token#token_n))
@@ -1360 +1360 @@
-and)
+and#and_conj1)
@@ -1363 +1363 @@
-your)
+your#your_pron)
@@ -1365 +1365 @@
-mother))
+mother#mother_n1))
@@ -1367 +1367 @@
-hath)
+hath#have_v)
@@ -1369 +1369 @@
-sent)
+sent#send_v1)
@@ -1372 +1372 @@
-you))
+you#you_pron))
@@ -1375 +1375 @@
-another))
+another#another_pron))
@@ -1383 +1383 @@
-and)
+and#and_conj1)
@@ -1386 +1386 @@
-wee))
+wee#we_pron))
@@ -1388 +1388 @@
-all)
+all#all_adj)
@@ -1390 +1390 @@
-do)
+do#do_v)
@@ -1392 +1392 @@
-ioyne)
+ioyne#join_v1)
@@ -1395 +1395 @@
-in)
+in#in_prep)
@@ -1398 +1398 @@
-prayer)
+prayer#prayer_n1)
@@ -1401 +1401 @@
-to)
+to#to_prep)
@@ -1404 +1404 @@
-God)))))
+God#NA)))))
@@ -1407 +1407 @@
-that)
+that#that_conj)
@@ -1411 +1411 @@
-it))
+it#it_pron))
@@ -1413 +1413 @@
-will)
+will#will_v1)
@@ -1415 +1415 @@
-please)
+please#please_v)
@@ -1418 +1418 @@
-him))
+him#him_pron))
@@ -1421 +1421 @@
-to)
+to#to_prep)
@@ -1423 +1423 @@
-send)
+send#send_v1)
@@ -1426 +1426 @@
-you))
+you#you_pron))
@@ -1429 +1429 @@
-youre)
+youre#your_pron)
@@ -1431 +1431 @@
-health))
+health#health_n))
@@ -1434 +1434 @@
-agayne)))))
+agayne#again_adv)))))
@@ -1443 +1443 @@
-Soe))
+Soe#so_adv))
@@ -1446 +1446 @@
-in)
+in#in_prep)
@@ -1449 +1449 @@
-hast)))
+hast#haste_n)))
@@ -1452 +1452 @@
-I))
+I#i_pron))
@@ -1454 +1454 @@
-rest)
+rest#rest_v2)
@@ -1457 +1457 @@
-Youer)
+Youer#your_pron)
@@ -1460 +1460 @@
-ever)
+ever#ever_adv)
@@ -1462 +1462 @@
-loveinge))
+loveinge#loving_adj))
@@ -1464 +1464 @@
-father)
+father#father_n)
@@ -1467 +1467 @@
-Richard)
+Richard#NA)
@@ -1469 +1469 @@
-Oxinden)))
+Oxinden#NA)))
@@ -1474 +1474 @@
-From)
+From#from_prep)
@@ -1477 +1477 @@
-Barham)))
+Barham#NA)))
@@ -1480 +1480 @@
-this)
+this#this_pron)
@@ -1482 +1482 @@
-7th)
+7th#NA)
@@ -1485 +1485 @@
-of)
+of#of_prep)
@@ -1488 +1488 @@
-July)))
+July#NA)))
@@ -1490 +1490 @@
-1627))
+1627#NA))
