--- rplumpt-1536-e1-h.psd.vert	2025-09-10 13:50:35
+++ rplumpt-1536-e1-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-To)
+To#to_prep)
@@ -17 +17 @@
-his)
+his#his_adj)
@@ -19 +19 @@
-mother)))
+mother#mother_n1)))
@@ -22 +22 @@
-at)
+at#at_prep)
@@ -25 +25 @@
-Plompton)))
+Plompton#NA)))
@@ -27 +27 @@
-be)
+be#be_v)
@@ -30 +30 @@
-this)
+this#this_pron)
@@ -32 +32 @@
-letter))
+letter#letter_n1))
@@ -34 +34 @@
-delivered)
+delivered#deliver_v1)
@@ -44 +44 @@
-Right)
+Right#right_adv)
@@ -46 +46 @@
-worshipful))
+worshipful#worshipful_adj))
@@ -48 +48 @@
-mother))
+mother#mother_n1))
@@ -53 +53 @@
-I))
+I#i_pron))
@@ -56 +56 @@
-humbly))
+humbly#humbly_adv))
@@ -58 +58 @@
-recommend)
+recommend#recommend_v1)
@@ -61 +61 @@
-mee))
+mee#me_pron1))
@@ -64 +64 @@
-unto)
+unto#unto_prep)
@@ -67 +67 @@
-you)))
+you#you_pron)))
@@ -72 +72 @@
-desiring)
+desiring#desire_v)
@@ -75 +75 @@
-you))
+you#you_pron))
@@ -78 +78 @@
-of)
+of#of_prep)
@@ -81 +81 @@
-your)
+your#your_pron)
@@ -83 +83 @@
-dayly)
+dayly#daily_adj)
@@ -85 +85 @@
-blessing))))
+blessing#blessing_n))))
@@ -90 +90 @@
-praing)
+praing#pray_v)
@@ -95 +95 @@
-Jesu))
+Jesu#NA))
@@ -101 +101 @@
-long))
+long#long_adj1))
@@ -103 +103 @@
-to)
+to#to_prep)
@@ -105 +105 @@
-continew)
+continew#continue_v)
@@ -108 +108 @@
-your)
+your#your_pron)
@@ -110 +110 @@
-helth))
+helth#health_n))
@@ -113 +113 @@
-to)
+to#to_prep)
@@ -116 +116 @@
-his)
+his#his_adj)
@@ -118 +118 @@
-pleasur)))))
+pleasur#pleasure_n)))))
@@ -127 +127 @@
-Mother))
+Mother#mother_n1))
@@ -132 +132 @@
-I))
+I#i_pron))
@@ -134 +134 @@
-thanke)
+thanke#thank_v)
@@ -137 +137 @@
-you))
+you#you_pron))
@@ -140 +140 @@
-for)
+for#for_prep)
@@ -143 +143 @@
-the)
+the#the_adj)
@@ -145 +145 @@
-.....)
+.....#NA)
@@ -156 +156 @@
-you))
+you#you_pron))
@@ -158 +158 @@
-send)
+send#send_v1)
@@ -161 +161 @@
-mee))))))
+mee#me_pron1))))))
@@ -169 +169 @@
-for)
+for#for_prep)
@@ -172 +172 @@
-yf)
+yf#if_conj)
@@ -179 +179 @@
-you))
+you#you_pron))
@@ -181 +181 @@
-were)
+were#be_v)
@@ -183 +183 @@
-not))))
+not#not_adv))))
@@ -188 +188 @@
-I))
+I#i_pron))
@@ -190 +190 @@
-were)
+were#be_v)
@@ -192 +192 @@
-not)
+not#not_adv)
@@ -195 +195 @@
-able)
+able#able_adj)
@@ -198 +198 @@
-to)
+to#to_prep)
@@ -200 +200 @@
-live)))
+live#live_v1)))
@@ -208 +208 @@
-for)
+for#for_prep)
@@ -211 +211 @@
-this)
+this#this_pron)
@@ -213 +213 @@
-same)
+same#same_adj)
@@ -215 +215 @@
-Christmasse))
+Christmasse#NA))
@@ -217 +217 @@
-hath)
+hath#have_v)
@@ -219 +219 @@
-cost)
+cost#cost_v)
@@ -222 +222 @@
-mee))
+mee#me_pron1))
@@ -226 +226 @@
-as)
+as#as_adv)
@@ -228 +228 @@
-much))
+much#much_adj))
@@ -231 +231 @@
-as)
+as#as_adv)
@@ -242 +242 @@
-you))
+you#you_pron))
@@ -244 +244 @@
-send)
+send#send_v1)
@@ -247 +247 @@
-mee))))))
+mee#me_pron1))))))
@@ -253 +253 @@
-Wherfore))
+Wherfore#wherefore_adv))
@@ -263 +263 @@
-I))
+I#i_pron))
@@ -265 +265 @@
-am)
+am#be_v)
@@ -268 +268 @@
-afraid))
+afraid#afraid_adj))
@@ -275 +275 @@
-I))
+I#i_pron))
@@ -277 +277 @@
-shal)
+shal#shall_v)
@@ -279 +279 @@
-not)
+not#not_adv)
@@ -281 +281 @@
-have)
+have#have_v)
@@ -284 +284 @@
-money)
+money#money_n)
@@ -292 +292 @@
-to)
+to#to_prep)
@@ -294 +294 @@
-serve)
+serve#serve_v1)
@@ -297 +297 @@
-mee))
+mee#me_pron1))
@@ -300 +300 @@
-to)
+to#to_prep)
@@ -303 +303 @@
-Easter))))))))))
+Easter#NA))))))))))
@@ -311 +311 @@
-Also)
+Also#also_adv)
@@ -314 +314 @@
-I))
+I#i_pron))
@@ -316 +316 @@
-wold)
+wold#will_v1)
@@ -318 +318 @@
-desire)
+desire#desire_v)
@@ -321 +321 @@
-you))
+you#you_pron))
@@ -324 +324 @@
-to)
+to#to_prep)
@@ -326 +326 @@
-send)
+send#send_v1)
@@ -329 +329 @@
-mee))
+mee#me_pron1))
@@ -332 +332 @@
-word)
+word#word_n)
@@ -335 +335 @@
-of)
+of#of_prep)
@@ -338 +338 @@
-the)
+the#the_adj)
@@ -340 +340 @@
-letter)
+letter#letter_n1)
@@ -345 +345 @@
-that)
+that#that_conj)
@@ -351 +351 @@
-I))
+I#i_pron))
@@ -353 +353 @@
-wrote)
+wrote#write_v)
@@ -356 +356 @@
-to)
+to#to_prep)
@@ -360 +360 @@
-my)
+my#my_adj)
@@ -362 +362 @@
-father))
+father#father_n))
@@ -365 +365 @@
-and)
+and#and_conj1)
@@ -368 +368 @@
-you)))))
+you#you_pron)))))
@@ -373 +373 @@
-for)
+for#for_prep)
@@ -375 +375 @@
-to)
+to#to_prep)
@@ -377 +377 @@
-moove)
+moove#move_v)
@@ -380 +380 @@
-my)
+my#my_adj)
@@ -382 +382 @@
-Lady)
+Lady#lady_n)
@@ -385 +385 @@
-Gascoin)))
+Gascoin#NA)))
@@ -388 +388 @@
-to)
+to#to_prep)
@@ -390 +390 @@
-write)
+write#write_v)
@@ -393 +393 @@
-to)
+to#to_prep)
@@ -398 +398 @@
-my)
+my#my_adj)
@@ -400 +400 @@
-lord)
+lord#lord_n)
@@ -405 +405 @@
-her)
+her#her_pron2)
@@ -407 +407 @@
-brother))))
+brother#brother_n))))
@@ -412 +412 @@
-not)
+not#not_adv)
@@ -414 +414 @@
-to)
+to#to_prep)
@@ -416 +416 @@
-bee)
+bee#be_v)
@@ -422 +422 @@
-only)
+only#only_adj)
@@ -424 +424 @@
-his)
+his#his_adj)
@@ -426 +426 @@
-servant))
+servant#servant_n))
@@ -431 +431 @@
-but)
+but#but_prep)
@@ -435 +435 @@
-of)
+of#of_prep)
@@ -438 +438 @@
-his)
+his#his_adj)
@@ -440 +440 @@
-houshold)))
+houshold#household_n)))
@@ -443 +443 @@
-and)
+and#and_conj1)
@@ -446 +446 @@
-attending)
+attending#attend_v)
@@ -449 +449 @@
-unto)
+unto#unto_prep)
@@ -452 +452 @@
-him)))))))))))))))))
+him#him_pron)))))))))))))))))
@@ -460 +460 @@
-for)
+for#for_prep)
@@ -463 +463 @@
-els))
+els#else_adv))
@@ -466 +466 @@
-he))
+he#he_pron))
@@ -468 +468 @@
-wold)
+wold#will_v1)
@@ -470 +470 @@
-do)
+do#do_v)
@@ -473 +473 @@
-as)
+as#as_adv)
@@ -484 +484 @@
-other)
+other#other_adj)
@@ -486 +486 @@
-lords)
+lords#lord_n)
@@ -490 +490 @@
-do)
+do#do_v)
@@ -502 +502 @@
-knowes)
+knowes#know_v)
@@ -504 +504 @@
-not)
+not#not_adv)
@@ -507 +507 @@
-half)
+half#half_adj)
@@ -509 +509 @@
-their)
+their#their_adj)
@@ -511 +511 @@
-servants)))))))
+servants#servant_n)))))))
@@ -517 +517 @@
-Wherfor))
+Wherfor#wherefore_adv))
@@ -527 +527 @@
-I))
+I#i_pron))
@@ -529 +529 @@
-desire)
+desire#desire_v)
@@ -532 +532 @@
-you))
+you#you_pron))
@@ -535 +535 @@
-that)
+that#that_conj)
@@ -539 +539 @@
-you))
+you#you_pron))
@@ -541 +541 @@
-wil)
+wil#will_v1)
@@ -543 +543 @@
-moove)
+moove#move_v)
@@ -546 +546 @@
-my)
+my#my_adj)
@@ -548 +548 @@
-lady)
+lady#lady_n)
@@ -551 +551 @@
-Gascoin)))
+Gascoin#NA)))
@@ -554 +554 @@
-to)
+to#to_prep)
@@ -556 +556 @@
-write)
+write#write_v)
@@ -559 +559 @@
-so))
+so#so_adv))
@@ -562 +562 @@
-to)
+to#to_prep)
@@ -565 +565 @@
-my)
+my#my_adj)
@@ -567 +567 @@
-lord)))
+lord#lord_n)))
@@ -572 +572 @@
-that)
+that#that_conj)
@@ -576 +576 @@
-I))
+I#i_pron))
@@ -578 +578 @@
-may)
+may#may_v1)
@@ -580 +580 @@
-bee)
+bee#be_v)
@@ -583 +583 @@
-his)
+his#his_adj)
@@ -585 +585 @@
-houshold)
+houshold#household_n)
@@ -587 +587 @@
-servant)))))))))
+servant#servant_n)))))))))
@@ -595 +595 @@
-Also)
+Also#also_adv)
@@ -600 +600 @@
-mother))
+mother#mother_n1))
@@ -605 +605 @@
-I))
+I#i_pron))
@@ -607 +607 @@
-wold)
+wold#will_v1)
@@ -609 +609 @@
-desire)
+desire#desire_v)
@@ -612 +612 @@
-you))
+you#you_pron))
@@ -615 +615 @@
-to)
+to#to_prep)
@@ -617 +617 @@
-mark)
+mark#mark_v)
@@ -620 +620 @@
-wel))
+wel#well_adv))
@@ -623 +623 @@
-my)
+my#my_adj)
@@ -625 +625 @@
-letter)
+letter#letter_n1)
@@ -632 +632 @@
-that)
+that#that_conj)
@@ -638 +638 @@
-I))
+I#i_pron))
@@ -640 +640 @@
-sent)
+sent#send_v1)
@@ -643 +643 @@
-you))
+you#you_pron))
@@ -646 +646 @@
-by)
+by#by_prep)
@@ -649 +649 @@
-Mr.)
+Mr.#NA)
@@ -651 +651 @@
-Oughtred)))))))
+Oughtred#NA)))))))
@@ -659 +659 @@
-and)
+and#and_conj1)
@@ -662 +662 @@
-here))
+here#here_adv))
@@ -665 +665 @@
-I))
+I#i_pron))
@@ -667 +667 @@
-send)
+send#send_v1)
@@ -670 +670 @@
-you))
+you#you_pron))
@@ -673 +673 @@
-a)
+a#a_adj2)
@@ -675 +675 @@
-godly)
+godly#godly_adj)
@@ -677 +677 @@
-New)
+New#NA)
@@ -679 +679 @@
-Testament))
+Testament#NA))
@@ -682 +682 @@
-by)
+by#by_prep)
@@ -685 +685 @@
-this)
+this#this_pron)
@@ -687 +687 @@
-bearer)))
+bearer#bearer_n)))
@@ -695 +695 @@
-And)
+And#and_conj1)
@@ -698 +698 @@
-yf)
+yf#if_conj)
@@ -705 +705 @@
-the)
+the#the_adj)
@@ -707 +707 @@
-prologue))
+prologue#prologue_n))
@@ -709 +709 @@
-bee)
+bee#be_v)
@@ -712 +712 @@
-so)
+so#so_adv)
@@ -714 +714 @@
-small)
+small#small_adj)
@@ -717 +717 @@
-that)
+that#that_conj)
@@ -721 +721 @@
-ye))
+ye#ye_pron))
@@ -723 +723 @@
-can@)
+can@#can_v1)
@@ -725 +725 @@
-@not)
+@not#not_adv)
@@ -728 +728 @@
-wel))
+wel#well_adv))
@@ -730 +730 @@
-reade)
+reade#read_v)
@@ -733 +733 @@
-them))))))))
+them#them_pron))))))))
@@ -738 +738 @@
-ther))
+ther#there_adv))
@@ -740 +740 @@
-is)
+is#be_v)
@@ -744 +744 @@
-my)
+my#my_adj)
@@ -746 +746 @@
-fathers))
+fathers#father_n))
@@ -748 +748 @@
-book))
+book#book_n))
@@ -756 +756 @@
-and)
+and#and_conj1)
@@ -759 +759 @@
-they))
+they#they_pron))
@@ -761 +761 @@
-are)
+are#be_v)
@@ -763 +763 @@
-bothe)
+bothe#both_pron)
@@ -766 +766 @@
-one))
+one#one_adj))
@@ -774 +774 @@
-and)
+and#and_conj1)
@@ -778 +778 @@
-my)
+my#my_adj)
@@ -780 +780 @@
-fathers))
+fathers#father_n))
@@ -782 +782 @@
-book))
+book#book_n))
@@ -784 +784 @@
-hath)
+hath#have_v)
@@ -787 +787 @@
-the)
+the#the_adj)
@@ -789 +789 @@
-prologue)
+prologue#prologue_n)
@@ -792 +792 @@
-printed)
+printed#print_v)
@@ -795 +795 @@
-in)
+in#in_prep)
@@ -798 +798 @@
-bigger)
+bigger#big_adj)
@@ -800 +800 @@
-letters)))))
+letters#letter_n1)))))
@@ -809 +809 @@
-Yf)
+Yf#if_conj)
@@ -816 +816 @@
-it))
+it#it_pron))
@@ -818 +818 @@
-wil)
+wil#will_v1)
@@ -820 +820 @@
-please)
+please#please_v)
@@ -823 +823 @@
-you))
+you#you_pron))
@@ -826 +826 @@
-to)
+to#to_prep)
@@ -828 +828 @@
-read)
+read#read_v)
@@ -831 +831 @@
-the)
+the#the_adj)
@@ -833 +833 @@
-introducement))))))
+introducement#introducement_n))))))
@@ -838 +838 @@
-ye))
+ye#ye_pron))
@@ -840 +840 @@
-shal)
+shal#shall_v)
@@ -842 +842 @@
-see)
+see#see_v)
@@ -845 +845 @@
-marvelous)
+marvelous#marvellous_adj)
@@ -847 +847 @@
-things)
+things#thing_n1)
@@ -850 +850 @@
-hyd)
+hyd#hide_v1)
@@ -853 +853 @@
-in)
+in#in_prep)
@@ -856 +856 @@
-it)))))
+it#it_pron)))))
@@ -864 +864 @@
-And)
+And#and_conj1)
@@ -867 +867 @@
-as)
+as#as_adv)
@@ -870 +870 @@
-for)
+for#for_prep)
@@ -873 +873 @@
-the)
+the#the_adj)
@@ -875 +875 @@
-understanding)
+understanding#understanding_n)
@@ -878 +878 @@
-of)
+of#of_prep)
@@ -881 +881 @@
-it))))))
+it#it_pron))))))
@@ -885 +885 @@
-dout)
+dout#doubt_v)
@@ -887 +887 @@
-not)
+not#not_adv)
@@ -895 +895 @@
-for)
+for#for_prep)
@@ -898 +898 @@
-God))
+God#NA))
@@ -900 +900 @@
-wil)
+wil#will_v1)
@@ -902 +902 @@
-give)
+give#give_v)
@@ -905 +905 @@
-knowledge)
+knowledge#knowledge_n)
@@ -909 +909 @@
-to)
+to#to_prep)
@@ -912 +912 @@
-whom)))
+whom#whom_pron)))
@@ -920 +920 @@
-he))
+he#he_pron))
@@ -922 +922 @@
-will)
+will#will_v1)
@@ -924 +924 @@
-give)
+give#give_v)
@@ -927 +927 @@
-knowledg)
+knowledg#knowledge_n)
@@ -930 +930 @@
-of)
+of#of_prep)
@@ -933 +933 @@
-the)
+the#the_adj)
@@ -935 +935 @@
-Scriptures))))
+Scriptures#NA))))
@@ -940 +940 @@
-as)
+as#as_adv)
@@ -942 +942 @@
-soon)
+soon#soon_adv)
@@ -947 +947 @@
-to)
+to#to_prep)
@@ -950 +950 @@
-a)
+a#a_adj2)
@@ -952 +952 @@
-shepperd)))
+shepperd#shepherd_n)))
@@ -955 +955 @@
-as)
+as#as_adv)
@@ -958 +958 @@
-to)
+to#to_prep)
@@ -961 +961 @@
-a)
+a#a_adj2)
@@ -963 +963 @@
-priest))))
+priest#priest_n))))
@@ -968 +968 @@
-yf)
+yf#if_conj)
@@ -975 +975 @@
-he))
+he#he_pron))
@@ -977 +977 @@
-ask)
+ask#ask_v)
@@ -980 +980 @@
-knowledg))
+knowledg#knowledge_n))
@@ -983 +983 @@
-of)
+of#of_prep)
@@ -986 +986 @@
-God)))
+God#NA)))
@@ -989 +989 @@
-faithfully))))))))
+faithfully#faithfully_adv))))))))
@@ -995 +995 @@
-Wherfor))
+Wherfor#wherefore_adv))
@@ -1003 +1003 @@
-pray)
+pray#pray_v)
@@ -1006 +1006 @@
-to)
+to#to_prep)
@@ -1009 +1009 @@
-God))))
+God#NA))))
@@ -1014 +1014 @@
-and)
+and#and_conj1)
@@ -1019 +1019 @@
-desire)
+desire#desire_v)
@@ -1022 +1022 @@
-Jesus)
+Jesus#NA)
@@ -1024 +1024 @@
-Christ))
+Christ#NA))
@@ -1027 +1027 @@
-to)
+to#to_prep)
@@ -1029 +1029 @@
-pray)
+pray#pray_v)
@@ -1033 +1033 @@
-for)
+for#for_prep)
@@ -1036 +1036 @@
-you)))
+you#you_pron)))
@@ -1039 +1039 @@
-and)
+and#and_conj1)
@@ -1042 +1042 @@
-with)
+with#with_prep)
@@ -1045 +1045 @@
-you))))))))))
+you#you_pron))))))))))
@@ -1055 +1055 @@
-No)
+No#no_adj)
@@ -1057 +1057 @@
-more)))
+more#more_adj)))
@@ -1060 +1060 @@
-to)
+to#to_prep)
@@ -1063 +1063 @@
-you)))
+you#you_pron)))
@@ -1066 +1066 @@
-at)
+at#at_prep)
@@ -1069 +1069 @@
-this)
+this#this_pron)
@@ -1071 +1071 @@
-tyme)))
+tyme#time_n)))
@@ -1079 +1079 @@
-but)
+but#but_prep)
@@ -1082 +1082 @@
-God))
+God#NA))
@@ -1084 +1084 @@
-fill)
+fill#fill_v)
@@ -1087 +1087 @@
-you))
+you#you_pron))
@@ -1092 +1092 @@
-with)
+with#with_prep)
@@ -1095 +1095 @@
-al)
+al#all_adj)
@@ -1097 +1097 @@
-spiritual)
+spiritual#spiritual_adj)
@@ -1099 +1099 @@
-knowledge)))
+knowledge#knowledge_n)))
@@ -1104 +1104 @@
-to)
+to#to_prep)
@@ -1107 +1107 @@
-the)
+the#the_adj)
@@ -1109 +1109 @@
-glory)
+glory#glory_n)
@@ -1112 +1112 @@
-of)
+of#of_prep)
@@ -1116 +1116 @@
-God))
+God#NA))
@@ -1122 +1122 @@
-the)
+the#the_adj)
@@ -1124 +1124 @@
-helth)
+helth#health_n)
@@ -1127 +1127 @@
-of)
+of#of_prep)
@@ -1130 +1130 @@
-your)
+your#your_pron)
@@ -1132 +1132 @@
-soule)))))
+soule#soul_n)))))
@@ -1137 +1137 @@
-and)
+and#and_conj1)
@@ -1140 +1140 @@
-the)
+the#the_adj)
@@ -1142 +1142 @@
-profit)
+profit#profit_n)
@@ -1145 +1145 @@
-of)
+of#of_prep)
@@ -1148 +1148 @@
-your)
+your#your_pron)
@@ -1150 +1150 @@
-poor)
+poor#poor_adj)
@@ -1152 +1152 @@
-nieghbor)))))))))
+nieghbor#neighbour_n)))))))))
@@ -1160 +1160 @@
-Written)
+Written#write_v)
@@ -1163 +1163 @@
-at)
+at#at_prep)
@@ -1166 +1166 @@
-the)
+the#the_adj)
@@ -1168 +1168 @@
-Temple)))
+Temple#NA)))
@@ -1173 +1173 @@
-the)
+the#the_adj)
@@ -1175 +1175 @@
-12)
+12#twelve_adj)
@@ -1177 +1177 @@
-day)
+day#day_n)
@@ -1180 +1180 @@
-of)
+of#of_prep)
@@ -1183 +1183 @@
-ianuary))))
+ianuary#NA))))
@@ -1191 +1191 @@
-By)
+By#by_prep)
@@ -1194 +1194 @@
-your)
+your#your_pron)
@@ -1196 +1196 @@
-sonn)
+sonn#son_n1)
@@ -1201 +1201 @@
-Robert)
+Robert#NA)
@@ -1203 +1203 @@
-Plompton)))
+Plompton#NA)))
