--- tbarring-1630-e2-h.psd.vert	2025-09-10 13:50:35
+++ tbarring-1630-e2-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-Madame))
+Madame#madam_n))
@@ -17 +17 @@
-My)
+My#my_adj)
@@ -19 +19 @@
-first)
+first#first_adj)
@@ -21 +21 @@
-knowledg)
+knowledg#knowledge_n)
@@ -24 +24 @@
-of)
+of#of_prep)
@@ -28 +28 @@
-this)
+this#this_pron)
@@ -30 +30 @@
-bearer's))
+bearer's#bearer_n))
@@ -32 +32 @@
-journye)
+journye#journey_n)
@@ -35 +35 @@
-towards)
+towards#towards_prep)
@@ -38 +38 @@
-you))))))
+you#you_pron))))))
@@ -40 +40 @@
-made)
+made#make_v1)
@@ -44 +44 @@
-me))
+me#me_pron1))
@@ -46 +46 @@
-differ)
+differ#differ_v)
@@ -49 +49 @@
-my)
+my#my_adj)
@@ -51 +51 @@
-intentions)
+intentions#intention_n)
@@ -54 +54 @@
-of)
+of#of_prep)
@@ -58 +58 @@
-sending)
+sending#send_v1)
@@ -61 +61 @@
-purposely))
+purposely#purposely_adv))
@@ -64 +64 @@
-to)
+to#to_prep)
@@ -67 +67 @@
-you))))
+you#you_pron))))
@@ -72 +72 @@
-and)
+and#and_conj1)
@@ -76 +76 @@
-by)
+by#by_prep)
@@ -79 +79 @@
-him)))
+him#him_pron)))
@@ -81 +81 @@
-to)
+to#to_prep)
@@ -83 +83 @@
-present)
+present#present_v)
@@ -86 +86 @@
-you))
+you#you_pron))
@@ -89 +89 @@
-with)
+with#with_prep)
@@ -93 +93 @@
-so)
+so#so_adv)
@@ -95 +95 @@
-much))
+much#much_adj))
@@ -97 +97 @@
-balsom)
+balsom#balsam_n)
@@ -100 +100 @@
-as)
+as#as_adv)
@@ -111 +111 @@
-I))
+I#i_pron))
@@ -113 +113 @@
-could)
+could#can_v1)
@@ -115 +115 @@
-gett)
+gett#get_v)
@@ -118 +118 @@
-in)
+in#in_prep)
@@ -121 +121 @@
-London))))))
+London#NA))))))
@@ -127 +127 @@
-which))
+which#which_adj))
@@ -133 +133 @@
-allthough)
+allthough#although_conj)
@@ -140 +140 @@
-it))
+it#it_pron))
@@ -142 +142 @@
-be)
+be#be_v)
@@ -144 +144 @@
-not)
+not#not_adv)
@@ -147 +147 @@
-answarable)
+answarable#answerable_adj)
@@ -150 +150 @@
-in)
+in#in_prep)
@@ -153 +153 @@
-quantitye)))
+quantitye#quantity_n)))
@@ -156 +156 @@
-to)
+to#to_prep)
@@ -159 +159 @@
-my)
+my#my_adj)
@@ -161 +161 @@
-desyres)
+desyres#desire_n)
@@ -164 +164 @@
-and)
+and#and_conj1)
@@ -167 +167 @@
-indevors)))))))))
+indevors#endeavour_n)))))))))
@@ -172 +172 @@
-yet))
+yet#yet_adv))
@@ -175 +175 @@
-the)
+the#the_adj)
@@ -177 +177 @@
-qualytie))
+qualytie#quality_n))
@@ -179 +179 @@
-may)
+may#may_v1)
@@ -181 +181 @@
-render)
+render#render_v)
@@ -184 +184 @@
-som)
+som#some_pron)
@@ -186 +186 @@
-answarable)
+answarable#answerable_adj)
@@ -188 +188 @@
-satisfaction))))))))))))
+satisfaction#satisfaction_n))))))))))))
@@ -193 +193 @@
-haveing)
+haveing#have_v)
@@ -196 +196 @@
-with)
+with#with_prep)
@@ -199 +199 @@
-som)
+som#some_pron)
@@ -201 +201 @@
-labor)))
+labor#labour_n)))
@@ -203 +203 @@
-obtained)
+obtained#obtain_v)
@@ -206 +206 @@
-it))
+it#it_pron))
@@ -209 +209 @@
-of)
+of#of_prep)
@@ -212 +212 @@
-a)
+a#a_adj2)
@@ -214 +214 @@
-freind)
+freind#friend_n)
@@ -218 +218 @@
-who))
+who#who_pron))
@@ -225 +225 @@
-hath)
+hath#have_v)
@@ -227 +227 @@
-made)
+made#make_v1)
@@ -230 +230 @@
-greate)
+greate#great_adj)
@@ -232 +232 @@
-tryall)
+tryall#trial_n1)
@@ -235 +235 @@
-of)
+of#of_prep)
@@ -238 +238 @@
-it)))))))))
+it#it_pron)))))))))
@@ -247 +247 @@
-that)
+that#that_pron)
@@ -251 +251 @@
-which))
+which#which_adj))
@@ -258 +258 @@
-wants))))
+wants#want_v))))
@@ -261 +261 @@
-tharefore))
+tharefore#thereforth_adv))
@@ -263 +263 @@
-makes)
+makes#make_v1)
@@ -267 +267 @@
-the)
+the#the_adj)
@@ -269 +269 @@
-worthe)
+worthe#worth_n1)
@@ -272 +272 @@
-of)
+of#of_prep)
@@ -275 +275 @@
-the)
+the#the_adj)
@@ -277 +277 @@
-remaineder))))
+remaineder#remainder_n))))
@@ -280 +280 @@
-the)
+the#the_adj)
@@ -282 +282 @@
-greater)))
+greater#great_adj)))
@@ -291 +291 @@
-I))
+I#i_pron))
@@ -294 +294 @@
-wish)
+wish#wish_v)
@@ -296 +296 @@
-and)
+and#and_conj1)
@@ -298 +298 @@
-praye))
+praye#pray_v))
@@ -301 +301 @@
-for)
+for#for_prep)
@@ -305 +305 @@
-a)
+a#a_adj2)
@@ -307 +307 @@
-merrye)
+merrye#merry_adj)
@@ -309 +309 @@
-new)
+new#new_adj)
@@ -311 +311 @@
-yeear))
+yeear#year_n))
@@ -318 +318 @@
-unto)
+unto#unto_prep)
@@ -321 +321 @@
-you)))
+you#you_pron)))
@@ -326 +326 @@
-and)
+and#and_conj1)
@@ -329 +329 @@
-your)
+your#your_pron)
@@ -332 +332 @@
-safe)
+safe#safe_adj)
@@ -334 +334 @@
-and)
+and#and_conj1)
@@ -336 +336 @@
-comfortable))
+comfortable#comfortable_adj))
@@ -338 +338 @@
-being)
+being#being_n)
@@ -341 +341 @@
-emong)
+emong#among_prep)
@@ -344 +344 @@
-us)))
+us#us_pron)))
@@ -347 +347 @@
-in)
+in#in_prep)
@@ -350 +350 @@
-it)))))
+it#it_pron)))))
@@ -355 +355 @@
-and)
+and#and_conj1)
@@ -359 +359 @@
-so)
+so#so_adv)
@@ -361 +361 @@
-many))
+many#many_adj))
@@ -364 +364 @@
-as)
+as#as_adv)
@@ -373 +373 @@
-God))
+God#NA))
@@ -375 +375 @@
-shall)
+shall#shall_v)
@@ -377 +377 @@
-see)
+see#see_v)
@@ -383 +383 @@
-best))
+best#best_adj))
@@ -386 +386 @@
-for)
+for#for_prep)
@@ -389 +389 @@
-us))
+us#us_pron))
@@ -391 +391 @@
-to)
+to#to_prep)
@@ -393 +393 @@
-enjoy)
+enjoy#enjoy_v)
@@ -397 +397 @@
-such))
+such#such_adj))
@@ -400 +400 @@
-and)
+and#and_conj1)
@@ -404 +404 @@
-so)
+so#so_adv)
@@ -406 +406 @@
-greate))
+greate#great_adj))
@@ -408 +408 @@
-a)
+a#a_adj2)
@@ -410 +410 @@
-blessing)))))))))))
+blessing#blessing_n)))))))))))
@@ -419 +419 @@
-The)
+The#the_adj)
@@ -421 +421 @@
-children))
+children#child_n))
@@ -424 +424 @@
-of)
+of#of_prep)
@@ -427 +427 @@
-theire)
+theire#their_adj)
@@ -429 +429 @@
-owne)
+owne#own_adj)
@@ -431 +431 @@
-ernest)
+ernest#earnest_adj)
@@ -433 +433 @@
-desyre)))
+desyre#desire_n)))
@@ -435 +435 @@
-have)
+have#have_v)
@@ -437 +437 @@
-presented)
+presented#present_v)
@@ -440 +440 @@
-you))
+you#you_pron))
@@ -443 +443 @@
-with)
+with#with_prep)
@@ -446 +446 @@
-theire)
+theire#their_adj)
@@ -448 +448 @@
-owne)))
+owne#own_adj)))
@@ -453 +453 @@
-and)
+and#and_conj1)
@@ -456 +456 @@
-that))
+that#that_pron))
@@ -459 +459 @@
-meearly)
+meearly#merely_adv2)
@@ -461 +461 @@
-so))
+so#so_adv))
@@ -464 +464 @@
-in)
+in#in_prep)
@@ -467 +467 @@
-matter)
+matter#matter_n1)
@@ -469 +469 @@
-and)
+and#and_conj1)
@@ -471 +471 @@
-forme)
+forme#form_n)
@@ -474 +474 @@
-too)))))
+too#too_adv)))))
@@ -479 +479 @@
-even)
+even#even_adj)
@@ -481 +481 @@
-to)
+to#to_prep)
@@ -484 +484 @@
-my)
+my#my_adj)
@@ -486 +486 @@
-sonn)
+sonn#son_n1)
@@ -489 +489 @@
-Oliver))
+Oliver#NA))
@@ -493 +493 @@
-who))
+who#who_pron))
@@ -500 +500 @@
-would)
+would#will_v1)
@@ -502 +502 @@
-not)
+not#not_adv)
@@ -506 +506 @@
-so)
+so#so_adv)
@@ -508 +508 @@
-much))
+much#much_adj))
@@ -511 +511 @@
-as)))
+as#as_adv)))
@@ -513 +513 @@
-have)
+have#have_v)
@@ -517 +517 @@
-it))
+it#it_pron))
@@ -519 +519 @@
-lookt)
+lookt#look_v)
@@ -521 +521 @@
-on)
+on#on_adv)
@@ -524 +524 @@
-till)
+till#till_prep)
@@ -531 +531 @@
-he))
+he#he_pron))
@@ -533 +533 @@
-had)
+had#have_v)
@@ -535 +535 @@
-finished)))))))))
+finished#finish_v)))))))))
@@ -544 +544 @@
-You))
+You#you_pron))
@@ -546 +546 @@
-will)
+will#will_v1)
@@ -548 +548 @@
-please)
+please#please_v)
@@ -551 +551 @@
-to)
+to#to_prep)
@@ -553 +553 @@
-thinke)
+thinke#think_v2)
@@ -560 +560 @@
-I))
+I#i_pron))
@@ -562 +562 @@
-would)
+would#will_v1)
@@ -564 +564 @@
-not)
+not#not_adv)
@@ -566 +566 @@
-perswade)
+perswade#persuade_v)
@@ -569 +569 @@
-them))
+them#them_pron))
@@ -574 +574 @@
-so)
+so#so_adv)
@@ -576 +576 @@
-much)))
+much#much_adj)))
@@ -578 +578 @@
-to)
+to#to_prep)
@@ -580 +580 @@
-trouble)
+trouble#trouble_v)
@@ -583 +583 @@
-you))))))
+you#you_pron))))))
@@ -591 +591 @@
-but)
+but#but_prep)
@@ -594 +594 @@
-I))
+I#i_pron))
@@ -596 +596 @@
-was)
+was#be_v)
@@ -599 +599 @@
-unwilling)
+unwilling#unwilling_adj)
@@ -602 +602 @@
-to)
+to#to_prep)
@@ -604 +604 @@
-discorage)
+discorage#discourage_v)
@@ -607 +607 @@
-them))
+them#them_pron))
@@ -610 +610 @@
-in)
+in#in_prep)
@@ -613 +613 @@
-that)
+that#that_pron)
@@ -615 +615 @@
-loveing)
+loveing#loving_adj)
@@ -617 +617 @@
-indevor)
+indevor#endeavour_n)
@@ -621 +621 @@
-which))
+which#which_adj))
@@ -628 +628 @@
-was)
+was#be_v)
@@ -631 +631 @@
-so)
+so#so_adv)
@@ -633 +633 @@
-meearly))
+meearly#merely_adv2))
@@ -635 +635 @@
-spunn)
+spunn#spin_v)
@@ -638 +638 @@
-oute)
+oute#out_adv)
@@ -640 +640 @@
-of)
+of#of_prep)
@@ -643 +643 @@
-theire)
+theire#their_adj)
@@ -645 +645 @@
-owne)
+owne#own_adj)
@@ -647 +647 @@
-good)
+good#good_adj)
@@ -649 +649 @@
-natures)))))))))
+natures#nature_n)))))))))
@@ -658 +658 @@
-My)
+My#my_adj)
@@ -660 +660 @@
-wife))
+wife#wife_n))
@@ -662 +662 @@
-offers)
+offers#offer_v)
@@ -665 +665 @@
-her)
+her#her_pron2)
@@ -667 +667 @@
-dewtie)
+dewtie#duty_n)
@@ -670 +670 @@
-and)
+and#and_conj1)
@@ -673 +673 @@
-love))))
+love#love_n1))))
@@ -676 +676 @@
-to)
+to#to_prep)
@@ -679 +679 @@
-you)))
+you#you_pron)))
@@ -685 +685 @@
-humblye))
+humblye#humbly_adv))
@@ -687 +687 @@
-desyreing)
+desyreing#desire_v)
@@ -690 +690 @@
-your)
+your#your_pron)
@@ -692 +692 @@
-excuse))
+excuse#excuse_n))
@@ -695 +695 @@
-for)
+for#for_prep)
@@ -699 +699 @@
-her))
+her#her_pron1))
@@ -701 +701 @@
-not)
+not#not_adv)
@@ -703 +703 @@
-wrighting))))
+wrighting#write_v))))
@@ -708 +708 @@
-for)
+for#for_prep)
@@ -711 +711 @@
-that)
+that#that_conj)
@@ -715 +715 @@
-in)
+in#in_prep)
@@ -718 +718 @@
-good)
+good#good_adj)
@@ -720 +720 @@
-truth)))
+truth#truth_n)))
@@ -723 +723 @@
-her)
+her#her_pron2)
@@ -725 +725 @@
-present)
+present#present_adj)
@@ -727 +727 @@
-toyle))
+toyle#toil_n1))
@@ -729 +729 @@
-is)
+is#be_v)
@@ -732 +732 @@
-verye)
+verye#very_adj)
@@ -734 +734 @@
-greate)))))
+greate#great_adj)))))
@@ -743 +743 @@
-My)
+My#my_adj)
@@ -745 +745 @@
-brother)
+brother#brother_n)
@@ -748 +748 @@
-Riche))
+Riche#NA))
@@ -752 +752 @@
-remembers)
+remembers#remember_v)
@@ -755 +755 @@
-you))
+you#you_pron))
@@ -758 +758 @@
-lovingly))
+lovingly#lovingly_adv))
@@ -764 +764 @@
-who))
+who#who_pron))
@@ -771 +771 @@
-hath)
+hath#have_v)
@@ -773 +773 @@
-ben)
+ben#be_v)
@@ -776 +776 @@
-heear))
+heear#here_adv))
@@ -779 +779 @@
-theise)
+theise#these_pron)
@@ -781 +781 @@
-two)
+two#two_adj)
@@ -783 +783 @@
-dayes))
+dayes#day_n))
@@ -787 +787 @@
-oute)
+oute#out_adv)
@@ -789 +789 @@
-of)
+of#of_prep)
@@ -792 +792 @@
-his)
+his#his_adj)
@@ -794 +794 @@
-love)))
+love#love_n1)))
@@ -797 +797 @@
-and)
+and#and_conj1)
@@ -800 +800 @@
-upon)
+upon#upon_prep)
@@ -803 +803 @@
-a)
+a#a_adj2)
@@ -805 +805 @@
-busines)
+busines#business_n)
@@ -808 +808 @@
-relateing)
+relateing#relate_v)
@@ -811 +811 @@
-to)
+to#to_prep)
@@ -814 +814 @@
-my)
+my#my_adj)
@@ -816 +816 @@
-neice)
+neice#niece_n)
@@ -819 +819 @@
-Alltham)))))
+Alltham#NA)))))
@@ -823 +823 @@
-which))
+which#which_adj))
@@ -831 +831 @@
-you))
+you#you_pron))
@@ -833 +833 @@
-shall)
+shall#shall_v)
@@ -835 +835 @@
-know)
+know#know_v)
@@ -838 +838 @@
-shortly))))
+shortly#shortly_adv))))
@@ -843 +843 @@
-one)
+one#one_adj)
@@ -845 +845 @@
-Mr)
+Mr#NA)
@@ -847 +847 @@
-St.)
+St.#NA)
@@ -849 +849 @@
-Johns)))))))))
+Johns#NA)))))))))
@@ -857 +857 @@
-but)
+but#but_prep)
@@ -860 +860 @@
-it))
+it#it_pron))
@@ -862 +862 @@
-must)
+must#must_v1)
@@ -865 +865 @@
-yet))
+yet#yet_adv))
@@ -867 +867 @@
-not)
+not#not_adv)
@@ -869 +869 @@
-be)
+be#be_v)
@@ -871 +871 @@
-imparted)
+imparted#impart_v)
@@ -874 +874 @@
-to)
+to#to_prep)
@@ -877 +877 @@
-anye)))
+anye#any_adj)))
@@ -880 +880 @@
-because)
+because#because_adv)
@@ -887 +887 @@
-the)
+the#the_adj)
@@ -889 +889 @@
-success))
+success#success_n))
@@ -891 +891 @@
-is)
+is#be_v)
@@ -894 +894 @@
-doubtfull)))))
+doubtfull#doubtful_adj)))))
@@ -905 +905 @@
-He))
+He#he_pron))
@@ -907 +907 @@
-is)
+is#be_v)
@@ -911 +911 @@
-religious))
+religious#religious_adj))
@@ -917 +917 @@
-honest)))
+honest#honest_adj)))
@@ -923 +923 @@
-of)
+of#of_prep)
@@ -926 +926 @@
-sweetness)
+sweetness#sweetness_n)
@@ -929 +929 @@
-in)
+in#in_prep)
@@ -932 +932 @@
-nature))))))
+nature#nature_n))))))
@@ -938 +938 @@
-discreet))))
+discreet#discreet_adj))))
@@ -947 +947 @@
-his)
+his#his_adj)
@@ -949 +949 @@
-estate)
+estate#estate_n)
@@ -952 +952 @@
-in)
+in#in_prep)
@@ -955 +955 @@
-land))))
+land#land_n1))))
@@ -960 +960 @@
-som)
+som#some_pron)
@@ -962 +962 @@
-300@)
+300@#NA)
@@ -964 +964 @@
-@=li=)
+@=li=#pound_n1)
@@ -967 +967 @@
-by)
+by#by_prep)
@@ -970 +970 @@
-the)
+the#the_adj)
@@ -972 +972 @@
-yeear))))
+yeear#year_n))))
@@ -981 +981 @@
-his)
+his#his_adj)
@@ -983 +983 @@
-practise))
+practise#practice_n))
@@ -989 +989 @@
-I))
+I#i_pron))
@@ -991 +991 @@
-beleive))
+beleive#believe_v))
@@ -994 +994 @@
-neear)
+neear#near_adv2)
@@ -996 +996 @@
-double))
+double#double_adj1))
@@ -1005 +1005 @@
-handsom)
+handsom#handsome_adj)
@@ -1008 +1008 @@
-for)
+for#for_prep)
@@ -1011 +1011 @@
-person))))
+person#person_n))))
@@ -1017 +1017 @@
-probable)
+probable#probable_adj)
@@ -1020 +1020 @@
-to)
+to#to_prep)
@@ -1022 +1022 @@
-rise))))
+rise#rise_v))))
@@ -1029 +1029 @@
-my)
+my#my_adj)
@@ -1031 +1031 @@
-Lord)
+Lord#lord_n)
@@ -1034 +1034 @@
-Bedford@))
+Bedford@#NA))
@@ -1036 +1036 @@
-@s)
+@s#NA)
@@ -1040 +1040 @@
-only)
+only#only_adj)
@@ -1042 +1042 @@
-favorite)
+favorite#favourite_n)
@@ -1048 +1048 @@
-who))
+who#who_pron))
@@ -1055 +1055 @@
-promises)
+promises#promise_v)
@@ -1058 +1058 @@
-that)
+that#that_conj)
@@ -1062 +1062 @@
-nothing))
+nothing#nothing_pron))
@@ -1064 +1064 @@
-shall)
+shall#shall_v)
@@ -1066 +1066 @@
-hinder)
+hinder#hinder_v)
@@ -1069 +1069 @@
-it))
+it#it_pron))
@@ -1072 +1072 @@
-for)
+for#for_prep)
@@ -1075 +1075 @@
-joyncture)))
+joyncture#jointure_n)))
@@ -1078 +1078 @@
-if)
+if#if_conj)
@@ -1085 +1085 @@
-his)
+his#his_adj)
@@ -1087 +1087 @@
-estate))
+estate#estate_n))
@@ -1089 +1089 @@
-will)
+will#will_v1)
@@ -1091 +1091 @@
-make)
+make#make_v1)
@@ -1095 +1095 @@
-it))
+it#it_pron))
@@ -1098 +1098 @@
-good))))))))))))
+good#good_adj))))))))))))
@@ -1107 +1107 @@
-I))
+I#i_pron))
@@ -1109 +1109 @@
-thinke)
+thinke#think_v2)
@@ -1116 +1116 @@
-it@))
+it@#it_pron))
@@ -1118 +1118 @@
-@'s)
+@'s#be_v)
@@ -1120 +1120 @@
-not)
+not#not_adv)
@@ -1123 +1123 @@
-to)
+to#to_prep)
@@ -1125 +1125 @@
-be)
+be#be_v)
@@ -1127 +1127 @@
-slighted))
+slighted#slight_v))
@@ -1130 +1130 @@
-considering)
+considering#consider_v)
@@ -1138 +1138 @@
-his)
+his#his_adj)
@@ -1140 +1140 @@
-present)
+present#present_adj)
@@ -1142 +1142 @@
-meanes))
+meanes#mean_n2))
@@ -1144 +1144 @@
-are)
+are#be_v)
@@ -1147 +1147 @@
-so)
+so#so_adv)
@@ -1151 +1151 @@
-competent)
+competent#competent_adj)
@@ -1154 +1154 @@
-by)
+by#by_prep)
@@ -1157 +1157 @@
-his)
+his#his_adj)
@@ -1159 +1159 @@
-prefession)))))
+prefession#profession_n)))))
@@ -1162 +1162 @@
-and)
+and#and_conj1)
@@ -1166 +1166 @@
-his)
+his#his_adj)
@@ -1170 +1170 @@
-likelyhood)
+likelyhood#likelihood_n)
@@ -1173 +1173 @@
-to)
+to#to_prep)
@@ -1175 +1175 @@
-rise)))
+rise#rise_v)))
@@ -1178 +1178 @@
-so)
+so#so_adv)
@@ -1180 +1180 @@
-greate))
+greate#great_adj))
@@ -1185 +1185 @@
-considring)
+considring#consider_v)
@@ -1189 +1189 @@
-how))
+how#how_adv))
@@ -1197 +1197 @@
-he))
+he#he_pron))
@@ -1199 +1199 @@
-is)
+is#be_v)
@@ -1201 +1201 @@
-befreinded)))))))))))
+befreinded#befriend_v)))))))))))
@@ -1210 +1210 @@
-I))
+I#i_pron))
@@ -1212 +1212 @@
-desyre)
+desyre#desire_v)
@@ -1215 +1215 @@
-to)
+to#to_prep)
@@ -1217 +1217 @@
-be)
+be#be_v)
@@ -1219 +1219 @@
-remembred))
+remembred#remember_v))
@@ -1224 +1224 @@
-and)
+and#and_conj1)
@@ -1227 +1227 @@
-so))
+so#so_adv))
@@ -1229 +1229 @@
-doe)
+doe#do_v)
@@ -1232 +1232 @@
-my)
+my#my_adj)
@@ -1234 +1234 @@
-wife)
+wife#wife_n)
@@ -1237 +1237 @@
-and)
+and#and_conj1)
@@ -1240 +1240 @@
-sisters)))))
+sisters#sister_n)))))
@@ -1245 +1245 @@
-to)
+to#to_prep)
@@ -1249 +1249 @@
-owr)
+owr#our_pron)
@@ -1251 +1251 @@
-brother)
+brother#brother_n)
@@ -1254 +1254 @@
-Gerard)))
+Gerard#NA)))
@@ -1257 +1257 @@
-and)
+and#and_conj1)
@@ -1260 +1260 @@
-our)
+our#our_pron)
@@ -1262 +1262 @@
-sisters)))
+sisters#sister_n)))
@@ -1267 +1267 @@
-and)
+and#and_conj1)
@@ -1270 +1270 @@
-my)
+my#my_adj)
@@ -1272 +1272 @@
-neice)
+neice#niece_n)
@@ -1275 +1275 @@
-and)
+and#and_conj1)
@@ -1278 +1278 @@
-uncle)))
+uncle#uncle_n)))
@@ -1283 +1283 @@
-and)
+and#and_conj1)
@@ -1286 +1286 @@
-cosen)
+cosen#cousin_n)
@@ -1289 +1289 @@
-Brewster))))
+Brewster#NA))))
@@ -1295 +1295 @@
-who))
+who#who_pron))
@@ -1301 +1301 @@
-I))
+I#i_pron))
@@ -1303 +1303 @@
-assure)
+assure#assure_v)
@@ -1306 +1306 @@
-me))
+me#me_pron1))
@@ -1314 +1314 @@
-is)
+is#be_v)
@@ -1317 +1317 @@
-now))
+now#now_adv))
@@ -1320 +1320 @@
-with)
+with#with_prep)
@@ -1323 +1323 @@
-you)))))))))))
+you#you_pron)))))))))))
@@ -1331 +1331 @@
-And)
+And#and_conj1)
@@ -1334 +1334 @@
-thus))
+thus#thus_adv))
@@ -1337 +1337 @@
-I))
+I#i_pron))
@@ -1339 +1339 @@
-kiss)
+kiss#kiss_v)
@@ -1342 +1342 @@
-your)
+your#your_pron)
@@ -1344 +1344 @@
-hands)))
+hands#hand_n)))
@@ -1350 +1350 @@
-and)
+and#and_conj1)
@@ -1354 +1354 @@
-pray)
+pray#pray_v)
@@ -1357 +1357 @@
-to)
+to#to_prep)
@@ -1360 +1360 @@
-God)))
+God#NA)))
@@ -1363 +1363 @@
-to)
+to#to_prep)
@@ -1365 +1365 @@
-bless)
+bless#bless_v1)
@@ -1369 +1369 @@
-you))
+you#you_pron))
@@ -1372 +1372 @@
-and)
+and#and_conj1)
@@ -1375 +1375 @@
-us)
+us#us_pron)
@@ -1378 +1378 @@
-all)))))))
+all#all_adj)))))))
@@ -1384 +1384 @@
-and)
+and#and_conj1)
@@ -1388 +1388 @@
-am)
+am#be_v)
@@ -1391 +1391 @@
-allwayes))
+allwayes#always_adv))
@@ -1394 +1394 @@
-Your)
+Your#your_pron)
@@ -1397 +1397 @@
-most)
+most#most_adj)
@@ -1399 +1399 @@
-dewtyfull)
+dewtyfull#dutiful_adj)
@@ -1402 +1402 @@
-and)
+and#and_conj1)
@@ -1405 +1405 @@
-loving))))
+loving#loving_adj))))
@@ -1407 +1407 @@
-sonn)
+sonn#son_n1)
@@ -1410 +1410 @@
-Thomas)
+Thomas#NA)
@@ -1412 +1412 @@
-Barrington))))
+Barrington#NA))))
@@ -1418 +1418 @@
-Hatfield)
+Hatfield#NA)
@@ -1420 +1420 @@
-Broad)
+Broad#NA)
@@ -1422 +1422 @@
-Oak)
+Oak#NA)
@@ -1430 +1430 @@
-December)
+December#NA)
@@ -1432 +1432 @@
-29)
+29#NA)
@@ -1436 +1436 @@
-1629))
+1629#NA))
@@ -1445 +1445 @@
-To)
+To#to_prep)
@@ -1449 +1449 @@
-the)
+the#the_adj)
@@ -1451 +1451 @@
-honourable))
+honourable#honourable_adj))
@@ -1454 +1454 @@
-and)
+and#and_conj1)
@@ -1457 +1457 @@
-my)
+my#my_adj)
@@ -1460 +1460 @@
-verie)
+verie#very_adj)
@@ -1462 +1462 @@
-good))
+good#good_adj))
@@ -1464 +1464 @@
-mother)))
+mother#mother_n1)))
@@ -1467 +1467 @@
-the)
+the#the_adj)
@@ -1469 +1469 @@
-Lady)
+Lady#lady_n)
@@ -1472 +1472 @@
-Johan)
+Johan#NA)
@@ -1474 +1474 @@
-Barrington)))))
+Barrington#NA)))))
