--- thoward-1517-e1-h.psd.vert	2025-09-10 13:50:35
+++ thoward-1517-e1-h.lem.vert	2025-09-10 13:50:45
@@ -18 +18 @@
-Right)
+Right#right_adv)
@@ -20 +20 @@
-highe)
+highe#high_adj)
@@ -23 +23 @@
-and)
+and#and_conj1)
@@ -26 +26 @@
-mightie))))
+mightie#mighty_adj))))
@@ -28 +28 @@
-Prince))
+Prince#prince_n))
@@ -33 +33 @@
-so))
+so#so_adv))
@@ -36 +36 @@
-itt))
+itt#it_pron))
@@ -38 +38 @@
-is)
+is#be_v)
@@ -41 +41 @@
-that)
+that#that_conj)
@@ -46 +46 @@
-latelie))
+latelie#lately_adv))
@@ -49 +49 @@
-I))
+I#i_pron))
@@ -51 +51 @@
-sent)
+sent#send_v1)
@@ -54 +54 @@
-unto)
+unto#unto_prep)
@@ -57 +57 @@
-you)))
+you#you_pron)))
@@ -60 +60 @@
-Rougecrosse)
+Rougecrosse#NA)
@@ -63 +63 @@
-pursevant)
+pursevant#pursuivant_n)
@@ -66 +66 @@
-at)
+at#at_prep)
@@ -69 +69 @@
-Armes))))))
+Armes#arms_n))))))
@@ -72 +72 @@
-and)
+and#and_conj1)
@@ -78 +78 @@
-by)
+by#by_prep)
@@ -81 +81 @@
-him)))
+him#him_pron)))
@@ -83 +83 @@
-advertised)
+advertised#advertise_v)
@@ -86 +86 @@
-yo=r=)
+yo=r=#your_pron)
@@ -88 +88 @@
-Grace))
+Grace#grace_n))
@@ -91 +91 @@
-that)
+that#that_conj)
@@ -95 +95 @@
-I)
+I#i_pron)
@@ -97 +97 @@
-and)
+and#and_conj1)
@@ -99 +99 @@
-other)
+other#other_adj)
@@ -105 +105 @@
-my)
+my#my_adj)
@@ -107 +107 @@
-Sovereigne)
+Sovereigne#sovereign_n)
@@ -109 +109 @@
-Lord's))
+Lord's#lord_n))
@@ -111 +111 @@
-subjects)))
+subjects#subject_n)))
@@ -115 +115 @@
-were)
+were#be_v)
@@ -117 +117 @@
-come)
+come#come_v)
@@ -120 +120 @@
-to)
+to#to_prep)
@@ -123 +123 @@
-represse)
+represse#repress_v1)
@@ -125 +125 @@
-and)
+and#and_conj1)
@@ -127 +127 @@
-resist))
+resist#resist_v))
@@ -130 +130 @@
-your)
+your#your_pron)
@@ -132 +132 @@
-Invasions)
+Invasions#invasion_n)
@@ -135 +135 @@
-of)
+of#of_prep)
@@ -138 +138 @@
-this)
+this#this_pron)
@@ -141 +141 @@
-the)
+the#the_adj)
@@ -143 +143 @@
-Kinges)
+Kinges#king_n)
@@ -146 +146 @@
-my)
+my#my_adj)
@@ -148 +148 @@
-Sovereigne)
+Sovereigne#sovereign_n)
@@ -150 +150 @@
-Lordes)))
+Lordes#lord_n)))
@@ -152 +152 @@
-realme)))))))))))
+realme#realm_n)))))))))))
@@ -160 +160 @@
-And)
+And#and_conj1)
@@ -163 +163 @@
-for)
+for#for_prep)
@@ -166 +166 @@
-that)
+that#that_pron)
@@ -168 +168 @@
-intent)))
+intent#intent_n)))
@@ -171 +171 @@
-I))
+I#i_pron))
@@ -173 +173 @@
-offered)
+offered#offer_v)
@@ -176 +176 @@
-to)
+to#to_prep)
@@ -178 +178 @@
-give)
+give#give_v)
@@ -181 +181 @@
-you))
+you#you_pron))
@@ -184 +184 @@
-battell))
+battell#battle_n))
@@ -187 +187 @@
-on)
+on#on_prep)
@@ -190 +190 @@
-this)
+this#this_pron)
@@ -192 +192 @@
-half)))
+half#half_n)))
@@ -197 +197 @@
-Friday)
+Friday#NA)
@@ -201 +201 @@
-next))
+next#next_adj))
@@ -203 +203 @@
-coming))))
+coming#come_v))))
@@ -209 +209 @@
-which)
+which#which_adj)
@@ -211 +211 @@
-my)
+my#my_adj)
@@ -213 +213 @@
-message))
+message#message_n))
@@ -219 +219 @@
-your)
+your#your_pron)
@@ -221 +221 @@
-Grace))
+Grace#grace_n))
@@ -223 +223 @@
-tooke)
+tooke#take_v)
@@ -226 +226 @@
-pleasure)
+pleasure#pleasure_n)
@@ -231 +231 @@
-to)
+to#to_prep)
@@ -233 +233 @@
-heare)))
+heare#hear_v)))
@@ -236 +236 @@
-as)
+as#as_adv)
@@ -247 +247 @@
-I))
+I#i_pron))
@@ -249 +249 @@
-am)
+am#be_v)
@@ -251 +251 @@
-enformed))))))
+enformed#inform_v))))))
@@ -259 +259 @@
-And)
+And#and_conj1)
@@ -262 +262 @@
-by)
+by#by_prep)
@@ -265 +265 @@
-your)
+your#your_pron)
@@ -267 +267 @@
-herauld)
+herauld#herald_n)
@@ -270 +270 @@
-Isley))))
+Isley#NA))))
@@ -273 +273 @@
-ye))
+ye#ye_pron))
@@ -275 +275 @@
-made)
+made#make_v1)
@@ -278 +278 @@
-answere))
+answere#answer_n))
@@ -281 +281 @@
-that)
+that#that_conj)
@@ -286 +286 @@
-you))
+you#you_pron))
@@ -288 +288 @@
-were)
+were#be_v)
@@ -291 +291 @@
-right)
+right#right_adv)
@@ -293 +293 @@
-joyeous)
+joyeous#joyous_adj)
@@ -296 +296 @@
-of)
+of#of_prep)
@@ -299 +299 @@
-my)
+my#my_adj)
@@ -301 +301 @@
-desire)))))
+desire#desire_n)))))
@@ -306 +306 @@
-and)
+and#and_conj1)
@@ -311 +311 @@
-would)
+would#will_v1)
@@ -313 +313 @@
-not)
+not#not_adv)
@@ -315 +315 @@
-faile)
+faile#fail_v)
@@ -319 +319 @@
-to)
+to#to_prep)
@@ -321 +321 @@
-accomplishe)
+accomplishe#accomplish_v)
@@ -324 +324 @@
-the)
+the#the_adj)
@@ -326 +326 @@
-same)))
+same#same_adj)))
@@ -329 +329 @@
-and)
+and#and_conj1)
@@ -332 +332 @@
-to)
+to#to_prep)
@@ -334 +334 @@
-abide)
+abide#abide_v)
@@ -337 +337 @@
-me))
+me#me_pron1))
@@ -340 +340 @@
-there)
+there#there_adv)
@@ -346 +346 @@
-where))
+where#where_adv))
@@ -354 +354 @@
-you))
+you#you_pron))
@@ -356 +356 @@
-were)
+were#be_v)
@@ -359 +359 @@
-at)
+at#at_prep)
@@ -362 +362 @@
-the)
+the#the_adj)
@@ -364 +364 @@
-tyme)
+tyme#time_n)
@@ -367 +367 @@
-of)
+of#of_prep)
@@ -370 +370 @@
-my)
+my#my_adj)
@@ -372 +372 @@
-message)
+message#message_n)
@@ -376 +376 @@
-so))
+so#so_adv))
@@ -378 +378 @@
-shewed)
+shewed#show_v)
@@ -381 +381 @@
-unto)
+unto#unto_prep)
@@ -384 +384 @@
-your)
+your#your_pron)
@@ -386 +386 @@
-Grace))))))))))))))))))
+Grace#grace_n))))))))))))))))))
@@ -394 +394 @@
-And)
+And#and_conj1)
@@ -397 +397 @@
-albeit)
+albeit#albeit_conj)
@@ -404 +404 @@
-it))
+it#it_pron))
@@ -406 +406 @@
-hathe)
+hathe#have_v)
@@ -408 +408 @@
-pleased)
+pleased#please_v)
@@ -411 +411 @@
-yow))
+yow#you_pron))
@@ -415 +415 @@
-to)
+to#to_prep)
@@ -417 +417 @@
-change)
+change#change_v)
@@ -420 +420 @@
-your)
+your#your_pron)
@@ -422 +422 @@
-said)
+said#said_adj)
@@ -424 +424 @@
-promise)))
+promise#promise_n)))
@@ -427 +427 @@
-and)
+and#and_conj1)
@@ -432 +432 @@
-put)
+put#put_v)
@@ -435 +435 @@
-your)
+your#your_pron)
@@ -437 +437 @@
-self))
+self#self_pron))
@@ -440 +440 @@
-into)
+into#into_prep)
@@ -443 +443 @@
-a)
+a#a_adj2)
@@ -445 +445 @@
-ground)
+ground#ground_n)
@@ -448 +448 @@
-more)
+more#more_adj)
@@ -450 +450 @@
-like)
+like#like_adj)
@@ -453 +453 @@
-a)
+a#a_adj2)
@@ -455 +455 @@
-fortresse)
+fortresse#fortress_n)
@@ -458 +458 @@
-or)
+or#or_conj2)
@@ -461 +461 @@
-Campe))))
+Campe#camp_n2))))
@@ -464 +464 @@
-then)
+then#than_conj)
@@ -467 +467 @@
-upon)
+upon#upon_prep)
@@ -470 +470 @@
-any)
+any#any_adj)
@@ -472 +472 @@
-indifferent)
+indifferent#indifferent_adj1)
@@ -474 +474 @@
-ground)
+ground#ground_n)
@@ -477 +477 @@
-for)
+for#for_prep)
@@ -480 +480 @@
-battel)
+battel#battle_n)
@@ -488 +488 @@
-to)
+to#to_prep)
@@ -490 +490 @@
-be)
+be#be_v)
@@ -492 +492 @@
-tryede)))))))))))))))))
+tryede#try_v)))))))))))))))))
@@ -498 +498 @@
-wherefore))
+wherefore#wherefore_adv))
@@ -506 +506 @@
-considering)
+considering#consider_v)
@@ -513 +513 @@
-the)
+the#the_adj)
@@ -515 +515 @@
-day)
+day#day_n)
@@ -518 +518 @@
-apointed)))
+apointed#appoint_v)))
@@ -520 +520 @@
-is)
+is#be_v)
@@ -524 +524 @@
-so)
+so#so_adv)
@@ -526 +526 @@
-nighe)))
+nighe#nigh_adv)))
@@ -530 +530 @@
-approching))))
+approching#approach_v))))
@@ -533 +533 @@
-I))
+I#i_pron))
@@ -535 +535 @@
-desire)
+desire#desire_v)
@@ -538 +538 @@
-now))
+now#now_adv))
@@ -541 +541 @@
-of)
+of#of_prep)
@@ -544 +544 @@
-your)
+your#your_pron)
@@ -546 +546 @@
-Grace)))
+Grace#grace_n)))
@@ -549 +549 @@
-for)
+for#for_prep)
@@ -552 +552 @@
-the)
+the#the_adj)
@@ -554 +554 @@
-accomplishment)
+accomplishment#accomplishment_n)
@@ -557 +557 @@
-of)
+of#of_prep)
@@ -560 +560 @@
-your)
+your#your_pron)
@@ -562 +562 @@
-honorable)
+honorable#honourable_adj)
@@ -564 +564 @@
-promise)))))
+promise#promise_n)))))
@@ -571 +571 @@
-yow))
+yow#you_pron))
@@ -573 +573 @@
-will)
+will#will_v1)
@@ -575 +575 @@
-despose)
+despose#despose_v)
@@ -578 +578 @@
-your)
+your#your_pron)
@@ -580 +580 @@
-self))
+self#self_pron))
@@ -583 +583 @@
-for)
+for#for_prep)
@@ -586 +586 @@
-yo=r=)
+yo=r=#your_pron)
@@ -588 +588 @@
-parte)))
+parte#part_n)))
@@ -593 +593 @@
-like)
+like#like_adj)
@@ -595 +595 @@
-as)
+as#as_adv)
@@ -606 +606 @@
-I))
+I#i_pron))
@@ -608 +608 @@
-shall)
+shall#shall_v)
@@ -610 +610 @@
-doo)
+doo#do_v)
@@ -613 +613 @@
-for)
+for#for_prep)
@@ -616 +616 @@
-myne))))))
+myne#mine_adj))))))
@@ -623 +623 @@
-to)
+to#to_prep)
@@ -625 +625 @@
-be)
+be#be_v)
@@ -628 +628 @@
-to_morrowe))
+to_morrowe#tomorrow_adv))
@@ -631 +631 @@
-with)
+with#with_prep)
@@ -634 +634 @@
-your)
+your#your_pron)
@@ -636 +636 @@
-host)))
+host#host_n1)))
@@ -639 +639 @@
-in)
+in#in_prep)
@@ -642 +642 @@
-your)
+your#your_pron)
@@ -644 +644 @@
-side)
+side#side_n1)
@@ -647 +647 @@
-of)
+of#of_prep)
@@ -650 +650 @@
-the)
+the#the_adj)
@@ -652 +652 @@
-plaine)
+plaine#plain_n1)
@@ -655 +655 @@
-of)
+of#of_prep)
@@ -658 +658 @@
-Milfeild)))))))
+Milfeild#NA)))))))
@@ -663 +663 @@
-in)
+in#in_prep)
@@ -666 +666 @@
-likewise)
+likewise#likewise_adv)
@@ -669 +669 @@
-as)
+as#as_adv)
@@ -681 +681 @@
-I))
+I#i_pron))
@@ -683 +683 @@
-shall)
+shall#shall_v)
@@ -685 +685 @@
-doo)
+doo#do_v)
@@ -688 +688 @@
-for)
+for#for_prep)
@@ -691 +691 @@
-myne))))
+myne#mine_adj))))
@@ -696 +696 @@
-and)
+and#and_conj1)
@@ -701 +701 @@
-shal@)
+shal@#shall_v)
@@ -703 +703 @@
-@be)
+@be#be_v)
@@ -706 +706 @@
-with)
+with#with_prep)
@@ -709 +709 @@
-the)
+the#the_adj)
@@ -711 +711 @@
-subjects)
+subjects#subject_n)
@@ -714 +714 @@
-of)
+of#of_prep)
@@ -717 +717 @@
-my)
+my#my_adj)
@@ -719 +719 @@
-sovereign)
+sovereign#sovereign_n)
@@ -721 +721 @@
-Lord)))))
+Lord#lord_n)))))
@@ -724 +724 @@
-on)
+on#on_prep)
@@ -727 +727 @@
-my)
+my#my_adj)
@@ -729 +729 @@
-side)
+side#side_n1)
@@ -732 +732 @@
-of)
+of#of_prep)
@@ -735 +735 @@
-the)
+the#the_adj)
@@ -737 +737 @@
-playne)
+playne#plain_n1)
@@ -740 +740 @@
-of)
+of#of_prep)
@@ -743 +743 @@
-the)
+the#the_adj)
@@ -745 +745 @@
-said)
+said#said_adj)
@@ -747 +747 @@
-feild)))))))
+feild#field_n1)))))))
@@ -750 +750 @@
-to)
+to#to_prep)
@@ -752 +752 @@
-give)
+give#give_v)
@@ -755 +755 @@
-yow))
+yow#you_pron))
@@ -758 +758 @@
-battell))
+battell#battle_n))
@@ -761 +761 @@
-betwixt)
+betwixt#betwixt_prep)
@@ -765 +765 @@
-xij.)
+xij.#twelve_adj)
@@ -768 +768 @@
-of)
+of#of_prep)
@@ -771 +771 @@
-the)
+the#the_adj)
@@ -773 +773 @@
-clock))))
+clock#clock_n1))))
@@ -776 +776 @@
-and)
+and#and_conj1)
@@ -779 +779 @@
-iij.)
+iij.#three_adj)
@@ -782 +782 @@
-in)
+in#in_prep)
@@ -785 +785 @@
-the)
+the#the_adj)
@@ -787 +787 @@
-afternoone)))))))
+afternoone#afternoon_n)))))))
@@ -792 +792 @@
-upon)
+upon#upon_prep)
@@ -795 +795 @@
-sufficient)
+sufficient#sufficient_adj)
@@ -797 +797 @@
-warning)
+warning#warning_n1)
@@ -806 +806 @@
-by)
+by#by_prep)
@@ -809 +809 @@
-yow)))
+yow#you_pron)))
@@ -811 +811 @@
-to)
+to#to_prep)
@@ -813 +813 @@
-be)
+be#be_v)
@@ -815 +815 @@
-given)
+given#give_v)
@@ -818 +818 @@
-by)
+by#by_prep)
@@ -822 +822 @@
-viij=th=.)
+viij=th=.#NA)
@@ -824 +824 @@
-or)
+or#or_conj2)
@@ -826 +826 @@
-ix))
+ix#nine_adj))
@@ -829 +829 @@
-of)
+of#of_prep)
@@ -832 +832 @@
-the)
+the#the_adj)
@@ -834 +834 @@
-Clock)))
+Clock#clock_n1)))
@@ -837 +837 @@
-in)
+in#in_prep)
@@ -840 +840 @@
-the)
+the#the_adj)
@@ -842 +842 @@
-Morning)))))
+Morning#morning_n)))))
@@ -845 +845 @@
-by)
+by#by_prep)
@@ -848 +848 @@
-the)
+the#the_adj)
@@ -850 +850 @@
-said)
+said#said_adj)
@@ -852 +852 @@
-Pursevant))))))))))))))))))))
+Pursevant#pursuivant_n))))))))))))))))))))
@@ -860 +860 @@
-And)
+And#and_conj1)
@@ -863 +863 @@
-like)
+like#like_adj)
@@ -865 +865 @@
-as)
+as#as_adv)
@@ -877 +877 @@
-I))
+I#i_pron))
@@ -880 +880 @@
-and)
+and#and_conj1)
@@ -883 +883 @@
-other)
+other#other_adj)
@@ -885 +885 @@
-Noblemen)
+Noblemen#nobleman_n)
@@ -888 +888 @@
-my)
+my#my_adj)
@@ -890 +890 @@
-company)))))
+company#company_n)))))
@@ -892 +892 @@
-binde)
+binde#bind_v)
@@ -895 +895 @@
-us))
+us#us_pron))
@@ -898 +898 @@
-by)
+by#by_prep)
@@ -901 +901 @@
-our)
+our#our_pron)
@@ -903 +903 @@
-writeing)
+writeing#writing_n)
@@ -906 +906 @@
-subscribed)
+subscribed#subscribe_v)
@@ -909 +909 @@
-with)
+with#with_prep)
@@ -912 +912 @@
-our)
+our#our_pron)
@@ -914 +914 @@
-hands))))))
+hands#hand_n))))))
@@ -917 +917 @@
-to)
+to#to_prep)
@@ -919 +919 @@
-keepe)
+keepe#keep_v)
@@ -922 +922 @@
-the)
+the#the_adj)
@@ -924 +924 @@
-same)
+same#same_adj)
@@ -926 +926 @@
-tyme))
+tyme#time_n))
@@ -929 +929 @@
-to)
+to#to_prep)
@@ -932 +932 @@
-the)
+the#the_adj)
@@ -934 +934 @@
-intent)
+intent#intent_n)
@@ -938 +938 @@
-above))
+above#above_adv))
@@ -940 +940 @@
-said))))))))
+said#say_v1))))))))
@@ -945 +945 @@
-if)
+if#if_conj)
@@ -954 +954 @@
-'t))
+'t#it_pron))
@@ -958 +958 @@
-may)
+may#may_v1)
@@ -960 +960 @@
-like)
+like#like_v1)
@@ -963 +963 @@
-your)
+your#your_pron)
@@ -965 +965 @@
-Grace))
+Grace#grace_n))
@@ -968 +968 @@
-by)
+by#by_prep)
@@ -971 +971 @@
-your)
+your#your_pron)
@@ -973 +973 @@
-honorable)
+honorable#honourable_adj)
@@ -975 +975 @@
-Letters)
+Letters#letter_n1)
@@ -978 +978 @@
-subscribed)
+subscribed#subscribe_v)
@@ -981 +981 @@
-with)
+with#with_prep)
@@ -984 +984 @@
-your)
+your#your_pron)
@@ -986 +986 @@
-hand))))))
+hand#hand_n))))))
@@ -989 +989 @@
-to)
+to#to_prep)
@@ -991 +991 @@
-bind)
+bind#bind_v)
@@ -994 +994 @@
-yo=r=)
+yo=r=#your_pron)
@@ -996 +996 @@
-Grace))
+Grace#grace_n))
@@ -999 +999 @@
-for)
+for#for_prep)
@@ -1002 +1002 @@
-the)
+the#the_adj)
@@ -1004 +1004 @@
-accomplishment)
+accomplishment#accomplishment_n)
@@ -1007 +1007 @@
-of)
+of#of_prep)
@@ -1010 +1010 @@
-this)
+this#this_pron)
@@ -1012 +1012 @@
-desire)))))))))
+desire#desire_n)))))))))
@@ -1017 +1017 @@
-trusting)
+trusting#trust_v)
@@ -1020 +1020 @@
-that)
+that#that_conj)
@@ -1024 +1024 @@
-yow))
+yow#you_pron))
@@ -1026 +1026 @@
-will)
+will#will_v1)
@@ -1028 +1028 @@
-depeach)
+depeach#depeach_v)
@@ -1031 +1031 @@
-our)
+our#our_pron)
@@ -1033 +1033 @@
-said)
+said#said_adj)
@@ -1035 +1035 @@
-Pursevant))
+Pursevant#pursuivant_n))
@@ -1038 +1038 @@
-immediatly)))))
+immediatly#immediately_adv)))))
@@ -1046 +1046 @@
-for)
+for#for_prep)
@@ -1049 +1049 @@
-the)
+the#the_adj)
@@ -1051 +1051 @@
-long)
+long#long_adj1)
@@ -1053 +1053 @@
-delay)
+delay#delay_n)
@@ -1056 +1056 @@
-of)
+of#of_prep)
@@ -1060 +1060 @@
-so)
+so#so_adv)
@@ -1062 +1062 @@
-honorable))
+honorable#honourable_adj))
@@ -1064 +1064 @@
-a)
+a#a_adj2)
@@ -1066 +1066 @@
-Journey))))
+Journey#journey_n))))
@@ -1069 +1069 @@
-wee))
+wee#we_pron))
@@ -1071 +1071 @@
-think)
+think#think_v2)
@@ -1079 +1079 @@
-should)
+should#shall_v)
@@ -1081 +1081 @@
-sound)
+sound#sound_v1)
@@ -1084 +1084 @@
-to)
+to#to_prep)
@@ -1087 +1087 @@
-your)
+your#your_pron)
@@ -1089 +1089 @@
-dishonor)))))
+dishonor#dishonour_n)))))
@@ -1097 +1097 @@
-Written)
+Written#write_v)
@@ -1100 +1100 @@
-in)
+in#in_prep)
@@ -1103 +1103 @@
-the)
+the#the_adj)
@@ -1105 +1105 @@
-feild)
+feild#field_n1)
@@ -1108 +1108 @@
-in)
+in#in_prep)
@@ -1111 +1111 @@
-Woller_haughe)))))
+Woller_haughe#NA)))))
@@ -1114 +1114 @@
-the)
+the#the_adj)
@@ -1116 +1116 @@
-7=th=.)
+7=th=.#NA)
@@ -1118 +1118 @@
-day)
+day#day_n)
@@ -1121 +1121 @@
-of)
+of#of_prep)
@@ -1124 +1124 @@
-September))))
+September#NA))))
@@ -1127 +1127 @@
-at)
+at#at_prep)
@@ -1130 +1130 @@
-five)
+five#five_v)
@@ -1133 +1133 @@
-of)
+of#of_prep)
@@ -1136 +1136 @@
-the)
+the#the_adj)
@@ -1138 +1138 @@
-Clock)))
+Clock#clock_n1)))
@@ -1141 +1141 @@
-in)
+in#in_prep)
@@ -1144 +1144 @@
-the)
+the#the_adj)
@@ -1146 +1146 @@
-afternoone))))))
+afternoone#afternoon_n))))))
@@ -1153 +1153 @@
-Thomas)
+Thomas#NA)
@@ -1155 +1155 @@
-Surrey))
+Surrey#NA))
@@ -1161 +1161 @@
-Thomas)
+Thomas#NA)
@@ -1163 +1163 @@
-Haward)))
+Haward#NA)))
@@ -1169 +1169 @@
-Thom.)
+Thom.#NA)
@@ -1171 +1171 @@
-Dacre)))
+Dacre#NA)))
@@ -1177 +1177 @@
-Clifford)))
+Clifford#NA)))
@@ -1183 +1183 @@
-Henerie)
+Henerie#NA)
@@ -1185 +1185 @@
-Scroope)))
+Scroope#NA)))
@@ -1191 +1191 @@
-Ralphe)
+Ralphe#NA)
@@ -1193 +1193 @@
-Scrope)))
+Scrope#NA)))
@@ -1199 +1199 @@
-Rich.)
+Rich.#NA)
@@ -1201 +1201 @@
-Latimer)))
+Latimer#NA)))
@@ -1207 +1207 @@
-William)
+William#NA)
@@ -1209 +1209 @@
-Conyers)))
+Conyers#NA)))
@@ -1215 +1215 @@
-J.)
+J.#NA)
@@ -1217 +1217 @@
-Lomley)))
+Lomley#NA)))
@@ -1223 +1223 @@
-R.)
+R.#NA)
@@ -1225 +1225 @@
-Ogle)))
+Ogle#NA)))
@@ -1231 +1231 @@
-W.)
+W.#NA)
@@ -1233 +1233 @@
-Percye)))
+Percye#NA)))
@@ -1239 +1239 @@
-E.)
+E.#NA)
@@ -1241 +1241 @@
-Stanley)))
+Stanley#NA)))
@@ -1247 +1247 @@
-William)
+William#NA)
@@ -1249 +1249 @@
-Molynex)))
+Molynex#NA)))
@@ -1255 +1255 @@
-Marmaduke)
+Marmaduke#NA)
@@ -1257 +1257 @@
-Constable)))
+Constable#NA)))
@@ -1263 +1263 @@
-W.)
+W.#NA)
@@ -1265 +1265 @@
-Gascoigne)))
+Gascoigne#NA)))
@@ -1271 +1271 @@
-W.)
+W.#NA)
@@ -1273 +1273 @@
-Griffith)))
+Griffith#NA)))
@@ -1279 +1279 @@
-George)
+George#NA)
@@ -1281 +1281 @@
-Darcy)))
+Darcy#NA)))
@@ -1287 +1287 @@
-W.)
+W.#NA)
@@ -1289 +1289 @@
-Bulmer)))
+Bulmer#NA)))
@@ -1295 +1295 @@
-Thom.)
+Thom.#NA)
@@ -1297 +1297 @@
-Strangwayes)))
+Strangwayes#NA)))
@@ -1301 +1301 @@
-&c.))
+&c.#et-cetera_n))
