--- wplumpt-1538-e1-h.psd.vert	2025-09-10 13:50:35
+++ wplumpt-1538-e1-h.lem.vert	2025-09-10 13:50:45
@@ -14 +14 @@
-To)
+To#to_prep)
@@ -17 +17 @@
-my)
+my#my_adj)
@@ -19 +19 @@
-welbeloved)
+welbeloved#well-beloved_adj)
@@ -21 +21 @@
-son)
+son#son_n1)
@@ -24 +24 @@
-Robart)
+Robart#NA)
@@ -26 +26 @@
-Plompton))))
+Plompton#NA))))
@@ -29 +29 @@
-at)
+at#at_prep)
@@ -32 +32 @@
-the)
+the#the_adj)
@@ -34 +34 @@
-Iner)
+Iner#NA)
@@ -36 +36 @@
-Temple)
+Temple#NA)
@@ -39 +39 @@
-in)
+in#in_prep)
@@ -42 +42 @@
-London)))))
+London#NA)))))
@@ -44 +44 @@
-be)
+be#be_v)
@@ -47 +47 @@
-this))
+this#this_pron))
@@ -56 +56 @@
-Son)
+Son#NA)
@@ -58 +58 @@
-Robart)
+Robart#NA)
@@ -60 +60 @@
-Plompton))
+Plompton#NA))
@@ -65 +65 @@
-I))
+I#i_pron))
@@ -68 +68 @@
-hertely))
+hertely#heartily_adv))
@@ -70 +70 @@
-recommend)
+recommend#recommend_v1)
@@ -73 +73 @@
-me))
+me#me_pron1))
@@ -76 +76 @@
-to)
+to#to_prep)
@@ -79 +79 @@
-you)))
+you#you_pron)))
@@ -87 +87 @@
-and)
+and#and_conj1)
@@ -90 +90 @@
-sending)
+sending#send_v1)
@@ -94 +94 @@
-you))
+you#you_pron))
@@ -97 +97 @@
-and)
+and#and_conj1)
@@ -100 +100 @@
-your)
+your#your_pron)
@@ -102 +102 @@
-brother))))
+brother#brother_n))))
@@ -106 +106 @@
-God)
+God#NA)
@@ -108 +108 @@
-blesing))
+blesing#blessing_n))
@@ -111 +111 @@
-and)
+and#and_conj1)
@@ -114 +114 @@
-mine)))))
+mine#mine_adj)))))
@@ -123 +123 @@
-The)
+The#the_adj)
@@ -125 +125 @@
-cause)
+cause#cause_n)
@@ -128 +128 @@
-of)
+of#of_prep)
@@ -131 +131 @@
-my)
+my#my_adj)
@@ -134 +134 @@
-writing)
+writing#write_v)
@@ -137 +137 @@
-to)
+to#to_prep)
@@ -140 +140 @@
-you)))
+you#you_pron)))
@@ -143 +143 @@
-now))))))
+now#now_adv))))))
@@ -148 +148 @@
-that)
+that#that_conj)
@@ -152 +152 @@
-I))
+I#i_pron))
@@ -154 +154 @@
-wold)
+wold#will_v1)
@@ -161 +161 @@
-you))
+you#you_pron))
@@ -163 +163 @@
-should)
+should#shall_v)
@@ -165 +165 @@
-helpe)
+helpe#help_v)
@@ -168 +168 @@
-this)
+this#this_pron)
@@ -170 +170 @@
-bearrer)
+bearrer#bearer_n)
@@ -175 +175 @@
-yong)
+yong#young_adj)
@@ -177 +177 @@
-Letham)))
+Letham#NA)))
@@ -182 +182 @@
-in)
+in#in_prep)
@@ -185 +185 @@
-such)
+such#such_adj)
@@ -187 +187 @@
-buisenes)
+buisenes#business_n)
@@ -190 +190 @@
-as)
+as#as_adv)
@@ -201 +201 @@
-he))
+he#he_pron))
@@ -203 +203 @@
-hath)
+hath#have_v)
@@ -206 +206 @@
-in)
+in#in_prep)
@@ -209 +209 @@
-the)
+the#the_adj)
@@ -211 +211 @@
-Court)
+Court#NA)
@@ -214 +214 @@
-of)
+of#of_prep)
@@ -217 +217 @@
-Augmentation)))))
+Augmentation#NA)))))
@@ -222 +222 @@
-for)
+for#for_prep)
@@ -225 +225 @@
-certaine)
+certaine#certain_adj)
@@ -227 +227 @@
-power)
+power#power_n1)
@@ -230 +230 @@
-for)
+for#for_prep)
@@ -233 +233 @@
-yong)
+yong#young_adj)
@@ -235 +235 @@
-children)
+children#child_n)
@@ -238 +238 @@
-of)
+of#of_prep)
@@ -241 +241 @@
-one)
+one#one_adj)
@@ -243 +243 @@
-Berkine)
+Berkine#NA)
@@ -248 +248 @@
-deceased))))))
+deceased#decease_v))))))
@@ -253 +253 @@
-as)
+as#as_adv)
@@ -256 +256 @@
-conserning)
+conserning#concerning_prep)
@@ -259 +259 @@
-one)
+one#one_adj)
@@ -261 +261 @@
-farme)
+farme#farm_n2)
@@ -263 +263 @@
-hold)
+hold#hold_n1)
@@ -269 +269 @@
-late))
+late#late_adv))
@@ -271 +271 @@
-belonging)
+belonging#belong_v)
@@ -274 +274 @@
-to)
+to#to_prep)
@@ -277 +277 @@
-the)
+the#the_adj)
@@ -279 +279 @@
-hold)
+hold#hold_n1)
@@ -282 +282 @@
-of)
+of#of_prep)
@@ -285 +285 @@
-St.)
+St.#NA)
@@ -287 +287 @@
-Robarts))))))))))))))))))))
+Robarts#NA))))))))))))))))))))
@@ -293 +293 @@
-which))
+which#which_adj))
@@ -301 +301 @@
-you))
+you#you_pron))
@@ -303 +303 @@
-know)
+know#know_v)
@@ -311 +311 @@
-I))
+I#i_pron))
@@ -313 +313 @@
-did)
+did#do_v)
@@ -315 +315 @@
-speake)
+speake#speak_v)
@@ -318 +318 @@
-to)
+to#to_prep)
@@ -321 +321 @@
-the)
+the#the_adj)
@@ -323 +323 @@
-Ansurer)))
+Ansurer#ensurer_n)))
@@ -326 +326 @@
-for)
+for#for_prep)
@@ -329 +329 @@
-the)
+the#the_adj)
@@ -331 +331 @@
-use)
+use#use_n)
@@ -334 +334 @@
-of)
+of#of_prep)
@@ -337 +337 @@
-the)
+the#the_adj)
@@ -339 +339 @@
-said)
+said#said_adj)
@@ -341 +341 @@
-children))))))
+children#child_n))))))
@@ -346 +346 @@
-and)
+and#and_conj1)
@@ -350 +350 @@
-he))
+he#he_pron))
@@ -352 +352 @@
-permised)
+permised#permise_v)
@@ -355 +355 @@
-not)
+not#not_adv)
@@ -357 +357 @@
-to)
+to#to_prep)
@@ -359 +359 @@
-suit)
+suit#suit_v)
@@ -362 +362 @@
-them)))))))))
+them#them_pron)))))))))
@@ -372 +372 @@
-That))
+That#that_pron))
@@ -375 +375 @@
-notwithstanding)))
+notwithstanding#notwithstanding_prep)))
@@ -380 +380 @@
-John)
+John#NA)
@@ -382 +382 @@
-Benson))
+Benson#NA))
@@ -384 +384 @@
-would)
+would#will_v1)
@@ -386 +386 @@
-have)
+have#have_v)
@@ -388 +388 @@
-entred)
+entred#enter_v)
@@ -399 +399 @@
-and)
+and#and_conj1)
@@ -404 +404 @@
-now))
+now#now_adv))
@@ -406 +406 @@
-made)
+made#make_v1)
@@ -409 +409 @@
-many)
+many#many_adj)
@@ -411 +411 @@
-great)
+great#great_adj)
@@ -413 +413 @@
-riots)
+riots#riot_n)
@@ -416 +416 @@
-upon)
+upon#upon_prep)
@@ -419 +419 @@
-the)
+the#the_adj)
@@ -421 +421 @@
-said)
+said#said_adj)
@@ -423 +423 @@
-children))))
+children#child_n))))
@@ -431 +431 @@
-and)
+and#and_conj1)
@@ -434 +434 @@
-therfore))
+therfore#therefore_adv))
@@ -437 +437 @@
-he))
+he#he_pron))
@@ -439 +439 @@
-is)
+is#be_v)
@@ -441 +441 @@
-indited)
+indited#indict_v1)
@@ -444 +444 @@
-with)
+with#with_prep)
@@ -447 +447 @@
-divers)
+divers#divers_adj)
@@ -449 +449 @@
-persons)
+persons#person_n)
@@ -452 +452 @@
-with)
+with#with_prep)
@@ -455 +455 @@
-him)))))
+him#him_pron)))))
@@ -463 +463 @@
-And)
+And#and_conj1)
@@ -466 +466 @@
-now))
+now#now_adv))
@@ -469 +469 @@
-forther))
+forther#further_adj))
@@ -474 +474 @@
-he))
+he#he_pron))
@@ -476 +476 @@
-hath)
+hath#have_v)
@@ -478 +478 @@
-brought)
+brought#bring_v)
@@ -481 +481 @@
-a)
+a#a_adj2)
@@ -483 +483 @@
-preve)
+preve#privy_adj)
@@ -485 +485 @@
-seale))
+seale#seal_n2))
@@ -488 +488 @@
-against)
+against#against_prep)
@@ -492 +492 @@
-old)
+old#old_adj)
@@ -494 +494 @@
-Leathom))
+Leathom#NA))
@@ -497 +497 @@
-and)
+and#and_conj1)
@@ -500 +500 @@
-yong)
+yong#young_adj)
@@ -502 +502 @@
-Leathom)))
+Leathom#NA)))
@@ -507 +507 @@
-and)
+and#and_conj1)
@@ -509 +509 @@
-also)
+also#also_adv)
@@ -512 +512 @@
-the)
+the#the_adj)
@@ -514 +514 @@
-eldest)
+eldest#eldest_adj)
@@ -516 +516 @@
-child)))))
+child#child_n)))))
@@ -524 +524 @@
-and)
+and#and_conj1)
@@ -527 +527 @@
-for)
+for#for_prep)
@@ -530 +530 @@
-that)))
+that#that_pron)))
@@ -534 +534 @@
-divers)
+divers#divers_adj)
@@ -537 +537 @@
-and)
+and#and_conj1)
@@ -539 +539 @@
-many)))
+many#many_adj)))
@@ -542 +542 @@
-of)
+of#of_prep)
@@ -545 +545 @@
-ther)
+ther#their_adj)
@@ -547 +547 @@
-frinds))))
+frinds#friend_n))))
@@ -549 +549 @@
-hath)
+hath#have_v)
@@ -551 +551 @@
-moved)
+moved#move_v)
@@ -554 +554 @@
-me))
+me#me_pron1))
@@ -557 +557 @@
-to)
+to#to_prep)
@@ -559 +559 @@
-wryte)
+wryte#write_v)
@@ -562 +562 @@
-to)
+to#to_prep)
@@ -565 +565 @@
-you)))
+you#you_pron)))
@@ -568 +568 @@
-to)
+to#to_prep)
@@ -570 +570 @@
-help)
+help#help_v)
@@ -573 +573 @@
-them))
+them#them_pron))
@@ -576 +576 @@
-in)
+in#in_prep)
@@ -579 +579 @@
-the)
+the#the_adj)
@@ -581 +581 @@
-said)
+said#said_adj)
@@ -583 +583 @@
-matter)))))
+matter#matter_n1)))))
@@ -591 +591 @@
-And)
+And#and_conj1)
@@ -594 +594 @@
-if)
+if#if_conj)
@@ -601 +601 @@
-it))
+it#it_pron))
@@ -603 +603 @@
-be)
+be#be_v)
@@ -606 +606 @@
-that)
+that#that_conj)
@@ -610 +610 @@
-you))
+you#you_pron))
@@ -612 +612 @@
-can)
+can#can_v1)
@@ -614 +614 @@
-make)
+make#make_v1)
@@ -618 +618 @@
-any)
+any#any_adj)
@@ -620 +620 @@
-frinds))
+frinds#friend_n))
@@ -624 +624 @@
-to)
+to#to_prep)
@@ -626 +626 @@
-shew)
+shew#show_v)
@@ -629 +629 @@
-Mr.)
+Mr.#NA)
@@ -631 +631 @@
-Chaunceler))
+Chaunceler#NA))
@@ -635 +635 @@
-the)
+the#the_adj)
@@ -637 +637 @@
-planer))
+planer#plain_adj1.B))
@@ -640 +640 @@
-and)
+and#and_conj1)
@@ -642 +642 @@
-through)))
+through#through_prep)))
@@ -645 +645 @@
-in)
+in#in_prep)
@@ -648 +648 @@
-every)
+every#every_adj)
@@ -650 +650 @@
-thing)))
+thing#thing_n1)))
@@ -655 +655 @@
-and)
+and#and_conj1)
@@ -658 +658 @@
-this)
+this#this_pron)
@@ -660 +660 @@
-bearrer))
+bearrer#bearer_n))
@@ -662 +662 @@
-can)
+can#can_v1)
@@ -664 +664 @@
-instruct)
+instruct#instruct_v)
@@ -667 +667 @@
-you)))))))))
+you#you_pron)))))))))
@@ -673 +673 @@
-and)
+and#and_conj1)
@@ -676 +676 @@
-then))
+then#then_adv))
@@ -682 +682 @@
-I))
+I#i_pron))
@@ -684 +684 @@
-pray)
+pray#pray_v)
@@ -687 +687 @@
-you)))
+you#you_pron)))
@@ -691 +691 @@
-do)
+do#do_v)
@@ -694 +694 @@
-the)
+the#the_adj)
@@ -696 +696 @@
-best)
+best#best_adj)
@@ -699 +699 @@
-for)
+for#for_prep)
@@ -702 +702 @@
-them))))
+them#them_pron))))
@@ -710 +710 @@
-And)
+And#and_conj1)
@@ -712 +712 @@
-also)
+also#also_adv)
@@ -715 +715 @@
-I))
+I#i_pron))
@@ -717 +717 @@
-would)
+would#will_v1)
@@ -719 +719 @@
-have)
+have#have_v)
@@ -724 +724 @@
-you))
+you#you_pron))
@@ -726 +726 @@
-to)
+to#to_prep)
@@ -728 +728 @@
-speak)
+speak#speak_v)
@@ -731 +731 @@
-with)
+with#with_prep)
@@ -734 +734 @@
-Mr.)
+Mr.#NA)
@@ -736 +736 @@
-Latham)
+Latham#NA)
@@ -741 +741 @@
-the)
+the#the_adj)
@@ -743 +743 @@
-goldsmith))
+goldsmith#goldsmith_n))
@@ -748 +748 @@
-Lanlord)
+Lanlord#landlord_n)
@@ -751 +751 @@
-to)
+to#to_prep)
@@ -754 +754 @@
-Robart)
+Robart#NA)
@@ -756 +756 @@
-Oliver)))))))
+Oliver#NA)))))))
@@ -761 +761 @@
-and)
+and#and_conj1)
@@ -768 +768 @@
-shew)
+shew#show_v)
@@ -771 +771 @@
-him))
+him#him_pron))
@@ -776 +776 @@
-how))
+how#how_adv))
@@ -778 +778 @@
-that)
+that#that_conj)
@@ -784 +784 @@
-he))
+he#he_pron))
@@ -786 +786 @@
-will)
+will#will_v1)
@@ -788 +788 @@
-not)
+not#not_adv)
@@ -790 +790 @@
-make)
+make#make_v1)
@@ -793 +793 @@
-his)
+his#his_adj)
@@ -795 +795 @@
-diches)
+diches#ditch_n1)
@@ -798 +798 @@
-and)
+and#and_conj1)
@@ -801 +801 @@
-fences)))
+fences#fence_n)))
@@ -804 +804 @@
-belonging)
+belonging#belong_v)
@@ -807 +807 @@
-to)
+to#to_prep)
@@ -810 +810 @@
-his)
+his#his_adj)
@@ -812 +812 @@
-farme)))))))
+farme#farm_n2)))))))
@@ -817 +817 @@
-but)
+but#but_prep)
@@ -820 +820 @@
-that)
+that#that_conj)
@@ -824 +824 @@
-my)
+my#my_adj)
@@ -826 +826 @@
-corne)
+corne#corn_n1)
@@ -829 +829 @@
-and)
+and#and_conj1)
@@ -832 +832 @@
-gras))))
+gras#grass_n1))))
@@ -834 +834 @@
-is)
+is#be_v)
@@ -836 +836 @@
-spoyled)
+spoyled#spoil_v1)
@@ -839 +839 @@
-at)
+at#at_prep)
@@ -842 +842 @@
-Watterton)))
+Watterton#NA)))
@@ -845 +845 @@
-by)
+by#by_prep)
@@ -848 +848 @@
-that)
+that#that_pron)
@@ -850 +850 @@
-meaner))))))))))
+meaner#mind_n1))))))))))
@@ -858 +858 @@
-And)
+And#and_conj1)
@@ -861 +861 @@
-if)
+if#if_conj)
@@ -868 +868 @@
-he))
+he#he_pron))
@@ -870 +870 @@
-will)
+will#will_v1)
@@ -872 +872 @@
-not)
+not#not_adv)
@@ -874 +874 @@
-seake)
+seake#seek_v)
@@ -877 +877 @@
-remedy)
+remedy#remedy_n)
@@ -880 +880 @@
-therof))))))
+therof#thereof_adv))))))
@@ -884 +884 @@
-let)
+let#let_v1)
@@ -888 +888 @@
-Mr.)
+Mr.#NA)
@@ -890 +890 @@
-Fox))
+Fox#NA))
@@ -892 +892 @@
-enter)
+enter#enter_v)
@@ -895 +895 @@
-a)
+a#a_adj2)
@@ -897 +897 @@
-action)
+action#action_n)
@@ -900 +900 @@
-of)
+of#of_prep)
@@ -903 +903 @@
-trespas)))
+trespas#trespass_n)))
@@ -906 +906 @@
-against)
+against#against_prep)
@@ -909 +909 @@
-the)
+the#the_adj)
@@ -911 +911 @@
-said)
+said#said_adj)
@@ -913 +913 @@
-Robart)
+Robart#NA)
@@ -915 +915 @@
-Oliver)))))
+Oliver#NA)))))
@@ -918 +918 @@
-for)
+for#for_prep)
@@ -921 +921 @@
-dispoyling)
+dispoyling#despoil_v)
@@ -924 +924 @@
-my)
+my#my_adj)
@@ -926 +926 @@
-gras))
+gras#grass_n1))
@@ -929 +929 @@
-at)
+at#at_prep)
@@ -932 +932 @@
-Watterton)))
+Watterton#NA)))
@@ -935 +935 @@
-to)
+to#to_prep)
@@ -938 +938 @@
-the)
+the#the_adj)
@@ -940 +940 @@
-valew)
+valew#value_n)
@@ -943 +943 @@
-of)
+of#of_prep)
@@ -946 +946 @@
-five)
+five#five_v)
@@ -948 +948 @@
-mark)))))))
+mark#mark_n2)))))))
@@ -956 +956 @@
-And)
+And#and_conj1)
@@ -960 +960 @@
-as)
+as#as_adv)
@@ -962 +962 @@
-far)
+far#far_adj)
@@ -965 +965 @@
-as)
+as#as_adv)
@@ -976 +976 @@
-I))
+I#i_pron))
@@ -978 +978 @@
-fele))))))
+fele#feel_v))))))
@@ -983 +983 @@
-Mr.)
+Mr.#NA)
@@ -985 +985 @@
-Norton))
+Norton#NA))
@@ -987 +987 @@
-comith)
+comith#come_v)
@@ -989 +989 @@
-not)
+not#not_adv)
@@ -991 +991 @@
-up)
+up#up_adv1)
@@ -1001 +1001 @@
-I))
+I#i_pron))
@@ -1003 +1003 @@
-shall)
+shall#shall_v)
@@ -1005 +1005 @@
-make)
+make#make_v1)
@@ -1009 +1009 @@
-your)
+your#your_pron)
@@ -1011 +1011 @@
-rents))
+rents#rent_n1))
@@ -1013 +1013 @@
-to)
+to#to_prep)
@@ -1015 +1015 @@
-be)
+be#be_v)
@@ -1017 +1017 @@
-gathered)))
+gathered#gather_v)))
@@ -1022 +1022 @@
-and)
+and#and_conj1)
@@ -1025 +1025 @@
-send)
+send#send_v1)
@@ -1028 +1028 @@
-it))
+it#it_pron))
@@ -1031 +1031 @@
-to)
+to#to_prep)
@@ -1034 +1034 @@
-you)))
+you#you_pron)))
@@ -1037 +1037 @@
-as)
+as#as_adv)
@@ -1039 +1039 @@
-shortly)
+shortly#shortly_adv)
@@ -1042 +1042 @@
-as)
+as#as_adv)
@@ -1053 +1053 @@
-I))
+I#i_pron))
@@ -1055 +1055 @@
-can)
+can#can_v1)
@@ -1065 +1065 @@
-And)
+And#and_conj1)
@@ -1068 +1068 @@
-thus))
+thus#thus_adv))
@@ -1071 +1071 @@
-hartely))
+hartely#heartily_adv))
@@ -1073 +1073 @@
-far)
+far#fare_v1)
@@ -1076 +1076 @@
-you))
+you#you_pron))
@@ -1079 +1079 @@
-well))
+well#well_adv))
@@ -1087 +1087 @@
-From)
+From#from_prep)
@@ -1090 +1090 @@
-Plompton))
+Plompton#NA))
@@ -1098 +1098 @@
-this)
+this#this_pron)
@@ -1100 +1100 @@
-14=th=)
+14=th=#NA)
@@ -1102 +1102 @@
-day)
+day#day_n)
@@ -1105 +1105 @@
-of)
+of#of_prep)
@@ -1108 +1108 @@
-November)))
+November#NA)))
@@ -1116 +1116 @@
-By)
+By#by_prep)
@@ -1119 +1119 @@
-your)
+your#your_pron)
@@ -1121 +1121 @@
-loving)
+loving#loving_adj)
@@ -1123 +1123 @@
-father)
+father#father_n)
@@ -1128 +1128 @@
-William)
+William#NA)
@@ -1130 +1130 @@
-Plompton)))
+Plompton#NA)))
